<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>The Human Connectome Project's neuroimaging approach | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"review article","legacy":{"webtrendsPrimaryArticleType":"reviews","webtrendsSubjectTerms":"language;neural-circuit;sensory-processing","webtrendsContentCategory":null,"webtrendsContentCollection":"Focus on the neuroscience toolbox","webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Review Article"}},"article":{"doi":"10.1038/nn.4361"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Matthew F Glasser","Stephen M Smith","Daniel S Marcus","Jesper L R Andersson","Edward J Auerbach","Timothy E J Behrens","Timothy S Coalson","Michael P Harms","Mark Jenkinson","Steen Moeller","Emma C Robinson","Stamatios N Sotiropoulos","Junqian Xu","Essa Yacoub","Kamil Ugurbil","David C Van Essen"],"publishedAt":1472169600,"publishedAtString":"2016-08-26","title":"The Human Connectome Project's neuroimaging approach","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"19","issue":"9"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":{"id":"dycqdqykfv"}},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"The Human Connectome Project's neuroimaging approach","description":"This paper describes an integrated approach for neuroimaging data acquisition, analysis and sharing. Building on methodological advances from the Human Connectome Project (HCP) and elsewhere, the HCP-style paradigm applies to new and existing data sets that meet core requirements and may accelerate progress in understanding the brain in health and disease. Noninvasive human neuroimaging has yielded many discoveries about the brain. Numerous methodological advances have also occurred, though inertia has slowed their adoption. This paper presents an integrated approach to data acquisition, analysis and sharing that builds upon recent advances, particularly from the Human Connectome Project (HCP). The 'HCP-style' paradigm has seven core tenets: (i) collect multimodal imaging data from many subjects; (ii) acquire data at high spatial and temporal resolution; (iii) preprocess data to minimize distortions, blurring and temporal artifacts; (iv) represent data using the natural geometry of cortical and subcortical structures; (v) accurately align corresponding brain areas across subjects and studies; (vi) analyze data using neurobiologically accurate brain parcellations; and (vii) share published data via user-friendly databases. We illustrate the HCP-style paradigm using existing HCP data sets and provide guidance for future research. Widespread adoption of this paradigm should accelerate progress in understanding the brain in health and disease.","datePublished":"2016-08-26T00:00:00Z","dateModified":"2016-08-26T00:00:00Z","pageStart":"1175","pageEnd":"1187","sameAs":"https://doi.org/10.1038/nn.4361","keywords":["Language","Neural circuits","Sensory processing","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig1_HTML.jpg","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig2_HTML.jpg","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig3_HTML.jpg","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig4_HTML.jpg","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig5_HTML.jpg","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig6_HTML.jpg"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"19","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Matthew F Glasser","affiliation":[{"name":"Washington University Medical School","address":{"name":"Department of Neuroscience, Washington University Medical School, St. Louis, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"glasserm@wustl.edu","@type":"Person"},{"name":"Stephen M Smith","url":"http://orcid.org/0000-0001-8166-069X","affiliation":[{"name":"FMRIB Centre, John Radcliffe Hospital, University of Oxford","address":{"name":"Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Daniel S Marcus","affiliation":[{"name":"Washington University Medical School","address":{"name":"Department of Radiology, Washington University Medical School, St. Louis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jesper L R Andersson","affiliation":[{"name":"FMRIB Centre, John Radcliffe Hospital, University of Oxford","address":{"name":"Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Edward J Auerbach","affiliation":[{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota","address":{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Timothy E J Behrens","affiliation":[{"name":"FMRIB Centre, John Radcliffe Hospital, University of Oxford","address":{"name":"Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Timothy S Coalson","affiliation":[{"name":"Washington University Medical School","address":{"name":"Department of Neuroscience, Washington University Medical School, St. Louis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael P Harms","affiliation":[{"name":"Washington University Medical School","address":{"name":"Department of Psychiatry, Washington University Medical School, St. Louis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Mark Jenkinson","affiliation":[{"name":"FMRIB Centre, John Radcliffe Hospital, University of Oxford","address":{"name":"Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Steen Moeller","affiliation":[{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota","address":{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Emma C Robinson","affiliation":[{"name":"Imperial College London","address":{"name":"Department of Computing, Imperial College London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stamatios N Sotiropoulos","affiliation":[{"name":"FMRIB Centre, John Radcliffe Hospital, University of Oxford","address":{"name":"Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Junqian Xu","url":"http://orcid.org/0000-0001-8438-2066","affiliation":[{"name":"7Translational and Molecular Imaging Institute, Icahn School of Medicine at Mount Sinai","address":{"name":"7Translational and Molecular Imaging Institute, Icahn School of Medicine at Mount Sinai, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Essa Yacoub","affiliation":[{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota","address":{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kamil Ugurbil","affiliation":[{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota","address":{"name":"Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"David C Van Essen","affiliation":[{"name":"Washington University Medical School","address":{"name":"Department of Neuroscience, Washington University Medical School, St. Louis, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"vanessen@wustl.edu","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/nn.4361">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="The Human Connectome Project's neuroimaging approach"/>
    <meta name="dc.source" content="Nature Neuroscience 2016 19:9"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2016-08-26"/>
    <meta name="dc.type" content="ReviewPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2016 Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2016 Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="This paper describes an integrated approach for neuroimaging data acquisition, analysis and sharing. Building on methodological advances from the Human Connectome Project (HCP) and elsewhere, the HCP-style paradigm applies to new and existing data sets that meet core requirements and may accelerate progress in understanding the brain in health and disease. Noninvasive human neuroimaging has yielded many discoveries about the brain. Numerous methodological advances have also occurred, though inertia has slowed their adoption. This paper presents an integrated approach to data acquisition, analysis and sharing that builds upon recent advances, particularly from the Human Connectome Project (HCP). The &#39;HCP-style&#39; paradigm has seven core tenets: (i) collect multimodal imaging data from many subjects; (ii) acquire data at high spatial and temporal resolution; (iii) preprocess data to minimize distortions, blurring and temporal artifacts; (iv) represent data using the natural geometry of cortical and subcortical structures; (v) accurately align corresponding brain areas across subjects and studies; (vi) analyze data using neurobiologically accurate brain parcellations; and (vii) share published data via user-friendly databases. We illustrate the HCP-style paradigm using existing HCP data sets and provide guidance for future research. Widespread adoption of this paradigm should accelerate progress in understanding the brain in health and disease."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2016-08-26"/>
    <meta name="prism.volume" content="19"/>
    <meta name="prism.number" content="9"/>
    <meta name="prism.section" content="ReviewPaper"/>
    <meta name="prism.startingPage" content="1175"/>
    <meta name="prism.endingPage" content="1187"/>
    <meta name="prism.copyright" content="2016 Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/nn.4361"/>
    <meta name="prism.doi" content="doi:10.1038/nn.4361"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/nn.4361.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/nn.4361"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="The Human Connectome Project's neuroimaging approach"/>
    <meta name="citation_volume" content="19"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_publication_date" content="2016/09"/>
    <meta name="citation_online_date" content="2016/08/26"/>
    <meta name="citation_firstpage" content="1175"/>
    <meta name="citation_lastpage" content="1187"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/nn.4361"/>
    <meta name="DOI" content="10.1038/nn.4361"/>
    <meta name="size" content="309127"/>
    <meta name="citation_doi" content="10.1038/nn.4361"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/nn.4361&amp;api_key="/>
    <meta name="description" content="This paper describes an integrated approach for neuroimaging data acquisition, analysis and sharing. Building on methodological advances from the Human Connectome Project (HCP) and elsewhere, the HCP-style paradigm applies to new and existing data sets that meet core requirements and may accelerate progress in understanding the brain in health and disease. Noninvasive human neuroimaging has yielded many discoveries about the brain. Numerous methodological advances have also occurred, though inertia has slowed their adoption. This paper presents an integrated approach to data acquisition, analysis and sharing that builds upon recent advances, particularly from the Human Connectome Project (HCP). The &#39;HCP-style&#39; paradigm has seven core tenets: (i) collect multimodal imaging data from many subjects; (ii) acquire data at high spatial and temporal resolution; (iii) preprocess data to minimize distortions, blurring and temporal artifacts; (iv) represent data using the natural geometry of cortical and subcortical structures; (v) accurately align corresponding brain areas across subjects and studies; (vi) analyze data using neurobiologically accurate brain parcellations; and (vii) share published data via user-friendly databases. We illustrate the HCP-style paradigm using existing HCP data sets and provide guidance for future research. Widespread adoption of this paradigm should accelerate progress in understanding the brain in health and disease."/>
    <meta name="dc.creator" content="Glasser, Matthew F"/>
    <meta name="dc.creator" content="Smith, Stephen M"/>
    <meta name="dc.creator" content="Marcus, Daniel S"/>
    <meta name="dc.creator" content="Andersson, Jesper L R"/>
    <meta name="dc.creator" content="Auerbach, Edward J"/>
    <meta name="dc.creator" content="Behrens, Timothy E J"/>
    <meta name="dc.creator" content="Coalson, Timothy S"/>
    <meta name="dc.creator" content="Harms, Michael P"/>
    <meta name="dc.creator" content="Jenkinson, Mark"/>
    <meta name="dc.creator" content="Moeller, Steen"/>
    <meta name="dc.creator" content="Robinson, Emma C"/>
    <meta name="dc.creator" content="Sotiropoulos, Stamatios N"/>
    <meta name="dc.creator" content="Xu, Junqian"/>
    <meta name="dc.creator" content="Yacoub, Essa"/>
    <meta name="dc.creator" content="Ugurbil, Kamil"/>
    <meta name="dc.creator" content="Van Essen, David C"/>
    <meta name="dc.subject" content="Language"/>
    <meta name="dc.subject" content="Neural circuits"/>
    <meta name="dc.subject" content="Sensory processing"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Neurobiol.; citation_title=Human connectomics; citation_author=TE Behrens, O Sporns; citation_volume=22; citation_publication_date=2012; citation_pages=144-153; citation_doi=10.1016/j.conb.2011.08.005; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Measuring macroscopic brain connections in vivo; citation_author=S Jbabdi, SN Sotiropoulos, SN Haber, DC Van Essen, TE Behrens; citation_volume=18; citation_publication_date=2015; citation_pages=1546-1555; citation_doi=10.1038/nn.4134; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Introduction to the NeuroImage special issue &#8220;Mapping the Connectome&#8221;; citation_author=S Smith; citation_volume=80; citation_publication_date=2013; citation_pages=1; citation_doi=10.1016/j.neuroimage.2013.07.012; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The WU-Minn Human Connectome Project: an overview; citation_author=DC Van Essen; citation_volume=80; citation_publication_date=2013; citation_pages=62-79; citation_doi=10.1016/j.neuroimage.2013.05.041; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Pushing spatial and temporal resolution for functional and diffusion MRI in the Human Connectome Project; citation_author=K Ug&#711;urbil; citation_volume=80; citation_publication_date=2013; citation_pages=80-104; citation_doi=10.1016/j.neuroimage.2013.05.012; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The minimal preprocessing pipelines for the Human Connectome Project; citation_author=MF Glasser; citation_volume=80; citation_publication_date=2013; citation_pages=105-124; citation_doi=10.1016/j.neuroimage.2013.04.127; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging; citation_author=L Griffanti; citation_volume=95; citation_publication_date=2014; citation_pages=232-247; citation_doi=10.1016/j.neuroimage.2014.03.034; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=et. al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers; citation_author=G Salimi-Khorshidi; citation_volume=90; citation_publication_date=2014; citation_pages=449-468; citation_doi=10.1016/j.neuroimage.2013.11.046; citation_id=CR8"/>
    <meta name="citation_reference" content="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=MSM: a new flexible framework for multimodal surface matching; citation_author=EC Robinson; citation_volume=100; citation_publication_date=2014; citation_pages=414-426; citation_doi=10.1016/j.neuroimage.2014.05.069; citation_id=CR10"/>
    <meta name="citation_reference" content="Van Essen, D.C. et al. The brain analysis of spatial maps and atlases (BALSA) database. Neuroimage j.neuroimage.2016.04.002 (2016)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Human Connectome Project informatics: quality control, database services, and data visualization; citation_author=DS Marcus; citation_volume=80; citation_publication_date=2013; citation_pages=202-219; citation_doi=10.1016/j.neuroimage.2013.05.077; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Cluster failure: why fMRI inferences for spatial extent have inflated false-positive rates; citation_author=A Eklund, TE Nichols, H Knutsson; citation_volume=28; citation_publication_date=2016; citation_pages=7900-7905; citation_doi=10.1073/pnas.1602413113; citation_id=CR13"/>
    <meta name="citation_reference" content="Turner, R. Where matters: new approaches to brain analysis. in Microstructural Parcellation of the Human Cerebral Cortex (eds. Geyer, S. &amp; Turner, R.) 179&#8211;196 (Springer, 2013)."/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Comparing like with like: the power of knowing where you are; citation_author=R Turner, S Geyer; citation_volume=4; citation_publication_date=2014; citation_pages=547-557; citation_doi=10.1089/brain.2014.0261; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Borders of multiple visual areas in humans revealed by functional magnetic resonance imaging; citation_author=MI Sereno; citation_volume=268; citation_publication_date=1995; citation_pages=889-893; citation_doi=10.1126/science.7754376; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Probabilistic maps of visual topography in human cortex; citation_author=L Wang, RE Mruczek, MJ Arcaro, S Kastner; citation_volume=25; citation_publication_date=2015; citation_pages=3911-3931; citation_doi=10.1093/cercor/bhu277; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Measuring the thickness of the human cerebral cortex from magnetic resonance images; citation_author=B Fischl, AM Dale; citation_volume=97; citation_publication_date=2000; citation_pages=11050-11055; citation_doi=10.1073/pnas.200033797; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Trends and properties of human cerebral cortex: correlations with cortical myelin content; citation_author=MF Glasser, MS Goyal, TM Preuss, ME Raichle, DC Van Essen; citation_volume=93; citation_publication_date=2014; citation_pages=165-175; citation_doi=10.1016/j.neuroimage.2013.03.060; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI; citation_author=MF Glasser, DC Van Essen; citation_volume=31; citation_publication_date=2011; citation_pages=11597-11616; citation_doi=10.1523/JNEUROSCI.2180-11.2011; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Cortical surface-based analysis. I. Segmentation and surface reconstruction; citation_author=AM Dale, B Fischl, MI Sereno; citation_volume=9; citation_publication_date=1999; citation_pages=179-194; citation_doi=10.1006/nimg.1998.0395; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=J. Cogn. Neurosci.; citation_title=Improved localizadon of cortical activity by combining EEG and MEG with MRI cortical surface reconstruction: a linear approach; citation_author=AM Dale, MI Sereno; citation_volume=5; citation_publication_date=1993; citation_pages=162-176; citation_doi=10.1162/jocn.1993.5.2.162; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=Automated manifold surgery: constructing geometrically accurate and topologically correct models of the human cerebral cortex; citation_author=B Fischl, A Liu, AM Dale; citation_volume=20; citation_publication_date=2001; citation_pages=70-80; citation_doi=10.1109/42.906426; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain; citation_author=B Fischl; citation_volume=33; citation_publication_date=2002; citation_pages=341-355; citation_doi=10.1016/S0896-6273(02)00569-X; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Sequence-independent segmentation of magnetic resonance images; citation_author=B Fischl; citation_volume=23; citation_issue=Suppl. 1; citation_publication_date=2004; citation_pages=S69-S84; citation_doi=10.1016/j.neuroimage.2004.07.016; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Cortical surface-based analysis. II: inflation, flattening, and a surface-based coordinate system; citation_author=B Fischl, MI Sereno, AM Dale; citation_volume=9; citation_publication_date=1999; citation_pages=195-207; citation_doi=10.1006/nimg.1998.0396; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=High-resolution intersubject averaging and a coordinate system for the cortical surface; citation_author=B Fischl, MI Sereno, RB Tootell, AM Dale; citation_volume=8; citation_publication_date=1999; citation_pages=272-284; citation_doi=10.1002/(SICI)1097-0193(1999)8:4&lt;272::AID-HBM10&gt;3.0.CO;2-4; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=Geometrically accurate topology-correction of cortical surfaces using nonseparating loops; citation_author=F S&#233;gonne, J Pacheco, B Fischl; citation_volume=26; citation_publication_date=2007; citation_pages=518-529; citation_doi=10.1109/TMI.2006.887364; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Function in the human connectome: task-fMRI and individual differences in behavior; citation_author=DM Barch; citation_volume=80; citation_publication_date=2013; citation_pages=169-189; citation_doi=10.1016/j.neuroimage.2013.05.033; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Vision Res.; citation_title=Imaging retinotopic maps in the human brain; citation_author=BA Wandell, J Winawer; citation_volume=51; citation_publication_date=2011; citation_pages=718-737; citation_doi=10.1016/j.visres.2010.08.004; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Psychol.; citation_title=The transition in the ventral stream from feature to real-world entity representations; citation_author=GA Orban, Q Zhu, W Vanduffel; citation_volume=5; citation_publication_date=2014; citation_pages=695; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Visual areas in macaque cortex measured using functional magnetic resonance imaging; citation_author=AA Brewer, WA Press, NK Logothetis, BA Wandell; citation_volume=22; citation_publication_date=2002; citation_pages=10416-10426; citation_doi=10.1523/JNEUROSCI.22-23-10416.2002; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Symp. Quant. Biol.; citation_title=Modular and hierarchical organization of extrastriate visual cortex in the macaque monkey; citation_author=DC Van Essen, DJ Felleman, EA DeYoe, J Olavarria, J Knierim; citation_volume=55; citation_publication_date=1990; citation_pages=679-696; citation_doi=10.1101/SQB.1990.055.01.064; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=A diffusion MRI tractography connectome of the mouse brain and comparison with neuronal tracer data; citation_author=E Calabrese, A Badea, G Cofer, Y Qi, GA Johnson; citation_volume=25; citation_publication_date=2015; citation_pages=4628-4637; citation_doi=10.1093/cercor/bhv121; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci; citation_title=Using diffusion tractography to predict cortical connection strength and distance: A quantitative comparison with tracers in the monkey; citation_author=CJ Donahue; citation_volume=36; citation_publication_date=2016; citation_pages=6758-6770; citation_doi=10.1523/JNEUROSCI.0493-16.2016; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=Biophys. J.; citation_title=MR diffusion tensor spectroscopy and imaging; citation_author=PJ Basser, J Mattiello, D LeBihan; citation_volume=66; citation_publication_date=1994; citation_pages=259-267; citation_doi=10.1016/S0006-3495(94)80775-1; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=White matter characterization with diffusional kurtosis imaging; citation_author=E Fieremans, JH Jensen, JA Helpern; citation_volume=58; citation_publication_date=2011; citation_pages=177-188; citation_doi=10.1016/j.neuroimage.2011.06.006; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=NODDI: practical in vivo neurite orientation dispersion and density imaging of the human brain; citation_author=H Zhang, T Schneider, CA Wheeler-Kingshott, DC Alexander; citation_volume=61; citation_publication_date=2012; citation_pages=1000-1016; citation_doi=10.1016/j.neuroimage.2012.03.072; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases; citation_author=DC Van Essen, MF Glasser, DL Dierker, J Harwell, T Coalson; citation_volume=22; citation_publication_date=2012; citation_pages=2241-2262; citation_doi=10.1093/cercor/bhr291; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Cybern.; citation_title=Distribution of axon diameters in cortical white matter: an electron-microscopic study on three human brains and a macaque; citation_author=D Liewald, R Miller, N Logothetis, HJ Wagner, A Sch&#252;z; citation_volume=108; citation_publication_date=2014; citation_pages=541-557; citation_doi=10.1007/s00422-014-0626-2; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The future of the human connectome; citation_author=DC Van Essen, K Ugurbil; citation_volume=62; citation_publication_date=2012; citation_pages=1299-1310; citation_doi=10.1016/j.neuroimage.2012.01.032; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The impact of gradient strength on in vivo diffusion MRI estimates of axon diameter; citation_author=SY Huang; citation_volume=106; citation_publication_date=2015; citation_pages=464-472; citation_doi=10.1016/j.neuroimage.2014.12.008; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The Human Connectome Project and beyond: initial applications of 300 mT/m gradients; citation_author=JA McNab; citation_volume=80; citation_publication_date=2013; citation_pages=234-245; citation_doi=10.1016/j.neuroimage.2013.05.074; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Blipped-controlled aliasing in parallel imaging for simultaneous multislice echo planar imaging with reduced g-factor penalty; citation_author=K Setsompop; citation_volume=67; citation_publication_date=2012; citation_pages=1210-1224; citation_doi=10.1002/mrm.23097; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Pushing the limits of in vivo diffusion MRI for the Human Connectome Project; citation_author=K Setsompop; citation_volume=80; citation_publication_date=2013; citation_pages=220-233; citation_doi=10.1016/j.neuroimage.2013.05.078; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Fusion in diffusion MRI for improved fibre orientation estimation: An application to the 3T and 7T data of the Human Connectome Project; citation_author=SN Sotiropoulos; citation_volume=134; citation_publication_date=2016; citation_pages=396-409; citation_doi=10.1016/j.neuroimage.2016.04.014; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Advances in diffusion MRI acquisition and processing in the Human Connectome Project; citation_author=SN Sotiropoulos; citation_volume=80; citation_publication_date=2013; citation_pages=125-143; citation_doi=10.1016/j.neuroimage.2013.05.057; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=High resolution whole brain diffusion imaging at 7T for the Human Connectome Project; citation_author=AT Vu; citation_volume=122; citation_publication_date=2015; citation_pages=318-331; citation_doi=10.1016/j.neuroimage.2015.08.004; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Annu. Rev. Neurosci.; citation_title=Coupling mechanism and significance of the BOLD signal: a status report; citation_author=EM Hillman; citation_volume=37; citation_publication_date=2014; citation_pages=161-181; citation_doi=10.1146/annurev-neuro-071013-014111; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Multiplexed echo planar imaging for sub-second whole brain FMRI and fast diffusion imaging; citation_author=DA Feinberg; citation_volume=5; citation_publication_date=2010; citation_pages=e15710; citation_doi=10.1371/journal.pone.0015710; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=BOLD fractional contribution to resting-state functional connectivity above 0.1 Hz; citation_author=JE Chen, GH Glover; citation_volume=107; citation_publication_date=2015; citation_pages=207-218; citation_doi=10.1016/j.neuroimage.2014.12.012; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Prog. Brain Res.; citation_title=Spectral characteristics of resting state networks; citation_author=RK Niazy, J Xie, K Miller, CF Beckmann, SM Smith; citation_volume=193; citation_publication_date=2011; citation_pages=259-276; citation_doi=10.1016/B978-0-444-53839-0.00017-X; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Multiband accelerated spin-echo echo planar imaging with reduced peak RF power using time-shifted RF pulses; citation_author=EJ Auerbach, J Xu, E Yacoub, S Moeller, K Ug&#711;urbil; citation_volume=69; citation_publication_date=2013; citation_pages=1261-1267; citation_doi=10.1002/mrm.24719; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Multiband multislice GE-EPI at 7 tesla, with 16-fold acceleration using partial parallel imaging with application to high spatial and temporal whole-brain fMRI; citation_author=S Moeller; citation_volume=63; citation_publication_date=2010; citation_pages=1144-1153; citation_doi=10.1002/mrm.22361; citation_id=CR54"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Evaluation of slice accelerations using multiband echo planar imaging at 3 T; citation_author=J Xu; citation_volume=83; citation_publication_date=2013; citation_pages=991-1001; citation_doi=10.1016/j.neuroimage.2013.07.055; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Functional system and areal organization of a highly sampled individual human brain; citation_author=TO Laumann; citation_volume=87; citation_publication_date=2015; citation_pages=657-670; citation_doi=10.1016/j.neuron.2015.06.037; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Model-based analysis of multishell diffusion MR data for tractography: how to get over fitting problems; citation_author=S Jbabdi, SN Sotiropoulos, AM Savio, M Gra&#241;a, TE Behrens; citation_volume=68; citation_publication_date=2012; citation_pages=1846-1855; citation_doi=10.1002/mrm.24204; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Multi-tissue constrained spherical deconvolution for improved analysis of multi-shell diffusion MRI data; citation_author=B Jeurissen, JD Tournier, T Dhollander, A Connelly, J Sijbers; citation_volume=103; citation_publication_date=2014; citation_pages=411-426; citation_doi=10.1016/j.neuroimage.2014.07.061; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Accurate and robust brain image alignment using boundary-based registration; citation_author=DN Greve, B Fischl; citation_volume=48; citation_publication_date=2009; citation_pages=63-72; citation_doi=10.1016/j.neuroimage.2009.06.060; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Non-parametric representation and prediction of single- and multi-shell diffusion-weighted MRI data using Gaussian processes; citation_author=JL Andersson, SN Sotiropoulos; citation_volume=122; citation_publication_date=2015; citation_pages=166-176; citation_doi=10.1016/j.neuroimage.2015.07.067; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An integrated approach to correction for off-resonance effects and subject movement in diffusion MR imaging; citation_author=JL Andersson, SN Sotiropoulos; citation_volume=125; citation_publication_date=2016; citation_pages=1063-1078; citation_doi=10.1016/j.neuroimage.2015.10.019; citation_id=CR61"/>
    <meta name="citation_reference" content="Andersson, J.L.R., Graham, M.S., Zsoldos, E. &amp; Sotiropoulos, S.N. Incorporating outlier detection and replacement into a non-parameteric framework for movement and distortion correction of diffusion MR images. Neuroimage j.neuroimage.2016.06.058 (2016)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Recent progress and outstanding issues in motion correction in resting state fMRI; citation_author=JD Power, BL Schlaggar, SE Petersen; citation_volume=105; citation_publication_date=2015; citation_pages=536-551; citation_doi=10.1016/j.neuroimage.2014.10.044; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion; citation_author=JD Power, KA Barnes, AZ Snyder, BL Schlaggar, SE Petersen; citation_volume=59; citation_publication_date=2012; citation_pages=2142-2154; citation_doi=10.1016/j.neuroimage.2011.10.018; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Impact of in-scanner head motion on multiple measures of functional connectivity: relevance for studies of neurodevelopment in youth; citation_author=TD Satterthwaite; citation_volume=60; citation_publication_date=2012; citation_pages=623-632; citation_doi=10.1016/j.neuroimage.2011.12.063; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=Probabilistic independent component analysis for functional magnetic resonance imaging; citation_author=CF Beckmann, SM Smith; citation_volume=23; citation_publication_date=2004; citation_pages=137-152; citation_doi=10.1109/TMI.2003.822821; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Multi-level block permutation; citation_author=AM Winkler, MA Webster, D Vidaurre, TE Nichols, SM Smith; citation_volume=123; citation_publication_date=2015; citation_pages=253-268; citation_doi=10.1016/j.neuroimage.2015.05.092; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=High-dimensional spatial normalization of diffusion tensor images improves the detection of white matter differences: an example study using amyotrophic lateral sclerosis; citation_author=H Zhang; citation_volume=26; citation_publication_date=2007; citation_pages=1585-1597; citation_doi=10.1109/TMI.2007.906784; citation_id=CR68"/>
    <meta name="citation_reference" content="citation_journal_title=Med. Image Anal.; citation_title=Deformable registration of diffusion tensor MR images with explicit orientation optimization; citation_author=H Zhang, PA Yushkevich, DC Alexander, JC Gee; citation_volume=10; citation_publication_date=2006; citation_pages=764-785; citation_doi=10.1016/j.media.2006.06.004; citation_id=CR69"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Brodmann&#39;s areas 17 and 18 brought into stereotaxic space-where and how variable?; citation_author=K Amunts, A Malikovic, H Mohlberg, T Schormann, K Zilles; citation_volume=11; citation_publication_date=2000; citation_pages=66-84; citation_doi=10.1006/nimg.1999.0516; citation_id=CR70"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data; citation_author=SB Eickhoff; citation_volume=25; citation_publication_date=2005; citation_pages=1325-1335; citation_doi=10.1016/j.neuroimage.2004.12.034; citation_id=CR71"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=A population-average, landmark- and surface-based (PALS) atlas of human cerebral cortex; citation_author=DC Van Essen; citation_volume=28; citation_publication_date=2005; citation_pages=635-662; citation_doi=10.1016/j.neuroimage.2005.06.058; citation_id=CR72"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Comparing surface-based and volume-based analyses of functional neuroimaging data in patients with schizophrenia; citation_author=A Anticevic; citation_volume=41; citation_publication_date=2008; citation_pages=835-848; citation_doi=10.1016/j.neuroimage.2008.02.052; citation_id=CR73"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Cortical folding patterns and predicting cytoarchitecture; citation_author=B Fischl; citation_volume=18; citation_publication_date=2008; citation_pages=1973-1980; citation_doi=10.1093/cercor/bhm225; citation_id=CR74"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Measuring structural-functional correspondence: spatial variability of specialised brain regions after macro-anatomical alignment; citation_author=MA Frost, R Goebel; citation_volume=59; citation_publication_date=2012; citation_pages=1369-1381; citation_doi=10.1016/j.neuroimage.2011.08.035; citation_id=CR75"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Resting-state fMRI in the Human Connectome Project; citation_author=SM Smith; citation_volume=80; citation_publication_date=2013; citation_pages=144-168; citation_doi=10.1016/j.neuroimage.2013.05.039; citation_id=CR76"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An empirical comparison of surface-based and volume-based group studies in neuroimaging; citation_author=A Tucholka, V Fritsch, JB Poline, B Thirion; citation_volume=63; citation_publication_date=2012; citation_pages=1443-1453; citation_doi=10.1016/j.neuroimage.2012.06.019; citation_id=CR77"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Distributed hierarchical processing in the primate cerebral cortex; citation_author=DJ Felleman, DC Van Essen; citation_volume=1; citation_publication_date=1991; citation_pages=1-47; citation_doi=10.1093/cercor/1.1.1; citation_id=CR78"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=A common, high-dimensional model of the representational space in human ventral temporal cortex; citation_author=JV Haxby; citation_volume=72; citation_publication_date=2011; citation_pages=404-416; citation_doi=10.1016/j.neuron.2011.08.026; citation_id=CR79"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=DTI tractography of the human brain&#39;s language pathways; citation_author=MF Glasser, JK Rilling; citation_volume=18; citation_publication_date=2008; citation_pages=2471-2482; citation_doi=10.1093/cercor/bhn011; citation_id=CR80"/>
    <meta name="citation_reference" content="Brodmann, K. Vergleichende Lokalisationslehre der Grosshirnrinde (Leipzig, 1909)."/>
    <meta name="citation_reference" content="citation_journal_title=Brain Struct. Funct.; citation_title=The myeloarchitectonic studies on the human cerebral cortex of the Vogt-Vogt school, and their significance for the interpretation of functional neuroimaging data; citation_author=R Nieuwenhuys; citation_volume=218; citation_publication_date=2013; citation_pages=303-352; citation_doi=10.1007/s00429-012-0460-z; citation_id=CR82"/>
    <meta name="citation_reference" content="citation_journal_title=J. Psychol. Neurol.; citation_title=Allgemeinere Ergebnisse unserer Hirnforschung; citation_author=C Vogt, O Vogt; citation_volume=25; citation_publication_date=1919; citation_pages=279-468; citation_id=CR83"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Struct. Funct.; citation_title=A new myeloarchitectonic map of the human neocortex based on data from the Vogt-Vogt school; citation_author=R Nieuwenhuys, CA Broere, L Cerliani; citation_volume=220; citation_publication_date=2015; citation_pages=2551-2573; citation_doi=10.1007/s00429-014-0806-9; citation_id=CR84"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Generation and evaluation of a cortical area parcellation from resting-state correlations; citation_author=EM Gordon; citation_volume=26; citation_publication_date=2016; citation_pages=288-303; citation_doi=10.1093/cercor/bhu239; citation_id=CR85"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Functional network organization of the human brain; citation_author=JD Power; citation_volume=72; citation_publication_date=2011; citation_pages=665-678; citation_doi=10.1016/j.neuron.2011.09.006; citation_id=CR86"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurophysiol.; citation_title=The organization of the human cerebral cortex estimated by intrinsic functional connectivity; citation_author=BT Yeo; citation_volume=106; citation_publication_date=2011; citation_pages=1125-1165; citation_doi=10.1152/jn.00338.2011; citation_id=CR87"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Resting state network estimation in individual subjects; citation_author=CD Hacker; citation_volume=82; citation_publication_date=2013; citation_pages=616-633; citation_doi=10.1016/j.neuroimage.2013.05.108; citation_id=CR88"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Intrinsic and task-evoked network architectures of the human brain; citation_author=MW Cole, DS Bassett, JD Power, TS Braver, SE Petersen; citation_volume=83; citation_publication_date=2014; citation_pages=238-251; citation_doi=10.1016/j.neuron.2014.05.014; citation_id=CR89"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Automatically parcellating the human cerebral cortex; citation_author=B Fischl; citation_volume=14; citation_publication_date=2004; citation_pages=11-22; citation_doi=10.1093/cercor/bhg087; citation_id=CR90"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=ConnectomeDB&#8212;sharing human brain connectivity data; citation_author=MR Hodge; citation_volume=124; citation_issue=Pt B,; citation_publication_date=2016; citation_pages=1102-1107; citation_doi=10.1016/j.neuroimage.2015.04.046; citation_id=CR91"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroinformatics; citation_title=The Extensible Neuroimaging Archive Toolkit: an informatics platform for managing, exploring, and sharing neuroimaging data; citation_author=DS Marcus, TR Olsen, M Ramaratnam, RL Buckner; citation_volume=5; citation_publication_date=2007; citation_pages=11-34; citation_doi=10.1385/NI:5:1:11; citation_id=CR92"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A positive-negative mode of population covariation links brain connectivity, demographics and behavior; citation_author=SM Smith; citation_volume=18; citation_publication_date=2015; citation_pages=1565-1567; citation_doi=10.1038/nn.4125; citation_id=CR93"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Estimates of segregation and overlap of functional connectivity networks in the human cerebral cortex; citation_author=BT Yeo, FM Krienen, MW Chee, RL Buckner; citation_volume=88; citation_publication_date=2014; citation_pages=212-227; citation_doi=10.1016/j.neuroimage.2013.10.046; citation_id=CR94"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Parcellating cortical functional networks in individuals; citation_author=D Wang; citation_volume=18; citation_publication_date=2015; citation_pages=1853-1860; citation_doi=10.1038/nn.4164; citation_id=CR95"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Canonical genetic signatures of the adult human brain; citation_author=M Hawrylycz; citation_volume=18; citation_publication_date=2015; citation_pages=1832-1844; citation_doi=10.1038/nn.4171; citation_id=CR96"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Task-free MRI predicts individual differences in brain activity during task performance; citation_author=I Tavor; citation_volume=352; citation_publication_date=2016; citation_pages=216-220; citation_doi=10.1126/science.aad8127; citation_id=CR97"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Superficial white matter fiber systems impede detection of long-range cortical connections in diffusion MR tractography; citation_author=C Reveley; citation_volume=112; citation_publication_date=2015; citation_pages=E2820-E2828; citation_doi=10.1073/pnas.1418198112; citation_id=CR98"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=18 years of science with the Hubble Space Telescope; citation_author=JJ Dalcanton; citation_volume=457; citation_publication_date=2009; citation_pages=41-50; citation_doi=10.1038/nature07621; citation_id=CR99"/>
    <meta name="citation_reference" content="Kuhn, T. The Structure of Scientific Revolutions (University of Chicago Press, 1962)."/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Frequency preference and attention effects across cortical depths in the human primary auditory cortex; citation_author=F De Martino; citation_volume=112; citation_publication_date=2015; citation_pages=16036-16041; citation_doi=10.1073/pnas.1507552112; citation_id=CR101"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Spatial organization of frequency preference and selectivity in the human inferior colliculus; citation_author=F De Martino; citation_volume=4; citation_publication_date=2013; citation_pages=1386; citation_doi=10.1038/ncomms2379; citation_id=CR102"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Biol.; citation_title=Contextual feedback to superficial layers of V1; citation_author=L Muckli; citation_volume=25; citation_publication_date=2015; citation_pages=2690-2695; citation_doi=10.1016/j.cub.2015.08.057; citation_id=CR103"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Cytoarchitectonic analysis of the human extrastriate cortex in the region of V5/MT+: a probabilistic, stereotaxic map of area hOc5; citation_author=A Malikovic; citation_volume=17; citation_publication_date=2007; citation_pages=562-574; citation_doi=10.1093/cercor/bhj181; citation_id=CR104"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Correspondences between retinotopic areas and myelin maps in human visual cortex; citation_author=RO Abdollahi; citation_volume=99; citation_publication_date=2014; citation_pages=509-524; citation_doi=10.1016/j.neuroimage.2014.06.042; citation_id=CR105"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The impact of global signal regression on resting state correlations: are anti-correlated networks introduced?; citation_author=K Murphy, RM Birn, DA Handwerker, TB Jones, PA Bandettini; citation_volume=44; citation_publication_date=2009; citation_pages=893-905; citation_doi=10.1016/j.neuroimage.2008.09.036; citation_id=CR106"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Methods to detect, characterize, and remove motion artifact in resting state fMRI; citation_author=JD Power; citation_volume=84; citation_publication_date=2014; citation_pages=320-341; citation_doi=10.1016/j.neuroimage.2013.08.048; citation_id=CR107"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Correcting brain-wide correlation differences in resting-state FMRI; citation_author=ZS Saad; citation_volume=3; citation_publication_date=2013; citation_pages=339-352; citation_doi=10.1089/brain.2013.0156; citation_id=CR108"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Hum. Neurosci.; citation_title=The perils of global signal regression for group comparisons: a case study of Autism Spectrum Disorders; citation_author=SJ Gotts; citation_volume=7; citation_publication_date=2013; citation_pages=356; citation_doi=10.3389/fnhum.2013.00356; citation_id=CR109"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Trouble at rest: how correlation patterns and group differences become distorted after global signal regression; citation_author=ZS Saad; citation_volume=2; citation_publication_date=2012; citation_pages=25-32; citation_doi=10.1089/brain.2012.0080; citation_id=CR110"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The role of physiological noise in resting-state functional connectivity; citation_author=RM Birn; citation_volume=62; citation_publication_date=2012; citation_pages=864-870; citation_doi=10.1016/j.neuroimage.2012.01.016; citation_id=CR111"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Relationship between respiration, end-tidal CO2, and BOLD signals in resting-state fMRI; citation_author=C Chang, GH Glover; citation_volume=47; citation_publication_date=2009; citation_pages=1381-1393; citation_doi=10.1016/j.neuroimage.2009.04.048; citation_id=CR112"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Mapping the end-tidal CO2 response function in the resting-state BOLD fMRI signal: spatial specificity, test-retest reliability and effect of fMRI sampling rate; citation_author=AM Golestani, C Chang, JB Kwinta, YB Khatamian, J Jean Chen; citation_volume=104; citation_publication_date=2015; citation_pages=266-277; citation_doi=10.1016/j.neuroimage.2014.10.031; citation_id=CR113"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Network modelling methods for FMRI; citation_author=SM Smith; citation_volume=54; citation_publication_date=2011; citation_pages=875-891; citation_doi=10.1016/j.neuroimage.2010.08.063; citation_id=CR114"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Adding dynamics to the Human Connectome Project with MEG; citation_author=LJ Larson-Prior; citation_volume=80; citation_publication_date=2013; citation_pages=190-201; citation_doi=10.1016/j.neuroimage.2013.05.056; citation_id=CR115"/>
    <meta name="citation_reference" content="citation_journal_title=J. Magn. Reson. Imaging; citation_title=The Alzheimer&#39;s Disease Neuroimaging Initiative (ADNI): MRI methods; citation_author=CR Jack; citation_volume=27; citation_publication_date=2008; citation_pages=685-691; citation_doi=10.1002/jmri.21049; citation_id=CR116"/>
    <meta name="citation_author" content="Glasser, Matthew F"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, Washington University Medical School, St. Louis, USA"/>
    <meta name="citation_author" content="Smith, Stephen M"/>
    <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK"/>
    <meta name="citation_author" content="Marcus, Daniel S"/>
    <meta name="citation_author_institution" content="Department of Radiology, Washington University Medical School, St. Louis, USA"/>
    <meta name="citation_author" content="Andersson, Jesper L R"/>
    <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK"/>
    <meta name="citation_author" content="Auerbach, Edward J"/>
    <meta name="citation_author_institution" content="Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA"/>
    <meta name="citation_author" content="Behrens, Timothy E J"/>
    <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK"/>
    <meta name="citation_author" content="Coalson, Timothy S"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, Washington University Medical School, St. Louis, USA"/>
    <meta name="citation_author" content="Harms, Michael P"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Washington University Medical School, St. Louis, USA"/>
    <meta name="citation_author" content="Jenkinson, Mark"/>
    <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK"/>
    <meta name="citation_author" content="Moeller, Steen"/>
    <meta name="citation_author_institution" content="Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA"/>
    <meta name="citation_author" content="Robinson, Emma C"/>
    <meta name="citation_author_institution" content="Department of Computing, Imperial College London, UK"/>
    <meta name="citation_author" content="Sotiropoulos, Stamatios N"/>
    <meta name="citation_author_institution" content="Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK"/>
    <meta name="citation_author" content="Xu, Junqian"/>
    <meta name="citation_author_institution" content="7Translational and Molecular Imaging Institute, Icahn School of Medicine at Mount Sinai, New York, USA"/>
    <meta name="citation_author" content="Yacoub, Essa"/>
    <meta name="citation_author_institution" content="Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA"/>
    <meta name="citation_author" content="Ugurbil, Kamil"/>
    <meta name="citation_author_institution" content="Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, USA"/>
    <meta name="citation_author" content="Van Essen, David C"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, Washington University Medical School, St. Louis, USA"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="The Human Connectome Project's neuroimaging approach"/>
    <meta name="twitter:description" content="Nature Neuroscience - This paper describes an integrated approach for neuroimaging data acquisition, analysis and sharing. Building on methodological advances from the Human Connectome Project..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig1_HTML.jpg"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/nn.4361"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="The Human Connectome Project's neuroimaging approach - Nature Neuroscience"/>
    <meta property="og:description" content="This paper describes an integrated approach for neuroimaging data acquisition, analysis and sharing. Building on methodological advances from the Human Connectome Project (HCP) and elsewhere, the HCP-style paradigm applies to new and existing data sets that meet core requirements and may accelerate progress in understanding the brain in health and disease."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig1_HTML.jpg"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=nn.4361;doi=10.1038/nn.4361;techmeta=57,59;subjmeta=1594,2649,378,3917,3920,631;kwrd=Language,Neural+circuits,Sensory+processing">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-12044720&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Dnn.4361%26doi%3D10.1038/nn.4361%26techmeta%3D57,59%26subjmeta%3D1594,2649,378,3917,3920,631%26kwrd%3DLanguage,Neural+circuits,Sensory+processing">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-12044720&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Dnn.4361%26doi%3D10.1038/nn.4361%26techmeta%3D57,59%26subjmeta%3D1594,2649,378,3917,3920,631%26kwrd%3DLanguage,Neural+circuits,Sensory+processing"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/nn.4361'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;review-article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:review articles"><span itemprop="name">review articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        The Human Connectome Project's neuroimaging approach
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/nn.4361.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Review Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2016-08-26">26 August 2016</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">The Human Connectome Project's neuroimaging approach</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Matthew_F-Glasser-Aff1" data-author-popup="auth-Matthew_F-Glasser-Aff1" data-author-search="Glasser, Matthew F" data-corresp-id="c1">Matthew F Glasser<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stephen_M-Smith-Aff2" data-author-popup="auth-Stephen_M-Smith-Aff2" data-author-search="Smith, Stephen M">Stephen M Smith</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-8166-069X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8166-069X</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Daniel_S-Marcus-Aff3" data-author-popup="auth-Daniel_S-Marcus-Aff3" data-author-search="Marcus, Daniel S">Daniel S Marcus</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jesper_L_R-Andersson-Aff2" data-author-popup="auth-Jesper_L_R-Andersson-Aff2" data-author-search="Andersson, Jesper L R">Jesper L R Andersson</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Edward_J-Auerbach-Aff4" data-author-popup="auth-Edward_J-Auerbach-Aff4" data-author-search="Auerbach, Edward J">Edward J Auerbach</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Timothy_E_J-Behrens-Aff2" data-author-popup="auth-Timothy_E_J-Behrens-Aff2" data-author-search="Behrens, Timothy E J">Timothy E J Behrens</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Timothy_S-Coalson-Aff1" data-author-popup="auth-Timothy_S-Coalson-Aff1" data-author-search="Coalson, Timothy S">Timothy S Coalson</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_P-Harms-Aff5" data-author-popup="auth-Michael_P-Harms-Aff5" data-author-search="Harms, Michael P">Michael P Harms</a><sup class="u-js-hide"><a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Mark-Jenkinson-Aff2" data-author-popup="auth-Mark-Jenkinson-Aff2" data-author-search="Jenkinson, Mark">Mark Jenkinson</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Steen-Moeller-Aff4" data-author-popup="auth-Steen-Moeller-Aff4" data-author-search="Moeller, Steen">Steen Moeller</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Emma_C-Robinson-Aff6" data-author-popup="auth-Emma_C-Robinson-Aff6" data-author-search="Robinson, Emma C">Emma C Robinson</a><sup class="u-js-hide"><a href="#Aff6">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stamatios_N-Sotiropoulos-Aff2" data-author-popup="auth-Stamatios_N-Sotiropoulos-Aff2" data-author-search="Sotiropoulos, Stamatios N">Stamatios N Sotiropoulos</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Junqian-Xu-Aff7" data-author-popup="auth-Junqian-Xu-Aff7" data-author-search="Xu, Junqian">Junqian Xu</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-8438-2066"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8438-2066</a></span><sup class="u-js-hide"><a href="#Aff7">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Essa-Yacoub-Aff4" data-author-popup="auth-Essa-Yacoub-Aff4" data-author-search="Yacoub, Essa">Essa Yacoub</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kamil-Ugurbil-Aff4" data-author-popup="auth-Kamil-Ugurbil-Aff4" data-author-search="Ugurbil, Kamil">Kamil Ugurbil</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 16 authors for this article" title="Show all 16 authors for this article"></li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-David_C-Van_Essen-Aff1" data-author-popup="auth-David_C-Van_Essen-Aff1" data-author-search="Van Essen, David C" data-corresp-id="c2">David C Van Essen<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><sup class="u-js-hide"><a href="#Aff1">1</a></sup></li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>19</b>,<span class="u-visually-hidden">pages </span>11751187 (<span data-test="article-publication-year">2016</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">31k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">566 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">67 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/nn.4361/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/language" data-track="click" data-track-action="view subject" data-track-label="link">Language</a></li><li class="c-article-subject-list__subject"><a href="/subjects/neural-circuit" data-track="click" data-track-action="view subject" data-track-label="link">Neural circuits</a></li><li class="c-article-subject-list__subject"><a href="/subjects/sensory-processing" data-track="click" data-track-action="view subject" data-track-label="link">Sensory processing</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs2" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs2">Abstract</h2><div class="c-article-section__content" id="Abs2-content"><p>Noninvasive human neuroimaging has yielded many discoveries about the brain. Numerous methodological advances have also occurred, though inertia has slowed their adoption. This paper presents an integrated approach to data acquisition, analysis and sharing that builds upon recent advances, particularly from the Human Connectome Project (HCP). The 'HCP-style' paradigm has seven core tenets: (i) collect multimodal imaging data from many subjects; (ii) acquire data at high spatial and temporal resolution; (iii) preprocess data to minimize distortions, blurring and temporal artifacts; (iv) represent data using the natural geometry of cortical and subcortical structures; (v) accurately align corresponding brain areas across subjects and studies; (vi) analyze data using neurobiologically accurate brain parcellations; and (vii) share published data via user-friendly databases. We illustrate the HCP-style paradigm using existing HCP data sets and provide guidance for future research. Widespread adoption of this paradigm should accelerate progress in understanding the brain in health and disease.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/nn.4361.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/nn.4361.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41597-023-02525-0/MediaObjects/41597_2023_2525_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41597-023-02525-0?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41597-023-02525-0">DISCOVER-EEG: an open, fully automated EEG pipeline for biomarker discovery in clinical neuroscience
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">11 September 2023</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Cristina Gil vila, Felix S. Bott,  Markus Ploner</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41592-022-01625-w/MediaObjects/41592_2022_1625_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41592-022-01625-w?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41592-022-01625-w">neuromaps: structural and functional interpretation of brain maps
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">06 October 2022</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Ross D. Markello, Justine Y. Hansen,  Bratislav Misic</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41593-020-00726-z?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41593-020-00726-z">Challenges and future directions for representations of functional brain organization
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">26 October 2020</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Janine Bijsterbosch, Samuel J. Harrison,  Eugene P. Duff</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711549893,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>The Human Connectome Project (HCP) began in 2010 when the National Institutes of Health (NIH) awarded <span class="stix"></span>$40 million to two consortia to develop improved neuroimaging methods and to acquire a data set of unprecedented size and quality for mapping the normal human macroscale connectome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Behrens, T.E. &amp; Sporns, O. Human connectomics. Curr. Opin. Neurobiol. 22, 144153 (2012)." href="/articles/nn.4361#ref-CR1" id="ref-link-section-d24703174e594">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Jbabdi, S., Sotiropoulos, S.N., Haber, S.N., Van Essen, D.C. &amp; Behrens, T.E. Measuring macroscopic brain connections in vivo. Nat. Neurosci. 18, 15461555 (2015)." href="/articles/nn.4361#ref-CR2" id="ref-link-section-d24703174e597">2</a></sup>; that is, the long-distance connections between all of the brain's areas. Better maps of the brain's areas and their connections will deepen our understanding of healthy brain function and may improve our ability to understand and treat neurological and psychiatric disorders. The 'WU-Minn-Ox' HCP consortium, centered at Washington University, University of Minnesota and Oxford University, spent its first 2 years developing the state-of-the-art methods discussed here and then proceeded to acquire, analyze and share high quality multimodal neuroimaging, behavioral and genotype data from <span class="stix"></span>1,100 healthy young adult twins and nontwin siblings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Smith, S. Introduction to the NeuroImage special issue Mapping the Connectome. Neuroimage 80, 1 (2013)." href="/articles/nn.4361#ref-CR3" id="ref-link-section-d24703174e601">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Van Essen, D.C. et al. The WU-Minn Human Connectome Project: an overview. Neuroimage 80, 6279 (2013)." href="/articles/nn.4361#ref-CR4" id="ref-link-section-d24703174e604">4</a></sup>. The HCP focused primarily on magnetic resonance imaging (MRI)-based imaging modalities to measure brain architecture, function and connectivity, as well as the regular, topographic organization of some brain areas; however, magnetoencephalography data were also acquired in some subjects (see below). As the HCP's core activities draw to a close in 2016, its contributions include (i) a set of broadly useful MRI acquisition protocols, consisting of thoroughly tested and optimized pulse sequences and image reconstruction algorithms; (ii) a collection of exceptionally high-quality, freely shared neuroimaging data; (iii) numerous publicly available neuroimaging software and informatics tools; (iv) a growing number of discoveries emerging from analyses of HCP data; (v) the emergence of an HCP-style paradigm for neuroimaging data acquisition, analysis and sharing; and (vi) a growing number of HCP-style projects that will study different age ranges and brain disorders.</p><p>This article is both a position paper and a primer, promoting awareness and adoption of the integrated HCP-style paradigm along with giving guidance on its use. We provide an accessible conceptual overview of seven core tenets of the paradigm and summarize several recent discoveries made possible through its application. Additional technical information and discussion is provided in the cited publications and in <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Notes 118</a>, which expand on specific topics raised in the main text. Investigators interested in practical aspects of implementing the HCP paradigm in their own laboratories can access the MRI protocols, HCP software and HCP course materials listed below.</p><p>The seven tenets of the HCP-style paradigm span the domains of acquisition, analysis and sharing of MRI-based data: (i) acquiring large amounts of high-quality, multimodal data on as many subjects as feasible to measure architectural, functional, connectional and topographical information<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Van Essen, D.C. et al. The WU-Minn Human Connectome Project: an overview. Neuroimage 80, 6279 (2013)." href="/articles/nn.4361#ref-CR4" id="ref-link-section-d24703174e617">4</a></sup> (ii) acquiring these MRI data with high spatial, temporal and angular resolution for structural, functional and diffusion MRI using cutting-edge accelerated acquisition protocols<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Ugurbil, K. et al. Pushing spatial and temporal resolution for functional and diffusion MRI in the Human Connectome Project. Neuroimage 80, 80104 (2013)." href="/articles/nn.4361#ref-CR5" id="ref-link-section-d24703174e621">5</a></sup>; (iii) minimizing blurring introduced by preprocessing and removing distortions, noise and temporal artifacts as selectively and completely as possible<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e625">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Griffanti, L. et al. ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging. Neuroimage 95, 232247 (2014)." href="/articles/nn.4361#ref-CR7" id="ref-link-section-d24703174e628">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Salimi-Khorshidi, G. et. al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. Neuroimage 90, 449468 (2014). This study demonstrated a method for automatically cleaning spatially specific structured noise out of fMRI data using an independent component analysis (ICA) decomposition and machine learning classification of the resulting components into signal and noise." href="/articles/nn.4361#ref-CR8" id="ref-link-section-d24703174e631">8</a></sup>; (iv) representing cortical data (surface vertices) and subcortical data (volume voxels) in a common geometrical framework ('CIFTI grayordinates') that is optimal for each<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e635">6</a></sup>; (v) accurately aligning corresponding brain areas across subjects and studies using areal features related to connectivity and architecture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e639">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Robinson, E.C. et al. MSM: a new flexible framework for multimodal surface matching. Neuroimage 100, 414426 (2014). This study introduced a powerful new method for aligning cortical surfaces from individuals to an atlas using areal features as well as folding patterns to constrain the alignment." href="/articles/nn.4361#ref-CR10" id="ref-link-section-d24703174e642">10</a></sup>; (vi) using structurally and functionally relevant brain parcellations (preferably based on multiple modalities) to provide a strong neuroanatomical framework for condensing complex neuroimaging data and for enhancing statistical sensitivity and power without blurring across areal boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e647">9</a></sup>; and (vii) routinely sharing extensively analyzed results such as statistical maps<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Van Essen, D.C. et al. The brain analysis of spatial maps and atlases (BALSA) database. Neuroimage j.neuroimage.2016.04.002 (2016)." href="/articles/nn.4361#ref-CR11" id="ref-link-section-d24703174e651">11</a></sup> (plus raw and preprocessed data when feasible<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Marcus, D.S. et al. Human Connectome Project informatics: quality control, database services, and data visualization. Neuroimage 80, 202219 (2013)." href="/articles/nn.4361#ref-CR12" id="ref-link-section-d24703174e655">12</a></sup>) together with the code used for the analysis, so that other neuroscientists can make precise comparisons across studies, along with replicating and extending findings.</p><p>The HCP-style paradigm builds on recent advances in data acquisition, analysis and sharing from many neuroimaging labs worldwide, both within and outside the HCP consortia. It benefitted greatly from the NIH's emphasis on improving MRI acquisition and analysis methods prior to initiating high-throughput data collection. The HCP-style paradigm differs from common practices (the 'traditional paradigm') in the human neuroimaging community that emerged during the early years of PET and functional MRI (fMRI). These include (i) voxel-based (volumetric) analysis of data acquired at relatively coarse spatial and temporal resolutions on small groups of subjects; (ii) extensive unconstrained spatial smoothing that attempts to compensate for imperfect cross-subject alignment, improve signal-to-noise ratio (SNR) and satisfy oversimplified statistical assumptions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Eklund, A., Nichols, T.E. &amp; Knutsson, H. Cluster failure: why fMRI inferences for spatial extent have inflated false-positive rates. Proc. Natl. Acad. Sci. USA 28, 79007905 (2016)." href="/articles/nn.4361#ref-CR13" id="ref-link-section-d24703174e662">13</a></sup>; (iii) analysis without using parcellations at all, or using parcellations either derived from Brodmann's 2D schematic drawing or from locations relative to gyri and sulci; and (iv) the reporting of summary results using 3D volumetric standard space coordinates while keeping the extensively analyzed data private. Relative to the new paradigm, the traditional paradigm has limitations in spatial fidelity, neuroanatomical fundamentals and the robustness of cross-study comparisons, along lines also noted by Turner<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Turner, R. Where matters: new approaches to brain analysis. in Microstructural Parcellation of the Human Cerebral Cortex (eds. Geyer, S. &amp; Turner, R.) 179196 (Springer, 2013)." href="/articles/nn.4361#ref-CR14" id="ref-link-section-d24703174e666">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Turner, R. &amp; Geyer, S. Comparing like with like: the power of knowing where you are. Brain Connect. 4, 547557 (2014). The authors emphasize the importance of high-quality data acquisition, careful surface-based analysis in individual subjects and localization in relation to the best available cortical parcellation. Hence they presage many of the points made in the present article." href="/articles/nn.4361#ref-CR15" id="ref-link-section-d24703174e669">15</a></sup>.</p><p>Though most neuroimaging studies still use the traditional paradigm, important exceptions dating as far back as the 1990s include numerous studies on retinotopy that used fMRI data to map the arrangement of visual areas on cortical surface models of individual subjects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Sereno, M.I. et al. Borders of multiple visual areas in humans revealed by functional magnetic resonance imaging. Science 268, 889893 (1995)." href="/articles/nn.4361#ref-CR16" id="ref-link-section-d24703174e677">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Wang, L., Mruczek, R.E., Arcaro, M.J. &amp; Kastner, S. Probabilistic maps of visual topography in human cortex. Cereb. Cortex 25, 39113931 (2015)." href="/articles/nn.4361#ref-CR17" id="ref-link-section-d24703174e680">17</a></sup>. Thanks in part to this emphasis on surface-based analyses, the visual system is one of the better-understood functional systems in the human brain. We anticipate accelerated progress in understanding other brain regions as the new paradigm is applied to them more routinely.</p><p>A solid conceptual understanding of MRI acquisition, preprocessing and analysis (tenets 15) is important for applying the HCP-style paradigm to one's own studies, as well as for understanding the methodological choices made by the HCP and its successors (see below). Many of these choices were driven by fundamental properties of human brain anatomy, the physics of MRI, and human physiology.</p><h3 class="c-article__sub-heading" id="Sec2">Tenets 1 and 2: multimodal data acquisition informed by brain anatomy and physiology</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec3">Comprehensive noninvasive brain mapping of many subjects using multimodal MRI.</h4><p>The HCP acquired <span class="stix"></span>4 h of structural, functional, and diffusion images (along with auxiliary 'field map' images to help with image preprocessing). The structural images were T1-weighted (T1w) and T2-weighted (T2w) images (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 1</a> for a discussion of the relationship between these terms and MRI contrast). These images allow noninvasive measurement of architectural properties such as myelin content within the cerebral cortex and cortical thickness that match well with corresponding invasive measures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Fischl, B. &amp; Dale, A.M. Measuring the thickness of the human cerebral cortex from magnetic resonance images. Proc. Natl. Acad. Sci. USA 97, 1105011055 (2000)." href="/articles/nn.4361#ref-CR18" id="ref-link-section-d24703174e701">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Glasser, M.F., Goyal, M.S., Preuss, T.M., Raichle, M.E. &amp; Van Essen, D.C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. Neuroimage 93, 165175 (2014)." href="/articles/nn.4361#ref-CR19" id="ref-link-section-d24703174e704">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Glasser, M.F. &amp; Van Essen, D.C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 1159711616 (2011)." href="/articles/nn.4361#ref-CR20" id="ref-link-section-d24703174e707">20</a></sup>. The structural images also generate geometrical models of brain anatomy (cortical surface meshes and subcortical segmentations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Fischl, B. &amp; Dale, A.M. Measuring the thickness of the human cerebral cortex from magnetic resonance images. Proc. Natl. Acad. Sci. USA 97, 1105011055 (2000)." href="/articles/nn.4361#ref-CR18" id="ref-link-section-d24703174e711">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Dale, A.M., Fischl, B. &amp; Sereno, M.I. Cortical surface-based analysis. I. Segmentation and surface reconstruction. Neuroimage 9, 179194 (1999)." href="/articles/nn.4361#ref-CR21" id="ref-link-section-d24703174e714">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Dale, A.M. &amp; Sereno, M.I. Improved localizadon of cortical activity by combining EEG and MEG with MRI cortical surface reconstruction: a linear approach. J. Cogn. Neurosci. 5, 162176 (1993)." href="/articles/nn.4361#ref-CR22" id="ref-link-section-d24703174e717">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Fischl, B., Liu, A. &amp; Dale, A.M. Automated manifold surgery: constructing geometrically accurate and topologically correct models of the human cerebral cortex. IEEE Trans. Med. Imaging 20, 7080 (2001)." href="/articles/nn.4361#ref-CR23" id="ref-link-section-d24703174e720">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Fischl, B. et al. Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. Neuron 33, 341355 (2002)." href="/articles/nn.4361#ref-CR24" id="ref-link-section-d24703174e723">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Fischl, B. et al. Sequence-independent segmentation of magnetic resonance images. Neuroimage 23 (Suppl. 1), S69S84 (2004)." href="/articles/nn.4361#ref-CR25" id="ref-link-section-d24703174e726">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Fischl, B., Sereno, M.I. &amp; Dale, A.M. Cortical surface-based analysis. II: inflation, flattening, and a surface-based coordinate system. Neuroimage 9, 195207 (1999)." href="/articles/nn.4361#ref-CR26" id="ref-link-section-d24703174e730">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Fischl, B., Sereno, M.I., Tootell, R.B. &amp; Dale, A.M. High-resolution intersubject averaging and a coordinate system for the cortical surface. Hum. Brain Mapp. 8, 272284 (1999). This study introduced an automated method for aligning cortical surfaces from individuals to a surface-based atlas using cortical folding to constrain the registration." href="/articles/nn.4361#ref-CR27" id="ref-link-section-d24703174e733">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Sgonne, F., Pacheco, J. &amp; Fischl, B. Geometrically accurate topology-correction of cortical surfaces using nonseparating loops. IEEE Trans. Med. Imaging 26, 518529 (2007)." href="/articles/nn.4361#ref-CR28" id="ref-link-section-d24703174e736">28</a></sup>) that are critical for all other analyses. The functional images included task-based fMRI, in which the subjects carried out behavioral tasks while in the scanner<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Barch, D.M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. Neuroimage 80, 169189 (2013)." href="/articles/nn.4361#ref-CR29" id="ref-link-section-d24703174e740">29</a></sup>, and resting-state scans, in which subjects fixated on a crosshair. These functional images enable measurement of functional activation within brain areas and functional connectivity based on correlations of the fMRI signal between brain areas in the absence of a task or controlled external stimuli. They can also map the internal topographic organization within brain areas, such as the ordered representation of visual space within visual areas using task-driven<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Sereno, M.I. et al. Borders of multiple visual areas in humans revealed by functional magnetic resonance imaging. Science 268, 889893 (1995)." href="/articles/nn.4361#ref-CR16" id="ref-link-section-d24703174e744">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Wang, L., Mruczek, R.E., Arcaro, M.J. &amp; Kastner, S. Probabilistic maps of visual topography in human cortex. Cereb. Cortex 25, 39113931 (2015)." href="/articles/nn.4361#ref-CR17" id="ref-link-section-d24703174e747">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wandell, B.A. &amp; Winawer, J. Imaging retinotopic maps in the human brain. Vision Res. 51, 718737 (2011)." href="/articles/nn.4361#ref-CR30" id="ref-link-section-d24703174e750">30</a></sup> or resting-state-based<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e755">9</a></sup> methods (visuotopic organization). The similarity of these detailed maps across species<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Orban, G.A., Zhu, Q. &amp; Vanduffel, W. The transition in the ventral stream from feature to real-world entity representations. Front. Psychol. 5, 695 (2014)." href="/articles/nn.4361#ref-CR31" id="ref-link-section-d24703174e759">31</a></sup> and in comparison with invasive studies in nonhuman primates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Brewer, A.A., Press, W.A., Logothetis, N.K. &amp; Wandell, B.A. Visual areas in macaque cortex measured using functional magnetic resonance imaging. J. Neurosci. 22, 1041610426 (2002)." href="/articles/nn.4361#ref-CR32" id="ref-link-section-d24703174e763">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Van Essen, D.C., Felleman, D.J., DeYoe, E.A., Olavarria, J. &amp; Knierim, J. Modular and hierarchical organization of extrastriate visual cortex in the macaque monkey. Cold Spring Harb. Symp. Quant. Biol. 55, 679696 (1990)." href="/articles/nn.4361#ref-CR33" id="ref-link-section-d24703174e766">33</a></sup> confirms that fMRI can accurately capture properties of brain organization. Diffusion MRI (dMRI) uses directionally dependent water diffusion within white matter to encode average axonal fiber orientations, which can be connected by streamline tractography algorithms to estimate brain structural connectivity, yielding results that are modestly correlated with invasively measured anatomical connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Calabrese, E., Badea, A., Cofer, G., Qi, Y. &amp; Johnson, G.A. A diffusion MRI tractography connectome of the mouse brain and comparison with neuronal tracer data. Cereb. Cortex 25, 46284637 (2015)." href="/articles/nn.4361#ref-CR34" id="ref-link-section-d24703174e770">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Donahue, C.J. et al. Using diffusion tractography to predict cortical connection strength and distance: A quantitative comparison with tracers in the monkey. J. Neurosci 36, 67586770 (2016)." href="/articles/nn.4361#ref-CR35" id="ref-link-section-d24703174e773">35</a></sup>. dMRI also allows estimates of microscopic structural properties of brain tissue, such as the integrity and orderliness of axons (for example, refs. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Basser, P.J., Mattiello, J. &amp; LeBihan, D. MR diffusion tensor spectroscopy and imaging. Biophys. J. 66, 259267 (1994)." href="/articles/nn.4361#ref-CR36" id="ref-link-section-d24703174e776">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Fieremans, E., Jensen, J.H. &amp; Helpern, J.A. White matter characterization with diffusional kurtosis imaging. Neuroimage 58, 177188 (2011)." href="/articles/nn.4361#ref-CR37" id="ref-link-section-d24703174e779">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Zhang, H., Schneider, T., Wheeler-Kingshott, C.A. &amp; Alexander, D.C. NODDI: practical in vivo neurite orientation dispersion and density imaging of the human brain. Neuroimage 61, 10001016 (2012)." href="/articles/nn.4361#ref-CR38" id="ref-link-section-d24703174e783">38</a>). Thus, the HCP MRI data enable brain architecture, function, connectivity and topography to be estimated noninvasively. The large number of subjects (<span class="stix"></span>1,100) imaged by the HCP enables statistically precise estimation of these measures in the healthy young adult population and investigation of the nature and extent of their individual variability, while the focus on twins (and their non-twin siblings) enables assessment of their heritability.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec4">How brain anatomy drives MRI spatial resolution.</h4><p>The cerebral cortex, the seat of human cognition, is a folded sheet 1.64 mm thick (mean <span class="stix"></span>2.6 mm) with a surface area of <span class="stix"></span>1,000 cm<sup>2</sup> per hemisphere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Glasser, M.F., Goyal, M.S., Preuss, T.M., Raichle, M.E. &amp; Van Essen, D.C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. Neuroimage 93, 165175 (2014)." href="/articles/nn.4361#ref-CR19" id="ref-link-section-d24703174e797">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Van Essen, D.C., Glasser, M.F., Dierker, D.L., Harwell, J. &amp; Coalson, T. Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases. Cereb. Cortex 22, 22412262 (2012)." href="/articles/nn.4361#ref-CR39" id="ref-link-section-d24703174e800">39</a></sup>. High quality structural images (both T1w and T2w) are vital for generating accurate geometrical models upon which all other analyses are based (surface meshes for the cerebral cortex and gray matter regions of interest for the subcortical structures)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e804">6</a></sup>. For the main HCP, we acquired these images at an isotropic spatial resolution of 0.7 mm (voxel cubes 0.7 mm on each side), enabling accurate individual subject maps of cortical myelin content and thickness together with precise models of the inner (white matter) and outer (pial) surfaces. In general, we recommend acquiring T1w and T2w images at 0.8 mm isotropic or better (half the minimum thickness of the cortex), in contrast to the single 1-mm T1w image traditionally acquired. Such high-resolution images benefit from tightly fitting, multichannel radio frequency receiving head coils, such as the Siemens 32-channel head coil used by the HCP.</p><p>For fMRI, it is important to acquire images of sufficient spatial resolution to distinguish among cortical gray matter, cerebrospinal fluid and white matter, and between opposite banks of cortical folds. The HCP acquired high-resolution fMRI data at 2 mm isotropic (with high resolution fMRI defined as having voxels smaller than the mean cortical thickness of 2.6 mm; <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig7">Supplementary Fig. 1</a>) on its customized 3-tesla (3 T) scanner (and at 1.6 mm isotropic at 7 T). High-resolution fMRI, in contrast to resolutions as low as 4 mm that have been traditionally used in fMRI, enables more precise localization of functional signals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e814">6</a></sup>.</p><p>dMRI aims to address a fundamentally resolution-starved problem: mapping connections through brain white matter whose individual axons are mostly &lt;1 m in diameter<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Liewald, D., Miller, R., Logothetis, N., Wagner, H.J. &amp; Schz, A. Distribution of axon diameters in cortical white matter: an electron-microscopic study on three human brains and a macaque. Biol. Cybern. 108, 541557 (2014)." href="/articles/nn.4361#ref-CR40" id="ref-link-section-d24703174e821">40</a></sup> with MRI voxels that exceed 1 mm isotropic, contain hundreds of thousands of axons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Van Essen, D.C. &amp; Ugurbil, K. The future of the human connectome. Neuroimage 62, 12991310 (2012)." href="/articles/nn.4361#ref-CR41" id="ref-link-section-d24703174e825">41</a></sup> and typically include multiple populations of fibers crossing in various directions. Both of the HCP consortia used customized MRI scanners and pulse sequences to acquire <i>in vivo</i> human diffusion data of unprecedented resolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Ugurbil, K. et al. Pushing spatial and temporal resolution for functional and diffusion MRI in the Human Connectome Project. Neuroimage 80, 80104 (2013)." href="/articles/nn.4361#ref-CR5" id="ref-link-section-d24703174e832">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Huang, S.Y. et al. The impact of gradient strength on in vivo diffusion MRI estimates of axon diameter. Neuroimage 106, 464472 (2015)." href="/articles/nn.4361#ref-CR42" id="ref-link-section-d24703174e835">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="McNab, J.A. et al. The Human Connectome Project and beyond: initial applications of 300 mT/m gradients. Neuroimage 80, 234245 (2013)." href="/articles/nn.4361#ref-CR43" id="ref-link-section-d24703174e838">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Setsompop, K. et al. Blipped-controlled aliasing in parallel imaging for simultaneous multislice echo planar imaging with reduced g-factor penalty. Magn. Reson. Med. 67, 12101224 (2012). This study introduced a slice gradient blip technique for simultaneous multislice (multiband) echo-planar imaging to significantly reduce image artifacts in unaliasing of simultaneously acquired slices, particularly for high multiband factors. This technique is adopted in the HCP pulse sequences for fMRI and dMRI." href="/articles/nn.4361#ref-CR44" id="ref-link-section-d24703174e841">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Setsompop, K. et al. Pushing the limits of in vivo diffusion MRI for the Human Connectome Project. Neuroimage 80, 220233 (2013)." href="/articles/nn.4361#ref-CR45" id="ref-link-section-d24703174e844">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Sotiropoulos, S.N. et al. Fusion in diffusion MRI for improved fibre orientation estimation: An application to the 3T and 7T data of the Human Connectome Project. Neuroimage 134, 396409 (2016)." href="/articles/nn.4361#ref-CR46" id="ref-link-section-d24703174e847">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Sotiropoulos, S.N. et al. Advances in diffusion MRI acquisition and processing in the Human Connectome Project. Neuroimage 80, 125143 (2013). The authors report on advances in acquisition and analysis of HCP diffusion imaging data. This includes use of scanners with high maximal gradient strength, multiband pulse sequences, and improvements in image reconstruction, distortion correction, fiber orientation modeling and probabilistic tractography algorithms." href="/articles/nn.4361#ref-CR47" id="ref-link-section-d24703174e851">47</a></sup>. The 3 T WU-Minn-Ox HCP scanner used very strong magnetic field gradients (100 mT/m) that were approximately double those available in commercial scanners when the project began in 2010. Along with new and optimized dMRI pulse sequences, this enabled 1.25 mm isotropic resolution for human dMRI <i>in vivo</i> at 3 T (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 2</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig8">Supplementary Figs. 2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">3</a>) and 1.05 mm isotropic at 7 T (refs. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Sotiropoulos, S.N. et al. Fusion in diffusion MRI for improved fibre orientation estimation: An application to the 3T and 7T data of the Human Connectome Project. Neuroimage 134, 396409 (2016)." href="/articles/nn.4361#ref-CR46" id="ref-link-section-d24703174e867">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Vu, A.T. et al. High resolution whole brain diffusion imaging at 7T for the Human Connectome Project. Neuroimage 122, 318331 (2015)." href="/articles/nn.4361#ref-CR48" id="ref-link-section-d24703174e870">48</a>), in contrast to the 22.5 mm resolution of traditional dMRI. The recently introduced Siemens Prisma MRI scanner (80 mT/m), developed in part because of the HCP, enables the community to acquire similar dMRI data.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec5">Faster, longer imaging for cleaner fMRI data and more robust estimates of white matter fiber orientation.</h4><p>Most MRI-based functional imaging relies on the blood oxygen leveldependent (BOLD) contrast mechanism, in which increased neural activity indirectly alters the regional deoxygenated hemoglobin content over a time course (hemodynamic response function) of many seconds<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Hillman, E.M. Coupling mechanism and significance of the BOLD signal: a status report. Annu. Rev. Neurosci. 37, 161181 (2014)." href="/articles/nn.4361#ref-CR49" id="ref-link-section-d24703174e882">49</a></sup>. Accordingly, one might think that it is sufficient to acquire fMRI images at intervals modestly faster than the hemodynamic response function (a repetition time (TR) of 23 s has been typical for whole brain imaging). However, fMRI images are corrupted by many sources of time-dependent (temporal) artifacts, including subject movement, physiology and scanner instability. Many artifacts occur more rapidly than the typical fMRI volume TR and thus are aliased into the signal. High-temporal-resolution fMRI data (i.e., TR &lt; 1 s) facilitates more selective and effective artifact removal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Griffanti, L. et al. ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging. Neuroimage 95, 232247 (2014)." href="/articles/nn.4361#ref-CR7" id="ref-link-section-d24703174e886">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Salimi-Khorshidi, G. et. al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. Neuroimage 90, 449468 (2014). This study demonstrated a method for automatically cleaning spatially specific structured noise out of fMRI data using an independent component analysis (ICA) decomposition and machine learning classification of the resulting components into signal and noise." href="/articles/nn.4361#ref-CR8" id="ref-link-section-d24703174e889">8</a></sup> and improves statistical efficiency, particularly for multivariate statistics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Feinberg, D.A. et al. Multiplexed echo planar imaging for sub-second whole brain FMRI and fast diffusion imaging. PLoS One 5, e15710 (2010)." href="/articles/nn.4361#ref-CR50" id="ref-link-section-d24703174e893">50</a></sup>. Recent data also suggest that there may be faster components to the BOLD response<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Chen, J.E. &amp; Glover, G.H. BOLD fractional contribution to resting-state functional connectivity above 0.1 Hz. Neuroimage 107, 207218 (2015)." href="/articles/nn.4361#ref-CR51" id="ref-link-section-d24703174e897">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Niazy, R.K., Xie, J., Miller, K., Beckmann, C.F. &amp; Smith, S.M. Spectral characteristics of resting state networks. Prog. Brain Res. 193, 259276 (2011)." href="/articles/nn.4361#ref-CR52" id="ref-link-section-d24703174e900">52</a></sup>, which would be better captured with faster scans. The HCP acquired fMRI data at TR = 0.72 s for 3 T and 1 s for 7 T using MRI pulse sequences that acquire many brain slices at a time ('multiband' or 'simultaneous multislice' echo planar imaging (EPI) sequences; <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 3</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig9">Supplementary Fig. 4</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Ugurbil, K. et al. Pushing spatial and temporal resolution for functional and diffusion MRI in the Human Connectome Project. Neuroimage 80, 80104 (2013)." href="/articles/nn.4361#ref-CR5" id="ref-link-section-d24703174e911">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Setsompop, K. et al. Blipped-controlled aliasing in parallel imaging for simultaneous multislice echo planar imaging with reduced g-factor penalty. Magn. Reson. Med. 67, 12101224 (2012). This study introduced a slice gradient blip technique for simultaneous multislice (multiband) echo-planar imaging to significantly reduce image artifacts in unaliasing of simultaneously acquired slices, particularly for high multiband factors. This technique is adopted in the HCP pulse sequences for fMRI and dMRI." href="/articles/nn.4361#ref-CR44" id="ref-link-section-d24703174e914">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Auerbach, E.J., Xu, J., Yacoub, E., Moeller, S. &amp; Ugurbil, K. Multiband accelerated spin-echo echo planar imaging with reduced peak RF power using time-shifted RF pulses. Magn. Reson. Med. 69, 12611267 (2013)." href="/articles/nn.4361#ref-CR53" id="ref-link-section-d24703174e917">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Moeller, S. et al. Multiband multislice GE-EPI at 7 tesla, with 16-fold acceleration using partial parallel imaging with application to high spatial and temporal whole-brain fMRI. Magn. Reson. Med. 63, 11441153 (2010). This study demonstrated the feasibility of improving the spatial and temporal resolution of whole-brain EPI-based fMRI by slice multiplexing, in which multiple slices are excited and acquired simultaneously, thus sharing a common contrast and common spatial encoding periods. Multiband EPI is a key strength of the HCP fMRI and dMRI pulse sequences." href="/articles/nn.4361#ref-CR54" id="ref-link-section-d24703174e920">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Xu, J. et al. Evaluation of slice accelerations using multiband echo planar imaging at 3 T. Neuroimage 83, 9911001 (2013)." href="/articles/nn.4361#ref-CR55" id="ref-link-section-d24703174e923">55</a></sup>. For diffusion imaging, using these sequences to reduce the TR allows more images to be acquired with diffusion weightings at different strengths and along a larger number of unique spatial directions in the same amount of time. The resulting higher angular resolution and effective SNR enables multiple fiber populations to be more robustly identified in each imaging voxel, helping to resolve crossing fiber pathways and improve tractography<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Ugurbil, K. et al. Pushing spatial and temporal resolution for functional and diffusion MRI in the Human Connectome Project. Neuroimage 80, 80104 (2013)." href="/articles/nn.4361#ref-CR5" id="ref-link-section-d24703174e927">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Setsompop, K. et al. Pushing the limits of in vivo diffusion MRI for the Human Connectome Project. Neuroimage 80, 220233 (2013)." href="/articles/nn.4361#ref-CR45" id="ref-link-section-d24703174e930">45</a></sup>.</p><p>The HCP used 1 h of scan time for each of three modalities: task fMRI, resting-state fMRI (rfMRI), and diffusion imaging. This enabled data for seven fMRI tasks to be acquired, covering multiple cognitive domains and involving most of the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Barch, D.M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. Neuroimage 80, 169189 (2013)." href="/articles/nn.4361#ref-CR29" id="ref-link-section-d24703174e937">29</a></sup> (<a href="http://protocols.humanconnectome.org/HCP/3T/task-fMRI-protocol-details.html">http://protocols.humanconnectome.org/HCP/3T/task-fMRI-protocol-details.html</a>), along with four 15-min resting-state fMRI runs (helping to provide more stable connectivity estimates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Laumann, T.O. et al. Functional system and areal organization of a highly sampled individual human brain. Neuron 87, 657670 (2015)." href="/articles/nn.4361#ref-CR56" id="ref-link-section-d24703174e948">56</a></sup>) and 270 different diffusion measures (90 unique diffusion directions for each of three different diffusion weighting strengths; i.e., 'shells'), enabling better fiber orientation and tissue microstructure modeling at high resolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Jbabdi, S., Sotiropoulos, S.N., Savio, A.M., Graa, M. &amp; Behrens, T.E. Model-based analysis of multishell diffusion MR data for tractography: how to get over fitting problems. Magn. Reson. Med. 68, 18461855 (2012)." href="/articles/nn.4361#ref-CR57" id="ref-link-section-d24703174e952">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Jeurissen, B., Tournier, J.D., Dhollander, T., Connelly, A. &amp; Sijbers, J. Multi-tissue constrained spherical deconvolution for improved analysis of multi-shell diffusion MRI data. Neuroimage 103, 411426 (2014)." href="/articles/nn.4361#ref-CR58" id="ref-link-section-d24703174e955">58</a></sup>. See <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 4</a> for particulars on various HCP-Short protocols optimized for subjects who are less tolerant of long scans than are healthy young adults.</p><h3 class="c-article__sub-heading" id="Sec6">Tenets 3 and 4: optimizing the precision of data analysis</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec7">Aligning all MRI modalities within the subject's physical space.</h4><p>A key early task of preprocessing is to align all MRI modalities so that every image volume is in the subject's undistorted physical space. Head movements occur in the scanner, both between acquisition of each dataset and during acquisition. Because head movement during T1w and T2w images causes uncorrectable blurring, the HCP acquired at least two pairs of T1w and T2w images and used all scans that were rated as good or excellent in overall quality<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Marcus, D.S. et al. Human Connectome Project informatics: quality control, database services, and data visualization. Neuroimage 80, 202219 (2013)." href="/articles/nn.4361#ref-CR12" id="ref-link-section-d24703174e975">12</a></sup> (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 5</a> for further discussion of this issue and potential solutions). Spatial effects of movement during fMRI or dMRI acquisitions can be corrected after the fact using image registration to realign the images; however, movement also induces temporal variations of image intensities that must be corrected through other means described below. Movement between images of different modalities is corrected with advanced registration algorithms that use knowledge of brain tissue boundaries and the expected image intensity gradients across those boundaries to align images<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Greve, D.N. &amp; Fischl, B. Accurate and robust brain image alignment using boundary-based registration. Neuroimage 48, 6372 (2009)." href="/articles/nn.4361#ref-CR59" id="ref-link-section-d24703174e982">59</a></sup>.</p><p>Ideally, image registration within each subject would entail simple translations or rotations, but MRI images have many forms of spatial distortion and must be unwarped before images faithfully represent the individual brain's real physical dimensions and thus can be precisely aligned. These distortions all derive from imperfections in the scanner's encoding of physical space using perturbations (linear gradients) of the main magnetic field. All MRI images are slightly distorted by small nonlinearities in the space-encoding gradients that are correctible using information measured and specified by the scanner manufacturer (gradient distortion; <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 6</a>). In addition, placement of the subject's head in the scanner induces idiosyncratic variations in the magnetic field (field inhomogeneities), particularly near the air-filled spaces in the head (sinuses and ear canals), which can be measured using the auxiliary field-map scan mentioned above. Correcting these distortions with a field map is critical for accurately aligning data across modalities within each subject, although traditionally this scan is often omitted, leading to poor alignment in some brain regions. Scanner vendors provide a standard field map scan (based on gradient echoes), but the HCP developed an alternative 'spin echo EPI field map' that is faster, at least as accurate and also enables correction of fMRI image intensity biases (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 7</a>). These field map scans should exactly match the geometry and distortion properties of the functional imaging scans. Diffusion imaging scans have additional 'eddy current' distortions caused by switching on and off the strong magnetic field gradients used to generate diffusion weighting. The HCP developed a new algorithm for correcting eddy current distortions that outperforms previous methods and allows quality control and outlier detection (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 8</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Andersson, J.L. &amp; Sotiropoulos, S.N. Non-parametric representation and prediction of single- and multi-shell diffusion-weighted MRI data using Gaussian processes. Neuroimage 122, 166176 (2015)." href="/articles/nn.4361#ref-CR60" id="ref-link-section-d24703174e998">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Andersson, J.L. &amp; Sotiropoulos, S.N. An integrated approach to correction for off-resonance effects and subject movement in diffusion MR imaging. Neuroimage 125, 10631078 (2016)." href="/articles/nn.4361#ref-CR61" id="ref-link-section-d24703174e1001">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Andersson, J.L.R., Graham, M.S., Zsoldos, E. &amp; Sotiropoulos, S.N. Incorporating outlier detection and replacement into a non-parameteric framework for movement and distortion correction of diffusion MR images. Neuroimage j.neuroimage.2016.06.058 (2016)." href="/articles/nn.4361#ref-CR62" id="ref-link-section-d24703174e1004">62</a></sup>. With field maps, advanced distortion correction methods and anatomically informed registration algorithms, the HCP's spatial preprocessing pipelines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e1008">6</a></sup> precisely align the T1w, T2w, fMRI and diffusion data to the subject's physical space while minimizing interpolation-induced blurring by combining all spatial transformations in a single spline interpolation.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec8">Cleaning time-dependent artifacts from MRI time series data.</h4><p>In functional imaging, many processes other than the BOLD-related fluctuations of interest cause time-dependent intensity fluctuations. This structured temporal noise is especially problematic for resting-state functional connectivity because it uses correlations of noisy signals in different parts of the brain rather than correlation of a predefined noiseless task design with noisy data. Temporal noise can either increase or reduce functional connectivity artifactually and lead to incorrect conclusions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Power, J.D., Schlaggar, B.L. &amp; Petersen, S.E. Recent progress and outstanding issues in motion correction in resting state fMRI. Neuroimage 105, 536551 (2015)." href="/articles/nn.4361#ref-CR63" id="ref-link-section-d24703174e1020">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Power, J.D., Barnes, K.A., Snyder, A.Z., Schlaggar, B.L. &amp; Petersen, S.E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. Neuroimage 59, 21422154 (2012)." href="/articles/nn.4361#ref-CR64" id="ref-link-section-d24703174e1023">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Satterthwaite, T.D. et al. Impact of in-scanner head motion on multiple measures of functional connectivity: relevance for studies of neurodevelopment in youth. Neuroimage 60, 623632 (2012)." href="/articles/nn.4361#ref-CR65" id="ref-link-section-d24703174e1026">65</a></sup>. Major sources of structured temporal noise include subject movement, subject physiology and the physics of MRI. It is useful to consider the fMRI time series as containing fluctuations arising from (i) the BOLD signals of interest, (ii) structured temporal noise and (iii) unstructured random Gaussian temporal noise<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Marcus, D.S. et al. Human Connectome Project informatics: quality control, database services, and data visualization. Neuroimage 80, 202219 (2013)." href="/articles/nn.4361#ref-CR12" id="ref-link-section-d24703174e1030">12</a></sup>. Temporal cleanup aims to remove temporal noise as selectively as possible while minimizing the impact on the BOLD signals of interest. The HCP devised a method based on independent components analysis (ICA)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Beckmann, C.F. &amp; Smith, S.M. Probabilistic independent component analysis for functional magnetic resonance imaging. IEEE Trans. Med. Imaging 23, 137152 (2004)." href="/articles/nn.4361#ref-CR66" id="ref-link-section-d24703174e1034">66</a></sup>, which splits the fMRI time series into structured components and unstructured noise, representing all of the temporal variance. The structured components are then categorized by a machine learning classifier (FIX) into signal and noise<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Griffanti, L. et al. ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging. Neuroimage 95, 232247 (2014)." href="/articles/nn.4361#ref-CR7" id="ref-link-section-d24703174e1038">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Salimi-Khorshidi, G. et. al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. Neuroimage 90, 449468 (2014). This study demonstrated a method for automatically cleaning spatially specific structured noise out of fMRI data using an independent component analysis (ICA) decomposition and machine learning classification of the resulting components into signal and noise." href="/articles/nn.4361#ref-CR8" id="ref-link-section-d24703174e1041">8</a></sup>, and the noise components are removed by regressing them out of the data. For HCP data, the ICA+FIX approach removes spatially specific (i.e., affecting only part of the brain in a particular way) structured noise components and retains spatially specific BOLD signal components (i.e., resting-state networks) with better than 99% accuracy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Griffanti, L. et al. ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging. Neuroimage 95, 232247 (2014)." href="/articles/nn.4361#ref-CR7" id="ref-link-section-d24703174e1045">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Salimi-Khorshidi, G. et. al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. Neuroimage 90, 449468 (2014). This study demonstrated a method for automatically cleaning spatially specific structured noise out of fMRI data using an independent component analysis (ICA) decomposition and machine learning classification of the resulting components into signal and noise." href="/articles/nn.4361#ref-CR8" id="ref-link-section-d24703174e1048">8</a></sup>. In contrast, traditional approaches, which include bandpass filtering, white matter and cerebrospinal fluid regression, and censoring of high-motion frames<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Power, J.D., Barnes, K.A., Snyder, A.Z., Schlaggar, B.L. &amp; Petersen, S.E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. Neuroimage 59, 21422154 (2012)." href="/articles/nn.4361#ref-CR64" id="ref-link-section-d24703174e1053">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Satterthwaite, T.D. et al. Impact of in-scanner head motion on multiple measures of functional connectivity: relevance for studies of neurodevelopment in youth. Neuroimage 60, 623632 (2012)." href="/articles/nn.4361#ref-CR65" id="ref-link-section-d24703174e1056">65</a></sup>, are not as selective at targeting structured noise while preserving BOLD signal. However, a controversy remains about 'global' fluctuations in fMRI (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/nn.4361#Sec9">Box 1</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 9</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig10">Supplementary Figs. 5</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">6</a>), as the spatial ICA+FIX approach is not designed to separate these global fluctuations in the average fMRI signal into signal and noise components. For dMRI, subject motion and physiology can cause signal loss. Such outliers can be detected using a new approach that builds a model of the diffusion signal from all of the data, allowing artifactual measurements to be identified and replaced with estimates of what the signal should have been (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 8</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Andersson, J.L. &amp; Sotiropoulos, S.N. Non-parametric representation and prediction of single- and multi-shell diffusion-weighted MRI data using Gaussian processes. Neuroimage 122, 166176 (2015)." href="/articles/nn.4361#ref-CR60" id="ref-link-section-d24703174e1076">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Andersson, J.L. &amp; Sotiropoulos, S.N. An integrated approach to correction for off-resonance effects and subject movement in diffusion MR imaging. Neuroimage 125, 10631078 (2016)." href="/articles/nn.4361#ref-CR61" id="ref-link-section-d24703174e1079">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Andersson, J.L.R., Graham, M.S., Zsoldos, E. &amp; Sotiropoulos, S.N. Incorporating outlier detection and replacement into a non-parameteric framework for movement and distortion correction of diffusion MR images. Neuroimage j.neuroimage.2016.06.058 (2016)." href="/articles/nn.4361#ref-CR62" id="ref-link-section-d24703174e1082">62</a></sup>.</p><p>The traditional approach to reducing unstructured Gaussian noise involves spatial and temporal smoothing, which unfortunately affect both signal and noise unselectively. A common problem in traditional neuroimaging is that smoothing mixes the gray matter functional signals of a given brain area with signals from white matter, cerebrospinal fluid and nearby brain areas, markedly reducing the spatial fidelity of neuroimaging data. Parcellation (tenet 6) enables averaging of data without mixing signal across different tissue types or between different brain areas to the extent that the parcellation is neurobiologically accurate. Thus, parcellation acts as a neuroanatomically informed method of nonlinear smoothing. Another option is to selectively smooth unstructured noise more than structured BOLD signal (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 10</a>). This reduces certain artifacts (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig11">Supplementary Fig. 7</a>) and enhances SNR without concomitant spatial blurring and without invoking a brain parcellation.</p><div class="c-article-box" data-expandable-box-container="true"><div class="c-article-box__container" data-expandable-box="true" aria-hidden="true" id="box-Sec9"><h3 class="c-article-box__container-title u-h3 js-expandable-title" id="Sec9">Box 1: Global fMRI temporal fluctuations</h3><div class="c-article-box__content"><p>How best to handle global signals and artifacts present in fMRI data is a hotly debated topic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Power, J.D., Schlaggar, B.L. &amp; Petersen, S.E. Recent progress and outstanding issues in motion correction in resting state fMRI. Neuroimage 105, 536551 (2015)." href="/articles/nn.4361#ref-CR63" id="ref-link-section-d24703174e1102">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Power, J.D., Barnes, K.A., Snyder, A.Z., Schlaggar, B.L. &amp; Petersen, S.E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. Neuroimage 59, 21422154 (2012)." href="/articles/nn.4361#ref-CR64" id="ref-link-section-d24703174e1105">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 106" title="Murphy, K., Birn, R.M., Handwerker, D.A., Jones, T.B. &amp; Bandettini, P.A. The impact of global signal regression on resting state correlations: are anti-correlated networks introduced? Neuroimage 44, 893905 (2009)." href="/articles/nn.4361#ref-CR106" id="ref-link-section-d24703174e1108">106</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 107" title="Power, J.D. et al. Methods to detect, characterize, and remove motion artifact in resting state fMRI. Neuroimage 84, 320341 (2014)." href="/articles/nn.4361#ref-CR107" id="ref-link-section-d24703174e1111">107</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 108" title="Saad, Z.S. et al. Correcting brain-wide correlation differences in resting-state FMRI. Brain Connect. 3, 339352 (2013)." href="/articles/nn.4361#ref-CR108" id="ref-link-section-d24703174e1114">108</a></sup>. The core phenomenon involves slow and spatially widespread fluctuations in the fMRI time series. The average time series across the brain includes both neural signal (for example, the average of all the resting-state networks) and structured noise (for example, physiological noise), and the debate centers on whether the average time series should be regressed out of the data to remove the effects of global noise<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Power, J.D., Schlaggar, B.L. &amp; Petersen, S.E. Recent progress and outstanding issues in motion correction in resting state fMRI. Neuroimage 105, 536551 (2015)." href="/articles/nn.4361#ref-CR63" id="ref-link-section-d24703174e1118">63</a></sup> or whether it should be preserved to avoid distorting neural signals by removing their average. Additive global noise will by definition artifactually increase 'full' correlations across the brain (especially problematic if it differs across groups), but removal of a global neural signal will artifactually decrease correlations across the brain and, more problematically, distort the pattern of correlations for particularly large or strong networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Power, J.D., Schlaggar, B.L. &amp; Petersen, S.E. Recent progress and outstanding issues in motion correction in resting state fMRI. Neuroimage 105, 536551 (2015)." href="/articles/nn.4361#ref-CR63" id="ref-link-section-d24703174e1122">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 109" title="Gotts, S.J. et al. The perils of global signal regression for group comparisons: a case study of Autism Spectrum Disorders. Front. Hum. Neurosci. 7, 356 (2013)." href="/articles/nn.4361#ref-CR109" id="ref-link-section-d24703174e1125">109</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 110" title="Saad, Z.S. et al. Trouble at rest: how correlation patterns and group differences become distorted after global signal regression. Brain Connect. 2, 2532 (2012)." href="/articles/nn.4361#ref-CR110" id="ref-link-section-d24703174e1128">110</a></sup></p><p>What is needed to get past the current impasse is a method that selectively separates global signal from global noise. Though no such method is yet available, we offer several observations about global fMRI fluctuations. (i) In HCP data, the global signal (after ICA+FIX cleanup) is primarily a gray matter signal, with a near-zero amplitude in white matter voxels and low-level anti-correlation with cerebrospinal fluid (Supplementary Fig. 5). (ii) Within gray matter, the mean time course has much higher amplitude in sensory regions (for example, visual, auditory, vestibular and somatosensory cortex, plus the lateral geniculate nucleus and other thalamic nuclei) and is lower in the cerebellum and many neocortical cognitive regions (Supplementary Fig. 6). Thus, the global signal is not uniform across the brain, or even across gray matter, and these observations constrain the possible sources of the global fluctuations, as follows:</p><p><b>Physiological effects.</b> Substantial fMRI time series variance is attributable to physiological variables, including rate and depth of breathing, heart rate and end tidal partial pressure of expired CO<sub>2</sub> (refs. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 111" title="Birn, R.M. The role of physiological noise in resting-state functional connectivity. Neuroimage 62, 864870 (2012)." href="/articles/nn.4361#ref-CR111" id="ref-link-section-d24703174e1140">111</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 112" title="Chang, C. &amp; Glover, G.H. Relationship between respiration, end-tidal CO2, and BOLD signals in resting-state fMRI. Neuroimage 47, 13811393 (2009)." href="/articles/nn.4361#ref-CR112" id="ref-link-section-d24703174e1143">112</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 113" title="Golestani, A.M., Chang, C., Kwinta, J.B., Khatamian, Y.B. &amp; Jean Chen, J. Mapping the end-tidal CO2 response function in the resting-state BOLD fMRI signal: spatial specificity, test-retest reliability and effect of fMRI sampling rate. Neuroimage 104, 266277 (2015)." href="/articles/nn.4361#ref-CR113" id="ref-link-section-d24703174e1146">113</a>). Indeed, any physiological process that affects global perfusion pressure of the brain (i.e., the above three or even yawning) may transiently alter the amount of deoxyhemoglobin in the brain and contribute to global, physiologically driven artifactual fluctuations of the BOLD signal. Different brain areas and tissue types may be differentially sensitive to such changes in perfusion pressure because of differing metabolic rates, which may explain regional differences in global signal amplitude.</p><p><b>Head movements (direct, biophysical).</b> Transient head movements are sometimes followed by global fluctuations in the fMRI signal, leading to suggestions that head movements could directly cause these fluctuations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Power, J.D., Schlaggar, B.L. &amp; Petersen, S.E. Recent progress and outstanding issues in motion correction in resting state fMRI. Neuroimage 105, 536551 (2015)." href="/articles/nn.4361#ref-CR63" id="ref-link-section-d24703174e1155">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Satterthwaite, T.D. et al. Impact of in-scanner head motion on multiple measures of functional connectivity: relevance for studies of neurodevelopment in youth. Neuroimage 60, 623632 (2012)." href="/articles/nn.4361#ref-CR65" id="ref-link-section-d24703174e1158">65</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 107" title="Power, J.D. et al. Methods to detect, characterize, and remove motion artifact in resting state fMRI. Neuroimage 84, 320341 (2014)." href="/articles/nn.4361#ref-CR107" id="ref-link-section-d24703174e1161">107</a></sup>. However, head movements are unlikely to cause global fluctuations directly, given the lack of a plausible biophysical mechanism for how head movement might induce tissue-specific and brain-area-specific effects on the fMRI time series, let alone globally uniform changes in signal intensity short of complete head coil exit and reentry (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 9</a>).</p><p><b>Head movements (indirect, neural).</b> Head movements may induce neural responses in somatosensory, vestibular and/or visual systems that may in turn modulate wider brain activity in a pattern very consistent with that mentioned above.</p><p><b>Widespread and/or high amplitude RSNs, average of the RSNs.</b> As discussed above, a portion of the global fluctuations will represent the average of the RSNs with particular weighting toward more widespread or stronger networks. It will be important to separate global physiological effects from spatially widespread neural effects (using either extrinsic physiological monitoring or intrinsic fMRI-data-driven means that have yet to be developed). Short of a complete solution to the problem of global fluctuations, we note that some approaches to analysis (such as the use of partial correlation when estimating functional connectivity and multivariate measures in general) are much more immune to the effects of global noise fluctuations than others (for example, full-correlation functional connectivity and univariate measures in general)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 114" title="Smith, S.M. et al. Network modelling methods for FMRI. Neuroimage 54, 875891 (2011)." href="/articles/nn.4361#ref-CR114" id="ref-link-section-d24703174e1179">114</a></sup>.</p></div></div></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec10">Preserving precision from voxels to CIFTI grayordinates.</h4><p>It is vital that the preprocessing and analysis techniques used on high-resolution HCP-style data maintain this hard-won spatial resolution. Besides removing distortions and accurately aligning data across modalities within each subject, the analysis must put the data into a framework that enables accurate cross-subject and cross-study comparisons (tenet 5, below). The traditional neuroimaging paradigm represents spatial data in a regular 3D array of voxels (an image volume) that reflects the output of the MRI image reconstruction process. Because the cerebral cortex has a folded, sheet-like geometry, it is advantageous to visualize and analyze cortical data on surface mesh vertices instead of volume voxels<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Sereno, M.I. et al. Borders of multiple visual areas in humans revealed by functional magnetic resonance imaging. Science 268, 889893 (1995)." href="/articles/nn.4361#ref-CR16" id="ref-link-section-d24703174e1192">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Fischl, B., Sereno, M.I. &amp; Dale, A.M. Cortical surface-based analysis. II: inflation, flattening, and a surface-based coordinate system. Neuroimage 9, 195207 (1999)." href="/articles/nn.4361#ref-CR26" id="ref-link-section-d24703174e1195">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Fischl, B., Sereno, M.I., Tootell, R.B. &amp; Dale, A.M. High-resolution intersubject averaging and a coordinate system for the cortical surface. Hum. Brain Mapp. 8, 272284 (1999). This study introduced an automated method for aligning cortical surfaces from individuals to a surface-based atlas using cortical folding to constrain the registration." href="/articles/nn.4361#ref-CR27" id="ref-link-section-d24703174e1198">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Van Essen, D.C., Glasser, M.F., Dierker, D.L., Harwell, J. &amp; Coalson, T. Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases. Cereb. Cortex 22, 22412262 (2012)." href="/articles/nn.4361#ref-CR39" id="ref-link-section-d24703174e1201">39</a></sup>. However, subcortical gray matter structures are mostly globular in shape and remain best represented using volume voxels. To represent the inherently dual geometry of the brain, the HCP developed the CIFTI file format<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e1205">6</a></sup>, which integrates surface vertices and subcortical voxels into a single file and represents each major gray matter structure using the appropriate geometry (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig1">Fig. 1</a>). CIFTI gray matter vertices and voxels are called grayordinates. Analyses focusing only on grayordinates (for example, fMRI) dramatically reduce file sizes and computational burdens (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig1">Fig. 1</a>). For dMRI, surface representations allow more accurate anatomical constraints in tractography (for example, the white matter surface can be used as the cortical destination for tractography). The final common outputs of the HCP preprocessing and analysis pipelines are data sets in the CIFTI standard space<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e1215">6</a></sup>. Connectome Workbench natively supports CIFTI, and other tools (for example, FSL, FreeSurfer, FieldTrip and nibabel) are currently implementing CIFTI compatibility. In particular, FSL's PALM software enables permutation-based statistical inference on CIFTI data sets (<a href="http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PALM">http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PALM</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Winkler, A.M., Webster, M.A., Vidaurre, D., Nichols, T.E. &amp; Smith, S.M. Multi-level block permutation. Neuroimage 123, 253268 (2015)." href="/articles/nn.4361#ref-CR67" id="ref-link-section-d24703174e1227">67</a></sup>. The CIFTI grayordinates analysis framework allows each brain structure to be registered using the approach best suited to it (tenet 5).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Combined representation of cortical surface vertices and subcortical voxels in the CIFTI grayordinates standard space."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 1: Combined representation of cortical surface vertices and subcortical voxels in the CIFTI grayordinates standard space.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/nn.4361/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig1_HTML.jpg?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig1_HTML.jpg" alt="figure 1" loading="lazy" width="675" height="1385"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>Left and right cerebral cortices contribute about 30,000 surface vertices each (fewer than the 32,000 vertex standard meshes for each hemisphere because the noncortical medial wall is not included). Additionally, 19 subcortical gray matter structures combine to contribute about 30,000 volume voxels. In total, there are 91,282 grayordinates corresponding to all of the gray matter sampled at a 2-mm average vertex spacing on the surface and as 2-mm voxels subcortically. The HCP's minimal preprocessing pipelines ensure that each subject has 91,282 aligned grayordinates, thereby facilitating cross-subject comparisons of data within this coordinate system. This entails a more than twofold reduction in file size relative to the &gt;200,000 voxels needed for an equivalent 2-mm isotropic volume representation. For the 1.6-mm 7 T HCP data, we have developed a 1.6-mm standard grayordinate space with 170,494 grayordinates using 59,000 vertex surface meshes. Adapted from ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013). This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format." href="/articles/nn.4361#ref-CR6" id="ref-link-section-d24703174e1242">6</a>, Elsevier.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/nn.4361/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec11">Tenet 5: accurately aligning brain areas across subjects and studies</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec12">Cross-subject comparisons.</h4><p>Analyses that combine neuroimaging data obtained from different individuals benefit greatly from accurately aligning corresponding brain areas so that like is compared with like<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Turner, R. &amp; Geyer, S. Comparing like with like: the power of knowing where you are. Brain Connect. 4, 547557 (2014). The authors emphasize the importance of high-quality data acquisition, careful surface-based analysis in individual subjects and localization in relation to the best available cortical parcellation. Hence they presage many of the points made in the present article." href="/articles/nn.4361#ref-CR15" id="ref-link-section-d24703174e1267">15</a></sup>. Alignment of subcortical and cortical structures is best achieved through differing means. Subcortical structures are less variable in size, shape and position across subjects, and reasonable alignment is achieved by nonlinear 3D registration to a standard template as in the traditional paradigm. The HCP currently uses FSL's FNIRT for nonlinear registration of T1w images to the MNI standard volumetric space, but registrations that also incorporate dMRI fiber orientations in the white matter may improve alignment of subcortical white matter fiber tracts (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 11</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Zhang, H. et al. High-dimensional spatial normalization of diffusion tensor images improves the detection of white matter differences: an example study using amyotrophic lateral sclerosis. IEEE Trans. Med. Imaging 26, 15851597 (2007)." href="/articles/nn.4361#ref-CR68" id="ref-link-section-d24703174e1274">68</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Zhang, H., Yushkevich, P.A., Alexander, D.C. &amp; Gee, J.C. Deformable registration of diffusion tensor MR images with explicit orientation optimization. Med. Image Anal. 10, 764785 (2006)." href="/articles/nn.4361#ref-CR69" id="ref-link-section-d24703174e1277">69</a></sup>.</p><p>For human cerebral cortex, the complexity and variability of folding patterns and of areas relative to folds<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Van Essen, D.C., Glasser, M.F., Dierker, D.L., Harwell, J. &amp; Coalson, T. Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases. Cereb. Cortex 22, 22412262 (2012)." href="/articles/nn.4361#ref-CR39" id="ref-link-section-d24703174e1284">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Amunts, K., Malikovic, A., Mohlberg, H., Schormann, T. &amp; Zilles, K. Brodmann's areas 17 and 18 brought into stereotaxic space-where and how variable? Neuroimage 11, 6684 (2000). This study (and many others from the same laboratory) used objective observer-independent methods to identify cytoarchitectonic areas in post-mortem human brains." href="/articles/nn.4361#ref-CR70" id="ref-link-section-d24703174e1287">70</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Eickhoff, S.B. et al. A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. Neuroimage 25, 13251335 (2005)." href="/articles/nn.4361#ref-CR71" id="ref-link-section-d24703174e1290">71</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Van Essen, D.C. A population-average, landmark- and surface-based (PALS) atlas of human cerebral cortex. Neuroimage 28, 635662 (2005)." href="/articles/nn.4361#ref-CR72" id="ref-link-section-d24703174e1293">72</a></sup> presents a profoundly challenging, ill-posed problem for nonlinear 3D registration. Though alternative approaches such as hyperalignment (where alignment is not spatially constrained) are useful in some contexts, many neuroscientific questions are best addressed by aligning subjects to a common space. Registering cortical surfaces rather than brain volumes enables better alignment because accurate cortical segmentation of each subject reduces the dimensionality of the registration problem from 3D to 2D: alignment need only occur along the cortical sheet, as opposed to a 3D alignment of the folded sheet itself<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Fischl, B., Sereno, M.I., Tootell, R.B. &amp; Dale, A.M. High-resolution intersubject averaging and a coordinate system for the cortical surface. Hum. Brain Mapp. 8, 272284 (1999). This study introduced an automated method for aligning cortical surfaces from individuals to a surface-based atlas using cortical folding to constrain the registration." href="/articles/nn.4361#ref-CR27" id="ref-link-section-d24703174e1297">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Van Essen, D.C., Glasser, M.F., Dierker, D.L., Harwell, J. &amp; Coalson, T. Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases. Cereb. Cortex 22, 22412262 (2012)." href="/articles/nn.4361#ref-CR39" id="ref-link-section-d24703174e1300">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Anticevic, A. et al. Comparing surface-based and volume-based analyses of functional neuroimaging data in patients with schizophrenia. Neuroimage 41, 835848 (2008)." href="/articles/nn.4361#ref-CR73" id="ref-link-section-d24703174e1303">73</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Fischl, B. et al. Cortical folding patterns and predicting cytoarchitecture. Cereb. Cortex 18, 19731980 (2008). The authors used folding-based alignment to register cytoarchitectonic areas to a surface-based atlas. In the resultant probabilistic maps, alignment was high for early sensory areas but much lower for 'higher' cortical areas, reflecting regional differences in the correlation between areal boundaries and cortical folding." href="/articles/nn.4361#ref-CR74" id="ref-link-section-d24703174e1306">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Frost, M.A. &amp; Goebel, R. Measuring structural-functional correspondence: spatial variability of specialised brain regions after macro-anatomical alignment. Neuroimage 59, 13691381 (2012)." href="/articles/nn.4361#ref-CR75" id="ref-link-section-d24703174e1309">75</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Smith, S.M. et al. Resting-state fMRI in the Human Connectome Project. Neuroimage 80, 144168 (2013)." href="/articles/nn.4361#ref-CR76" id="ref-link-section-d24703174e1312">76</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Tucholka, A., Fritsch, V., Poline, J.B. &amp; Thirion, B. An empirical comparison of surface-based and volume-based group studies in neuroimaging. Neuroimage 63, 14431453 (2012)." href="/articles/nn.4361#ref-CR77" id="ref-link-section-d24703174e1316">77</a></sup>.</p><p>Though surface-based registration of the cerebral cortex to an atlas is not a part of the traditional paradigm, it has been adopted by a growing number of investigators. Most use surface registration based on cortical folding patterns<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Fischl, B., Sereno, M.I., Tootell, R.B. &amp; Dale, A.M. High-resolution intersubject averaging and a coordinate system for the cortical surface. Hum. Brain Mapp. 8, 272284 (1999). This study introduced an automated method for aligning cortical surfaces from individuals to a surface-based atlas using cortical folding to constrain the registration." href="/articles/nn.4361#ref-CR27" id="ref-link-section-d24703174e1323">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Van Essen, D.C., Glasser, M.F., Dierker, D.L., Harwell, J. &amp; Coalson, T. Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases. Cereb. Cortex 22, 22412262 (2012)." href="/articles/nn.4361#ref-CR39" id="ref-link-section-d24703174e1326">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Anticevic, A. et al. Comparing surface-based and volume-based analyses of functional neuroimaging data in patients with schizophrenia. Neuroimage 41, 835848 (2008)." href="/articles/nn.4361#ref-CR73" id="ref-link-section-d24703174e1329">73</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Fischl, B. et al. Cortical folding patterns and predicting cytoarchitecture. Cereb. Cortex 18, 19731980 (2008). The authors used folding-based alignment to register cytoarchitectonic areas to a surface-based atlas. In the resultant probabilistic maps, alignment was high for early sensory areas but much lower for 'higher' cortical areas, reflecting regional differences in the correlation between areal boundaries and cortical folding." href="/articles/nn.4361#ref-CR74" id="ref-link-section-d24703174e1332">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Frost, M.A. &amp; Goebel, R. Measuring structural-functional correspondence: spatial variability of specialised brain regions after macro-anatomical alignment. Neuroimage 59, 13691381 (2012)." href="/articles/nn.4361#ref-CR75" id="ref-link-section-d24703174e1335">75</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Smith, S.M. et al. Resting-state fMRI in the Human Connectome Project. Neuroimage 80, 144168 (2013)." href="/articles/nn.4361#ref-CR76" id="ref-link-section-d24703174e1338">76</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Tucholka, A., Fritsch, V., Poline, J.B. &amp; Thirion, B. An empirical comparison of surface-based and volume-based group studies in neuroimaging. Neuroimage 63, 14431453 (2012)." href="/articles/nn.4361#ref-CR77" id="ref-link-section-d24703174e1342">77</a></sup>, which achieves geographic correspondence in regions where folding patterns are consistent across individuals and functional correspondence in regions where areal boundaries are consistently related to folding. Because cortical folding patterns are highly variable in many regions of cortex and their relationships to cortical areas are similarly variable, folding patterns alone are not enough to achieve alignment of (functional) cortical areas across most of the cerebral cortex. Registration needs instead to be driven by features that are more closely tied to cortical areas, such as architecture, function, connectivity and topography<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Felleman, D.J. &amp; Van Essen, D.C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 147 (1991)." href="/articles/nn.4361#ref-CR78" id="ref-link-section-d24703174e1346">78</a></sup>. Such areal-feature-based cortical surface registration can dramatically improve the sharpness of group average maps and enhance group statistics, as it largely compensates for differences across subjects in cortical areal size, shape and position. Indeed, alignment based on myelin maps, resting-state network maps and visuotopic connectivity maps yields sharper task fMRI contrast maps with higher statistics, even though the task fMRI data are not used in the registration (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig2">Fig. 2</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e1353">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Robinson, E.C. et al. MSM: a new flexible framework for multimodal surface matching. Neuroimage 100, 414426 (2014). This study introduced a powerful new method for aligning cortical surfaces from individuals to an atlas using areal features as well as folding patterns to constrain the alignment." href="/articles/nn.4361#ref-CR10" id="ref-link-section-d24703174e1356">10</a></sup>. Areal-feature aligned data were recently released by the HCP (dubbed MSMAll because they use the multimodal surface matching algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Robinson, E.C. et al. MSM: a new flexible framework for multimodal surface matching. Neuroimage 100, 414426 (2014). This study introduced a powerful new method for aligning cortical surfaces from individuals to an atlas using areal features as well as folding patterns to constrain the alignment." href="/articles/nn.4361#ref-CR10" id="ref-link-section-d24703174e1360">10</a></sup> and all of the useful modalities listed above). We believe that many studies will benefit from using areal-feature-based surface registration as a starting point, even if they choose to make final comparisons across subjects using other means; for example, hyperalignment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Haxby, J.V. et al. A common, high-dimensional model of the representational space in human ventral temporal cortex. Neuron 72, 404416 (2011)." href="/articles/nn.4361#ref-CR79" id="ref-link-section-d24703174e1365">79</a></sup> or individual subject parcellation (see below and ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e1368">9</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Improved intersubject registration using information based on areal features in addition to cortical folding."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 2: Improved intersubject registration using information based on areal features in addition to cortical folding.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/nn.4361/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig2_HTML.jpg?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig2_HTML.jpg" alt="figure 2" loading="lazy" width="685" height="515"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>Top row: group task-fMRI <i>z</i>-statistic maps (story vs. baseline contrast from the language task on the left and the two-back (2BK) vs. zero-back (0BK) contrast in the working memory task on the right) from 120 HCP subjects in the first and second quarter data releases after intersubject registration using the multimodal surface matching (MSM) method constrained only by folding (FreeSurfer's 'sulc' maps). Bottom row: sharper group task-fMRI maps and higher <i>z</i>-statistics obtained when using resting-state networks (RSNs) along with myelin maps to constrain the registration. Circled areas and arrows indicate regions of increased contrast in the areal-feature-based registration compared to the folding-based registration. Subject recruitment procedures and informed consent forms, including consent to share de-identified data, were approved by the Washington University institutional review board. Data at <a href="http://balsa.wustl.edu/97V4">http://balsa.wustl.edu/97V4</a>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/nn.4361/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec13">Cross-study comparisons.</h4><p>Traditionally, cross-study comparisons based on 3D stereotaxic coordinates were assumed to align if everyone used a common standard volume space (for example, MNI). These 3D coordinates were typically derived from thresholded statistical maps of the 'significant' activations (or connectivity seed locations). Though such standard space coordinates allow gross comparisons across studies at the gyral level (for example, ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Glasser, M.F. &amp; Rilling, J.K. DTI tractography of the human brain's language pathways. Cereb. Cortex 18, 24712482 (2008)." href="/articles/nn.4361#ref-CR80" id="ref-link-section-d24703174e1412">80</a>), questions of whether a particular cortical area was activated in particular task contrast typically reveal clouds of points whose distributions may be suggestive but are often inconclusive (for example, ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Glasser, M.F. &amp; Van Essen, D.C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 1159711616 (2011)." href="/articles/nn.4361#ref-CR20" id="ref-link-section-d24703174e1415">20</a>). Stereotaxic coordinates have high spatial uncertainty (relatively small changes in the thresholded statistical maps used to produce them can have large effects on the resulting coordinate locations; <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 12</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig12">Supplementary Fig. 8</a>), and they do not contain information about the spatial extent or boundaries of activation. Conclusive cross-study comparisons require assessment of the overlap of activated regions and whether their boundaries align or not. Cross-study comparisons also rely on the absence of any 'drift' in alignment between the studies (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 13</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig13">Supplementary Fig. 9</a>). When drift is eliminated and boundaries are compared, direct and conclusive assessments of cross-study agreement are possible (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig14">Supplementary Fig. 10</a> and see below).</p><h3 class="c-article__sub-heading" id="Sec14">Tenet 6: neuroanatomically accurate maps of brain areas</h3><p>Accurately subdividing the brain into its constituent parts (areas or parcels) has been a holy grail for brain cartographers and other neuroanatomists for more than a century<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Brodmann, K. Vergleichende Lokalisationslehre der Grosshirnrinde (Leipzig, 1909)." href="/articles/nn.4361#ref-CR81" id="ref-link-section-d24703174e1444">81</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Nieuwenhuys, R. The myeloarchitectonic studies on the human cerebral cortex of the Vogt-Vogt school, and their significance for the interpretation of functional neuroimaging data. Brain Struct. Funct. 218, 303352 (2013)." href="/articles/nn.4361#ref-CR82" id="ref-link-section-d24703174e1447">82</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Vogt, C. &amp; Vogt, O. Allgemeinere Ergebnisse unserer Hirnforschung. J. Psychol. Neurol. 25, 279468 (1919)." href="/articles/nn.4361#ref-CR83" id="ref-link-section-d24703174e1450">83</a></sup>. The goal is to reliably identify parcels that are distinct from one another in architecture, function, connectivity and/or topographic organization and are consistently identifiable in most or all individuals. For neuroimaging, parcellation serves to (i) clarify 'where we are' in the brain when describing results<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Turner, R. &amp; Geyer, S. Comparing like with like: the power of knowing where you are. Brain Connect. 4, 547557 (2014). The authors emphasize the importance of high-quality data acquisition, careful surface-based analysis in individual subjects and localization in relation to the best available cortical parcellation. Hence they presage many of the points made in the present article." href="/articles/nn.4361#ref-CR15" id="ref-link-section-d24703174e1454">15</a></sup>, (ii) facilitate comparisons across individuals and studies, and (iii) reduce the dimensionality of complex and noisy data sets while respecting neuroanatomically distinct boundaries. Parcellation can dramatically improve statistical sensitivity and power by averaging across neuroanatomically similar units of the brain, enhancing SNR and reducing the vast number of statistical comparisons often carried out in neuroimaging (circumventing problematic corrections for multiple comparisons that have affected a substantial fraction of the neuroimaging literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Eklund, A., Nichols, T.E. &amp; Knutsson, H. Cluster failure: why fMRI inferences for spatial extent have inflated false-positive rates. Proc. Natl. Acad. Sci. USA 28, 79007905 (2016)." href="/articles/nn.4361#ref-CR13" id="ref-link-section-d24703174e1458">13</a></sup>). As noted above, parcellation is an attractive alternative to traditional spatial smoothing for analyses that are interested in effects at the areal level (in practice, most studies that provide a standard space coordinate table).</p><p>A cortical areal parcellation should ideally have four key qualities. (i) It should be based on many (hundreds of) well-aligned subjects so that it represents the typical areal arrangement in the studied population (for example, healthy young adults for the HCP). (ii) It should reflect complementary and converging evidence from multiple modalities across the whole cerebral neocortex for completeness and for higher confidence in boundaries. Most parcellation efforts to date have relied instead on information from a single modality, such as architecture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Eickhoff, S.B. et al. A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. Neuroimage 25, 13251335 (2005)." href="/articles/nn.4361#ref-CR71" id="ref-link-section-d24703174e1465">71</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Nieuwenhuys, R., Broere, C.A. &amp; Cerliani, L. A new myeloarchitectonic map of the human neocortex based on data from the Vogt-Vogt school. Brain Struct. Funct. 220, 25512573 (2015)." href="/articles/nn.4361#ref-CR84" id="ref-link-section-d24703174e1468">84</a></sup>, retinotopy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Wang, L., Mruczek, R.E., Arcaro, M.J. &amp; Kastner, S. Probabilistic maps of visual topography in human cortex. Cereb. Cortex 25, 39113931 (2015)." href="/articles/nn.4361#ref-CR17" id="ref-link-section-d24703174e1472">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wandell, B.A. &amp; Winawer, J. Imaging retinotopic maps in the human brain. Vision Res. 51, 718737 (2011)." href="/articles/nn.4361#ref-CR30" id="ref-link-section-d24703174e1475">30</a></sup> or resting-state fMRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Gordon, E.M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288303 (2016)." href="/articles/nn.4361#ref-CR85" id="ref-link-section-d24703174e1479">85</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Power, J.D. et al. Functional network organization of the human brain. Neuron 72, 665678 (2011)." href="/articles/nn.4361#ref-CR86" id="ref-link-section-d24703174e1482">86</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Yeo, B.T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 11251165 (2011)." href="/articles/nn.4361#ref-CR87" id="ref-link-section-d24703174e1485">87</a></sup>, and often do not cover the whole hemisphere. (iii) It should reflect existing terminology for areas previously reported in the neuroanatomical literature and rational terminology for newly defined areas. (iv) It should be possible to automatically replicate the parcellation in individual subjects' data sets on the basis of multimodal areal fingerprints, even in subjects with atypical areal arrangements or those from future studies. The recently reported HCP Multi-Modal Parcellation, version 1.0 (HCP_MMP1.0) meets these criteria<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e1489">9</a></sup>, but we expect that this will not be the final word on cortical parcellation. Analogous multimodal parcellations are also needed for subcortical structures and the cerebellum.</p><p>The HCP_MMP1.0 parcellation (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig3">Fig. 3</a>) contains 180 distinct areas per hemisphere, symmetrically arranged across the two hemispheres. Eighty-three areas correspond reasonably well with areas defined in one or more prior studies and were assigned the same names, while 97 were assigned new names. The group average parcellation was used to train a machine-learning classifier (a multilayer perceptron<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Hacker, C.D. et al. Resting state network estimation in individual subjects. Neuroimage 82, 616633 (2013). This study used a supervised classifier (multilayer perceptron) to identify resting-state networks in individual subjects. Classification was reliable in individuals not used for classifier training and generated more spatially specific resting-state networks than alternative methods tested." href="/articles/nn.4361#ref-CR88" id="ref-link-section-d24703174e1499">88</a></sup>) to enable a fully automated process for identifying areas in individual subjects on the basis of each area's multimodal fingerprint, even in individuals having atypical topological arrangements of areas (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig15">Supplementary Fig. 11</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="The HCP_MMP1.0."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 3: The HCP_MMP1.0.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/nn.4361/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig3_HTML.jpg?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig3_HTML.jpg" alt="figure 3" loading="lazy" width="685" height="459"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>Each panel shows 180 cortical areas delineated and identified in the left or right hemisphere, displayed on an inflated or flattened cortical surface. Black outlines indicate areal borders. Colors indicate to what extent the areas are associated in the resting state to auditory (red), somatosensory (green), visual (blue), task-positive (paler shades) or task-negative (darker shades) systems. The legend on the bottom right illustrates the 3D color space used in the figure. Hopf's area 55b, Brodmann's area 44, retrosplenial cortex (RSC), ventral lateral intraparietal cortex (LIPv), the superior frontal language area (SFL), the second parieto-occipital sulcus area (POS2), the perisylvian language area (PSL), the middle temporal area (MT), primary visual cortex (V1) and primary auditory cortex (A1) are labeled. Reproduced from ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e1517">9</a>, Nature Publishing Group. Data at <a href="http://balsa.wustl.edu/WN56">http://balsa.wustl.edu/WN56</a>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/nn.4361/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>More generally, the classifier can reliably detect cortical areas in individual subjects without using the HCP's task fMRI data (using only myelin maps, thickness maps, resting-state network maps and resting-state visuotopic maps), enabling generalization of the approach to many non-HCP neuroimaging data sets. Individual investigators wishing to apply the HCP Pipelines, areal-feature-based registration and trained areal classifier to their own individual subjects need only acquire T1w, T2w, field map and fMRI data as described above, as even non-resting-state fMRI data include similar patterns of connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Cole, M.W., Bassett, D.S., Power, J.D., Braver, T.S. &amp; Petersen, S.E. Intrinsic and task-evoked network architectures of the human brain. Neuron 83, 238251 (2014)." href="/articles/nn.4361#ref-CR89" id="ref-link-section-d24703174e1539">89</a></sup>. The population-based parcellation can also be applied directly to registered individual subjects, though it will not be accurate where individual subjects deviate from the typical layout of areas. The HCP's multimodal 'hard' parcellation v1.0 provides a neuroanatomical foundation for future neuroimaging analyses, though other 'weighted' approaches provide valuable complementary information as well (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 14</a>). It is worth noting that FreeSurfer uses an algorithm to automatically label gyri and sulci in individual subjects based on manually generated training labels that is similar in spirit to our areal classifier<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Fischl, B. et al. Automatically parcellating the human cerebral cortex. Cereb. Cortex 14, 1122 (2004)." href="/articles/nn.4361#ref-CR90" id="ref-link-section-d24703174e1546">90</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec15">Tenet 7: routine data sharing and advanced informatics for data management</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec16">Data sharing for individual laboratories.</h4><p>Data may be publicly shared at many levels, including extensively processed, preprocessed and unprocessed stages, together with the code used for analysis. Data sharing is increasingly mandated by journals and funding agencies. Although the HCP has shared its data using an advanced informatics infrastructure (described below), most individual investigators only publish their results in journal articles, as the additional time and informatics skills required for more expansive data sharing have made it impractical. Generating and editing manuscript figures is typically a tedious process involving multiple software applications that neither work efficiently together nor aid in preparing data for sharing after publication. Routine sharing of extensively processed data would be expedited by a streamlined process for generating publication-ready figures that is integrated with a service for easy post-publication sharing of the associated data.</p><p>An integrated approach to data visualization, figure generation and data sharing is now available. Connectome Workbench, a freely available general-purpose neuroimaging visualization platform, provides a flexible interface for displaying rich, multimodal neuroimaging data sets. Workbench enables visualization of various data types overlaid on surfaces or volumes (NIFTI, GIFTI or CIFTI files). Multipanel montage figures can be generated by combining web-browser-like tabs that can also be annotated (for example, with text, symbols and imported image panels) directly in Workbench. For example, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig4">Figure 4</a> is one of 42 published figures in one study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e1569">9</a></sup> exported from Workbench without further editing. Importantly, Workbench allows such complex displays to be saved as 'scenes', which can be quickly reopened, edited, resaved and exported as publication-ready images. Data from such figures can be shared by uploading the scene file to the BALSA database (<a href="http://balsa.wustl.edu/">http://balsa.wustl.edu/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Van Essen, D.C. et al. The brain analysis of spatial maps and atlases (BALSA) database. Neuroimage j.neuroimage.2016.04.002 (2016)." href="/articles/nn.4361#ref-CR11" id="ref-link-section-d24703174e1580">11</a></sup> directly from Workbench, together with basic information about the publication. Scene-specific URLs enable one-click linking from a published figure to the corresponding page in BALSA (see figure legends). Adoption of similar strategies by other neuroimaging software and databases would accelerate sharing of extensively analyzed data and facilitate more accurate and diverse comparisons across studies.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Example parcellated analyses run using HCP data and the HCP_MMP1.0 cortical parcellation."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 4: Example parcellated analyses run using HCP data and the HCP_MMP1.0 cortical parcellation.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/nn.4361/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig4_HTML.jpg?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig4_HTML.jpg" alt="figure 4" loading="lazy" width="685" height="402"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>(<b>a</b>) Dense myelin map, very similar to a parcellated myelin map in <b>f</b> despite a dramatic dimensionality reduction. (<b>b</b>,<b>c</b>) Example dense (<b>b</b>) and parcellated (<b>c</b>) task fMRI analysis (language; story vs. baseline). (<b>d</b>) The entire HCP task fMRI battery's <i>z</i>-statistics (86 contrasts; 47 distinct and 39 sign-reversed versions) analyzed in parcellated form and displayed as a matrix (rows are parcels, columns are contrasts, white outline indicates the map displayed in <b>c</b>). (<b>e</b>) Fitting a task design to parcellated time series data provides a major improvement in <i>z</i>-statistics compared to fitting it to dense time series and then parcellating the resulting statistical maps. (<b>f</b>) Parcellated myelin map, very similar to the dense map in <b>a</b> despite a dramatic dimensionality reduction. (<b>g</b>) Map of parcellated cortical thickness (in mm; corrected for folding by regressing out mean surface curvature). (<b>h</b>,<b>i</b>) Parcellated functional connectivity maps on the brain (seeded from area PGi, black dot). In both cases, the task-negative (default mode) network is apparent. (<b>j</b>) Parcellated connectome matrix view with the full correlation connectome below the diagonal and the partial correlation connectome above the diagonal (white line is the displayed partial correlation brain map). The figure matches the Connectome Workbench scene available on the BALSA database (<a href="http://balsa.wustl.edu/RG0x">http://balsa.wustl.edu/RG0x</a>). Panel <b>e</b> was generated in Matlab (MathWorks), saved as a PNG file and then loaded into Workbench as its own tab. Reprinted from ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. Nature nature18933 (2016). This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere." href="/articles/nn.4361#ref-CR9" id="ref-link-section-d24703174e1659">9</a>, Nature Publishing Group.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/nn.4361/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">Large-scale data sharing and an advanced informatics infrastructure.</h4><p>Well-designed data management tools and processes are vital for the successful collection and analysis of high-quality HCP-style data on a large scale. ConnectomeDB (<a href="https://db.humanconnectome.org/">https://db.humanconnectome.org/</a>), the primary database for HCP data, was designed for user-friendly sharing of unprocessed and preprocessed data from a large number of individual subjects, which can be selectively downloaded using a variety of flexible criteria<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Marcus, D.S. et al. Human Connectome Project informatics: quality control, database services, and data visualization. Neuroimage 80, 202219 (2013)." href="/articles/nn.4361#ref-CR12" id="ref-link-section-d24703174e1686">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Hodge, M.R. et al. ConnectomeDBsharing human brain connectivity data. Neuroimage 124 Pt B, 11021107 (2016)." href="/articles/nn.4361#ref-CR91" id="ref-link-section-d24703174e1689">91</a></sup>. ConnectomeDB is based on the XNAT informatics platform (<a href="http://www.xnat.org/">http://www.xnat.org/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Marcus, D.S., Olsen, T.R., Ramaratnam, M. &amp; Buckner, R.L. The Extensible Neuroimaging Archive Toolkit: an informatics platform for managing, exploring, and sharing neuroimaging data. Neuroinformatics 5, 1134 (2007)." href="/articles/nn.4361#ref-CR92" id="ref-link-section-d24703174e1700">92</a></sup>, which manages imaging data through a structured workflow from acquisition to quality control to automated processing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Hodge, M.R. et al. ConnectomeDBsharing human brain connectivity data. Neuroimage 124 Pt B, 11021107 (2016)." href="/articles/nn.4361#ref-CR91" id="ref-link-section-d24703174e1704">91</a></sup>. It also provides data entry and import tools for task and biometric data collected during image acquisition, as well as behavioral and other nonimaging data. ConnectomeDB also serves as the data management platform for the Lifespan Connectome projects, has been deployed by some Connectomes Related to Human Disease projects (see below), and can benefit other groups carrying out large-scale HCP-style studies (source code at <a href="http://www.xnat.org/download/">http://www.xnat.org/download/</a>). Given the complexity of the HCP project and the massive size of the HCP datasets, documentation and distribution required a multifaceted approach (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/nn.4361#Sec18">Box 2</a>).</p><div class="c-article-box" data-expandable-box-container="true"><div class="c-article-box__container" data-expandable-box="true" aria-hidden="true" id="box-Sec18"><h3 class="c-article-box__container-title u-h3 js-expandable-title" id="Sec18">Box 2: Accessing and using HCP data, software and protocols</h3><div class="c-article-box__content"><p><b>Overview of the HCP's data sharing.</b> ConnectomeDB (<a href="http://db.humanconnectome.org/">http://db.humanconnectome.org/</a>) is the primary repository of HCP imaging, behavioral and demographic data. Three-tesla structural, functional and diffusion imaging data for <span class="stix"></span>1,100 subjects are shared in unprocessed, minimally preprocessed, and more extensively processed forms and staged over four major releases (Q1, Q1Q3, S500, S900), with the final S1100 release slated for summer 2016. Seven-tesla data acquired on 180 subjects have been released in part, with the rest to follow shortly. Magnetoencephalography data acquired from 95 subjects give a window on fast temporal dynamics of the brain and were released in the fall of 2015 (ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 115" title="Larson-Prior, L.J. et al. Adding dynamics to the Human Connectome Project with MEG. Neuroimage 80, 190201 (2013)." href="/articles/nn.4361#ref-CR115" id="ref-link-section-d24703174e1734">115</a>). Notably, the S900 release and the forthcoming S1100 release include the improved intersubject registration provided by MSMAll. The HCP_MMP1.0 parcellation is available in BALSA (<a href="https://balsa.wustl.edu/study/show/RVVG">https://balsa.wustl.edu/study/show/RVVG</a>), and individual subject parcellations for each HCP subject will be released on ConnectomeDB in the future (anticipated in fall 2016).</p><p><b>HCP data set is much larger than those of past studies.</b> With the high spatial and temporal resolution of HCP data, the data set is at least an order of magnitude larger than widely used open-access neuroimaging data sets such as ADNI (<a href="http://adni.loni.usc.edu">http://adni.loni.usc.edu</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 116" title="Jack, C.R. Jr. et al. The Alzheimer's Disease Neuroimaging Initiative (ADNI): MRI methods. J. Magn. Reson. Imaging 27, 685691 (2008)." href="/articles/nn.4361#ref-CR116" id="ref-link-section-d24703174e1757">116</a></sup>. For a single subject, the compressed NIFTI-formatted unprocessed data are around 10 GB, and the preprocessed data are nearly 30 GB.</p><p><b>HCP data are widely used via multiple modes of data access.</b> To date, &gt;5,200 investigators have agreed to HCP open-access data-use terms (<span class="stix"></span>520 to restricted-access terms for accessing family structure and other sensitive data). Users can (i) access ConnectomeDB directly to download packages for individual subjects, user-selected subject groups or HCP-specified groups; (ii) purchase Connectome-in-a-Box for the cost of the hard drives, which can be shared by investigators at a given institution; and (iii) access data via the Amazon cloud for processing on the cloud (or for download). Data downloaded directly from ConnectomeDB exceed 5,400 terabytes (TB), with an additional 2,000 TB transferred via hard drives (Connectome-in-a-Box) and the Amazon cloud.</p><p><b>Data documentation.</b> The richness and complexity of the HCP data require extensive documentation for users to understand what is available, how the data sets are organized and how they were processed, including quality control measures. Available resources include (i) a Reference Manual associated with each data release (<a href="http://humanconnectome.org/documentation/">http://humanconnectome.org/documentation/</a>); (ii) HCP course materials (lectures, tutorials and associated data, available at <a href="http://humanconnectome.org/courses/">http://humanconnectome.org/courses/</a>); (iii) publications (for example, for database organization)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Hodge, M.R. et al. ConnectomeDBsharing human brain connectivity data. Neuroimage 124 Pt B, 11021107 (2016)." href="/articles/nn.4361#ref-CR91" id="ref-link-section-d24703174e1785">91</a></sup>; and (iv) the HCP public wiki (<a href="https://wiki.humanconnectome.org/display/PublicData/Home/">https://wiki.humanconnectome.org/display/PublicData/Home/</a>), which provides additional documentation, answers to frequently asked questions, and updates (including known issues and planned fixes).</p><p><b>Software sharing.</b> Also important to the replicability of neuroimaging studies is the sharing of the software used for analysis. Scripts for HCP pipelines are available on GitHub (<a href="https://github.com/Washington-University/Pipelines/">https://github.com/Washington-University/Pipelines/</a>), and their usage is described in the HCP course materials. This includes pipelines for magnetoencephalography as well as MRI data. Connectome Workbench is available as binaries and source code (<a href="http://www.humanconnectome.org/software/get-connectome-workbench.html">http://www.humanconnectome.org/software/get-connectome-workbench.html</a>, on GitHub, and at <a href="http://neuro.debian.net/">http://neuro.debian.net/</a>) along with tutorials. Many of the labs that make up the HCP consortia also share their software on either their own websites or the HCP website.</p><p><b>MRI protocols.</b> The MRI protocols used for the main HCP 3 T and 7 T scans are available at <a href="http://protocols.humanconnectome.org/HCP/">http://protocols.humanconnectome.org/HCP/</a>; HCP-style pulse sequences have been widely distributed (<a href="http://www.cmrr.umn.edu/multiband/index.shtml">http://www.cmrr.umn.edu/multiband/index.shtml</a>). The HCP-Short protocols used for the HCP Lifespan project (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 4</a>) are at <a href="http://protocols.humanconnectome.org/lifespan/">http://protocols.humanconnectome.org/lifespan/</a>.</p><p><b>User Support.</b> Users of the HCP's data and software are supported on the 'HCP-Users' mailing list (<a href="https://www.humanconnectome.org/contact/hcp-users-request.php">https://www.humanconnectome.org/contact/hcp-users-request.php</a>), where support requests are answered by HCP consortium members and other community members in a public forum so that all users may benefit.</p></div></div></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec19">HCP-style follow-up studies.</h4><p>The original (healthy young adult) HCP will be succeeded by several large-scale imaging projects that use HCP-style data acquisition, analysis and sharing to study different populations in health and disease. Data sharing for many of these projects will be managed by the Connectome Coordination Facility (CCF; <a href="http://www.humanconnectome.org/ccf">http://www.humanconnectome.org/ccf</a>), an NIH-funded resource devoted to supporting data sharing from the Connectomes Related to Human Disease projects (11 currently funded, with more anticipated in 2017) plus three Lifespan HCP projects (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig5">Fig. 5</a>). The CCF is an extension of the ConnectomeDB infrastructure and will provide a common interface for searching within and across projects. In addition, it is developing common acquisition protocols and maintaining the HCP's processing pipelines to maximize cross-study comparisons and analysis, and it is partnering with other connectomics studies, such as the Adolescent Brain Cognitive Development study and UK Developing HCP (<a href="http://www.developingconnectome.org/">http://www.developingconnectome.org/</a>), to encourage harmonization of acquisition and analysis methods. Versions of the HCP-Short protocol (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 4</a>) have been adapted for a variety of vendors and scanner models, with the goal of producing images of similar spatial and temporal resolution.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Current and future projects that use HCP-style data acquisition, analysis and sharing."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 5: Current and future projects that use HCP-style data acquisition, analysis and sharing.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/nn.4361/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig5_HTML.jpg?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig5_HTML.jpg" alt="figure 5" loading="lazy" width="685" height="308"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>Projects shown in green will use the Connectome Coordination Facility (CCF, purple) as their primary mode of data sharing. MGH, Massachusetts General Hospital; UCLA, University of California, Los Angeles. WU, Washington University; in St. Louis; Minn, University of Minnesota; Ox, Oxford University; MEG, magnetoencephalography.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/nn.4361/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Concluding comments"><div class="c-article-section" id="Sec20-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec20">Concluding comments</h2><div class="c-article-section__content" id="Sec20-content"><p>The NIH had two interrelated core goals in launching the HCP: (i) to generate a publicly shared, widely used repository of high-quality multimodal neuroimaging data suitable for mapping the human connectome and (ii) to advance the methods used in neuroimaging research. More than 140 studies have acknowledged the use of HCP data, and for half of these the authors were largely or entirely outside the HCP consortium (<a href="https://wiki.humanconnectome.org/x/YQDJAw">https://wiki.humanconnectome.org/x/YQDJAw</a>). These studies address a wide range of questions using various imaging modalities. Five brief examples highlight some of this progress. Smith <i>et al</i>.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Smith, S.M. et al. A positive-negative mode of population covariation links brain connectivity, demographics and behavior. Nat. Neurosci. 18, 15651567 (2015)." href="/articles/nn.4361#ref-CR93" id="ref-link-section-d24703174e1932">93</a></sup> identified a positivenegative phenotypic axis that covaries with differences in the strength of functional connectivity in a subset of brain regions that are generally implicated in higher cognitive functions. Yeo <i>et al</i>.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Yeo, B.T., Krienen, F.M., Chee, M.W. &amp; Buckner, R.L. Estimates of segregation and overlap of functional connectivity networks in the human cerebral cortex. Neuroimage 88, 212227 (2014)." href="/articles/nn.4361#ref-CR94" id="ref-link-section-d24703174e1939">94</a></sup> investigated the overlap and segregation of the brain's functional networks, finding that association regions tend to overlap with at least two networks whereas somatosensory and visual regions are more isolated. Wang <i>et al</i>.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Wang, D. et al. Parcellating cortical functional networks in individuals. Nat. Neurosci. 18, 18531860 (2015b)." href="/articles/nn.4361#ref-CR95" id="ref-link-section-d24703174e1947">95</a></sup> investigated functional network parcellations at the individual level using HCP data. Hawrylycz <i>et al</i>.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 18321844 (2015)." href="/articles/nn.4361#ref-CR96" id="ref-link-section-d24703174e1954">96</a></sup> demonstrated correlations between gene expression patterns in post-mortem human cortex and HCP-based <i>in vivo</i> functional connectivity. Tavor <i>et al</i>.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Tavor, I. et al. Task-free MRI predicts individual differences in brain activity during task performance. Science 352, 216220 (2016). The authors show that a model relating task-independent (rfMRI) measurements to task activity can accurately predict task activation maps for unseen subjects, suggesting a coupling between brain connectivity and function at the level of individual subjects." href="/articles/nn.4361#ref-CR97" id="ref-link-section-d24703174e1965">97</a></sup> showed that a model relating task-independent (rfMRI) measurements to task activity can accurately predict task activation maps for unseen subjects, suggesting a coupling between brain connectivity and function at the level of individual subjects. After release of the final S1100 imaging and genotyping data (summer 2016) and the individual-subject cortical parcellations (projected autumn 2016), we anticipate that a growing fraction of studies using HCP data will use the full set of available subjects, better-aligned data, multimodal parcellations of group average and individual subjects, and family structure (for example, twins). Capitalizing on the full richness of the HCP data offers the best prospects for robust and neurobiologically grounded findings.</p><p>Even with large amounts of high-quality data, it remains extremely important to use caution in interpreting complex neuroimaging data sets and the indirect inferences related to brain connectivity available from neuroimaging. Neither functional connectivity based on rfMRI nor structural connectivity based on dMRI tractography provides good quantitative estimates of the actual strength of direct anatomical connectivity between areas or regions. This reflects a variety of methodological biases and other limitations that have become better understood in recent years (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 15</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Jbabdi, S., Sotiropoulos, S.N., Haber, S.N., Van Essen, D.C. &amp; Behrens, T.E. Measuring macroscopic brain connections in vivo. Nat. Neurosci. 18, 15461555 (2015)." href="/articles/nn.4361#ref-CR2" id="ref-link-section-d24703174e1975">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Donahue, C.J. et al. Using diffusion tractography to predict cortical connection strength and distance: A quantitative comparison with tracers in the monkey. J. Neurosci 36, 67586770 (2016)." href="/articles/nn.4361#ref-CR35" id="ref-link-section-d24703174e1978">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Reveley, C. et al. Superficial white matter fiber systems impede detection of long-range cortical connections in diffusion MR tractography. Proc. Natl. Acad. Sci. USA 112, E2820E2828 (2015)." href="/articles/nn.4361#ref-CR98" id="ref-link-section-d24703174e1981">98</a></sup>, but remain serious challenges even with the advances provided by the HCP-style paradigm. Research continues to further validate noninvasive connectivity methods and maximize their accuracy.</p><p>The HCP-style paradigm brings together a wide range of methodological advances, many provided by investigators or projects outside the HCP and others reflecting direct HCP innovations. New MRI scanner technology (Siemens Prisma) and pulse sequence technology (multiband) were developed in part through the HCP, enabling faster imaging of finer details. Many improvements in image distortion correction, registration and data-driven image denoising enable selective removal of artifacts while preserving the precision of the signals of interest. The CIFTI grayordinates analysis framework, together with areal-feature-based surface registration, largely compensates for individual variability across subjects without causing or resorting to blurring of the data. A multimodal cortical parcellation provides a new neuroanatomical foundation for studies of the human cerebral cortex and is a valuable prerequisite for generating area-to-area connectomes in individuals and group averages. Intuitive, user-friendly data visualization software together with integrated neuroimaging databases and software pipelines enable easier data processing and sharing.</p><p>The overarching objectives of the HCP-style paradigm are (i) to acquire and analyze neuroimaging data as accurately as possible in each individual from their original brain imaging voxels to their individually mapped brain areas; (ii) to use precise intersubject alignment while avoiding smoothing to minimize blurring across brain areal boundaries when comparing results across individuals, groups and studies; and (iii) to routinely share neuroimaging data in a way that facilitates conclusive cross-study comparisons. Unlike traditional volume-based approaches that often result in statistically significant 'blobs' or 3D coordinates of unclear neuroanatomical identity, the HCP-style paradigm aims to remain as faithful as possible to the underlying neuroanatomy, even in the face of the remarkable individual variability of human brains. Indeed, the gains in spatial sharpness and clarity provided by the HCP-style paradigm (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig6">Fig. 6</a>) are qualitatively analogous to those made in astronomy after the introduction of adaptive optics and space telescopes to overcome the atmospheric blurring that plagues conventional ground-based telescopes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Dalcanton, J.J. 18 years of science with the Hubble Space Telescope. Nature 457, 4150 (2009)." href="/articles/nn.4361#ref-CR99" id="ref-link-section-d24703174e1994">99</a></sup>. We provide concrete, generally applicable guidelines for individual laboratories wishing to embark on a new study using the HCP-style paradigm, though of course not all recommendations will fit every study, and methods will continue to improve (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 16</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Fidelity of localizing area MT+ (architectonic area hOc5) when mapped to the cortical surface by different methods."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Figure 6: Fidelity of localizing area MT+ (architectonic area hOc5) when mapped to the cortical surface by different methods.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/nn.4361/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig6_HTML.jpg?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig6_HTML.jpg" alt="figure 6" loading="lazy" width="685" height="248"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p>Left: volume-based mapping of probabilistic cytoarchitectonic area hOc5 (ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 104" title="Malikovic, A. et al. Cytoarchitectonic analysis of the human extrastriate cortex in the region of V5/MT+: a probabilistic, stereotaxic map of area hOc5. Cereb. Cortex 17, 562574 (2007)." href="/articles/nn.4361#ref-CR104" id="ref-link-section-d24703174e2012">104</a>) from 10 post-mortem subjects mapped to a cortical atlas surface. Black arrows, locations where the volume-based mapping spreads across gyral and sulcal folds. Center: Surface-based registration of hOc5 from the same 10 subjects mapped to individual surface reconstructions, then to a surface-based atlas using FreeSurfer's folding-based surface registration method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Fischl, B. et al. Cortical folding patterns and predicting cytoarchitecture. Cereb. Cortex 18, 19731980 (2008). The authors used folding-based alignment to register cytoarchitectonic areas to a surface-based atlas. In the resultant probabilistic maps, alignment was high for early sensory areas but much lower for 'higher' cortical areas, reflecting regional differences in the correlation between areal boundaries and cortical folding." href="/articles/nn.4361#ref-CR74" id="ref-link-section-d24703174e2016">74</a></sup>. White arrows, outlier hOc5 from individual subjects that are not well aligned to the FreeSurfer group average owing to imperfect correspondence between areal boundaries and sulcal folds. The white oval is in the same location across all panels, showing how the volume-based alignment drifts away from the surface-based alignments, in addition to having substantially lower cross-subject overlap. Right: group average cortical myelin map (from 196 HCP subjects) with a yelloworangered hotspot indicating the MT+ complex and retinotopic areal maximum probability maps (MPMs) from 12 non-HCP subjects, both registered independently using areal-feature-based surface registration and de-drifted (<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Note 13</a>). As shown in <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig14">Supplementary Figure 10</a>, the white dots represent a contour along which functional connectivity rapidly changes and which aligns with the border between retinotopic areas MT and pMST from a separate study. This is an example of the more conclusive cross-study, cross-modal boundary comparisons made possible by HCP-style analysis. Data at <a href="http://balsa.wustl.edu/kNpD">http://balsa.wustl.edu/kNpD</a>. Left and center panels adapted from ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Van Essen, D.C., Glasser, M.F., Dierker, D.L., Harwell, J. &amp; Coalson, T. Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases. Cereb. Cortex 22, 22412262 (2012)." href="/articles/nn.4361#ref-CR39" id="ref-link-section-d24703174e2033">39</a>, D.C. Van Essen, M.F. Glasser, D.L. Dierker, J. Harwell &amp; T. Coalson, Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases, <i>Cerebral Cortex</i> 2012, <b>22</b>(10), 22412262, by permission of Oxford University Press. Right panel adapted from ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 105" title="Abdollahi, R.O. et al. Correspondences between retinotopic areas and myelin maps in human visual cortex. Neuroimage 99, 509524 (2014)." href="/articles/nn.4361#ref-CR105" id="ref-link-section-d24703174e2042">105</a>, Elsevier.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/nn.4361/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We propose that the diverse technological and conceptual advances in imaging integrated by and developed within the HCP meet the criteria of a new paradigm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Kuhn, T. The Structure of Scientific Revolutions (University of Chicago Press, 1962)." href="/articles/nn.4361#ref-CR100" id="ref-link-section-d24703174e2058">100</a></sup> for neuroimaging research. The HCP-style paradigm has a differing set of assumptions (for example, that blurriness in brain images is not a fact of life but an avoidable artifact of traditional methods and that neuroanatomical localization is critical to understanding the brain). It invokes new terminology and provides substantial new capabilities for further research. It is also disruptive to business as usual as practiced by the majority of neuroimaging investigators, and legacy data may not be easily comparable with data from the HCP-style paradigm while maintaining the new paradigm's improved standards of spatial fidelity and neuroanatomical localization (<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/nn.4361#Fig16">Supplementary Fig. 12</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">Supplementary Notes 17</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/nn.4361#MOESM36">18</a>). Thus moving to the new paradigm should not be undertaken lightly. That said, we believe the benefits decidedly outweigh the costs, as the new paradigm opens up a variety of novel analysis strategies that will likely accelerate our understanding of human brain structure and function and may even prove useful in a clinical setting. Instead of relying on rough evidence of spatial proximity as in the old model, we aim in the new paradigm to precisely compare the overlap and boundaries of results from differing studies to see if they likely refer to the same brain areas or not. Additionally, we are able to separate differences in brain function or connectivity much more precisely from differences in brain areal size, shape and position. We can investigate the significance of variations in individual subject areal topologies from the population's typical topology. Finally, the HCP-style paradigm is better positioned to exploit further improvements in brain image acquisition that will enable even finer-grained study of the brain at the level of cortical layers or columns, as well as finer subcortical organization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 101" title="De Martino, F. et al. Frequency preference and attention effects across cortical depths in the human primary auditory cortex. Proc. Natl. Acad. Sci. USA 112, 1603616041 (2015)." href="/articles/nn.4361#ref-CR101" id="ref-link-section-d24703174e2071">101</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="De Martino, F. et al. Spatial organization of frequency preference and selectivity in the human inferior colliculus. Nat. Commun. 4, 1386 (2013)." href="/articles/nn.4361#ref-CR102" id="ref-link-section-d24703174e2074">102</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Muckli, L. et al. Contextual feedback to superficial layers of V1. Curr. Biol. 25, 26902695 (2015)." href="/articles/nn.4361#ref-CR103" id="ref-link-section-d24703174e2077">103</a></sup>. Nonetheless, the challenges to neuroimaging remain daunting, and we hope that new technologies and ideas will lead to further paradigmatic advances in the future.</p></div></div></section>
                </div>
            

            <div>
                <div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1"><p class="c-article-references__text" id="ref-CR1">Behrens, T.E. &amp; Sporns, O. Human connectomics. <i>Curr. Opin. Neurobiol.</i> <b>22</b>, 144153 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.conb.2011.08.005" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.conb.2011.08.005" aria-label="Article reference 1" data-doi="10.1016/j.conb.2011.08.005">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xjt1Glur8%3D" aria-label="CAS reference 1">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21908183" aria-label="PubMed reference 1">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20connectomics&amp;journal=Curr.%20Opin.%20Neurobiol.&amp;doi=10.1016%2Fj.conb.2011.08.005&amp;volume=22&amp;pages=144-153&amp;publication_year=2012&amp;author=Behrens%2CTE&amp;author=Sporns%2CO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2"><p class="c-article-references__text" id="ref-CR2">Jbabdi, S., Sotiropoulos, S.N., Haber, S.N., Van Essen, D.C. &amp; Behrens, T.E. Measuring macroscopic brain connections <i>in vivo</i>. <i>Nat. Neurosci.</i> <b>18</b>, 15461555 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4134" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4134" aria-label="Article reference 2" data-doi="10.1038/nn.4134">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhslentr%2FN" aria-label="CAS reference 2">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26505566" aria-label="PubMed reference 2">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20macroscopic%20brain%20connections%20in%20vivo&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4134&amp;volume=18&amp;pages=1546-1555&amp;publication_year=2015&amp;author=Jbabdi%2CS&amp;author=Sotiropoulos%2CSN&amp;author=Haber%2CSN&amp;author=Van%20Essen%2CDC&amp;author=Behrens%2CTE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3"><p class="c-article-references__text" id="ref-CR3">Smith, S. Introduction to the <i>NeuroImage</i> special issue Mapping the Connectome. <i>Neuroimage</i> <b>80</b>, 1 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.07.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.07.012" aria-label="Article reference 3" data-doi="10.1016/j.neuroimage.2013.07.012">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23870176" aria-label="PubMed reference 3">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Introduction%20to%20the%20NeuroImage%20special%20issue%20%E2%80%9CMapping%20the%20Connectome%E2%80%9D&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.07.012&amp;volume=80&amp;publication_year=2013&amp;author=Smith%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4"><p class="c-article-references__text" id="ref-CR4">Van Essen, D.C. et al. The WU-Minn Human Connectome Project: an overview. <i>Neuroimage</i> <b>80</b>, 6279 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.041" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.041" aria-label="Article reference 4" data-doi="10.1016/j.neuroimage.2013.05.041">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23684880" aria-label="PubMed reference 4">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20WU-Minn%20Human%20Connectome%20Project%3A%20an%20overview&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.041&amp;volume=80&amp;pages=62-79&amp;publication_year=2013&amp;author=Van%20Essen%2CDC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5"><p class="c-article-references__text" id="ref-CR5">Ugurbil, K. et al. Pushing spatial and temporal resolution for functional and diffusion MRI in the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 80104 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.012" aria-label="Article reference 5" data-doi="10.1016/j.neuroimage.2013.05.012">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVOjsbzE" aria-label="CAS reference 5">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Pushing%20spatial%20and%20temporal%20resolution%20for%20functional%20and%20diffusion%20MRI%20in%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.012&amp;volume=80&amp;pages=80-104&amp;publication_year=2013&amp;author=Ug%CB%87urbil%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6"><p class="c-article-references__text" id="ref-CR6">Glasser, M.F. et al. The minimal preprocessing pipelines for the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 105124 (2013). <b>This study provides an overview of the preprocessing pipelines used for the structural, functional and diffusion MRI modalities acquired by the HCP and by a growing number of HCP-style projects. It also introduces the concept of grayordinates, representing surface and volume data in a combined CIFTI data format</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.04.127" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.04.127" aria-label="Article reference 6" data-doi="10.1016/j.neuroimage.2013.04.127">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23668970" aria-label="PubMed reference 6">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20minimal%20preprocessing%20pipelines%20for%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.04.127&amp;volume=80&amp;pages=105-124&amp;publication_year=2013&amp;author=Glasser%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7"><p class="c-article-references__text" id="ref-CR7">Griffanti, L. et al. ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging. <i>Neuroimage</i> <b>95</b>, 232247 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.03.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.03.034" aria-label="Article reference 7" data-doi="10.1016/j.neuroimage.2014.03.034">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24657355" aria-label="PubMed reference 7">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=ICA-based%20artefact%20removal%20and%20accelerated%20fMRI%20acquisition%20for%20improved%20resting%20state%20network%20imaging&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.03.034&amp;volume=95&amp;pages=232-247&amp;publication_year=2014&amp;author=Griffanti%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8"><p class="c-article-references__text" id="ref-CR8">Salimi-Khorshidi, G. <i>et. al</i>. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. <i>Neuroimage</i> <b>90</b>, 449468 (2014). <b>This study demonstrated a method for automatically cleaning spatially specific structured noise out of fMRI data using an independent component analysis (ICA)</b> <b>decomposition and machine learning classification of the resulting components into signal and noise</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.11.046" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.11.046" aria-label="Article reference 8" data-doi="10.1016/j.neuroimage.2013.11.046">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24389422" aria-label="PubMed reference 8">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=et.%20al.%20Automatic%20denoising%20of%20functional%20MRI%20data%3A%20combining%20independent%20component%20analysis%20and%20hierarchical%20fusion%20of%20classifiers&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.11.046&amp;volume=90&amp;pages=449-468&amp;publication_year=2014&amp;author=Salimi-Khorshidi%2CG">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9"><p class="c-article-references__text" id="ref-CR9">Glasser, M.F. et al. A multi-modal parcellation of human cerebral cortex. <i>Nature</i> nature18933 (2016). <b>This study describes the HCP_MMP1.0 multimodal human cortical parcellation in a group average and in individual subjects. Extensive supplemental data detail the analysis methods used and the evidence supporting the identification of all 180 areas in each hemisphere</b>.</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10"><p class="c-article-references__text" id="ref-CR10">Robinson, E.C. et al. MSM: a new flexible framework for multimodal surface matching. <i>Neuroimage</i> <b>100</b>, 414426 (2014). <b>This study introduced a powerful new method for aligning cortical surfaces from individuals to an atlas using areal features as well as folding patterns to constrain the alignment</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.05.069" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.05.069" aria-label="Article reference 10" data-doi="10.1016/j.neuroimage.2014.05.069">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24939340" aria-label="PubMed reference 10">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=MSM%3A%20a%20new%20flexible%20framework%20for%20multimodal%20surface%20matching&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.05.069&amp;volume=100&amp;pages=414-426&amp;publication_year=2014&amp;author=Robinson%2CEC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11"><p class="c-article-references__text" id="ref-CR11">Van Essen, D.C. et al. The brain analysis of spatial maps and atlases (BALSA) database. <i>Neuroimage</i> j.neuroimage.2016.04.002 (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12"><p class="c-article-references__text" id="ref-CR12">Marcus, D.S. et al. Human Connectome Project informatics: quality control, database services, and data visualization. <i>Neuroimage</i> <b>80</b>, 202219 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.077" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.077" aria-label="Article reference 12" data-doi="10.1016/j.neuroimage.2013.05.077">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23707591" aria-label="PubMed reference 12">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20Connectome%20Project%20informatics%3A%20quality%20control%2C%20database%20services%2C%20and%20data%20visualization&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.077&amp;volume=80&amp;pages=202-219&amp;publication_year=2013&amp;author=Marcus%2CDS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13"><p class="c-article-references__text" id="ref-CR13">Eklund, A., Nichols, T.E. &amp; Knutsson, H. Cluster failure: why fMRI inferences for spatial extent have inflated false-positive rates. <i>Proc. Natl. Acad. Sci. USA</i> <b>28</b>, 79007905 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1602413113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1602413113" aria-label="Article reference 13" data-doi="10.1073/pnas.1602413113">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFentrvN" aria-label="CAS reference 13">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Cluster%20failure%3A%20why%20fMRI%20inferences%20for%20spatial%20extent%20have%20inflated%20false-positive%20rates&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1602413113&amp;volume=28&amp;pages=7900-7905&amp;publication_year=2016&amp;author=Eklund%2CA&amp;author=Nichols%2CTE&amp;author=Knutsson%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14"><p class="c-article-references__text" id="ref-CR14">Turner, R. Where matters: new approaches to brain analysis. in <i>Microstructural Parcellation of the Human Cerebral Cortex</i> (eds. Geyer, S. &amp; Turner, R.) 179196 (Springer, 2013).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15"><p class="c-article-references__text" id="ref-CR15">Turner, R. &amp; Geyer, S. Comparing like with like: the power of knowing where you are. <i>Brain Connect.</i> <b>4</b>, 547557 (2014). <b>The authors emphasize the importance of high-quality data acquisition, careful surface-based analysis in individual subjects and localization in relation to the best available cortical parcellation. Hence they presage many of the points made in the present article</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1089/brain.2014.0261" data-track-action="article reference" href="https://doi.org/10.1089%2Fbrain.2014.0261" aria-label="Article reference 15" data-doi="10.1089/brain.2014.0261">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24999746" aria-label="PubMed reference 15">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4146387" aria-label="PubMed Central reference 15">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparing%20like%20with%20like%3A%20the%20power%20of%20knowing%20where%20you%20are&amp;journal=Brain%20Connect.&amp;doi=10.1089%2Fbrain.2014.0261&amp;volume=4&amp;pages=547-557&amp;publication_year=2014&amp;author=Turner%2CR&amp;author=Geyer%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16"><p class="c-article-references__text" id="ref-CR16">Sereno, M.I. et al. Borders of multiple visual areas in humans revealed by functional magnetic resonance imaging. <i>Science</i> <b>268</b>, 889893 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.7754376" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.7754376" aria-label="Article reference 16" data-doi="10.1126/science.7754376">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2MXls1SgsLs%3D" aria-label="CAS reference 16">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7754376" aria-label="PubMed reference 16">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Borders%20of%20multiple%20visual%20areas%20in%20humans%20revealed%20by%20functional%20magnetic%20resonance%20imaging&amp;journal=Science&amp;doi=10.1126%2Fscience.7754376&amp;volume=268&amp;pages=889-893&amp;publication_year=1995&amp;author=Sereno%2CMI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17"><p class="c-article-references__text" id="ref-CR17">Wang, L., Mruczek, R.E., Arcaro, M.J. &amp; Kastner, S. Probabilistic maps of visual topography in human cortex. <i>Cereb. Cortex</i> <b>25</b>, 39113931 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhu277" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhu277" aria-label="Article reference 17" data-doi="10.1093/cercor/bhu277">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ektbs%3D" aria-label="CAS reference 17">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25452571" aria-label="PubMed reference 17">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Probabilistic%20maps%20of%20visual%20topography%20in%20human%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhu277&amp;volume=25&amp;pages=3911-3931&amp;publication_year=2015&amp;author=Wang%2CL&amp;author=Mruczek%2CRE&amp;author=Arcaro%2CMJ&amp;author=Kastner%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18"><p class="c-article-references__text" id="ref-CR18">Fischl, B. &amp; Dale, A.M. Measuring the thickness of the human cerebral cortex from magnetic resonance images. <i>Proc. Natl. Acad. Sci. USA</i> <b>97</b>, 1105011055 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.200033797" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.200033797" aria-label="Article reference 18" data-doi="10.1073/pnas.200033797">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3cXnt1ahtrg%3D" aria-label="CAS reference 18">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10984517" aria-label="PubMed reference 18">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC27146" aria-label="PubMed Central reference 18">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20the%20thickness%20of%20the%20human%20cerebral%20cortex%20from%20magnetic%20resonance%20images&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.200033797&amp;volume=97&amp;pages=11050-11055&amp;publication_year=2000&amp;author=Fischl%2CB&amp;author=Dale%2CAM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19"><p class="c-article-references__text" id="ref-CR19">Glasser, M.F., Goyal, M.S., Preuss, T.M., Raichle, M.E. &amp; Van Essen, D.C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. <i>Neuroimage</i> <b>93</b>, 165175 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.03.060" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.03.060" aria-label="Article reference 19" data-doi="10.1016/j.neuroimage.2013.03.060">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXms1Krs7k%3D" aria-label="CAS reference 19">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23567887" aria-label="PubMed reference 19">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Trends%20and%20properties%20of%20human%20cerebral%20cortex%3A%20correlations%20with%20cortical%20myelin%20content&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.03.060&amp;volume=93&amp;pages=165-175&amp;publication_year=2014&amp;author=Glasser%2CMF&amp;author=Goyal%2CMS&amp;author=Preuss%2CTM&amp;author=Raichle%2CME&amp;author=Van%20Essen%2CDC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20"><p class="c-article-references__text" id="ref-CR20">Glasser, M.F. &amp; Van Essen, D.C. Mapping human cortical areas <i>in vivo</i> based on myelin content as revealed by T1- and T2-weighted MRI. <i>J. Neurosci.</i> <b>31</b>, 1159711616 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2180-11.2011" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2180-11.2011" aria-label="Article reference 20" data-doi="10.1523/JNEUROSCI.2180-11.2011">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtVOru7jJ" aria-label="CAS reference 20">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21832190" aria-label="PubMed reference 20">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3167149" aria-label="PubMed Central reference 20">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20human%20cortical%20areas%20in%20vivo%20based%20on%20myelin%20content%20as%20revealed%20by%20T1-%20and%20T2-weighted%20MRI&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2180-11.2011&amp;volume=31&amp;pages=11597-11616&amp;publication_year=2011&amp;author=Glasser%2CMF&amp;author=Van%20Essen%2CDC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21"><p class="c-article-references__text" id="ref-CR21">Dale, A.M., Fischl, B. &amp; Sereno, M.I. Cortical surface-based analysis. I. Segmentation and surface reconstruction. <i>Neuroimage</i> <b>9</b>, 179194 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.1998.0395" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.1998.0395" aria-label="Article reference 21" data-doi="10.1006/nimg.1998.0395">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1M7jt1Gisg%3D%3D" aria-label="CAS reference 21">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9931268" aria-label="PubMed reference 21">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20surface-based%20analysis.%20I.%20Segmentation%20and%20surface%20reconstruction&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.1998.0395&amp;volume=9&amp;pages=179-194&amp;publication_year=1999&amp;author=Dale%2CAM&amp;author=Fischl%2CB&amp;author=Sereno%2CMI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22"><p class="c-article-references__text" id="ref-CR22">Dale, A.M. &amp; Sereno, M.I. Improved localizadon of cortical activity by combining EEG and MEG with MRI cortical surface reconstruction: a linear approach. <i>J. Cogn. Neurosci.</i> <b>5</b>, 162176 (1993).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1162/jocn.1993.5.2.162" data-track-action="article reference" href="https://doi.org/10.1162%2Fjocn.1993.5.2.162" aria-label="Article reference 22" data-doi="10.1162/jocn.1993.5.2.162">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC3sbitlSqtw%3D%3D" aria-label="CAS reference 22">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23972151" aria-label="PubMed reference 22">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20localizadon%20of%20cortical%20activity%20by%20combining%20EEG%20and%20MEG%20with%20MRI%20cortical%20surface%20reconstruction%3A%20a%20linear%20approach&amp;journal=J.%20Cogn.%20Neurosci.&amp;doi=10.1162%2Fjocn.1993.5.2.162&amp;volume=5&amp;pages=162-176&amp;publication_year=1993&amp;author=Dale%2CAM&amp;author=Sereno%2CMI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23"><p class="c-article-references__text" id="ref-CR23">Fischl, B., Liu, A. &amp; Dale, A.M. Automated manifold surgery: constructing geometrically accurate and topologically correct models of the human cerebral cortex. <i>IEEE Trans. Med. Imaging</i> <b>20</b>, 7080 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1109/42.906426" data-track-action="article reference" href="https://doi.org/10.1109%2F42.906426" aria-label="Article reference 23" data-doi="10.1109/42.906426">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3MzjvV2ksw%3D%3D" aria-label="CAS reference 23">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11293693" aria-label="PubMed reference 23">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20manifold%20surgery%3A%20constructing%20geometrically%20accurate%20and%20topologically%20correct%20models%20of%20the%20human%20cerebral%20cortex&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;doi=10.1109%2F42.906426&amp;volume=20&amp;pages=70-80&amp;publication_year=2001&amp;author=Fischl%2CB&amp;author=Liu%2CA&amp;author=Dale%2CAM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24"><p class="c-article-references__text" id="ref-CR24">Fischl, B. et al. Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. <i>Neuron</i> <b>33</b>, 341355 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0896-6273(02)00569-X" data-track-action="article reference" href="https://doi.org/10.1016%2FS0896-6273%2802%2900569-X" aria-label="Article reference 24" data-doi="10.1016/S0896-6273(02)00569-X">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XjtlWnsbg%3D" aria-label="CAS reference 24">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11832223" aria-label="PubMed reference 24">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole%20brain%20segmentation%3A%20automated%20labeling%20of%20neuroanatomical%20structures%20in%20the%20human%20brain&amp;journal=Neuron&amp;doi=10.1016%2FS0896-6273%2802%2900569-X&amp;volume=33&amp;pages=341-355&amp;publication_year=2002&amp;author=Fischl%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25"><p class="c-article-references__text" id="ref-CR25">Fischl, B. et al. Sequence-independent segmentation of magnetic resonance images. <i>Neuroimage</i> <b>23</b> (Suppl. 1), S69S84 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2004.07.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2004.07.016" aria-label="Article reference 25" data-doi="10.1016/j.neuroimage.2004.07.016">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15501102" aria-label="PubMed reference 25">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Sequence-independent%20segmentation%20of%20magnetic%20resonance%20images&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2004.07.016&amp;volume=23&amp;issue=Suppl.%201&amp;pages=S69-S84&amp;publication_year=2004&amp;author=Fischl%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26"><p class="c-article-references__text" id="ref-CR26">Fischl, B., Sereno, M.I. &amp; Dale, A.M. Cortical surface-based analysis. II: inflation, flattening, and a surface-based coordinate system. <i>Neuroimage</i> <b>9</b>, 195207 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.1998.0396" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.1998.0396" aria-label="Article reference 26" data-doi="10.1006/nimg.1998.0396">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1M7jt1Gisw%3D%3D" aria-label="CAS reference 26">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9931269" aria-label="PubMed reference 26">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20surface-based%20analysis.%20II%3A%20inflation%2C%20flattening%2C%20and%20a%20surface-based%20coordinate%20system&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.1998.0396&amp;volume=9&amp;pages=195-207&amp;publication_year=1999&amp;author=Fischl%2CB&amp;author=Sereno%2CMI&amp;author=Dale%2CAM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27"><p class="c-article-references__text" id="ref-CR27">Fischl, B., Sereno, M.I., Tootell, R.B. &amp; Dale, A.M. High-resolution intersubject averaging and a coordinate system for the cortical surface. <i>Hum. Brain Mapp.</i> <b>8</b>, 272284 (1999). <b>This study introduced an automated method for aligning cortical surfaces from individuals to a surface-based atlas using cortical folding to constrain the registration</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/(SICI)1097-0193(1999)8:4<272::AID-HBM10&gt;3.0.CO;2-4" data-track-action="article reference" href="https://doi.org/10.1002%2F%28SICI%291097-0193%281999%298%3A4%3C272%3A%3AAID-HBM10%3E3.0.CO%3B2-4" aria-label="Article reference 27" data-doi="10.1002/(SICI)1097-0193(1999)8:4<272::AID-HBM10&gt;3.0.CO;2-4">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3c%2FosVCgug%3D%3D" aria-label="CAS reference 27">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10619420" aria-label="PubMed reference 27">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6873338" aria-label="PubMed Central reference 27">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=High-resolution%20intersubject%20averaging%20and%20a%20coordinate%20system%20for%20the%20cortical%20surface&amp;journal=Hum.%20Brain%20Mapp.&amp;doi=10.1002%2F%28SICI%291097-0193%281999%298%3A4%3C272%3A%3AAID-HBM10%3E3.0.CO%3B2-4&amp;volume=8&amp;pages=272-284&amp;publication_year=1999&amp;author=Fischl%2CB&amp;author=Sereno%2CMI&amp;author=Tootell%2CRB&amp;author=Dale%2CAM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28"><p class="c-article-references__text" id="ref-CR28">Sgonne, F., Pacheco, J. &amp; Fischl, B. Geometrically accurate topology-correction of cortical surfaces using nonseparating loops. <i>IEEE Trans. Med. Imaging</i> <b>26</b>, 518529 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1109/TMI.2006.887364" data-track-action="article reference" href="https://doi.org/10.1109%2FTMI.2006.887364" aria-label="Article reference 28" data-doi="10.1109/TMI.2006.887364">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17427739" aria-label="PubMed reference 28">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Geometrically%20accurate%20topology-correction%20of%20cortical%20surfaces%20using%20nonseparating%20loops&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;doi=10.1109%2FTMI.2006.887364&amp;volume=26&amp;pages=518-529&amp;publication_year=2007&amp;author=S%C3%A9gonne%2CF&amp;author=Pacheco%2CJ&amp;author=Fischl%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29"><p class="c-article-references__text" id="ref-CR29">Barch, D.M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. <i>Neuroimage</i> <b>80</b>, 169189 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.033" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.033" aria-label="Article reference 29" data-doi="10.1016/j.neuroimage.2013.05.033">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23684877" aria-label="PubMed reference 29">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Function%20in%20the%20human%20connectome%3A%20task-fMRI%20and%20individual%20differences%20in%20behavior&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.033&amp;volume=80&amp;pages=169-189&amp;publication_year=2013&amp;author=Barch%2CDM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30"><p class="c-article-references__text" id="ref-CR30">Wandell, B.A. &amp; Winawer, J. Imaging retinotopic maps in the human brain. <i>Vision Res.</i> <b>51</b>, 718737 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.visres.2010.08.004" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.visres.2010.08.004" aria-label="Article reference 30" data-doi="10.1016/j.visres.2010.08.004">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20692278" aria-label="PubMed reference 30">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Imaging%20retinotopic%20maps%20in%20the%20human%20brain&amp;journal=Vision%20Res.&amp;doi=10.1016%2Fj.visres.2010.08.004&amp;volume=51&amp;pages=718-737&amp;publication_year=2011&amp;author=Wandell%2CBA&amp;author=Winawer%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31"><p class="c-article-references__text" id="ref-CR31">Orban, G.A., Zhu, Q. &amp; Vanduffel, W. The transition in the ventral stream from feature to real-world entity representations. <i>Front. Psychol.</i> <b>5</b>, 695 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25071663" aria-label="PubMed reference 31">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4079243" aria-label="PubMed Central reference 31">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20transition%20in%20the%20ventral%20stream%20from%20feature%20to%20real-world%20entity%20representations&amp;journal=Front.%20Psychol.&amp;volume=5&amp;publication_year=2014&amp;author=Orban%2CGA&amp;author=Zhu%2CQ&amp;author=Vanduffel%2CW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32"><p class="c-article-references__text" id="ref-CR32">Brewer, A.A., Press, W.A., Logothetis, N.K. &amp; Wandell, B.A. Visual areas in macaque cortex measured using functional magnetic resonance imaging. <i>J. Neurosci.</i> <b>22</b>, 1041610426 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.22-23-10416.2002" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.22-23-10416.2002" aria-label="Article reference 32" data-doi="10.1523/JNEUROSCI.22-23-10416.2002">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XpsVamsbc%3D" aria-label="CAS reference 32">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12451141" aria-label="PubMed reference 32">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6758759" aria-label="PubMed Central reference 32">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Visual%20areas%20in%20macaque%20cortex%20measured%20using%20functional%20magnetic%20resonance%20imaging&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.22-23-10416.2002&amp;volume=22&amp;pages=10416-10426&amp;publication_year=2002&amp;author=Brewer%2CAA&amp;author=Press%2CWA&amp;author=Logothetis%2CNK&amp;author=Wandell%2CBA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33"><p class="c-article-references__text" id="ref-CR33">Van Essen, D.C., Felleman, D.J., DeYoe, E.A., Olavarria, J. &amp; Knierim, J. Modular and hierarchical organization of extrastriate visual cortex in the macaque monkey. <i>Cold Spring Harb. Symp. Quant. Biol.</i> <b>55</b>, 679696 (1990).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1101/SQB.1990.055.01.064" data-track-action="article reference" href="https://doi.org/10.1101%2FSQB.1990.055.01.064" aria-label="Article reference 33" data-doi="10.1101/SQB.1990.055.01.064">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK387htFWrtw%3D%3D" aria-label="CAS reference 33">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1966771" aria-label="PubMed reference 33">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Modular%20and%20hierarchical%20organization%20of%20extrastriate%20visual%20cortex%20in%20the%20macaque%20monkey&amp;journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol.&amp;doi=10.1101%2FSQB.1990.055.01.064&amp;volume=55&amp;pages=679-696&amp;publication_year=1990&amp;author=Van%20Essen%2CDC&amp;author=Felleman%2CDJ&amp;author=DeYoe%2CEA&amp;author=Olavarria%2CJ&amp;author=Knierim%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34"><p class="c-article-references__text" id="ref-CR34">Calabrese, E., Badea, A., Cofer, G., Qi, Y. &amp; Johnson, G.A. A diffusion MRI tractography connectome of the mouse brain and comparison with neuronal tracer data. <i>Cereb. Cortex</i> <b>25</b>, 46284637 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhv121" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhv121" aria-label="Article reference 34" data-doi="10.1093/cercor/bhv121">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26048951" aria-label="PubMed reference 34">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4715247" aria-label="PubMed Central reference 34">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20diffusion%20MRI%20tractography%20connectome%20of%20the%20mouse%20brain%20and%20comparison%20with%20neuronal%20tracer%20data&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhv121&amp;volume=25&amp;pages=4628-4637&amp;publication_year=2015&amp;author=Calabrese%2CE&amp;author=Badea%2CA&amp;author=Cofer%2CG&amp;author=Qi%2CY&amp;author=Johnson%2CGA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35"><p class="c-article-references__text" id="ref-CR35">Donahue, C.J. et al. Using diffusion tractography to predict cortical connection strength and distance: A quantitative comparison with tracers in the monkey. <i>J. Neurosci</i> <b>36</b>, 67586770 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.0493-16.2016" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.0493-16.2016" aria-label="Article reference 35" data-doi="10.1523/JNEUROSCI.0493-16.2016">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGlsr%2FE" aria-label="CAS reference 35">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27335406" aria-label="PubMed reference 35">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4916250" aria-label="PubMed Central reference 35">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20diffusion%20tractography%20to%20predict%20cortical%20connection%20strength%20and%20distance%3A%20A%20quantitative%20comparison%20with%20tracers%20in%20the%20monkey&amp;journal=J.%20Neurosci&amp;doi=10.1523%2FJNEUROSCI.0493-16.2016&amp;volume=36&amp;pages=6758-6770&amp;publication_year=2016&amp;author=Donahue%2CCJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36"><p class="c-article-references__text" id="ref-CR36">Basser, P.J., Mattiello, J. &amp; LeBihan, D. MR diffusion tensor spectroscopy and imaging. <i>Biophys. J.</i> <b>66</b>, 259267 (1994).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0006-3495(94)80775-1" data-track-action="article reference" href="https://doi.org/10.1016%2FS0006-3495%2894%2980775-1" aria-label="Article reference 36" data-doi="10.1016/S0006-3495(94)80775-1">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK2c7ns1Kgtw%3D%3D" aria-label="CAS reference 36">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8130344" aria-label="PubMed reference 36">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1275686" aria-label="PubMed Central reference 36">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=MR%20diffusion%20tensor%20spectroscopy%20and%20imaging&amp;journal=Biophys.%20J.&amp;doi=10.1016%2FS0006-3495%2894%2980775-1&amp;volume=66&amp;pages=259-267&amp;publication_year=1994&amp;author=Basser%2CPJ&amp;author=Mattiello%2CJ&amp;author=LeBihan%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37"><p class="c-article-references__text" id="ref-CR37">Fieremans, E., Jensen, J.H. &amp; Helpern, J.A. White matter characterization with diffusional kurtosis imaging. <i>Neuroimage</i> <b>58</b>, 177188 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.06.006" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.06.006" aria-label="Article reference 37" data-doi="10.1016/j.neuroimage.2011.06.006">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21699989" aria-label="PubMed reference 37">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=White%20matter%20characterization%20with%20diffusional%20kurtosis%20imaging&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2011.06.006&amp;volume=58&amp;pages=177-188&amp;publication_year=2011&amp;author=Fieremans%2CE&amp;author=Jensen%2CJH&amp;author=Helpern%2CJA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38"><p class="c-article-references__text" id="ref-CR38">Zhang, H., Schneider, T., Wheeler-Kingshott, C.A. &amp; Alexander, D.C. NODDI: practical <i>in vivo</i> neurite orientation dispersion and density imaging of the human brain. <i>Neuroimage</i> <b>61</b>, 10001016 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.03.072" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.03.072" aria-label="Article reference 38" data-doi="10.1016/j.neuroimage.2012.03.072">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22484410" aria-label="PubMed reference 38">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=NODDI%3A%20practical%20in%20vivo%20neurite%20orientation%20dispersion%20and%20density%20imaging%20of%20the%20human%20brain&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.03.072&amp;volume=61&amp;pages=1000-1016&amp;publication_year=2012&amp;author=Zhang%2CH&amp;author=Schneider%2CT&amp;author=Wheeler-Kingshott%2CCA&amp;author=Alexander%2CDC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39"><p class="c-article-references__text" id="ref-CR39">Van Essen, D.C., Glasser, M.F., Dierker, D.L., Harwell, J. &amp; Coalson, T. Parcellations and hemispheric asymmetries of human cerebral cortex analyzed on surface-based atlases. <i>Cereb. Cortex</i> <b>22</b>, 22412262 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhr291" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhr291" aria-label="Article reference 39" data-doi="10.1093/cercor/bhr291">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22047963" aria-label="PubMed reference 39">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Parcellations%20and%20hemispheric%20asymmetries%20of%20human%20cerebral%20cortex%20analyzed%20on%20surface-based%20atlases&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhr291&amp;volume=22&amp;pages=2241-2262&amp;publication_year=2012&amp;author=Van%20Essen%2CDC&amp;author=Glasser%2CMF&amp;author=Dierker%2CDL&amp;author=Harwell%2CJ&amp;author=Coalson%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40"><p class="c-article-references__text" id="ref-CR40">Liewald, D., Miller, R., Logothetis, N., Wagner, H.J. &amp; Schz, A. Distribution of axon diameters in cortical white matter: an electron-microscopic study on three human brains and a macaque. <i>Biol. Cybern.</i> <b>108</b>, 541557 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1007/s00422-014-0626-2" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s00422-014-0626-2" aria-label="Article reference 40" data-doi="10.1007/s00422-014-0626-2">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25142940" aria-label="PubMed reference 40">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4228120" aria-label="PubMed Central reference 40">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Distribution%20of%20axon%20diameters%20in%20cortical%20white%20matter%3A%20an%20electron-microscopic%20study%20on%20three%20human%20brains%20and%20a%20macaque&amp;journal=Biol.%20Cybern.&amp;doi=10.1007%2Fs00422-014-0626-2&amp;volume=108&amp;pages=541-557&amp;publication_year=2014&amp;author=Liewald%2CD&amp;author=Miller%2CR&amp;author=Logothetis%2CN&amp;author=Wagner%2CHJ&amp;author=Sch%C3%BCz%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41"><p class="c-article-references__text" id="ref-CR41">Van Essen, D.C. &amp; Ugurbil, K. The future of the human connectome. <i>Neuroimage</i> <b>62</b>, 12991310 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.01.032" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.01.032" aria-label="Article reference 41" data-doi="10.1016/j.neuroimage.2012.01.032">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC38vgsVSkuw%3D%3D" aria-label="CAS reference 41">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22245355" aria-label="PubMed reference 41">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20future%20of%20the%20human%20connectome&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.01.032&amp;volume=62&amp;pages=1299-1310&amp;publication_year=2012&amp;author=Van%20Essen%2CDC&amp;author=Ugurbil%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42"><p class="c-article-references__text" id="ref-CR42">Huang, S.Y. et al. The impact of gradient strength on <i>in vivo</i> diffusion MRI estimates of axon diameter. <i>Neuroimage</i> <b>106</b>, 464472 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.12.008" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.12.008" aria-label="Article reference 42" data-doi="10.1016/j.neuroimage.2014.12.008">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25498429" aria-label="PubMed reference 42">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20gradient%20strength%20on%20in%20vivo%20diffusion%20MRI%20estimates%20of%20axon%20diameter&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.12.008&amp;volume=106&amp;pages=464-472&amp;publication_year=2015&amp;author=Huang%2CSY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43"><p class="c-article-references__text" id="ref-CR43">McNab, J.A. et al. The Human Connectome Project and beyond: initial applications of 300 mT/m gradients. <i>Neuroimage</i> <b>80</b>, 234245 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.074" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.074" aria-label="Article reference 43" data-doi="10.1016/j.neuroimage.2013.05.074">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23711537" aria-label="PubMed reference 43">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Human%20Connectome%20Project%20and%20beyond%3A%20initial%20applications%20of%20300%20mT%2Fm%20gradients&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.074&amp;volume=80&amp;pages=234-245&amp;publication_year=2013&amp;author=McNab%2CJA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44"><p class="c-article-references__text" id="ref-CR44">Setsompop, K. et al. Blipped-controlled aliasing in parallel imaging for simultaneous multislice echo planar imaging with reduced g-factor penalty. <i>Magn. Reson. Med.</i> <b>67</b>, 12101224 (2012). <b>This study introduced a slice gradient blip technique for simultaneous multislice (multiband) echo-planar imaging to significantly reduce image artifacts in unaliasing of simultaneously acquired slices, particularly for high multiband factors. This technique is adopted in the HCP pulse sequences for fMRI and dMRI</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.23097" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.23097" aria-label="Article reference 44" data-doi="10.1002/mrm.23097">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21858868" aria-label="PubMed reference 44">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Blipped-controlled%20aliasing%20in%20parallel%20imaging%20for%20simultaneous%20multislice%20echo%20planar%20imaging%20with%20reduced%20g-factor%20penalty&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.23097&amp;volume=67&amp;pages=1210-1224&amp;publication_year=2012&amp;author=Setsompop%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45"><p class="c-article-references__text" id="ref-CR45">Setsompop, K. et al. Pushing the limits of <i>in vivo</i> diffusion MRI for the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 220233 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.078" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.078" aria-label="Article reference 45" data-doi="10.1016/j.neuroimage.2013.05.078">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC3snmvVWmsQ%3D%3D" aria-label="CAS reference 45">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23707579" aria-label="PubMed reference 45">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Pushing%20the%20limits%20of%20in%20vivo%20diffusion%20MRI%20for%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.078&amp;volume=80&amp;pages=220-233&amp;publication_year=2013&amp;author=Setsompop%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46"><p class="c-article-references__text" id="ref-CR46">Sotiropoulos, S.N. et al. Fusion in diffusion MRI for improved fibre orientation estimation: An application to the 3T and 7T data of the Human Connectome Project. <i>Neuroimage</i> <b>134</b>, 396409 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2016.04.014" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2016.04.014" aria-label="Article reference 46" data-doi="10.1016/j.neuroimage.2016.04.014">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27071694" aria-label="PubMed reference 46">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Fusion%20in%20diffusion%20MRI%20for%20improved%20fibre%20orientation%20estimation%3A%20An%20application%20to%20the%203T%20and%207T%20data%20of%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2016.04.014&amp;volume=134&amp;pages=396-409&amp;publication_year=2016&amp;author=Sotiropoulos%2CSN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47"><p class="c-article-references__text" id="ref-CR47">Sotiropoulos, S.N. et al. Advances in diffusion MRI acquisition and processing in the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 125143 (2013). <b>The authors report on advances in acquisition and analysis of HCP diffusion imaging data. This includes use of scanners with high maximal gradient strength, multiband pulse sequences, and improvements in image reconstruction, distortion correction, fiber orientation modeling and probabilistic tractography algorithms</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.057" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.057" aria-label="Article reference 47" data-doi="10.1016/j.neuroimage.2013.05.057">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23702418" aria-label="PubMed reference 47">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Advances%20in%20diffusion%20MRI%20acquisition%20and%20processing%20in%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.057&amp;volume=80&amp;pages=125-143&amp;publication_year=2013&amp;author=Sotiropoulos%2CSN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48"><p class="c-article-references__text" id="ref-CR48">Vu, A.T. et al. High resolution whole brain diffusion imaging at 7T for the Human Connectome Project. <i>Neuroimage</i> <b>122</b>, 318331 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.08.004" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.08.004" aria-label="Article reference 48" data-doi="10.1016/j.neuroimage.2015.08.004">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC287hsF2ntQ%3D%3D" aria-label="CAS reference 48">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26260428" aria-label="PubMed reference 48">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20resolution%20whole%20brain%20diffusion%20imaging%20at%207T%20for%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.08.004&amp;volume=122&amp;pages=318-331&amp;publication_year=2015&amp;author=Vu%2CAT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49"><p class="c-article-references__text" id="ref-CR49">Hillman, E.M. Coupling mechanism and significance of the BOLD signal: a status report. <i>Annu. Rev. Neurosci.</i> <b>37</b>, 161181 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1146/annurev-neuro-071013-014111" data-track-action="article reference" href="https://doi.org/10.1146%2Fannurev-neuro-071013-014111" aria-label="Article reference 49" data-doi="10.1146/annurev-neuro-071013-014111">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVKgt7fL" aria-label="CAS reference 49">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25032494" aria-label="PubMed reference 49">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4147398" aria-label="PubMed Central reference 49">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Coupling%20mechanism%20and%20significance%20of%20the%20BOLD%20signal%3A%20a%20status%20report&amp;journal=Annu.%20Rev.%20Neurosci.&amp;doi=10.1146%2Fannurev-neuro-071013-014111&amp;volume=37&amp;pages=161-181&amp;publication_year=2014&amp;author=Hillman%2CEM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50"><p class="c-article-references__text" id="ref-CR50">Feinberg, D.A. et al. Multiplexed echo planar imaging for sub-second whole brain FMRI and fast diffusion imaging. <i>PLoS One</i> <b>5</b>, e15710 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0015710" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0015710" aria-label="Article reference 50" data-doi="10.1371/journal.pone.0015710">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXnsVGh" aria-label="CAS reference 50">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21187930" aria-label="PubMed reference 50">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3004955" aria-label="PubMed Central reference 50">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20echo%20planar%20imaging%20for%20sub-second%20whole%20brain%20FMRI%20and%20fast%20diffusion%20imaging&amp;journal=PLoS%20One&amp;doi=10.1371%2Fjournal.pone.0015710&amp;volume=5&amp;publication_year=2010&amp;author=Feinberg%2CDA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51"><p class="c-article-references__text" id="ref-CR51">Chen, J.E. &amp; Glover, G.H. BOLD fractional contribution to resting-state functional connectivity above 0.1 Hz. <i>Neuroimage</i> <b>107</b>, 207218 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.12.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.12.012" aria-label="Article reference 51" data-doi="10.1016/j.neuroimage.2014.12.012">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25497686" aria-label="PubMed reference 51">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=BOLD%20fractional%20contribution%20to%20resting-state%20functional%20connectivity%20above%200.1%20Hz&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.12.012&amp;volume=107&amp;pages=207-218&amp;publication_year=2015&amp;author=Chen%2CJE&amp;author=Glover%2CGH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52"><p class="c-article-references__text" id="ref-CR52">Niazy, R.K., Xie, J., Miller, K., Beckmann, C.F. &amp; Smith, S.M. Spectral characteristics of resting state networks. <i>Prog. Brain Res.</i> <b>193</b>, 259276 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/B978-0-444-53839-0.00017-X" data-track-action="article reference" href="https://doi.org/10.1016%2FB978-0-444-53839-0.00017-X" aria-label="Article reference 52" data-doi="10.1016/B978-0-444-53839-0.00017-X">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21854968" aria-label="PubMed reference 52">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Spectral%20characteristics%20of%20resting%20state%20networks&amp;journal=Prog.%20Brain%20Res.&amp;doi=10.1016%2FB978-0-444-53839-0.00017-X&amp;volume=193&amp;pages=259-276&amp;publication_year=2011&amp;author=Niazy%2CRK&amp;author=Xie%2CJ&amp;author=Miller%2CK&amp;author=Beckmann%2CCF&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53"><p class="c-article-references__text" id="ref-CR53">Auerbach, E.J., Xu, J., Yacoub, E., Moeller, S. &amp; Ugurbil, K. Multiband accelerated spin-echo echo planar imaging with reduced peak RF power using time-shifted RF pulses. <i>Magn. Reson. Med.</i> <b>69</b>, 12611267 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.24719" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.24719" aria-label="Article reference 53" data-doi="10.1002/mrm.24719">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23468087" aria-label="PubMed reference 53">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3769699" aria-label="PubMed Central reference 53">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiband%20accelerated%20spin-echo%20echo%20planar%20imaging%20with%20reduced%20peak%20RF%20power%20using%20time-shifted%20RF%20pulses&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.24719&amp;volume=69&amp;pages=1261-1267&amp;publication_year=2013&amp;author=Auerbach%2CEJ&amp;author=Xu%2CJ&amp;author=Yacoub%2CE&amp;author=Moeller%2CS&amp;author=Ug%CB%87urbil%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54"><p class="c-article-references__text" id="ref-CR54">Moeller, S. et al. Multiband multislice GE-EPI at 7 tesla, with 16-fold acceleration using partial parallel imaging with application to high spatial and temporal whole-brain fMRI. <i>Magn. Reson. Med.</i> <b>63</b>, 11441153 (2010). <b>This study demonstrated the feasibility of improving the spatial and temporal resolution of whole-brain EPI</b>-<b>based fMRI by slice multiplexing, in which multiple slices are excited and acquired simultaneously, thus sharing a common contrast and common spatial encoding periods. Multiband EPI is a key strength of the HCP fMRI and dMRI pulse sequences</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.22361" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.22361" aria-label="Article reference 54" data-doi="10.1002/mrm.22361">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20432285" aria-label="PubMed reference 54">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2906244" aria-label="PubMed Central reference 54">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiband%20multislice%20GE-EPI%20at%207%20tesla%2C%20with%2016-fold%20acceleration%20using%20partial%20parallel%20imaging%20with%20application%20to%20high%20spatial%20and%20temporal%20whole-brain%20fMRI&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.22361&amp;volume=63&amp;pages=1144-1153&amp;publication_year=2010&amp;author=Moeller%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55"><p class="c-article-references__text" id="ref-CR55">Xu, J. et al. Evaluation of slice accelerations using multiband echo planar imaging at 3 T. <i>Neuroimage</i> <b>83</b>, 9911001 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.07.055" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.07.055" aria-label="Article reference 55" data-doi="10.1016/j.neuroimage.2013.07.055">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23899722" aria-label="PubMed reference 55">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluation%20of%20slice%20accelerations%20using%20multiband%20echo%20planar%20imaging%20at%203%20T&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.07.055&amp;volume=83&amp;pages=991-1001&amp;publication_year=2013&amp;author=Xu%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56"><p class="c-article-references__text" id="ref-CR56">Laumann, T.O. et al. Functional system and areal organization of a highly sampled individual human brain. <i>Neuron</i> <b>87</b>, 657670 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2015.06.037" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.06.037" aria-label="Article reference 56" data-doi="10.1016/j.neuron.2015.06.037">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1ChsLrP" aria-label="CAS reference 56">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26212711" aria-label="PubMed reference 56">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4642864" aria-label="PubMed Central reference 56">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20system%20and%20areal%20organization%20of%20a%20highly%20sampled%20individual%20human%20brain&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2015.06.037&amp;volume=87&amp;pages=657-670&amp;publication_year=2015&amp;author=Laumann%2CTO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57"><p class="c-article-references__text" id="ref-CR57">Jbabdi, S., Sotiropoulos, S.N., Savio, A.M., Graa, M. &amp; Behrens, T.E. Model-based analysis of multishell diffusion MR data for tractography: how to get over fitting problems. <i>Magn. Reson. Med.</i> <b>68</b>, 18461855 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.24204" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.24204" aria-label="Article reference 57" data-doi="10.1002/mrm.24204">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22334356" aria-label="PubMed reference 57">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3359399" aria-label="PubMed Central reference 57">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Model-based%20analysis%20of%20multishell%20diffusion%20MR%20data%20for%20tractography%3A%20how%20to%20get%20over%20fitting%20problems&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.24204&amp;volume=68&amp;pages=1846-1855&amp;publication_year=2012&amp;author=Jbabdi%2CS&amp;author=Sotiropoulos%2CSN&amp;author=Savio%2CAM&amp;author=Gra%C3%B1a%2CM&amp;author=Behrens%2CTE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58"><p class="c-article-references__text" id="ref-CR58">Jeurissen, B., Tournier, J.D., Dhollander, T., Connelly, A. &amp; Sijbers, J. Multi-tissue constrained spherical deconvolution for improved analysis of multi-shell diffusion MRI data. <i>Neuroimage</i> <b>103</b>, 411426 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.07.061" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.07.061" aria-label="Article reference 58" data-doi="10.1016/j.neuroimage.2014.07.061">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25109526" aria-label="PubMed reference 58">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-tissue%20constrained%20spherical%20deconvolution%20for%20improved%20analysis%20of%20multi-shell%20diffusion%20MRI%20data&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.07.061&amp;volume=103&amp;pages=411-426&amp;publication_year=2014&amp;author=Jeurissen%2CB&amp;author=Tournier%2CJD&amp;author=Dhollander%2CT&amp;author=Connelly%2CA&amp;author=Sijbers%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59"><p class="c-article-references__text" id="ref-CR59">Greve, D.N. &amp; Fischl, B. Accurate and robust brain image alignment using boundary-based registration. <i>Neuroimage</i> <b>48</b>, 6372 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2009.06.060" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2009.06.060" aria-label="Article reference 59" data-doi="10.1016/j.neuroimage.2009.06.060">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19573611" aria-label="PubMed reference 59">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20and%20robust%20brain%20image%20alignment%20using%20boundary-based%20registration&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2009.06.060&amp;volume=48&amp;pages=63-72&amp;publication_year=2009&amp;author=Greve%2CDN&amp;author=Fischl%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60"><p class="c-article-references__text" id="ref-CR60">Andersson, J.L. &amp; Sotiropoulos, S.N. Non-parametric representation and prediction of single- and multi-shell diffusion-weighted MRI data using Gaussian processes. <i>Neuroimage</i> <b>122</b>, 166176 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.07.067" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.07.067" aria-label="Article reference 60" data-doi="10.1016/j.neuroimage.2015.07.067">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26236030" aria-label="PubMed reference 60">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Non-parametric%20representation%20and%20prediction%20of%20single-%20and%20multi-shell%20diffusion-weighted%20MRI%20data%20using%20Gaussian%20processes&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.07.067&amp;volume=122&amp;pages=166-176&amp;publication_year=2015&amp;author=Andersson%2CJL&amp;author=Sotiropoulos%2CSN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61"><p class="c-article-references__text" id="ref-CR61">Andersson, J.L. &amp; Sotiropoulos, S.N. An integrated approach to correction for off-resonance effects and subject movement in diffusion MR imaging. <i>Neuroimage</i> <b>125</b>, 10631078 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.10.019" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.10.019" aria-label="Article reference 61" data-doi="10.1016/j.neuroimage.2015.10.019">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26481672" aria-label="PubMed reference 61">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20integrated%20approach%20to%20correction%20for%20off-resonance%20effects%20and%20subject%20movement%20in%20diffusion%20MR%20imaging&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.10.019&amp;volume=125&amp;pages=1063-1078&amp;publication_year=2016&amp;author=Andersson%2CJL&amp;author=Sotiropoulos%2CSN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62"><p class="c-article-references__text" id="ref-CR62">Andersson, J.L.R., Graham, M.S., Zsoldos, E. &amp; Sotiropoulos, S.N. Incorporating outlier detection and replacement into a non-parameteric framework for movement and distortion correction of diffusion MR images. <i>Neuroimage</i> j.neuroimage.2016.06.058 (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63"><p class="c-article-references__text" id="ref-CR63">Power, J.D., Schlaggar, B.L. &amp; Petersen, S.E. Recent progress and outstanding issues in motion correction in resting state fMRI. <i>Neuroimage</i> <b>105</b>, 536551 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.10.044" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.10.044" aria-label="Article reference 63" data-doi="10.1016/j.neuroimage.2014.10.044">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25462692" aria-label="PubMed reference 63">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Recent%20progress%20and%20outstanding%20issues%20in%20motion%20correction%20in%20resting%20state%20fMRI&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.10.044&amp;volume=105&amp;pages=536-551&amp;publication_year=2015&amp;author=Power%2CJD&amp;author=Schlaggar%2CBL&amp;author=Petersen%2CSE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64"><p class="c-article-references__text" id="ref-CR64">Power, J.D., Barnes, K.A., Snyder, A.Z., Schlaggar, B.L. &amp; Petersen, S.E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. <i>Neuroimage</i> <b>59</b>, 21422154 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.10.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.10.018" aria-label="Article reference 64" data-doi="10.1016/j.neuroimage.2011.10.018">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22019881" aria-label="PubMed reference 64">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Spurious%20but%20systematic%20correlations%20in%20functional%20connectivity%20MRI%20networks%20arise%20from%20subject%20motion&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2011.10.018&amp;volume=59&amp;pages=2142-2154&amp;publication_year=2012&amp;author=Power%2CJD&amp;author=Barnes%2CKA&amp;author=Snyder%2CAZ&amp;author=Schlaggar%2CBL&amp;author=Petersen%2CSE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65"><p class="c-article-references__text" id="ref-CR65">Satterthwaite, T.D. et al. Impact of in-scanner head motion on multiple measures of functional connectivity: relevance for studies of neurodevelopment in youth. <i>Neuroimage</i> <b>60</b>, 623632 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.12.063" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.12.063" aria-label="Article reference 65" data-doi="10.1016/j.neuroimage.2011.12.063">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22233733" aria-label="PubMed reference 65">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Impact%20of%20in-scanner%20head%20motion%20on%20multiple%20measures%20of%20functional%20connectivity%3A%20relevance%20for%20studies%20of%20neurodevelopment%20in%20youth&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2011.12.063&amp;volume=60&amp;pages=623-632&amp;publication_year=2012&amp;author=Satterthwaite%2CTD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66"><p class="c-article-references__text" id="ref-CR66">Beckmann, C.F. &amp; Smith, S.M. Probabilistic independent component analysis for functional magnetic resonance imaging. <i>IEEE Trans. Med. Imaging</i> <b>23</b>, 137152 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1109/TMI.2003.822821" data-track-action="article reference" href="https://doi.org/10.1109%2FTMI.2003.822821" aria-label="Article reference 66" data-doi="10.1109/TMI.2003.822821">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14964560" aria-label="PubMed reference 66">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Probabilistic%20independent%20component%20analysis%20for%20functional%20magnetic%20resonance%20imaging&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;doi=10.1109%2FTMI.2003.822821&amp;volume=23&amp;pages=137-152&amp;publication_year=2004&amp;author=Beckmann%2CCF&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67"><p class="c-article-references__text" id="ref-CR67">Winkler, A.M., Webster, M.A., Vidaurre, D., Nichols, T.E. &amp; Smith, S.M. Multi-level block permutation. <i>Neuroimage</i> <b>123</b>, 253268 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.05.092" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.05.092" aria-label="Article reference 67" data-doi="10.1016/j.neuroimage.2015.05.092">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26074200" aria-label="PubMed reference 67">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-level%20block%20permutation&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.05.092&amp;volume=123&amp;pages=253-268&amp;publication_year=2015&amp;author=Winkler%2CAM&amp;author=Webster%2CMA&amp;author=Vidaurre%2CD&amp;author=Nichols%2CTE&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68"><p class="c-article-references__text" id="ref-CR68">Zhang, H. et al. High-dimensional spatial normalization of diffusion tensor images improves the detection of white matter differences: an example study using amyotrophic lateral sclerosis. <i>IEEE Trans. Med. Imaging</i> <b>26</b>, 15851597 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1109/TMI.2007.906784" data-track-action="article reference" href="https://doi.org/10.1109%2FTMI.2007.906784" aria-label="Article reference 68" data-doi="10.1109/TMI.2007.906784">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXht1alt7zE" aria-label="CAS reference 68">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18041273" aria-label="PubMed reference 68">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=High-dimensional%20spatial%20normalization%20of%20diffusion%20tensor%20images%20improves%20the%20detection%20of%20white%20matter%20differences%3A%20an%20example%20study%20using%20amyotrophic%20lateral%20sclerosis&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;doi=10.1109%2FTMI.2007.906784&amp;volume=26&amp;pages=1585-1597&amp;publication_year=2007&amp;author=Zhang%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69"><p class="c-article-references__text" id="ref-CR69">Zhang, H., Yushkevich, P.A., Alexander, D.C. &amp; Gee, J.C. Deformable registration of diffusion tensor MR images with explicit orientation optimization. <i>Med. Image Anal.</i> <b>10</b>, 764785 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.media.2006.06.004" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.media.2006.06.004" aria-label="Article reference 69" data-doi="10.1016/j.media.2006.06.004">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16899392" aria-label="PubMed reference 69">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Deformable%20registration%20of%20diffusion%20tensor%20MR%20images%20with%20explicit%20orientation%20optimization&amp;journal=Med.%20Image%20Anal.&amp;doi=10.1016%2Fj.media.2006.06.004&amp;volume=10&amp;pages=764-785&amp;publication_year=2006&amp;author=Zhang%2CH&amp;author=Yushkevich%2CPA&amp;author=Alexander%2CDC&amp;author=Gee%2CJC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70"><p class="c-article-references__text" id="ref-CR70">Amunts, K., Malikovic, A., Mohlberg, H., Schormann, T. &amp; Zilles, K. Brodmann's areas 17 and 18 brought into stereotaxic space-where and how variable? <i>Neuroimage</i> <b>11</b>, 6684 (2000). <b>This study (and many others from the same laboratory) used objective observer-independent methods to identify cytoarchitectonic areas in post-mortem human brains</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.1999.0516" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.1999.0516" aria-label="Article reference 70" data-doi="10.1006/nimg.1999.0516">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3c7ltl2ksA%3D%3D" aria-label="CAS reference 70">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10686118" aria-label="PubMed reference 70">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Brodmann%27s%20areas%2017%20and%2018%20brought%20into%20stereotaxic%20space-where%20and%20how%20variable%3F&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.1999.0516&amp;volume=11&amp;pages=66-84&amp;publication_year=2000&amp;author=Amunts%2CK&amp;author=Malikovic%2CA&amp;author=Mohlberg%2CH&amp;author=Schormann%2CT&amp;author=Zilles%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71"><p class="c-article-references__text" id="ref-CR71">Eickhoff, S.B. et al. A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. <i>Neuroimage</i> <b>25</b>, 13251335 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2004.12.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2004.12.034" aria-label="Article reference 71" data-doi="10.1016/j.neuroimage.2004.12.034">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15850749" aria-label="PubMed reference 71">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20SPM%20toolbox%20for%20combining%20probabilistic%20cytoarchitectonic%20maps%20and%20functional%20imaging%20data&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2004.12.034&amp;volume=25&amp;pages=1325-1335&amp;publication_year=2005&amp;author=Eickhoff%2CSB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72"><p class="c-article-references__text" id="ref-CR72">Van Essen, D.C. A population-average, landmark- and surface-based (PALS) atlas of human cerebral cortex. <i>Neuroimage</i> <b>28</b>, 635662 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2005.06.058" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2005.06.058" aria-label="Article reference 72" data-doi="10.1016/j.neuroimage.2005.06.058">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16172003" aria-label="PubMed reference 72">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20population-average%2C%20landmark-%20and%20surface-based%20%28PALS%29%20atlas%20of%20human%20cerebral%20cortex&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2005.06.058&amp;volume=28&amp;pages=635-662&amp;publication_year=2005&amp;author=Van%20Essen%2CDC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73"><p class="c-article-references__text" id="ref-CR73">Anticevic, A. et al. Comparing surface-based and volume-based analyses of functional neuroimaging data in patients with schizophrenia. <i>Neuroimage</i> <b>41</b>, 835848 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2008.02.052" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2008.02.052" aria-label="Article reference 73" data-doi="10.1016/j.neuroimage.2008.02.052">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18434199" aria-label="PubMed reference 73">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparing%20surface-based%20and%20volume-based%20analyses%20of%20functional%20neuroimaging%20data%20in%20patients%20with%20schizophrenia&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2008.02.052&amp;volume=41&amp;pages=835-848&amp;publication_year=2008&amp;author=Anticevic%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74"><p class="c-article-references__text" id="ref-CR74">Fischl, B. et al. Cortical folding patterns and predicting cytoarchitecture. <i>Cereb. Cortex</i> <b>18</b>, 19731980 (2008). <b>The authors used folding-based alignment to register cytoarchitectonic areas to a surface-based atlas. In the resultant probabilistic maps, alignment was high for early sensory areas but much lower for 'higher' cortical areas, reflecting regional differences in the correlation between areal boundaries and cortical folding</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhm225" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhm225" aria-label="Article reference 74" data-doi="10.1093/cercor/bhm225">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18079129" aria-label="PubMed reference 74">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20folding%20patterns%20and%20predicting%20cytoarchitecture&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhm225&amp;volume=18&amp;pages=1973-1980&amp;publication_year=2008&amp;author=Fischl%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75"><p class="c-article-references__text" id="ref-CR75">Frost, M.A. &amp; Goebel, R. Measuring structural-functional correspondence: spatial variability of specialised brain regions after macro-anatomical alignment. <i>Neuroimage</i> <b>59</b>, 13691381 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.08.035" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.08.035" aria-label="Article reference 75" data-doi="10.1016/j.neuroimage.2011.08.035">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21875671" aria-label="PubMed reference 75">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20structural-functional%20correspondence%3A%20spatial%20variability%20of%20specialised%20brain%20regions%20after%20macro-anatomical%20alignment&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2011.08.035&amp;volume=59&amp;pages=1369-1381&amp;publication_year=2012&amp;author=Frost%2CMA&amp;author=Goebel%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76"><p class="c-article-references__text" id="ref-CR76">Smith, S.M. et al. Resting-state fMRI in the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 144168 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.039" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.039" aria-label="Article reference 76" data-doi="10.1016/j.neuroimage.2013.05.039">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23702415" aria-label="PubMed reference 76">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting-state%20fMRI%20in%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.039&amp;volume=80&amp;pages=144-168&amp;publication_year=2013&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77"><p class="c-article-references__text" id="ref-CR77">Tucholka, A., Fritsch, V., Poline, J.B. &amp; Thirion, B. An empirical comparison of surface-based and volume-based group studies in neuroimaging. <i>Neuroimage</i> <b>63</b>, 14431453 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.06.019" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.06.019" aria-label="Article reference 77" data-doi="10.1016/j.neuroimage.2012.06.019">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22732555" aria-label="PubMed reference 77">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20empirical%20comparison%20of%20surface-based%20and%20volume-based%20group%20studies%20in%20neuroimaging&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.06.019&amp;volume=63&amp;pages=1443-1453&amp;publication_year=2012&amp;author=Tucholka%2CA&amp;author=Fritsch%2CV&amp;author=Poline%2CJB&amp;author=Thirion%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78"><p class="c-article-references__text" id="ref-CR78">Felleman, D.J. &amp; Van Essen, D.C. Distributed hierarchical processing in the primate cerebral cortex. <i>Cereb. Cortex</i> <b>1</b>, 147 (1991).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/1.1.1" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2F1.1.1" aria-label="Article reference 78" data-doi="10.1093/cercor/1.1.1">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK38zltlGmsg%3D%3D" aria-label="CAS reference 78">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1822724" aria-label="PubMed reference 78">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Distributed%20hierarchical%20processing%20in%20the%20primate%20cerebral%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2F1.1.1&amp;volume=1&amp;pages=1-47&amp;publication_year=1991&amp;author=Felleman%2CDJ&amp;author=Van%20Essen%2CDC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79"><p class="c-article-references__text" id="ref-CR79">Haxby, J.V. et al. A common, high-dimensional model of the representational space in human ventral temporal cortex. <i>Neuron</i> <b>72</b>, 404416 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2011.08.026" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2011.08.026" aria-label="Article reference 79" data-doi="10.1016/j.neuron.2011.08.026">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtlKrtLzJ" aria-label="CAS reference 79">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22017997" aria-label="PubMed reference 79">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3201764" aria-label="PubMed Central reference 79">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20common%2C%20high-dimensional%20model%20of%20the%20representational%20space%20in%20human%20ventral%20temporal%20cortex&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2011.08.026&amp;volume=72&amp;pages=404-416&amp;publication_year=2011&amp;author=Haxby%2CJV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80"><p class="c-article-references__text" id="ref-CR80">Glasser, M.F. &amp; Rilling, J.K. DTI tractography of the human brain's language pathways. <i>Cereb. Cortex</i> <b>18</b>, 24712482 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhn011" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhn011" aria-label="Article reference 80" data-doi="10.1093/cercor/bhn011">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18281301" aria-label="PubMed reference 80">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=DTI%20tractography%20of%20the%20human%20brain%27s%20language%20pathways&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhn011&amp;volume=18&amp;pages=2471-2482&amp;publication_year=2008&amp;author=Glasser%2CMF&amp;author=Rilling%2CJK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81"><p class="c-article-references__text" id="ref-CR81">Brodmann, K. <i>Vergleichende Lokalisationslehre der Grosshirnrinde</i> (Leipzig, 1909).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82"><p class="c-article-references__text" id="ref-CR82">Nieuwenhuys, R. The myeloarchitectonic studies on the human cerebral cortex of the Vogt-Vogt school, and their significance for the interpretation of functional neuroimaging data. <i>Brain Struct. Funct.</i> <b>218</b>, 303352 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1007/s00429-012-0460-z" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s00429-012-0460-z" aria-label="Article reference 82" data-doi="10.1007/s00429-012-0460-z">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23076375" aria-label="PubMed reference 82">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20myeloarchitectonic%20studies%20on%20the%20human%20cerebral%20cortex%20of%20the%20Vogt-Vogt%20school%2C%20and%20their%20significance%20for%20the%20interpretation%20of%20functional%20neuroimaging%20data&amp;journal=Brain%20Struct.%20Funct.&amp;doi=10.1007%2Fs00429-012-0460-z&amp;volume=218&amp;pages=303-352&amp;publication_year=2013&amp;author=Nieuwenhuys%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83"><p class="c-article-references__text" id="ref-CR83">Vogt, C. &amp; Vogt, O. Allgemeinere Ergebnisse unserer Hirnforschung. <i>J. Psychol. Neurol.</i> <b>25</b>, 279468 (1919).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Allgemeinere%20Ergebnisse%20unserer%20Hirnforschung&amp;journal=J.%20Psychol.%20Neurol.&amp;volume=25&amp;pages=279-468&amp;publication_year=1919&amp;author=Vogt%2CC&amp;author=Vogt%2CO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84"><p class="c-article-references__text" id="ref-CR84">Nieuwenhuys, R., Broere, C.A. &amp; Cerliani, L. A new myeloarchitectonic map of the human neocortex based on data from the Vogt-Vogt school. <i>Brain Struct. Funct.</i> <b>220</b>, 25512573 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1007/s00429-014-0806-9" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s00429-014-0806-9" aria-label="Article reference 84" data-doi="10.1007/s00429-014-0806-9">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24924165" aria-label="PubMed reference 84">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20myeloarchitectonic%20map%20of%20the%20human%20neocortex%20based%20on%20data%20from%20the%20Vogt-Vogt%20school&amp;journal=Brain%20Struct.%20Funct.&amp;doi=10.1007%2Fs00429-014-0806-9&amp;volume=220&amp;pages=2551-2573&amp;publication_year=2015&amp;author=Nieuwenhuys%2CR&amp;author=Broere%2CCA&amp;author=Cerliani%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85"><p class="c-article-references__text" id="ref-CR85">Gordon, E.M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. <i>Cereb. Cortex</i> <b>26</b>, 288303 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhu239" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhu239" aria-label="Article reference 85" data-doi="10.1093/cercor/bhu239">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25316338" aria-label="PubMed reference 85">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20and%20evaluation%20of%20a%20cortical%20area%20parcellation%20from%20resting-state%20correlations&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhu239&amp;volume=26&amp;pages=288-303&amp;publication_year=2016&amp;author=Gordon%2CEM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86"><p class="c-article-references__text" id="ref-CR86">Power, J.D. et al. Functional network organization of the human brain. <i>Neuron</i> <b>72</b>, 665678 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2011.09.006" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2011.09.006" aria-label="Article reference 86" data-doi="10.1016/j.neuron.2011.09.006">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsV2rtrnN" aria-label="CAS reference 86">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22099467" aria-label="PubMed reference 86">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3222858" aria-label="PubMed Central reference 86">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20network%20organization%20of%20the%20human%20brain&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2011.09.006&amp;volume=72&amp;pages=665-678&amp;publication_year=2011&amp;author=Power%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87"><p class="c-article-references__text" id="ref-CR87">Yeo, B.T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. <i>J. Neurophysiol.</i> <b>106</b>, 11251165 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1152/jn.00338.2011" data-track-action="article reference" href="https://doi.org/10.1152%2Fjn.00338.2011" aria-label="Article reference 87" data-doi="10.1152/jn.00338.2011">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21653723" aria-label="PubMed reference 87">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20organization%20of%20the%20human%20cerebral%20cortex%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=J.%20Neurophysiol.&amp;doi=10.1152%2Fjn.00338.2011&amp;volume=106&amp;pages=1125-1165&amp;publication_year=2011&amp;author=Yeo%2CBT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88"><p class="c-article-references__text" id="ref-CR88">Hacker, C.D. et al. Resting state network estimation in individual subjects. <i>Neuroimage</i> <b>82</b>, 616633 (2013). <b>This study used a supervised classifier (multilayer perceptron) to identify resting-state networks in individual subjects. Classification was reliable in individuals not used for classifier training and generated more spatially specific resting-state networks than alternative methods tested</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.108" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.108" aria-label="Article reference 88" data-doi="10.1016/j.neuroimage.2013.05.108">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23735260" aria-label="PubMed reference 88">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting%20state%20network%20estimation%20in%20individual%20subjects&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.108&amp;volume=82&amp;pages=616-633&amp;publication_year=2013&amp;author=Hacker%2CCD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89"><p class="c-article-references__text" id="ref-CR89">Cole, M.W., Bassett, D.S., Power, J.D., Braver, T.S. &amp; Petersen, S.E. Intrinsic and task-evoked network architectures of the human brain. <i>Neuron</i> <b>83</b>, 238251 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2014.05.014" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2014.05.014" aria-label="Article reference 89" data-doi="10.1016/j.neuron.2014.05.014">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFWgt7bL" aria-label="CAS reference 89">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24991964" aria-label="PubMed reference 89">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4082806" aria-label="PubMed Central reference 89">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=Intrinsic%20and%20task-evoked%20network%20architectures%20of%20the%20human%20brain&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2014.05.014&amp;volume=83&amp;pages=238-251&amp;publication_year=2014&amp;author=Cole%2CMW&amp;author=Bassett%2CDS&amp;author=Power%2CJD&amp;author=Braver%2CTS&amp;author=Petersen%2CSE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90"><p class="c-article-references__text" id="ref-CR90">Fischl, B. et al. Automatically parcellating the human cerebral cortex. <i>Cereb. Cortex</i> <b>14</b>, 1122 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhg087" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhg087" aria-label="Article reference 90" data-doi="10.1093/cercor/bhg087">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14654453" aria-label="PubMed reference 90">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=Automatically%20parcellating%20the%20human%20cerebral%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhg087&amp;volume=14&amp;pages=11-22&amp;publication_year=2004&amp;author=Fischl%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91"><p class="c-article-references__text" id="ref-CR91">Hodge, M.R. et al. ConnectomeDBsharing human brain connectivity data. <i>Neuroimage</i> <b>124</b> Pt B, 11021107 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.04.046" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.04.046" aria-label="Article reference 91" data-doi="10.1016/j.neuroimage.2015.04.046">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25934470" aria-label="PubMed reference 91">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=ConnectomeDB%E2%80%94sharing%20human%20brain%20connectivity%20data&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.04.046&amp;volume=124&amp;issue=Pt%20B%2C&amp;pages=1102-1107&amp;publication_year=2016&amp;author=Hodge%2CMR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92"><p class="c-article-references__text" id="ref-CR92">Marcus, D.S., Olsen, T.R., Ramaratnam, M. &amp; Buckner, R.L. The Extensible Neuroimaging Archive Toolkit: an informatics platform for managing, exploring, and sharing neuroimaging data. <i>Neuroinformatics</i> <b>5</b>, 1134 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1385/NI:5:1:11" data-track-action="article reference" href="https://doi.org/10.1385%2FNI%3A5%3A1%3A11" aria-label="Article reference 92" data-doi="10.1385/NI:5:1:11">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17426351" aria-label="PubMed reference 92">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Extensible%20Neuroimaging%20Archive%20Toolkit%3A%20an%20informatics%20platform%20for%20managing%2C%20exploring%2C%20and%20sharing%20neuroimaging%20data&amp;journal=Neuroinformatics&amp;doi=10.1385%2FNI%3A5%3A1%3A11&amp;volume=5&amp;pages=11-34&amp;publication_year=2007&amp;author=Marcus%2CDS&amp;author=Olsen%2CTR&amp;author=Ramaratnam%2CM&amp;author=Buckner%2CRL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93"><p class="c-article-references__text" id="ref-CR93">Smith, S.M. et al. A positive-negative mode of population covariation links brain connectivity, demographics and behavior. <i>Nat. Neurosci.</i> <b>18</b>, 15651567 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4125" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4125" aria-label="Article reference 93" data-doi="10.1038/nn.4125">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7vN" aria-label="CAS reference 93">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26414616" aria-label="PubMed reference 93">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4625579" aria-label="PubMed Central reference 93">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20positive-negative%20mode%20of%20population%20covariation%20links%20brain%20connectivity%2C%20demographics%20and%20behavior&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4125&amp;volume=18&amp;pages=1565-1567&amp;publication_year=2015&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94"><p class="c-article-references__text" id="ref-CR94">Yeo, B.T., Krienen, F.M., Chee, M.W. &amp; Buckner, R.L. Estimates of segregation and overlap of functional connectivity networks in the human cerebral cortex. <i>Neuroimage</i> <b>88</b>, 212227 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.10.046" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.10.046" aria-label="Article reference 94" data-doi="10.1016/j.neuroimage.2013.10.046">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24185018" aria-label="PubMed reference 94">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimates%20of%20segregation%20and%20overlap%20of%20functional%20connectivity%20networks%20in%20the%20human%20cerebral%20cortex&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.10.046&amp;volume=88&amp;pages=212-227&amp;publication_year=2014&amp;author=Yeo%2CBT&amp;author=Krienen%2CFM&amp;author=Chee%2CMW&amp;author=Buckner%2CRL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95"><p class="c-article-references__text" id="ref-CR95">Wang, D. et al. Parcellating cortical functional networks in individuals. <i>Nat. Neurosci.</i> <b>18</b>, 18531860 (2015b).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4164" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4164" aria-label="Article reference 95" data-doi="10.1038/nn.4164">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVSgt7bF" aria-label="CAS reference 95">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26551545" aria-label="PubMed reference 95">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4661084" aria-label="PubMed Central reference 95">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Parcellating%20cortical%20functional%20networks%20in%20individuals&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4164&amp;volume=18&amp;pages=1853-1860&amp;publication_year=2015&amp;author=Wang%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96"><p class="c-article-references__text" id="ref-CR96">Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. <i>Nat. Neurosci.</i> <b>18</b>, 18321844 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4171" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4171" aria-label="Article reference 96" data-doi="10.1038/nn.4171">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVOgtLfI" aria-label="CAS reference 96">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26571460" aria-label="PubMed reference 96">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4700510" aria-label="PubMed Central reference 96">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=Canonical%20genetic%20signatures%20of%20the%20adult%20human%20brain&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4171&amp;volume=18&amp;pages=1832-1844&amp;publication_year=2015&amp;author=Hawrylycz%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97"><p class="c-article-references__text" id="ref-CR97">Tavor, I. et al. Task-free MRI predicts individual differences in brain activity during task performance. <i>Science</i> <b>352</b>, 216220 (2016). <b>The authors show that a model relating task-independent (rfMRI) measurements to task activity can accurately predict task activation maps for unseen subjects, suggesting a coupling between brain connectivity and function at the level of individual subjects</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aad8127" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aad8127" aria-label="Article reference 97" data-doi="10.1126/science.aad8127">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XlsFSmtbc%3D" aria-label="CAS reference 97">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27124457" aria-label="PubMed reference 97">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6309730" aria-label="PubMed Central reference 97">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 97" href="http://scholar.google.com/scholar_lookup?&amp;title=Task-free%20MRI%20predicts%20individual%20differences%20in%20brain%20activity%20during%20task%20performance&amp;journal=Science&amp;doi=10.1126%2Fscience.aad8127&amp;volume=352&amp;pages=216-220&amp;publication_year=2016&amp;author=Tavor%2CI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98"><p class="c-article-references__text" id="ref-CR98">Reveley, C. et al. Superficial white matter fiber systems impede detection of long-range cortical connections in diffusion MR tractography. <i>Proc. Natl. Acad. Sci. USA</i> <b>112</b>, E2820E2828 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1418198112" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1418198112" aria-label="Article reference 98" data-doi="10.1073/pnas.1418198112">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXotFCmsLY%3D" aria-label="CAS reference 98">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25964365" aria-label="PubMed reference 98">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4450402" aria-label="PubMed Central reference 98">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Superficial%20white%20matter%20fiber%20systems%20impede%20detection%20of%20long-range%20cortical%20connections%20in%20diffusion%20MR%20tractography&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1418198112&amp;volume=112&amp;pages=E2820-E2828&amp;publication_year=2015&amp;author=Reveley%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99"><p class="c-article-references__text" id="ref-CR99">Dalcanton, J.J. 18 years of science with the Hubble Space Telescope. <i>Nature</i> <b>457</b>, 4150 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature07621" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature07621" aria-label="Article reference 99" data-doi="10.1038/nature07621">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVGhsw%3D%3D" aria-label="CAS reference 99">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19122634" aria-label="PubMed reference 99">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 99" href="http://scholar.google.com/scholar_lookup?&amp;title=18%20years%20of%20science%20with%20the%20Hubble%20Space%20Telescope&amp;journal=Nature&amp;doi=10.1038%2Fnature07621&amp;volume=457&amp;pages=41-50&amp;publication_year=2009&amp;author=Dalcanton%2CJJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100"><p class="c-article-references__text" id="ref-CR100">Kuhn, T. <i>The Structure of Scientific Revolutions</i> (University of Chicago Press, 1962).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101"><p class="c-article-references__text" id="ref-CR101">De Martino, F. et al. Frequency preference and attention effects across cortical depths in the human primary auditory cortex. <i>Proc. Natl. Acad. Sci. USA</i> <b>112</b>, 1603616041 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1507552112" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1507552112" aria-label="Article reference 101" data-doi="10.1073/pnas.1507552112">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXitVWjtLvF" aria-label="CAS reference 101">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26668397" aria-label="PubMed reference 101">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4702984" aria-label="PubMed Central reference 101">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 101" href="http://scholar.google.com/scholar_lookup?&amp;title=Frequency%20preference%20and%20attention%20effects%20across%20cortical%20depths%20in%20the%20human%20primary%20auditory%20cortex&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1507552112&amp;volume=112&amp;pages=16036-16041&amp;publication_year=2015&amp;author=De%20Martino%2CF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102"><p class="c-article-references__text" id="ref-CR102">De Martino, F. et al. Spatial organization of frequency preference and selectivity in the human inferior colliculus. <i>Nat. Commun.</i> <b>4</b>, 1386 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms2379" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms2379" aria-label="Article reference 102" data-doi="10.1038/ncomms2379">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXptFyjsL4%3D" aria-label="CAS reference 102">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23340426" aria-label="PubMed reference 102">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 102" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20organization%20of%20frequency%20preference%20and%20selectivity%20in%20the%20human%20inferior%20colliculus&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms2379&amp;volume=4&amp;publication_year=2013&amp;author=De%20Martino%2CF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103"><p class="c-article-references__text" id="ref-CR103">Muckli, L. et al. Contextual feedback to superficial layers of V1. <i>Curr. Biol.</i> <b>25</b>, 26902695 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cub.2015.08.057" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2015.08.057" aria-label="Article reference 103" data-doi="10.1016/j.cub.2015.08.057">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1SqsL7F" aria-label="CAS reference 103">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26441356" aria-label="PubMed reference 103">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4612466" aria-label="PubMed Central reference 103">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 103" href="http://scholar.google.com/scholar_lookup?&amp;title=Contextual%20feedback%20to%20superficial%20layers%20of%20V1&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2015.08.057&amp;volume=25&amp;pages=2690-2695&amp;publication_year=2015&amp;author=Muckli%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="104"><p class="c-article-references__text" id="ref-CR104">Malikovic, A. et al. Cytoarchitectonic analysis of the human extrastriate cortex in the region of V5/MT+: a probabilistic, stereotaxic map of area hOc5. <i>Cereb. Cortex</i> <b>17</b>, 562574 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhj181" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhj181" aria-label="Article reference 104" data-doi="10.1093/cercor/bhj181">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16603710" aria-label="PubMed reference 104">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 104" href="http://scholar.google.com/scholar_lookup?&amp;title=Cytoarchitectonic%20analysis%20of%20the%20human%20extrastriate%20cortex%20in%20the%20region%20of%20V5%2FMT%2B%3A%20a%20probabilistic%2C%20stereotaxic%20map%20of%20area%20hOc5&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhj181&amp;volume=17&amp;pages=562-574&amp;publication_year=2007&amp;author=Malikovic%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="105"><p class="c-article-references__text" id="ref-CR105">Abdollahi, R.O. et al. Correspondences between retinotopic areas and myelin maps in human visual cortex. <i>Neuroimage</i> <b>99</b>, 509524 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.06.042" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.06.042" aria-label="Article reference 105" data-doi="10.1016/j.neuroimage.2014.06.042">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24971513" aria-label="PubMed reference 105">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 105" href="http://scholar.google.com/scholar_lookup?&amp;title=Correspondences%20between%20retinotopic%20areas%20and%20myelin%20maps%20in%20human%20visual%20cortex&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.06.042&amp;volume=99&amp;pages=509-524&amp;publication_year=2014&amp;author=Abdollahi%2CRO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="106"><p class="c-article-references__text" id="ref-CR106">Murphy, K., Birn, R.M., Handwerker, D.A., Jones, T.B. &amp; Bandettini, P.A. The impact of global signal regression on resting state correlations: are anti-correlated networks introduced? <i>Neuroimage</i> <b>44</b>, 893905 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2008.09.036" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2008.09.036" aria-label="Article reference 106" data-doi="10.1016/j.neuroimage.2008.09.036">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18976716" aria-label="PubMed reference 106">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 106" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20global%20signal%20regression%20on%20resting%20state%20correlations%3A%20are%20anti-correlated%20networks%20introduced%3F&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2008.09.036&amp;volume=44&amp;pages=893-905&amp;publication_year=2009&amp;author=Murphy%2CK&amp;author=Birn%2CRM&amp;author=Handwerker%2CDA&amp;author=Jones%2CTB&amp;author=Bandettini%2CPA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="107"><p class="c-article-references__text" id="ref-CR107">Power, J.D. et al. Methods to detect, characterize, and remove motion artifact in resting state fMRI. <i>Neuroimage</i> <b>84</b>, 320341 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.08.048" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.08.048" aria-label="Article reference 107" data-doi="10.1016/j.neuroimage.2013.08.048">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23994314" aria-label="PubMed reference 107">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 107" href="http://scholar.google.com/scholar_lookup?&amp;title=Methods%20to%20detect%2C%20characterize%2C%20and%20remove%20motion%20artifact%20in%20resting%20state%20fMRI&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.08.048&amp;volume=84&amp;pages=320-341&amp;publication_year=2014&amp;author=Power%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="108"><p class="c-article-references__text" id="ref-CR108">Saad, Z.S. et al. Correcting brain-wide correlation differences in resting-state FMRI. <i>Brain Connect.</i> <b>3</b>, 339352 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1089/brain.2013.0156" data-track-action="article reference" href="https://doi.org/10.1089%2Fbrain.2013.0156" aria-label="Article reference 108" data-doi="10.1089/brain.2013.0156">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23705677" aria-label="PubMed reference 108">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3749702" aria-label="PubMed Central reference 108">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 108" href="http://scholar.google.com/scholar_lookup?&amp;title=Correcting%20brain-wide%20correlation%20differences%20in%20resting-state%20FMRI&amp;journal=Brain%20Connect.&amp;doi=10.1089%2Fbrain.2013.0156&amp;volume=3&amp;pages=339-352&amp;publication_year=2013&amp;author=Saad%2CZS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="109"><p class="c-article-references__text" id="ref-CR109">Gotts, S.J. et al. The perils of global signal regression for group comparisons: a case study of Autism Spectrum Disorders. <i>Front. Hum. Neurosci.</i> <b>7</b>, 356 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fnhum.2013.00356" data-track-action="article reference" href="https://doi.org/10.3389%2Ffnhum.2013.00356" aria-label="Article reference 109" data-doi="10.3389/fnhum.2013.00356">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23874279" aria-label="PubMed reference 109">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3709423" aria-label="PubMed Central reference 109">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 109" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20perils%20of%20global%20signal%20regression%20for%20group%20comparisons%3A%20a%20case%20study%20of%20Autism%20Spectrum%20Disorders&amp;journal=Front.%20Hum.%20Neurosci.&amp;doi=10.3389%2Ffnhum.2013.00356&amp;volume=7&amp;publication_year=2013&amp;author=Gotts%2CSJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="110"><p class="c-article-references__text" id="ref-CR110">Saad, Z.S. et al. Trouble at rest: how correlation patterns and group differences become distorted after global signal regression. <i>Brain Connect.</i> <b>2</b>, 2532 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1089/brain.2012.0080" data-track-action="article reference" href="https://doi.org/10.1089%2Fbrain.2012.0080" aria-label="Article reference 110" data-doi="10.1089/brain.2012.0080">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22432927" aria-label="PubMed reference 110">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3484684" aria-label="PubMed Central reference 110">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 110" href="http://scholar.google.com/scholar_lookup?&amp;title=Trouble%20at%20rest%3A%20how%20correlation%20patterns%20and%20group%20differences%20become%20distorted%20after%20global%20signal%20regression&amp;journal=Brain%20Connect.&amp;doi=10.1089%2Fbrain.2012.0080&amp;volume=2&amp;pages=25-32&amp;publication_year=2012&amp;author=Saad%2CZS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="111"><p class="c-article-references__text" id="ref-CR111">Birn, R.M. The role of physiological noise in resting-state functional connectivity. <i>Neuroimage</i> <b>62</b>, 864870 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.01.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.01.016" aria-label="Article reference 111" data-doi="10.1016/j.neuroimage.2012.01.016">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22245341" aria-label="PubMed reference 111">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 111" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20physiological%20noise%20in%20resting-state%20functional%20connectivity&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.01.016&amp;volume=62&amp;pages=864-870&amp;publication_year=2012&amp;author=Birn%2CRM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="112"><p class="c-article-references__text" id="ref-CR112">Chang, C. &amp; Glover, G.H. Relationship between respiration, end-tidal CO2, and BOLD signals in resting-state fMRI. <i>Neuroimage</i> <b>47</b>, 13811393 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2009.04.048" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2009.04.048" aria-label="Article reference 112" data-doi="10.1016/j.neuroimage.2009.04.048">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19393322" aria-label="PubMed reference 112">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 112" href="http://scholar.google.com/scholar_lookup?&amp;title=Relationship%20between%20respiration%2C%20end-tidal%20CO2%2C%20and%20BOLD%20signals%20in%20resting-state%20fMRI&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2009.04.048&amp;volume=47&amp;pages=1381-1393&amp;publication_year=2009&amp;author=Chang%2CC&amp;author=Glover%2CGH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="113"><p class="c-article-references__text" id="ref-CR113">Golestani, A.M., Chang, C., Kwinta, J.B., Khatamian, Y.B. &amp; Jean Chen, J. Mapping the end-tidal CO2 response function in the resting-state BOLD fMRI signal: spatial specificity, test-retest reliability and effect of fMRI sampling rate. <i>Neuroimage</i> <b>104</b>, 266277 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.10.031" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.10.031" aria-label="Article reference 113" data-doi="10.1016/j.neuroimage.2014.10.031">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25462695" aria-label="PubMed reference 113">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 113" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20the%20end-tidal%20CO2%20response%20function%20in%20the%20resting-state%20BOLD%20fMRI%20signal%3A%20spatial%20specificity%2C%20test-retest%20reliability%20and%20effect%20of%20fMRI%20sampling%20rate&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.10.031&amp;volume=104&amp;pages=266-277&amp;publication_year=2015&amp;author=Golestani%2CAM&amp;author=Chang%2CC&amp;author=Kwinta%2CJB&amp;author=Khatamian%2CYB&amp;author=Jean%20Chen%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="114"><p class="c-article-references__text" id="ref-CR114">Smith, S.M. et al. Network modelling methods for FMRI. <i>Neuroimage</i> <b>54</b>, 875891 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2010.08.063" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2010.08.063" aria-label="Article reference 114" data-doi="10.1016/j.neuroimage.2010.08.063">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20817103" aria-label="PubMed reference 114">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 114" href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20modelling%20methods%20for%20FMRI&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2010.08.063&amp;volume=54&amp;pages=875-891&amp;publication_year=2011&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="115"><p class="c-article-references__text" id="ref-CR115">Larson-Prior, L.J. et al. Adding dynamics to the Human Connectome Project with MEG. <i>Neuroimage</i> <b>80</b>, 190201 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.056" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.056" aria-label="Article reference 115" data-doi="10.1016/j.neuroimage.2013.05.056">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVOjsL3J" aria-label="CAS reference 115">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23702419" aria-label="PubMed reference 115">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 115" href="http://scholar.google.com/scholar_lookup?&amp;title=Adding%20dynamics%20to%20the%20Human%20Connectome%20Project%20with%20MEG&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.056&amp;volume=80&amp;pages=190-201&amp;publication_year=2013&amp;author=Larson-Prior%2CLJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="116"><p class="c-article-references__text" id="ref-CR116">Jack, C.R. Jr. et al. The Alzheimer's Disease Neuroimaging Initiative (ADNI): MRI methods. <i>J. Magn. Reson. Imaging</i> <b>27</b>, 685691 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/jmri.21049" data-track-action="article reference" href="https://doi.org/10.1002%2Fjmri.21049" aria-label="Article reference 116" data-doi="10.1002/jmri.21049">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18302232" aria-label="PubMed reference 116">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2544629" aria-label="PubMed Central reference 116">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 116" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Alzheimer%27s%20Disease%20Neuroimaging%20Initiative%20%28ADNI%29%3A%20MRI%20methods&amp;journal=J.%20Magn.%20Reson.%20Imaging&amp;doi=10.1002%2Fjmri.21049&amp;volume=27&amp;pages=685-691&amp;publication_year=2008&amp;author=Jack%2CCR">
                    Google Scholar</a>
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/nn.4361?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank the other investigators and staff members of the Human Connectome Project consortium for invaluable contributions to data acquisition, analysis and sharing. Additionally, we thank the many colleagues outside the HCP upon whose methodological contributions the paradigm espoused in this paper are also based. We thank S. Danker for assistance in manuscript preparation. Supported in part by the Human Connectome Project, WU-Minn-Ox Consortium (1U54MH091657) funded by the 16 NIH Institutes and Centers that support the NIH Blueprint for Neuroscience Research; the McDonnell Center for Systems Neuroscience at Washington University; and NIH F30 MH097312 (M.F.G.), RO1 MH-60974 (D.C.V.E.), P41 EB015894 (NIBIB; K.U.), Wellcome Trust 098369/Z/12/Z (S.M.S., J.L.R.A., T.E.J.B., M.J., E.C.R., S.N.S.), 5R01EB009352 (D.S.M.), 5P30NS048056 (D.S.M.) and 5R24MH108315 (D.S.M.).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Department of Neuroscience, Washington University Medical School, St. Louis, Missouri, USA</p><p class="c-article-author-affiliation__authors-list">Matthew F Glasser,Timothy S Coalson&amp;David C Van Essen</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Nuffield Department of Clinical Neurosciences, FMRIB Centre, John Radcliffe Hospital, University of Oxford, Oxford, UK</p><p class="c-article-author-affiliation__authors-list">Stephen M Smith,Jesper L R Andersson,Timothy E J Behrens,Mark Jenkinson&amp;Stamatios N Sotiropoulos</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Department of Radiology, Washington University Medical School, St. Louis, Missouri, USA</p><p class="c-article-author-affiliation__authors-list">Daniel S Marcus</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Center for Magnetic Resonance Research (CMRR), University of Minnesota, Minneapolis, Minnesota, USA</p><p class="c-article-author-affiliation__authors-list">Edward J Auerbach,Steen Moeller,Essa Yacoub&amp;Kamil Ugurbil</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Department of Psychiatry, Washington University Medical School, St. Louis, Missouri, USA</p><p class="c-article-author-affiliation__authors-list">Michael P Harms</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Department of Computing, Imperial College London, London, UK</p><p class="c-article-author-affiliation__authors-list">Emma C Robinson</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">7Translational and Molecular Imaging Institute, Icahn School of Medicine at Mount Sinai, New York, New York, USA</p><p class="c-article-author-affiliation__authors-list">Junqian Xu</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Matthew_F-Glasser-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Matthew F Glasser</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Matthew%20F%20Glasser" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Matthew%20F%20Glasser" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Matthew%20F%20Glasser%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stephen_M-Smith-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Stephen M Smith</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stephen%20M%20Smith" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stephen%20M%20Smith" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stephen%20M%20Smith%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Daniel_S-Marcus-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Daniel S Marcus</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Daniel%20S%20Marcus" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Daniel%20S%20Marcus" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Daniel%20S%20Marcus%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jesper_L_R-Andersson-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Jesper L R Andersson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jesper%20L%20R%20Andersson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jesper%20L%20R%20Andersson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jesper%20L%20R%20Andersson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Edward_J-Auerbach-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Edward J Auerbach</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Edward%20J%20Auerbach" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Edward%20J%20Auerbach" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Edward%20J%20Auerbach%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Timothy_E_J-Behrens-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Timothy E J Behrens</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Timothy%20E%20J%20Behrens" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Timothy%20E%20J%20Behrens" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Timothy%20E%20J%20Behrens%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Timothy_S-Coalson-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Timothy S Coalson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Timothy%20S%20Coalson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Timothy%20S%20Coalson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Timothy%20S%20Coalson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael_P-Harms-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Michael P Harms</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20P%20Harms" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20P%20Harms" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20P%20Harms%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Mark-Jenkinson-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Mark Jenkinson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Mark%20Jenkinson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mark%20Jenkinson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mark%20Jenkinson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Steen-Moeller-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Steen Moeller</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Steen%20Moeller" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Steen%20Moeller" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Steen%20Moeller%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Emma_C-Robinson-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Emma C Robinson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Emma%20C%20Robinson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Emma%20C%20Robinson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Emma%20C%20Robinson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stamatios_N-Sotiropoulos-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Stamatios N Sotiropoulos</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stamatios%20N%20Sotiropoulos" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stamatios%20N%20Sotiropoulos" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stamatios%20N%20Sotiropoulos%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Junqian-Xu-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Junqian Xu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Junqian%20Xu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Junqian%20Xu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Junqian%20Xu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Essa-Yacoub-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Essa Yacoub</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Essa%20Yacoub" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Essa%20Yacoub" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Essa%20Yacoub%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kamil-Ugurbil-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Kamil Ugurbil</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kamil%20Ugurbil" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kamil%20Ugurbil" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kamil%20Ugurbil%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-David_C-Van_Essen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">David C Van Essen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=David%20C%20Van%20Essen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=David%20C%20Van%20Essen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22David%20C%20Van%20Essen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>M.F.G., S.M.S., D.S.M., K.U. and D.C.V.E. framed the issues and generated the initial draft. M.F.G., S.M.S., D.S.M., J.L.R.A., E.J.A., T.E.J.B., T.S.C., M.P.H., M.J., S.M., E.C.R., S.N.S., J.X., E.Y., K.U. and D.C.V.E. contributed novel methods or analyses. M.F.G., S.M.S., D.S.M., T.E.J.B., T.S.C., M.P.H., E.C.R., S.N.S., J.X., E.Y., K.U. and D.C.V.E. wrote the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:glasserm@wustl.edu">Matthew F Glasser</a> or <a id="corresp-c2" href="mailto:vanessen@wustl.edu">David C Van Essen</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading">Competing interests</h3>
                <p>The authors declare no competing financial interests.</p>
              
            </div></div></section><section data-title="Integrated supplementary information"><div class="c-article-section" id="Sec22-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec22">Integrated supplementary information</h2><div class="c-article-section__content" id="Sec22-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 1 average cortical thickness " href="/articles/nn.4361/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig7_ESM.jpg">Supplementary Figure 1 Average cortical thickness map of 210 HCP subjects at each left hemisphere vertex and the associated colorized histogram.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The mean cortical thickness is around 2.6 mm, and this roughly divides fMRI data into high resolution (less than the mean cortical thickness) and low resolution (more than the mean cortical thickness). The HCPs 3T and 7T chosen resolutions are also plotted. For the data in this figure and other Supplementary Figures, subject recruitment procedures and informed consent forms, including consent to share de-identified data, were approved by the Washington University institutional review board.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 2 dmri data for 3t and 7t sca" href="/articles/nn.4361/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig8_ESM.jpg">Supplementary Figure 2 dMRI data for 3T and 7T scans of the same subject (HCP Subject 158035).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Top row: Fractional anisotropy (FA) maps (axial, coronal and sagittal views). Notice that B1 inhomogeneities at 7T lead to poor SNR and noisy FA estimates in the inferior temporal regions (evident in the coronal views), but efforts have been taken to minimize them. Bottom row: DTI principal fiber orientations (coronal zoomed view of the area delineated by the yellow box). The orientations are RGB color-coded (Red: LeftRight, Green: AnteriorPosterior, Blue: SuperiorInferior) and superimposed on the structural T1w image. The pial surface and the WM/GM boundary surface are also shown. Reproduced, with permission, from Ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Felleman, D.J. &amp; Van Essen, D.C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 147 (1991)." href="/articles/nn.4361#ref-CR78" id="ref-link-section-d24703174e2152">78</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig17"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 3 patterns of cortico-striata" href="/articles/nn.4361/figures/17" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig9_ESM.jpg">Supplementary Figure 3 Patterns of cortico-striatal connectivity revealed by tractography.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Seed locations were in different cortical regions, including vmPFC (ventromedial prefrontal cortex), OFC (orbito-frontal cortex), dACC (dorsal anterior cingulate cortex), dPFC (dorsal prefrontal cortex) and Premotor cortex. Path probabilities (yellow: high, red: intermediate, black: low) are obtained using probabilistic tractography (FSLs probtracx2, <a href="http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FDT/UserGuide">http://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FDT/UserGuide</a>) and the Matrix 3 (bidirectional white matter voxel to gray-matter terminations) algorithm to compute dense connectomes. N=150 HCP subjects were analyzed and averaged. Note the strong similarity to patterns of tracer-based connectivity reported in the macaque monkey, shown in the sketch on the right (reproduced with permission from Ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Kuhn, T. The Structure of Scientific Revolutions (University of Chicago Press, 1962)." href="/articles/nn.4361#ref-CR100" id="ref-link-section-d24703174e2183">100</a>).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 4 multi-band imaging schemati" href="/articles/nn.4361/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig10_ESM.jpg">Supplementary Figure 4 Multi-band imaging schematic and exemplar results.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Left: A schematic representation of the array coil elements over a human head image and a multi-band (MB) excitation (8 slices, in red). Each coil detects a linear combination of signals from each slice weighted by the sensitivity profile of that coil. Right. Four slices from a whole brain data set with standard acquisition (upper left, MB1) and with SMS/MB acquisition with MB=8, MB=12, and MB=16, all obtained using a 32-channel coil on a 3T scanner. Visual inspection of the M=16 image reveals discernible artifacts (e.g., ghosting), whereas the MB=12 and MB=8 images appear much cleaner. Quantitative estimates of cross slice contamination and reconstruction noise can be made. The HCP used MB=8 for fMRI and MB=3 for dMRI (now MB=4 or more for HCP Short).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 5 beta map of the mean fmri t" href="/articles/nn.4361/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig11_ESM.jpg">Supplementary Figure 5 Beta map of the mean fMRI timeseries.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The timecourses were averaged over the whole brain (including gray matter, white matter, and CSF), regressed into the data of each subject, and then averaged across (n = 210 HCP subjects). One particularly striking characteristic of this map is how tissue-specific the global signal is (after ICA+FIX data cleanup), being generally most positive in grey matter, close to zero in white matter, and slightly negative in the ventricles. The tissue specificity of this signal argues against a non-physiologic, non-BOLD contrast-based cause of the signal (e.g., direct biophysical effects of subject head motion). This map by itself does not tell us to what extent the global signal is physiological noise vs. neural signal. Although the data are averaged in the volume across subjects, they still appear relatively sharp because they are not smoothed. Data at <a href="http://balsa.wustl.edu/0L1m">http://balsa.wustl.edu/0L1m</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig18"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 6 visualization of the mean g" href="/articles/nn.4361/figures/18" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig12_ESM.jpg">Supplementary Figure 6 Visualization of the mean grey signal.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The mean grey timeseries beta map (top rows) and the ratio of the variance of the mean grey timeseries to the total BOLD variance (i.e., variance classified as signal by ICA+FIX; bottom rows). The absolute magnitude of the mean grey signal is highest in sensory regions including visual cortex, early somatosensory cortex (particularly of the face), early auditory cortex, and several thalamic nuclei including the LGN/MGN. Visual, somatosensory, auditory, and likely vestibular cortical areas are highlighted with black outlines. Data based on averaging across 210 HCP subjects, aligned using MSMAll for the cortical surface. When the global signal is strong in individual subjects, it closely matches the group average pattern, however when it is weaker, it may or may not match the group pattern. In these cases, the global signal often looks like one or another widespread RSN (e.g., the task positive or task negative (default mode) network). This is further evidence that removing the global signal as a preprocessing step may distort resting state functional connectivity and hence that we need a more selective way to clean global noise out of the data. The bottom row is a relative measure of how much the fMRI timeseries will be altered by regressing out the mean grey timeseries (on average across subjects), as it is a measure of the proportion of the total BOLD signal represented by the mean grey timeseries. Regressing out the mean grey signal will also tend to cause resting state gradient boundaries to move somewhat in the regions where this map has sharp gradients. Data at <a href="http://balsa.wustl.edu/2VnN">http://balsa.wustl.edu/2VnN</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 7 effects of the wishart roll" href="/articles/nn.4361/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig13_ESM.jpg">Supplementary Figure 7 Effects of the Wishart rolloff on dense functional connectivity maps of both an individual subject and group data (210 HCP subjects; MSMAll surface registration).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Top rows show an individual subject, before (column 1) and after (column 2) Wishart rolloff for a seed location in lateral parietal cortex (white dot in upper left panels). The correlation increases dramatically as unstructured spatio-temporal noise is reduced, however the map is not substantially smoothed as it would be with typical smoothing algorithms. Bottom rows show a group dataset before and after Wishart rolloff for a seed location in the posterior cingulate sulcus (white dot in lower left panels). The dataset has been created using the MIGP algorithm to generate a group PCA series (d=4500) that represents the group concatenated timeseries. Because of the hard cutoff at PCA component number 4500, there is a ringing pattern resulting from spatial autocorrelation in the spatio-temporal noise that is represented by the PCA components with the lowest eigenvalues. If a Gaussian filter had been applied, this pattern of local connectivity would be a blob instead of rings. The Wishart rolloff eliminates these rings and again dramatically increases the SNR of the data. Data at <a href="http://balsa.wustl.edu/rrpl">http://balsa.wustl.edu/rrpl</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 8 the hcp language task (stor" href="/articles/nn.4361/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig14_ESM.jpg">Supplementary Figure 8 The HCP language task (story vs baseline) beta maps and their spatial gradients.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Beta maps (rows 1 and 3) and gradient maps (rows 2 and 4) are from two independent groups of 210 HCP subjects, 210P (rows 1 and 2) and 210V (rows 3 and 4). Because of the large number of high quality HCP subjects, the beta maps are very similar across the two groups, and the strong gradients in the beta maps are also very similar. Also shown are white contours of a Bonferroni corrected significance threshold across all 91282 grayordinates (z+/- ~5). Two things are apparent: 1) Because of the large amount of high quality data, most of the brain is either significantly activated or deactivated (an issue that has been discussed before, see Ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Andersson, J.L. &amp; Sotiropoulos, S.N. Non-parametric representation and prediction of single- and multi-shell diffusion-weighted MRI data using Gaussian processes. Neuroimage 122, 166176 (2015)." href="/articles/nn.4361#ref-CR60" id="ref-link-section-d24703174e2312">60</a>). Thus the statistical threshold is not particularly biologically meaningful (a point about statistical thresholds that generally applies). At the same time, the statistical threshold is also not strongly reproducible, in spite of the large amount of high quality data (highlighted ellipses show large differences in the area of activation classified as significant that are not particularly impressive when viewing the unthresholded beta maps). In contrast, the strong gradients in the beta maps are much more reproducible, are likely also more biologically meaningful, and hence provide a better substrate for defining regions of activation or comparing across studies. Data at <a href="http://balsa.wustl.edu/PrmK">http://balsa.wustl.edu/PrmK</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 9 effects on average brain vo" href="/articles/nn.4361/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig15_ESM.jpg">Supplementary Figure 9 Effects on average brain volume of registering 196 HCP brains to MNI space.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The total brain volume (minimal preprocessing pipelines whole brain mask) measured in the subjects native physical space is around ~1350 cc; however, after registration to MNI space it is ~1800 cc, though the variability in brain volumes goes down as indicated by the narrower standard deviation error bars. This reduction in variability is the intended effect of registration, but the increase in brain volume is group average registration drift that was built into MNI standard space during the non-rigid registrations of the template generation process.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 10 a comparison between the h" href="/articles/nn.4361/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig16_ESM.jpg">Supplementary Figure 10 A comparison between the HCP data and published retinotopic parcellation data.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Data from Ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Power, J.D., Barnes, K.A., Snyder, A.Z., Schlaggar, B.L. &amp; Petersen, S.E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. Neuroimage 59, 21422154 (2012)." href="/articles/nn.4361#ref-CR64" id="ref-link-section-d24703174e2363">64</a> and from 120 HCP subjects (from Q1-2) were registered using MSM areal-feature-based registration and group average registration drift was removed from both. Because of this, a contour in functional connectivity in the MT+ region distinguishing strong connectivity to the heavily myelinated IPS hotspot (LIPv, column 1) and to the STS (column 3) lines up with the border between MT+V4t (orange and yellow) and MST+FST (red and maroon, middle column). This illustrates the kind of precise cross-modal, cross-study boundary comparisons possible using the HCP-Style paradigm. Data at <a href="http://balsa.wustl.edu/x2Lz">http://balsa.wustl.edu/x2Lz</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 11 classification of area 55b" href="/articles/nn.4361/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig17_ESM.jpg">Supplementary Figure 11 Classification of area 55b in individual subjects by the areal classifier.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The typical location of area 55b is shown in black or white outlines on the inflated left hemisphere surface. The top two rows show a subject having an area 55b in the typical location found in the population. Rows 1 and 2 are entirely separate test/retest runs of this subject through the full HCP MRI acquisition and analysis protocol. Column 1 is the subjects individual curvature map, column 2 is the subjects myelin map, column 3 is a d=40 RSN map that shows strong connectivity between 55b and other areas of the language network, column 4 is the raw probabilistic output of the areal classifier, column 5 is the final output of the areal classifier. Rows 3 and 4 show a different subject whose 55b is atypically split (heavy myelination running through the population average location of 55b and a concomitant lack of resting state connectivity). In both runs through the protocol, this subject shows a split 55b that is accurately detected by the areal classifier, showing that the classifier can accurately classify atypical subjects and that these atypical patterns are stable across time. Rows 5 and 6 show a third subject whose 55b is atypically shifted relative to its neighbors. Again the pattern is stable across time and the areal classifier is able to accurately detect the area. Reprinted from Ref. <a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Calabrese, E., Badea, A., Cofer, G., Qi, Y. &amp; Johnson, G.A. A diffusion MRI tractography connectome of the mouse brain and comparison with neuronal tracer data. Cereb. Cortex 25, 46284637 (2015)." href="/articles/nn.4361#ref-CR34" id="ref-link-section-d24703174e2394">34</a>. Data at <a href="http://balsa.wustl.edu/WPPn">http://balsa.wustl.edu/WPPn</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig16"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 12 effects of averaging surfa" href="/articles/nn.4361/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_Article_BFnn4361_Fig18_ESM.jpg">Supplementary Figure 12 Effects of averaging surface coordinates and folding maps after areal feature-based registration (MSMAll).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The first row shows the group average midthickness surface (left), and the group average curvature map also displayed on inflated and flat surfaces (center and right). The group consisted of 210 subjects. The second row shows an individual subjects midthickness surface (left), and the individuals curvature map displayed also on inflated and flat surfaces (center and right). Note how much less detailed the group average surfaces and curvature maps are in most regions of cortex. However, in regions with consistent folding patterns across subjects and consistent relationships between cortical areas and folds, the group average patterns remain sharp (e.g. the central and calcarine sulci and the insula). The third row shows the group average T1w volume (after FNIRT nonlinear volume registration to MNI152 space of each subject in the group) together with the group average white (green) and pial (blue) surface contours. The fourth row shows the same individual subjects T1w volume together with the individuals white and pial surfaces (after aligning both the T1w volume and the surfaces to the group average in MNI space using FNIRT nonlinear volume registration). Despite the high precision of the white and pial surfaces in following the grey matter ribbon in the individual, the group average surfaces do not follow the group average volume particularly well, except in the regions where there are consistent folding patterns across subjects and consistent relationships between cortical areas and folds (as mentioned above). These issues also occur with folding-based surface registration (not shown), though they are less severe, because for folding-based registration the dominant factor is inconsistency in folding patterns across subjects (as no attempt is made to directly align cortical areas). The midthickness surfaces are the average of the white and pial surfaces (this average is performed on each individual, and the group midthickness surface is the average of the individual midthickness surfaces). Data at <a href="http://balsa.wustl.edu/7qP3">http://balsa.wustl.edu/7qP3</a>.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec23-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec23">Supplementary information</h2><div class="c-article-section__content" id="Sec23-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM36"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary text and figures" href="https://static-content.springer.com/esm/art%3A10.1038%2Fnn.4361/MediaObjects/41593_2016_BFnn4361_MOESM36_ESM.pdf" data-supp-info-image="">Supplementary Text and Figures</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Notes and Supplementary Figures 112 (PDF 3941 kb)</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=The%20Human%20Connectome%20Project%27s%20neuroimaging%20approach&amp;author=Matthew%20F%20Glasser%20et%20al&amp;contentID=10.1038%2Fnn.4361&amp;copyright=Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2016-08-26&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/nn.4361" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/nn.4361" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Glasser, M., Smith, S., Marcus, D. <i>et al.</i> The Human Connectome Project's neuroimaging approach.
                    <i>Nat Neurosci</i> <b>19</b>, 11751187 (2016). https://doi.org/10.1038/nn.4361</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/nn.4361?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2016-03-23">23 March 2016</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2016-07-18">18 July 2016</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2016-08-26">26 August 2016</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2016-09">September 2016</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/nn.4361</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Choroid plexus volume enlargement in first-episode antipsychotic-nave schizophrenia" href="https://doi.org/10.1038/s41537-023-00424-2">
                                        Choroid plexus volume enlargement in first-episode antipsychotic-nave schizophrenia
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jiaxin Zeng</li><li>Tianwei Zhang</li><li>Su Lui</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Schizophrenia</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Network Representation of fMRI Data Using Visibility Graphs: The Impact of Motion and Test-Retest Reliability" href="https://doi.org/10.1007/s12021-024-09652-y">
                                        Network Representation of fMRI Data Using Visibility Graphs: The Impact of Motion and Test-Retest Reliability
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Govinda R. Poudel</li><li>Prabin Sharma</li><li>Ester Cerin</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Neuroinformatics</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Larger left hippocampal presubiculum is associated with lower risk of antisocial behavior in healthy adults with childhood conduct history" href="https://doi.org/10.1038/s41598-023-33198-9">
                                        Larger left hippocampal presubiculum is associated with lower risk of antisocial behavior in healthy adults with childhood conduct history
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>AmirHussein Abdolalizadeh</li><li>Kamyar Moradi</li><li>Reza Rajimehr</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Reports</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Reproducible Tract Profiles2 (RTP2) suite, from diffusion MRI acquisition to clinical practice and research" href="https://doi.org/10.1038/s41598-023-32924-7">
                                        Reproducible Tract Profiles2 (RTP2) suite, from diffusion MRI acquisition to clinical practice and research
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Garikoitz Lerma-Usabiaga</li><li>Mengxing Liu</li><li>Brian A. Wandell</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Reports</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Physical symptoms and brain morphology: a population neuroimaging study in 12,286 pre-adolescents" href="https://doi.org/10.1038/s41398-023-02528-w">
                                        Physical symptoms and brain morphology: a population neuroimaging study in 12,286 pre-adolescents
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Fernando Estvez-Lpez</li><li>Hannah H. Kim</li><li>Ryan L. Muetzel</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Translational Psychiatry</i> (2023)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/nn.4361.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/nn.4361.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                        <div class="c-article-associated-content__collection focus u-mb-24">
                            <section>
                                <p class="c-article-associated-content__collection-label u-sans-serif u-text-bold u-mb-8">Focus</p>
                                <h3 class="c-article-associated-content__collection-title u-h3 u-mb-8">
                                    <a href="https://www.nature.com/collections/dycqdqykfv"
                                       class="u-link-inherit"
                                       data-track="click"
                                       data-track-action="view focus"
                                       data-track-category="associated content"
                                       data-track-label="focus"
                                       data-test="collection-link">Focus on the neuroscience toolbox</a>
                                </h3>
                            </section>
                        </div>
                    
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "focus";
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=nn.4361;doi=10.1038/nn.4361;techmeta=57,59;subjmeta=1594,2649,378,3917,3920,631;kwrd=Language,Neural+circuits,Sensory+processing">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=2038986225&amp;t=pos%3Dright%26type%3Darticle%26artid%3Dnn.4361%26doi%3D10.1038/nn.4361%26techmeta%3D57,59%26subjmeta%3D1594,2649,378,3917,3920,631%26kwrd%3DLanguage,Neural+circuits,Sensory+processing">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=2038986225&amp;t=pos%3Dright%26type%3Darticle%26artid%3Dnn.4361%26doi%3D10.1038/nn.4361%26techmeta%3D57,59%26subjmeta%3D1594,2649,378,3917,3920,631%26kwrd%3DLanguage,Neural+circuits,Sensory+processing"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter  what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/nn.4361&amp;format=js&amp;last_modified=2016-09-01" async></script>
<img src="/n6iz4e0d/article/nn.4361" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>