<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"epigenetics-in-the-nervous-system;neural-circuit","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41593-018-0195-0"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Joshua B. Burt","Murat Demirtaş","William J. Eckner","Natasha M. Navejar","Jie Lisa Ji","William J. Martin","Alberto Bernacchia","Alan Anticevic","John D. Murray"],"publishedAt":1533513600,"publishedAtString":"2018-08-06","title":"Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"21","issue":"9"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography","description":"Hierarchy provides a unifying principle for the macroscale organization of anatomical and functional properties across primate cortex, yet microscale bases of specialization across human cortex are poorly understood. Anatomical hierarchy is conventionally informed by invasive tract-tracing measurements, creating a need for a principled proxy measure in humans. Moreover, cortex exhibits marked interareal variation in gene expression, yet organizing principles of cortical transcription remain unclear. We hypothesized that specialization of cortical microcircuitry involves hierarchical gradients of gene expression. We found that a noninvasive neuroimaging measure—MRI-derived T1-weighted/T2-weighted (T1w/T2w) mapping—reliably indexes anatomical hierarchy, and it captures the dominant pattern of transcriptional variation across human cortex. We found hierarchical gradients in expression profiles of genes related to microcircuit function, consistent with monkey microanatomy, and implicated in neuropsychiatric disorders. Our findings identify a hierarchical axis linking cortical transcription and anatomy, along which gradients of microscale properties may contribute to the macroscale specialization of cortical function. Burt et al. analyze patterns of gene expression across human cortex and show expression primarily varies along a sensory-association hierarchy captured by noninvasive neuroimaging, suggesting an organizing principle for microcircuit specialization.","datePublished":"2018-08-06T00:00:00Z","dateModified":"2018-08-06T00:00:00Z","pageStart":"1251","pageEnd":"1259","sameAs":"https://doi.org/10.1038/s41593-018-0195-0","keywords":["Epigenetics in the nervous system","Neural circuits","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig7_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig8_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"21","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Joshua B. Burt","affiliation":[{"name":"Yale University","address":{"name":"Department of Physics, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Murat Demirtaş","affiliation":[{"name":"Yale University School of Medicine","address":{"name":"Department of Psychiatry, Yale University School of Medicine, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"William J. Eckner","affiliation":[{"name":"Yale University","address":{"name":"Department of Physics, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Natasha M. Navejar","url":"http://orcid.org/0000-0002-2495-6061","affiliation":[{"name":"Tulane University","address":{"name":"Neuroscience Program, Tulane University, New Orleans, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jie Lisa Ji","affiliation":[{"name":"Yale University","address":{"name":"Interdepartmental Neuroscience Program, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"William J. Martin","url":"http://orcid.org/0000-0002-2749-3365","affiliation":[{"name":"BlackThorn Therapeutics","address":{"name":"BlackThorn Therapeutics, San Francisco, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alberto Bernacchia","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Engineering, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alan Anticevic","affiliation":[{"name":"Yale University School of Medicine","address":{"name":"Department of Psychiatry, Yale University School of Medicine, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"John D. Murray","url":"http://orcid.org/0000-0003-4115-8181","affiliation":[{"name":"Yale University","address":{"name":"Department of Physics, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yale University School of Medicine","address":{"name":"Department of Psychiatry, Yale University School of Medicine, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"john.murray@yale.edu","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-018-0195-0">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography"/>
    <meta name="dc.source" content="Nature Neuroscience 2018 21:9"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2018-08-06"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2018 The Author(s)"/>
    <meta name="dc.rights" content="2018 The Author(s)"/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="Hierarchy provides a unifying principle for the macroscale organization of anatomical and functional properties across primate cortex, yet microscale bases of specialization across human cortex are poorly understood. Anatomical hierarchy is conventionally informed by invasive tract-tracing measurements, creating a need for a principled proxy measure in humans. Moreover, cortex exhibits marked interareal variation in gene expression, yet organizing principles of cortical transcription remain unclear. We hypothesized that specialization of cortical microcircuitry involves hierarchical gradients of gene expression. We found that a noninvasive neuroimaging measure&#8212;MRI-derived T1-weighted/T2-weighted (T1w/T2w) mapping&#8212;reliably indexes anatomical hierarchy, and it captures the dominant pattern of transcriptional variation across human cortex. We found hierarchical gradients in expression profiles of genes related to microcircuit function, consistent with monkey microanatomy, and implicated in neuropsychiatric disorders. Our findings identify a hierarchical axis linking cortical transcription and anatomy, along which gradients of microscale properties may contribute to the macroscale specialization of cortical function. Burt et al. analyze patterns of gene expression across human cortex and show expression primarily varies along a sensory-association hierarchy captured by noninvasive neuroimaging, suggesting an organizing principle for microcircuit specialization."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2018-08-06"/>
    <meta name="prism.volume" content="21"/>
    <meta name="prism.number" content="9"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="1251"/>
    <meta name="prism.endingPage" content="1259"/>
    <meta name="prism.copyright" content="2018 The Author(s)"/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-018-0195-0"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-018-0195-0"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-018-0195-0.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-018-0195-0"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography"/>
    <meta name="citation_volume" content="21"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_publication_date" content="2018/09"/>
    <meta name="citation_online_date" content="2018/08/06"/>
    <meta name="citation_firstpage" content="1251"/>
    <meta name="citation_lastpage" content="1259"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-018-0195-0"/>
    <meta name="DOI" content="10.1038/s41593-018-0195-0"/>
    <meta name="size" content="323450"/>
    <meta name="citation_doi" content="10.1038/s41593-018-0195-0"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-018-0195-0&amp;api_key="/>
    <meta name="description" content="Hierarchy provides a unifying principle for the macroscale organization of anatomical and functional properties across primate cortex, yet microscale bases of specialization across human cortex are poorly understood. Anatomical hierarchy is conventionally informed by invasive tract-tracing measurements, creating a need for a principled proxy measure in humans. Moreover, cortex exhibits marked interareal variation in gene expression, yet organizing principles of cortical transcription remain unclear. We hypothesized that specialization of cortical microcircuitry involves hierarchical gradients of gene expression. We found that a noninvasive neuroimaging measure&#8212;MRI-derived T1-weighted/T2-weighted (T1w/T2w) mapping&#8212;reliably indexes anatomical hierarchy, and it captures the dominant pattern of transcriptional variation across human cortex. We found hierarchical gradients in expression profiles of genes related to microcircuit function, consistent with monkey microanatomy, and implicated in neuropsychiatric disorders. Our findings identify a hierarchical axis linking cortical transcription and anatomy, along which gradients of microscale properties may contribute to the macroscale specialization of cortical function. Burt et al. analyze patterns of gene expression across human cortex and show expression primarily varies along a sensory-association hierarchy captured by noninvasive neuroimaging, suggesting an organizing principle for microcircuit specialization."/>
    <meta name="dc.creator" content="Burt, Joshua B."/>
    <meta name="dc.creator" content="Demirta&#351;, Murat"/>
    <meta name="dc.creator" content="Eckner, William J."/>
    <meta name="dc.creator" content="Navejar, Natasha M."/>
    <meta name="dc.creator" content="Ji, Jie Lisa"/>
    <meta name="dc.creator" content="Martin, William J."/>
    <meta name="dc.creator" content="Bernacchia, Alberto"/>
    <meta name="dc.creator" content="Anticevic, Alan"/>
    <meta name="dc.creator" content="Murray, John D."/>
    <meta name="dc.subject" content="Epigenetics in the nervous system"/>
    <meta name="dc.subject" content="Neural circuits"/>
    <meta name="citation_reference" content="citation_journal_title=Perception; citation_title=Single units and visual cortical organization; citation_author=P Lennie; citation_volume=27; citation_publication_date=1998; citation_pages=889-935; citation_doi=10.1068/p270889; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=A hierarchy of temporal receptive windows in human cortex; citation_author=U Hasson, E Yang, I Vallines, DJ Heeger, N Rubin; citation_volume=28; citation_publication_date=2008; citation_pages=2539-2550; citation_doi=10.1523/JNEUROSCI.5487-07.2008; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Topographic mapping of a hierarchy of temporal receptive windows using a narrated story; citation_author=Y Lerner, CJ Honey, LJ Silbert, U Hasson; citation_volume=31; citation_publication_date=2011; citation_pages=2906-2915; citation_doi=10.1523/JNEUROSCI.3684-10.2011; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A hierarchy of intrinsic timescales across primate cortex; citation_author=JD Murray; citation_volume=17; citation_publication_date=2014; citation_pages=1661-1663; citation_doi=10.1038/nn.3862; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Slow cortical dynamics and the accumulation of information over long timescales; citation_author=CJ Honey; citation_volume=76; citation_publication_date=2012; citation_pages=423-434; citation_doi=10.1016/j.neuron.2012.08.011; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=An anatomically comprehensive atlas of the adult human brain transcriptome; citation_author=MJ Hawrylycz; citation_volume=489; citation_publication_date=2012; citation_pages=391-399; citation_doi=10.1038/nature11405; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Canonical genetic signatures of the adult human brain; citation_author=M Hawrylycz; citation_volume=18; citation_publication_date=2015; citation_pages=1832-1844; citation_doi=10.1038/nn.4171; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Annu. Rev. Neurosci.; citation_title=Transcriptomic perspectives on neocortical structure, development, evolution, and disease; citation_author=ES Lein, TG Belgard, M Hawrylycz, Z Moln&#225;r; citation_volume=40; citation_publication_date=2017; citation_pages=629-652; citation_doi=10.1146/annurev-neuro-070815-013858; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Correspondence between resting-state activity and brain gene expression; citation_author=GZ Wang; citation_volume=88; citation_publication_date=2015; citation_pages=659-666; citation_doi=10.1016/j.neuron.2015.10.022; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Transcriptional profiles of supragranular-enriched genes associate with corticocortical network architecture in the human brain; citation_author=FM Krienen, BTT Yeo, T Ge, RL Buckner, CC Sherwood; citation_volume=113; citation_publication_date=2016; citation_pages=E469-E478; citation_doi=10.1073/pnas.1510903113; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Brain networks. Correlated gene expression supports synchronous activity in brain networks; citation_author=J Richiardi; citation_volume=348; citation_publication_date=2015; citation_pages=1241-1244; citation_doi=10.1126/science.1255905; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=A transcriptional signature of hub connectivity in the mouse connectome.; citation_author=BD Fulcher, A Fornito; citation_volume=113; citation_publication_date=2016; citation_pages=1435-1440; citation_doi=10.1073/pnas.1513302113; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Distributed hierarchical processing in the primate cerebral cortex; citation_author=DJ Felleman, DC Essen; citation_volume=1; citation_publication_date=1991; citation_pages=1-47; citation_doi=10.1093/cercor/1.1.1; citation_id=CR13"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Cortical structure predicts the pattern of corticocortical connections; citation_author=H Barbas, N Rempel-Clower; citation_volume=7; citation_publication_date=1997; citation_pages=635-646; citation_doi=10.1093/cercor/7.7.635; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=J. Comp. Neurol.; citation_title=Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex; citation_author=NT Markov; citation_volume=522; citation_publication_date=2014; citation_pages=225-259; citation_doi=10.1002/cne.23458; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Is the rostro-caudal axis of the frontal lobe hierarchical?; citation_author=D Badre, M D&#8217;Esposito; citation_volume=10; citation_publication_date=2009; citation_pages=659-669; citation_doi=10.1038/nrn2667; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI; citation_author=MF Glasser, DC Essen; citation_volume=31; citation_publication_date=2011; citation_pages=11597-11616; citation_doi=10.1523/JNEUROSCI.2180-11.2011; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Trends and properties of human cerebral cortex: correlations with cortical myelin content; citation_author=MF Glasser, MS Goyal, TM Preuss, ME Raichle, DC Essen; citation_volume=93; citation_publication_date=2014; citation_pages=165-175; citation_doi=10.1016/j.neuroimage.2013.03.060; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex; citation_author=R Chaudhuri, K Knoblauch, MA Gariel, H Kennedy, XJ Wang; citation_volume=88; citation_publication_date=2015; citation_pages=419-431; citation_doi=10.1016/j.neuron.2015.09.008; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Cortical thickness gradients in structural hierarchies; citation_author=K Wagstyl, L Ronan, IM Goodyer, PC Fletcher; citation_volume=111; citation_publication_date=2015; citation_pages=241-250; citation_doi=10.1016/j.neuroimage.2015.02.036; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A multi-modal parcellation of human cerebral cortex; citation_author=MF Glasser; citation_volume=536; citation_publication_date=2016; citation_pages=171-178; citation_doi=10.1038/nature18933; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The primate connectome in context: Principles of connections of the cortical visual system; citation_author=CC Hilgetag, M Medalla, SF Beul, H Barbas; citation_volume=134; citation_publication_date=2016; citation_pages=685-702; citation_doi=10.1016/j.neuroimage.2016.04.017; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Large-scale cellular-resolution gene profiling in human neocortex reveals species-specific molecular signatures; citation_author=H Zeng; citation_volume=149; citation_publication_date=2012; citation_pages=483-496; citation_doi=10.1016/j.cell.2012.02.052; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Interneurons of the neocortical inhibitory system; citation_author=H Markram; citation_volume=5; citation_publication_date=2004; citation_pages=793-807; citation_doi=10.1038/nrn1519; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Interneuron cell types are fit to function; citation_author=A Kepecs, G Fishell; citation_volume=505; citation_publication_date=2014; citation_pages=318-326; citation_doi=10.1038/nature12983; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain; citation_author=BB Lake; citation_volume=352; citation_publication_date=2016; citation_pages=1586-1590; citation_doi=10.1126/science.aaf1204; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Neurosci.; citation_title=Synaptic reverberation underlying mnemonic persistent activity; citation_author=XJ Wang; citation_volume=24; citation_publication_date=2001; citation_pages=455-463; citation_doi=10.1016/S0166-2236(00)01868-3; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Cortex, cognition and the cell: new insights into the pyramidal neuron and prefrontal function; citation_author=GN Elston; citation_volume=13; citation_publication_date=2003; citation_pages=1124-1138; citation_doi=10.1093/cercor/bhg093; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=A specialized NMDA receptor function in layer 5 recurrent microcircuitry of the adult rat prefrontal cortex.; citation_author=H Wang, GG Stradtman, XJ Wang, WJ Gao; citation_volume=105; citation_publication_date=2008; citation_pages=16791-16796; citation_doi=10.1073/pnas.0804318105; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Increased burden of ultra-rare protein-altering variants among 4,877 individuals with schizophrenia; citation_author=G Genovese; citation_volume=19; citation_publication_date=2016; citation_pages=1433-1441; citation_doi=10.1038/nn.4402; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SynaptomeDB: an ontology-based knowledgebase for synaptic genes; citation_author=M Pirooznia; citation_volume=28; citation_publication_date=2012; citation_pages=897-899; citation_doi=10.1093/bioinformatics/bts040; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=ToppGene Suite for gene list enrichment analysis and candidate gene prioritization; citation_author=J Chen, EE Bardes, BJ Aronow, AG Jegga; citation_volume=37; citation_publication_date=2009; citation_pages=W305-11; citation_doi=10.1093/nar/gkp427; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.; citation_title=Genetic analysis implicates APOE, SNCA and suggests lysosomal dysfunction in the etiology of dementia with Lewy bodies; citation_author=J Bras; citation_volume=23; citation_publication_date=2014; citation_pages=6139-6146; citation_doi=10.1093/hmg/ddu334; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants; citation_author=J Pi&#241;ero; citation_volume=45; citation_publication_date=2017; citation_pages=D833-D839; citation_doi=10.1093/nar/gkw943; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Working memory and decision-making in a frontoparietal circuit model; citation_author=JD Murray, J Jaramillo, XJ Wang; citation_volume=37; citation_publication_date=2017; citation_pages=12167-12186; citation_doi=10.1523/JNEUROSCI.0343-17.2017; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=A dendritic disinhibitory circuit mechanism for pathway-specific gating; citation_author=GR Yang, JD Murray, XJ Wang; citation_volume=7; citation_publication_date=2016; citation_doi=10.1038/ncomms12815; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Neurobiological origin of spurious brain morphological changes: A quantitative MRI study; citation_author=S Lorio; citation_volume=37; citation_publication_date=2016; citation_pages=1801-1815; citation_doi=10.1002/hbm.23137; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Myelin and iron concentration in the human brain: a quantitative study of MRI contrast; citation_author=C St&#252;ber; citation_volume=93; citation_publication_date=2014; citation_pages=95-106; citation_doi=10.1016/j.neuroimage.2014.02.026; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Mapping the human cortical surface by combining quantitative T(1) with retinotopy; citation_author=MI Sereno, A Lutti, N Weiskopf, F Dick; citation_volume=23; citation_publication_date=2013; citation_pages=2261-2268; citation_doi=10.1093/cercor/bhs213; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Using high-resolution quantitative mapping of R1 as an index of cortical myelination; citation_author=A Lutti, F Dick, MI Sereno, N Weiskopf; citation_volume=93; citation_publication_date=2014; citation_pages=176-188; citation_doi=10.1016/j.neuroimage.2013.06.005; citation_id=CR40"/>
    <meta name="citation_reference" content="Carey, D. et al. Quantitative MRI provides markers of intra-, inter-regional, and age-related differences in young adult cortical microstructure. Neuroimage S1053-8119 (17)31012-1 (2017)."/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Microstructural proliferation in human cortex is coupled with the development of face processing; citation_author=J Gomez; citation_volume=355; citation_publication_date=2017; citation_pages=68-71; citation_doi=10.1126/science.aag0311; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Situating the default-mode network along a principal gradient of macroscale cortical organization.; citation_author=DS Margulies; citation_volume=113; citation_publication_date=2016; citation_pages=12574-12579; citation_doi=10.1073/pnas.1608282113; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=Large-scale gradients in human cortical organization; citation_author=JM Huntenburg, PL Bazin, DS Margulies; citation_volume=22; citation_publication_date=2018; citation_pages=21-31; citation_doi=10.1016/j.tics.2017.11.002; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Visual field maps in human cortex; citation_author=BA Wandell, SO Dumoulin, AA Brewer; citation_volume=56; citation_publication_date=2007; citation_pages=366-383; citation_doi=10.1016/j.neuron.2007.10.012; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome; citation_author=KJ Whitaker; citation_volume=113; citation_publication_date=2016; citation_pages=9105-9110; citation_doi=10.1073/pnas.1601745113; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiatry; citation_title=Connectome disconnectivity and cortical gene expression in patients with schizophrenia; citation_author=IAC Romme, MA Reus, RA Ophoff, RS Kahn, MP Heuvel; citation_volume=81; citation_publication_date=2017; citation_pages=495-502; citation_doi=10.1016/j.biopsych.2016.07.012; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Functional and evolutionary insights into human brain development through global transcriptome analysis; citation_author=MB Johnson; citation_volume=62; citation_publication_date=2009; citation_pages=494-509; citation_doi=10.1016/j.neuron.2009.03.027; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A comprehensive transcriptional map of primate brain development; citation_author=TE Bakken; citation_volume=535; citation_publication_date=2016; citation_pages=367-375; citation_doi=10.1038/nature18637; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Lifespan maturation and degeneration of human brain white matter; citation_author=JD Yeatman, BA Wandell, AA Mezer; citation_volume=5; citation_publication_date=2014; citation_doi=10.1038/ncomms5932; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The WU-Minn human connectome project: an overview; citation_author=DC Essen; citation_volume=80; citation_publication_date=2013; citation_pages=62-79; citation_doi=10.1016/j.neuroimage.2013.05.041; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=MSM: a new flexible framework for Multimodal Surface Matching; citation_author=EC Robinson; citation_volume=100; citation_publication_date=2014; citation_pages=414-426; citation_doi=10.1016/j.neuroimage.2014.05.069; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The minimal preprocessing pipelines for the Human Connectome Project; citation_author=MF Glasser; citation_volume=80; citation_publication_date=2013; citation_pages=105-124; citation_doi=10.1016/j.neuroimage.2013.04.127; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Cognitive task information is transferred between brain regions via resting-state network topology; citation_author=T Ito; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/s41467-017-01000-w; citation_id=CR54"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Using diffusion tractography to predict cortical connection strength and distance: a quantitative comparison with tracers in the monkey; citation_author=CJ Donahue; citation_volume=36; citation_publication_date=2016; citation_pages=6758-6770; citation_doi=10.1523/JNEUROSCI.0493-16.2016; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=A weighted and directed interareal connectivity matrix for macaque cerebral cortex; citation_author=NT Markov; citation_volume=24; citation_publication_date=2014; citation_pages=17-36; citation_doi=10.1093/cercor/bhs270; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=J. Comp. Neurol.; citation_title=Local circuit neurons immunoreactive for calretinin, calbindin D-28k or parvalbumin in monkey prefrontal cortex: distribution and morphology; citation_author=F Cond&#233;, JS Lund, DM Jacobowitz, KG Baimbridge, DA Lewis; citation_volume=341; citation_publication_date=1994; citation_pages=95-116; citation_doi=10.1002/cne.903410109; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=J. Comp. Neurol.; citation_title=Local circuit neurons in the medial prefrontal cortex (areas 24a,b,c, 25 and 32) in the monkey: II. Quantitative areal and laminar distributions; citation_author=PL Gabbott, SJ Bacon; citation_volume=364; citation_publication_date=1996; citation_pages=609-636; citation_doi=10.1002/(SICI)1096-9861(19960122)364:4&lt;609::AID-CNE2&gt;3.0.CO;2-7; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=Eur. J. Neurosci.; citation_title=Neurochemical gradients along monkey sensory cortical pathways: calbindin-immunoreactive pyramidal neurons in layers II and III; citation_author=H Kondo, K Tanaka, T Hashikawa, EG Jones; citation_volume=11; citation_publication_date=1999; citation_pages=4197-4203; citation_doi=10.1046/j.1460-9568.1999.00844.x; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Quantitative architecture distinguishes prefrontal cortical systems in the rhesus monkey; citation_author=SM Dombrowski, CC Hilgetag, H Barbas; citation_volume=11; citation_publication_date=2001; citation_pages=975-988; citation_doi=10.1093/cercor/11.10.975; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=The occipitoparietal pathway of the macaque monkey: comparison of pyramidal cell morphology in layer III of functionally related cortical visual areas; citation_author=GN Elston, MG Rosa; citation_volume=7; citation_publication_date=1997; citation_pages=432-452; citation_doi=10.1093/cercor/7.5.432; citation_id=CR61"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Morphological variation of layer III pyramidal neurones in the occipitotemporal pathway of the macaque monkey visual cortex; citation_author=GN Elston, MG Rosa; citation_volume=8; citation_publication_date=1998; citation_pages=278-294; citation_doi=10.1093/cercor/8.3.278; citation_id=CR62"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Biol. Sci.; citation_title=Cortical integration in the visual system of the macaque monkey: large-scale morphological differences in the pyramidal neurons in the occipital, parietal and temporal lobes; citation_author=GN Elston, R Tweedale, MG Rosa; citation_volume=266; citation_publication_date=1999; citation_pages=1367-1374; citation_doi=10.1098/rspb.1999.0789; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=The pyramidal cell of the sensorimotor cortex of the macaque monkey: phenotypic variation; citation_author=GN Elston, KS Rockland; citation_volume=12; citation_publication_date=2002; citation_pages=1071-1078; citation_doi=10.1093/cercor/12.10.1071; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=A study of pyramidal cell structure in the cingulate cortex of the macaque monkey with comparative notes on inferotemporal and primary visual cortex; citation_author=GN Elston, R Benavides-Piccione, J Defelipe; citation_volume=15; citation_publication_date=2005; citation_pages=64-73; citation_doi=10.1093/cercor/bhh109; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Neuroanat.; citation_title=Spinogenesis and pruning in the anterior ventral inferotemporal cortex of the macaque monkey: An intracellular injection study of layer III pyramidal cells; citation_author=GN Elston, T Oga, T Okamoto, I Fujita; citation_volume=5; citation_publication_date=2011; citation_pages=42; citation_doi=10.3389/fnana.2011.00042; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Cell. Proteom.; citation_title=Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics; citation_author=L Fagerberg; citation_volume=13; citation_publication_date=2014; citation_pages=397-406; citation_doi=10.1074/mcp.M113.035600; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function; citation_author=JD Cahoy; citation_volume=28; citation_publication_date=2008; citation_pages=264-278; citation_doi=10.1523/JNEUROSCI.4178-07.2008; citation_id=CR68"/>
    <meta name="citation_reference" content="Anselin, L. Spatial econometrics. in A Companion to Theoretical Econometrics (ed. Baltagi, B.H.) 310&#8211;330 (Blackwell, Oxford, UK, 2001)."/>
    <meta name="citation_reference" content="Fischer, M. M. &amp; Getis, A. Handbook of Applied Spatial Analysis: Software Tools, Methods and Applications. (Springer, Heidelberg, Germany, 2010)."/>
    <meta name="citation_reference" content="citation_journal_title=J. Am. Stat. Assoc.; citation_title=Estimates of the regression coefficient based on Kendall&#8217;s tau; citation_author=PK Sen; citation_volume=63; citation_publication_date=1968; citation_pages=1379-1389; citation_doi=10.1080/01621459.1968.10480934; citation_id=CR71"/>
    <meta name="citation_reference" content="citation_journal_title=Ann. Math. Stat.; citation_title=Bias and confidence in not-quite large samples; citation_author=JW Tukey; citation_volume=29; citation_publication_date=1958; citation_pages=614; citation_doi=10.1214/aoms/1177706647; citation_id=CR72"/>
    <meta name="citation_reference" content="Davison, A.C. &amp; Hinkley, D.V. Bootstrap Methods and Their Application (Cambridge Univ. Press, Cambridge, UK, 1997)."/>
    <meta name="citation_reference" content="citation_journal_title=Br. J. Math. Stat. Psychol.; citation_title=Estimating confidence intervals for principal component loadings: a comparison between the bootstrap and asymptotic results; citation_author=ME Timmerman, HA Kiers, AK Smilde; citation_volume=60; citation_publication_date=2007; citation_pages=295-314; citation_doi=10.1348/000711006X109636; citation_id=CR74"/>
    <meta name="citation_author" content="Burt, Joshua B."/>
    <meta name="citation_author_institution" content="Department of Physics, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Demirta&#351;, Murat"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University School of Medicine, New Haven, USA"/>
    <meta name="citation_author" content="Eckner, William J."/>
    <meta name="citation_author_institution" content="Department of Physics, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Navejar, Natasha M."/>
    <meta name="citation_author_institution" content="Neuroscience Program, Tulane University, New Orleans, USA"/>
    <meta name="citation_author" content="Ji, Jie Lisa"/>
    <meta name="citation_author_institution" content="Interdepartmental Neuroscience Program, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Martin, William J."/>
    <meta name="citation_author_institution" content="BlackThorn Therapeutics, San Francisco, USA"/>
    <meta name="citation_author" content="Bernacchia, Alberto"/>
    <meta name="citation_author_institution" content="Department of Engineering, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Anticevic, Alan"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University School of Medicine, New Haven, USA"/>
    <meta name="citation_author" content="Murray, John D."/>
    <meta name="citation_author_institution" content="Department of Physics, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University School of Medicine, New Haven, USA"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography"/>
    <meta name="twitter:description" content="Nature Neuroscience - Burt et al. analyze patterns of gene expression across human cortex and show expression primarily varies along a sensory-association hierarchy captured by noninvasive..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-018-0195-0"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography - Nature Neuroscience"/>
    <meta property="og:description" content="Burt et al. analyze patterns of gene expression across human cortex and show expression primarily varies along a sensory-association hierarchy captured by noninvasive neuroimaging, suggesting an organizing principle for microcircuit specialization."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-018-0195-0;doi=10.1038/s41593-018-0195-0;techmeta=38,57,59;subjmeta=2584,378,3920,631;kwrd=Epigenetics+in+the+nervous+system,Neural+circuits">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-48570271&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-018-0195-0%26doi%3D10.1038/s41593-018-0195-0%26techmeta%3D38,57,59%26subjmeta%3D2584,378,3920,631%26kwrd%3DEpigenetics+in+the+nervous+system,Neural+circuits">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-48570271&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-018-0195-0%26doi%3D10.1038/s41593-018-0195-0%26techmeta%3D38,57,59%26subjmeta%3D2584,378,3920,631%26kwrd%3DEpigenetics+in+the+nervous+system,Neural+circuits"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-018-0195-0'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0195-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2018-08-06">06 August 2018</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Joshua_B_-Burt-Aff1" data-author-popup="auth-Joshua_B_-Burt-Aff1" data-author-search="Burt, Joshua B.">Joshua B. Burt</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Murat-Demirta_-Aff2" data-author-popup="auth-Murat-Demirta_-Aff2" data-author-search="Demirtaş, Murat">Murat Demirtaş</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-William_J_-Eckner-Aff1" data-author-popup="auth-William_J_-Eckner-Aff1" data-author-search="Eckner, William J.">William J. Eckner</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Natasha_M_-Navejar-Aff3" data-author-popup="auth-Natasha_M_-Navejar-Aff3" data-author-search="Navejar, Natasha M.">Natasha M. Navejar</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-2495-6061"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2495-6061</a></span><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jie_Lisa-Ji-Aff4" data-author-popup="auth-Jie_Lisa-Ji-Aff4" data-author-search="Ji, Jie Lisa">Jie Lisa Ji</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-William_J_-Martin-Aff5" data-author-popup="auth-William_J_-Martin-Aff5" data-author-search="Martin, William J.">William J. Martin</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-2749-3365"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2749-3365</a></span><sup class="u-js-hide"><a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alberto-Bernacchia-Aff6" data-author-popup="auth-Alberto-Bernacchia-Aff6" data-author-search="Bernacchia, Alberto">Alberto Bernacchia</a><sup class="u-js-hide"><a href="#Aff6">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alan-Anticevic-Aff2" data-author-popup="auth-Alan-Anticevic-Aff2" data-author-search="Anticevic, Alan">Alan Anticevic</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 9 authors for this article" title="Show all 9 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John_D_-Murray-Aff1-Aff2" data-author-popup="auth-John_D_-Murray-Aff1-Aff2" data-author-search="Murray, John D." data-corresp-id="c1">John D. Murray<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-4115-8181"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4115-8181</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 21</b>, <span class="u-visually-hidden">pages </span>1251–1259 (<span data-test="article-publication-year">2018</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">13k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">277 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">152 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-018-0195-0/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/epigenetics-in-the-nervous-system" data-track="click" data-track-action="view subject" data-track-label="link">Epigenetics in the nervous system</a></li><li class="c-article-subject-list__subject"><a href="/subjects/neural-circuit" data-track="click" data-track-action="view subject" data-track-label="link">Neural circuits</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Hierarchy provides a unifying principle for the macroscale organization of anatomical and functional properties across primate cortex, yet microscale bases of specialization across human cortex are poorly understood. Anatomical hierarchy is conventionally informed by invasive tract-tracing measurements, creating a need for a principled proxy measure in humans. Moreover, cortex exhibits marked interareal variation in gene expression, yet organizing principles of cortical transcription remain unclear. We hypothesized that specialization of cortical microcircuitry involves hierarchical gradients of gene expression. We found that a noninvasive neuroimaging measure—MRI-derived T1-weighted/T2-weighted (T1w/T2w) mapping—reliably indexes anatomical hierarchy, and it captures the dominant pattern of transcriptional variation across human cortex. We found hierarchical gradients in expression profiles of genes related to microcircuit function, consistent with monkey microanatomy, and implicated in neuropsychiatric disorders. Our findings identify a hierarchical axis linking cortical transcription and anatomy, along which gradients of microscale properties may contribute to the macroscale specialization of cortical function.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0195-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0195-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-024-07178-6/MediaObjects/41586_2024_7178_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-024-07178-6?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41586-024-07178-6">Neural signatures of natural behaviour in socializing macaques
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">13 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Camille Testard, Sébastien Tremblay, … Michael L. Platt</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-024-07192-8/MediaObjects/41586_2024_7192_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-024-07192-8?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41586-024-07192-8">Neural and behavioural state switching during hippocampal dentate spikes
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">13 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Jordan S. Farrell, Ernie Hwaun, … Ivan Soltesz</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-024-07108-6/MediaObjects/41586_2024_7108_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-024-07108-6?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41586-024-07108-6">Neuronal dynamics direct cerebrospinal fluid perfusion and brain clearance
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">28 February 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Li-Feng Jiang-Xie, Antoine Drieu, … Jonathan Kipnis</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'topic',
                        model: 'visits_v2',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711536842,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>The neocortices of human and nonhuman primates exhibit interareal patterns of structural and functional variation. Cortical areas are distinguished by differences in their cellular composition, laminar differentiation, and long-range anatomical connectivity. Primate cortex is characterized by large-scale gradients of specialization in physiology and function, including in representational selectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lennie, P. Single units and visual cortical organization. Perception 27, 889–935 (1998)." href="#ref-CR1" id="ref-link-section-d1625537e518">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hasson, U., Yang, E., Vallines, I., Heeger, D. J. &amp; Rubin, N. A hierarchy of temporal receptive windows in human cortex. J. Neurosci. 28, 2539–2550 (2008)." href="#ref-CR2" id="ref-link-section-d1625537e518_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Lerner, Y., Honey, C. J., Silbert, L. J. &amp; Hasson, U. Topographic mapping of a hierarchy of temporal receptive windows using a narrated story. J. Neurosci. 31, 2906–2915 (2011)." href="/articles/s41593-018-0195-0#ref-CR3" id="ref-link-section-d1625537e521">3</a></sup> and dynamics of intrinsic activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 1661–1663 (2014)." href="/articles/s41593-018-0195-0#ref-CR4" id="ref-link-section-d1625537e525">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Honey, C. J. et al. Slow cortical dynamics and the accumulation of information over long timescales. Neuron 76, 423–434 (2012)." href="/articles/s41593-018-0195-0#ref-CR5" id="ref-link-section-d1625537e528">5</a></sup>. Recent advances in large-scale, high-throughput transcriptomics, which can produce genome-wide atlases of spatially distributed gene expression profiles, have also revealed a rich transcriptional architecture in humans, characterized by spatially heterogeneous gene expression levels across brain areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="#ref-CR6" id="ref-link-section-d1625537e532">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="#ref-CR7" id="ref-link-section-d1625537e532_1">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Lein, E. S., Belgard, T. G., Hawrylycz, M. &amp; Molnár, Z. Transcriptomic perspectives on neocortical structure, development, evolution, and disease. Annu. Rev. Neurosci. 40, 629–652 (2017)." href="/articles/s41593-018-0195-0#ref-CR8" id="ref-link-section-d1625537e535">8</a></sup>. Interareal transcriptional diversity has been related to differences in cortical function, including the spatiotemporal structure of intrinsic network activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e539">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, G.-Z. et al. Correspondence between resting-state activity and brain gene expression. Neuron 88, 659–666 (2015)." href="#ref-CR9" id="ref-link-section-d1625537e542">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Krienen, F. M., Yeo, B. T. T., Ge, T., Buckner, R. L., &amp; Sherwood, C. C. Transcriptional profiles of supragranular-enriched genes associate with corticocortical network architecture in the human brain. Proc. Natl Acad. Sci. USA 113, E469–E478 (2016)." href="#ref-CR10" id="ref-link-section-d1625537e542_1">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Richiardi, J. et al. Brain networks. Correlated gene expression supports synchronous activity in brain networks. Science 348, 1241–1244 (2015)." href="/articles/s41593-018-0195-0#ref-CR11" id="ref-link-section-d1625537e545">11</a></sup>, and to spatially heterogeneous patterns of anatomical connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Richiardi, J. et al. Brain networks. Correlated gene expression supports synchronous activity in brain networks. Science 348, 1241–1244 (2015)." href="/articles/s41593-018-0195-0#ref-CR11" id="ref-link-section-d1625537e549">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Fulcher, B. D. &amp; Fornito, A. A transcriptional signature of hub connectivity in the mouse connectome. Proc. Natl Acad. Sci. USA 113, 1435–1440 (2016)." href="/articles/s41593-018-0195-0#ref-CR12" id="ref-link-section-d1625537e552">12</a></sup>. Yet unifying principles for the macroscale organization of structural, functional, and transcriptional differences across human and nonhuman primate cortexes are still unknown.</p><p>A parsimonious principle for the large-scale anatomical and functional organization of nonhuman primate cortex is the concept of cortical hierarchy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hasson, U., Yang, E., Vallines, I., Heeger, D. J. &amp; Rubin, N. A hierarchy of temporal receptive windows in human cortex. J. Neurosci. 28, 2539–2550 (2008)." href="#ref-CR2" id="ref-link-section-d1625537e559">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lerner, Y., Honey, C. J., Silbert, L. J. &amp; Hasson, U. Topographic mapping of a hierarchy of temporal receptive windows using a narrated story. J. Neurosci. 31, 2906–2915 (2011)." href="#ref-CR3" id="ref-link-section-d1625537e559_1">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 1661–1663 (2014)." href="/articles/s41593-018-0195-0#ref-CR4" id="ref-link-section-d1625537e562">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 1–47 (1991)." href="#ref-CR13" id="ref-link-section-d1625537e565">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Barbas, H. &amp; Rempel-Clower, N. Cortical structure predicts the pattern of corticocortical connections. Cereb. Cortex 7, 635–646 (1997)." href="#ref-CR14" id="ref-link-section-d1625537e565_1">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="#ref-CR15" id="ref-link-section-d1625537e565_2">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Badre, D. &amp; D’Esposito, M. Is the rostro-caudal axis of the frontal lobe hierarchical? Nat. Rev. Neurosci. 10, 659–669 (2009)." href="/articles/s41593-018-0195-0#ref-CR16" id="ref-link-section-d1625537e568">16</a></sup>. Anatomical hierarchy, defined as a globally self-consistent ordering of cortical areas according to characteristic laminar patterns of interareal projections, has been studied extensively in monkeys through histological tract-tracing methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 1–47 (1991)." href="#ref-CR13" id="ref-link-section-d1625537e572">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Barbas, H. &amp; Rempel-Clower, N. Cortical structure predicts the pattern of corticocortical connections. Cereb. Cortex 7, 635–646 (1997)." href="#ref-CR14" id="ref-link-section-d1625537e572_1">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e575">15</a></sup>. The ordering of cortical areas along the anatomical hierarchy, which situates early sensory areas toward the bottom and higher-order association areas toward the top of the hierarchical levels, has also been found to align with areas’ functional organization in sensory processing hierarchies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 1–47 (1991)." href="/articles/s41593-018-0195-0#ref-CR13" id="ref-link-section-d1625537e579">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e582">15</a></sup>. We hypothesized that the transcriptional architecture of human cortex is also hierarchically organized, such that the functional specialization of human cortical microcircuitry involves hierarchical gradients of gene expression levels. However, the highly invasive nature of the tract-tracing data-acquisition procedures required to index hierarchy in nonhuman primates has thus far precluded analogous investigations of cortical organization in humans, thereby creating the need for noninvasive alternative measures.</p><p>To address these open questions, we analyzed transcriptional, anatomical, and neuroimaging data from humans and monkeys to study the hierarchical organization of cortical microcircuit specialization. We found that a structural neuroimaging measure, the MRI-derived T1w/T2w map<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e589">17</a></sup>, provides a noninvasive proxy for anatomical hierarchy in primate cortex. To test for hierarchical gradients in gene expression, we then compared the spatial expression profiles of genes in the Allen Human Brain Atlas (AHBA) to the topography of the human T1w/T2w map. We found strong hierarchical gradients in expression profiles of genes related to synaptic physiology, cell-type specificity, and cortical cytoarchitecture, in line with monkey microanatomical measurements. Furthermore, we observed a remarkably close topographic correspondence between the T1w/T2w map and the dominant spatial pattern of gene expression variation across human cortex. Finally, we found that hierarchically patterned genes were preferentially associated with functional processes and brain disorders. Taken together, these findings suggest that the transcriptional and anatomical architectures of human cortex share a common principal axis of areal variation related to hierarchy and that hierarchical gradients of microscale properties contribute to the macroscale specialization of cortical function.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">T1w/T2w maps noninvasively capture anatomical hierarchy</h3><p>To enable the study of hierarchy in human cortex, we first sought to establish a neuroimaging measure that can serve as a noninvasive proxy for indexing anatomical hierarchy. One measure we examined was the cortical T1w/T2w map. The cortical T1w/T2w map has been proposed as an in vivo measure that is sensitive to regional variation in gray-matter myelin content, based on its close correspondence with myelin-stained sections in histological validation studies and its recapitulation of known neuroanatomical borders between cyto- and myeloarchitecturally delineated areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e605">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Glasser, M. F., Goyal, M. S., Preuss, T. M., Raichle, M. E. &amp; Van Essen, D. C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. Neuroimage 93, 165–175 (2014)." href="/articles/s41593-018-0195-0#ref-CR18" id="ref-link-section-d1625537e608">18</a></sup> (see Discussion). Motivated by the empirical observation that T1w/T2w map values are high in primary sensory cortex (visual, somatosensory, auditory) and low in association cortex, homologously in human and monkey (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1a–c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig9">1</a>) and stably across individuals, we hypothesized that the group-averaged cortical T1w/T2w map, through an inverse relationship with hierarchy, provides a noninvasive correlate for areas’ hierarchical positions.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="T1w/T2w neuroimaging maps noninvasively capture the hierarchical organization of primate cortex."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: T1w/T2w neuroimaging maps noninvasively capture the hierarchical organization of primate cortex.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="1115"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, The parcellated group-averaged (<i>N</i> = 339) human T1w/T2w map exhibits high values in primary sensory cortical areas relative to association areas. <b>b</b>, Human T1w/T2w map values are significantly lower in functionally defined association networks than in sensory networks (***<i>P</i> <i>&lt;</i> 10<sup>–5</sup>; two-sided Wilcoxon signed-rank test on <i>N</i> = 6,608 paired differences; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig9">1c,d</a>) AUD, auditory; VIS, visual; SOM, somatomotor; DAN, dorsal attention; FPN, frontoparietal; VAN, ventral attention; DMN, default mode; CON, cingulo-opercular. Box plots mark the median and inner quartile ranges for areas in each network, and whiskers indicate the 95% confidence interval. <b>c</b>, The parcellated group-averaged (<i>N</i> = 19) macaque monkey T1w/T2w map topography is similar to that of the human. <b>d</b>, Interareal variation in the T1w/T2w map correlates with the laminar specificity of directed feedforward (FF) and feedback (FB) projections in monkey cortex, as quantified by the fraction of labeled supragranular layer neurons (SLN) in the source area. High and low SLN correspond to FF and FB projection motifs, respectively. SLN significantly correlates with pairwise difference (target minus source) in areal T1w/T2w map values across <i>N</i> = 1,243 directed projections (<i>r</i><sub>s</sub> = –0.44, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Spearman rank correlation). <i>N</i><sub>sup</sub> (<i>N</i><sub>inf</sub>), number of supragranular (infragranular) labeled neurons in the source area. <b>e</b>, Anatomical hierarchy levels across cortical areas are estimated by fitting a generalized linear model to predict projections’ SLNs as a function of pairwise hierarchical distance. <b>f</b>, Model-estimated anatomical hierarchy levels are highly anticorrelated with T1w/T2w map values across <i>N</i> = 89 areas of monkey cortex (<i>r</i><sub>s</sub> = –0.76; <i>P</i> <i>&lt;</i> 10<sup>–5</sup>). T1w/T2w map values and model-estimated hierarchy levels in <b>a</b>, <b>c</b>, and <b>e</b> are standardized (i.e., <i>z</i>-scored) and shown in units of s.d. (σ) from the mean.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We validated the T1w/T2w map as a proxy for hierarchy in monkey cortex by comparing T1w/T2w map values to model-estimated anatomical hierarchy levels, derived from conventional tract-tracing approaches that quantify long-range interareal projections and their laminar specificity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e733">15</a></sup>. These laminar connectivity data, which include only direct corticocortical projections, are used to specify a globally optimal hierarchical ordering of cortical areas, such that lower areas send feedforward projections to higher areas and higher areas send feedback projections to lower areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 1–47 (1991)." href="#ref-CR13" id="ref-link-section-d1625537e737">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Barbas, H. &amp; Rempel-Clower, N. Cortical structure predicts the pattern of corticocortical connections. Cereb. Cortex 7, 635–646 (1997)." href="#ref-CR14" id="ref-link-section-d1625537e737_1">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e740">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron 88, 419–431 (2015)." href="/articles/s41593-018-0195-0#ref-CR19" id="ref-link-section-d1625537e743">19</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig10">2</a>). Feedforward and feedback projections primarily originate from the supragranular and infragranular cortical layers, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 1–47 (1991)." href="/articles/s41593-018-0195-0#ref-CR13" id="ref-link-section-d1625537e750">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e753">15</a></sup>. At the level of individual projections, we found that the difference in T1w/T2w map values between connected areas was correlated with the laminar feedforward or feedback structure of the connection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1d</a>), more strongly in high-T1w/T2w sensory areas than in low-T1w/T2w association areas (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig11">3</a>). Global anatomical hierarchy levels were estimated by fitting a generalized linear model to pairwise laminar projection data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e764">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron 88, 419–431 (2015)." href="/articles/s41593-018-0195-0#ref-CR19" id="ref-link-section-d1625537e767">19</a></sup> (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>). We found a strong negative correlation between model-estimated anatomical hierarchy levels and T1w/T2w map values (<i>r</i><sub>s</sub> = –0.76, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Spearman rank correlation; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1e,f</a>).</p><p>How well does the T1w/T2w map capture estimated anatomical hierarchy levels, relative to other putative proxy measures? We compared the performance of the T1w/T2w map against two alternative proxy candidates derived from structural MRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Wagstyl, K., Ronan, L., Goodyer, I. M. &amp; Fletcher, P. C. Cortical thickness gradients in structural hierarchies. Neuroimage 111, 241–250 (2015)." href="/articles/s41593-018-0195-0#ref-CR20" id="ref-link-section-d1625537e793">20</a></sup>: the map of cortical thickness, as cortex is generally thicker in association cortex than sensory cortex; and the map of geodesic distance from primary visual cortex, which defines a posterior–anterior gradient. We found that the T1w/T2w map was more strongly correlated with model-estimated hierarchy than were either of the two other candidate proxy measures (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig2">2</a>). This finding, together with the observed interspecies homology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Glasser, M. F., Goyal, M. S., Preuss, T. M., Raichle, M. E. &amp; Van Essen, D. C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. Neuroimage 93, 165–175 (2014)." href="/articles/s41593-018-0195-0#ref-CR18" id="ref-link-section-d1625537e800">18</a></sup>, supports the use of the T1w/T2w map as a noninvasive correlate of hierarchy across human cortex, for which lack of invasive tract-tracing data precludes a more direct characterization of anatomical hierarchy per conventional approaches.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Model-estimated anatomical hierarchy in monkey cortex is better captured by the group-averaged T1w/T2w map than by two other candidate proxy measures derived from structural MRI."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Model-estimated anatomical hierarchy in monkey cortex is better captured by the group-averaged T1w/T2w map than by two other candidate proxy measures derived from structural MRI.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="168"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Correlation between hierarchy and cortical thickness. <b>b</b>, Correlation between hierarchy and geodesic distance from primary visual cortex (V1), which follows a rostrocaudal gradient. <b>c</b>, Comparison of hierarchy correlation values for the T1w/T2w map, cortical thickness map, and map of geodesic distance from area V1. The T1w/T2w map is much more strongly correlated with model-estimated anatomical hierarchy than the other two maps (<i>P</i> <i>&lt;</i> 10<sup>–5</sup> for both maps). Grey lines in <b>c</b> mark the jackknife estimate of standard error. Statistical significance is calculated by a two-sided test of the difference between dependent correlations (***<i>P</i> <i>&lt;</i> 10<sup>–3</sup>). Correlations and statistical significance values were computed across <i>N</i> = 89 cortical areas.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec4">Hierarchical gradients in cortical microcircuit specialization</h3><p>We hypothesized that the large-scale organization of cortical microcircuit specialization—that is, areal variation in synaptic and cellular composition with functional relevance—may involve hierarchical gradients of gene expression levels across human cortex. To test this hypothesis, we examined areal patterns of cortical gene expression variation from the AHBA in relation to the T1w/T2w map. The AHBA is a transcriptional atlas that contains gene expression levels measured with DNA microarray probes and sampled from hundreds of neuroanatomical structures in the left hemisphere across six normal postmortem human brains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-018-0195-0#ref-CR6" id="ref-link-section-d1625537e863">6</a></sup>. From these data, we calculated group-averaged gene expression profiles across 180 unilateral cortical areas using a multimodal parcellation from the Human Connectome Project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-018-0195-0#ref-CR21" id="ref-link-section-d1625537e867">21</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig3">3</a> and see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>). We then computed correlations of these expression profiles with the human T1w/T2w map. Because of the strong inverse relationship found between the T1w/T2w map and model-estimated hierarchy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1</a>), by extension, expression levels of genes that negatively correlate with the T1w/T2w map tend to increase with progression to higher hierarchical levels—i.e., from sensory to association cortex—and thus exhibit a positive hierarchical gradient; conversely, genes with positive T1w/T2w map correlations (TMCs) exhibit decreasing expression levels along the hierarchical axis. To support the validity of our interpretations, we compared the TMCs of microcircuitry-related gene expression profiles in human cortex to TMCs with more direct anatomical measures in monkey cortex, with a focus on cytoarchitecture, inhibitory interneuron densities, and synaptic processes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Procedure for generating group-averaged parcellated maps of gene expression levels."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Procedure for generating group-averaged parcellated maps of gene expression levels.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="394"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>All analyses of gene expression patterns used group-averaged parcellated expression maps derived from the AHBA (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods </a> for details). The AHBA contains genes expression levels measured with DNA microarray probes and sampled from hundreds of neuroanatomical structures in the left hemisphere across six normal postmortem human brains. First, cortical samples for each subject were mapped from volumetric space onto that subject’s native reconstructed two-dimensional cortical surface. Second, parcellated gene expression maps were constructed for each subject, using the HCP multimodal parcellation (MMP1.0) of the left cortical surface into 180 contiguous areas. For genes profiled by multiple microarray probes, we selected a single representative probe for each subject. Finally, a group-level parcellated expression map for each unique gene was computed by averaging parcellated expression levels across subjects’ selected gene probes (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Group-averaged T1w/T2w maps capture specialization of cortical microcircuitry in humans and nonhuman primates."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Group-averaged T1w/T2w maps capture specialization of cortical microcircuitry in humans and nonhuman primates.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="442"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Cortical cytoarchitectural type is very strongly correlated with the macaque monkey T1w/T2w map across <i>N</i> = 29 areas (τ = 0.87, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; two-sided Kendall’s Tau-b correlation; <i>r</i><sub>s</sub> = 0.96, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Spearman rank correlation). <b>b</b>, The average expression map of 5 genes preferentially expressed in human L4 is positively correlated with the human cortical T1w/T2w map (<i>r</i><sub>s</sub> = 0.68; <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Spearman rank correlation), consistent with a more prominent granular L4 in sensory than association cortex. Expression is plotted in units of s.d. (σ) from the mean. <b>c</b>, Average expression maps of laminar-specific genes show significant TMCs. L1–L3: supragranular layers 1–3 (<i>r</i><sub>s</sub> = –0.42; <i>P</i> <i>&lt;</i> 10<sup>–5</sup>); L5/6: infragranular layers 5 and 6 (<i>r</i><sub>s</sub> = –0.44; <i>P</i> <i>=</i> 2.49 × 10<sup>–3</sup>). <b>d</b>, The T1w/T2w map captures areal variation in the relative proportions of calretinin-expressing (<i>r</i><sub>s</sub> = –0.72; <i>P</i> <i>&lt;</i> 10<sup>–5</sup>) and parvalbumin-expressing (<i>r</i><sub>s</sub> = 0.58; <i>P</i> <i>=</i> 1.7 × 10<sup>–5</sup>) inhibitory interneurons across <i>N</i> = 47 areas of monkey cortex. <b>e</b>, Genes coding for calretinin (<i>CALB2</i>; <i>r</i><sub>s</sub> = –0.45, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>) and parvalbumin (<i>PVALB</i>; <i>r</i><sub>s</sub> = 0.70, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>) exhibit homologous hierarchical gradients in human cortex. <b>f</b>, TMCs of genes coding for markers of specific inhibitory interneuron cell types. <b>g</b>, Basal-dendritic spine counts on pyramidal cells are significantly anticorrelated with the monkey T1w/T2w map across <i>N</i> = 23 areas (<i>r</i><sub>s</sub> = –0.71; <i>P</i> <i>=</i> 1.6 × 10<sup>–4</sup>). <b>h</b>, <i>GRIN2B</i>, which codes for the NMDA receptor subunit NR2B, exhibits a negative TMC (<i>r</i><sub>s</sub> = –0.63; <i>P</i> <i>&lt;</i> 10<sup>–5</sup>). <b>i</b>,<b>j</b>, TMCs of genes coding for distinct subunits of the excitatory NMDA receptor and inhibitory GABA<sub>A</sub> receptor. For comparison with monkey measurements in <b>a</b>, <b>d</b>, and <b>g</b>, Spearman rank correlations with model-estimated hierarchy levels (instead of T1w/T2w map values) were –0.92 for cytoarchitectural type; 0.72 (–0.77) for relative calretinin (parvalbumin) expressing interneuron proportion; and 0.78 for spine count. Statistical significance in <b>c</b>, <b>f</b>, <b>i</b>, and <b>j</b> is calculated using a spatial autoregressive model to account for spatial autocorrelation, Bonferroni-corrected by the number of genes in each set (*<i>P</i> <i>&lt;</i> 0.05; **<i>P</i> <i>&lt;</i> 10<sup>–2</sup>; ***<i>P</i> <i>&lt;</i> 10<sup>–3</sup>), and gray lines mark the jackknife estimate of standard error (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>An established feature of microcircuit specialization that varies along the cortical hierarchy is the degree of laminar differentiation in local cytoarchitecture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Hilgetag, C. C., Medalla, M., Beul, S. F. &amp; Barbas, H. The primate connectome in context: Principles of connections of the cortical visual system. Neuroimage 134, 685–702 (2016)." href="/articles/s41593-018-0195-0#ref-CR22" id="ref-link-section-d1625537e1170">22</a></sup>: primary sensory cortex is highly laminated and exhibits a thick and well-defined granular layer, whereas association cortex is characterized by decreasing laminar differentiation and a gradual loss of the granular layer with progression up hierarchical levels. In monkey cortex, we found that areas’ cytoarchitectural types<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Hilgetag, C. C., Medalla, M., Beul, S. F. &amp; Barbas, H. The primate connectome in context: Principles of connections of the cortical visual system. Neuroimage 134, 685–702 (2016)." href="/articles/s41593-018-0195-0#ref-CR22" id="ref-link-section-d1625537e1174">22</a></sup> correlated strongly with their T1w/T2w map values (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4a</a>). In human cortex, we examined average expression profiles of genes reported to be preferentially expressed in specific cortical layers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Zeng, H. et al. Large-scale cellular-resolution gene profiling in human neocortex reveals species-specific molecular signatures. Cell 149, 483–496 (2012)." href="/articles/s41593-018-0195-0#ref-CR23" id="ref-link-section-d1625537e1181">23</a></sup>. Consistent with the cytoarchitectural trends observed in monkey cortex, we found positive TMCs for genes preferentially expressed in granular layer 4 (L4) and found negative TMCs for supragranular (L1–L3) and infragranular (L5/6) layer-specific genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4b,c</a>). These findings demonstrate that the noninvasive T1w/T2w map captures anatomical gradients related to cortical hierarchy in humans and nonhuman primates.</p><p>To gain further insight into microcircuit bases of hierarchical specialization, we examined the spatial distributions of markers for different inhibitory interneuron cell types. Inhibitory interneuron cell types fall into several biophysically distinct classes that differ in their synaptic connectivity patterns, morphology, electrophysiology, and functional roles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Markram, H. et al. Interneurons of the neocortical inhibitory system. Nat. Rev. Neurosci. 5, 793–807 (2004)." href="/articles/s41593-018-0195-0#ref-CR24" id="ref-link-section-d1625537e1191">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Kepecs, A. &amp; Fishell, G. Interneuron cell types are fit to function. Nature 505, 318–326 (2014)." href="/articles/s41593-018-0195-0#ref-CR25" id="ref-link-section-d1625537e1194">25</a></sup>. In monkey cortex, we found that immunohistochemically measured densities of parvalbumin- and calretinin-expressing interneurons exhibited positive and negative TMCs, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4d</a>). Consistent with these trends, in human cortex we found corresponding negative and positive hierarchical gradients in the expression profiles for the genes that code for parvalbumin and calretinin (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4e</a>). In general, we observed strong hierarchical gradients in transcriptional markers for a number of inhibitory interneuron cell types<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Markram, H. et al. Interneurons of the neocortical inhibitory system. Nat. Rev. Neurosci. 5, 793–807 (2004)." href="/articles/s41593-018-0195-0#ref-CR24" id="ref-link-section-d1625537e1204">24</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4f</a>), as well as for composite gene expression profiles associated with specific neuronal cell types derived from RNA-sequencing in individual human neurons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Lake, B. B. et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. Science 352, 1586–1590 (2016)." href="/articles/s41593-018-0195-0#ref-CR26" id="ref-link-section-d1625537e1212">26</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig12">4</a>). These findings suggest that hierarchical gradients in neuronal cell-type distributions may contribute to sensory–association specialization of cortical microcircuit function.</p><p>Gradients in the composition of synapses may endow cortical areas with the diverse physiological properties required to perform the various computations underlying specialized cognitive and behavioral functions. For instance, local increases in the strength of recurrent excitatory connectivity may endow cortical circuits in association cortex with extended temporal integration, supporting cognitive computations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 1661–1663 (2014)." href="/articles/s41593-018-0195-0#ref-CR4" id="ref-link-section-d1625537e1222">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron 88, 419–431 (2015)." href="/articles/s41593-018-0195-0#ref-CR19" id="ref-link-section-d1625537e1225">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Wang, X.-J. Synaptic reverberation underlying mnemonic persistent activity. Trends Neurosci. 24, 455–463 (2001)." href="/articles/s41593-018-0195-0#ref-CR27" id="ref-link-section-d1625537e1228">27</a></sup>. One putative microanatomical correlate for the strength of recurrent synaptic excitation in local cortical microcircuits is the number of excitatory synapses on pyramidal neurons, which can be quantified by counting the number of spines on pyramidal cell dendrites. In monkey cortex, we found a strong negative TMC for basal-dendritic spine counts on cortical pyramidal neurons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Elston, G. N. Cortex, cognition and the cell: new insights into the pyramidal neuron and prefrontal function. Cereb. Cortex 13, 1124–1138 (2003)." href="/articles/s41593-018-0195-0#ref-CR28" id="ref-link-section-d1625537e1232">28</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4g</a>). This finding suggests a gradient of increasing local recurrent excitation strength along the cortical hierarchy in primates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron 88, 419–431 (2015)." href="/articles/s41593-018-0195-0#ref-CR19" id="ref-link-section-d1625537e1239">19</a></sup>.</p><p>Distinct subunits of synaptic receptor proteins that mediate neurotransmission are differentially expressed across neuronal cell types and produce physiologically diverse synaptic properties. In the AHBA dataset, we examined expression profiles of genes that code for various excitatory and inhibitory synaptic receptor subunits (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4h–j</a>). The gene <i>GRIN2B</i>, which codes for a glutamatergic NMDA receptor subunit that mediates synaptic excitation preferentially in association cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Wang, H., Stradtman, G. G.III, Wang, X.-J. &amp; Gao, W.-J. A specialized NMDA receptor function in layer 5 recurrent microcircuitry of the adult rat prefrontal cortex. Proc. Natl Acad. Sci. USA 105, 16791–16796 (2008)." href="/articles/s41593-018-0195-0#ref-CR29" id="ref-link-section-d1625537e1253">29</a></sup>, exhibited a strong negative TMC. This result suggests increased recurrent excitatory strength in association cortical areas and is consistent with the spine count gradient observed in monkey. Gene sets coding for neuromodulatory synaptic receptor subunits also contain strong positive and negative TMCs (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig13">5</a>). The positive and negative TMCs reported in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4i,j</a> suggest that hierarchical gradients in local excitatory and inhibitory synaptic machinery contribute to the functional specialization of cortical microcircuitry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 1661–1663 (2014)." href="/articles/s41593-018-0195-0#ref-CR4" id="ref-link-section-d1625537e1264">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron 88, 419–431 (2015)." href="/articles/s41593-018-0195-0#ref-CR19" id="ref-link-section-d1625537e1267">19</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec5">T1w/T2w topography captures the dominant axis of transcriptional variation across human cortex</h3><p>How well does the T1w/T2w map capture areal variation in the transcriptional architecture of human cortex in general? We performed principal component analysis to identify the dominant areal patterns of gene expression variation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig14">6</a>). To test for generality of effects, we analyzed categorical sets of genes that are preferentially expressed in human brain tissue, neurons, oligodendrocytes, and synaptic compartments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Genovese, G. et al. Increased burden of ultra-rare protein-altering variants among 4,877 individuals with schizophrenia. Nat. Neurosci. 19, 1433–1441 (2016)." href="/articles/s41593-018-0195-0#ref-CR30" id="ref-link-section-d1625537e1285">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Pirooznia, M. et al. SynaptomeDB: an ontology-based knowledgebase for synaptic genes. Bioinformatics 28, 897–899 (2012)." href="/articles/s41593-018-0195-0#ref-CR31" id="ref-link-section-d1625537e1288">31</a></sup>. To assess the statistical significance of effects, we developed a method for spatial autocorrelation-preserving permutation testing to generate random surrogate maps (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig15">7</a> and see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>). The first principal component (PC1) is defined as the spatial map that captures the greatest fraction of total gene expression variance across cortical areas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5a</a>). Across all five gene sets, PC1 captured a large fraction of gene expression variance (range: 21–27%, more than twice that of PC2; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig14">6</a>), revealing that cortical gene expression patterns are effectively low-dimensional.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="The group-averaged T1w/T2w map captures the dominant axis of gene expression variation across human cortex."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: The group-averaged T1w/T2w map captures the dominant axis of gene expression variation across human cortex.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="179"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, PC1, here for a set of brain-specific genes, is the areal map that linearly captures the maximum variation in gene expression. Both maps are standardized (i.e., <i>z</i>-scored) and shown in units of s.d. (σ) from the mean. <b>b</b>, PC1 captures a large fraction of total gene expression variance. Inset: variance captured by PC1 for five categorical gene sets: all genes; genes preferentially expressed in brain, neurons, oligodendrocytes, and synaptic compartments (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>). <b>c</b>, PC1 for the brain-specific gene set is highly correlated with the T1w/T2w map (<i>r</i><sub>s</sub> = 0.81, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Spearman rank correlation). <b>d</b>, Across all sets, PC1 exhibits a highly similar areal topography to the T1w/T2w map (TMC range: 0.80–0.81; <i>P</i> <i>&lt;</i> 10<sup>–5</sup> for each). <b>e</b>, Gene expression variance captured by the T1w/T2w map (<span class="mathjax-tex">\(\sigma _{\mathrm{T1w/T2w}}^2\)</span>) relative to PC1 (<span class="mathjax-tex">\(\sigma _{\mathrm{PC1}}^2\)</span>). Statistical significance in <b>d</b> and <b>e</b> is calculated through permutation testing with surrogate maps that preserve spatial autocorrelation structure (***<i>P</i> <i>&lt;</i> 10<sup>–3</sup>), and gray lines in <b>b</b>, <b>d</b>, and <b>e</b> mark the bootstrap-estimated 95% confidence interval (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Notably, we found that T1w/T2w map topography was strongly correlated with PC1, i.e., the dominant spatial pattern of gene expression variation, across all tested gene sets (TMC range: 0.80–0.81; <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5c,d</a>). Like the T1w/T2w map, PC1 differentiates primary sensorimotor areas from association areas (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig16">8</a>), consistent with a prior report on a subset of the AHBA dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-018-0195-0#ref-CR6" id="ref-link-section-d1625537e1488">6</a></sup>. We also quantified how much gene expression variance was captured by the T1w/T2w map (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>). We found that, across all gene sets, the T1w/T2w map captured more than half as much variance as PC1, which by construction was the spatial map that captured the maximum possible gene expression variation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5e</a>). We then compared performance of the T1w/T2w map against the two alternative candidate proxy maps, cortical thickness and geodesic distance from primary visual cortex (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig6">6</a>). Across all gene sets, the T1w/T2w map was more strongly correlated with PC1 and captured more gene expression variance than either alternative map. The close alignment between T1w/T2w map topography and spatial gene expression variation suggests that the dominant axis of transcriptional variation in human cortex relates to hierarchy. Furthermore, the robustness of our findings across gene sets demonstrates that this axis captures areal variation in general across a number of neurobiological processes.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Principal component analysis (PCA) shows that the dominant mode of gene expression variation (PC1) is better captured by the group-averaged T1w/T2w map than by other candidate proxies."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: Principal component analysis (PCA) shows that the dominant mode of gene expression variation (PC1) is better captured by the group-averaged T1w/T2w map than by other candidate proxies.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="289"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Parcellated group-averaged (<i>N</i> = 339) map of human cortical thickness. <b>b</b>, The difference in correlation with PC1 between the cortical thickness map and the T1w/T2w map, i.e., <i>r</i><sub>s</sub>(Thickness, PC1) – <i>r</i><sub>s</sub>(T1w/T2w, PC1), across several categorical gene sets. Negative values indicate that the T1w/T2w map is more strongly correlated with PC1 than is the thickness map. <b>c</b>, The difference in the fraction of gene expression variance captured, relative to the variance captured by PC1, between the T1w/T2w map and the cortical thickness map, i.e., <span class="mathjax-tex">\(\left( {\sigma _{\mathrm{Thickness}}^2 - \sigma _{\mathrm{T1w/T2w}}^2} \right)/\sigma _{\mathrm{PC1}}^2\)</span>, across several categorical gene sets. Negative values indicate that the T1w/T2w map captures more gene expression variance than does the thickness map. <b>d</b>, Parcellated map of geodesic distance from primary visual cortical area V1. Maps in <b>a</b> and <b>d</b> are standardized (i.e., <i>z</i>-scored) and shown in units of s.d. (σ) from the mean. <b>e</b>, The difference in correlation with PC1 between the T1w/T2w map and the map of distance from area V1. <b>f</b>, The difference in the fraction of gene expression variance captured, relative to the variance captured by PC1, between the T1w/T2w map and the map of distance from V1. Statistical significance in <b>b</b> and <b>e</b> is calculated by a two-sided test of the difference between dependent correlations (<i>N</i> = 180); in <b>c</b> and <b>f</b> it is calculated through permutation testing with surrogate maps that preserve spatial autocorrelation structure (***<i>P</i> <i>&lt;</i> 10<sup>–3</sup>). Gray lines in <b>b</b>, <b>c</b>, <b>e</b>, and <b>f</b> mark the bootstrap-estimated 95% confidence interval.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>What effects may be driving the outliers that deviate from the otherwise strikingly strong correspondence between gene expression PC1 and the T1w/T2w map shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5c</a>? We constructed a map of the absolute deviation (i.e., residual) of each cortical area from the best-fit line illustrated in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5c</a> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig17">9a</a>) and noticed that the anomalously large residuals were preferentially located in areas of cortex with large gradients in local T1w/T2w map values (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig9">1a</a>). Due to sparse cortical sampling in the AHBA (203 ± 27 samples per subject; 1,220 total samples across 6 subjects), substantial spatial interpolation was required to produce our parcellated gene expression maps (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig3">3</a>). We therefore hypothesized that large discrepancies were due to relatively poor gene expression estimates in regions with large T1w/T2w gradients. To test this quantitatively, for each area we computed a measure of local T1w/T2w gradient and compared these values to the PC1 residuals. We indeed found a strong correlation between local T1w/T2w gradient and PC1 residual magnitude (<i>r</i><sub>p</sub> = 0.70, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Pearson correlation; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig17">9b,c</a>). Our prediction was further supported by a validation of our key results with an earlier group-averaged (<i>N</i> = 69) T1w/T2w map from the Conte69 dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e1722">17</a></sup>. Results were highly consistent, and the Conte69 T1w/T2w map tended to yield stronger TMC values than did the Human Connectome Project’s (HCP) T1w/T2w map (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig18">10</a>). Compared to the smoother Conte69 map, the HCP map contains more sharply separated T1w/T2w values among neighboring cortical parcels (autocorrelation space constant: 6.38 mm for Conte69 versus 6.16 mm for HCP), likely due to differences in surface registration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-018-0195-0#ref-CR21" id="ref-link-section-d1625537e1729">21</a></sup> and smoothing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e1733">17</a></sup>. Together, these findings suggest that the relationships reported in this study, between cortical structure and transcription in humans, may be systematically underestimated, due to the limited spatial resolution in the AHBA dataset.</p><h3 class="c-article__sub-heading" id="Sec6">Stably expressed genes preferentially exhibit hierarchical gradients</h3><p>Genes that are especially vital to normal healthy cortical function may be more likely to have consistent spatial expression profiles across individual subjects. Hawrylycz and colleagues defined differential stability (DS) as the mean pairwise correlation between subjects’ individual gene expression profiles, which they found predicts association with key neurobiological functions when computed across all (i.e., both cortical and subcortical) brain structures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e1745">7</a></sup>. We found a strongly positive, nonlinear relationship between DS computed across cortical areas (DS<sub>c</sub>) and TMC magnitude (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig7">7a</a>). To gain additional insight into this relationship, we explored the impact of filtering genes through progressively higher DS<sub>c</sub> thresholds on the TMC distribution. Exclusion of low-DS<sub>c</sub> genes greatly altered the shape of the TMC distribution, collapsing the prominent peak centered near zero while progressively producing two roughly symmetric, bimodal peaks at strong TMCs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig7">7b</a>). Furthermore, exclusion of low-DS<sub>c</sub> genes strongly increased the fraction of transcriptional variance captured by PC1 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig7">7c</a>), rendering gene expression patterns more quasi-one-dimensional. Together, these findings suggest that high-DS<sub>c</sub> genes—that is, genes whose spatial expression profiles in cortex are highly stable across individuals—preferentially exhibit strong positive and negative hierarchical gradients.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7" data-title="Expression profiles of genes that exhibit strong hierarchical gradients tend to be relatively stable across individuals."><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 7: Expression profiles of genes that exhibit strong hierarchical gradients tend to be relatively stable across individuals.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig7_HTML.png?as=webp"><img aria-describedby="Fig7" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig7_HTML.png" alt="figure 7" loading="lazy" width="685" height="178"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p><b>a</b>, DS<sub>c</sub>, the mean pairwise Spearman rank correlation between subjects’ cortical gene expression maps, as a function of the TMC magnitude for all 16,088 genes (<i>r</i><sub>s</sub> = 0.66, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Spearman rank correlation). Each gray dot represents a single gene. The black line indicates the average value in a sliding window of size 600 points. Abs, absolute value. <b>b</b>, Filtering genes by a threshold on DS<sub>c</sub> alters the shape of the TMC distribution. Increasing the DS<sub>c</sub> threshold filters out genes whose cortical expression profiles are not relatively consistent across subjects. The trough that develops near TMC = 0 suggests that high-DS<sub>c</sub> genes preferentially exhibit strong hierarchical gradients. Min, minimum. <b>c</b>, Thresholding genes by DS<sub>c</sub> substantially increases variance captured by PC1 of gene expression variation (blue), whereas it has little effect on PC1’s TMC (red). Shaded regions in <b>b</b> and <b>c</b> mark the bootstrap-estimated 95% confidence interval. Number of genes that exceed each DS<sub>c</sub> threshold: DS<sub>c</sub> &gt; 0, <i>N</i> =14,509; DS<sub>c</sub> &gt; 0.025, <i>N</i> =12,169; DS<sub>c</sub> &gt; 0.05, <i>N</i> =9,494; DS<sub>c</sub> &gt; 0.075, <i>N</i> =7,332; DS<sub>c</sub> &gt; 0.1, <i>N</i> =5,853.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/7" data-track-dest="link:Figure7 Full size image" aria-label="Full size image figure 7" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec7">Hierarchically expressed genes are enriched for functional and disease annotations</h3><p>To examine the functional roles of genes with strong hierarchical gradients, we tested for their preferential enrichment in gene sets defined by functional and disease ontologies. We found that genes with stronger TMCs were enriched in more functional categories, relative to genes with weaker TMCs, for all functional gene ontologies tested<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e1865">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritization. Nucleic Acids Res. 37, W305–11 (2009)." href="/articles/s41593-018-0195-0#ref-CR32" id="ref-link-section-d1625537e1868">32</a></sup>: biological processes, cellular components, molecular functions, microRNA binding sites, and drug targets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig8">8a</a>). These results suggest that diverse key cell-biological processes contribute to hierarchical differentiation of cortical microcircuitry. Finally, we examined whether hierarchical expression is a property preferentially found in group-averaged profiles of genes associated with psychiatric and neurological disorders. For instance, we found that the genes <i>APOE</i> and <i>SNCA</i>, which are strongly linked to Alzheimer’s and Parkinson’s diseases, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Bras, J. et al. Genetic analysis implicates APOE, SNCA and suggests lysosomal dysfunction in the etiology of dementia with Lewy bodies. Hum. Mol. Genet. 23, 6139–6146 (2014)." href="/articles/s41593-018-0195-0#ref-CR33" id="ref-link-section-d1625537e1881">33</a></sup>, exhibited robust negative TMCs and were therefore more highly expressed in association cortex (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig8">8b,c</a>). For a systematic examination, we statistically quantified the enrichment of genes with strong hierarchical variation in disease-related gene sets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e1889">7</a></sup>, obtained from the DisGeNet database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Piñero, J. et al. DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants. Nucleic Acids Res. 45, D833–D839 (2017). D1." href="/articles/s41593-018-0195-0#ref-CR34" id="ref-link-section-d1625537e1893">34</a></sup>. We found that genes with strongly negative TMCs were significantly over-represented across multiple disease-related gene sets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig8">8d</a>); in particular, gene sets for schizophrenia, bipolar disorder, autistic disorders, and depressive disorders were significantly enriched with strongly negative TMC genes (<i>P</i> &lt; 10<sup>–2</sup> for each), which are more highly expressed in association cortex. These findings suggest that brain disorders involve differential impacts to areas along the cortical hierarchy.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-8" data-title="Hierarchical variation relates to neurobiological function and brain disorders."><figure><figcaption><b id="Fig8" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 8: Hierarchical variation relates to neurobiological function and brain disorders.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/8" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig8_HTML.png?as=webp"><img aria-describedby="Fig8" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig8_HTML.png" alt="figure 8" loading="lazy" width="685" height="979"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p><b>a</b>, Genes with strong TMCs are over-represented in functional annotations across multiple gene ontologies (GOs). BP, biological process; CC, cellular component; MF, molecular function; miRNA, microRNA binding sites; Drug, drug targets. <b>b</b>,<b>c</b>, Two key risk genes for neurodegenerative disorders, <i>APOE</i> for Alzheimer’s disease and <i>SNCA</i> for Parkinson’s disease, exhibit strongly negative TMCs, with higher expression levels in association cortex relative to sensory cortex (<i>APOE</i>: <i>r</i><sub>s</sub> = –0.62, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; <i>SNCA</i>: <i>r</i><sub>s</sub> = –0.72, <i>P</i> <i>&lt;</i> 10<sup>–5</sup>; Spearman rank correlation). <i>APOE</i> is a leading risk gene for Alzheimer’s disease. The ε4 allele of <i>APOE</i> is the largest genetic risk factor for late-onset Alzheimer’s disease. <i>SNCA</i> (<i>PARK1</i>/<i>PARK4</i>) is a key risk gene for Parkinson’s disease. Duplication of <i>SNCA</i> is risk factor for familial Parkinson’s disease with dominant inheritance. <i>SNCA</i> codes for the alpha-synuclein protein, which is the primary component of Lewy bodies, a biomarker of Parkinson’s disease. <b>d</b>, Genes with strong negative TMCs are over-represented in multiple gene sets associated with neuropsychiatric disorders. Left: 20–80% range of TMCs for each disease gene set. Right: enrichment is quantified by the hypergeometric test, which assesses the statistical significance of overlap between each gene set and the top (red) or bottom (blue) 20% TMC genes. Inset: distribution of TMCs across all genes. Expression in <b>b</b> and <b>c</b> is plotted in units of s.d. (σ) from the mean for each map.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0195-0/figures/8" data-track-dest="link:Figure8 Full size image" aria-label="Full size image figure 8" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Discussion</h2><div class="c-article-section__content" id="Sec8-content"><p>Taken together, our findings show that multiple complementary measurement approaches reveal a robust hierarchical organization of microscale variation that may contribute to the macroscale specialization of primate cortical function. First, the MRI-derived T1w/T2w map provides a noninvasive neuroimaging proxy for anatomical hierarchy in the absence of axonal tract-tracing data. Second, the principal axis of transcriptional variation across human cortex aligns with cortical hierarchy, as captured by the T1w/T2w map. Third, this hierarchical axis reflects a gradient of local microcircuit specialization involving synapses and cell types, with relevance to brain disease pathophysiology. Strong similarities between the patterns of anatomical, functional, and transcriptional variation suggest that hierarchical gradients of microcircuit properties play key roles in the functional specialization of large-scale networks across the human cortex. Moreover, the agreement between human transcriptional and monkey anatomical measures suggests conserved organizing principles in human and nonhuman primate cortex.</p><p>Specialization of cortical function may derive, in part, from the multiple features of microcircuitry identified here to exhibit hierarchical gradients. For instance, stronger recurrent excitation in association cortex can endow association circuits with longer timescales of intrinsic activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron 88, 419–431 (2015)." href="/articles/s41593-018-0195-0#ref-CR19" id="ref-link-section-d1625537e2016">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Murray, J. D., Jaramillo, J. &amp; Wang, X.-J. Working memory and decision-making in a frontoparietal circuit model. J. Neurosci. 37, 12167–12186 (2017)." href="/articles/s41593-018-0195-0#ref-CR35" id="ref-link-section-d1625537e2019">35</a></sup>, as observed empirically<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 1661–1663 (2014)." href="/articles/s41593-018-0195-0#ref-CR4" id="ref-link-section-d1625537e2023">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Honey, C. J. et al. Slow cortical dynamics and the accumulation of information over long timescales. Neuron 76, 423–434 (2012)." href="/articles/s41593-018-0195-0#ref-CR5" id="ref-link-section-d1625537e2026">5</a></sup>, which subserve the prolonged integration of signals in these areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Lerner, Y., Honey, C. J., Silbert, L. J. &amp; Hasson, U. Topographic mapping of a hierarchy of temporal receptive windows using a narrated story. J. Neurosci. 31, 2906–2915 (2011)." href="/articles/s41593-018-0195-0#ref-CR3" id="ref-link-section-d1625537e2030">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Badre, D. &amp; D’Esposito, M. Is the rostro-caudal axis of the frontal lobe hierarchical? Nat. Rev. Neurosci. 10, 659–669 (2009)." href="/articles/s41593-018-0195-0#ref-CR16" id="ref-link-section-d1625537e2033">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Wang, X.-J. Synaptic reverberation underlying mnemonic persistent activity. Trends Neurosci. 24, 455–463 (2001)." href="/articles/s41593-018-0195-0#ref-CR27" id="ref-link-section-d1625537e2036">27</a></sup>. Furthermore, computational modeling of cortical circuits identifies recurrent excitation strength as a key property governing functional specialization across areas for core cognitive computations such as working memory and decision-making<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Wang, X.-J. Synaptic reverberation underlying mnemonic persistent activity. Trends Neurosci. 24, 455–463 (2001)." href="/articles/s41593-018-0195-0#ref-CR27" id="ref-link-section-d1625537e2040">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Murray, J. D., Jaramillo, J. &amp; Wang, X.-J. Working memory and decision-making in a frontoparietal circuit model. J. Neurosci. 37, 12167–12186 (2017)." href="/articles/s41593-018-0195-0#ref-CR35" id="ref-link-section-d1625537e2043">35</a></sup>. Hierarchical gradients of inhibitory interneuron cell types can additionally shape regional specialization of dynamics and function, due to cell-type differences in physiology and synaptic connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Markram, H. et al. Interneurons of the neocortical inhibitory system. Nat. Rev. Neurosci. 5, 793–807 (2004)." href="/articles/s41593-018-0195-0#ref-CR24" id="ref-link-section-d1625537e2047">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Kepecs, A. &amp; Fishell, G. Interneuron cell types are fit to function. Nature 505, 318–326 (2014)." href="/articles/s41593-018-0195-0#ref-CR25" id="ref-link-section-d1625537e2050">25</a></sup>. For example, parvalbumin-expressing inhibitory interneurons preferentially target the perisomatic areas of pyramidal neurons, where they can gate pyramidal neuron outputs. In contrast, calretinin-expressing inhibitory interneurons preferentially target distal dendrites of pyramidal neurons and other inhibitory interneurons, where they may play key computational roles in disinhibition-mediated gating of dendritic inputs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Yang, G. R., Murray, J. D. &amp; Wang, X.-J. A dendritic disinhibitory circuit mechanism for pathway-specific gating. Nat. Commun. 7, 12815 (2016)." href="/articles/s41593-018-0195-0#ref-CR36" id="ref-link-section-d1625537e2055">36</a></sup>. Cytoarchitectural differences between areas correlate with their pairwise laminar-projection profiles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Barbas, H. &amp; Rempel-Clower, N. Cortical structure predicts the pattern of corticocortical connections. Cereb. Cortex 7, 635–646 (1997)." href="/articles/s41593-018-0195-0#ref-CR14" id="ref-link-section-d1625537e2059">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Hilgetag, C. C., Medalla, M., Beul, S. F. &amp; Barbas, H. The primate connectome in context: Principles of connections of the cortical visual system. Neuroimage 134, 685–702 (2016)." href="/articles/s41593-018-0195-0#ref-CR22" id="ref-link-section-d1625537e2062">22</a></sup>, linking local microcircuit specialization of areas to their hierarchical long-range interactions.</p><p>Our study adds to a growing understanding of how transcriptional specialization shapes cortical function. Transcriptional diversity, particularly of genes that regulate synaptic function and ion channel activity, relates to the spatiotemporal organization of intrinsic activity in large-scale cortical networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e2069">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wang, G.-Z. et al. Correspondence between resting-state activity and brain gene expression. Neuron 88, 659–666 (2015)." href="#ref-CR9" id="ref-link-section-d1625537e2072">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Krienen, F. M., Yeo, B. T. T., Ge, T., Buckner, R. L., &amp; Sherwood, C. C. Transcriptional profiles of supragranular-enriched genes associate with corticocortical network architecture in the human brain. Proc. Natl Acad. Sci. USA 113, E469–E478 (2016)." href="#ref-CR10" id="ref-link-section-d1625537e2072_1">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Richiardi, J. et al. Brain networks. Correlated gene expression supports synchronous activity in brain networks. Science 348, 1241–1244 (2015)." href="/articles/s41593-018-0195-0#ref-CR11" id="ref-link-section-d1625537e2075">11</a></sup>; and transcriptional markers for synaptic, neuronal, and axonal structure relate to patterns of anatomical connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Richiardi, J. et al. Brain networks. Correlated gene expression supports synchronous activity in brain networks. Science 348, 1241–1244 (2015)." href="/articles/s41593-018-0195-0#ref-CR11" id="ref-link-section-d1625537e2079">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Fulcher, B. D. &amp; Fornito, A. A transcriptional signature of hub connectivity in the mouse connectome. Proc. Natl Acad. Sci. USA 113, 1435–1440 (2016)." href="/articles/s41593-018-0195-0#ref-CR12" id="ref-link-section-d1625537e2082">12</a></sup>. Of note, Hawrylycz et al. found that genes most strongly predictive of functional connectivity patterns in cortex were shifted toward high DS<sub>c</sub> and that, across all brain regions, high-DS genes were enriched in gene sets related to functional ontologies and brain diseases, leading the authors to suggest these genes constitute a “canonical transcriptional blueprint” for the human brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e2088">7</a></sup>. We found that high-DS<sub>c</sub> genes exhibited strong hierarchical gradients across human cortex (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig7">7a</a>) and that these strong-TMC genes exhibited similar functional and brain disease-related enrichments. These results suggest that hierarchically and stably expressed genes across human cortex contribute to the transcriptional regulation of cortical function, and to its pathophysiological disruption in disease.</p><p>Our findings demonstrate that the T1w/T2w map generally captures an axis of hierarchical differentiation across cortex that reflects multiple features of interareal variation. The T1w/T2w map—an MRI contrast map that removes shared imaging intensity biases and increases image contrast—is sensitive to gray-matter myelin content<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e2101">17</a></sup>, which may itself contribute to functional specialization in several ways<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Glasser, M. F., Goyal, M. S., Preuss, T. M., Raichle, M. E. &amp; Van Essen, D. C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. Neuroimage 93, 165–175 (2014)." href="/articles/s41593-018-0195-0#ref-CR18" id="ref-link-section-d1625537e2105">18</a></sup>. However, both T1- and T2-weighted image intensities depend on multiple MRI parameters, each of which is sensitive to several other brain microstructural properties, including cell size and density, degree of dendritic arborization, iron, and water<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Lorio, S. et al. Neurobiological origin of spurious brain morphological changes: A quantitative MRI study. Hum. Brain Mapp. 37, 1801–1815 (2016)." href="/articles/s41593-018-0195-0#ref-CR37" id="ref-link-section-d1625537e2109">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Stüber, C. et al. Myelin and iron concentration in the human brain: a quantitative study of MRI contrast. Neuroimage 93, 95–106 (2014)." href="/articles/s41593-018-0195-0#ref-CR38" id="ref-link-section-d1625537e2112">38</a></sup>. Further in vivo characterization of microstructural variation can be provided by quantitative MRI techniques such as T1 mapping<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sereno, M. I., Lutti, A., Weiskopf, N. &amp; Dick, F. Mapping the human cortical surface by combining quantitative T(1) with retinotopy. Cereb. Cortex 23, 2261–2268 (2013)." href="#ref-CR39" id="ref-link-section-d1625537e2116">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lutti, A., Dick, F., Sereno, M. I. &amp; Weiskopf, N. Using high-resolution quantitative mapping of R1 as an index of cortical myelination. Neuroimage 93, 176–188 (2014)." href="#ref-CR40" id="ref-link-section-d1625537e2116_1">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Carey, D. et al. Quantitative MRI provides markers of intra-, inter-regional, and age-related differences in young adult cortical microstructure. Neuroimage S1053-8119 (17)31012-1 (2017)." href="#ref-CR41" id="ref-link-section-d1625537e2116_2">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Gomez, J. et al. Microstructural proliferation in human cortex is coupled with the development of face processing. Science 355, 68–71 (2017)." href="/articles/s41593-018-0195-0#ref-CR42" id="ref-link-section-d1625537e2119">42</a></sup>. Thus, the T1w/T2w map provides a readily acquired, noninvasive neuroimaging measure that is sensitive to areal variation in not one but several structural components of local cortical microarchitecture. We note that there are interesting deviations between the topographies of the T1w/T2w map and other hierarchical features. For instance, primary motor cortex and retrosplenial cortex exhibit high T1w/T2w map values, yet differ from primary sensory areas in their laminar structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e2123">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Glasser, M. F., Goyal, M. S., Preuss, T. M., Raichle, M. E. &amp; Van Essen, D. C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. Neuroimage 93, 165–175 (2014)." href="/articles/s41593-018-0195-0#ref-CR18" id="ref-link-section-d1625537e2126">18</a></sup>.</p><p>Multiple functionally defined hierarchies in human and nonhuman primate cortices have been proposed, none of which are mutually exclusive with the anatomical hierarchy informed by long-range laminar projection patterns. For instance, studies have found hierarchical differences across areas in the temporal selectivity of spontaneous dynamics and sensory processing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 1661–1663 (2014)." href="/articles/s41593-018-0195-0#ref-CR4" id="ref-link-section-d1625537e2134">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Honey, C. J. et al. Slow cortical dynamics and the accumulation of information over long timescales. Neuron 76, 423–434 (2012)." href="/articles/s41593-018-0195-0#ref-CR5" id="ref-link-section-d1625537e2137">5</a></sup>, but it remains unclear how these differences relate to hierarchies of microcircuit specialization. In this study, we have identified multiple microanatomical and transcriptional properties of cortical microcircuitry that exhibit hierarchical gradients and may contribute to physiological and functional specialization. Notably, cortical function has a complex multidimensional organization with multiple axes of areal variation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-018-0195-0#ref-CR43" id="ref-link-section-d1625537e2141">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Huntenburg, J. M., Bazin, P. L. &amp; Margulies, D. S. Large-scale gradients in human cortical organization. Trends Cogn. Sci. 22, 21–31 (2018)." href="/articles/s41593-018-0195-0#ref-CR44" id="ref-link-section-d1625537e2144">44</a></sup>, each of which can be represented by a scalar-valued map. Distinct information-processing hierarchies can be defined for different sensory modalities, as well as within a modality, such as the dorsal and ventral processing streams in the primate visual system<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 1–47 (1991)." href="/articles/s41593-018-0195-0#ref-CR13" id="ref-link-section-d1625537e2148">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e2151">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Wandell, B. A., Dumoulin, S. O. &amp; Brewer, A. A. Visual field maps in human cortex. Neuron 56, 366–383 (2007)." href="/articles/s41593-018-0195-0#ref-CR45" id="ref-link-section-d1625537e2154">45</a></sup>. Future studies can investigate how integration of multiple neuroimaging measures—for instance, combining T1w/T2w imaging with diffusion-weighted imaging—can reveal new multidimensional principles of cortical organization, both within and across functionally specialized networks.</p><p>Multiple lines of evidence point to a transcriptional basis for disease phenotypic variation, linking white matter dysconnectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-018-0195-0#ref-CR46" id="ref-link-section-d1625537e2161">46</a></sup> and developmental changes in structural topology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Romme, I. A. C., de Reus, M. A., Ophoff, R. A., Kahn, R. S. &amp; van den Heuvel, M. P. Connectome disconnectivity and cortical gene expression in patients with schizophrenia. Biol. Psychiatry 81, 495–502 (2017)." href="/articles/s41593-018-0195-0#ref-CR47" id="ref-link-section-d1625537e2165">47</a></sup> to genes implicated in schizophrenia. Further characterization of the developmental trajectory of hierarchical transcriptional specialization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-018-0195-0#ref-CR46" id="ref-link-section-d1625537e2169">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Johnson, M. B. et al. Functional and evolutionary insights into human brain development through global transcriptome analysis. Neuron 62, 494–509 (2009)." href="/articles/s41593-018-0195-0#ref-CR48" id="ref-link-section-d1625537e2172">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Bakken, T. E. et al. A comprehensive transcriptional map of primate brain development. Nature 535, 367–375 (2016)." href="/articles/s41593-018-0195-0#ref-CR49" id="ref-link-section-d1625537e2175">49</a></sup> and structural brain tissue degeneration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Yeatman, J. D., Wandell, B. A. &amp; Mezer, A. A. Lifespan maturation and degeneration of human brain white matter. Nat. Commun. 5, 4932 (2014)." href="/articles/s41593-018-0195-0#ref-CR50" id="ref-link-section-d1625537e2179">50</a></sup> may inform the progression of neurodevelopmental disorders. Strong hierarchical gradients in drug targets, such as receptor subunits, could enable preferential modulation of sensory or association cortical areas at the group level through targeted pharmacology. This may guide future rational design of drug treatments to target specific macroscale cortical circuits. Large-scale mapping of the cortical transcriptome at finer spatial resolution will further elucidate the microcircuit basis of hierarchical specialization with laminar<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Zeng, H. et al. Large-scale cellular-resolution gene profiling in human neocortex reveals species-specific molecular signatures. Cell 149, 483–496 (2012)." href="/articles/s41593-018-0195-0#ref-CR23" id="ref-link-section-d1625537e2183">23</a></sup> and cell-type<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Lein, E. S., Belgard, T. G., Hawrylycz, M. &amp; Molnár, Z. Transcriptomic perspectives on neocortical structure, development, evolution, and disease. Annu. Rev. Neurosci. 40, 629–652 (2017)." href="/articles/s41593-018-0195-0#ref-CR8" id="ref-link-section-d1625537e2188">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Lake, B. B. et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. Science 352, 1586–1590 (2016)." href="/articles/s41593-018-0195-0#ref-CR26" id="ref-link-section-d1625537e2191">26</a></sup> specificity.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Methods</h2><div class="c-article-section__content" id="Sec9-content"><h3 class="c-article__sub-heading" id="Sec10">Parcellated structural neuroimaging maps</h3><p>Human T1w/T2w and cortical thickness maps were obtained from the Human Connectome Project (HCP)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Van Essen, D. C. et al. The WU-Minn human connectome project: an overview. Neuroimage 80, 62–79 (2013)." href="/articles/s41593-018-0195-0#ref-CR51" id="ref-link-section-d1625537e2207">51</a></sup> in the surface-based CIFTI file format<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-018-0195-0#ref-CR21" id="ref-link-section-d1625537e2211">21</a></sup>. To produce the T1w/T2w maps, high resolution T1- and T2-weighted images were first registered to a standard reference space using a state-of-the-art areal-feature-based technique<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-018-0195-0#ref-CR21" id="ref-link-section-d1625537e2215">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Robinson, E. C. et al. MSM: a new flexible framework for Multimodal Surface Matching. Neuroimage 100, 414–426 (2014)." href="/articles/s41593-018-0195-0#ref-CR52" id="ref-link-section-d1625537e2218">52</a></sup>, which precluded the need for spatial smoothing, and then corrected for bias-field intensity inhomogeneities, yielding dimensionless quantities defined with respect to a reference group-averaged map (for more details, see refs <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e2222">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-018-0195-0#ref-CR21" id="ref-link-section-d1625537e2225">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-018-0195-0#ref-CR53" id="ref-link-section-d1625537e2228">53</a></sup>). Group-averaged (<i>N</i> = 339) left-hemispheric T1w/T2w and thickness maps were parcellated into 180 areas using the HCP’s multimodal parcellation (MMP1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-018-0195-0#ref-CR21" id="ref-link-section-d1625537e2236">21</a></sup>. Parcellated maps were highly stable across individual subjects: the mean pairwise Spearman rank correlation between subjects’ individual maps was 0.94 (or 0.76) for the T1w/T2w (or thickness) map. We note that the areal-feature-based surface-registration technique is informed, in part, by alignment of subjects’ T1w/T2w maps<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Robinson, E. C. et al. MSM: a new flexible framework for Multimodal Surface Matching. Neuroimage 100, 414–426 (2014)." href="/articles/s41593-018-0195-0#ref-CR52" id="ref-link-section-d1625537e2240">52</a></sup>, which likely contributes to the discrepancy in stability observed between the two structural maps. Assignment of MMP1.0 parcels to functional networks (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig9">1d</a>) was performed through community detection analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Ito, T. et al. Cognitive task information is transferred between brain regions via resting-state network topology. Nat. Commun. 8, 1027 (2017)." href="/articles/s41593-018-0195-0#ref-CR54" id="ref-link-section-d1625537e2250">54</a></sup> on time-series correlations in the HCP resting-state fMRI dataset. Six of 180 parcels were not assigned to one of the eight networks analyzed in this study.</p><p>For validation, key findings reported for human cortex were replicated using group-averaged (<i>N</i> = 69) T1w/T2w maps from the publicly available Conte69 dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e2260">17</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig18">10</a>). In contrast to the HCP maps, both individual and group-averaged T1w/T2w maps in the Conte69 dataset were smoothed using Gaussian filters weighted by geodesic distance to reduce high frequency spatial artifacts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. J. Neurosci. 31, 11597–11616 (2011)." href="/articles/s41593-018-0195-0#ref-CR17" id="ref-link-section-d1625537e2267">17</a></sup>.</p><p>The group-averaged (<i>N</i> = 19) T1w/T2w and thickness maps for macaque monkey cortex were obtained from the publicly available BALSA database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Donahue, C. J. et al. Using diffusion tractography to predict cortical connection strength and distance: a quantitative comparison with tracers in the monkey. J. Neurosci. 36, 6758–6770 (2016)." href="/articles/s41593-018-0195-0#ref-CR55" id="ref-link-section-d1625537e2277">55</a></sup> (<a href="https://balsa.wustl.edu/study/show/W336">https://balsa.wustl.edu/study/show/W336</a>) and were produced by adapting the HCP preprocessing pipelines to work with monkey MRI data (see ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Donahue, C. J. et al. Using diffusion tractography to predict cortical connection strength and distance: a quantitative comparison with tracers in the monkey. J. Neurosci. 36, 6758–6770 (2016)." href="/articles/s41593-018-0195-0#ref-CR55" id="ref-link-section-d1625537e2288">55</a></sup> for more details). Monkey T1w/T2w map values for the left cortical hemisphere were parcellated into 91 areas using the M132 parcellation, which was used for the anatomical tract-tracing dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Donahue, C. J. et al. Using diffusion tractography to predict cortical connection strength and distance: a quantitative comparison with tracers in the monkey. J. Neurosci. 36, 6758–6770 (2016)." href="/articles/s41593-018-0195-0#ref-CR55" id="ref-link-section-d1625537e2292">55</a></sup>.</p><p>To construct maps of geodesic distance from primary visual area V1 in human and monkey cortex, pairwise geodesic distance between two parcels <i>i</i> and <i>j</i> was calculated as the average of all pairwise surface-based distances between grayordinate vertices in parcel <i>i</i> and vertices in parcel <i>j</i>.</p><h3 class="c-article__sub-heading" id="Sec11">Anatomical hierarchy levels in monkey cortex</h3><p>To assess whether macaque cortical T1w/T2w maps could reliably capture the laminar-specific interareal projection patterns conventionally used to define anatomical hierarchy, we fit a generalized linear model (GLM) to quantitative laminar projection data, yielding ordinal hierarchy values in 89 cortical areas, following the procedure in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e2319">15</a></sup>. Anatomical tract-tracing data, derived from retrograde tracers, was obtained from the publicly available Core-Nets database (<a href="http://core-nets.org">http://core-nets.org</a>). Retrograde tracer was injected into a target area <i>i</i>, and the number of tracer-labeled neurons in source area <i>j</i> were counted. The fraction of external labeled neurons, <i>FLNe</i><sub><i>ij</i></sub>, provides a quantitative measure of connection strength, defined as the number of labeled neurons in the source area normalized by the total number of labeled neurons in all external cortical source areas for a given injection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Markov, N. T. et al. A weighted and directed interareal connectivity matrix for macaque cerebral cortex. Cereb. Cortex 24, 17–36 (2014)." href="/articles/s41593-018-0195-0#ref-CR56" id="ref-link-section-d1625537e2343">56</a></sup>. Labeled neurons in source areas are classified by their location in either supragranular or infragranular layers. For a given projection, the proportion of supragranular labeled neurons, <i>SLN</i><sub><i>ij</i></sub>, is defined as the ratio of <i>N</i><sub>supra</sub> to <i>N</i><sub>supra + </sub><i>N</i><sub>infra</sub> for neurons labeled in source area <i>j</i>. As feedforward and feedback connections preferentially originate in supragranular and infragranular layers, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. Cereb. Cortex 1, 1–47 (1991)." href="#ref-CR13" id="ref-link-section-d1625537e2368">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Barbas, H. &amp; Rempel-Clower, N. Cortical structure predicts the pattern of corticocortical connections. Cereb. Cortex 7, 635–646 (1997)." href="#ref-CR14" id="ref-link-section-d1625537e2368_1">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e2371">15</a></sup>, SLN is a quantitative measure of hierarchical distance between two cortical areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e2375">15</a></sup>: within this model for laminar-specific projection motifs, a pure feedforward connection from source area <i>j</i> to target area <i>i</i> would originate entirely in the superficial layers, resulting in an SLN of 1. Conversely, a pure feedback projection originating entirely in deep infragranular layers would result in an SLN of 0. We note that two of the 91 areas in the M132 parcellation—subiculum and piriform—were excluded from the model-fitting procedure, as SLN for these two areas was undefined.</p><p>The GLM procedure for estimating hierarchy levels from SLN data is described in detail in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e2388">15</a></sup>. In brief, the hypothesis that SLN is a measure of hierarchical distance can be expressed as <i>g</i>(<i>SLN</i><sub><i>ij</i></sub>) = <i>H</i><sub><i>i</i></sub> – <i>H</i><sub><i>j</i></sub>, where <i>H</i><sub><i>i</i></sub> corresponds to the hierarchical level of area <i>i</i> and <i>g</i> is an arbitrary and possibly nonlinear function linking SLN values on the unit interval (0, 1) to their corresponding hierarchical distance. We used a logit link function to map SLN values from the unit interval to the entire real number line following the procedure in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. Neuron 88, 419–431 (2015)." href="/articles/s41593-018-0195-0#ref-CR19" id="ref-link-section-d1625537e2426">19</a></sup>. Fitting linear predictors (i.e., hierarchical levels) to logit-transformed SLN values constitutes a type of generalized linear model, with maximum likelihood estimation assuming a binomial family probability distribution for the supra- and infragranular neuron counts. To assign more weight to stronger connections during model estimation of hierarchical levels, we also weighted each pathway in the model by the negative logarithm of the FLNe value. We clipped SLN values to lie in the interval (0.01, 0.99) so the logit-transformed SLN value was well-defined for all pathways used to fit the model. Furthermore, to reduce the impact of noise on model parameter estimation, we only included those <i>N</i> = 1,243 pathways that contained at least 10 projection neurons when fitting the GLM; we confirmed that results were generally robust to the choice of neuron count threshold.</p><p>Maximum likelihood estimation of model parameters was done in the R programming language using the glm function. The model-estimated hierarchy levels, invariant under linear transformations, were shifted and rescaled to span the unit interval [0,1]. To assess the statistical relationship between T1w/T2w map value and hierarchy level, we calculated the Spearman rank correlation between the 89 ordinal hierarchy values and their corresponding parcellated T1w/T2w map values (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1f</a>). For visual clarity in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1e,f</a> we removed the nonlinear logit transformation by displaying model-estimated hierarchy levels after applying the inverse-logit (i.e., logistic) transformation. This rescaling preserves the ordering of areas and therefore does not affect the reported Spearman rank correlations.</p><h3 class="c-article__sub-heading" id="Sec12">Macaque monkey anatomical data: cytoarchitectural types, inhibitory interneuron densities, and pyramidal neuron spine counts</h3><p>To quantify the statistical relationship between T1w/T2w map value and categorical cytoarchitectural type (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4a</a>), we compared T1w/T2w map values to structural classification values reported for 29 regions of primate visual cortex, obtained from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Hilgetag, C. C., Medalla, M., Beul, S. F. &amp; Barbas, H. The primate connectome in context: Principles of connections of the cortical visual system. Neuroimage 134, 685–702 (2016)." href="/articles/s41593-018-0195-0#ref-CR22" id="ref-link-section-d1625537e2454">22</a></sup>. To characterize hierarchical distributions of cortical inhibitory interneuron cell types (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4d</a>), we compiled, from multiple immunohistochemical studies, the relative densities of inhibitory interneurons which are immunoreactive (ir) to the calcium-binding proteins parvalbumin (PV) and calretinin (CR)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Condé, F., Lund, J. S., Jacobowitz, D. M., Baimbridge, K. G. &amp; Lewis, D. A. Local circuit neurons immunoreactive for calretinin, calbindin D-28k or parvalbumin in monkey prefrontal cortex: distribution and morphology. J. Comp. Neurol. 341, 95–116 (1994)." href="#ref-CR57" id="ref-link-section-d1625537e2461">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gabbott, P. L. &amp; Bacon, S. J. Local circuit neurons in the medial prefrontal cortex (areas 24a,b,c, 25 and 32) in the monkey: II. Quantitative areal and laminar distributions. J. Comp. Neurol. 364, 609–636 (1996)." href="#ref-CR58" id="ref-link-section-d1625537e2461_1">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kondo, H., Tanaka, K., Hashikawa, T. &amp; Jones, E. G. Neurochemical gradients along monkey sensory cortical pathways: calbindin-immunoreactive pyramidal neurons in layers II and III. Eur. J. Neurosci. 11, 4197–4203 (1999)." href="#ref-CR59" id="ref-link-section-d1625537e2461_2">59</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Dombrowski, S. M., Hilgetag, C. C. &amp; Barbas, H. Quantitative architecture distinguishes prefrontal cortical systems in the rhesus monkey. Cereb. Cortex 11, 975–988 (2001)." href="/articles/s41593-018-0195-0#ref-CR60" id="ref-link-section-d1625537e2464">60</a></sup>. To characterize hierarchical variation in pyramidal neuron excitatory synaptic connectivity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4g</a>), we compiled, from multiple studies by Elston and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Elston, G. N. &amp; Rosa, M. G. The occipitoparietal pathway of the macaque monkey: comparison of pyramidal cell morphology in layer III of functionally related cortical visual areas. Cereb. Cortex 7, 432–452 (1997)." href="#ref-CR61" id="ref-link-section-d1625537e2472">61</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Elston, G. N. &amp; Rosa, M. G. Morphological variation of layer III pyramidal neurones in the occipitotemporal pathway of the macaque monkey visual cortex. Cereb. Cortex 8, 278–294 (1998)." href="#ref-CR62" id="ref-link-section-d1625537e2472_1">62</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Elston, G. N., Tweedale, R. &amp; Rosa, M. G. Cortical integration in the visual system of the macaque monkey: large-scale morphological differences in the pyramidal neurons in the occipital, parietal and temporal lobes. Proc. Biol. Sci. 266, 1367–1374 (1999)." href="#ref-CR63" id="ref-link-section-d1625537e2472_2">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Elston, G. N. &amp; Rockland, K. S. The pyramidal cell of the sensorimotor cortex of the macaque monkey: phenotypic variation. Cereb. Cortex 12, 1071–1078 (2002)." href="#ref-CR64" id="ref-link-section-d1625537e2472_3">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Elston, G. N., Benavides-Piccione, R. &amp; Defelipe, J. A study of pyramidal cell structure in the cingulate cortex of the macaque monkey with comparative notes on inferotemporal and primary visual cortex. Cereb. Cortex 15, 64–73 (2005)." href="#ref-CR65" id="ref-link-section-d1625537e2472_4">65</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Elston, G. N., Oga, T., Okamoto, T. &amp; Fujita, I. Spinogenesis and pruning in the anterior ventral inferotemporal cortex of the macaque monkey: An intracellular injection study of layer III pyramidal cells. Front. Neuroanat. 5, 42 (2011)." href="/articles/s41593-018-0195-0#ref-CR66" id="ref-link-section-d1625537e2475">66</a></sup>, the number of spines of basal-dendritic trees of L3 pyramidal neurons.</p><p>For each of these three analyses, we produced a mapping between the 91 areas in the M132 atlas parcellation, which was used to calculate parcellated T1w/T2w map values in monkey cortex, to the architectonic areas reported in these collated studies (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM3">1</a>). Where the anatomical mapping was not a one-to-one correspondence, we mapped the reported architectonic area onto the set of all M132 parcels with nonzero spatial overlap, and the corresponding T1w/T2w map value was calculated as the average across these M132 parcels.</p><h3 class="c-article__sub-heading" id="Sec13">Gene expression preprocessing</h3><p>The Allen Human Brain Atlas (AHBA) is a publicly available transcriptional atlas containing gene expression data measured with DNA microarrays and sampled from hundreds of histologically validated neuroanatomical structures across six (five male and one female) normal postmortem human brains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-018-0195-0#ref-CR6" id="ref-link-section-d1625537e2493">6</a></sup>. After no significant interhemispheric transcriptional differences were observed in the first two bilaterally profiled brains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-018-0195-0#ref-CR6" id="ref-link-section-d1625537e2497">6</a></sup>, the remaining four donor brains were profiled only in the left cortical hemisphere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e2501">7</a></sup>. To construct parcellated group-averaged gene expression profiles, we therefore restricted all analyses to microarray data sampled from the left cortical hemisphere in each of the six brains. Microarray expression data and all accompanying metadata were downloaded from the AHBA (<a href="http://human.brain-map.org">http://human.brain-map.org</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-018-0195-0#ref-CR6" id="ref-link-section-d1625537e2512">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e2515">7</a></sup>. The raw microarray expression data for each of the six donors includes expression levels of 20,737 genes, profiled by 58,692 microarray probes. These data were preprocessed according to the following procedure:</p><ol class="u-list-style-none">
                  <li>
                    <span class="u-custom-list-number">1.</span>
                    
                      <p>Gene probes without a valid Entrez Gene ID were excluded.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">2.</span>
                    
                      <p>Microarray samples exhibiting exceptionally low interareal similarity were excluded. We first computed the spatial correlation matrix of expression values between samples using the remaining 48,170 probes, then summed this matrix across all samples. Samples whose similarity measure was more than 5 s.d. below the mean across all samples were excluded. At most, this step excluded three samples within a subject.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">3.</span>
                    
                      <p>Samples whose annotations did not indicate that they originated in the left hemisphere of the cerebral cortex were excluded. To focus analysis on neocortex, we also excluded samples taken from cortical structures that are cytoarchitecturally similar to the hippocampus, including piriform cortex, the parahippocampal gyrus, and the hippocampal formation.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">4.</span>
                    
                      <p>Samples whose measured expression level was not well above background, as provided in the AHBA dataset, were excluded<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e2563">7</a></sup>. Samples surviving this step (i) belonged to a probe whose mean signal was significantly different from the corresponding background, and (ii) had a background-subtracted signal which was at minimum 2.6 times greater than the s.d. of the background.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">5.</span>
                    
                      <p>The remaining cortical samples were mapped from volumetric space to the two-dimensional cortical surface by minimizing the pairwise Euclidean distance between the stereotaxic MNI coordinates reported for each cortical sample and the coordinates of grayordinate vertices in each subject’s native cortical surface mesh (which was constructed using the procedure described in the following section below). Samples whose Euclidean distance to the nearest surface vertex was more than 2 s.d. above the mean distance computed across all samples were excluded (excluding between two and 15 samples per subject). An average of 203 ± 27 samples per subject, yielding 1,220 total samples across all six subjects, remained at this stage.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">6.</span>
                    
                      <p>Expression levels for samples mapped onto the same surface vertex were averaged. Then expression levels within each remaining sample were <i>z</i>-scored across all gene probes.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">7.</span>
                    
                      <p>Using cortical samples mapped onto subjects’ native surface meshes, expression profiles for each of the 180 unilateral parcels in the HCP’s MMP1.0 cortical parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-018-0195-0#ref-CR21" id="ref-link-section-d1625537e2604">21</a></sup> were computed in one of the two following ways. (i) For parcels that had at least one sample mapped directly onto one of their constituent surface vertices, parcellated expression values were computed by averaging expression levels across all samples mapped directly onto the parcel. (ii) For parcels that had no samples mapped onto any of their constituent vertices, we first created densely interpolated expression maps, in which each vertex in the native surface mesh was assigned the expression level associated with the most proximal surface vertex onto which a sample had been directly mapped, determined using surface-based geodesic distance along each subject’s cortical surface mesh (i.e., a Voronoi diagram approach); the average of expression levels across parcels’ constituent vertices was then computed to obtain parcellated expression values, effectively equivalent to performing a weighted average.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">8.</span>
                    
                      <p>A coverage score was also assigned to each gene probe, defined as the fraction of 180 parcels that had at least one sample mapped directly onto one of its constituent surface vertices. Probes with coverage below 0.4 (i.e., probes for which fewer than 72 of the 180 parcels contained samples) were excluded from further analysis.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">9.</span>
                    
                      <p>For each gene profiled by multiple gene probes, we selected and used the expression profile of a single representative probe. If two probes were available, we selected the probe with maximum gene expression variance across sampled cortical structures to more reliably capture spatial patterns of areal heterogeneity. If three or more probes were available, we computed a correlation matrix of parcellated gene expression values across the available gene probes, summed the resultant matrix along one of its dimensions to obtain a quantitative similarity measure for each probe relative to the other gene probes, and selected the probe with the highest similarity measure, as it is most highly representative among all available gene probes.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">10.</span>
                    
                      <p>Each subject-level gene expression profile was <i>z</i>-scored before we computed group-level expression profiles, which were obtained by computing the mean across subjects which were assigned a probe for that gene. Group-level gene expression profiles were not computed if fewer than four subjects had an available gene probe. Finally, group-level expression profiles were <i>z</i>-scored across all 180 areas for each gene.</p>
                    
                  </li>
                </ol><p>These steps yielded group-averaged expression values for 16,088 genes across 180 cortical areas, which were used for all reported analyses. The T1w/T2w map correlation (TMC) for each gene is reported in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM4">2</a>. We also replicated all reported findings after mapping subjects’ gene expression samples to the HCP’s group-averaged surface mesh instead of subjects’ native surface meshes in step 5 above. However, we found that native surface-based expression sample mapping yielded slightly stronger TMCs and improved spatial registration in general (not shown).</p><h3 class="c-article__sub-heading" id="Sec14">Native surface mesh construction</h3><p>Single-subject surface registration for each of the six subjects in the AHBA was performed following a procedure adapted from the HCP’s minimal preprocessing pipelines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-018-0195-0#ref-CR53" id="ref-link-section-d1625537e2665">53</a></sup>. Briefly, the T1w image was first rigidly aligned to the MNI coordinate axes to produce a native space volume, which was then nonlinearly registered to the standard MNI template using FSL’s FLIRT and FNIRT. The native space image was run through FreeSurfer’s recon-all pipeline, which performs automated segmentation of brain structures to reconstruct the white matter and pial surfaces. The FreeSurfer output surface was then converted to standard GIFTI format to produce each subject’s native surface mesh. Finally, subjects’ native surface meshes were registered to the standard HCP surface mesh.</p><h3 class="c-article__sub-heading" id="Sec15">Categorical gene sets</h3><p>We conducted analyses on biologically and physiologically meaningful gene sets extracted from existing databases and neuroscientific literature, reported below (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM4">2</a>).</p><ol class="u-list-style-none">
                  <li>
                    <span class="u-custom-list-number">1.</span>
                    
                      <p><b>Brain-specific</b>. We obtained <i>N</i> = 2,413 genes with expression specific to human brain tissue, relative to other tissues, from supplementary dataset 1 of ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Fagerberg, L. et al. Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics. Mol. Cell. Proteom. 13, 397–406 (2014)." href="/articles/s41593-018-0195-0#ref-CR67" id="ref-link-section-d1625537e2695">67</a></sup>. Following ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Genovese, G. et al. Increased burden of ultra-rare protein-altering variants among 4,877 individuals with schizophrenia. Nat. Neurosci. 19, 1433–1441 (2016)." href="/articles/s41593-018-0195-0#ref-CR30" id="ref-link-section-d1625537e2699">30</a></sup>, brain-specific genes were selected, for which expression in brain tissue was four times higher than the median expression across all 27 different tissues.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">2.</span>
                    
                      <p><b>Neuron- and oligodendrocyte-specific</b>. Brain genes with expression specific to neurons (<i>N</i> = 2,530) or oligodendrocytes (<i>N</i> = 1,769), relative to other CNS cell types, were obtained from supplementary dataset S3b of ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Cahoy, J. D. et al. A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. J. Neurosci. 28, 264–278 (2008)." href="/articles/s41593-018-0195-0#ref-CR68" id="ref-link-section-d1625537e2722">68</a></sup>. Following ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Genovese, G. et al. Increased burden of ultra-rare protein-altering variants among 4,877 individuals with schizophrenia. Nat. Neurosci. 19, 1433–1441 (2016)." href="/articles/s41593-018-0195-0#ref-CR30" id="ref-link-section-d1625537e2726">30</a></sup>, neuron-specific genes were selected, for which log-expression in neurons of P7n cell type in the mouse was 0.5 times greater than the median log-expression across 11 CNS cell types.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">3.</span>
                    
                      <p><b>Synaptome</b>. We aggregated four sets of synaptic genes (<i>N</i> = 1,886 in total) encoding proteins found in the presynaptic nerve terminal, presynaptic active zone, synaptic vesicles, and postsynaptic density, which were obtained from SynaptomeDB, an ontology-based database of genes in the human synaptome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Pirooznia, M. et al. SynaptomeDB: an ontology-based knowledgebase for synaptic genes. Bioinformatics 28, 897–899 (2012)." href="/articles/s41593-018-0195-0#ref-CR31" id="ref-link-section-d1625537e2746">31</a></sup>.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">4.</span>
                    
                      <p><b>Neuron subtype-specific</b>. Gene sets representing distinct classes of neuronal subtypes were obtained from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Lake, B. B. et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. Science 352, 1586–1590 (2016)." href="/articles/s41593-018-0195-0#ref-CR26" id="ref-link-section-d1625537e2763">26</a></sup>, in which clustering and classification analyses yielded 16 distinct neuron subtypes, on the basis of differential gene expression measured by RNA sequencing from single neurons in human cortex. The fraction of positive values using exon-only derived transcripts per million (TPM) associated with each subtype-specific gene were obtained from supplementary table S5 of ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Lake, B. B. et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. Science 352, 1586–1590 (2016)." href="/articles/s41593-018-0195-0#ref-CR26" id="ref-link-section-d1625537e2767">26</a></sup>; within each neuronal subtype cluster, the TPM values for the cluster genes were normalized and used to create a weighted gene expression profile representative of each subtype’s spatial topography (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig12">4</a>).</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">5.</span>
                    
                      <p><b>Layer-specific</b>. Sets of laminar-specific genes localized to different layers of human neocortex were obtained from supplementary table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM4">S2</a> of ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Zeng, H. et al. Large-scale cellular-resolution gene profiling in human neocortex reveals species-specific molecular signatures. Cell 149, 483–496 (2012)." href="/articles/s41593-018-0195-0#ref-CR23" id="ref-link-section-d1625537e2790">23</a></sup>. Genes were broadly grouped into sets representative of supragranular (L1–L3), granular (L4), and infragranular (L5/6) layers.</p>
                    
                  </li>
                </ol><h3 class="c-article__sub-heading" id="Sec16">Spatial autoregressive modeling</h3><p>Significance values indicated by the number of stars reported on bar plots for T1w/T2w map correlations (TMCs) were corrected to account for spatial autocorrelation structure in parcellated T1w/T2w maps and gene expression maps. Because physical quantities like microstructural tissue composition and gene expression must vary smoothly and continuously in space, measurements recorded from proximal cortical areas tend to be more similar than measurements recorded from distal areas of cortex. This departure from the assumption of independent observations biases calculations of statistical significance. To model this spatial autocorrelation, we used a spatial lag model (SLM) commonly applied in the spatial econometrics literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Anselin, L. Spatial econometrics. in A Companion to Theoretical Econometrics (ed. Baltagi, B.H.) 310–330 (Blackwell, Oxford, UK, 2001)." href="/articles/s41593-018-0195-0#ref-CR69" id="ref-link-section-d1625537e2805">69</a></sup>, of the form <i>y</i> <i>=</i> ρ<i>Wy</i> <i>+</i> <i>Xβ</i> <i>+</i> <i>ν</i>, where <i>W</i> is a user-defined weight matrix implicitly specifying the form of spatial structure in the data, and <i>ν</i> is normally distributed.</p><p>To implement a spatial lag model in the python programming language, we used the maximum likelihood estimation routine defined in the Python Spatial Analysis Library (pysal)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Fischer, M. M. &amp; Getis, A. Handbook of Applied Spatial Analysis: Software Tools, Methods and Applications. (Springer, Heidelberg, Germany, 2010)." href="/articles/s41593-018-0195-0#ref-CR70" id="ref-link-section-d1625537e2840">70</a></sup>. We first determined the surface-based spatial separation between each pair of cortical parcels by computing the mean of the pairwise geodesic distances between each vertex in parcel <i>i</i> and each vertex in parcel <i>j</i>, from which we constructed a pairwise parcel distance matrix, <i>D</i>.</p><p>Similarity of gene expression profiles was well-approximated by an exponential decaying spatial autocorrelation function (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig15">7a,b</a>), as was found in mouse cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Fulcher, B. D. &amp; Fornito, A. A transcriptional signature of hub connectivity in the mouse connectome. Proc. Natl Acad. Sci. USA 113, 1435–1440 (2016)." href="/articles/s41593-018-0195-0#ref-CR12" id="ref-link-section-d1625537e2859">12</a></sup>. We fit the correlation of gene expression profiles between two areas with the exponential function Corr(<i>x</i><sub><i>i</i></sub>,<i>x</i><sub><i>j</i></sub>) ~ exp(–<i>D</i><sub><i>ij</i></sub>/<i>d</i><sub>0</sub>), where <i>x</i><sub><i>i</i></sub> and <i>x</i><sub><i>j</i></sub> are vectors containing the parcellated gene expression values at parcels <i>i</i> and <i>j</i>, <i>D</i><sub><i>ij</i></sub> is the geodesic distance between the parcels, and <i>d</i><sub>0</sub> is the characteristic spatial scale of autocorrelation. We empirically determined <i>d</i><sub>0</sub> by first computing the pairwise gene co-expression matrix <i>C</i><sub><i>ij</i></sub> ≡ Corr(<i>x</i><sub><i>i</i></sub>,<i>x</i><sub><i>j</i></sub>). We then fit the free parameter <i>d</i><sub>0</sub> using ordinary least-squares (OLS) regression on the off-diagonal (upper-triangular) elements of the gene co-expression and parcel distance matrices, to minimize the sum-of-squared-residuals between empirical and model-estimated gene co-expression values over all pairs of cortical parcels, <span class="mathjax-tex">\(S = \mathop {\sum }\limits_{i &gt; j} r_{ij}^2 = \mathop {\sum }\limits_{i &gt; j} \left[ {C_{ij} - {\mathrm{exp}}\left( { - D_{ij}/d_0} \right)} \right]^2\)</span>. This empirical fit was performed on the gene co-expression matrix computed using the set of brain-specific genes. Using the OLS estimate of the spatial autocorrelation scale from the fit to the empirical gene expression data, we calculated the elements of the spatial weight matrix, <i>W</i><sub><i>ij</i></sub> = exp(–<i>D</i><sub><i>ij</i></sub>/<i>d</i><sub>0</sub>). Finally, we fit the SLM to parcellated gene expression profiles, using the maximum-likelihood estimator routine (pysal.spreg.ml_lag.ML_Lag) in pysal. <i>P</i> values indicated by the number of stars in the bar plots of T1w/T2w map correlations (TMCs) correspond to <i>P</i> values for model parameter β defined above.</p><p>Of note, spatial autoregressive model parameters do not have the same interpretation as they do in OLS regression. The parameter β reflects the direct (i.e., local) impact on the dependent variable <i>y</i> due to a unit change in the independent variable <i>x</i>. In addition, because of the underlying spatial structure, the direct impact of <i>x</i><sub><i>i</i></sub> on <i>y</i><sub><i>i</i></sub> results in an indirect effect of <i>y</i><sub><i>i</i></sub> on neighboring <i>y</i><sub><i>j</i></sub>. Therefore β cannot be interpreted as a corrected, global correlation coefficient, and we restrict our use of the SLM to correcting for the biasing effect of spatially autocorrelated samples on reported significance values.</p><h3 class="c-article__sub-heading" id="Sec17">Theil–Sen estimator</h3><p>Grey trend lines in all figures were calculated using the Theil–Sen estimator, a nonparametric estimator of linear slope based on Kendall’s tau rank correlation that is insensitive to the underlying distribution and robust to statistical outliers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Sen, P. K. Estimates of the regression coefficient based on Kendall’s tau. J. Am. Stat. Assoc. 63, 1379–1389 (1968)." href="/articles/s41593-018-0195-0#ref-CR71" id="ref-link-section-d1625537e3164">71</a></sup>. It is defined as the median of the set of slopes computed between all pairs of points. From the Theil–Sen estimate of linear slope, <i>m</i>, we define the y-intercept used for the best-fit trend line as <i>y</i><sub>int</sub> ≡ median(<i>y</i>) – <i>m</i> median(<i>x</i>), such that the trend line runs through the median of the fitted data.</p><h3 class="c-article__sub-heading" id="Sec18">Principal components analysis</h3><p>We used principal component analysis (PCA) to identify the dominant modes of spatial variation in the transcriptional profiles of gene expression in the human cortex. For a set of <i>N</i> genes, each with group-averaged expression values for <i>P</i> cortical parcels, we constructed a gene expression matrix <b>G</b> with one row for each cortical parcel and one column for each unique gene (i.e., with dimensions <i>P</i> <i>×</i> <i>N</i>). The <i>P</i> × <i>P</i> spatial covariance matrix <b>C</b> was constructed by computing the covariance between vectors of gene expression values for each pair of cortical parcels: <i>C</i><sub><i>ij</i></sub> = Cov(<i>G</i><sub><i>i</i></sub>,<i>G</i><sub><i>j</i></sub>), where <i>G</i><sub><i>i</i></sub> is the <i>i</i>th row in the matrix G, corresponding to the vector of <i>N</i> gene expression values for the <i>i</i>th cortical parcel. Eigendecomposition is performed on the spatial covariance matrix to obtain the matrix eigenvectors (i.e., the principal components, PCs) and their corresponding eigenvalues, which are proportional to the amount of variance captured by the corresponding PC. To enumerate each principal component, eigenvalues are ranked in descending order of absolute magnitude, with larger magnitudes indicating a greater proportion of the total variance captured by the associated PC (i.e., the associated mode of spatial covariation). PCA therefore allows for simultaneous identification of spatial patterns of covariation and quantification of the extent to which these spatial modes capture variance in cortical gene expression profiles.</p><p>To quantify the overlap of these spatial PCs with the cortical T1w/T2w map, we compute the Spearman rank correlation coefficient between each <i>P</i>-dimensional PC and the <i>P</i>-dimensional vector of T1w/T2w map values, across all cortical parcels. We can quantify the amount of gene expression variance that is captured by any given spatial map, such as the T1w/T2w map (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5e</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig14">6f–j</a>): from the spatial covariance matrix <b>C</b>, the variance captured along a unit-length vector <b>a</b>, here a demeaned and normalized map, is given by <b>a</b><sup><i>┬</i></sup><b>Ca</b>.</p><h3 class="c-article__sub-heading" id="Sec19">Differential stability</h3><p>Differential stability (DS) is a correlation-based metric which quantifies the consistency of spatial gene expression patterns across individual brains. DS was originally defined in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e3294">7</a></sup> as “the tendency for a gene to exhibit reproducible differential expression relationships across brain structures.” To compute DS for a gene, we calculated the average pairwise Spearman rank correlation (<i>r</i><sub><i>s</i></sub>) across all subject-level gene expression profiles, for the (four to six) AHBA subjects with an available gene probe, for a maximum of 15 possible pairs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM4">2</a>). That is, for gene <i>g</i> whose expression profile across 180 cortical areas in brain <i>i</i> is the vector <i>b</i><sub><i>i</i></sub>(<i>g</i>), we define the DS in cortex (DS<sub>c</sub>) by <span class="mathjax-tex">\({\mathrm{DS}}_{\mathrm{c}}\left( g \right) \equiv \frac{2}{{n(n-1)}}\mathop {\sum }\limits_{i = 1}^n \mathop {\sum }\limits_{j &gt; i} r_s\left( {b_i\left( g \right),b_j\left( g \right)} \right)\)</span>, where <i>n</i> is the number of subjects with an available gene probe. We note that DS is therefore defined with respect to a specified set of brain structures, in this case 180 unilateral cortical areas. We note that any differences between cortical DS values shown in our Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig7">7a</a> and those shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig7">7b</a> of ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e3503">7</a></sup> are due to (i) different cortical parcellations (containing 180 vs. 52 parcels, respectively); (ii) different preprocessing procedures; and (iii) different correlation coefficients (Spearman vs. Pearson, respectively).</p><h3 class="c-article__sub-heading" id="Sec20">Functional enrichment analyses</h3><p>Functional enrichments were determined using the ToppGene (<a href="https://toppgene.cchmc.org/">https://toppgene.cchmc.org/</a>) web portal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritization. Nucleic Acids Res. 37, W305–11 (2009)." href="/articles/s41593-018-0195-0#ref-CR32" id="ref-link-section-d1625537e3523">32</a></sup>, including gene ontology annotations (biological process, cellular component, and molecular function); microRNA targets (from all sources indicated on <a href="https://toppgene.cchmc.org/navigation/database.jsp">https://toppgene.cchmc.org/navigation/database.jsp</a>); and drug annotations (from DrugBank, Comparative Toxicogenomics Database, including marker and therapeutic, and Broad Institute CMAP). Significant genes in each category were identified using the ToppFun utility. Disease annotations were determined using curated disease gene associations in the DisGeNet database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Piñero, J. et al. DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants. Nucleic Acids Res. 45, D833–D839 (2017). D1." href="/articles/s41593-018-0195-0#ref-CR34" id="ref-link-section-d1625537e3534">34</a></sup> (<a href="http://www.disgenet.org/web/DisGeNET/menu/home">http://www.disgenet.org/web/DisGeNET/menu/home)</a>. Hypergeometric testing was used to determine significant over-representation of brain-related disease genes in the top and bottom gene quintiles (20%, 3,218 genes) ranked by T1w/T2w map correlation, following ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e3546">7</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec21">Statistical methods</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">Multiple comparisons corrections</h4><p>Significance values indicated by the number of stars reported on bar plots for T1w/T2w map correlations (TMCs) were Bonferroni-corrected for multiple comparisons. Specifically, statistical significance thresholds (*<i>P</i> <i>&lt;</i> 0.05, **<i>P</i> <i>&lt;</i> 10<sup>–2</sup>, ***<i>P</i> <i>&lt;</i> 10<sup>–3</sup>) were divided by the number of null hypotheses tested, i.e., by the number of constituent bars contained in each bar plot (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig12">4</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig13">5</a>, and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig18">10</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec23">Surrogate data generation</h4><p>To nonparametrically determine significance values in our PCA results, for Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig14">6</a>, we generated surrogate maps with a spatial autocorrelation structure that matched the empirical data (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig15">7c</a>). Parameters characterizing the empirical spatial autocorrelation were determined numerically for the cortical T1w/T2w map, cortical thickness map, and the map of surface-based geodesic distance from area V1; in each case, we fit the data using a spatial lag model of the form <b>y</b> <b>=</b> ρ<b>Wy</b>, where <b>y</b> is a vector of first Box–Cox transformed and then mean-subtracted map values. The Box–Cox transformation was first applied to the maps so their values were approximately normally distributed. <b>W</b> is the row-normalized weight matrix with zero diagonal and off-diagonal elements proportional to <span class="mathjax-tex">\(W_{ij} = z_i^{ - 1}{\mathrm{exp}}\left( { - D_{ij}/d_0} \right)\)</span>, where <i>D</i><sub><i>ij</i></sub> is the surface-based geodesic distance between cortical areas <i>i</i> and <i>j</i>, and <span class="mathjax-tex">\(z_i \equiv \mathop {\sum }\limits_j {\mathrm{exp}}\left( { - D_{ij}/d_0} \right)\)</span> is a row-wise normalization factor. Weights <i>W</i><sub><i>ij</i></sub> define the fraction of spatial influence on area <i>i</i> attributable to area <i>j</i>. Two free parameters, ρ and <i>d</i><sub>0</sub>, are estimated by minimizing the residual sum-of-squares<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Anselin, L. Spatial econometrics. in A Companion to Theoretical Econometrics (ed. Baltagi, B.H.) 310–330 (Blackwell, Oxford, UK, 2001)." href="/articles/s41593-018-0195-0#ref-CR69" id="ref-link-section-d1625537e3848">69</a></sup>. Using best-fit parameter values <span class="mathjax-tex">\(\hat \rho\)</span> and <span class="mathjax-tex">\(\hat{d}_0\)</span>, surrogate maps <b>y</b><sub>surr</sub> are generated according to <span class="mathjax-tex">\({\mathbf{y}}_{{\mathrm{surr}}} = \left( {{\Bbb I} - \hat \rho {\mathbf{W}}\left[ {\hat d_0} \right]} \right)^{ - 1}{\mathbf{u}}\)</span>, where <b>u</b> ~ <span class="stix">𝒩</span>(0,1). To match surrogate map value distributions to the distribution of values in the corresponding empirical map (for example, the T1w/T2w map), rank-ordered surrogate map values were re-assigned the corresponding rank-ordered values in the empirical map. Note that this approach to surrogate data generation approximates a spatial autocorrelation-preserving permutation test of the empirical neuroimaging map. Using these surrogate maps, we constructed null distributions for <i>N</i> <i>=</i> 10,000 statistics and report significance values as the proportion of samples in the null distributions whose absolute value is greater than or equal to the absolute value of the test statistic. To compute significance values reported in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig6">6c,f</a>, we first constructed null distributions of the statistic <span class="mathjax-tex">\(\sigma _{{\mathrm{Map}}}^2/\sigma _{{\mathrm{PC}}1}^2\)</span> using surrogate maps constructed for each neuroimaging map. For each neuroimaging map, we then computed distributions of the difference between the test statistic and each sample statistic in the null distribution. Finally, we used the nonparametric Wilcoxon signed-rank test on these difference distributions, one for the T1w/T2w map and one for either the cortical thickness or geodesic distance map, to test for statistically significant differences in the means of the distributions. The interpretation of our statistically significant results reported in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig6">6c,f</a> is that the T1w/T2w map tends to capture a more appreciable fraction of gene expression variance, relative to its surrogate maps, than do either of the other two candidate neuroimaging maps.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">Jackknife estimate of standard error</h4><p>To nonparametrically estimate the error on reported Spearman rank correlations between length-<i>N</i> vectors <b><i>x</i></b> and <b><i>y</i></b>, we used a leave-one-out jackknife replication procedure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Tukey, J. W. Bias and confidence in not-quite large samples. Ann. Math. Stat. 29, 614 (1958)." href="/articles/s41593-018-0195-0#ref-CR72" id="ref-link-section-d1625537e4115">72</a></sup>. We generated <i>N</i> jackknife samples, denoted <i>θ</i><sub>−<i>i</i></sub>, by removing the <i>i</i>th element from vectors <b><i>x</i></b> and <b><i>y</i></b>, recomputing the rank correlation between the two new length-(<i>N</i> – 1) vectors <b><i>x</i></b><sub><b><i>–i</i></b></sub> and <b><i>y</i></b><sub><b><i>–i</i></b></sub>, and repeating for <i>i</i> <i>=</i> 1…<i>N</i>. Because the jackknife estimate consists of a linear operation (i.e., subtraction), we applied the Fisher <i>z</i>-transformation to the <i>N</i> jackknife-sampled correlation coefficients. The jackknife estimate of standard error on the Fisher <i>z</i>-transformed jackknife samples, <i>z</i>(<i>θ</i><sub>−<i>i</i></sub>), was then computed as <span class="mathjax-tex">\(\widehat {SE} = \sqrt {\frac{{N - 1}}{N}\mathop {\sum }\limits_{i = 1}^N \left[ {z\left( {\theta _{ - i}} \right) - \overline {z\left( \theta \right)} } \right]^2}\)</span>, where <span class="mathjax-tex">\(\overline {z\left( \theta \right)} \equiv \frac{1}{N}\mathop {\sum }\limits_{i = 1}^N z\left( {\theta _{ - i}} \right)\)</span> is the mean of the <i>N z</i>-transformed jackknife samples. We then added (or subtracted) <span class="mathjax-tex">\(\widehat {SE}\)</span> from the Fisher <i>z</i>-transformed sample statistic, <i>z</i>(<i>r</i><sub><i>s</i></sub>), before applying the inverse transformation to obtain the upper (or lower) bound on each reported error bar, i.e., <span class="mathjax-tex">\(z^{ - 1}\left( {z\left( {r_s} \right) \pm \widehat {SE}} \right)\)</span>. For all human analyses (i.e., for all correlations performed between principal components, neuroimaging maps, random surrogate maps, and gene expression profiles in Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig8">8</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig12">4</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig15">7</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig17">9</a>, and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig18">10</a>), correlations and their jackknife estimates were computed using <i>N</i> = 180 cortical areas. For monkey analyses (i.e., Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig1">1</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig2">2</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4</a>, and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig11">3</a>), we used the number of areas reported in figure legends. The interpretation of the jackknife estimate of standard error is the amount by which the correlation coefficient would change with the addition or removal of one area.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec25">Bootstrap-estimated confidence intervals</h4><p>To nonparametrically estimate confidence intervals on reported statistics derived from PCA (i.e., Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig5">5</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig7">7</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig14">6</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig18">10</a>) we performed a percentile bootstrap<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Davison, A.C. &amp; Hinkley, D.V. Bootstrap Methods and Their Application (Cambridge Univ. Press, Cambridge, UK, 1997)." href="/articles/s41593-018-0195-0#ref-CR73" id="ref-link-section-d1625537e4598">73</a></sup>. We constructed bootstrap distributions for each sample statistic by repeatedly resampling with replacement from the sample data distribution—for example, expression profiles of genes in each categorical gene set—and recomputing the test statistic on each of the resampled datasets. Specifically, we first performed PCA on the resampled data to obtain the first ten PCs and percent variance captured by each. The Procrustes rotation was used to align the bootstrap loading matrix to the sample loading matrix<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Timmerman, M. E., Kiers, H. A. &amp; Smilde, A. K. Estimating confidence intervals for principal component loadings: a comparison between the bootstrap and asymptotic results. Br. J. Math. Stat. Psychol. 60, 295–314 (2007)." href="/articles/s41593-018-0195-0#ref-CR74" id="ref-link-section-d1625537e4603">74</a></sup>, i.e., to establish the correspondence between bootstrap and sample PCs, and then the bootstrap distributions of all test statistics derived from the PCA eigenspectrum were recomputed (for example, PC1 TMCs and <span class="mathjax-tex">\(\sigma _{{\mathrm{Map}}}^2/\sigma _{{\mathrm{PC}}1}^2\)</span>). We estimated confidence intervals on sample statistics using percentiles of bootstrap distributions, i.e., the 2.5% and 97.5% percentiles for the upper and lower bounds on the 95% confidence interval, respectively. All bootstrap distributions were computed using <i>N</i> = 1,000 bootstrap samples. The bootstrap technique rests on the assumption that the sample distribution is a reasonable approximation of the ‘true’ population distribution, which is unknown; thus, statistical inference on the resampled sample data is used to approximate statistical inference on the population.</p><p>No statistical methods were used to predetermine sample sizes, but our sample sizes are the same as those reported in previous publications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-018-0195-0#ref-CR6" id="ref-link-section-d1625537e4675">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. Nat. Neurosci. 18, 1832–1844 (2015)." href="/articles/s41593-018-0195-0#ref-CR7" id="ref-link-section-d1625537e4678">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. J. Comp. Neurol. 522, 225–259 (2014)." href="/articles/s41593-018-0195-0#ref-CR15" id="ref-link-section-d1625537e4681">15</a></sup>. Nonparametric tests that do not make assumptions about the form of the data distribution were used to compute statistical significance values. Spatial autocorrelation structure in the data violated the assumption that samples were independent, though we note that this is a feature which is frequently present in neuroimaging and transcriptional data, yet rarely addressed in the neuroscience literature. No data were excluded from our analyses unless otherwise noted. Data analyses were not performed blind to the conditions of the experiments. There were no allocated experimental groups or new data collected in this study.</p><h3 class="c-article__sub-heading" id="Sec26">Reporting Summary</h3><p>Further information on experimental design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p><h3 class="c-article__sub-heading" id="Sec27">Code availability</h3><p>Custom analysis codes written in Python are available from the corresponding author upon reasonable request.</p><h3 class="c-article__sub-heading" id="Sec28">Data availability</h3><p>All results derive from data that is publicly available from sources described above. Monkey neuroimaging maps were obtained from <a href="https://balsa.wustl.edu/study/show/W336">https://balsa.wustl.edu/study/show/W336</a>. Monkey tract-tracing data were obtained from <a href="http://core-nets.org">http://core-nets.org</a>. Human gene expression data were obtained from <a href="http://human.brain-map.org">http://human.brain-map.org</a>. Annotated disease gene sets were obtained from <a href="http://www.disgenet.org/web/DisGeNET/menu/home">http://www.disgenet.org/web/DisGeNET/menu/home</a>. Monkey microanatomical data used in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig4">4</a>; correlations for each target area in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig11">3</a>; TMC and DS<sub>C</sub> values for all 16,088 genes; and constituent genes in the brain-, neuron-, oligodendrocyte-, synaptome-, and layer-specific sets are included in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM3">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM4">2</a>. Parcellated maps and connectivity matrices related to this study are available via the BALSA database (<a href="https://balsa.wustl.edu/">https://balsa.wustl.edu/</a>).</p></div></div></section>
                </div>
            

            <div>
                <div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Lennie, P. Single units and visual cortical organization. <i>Perception</i> <b>27</b>, 889–935 (1998).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1068/p270889" data-track-action="article reference" href="https://doi.org/10.1068%2Fp270889" aria-label="Article reference 1" data-doi="10.1068/p270889">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1M3is1WjtQ%3D%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10209632" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Single%20units%20and%20visual%20cortical%20organization&amp;journal=Perception&amp;doi=10.1068%2Fp270889&amp;volume=27&amp;pages=889-935&amp;publication_year=1998&amp;author=Lennie%2CP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Hasson, U., Yang, E., Vallines, I., Heeger, D. J. &amp; Rubin, N. A hierarchy of temporal receptive windows in human cortex. <i>J. Neurosci.</i> <b>28</b>, 2539–2550 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.5487-07.2008" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.5487-07.2008" aria-label="Article reference 2" data-doi="10.1523/JNEUROSCI.5487-07.2008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXjt1Ggsrk%3D" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18322098" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2556707" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20hierarchy%20of%20temporal%20receptive%20windows%20in%20human%20cortex&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.5487-07.2008&amp;volume=28&amp;pages=2539-2550&amp;publication_year=2008&amp;author=Hasson%2CU&amp;author=Yang%2CE&amp;author=Vallines%2CI&amp;author=Heeger%2CDJ&amp;author=Rubin%2CN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Lerner, Y., Honey, C. J., Silbert, L. J. &amp; Hasson, U. Topographic mapping of a hierarchy of temporal receptive windows using a narrated story. <i>J. Neurosci.</i> <b>31</b>, 2906–2915 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.3684-10.2011" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.3684-10.2011" aria-label="Article reference 3" data-doi="10.1523/JNEUROSCI.3684-10.2011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXislGitrs%3D" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21414912" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3089381" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Topographic%20mapping%20of%20a%20hierarchy%20of%20temporal%20receptive%20windows%20using%20a%20narrated%20story&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.3684-10.2011&amp;volume=31&amp;pages=2906-2915&amp;publication_year=2011&amp;author=Lerner%2CY&amp;author=Honey%2CCJ&amp;author=Silbert%2CLJ&amp;author=Hasson%2CU">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. <i>Nat. Neurosci.</i> <b>17</b>, 1661–1663 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.3862" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.3862" aria-label="Article reference 4" data-doi="10.1038/nn.3862">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFGltLbN" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25383900" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4241138" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20hierarchy%20of%20intrinsic%20timescales%20across%20primate%20cortex&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.3862&amp;volume=17&amp;pages=1661-1663&amp;publication_year=2014&amp;author=Murray%2CJD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Honey, C. J. et al. Slow cortical dynamics and the accumulation of information over long timescales. <i>Neuron</i> <b>76</b>, 423–434 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2012.08.011" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2012.08.011" aria-label="Article reference 5" data-doi="10.1016/j.neuron.2012.08.011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFClt7jN" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23083743" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517908" aria-label="PubMed Central reference 5">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Slow%20cortical%20dynamics%20and%20the%20accumulation%20of%20information%20over%20long%20timescales&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2012.08.011&amp;volume=76&amp;pages=423-434&amp;publication_year=2012&amp;author=Honey%2CCJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. <i>Nature</i> <b>489</b>, 391–399 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature11405" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature11405" aria-label="Article reference 6" data-doi="10.1038/nature11405">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xhtlymu77N" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22996553" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4243026" aria-label="PubMed Central reference 6">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20anatomically%20comprehensive%20atlas%20of%20the%20adult%20human%20brain%20transcriptome&amp;journal=Nature&amp;doi=10.1038%2Fnature11405&amp;volume=489&amp;pages=391-399&amp;publication_year=2012&amp;author=Hawrylycz%2CMJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Hawrylycz, M. et al. Canonical genetic signatures of the adult human brain. <i>Nat. Neurosci.</i> <b>18</b>, 1832–1844 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4171" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4171" aria-label="Article reference 7" data-doi="10.1038/nn.4171">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVOgtLfI" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26571460" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4700510" aria-label="PubMed Central reference 7">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Canonical%20genetic%20signatures%20of%20the%20adult%20human%20brain&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4171&amp;volume=18&amp;pages=1832-1844&amp;publication_year=2015&amp;author=Hawrylycz%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Lein, E. S., Belgard, T. G., Hawrylycz, M. &amp; Molnár, Z. Transcriptomic perspectives on neocortical structure, development, evolution, and disease. <i>Annu. Rev. Neurosci.</i> <b>40</b>, 629–652 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1146/annurev-neuro-070815-013858" data-track-action="article reference" href="https://doi.org/10.1146%2Fannurev-neuro-070815-013858" aria-label="Article reference 8" data-doi="10.1146/annurev-neuro-070815-013858">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVyhur3K" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28661727" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptomic%20perspectives%20on%20neocortical%20structure%2C%20development%2C%20evolution%2C%20and%20disease&amp;journal=Annu.%20Rev.%20Neurosci.&amp;doi=10.1146%2Fannurev-neuro-070815-013858&amp;volume=40&amp;pages=629-652&amp;publication_year=2017&amp;author=Lein%2CES&amp;author=Belgard%2CTG&amp;author=Hawrylycz%2CM&amp;author=Moln%C3%A1r%2CZ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Wang, G.-Z. et al. Correspondence between resting-state activity and brain gene expression. <i>Neuron</i> <b>88</b>, 659–666 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2015.10.022" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.10.022" aria-label="Article reference 9" data-doi="10.1016/j.neuron.2015.10.022">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVOqsrvE" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26590343" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4694561" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Correspondence%20between%20resting-state%20activity%20and%20brain%20gene%20expression&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2015.10.022&amp;volume=88&amp;pages=659-666&amp;publication_year=2015&amp;author=Wang%2CGZ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Krienen, F. M., Yeo, B. T. T., Ge, T., Buckner, R. L., &amp; Sherwood, C. C. Transcriptional profiles of supragranular-enriched genes associate with corticocortical network architecture in the human brain. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, E469–E478 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1510903113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1510903113" aria-label="Article reference 10" data-doi="10.1073/pnas.1510903113">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XjvFyisg%3D%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26739559" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4739529" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptional%20profiles%20of%20supragranular-enriched%20genes%20associate%20with%20corticocortical%20network%20architecture%20in%20the%20human%20brain&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1510903113&amp;volume=113&amp;pages=E469-E478&amp;publication_year=2016&amp;author=Krienen%2CFM&amp;author=Yeo%2CBTT&amp;author=Ge%2CT&amp;author=Buckner%2CRL&amp;author=Sherwood%2CCC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Richiardi, J. et al. Brain networks. Correlated gene expression supports synchronous activity in brain networks. <i>Science</i> <b>348</b>, 1241–1244 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1255905" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1255905" aria-label="Article reference 11" data-doi="10.1126/science.1255905">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXps1OltLs%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26068849" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4829082" aria-label="PubMed Central reference 11">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Brain%20networks.%20Correlated%20gene%20expression%20supports%20synchronous%20activity%20in%20brain%20networks&amp;journal=Science&amp;doi=10.1126%2Fscience.1255905&amp;volume=348&amp;pages=1241-1244&amp;publication_year=2015&amp;author=Richiardi%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Fulcher, B. D. &amp; Fornito, A. A transcriptional signature of hub connectivity in the mouse connectome. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, 1435–1440 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1513302113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1513302113" aria-label="Article reference 12" data-doi="10.1073/pnas.1513302113">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XovVCmug%3D%3D" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26772314" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4747775" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20transcriptional%20signature%20of%20hub%20connectivity%20in%20the%20mouse%20connectome.&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1513302113&amp;volume=113&amp;pages=1435-1440&amp;publication_year=2016&amp;author=Fulcher%2CBD&amp;author=Fornito%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Felleman, D. J. &amp; Van Essen, D. C. Distributed hierarchical processing in the primate cerebral cortex. <i>Cereb. Cortex</i> <b>1</b>, 1–47 (1991).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/1.1.1" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2F1.1.1" aria-label="Article reference 13" data-doi="10.1093/cercor/1.1.1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK38zltlGmsg%3D%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1822724" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Distributed%20hierarchical%20processing%20in%20the%20primate%20cerebral%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2F1.1.1&amp;volume=1&amp;pages=1-47&amp;publication_year=1991&amp;author=Felleman%2CDJ&amp;author=Essen%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Barbas, H. &amp; Rempel-Clower, N. Cortical structure predicts the pattern of corticocortical connections. <i>Cereb. Cortex</i> <b>7</b>, 635–646 (1997).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/7.7.635" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2F7.7.635" aria-label="Article reference 14" data-doi="10.1093/cercor/7.7.635">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1c%2FjvVymtA%3D%3D" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9373019" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20structure%20predicts%20the%20pattern%20of%20corticocortical%20connections&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2F7.7.635&amp;volume=7&amp;pages=635-646&amp;publication_year=1997&amp;author=Barbas%2CH&amp;author=Rempel-Clower%2CN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Markov, N. T. et al. Anatomy of hierarchy: feedforward and feedback pathways in macaque visual cortex. <i>J. Comp. Neurol.</i> <b>522</b>, 225–259 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/cne.23458" data-track-action="article reference" href="https://doi.org/10.1002%2Fcne.23458" aria-label="Article reference 15" data-doi="10.1002/cne.23458">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23983048" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Anatomy%20of%20hierarchy%3A%20feedforward%20and%20feedback%20pathways%20in%20macaque%20visual%20cortex&amp;journal=J.%20Comp.%20Neurol.&amp;doi=10.1002%2Fcne.23458&amp;volume=522&amp;pages=225-259&amp;publication_year=2014&amp;author=Markov%2CNT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Badre, D. &amp; D’Esposito, M. Is the rostro-caudal axis of the frontal lobe hierarchical? <i>Nat. Rev. Neurosci.</i> <b>10</b>, 659–669 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn2667" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn2667" aria-label="Article reference 16" data-doi="10.1038/nrn2667">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXpsleqsrw%3D" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19672274" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3258028" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Is%20the%20rostro-caudal%20axis%20of%20the%20frontal%20lobe%20hierarchical%3F&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn2667&amp;volume=10&amp;pages=659-669&amp;publication_year=2009&amp;author=Badre%2CD&amp;author=D%E2%80%99Esposito%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Glasser, M. F. &amp; Van Essen, D. C. Mapping human cortical areas in vivo based on myelin content as revealed by T1- and T2-weighted MRI. <i>J. Neurosci.</i> <b>31</b>, 11597–11616 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2180-11.2011" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2180-11.2011" aria-label="Article reference 17" data-doi="10.1523/JNEUROSCI.2180-11.2011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtVOru7jJ" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21832190" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3167149" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20human%20cortical%20areas%20in%20vivo%20based%20on%20myelin%20content%20as%20revealed%20by%20T1-%20and%20T2-weighted%20MRI&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2180-11.2011&amp;volume=31&amp;pages=11597-11616&amp;publication_year=2011&amp;author=Glasser%2CMF&amp;author=Essen%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Glasser, M. F., Goyal, M. S., Preuss, T. M., Raichle, M. E. &amp; Van Essen, D. C. Trends and properties of human cerebral cortex: correlations with cortical myelin content. <i>Neuroimage</i> <b>93</b>, 165–175 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.03.060" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.03.060" aria-label="Article reference 18" data-doi="10.1016/j.neuroimage.2013.03.060">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXms1Krs7k%3D" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23567887" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Trends%20and%20properties%20of%20human%20cerebral%20cortex%3A%20correlations%20with%20cortical%20myelin%20content&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.03.060&amp;volume=93&amp;pages=165-175&amp;publication_year=2014&amp;author=Glasser%2CMF&amp;author=Goyal%2CMS&amp;author=Preuss%2CTM&amp;author=Raichle%2CME&amp;author=Essen%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Chaudhuri, R., Knoblauch, K., Gariel, M.-A., Kennedy, H. &amp; Wang, X.-J. A large-scale circuit mechanism for hierarchical dynamical processing in the primate cortex. <i>Neuron</i> <b>88</b>, 419–431 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2015.09.008" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.09.008" aria-label="Article reference 19" data-doi="10.1016/j.neuron.2015.09.008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1SltbfJ" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26439530" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4630024" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20large-scale%20circuit%20mechanism%20for%20hierarchical%20dynamical%20processing%20in%20the%20primate%20cortex&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2015.09.008&amp;volume=88&amp;pages=419-431&amp;publication_year=2015&amp;author=Chaudhuri%2CR&amp;author=Knoblauch%2CK&amp;author=Gariel%2CMA&amp;author=Kennedy%2CH&amp;author=Wang%2CXJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Wagstyl, K., Ronan, L., Goodyer, I. M. &amp; Fletcher, P. C. Cortical thickness gradients in structural hierarchies. <i>Neuroimage</i> <b>111</b>, 241–250 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.02.036" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.02.036" aria-label="Article reference 20" data-doi="10.1016/j.neuroimage.2015.02.036">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25725468" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20thickness%20gradients%20in%20structural%20hierarchies&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.02.036&amp;volume=111&amp;pages=241-250&amp;publication_year=2015&amp;author=Wagstyl%2CK&amp;author=Ronan%2CL&amp;author=Goodyer%2CIM&amp;author=Fletcher%2CPC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. <i>Nature</i> <b>536</b>, 171–178 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature18933" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature18933" aria-label="Article reference 21" data-doi="10.1038/nature18933">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1GrurnK" aria-label="CAS reference 21">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27437579" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990127" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-modal%20parcellation%20of%20human%20cerebral%20cortex&amp;journal=Nature&amp;doi=10.1038%2Fnature18933&amp;volume=536&amp;pages=171-178&amp;publication_year=2016&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Hilgetag, C. C., Medalla, M., Beul, S. F. &amp; Barbas, H. The primate connectome in context: Principles of connections of the cortical visual system. <i>Neuroimage</i> <b>134</b>, 685–702 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2016.04.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2016.04.017" aria-label="Article reference 22" data-doi="10.1016/j.neuroimage.2016.04.017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27083526" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20primate%20connectome%20in%20context%3A%20Principles%20of%20connections%20of%20the%20cortical%20visual%20system&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2016.04.017&amp;volume=134&amp;pages=685-702&amp;publication_year=2016&amp;author=Hilgetag%2CCC&amp;author=Medalla%2CM&amp;author=Beul%2CSF&amp;author=Barbas%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Zeng, H. et al. Large-scale cellular-resolution gene profiling in human neocortex reveals species-specific molecular signatures. <i>Cell</i> <b>149</b>, 483–496 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2012.02.052" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2012.02.052" aria-label="Article reference 23" data-doi="10.1016/j.cell.2012.02.052">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xls1entLo%3D" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22500809" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3328777" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20cellular-resolution%20gene%20profiling%20in%20human%20neocortex%20reveals%20species-specific%20molecular%20signatures&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2012.02.052&amp;volume=149&amp;pages=483-496&amp;publication_year=2012&amp;author=Zeng%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Markram, H. et al. Interneurons of the neocortical inhibitory system. <i>Nat. Rev. Neurosci.</i> <b>5</b>, 793–807 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn1519" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn1519" aria-label="Article reference 24" data-doi="10.1038/nrn1519">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXns1Kiurc%3D" aria-label="CAS reference 24">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15378039" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Interneurons%20of%20the%20neocortical%20inhibitory%20system&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn1519&amp;volume=5&amp;pages=793-807&amp;publication_year=2004&amp;author=Markram%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Kepecs, A. &amp; Fishell, G. Interneuron cell types are fit to function. <i>Nature</i> <b>505</b>, 318–326 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature12983" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature12983" aria-label="Article reference 25" data-doi="10.1038/nature12983">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXotFaksQ%3D%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24429630" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4349583" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Interneuron%20cell%20types%20are%20fit%20to%20function&amp;journal=Nature&amp;doi=10.1038%2Fnature12983&amp;volume=505&amp;pages=318-326&amp;publication_year=2014&amp;author=Kepecs%2CA&amp;author=Fishell%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Lake, B. B. et al. Neuronal subtypes and diversity revealed by single-nucleus RNA sequencing of the human brain. <i>Science</i> <b>352</b>, 1586–1590 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aaf1204" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaf1204" aria-label="Article reference 26" data-doi="10.1126/science.aaf1204">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhtVaitLzF" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27339989" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5038589" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Neuronal%20subtypes%20and%20diversity%20revealed%20by%20single-nucleus%20RNA%20sequencing%20of%20the%20human%20brain&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf1204&amp;volume=352&amp;pages=1586-1590&amp;publication_year=2016&amp;author=Lake%2CBB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Wang, X.-J. Synaptic reverberation underlying mnemonic persistent activity. <i>Trends Neurosci.</i> <b>24</b>, 455–463 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0166-2236(00)01868-3" data-track-action="article reference" href="https://doi.org/10.1016%2FS0166-2236%2800%2901868-3" aria-label="Article reference 27" data-doi="10.1016/S0166-2236(00)01868-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXltlymsL0%3D" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11476885" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Synaptic%20reverberation%20underlying%20mnemonic%20persistent%20activity&amp;journal=Trends%20Neurosci.&amp;doi=10.1016%2FS0166-2236%2800%2901868-3&amp;volume=24&amp;pages=455-463&amp;publication_year=2001&amp;author=Wang%2CXJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Elston, G. N. Cortex, cognition and the cell: new insights into the pyramidal neuron and prefrontal function. <i>Cereb. Cortex</i> <b>13</b>, 1124–1138 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhg093" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhg093" aria-label="Article reference 28" data-doi="10.1093/cercor/bhg093">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14576205" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortex%2C%20cognition%20and%20the%20cell%3A%20new%20insights%20into%20the%20pyramidal%20neuron%20and%20prefrontal%20function&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhg093&amp;volume=13&amp;pages=1124-1138&amp;publication_year=2003&amp;author=Elston%2CGN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Wang, H., Stradtman, G. G.III, Wang, X.-J. &amp; Gao, W.-J. A specialized NMDA receptor function in layer 5 recurrent microcircuitry of the adult rat prefrontal cortex. <i>Proc. Natl Acad. Sci. USA</i> <b>105</b>, 16791–16796 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0804318105" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0804318105" aria-label="Article reference 29" data-doi="10.1073/pnas.0804318105">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlertrfK" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18922773" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2575498" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20specialized%20NMDA%20receptor%20function%20in%20layer%205%20recurrent%20microcircuitry%20of%20the%20adult%20rat%20prefrontal%20cortex.&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0804318105&amp;volume=105&amp;pages=16791-16796&amp;publication_year=2008&amp;author=Wang%2CH&amp;author=Stradtman%2CGG&amp;author=Wang%2CXJ&amp;author=Gao%2CWJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Genovese, G. et al. Increased burden of ultra-rare protein-altering variants among 4,877 individuals with schizophrenia. <i>Nat. Neurosci.</i> <b>19</b>, 1433–1441 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4402" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4402" aria-label="Article reference 30" data-doi="10.1038/nn.4402">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1SltLrF" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27694994" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5104192" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Increased%20burden%20of%20ultra-rare%20protein-altering%20variants%20among%204%2C877%20individuals%20with%20schizophrenia&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4402&amp;volume=19&amp;pages=1433-1441&amp;publication_year=2016&amp;author=Genovese%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Pirooznia, M. et al. SynaptomeDB: an ontology-based knowledgebase for synaptic genes. <i>Bioinformatics</i> <b>28</b>, 897–899 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bts040" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts040" aria-label="Article reference 31" data-doi="10.1093/bioinformatics/bts040">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xks1Sisrs%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22285564" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3307115" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=SynaptomeDB%3A%20an%20ontology-based%20knowledgebase%20for%20synaptic%20genes&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbts040&amp;volume=28&amp;pages=897-899&amp;publication_year=2012&amp;author=Pirooznia%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Chen, J., Bardes, E. E., Aronow, B. J. &amp; Jegga, A. G. ToppGene Suite for gene list enrichment analysis and candidate gene prioritization. <i>Nucleic Acids Res.</i> <b>37</b>, W305–11 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkp427" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkp427" aria-label="Article reference 32" data-doi="10.1093/nar/gkp427">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXosFSrs7g%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19465376" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2703978" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=ToppGene%20Suite%20for%20gene%20list%20enrichment%20analysis%20and%20candidate%20gene%20prioritization&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkp427&amp;volume=37&amp;pages=W305-11&amp;publication_year=2009&amp;author=Chen%2CJ&amp;author=Bardes%2CEE&amp;author=Aronow%2CBJ&amp;author=Jegga%2CAG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Bras, J. et al. Genetic analysis implicates APOE, SNCA and suggests lysosomal dysfunction in the etiology of dementia with Lewy bodies. <i>Hum. Mol. Genet.</i> <b>23</b>, 6139–6146 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/hmg/ddu334" data-track-action="article reference" href="https://doi.org/10.1093%2Fhmg%2Fddu334" aria-label="Article reference 33" data-doi="10.1093/hmg/ddu334">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFOrurbO" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24973356" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4222357" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20analysis%20implicates%20APOE%2C%20SNCA%20and%20suggests%20lysosomal%20dysfunction%20in%20the%20etiology%20of%20dementia%20with%20Lewy%20bodies&amp;journal=Hum.%20Mol.%20Genet.&amp;doi=10.1093%2Fhmg%2Fddu334&amp;volume=23&amp;pages=6139-6146&amp;publication_year=2014&amp;author=Bras%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Piñero, J. et al. DisGeNET: a comprehensive platform integrating information on human disease-associated genes and variants. <i>Nucleic Acids Res.</i> <b>45</b>, D833–D839 (2017). D1.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkw943" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw943" aria-label="Article reference 34" data-doi="10.1093/nar/gkw943">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslWhtL4%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27924018" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=DisGeNET%3A%20a%20comprehensive%20platform%20integrating%20information%20on%20human%20disease-associated%20genes%20and%20variants&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw943&amp;volume=45&amp;pages=D833-D839&amp;publication_year=2017&amp;author=Pi%C3%B1ero%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Murray, J. D., Jaramillo, J. &amp; Wang, X.-J. Working memory and decision-making in a frontoparietal circuit model. <i>J. Neurosci.</i> <b>37</b>, 12167–12186 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.0343-17.2017" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.0343-17.2017" aria-label="Article reference 35" data-doi="10.1523/JNEUROSCI.0343-17.2017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVWntrjO" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29114071" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5729190" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Working%20memory%20and%20decision-making%20in%20a%20frontoparietal%20circuit%20model&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.0343-17.2017&amp;volume=37&amp;pages=12167-12186&amp;publication_year=2017&amp;author=Murray%2CJD&amp;author=Jaramillo%2CJ&amp;author=Wang%2CXJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Yang, G. R., Murray, J. D. &amp; Wang, X.-J. A dendritic disinhibitory circuit mechanism for pathway-specific gating. <i>Nat. Commun.</i> <b>7</b>, 12815 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms12815" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms12815" aria-label="Article reference 36" data-doi="10.1038/ncomms12815">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGrtr3I" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27649374" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5034308" aria-label="PubMed Central reference 36">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20dendritic%20disinhibitory%20circuit%20mechanism%20for%20pathway-specific%20gating&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms12815&amp;volume=7&amp;publication_year=2016&amp;author=Yang%2CGR&amp;author=Murray%2CJD&amp;author=Wang%2CXJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Lorio, S. et al. Neurobiological origin of spurious brain morphological changes: A quantitative MRI study. <i>Hum. Brain Mapp.</i> <b>37</b>, 1801–1815 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/hbm.23137" data-track-action="article reference" href="https://doi.org/10.1002%2Fhbm.23137" aria-label="Article reference 37" data-doi="10.1002/hbm.23137">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26876452" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4855623" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Neurobiological%20origin%20of%20spurious%20brain%20morphological%20changes%3A%20A%20quantitative%20MRI%20study&amp;journal=Hum.%20Brain%20Mapp.&amp;doi=10.1002%2Fhbm.23137&amp;volume=37&amp;pages=1801-1815&amp;publication_year=2016&amp;author=Lorio%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Stüber, C. et al. Myelin and iron concentration in the human brain: a quantitative study of MRI contrast. <i>Neuroimage</i> <b>93</b>, 95–106 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.02.026" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.02.026" aria-label="Article reference 38" data-doi="10.1016/j.neuroimage.2014.02.026">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXns1Wrtrg%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24607447" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Myelin%20and%20iron%20concentration%20in%20the%20human%20brain%3A%20a%20quantitative%20study%20of%20MRI%20contrast&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.02.026&amp;volume=93&amp;pages=95-106&amp;publication_year=2014&amp;author=St%C3%BCber%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Sereno, M. I., Lutti, A., Weiskopf, N. &amp; Dick, F. Mapping the human cortical surface by combining quantitative T(1) with retinotopy. <i>Cereb. Cortex</i> <b>23</b>, 2261–2268 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhs213" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhs213" aria-label="Article reference 39" data-doi="10.1093/cercor/bhs213">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22826609" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20the%20human%20cortical%20surface%20by%20combining%20quantitative%20T%281%29%20with%20retinotopy&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhs213&amp;volume=23&amp;pages=2261-2268&amp;publication_year=2013&amp;author=Sereno%2CMI&amp;author=Lutti%2CA&amp;author=Weiskopf%2CN&amp;author=Dick%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Lutti, A., Dick, F., Sereno, M. I. &amp; Weiskopf, N. Using high-resolution quantitative mapping of R1 as an index of cortical myelination. <i>Neuroimage</i> <b>93</b>, 176–188 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.06.005" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.06.005" aria-label="Article reference 40" data-doi="10.1016/j.neuroimage.2013.06.005">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVOjt7%2FL" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23756203" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20high-resolution%20quantitative%20mapping%20of%20R1%20as%20an%20index%20of%20cortical%20myelination&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.06.005&amp;volume=93&amp;pages=176-188&amp;publication_year=2014&amp;author=Lutti%2CA&amp;author=Dick%2CF&amp;author=Sereno%2CMI&amp;author=Weiskopf%2CN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Carey, D. et al. Quantitative MRI provides markers of intra-, inter-regional, and age-related differences in young adult cortical microstructure. <i>Neuroimage</i> S1053-8119 (17)31012-1 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Gomez, J. et al. Microstructural proliferation in human cortex is coupled with the development of face processing. <i>Science</i> <b>355</b>, 68–71 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aag0311" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aag0311" aria-label="Article reference 42" data-doi="10.1126/science.aag0311">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1WjsA%3D%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28059764" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5373008" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Microstructural%20proliferation%20in%20human%20cortex%20is%20coupled%20with%20the%20development%20of%20face%20processing&amp;journal=Science&amp;doi=10.1126%2Fscience.aag0311&amp;volume=355&amp;pages=68-71&amp;publication_year=2017&amp;author=Gomez%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, 12574–12579 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1608282113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1608282113" aria-label="Article reference 43" data-doi="10.1073/pnas.1608282113">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ykurnM" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27791099" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5098630" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Situating%20the%20default-mode%20network%20along%20a%20principal%20gradient%20of%20macroscale%20cortical%20organization.&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1608282113&amp;volume=113&amp;pages=12574-12579&amp;publication_year=2016&amp;author=Margulies%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Huntenburg, J. M., Bazin, P. L. &amp; Margulies, D. S. Large-scale gradients in human cortical organization. <i>Trends Cogn. Sci.</i> <b>22</b>, 21–31 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.tics.2017.11.002" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.tics.2017.11.002" aria-label="Article reference 44" data-doi="10.1016/j.tics.2017.11.002">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29203085" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20gradients%20in%20human%20cortical%20organization&amp;journal=Trends%20Cogn.%20Sci.&amp;doi=10.1016%2Fj.tics.2017.11.002&amp;volume=22&amp;pages=21-31&amp;publication_year=2018&amp;author=Huntenburg%2CJM&amp;author=Bazin%2CPL&amp;author=Margulies%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Wandell, B. A., Dumoulin, S. O. &amp; Brewer, A. A. Visual field maps in human cortex. <i>Neuron</i> <b>56</b>, 366–383 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2007.10.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2007.10.012" aria-label="Article reference 45" data-doi="10.1016/j.neuron.2007.10.012">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXht12jsrvP" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17964252" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Visual%20field%20maps%20in%20human%20cortex&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2007.10.012&amp;volume=56&amp;pages=366-383&amp;publication_year=2007&amp;author=Wandell%2CBA&amp;author=Dumoulin%2CSO&amp;author=Brewer%2CAA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, 9105–9110 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1601745113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1601745113" aria-label="Article reference 46" data-doi="10.1073/pnas.1601745113">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1ehurbM" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27457931" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987797" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Adolescence%20is%20associated%20with%20genomically%20patterned%20consolidation%20of%20the%20hubs%20of%20the%20human%20brain%20connectome&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1601745113&amp;volume=113&amp;pages=9105-9110&amp;publication_year=2016&amp;author=Whitaker%2CKJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Romme, I. A. C., de Reus, M. A., Ophoff, R. A., Kahn, R. S. &amp; van den Heuvel, M. P. Connectome disconnectivity and cortical gene expression in patients with schizophrenia. <i>Biol. Psychiatry</i> <b>81</b>, 495–502 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2016.07.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2016.07.012" aria-label="Article reference 47" data-doi="10.1016/j.biopsych.2016.07.012">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1CgtrrK" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27720199" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Connectome%20disconnectivity%20and%20cortical%20gene%20expression%20in%20patients%20with%20schizophrenia&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2016.07.012&amp;volume=81&amp;pages=495-502&amp;publication_year=2017&amp;author=Romme%2CIAC&amp;author=Reus%2CMA&amp;author=Ophoff%2CRA&amp;author=Kahn%2CRS&amp;author=Heuvel%2CMP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Johnson, M. B. et al. Functional and evolutionary insights into human brain development through global transcriptome analysis. <i>Neuron</i> <b>62</b>, 494–509 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2009.03.027" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2009.03.027" aria-label="Article reference 48" data-doi="10.1016/j.neuron.2009.03.027">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXosVCkt70%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19477152" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2739738" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20and%20evolutionary%20insights%20into%20human%20brain%20development%20through%20global%20transcriptome%20analysis&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2009.03.027&amp;volume=62&amp;pages=494-509&amp;publication_year=2009&amp;author=Johnson%2CMB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Bakken, T. E. et al. A comprehensive transcriptional map of primate brain development. <i>Nature</i> <b>535</b>, 367–375 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature18637" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature18637" aria-label="Article reference 49" data-doi="10.1038/nature18637">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1yqsLvF" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27409810" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5325728" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20comprehensive%20transcriptional%20map%20of%20primate%20brain%20development&amp;journal=Nature&amp;doi=10.1038%2Fnature18637&amp;volume=535&amp;pages=367-375&amp;publication_year=2016&amp;author=Bakken%2CTE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Yeatman, J. D., Wandell, B. A. &amp; Mezer, A. A. Lifespan maturation and degeneration of human brain white matter. <i>Nat. Commun.</i> <b>5</b>, 4932 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms5932" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms5932" aria-label="Article reference 50" data-doi="10.1038/ncomms5932">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvF2mu7vJ" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25230200" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Lifespan%20maturation%20and%20degeneration%20of%20human%20brain%20white%20matter&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms5932&amp;volume=5&amp;publication_year=2014&amp;author=Yeatman%2CJD&amp;author=Wandell%2CBA&amp;author=Mezer%2CAA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Van Essen, D. C. et al. The WU-Minn human connectome project: an overview. <i>Neuroimage</i> <b>80</b>, 62–79 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.041" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.041" aria-label="Article reference 51" data-doi="10.1016/j.neuroimage.2013.05.041">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23684880" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20WU-Minn%20human%20connectome%20project%3A%20an%20overview&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.041&amp;volume=80&amp;pages=62-79&amp;publication_year=2013&amp;author=Essen%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Robinson, E. C. et al. MSM: a new flexible framework for Multimodal Surface Matching. <i>Neuroimage</i> <b>100</b>, 414–426 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.05.069" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.05.069" aria-label="Article reference 52" data-doi="10.1016/j.neuroimage.2014.05.069">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24939340" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=MSM%3A%20a%20new%20flexible%20framework%20for%20Multimodal%20Surface%20Matching&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.05.069&amp;volume=100&amp;pages=414-426&amp;publication_year=2014&amp;author=Robinson%2CEC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 105–124 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.04.127" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.04.127" aria-label="Article reference 53" data-doi="10.1016/j.neuroimage.2013.04.127">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23668970" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20minimal%20preprocessing%20pipelines%20for%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.04.127&amp;volume=80&amp;pages=105-124&amp;publication_year=2013&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Ito, T. et al. Cognitive task information is transferred between brain regions via resting-state network topology. <i>Nat. Commun.</i> <b>8</b>, 1027 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-017-01000-w" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-017-01000-w" aria-label="Article reference 54" data-doi="10.1038/s41467-017-01000-w">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXptVCls7Y%3D" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29044112" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5715061" aria-label="PubMed Central reference 54">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Cognitive%20task%20information%20is%20transferred%20between%20brain%20regions%20via%20resting-state%20network%20topology&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-01000-w&amp;volume=8&amp;publication_year=2017&amp;author=Ito%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Donahue, C. J. et al. Using diffusion tractography to predict cortical connection strength and distance: a quantitative comparison with tracers in the monkey. <i>J. Neurosci.</i> <b>36</b>, 6758–6770 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.0493-16.2016" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.0493-16.2016" aria-label="Article reference 55" data-doi="10.1523/JNEUROSCI.0493-16.2016">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGlsr%2FE" aria-label="CAS reference 55">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27335406" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4916250" aria-label="PubMed Central reference 55">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20diffusion%20tractography%20to%20predict%20cortical%20connection%20strength%20and%20distance%3A%20a%20quantitative%20comparison%20with%20tracers%20in%20the%20monkey&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.0493-16.2016&amp;volume=36&amp;pages=6758-6770&amp;publication_year=2016&amp;author=Donahue%2CCJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Markov, N. T. et al. A weighted and directed interareal connectivity matrix for macaque cerebral cortex. <i>Cereb. Cortex</i> <b>24</b>, 17–36 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhs270" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhs270" aria-label="Article reference 56" data-doi="10.1093/cercor/bhs270">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC3s%2FgtVykuw%3D%3D" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23010748" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20weighted%20and%20directed%20interareal%20connectivity%20matrix%20for%20macaque%20cerebral%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhs270&amp;volume=24&amp;pages=17-36&amp;publication_year=2014&amp;author=Markov%2CNT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Condé, F., Lund, J. S., Jacobowitz, D. M., Baimbridge, K. G. &amp; Lewis, D. A. Local circuit neurons immunoreactive for calretinin, calbindin D-28k or parvalbumin in monkey prefrontal cortex: distribution and morphology. <i>J. Comp. Neurol.</i> <b>341</b>, 95–116 (1994).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/cne.903410109" data-track-action="article reference" href="https://doi.org/10.1002%2Fcne.903410109" aria-label="Article reference 57" data-doi="10.1002/cne.903410109">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8006226" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Local%20circuit%20neurons%20immunoreactive%20for%20calretinin%2C%20calbindin%20D-28k%20or%20parvalbumin%20in%20monkey%20prefrontal%20cortex%3A%20distribution%20and%20morphology&amp;journal=J.%20Comp.%20Neurol.&amp;doi=10.1002%2Fcne.903410109&amp;volume=341&amp;pages=95-116&amp;publication_year=1994&amp;author=Cond%C3%A9%2CF&amp;author=Lund%2CJS&amp;author=Jacobowitz%2CDM&amp;author=Baimbridge%2CKG&amp;author=Lewis%2CDA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Gabbott, P. L. &amp; Bacon, S. J. Local circuit neurons in the medial prefrontal cortex (areas 24a,b,c, 25 and 32) in the monkey: II. Quantitative areal and laminar distributions. <i>J. Comp. Neurol.</i> <b>364</b>, 609–636 (1996).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/(SICI)1096-9861(19960122)364:4<609::AID-CNE2&gt;3.0.CO;2-7" data-track-action="article reference" href="https://doi.org/10.1002%2F%28SICI%291096-9861%2819960122%29364%3A4%3C609%3A%3AAID-CNE2%3E3.0.CO%3B2-7" aria-label="Article reference 58" data-doi="10.1002/(SICI)1096-9861(19960122)364:4<609::AID-CNE2&gt;3.0.CO;2-7">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK28vhvVKltw%3D%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8821450" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Local%20circuit%20neurons%20in%20the%20medial%20prefrontal%20cortex%20%28areas%2024a%2Cb%2Cc%2C%2025%20and%2032%29%20in%20the%20monkey%3A%20II.%20Quantitative%20areal%20and%20laminar%20distributions&amp;journal=J.%20Comp.%20Neurol.&amp;doi=10.1002%2F%28SICI%291096-9861%2819960122%29364%3A4%3C609%3A%3AAID-CNE2%3E3.0.CO%3B2-7&amp;volume=364&amp;pages=609-636&amp;publication_year=1996&amp;author=Gabbott%2CPL&amp;author=Bacon%2CSJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Kondo, H., Tanaka, K., Hashikawa, T. &amp; Jones, E. G. Neurochemical gradients along monkey sensory cortical pathways: calbindin-immunoreactive pyramidal neurons in layers II and III. <i>Eur. J. Neurosci.</i> <b>11</b>, 4197–4203 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1046/j.1460-9568.1999.00844.x" data-track-action="article reference" href="https://doi.org/10.1046%2Fj.1460-9568.1999.00844.x" aria-label="Article reference 59" data-doi="10.1046/j.1460-9568.1999.00844.x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3c%2Fmt1CrtA%3D%3D" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10594645" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Neurochemical%20gradients%20along%20monkey%20sensory%20cortical%20pathways%3A%20calbindin-immunoreactive%20pyramidal%20neurons%20in%20layers%20II%20and%20III&amp;journal=Eur.%20J.%20Neurosci.&amp;doi=10.1046%2Fj.1460-9568.1999.00844.x&amp;volume=11&amp;pages=4197-4203&amp;publication_year=1999&amp;author=Kondo%2CH&amp;author=Tanaka%2CK&amp;author=Hashikawa%2CT&amp;author=Jones%2CEG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Dombrowski, S. M., Hilgetag, C. C. &amp; Barbas, H. Quantitative architecture distinguishes prefrontal cortical systems in the rhesus monkey. <i>Cereb. Cortex</i> <b>11</b>, 975–988 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/11.10.975" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2F11.10.975" aria-label="Article reference 60" data-doi="10.1093/cercor/11.10.975">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3Mrks12hug%3D%3D" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11549620" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantitative%20architecture%20distinguishes%20prefrontal%20cortical%20systems%20in%20the%20rhesus%20monkey&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2F11.10.975&amp;volume=11&amp;pages=975-988&amp;publication_year=2001&amp;author=Dombrowski%2CSM&amp;author=Hilgetag%2CCC&amp;author=Barbas%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Elston, G. N. &amp; Rosa, M. G. The occipitoparietal pathway of the macaque monkey: comparison of pyramidal cell morphology in layer III of functionally related cortical visual areas. <i>Cereb. Cortex</i> <b>7</b>, 432–452 (1997).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/7.5.432" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2F7.5.432" aria-label="Article reference 61" data-doi="10.1093/cercor/7.5.432">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK2svgsVykuw%3D%3D" aria-label="CAS reference 61">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9261573" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20occipitoparietal%20pathway%20of%20the%20macaque%20monkey%3A%20comparison%20of%20pyramidal%20cell%20morphology%20in%20layer%20III%20of%20functionally%20related%20cortical%20visual%20areas&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2F7.5.432&amp;volume=7&amp;pages=432-452&amp;publication_year=1997&amp;author=Elston%2CGN&amp;author=Rosa%2CMG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Elston, G. N. &amp; Rosa, M. G. Morphological variation of layer III pyramidal neurones in the occipitotemporal pathway of the macaque monkey visual cortex. <i>Cereb. Cortex</i> <b>8</b>, 278–294 (1998).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/8.3.278" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2F8.3.278" aria-label="Article reference 62" data-doi="10.1093/cercor/8.3.278">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1c3nvVCrtw%3D%3D" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9617923" aria-label="PubMed reference 62">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Morphological%20variation%20of%20layer%20III%20pyramidal%20neurones%20in%20the%20occipitotemporal%20pathway%20of%20the%20macaque%20monkey%20visual%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2F8.3.278&amp;volume=8&amp;pages=278-294&amp;publication_year=1998&amp;author=Elston%2CGN&amp;author=Rosa%2CMG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Elston, G. N., Tweedale, R. &amp; Rosa, M. G. Cortical integration in the visual system of the macaque monkey: large-scale morphological differences in the pyramidal neurons in the occipital, parietal and temporal lobes. <i>Proc. Biol. Sci.</i> <b>266</b>, 1367–1374 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rspb.1999.0789" data-track-action="article reference" href="https://doi.org/10.1098%2Frspb.1999.0789" aria-label="Article reference 63" data-doi="10.1098/rspb.1999.0789">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1MznsVeitQ%3D%3D" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10445291" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1690073" aria-label="PubMed Central reference 63">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20integration%20in%20the%20visual%20system%20of%20the%20macaque%20monkey%3A%20large-scale%20morphological%20differences%20in%20the%20pyramidal%20neurons%20in%20the%20occipital%2C%20parietal%20and%20temporal%20lobes&amp;journal=Proc.%20Biol.%20Sci.&amp;doi=10.1098%2Frspb.1999.0789&amp;volume=266&amp;pages=1367-1374&amp;publication_year=1999&amp;author=Elston%2CGN&amp;author=Tweedale%2CR&amp;author=Rosa%2CMG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Elston, G. N. &amp; Rockland, K. S. The pyramidal cell of the sensorimotor cortex of the macaque monkey: phenotypic variation. <i>Cereb. Cortex</i> <b>12</b>, 1071–1078 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/12.10.1071" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2F12.10.1071" aria-label="Article reference 64" data-doi="10.1093/cercor/12.10.1071">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12217971" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20pyramidal%20cell%20of%20the%20sensorimotor%20cortex%20of%20the%20macaque%20monkey%3A%20phenotypic%20variation&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2F12.10.1071&amp;volume=12&amp;pages=1071-1078&amp;publication_year=2002&amp;author=Elston%2CGN&amp;author=Rockland%2CKS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Elston, G. N., Benavides-Piccione, R. &amp; Defelipe, J. A study of pyramidal cell structure in the cingulate cortex of the macaque monkey with comparative notes on inferotemporal and primary visual cortex. <i>Cereb. Cortex</i> <b>15</b>, 64–73 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhh109" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhh109" aria-label="Article reference 65" data-doi="10.1093/cercor/bhh109">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15238445" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20study%20of%20pyramidal%20cell%20structure%20in%20the%20cingulate%20cortex%20of%20the%20macaque%20monkey%20with%20comparative%20notes%20on%20inferotemporal%20and%20primary%20visual%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhh109&amp;volume=15&amp;pages=64-73&amp;publication_year=2005&amp;author=Elston%2CGN&amp;author=Benavides-Piccione%2CR&amp;author=Defelipe%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Elston, G. N., Oga, T., Okamoto, T. &amp; Fujita, I. Spinogenesis and pruning in the anterior ventral inferotemporal cortex of the macaque monkey: An intracellular injection study of layer III pyramidal cells. <i>Front. Neuroanat.</i> <b>5</b>, 42 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fnana.2011.00042" data-track-action="article reference" href="https://doi.org/10.3389%2Ffnana.2011.00042" aria-label="Article reference 66" data-doi="10.3389/fnana.2011.00042">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21811440" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3143722" aria-label="PubMed Central reference 66">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Spinogenesis%20and%20pruning%20in%20the%20anterior%20ventral%20inferotemporal%20cortex%20of%20the%20macaque%20monkey%3A%20An%20intracellular%20injection%20study%20of%20layer%20III%20pyramidal%20cells&amp;journal=Front.%20Neuroanat.&amp;doi=10.3389%2Ffnana.2011.00042&amp;volume=5&amp;publication_year=2011&amp;author=Elston%2CGN&amp;author=Oga%2CT&amp;author=Okamoto%2CT&amp;author=Fujita%2CI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Fagerberg, L. et al. Analysis of the human tissue-specific expression by genome-wide integration of transcriptomics and antibody-based proteomics. <i>Mol. Cell. Proteom.</i> <b>13</b>, 397–406 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1074/mcp.M113.035600" data-track-action="article reference" href="https://doi.org/10.1074%2Fmcp.M113.035600" aria-label="Article reference 67" data-doi="10.1074/mcp.M113.035600">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsl2nsrg%3D" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20the%20human%20tissue-specific%20expression%20by%20genome-wide%20integration%20of%20transcriptomics%20and%20antibody-based%20proteomics&amp;journal=Mol.%20Cell.%20Proteom.&amp;doi=10.1074%2Fmcp.M113.035600&amp;volume=13&amp;pages=397-406&amp;publication_year=2014&amp;author=Fagerberg%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Cahoy, J. D. et al. A transcriptome database for astrocytes, neurons, and oligodendrocytes: a new resource for understanding brain development and function. <i>J. Neurosci.</i> <b>28</b>, 264–278 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.4178-07.2008" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.4178-07.2008" aria-label="Article reference 68" data-doi="10.1523/JNEUROSCI.4178-07.2008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtV2ltw%3D%3D" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18171944" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6671143" aria-label="PubMed Central reference 68">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20transcriptome%20database%20for%20astrocytes%2C%20neurons%2C%20and%20oligodendrocytes%3A%20a%20new%20resource%20for%20understanding%20brain%20development%20and%20function&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.4178-07.2008&amp;volume=28&amp;pages=264-278&amp;publication_year=2008&amp;author=Cahoy%2CJD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Anselin, L. Spatial econometrics. in <i>A Companion to Theoretical Econometrics</i> (ed. Baltagi, B.H.) 310–330 (Blackwell, Oxford, UK, 2001).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Fischer, M. M. &amp; Getis, A. <i>Handbook of Applied Spatial Analysis: Software Tools, Methods and Applications</i>. (Springer, Heidelberg, Germany, 2010).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Sen, P. K. Estimates of the regression coefficient based on Kendall’s tau. <i>J. Am. Stat. Assoc.</i> <b>63</b>, 1379–1389 (1968).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/01621459.1968.10480934" data-track-action="article reference" href="https://doi.org/10.1080%2F01621459.1968.10480934" aria-label="Article reference 71" data-doi="10.1080/01621459.1968.10480934">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimates%20of%20the%20regression%20coefficient%20based%20on%20Kendall%E2%80%99s%20tau&amp;journal=J.%20Am.%20Stat.%20Assoc.&amp;doi=10.1080%2F01621459.1968.10480934&amp;volume=63&amp;pages=1379-1389&amp;publication_year=1968&amp;author=Sen%2CPK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Tukey, J. W. Bias and confidence in not-quite large samples. <i>Ann. Math. Stat.</i> <b>29</b>, 614 (1958).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1214/aoms/1177706647" data-track-action="article reference" href="https://doi.org/10.1214%2Faoms%2F1177706647" aria-label="Article reference 72" data-doi="10.1214/aoms/1177706647">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Bias%20and%20confidence%20in%20not-quite%20large%20samples&amp;journal=Ann.%20Math.%20Stat.&amp;doi=10.1214%2Faoms%2F1177706647&amp;volume=29&amp;publication_year=1958&amp;author=Tukey%2CJW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Davison, A.C. &amp; Hinkley, D.V. <i>Bootstrap Methods and Their Application</i> (Cambridge Univ. Press, Cambridge, UK, 1997).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Timmerman, M. E., Kiers, H. A. &amp; Smilde, A. K. Estimating confidence intervals for principal component loadings: a comparison between the bootstrap and asymptotic results. <i>Br. J. Math. Stat. Psychol.</i> <b>60</b>, 295–314 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1348/000711006X109636" data-track-action="article reference" href="https://doi.org/10.1348%2F000711006X109636" aria-label="Article reference 74" data-doi="10.1348/000711006X109636">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17971271" aria-label="PubMed reference 74">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimating%20confidence%20intervals%20for%20principal%20component%20loadings%3A%20a%20comparison%20between%20the%20bootstrap%20and%20asymptotic%20results&amp;journal=Br.%20J.%20Math.%20Stat.%20Psychol.&amp;doi=10.1348%2F000711006X109636&amp;volume=60&amp;pages=295-314&amp;publication_year=2007&amp;author=Timmerman%2CME&amp;author=Kiers%2CHA&amp;author=Smilde%2CAK">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-018-0195-0?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank B.D. Fulcher, X.-J. Wang, R. Chaudhuri, and D.C. Glahn for useful discussions. This research was supported by NIH grants R01MH112746 (J.D.M.), R01MH108590 (A.A.), and TL1TR000141 (J.D.M.), and by BlackThorn Therapeutics (J.D.M., A.A.).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Department of Physics, Yale University, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">Joshua B. Burt, William J. Eckner &amp; John D. Murray</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Psychiatry, Yale University School of Medicine, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">Murat Demirtaş, Alan Anticevic &amp; John D. Murray</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Neuroscience Program, Tulane University, New Orleans, LA, USA</p><p class="c-article-author-affiliation__authors-list">Natasha M. Navejar</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Interdepartmental Neuroscience Program, Yale University, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">Jie Lisa Ji</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">BlackThorn Therapeutics, San Francisco, CA, USA</p><p class="c-article-author-affiliation__authors-list">William J. Martin</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Department of Engineering, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Alberto Bernacchia</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Joshua_B_-Burt-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Joshua B. Burt</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Joshua%20B.%20Burt" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joshua%20B.%20Burt" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joshua%20B.%20Burt%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Murat-Demirta_-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Murat Demirtaş</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Murat%20Demirta%C5%9F" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Murat%20Demirta%C5%9F" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Murat%20Demirta%C5%9F%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-William_J_-Eckner-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">William J. Eckner</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=William%20J.%20Eckner" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=William%20J.%20Eckner" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22William%20J.%20Eckner%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Natasha_M_-Navejar-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Natasha M. Navejar</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Natasha%20M.%20Navejar" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Natasha%20M.%20Navejar" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Natasha%20M.%20Navejar%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jie_Lisa-Ji-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Jie Lisa Ji</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jie%20Lisa%20Ji" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jie%20Lisa%20Ji" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jie%20Lisa%20Ji%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-William_J_-Martin-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">William J. Martin</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=William%20J.%20Martin" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=William%20J.%20Martin" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22William%20J.%20Martin%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Alberto-Bernacchia-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Alberto Bernacchia</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Alberto%20Bernacchia" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alberto%20Bernacchia" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alberto%20Bernacchia%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Alan-Anticevic-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Alan Anticevic</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Alan%20Anticevic" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alan%20Anticevic" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alan%20Anticevic%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-John_D_-Murray-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">John D. Murray</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=John%20D.%20Murray" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=John%20D.%20Murray" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20D.%20Murray%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>J.B.B., W.J.M., A.B., A.A., and J.D.M. designed the research. J.B.B., M.D., W.J.E., N.M.N., and J.L.J. analyzed the data. J.D.M. supervised the project. J.B.B. and J.D.M. wrote the manuscript and prepared the figures. All authors contributed to editing the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:john.murray@yale.edu">John D. Murray</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
                <p>W.J.M. is an employee for BlackThorn Therapeutics. A.A. and J.D.M. are consultants for BlackThorn Therapeutics. W.J.M., A.A., and J.D.M. are co-inventors on a provisional patent application #62567087 related to using gene expression topography for predictive therapeutic applications.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Integrated supplementary information"><div class="c-article-section" id="Sec31-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec31">Integrated supplementary information</h2><div class="c-article-section__content" id="Sec31-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 1 group-averaged cortical t1w" href="/articles/s41593-018-0195-0/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig9_ESM.jpg">Supplementary Figure 1 Group-averaged cortical T1w/T2w maps exhibit interspecies homology and interhemispheric symmetry.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> Unparcellated group-averaged (N=339) T1w/T2w map in human cortex visualized bilaterally on an inflated cortical surface. <b>(b)</b> Unparcellated group-averaged (N=19) T1w/T2w map in monkey cortex visualized bilaterally on an inflated cortical surface. <b>(c)</b> Primary sensory (visual, V1; auditory, A1; somatosensory, S1) and primary motor (M1) homologues in human and monkey cortex, corresponding to the four bordered areas in panels <b>a</b> and <b>b</b>, exhibit high T1w/T2w map values relative to the average value computed across all higher-order sensory and association (Non-Primary) areas. Grey bars mark the standard error on the mean. The low T1w/T2w map value found in monkey area A1 (relative to human area A1) is likely driven by the size and lack of spatial specificity of the corresponding M132 parcel which extends deep down into temporal cortex. Note that human Brodmann area 3 is further subdivided into areas 3a and 3b in the HCP parcellation, the latter which was used for this analysis. <b>(d)</b> Functional networks derived from resting-state functional connectivity from the Human Connectome Project (HCP). Cortical areas are parcellated using the HCP multi-modal parcellation (MMP1.0). We assigned each region to a functional network using a community detection method applied to resting-state fMRI data from the HCP, and designated functional labels to networks, including three sensory and five association, that align with previously reported functional networks (with abbreviations labeled in Fig. 1b): Auditory (AUD), Visual (VIS), Somatomotor (SOM), Dorsal Attention (DAN), Frontoparietal (FPN), Ventral Attention (VAN), Default (DMN), and Cingulo-Opercular (CON).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 2 anatomical cortical hierarc" href="/articles/s41593-018-0195-0/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig10_ESM.jpg">Supplementary Figure 2 Anatomical cortical hierarchy derived from <i>N</i> = 1,243 laminar-specific interareal projections in monkey cortex.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> Fraction of external labeled neurons (FLNe). Target area <i>i</i> is injected with a retrograde tracer that labels neurons in many source areas; the FLNe in source area <i>j</i> is then defined as the fraction of all external labeled neurons terminating in area <i>i</i> that originated in source area <i>j</i>. Each row of the FLN matrix is therefore normalized to 1. Measurements which yielded no labeled neurons are marked in grey. <b>(b)</b> Fraction of supragranular layer neurons (SLN), defined as the fraction of neurons in an interareal projection (to target area <i>i</i> from source area <i>j</i>) originating in supragranular layers. An SLN of 1 indicates that all labeled projection neurons were of supragranular origin, reflecting a pure feedforward connection; an SLN of 0 indicates that all projection neurons originated in deep infragranular layers, reflecting a pure feedback connection. Measurements which yielded no labeled neurons are marked in grey. <b>(c)</b> Model-estimated hierarchy levels for 89 cortical regions. The blue line indicates hierarchy levels estimated by the model after shifting and re-scaling them to lie on the unit interval. The red indicates hierarchy values passed through a logistic function to remove the nonlinearity introduced by the logit link function in the GLM fitting procedure. The monotonicity of this transformation preserves the order of the cortical regions and therefore does not affect the Spearman rank correlations reported in the main text. <b>(d)</b> Group-averaged T1w/T2w map values for 89 cortical areas in the monkey.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 3 the strength of the corresp" href="/articles/s41593-018-0195-0/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig11_ESM.jpg">Supplementary Figure 3 The strength of the correspondence between laminar projection profiles and T1w/T2w value is strongest in high-T1w/T2w areas.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> 1243 retrograde tracer-labeled projections are separated by target area (i.e., by N=29 unique injection sites; see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0195-0#MOESM3">1</a>). The laminar profile of a projection is quantified by the fraction of labeled supragranular layer neurons (SLN) in the source area. We found the Spearman rank correlation between SLN and the areal difference in T1w/T2w map values (ΔT1w/T2w; target minus source) is stronger for areas with higher T1w/T2w values (<i>r</i><sub><i>s</i></sub> = −0.63; <i>P</i> = 2.9*10<sup>−4</sup>; Spearman rank correlation). This indicates that the relationship between laminar specificity and structural dissimilarity holds more strongly in sensory areas at lower levels of the anatomical hierarchy, which have high T1w/T2w values. <b>(b)</b> Hierarchically organized structure in the strength of the relationship between laminar specificity (SLN) and structural dissimilarity (ΔT1w/T2w) for the 29 unique target areas, visualized on the inflated monkey cortical surface.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 4 expression maps and tmcs fo" href="/articles/s41593-018-0195-0/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig12_ESM.jpg">Supplementary Figure 4 Expression maps and TMCs for genes that encode markers of distinct inhibitory interneuron cell types, and for weighted profiles characteristic of distinct neuronal cell types derived from single-cell RNA-seq of human cortical neurons.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> Markers for inhibitory interneuron cell types. <b>(b)</b> Weighted gene sets for excitatory neuronal cell types, derived from single-cell RNA sequencing. <b>(c)</b> Weighted gene sets for inhibitory neuronal cell types, derived from single-cell RNA sequencing. Expression map values are standardized (i.e., z-scored) and shown in units of standard deviations from the mean. Expression maps use the same colormap and scale as Fig. 1a. Statistical significance is calculated using a spatial autoregressive model to account for spatial autocorrelation, Bonferroni-corrected by the number of genes in each set (*, <i>P</i> &lt; 0.05; **, <i>P</i> &lt; 10<sup>−2</sup>; ***, <i>P</i> &lt; 10<sup>−3</sup>). Grey lines mark the jackknife estimate of standard error.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 5 expression maps and tmcs fo" href="/articles/s41593-018-0195-0/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig13_ESM.jpg">Supplementary Figure 5 Expression maps and TMCs for genes encoding synaptic receptor subunits and neuromodulator receptor subunits.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> NMDA receptor subunits. <b>(b)</b> GABA<sub>A</sub> receptor subunits. <b>(c)</b> Muscarinic acetylcholine receptors (CHRM). <b>(d)</b> Nicotinic acetylcholine receptors (CHRN). <b>(e)</b> Norepinephrine receptors (ADR). <b>(f)</b> Dopamine receptors (DRD). <b>(g)</b> Serotonin receptors (HTR). Expression map values are standardized (i.e., z-scored) and shown in units of standard deviations from the mean. All maps use the same colormap and scale as Fig. 1a. Statistical significance is calculated using a spatial autoregressive model to account for spatial autocorrelation, Bonferroni-corrected by the number of genes in each set (*, <i>P</i> &lt; 0.05; **, <i>P</i> &lt; 10<sup>−2</sup>; ***, <i>P</i> &lt; 10<sup>−3</sup>). Grey lines mark the jackknife estimate of standard error.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 6 the dominant axis of gene e" href="/articles/s41593-018-0195-0/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig14_ESM.jpg">Supplementary Figure 6 The dominant axis of gene expression variation is better captured by the group-averaged T1w/T2w map than by two alternative candidate proxies: cortical thickness and distance from primary visual cortex (V1).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a–e)</b> For five categorical gene sets, the Spearman rank correlation between the first principal component (PC1) and the T1w/T2w map, the map of cortical thickness, and the map of geodesic distance from primary visual cortex. For each gene set, PC1 is more strongly correlated with the T1w/T2w map than with the two other candidate maps. <b>(f–j)</b> For five gene sets, the amount of gene expression variance captured, relative to PC1, for the three candidate maps. For each gene set, the T1w/T2w map captures more gene expression variance than do the other two maps. <b>(k–o)</b> Percentage of gene expression variance captured by the top 10 of 179 total PCs. For all five gene sets, PC1 captures between 21% and 27% of the variance, roughly two to three times the amount captured by PC2. <b>(p–t)</b> Distribution of T1w/T2w map correlations (TMCs) across genes in each gene set. Dashed lines mark the mean of the distribution. For all five gene sets, the distributions are broad, containing large fractions of strong positive and negative TMCs, and centered near zero, with a range of means (-0.05, +0.05). Statistical significance is calculated through permutation testing with surrogate maps that preserve the spatial autocorrelation structure of each map (*, <i>P</i> &lt; 0.05; **, <i>P</i> &lt; 10<sup>−2</sup>; ***, <i>P</i> &lt; 10<sup>−3</sup>). Grey lines mark the bootstrap estimated 95% confidence interval.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 7 spatial autocorrelation str" href="/articles/s41593-018-0195-0/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig15_ESM.jpg">Supplementary Figure 7 Spatial autocorrelation structure in gene expression and group-averaged T1w/T2w maps.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> Spatial autocorrelation in the parcellated cortical gene expression data is well-approximated by a decaying exponential function of distance. Gene co-expression is defined as the pairwise Spearman rank correlation between cortical parcels’ gene expression values, here for the 2413 brain-specific genes. Proximal cortical parcels exhibit more similar gene expression values compared to distal parcels. Pairs of parcel with geodesic separation distance less than 100 mm (8247 of 16,110 total pairs) were used to fit the characteristic scale of spatial autocorrelation, illustrated in red (i.e., <i>exp</i>(−<i>d</i>/<i>d</i><sub>0</sub>)), where <i>d</i> is geodesic distance and <i>d</i><sub>0</sub> = 25 mm. Each data point corresponds to the co-expression of a pair of cortical parcels. Top: Mean co-expression value as a function of geodesic distance bin. <b>(b)</b> Gene co-expression values after correcting for spatial autocorrelation structure by subtraction of the fitted exponential in <b>a</b>. After correction, the mean co-expression value is near zero across all geodesic distance bins. <b>(c)</b> Example randomized surrogate maps with spatial autocorrelation structure matched to the empirical T1w/T2w map (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-018-0195-0#Sec9">Methods</a>). Autocorrelation structure-preserving surrogate maps are used for nonparametric calculation of statistical significance values for PCA results in Figs. 5 and 6, and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig14">6</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0195-0#Fig18">10</a>. <b>(d)</b> Distribution of pairwise Spearman rank correlations between 39,800 pairs of surrogate T1w/T2w maps.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig16"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 8 pca reveals that the domina" href="/articles/s41593-018-0195-0/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig16_ESM.jpg">Supplementary Figure 8 PCA reveals that the dominant axis of gene expression variation (i.e., the first principal component, PC1) is conserved across five categorical gene sets.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>PC1, which aligns with the T1w/T2w map, combines sensory areas (e.g., primary visual, somatosensory, and auditory cortical areas), and separates sensory areas from association areas. In contrast, the secondary mode of gene expression tends to fractionate sensorimotor cortical areas by modality, separating early visual cortex from somatomotor cortex. Rows correspond to the first three PCs, respectively, across gene sets.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig17"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 9 evidence for spatial struct" href="/articles/s41593-018-0195-0/figures/17" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig17_ESM.jpg">Supplementary Figure 9 Evidence for spatial structure in residual gene expression variance not captured by the group-averaged T1w/T2w map.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> Map of the brain-specific PC1 residual, defined as the magnitude of the vertical distance between an area’s brain-specific PC1 weight and the best-fit Theil-Sen estimator of linear slope for the PC1 vs. T1w/T2w relationship (i.e., absolute vertical distance between scatter points and best-fit line in Fig. 5c). <b>(b)</b> Map of the local T1w/T2w gradient, defined as the mean absolute difference between an area’s T1w/T2w map value and the values of all neighboring areas, i.e. <span class="mathjax-tex">\(x_i = \frac{1}{N}\mathop {\sum }\limits_{n \in \{ N_i\} } \left| {T_i - T_n} \right|\)</span>, where <i>T</i><sub><i>i</i></sub> is the T1w/T2w value for area <i>i</i>, and {<i>N</i><sub><i>i</i></sub>} is the set of all areas bordering area <i>i</i>. <b>(c)</b> The brain-specific PC1 residual is highly correlated with the magnitude of the local T1w/T2w gradient (<i>r</i><sub><i>p</i></sub> = 0.70; <i>P</i> &lt; 10<sup>−5</sup>; Pearson correlation). This relationship shows that areas with the strongest T1w/T2w gradient tend to be those that drive the discrepancy between the brain-specific gene PC1 and T1w/T2w map topographies. This is consistent with the reported correspondence shown in Fig. 5c being limited by the relatively poor spatial resolution of gene expression sampling in the AHBA.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig18"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 10 validation of key results " href="/articles/s41593-018-0195-0/figures/18" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_Fig18_ESM.jpg">Supplementary Figure 10 Validation of key results in human cortex using the group-averaged (<i>N</i> = 69) Conte69 T1w/T2w map.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> The parcellated T1w/T2w map. <b>(b)</b> Human T1w/T2w map values are significantly lower in functionally defined association networks than in sensory networks (<i>P</i> &lt; 10<sup>−5</sup>; two-sided Wilcoxon signed-rank test on N=6608 paired differences). Box plots mark the median and inner quartile ranges for areas in each network, and whiskers indicate the 95% confidence interval. <b>(c)</b> The average expression map of 5 genes preferentially expressed in human granular layer 4 (L4) is positively correlated with the T1w/T2w map (<i>r</i><sub><i>s</i></sub> = 0.73; <i>P</i> &lt; 10<sup>−5</sup>; Spearman rank correlation). <b>(d)</b> Average expression maps of laminar-specific genes show significant T1w/T2w map correlations (TMCs). L1-3: supragranular layers 1-3 (<i>r</i><sub><i>s</i></sub> = −0.49; <i>P</i> &lt; 10<sup>−5</sup>); L5/6: infragranular layers 5 and 6 (<i>r</i><sub><i>s</i></sub> = −0.44; <i>P</i> &lt; 10<sup>−5</sup><i>)</i> <b>(e)</b> Genes coding for calretinin <i>(CALB2; r</i><sub><i>s</i></sub> = −0.55<i>; P</i> &lt; 10<sup>−5</sup>) and parvalbumin (<i>PVALB; r</i><sub><i>s</i></sub> = 0.78<i>; P</i> &lt; 10<sup>−5</sup><i>)</i> exhibit homologous hierarchical gradients in human cortex. <b>(f)</b> TMCs of genes coding for markers of specific inhibitory interneuron cell types. <b>(g)</b> The gene coding for the NMDA receptor subunit NR2B (<i>GRIN2B</i>) exhibits a negative TMC (<i>r</i><sub><i>s</i></sub> = −0.69; <i>P</i> &lt; 10<sup>−5</sup>). <b>(h, i)</b> TMCs of genes coding for distinct subunits of NMDA &amp; GABA<sub>A</sub> receptors. <b>(j)</b> PC1 for the brain-specific gene set is highly correlated with the T1w/T2w map (<i>r</i><sub><i>s</i></sub> = 0.88; <i>P</i> &lt; 10<sup>−5</sup>). <b>(k)</b> Across all five gene sets, PC1 exhibits a highly similar areal topography to the T1w/T2w map (TMC range: 0.86–0.88; <i>P</i> &lt; 10<sup>−5</sup> for each). <b>(l)</b> Gene expression variance captured by the T1w/T2w map (<span class="mathjax-tex">\(_{{\mathrm{T}}1{\mathrm{w}}/{\mathrm{T}}2{\mathrm{w}}}^2\)</span>) relative to PC1 (<span class="mathjax-tex">\(_{{\mathrm{PC}}1}^2\)</span>). Statistical significance in panels <b>d</b>, <b>f</b>, <b>h</b>, and <b>i</b> is calculated with a spatial autoregressive model to account for spatial autocorrelation, and in panels <b>k</b> and <b>l</b>, through permutation testing with surrogate maps that preserve spatial autocorrelation structure, Bonferroni-corrected by the number of genes in each set (*, <i>P</i> &lt; 0.05; **, <i>P</i> &lt; 10<sup>−2</sup>; ***, <i>P</i> &lt; 10<sup>−3</sup>). Grey lines in panels <b>d</b>, <b>f</b>, <b>h</b>, and <b>i</b> mark the jackknife estimate of standard error, and in panels <b>k</b> and <b>l</b>, the bootstrap estimated 95% confidence interval. Expression in panels <b>c</b>, <b>e</b>, and <b>g</b> is plotted in units of standard deviations (s.d.; σ) from the mean.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec32-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec32">Supplementary information</h2><div class="c-article-section__content" id="Sec32-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary text and figures" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Text and Figures</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figures 1–10</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Table 1</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Compiled monkey microanatomical data for cytoarchitectural type, Spearman rank correlations for projections terminating in each target area, inhibitory interneuron proportions, and pyramidal neuron dendritic spine counts (Figs. 4a,d,g; Supplementary Fig. 3).</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0195-0/MediaObjects/41593_2018_195_MOESM4_ESM.xlsx" data-supp-info-image="">Supplementary Table 2</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>T1w/T2w map correlation (TMC) values and cortical differential stability (DS<sub>c</sub>) for all genes we analyzed, and sets of brain-, neuron-, oligodendrocyte-, synaptome-, and layer-specific genes.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Hierarchy%20of%20transcriptomic%20specialization%20across%20human%20cortex%20captured%20by%20structural%20neuroimaging%20topography&amp;author=Joshua%20B.%20Burt%20et%20al&amp;contentID=10.1038%2Fs41593-018-0195-0&amp;copyright=The%20Author%28s%29&amp;publication=1097-6256&amp;publicationDate=2018-08-06&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-018-0195-0" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-018-0195-0" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Burt, J.B., Demirtaş, M., Eckner, W.J. <i>et al.</i> Hierarchy of transcriptomic specialization across human cortex captured by structural neuroimaging topography.
                    <i>Nat Neurosci</i> <b>21</b>, 1251–1259 (2018). https://doi.org/10.1038/s41593-018-0195-0</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-018-0195-0?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-03-02">02 March 2018</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-05-30">30 May 2018</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-08-06">06 August 2018</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-09">September 2018</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-018-0195-0</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:A systems identification approach using Bayes factors to deconstruct the brain bases of emotion regulation" href="https://doi.org/10.1038/s41593-024-01605-7">
                                        A systems identification approach using Bayes factors to deconstruct the brain bases of emotion regulation
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Ke Bo</li><li>Thomas E. Kraynak</li><li>Tor D. Wager</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Neuroscience</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Decomposing cortical activity through neuronal tracing connectome-eigenmodes in marmosets" href="https://doi.org/10.1038/s41467-024-46651-8">
                                        Decomposing cortical activity through neuronal tracing connectome-eigenmodes in marmosets
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jie Xia</li><li>Cirong Liu</li><li>Wei Liao</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Macroscale intrinsic dynamics are associated with microcircuit function in focal and generalized epilepsies" href="https://doi.org/10.1038/s42003-024-05819-0">
                                        Macroscale intrinsic dynamics are associated with microcircuit function in focal and generalized epilepsies
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Siqi Yang</li><li>Yimin Zhou</li><li>Zhiqiang Zhang</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Transcriptomic contributions to a modern cytoarchitectonic parcellation of the human cerebral cortex" href="https://doi.org/10.1007/s00429-023-02754-4">
                                        Transcriptomic contributions to a modern cytoarchitectonic parcellation of the human cerebral cortex
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Leana King</li><li>Kevin S. Weiner</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Brain Structure and Function</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Reshaping the Cortical Connectivity Gradient by Long-Term Cognitive Training During Development" href="https://doi.org/10.1007/s12264-023-01108-8">
                                        Reshaping the Cortical Connectivity Gradient by Long-Term Cognitive Training During Development
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Tianyong Xu</li><li>Yunying Wu</li><li>Changsong Zhou</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Neuroscience Bulletin</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0195-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0195-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-018-0195-0;doi=10.1038/s41593-018-0195-0;techmeta=38,57,59;subjmeta=2584,378,3920,631;kwrd=Epigenetics+in+the+nervous+system,Neural+circuits">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=1733665675&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-018-0195-0%26doi%3D10.1038/s41593-018-0195-0%26techmeta%3D38,57,59%26subjmeta%3D2584,378,3920,631%26kwrd%3DEpigenetics+in+the+nervous+system,Neural+circuits">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=1733665675&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-018-0195-0%26doi%3D10.1038/s41593-018-0195-0%26techmeta%3D38,57,59%26subjmeta%3D2584,378,3920,631%26kwrd%3DEpigenetics+in+the+nervous+system,Neural+circuits"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-018-0195-0&amp;format=js&amp;last_modified=2018-08-06" async></script>
<img src="/z4wsjeyz/article/s41593-018-0195-0" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>