<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"autism-spectrum-disorders;psychiatric-disorders","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41593-018-0281-3"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Michael V. Lombardo","Tiziano Pramparo","Vahid Gazestani","Varun Warrier","Richard A. I. Bethlehem","Cynthia Carter Barnes","Linda Lopez","Nathan E. Lewis","Lisa Eyler","Karen Pierce","Eric Courchesne"],"publishedAt":1543190400,"publishedAtString":"2018-11-26","title":"Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"21","issue":"12"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes","description":"Heterogeneity in early language development in autism spectrum disorder (ASD) is clinically important and may reflect neurobiologically distinct subtypes. Here, we identified a large-scale association between multiple coordinated blood leukocyte gene coexpression modules and the multivariate functional neuroimaging (fMRI) response to speech. Gene coexpression modules associated with the multivariate fMRI response to speech were different for all pairwise comparisons between typically developing toddlers and toddlers with ASD and poor versus good early language outcome. Associated coexpression modules were enriched in genes that are broadly expressed in the brain and many other tissues. These coexpression modules were also enriched in ASD-associated, prenatal, human-specific, and language-relevant genes. This work highlights distinctive neurobiology in ASD subtypes with different early language outcomes that is present well before such outcomes are known. Associations between neuroimaging measures and gene expression levels in blood leukocytes may offer a unique in vivo window into identifying brain-relevant molecular mechanisms in ASD. Lombardo et al. find large-scale associations between the leukocyte transcriptome and neural responses to speech in toddlers, and these associations differ between toddlers with ASD who have good versus poor language outcomes.","datePublished":"2018-11-26T00:00:00Z","dateModified":"2018-11-26T00:00:00Z","pageStart":"1680","pageEnd":"1688","sameAs":"https://doi.org/10.1038/s41593-018-0281-3","keywords":["Autism spectrum disorders","Psychiatric disorders","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig5_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"21","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Michael V. Lombardo","url":"http://orcid.org/0000-0001-6780-8619","affiliation":[{"name":"University of Cyprus","address":{"name":"Department of Psychology, University of Cyprus, Nicosia, Cyprus","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Autism Research Centre, Department of Psychiatry, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"mvlombardo@gmail.com","@type":"Person"},{"name":"Tiziano Pramparo","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Neuroscience, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Vahid Gazestani","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Pediatrics, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Varun Warrier","affiliation":[{"name":"University of Cambridge","address":{"name":"Autism Research Centre, Department of Psychiatry, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Richard A. I. Bethlehem","url":"http://orcid.org/0000-0002-0714-0685","affiliation":[{"name":"University of Cambridge","address":{"name":"Autism Research Centre, Department of Psychiatry, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Cynthia Carter Barnes","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Neuroscience, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Linda Lopez","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Neuroscience, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Nathan E. Lewis","url":"http://orcid.org/0000-0001-7700-3654","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Pediatrics, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of California, San Diego","address":{"name":"Department of Bioengineering, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Novo Nordisk Foundation Center for Biosustainability at the University of California, San Diego","address":{"name":"Novo Nordisk Foundation Center for Biosustainability at the University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lisa Eyler","url":"http://orcid.org/0000-0002-7783-8798","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Psychiatry, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Veterans Affairs San Diego Healthcare System","address":{"name":"Veterans Affairs San Diego Healthcare System, San Diego, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Karen Pierce","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Neuroscience, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Eric Courchesne","url":"http://orcid.org/0000-0002-3772-5799","affiliation":[{"name":"University of California, San Diego","address":{"name":"Department of Neuroscience, University of California, San Diego, La Jolla, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"ecourchesne@ucsd.edu","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-018-0281-3">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes"/>
    <meta name="dc.source" content="Nature Neuroscience 2018 21:12"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2018-11-26"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2018 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2018 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="Heterogeneity in early language development in autism spectrum disorder (ASD) is clinically important and may reflect neurobiologically distinct subtypes. Here, we identified a large-scale association between multiple coordinated blood leukocyte gene coexpression modules and the multivariate functional neuroimaging (fMRI) response to speech. Gene coexpression modules associated with the multivariate fMRI response to speech were different for all pairwise comparisons between typically developing toddlers and toddlers with ASD and poor versus good early language outcome. Associated coexpression modules were enriched in genes that are broadly expressed in the brain and many other tissues. These coexpression modules were also enriched in ASD-associated, prenatal, human-specific, and language-relevant genes. This work highlights distinctive neurobiology in ASD subtypes with different early language outcomes that is present well before such outcomes are known. Associations between neuroimaging measures and gene expression levels in blood leukocytes may offer a unique in vivo window into identifying brain-relevant molecular mechanisms in ASD. Lombardo et al. find large-scale associations between the leukocyte transcriptome and neural responses to speech in toddlers, and these associations differ between toddlers with ASD who have good versus poor language outcomes."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2018-11-26"/>
    <meta name="prism.volume" content="21"/>
    <meta name="prism.number" content="12"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="1680"/>
    <meta name="prism.endingPage" content="1688"/>
    <meta name="prism.copyright" content="2018 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-018-0281-3"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-018-0281-3"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-018-0281-3.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-018-0281-3"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes"/>
    <meta name="citation_volume" content="21"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_publication_date" content="2018/12"/>
    <meta name="citation_online_date" content="2018/11/26"/>
    <meta name="citation_firstpage" content="1680"/>
    <meta name="citation_lastpage" content="1688"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-018-0281-3"/>
    <meta name="DOI" content="10.1038/s41593-018-0281-3"/>
    <meta name="size" content="228814"/>
    <meta name="citation_doi" content="10.1038/s41593-018-0281-3"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-018-0281-3&amp;api_key="/>
    <meta name="description" content="Heterogeneity in early language development in autism spectrum disorder (ASD) is clinically important and may reflect neurobiologically distinct subtypes. Here, we identified a large-scale association between multiple coordinated blood leukocyte gene coexpression modules and the multivariate functional neuroimaging (fMRI) response to speech. Gene coexpression modules associated with the multivariate fMRI response to speech were different for all pairwise comparisons between typically developing toddlers and toddlers with ASD and poor versus good early language outcome. Associated coexpression modules were enriched in genes that are broadly expressed in the brain and many other tissues. These coexpression modules were also enriched in ASD-associated, prenatal, human-specific, and language-relevant genes. This work highlights distinctive neurobiology in ASD subtypes with different early language outcomes that is present well before such outcomes are known. Associations between neuroimaging measures and gene expression levels in blood leukocytes may offer a unique in vivo window into identifying brain-relevant molecular mechanisms in ASD. Lombardo et al. find large-scale associations between the leukocyte transcriptome and neural responses to speech in toddlers, and these associations differ between toddlers with ASD who have good versus poor language outcomes."/>
    <meta name="dc.creator" content="Lombardo, Michael V."/>
    <meta name="dc.creator" content="Pramparo, Tiziano"/>
    <meta name="dc.creator" content="Gazestani, Vahid"/>
    <meta name="dc.creator" content="Warrier, Varun"/>
    <meta name="dc.creator" content="Bethlehem, Richard A. I."/>
    <meta name="dc.creator" content="Carter Barnes, Cynthia"/>
    <meta name="dc.creator" content="Lopez, Linda"/>
    <meta name="dc.creator" content="Lewis, Nathan E."/>
    <meta name="dc.creator" content="Eyler, Lisa"/>
    <meta name="dc.creator" content="Pierce, Karen"/>
    <meta name="dc.creator" content="Courchesne, Eric"/>
    <meta name="dc.subject" content="Autism spectrum disorders"/>
    <meta name="dc.subject" content="Psychiatric disorders"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Neurobiol.; citation_title=Autism spectrum disorders: developmental disconnection syndromes; citation_author=DH Geschwind, P Levitt; citation_volume=17; citation_publication_date=2007; citation_pages=103-111; citation_doi=10.1016/j.conb.2007.01.009; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Time to give up on a single explanation for autism; citation_author=F Happ&#233;, A Ronald, R Plomin; citation_volume=9; citation_publication_date=2006; citation_pages=1218-1220; citation_doi=10.1038/nn1770; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Biol.; citation_title=Subgrouping the autism &#8220;spectrum&#8221;: reflections on DSM-5; citation_author=MC Lai, MV Lombardo, B Chakrabarti, S Baron-Cohen; citation_volume=11; citation_publication_date=2013; citation_pages=e1001544; citation_doi=10.1371/journal.pbio.1001544; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Psychiatry; citation_title=Why has it taken so long for biological psychiatry to develop clinical tests and what to do about it?; citation_author=S Kapur, AG Phillips, TR Insel; citation_volume=17; citation_publication_date=2012; citation_pages=1174-1179; citation_doi=10.1038/mp.2012.105; citation_id=CR4"/>
    <meta name="citation_reference" content="Lombardo, M. V., Lai, M. C. &amp; Baron-Cohen, S. Big data approaches to decomposing heterogeneity across the autism spectrum. Preprint at 
                    https://www.biorxiv.org/content/early/2018/03/17/278788
                    
                   (2018)."/>
    <meta name="citation_reference" content="citation_journal_title=Lang. Cogn. Process.; citation_title=An investigation of language impairment in autism: implications for genetic subgroups; citation_author=MM Kjelgaard, H Tager-Flusberg; citation_volume=16; citation_publication_date=2001; citation_pages=287-308; citation_doi=10.1080/01690960042000058; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=Autism Res.; citation_title=Minimally verbal school-aged children with autism spectrum disorder: the neglected end of the spectrum; citation_author=H Tager-Flusberg, C Kasari; citation_volume=6; citation_publication_date=2013; citation_pages=468-478; citation_doi=10.1002/aur.1329; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=J. Autism. Dev. Disord.; citation_title=Outcome in high-functioning adults with autism with and without early language delays: implications for the differentiation between autism and Asperger syndrome; citation_author=P Howlin; citation_volume=33; citation_publication_date=2003; citation_pages=3-13; citation_doi=10.1023/A:1022270118899; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Res. Autism Spectr. Disord.; citation_title=Predictors of outcome for children receiving intensive behavioral intervention in a large, community-based program; citation_author=A Perry; citation_volume=5; citation_publication_date=2011; citation_pages=592-603; citation_doi=10.1016/j.rasd.2010.07.003; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Pediatrics; citation_title=Trajectories of autism severity in children using standardized ADOS scores; citation_author=K Gotham, A Pickles, C Lord; citation_volume=130; citation_publication_date=2012; citation_pages=e1278-e1284; citation_doi=10.1542/peds.2011-3668; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=J. Child Psychol. Psychiatry; citation_title=A follow-up study of high-functioning autistic children; citation_author=A Venter, C Lord, E Schopler; citation_volume=33; citation_publication_date=1992; citation_pages=489-507; citation_doi=10.1111/j.1469-7610.1992.tb00887.x; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=J. Child Psychol. Psychiatry; citation_title=Similar developmental trajectories in autism and Asperger syndrome: from early childhood to adolescence; citation_author=P Szatmari; citation_volume=50; citation_publication_date=2009; citation_pages=1459-1467; citation_doi=10.1111/j.1469-7610.2009.02123.x; citation_id=CR12"/>
    <meta name="citation_reference" content="Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry 
                    https://doi.org/10.1038/s41380-018-0056-y
                    
                   (2018)."/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.; citation_title=Cell cycle networks link gene expression dysregulation, mutation, and brain maldevelopment in autistic toddlers; citation_author=T Pramparo; citation_volume=11; citation_publication_date=2015; citation_pages=841; citation_doi=10.15252/msb.20156108; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=JAMA Psychiatry; citation_title=Prediction of autism by translation and immune/inflammation coexpressed genes in toddlers from pediatric community practices; citation_author=T Pramparo; citation_volume=72; citation_publication_date=2015; citation_pages=386-394; citation_doi=10.1001/jamapsychiatry.2014.3008; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Lancet. Neurol.; citation_title=Gene hunting in autism spectrum disorder: on the path to precision medicine; citation_author=DH Geschwind, MW State; citation_volume=14; citation_publication_date=2015; citation_pages=1109-1120; citation_doi=10.1016/S1474-4422(15)00044-7; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=J. Am. Med. Assoc.; citation_title=The heritability of autism spectrum disorder; citation_author=S Sandin; citation_volume=318; citation_publication_date=2017; citation_pages=1182-1184; citation_doi=10.1001/jama.2017.12141; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Genetic effects on gene expression across human tissues; citation_author=A Battle, CD Brown, BE Engelhardt, SB Montgomery; citation_volume=550; citation_publication_date=2017; citation_pages=204-213; citation_doi=10.1038/nature24277; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Different functional neural substrates for good and poor language outcome in autism; citation_author=MV Lombardo; citation_volume=86; citation_publication_date=2015; citation_pages=567-577; citation_doi=10.1016/j.neuron.2015.03.023; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=An expanded view of complex traits: from polygenic to omnigenic; citation_author=EA Boyle, YI Li, JK Pritchard; citation_volume=169; citation_publication_date=2017; citation_pages=1177-1186; citation_doi=10.1016/j.cell.2017.05.038; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Common disease is more complex than implied by the core gene omnigenic model; citation_author=NR Wray, C Wijmenga, PF Sullivan, J Yang, PM Visscher; citation_volume=173; citation_publication_date=2018; citation_pages=1573-1580; citation_doi=10.1016/j.cell.2018.05.051; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE.; citation_title=The relation between statistical power and inference in fMRI; citation_author=HR Cremers, TD Wager, T Yarkoni; citation_volume=12; citation_publication_date=2017; citation_pages=e0184923; citation_doi=10.1371/journal.pone.0184923; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=WGCNA: an R package for weighted correlation network analysis; citation_author=P Langfelder, S Horvath; citation_volume=9; citation_publication_date=2008; citation_doi=10.1186/1471-2105-9-559; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=The cortical organization of speech processing; citation_author=G Hickok, D Poeppel; citation_volume=8; citation_publication_date=2007; citation_pages=393-402; citation_doi=10.1038/nrn2113; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Insights into the neural and genetic basis of vocal communication; citation_author=G Konopka, TF Roberts; citation_volume=164; citation_publication_date=2016; citation_pages=1269-1276; citation_doi=10.1016/j.cell.2016.02.039; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Convergent transcriptional specializations in the brains of humans and song-learning birds; citation_author=AR Pfenning; citation_volume=346; citation_publication_date=2014; citation_pages=1256846; citation_doi=10.1126/science.1256846; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=The autistic neuron: troubled translation?; citation_author=RJ Kelleher, MF Bear; citation_volume=135; citation_publication_date=2008; citation_pages=401-406; citation_doi=10.1016/j.cell.2008.10.017; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Synaptic, transcriptional and chromatin genes disrupted in autism; citation_author=S Rubeis; citation_volume=515; citation_publication_date=2014; citation_pages=209-215; citation_doi=10.1038/nature13772; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Disruptive CHD8 mutations define a subtype of autism early in development; citation_author=R Bernier; citation_volume=158; citation_publication_date=2014; citation_pages=263-276; citation_doi=10.1016/j.cell.2014.06.017; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiatry; citation_title=Microglial activation and increased microglial density observed in the dorsolateral prefrontal cortex in autism; citation_author=JT Morgan; citation_volume=68; citation_publication_date=2010; citation_pages=368-376; citation_doi=10.1016/j.biopsych.2010.05.024; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Ann. Neurol.; citation_title=Neuroglial activation and neuroinflammation in the brain of patients with autism; citation_author=DL Vargas, C Nascimbene, C Krishnan, AW Zimmerman, CA Pardo; citation_volume=57; citation_publication_date=2005; citation_pages=67-81; citation_doi=10.1002/ana.20315; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Transcriptomic analysis of autistic brain reveals convergent molecular pathology; citation_author=I Voineagu; citation_volume=474; citation_publication_date=2011; citation_pages=380-384; citation_doi=10.1038/nature10110; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Molecular microcircuitry underlies functional specification in a basal ganglia circuit dedicated to vocal learning; citation_author=AT Hilliard, JE Miller, ER Fraley, S Horvath, SA White; citation_volume=73; citation_publication_date=2012; citation_pages=537-552; citation_doi=10.1016/j.neuron.2012.01.005; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Human-specific transcriptional networks in the brain; citation_author=G Konopka; citation_volume=75; citation_publication_date=2012; citation_pages=601-617; citation_doi=10.1016/j.neuron.2012.05.034; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Biol.; citation_title=Disruption of an evolutionarily novel synaptic expression pattern in autism; citation_author=X Liu; citation_volume=14; citation_publication_date=2016; citation_pages=e1002558; citation_doi=10.1371/journal.pbio.1002558; citation_id=CR35"/>
    <meta name="citation_reference" content="Eising, E. et al. A set of regulatory genes co-expressed in embryonic human brain is implicated in disrupted speech development. Mol. Psychiatry 
                    https://doi.org/10.1038/s41380-018-0020-x
                    
                   (2018)."/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism; citation_author=NN Parikshak; citation_volume=155; citation_publication_date=2013; citation_pages=1008-1021; citation_doi=10.1016/j.cell.2013.10.031; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism; citation_author=AJ Willsey; citation_volume=155; citation_publication_date=2013; citation_pages=997-1007; citation_doi=10.1016/j.cell.2013.10.020; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Transcriptional landscape of the prenatal human brain; citation_author=JA Miller; citation_volume=508; citation_publication_date=2014; citation_pages=199-206; citation_doi=10.1038/nature13185; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism; citation_author=NN Parikshak; citation_volume=540; citation_publication_date=2016; citation_pages=423-427; citation_doi=10.1038/nature20612; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Psychiatry; citation_title=Mitochondrial dysfunction in autism spectrum disorders: a systematic review and meta-analysis; citation_author=DA Rossignol, RE Frye; citation_volume=17; citation_publication_date=2012; citation_pages=290-314; citation_doi=10.1038/mp.2010.136; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci; citation_author=SJ Sanders; citation_volume=87; citation_publication_date=2015; citation_pages=1215-1233; citation_doi=10.1016/j.neuron.2015.09.016; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Differential L1 regulation in pluripotent stem cells of humans and apes; citation_author=MCN Marchetto; citation_volume=503; citation_publication_date=2013; citation_pages=525-529; citation_doi=10.1038/nature12686; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Emergence of a Homo sapiens&#8211;specific gene family and chromosome 16p11.2 CNV susceptibility; citation_author=X Nuttle; citation_volume=536; citation_publication_date=2016; citation_pages=205-209; citation_doi=10.1038/nature19075; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiatry; citation_title=The number of genomic copies at the 16p11.2 locus modulates language, verbal memory, and inhibition; citation_author=L Hippolyte; citation_volume=80; citation_publication_date=2016; citation_pages=129-139; citation_doi=10.1016/j.biopsych.2015.10.021; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Sci. Rep.; citation_title=Abnormal speech motor control in individuals with 16p11.2 deletions; citation_author=C Demopoulos; citation_volume=8; citation_publication_date=2018; citation_doi=10.1038/s41598-018-19751-x; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage Clin.; citation_title=Abnormal auditory and language pathways in children with 16p11.2 deletion; citation_author=JI Berman; citation_volume=9; citation_publication_date=2015; citation_pages=50-57; citation_doi=10.1016/j.nicl.2015.07.006; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples; citation_author=JA Kosmicki; citation_volume=49; citation_publication_date=2017; citation_pages=504-510; citation_doi=10.1038/ng.3789; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Analysis of protein-coding genetic variation in 60,706 humans; citation_author=M Lek; citation_volume=536; citation_publication_date=2016; citation_pages=285-291; citation_doi=10.1038/nature19057; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Autism; citation_title=SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs); citation_author=BS Abrahams; citation_volume=4; citation_publication_date=2013; citation_doi=10.1186/2040-2392-4-36; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=Arch. Gen. Psychiatry; citation_title=Preference for geometric patterns early in life as a risk factor for autism; citation_author=K Pierce, D Conant, R Hazin, R Stoner, J Desmond; citation_volume=68; citation_publication_date=2011; citation_pages=101-109; citation_doi=10.1001/archgenpsychiatry.2010.113; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiatry; citation_title=Eye tracking reveals abnormal visual preference for geometric images as an early biomarker of an autism spectrum disorder subtype associated with increased symptom severity; citation_author=K Pierce; citation_volume=79; citation_publication_date=2016; citation_pages=657-666; citation_doi=10.1016/j.biopsych.2015.03.032; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=J. Pediatr.; citation_title=Detecting, studying, and treating autism early: the One-Year Well-Baby Check-Up Approach; citation_author=K Pierce; citation_volume=159; citation_publication_date=2011; citation_pages=458-465.e6; citation_doi=10.1016/j.jpeds.2011.02.036; citation_id=CR53"/>
    <meta name="citation_reference" content="Wetherby, A. &amp; Prizant, B. Communication and Symbolic Behavior Scales Developmental Profile, First Normed Edition (Paul H. Brookes, Baltimore, 2002)."/>
    <meta name="citation_reference" content="citation_journal_title=Autism; citation_title=Validation of the Infant-Toddler Checklist as a broadband screener for autism spectrum disorders from 9 to 24 months of age; citation_author=AM Wetherby, S Brosnan-Maddox, V Peace, L Newton; citation_volume=12; citation_publication_date=2008; citation_pages=487-511; citation_doi=10.1177/1362361308094501; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=J. Autism. Dev. Disord.; citation_title=The autism diagnostic observation schedule-generic: a standard measure of social and communication deficits associated with the spectrum of autism; citation_author=C Lord; citation_volume=30; citation_publication_date=2000; citation_pages=205-223; citation_doi=10.1023/A:1005592401947; citation_id=CR56"/>
    <meta name="citation_reference" content="Mullen, E. M. Mullen Scales of Early Learning (American Guidance Service, Circle Pine, MN, USA, 1995)."/>
    <meta name="citation_reference" content="Sparrow, S., Cicchetti, D. &amp; Balla, D. Vineland-II Scales Of Adaptive Behavior: Survey Form Manual (American Guidance Service, Circle Pines, MN, USA, 2005)."/>
    <meta name="citation_reference" content="citation_journal_title=BMC. Mol. Biol.; citation_title=The RIN: an RNA integrity number for assigning integrity values to RNA measurements; citation_author=A Schroeder; citation_volume=7; citation_publication_date=2006; citation_doi=10.1186/1471-2199-7-3; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=lumi: a pipeline for processing Illumina microarray; citation_author=P Du, WA Kibbe, SM Lin; citation_volume=24; citation_publication_date=2008; citation_pages=1547-1548; citation_doi=10.1093/bioinformatics/btn224; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=Genome. Biol.; citation_title=Bioconductor: open software development for computational biology and bioinformatics; citation_author=RC Gentleman; citation_volume=5; citation_publication_date=2004; citation_doi=10.1186/gb-2004-5-10-r80; citation_id=CR61"/>
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=CellCODE: a robust latent variable approach to differential expression analysis for heterogeneous cell populations; citation_author=M Chikina, E Zaslavsky, SC Sealfon; citation_volume=31; citation_publication_date=2015; citation_pages=1584-1591; citation_doi=10.1093/bioinformatics/btv015; citation_id=CR62"/>
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=limma powers differential expression analyses for RNA-sequencing and microarray studies; citation_author=ME Ritchie; citation_volume=43; citation_publication_date=2015; citation_pages=e47; citation_doi=10.1093/nar/gkv007; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=J. R. Statist. Soc. B; citation_title=A direct approach to false discovery rates; citation_author=JD Storey; citation_volume=64; citation_publication_date=2002; citation_pages=479-498; citation_doi=10.1111/1467-9868.00346; citation_id=CR64"/>
    <meta name="citation_reference" content="Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, Article17 (2005)."/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Conservation and evolution of gene coexpression networks in human and chimpanzee brains; citation_author=MC Oldham, S Horvath, DH Geschwind; citation_volume=103; citation_publication_date=2006; citation_pages=17973-17978; citation_doi=10.1073/pnas.0605938103; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS. Comput. Biol.; citation_title=Is my network module preserved and reproducible?; citation_author=P Langfelder, R Luo, MC Oldham, S Horvath; citation_volume=7; citation_publication_date=2011; citation_pages=e1001057; citation_doi=10.1371/journal.pcbi.1001057; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=Brain; citation_title=A failure of left temporal cortex to specialize for language is an early emerging and fundamental property of autism; citation_author=LT Eyler, K Pierce, E Courchesne; citation_volume=135; citation_publication_date=2012; citation_pages=949-960; citation_doi=10.1093/brain/awr364; citation_id=CR68"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiatry; citation_title=Deviant functional magnetic resonance imaging patterns of brain activity to speech in 2-3-year-old children with autism spectrum disorder; citation_author=E Redcay, E Courchesne; citation_volume=64; citation_publication_date=2008; citation_pages=589-598; citation_doi=10.1016/j.biopsych.2008.05.020; citation_id=CR69"/>
    <meta name="citation_reference" content="citation_journal_title=Dev. Sci.; citation_title=Functional neuroimaging of speech perception during a pivotal period in language acquisition; citation_author=E Redcay, F Haist, E Courchesne; citation_volume=11; citation_publication_date=2008; citation_pages=237-252; citation_doi=10.1111/j.1467-7687.2008.00674.x; citation_id=CR70"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Functional neuroimaging of speech perception in infants; citation_author=G Dehaene-Lambertz, S Dehaene, L Hertz-Pannier; citation_volume=298; citation_publication_date=2002; citation_pages=2013-2015; citation_doi=10.1126/science.1077066; citation_id=CR71"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Thresholding of statistical maps in functional neuroimaging using the false discovery rate; citation_author=CR Genovese, NA Lazar, T Nichols; citation_volume=15; citation_publication_date=2002; citation_pages=870-878; citation_doi=10.1006/nimg.2001.1037; citation_id=CR72"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Large-scale automated synthesis of human functional neuroimaging data; citation_author=T Yarkoni, RA Poldrack, TE Nichols, DC Essen, TD Wager; citation_volume=8; citation_publication_date=2011; citation_pages=665-670; citation_doi=10.1038/nmeth.1635; citation_id=CR73"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Partial Least Squares (PLS) methods for neuroimaging: a tutorial and review; citation_author=A Krishnan, LJ Williams, AR McIntosh, H Abdi; citation_volume=56; citation_publication_date=2011; citation_pages=455-475; citation_doi=10.1016/j.neuroimage.2010.07.034; citation_id=CR74"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Partial least squares analysis of neuroimaging data: applications and advances; citation_author=AR McIntosh, NJ Lobaugh; citation_volume=23; citation_publication_date=2004; citation_pages=S250-S263; citation_doi=10.1016/j.neuroimage.2004.07.020; citation_id=CR75"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment; citation_author=J Cotney; citation_volume=6; citation_publication_date=2015; citation_doi=10.1038/ncomms7404; citation_id=CR76"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism; citation_author=JC Darnell; citation_volume=146; citation_publication_date=2011; citation_pages=247-261; citation_doi=10.1016/j.cell.2011.06.013; citation_id=CR77"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=CHD8 regulates neurodevelopmental pathways associated with autism spectrum disorder in neural progenitors; citation_author=A Sugathan; citation_volume=111; citation_publication_date=2014; citation_pages=E4468-E4477; citation_doi=10.1073/pnas.1405266111; citation_id=CR78"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Fragile X mental retardation 1 gene enhances the translation of large autism-related proteins; citation_author=EJ Greenblatt, AC Spradling; citation_volume=361; citation_publication_date=2018; citation_pages=709-712; citation_doi=10.1126/science.aas9963; citation_id=CR79"/>
    <meta name="citation_author" content="Lombardo, Michael V."/>
    <meta name="citation_author_institution" content="Department of Psychology, University of Cyprus, Nicosia, Cyprus"/>
    <meta name="citation_author_institution" content="Autism Research Centre, Department of Psychiatry, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Pramparo, Tiziano"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author" content="Gazestani, Vahid"/>
    <meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author" content="Warrier, Varun"/>
    <meta name="citation_author_institution" content="Autism Research Centre, Department of Psychiatry, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Bethlehem, Richard A. I."/>
    <meta name="citation_author_institution" content="Autism Research Centre, Department of Psychiatry, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Carter Barnes, Cynthia"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author" content="Lopez, Linda"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author" content="Lewis, Nathan E."/>
    <meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author_institution" content="Department of Bioengineering, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author_institution" content="Novo Nordisk Foundation Center for Biosustainability at the University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author" content="Eyler, Lisa"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author_institution" content="Veterans Affairs San Diego Healthcare System, San Diego, USA"/>
    <meta name="citation_author" content="Pierce, Karen"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, University of California, San Diego, La Jolla, USA"/>
    <meta name="citation_author" content="Courchesne, Eric"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, University of California, San Diego, La Jolla, USA"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes"/>
    <meta name="twitter:description" content="Nature Neuroscience - Lombardo et al. find large-scale associations between the leukocyte transcriptome and neural responses to speech in toddlers, and these associations differ between toddlers..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-018-0281-3"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes - Nature Neuroscience"/>
    <meta property="og:description" content="Lombardo et al. find large-scale associations between the leukocyte transcriptome and neural responses to speech in toddlers, and these associations differ between toddlers with ASD who have good versus poor language outcomes."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-018-0281-3;doi=10.1038/s41593-018-0281-3;techmeta=36,38,45,59,61;subjmeta=1373,1689,378,476,631,692,699;kwrd=Autism+spectrum+disorders,Psychiatric+disorders">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1510801820&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-018-0281-3%26doi%3D10.1038/s41593-018-0281-3%26techmeta%3D36,38,45,59,61%26subjmeta%3D1373,1689,378,476,631,692,699%26kwrd%3DAutism+spectrum+disorders,Psychiatric+disorders">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1510801820&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-018-0281-3%26doi%3D10.1038/s41593-018-0281-3%26techmeta%3D36,38,45,59,61%26subjmeta%3D1373,1689,378,476,631,692,699%26kwrd%3DAutism+spectrum+disorders,Psychiatric+disorders"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-018-0281-3'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0281-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2018-11-26">26 November 2018</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_V_-Lombardo-Aff1-Aff2" data-author-popup="auth-Michael_V_-Lombardo-Aff1-Aff2" data-author-search="Lombardo, Michael V." data-corresp-id="c1">Michael V. Lombardo<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-6780-8619"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6780-8619</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tiziano-Pramparo-Aff3" data-author-popup="auth-Tiziano-Pramparo-Aff3" data-author-search="Pramparo, Tiziano">Tiziano Pramparo</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Vahid-Gazestani-Aff4" data-author-popup="auth-Vahid-Gazestani-Aff4" data-author-search="Gazestani, Vahid">Vahid Gazestani</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Varun-Warrier-Aff2" data-author-popup="auth-Varun-Warrier-Aff2" data-author-search="Warrier, Varun">Varun Warrier</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Richard_A__I_-Bethlehem-Aff2" data-author-popup="auth-Richard_A__I_-Bethlehem-Aff2" data-author-search="Bethlehem, Richard A. I.">Richard A. I. Bethlehem</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-0714-0685"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0714-0685</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Cynthia-Carter_Barnes-Aff3" data-author-popup="auth-Cynthia-Carter_Barnes-Aff3" data-author-search="Carter Barnes, Cynthia">Cynthia Carter Barnes</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Linda-Lopez-Aff3" data-author-popup="auth-Linda-Lopez-Aff3" data-author-search="Lopez, Linda">Linda Lopez</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nathan_E_-Lewis-Aff4-Aff5-Aff6" data-author-popup="auth-Nathan_E_-Lewis-Aff4-Aff5-Aff6" data-author-search="Lewis, Nathan E.">Nathan E. Lewis</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-7700-3654"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7700-3654</a></span><sup class="u-js-hide"><a href="#Aff4">4</a>,<a href="#Aff5">5</a>,<a href="#Aff6">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lisa-Eyler-Aff7-Aff8" data-author-popup="auth-Lisa-Eyler-Aff7-Aff8" data-author-search="Eyler, Lisa">Lisa Eyler</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-7783-8798"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-7783-8798</a></span><sup class="u-js-hide"><a href="#Aff7">7</a>,<a href="#Aff8">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Karen-Pierce-Aff3" data-author-popup="auth-Karen-Pierce-Aff3" data-author-search="Pierce, Karen">Karen Pierce</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 11 authors for this article" title="Show all 11 authors for this article"></li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Eric-Courchesne-Aff3" data-author-popup="auth-Eric-Courchesne-Aff3" data-author-search="Courchesne, Eric" data-corresp-id="c2">Eric Courchesne<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-3772-5799"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3772-5799</a></span><sup class="u-js-hide"><a href="#Aff3">3</a></sup></li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>21</b>,<span class="u-visually-hidden">pages </span>16801688 (<span data-test="article-publication-year">2018</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">4563 <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">47 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">96 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-018-0281-3/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/autism-spectrum-disorders" data-track="click" data-track-action="view subject" data-track-label="link">Autism spectrum disorders</a></li><li class="c-article-subject-list__subject"><a href="/subjects/psychiatric-disorders" data-track="click" data-track-action="view subject" data-track-label="link">Psychiatric disorders</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Heterogeneity in early language development in autism spectrum disorder (ASD) is clinically important and may reflect neurobiologically distinct subtypes. Here, we identified a large-scale association between multiple coordinated blood leukocyte gene coexpression modules and the multivariate functional neuroimaging (fMRI) response to speech. Gene coexpression modules associated with the multivariate fMRI response to speech were different for all pairwise comparisons between typically developing toddlers and toddlers with ASD and poor versus good early language outcome. Associated coexpression modules were enriched in genes that are broadly expressed in the brain and many other tissues. These coexpression modules were also enriched in ASD-associated, prenatal, human-specific, and language-relevant genes. This work highlights distinctive neurobiology in ASD subtypes with different early language outcomes that is present well before such outcomes are known. Associations between neuroimaging measures and gene expression levels in blood leukocytes may offer a unique in vivo window into identifying brain-relevant molecular mechanisms in ASD.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0281-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0281-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-024-07109-5/MediaObjects/41586_2024_7109_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-024-07109-5?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41586-024-07109-5">A concerted neuronastrocyte program declines in ageing and schizophrenia
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">06 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Emi Ling, James Nemesh,  Steven A. McCarroll</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01668-z/MediaObjects/41588_2024_1668_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41588-024-01668-z?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41588-024-01668-z">Epigenetic variation impacts individual differences in the transcriptional response to influenza infection
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">29 February 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Katherine A. Aracena, Yen-Lung Lin,  Luis B. Barreiro</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-023-06812-z/MediaObjects/41586_2023_6812_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-023-06812-z?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41586-023-06812-z">A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">13 December 2023</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Zizhen Yao, Cindy T. J. van Velthoven,  Hongkui Zeng</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'topic',
                        model: 'visits_v2',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711582515,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>ASD is heterogeneous at multiple levels (e.g., genetics, cellular and neural systems, cognition, behavior, developmental trajectories, prognosis, and response to treatment)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Geschwind, D. H. &amp; Levitt, P. Autism spectrum disorders: developmental disconnection syndromes. Curr. Opin. Neurobiol. 17, 103111 (2007)." href="#ref-CR1" id="ref-link-section-d136720129e640">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Happ, F., Ronald, A. &amp; Plomin, R. Time to give up on a single explanation for autism. Nat. Neurosci. 9, 12181220 (2006)." href="#ref-CR2" id="ref-link-section-d136720129e640_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Lai, M. C., Lombardo, M. V., Chakrabarti, B. &amp; Baron-Cohen, S. Subgrouping the autism spectrum: reflections on DSM-5. PLoS Biol. 11, e1001544 (2013)." href="/articles/s41593-018-0281-3#ref-CR3" id="ref-link-section-d136720129e643">3</a></sup>. This multilevel heterogeneity presents a major challenge on the path toward stratified psychiatry and precision medicine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Kapur, S., Phillips, A. G. &amp; Insel, T. R. Why has it taken so long for biological psychiatry to develop clinical tests and what to do about it? Mol. Psychiatry 17, 11741179 (2012)." href="/articles/s41593-018-0281-3#ref-CR4" id="ref-link-section-d136720129e647">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Lombardo, M. V., Lai, M. C. &amp; Baron-Cohen, S. Big data approaches to decomposing heterogeneity across the autism spectrum. Preprint at &#xA;                    https://www.biorxiv.org/content/early/2018/03/17/278788&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR5" id="ref-link-section-d136720129e650">5</a></sup>. One dimension of heterogeneity with clinical importance for ASD is early language development and outcome. There is a wide spectrum of variability in early language abilities in the population with ASD, from individuals who remain minimally verbal to those who have difficulties similar to specific language impairment, to those who develop near-typical levels of language function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Kjelgaard, M. M. &amp; Tager-Flusberg, H. An investigation of language impairment in autism: implications for genetic subgroups. Lang. Cogn. Process. 16, 287308 (2001)." href="/articles/s41593-018-0281-3#ref-CR6" id="ref-link-section-d136720129e654">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Tager-Flusberg, H. &amp; Kasari, C. Minimally verbal school-aged children with autism spectrum disorder: the neglected end of the spectrum. Autism Res. 6, 468478 (2013)." href="/articles/s41593-018-0281-3#ref-CR7" id="ref-link-section-d136720129e657">7</a></sup>. Early language ability is paramount for better understanding a range of clinical phenomena. For example, early language ability is one of the most important predictors of early intervention response and later-life outcomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Howlin, P. Outcome in high-functioning adults with autism with and without early language delays: implications for the differentiation between autism and Asperger syndrome. J. Autism. Dev. Disord. 33, 313 (2003)." href="#ref-CR8" id="ref-link-section-d136720129e661">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Perry, A. et al. Predictors of outcome for children receiving intensive behavioral intervention in a large, community-based program. Res. Autism Spectr. Disord. 5, 592603 (2011)." href="#ref-CR9" id="ref-link-section-d136720129e661_1">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gotham, K., Pickles, A. &amp; Lord, C. Trajectories of autism severity in children using standardized ADOS scores. Pediatrics 130, e1278e1284 (2012)." href="#ref-CR10" id="ref-link-section-d136720129e661_2">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Venter, A., Lord, C. &amp; Schopler, E. A follow-up study of high-functioning autistic children. J. Child Psychol. Psychiatry 33, 489507 (1992)." href="#ref-CR11" id="ref-link-section-d136720129e661_3">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Szatmari, P. et al. Similar developmental trajectories in autism and Asperger syndrome: from early childhood to adolescence. J. Child Psychol. Psychiatry 50, 14591467 (2009)." href="/articles/s41593-018-0281-3#ref-CR12" id="ref-link-section-d136720129e664">12</a></sup>.</p><p>An additional challenge lies in studying the relationships between the macroscale properties of the brain and the molecular mechanisms at play in early development, and how this relationship may be altered in ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0056-y&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR13" id="ref-link-section-d136720129e671">13</a></sup>. fMRI can be used to gain insight into the macroscale, neural systems level of organization and its association with cognitive and behavioral functioning. However, the molecular biological underpinnings of this organization are not well understood. Although blood samples are a practical source for assaying atypical gene expression in early ASD development<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pramparo, T. et al. Cell cycle networks link gene expression dysregulation, mutation, and brain maldevelopment in autistic toddlers. Mol. Syst. Biol. 11, 841 (2015)." href="/articles/s41593-018-0281-3#ref-CR14" id="ref-link-section-d136720129e675">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Pramparo, T. et al. Prediction of autism by translation and immune/inflammation coexpressed genes in toddlers from pediatric community practices. JAMA Psychiatry 72, 386394 (2015)." href="/articles/s41593-018-0281-3#ref-CR15" id="ref-link-section-d136720129e678">15</a></sup>, a common question is how relevant they are for understanding atypical neural processes in ASD. The evidence for a genetic basis of ASD is strong<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Geschwind, D. H. &amp; State, M. W. Gene hunting in autism spectrum disorder: on the path to precision medicine. Lancet. Neurol. 14, 11091120 (2015)." href="/articles/s41593-018-0281-3#ref-CR16" id="ref-link-section-d136720129e682">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Sandin, S. et al. The heritability of autism spectrum disorder. J. Am. Med. Assoc. 318, 11821184 (2017)." href="/articles/s41593-018-0281-3#ref-CR17" id="ref-link-section-d136720129e685">17</a></sup>, and genetic variation is likely to affect gene expression levels in multiple tissues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Battle, A., Brown, C. D., Engelhardt, B. E. &amp; Montgomery, S. B. Genetic effects on gene expression across human tissues. Nature 550, 204213 (2017)." href="/articles/s41593-018-0281-3#ref-CR18" id="ref-link-section-d136720129e689">18</a></sup>, including the brain and blood. Thus, identifying associations between the blood leukocyte transcriptome and neuroimaging phenotypes may help shed light on mechanisms affecting early neural systems development in toddlers with ASD compared with typically developing toddlers (TD). Such an in vivo window into the biology of ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0056-y&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR13" id="ref-link-section-d136720129e693">13</a></sup> may be able to not only advance understanding of the mechanisms underlying atypical brain development in heterogeneous ASD patients but also advance translational work targeted at better monitoring treatment response, predicting prognosis, and evaluating clinical trials.</p><p>Here, we asked whether large-scale coordinated gene expression in blood leukocytes might be associated with neural responses to speech, as measured with fMRI, and whether this association might differ between TD and toddlers with ASD and either poor or good early language outcome. A fundamental question in this work was whether differences in early language outcomes might be a biologically relevant basis for stratifying ASD. On the basis of prior work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. Neuron 86, 567577 (2015)." href="/articles/s41593-018-0281-3#ref-CR19" id="ref-link-section-d136720129e700">19</a></sup> suggesting that early language outcome subtypes are underpinned by distinct biology, we predicted that early language outcome ASD subtypes would show different profiles of associations between blood leukocyte gene expression and the functional neural systems response to speech.</p><p>Examining large-scale blood leukocyte transcriptome associations with neuroimaging phenotypes in ASD may also identify novel mechanisms involved in ASD. The omnigenic model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Cell 169, 11771186 (2017)." href="/articles/s41593-018-0281-3#ref-CR20" id="ref-link-section-d136720129e707">20</a></sup>, much like other viewpoints on polygenic risk<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Wray, N. R., Wijmenga, C., Sullivan, P. F., Yang, J. &amp; Visscher, P. M. Common disease is more complex than implied by the core gene omnigenic model. Cell 173, 15731580 (2018)." href="/articles/s41593-018-0281-3#ref-CR21" id="ref-link-section-d136720129e711">21</a></sup>, predicts that large numbers of genes are relevant to complex traits such as those in ASD. However, the omnigenic model suggests that these genes do not necessarily need to be specific to brain tissue. Genes that are broadly expressed in one or more tissues, including the brain and blood, are predicted to contain a large amount of the heritability signal and may contribute more to overall risk than the smaller number of tissue-specific genes implicated in a complex trait<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Cell 169, 11771186 (2017)." href="/articles/s41593-018-0281-3#ref-CR20" id="ref-link-section-d136720129e715">20</a></sup>. Applying these ideas to the current study, we predicted that large-scale coordinated transcriptional activity in the blood leukocyte transcriptome might be relevant for explaining neural phenotypes relevant to ASD. The omnigenic model predicts that this large-scale transcriptomic signal would be enriched for genes that are broadly expressed in the brain and many other tissues.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Group differentiation in the superior temporal cortex response to speech and clinical behavioral trajectories over the first 4 years of life</h3><p>In this study, we compared TD and age-matched toddlers with ASD whose language abilities were assessed around 34 years of age. Toddlers with ASD were stratified by poor (ASD Poor) or good (ASD Good) language outcome. ASD Poor was defined by Mullen Expressive Language (EL) and Receptive Language (RL) <i>T</i> scores below 1s.d. of typical developing age norms (<i>T</i>&lt;40). In contrast, ASD Good was defined by outcome Mullen EL or RL within 1s.d. of typical age norms (T 40) (Methods). In prior work, we have shown that this stratification identifies an ASD subtype with different developmental trajectories and a decreased left-hemisphere superior temporal cortex response to speech, as measured with sleep fMRI, before the diagnosis and outcome are known<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. Neuron 86, 567577 (2015)." href="/articles/s41593-018-0281-3#ref-CR19" id="ref-link-section-d136720129e737">19</a></sup>. The current dataset included a subset of toddlers from the prior paper (<i>n</i>=19 ASD Poor; <i>n</i>=24 ASD Good; <i>n</i>=21 TD) and adds a similar number of new toddlers (<i>n</i>=22 ASD Poor, <i>n</i>=16 ASD Good, and <i>n</i>=16 TD). Therefore, we first reran longitudinal clinical trajectory and fMRI analyses on this combined dataset.</p><p>As previously reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. Neuron 86, 567577 (2015)." href="/articles/s41593-018-0281-3#ref-CR19" id="ref-link-section-d136720129e763">19</a></sup>, all longitudinal clinical measures showed evidence of subtypeage interactions (except for Mullen Fine Motor and Autism Diagnostic Observation Schedule (ADOS) total), thus indicating that the groups differed in the slopes of the trajectories (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig1">1</a>; statistics in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM3">1</a>). This difference was generally driven by the ASD Poor group, whose downward trajectories were indicative of falling further behind age-appropriate norms. All measures also showed main effects of group and were generally due to all groups differing from one another in a hierarchy of ASD Poor as most severe, ASD Good as intermediate, and TD as least severe. With the fMRI data, we also found that the previously reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. Neuron 86, 567577 (2015)." href="/articles/s41593-018-0281-3#ref-CR19" id="ref-link-section-d136720129e773">19</a></sup> hypoactivation in the left-hemisphere superior temporal cortex remained stable in this combined sample (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig2">2</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM4">2</a>). Whole-brain between-group analyses did not reveal any regions differentiating the groups. However, the lack of effects in this context are likely to be due to low statistical power for whole-brain between-group comparisons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Cremers, H. R., Wager, T. D. &amp; Yarkoni, T. The relation between statistical power and inference in fMRI. PLoS ONE. 12, e0184923 (2017)." href="/articles/s41593-018-0281-3#ref-CR22" id="ref-link-section-d136720129e784">22</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Clinical behavioral trajectories over the first 4 years of life in TD toddlers and toddlers with ASD plus good or poor early language outcome."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Clinical behavioral trajectories over the first 4 years of life in TD toddlers and toddlers with ASD plus good or poor early language outcome.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="243"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>This figure shows developmental trajectories over the first 4 years of life for TD toddlers, toddlers with ASD plus good early language outcome (ASD Good), and toddlers with ASD plus poor early language outcome (ASD Poor) on clinical-behavioral-assessment measures such as ADOS total scores, Mullen Scales of Early Learning subscales (Expressive and Receptive Language, Visual Reception, and Fine Motor) and Vineland Adaptive Behavioral Scales (Communication, Socialization, Daily Living Skills, Motor, and Adaptive Behavior). TD (<i>n</i>=35), blue; ASD Good (<i>n</i>=40), pink; ASD Poor (<i>n</i>=41), green. Individual-level trajectories are plotted, including the group-level trajectory and 95%-confidence band.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Decreased fMRI response to speech in toddlers with ASD and poor early language outcome."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Decreased fMRI response to speech in toddlers with ASD and poor early language outcome.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="297"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Results of whole-brain analyses (one-tailed <i>t</i> test) on each group separately (results shown at FDR <i>q</i>&lt;0.05) (TD, <i>n</i>=37; ASD Good, <i>n</i>=40; ASD Poor, <i>n</i>=41). <b>b</b>, Results of region of interest (ROI) analyses testing for subtype differences. ROIs are defined by four regions within the Neurosynth Language meta-analysis map in the left hemisphere (LH) or right hemisphere (RH) frontal and temporal cortex. ROI data are shown for each individual in the scatter box plots (TD, blue, <i>n</i>=37; ASD Good, pink, <i>n</i>=40; ASD Poor, green, <i>n</i>=41). The boxes indicate interquartile range (IQR; first quartile, twenty-fifth percentile; third quartile, seventy-fifth percentile), and the whiskers indicate Q1  (1.5IQR) or Q3+(1.5IQR). The line within the box represents the median. Matrices next to the scatter box plots show standardized effect sizes (Cohens <i>d</i>) for each pairwise group comparison. Cohens <i>d</i> is shown in each cell and is also indicated by the color of the cell. Within each cell, *<i>P</i>&lt;0.05; **<i>P</i>&lt;0.005.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec4">Lack of group differentiation within gene expression data alone</h3><p>The total sample of <i>n</i>=41 ASD Poor, <i>n</i>=40 ASD Good, and <i>n</i>=37 TD individuals was the largest dataset of toddlers for whom both fMRI and gene expression data were available. We next assessed whether differences in blood leukocyte gene expression might reflect different neural systems organization between the two ASD language outcome subtypes. First, we assessed differential expression (DE) between subtypes at the level of individual genes. After correction for multiple comparison, no genes were identified as DE for any pairwise group comparison (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM5">3</a>). Next, we used weighted gene coexpression network analysis (WGCNA) to decrease redundancy among the 14,313 genes, down to 21 discrete coexpression modules. The coexpression modules were summarized by the first principal component score, also known as the module eigengene (ME)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Langfelder, P. &amp; Horvath, S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics 9, 559 (2008)." href="/articles/s41593-018-0281-3#ref-CR23" id="ref-link-section-d136720129e900">23</a></sup>. Similarly to the DE analysis at the gene level, there were no ME differences between the two ASD subtypes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM5">3</a>). Thus, examining blood leukocyte gene expression data in isolation did not significantly differentiate the groups at the current sample sizes. We next turned to examining associations between gene expression and functional neuroimaging phenotypes.</p><h3 class="c-article__sub-heading" id="Sec5">Large-scale blood leukocyte gene coexpression module association with fMRI response to speech</h3><p>Multivariate analysis of the association between coexpression modules and whole-brain voxel-wise patterns of activation was implemented with partial least-squares (PLS) analyses. Of the 63 total latent variable (LV) coexpressionfMRI pairs, PLS identified only one LV pair with a statistically significant association after multiple-comparison correction (LV1: <i>d</i>=65.47, <i>P</i>=1.9910<sup>4</sup>, false discovery rate (FDR) <i>q</i>=0.0125; statistics for all PLS LV pairs in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM6">4</a>). LV1 accounted for 20.13% of the covariance between gene expression and fMRI data and was spatially distributed across a number of cortical regions that have been found to be highly relevant to speech, language<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Hickok, G. &amp; Poeppel, D. The cortical organization of speech processing. Nat. Rev. Neurosci. 8, 393402 (2007)." href="/articles/s41593-018-0281-3#ref-CR24" id="ref-link-section-d136720129e931">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Konopka, G. &amp; Roberts, T. F. Insights into the neural and genetic basis of vocal communication. Cell 164, 12691276 (2016)." href="/articles/s41593-018-0281-3#ref-CR25" id="ref-link-section-d136720129e934">25</a></sup> (e.g., the primary auditory cortex, superior temporal sulcus, inferior frontal gyrus, ventral premotor cortex, and insula), visual and sensorimotor areas (e.g., the primary visual cortex, superior parietal cortex, primary somatomotor cortex, and premotor cortex), cognitive control (dorsolateral prefrontal cortex), and social brain circuitry overlapping with key areas of the default mode network (e.g., the posterior cingulate cortex, medial prefrontal cortex, right temporoparietal junction, and superior temporal sulcus) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig3">3a</a>). Subcortical regions such as the striatum and thalamus were also implicated and are highly relevant for language processes such as vocal learning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Konopka, G. &amp; Roberts, T. F. Insights into the neural and genetic basis of vocal communication. Cell 164, 12691276 (2016)." href="/articles/s41593-018-0281-3#ref-CR25" id="ref-link-section-d136720129e941">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Pfenning, A. R. et al. Convergent transcriptional specializations in the brains of humans and song-learning birds. Science 346, 1256846 (2014)." href="/articles/s41593-018-0281-3#ref-CR26" id="ref-link-section-d136720129e944">26</a></sup>. For example, Area X in songbirds is linked to vocal learning and is homologous to the human dorsal striatum<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Pfenning, A. R. et al. Convergent transcriptional specializations in the brains of humans and song-learning birds. Science 346, 1256846 (2014)." href="/articles/s41593-018-0281-3#ref-CR26" id="ref-link-section-d136720129e948">26</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Multivariate gene coexpressionfMRI association in ASD with good or poor early language outcome and typically developing control toddlers."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Multivariate gene coexpressionfMRI association in ASD with good or poor early language outcome and typically developing control toddlers.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="447"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Brain regions with the strongest contributions to the multivariate gene coexpressionfMRI association present in the LV1 PLS results. The coloring in each region indicates the bootstrap ratio (BSR) and reflects how important each voxel is to the LV1 PLS results. Areas are shown if the BSR1.96 or BSR1.96. Hot-colored regions are interpreted as showing a positive gene coexpressionfMRI correlationthat is, as a modules eigengene increases, functional activation in response to speech also increases. In contrast, cool-colored areas indicate a negative correlation between a modules eigengene and functional activation response to speech. <b>b</b>, Table describing which modules were the strongest contributors to the LV1 PLS results. Each row indicates one of the 21 coexpression modules used in the PLS analysis. The columns labeled with the heading nonzero modules are broken down to indicate gene coexpressionfMRI correlations by group. Cells in these columns are colored red or blue if the gene coexpressionfMRI correlation was nonzero and had 95%CIs (estimated from bootstrapping) that did not include a correlation of 0. These modules are called nonzero modules because they were the strongest contributors or modules important to the LV1 PLS results. All other modules with white cells are labeled zero modules because the 95%CIs for the gene coexpressionfMRI correlation included 0. Nonzero modules have cells colored in red to indicate a positive gene coexpressionfMRI correlation (that is, congruent with the interpretation already stated for the hot- and cool-colored regions in <b>a</b>). However, in the case of nonzero modules with cells colored in blue, the previously stated way to interpret the hot- and cool-colored regions in <b>a</b> should reverse (for example, cool-colored regions in <b>a</b> reflect positive correlations with a modules eigengene, and hot-colored regions in <b>a</b> reflect negative correlations with a modules eigengene). The remaining columns in <b>b</b> with the heading biological processes annotate each module for enrichment in biological process terms from Metacore GeneGO software. Cyan-colored cells indicate modules with enrichments passing FDR <i>q</i>&lt;0.05 for multiple-comparison correction. A complete description of these biological process enrichments is available in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM7">5</a>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec6">Extent of nonzero associations across coexpression modules</h3><p>To better understand the most important coexpression modules for the PLS LV1 results, we first identified what we termed nonzero association modules, which were defined as gene coexpression modules with gene coexpressionfMRI correlations with 95% confidence intervals (CIs) that excluded a correlation of 0. Nonzero modules accounted for approximately half of all modules analyzed (11/21; 52%) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig3">3b</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM6">4</a>). The remaining modules (10/21; 48%) were referred to as zero modules, defined as gene coexpression modules for which a correlation of 0 lay within the 95%CIs. Zero modules contributed little to and/or were unreliable in how they contributed to LV1. Nonzero modules covered most (61%) of the transcriptome considered for the WGCNA analysis. This widespread coverage indicated a coordinated and large-scale signal spanning large parts of the blood leukocyte transcriptome associated with macroscale functional neural response to speech, as measured with fMRI.</p><p>Most nonzero modules were characterized by a variety of biological processes generally falling within categories such as translation, transcription, cell-cycle, immunological, inflammatory, signal-transduction, and cytoskeleton processes. However, the enrichments differed substantially depending on the module (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig3">3b</a>; complete description of enrichments for each module in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM7">5</a>). For instance, nonzero modules M2, M8, and M11 were primarily translation and transcription modules, whereas M1 and M10 were enriched in many of these terms but not translation and transcription. These biological processes have all been implicated in autism. For example, translation processes are affected in many syndromic forms of ASD (such as fragile X syndrome and tuberous sclerosis)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Kelleher, R. J. III &amp; Bear, M. F. The autistic neuron: troubled translation? Cell 135, 401406 (2008)." href="/articles/s41593-018-0281-3#ref-CR27" id="ref-link-section-d136720129e1022">27</a></sup>. Many high-confidence ASD-risk genes are known to affect transcription processes (e.g., <i>CHD8</i>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="De Rubeis, S. et al. Synaptic, transcriptional and chromatin genes disrupted in autism. Nature 515, 209215 (2014)." href="/articles/s41593-018-0281-3#ref-CR28" id="ref-link-section-d136720129e1029">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Bernier, R. et al. Disruptive CHD8 mutations define a subtype of autism early in development. Cell 158, 263276 (2014)." href="/articles/s41593-018-0281-3#ref-CR29" id="ref-link-section-d136720129e1032">29</a></sup>. Cell-cycle processes are involved in aberrant early cell proliferation and increased early brain growth in ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0056-y&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR13" id="ref-link-section-d136720129e1037">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pramparo, T. et al. Cell cycle networks link gene expression dysregulation, mutation, and brain maldevelopment in autistic toddlers. Mol. Syst. Biol. 11, 841 (2015)." href="/articles/s41593-018-0281-3#ref-CR14" id="ref-link-section-d136720129e1040">14</a></sup>. Immunological and inflammatory processes have been linked to ASD via various lines of evidence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Morgan, J. T. et al. Microglial activation and increased microglial density observed in the dorsolateral prefrontal cortex in autism. Biol. Psychiatry 68, 368376 (2010)." href="#ref-CR30" id="ref-link-section-d136720129e1044">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vargas, D. L., Nascimbene, C., Krishnan, C., Zimmerman, A. W. &amp; Pardo, C. A. Neuroglial activation and neuroinflammation in the brain of patients with autism. Ann. Neurol. 57, 6781 (2005)." href="#ref-CR31" id="ref-link-section-d136720129e1044_1">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Voineagu, I. et al. Transcriptomic analysis of autistic brain reveals convergent molecular pathology. Nature 474, 380384 (2011)." href="/articles/s41593-018-0281-3#ref-CR32" id="ref-link-section-d136720129e1047">32</a></sup>. These results supports the idea that ASD-relevant biological processes can be assayed in blood leukocytes and are associated with early-developing large-scale functional neuroimaging phenotypes.</p><h3 class="c-article__sub-heading" id="Sec7">Lack of overlap in nonzero modules across ASD subtypes and TD</h3><p>Most nonzero modules were present only in one group (9/11; 81%). In fact, only TD and ASD Poor showed evidence of nonzero modules. No nonzero modules were present in the ASD Good subtype. Two (18%) nonzero modules were present in both TD and ASD Poor and were correlated in the same direction. However, the extent of this overlap was not statistically significant (enrichment odds ratio (OR)=1.67, <i>P</i>=0.65) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig3">3b</a>). This result suggests that different biological mechanisms within each group may underpin the variability observed in macroscale language-relevant fMRI phenotypes. To further test the importance of the ASD language outcome subtype distinction, we next tested whether a simple casecontrol distinction might enhance sensitivity in detecting gene coexpressionfMRI relationships. This casecontrol PLS analysis was not able to identify any statistically significant LV pairs at FDR <i>q</i>&lt;0.05 with correction for multiple comparisons (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM6">4</a>). Thus, using early language outcomes as a stratifier in ASD appears to substantially enhance sensitivity for detecting gene coexpressionfMRI relationships.</p><h3 class="c-article__sub-heading" id="Sec8">Nonzero modules are enriched in broadly expressed genes</h3><p>We next examined what class of genes was likely to strongly contribute to the nonzero modules. On the basis of ideas from the omnigenic model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Cell 169, 11771186 (2017)." href="/articles/s41593-018-0281-3#ref-CR20" id="ref-link-section-d136720129e1080">20</a></sup>, genes that are broadly expressed in many tissues including the brain, could also be expressed and measurable in blood leukocytes and therefore may be highly relevant for these nonzero modules associated with a functional neuroimaging phenotype. Remarkably, we found that 81% (9/11) of nonzero modules were enriched in broadly expressed genes (OR=184.5, <i>P</i>=1.8710<sup>4</sup>). All modules enriched in broadly expressed genes were also nonzero modules (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig4">4b</a>). In contrast, tissue-specific gene lists (for example, brain, whole-blood, and lymphocyte lists) were not heavily enriched in many modules nor were they overrepresented in nonzero modules (brain-specific modules: OR=0, <i>P</i>=1; whole-blood-specific modules: OR=0.6, <i>P</i>=0.96; lymphocyte-specific modules: OR=4.44, <i>P</i>=0.53) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig4">4b</a>). In addition to running these enrichments at the level of overlap among modules, we also ran tests for overlap at the gene level. Approximately 44% of all broadly expressed genes were present in nonzero modules, amounting to a highly significant enrichment (OR=3.58, <i>P</i>=1.4810<sup>93</sup>). Whole-blood and lymphocyte-specific genes also showed evidence of enrichment in nonzero modules (blood OR=4.79, <i>P</i>=1.5710<sup>18</sup>; lymphocyte OR=2.82, <i>P</i>=1.9410<sup>8</sup>), although whole-blood-specific genes also showed enrichment in zero modules (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig4">4a</a>). The whole-blood and lymphocyte-specific enrichments were likely to be driven by genes within one or two nonzero modules (e.g., M1, M6, and M17) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig4">4b</a>). In contrast, the enrichment in broadly expressed genes was driven by genes spread across nearly every single nonzero module.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Tissue-class enrichments with sets of nonzero or zero association modules."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Tissue-class enrichments with sets of nonzero or zero association modules.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="466"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>Enrichments in different classes of genes taken from the analysis by Boyle et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Cell 169, 11771186 (2017)." href="/articles/s41593-018-0281-3#ref-CR20" id="ref-link-section-d136720129e1139">20</a></sup> of tissue-specific or broadly expressed genes from GTEx data. <b>a</b>, Enrichment ORs (numbers) and log10 <i>P</i> values (coloring) for each hypergeometric test for enrichment. Cells outlined in green passed multiple-comparison correction at FDR <i>q</i>&lt;0.05. <b>b</b>, All gene coexpression modules (rows) and whether they were enriched in each tissue class (columns). Modules with enrichments passing FDR <i>q</i>&lt;0.05 for multiple-comparison correction are indicated as colored cells. The first three columns show which modules had nonzero associations (colored cells), as shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig3">3b</a>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec9">Nonzero modules are enriched in differentially expressed genes in a songbird vocal-learning model</h3><p>Given the relationship between the expression of genes in nonzero modules and language-relevant functional neuroimaging phenotypes, we next sought to validate whether these genes were brain relevant and conserved in an animal model of vocal learning. Vocal learning is a language-relevant ability shared between humans and songbirds and has been extensively examined<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Konopka, G. &amp; Roberts, T. F. Insights into the neural and genetic basis of vocal communication. Cell 164, 12691276 (2016)." href="/articles/s41593-018-0281-3#ref-CR25" id="ref-link-section-d136720129e1178">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Pfenning, A. R. et al. Convergent transcriptional specializations in the brains of humans and song-learning birds. Science 346, 1256846 (2014)." href="/articles/s41593-018-0281-3#ref-CR26" id="ref-link-section-d136720129e1181">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Hilliard, A. T., Miller, J. E., Fraley, E. R., Horvath, S. &amp; White, S. A. Molecular microcircuitry underlies functional specification in a basal ganglia circuit dedicated to vocal learning. Neuron 73, 537552 (2012)." href="/articles/s41593-018-0281-3#ref-CR33" id="ref-link-section-d136720129e1184">33</a></sup>. Here, we tested whether genes in our nonzero modules showed overlap with DE genes from subcortical Area X of singing versus nonsinging songbirds. A reanalysis of data from Hilliard et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Hilliard, A. T., Miller, J. E., Fraley, E. R., Horvath, S. &amp; White, S. A. Molecular microcircuitry underlies functional specification in a basal ganglia circuit dedicated to vocal learning. Neuron 73, 537552 (2012)." href="/articles/s41593-018-0281-3#ref-CR33" id="ref-link-section-d136720129e1188">33</a></sup> identified 1,267 DE genes in Area X, among which 902 overlapped with the genes examined in the main PLS analysis. Area X is thought to be homologous with human striatal areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Pfenning, A. R. et al. Convergent transcriptional specializations in the brains of humans and song-learning birds. Science 346, 1256846 (2014)." href="/articles/s41593-018-0281-3#ref-CR26" id="ref-link-section-d136720129e1192">26</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig1">1a</a>). Strikingly, 33% of the DE genes in Area X were present in our nonzero modules (OR=1.77, <i>P</i>=0.002). In contrast, no such enrichment was present in zero modules (OR=1.37, <i>P</i>=0.13) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). Most of the nonzero enrichment was driven specifically by module M10, because no other nonzero modules were specifically enriched in songbird DE genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5b</a>). These results suggest that a subset of genes in nonzero modules are indeed brain relevant and conserved between humans and songbirds, both of which have vocal learning ability.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Vocal learning, human-specific, and ASD-associated enrichments with sets of broadly expressed genes and nonzero or zero association modules."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Vocal learning, human-specific, and ASD-associated enrichments with sets of broadly expressed genes and nonzero or zero association modules.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="325"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Results of hypergeometric tests for enrichment among broadly expressed genes, nonzero and zero modules (columns), and a variety of different lists (rows) of genes relevant to vocal learning, human-specific genes, or genes relevant to ASD. The numbers in each cell represent the enrichment OR, and the coloring represents the log10 <i>P</i> value for each hypergeometric test for enrichment. Details about the gene lists specified in each row are provided in Methods. Cells outlined in green passed multiple-comparison correction at FDR <i>q</i>&lt;0.05. <b>b</b>, Table indicating which gene coexpression modules (rows) were enriched in a variety of different gene lists (columns). Modules with enrichments passing FDR <i>q</i>&lt;0.05 for multiple-comparison correction are indicated in cyan. The first three columns show which modules had nonzero associations (colored cells), as shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig3">3b</a>. Upreg, upregulated; downreg, downregulated; CTX, cortex; ModPres, module preservation.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-018-0281-3/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec10">Nonzero modules are enriched in transcriptionally human-specific genes</h3><p>Language requires more than vocal learning and is indeed a uniquely human ability. Therefore, components of language ability may possibly be reflected in neural differences between humans and our closest nonhuman primate relatives (e.g., chimpanzees), which do not possess language<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Konopka, G. et al. Human-specific transcriptional networks in the brain. Neuron 75, 601617 (2012)." href="/articles/s41593-018-0281-3#ref-CR34" id="ref-link-section-d136720129e1257">34</a></sup>. We therefore investigated whether nonzero modules were enriched in DE genes in the cortical tissue of humans versus chimpanzees (human-specific genes). Using two lists of such human-specific genes that were obtained from independent studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Konopka, G. et al. Human-specific transcriptional networks in the brain. Neuron 75, 601617 (2012)." href="/articles/s41593-018-0281-3#ref-CR34" id="ref-link-section-d136720129e1261">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Liu, X. et al. Disruption of an evolutionarily novel synaptic expression pattern in autism. PLoS Biol. 14, e1002558 (2016)." href="/articles/s41593-018-0281-3#ref-CR35" id="ref-link-section-d136720129e1264">35</a></sup> and minimally overlapped (4.38%), we found that nonzero modules were significantly enriched in both lists, and 3334% of human-specific genes overlapped with genes from nonzero modules (OR&gt;1.73, <i>P</i>&lt;0.0115) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). These enrichments were driven by M13, because no other specific modules were enriched across both human-specific gene lists (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5b</a>). In contrast, no enrichment of human-specific genes was present in zero modules (OR&lt;1.11, <i>P</i>&gt;0.85; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). These results suggest that the transcriptional activity of human-specific genes in blood leukocytes is linked to the language-relevant fMRI phenotypes measured in TD and toddlers with ASD with varying early language abilities.</p><h3 class="c-article__sub-heading" id="Sec11">Nonzero modules are enriched in highly active prenatal coexpression modules associated with ASD</h3><p>Several lines of evidence suggest that ASD pathophysiology can manifest during prenatal brain development<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0056-y&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR13" id="ref-link-section-d136720129e1292">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Eising, E. et al. A set of regulatory genes co-expressed in embryonic human brain is implicated in disrupted speech development. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0020-x&#xA;                    &#xA;                   (2018)." href="#ref-CR36" id="ref-link-section-d136720129e1295">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 10081021 (2013)." href="#ref-CR37" id="ref-link-section-d136720129e1295_1">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Willsey, A. J. et al. Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism. Cell 155, 9971007 (2013)." href="/articles/s41593-018-0281-3#ref-CR38" id="ref-link-section-d136720129e1298">38</a></sup>. We therefore examined whether nonzero modules might be enriched in genes that were members of coexpression modules that showed high levels of prenatal expression and that possessed a number of highly penetrant ASD-associated genes. Using lists from two independent studies of the BrainSpan atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Miller, J. A. et al. Transcriptional landscape of the prenatal human brain. Nature 508, 199206 (2014)." href="/articles/s41593-018-0281-3#ref-CR39" id="ref-link-section-d136720129e1302">39</a></sup> examining either cortical-only<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 10081021 (2013)." href="/articles/s41593-018-0281-3#ref-CR37" id="ref-link-section-d136720129e1306">37</a></sup> or cortical and subcortical regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Eising, E. et al. A set of regulatory genes co-expressed in embryonic human brain is implicated in disrupted speech development. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0020-x&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR36" id="ref-link-section-d136720129e1310">36</a></sup>, we found that approximately 32% of the genes in prenatal and ASD-associated coexpression modules also appeared in nonzero modules (OR&gt;1.7, <i>P</i>&lt;0.0056) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>), whereas only 1517% were present in zero modules (OR&lt;1.19, <i>P</i>&gt;0.74). Nonzero modules M15 and M10 drove the enrichment, because no other nonzero modules showed evidence of enrichment in genes in either ASD-associated prenatal-gene list (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5b</a>). Overall, this evidence supports the idea that some of the genes present in nonzero modules may also be members of prenatally active and ASD-associated coexpression modules.</p><h3 class="c-article__sub-heading" id="Sec12">Nonzero modules are enriched in genes from ASD-downregulated coexpression modules from frontal and temporal cortex tissue</h3><p>Although we established that nonzero modules overlapped with prenatally relevant coexpression modules bearing ASD-relevant genes, a caveat to this result is that those prenatal, ASD-associated coexpression modules were identified from the BrainSpan dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Miller, J. A. et al. Transcriptional landscape of the prenatal human brain. Nature 508, 199206 (2014)." href="/articles/s41593-018-0281-3#ref-CR39" id="ref-link-section-d136720129e1335">39</a></sup>, which for obvious reasons does not contain prenatal tissue from donors with ASD. Thus, to more directly connect nonzero modules with cortical gene expression in patients diagnosed with ASD, we used gene expression data from the postmortem frontal and temporal cortical tissue of patients with ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Parikshak, N. N. et al. Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism. Nature 540, 423427 (2016)." href="/articles/s41593-018-0281-3#ref-CR40" id="ref-link-section-d136720129e1339">40</a></sup>. Nonzero modules were enriched in genes that were members of ASD-downregulated frontal and temporal cortex coexpression modules (OR=1.70, <i>P</i>=0.03). Enrichments at trend levels were also seen for genes from ASD-upregulated coexpression modules (OR=1.64, <i>P</i>=0.0502, FDR <i>q</i>=0.0586) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). However, no specific nonzero modules seemed to drive this enrichment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5b</a>). Although zero modules were not enriched in genes from ASD-downregulated modules (OR=1.15, <i>P</i>=0.75), zero modules were enriched in genes from ASD-upregulated modules (OR=1.79, <i>P</i>=2.8010<sup>5</sup>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). These results further suggest the ASD and brain relevance of genes identified via their nonzero association between expression in blood leukocytes and language-relevant functional neuroimaging phenotypes.</p><h3 class="c-article__sub-heading" id="Sec13">Nonzero modules with preservation of network structure between ASD blood and cortical tissue</h3><p>Using the same gene expression dataset from postmortem cortical tissue from patients with ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Parikshak, N. N. et al. Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism. Nature 540, 423427 (2016)." href="/articles/s41593-018-0281-3#ref-CR40" id="ref-link-section-d136720129e1379">40</a></sup>, we next examined whether the coexpression network structure of nonzero modules identified in blood might be preserved in ASD frontal and temporal cortical tissue. This step is important because it highlights specific modules whose coexpression network connectivity patterns are similar between blood leukocytes and brain tissue. Whereas nonzero modules M8 and M11 showed moderate evidence of preservation (2&lt;Zsummary&lt;6), the nonzero M2 module had the highest ranking of all modules with evidence of high-to-moderate preservation (Zsummary=8.1) (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM1">1</a>). M2 is highly enriched in the term translation in mitochondria (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM7">5</a>), and many M2 hub genes (for example, <i>MRPS12</i>, <i>NDUFS3</i>, <i>NDUFB8</i>, <i>HINT2</i>, and <i>MRPL14</i>) encode proteins that localize to mitochondria (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM8">6</a>). This evidence may be relevant in light of possible mitochondrial dysfunction in ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Rossignol, D. A. &amp; Frye, R. E. Mitochondrial dysfunction in autism spectrum disorders: a systematic review and meta-analysis. Mol. Psychiatry 17, 290314 (2012)." href="/articles/s41593-018-0281-3#ref-CR41" id="ref-link-section-d136720129e1408">41</a></sup>. Other notable M2 hub genes were <i>DGCR6</i> and <i>BOLA2</i>. Both are located within prominent ASD-associated copy number variant (CNV) regions of 22q11.21 (<i>DGCR6</i>) and 16p11.2 (<i>BOLA2</i>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Sanders, S. J. et al. Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci. Neuron 87, 12151233 (2015)." href="/articles/s41593-018-0281-3#ref-CR42" id="ref-link-section-d136720129e1425">42</a></sup>. Interestingly with regard to evolutionarily accelerated human-specific genes, <i>BOLA2</i> is known for human-specific duplications and shows upregulated expression in human versus chimpanzee induced pluripotent stem cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Marchetto, M. C. N. et al. Differential L1 regulation in pluripotent stem cells of humans and apes. Nature 503, 525529 (2013)." href="/articles/s41593-018-0281-3#ref-CR43" id="ref-link-section-d136720129e1432">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Nuttle, X. et al. Emergence of a Homo sapiensspecific gene family and chromosome 16p11.2 CNV susceptibility. Nature 536, 205209 (2016)." href="/articles/s41593-018-0281-3#ref-CR44" id="ref-link-section-d136720129e1435">44</a></sup>. In patients with 16p11.2 CNVs, 96% of breakpoints include human-specific duplications of <i>BOLA2</i> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Nuttle, X. et al. Emergence of a Homo sapiensspecific gene family and chromosome 16p11.2 CNV susceptibility. Nature 536, 205209 (2016)." href="/articles/s41593-018-0281-3#ref-CR44" id="ref-link-section-d136720129e1443">44</a></sup>). Deletions and duplications of 16p11.2 are linked to language and its associated neural circuitry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hippolyte, L. et al. The number of genomic copies at the 16p11.2 locus modulates language, verbal memory, and inhibition. Biol. Psychiatry 80, 129139 (2016)." href="#ref-CR45" id="ref-link-section-d136720129e1447">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Demopoulos, C. et al. Abnormal speech motor control in individuals with 16p11.2 deletions. Sci. Rep. 8, 1274 (2018)." href="#ref-CR46" id="ref-link-section-d136720129e1447_1">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Berman, J. I. et al. Abnormal auditory and language pathways in children with 16p11.2 deletion. Neuroimage Clin. 9, 5057 (2015)." href="/articles/s41593-018-0281-3#ref-CR47" id="ref-link-section-d136720129e1450">47</a></sup>. Thus, the evidence may suggest that <i>BOLA2</i> is an important ASD-relevant 16p11.2 locus but also is more generally relevant for the human-specific ability to develop language and the neural systems supporting that development.</p><h3 class="c-article__sub-heading" id="Sec14">Nonzero modules are enriched in ASD de novo protein-truncating variants</h3><p>We next tested nonzero modules for enrichment in different classes of genetic variants associated with ASD. We first examined enrichment in high-penetrance rare de novo protein truncating variants (dnPTVs). Among the genes highlighted by Kosmicki et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Kosmicki, J. A. et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. Nat. Genet. 49, 504510 (2017)." href="/articles/s41593-018-0281-3#ref-CR48" id="ref-link-section-d136720129e1465">48</a></sup> with two or more dnPTVs in ASD, 43% were also present in nonzero modules, thus resulting in an enrichment at trend-level significance (OR=2.58, <i>P</i>=0.08, FDR <i>q</i>=0.0915). The lack of significant enrichment may have been due to the limited number of known dnPTVs that overlap with the subset of genes considered in our analysis (i.e., 28). When we relaxed the criterion to one or more dnPTVs in ASD but added the constraint that the gene had to have a probability of loss-of-function intolerance (pLI)0.9 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285291 (2016)." href="/articles/s41593-018-0281-3#ref-CR49" id="ref-link-section-d136720129e1475">49</a></sup>), we were able to study a larger set of 155 genes with putative ASD-relevant dnPTVs. Under this criterion, we found a significant enrichment of these ASD-risk genes in nonzero modules (OR=2.01, <i>P</i>=0.02) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>), including genes such as <i>ADNP</i>, <i>ANKRD11</i>, <i>DYRK1A</i>, <i>ILF2</i>, <i>KDM5B</i>, <i>KDM6B</i>, <i>MED13L</i>, <i>PHF2</i>, <i>PTEN</i>, <i>SPAST</i>, <i>SUV420H1</i>, <i>TRIP12</i>, <i>WDFY3</i>, and <i>ZC3H4</i>. Nonzero module M10 is the primary driver of this enrichment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5b</a>) and includes <i>ADNP</i>, <i>ANKRD11</i>, <i>DYRK1A</i>, <i>KDM5B</i>, <i>TRIP12</i>, and <i>ZC3H4</i>. Of these notable M10 genes, <i>ADNP</i> is within the top 20 hub genes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM8">6</a>). In contrast, zero modules were not enriched in these ASD-risk genes, either among the criteria of two or more dnPTVs or one or more dnPTVs and pLI 0.9 (OR&lt;1.82, <i>P</i>&gt;0.24) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). In addition, and contrary to the enrichment in ASD-associated dnPTVs, we did not find any enrichment among the 543 ASD-associated genes annotated in SFARI Gene (<a href="https://gene.sfari.org/">https://gene.sfari.org/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). Mol. Autism 4, 36 (2013)." href="/articles/s41593-018-0281-3#ref-CR50" id="ref-link-section-d136720129e1572">50</a></sup> for nonzero (OR=1.36, <i>P</i>=0.66) or zero modules (OR=1.26, <i>P</i>=0.44) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). This evidence suggests that some high-penetrance ASD-associated genes are detectable within blood leukocyte gene expression data and show strong association with in vivo functional neuroimaging phenotypes relevant to early language heterogeneity in ASD.</p><h3 class="c-article__sub-heading" id="Sec15">Nonzero modules are enriched in targets of FMRP and CHD8</h3><p>Although nonzero modules do not contain some of the most well-known and highly penetrant ASD-associated genes, such as <i>FMR1</i> and <i>CHD8</i>, these modules may nevertheless overlap with the molecular networks linked to these genes. One way to examine this hypothesis is through testing nonzero modules for enrichment in downstream targets of these highly important genes. Nonzero modules were highly enriched in targets of both FMRP and CHD8 across two different target lists (OR&gt;1.89, <i>P</i>&lt;0.0269) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). Numerous modules drive these enrichments, such as M10 and M15 for FMRP targets and M10, M8, M13, and M15 for CHD8 targets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5b</a>). In contrast, zero modules were not enriched in target genes of either FMRP or CHD8 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). These results suggest that nonzero modules also contain genes that are members of FMRP- and CHD8-related networks.</p><h3 class="c-article__sub-heading" id="Sec16">Broadly expressed genes are a prominent source of signal driving enrichments</h3><p>Finally, given the prominent overlap between broadly expressed genes and nonzero modules, we tested whether many of the other enrichments with nonzero modules might be driven by broadly expressed genes. We first examined the enrichment of broadly expressed genes with all of the gene lists already tested. Notably, we found that nearly all gene lists enriched in nonzero modules were also highly enriched in broadly expressed genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig5">5a</a>). Furthermore, after broadly expressed genes were removed from these lists, the enrichments with nonzero modules largely disappeared (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM1">2</a>). This result suggests that broadly expressed genes drive the enrichments of these lists in nonzero modules.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec17-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec17">Discussion</h2><div class="c-article-section__content" id="Sec17-content"><p>Here, we found one large-scale association between coordinated gene coexpression modules in blood leukocytes and multivariate fMRI response to speech. Highlighting the distinctiveness of ASD language outcome subtypes, we found that blood leukocyte coexpression modules associated with the multivariate fMRI response to speech were different for all pairwise comparisons between groups of TD toddlers and toddlers with ASD plus either poor or good language outcome. Given the early ages at which the blood samples and fMRI data were collected, this association clearly manifests well before stable diagnoses and final language outcomes are known. Coexpression modules of importance in TD but not ASD may signal normative biological processes associated with the development of language-related neural circuitry. These normative processes may be affected in ASD. In addition, modules that diverge among ASD subtypes may indicate risk or protective mechanisms that push different individuals with ASD toward different early developmental language outcomes. Thus, in contrast to the idea that ASD is a uniform condition with similar underlying biological mechanisms in all diagnosed individuals, these results indicate that a behavioral stratifier such as early language outcome holds important information to aid in understanding how the underlying biology may be differentially linked to the way in which macroscale neural systems develop.</p><p>These findings may have high translational importance. Both neuroimaging methods and blood sampling to quantify the leukocyte transcriptome with high-throughput techniques are feasible for patients with ASD with different levels of impairment and at early ages. In vivo examination of the molecular mechanisms and their associations with higher-level macroscale neural systems and heterogeneity in clinical phenotypes will be important for furthering progress toward precision medicine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0056-y&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR13" id="ref-link-section-d136720129e1639">13</a></sup>. Endeavors such as evaluating early age treatment response, monitoring clinical trials, and developing prediction tools for diagnosis and prognosis can all be facilitated with this approach to understanding the links among gene expression, macroscale neural systems, and behavioral levels of analysis. Future work will be necessary to determine whether similar associations are present in older children and adults with ASD. Given the inability to directly and noninvasively assay gene expression from brain tissue in living patients, the current approach offers a novel in vivo window into how molecular mechanisms are associated with ongoing and dynamic macroscale neural systems development across the lifespan in ASD.</p><p>Another striking feature of these results is the large-scale nature of the association that covers most of the blood leukocyte transcriptome considered by the coexpression analysis. This feature matches predictions from the omnigenic model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Cell 169, 11771186 (2017)." href="/articles/s41593-018-0281-3#ref-CR20" id="ref-link-section-d136720129e1646">20</a></sup>. The omnigenic model suggests that for any complex trait or disorder, most of the heritability signal is spread widely throughout most of the genome. The omnigenic model also suggests that the numerous widespread peripheral genes with small effect probably interact within gene regulatory networks with a smaller set of core genes with much larger effect. Here, we found evidence that higher-effect rare dnPTVs in ASD that are intolerant to loss-of-function mutations are enriched among nonzero modules. Furthermore, we also found that many targets of FMRP and CHD8 were enriched in nonzero modules. Thus, the massive number of genes present within nonzero modules may indicate a large peripheral background of small-risk common variants that may work en masse and interact in important ways with higher-effect core mechanisms.</p><p>The omnigenic model makes another key prediction, namely that such associations may be detectable in many tissue types other than the brain, such as blood leukocytes. The omnigenic model suggests that a large percentage of the genes associated with a complex trait are likely to be broadly expressed genes. Here, we found evidence of large overlap between broadly expressed genes and nonzero modulesapproximately 44% of all broadly expressed genes were found within nonzero association modules. In contrast to the observation that nearly all nonzero modules (e.g., 81%) were enriched in broadly expressed genes, only two nonzero modules (e.g., 18%) were enriched in lymphocyte-specific genes. Thus, this large-scale gene coexpressionfMRI association is largely driven by genes broadly expressed in the brain and many other tissues rather than lymphocyte-specific genes. Although we observed enrichments among nonzero modules and genes implicated in vocal learning, human-specific genes, ASD-associated prenatal coexpression modules, cortically ASD-downregulated coexpression modules, ASD dnPTVs, and FMRP and CHD8 targets, most of these enrichments probably emerged because each gene list was heavily enriched in broadly expressed genes. Removing broadly expressed genes from these lists resulted in the elimination of nearly all significant enrichments with nonzero modules. Overall, these results highlight the importance of broadly expressed genes as a novel class of mechanisms for further study in ASD.</p><p>This study has some limitations and caveats to bear in mind. First, the genes investigated in the final coexpression and PLS analyses were a subset of all genes in the entire genome that could be considered. Therefore, although nonzero modules did cover a large proportion of the genes examined in the analysis, they did not cover most of the entire genome. The extent of coverage of nonzero modules was certainly compatible with ideas about polygenic architecture underlying complex neural phenotypes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Wray, N. R., Wijmenga, C., Sullivan, P. F., Yang, J. &amp; Visscher, P. M. Common disease is more complex than implied by the core gene omnigenic model. Cell 173, 15731580 (2018)." href="/articles/s41593-018-0281-3#ref-CR21" id="ref-link-section-d136720129e1657">21</a></sup>. However, the coverage of nonzero modules cannot be interpreted with respect to the omnigenic model in terms of sheer size. The current study did, however, evaluate predictions from the omnigenic model, particularly with respect to the importance of broadly expressed genes. However, this result can also be consistent with polygenic viewpoints, particularly if most of the polygenic associations reside within broadly expressed genes. Second, because the expression data were measured from a non-neural tissue, many brain-specific genes were not considered in the analyses. Thus, the current dataset cannot indicate the importance or lack thereof with regard to brain-specific genes, nor can it allow for comparisons of the relative importance of broadly expressed genes versus brain-specific genes.</p><p>In summary, we identified a large-scale association between multiple coordinated blood leukocyte gene coexpression modules and the multivariate fMRI response to speech. The associated coexpression modules were different for all pairwise comparisons between TD toddlers and toddlers with ASD plus good versus poor early language outcome. The associated coexpression modules were highly enriched in broadly expressed genes as well as ASD, prenatal, human-specific, and language-relevant genes. These results are congruent with predictions from polygenic and omnigenic models and suggest that gene expression in peripheral cells such as blood leukocytes are associated with the in vivo functional neural response to language that differentiates toddlers with ASD and poor versus good early language outcomes. The study showcases a novel in vivo approach that may be used in future work toward precision medicine goals.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec18-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec18">Methods</h2><div class="c-article-section__content" id="Sec18-content"><h3 class="c-article__sub-heading" id="Sec19">Participants</h3><p>This study was approved by the Institutional Review Board at the University of California, San Diego. Parents provided written informed consent according to the Declaration of Helsinki and were paid for their participation. Identically to the approach used in our earlier studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pramparo, T. et al. Cell cycle networks link gene expression dysregulation, mutation, and brain maldevelopment in autistic toddlers. Mol. Syst. Biol. 11, 841 (2015)." href="/articles/s41593-018-0281-3#ref-CR14" id="ref-link-section-d136720129e1676">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Pramparo, T. et al. Prediction of autism by translation and immune/inflammation coexpressed genes in toddlers from pediatric community practices. JAMA Psychiatry 72, 386394 (2015)." href="/articles/s41593-018-0281-3#ref-CR15" id="ref-link-section-d136720129e1679">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. Neuron 86, 567577 (2015)." href="/articles/s41593-018-0281-3#ref-CR19" id="ref-link-section-d136720129e1682">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Pierce, K., Conant, D., Hazin, R., Stoner, R. &amp; Desmond, J. Preference for geometric patterns early in life as a risk factor for autism. Arch. Gen. Psychiatry 68, 101109 (2011)." href="/articles/s41593-018-0281-3#ref-CR51" id="ref-link-section-d136720129e1685">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Pierce, K. et al. Eye tracking reveals abnormal visual preference for geometric images as an early biomarker of an autism spectrum disorder subtype associated with increased symptom severity. Biol. Psychiatry 79, 657666 (2016)." href="/articles/s41593-018-0281-3#ref-CR52" id="ref-link-section-d136720129e1688">52</a></sup>, toddlers were recruited through two mechanisms: community referrals (for example, website) or a general population-based screening method called the 1-Year Well-Baby Check-Up Approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Pierce, K. et al. Detecting, studying, and treating autism early: the One-Year Well-Baby Check-Up Approach. J. Pediatr. 159, 458465.e6 (2011)." href="/articles/s41593-018-0281-3#ref-CR53" id="ref-link-section-d136720129e1692">53</a></sup> (now called the Get S.E.T. Early model) that allowed for the prospective study of ASD beginning as young as ages 12 to 24 months, on the basis of a toddlers failure of the CSBS-DP Infant-Toddler Checklist<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Wetherby, A. &amp; Prizant, B. Communication and Symbolic Behavior Scales Developmental Profile, First Normed Edition (Paul H. Brookes, Baltimore, 2002)." href="/articles/s41593-018-0281-3#ref-CR54" id="ref-link-section-d136720129e1696">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Wetherby, A. M., Brosnan-Maddox, S., Peace, V. &amp; Newton, L. Validation of the Infant-Toddler Checklist as a broadband screener for autism spectrum disorders from 9 to 24 months of age. Autism 12, 487511 (2008)." href="/articles/s41593-018-0281-3#ref-CR55" id="ref-link-section-d136720129e1699">55</a></sup>. All toddlers, including control subjects, participated in a series of tests collected longitudinally across all visits, including the ADOS (Module T, 1, or 2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Lord, C. et al. The autism diagnostic observation schedule-generic: a standard measure of social and communication deficits associated with the spectrum of autism. J. Autism. Dev. Disord. 30, 205223 (2000)." href="/articles/s41593-018-0281-3#ref-CR56" id="ref-link-section-d136720129e1703">56</a></sup>, Mullen Scales of Early Learning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Mullen, E. M. Mullen Scales of Early Learning (American Guidance Service, Circle Pine, MN, USA, 1995)." href="/articles/s41593-018-0281-3#ref-CR57" id="ref-link-section-d136720129e1707">57</a></sup>, and Vineland Adaptive Behavior Scales<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Sparrow, S., Cicchetti, D. &amp; Balla, D. Vineland-II Scales Of Adaptive Behavior: Survey Form Manual (American Guidance Service, Circle Pines, MN, USA, 2005)." href="/articles/s41593-018-0281-3#ref-CR58" id="ref-link-section-d136720129e1712">58</a></sup>. All testing occurred at the University of California, San Diego Autism Center of Excellence (ACE). Toddlers who received their first diagnostic evaluation at younger than 36 months of age were invited to return for repeat diagnostic testing once every 12 months until age 34 years, when a final diagnosis was given. No randomization procedures were implemented as part of the data-collection process. Data collection and analyses were not performed blind to the conditions of the experiment.</p><p>A total of <i>n</i>=118 toddlers were scanned with fMRI and had available gene expression data. No statistical methods were used to predetermine sample sizes, but our sample sizes are currently among the largest of any fMRI study to date on ASD at very early ages in toddlerhood. From these 118 toddlers, <i>n</i>=81 individuals with ASD were examined and were split into two language-outcome subtypes. <i>n</i>=41 individuals with ASD (34 males and 7 females) were classified as having poor language outcome (ASD Poor), on the basis of the criteria of having both Mullen EL and RL <i>T</i> scores more than 1s.d. below the norm of 50 (that is, <i>T</i>&lt;40) at the final testing time point (mean age at fMRI scan=29.53 months, s.d. at fMRI scan=8.04, range=1246 months). Another <i>n</i>=40 individuals with ASD (30 males and 10 females) were classified as having good language outcome (ASD Good), on the basis of having either Mullen EL or RL <i>T</i> scores greater than or equal to 40 (that is, <i>T</i>  40) at the final testing time point (mean age at fMRI scan=29.73 months, s.d. at fMRI scan=8.51, range=1245 months). The usage of the term Good here is not used to refer to ability level in absolute terms but instead reflects ability relative to the ASD Poor subgroup. These ASD subtypes were compared to <i>n</i>=37 typically developing toddlers (21 males and 16 females; mean age at fMRI scan=26.19 months, s.d. at fMRI scan=10.20, range=1245 months). ASD subtypes and TD did not statistically differ in age at the time of scanning (<i>F</i>(2,115)=1.87, <i>P</i>=0.15). Additional demographic and phenotypic information can be found in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM9">7</a>.</p><h3 class="c-article__sub-heading" id="Sec20">Blood sample collection, RNA extraction, quality control, and sample preparation</h3><p>Four to six milliliters of blood was collected into EDTA-coated tubes from toddlers on visits when they had no fever, cold, flu, infections, or other illnesses, or use of medications for illnesses 72h before the blood draw. Blood samples were passed over a LeukoLOCK filter (Ambion) to capture and stabilize leukocytes and immediately placed in a 20C freezer. Total RNA was extracted according to standard procedures and manufacturers instructions (Ambion). LeukoLOCK disks (Ambion cat no. 1933) were freed from RNAlater, and Tri-reagent (Ambion cat no. 9738) was used to flush out the captured lymphocytes and lyse the cells. RNA was subsequently precipitated with ethanol and purified though washing and cartridge-based steps. The quality of the mRNA samples was quantified according to the RNA Integrity Number (RIN): values of 7.0 or greater were considered acceptable<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Schroeder, A. et al. The RIN: an RNA integrity number for assigning integrity values to RNA measurements. BMC. Mol. Biol. 7, 3 (2006)." href="/articles/s41593-018-0281-3#ref-CR59" id="ref-link-section-d136720129e1765">59</a></sup>, and all processed RNA samples passed RIN quality control. Quantification of RNA was performed with a NanoDrop spectrophotometer (Thermo Scientific). Samples were prepared in 96-well plates at a concentration of 25ng/l.</p><h3 class="c-article__sub-heading" id="Sec21">Gene expression and data processing</h3><p>RNA was assayed at Scripps Genomic Medicine for labeling, hybridization, and scanning with the Illumina BeadChip pipeline (Illumina) according to the manufacturers instructions. All arrays were scanned with the Illumina BeadArray Reader and read into Illumina GenomeStudio software (version 1.1.1). Raw data were exported from Illumina GenomeStudio, and data preprocessing was performed with the <i>lumi</i> package<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Du, P., Kibbe, W. A. &amp; Lin, S. M. lumi: a pipeline for processing Illumina microarray. Bioinformatics 24, 15471548 (2008)." href="/articles/s41593-018-0281-3#ref-CR60" id="ref-link-section-d136720129e1780">60</a></sup> for R (<a href="http://www.R-project.org/">http://www.R-project.org/</a>) and Bioconductor (<a href="http://www.bioconductor.org/">http://www.bioconductor.org/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Gentleman, R. C. et al. Bioconductor: open software development for computational biology and bioinformatics. Genome. Biol. 5, R80 (2004)." href="/articles/s41593-018-0281-3#ref-CR61" id="ref-link-section-d136720129e1798">61</a></sup>. Raw and normalized data are part of larger sets deposited in the Gene Expression Omnibus database (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE42133">GSE42133</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111175">GSE111175</a>).</p><p>A larger primary dataset of blood leukocyte gene expression was available from 383 samples from 314 toddlers 14 years old. The samples were assayed with the Illumina microarray platform in three batches. The datasets were combined by matching the Illumina Probe ID and probe nucleotide sequences. The final set included a total of 20,194 gene probes. Quality-control analysis was performed to identify and remove 23 outlier samples from the dataset. Samples were marked as outliers if they showed low signal intensity (average signal two s.d. lower than the overall mean), deviant pairwise correlations, deviant cumulative distributions, deviant multidimensional scaling plots, or poor hierarchical clustering, as described elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Pramparo, T. et al. Cell cycle networks link gene expression dysregulation, mutation, and brain maldevelopment in autistic toddlers. Mol. Syst. Biol. 11, 841 (2015)." href="/articles/s41593-018-0281-3#ref-CR14" id="ref-link-section-d136720129e1820">14</a></sup>. The high-quality dataset included 360 samples from 299 toddlers. High reproducibility was observed across technical replicates (mean Spearman correlation of 0.97 and median of 0.98). Thus, we randomly removed one of each of two technical replicates from the primary dataset. From the subjects in the larger primary dataset, <i>n</i>=118 also had task fMRI data, and thus a total of <i>n</i>=105 from the Illumina HT12 platform along with <i>n</i>=13 from the Illumina WG6 platform were used in this study. The batch was not asymmetrically distributed across one subgroup more than another, because chi-square analyses on the contingency table between subgroup and batch showed no effect (<i></i><sup>2</sup>(4)=4.772, <i>P</i>=0.3115). ASD subtypes and TD toddlers also did not statistically differ in age at the time of blood sampling (<i>F</i>(2,115)=1.74, <i>P</i>=0.17). The 20,194 probes were then collapsed to 14,313 genes on the basis of picking the probe with maximal mean expression across samples. Data were quantile normalized and then adjusted for batch effects, sex, and RIN. The batch-, sex-, and RIN-adjusted data were used in all further downstream analyses. We also checked for differences in the proportion estimates of different leukocyte cell types (neutrophils, B cells, T cells, NK cells, and monocytes) with the CellCODE deconvolution method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Chikina, M., Zaslavsky, E. &amp; Sealfon, S. C. CellCODE: a robust latent variable approach to differential expression analysis for heterogeneous cell populations. Bioinformatics 31, 15841591 (2015)." href="/articles/s41593-018-0281-3#ref-CR62" id="ref-link-section-d136720129e1847">62</a></sup>, but we found no evidence of differences across groups for any cell type (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM10">8</a>). In addition to the primary analyses with WGCNA, differential expression analysis at the level of individual genes was also conducted with <i>limma</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47 (2015)." href="/articles/s41593-018-0281-3#ref-CR63" id="ref-link-section-d136720129e1857">63</a></sup>, and DE genes were identified if they passed Storey FDR <i>q</i>&lt;0.05 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Storey, J. D. A direct approach to false discovery rates. J. R. Statist. Soc. B 64, 479498 (2002)." href="/articles/s41593-018-0281-3#ref-CR64" id="ref-link-section-d136720129e1864">64</a></sup>). Data distributions were assumed to be normal, but normality was not formally tested for each gene.</p><h3 class="c-article__sub-heading" id="Sec22">Weighted gene coexpression-network analysis</h3><p>We decreased the number of features in the gene expression dataset from 14,313 genes down to 21 modules of tightly coexpressed genes. This data-reduction step was achieved with WGCNA, implemented within the <i>WGCNA</i> library in R<sup>23</sup>. Correlation matrices estimated with the robust correlation measure of biweight midcorrelation were computed and then converted into adjacency matrices that retained the sign of the correlation. These adjacency matrices were then raised to a soft power of 16 (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM1">3a</a>). This soft power was chosen by finding the first soft power for which a measure of <i>R</i><sup>2</sup> scale-free topology-model fit saturated at least above <i>R</i><sup>2</sup>&gt;0.8 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. Stat. Appl. Genet. Mol. Biol. 4, Article17 (2005)." href="/articles/s41593-018-0281-3#ref-CR65" id="ref-link-section-d136720129e1893">65</a></sup>), and the slope was between 1 and 2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Oldham, M. C., Horvath, S. &amp; Geschwind, D. H. Conservation and evolution of gene coexpression networks in human and chimpanzee brains. Proc. Natl Acad. Sci. USA 103, 1797317978 (2006)." href="/articles/s41593-018-0281-3#ref-CR66" id="ref-link-section-d136720129e1897">66</a></sup>). The soft-power-thresholded adjacency matrix was then converted into a topological overlap matrix (TOM) and then a TOM dissimilarity matrix (1  TOM). The TOM dissimilarity matrix was then input into agglomerative hierarchical clustering with the average linkage method. Gene modules were defined from the resulting clustering tree, and branches were cut with a hybrid dynamic tree-cutting algorithm (deepSplit parameter=4) (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM1">3b</a>). Modules were merged at a cut height of 0.2, and the minimum module size was set to 100. Only genes with a module membership of <i>r</i>&gt;0.3 were retained within modules. For each gene module, a summary measure called the ME was computed as the first principal component of the scaled (standardized) module expression profiles. We also computed module membership for each gene and module. Module membership indicated the correlation between each gene and the ME (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM8">6</a>). Genes that could not be clustered into any specific module were left within the M0 module, and this module was not considered in any further analyses. Analysis of group differences in MEs were also conducted with linear models and correction for multiple comparisons at FDR <i>q</i>&lt;0.05 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM5">3</a>; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM1">4</a>). Data distributions were assumed to be normal, but normality was not formally tested for each module. Further WGCNA analyses were run separately within each group to check for preservation of detected modules across groups. These analyses all indicated high levels of preservation (Zsummary&gt;10)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Langfelder, P., Luo, R., Oldham, M. C. &amp; Horvath, S. Is my network module preserved and reproducible? PLoS. Comput. Biol. 7, e1001057 (2011)." href="/articles/s41593-018-0281-3#ref-CR67" id="ref-link-section-d136720129e1920">67</a></sup> across nearly all detected modules for each pairwise group comparison (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM1">5</a>).</p><h3 class="c-article__sub-heading" id="Sec23">fMRI data acquisition and task design</h3><p>The fMRI task was identical to that used in our previously published studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. Neuron 86, 567577 (2015)." href="/articles/s41593-018-0281-3#ref-CR19" id="ref-link-section-d136720129e1936">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Eyler, L. T., Pierce, K. &amp; Courchesne, E. A failure of left temporal cortex to specialize for language is an early emerging and fundamental property of autism. Brain 135, 949960 (2012)." href="#ref-CR68" id="ref-link-section-d136720129e1939">68</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Redcay, E. &amp; Courchesne, E. Deviant functional magnetic resonance imaging patterns of brain activity to speech in 2-3-year-old children with autism spectrum disorder. Biol. Psychiatry 64, 589598 (2008)." href="#ref-CR69" id="ref-link-section-d136720129e1939_1">69</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Redcay, E., Haist, F. &amp; Courchesne, E. Functional neuroimaging of speech perception during a pivotal period in language acquisition. Dev. Sci. 11, 237252 (2008)." href="/articles/s41593-018-0281-3#ref-CR70" id="ref-link-section-d136720129e1942">70</a></sup> and consisted of three types of speech stimuli (complex forward speech, simple forward speech, and backward speech) as well as rest blocks interspersed between task blocks to forestall possible habituation across blocks. Blocks were 20s in duration. All speech conditions were created with the same female speaker. One contrast of interest was analyzed in this study: all speech conditions versus rest. At early language learning ages, when neonates, infants, and toddlers are not yet experts at language, forward and backward speech both activate language-relevant temporal areas; consequently, specific comparisons between them tend to be nonsignificant<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Redcay, E., Haist, F. &amp; Courchesne, E. Functional neuroimaging of speech perception during a pivotal period in language acquisition. Dev. Sci. 11, 237252 (2008)." href="/articles/s41593-018-0281-3#ref-CR70" id="ref-link-section-d136720129e1946">70</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Dehaene-Lambertz, G., Dehaene, S. &amp; Hertz-Pannier, L. Functional neuroimaging of speech perception in infants. Science 298, 20132015 (2002)." href="/articles/s41593-018-0281-3#ref-CR71" id="ref-link-section-d136720129e1949">71</a></sup>. Therefore, forward and backward speech stimuli both appear to be effective in stimulating language-sensitive cortices, by perhaps both being treated as potentially language relevant by the language-inexperienced infant and toddler brain. Thus, because of this age-related caveat for forward versus backward speech, our main contrast of interest was all speech versus rest.</p><p>Imaging data were collected on a 1.5 Tesla General Electric MRI scanner during natural sleep at night; no sedation was used. High-resolution T1-weighted anatomical scans were collected for warping fMRI data into standard atlas space. Blood oxygenation leveldependent (BOLD) signal was measured across the whole brain with echoplanar imaging during the language paradigm (echo time=30ms, repetition time=2,500ms, flip angle=90, bandwidth=70kHz, field of view=25.6cm, in-plane resolution=44mm, slice thickness=4mm, 31 slices).</p><p>Analysis of head motion via framewise displacement (FD) and DVARS indicated that the head motion was minimal (mean FD&lt;0.25) for nearly all subjects in all groups (ASD Good, mean=0.11mm, s.d.=0.23; ASD Poor, mean=0.07mm, s.d.=0.08; TD, mean=0.07mm, s.d.=0.03) and that groups did not differ in either mean FD (<i>F</i>(2,115)=1.12, <i>P</i>=0.33) or mean DVARS (<i>F</i>(2,115)=1.93, <i>P</i>=0.15; ASD Good, mean=8.81, s.d.=2.85; ASD Poor, mean=8.61, s.d.=2.57; TD, mean=7.75, s.d.=2.01).</p><h3 class="c-article__sub-heading" id="Sec24">fMRI data analyses</h3><p>Preprocessing of functional imaging data was implemented within the Analysis of Functional NeuroImages (AFNI) software package. The preprocessing pipeline comprised motion correction, normalization to Talairach space, and smoothing (Gaussian kernel with 8-mm full width at half maximum). First-level and second-level mass-univariate whole-brain activation analyses were modeled with the general linear model (GLM) in SPM8 (<a href="http://www.fil.ion.ucl.ac.uk/spm/">http://www.fil.ion.ucl.ac.uk/spm/</a>). Events in first-level models were modeled with the canonical hemodynamic response function and its temporal derivative. All first-level GLMs included motion parameters as covariates of no interest. High-pass temporal filtering was applied with a cutoff of 0.0078Hz (1/128s) to remove low-frequency drift in the time series. For whole-brain analyses, the distributions were assumed to be normal, but normality was not formally tested for every voxel.</p><p>Group-level analyses were implemented with the general linear model in SPM8. We ran whole-brain analyses for the contrast of all speech versus rest within and between-groups and thresholded at a voxelwise FDR <i>q</i>&lt;0.05 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Genovese, C. R., Lazar, N. A. &amp; Nichols, T. Thresholding of statistical maps in functional neuroimaging using the false discovery rate. Neuroimage 15, 870878 (2002)." href="/articles/s41593-018-0281-3#ref-CR72" id="ref-link-section-d136720129e1992">72</a></sup>). For between-group ROI analysis, we used meta-analytic ROIs from the Neurosynth term language<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. Nat. Methods 8, 665670 (2011)." href="/articles/s41593-018-0281-3#ref-CR73" id="ref-link-section-d136720129e1996">73</a></sup> of frontal and temporal cortex areas in both hemispheres, which were identical to those used in a prior paper<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. Neuron 86, 567577 (2015)." href="/articles/s41593-018-0281-3#ref-CR19" id="ref-link-section-d136720129e2000">19</a></sup>. We computed the difference in percentage signal change for all speech versus rest and used this difference as the dependent variable in a linear model testing subtype membership as the main independent variable of interest while covarying for sex. The data are plotted in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-018-0281-3#Fig2">2b</a> for each individual to show the distribution of the data. No group showed heavy deviations from normality, and all regions showed evidence of homogeneity of variance between groups. Follow-up tests for pairwise group comparisons used Welchs <i>t</i> test.</p><h3 class="c-article__sub-heading" id="Sec25">fMRIgene expression association analysis</h3><p>To assess multivariate fMRIgene expression relationships, we used PLS analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Krishnan, A., Williams, L. J., McIntosh, A. R. &amp; Abdi, H. Partial Least Squares (PLS) methods for neuroimaging: a tutorial and review. Neuroimage 56, 455475 (2011)." href="/articles/s41593-018-0281-3#ref-CR74" id="ref-link-section-d136720129e2019">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="McIntosh, A. R. &amp; Lobaugh, N. J. Partial least squares analysis of neuroimaging data: applications and advances. Neuroimage 23, S250S263 (2004)." href="/articles/s41593-018-0281-3#ref-CR75" id="ref-link-section-d136720129e2022">75</a></sup>. PLS is widely used in the neuroimaging literature, particularly to explain multivariate neural responses in terms of multivariate behavioral patterns of variation or a design matrix. Given that the current dataset was massively multivariate in terms of both fMRI and gene expression datasets, we used PLS to elucidate how variation in neural response to speech across large-scale neural systems covaried with gene expression, as measured by ME values of coexpression modules. PLS allows for identification of such relationships by finding latent fMRIgene expression variable pairs (LV) that maximally explained covariation in the dataset and were uncorrelated with other latent fMRIgene expression variable pairs. The strength of such covariation was denoted by the singular value (<i>d</i>) for each brainbehavior LV, and hypothesis tests were made via with permutation tests on the singular values. Furthermore, identifying brain regions that most strongly contribute to each LV pair was made via bootstrapping, whereby a bootstrap ratio was created for each voxel and represented the reliability of that voxel for contributing strongly to the LV pattern identified. The bootstrap ratio was roughly equivalent to a <i>Z</i> statistic and could be used to threshold data to find voxels that reliably contribute to an LV pair.</p><p>The PLS analyses reported here were implemented within the plsgui MATLAB toolbox (<a href="http://www.rotman-baycrest.on.ca/pls/">http://www.rotman-baycrest.on.ca/pls/</a>). Here we input first-level speech versus rest contrast images into the PLS. For gene expression data, we input ME values for all 21 coexpression modules. For statistical inference on identified fMRIgene expression LV pairs, a permutation test was run with 10,000 permutations. To identify reliably contributing voxels for fMRIgene expression LVs and to compute 95%CIs on fMRIgene expression correlations, bootstrapping was used with 10,000 resamples. To show voxels that most reliably contributed to significant fMRIgene expression LVs, we thresholded data for visualization at a BSR of 1.96 and 1.96. The strength of fMRIgene expression correlations for significant LVs was displayed as a bar graph with 95% bootstrap CIs as error bars. Gene coexpression modules in which the 95%CIs did not encompass 0 were denoted nonzero association modules. All other modules in which the 95%CIs included 0 were denoted zero modules.</p><p>From the PLS results, we tested whether the nonzero associations across modules were common across ASD subtypes or common across ASD subtypes and TD. To test for this question, we counted the overlap among nonzero association modules in each group and ran hypergeometric tests that explicitly tested for statistically significant overlap or commonality of nonzero associations across groups.</p><h3 class="c-article__sub-heading" id="Sec26">Enrichment tests</h3><p>Tests for functional (process-level) enrichment across all modules were implemented with the MetaCore GeneGO software platform. Further gene-set enrichment tests (hypergeometric tests and enrichment OR) were done on tissue-specific gene lists. First, we annotated each coexpression module by enrichment in four types of gene classes of relevance, as defined by GTEx data reported from Boyle et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Cell 169, 11771186 (2017)." href="/articles/s41593-018-0281-3#ref-CR20" id="ref-link-section-d136720129e2053">20</a></sup>. These classes were (i) broadly expressed genes, (ii) brain-specific genes, (iii) whole-blood-specific genes, and (iv) lymphocyte-specific genes. The background pool number for these hypergeometric tests was 14,313. Next, we tested whether nonzero modules were heavily enriched in modules from one or more of these gene classes. The background total for these tests was set to the total number of coexpression modules (e.g., 21).</p><p>Further enrichment tests were done across a wider range of gene lists of theoretical importance. Songbirds are often used as animal models relevant to the vocal-learning component of language<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Konopka, G. &amp; Roberts, T. F. Insights into the neural and genetic basis of vocal communication. Cell 164, 12691276 (2016)." href="/articles/s41593-018-0281-3#ref-CR25" id="ref-link-section-d136720129e2060">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Pfenning, A. R. et al. Convergent transcriptional specializations in the brains of humans and song-learning birds. Science 346, 1256846 (2014)." href="/articles/s41593-018-0281-3#ref-CR26" id="ref-link-section-d136720129e2063">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Hilliard, A. T., Miller, J. E., Fraley, E. R., Horvath, S. &amp; White, S. A. Molecular microcircuitry underlies functional specification in a basal ganglia circuit dedicated to vocal learning. Neuron 73, 537552 (2012)." href="/articles/s41593-018-0281-3#ref-CR33" id="ref-link-section-d136720129e2066">33</a></sup>. We investigated enrichments with differentially expressed genes taken from a microarray dataset of Area X in songbirds<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Hilliard, A. T., Miller, J. E., Fraley, E. R., Horvath, S. &amp; White, S. A. Molecular microcircuitry underlies functional specification in a basal ganglia circuit dedicated to vocal learning. Neuron 73, 537552 (2012)." href="/articles/s41593-018-0281-3#ref-CR33" id="ref-link-section-d136720129e2070">33</a></sup>. To identify DE genes between singing versus nonsinging birds, we reanalyzed this dataset (GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE34819">GSE34819</a>) with <i>limma</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. Nucleic Acids Res. 43, e47 (2015)." href="/articles/s41593-018-0281-3#ref-CR63" id="ref-link-section-d136720129e2083">63</a></sup>, and DE genes were identified if they passed Storey FDR <i>q</i>&lt;0.05 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Storey, J. D. A direct approach to false discovery rates. J. R. Statist. Soc. B 64, 479498 (2002)." href="/articles/s41593-018-0281-3#ref-CR64" id="ref-link-section-d136720129e2091">64</a></sup>). Given the uniquely human nature of language, we also tested hypotheses regarding enrichments among genes that were transcriptionally different in the cortical tissue between humans and chimpanzees (that is, human-specific genes). These tests were performed across gene lists from two independent investigations on human-specific gene expression differences, in which the common overlap among the two lists was small (4.38%)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Konopka, G. et al. Human-specific transcriptional networks in the brain. Neuron 75, 601617 (2012)." href="/articles/s41593-018-0281-3#ref-CR34" id="ref-link-section-d136720129e2095">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Liu, X. et al. Disruption of an evolutionarily novel synaptic expression pattern in autism. PLoS Biol. 14, e1002558 (2016)." href="/articles/s41593-018-0281-3#ref-CR35" id="ref-link-section-d136720129e2098">35</a></sup>. Ample evidence suggests that prenatal brain developmental periods are critical for ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0056-y&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR13" id="ref-link-section-d136720129e2102">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Eising, E. et al. A set of regulatory genes co-expressed in embryonic human brain is implicated in disrupted speech development. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0020-x&#xA;                    &#xA;                   (2018)." href="#ref-CR36" id="ref-link-section-d136720129e2105">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 10081021 (2013)." href="#ref-CR37" id="ref-link-section-d136720129e2105_1">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Willsey, A. J. et al. Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism. Cell 155, 9971007 (2013)." href="/articles/s41593-018-0281-3#ref-CR38" id="ref-link-section-d136720129e2108">38</a></sup>. To test for enrichment in prenatal ASD-associated coexpression modules, we used coexpression modules from two independent studies that analyzed the Allen Institute BrainSpan dataset:<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Miller, J. A. et al. Transcriptional landscape of the prenatal human brain. Nature 508, 199206 (2014)." href="/articles/s41593-018-0281-3#ref-CR39" id="ref-link-section-d136720129e2112">39</a></sup> (i) Eising et al., who analyzed data from both subcortical and cortical regions and identified modules M3, M9, and M12 as ASD associated and prenatally active<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Eising, E. et al. A set of regulatory genes co-expressed in embryonic human brain is implicated in disrupted speech development. Mol. Psychiatry &#xA;                    https://doi.org/10.1038/s41380-018-0020-x&#xA;                    &#xA;                   (2018)." href="/articles/s41593-018-0281-3#ref-CR36" id="ref-link-section-d136720129e2116">36</a></sup>, and (ii) Parikshak et al., who analyzed only cortical regions and identified M2 and M3 as ASD associated and prenatally active<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. Cell 155, 10081021 (2013)." href="/articles/s41593-018-0281-3#ref-CR37" id="ref-link-section-d136720129e2120">37</a></sup>. There is a 23% overlap between these two gene lists. We also tested for enrichments with gene lists known to be associated with ASD, either via genetic evidence or evidence from cortical transcriptomic dysregulation. In particular, we examined dnPTVs associated with ASD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Kosmicki, J. A. et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. Nat. Genet. 49, 504510 (2017)." href="/articles/s41593-018-0281-3#ref-CR48" id="ref-link-section-d136720129e2125">48</a></sup>, ASD-associated genes from SFARI Gene (<a href="https://gene.sfari.org/">https://gene.sfari.org/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). Mol. Autism 4, 36 (2013)." href="/articles/s41593-018-0281-3#ref-CR50" id="ref-link-section-d136720129e2136">50</a></sup>, and differentially expressed cortical coexpression modules measured from ASD postmortem frontal cortex and temporal cortex tissue<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Parikshak, N. N. et al. Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism. Nature 540, 423427 (2016)." href="/articles/s41593-018-0281-3#ref-CR40" id="ref-link-section-d136720129e2140">40</a></sup>. For ASD-associated dnPTVs, we used a list of 38 genes from Kosmicki et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Kosmicki, J. A. et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. Nat. Genet. 49, 504510 (2017)." href="/articles/s41593-018-0281-3#ref-CR48" id="ref-link-section-d136720129e2144">48</a></sup>, with two or more dnPTVs in ASD and that also showed zero dnPTVs in the normative ExAC database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285291 (2016)." href="/articles/s41593-018-0281-3#ref-CR49" id="ref-link-section-d136720129e2148">49</a></sup>. We additionally used more relaxed criteria of one or more dnPTVs in ASD and zero dnPTVs in ExAC combined with a pLI 0.9 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. Nature 536, 285291 (2016)." href="/articles/s41593-018-0281-3#ref-CR49" id="ref-link-section-d136720129e2153">49</a></sup>), thus resulting in 211 genes. Finally, we tested for enrichments with known downstream targets of highly penetrant mutations (in <i>FMR1</i> and <i>CHD8</i>) known to be associated with ASD. For each, we had lists of downstream targets from two independent studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cotney, J. et al. The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment. Nat. Commun. 6, 6404 (2015)." href="#ref-CR76" id="ref-link-section-d136720129e2163">76</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. Cell 146, 247261 (2011)." href="#ref-CR77" id="ref-link-section-d136720129e2163_1">77</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sugathan, A. et al. CHD8 regulates neurodevelopmental pathways associated with autism spectrum disorder in neural progenitors. Proc. Natl Acad. Sci. USA 111, E4468E4477 (2014)." href="#ref-CR78" id="ref-link-section-d136720129e2163_2">78</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Greenblatt, E. J. &amp; Spradling, A. C. Fragile X mental retardation 1 gene enhances the translation of large autism-related proteins. Science 361, 709712 (2018)." href="/articles/s41593-018-0281-3#ref-CR79" id="ref-link-section-d136720129e2166">79</a></sup>, among which the overlap for FMRP targets was 3.71% and 27.61% for CHD8 targets. FDR <i>q</i>&lt;0.05 was used to identify significant enrichments after multiple-comparison correction.</p><h3 class="c-article__sub-heading" id="Sec27">Coexpression-network preservation across ASD brain and blood datasets</h3><p>We also sought to understand whether coexpression modules detected in blood leukocytes showed preservation of coexpression-network patterns in ASD postmortem cortical tissue from the frontal cortex and temporal cortex. To achieve this aim, we used ASD postmortem frontal and temporal cortex RNA-seq data from Parikshak et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Parikshak, N. N. et al. Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism. Nature 540, 423427 (2016)." href="/articles/s41593-018-0281-3#ref-CR40" id="ref-link-section-d136720129e2181">40</a></sup>. With the same preprocessed data as Parikshak et al., we computed Zsummary module-preservation statistics and evaluated which modules detected from ASD blood leukocyte datasets were preserved in ASD cortical frontal and temporal cortical tissue sampled from similar sites to those detected in the PLS LV1 map. A Zsummary&gt;10 indicates strong preservation, whereas a Zsummary between 2 and 10 indicates moderate preservation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Langfelder, P., Luo, R., Oldham, M. C. &amp; Horvath, S. Is my network module preserved and reproducible? PLoS. Comput. Biol. 7, e1001057 (2011)." href="/articles/s41593-018-0281-3#ref-CR67" id="ref-link-section-d136720129e2185">67</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec28">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-018-0281-3#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p><h3 class="c-article__sub-heading" id="Sec29">Code availability</h3><p>Code for implementing all analyses can be found at <a href="https://github.com/mvlombardo/asdlangoutcomebloodgexfmripls/">https://github.com/mvlombardo/asdlangoutcomebloodgexfmripls/</a>.</p></div></div></section>
                </div>
            

            <div>
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>The raw data that support the findings from this study are publicly available from the NIH National Database for Autism Research (NDAR). Raw blood leukocyte gene expression data are publicly available via Gene Expression Omnibus (GEO) (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE42133">GSE42133</a>; <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE111175">GSE111175</a>). Songbird Area X gene expression data are publicly available via GEO (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE34819">GSE34819</a>). GTEx data are publicly available at <a href="https://gtexportal.org/">https://gtexportal.org/</a>. ASD postmortem cortical gene expression can be found at <a href="https://github.com/dhglab/Genome-wide-changes-in-lncRNA-alternative-splicing-and-cortical-patterning-in-autism/">https://github.com/dhglab/Genome-wide-changes-in-lncRNA-alternative-splicing-and-cortical-patterning-in-autism/</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Geschwind, D. H. &amp; Levitt, P. Autism spectrum disorders: developmental disconnection syndromes. <i>Curr. Opin. Neurobiol.</i> <b>17</b>, 103111 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.conb.2007.01.009" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.conb.2007.01.009" aria-label="Article reference 1" data-doi="10.1016/j.conb.2007.01.009">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhvVOqtLo%3D" aria-label="CAS reference 1">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Autism%20spectrum%20disorders%3A%20developmental%20disconnection%20syndromes&amp;journal=Curr.%20Opin.%20Neurobiol.&amp;doi=10.1016%2Fj.conb.2007.01.009&amp;volume=17&amp;pages=103-111&amp;publication_year=2007&amp;author=Geschwind%2CDH&amp;author=Levitt%2CP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Happ, F., Ronald, A. &amp; Plomin, R. Time to give up on a single explanation for autism. <i>Nat. Neurosci.</i> <b>9</b>, 12181220 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn1770" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn1770" aria-label="Article reference 2" data-doi="10.1038/nn1770">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Time%20to%20give%20up%20on%20a%20single%20explanation%20for%20autism&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn1770&amp;volume=9&amp;pages=1218-1220&amp;publication_year=2006&amp;author=Happ%C3%A9%2CF&amp;author=Ronald%2CA&amp;author=Plomin%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Lai, M. C., Lombardo, M. V., Chakrabarti, B. &amp; Baron-Cohen, S. Subgrouping the autism spectrum: reflections on DSM-5. <i>PLoS Biol.</i> <b>11</b>, e1001544 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.1001544" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.1001544" aria-label="Article reference 3" data-doi="10.1371/journal.pbio.1001544">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXnsFKmsr8%3D" aria-label="CAS reference 3">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Subgrouping%20the%20autism%20%E2%80%9Cspectrum%E2%80%9D%3A%20reflections%20on%20DSM-5&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.1001544&amp;volume=11&amp;publication_year=2013&amp;author=Lai%2CMC&amp;author=Lombardo%2CMV&amp;author=Chakrabarti%2CB&amp;author=Baron-Cohen%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Kapur, S., Phillips, A. G. &amp; Insel, T. R. Why has it taken so long for biological psychiatry to develop clinical tests and what to do about it? <i>Mol. Psychiatry</i> <b>17</b>, 11741179 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/mp.2012.105" data-track-action="article reference" href="https://doi.org/10.1038%2Fmp.2012.105" aria-label="Article reference 4" data-doi="10.1038/mp.2012.105">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC38flvFOqug%3D%3D" aria-label="CAS reference 4">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Why%20has%20it%20taken%20so%20long%20for%20biological%20psychiatry%20to%20develop%20clinical%20tests%20and%20what%20to%20do%20about%20it%3F&amp;journal=Mol.%20Psychiatry&amp;doi=10.1038%2Fmp.2012.105&amp;volume=17&amp;pages=1174-1179&amp;publication_year=2012&amp;author=Kapur%2CS&amp;author=Phillips%2CAG&amp;author=Insel%2CTR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Lombardo, M. V., Lai, M. C. &amp; Baron-Cohen, S. Big data approaches to decomposing heterogeneity across the autism spectrum. Preprint at <a href="https://www.biorxiv.org/content/early/2018/03/17/278788" data-track="click" data-track-action="external reference" data-track-label="https://www.biorxiv.org/content/early/2018/03/17/278788">https://www.biorxiv.org/content/early/2018/03/17/278788</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Kjelgaard, M. M. &amp; Tager-Flusberg, H. An investigation of language impairment in autism: implications for genetic subgroups. <i>Lang. Cogn. Process.</i> <b>16</b>, 287308 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/01690960042000058" data-track-action="article reference" href="https://doi.org/10.1080%2F01690960042000058" aria-label="Article reference 6" data-doi="10.1080/01690960042000058">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20investigation%20of%20language%20impairment%20in%20autism%3A%20implications%20for%20genetic%20subgroups&amp;journal=Lang.%20Cogn.%20Process.&amp;doi=10.1080%2F01690960042000058&amp;volume=16&amp;pages=287-308&amp;publication_year=2001&amp;author=Kjelgaard%2CMM&amp;author=Tager-Flusberg%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Tager-Flusberg, H. &amp; Kasari, C. Minimally verbal school-aged children with autism spectrum disorder: the neglected end of the spectrum. <i>Autism Res.</i> <b>6</b>, 468478 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/aur.1329" data-track-action="article reference" href="https://doi.org/10.1002%2Faur.1329" aria-label="Article reference 7" data-doi="10.1002/aur.1329">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Minimally%20verbal%20school-aged%20children%20with%20autism%20spectrum%20disorder%3A%20the%20neglected%20end%20of%20the%20spectrum&amp;journal=Autism%20Res.&amp;doi=10.1002%2Faur.1329&amp;volume=6&amp;pages=468-478&amp;publication_year=2013&amp;author=Tager-Flusberg%2CH&amp;author=Kasari%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Howlin, P. Outcome in high-functioning adults with autism with and without early language delays: implications for the differentiation between autism and Asperger syndrome. <i>J. Autism. Dev. Disord.</i> <b>33</b>, 313 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1023/A:1022270118899" data-track-action="article reference" href="https://doi.org/10.1023%2FA%3A1022270118899" aria-label="Article reference 8" data-doi="10.1023/A:1022270118899">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Outcome%20in%20high-functioning%20adults%20with%20autism%20with%20and%20without%20early%20language%20delays%3A%20implications%20for%20the%20differentiation%20between%20autism%20and%20Asperger%20syndrome&amp;journal=J.%20Autism.%20Dev.%20Disord.&amp;doi=10.1023%2FA%3A1022270118899&amp;volume=33&amp;pages=3-13&amp;publication_year=2003&amp;author=Howlin%2CP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Perry, A. et al. Predictors of outcome for children receiving intensive behavioral intervention in a large, community-based program. <i>Res. Autism Spectr. Disord.</i> <b>5</b>, 592603 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.rasd.2010.07.003" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.rasd.2010.07.003" aria-label="Article reference 9" data-doi="10.1016/j.rasd.2010.07.003">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Predictors%20of%20outcome%20for%20children%20receiving%20intensive%20behavioral%20intervention%20in%20a%20large%2C%20community-based%20program&amp;journal=Res.%20Autism%20Spectr.%20Disord.&amp;doi=10.1016%2Fj.rasd.2010.07.003&amp;volume=5&amp;pages=592-603&amp;publication_year=2011&amp;author=Perry%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Gotham, K., Pickles, A. &amp; Lord, C. Trajectories of autism severity in children using standardized ADOS scores. <i>Pediatrics</i> <b>130</b>, e1278e1284 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1542/peds.2011-3668" data-track-action="article reference" href="https://doi.org/10.1542%2Fpeds.2011-3668" aria-label="Article reference 10" data-doi="10.1542/peds.2011-3668">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Trajectories%20of%20autism%20severity%20in%20children%20using%20standardized%20ADOS%20scores&amp;journal=Pediatrics&amp;doi=10.1542%2Fpeds.2011-3668&amp;volume=130&amp;pages=e1278-e1284&amp;publication_year=2012&amp;author=Gotham%2CK&amp;author=Pickles%2CA&amp;author=Lord%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Venter, A., Lord, C. &amp; Schopler, E. A follow-up study of high-functioning autistic children. <i>J. Child Psychol. Psychiatry</i> <b>33</b>, 489507 (1992).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1469-7610.1992.tb00887.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1469-7610.1992.tb00887.x" aria-label="Article reference 11" data-doi="10.1111/j.1469-7610.1992.tb00887.x">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK383ltVOksw%3D%3D" aria-label="CAS reference 11">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20follow-up%20study%20of%20high-functioning%20autistic%20children&amp;journal=J.%20Child%20Psychol.%20Psychiatry&amp;doi=10.1111%2Fj.1469-7610.1992.tb00887.x&amp;volume=33&amp;pages=489-507&amp;publication_year=1992&amp;author=Venter%2CA&amp;author=Lord%2CC&amp;author=Schopler%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Szatmari, P. et al. Similar developmental trajectories in autism and Asperger syndrome: from early childhood to adolescence. <i>J. Child Psychol. Psychiatry</i> <b>50</b>, 14591467 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1469-7610.2009.02123.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1469-7610.2009.02123.x" aria-label="Article reference 12" data-doi="10.1111/j.1469-7610.2009.02123.x">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Similar%20developmental%20trajectories%20in%20autism%20and%20Asperger%20syndrome%3A%20from%20early%20childhood%20to%20adolescence&amp;journal=J.%20Child%20Psychol.%20Psychiatry&amp;doi=10.1111%2Fj.1469-7610.2009.02123.x&amp;volume=50&amp;pages=1459-1467&amp;publication_year=2009&amp;author=Szatmari%2CP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Courchesne, E. et al. TheASD Living Biology: from cell proliferation to clinical phenotype. M<i>ol. Psychiatry</i> <a href="https://doi.org/10.1038/s41380-018-0056-y" data-track="click" data-track-action="external reference" data-track-label="10.1038/s41380-018-0056-y">https://doi.org/10.1038/s41380-018-0056-y</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Pramparo, T. et al. Cell cycle networks link gene expression dysregulation, mutation, and brain maldevelopment in autistic toddlers. <i>Mol. Syst. Biol.</i> <b>11</b>, 841 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.15252/msb.20156108" data-track-action="article reference" href="https://doi.org/10.15252%2Fmsb.20156108" aria-label="Article reference 14" data-doi="10.15252/msb.20156108">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20cycle%20networks%20link%20gene%20expression%20dysregulation%2C%20mutation%2C%20and%20brain%20maldevelopment%20in%20autistic%20toddlers&amp;journal=Mol.%20Syst.%20Biol.&amp;doi=10.15252%2Fmsb.20156108&amp;volume=11&amp;publication_year=2015&amp;author=Pramparo%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Pramparo, T. et al. Prediction of autism by translation and immune/inflammation coexpressed genes in toddlers from pediatric community practices. <i>JAMA Psychiatry</i> <b>72</b>, 386394 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1001/jamapsychiatry.2014.3008" data-track-action="article reference" href="https://doi.org/10.1001%2Fjamapsychiatry.2014.3008" aria-label="Article reference 15" data-doi="10.1001/jamapsychiatry.2014.3008">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20autism%20by%20translation%20and%20immune%2Finflammation%20coexpressed%20genes%20in%20toddlers%20from%20pediatric%20community%20practices&amp;journal=JAMA%20Psychiatry&amp;doi=10.1001%2Fjamapsychiatry.2014.3008&amp;volume=72&amp;pages=386-394&amp;publication_year=2015&amp;author=Pramparo%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Geschwind, D. H. &amp; State, M. W. Gene hunting in autism spectrum disorder: on the path to precision medicine. <i>Lancet. Neurol.</i> <b>14</b>, 11091120 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S1474-4422(15)00044-7" data-track-action="article reference" href="https://doi.org/10.1016%2FS1474-4422%2815%2900044-7" aria-label="Article reference 16" data-doi="10.1016/S1474-4422(15)00044-7">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20hunting%20in%20autism%20spectrum%20disorder%3A%20on%20the%20path%20to%20precision%20medicine&amp;journal=Lancet.%20Neurol.&amp;doi=10.1016%2FS1474-4422%2815%2900044-7&amp;volume=14&amp;pages=1109-1120&amp;publication_year=2015&amp;author=Geschwind%2CDH&amp;author=State%2CMW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Sandin, S. et al. The heritability of autism spectrum disorder. <i>J. Am. Med. Assoc.</i> <b>318</b>, 11821184 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1001/jama.2017.12141" data-track-action="article reference" href="https://doi.org/10.1001%2Fjama.2017.12141" aria-label="Article reference 17" data-doi="10.1001/jama.2017.12141">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20heritability%20of%20autism%20spectrum%20disorder&amp;journal=J.%20Am.%20Med.%20Assoc.&amp;doi=10.1001%2Fjama.2017.12141&amp;volume=318&amp;pages=1182-1184&amp;publication_year=2017&amp;author=Sandin%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Battle, A., Brown, C. D., Engelhardt, B. E. &amp; Montgomery, S. B. Genetic effects on gene expression across human tissues. <i>Nature</i> <b>550</b>, 204213 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature24277" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature24277" aria-label="Article reference 18" data-doi="10.1038/nature24277">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20effects%20on%20gene%20expression%20across%20human%20tissues&amp;journal=Nature&amp;doi=10.1038%2Fnature24277&amp;volume=550&amp;pages=204-213&amp;publication_year=2017&amp;author=Battle%2CA&amp;author=Brown%2CCD&amp;author=Engelhardt%2CBE&amp;author=Montgomery%2CSB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Lombardo, M. V. et al. Different functional neural substrates for good and poor language outcome in autism. <i>Neuron</i> <b>86</b>, 567577 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2015.03.023" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.03.023" aria-label="Article reference 19" data-doi="10.1016/j.neuron.2015.03.023">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXmsVeqtLk%3D" aria-label="CAS reference 19">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20functional%20neural%20substrates%20for%20good%20and%20poor%20language%20outcome%20in%20autism&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2015.03.023&amp;volume=86&amp;pages=567-577&amp;publication_year=2015&amp;author=Lombardo%2CMV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. <i>Cell</i> <b>169</b>, 11771186 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.05.038" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.05.038" aria-label="Article reference 20" data-doi="10.1016/j.cell.2017.05.038">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVWrtrjM" aria-label="CAS reference 20">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20expanded%20view%20of%20complex%20traits%3A%20from%20polygenic%20to%20omnigenic&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.05.038&amp;volume=169&amp;pages=1177-1186&amp;publication_year=2017&amp;author=Boyle%2CEA&amp;author=Li%2CYI&amp;author=Pritchard%2CJK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Wray, N. R., Wijmenga, C., Sullivan, P. F., Yang, J. &amp; Visscher, P. M. Common disease is more complex than implied by the core gene omnigenic model. <i>Cell</i> <b>173</b>, 15731580 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2018.05.051" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.05.051" aria-label="Article reference 21" data-doi="10.1016/j.cell.2018.05.051">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFGjsrvP" aria-label="CAS reference 21">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Common%20disease%20is%20more%20complex%20than%20implied%20by%20the%20core%20gene%20omnigenic%20model&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.05.051&amp;volume=173&amp;pages=1573-1580&amp;publication_year=2018&amp;author=Wray%2CNR&amp;author=Wijmenga%2CC&amp;author=Sullivan%2CPF&amp;author=Yang%2CJ&amp;author=Visscher%2CPM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Cremers, H. R., Wager, T. D. &amp; Yarkoni, T. The relation between statistical power and inference in fMRI. <i>PLoS ONE.</i> <b>12</b>, e0184923 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0184923" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0184923" aria-label="Article reference 22" data-doi="10.1371/journal.pone.0184923">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20relation%20between%20statistical%20power%20and%20inference%20in%20fMRI&amp;journal=PLoS%20ONE.&amp;doi=10.1371%2Fjournal.pone.0184923&amp;volume=12&amp;publication_year=2017&amp;author=Cremers%2CHR&amp;author=Wager%2CTD&amp;author=Yarkoni%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Langfelder, P. &amp; Horvath, S. WGCNA: an R package for weighted correlation network analysis. <i>BMC Bioinformatics</i> <b>9</b>, 559 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/1471-2105-9-559" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2105-9-559" aria-label="Article reference 23" data-doi="10.1186/1471-2105-9-559">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=WGCNA%3A%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-9-559&amp;volume=9&amp;publication_year=2008&amp;author=Langfelder%2CP&amp;author=Horvath%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Hickok, G. &amp; Poeppel, D. The cortical organization of speech processing. <i>Nat. Rev. Neurosci.</i> <b>8</b>, 393402 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn2113" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn2113" aria-label="Article reference 24" data-doi="10.1038/nrn2113">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXksFSis7w%3D" aria-label="CAS reference 24">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20cortical%20organization%20of%20speech%20processing&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn2113&amp;volume=8&amp;pages=393-402&amp;publication_year=2007&amp;author=Hickok%2CG&amp;author=Poeppel%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Konopka, G. &amp; Roberts, T. F. Insights into the neural and genetic basis of vocal communication. <i>Cell</i> <b>164</b>, 12691276 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2016.02.039" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2016.02.039" aria-label="Article reference 25" data-doi="10.1016/j.cell.2016.02.039">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XktFKjsrk%3D" aria-label="CAS reference 25">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Insights%20into%20the%20neural%20and%20genetic%20basis%20of%20vocal%20communication&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.02.039&amp;volume=164&amp;pages=1269-1276&amp;publication_year=2016&amp;author=Konopka%2CG&amp;author=Roberts%2CTF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Pfenning, A. R. et al. Convergent transcriptional specializations in the brains of humans and song-learning birds. <i>Science</i> <b>346</b>, 1256846 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1256846" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1256846" aria-label="Article reference 26" data-doi="10.1126/science.1256846">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Convergent%20transcriptional%20specializations%20in%20the%20brains%20of%20humans%20and%20song-learning%20birds&amp;journal=Science&amp;doi=10.1126%2Fscience.1256846&amp;volume=346&amp;publication_year=2014&amp;author=Pfenning%2CAR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Kelleher, R. J. III &amp; Bear, M. F. The autistic neuron: troubled translation? <i>Cell</i> <b>135</b>, 401406 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2008.10.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2008.10.017" aria-label="Article reference 27" data-doi="10.1016/j.cell.2008.10.017">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtlyqtr%2FE" aria-label="CAS reference 27">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20autistic%20neuron%3A%20troubled%20translation%3F&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2008.10.017&amp;volume=135&amp;pages=401-406&amp;publication_year=2008&amp;author=Kelleher%2CRJ&amp;author=Bear%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">De Rubeis, S. et al. Synaptic, transcriptional and chromatin genes disrupted in autism. <i>Nature</i> <b>515</b>, 209215 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature13772" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature13772" aria-label="Article reference 28" data-doi="10.1038/nature13772">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Synaptic%2C%20transcriptional%20and%20chromatin%20genes%20disrupted%20in%20autism&amp;journal=Nature&amp;doi=10.1038%2Fnature13772&amp;volume=515&amp;pages=209-215&amp;publication_year=2014&amp;author=Rubeis%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Bernier, R. et al. Disruptive CHD8 mutations define a subtype of autism early in development. <i>Cell</i> <b>158</b>, 263276 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2014.06.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2014.06.017" aria-label="Article reference 29" data-doi="10.1016/j.cell.2014.06.017">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFWgtbbM" aria-label="CAS reference 29">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Disruptive%20CHD8%20mutations%20define%20a%20subtype%20of%20autism%20early%20in%20development&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2014.06.017&amp;volume=158&amp;pages=263-276&amp;publication_year=2014&amp;author=Bernier%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Morgan, J. T. et al. Microglial activation and increased microglial density observed in the dorsolateral prefrontal cortex in autism. <i>Biol. Psychiatry</i> <b>68</b>, 368376 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2010.05.024" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2010.05.024" aria-label="Article reference 30" data-doi="10.1016/j.biopsych.2010.05.024">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Microglial%20activation%20and%20increased%20microglial%20density%20observed%20in%20the%20dorsolateral%20prefrontal%20cortex%20in%20autism&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2010.05.024&amp;volume=68&amp;pages=368-376&amp;publication_year=2010&amp;author=Morgan%2CJT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Vargas, D. L., Nascimbene, C., Krishnan, C., Zimmerman, A. W. &amp; Pardo, C. A. Neuroglial activation and neuroinflammation in the brain of patients with autism. <i>Ann. Neurol.</i> <b>57</b>, 6781 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/ana.20315" data-track-action="article reference" href="https://doi.org/10.1002%2Fana.20315" aria-label="Article reference 31" data-doi="10.1002/ana.20315">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXpsFSktA%3D%3D" aria-label="CAS reference 31">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Neuroglial%20activation%20and%20neuroinflammation%20in%20the%20brain%20of%20patients%20with%20autism&amp;journal=Ann.%20Neurol.&amp;doi=10.1002%2Fana.20315&amp;volume=57&amp;pages=67-81&amp;publication_year=2005&amp;author=Vargas%2CDL&amp;author=Nascimbene%2CC&amp;author=Krishnan%2CC&amp;author=Zimmerman%2CAW&amp;author=Pardo%2CCA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Voineagu, I. et al. Transcriptomic analysis of autistic brain reveals convergent molecular pathology. <i>Nature</i> <b>474</b>, 380384 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature10110" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature10110" aria-label="Article reference 32" data-doi="10.1038/nature10110">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXmsFyqsLs%3D" aria-label="CAS reference 32">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptomic%20analysis%20of%20autistic%20brain%20reveals%20convergent%20molecular%20pathology&amp;journal=Nature&amp;doi=10.1038%2Fnature10110&amp;volume=474&amp;pages=380-384&amp;publication_year=2011&amp;author=Voineagu%2CI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Hilliard, A. T., Miller, J. E., Fraley, E. R., Horvath, S. &amp; White, S. A. Molecular microcircuitry underlies functional specification in a basal ganglia circuit dedicated to vocal learning. <i>Neuron</i> <b>73</b>, 537552 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2012.01.005" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2012.01.005" aria-label="Article reference 33" data-doi="10.1016/j.neuron.2012.01.005">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XitFOjtbY%3D" aria-label="CAS reference 33">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20microcircuitry%20underlies%20functional%20specification%20in%20a%20basal%20ganglia%20circuit%20dedicated%20to%20vocal%20learning&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2012.01.005&amp;volume=73&amp;pages=537-552&amp;publication_year=2012&amp;author=Hilliard%2CAT&amp;author=Miller%2CJE&amp;author=Fraley%2CER&amp;author=Horvath%2CS&amp;author=White%2CSA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Konopka, G. et al. Human-specific transcriptional networks in the brain. <i>Neuron</i> <b>75</b>, 601617 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2012.05.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2012.05.034" aria-label="Article reference 34" data-doi="10.1016/j.neuron.2012.05.034">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1Kju77N" aria-label="CAS reference 34">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Human-specific%20transcriptional%20networks%20in%20the%20brain&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2012.05.034&amp;volume=75&amp;pages=601-617&amp;publication_year=2012&amp;author=Konopka%2CG">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Liu, X. et al. Disruption of an evolutionarily novel synaptic expression pattern in autism. <i>PLoS Biol.</i> <b>14</b>, e1002558 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.1002558" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.1002558" aria-label="Article reference 35" data-doi="10.1371/journal.pbio.1002558">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Disruption%20of%20an%20evolutionarily%20novel%20synaptic%20expression%20pattern%20in%20autism&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.1002558&amp;volume=14&amp;publication_year=2016&amp;author=Liu%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Eising, E. et al. A set of regulatory genes co-expressed in embryonic human brain is implicated in disrupted speech development. <i>Mol. Psychiatry</i> <a href="https://doi.org/10.1038/s41380-018-0020-x" data-track="click" data-track-action="external reference" data-track-label="10.1038/s41380-018-0020-x">https://doi.org/10.1038/s41380-018-0020-x</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Parikshak, N. N. et al. Integrative functional genomic analyses implicate specific molecular pathways and circuits in autism. <i>Cell</i> <b>155</b>, 10081021 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2013.10.031" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.10.031" aria-label="Article reference 37" data-doi="10.1016/j.cell.2013.10.031">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFWjt73N" aria-label="CAS reference 37">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrative%20functional%20genomic%20analyses%20implicate%20specific%20molecular%20pathways%20and%20circuits%20in%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.10.031&amp;volume=155&amp;pages=1008-1021&amp;publication_year=2013&amp;author=Parikshak%2CNN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Willsey, A. J. et al. Coexpression networks implicate human midfetal deep cortical projection neurons in the pathogenesis of autism. <i>Cell</i> <b>155</b>, 9971007 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2013.10.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.10.020" aria-label="Article reference 38" data-doi="10.1016/j.cell.2013.10.020">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhvFWis7zP" aria-label="CAS reference 38">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Coexpression%20networks%20implicate%20human%20midfetal%20deep%20cortical%20projection%20neurons%20in%20the%20pathogenesis%20of%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.10.020&amp;volume=155&amp;pages=997-1007&amp;publication_year=2013&amp;author=Willsey%2CAJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Miller, J. A. et al. Transcriptional landscape of the prenatal human brain. <i>Nature</i> <b>508</b>, 199206 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature13185" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature13185" aria-label="Article reference 39" data-doi="10.1038/nature13185">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXmtlWnsrY%3D" aria-label="CAS reference 39">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Transcriptional%20landscape%20of%20the%20prenatal%20human%20brain&amp;journal=Nature&amp;doi=10.1038%2Fnature13185&amp;volume=508&amp;pages=199-206&amp;publication_year=2014&amp;author=Miller%2CJA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Parikshak, N. N. et al. Genome-wide changes in lncRNA, splicing, and regional gene expression patterns in autism. <i>Nature</i> <b>540</b>, 423427 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature20612" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature20612" aria-label="Article reference 40" data-doi="10.1038/nature20612">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XitVWmurbF" aria-label="CAS reference 40">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20changes%20in%20lncRNA%2C%20splicing%2C%20and%20regional%20gene%20expression%20patterns%20in%20autism&amp;journal=Nature&amp;doi=10.1038%2Fnature20612&amp;volume=540&amp;pages=423-427&amp;publication_year=2016&amp;author=Parikshak%2CNN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Rossignol, D. A. &amp; Frye, R. E. Mitochondrial dysfunction in autism spectrum disorders: a systematic review and meta-analysis. <i>Mol. Psychiatry</i> <b>17</b>, 290314 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/mp.2010.136" data-track-action="article reference" href="https://doi.org/10.1038%2Fmp.2010.136" aria-label="Article reference 41" data-doi="10.1038/mp.2010.136">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XivVansLs%3D" aria-label="CAS reference 41">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Mitochondrial%20dysfunction%20in%20autism%20spectrum%20disorders%3A%20a%20systematic%20review%20and%20meta-analysis&amp;journal=Mol.%20Psychiatry&amp;doi=10.1038%2Fmp.2010.136&amp;volume=17&amp;pages=290-314&amp;publication_year=2012&amp;author=Rossignol%2CDA&amp;author=Frye%2CRE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Sanders, S. J. et al. Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci. <i>Neuron</i> <b>87</b>, 12151233 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2015.09.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.09.016" aria-label="Article reference 42" data-doi="10.1016/j.neuron.2015.09.016">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFGhu7%2FN" aria-label="CAS reference 42">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Insights%20into%20autism%20spectrum%20disorder%20genomic%20architecture%20and%20biology%20from%2071%20risk%20loci&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2015.09.016&amp;volume=87&amp;pages=1215-1233&amp;publication_year=2015&amp;author=Sanders%2CSJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Marchetto, M. C. N. et al. Differential L1 regulation in pluripotent stem cells of humans and apes. <i>Nature</i> <b>503</b>, 525529 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature12686" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature12686" aria-label="Article reference 43" data-doi="10.1038/nature12686">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslSnsbzL" aria-label="CAS reference 43">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20L1%20regulation%20in%20pluripotent%20stem%20cells%20of%20humans%20and%20apes&amp;journal=Nature&amp;doi=10.1038%2Fnature12686&amp;volume=503&amp;pages=525-529&amp;publication_year=2013&amp;author=Marchetto%2CMCN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Nuttle, X. et al. Emergence of a <i>Homo sapiens</i>specific gene family and chromosome 16p11.2 CNV susceptibility. <i>Nature</i> <b>536</b>, 205209 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature19075" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature19075" aria-label="Article reference 44" data-doi="10.1038/nature19075">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1yqsbzL" aria-label="CAS reference 44">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Emergence%20of%20a%20Homo%20sapiens%E2%80%93specific%20gene%20family%20and%20chromosome%2016p11.2%20CNV%20susceptibility&amp;journal=Nature&amp;doi=10.1038%2Fnature19075&amp;volume=536&amp;pages=205-209&amp;publication_year=2016&amp;author=Nuttle%2CX">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Hippolyte, L. et al. The number of genomic copies at the 16p11.2 locus modulates language, verbal memory, and inhibition. <i>Biol. Psychiatry</i> <b>80</b>, 129139 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2015.10.021" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2015.10.021" aria-label="Article reference 45" data-doi="10.1016/j.biopsych.2015.10.021">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xis1ymsA%3D%3D" aria-label="CAS reference 45">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20number%20of%20genomic%20copies%20at%20the%2016p11.2%20locus%20modulates%20language%2C%20verbal%20memory%2C%20and%20inhibition&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2015.10.021&amp;volume=80&amp;pages=129-139&amp;publication_year=2016&amp;author=Hippolyte%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Demopoulos, C. et al. Abnormal speech motor control in individuals with 16p11.2 deletions. <i>Sci. Rep.</i> <b>8</b>, 1274 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41598-018-19751-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-018-19751-x" aria-label="Article reference 46" data-doi="10.1038/s41598-018-19751-x">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Abnormal%20speech%20motor%20control%20in%20individuals%20with%2016p11.2%20deletions&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-19751-x&amp;volume=8&amp;publication_year=2018&amp;author=Demopoulos%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Berman, J. I. et al. Abnormal auditory and language pathways in children with 16p11.2 deletion. <i>Neuroimage Clin.</i> <b>9</b>, 5057 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.nicl.2015.07.006" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.nicl.2015.07.006" aria-label="Article reference 47" data-doi="10.1016/j.nicl.2015.07.006">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Abnormal%20auditory%20and%20language%20pathways%20in%20children%20with%2016p11.2%20deletion&amp;journal=Neuroimage%20Clin.&amp;doi=10.1016%2Fj.nicl.2015.07.006&amp;volume=9&amp;pages=50-57&amp;publication_year=2015&amp;author=Berman%2CJI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Kosmicki, J. A. et al. Refining the role of de novo protein-truncating variants in neurodevelopmental disorders by using population reference samples. <i>Nat. Genet.</i> <b>49</b>, 504510 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.3789" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3789" aria-label="Article reference 48" data-doi="10.1038/ng.3789">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitlGiu7Y%3D" aria-label="CAS reference 48">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Refining%20the%20role%20of%20de%20novo%20protein-truncating%20variants%20in%20neurodevelopmental%20disorders%20by%20using%20population%20reference%20samples&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3789&amp;volume=49&amp;pages=504-510&amp;publication_year=2017&amp;author=Kosmicki%2CJA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Lek, M. et al. Analysis of protein-coding genetic variation in 60,706 humans. <i>Nature</i> <b>536</b>, 285291 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature19057" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature19057" aria-label="Article reference 49" data-doi="10.1038/nature19057">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlOnsbbP" aria-label="CAS reference 49">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20protein-coding%20genetic%20variation%20in%2060%2C706%20humans&amp;journal=Nature&amp;doi=10.1038%2Fnature19057&amp;volume=536&amp;pages=285-291&amp;publication_year=2016&amp;author=Lek%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Abrahams, B. S. et al. SFARI Gene 2.0: a community-driven knowledgebase for the autism spectrum disorders (ASDs). <i>Mol. Autism</i> <b>4</b>, 36 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/2040-2392-4-36" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/2040-2392-4-36" aria-label="Article reference 50" data-doi="10.1186/2040-2392-4-36">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=SFARI%20Gene%202.0%3A%20a%20community-driven%20knowledgebase%20for%20the%20autism%20spectrum%20disorders%20%28ASDs%29&amp;journal=Mol.%20Autism&amp;doi=10.1186%2F2040-2392-4-36&amp;volume=4&amp;publication_year=2013&amp;author=Abrahams%2CBS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Pierce, K., Conant, D., Hazin, R., Stoner, R. &amp; Desmond, J. Preference for geometric patterns early in life as a risk factor for autism. <i>Arch. Gen. Psychiatry</i> <b>68</b>, 101109 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1001/archgenpsychiatry.2010.113" data-track-action="article reference" href="https://doi.org/10.1001%2Farchgenpsychiatry.2010.113" aria-label="Article reference 51" data-doi="10.1001/archgenpsychiatry.2010.113">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Preference%20for%20geometric%20patterns%20early%20in%20life%20as%20a%20risk%20factor%20for%20autism&amp;journal=Arch.%20Gen.%20Psychiatry&amp;doi=10.1001%2Farchgenpsychiatry.2010.113&amp;volume=68&amp;pages=101-109&amp;publication_year=2011&amp;author=Pierce%2CK&amp;author=Conant%2CD&amp;author=Hazin%2CR&amp;author=Stoner%2CR&amp;author=Desmond%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Pierce, K. et al. Eye tracking reveals abnormal visual preference for geometric images as an early biomarker of an autism spectrum disorder subtype associated with increased symptom severity. <i>Biol. Psychiatry</i> <b>79</b>, 657666 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2015.03.032" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2015.03.032" aria-label="Article reference 52" data-doi="10.1016/j.biopsych.2015.03.032">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Eye%20tracking%20reveals%20abnormal%20visual%20preference%20for%20geometric%20images%20as%20an%20early%20biomarker%20of%20an%20autism%20spectrum%20disorder%20subtype%20associated%20with%20increased%20symptom%20severity&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2015.03.032&amp;volume=79&amp;pages=657-666&amp;publication_year=2016&amp;author=Pierce%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Pierce, K. et al. Detecting, studying, and treating autism early: the One-Year Well-Baby Check-Up Approach. <i>J. Pediatr.</i> <b>159</b>, 458465.e6 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.jpeds.2011.02.036" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jpeds.2011.02.036" aria-label="Article reference 53" data-doi="10.1016/j.jpeds.2011.02.036">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Detecting%2C%20studying%2C%20and%20treating%20autism%20early%3A%20the%20One-Year%20Well-Baby%20Check-Up%20Approach&amp;journal=J.%20Pediatr.&amp;doi=10.1016%2Fj.jpeds.2011.02.036&amp;volume=159&amp;pages=458-465.e6&amp;publication_year=2011&amp;author=Pierce%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Wetherby, A. &amp; Prizant, B. <i>Communication and Symbolic Behavior Scales Developmental Profile</i>, <i>First Normed Edition</i> (Paul H. Brookes, Baltimore, 2002).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Wetherby, A. M., Brosnan-Maddox, S., Peace, V. &amp; Newton, L. Validation of the Infant-Toddler Checklist as a broadband screener for autism spectrum disorders from 9 to 24 months of age. <i>Autism</i> <b>12</b>, 487511 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1177/1362361308094501" data-track-action="article reference" href="https://doi.org/10.1177%2F1362361308094501" aria-label="Article reference 55" data-doi="10.1177/1362361308094501">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Validation%20of%20the%20Infant-Toddler%20Checklist%20as%20a%20broadband%20screener%20for%20autism%20spectrum%20disorders%20from%209%20to%2024%20months%20of%20age&amp;journal=Autism&amp;doi=10.1177%2F1362361308094501&amp;volume=12&amp;pages=487-511&amp;publication_year=2008&amp;author=Wetherby%2CAM&amp;author=Brosnan-Maddox%2CS&amp;author=Peace%2CV&amp;author=Newton%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Lord, C. et al. The autism diagnostic observation schedule-generic: a standard measure of social and communication deficits associated with the spectrum of autism. <i>J. Autism. Dev. Disord.</i> <b>30</b>, 205223 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1023/A:1005592401947" data-track-action="article reference" href="https://doi.org/10.1023%2FA%3A1005592401947" aria-label="Article reference 56" data-doi="10.1023/A:1005592401947">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3crgsVOktA%3D%3D" aria-label="CAS reference 56">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20autism%20diagnostic%20observation%20schedule-generic%3A%20a%20standard%20measure%20of%20social%20and%20communication%20deficits%20associated%20with%20the%20spectrum%20of%20autism&amp;journal=J.%20Autism.%20Dev.%20Disord.&amp;doi=10.1023%2FA%3A1005592401947&amp;volume=30&amp;pages=205-223&amp;publication_year=2000&amp;author=Lord%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Mullen, E. M. <i>Mullen Scales of Early Learning</i> (American Guidance Service, Circle Pine, MN, USA, 1995).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Sparrow, S., Cicchetti, D. &amp; Balla, D. <i>Vineland-II Scales Of Adaptive Behavior: Survey Form Manual</i> (American Guidance Service, Circle Pines, MN, USA, 2005).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Schroeder, A. et al. The RIN: an RNA integrity number for assigning integrity values to RNA measurements. <i>BMC. Mol. Biol.</i> <b>7</b>, 3 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/1471-2199-7-3" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2199-7-3" aria-label="Article reference 59" data-doi="10.1186/1471-2199-7-3">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20RIN%3A%20an%20RNA%20integrity%20number%20for%20assigning%20integrity%20values%20to%20RNA%20measurements&amp;journal=BMC.%20Mol.%20Biol.&amp;doi=10.1186%2F1471-2199-7-3&amp;volume=7&amp;publication_year=2006&amp;author=Schroeder%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Du, P., Kibbe, W. A. &amp; Lin, S. M. lumi: a pipeline for processing Illumina microarray. <i>Bioinformatics</i> <b>24</b>, 15471548 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btn224" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtn224" aria-label="Article reference 60" data-doi="10.1093/bioinformatics/btn224">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXnvV2rtLg%3D" aria-label="CAS reference 60">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=lumi%3A%20a%20pipeline%20for%20processing%20Illumina%20microarray&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtn224&amp;volume=24&amp;pages=1547-1548&amp;publication_year=2008&amp;author=Du%2CP&amp;author=Kibbe%2CWA&amp;author=Lin%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Gentleman, R. C. et al. Bioconductor: open software development for computational biology and bioinformatics. <i>Genome. Biol.</i> <b>5</b>, R80 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/gb-2004-5-10-r80" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/gb-2004-5-10-r80" aria-label="Article reference 61" data-doi="10.1186/gb-2004-5-10-r80">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Bioconductor%3A%20open%20software%20development%20for%20computational%20biology%20and%20bioinformatics&amp;journal=Genome.%20Biol.&amp;doi=10.1186%2Fgb-2004-5-10-r80&amp;volume=5&amp;publication_year=2004&amp;author=Gentleman%2CRC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Chikina, M., Zaslavsky, E. &amp; Sealfon, S. C. CellCODE: a robust latent variable approach to differential expression analysis for heterogeneous cell populations. <i>Bioinformatics</i> <b>31</b>, 15841591 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btv015" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv015" aria-label="Article reference 62" data-doi="10.1093/bioinformatics/btv015">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ais7nM" aria-label="CAS reference 62">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=CellCODE%3A%20a%20robust%20latent%20variable%20approach%20to%20differential%20expression%20analysis%20for%20heterogeneous%20cell%20populations&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv015&amp;volume=31&amp;pages=1584-1591&amp;publication_year=2015&amp;author=Chikina%2CM&amp;author=Zaslavsky%2CE&amp;author=Sealfon%2CSC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. <i>Nucleic Acids Res.</i> <b>43</b>, e47 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkv007" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkv007" aria-label="Article reference 63" data-doi="10.1093/nar/gkv007">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkv007&amp;volume=43&amp;publication_year=2015&amp;author=Ritchie%2CME">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Storey, J. D. A direct approach to false discovery rates. <i>J. R. Statist. Soc. B</i> <b>64</b>, 479498 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/1467-9868.00346" data-track-action="article reference" href="https://doi.org/10.1111%2F1467-9868.00346" aria-label="Article reference 64" data-doi="10.1111/1467-9868.00346">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20direct%20approach%20to%20false%20discovery%20rates&amp;journal=J.%20R.%20Statist.%20Soc.%20B&amp;doi=10.1111%2F1467-9868.00346&amp;volume=64&amp;pages=479-498&amp;publication_year=2002&amp;author=Storey%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Zhang, B. &amp; Horvath, S. A general framework for weighted gene co-expression network analysis. <i>Stat. Appl. Genet. Mol. Biol</i>. <b>4</b>, Article17 (2005).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Oldham, M. C., Horvath, S. &amp; Geschwind, D. H. Conservation and evolution of gene coexpression networks in human and chimpanzee brains. <i>Proc. Natl Acad. Sci. USA</i> <b>103</b>, 1797317978 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0605938103" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0605938103" aria-label="Article reference 66" data-doi="10.1073/pnas.0605938103">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xht1Kmsr7F" aria-label="CAS reference 66">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Conservation%20and%20evolution%20of%20gene%20coexpression%20networks%20in%20human%20and%20chimpanzee%20brains&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0605938103&amp;volume=103&amp;pages=17973-17978&amp;publication_year=2006&amp;author=Oldham%2CMC&amp;author=Horvath%2CS&amp;author=Geschwind%2CDH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Langfelder, P., Luo, R., Oldham, M. C. &amp; Horvath, S. Is my network module preserved and reproducible? <i>PLoS. Comput. Biol.</i> <b>7</b>, e1001057 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1001057" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1001057" aria-label="Article reference 67" data-doi="10.1371/journal.pcbi.1001057">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtlyjs7o%3D" aria-label="CAS reference 67">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Is%20my%20network%20module%20preserved%20and%20reproducible%3F&amp;journal=PLoS.%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1001057&amp;volume=7&amp;publication_year=2011&amp;author=Langfelder%2CP&amp;author=Luo%2CR&amp;author=Oldham%2CMC&amp;author=Horvath%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Eyler, L. T., Pierce, K. &amp; Courchesne, E. A failure of left temporal cortex to specialize for language is an early emerging and fundamental property of autism. <i>Brain</i> <b>135</b>, 949960 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/brain/awr364" data-track-action="article reference" href="https://doi.org/10.1093%2Fbrain%2Fawr364" aria-label="Article reference 68" data-doi="10.1093/brain/awr364">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20failure%20of%20left%20temporal%20cortex%20to%20specialize%20for%20language%20is%20an%20early%20emerging%20and%20fundamental%20property%20of%20autism&amp;journal=Brain&amp;doi=10.1093%2Fbrain%2Fawr364&amp;volume=135&amp;pages=949-960&amp;publication_year=2012&amp;author=Eyler%2CLT&amp;author=Pierce%2CK&amp;author=Courchesne%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Redcay, E. &amp; Courchesne, E. Deviant functional magnetic resonance imaging patterns of brain activity to speech in 2-3-year-old children with autism spectrum disorder. <i>Biol. Psychiatry</i> <b>64</b>, 589598 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2008.05.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2008.05.020" aria-label="Article reference 69" data-doi="10.1016/j.biopsych.2008.05.020">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Deviant%20functional%20magnetic%20resonance%20imaging%20patterns%20of%20brain%20activity%20to%20speech%20in%202-3-year-old%20children%20with%20autism%20spectrum%20disorder&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2008.05.020&amp;volume=64&amp;pages=589-598&amp;publication_year=2008&amp;author=Redcay%2CE&amp;author=Courchesne%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Redcay, E., Haist, F. &amp; Courchesne, E. Functional neuroimaging of speech perception during a pivotal period in language acquisition. <i>Dev. Sci.</i> <b>11</b>, 237252 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1467-7687.2008.00674.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1467-7687.2008.00674.x" aria-label="Article reference 70" data-doi="10.1111/j.1467-7687.2008.00674.x">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20neuroimaging%20of%20speech%20perception%20during%20a%20pivotal%20period%20in%20language%20acquisition&amp;journal=Dev.%20Sci.&amp;doi=10.1111%2Fj.1467-7687.2008.00674.x&amp;volume=11&amp;pages=237-252&amp;publication_year=2008&amp;author=Redcay%2CE&amp;author=Haist%2CF&amp;author=Courchesne%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Dehaene-Lambertz, G., Dehaene, S. &amp; Hertz-Pannier, L. Functional neuroimaging of speech perception in infants. <i>Science</i> <b>298</b>, 20132015 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1077066" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1077066" aria-label="Article reference 71" data-doi="10.1126/science.1077066">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38Xpt1Wisbw%3D" aria-label="CAS reference 71">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20neuroimaging%20of%20speech%20perception%20in%20infants&amp;journal=Science&amp;doi=10.1126%2Fscience.1077066&amp;volume=298&amp;pages=2013-2015&amp;publication_year=2002&amp;author=Dehaene-Lambertz%2CG&amp;author=Dehaene%2CS&amp;author=Hertz-Pannier%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Genovese, C. R., Lazar, N. A. &amp; Nichols, T. Thresholding of statistical maps in functional neuroimaging using the false discovery rate. <i>Neuroimage</i> <b>15</b>, 870878 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.2001.1037" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.2001.1037" aria-label="Article reference 72" data-doi="10.1006/nimg.2001.1037">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Thresholding%20of%20statistical%20maps%20in%20functional%20neuroimaging%20using%20the%20false%20discovery%20rate&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.2001.1037&amp;volume=15&amp;pages=870-878&amp;publication_year=2002&amp;author=Genovese%2CCR&amp;author=Lazar%2CNA&amp;author=Nichols%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. <i>Nat. Methods</i> <b>8</b>, 665670 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmeth.1635" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.1635" aria-label="Article reference 73" data-doi="10.1038/nmeth.1635">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXotV2gs7w%3D" aria-label="CAS reference 73">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20automated%20synthesis%20of%20human%20functional%20neuroimaging%20data&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1635&amp;volume=8&amp;pages=665-670&amp;publication_year=2011&amp;author=Yarkoni%2CT&amp;author=Poldrack%2CRA&amp;author=Nichols%2CTE&amp;author=Essen%2CDC&amp;author=Wager%2CTD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Krishnan, A., Williams, L. J., McIntosh, A. R. &amp; Abdi, H. Partial Least Squares (PLS) methods for neuroimaging: a tutorial and review. <i>Neuroimage</i> <b>56</b>, 455475 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2010.07.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2010.07.034" aria-label="Article reference 74" data-doi="10.1016/j.neuroimage.2010.07.034">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Partial%20Least%20Squares%20%28PLS%29%20methods%20for%20neuroimaging%3A%20a%20tutorial%20and%20review&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2010.07.034&amp;volume=56&amp;pages=455-475&amp;publication_year=2011&amp;author=Krishnan%2CA&amp;author=Williams%2CLJ&amp;author=McIntosh%2CAR&amp;author=Abdi%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">McIntosh, A. R. &amp; Lobaugh, N. J. Partial least squares analysis of neuroimaging data: applications and advances. <i>Neuroimage</i> <b>23</b>, S250S263 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2004.07.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2004.07.020" aria-label="Article reference 75" data-doi="10.1016/j.neuroimage.2004.07.020">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Partial%20least%20squares%20analysis%20of%20neuroimaging%20data%3A%20applications%20and%20advances&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2004.07.020&amp;volume=23&amp;pages=S250-S263&amp;publication_year=2004&amp;author=McIntosh%2CAR&amp;author=Lobaugh%2CNJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Cotney, J. et al. The autism-associated chromatin modifier CHD8 regulates other autism risk genes during human neurodevelopment. <i>Nat. Commun.</i> <b>6</b>, 6404 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms7404" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms7404" aria-label="Article reference 76" data-doi="10.1038/ncomms7404">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXosVCrsL4%3D" aria-label="CAS reference 76">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20autism-associated%20chromatin%20modifier%20CHD8%20regulates%20other%20autism%20risk%20genes%20during%20human%20neurodevelopment&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms7404&amp;volume=6&amp;publication_year=2015&amp;author=Cotney%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Darnell, J. C. et al. FMRP stalls ribosomal translocation on mRNAs linked to synaptic function and autism. <i>Cell</i> <b>146</b>, 247261 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2011.06.013" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2011.06.013" aria-label="Article reference 77" data-doi="10.1016/j.cell.2011.06.013">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXptleksbo%3D" aria-label="CAS reference 77">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=FMRP%20stalls%20ribosomal%20translocation%20on%20mRNAs%20linked%20to%20synaptic%20function%20and%20autism&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2011.06.013&amp;volume=146&amp;pages=247-261&amp;publication_year=2011&amp;author=Darnell%2CJC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Sugathan, A. et al. CHD8 regulates neurodevelopmental pathways associated with autism spectrum disorder in neural progenitors. <i>Proc. Natl Acad. Sci. USA</i> <b>111</b>, E4468E4477 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1405266111" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1405266111" aria-label="Article reference 78" data-doi="10.1073/pnas.1405266111">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhslSrt73P" aria-label="CAS reference 78">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=CHD8%20regulates%20neurodevelopmental%20pathways%20associated%20with%20autism%20spectrum%20disorder%20in%20neural%20progenitors&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1405266111&amp;volume=111&amp;pages=E4468-E4477&amp;publication_year=2014&amp;author=Sugathan%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Greenblatt, E. J. &amp; Spradling, A. C. Fragile X mental retardation 1 gene enhances the translation of large autism-related proteins. <i>Science</i> <b>361</b>, 709712 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aas9963" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aas9963" aria-label="Article reference 79" data-doi="10.1126/science.aas9963">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFagt7zI" aria-label="CAS reference 79">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Fragile%20X%20mental%20retardation%201%20gene%20enhances%20the%20translation%20of%20large%20autism-related%20proteins&amp;journal=Science&amp;doi=10.1126%2Fscience.aas9963&amp;volume=361&amp;pages=709-712&amp;publication_year=2018&amp;author=Greenblatt%2CEJ&amp;author=Spradling%2CAC">
                    Google Scholar</a>
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-018-0281-3?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>This research was supported by grants to E.C. and K.P. (NIMH R01-MH080134 (K.P.), NIMH R01-MH104446 (K.P.), NFAR grant (K.P.), NIMH Autism Center of Excellence grant P50-MH081755 (E.C. and K.P.), NIMH R01-MH036840 (E.C.), NIMH R01-MH110558 (E.C. and N.E.L.), NIMH U01-MH108898 (E.C.), NIDCD R01-DC016385 (E.C., K.P., and M.V.L.), CDMRP AR130409 (E.C.), and Simons Foundation 176540 (E.C.)). The work was additionally supported by an ERC Starting Grant (ERC-2017-STG; 755816) to M.V.L. and a grant from the Brain &amp; Behavior Research Foundation (NARSAD) to T.P. We thank R. Znamirowski, C. Ahrens-Barbeau, S. Solso, K. Campbell, M. Mayo, and J. Young for help with data collection, and S. Spendlove and M. Weinfeld for assistance with clinical characterization of subjects.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Department of Psychology, University of Cyprus, Nicosia, Cyprus</p><p class="c-article-author-affiliation__authors-list">Michael V. Lombardo</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Autism Research Centre, Department of Psychiatry, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Michael V. Lombardo,Varun Warrier&amp;Richard A. I. Bethlehem</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Department of Neuroscience, University of California, San Diego, La Jolla, CA, USA</p><p class="c-article-author-affiliation__authors-list">Tiziano Pramparo,Cynthia Carter Barnes,Linda Lopez,Karen Pierce&amp;Eric Courchesne</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Department of Pediatrics, University of California, San Diego, La Jolla, CA, USA</p><p class="c-article-author-affiliation__authors-list">Vahid Gazestani&amp;Nathan E. Lewis</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Department of Bioengineering, University of California, San Diego, La Jolla, CA, USA</p><p class="c-article-author-affiliation__authors-list">Nathan E. Lewis</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Novo Nordisk Foundation Center for Biosustainability at the University of California, San Diego, La Jolla, CA, USA</p><p class="c-article-author-affiliation__authors-list">Nathan E. Lewis</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Department of Psychiatry, University of California, San Diego, La Jolla, CA, USA</p><p class="c-article-author-affiliation__authors-list">Lisa Eyler</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Veterans Affairs San Diego Healthcare System, San Diego, CA, USA</p><p class="c-article-author-affiliation__authors-list">Lisa Eyler</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Michael_V_-Lombardo-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Michael V. Lombardo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20V.%20Lombardo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20V.%20Lombardo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20V.%20Lombardo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tiziano-Pramparo-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Tiziano Pramparo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Tiziano%20Pramparo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tiziano%20Pramparo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tiziano%20Pramparo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Vahid-Gazestani-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Vahid Gazestani</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Vahid%20Gazestani" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Vahid%20Gazestani" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Vahid%20Gazestani%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Varun-Warrier-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Varun Warrier</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Varun%20Warrier" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Varun%20Warrier" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Varun%20Warrier%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Richard_A__I_-Bethlehem-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Richard A. I. Bethlehem</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Richard%20A.%20I.%20Bethlehem" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Richard%20A.%20I.%20Bethlehem" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Richard%20A.%20I.%20Bethlehem%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Cynthia-Carter_Barnes-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Cynthia Carter Barnes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Cynthia%20Carter%20Barnes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Cynthia%20Carter%20Barnes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Cynthia%20Carter%20Barnes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Linda-Lopez-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Linda Lopez</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Linda%20Lopez" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Linda%20Lopez" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Linda%20Lopez%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Nathan_E_-Lewis-Aff4-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Nathan E. Lewis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nathan%20E.%20Lewis" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nathan%20E.%20Lewis" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nathan%20E.%20Lewis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lisa-Eyler-Aff7-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Lisa Eyler</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lisa%20Eyler" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lisa%20Eyler" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lisa%20Eyler%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Karen-Pierce-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Karen Pierce</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Karen%20Pierce" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Karen%20Pierce" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Karen%20Pierce%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Eric-Courchesne-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Eric Courchesne</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Eric%20Courchesne" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Eric%20Courchesne" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Eric%20Courchesne%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>E.C., K.P., L.E., M.V.L., and T.P. conceived the idea and designed the study. M.V.L. conceived and performed all analyses. T.P., V.G., V.W., R.A.I.B., and N.E.L. aided in data analyses. E.C., K.P., L.E., L.L., and C.C.B. collected data. E.C., K.P., L.E., N.E.L., T.P., and M.V.L. obtained grant funding. M.V.L. and E.C. wrote the manuscript. All authors contributed to editing the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:mvlombardo@gmail.com">Michael V. Lombardo</a> or <a id="corresp-c2" href="mailto:ecourchesne@ucsd.edu">Eric Courchesne</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publishers note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Integrated supplementary information"><div class="c-article-section" id="Sec31-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec31">Integrated supplementary information</h2><div class="c-article-section__content" id="Sec31-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 1 preservation of asd blood l" href="/articles/s41593-018-0281-3/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig6_ESM.jpg">Supplementary Figure 1 Preservation of ASD blood leukocyte modules in ASD frontal and temporal cortical tissue.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Zsummary preservation statistics for all ASD blood leukocyte modules. Zsummary&gt;10 indicates strong preservation, while 2&lt;Zsummary&lt;10 indicates moderate preservation. Non-zero modules M2, M8, and M11 are annotated on this plot, since they show Zsummary&gt;2. M2 has Zsummary=8.1, which indicates high-moderate preservation.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 2 enrichments between non-zer" href="/articles/s41593-018-0281-3/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig7_ESM.jpg">Supplementary Figure 2 Enrichments between non-zero and zero modules with vocal learning, human-specific, and ASD-associated gene lists after broadly expressed genes are removed.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Numbers within the cells indicate the enrichment odds ratio, while the color indicates the -log10(p-value). Cells highlighted in green pass multiple comparison correction at FDR <i>q</i>&lt;0.05.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 3 soft power, tom dendrogram," href="/articles/s41593-018-0281-3/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig8_ESM.jpg">Supplementary Figure 3 Soft power, TOM dendrogram, and module preservation plots for comparisons of each group.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Panel A shows the soft power plot for the main WGCNA analysis including data from all groups. A horizontal red line depicts soft power topology model fit R<sup>2</sup> of 0.9, where the chosen soft power of 16 is located. Panel B shows the TOM dendrogram with modules labeled at the bottom. Panel C shows the module preservation Zsummary statistic for WGCNA analyses run separately on each group in order to show that networks are highly preserved (Zsummary&gt;10) across groups.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 4 plots of module eigengenes " href="/articles/s41593-018-0281-3/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig9_ESM.jpg">Supplementary Figure 4 Plots of module eigengenes for all modules and all groups.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>For all comparisons shown below, the sample sizes are TD n=37 (blue), ASD Good n=40 (pink), ASD Poor n=41 (green). The box in the boxplots indicates the interquartile range (IQR; Q1 indicates the 25<sup>th</sup>, while Q3 indicates the 75<sup>th</sup> percentile) and the whiskers indicate Q1-(1.5*IQR) or Q3+(1.5*IQR). The line within the box represents the median. The y-axis indicates the module eigengene, while the x-axis shows each group.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figure 5 module preservation between" href="/articles/s41593-018-0281-3/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_Fig10_ESM.jpg">Supplementary Figure 5 Module preservation between groups.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>This figure shows the module preservation Zsummary statistic for WGCNA analyses run separately on each group in order to show that networks are highly preserved (Zsummary&gt;10) across groups.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec32-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec32">Supplementary information</h2><div class="c-article-section__content" id="Sec32-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary text and figures" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Text and Figures</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figures 15</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Table 1</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Statistics for linear mixed effect model analysis of clinical trajectory differences. Sheet 1 (Trajectories_ANOVA_allGrps) shows the F-statistic and p-values for main effects of Age and Group and the Age*Group interaction. Sheets 24 are the same F-statistic and p-values for analyses on pairwise group comparisons. Any comparison passing multiple comparison correction at FDR q&lt;0.05 is highlighted in red</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM4_ESM.xlsx" data-supp-info-image="">Supplementary Table 2</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Statistics for ROI analyses. Sheet 1 called ROI_ANOVAstats are the ANOVA statistics from ROI analyses. Sheet 2 called ROI_PairwiseGroupComparisons are the statistics for the pairwise group-comparisons for ROI analyses</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 3" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM5_ESM.xlsx" data-supp-info-image="">Supplementary Table 3</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Differential expression analysis results. Sheet 1 called DE_subtypes_genes shows results from differential expression analysis at the gene-level. The main model evaluates the effect of group (TD, ASD Good, ASD Poor) with an F-test and corresponding p-value. The FDR q-value is also shown for each gene. No gene passes for a statistically significant of group at FDR q&lt;0.05. All other columns in Sheet 1 show the pairwise group comparisons (t-statistic, p-value, and FDR q per each gene). Sheet 2 called DE_subtypes_ME shows results from differentially expression analysis on module eigengenes from the WGCNA analysis. The main model evaluated is the same model as the gene-level DE analysis, testing an effect of group (TD, ASD Good, ASD Poor), and the remaining columns show statistics for each pairwise group comparison. Just like the DE analysis at the gene-level, no modules show statistically significant effects of group surviving FDR q&lt;0.05</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 4" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM6_ESM.xlsx" data-supp-info-image="">Supplementary Table 4</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Statistics from PLS analyses. Sheet 1 (Subtype_PLS) shows statistics for the PLS model that included TD and ASD subtypes (ASD Good and ASD Poor) as separate groups. Sheet 2 (CaseCtrl_PLS) shows statistics for the PLS model that included TD and ASD as the main group distinction (with no distinction for ASD language outcome subtypes). Sheet 3 (Subtype_PLS_LV1_moduleCorr) shows the fMRI-gene co-expression correlations and 95% confidence intervals for each group and each module and indicates which modules are non-zero or not</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM7_ESM.xlsx" data-supp-info-image="">Supplementary Table 5</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Biological process enrichment results for each module. Each sheet shows the results of biological process enrichment tests for each module from MetaCore GeneGO</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 6" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM8_ESM.xlsx" data-supp-info-image="">Supplementary Table 6</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Lists of genes within each module along with module membership statistics for each module. Each row shows a gene and each column shows the module membership value of that gene to each module. The column labeled Module shows the module label assigned to each gene</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 7" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM9_ESM.xlsx" data-supp-info-image="">Supplementary Table 7</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Demographic and clinical behavioral variables summary. This table shows demographic information such as sample size, sex, age, and descriptive statistics (mean with standard deviation in parentheses) for each clinical variable at the intake and outcome assessments</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 8" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-018-0281-3/MediaObjects/41593_2018_281_MOESM10_ESM.xlsx" data-supp-info-image="">Supplementary Table 8</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>ANOVA statistics for cell type surrogate proportion variables estimated with CellCODE. Surrogate proportion variables of different leukocyte cell types were estimated with CellCODE and tested for a group difference with an ANOVA. The table reports the F-statistic and p-value for the group effect from these ANOVAs</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Large-scale%20associations%20between%20the%20leukocyte%20transcriptome%20and%20BOLD%20responses%20to%20speech%20differ%20in%20autism%20early%20language%20outcome%20subtypes&amp;author=Michael%20V.%20Lombardo%20et%20al&amp;contentID=10.1038%2Fs41593-018-0281-3&amp;copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2018-11-26&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-018-0281-3" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-018-0281-3" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Lombardo, M.V., Pramparo, T., Gazestani, V. <i>et al.</i> Large-scale associations between the leukocyte transcriptome and BOLD responses to speech differ in autism early language outcome subtypes.
                    <i>Nat Neurosci</i> <b>21</b>, 16801688 (2018). https://doi.org/10.1038/s41593-018-0281-3</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-018-0281-3?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-04-19">19 April 2018</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-10-24">24 October 2018</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-11-26">26 November 2018</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-12">December 2018</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-018-0281-3</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Language Skill Differences Further Distinguish Social Sub-types in Children with Autism" href="https://doi.org/10.1007/s10803-022-05759-w">
                                        Language Skill Differences Further Distinguish Social Sub-types in Children with Autism
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Weihua Zhao</li><li>Qin Li</li><li>Keith M Kendrick</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Journal of Autism and Developmental Disorders</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Atypical functional connectivity of temporal cortex with precuneus and visual regions may be an early-age signature of ASD" href="https://doi.org/10.1186/s13229-023-00543-8">
                                        Atypical functional connectivity of temporal cortex with precuneus and visual regions may be an early-age signature of ASD
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Yaqiong Xiao</li><li>Teresa H. Wen</li><li>Eric Courchesne</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Molecular Autism</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:A predictive ensemble classifier for the gene expression diagnosis of ASD at ages 1 to 4 years" href="https://doi.org/10.1038/s41380-022-01826-x">
                                        A predictive ensemble classifier for the gene expression diagnosis of ASD at ages 1 to 4 years
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Bokan Bao</li><li>Javad Zahiri</li><li>Eric Courchesne</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Molecular Psychiatry</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Neuroimaging (Resting-State fMRI) and Neuropsychological Characteristics of Non-Converted Patients from a Group at Clinical High Risk for Schizophrenia" href="https://doi.org/10.1007/s11055-023-01538-x">
                                        Neuroimaging (Resting-State fMRI) and Neuropsychological Characteristics of Non-Converted Patients from a Group at Clinical High Risk for Schizophrenia
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>I. S. Lebedeva</li><li>Ya. R. Panikratova</li><li>V. G. Kaleda</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Neuroscience and Behavioral Physiology</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Language Networks in Autism Spectrum Disorder: A systematic review of connectivity-based fMRI studies" href="https://doi.org/10.1007/s40489-023-00382-6">
                                        Language Networks in Autism Spectrum Disorder: A systematic review of connectivity-based fMRI studies
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Caroline Larson</li><li>Hannah R. Thomas</li><li>Inge-Marie Eigsti</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Review Journal of Autism and Developmental Disorders</i> (2023)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0281-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-018-0281-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-018-0281-3;doi=10.1038/s41593-018-0281-3;techmeta=36,38,45,59,61;subjmeta=1373,1689,378,476,631,692,699;kwrd=Autism+spectrum+disorders,Psychiatric+disorders">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=1082332347&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-018-0281-3%26doi%3D10.1038/s41593-018-0281-3%26techmeta%3D36,38,45,59,61%26subjmeta%3D1373,1689,378,476,631,692,699%26kwrd%3DAutism+spectrum+disorders,Psychiatric+disorders">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=1082332347&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-018-0281-3%26doi%3D10.1038/s41593-018-0281-3%26techmeta%3D36,38,45,59,61%26subjmeta%3D1373,1689,378,476,631,692,699%26kwrd%3DAutism+spectrum+disorders,Psychiatric+disorders"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter  what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-018-0281-3&amp;format=js&amp;last_modified=2018-11-26" async></script>
<img src="/i7shop5l/article/s41593-018-0281-3" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>