<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"genetics;neuroscience","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41593-019-0379-2"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Jeroen Mollink","Stephen M. Smith","Lloyd T. Elliott","Michiel Kleinnijenhuis","Marlies Hiemstra","Fidel Alfaro-Almagro","Jonathan Marchini","Anne-Marie van Cappellen van Walsum","Saad Jbabdi","Karla L. Miller"],"publishedAt":1555286400,"publishedAtString":"2019-04-15","title":"The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"22","issue":"5"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure","description":"Microscopic features (that is, microstructure) of axons affect neural circuit activity through characteristics such as conduction speed. To what extent axonal microstructure in white matter relates to functional connectivity (synchrony) between brain regions is largely unknown. Using MRI data in 11,354 subjects, we constructed multivariate models that predict functional connectivity of pairs of brain regions from the microstructural signature of white matter pathways that connect them. Microstructure-derived models provided predictions of functional connectivity that explained 3.5% of cross-subject variance on average (ranging from 1–13%, or r = 0.1–0.36) and reached statistical significance in 90% of the brain regions considered. The microstructure–function relationships were associated with genetic variants, co-located with genes DAAM1 and LPAR1, that have previously been linked to neural development. Our results demonstrate that variation in white matter microstructure predicts a fraction of functional connectivity across individuals, and that this relationship is underpinned by genetic variability in certain brain areas. Mollink et al. establish a link between white matter microstructure and functional connectivity using MRI data from 11,354 individuals in the UK Biobank. GWAS identified genetic associations of these microstructure–function relationships.","datePublished":"2019-04-15T00:00:00Z","dateModified":"2019-04-15T00:00:00Z","pageStart":"809","pageEnd":"819","sameAs":"https://doi.org/10.1038/s41593-019-0379-2","keywords":["Genetics","Neuroscience","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig7_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"22","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Jeroen Mollink","url":"http://orcid.org/0000-0003-1199-324X","affiliation":[{"name":"University of Oxford","address":{"name":"Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"},{"name":"Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center","address":{"name":"Department of Anatomy, Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"email":"jeroen.mollink@radboudumc.nl","@type":"Person"},{"name":"Stephen M. Smith","url":"http://orcid.org/0000-0001-8166-069X","affiliation":[{"name":"University of Oxford","address":{"name":"Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lloyd T. Elliott","affiliation":[{"name":"Simon Fraser University","address":{"name":"Department of Statistics and Actuarial Science, Simon Fraser University, Burnaby, Canada","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Oxford","address":{"name":"Department of Statistics, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michiel Kleinnijenhuis","affiliation":[{"name":"University of Oxford","address":{"name":"Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marlies Hiemstra","affiliation":[{"name":"Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center","address":{"name":"Department of Anatomy, Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Fidel Alfaro-Almagro","affiliation":[{"name":"University of Oxford","address":{"name":"Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jonathan Marchini","url":"http://orcid.org/0000-0003-0610-8322","affiliation":[{"name":"University of Oxford","address":{"name":"Department of Statistics, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Oxford","address":{"name":"The Wellcome Centre for Human Genetics, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anne-Marie van Cappellen van Walsum","affiliation":[{"name":"Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center","address":{"name":"Department of Anatomy, Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Saad Jbabdi","affiliation":[{"name":"University of Oxford","address":{"name":"Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Karla L. Miller","affiliation":[{"name":"University of Oxford","address":{"name":"Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-019-0379-2">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure"/>
    <meta name="dc.source" content="Nature Neuroscience 2019 22:5"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2019-04-15"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2019 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2019 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="Microscopic features (that is, microstructure) of axons affect neural circuit activity through characteristics such as conduction speed. To what extent axonal microstructure in white matter relates to functional connectivity (synchrony) between brain regions is largely unknown. Using MRI data in 11,354&#8201;subjects, we constructed multivariate models that predict functional connectivity of pairs of brain regions from the microstructural signature of white matter pathways that connect them. Microstructure-derived models provided predictions of functional connectivity that explained 3.5% of cross-subject variance on average (ranging from 1&#8211;13%, or r&#8201;=&#8201;0.1&#8211;0.36) and reached statistical significance in 90% of the brain regions considered. The microstructure&#8211;function relationships were associated with genetic variants, co-located with genes DAAM1 and LPAR1, that have previously been linked to neural development. Our results demonstrate that variation in white matter microstructure predicts a fraction of functional connectivity across individuals, and that this relationship is underpinned by genetic variability in certain brain areas. Mollink et al. establish a link between white matter microstructure and functional connectivity using MRI data from 11,354&#8201;individuals in the UK Biobank. GWAS identified genetic associations of these microstructure&#8211;function relationships."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2019-04-15"/>
    <meta name="prism.volume" content="22"/>
    <meta name="prism.number" content="5"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="809"/>
    <meta name="prism.endingPage" content="819"/>
    <meta name="prism.copyright" content="2019 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-019-0379-2"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-019-0379-2"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-019-0379-2.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-019-0379-2"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure"/>
    <meta name="citation_volume" content="22"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_publication_date" content="2019/05"/>
    <meta name="citation_online_date" content="2019/04/15"/>
    <meta name="citation_firstpage" content="809"/>
    <meta name="citation_lastpage" content="819"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-019-0379-2"/>
    <meta name="DOI" content="10.1038/s41593-019-0379-2"/>
    <meta name="size" content="212891"/>
    <meta name="citation_doi" content="10.1038/s41593-019-0379-2"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-019-0379-2&amp;api_key="/>
    <meta name="description" content="Microscopic features (that is, microstructure) of axons affect neural circuit activity through characteristics such as conduction speed. To what extent axonal microstructure in white matter relates to functional connectivity (synchrony) between brain regions is largely unknown. Using MRI data in 11,354&#8201;subjects, we constructed multivariate models that predict functional connectivity of pairs of brain regions from the microstructural signature of white matter pathways that connect them. Microstructure-derived models provided predictions of functional connectivity that explained 3.5% of cross-subject variance on average (ranging from 1&#8211;13%, or r&#8201;=&#8201;0.1&#8211;0.36) and reached statistical significance in 90% of the brain regions considered. The microstructure&#8211;function relationships were associated with genetic variants, co-located with genes DAAM1 and LPAR1, that have previously been linked to neural development. Our results demonstrate that variation in white matter microstructure predicts a fraction of functional connectivity across individuals, and that this relationship is underpinned by genetic variability in certain brain areas. Mollink et al. establish a link between white matter microstructure and functional connectivity using MRI data from 11,354&#8201;individuals in the UK Biobank. GWAS identified genetic associations of these microstructure&#8211;function relationships."/>
    <meta name="dc.creator" content="Mollink, Jeroen"/>
    <meta name="dc.creator" content="Smith, Stephen M."/>
    <meta name="dc.creator" content="Elliott, Lloyd T."/>
    <meta name="dc.creator" content="Kleinnijenhuis, Michiel"/>
    <meta name="dc.creator" content="Hiemstra, Marlies"/>
    <meta name="dc.creator" content="Alfaro-Almagro, Fidel"/>
    <meta name="dc.creator" content="Marchini, Jonathan"/>
    <meta name="dc.creator" content="van Cappellen van Walsum, Anne-Marie"/>
    <meta name="dc.creator" content="Jbabdi, Saad"/>
    <meta name="dc.creator" content="Miller, Karla L."/>
    <meta name="dc.subject" content="Genetics"/>
    <meta name="dc.subject" content="Neuroscience"/>
    <meta name="citation_reference" content="Buzs&#225;ki, G. Rhythms of the Brain (Oxford University Press, 2006)."/>
    <meta name="citation_reference" content="citation_journal_title=Phil. Trans. R. Soc. Lond. B; citation_title=Global relationship between anatomical connectivity and activity propagation in the cerebral cortex; citation_author=R K&#246;tter, FT Sommer; citation_volume=355; citation_publication_date=2000; citation_pages=127-134; citation_doi=10.1098/rstb.2000.0553; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Using human brain lesions to infer function: a relic from a past era in the fMRI age?; citation_author=C Rorden, HO Karnath; citation_volume=5; citation_publication_date=2004; citation_pages=813-819; citation_doi=10.1038/nrn1521; citation_id=CR3"/>
    <meta name="citation_reference" content="Schmahmann, J. D. &amp; Pandya, D. N. in Fiber Pathways of the Brain 1&#8211;37 (Oxford University Press, 2006)."/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=In vivo fiber tractography using DT-MRI data; citation_author=PJ Basser, S Pajevic, C Pierpaoli, J Duda, A Aldroubi; citation_volume=44; citation_publication_date=2000; citation_pages=625-632; citation_doi=10.1002/1522-2594(200010)44:4&lt;625::AID-MRM17&gt;3.0.CO;2-O; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Tractography: where do we go from here?; citation_author=S Jbabdi, H Johansen-Berg; citation_volume=1; citation_publication_date=2011; citation_pages=169-183; citation_doi=10.1089/brain.2011.0033; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Multimodal population brain imaging in the UK Biobank prospective epidemiological study; citation_author=KL Miller; citation_volume=19; citation_publication_date=2016; citation_pages=1523-1536; citation_doi=10.1038/nn.4393; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Resting-state functional connectivity reflects structural connectivity in the default mode network; citation_author=MD Greicius, K Supekar, V Menon, RF Dougherty; citation_volume=19; citation_publication_date=2009; citation_pages=72-78; citation_doi=10.1093/cercor/bhn059; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Functionally linked resting-state networks reflect the underlying structural connectivity architecture of the human brain; citation_author=MP Heuvel, RCW Mandl, RS Kahn, HE Hulshoff Pol; citation_volume=30; citation_publication_date=2009; citation_pages=3127-3141; citation_doi=10.1002/hbm.20737; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Biol.; citation_title=Mapping the structural core of human cerebral cortex; citation_author=P Hagmann; citation_volume=6; citation_publication_date=2008; citation_pages=e159; citation_doi=10.1371/journal.pbio.0060159; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An investigation of functional and anatomical connectivity using magnetic resonance imaging; citation_author=MA Koch, DG Norris, M Hund-Georgiadis; citation_volume=16; citation_publication_date=2002; citation_pages=241-250; citation_doi=10.1006/nimg.2001.1052; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=NMR Biomed.; citation_title=The basis of anisotropic water diffusion in the nervous system - a technical review; citation_author=C Beaulieu; citation_volume=15; citation_publication_date=2002; citation_pages=435-455; citation_doi=10.1002/nbm.782; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Microstructural organization of the cingulum tract and the level of default mode functional connectivity; citation_author=M Heuvel, R Mandl, J Luigjes, H Hulshoff Pol; citation_volume=28; citation_publication_date=2008; citation_pages=10844-10851; citation_doi=10.1523/JNEUROSCI.2964-08.2008; citation_id=CR13"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Human motor corpus callosum: topography, somatotopy, and link between microstructure and function; citation_author=M Wahl; citation_volume=27; citation_publication_date=2007; citation_pages=12132-12138; citation_doi=10.1523/JNEUROSCI.2320-07.2007; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=NODDI: practical in vivo neurite orientation dispersion and density imaging of the human brain; citation_author=H Zhang, T Schneider, CA Wheeler-Kingshott, DC Alexander; citation_volume=61; citation_publication_date=2012; citation_pages=1000-1016; citation_doi=10.1016/j.neuroimage.2012.03.072; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Genome-wide association studies of brain imaging phenotypes in UK Biobank; citation_author=LT Elliott; citation_volume=562; citation_publication_date=2018; citation_pages=210-216; citation_doi=10.1038/s41586-018-0571-7; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Regional variation in interhemispheric coordination of intrinsic hemodynamic fluctuations; citation_author=DE Stark; citation_volume=28; citation_publication_date=2008; citation_pages=13754-13764; citation_doi=10.1523/JNEUROSCI.4544-08.2008; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Stable long-range interhemispheric coordination is supported by direct anatomical projections; citation_author=K Shen; citation_volume=112; citation_publication_date=2015; citation_pages=6473-6478; citation_doi=10.1073/pnas.1503436112; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank; citation_author=F Alfaro-Almagro; citation_volume=166; citation_publication_date=2018; citation_pages=400-424; citation_doi=10.1016/j.neuroimage.2017.10.034; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=J. Magn. Reson.; citation_title=Estimation of the effective self-diffusion tensor from the NMR spin echo; citation_author=PJ Basser, J Mattiello, D LeBihan; citation_volume=103; citation_publication_date=1994; citation_pages=247-254; citation_doi=10.1006/jmrb.1994.1037; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Orthogonal tensor invariants and the analysis of diffusion tensor magnetic resonance images; citation_author=DB Ennis, G Kindlmann; citation_volume=55; citation_publication_date=2006; citation_pages=136-146; citation_doi=10.1002/mrm.20741; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Probabilistic diffusion tractography with multiple fibre orientations: what can we gain?; citation_author=TEJ Behrens, HJ Berg, S Jbabdi, MFS Rushworth, MW Woolrich; citation_volume=34; citation_publication_date=2007; citation_pages=144-155; citation_doi=10.1016/j.neuroimage.2006.09.018; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data; citation_author=SM Smith; citation_volume=31; citation_publication_date=2006; citation_pages=1487-1505; citation_doi=10.1016/j.neuroimage.2006.02.024; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Evaluating fibre orientation dispersion in white matter: comparison of diffusion MRI, histology and polarized light imaging; citation_author=J Mollink; citation_volume=157; citation_publication_date=2017; citation_pages=561-574; citation_doi=10.1016/j.neuroimage.2017.06.001; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The UK Biobank resource with deep phenotyping and genomic data; citation_author=C Bycroft; citation_volume=562; citation_publication_date=2018; citation_pages=203-209; citation_doi=10.1038/s41586-018-0579-z; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=The human genome browser at UCSC; citation_author=WJ Kent; citation_volume=12; citation_publication_date=2002; citation_pages=996-1006; citation_doi=10.1101/gr.229102; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Formin proteins of the DAAM subfamily play a role during axon growth; citation_author=T Matusek; citation_volume=28; citation_publication_date=2008; citation_pages=13310-13319; citation_doi=10.1523/JNEUROSCI.2727-08.2008; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=Dev. Neurobiol.; citation_title=Canonical wnt signaling is required for commissural axon guidance; citation_author=EC Avil&#233;s, ET Stoeckli; citation_volume=76; citation_publication_date=2016; citation_pages=190-208; citation_doi=10.1002/dneu.22307; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Chromosome conformation elucidates regulatory relationships in developing human brain; citation_author=H Won; citation_volume=538; citation_publication_date=2016; citation_pages=523-527; citation_doi=10.1038/nature19847; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions; citation_author=Y Wang; citation_volume=19; citation_publication_date=2018; citation_pages=151; citation_doi=10.1186/s13059-018-1519-9; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Lysophosphatidic acid signaling in the nervous system; citation_author=YC Yung, NC Stoddard, H Mirendil, J Chun; citation_volume=85; citation_publication_date=2015; citation_pages=669-682; citation_doi=10.1016/j.neuron.2015.01.009; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Common genetic variants influence human subcortical brain structures; citation_author=DP Hibar; citation_volume=520; citation_publication_date=2015; citation_pages=224-229; citation_doi=10.1038/nature14101; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Predicting human resting-state functional connectivity from structural connectivity; citation_author=CJ Honey; citation_volume=106; citation_publication_date=2009; citation_pages=2035-2040; citation_doi=10.1073/pnas.0811168106; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Causal effect of disconnection lesions on interhemispheric functional connectivity in rhesus monkeys; citation_author=JX O&#8217;Reilly; citation_volume=110; citation_publication_date=2013; citation_pages=13982-13987; citation_doi=10.1073/pnas.1305062110; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=On the role of the corpus callosum in interhemispheric functional connectivity in humans; citation_author=JL Roland; citation_volume=114; citation_publication_date=2017; citation_pages=13278-13283; citation_doi=10.1073/pnas.1707050114; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Statistical challenges in &#8220;big data&#8221; human neuroimaging; citation_author=SM Smith, TE Nichols; citation_volume=97; citation_publication_date=2018; citation_pages=263-268; citation_doi=10.1016/j.neuron.2017.12.018; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.; citation_title=The personal and clinical utility of polygenic risk scores; citation_author=A Torkamani, NE Wineinger, EJ Topol; citation_volume=19; citation_publication_date=2018; citation_pages=581-590; citation_doi=10.1038/s41576-018-0018-x; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=A surface-based technique for mapping homotopic interhemispheric connectivity:development, characterization, and clinical application; citation_author=SM Tobyne; citation_volume=37; citation_publication_date=2016; citation_pages=2849-2868; citation_doi=10.1002/hbm.23214; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Brain; citation_title=Association fibre pathways of the brain: parallel observations from diffusion spectrum imaging and autoradiography; citation_author=JD Schmahmann; citation_volume=130; citation_publication_date=2007; citation_pages=630-653; citation_doi=10.1093/brain/awl359; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The future of FMRI connectivity; citation_author=SM Smith; citation_volume=62; citation_publication_date=2012; citation_pages=1257-1266; citation_doi=10.1016/j.neuroimage.2012.01.022; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=Am. J. Neuroradiol.; citation_title=Frequencies contributing to functional connectivity in the cerebral cortex in &#8220;resting-state&#8221; data; citation_author=D Cordes; citation_volume=22; citation_publication_date=2001; citation_pages=1326-1333; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Functional and effective connectivity: a review; citation_author=KJ Friston; citation_volume=1; citation_publication_date=2011; citation_pages=13-36; citation_doi=10.1089/brain.2011.0008; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Integr. Neurosci.; citation_title=Quantification of anisotropy and fiber orientation in human brain histological sections; citation_author=MD Budde, J Annese; citation_volume=7; citation_publication_date=2013; citation_pages=3; citation_doi=10.3389/fnint.2013.00003; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Novel genetic loci associated with hippocampal volume; citation_author=DP Hibar; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/ncomms13624; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Wnt/Frizzled activation of Rho regulates vertebrate gastrulation and requires a novel Formin homology protein Daam1; citation_author=R Habas, Y Kato, X He; citation_volume=107; citation_publication_date=2001; citation_pages=843-854; citation_doi=10.1016/S0092-8674(01)00614-6; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Neurosci. Lett.; citation_title=Localization of the Diaphanous-related formin Daam1 to neuronal dendrites; citation_author=SN Salomon, M Haber, KK Murai, RJ Dunn; citation_volume=447; citation_publication_date=2008; citation_pages=62-67; citation_doi=10.1016/j.neulet.2008.09.051; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Res. Dev. Brain Res.; citation_title=Identification of chick and mouse Daam1 and Daam2 genes and their expression patterns in the central nervous system; citation_author=Y Kida, T Shiraishi, T Ogura; citation_volume=153; citation_publication_date=2004; citation_pages=143-150; citation_doi=10.1016/j.devbrainres.2004.07.014; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Int. J. Cancer; citation_title=Novel tumor antigens identified by autologous antibody screening of childhood medulloblastoma cDNA libraries; citation_author=U Behrends; citation_volume=106; citation_publication_date=2003; citation_pages=244-251; citation_doi=10.1002/ijc.11208; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurochem.; citation_title=Anatomical location of LPA1 activation and LPA phospholipid precursors in rodent and human brain; citation_author=E Gonz&#225;lez de San Rom&#225;n; citation_volume=134; citation_publication_date=2015; citation_pages=471-485; citation_doi=10.1111/jnc.13112; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=A new mechanism of nervous system plasticity: activity-dependent myelination; citation_author=RD Fields; citation_volume=16; citation_publication_date=2015; citation_pages=756-767; citation_doi=10.1038/nrn4023; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=FSL; citation_author=M Jenkinson, CF Beckmann, TEJ Behrens, MW Woolrich, SM Smith; citation_volume=62; citation_publication_date=2012; citation_pages=782-790; citation_doi=10.1016/j.neuroimage.2011.09.015; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Improved optimization for the robust and accurate linear registration and motion correction of brain images; citation_author=M Jenkinson, P Bannister, M Brady, S Smith; citation_volume=17; citation_publication_date=2002; citation_pages=825-841; citation_doi=10.1006/nimg.2002.1132; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers; citation_author=G Salimi-Khorshidi; citation_volume=90; citation_publication_date=2014; citation_pages=449-468; citation_doi=10.1016/j.neuroimage.2013.11.046; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An integrated approach to correction for off-resonance effects and subject movement in diffusion MR imaging; citation_author=JLR Andersson, SN Sotiropoulos; citation_volume=125; citation_publication_date=2016; citation_pages=1063-1078; citation_doi=10.1016/j.neuroimage.2015.10.019; citation_id=CR54"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Accelerated microstructure imaging via convex pptimization (AMICO) from diffusion MRI data; citation_author=A Daducci; citation_volume=105; citation_publication_date=2015; citation_pages=32-44; citation_doi=10.1016/j.neuroimage.2014.10.026; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=BIANCA (Brain Intensity AbNormality Classification Algorithm): a new tool for automated segmentation of white matter hyperintensities; citation_author=L Griffanti; citation_volume=141; citation_publication_date=2016; citation_pages=191-205; citation_doi=10.1016/j.neuroimage.2016.07.018; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=Probabilistic independent component analysis for functional magnetic resonance imaging; citation_author=CF Beckmann, SM Smith; citation_volume=23; citation_publication_date=2004; citation_pages=137-152; citation_doi=10.1109/TMI.2003.822821; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Characterization and propagation of uncertainty in diffusion-weighted MR imaging; citation_author=TEJ Behrens; citation_volume=50; citation_publication_date=2003; citation_pages=1077-1088; citation_doi=10.1002/mrm.10609; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Model-based analysis of multishell diffusion MR data for tractography: how to get over fitting problems; citation_author=S Jbabdi, SN Sotiropoulos, AM Savio, M Gra&#241;a, TEJ Behrens; citation_volume=68; citation_publication_date=2012; citation_pages=1846-1855; citation_doi=10.1002/mrm.24204; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Improving alignment in tract-based spatial statistics: evaluation and optimization of image registration; citation_author=M Groot; citation_volume=76; citation_publication_date=2013; citation_pages=400-411; citation_doi=10.1016/j.neuroimage.2013.03.015; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Methodological considerations on tract-based spatial statistics (TBSS); citation_author=M Bach; citation_volume=100; citation_publication_date=2014; citation_pages=358-369; citation_doi=10.1016/j.neuroimage.2014.06.021; citation_id=CR61"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Permutation inference for the general linear model; citation_author=AM Winkler, GR Ridgway, MA Webster, SM Smith, TE Nichols; citation_volume=92; citation_publication_date=2014; citation_pages=381-397; citation_doi=10.1016/j.neuroimage.2014.01.060; citation_id=CR62"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Ball and rackets: inferring fiber fanning from diffusion-weighted MRI; citation_author=SN Sotiropoulos, TEJ Behrens, S Jbabdi; citation_volume=60; citation_publication_date=2012; citation_pages=1412-1425; citation_doi=10.1016/j.neuroimage.2012.01.056; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Examining brain microstructure using structure tensor analysis of histological sections; citation_author=MD Budde, JA Frank; citation_volume=63; citation_publication_date=2012; citation_pages=1-10; citation_doi=10.1016/j.neuroimage.2012.06.042; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Med. Image Anal.; citation_title=MIND: modality independent neighbourhood descriptor for multi-modal deformable registration; citation_author=MP Heinrich; citation_volume=16; citation_publication_date=2012; citation_pages=1423-1435; citation_doi=10.1016/j.media.2012.05.008; citation_id=CR65"/>
    <meta name="citation_author" content="Mollink, Jeroen"/>
    <meta name="citation_author_institution" content="Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author_institution" content="Department of Anatomy, Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands"/>
    <meta name="citation_author" content="Smith, Stephen M."/>
    <meta name="citation_author_institution" content="Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author" content="Elliott, Lloyd T."/>
    <meta name="citation_author_institution" content="Department of Statistics and Actuarial Science, Simon Fraser University, Burnaby, Canada"/>
    <meta name="citation_author_institution" content="Department of Statistics, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author" content="Kleinnijenhuis, Michiel"/>
    <meta name="citation_author_institution" content="Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author" content="Hiemstra, Marlies"/>
    <meta name="citation_author_institution" content="Department of Anatomy, Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands"/>
    <meta name="citation_author" content="Alfaro-Almagro, Fidel"/>
    <meta name="citation_author_institution" content="Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author" content="Marchini, Jonathan"/>
    <meta name="citation_author_institution" content="Department of Statistics, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author_institution" content="The Wellcome Centre for Human Genetics, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author" content="van Cappellen van Walsum, Anne-Marie"/>
    <meta name="citation_author_institution" content="Department of Anatomy, Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands"/>
    <meta name="citation_author" content="Jbabdi, Saad"/>
    <meta name="citation_author_institution" content="Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom"/>
    <meta name="citation_author" content="Miller, Karla L."/>
    <meta name="citation_author_institution" content="Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure"/>
    <meta name="twitter:description" content="Nature Neuroscience - Mollink et al. establish a link between white matter microstructure and functional connectivity using MRI data from 11,354&#8201;individuals in the UK Biobank. GWAS identified..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-019-0379-2"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure - Nature Neuroscience"/>
    <meta property="og:description" content="Mollink et al. establish a link between white matter microstructure and functional connectivity using MRI data from 11,354&#8201;individuals in the UK Biobank. GWAS identified genetic associations of these microstructure&#8211;function relationships."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-019-0379-2;doi=10.1038/s41593-019-0379-2;techmeta=36,43,45,59;subjmeta=208,378,631;kwrd=Genetics,Neuroscience">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1088192061&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-019-0379-2%26doi%3D10.1038/s41593-019-0379-2%26techmeta%3D36,43,45,59%26subjmeta%3D208,378,631%26kwrd%3DGenetics,Neuroscience">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1088192061&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-019-0379-2%26doi%3D10.1038/s41593-019-0379-2%26techmeta%3D36,43,45,59%26subjmeta%3D208,378,631%26kwrd%3DGenetics,Neuroscience"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-019-0379-2'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-019-0379-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2019-04-15">15 April 2019</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jeroen-Mollink-Aff1-Aff2" data-author-popup="auth-Jeroen-Mollink-Aff1-Aff2" data-author-search="Mollink, Jeroen" data-corresp-id="c1">Jeroen Mollink<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-1199-324X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1199-324X</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stephen_M_-Smith-Aff1" data-author-popup="auth-Stephen_M_-Smith-Aff1" data-author-search="Smith, Stephen M.">Stephen M. Smith</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-8166-069X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8166-069X</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lloyd_T_-Elliott-Aff3-Aff4" data-author-popup="auth-Lloyd_T_-Elliott-Aff3-Aff4" data-author-search="Elliott, Lloyd T.">Lloyd T. Elliott</a><sup class="u-js-hide"><a href="#Aff3">3</a>,<a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michiel-Kleinnijenhuis-Aff1" data-author-popup="auth-Michiel-Kleinnijenhuis-Aff1" data-author-search="Kleinnijenhuis, Michiel">Michiel Kleinnijenhuis</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marlies-Hiemstra-Aff2" data-author-popup="auth-Marlies-Hiemstra-Aff2" data-author-search="Hiemstra, Marlies">Marlies Hiemstra</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Fidel-Alfaro_Almagro-Aff1" data-author-popup="auth-Fidel-Alfaro_Almagro-Aff1" data-author-search="Alfaro-Almagro, Fidel">Fidel Alfaro-Almagro</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jonathan-Marchini-Aff4-Aff5" data-author-popup="auth-Jonathan-Marchini-Aff4-Aff5" data-author-search="Marchini, Jonathan">Jonathan Marchini</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-0610-8322"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0610-8322</a></span><sup class="u-js-hide"><a href="#Aff4">4</a>,<a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anne_Marie-van_Cappellen_van_Walsum-Aff2" data-author-popup="auth-Anne_Marie-van_Cappellen_van_Walsum-Aff2" data-author-search="van Cappellen van Walsum, Anne-Marie">Anne-Marie van Cappellen van Walsum</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Saad-Jbabdi-Aff1" data-author-popup="auth-Saad-Jbabdi-Aff1" data-author-search="Jbabdi, Saad">Saad Jbabdi</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#na1">na1</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 10 authors for this article" title="Show all 10 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Karla_L_-Miller-Aff1" data-author-popup="auth-Karla_L_-Miller-Aff1" data-author-search="Miller, Karla L.">Karla L. Miller</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup><sup class="u-js-hide"> <a href="#na1">na1</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 22</b>, <span class="u-visually-hidden">pages </span>809–819 (<span data-test="article-publication-year">2019</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">4733 <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">38 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">120 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-019-0379-2/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/genetics" data-track="click" data-track-action="view subject" data-track-label="link">Genetics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/neuroscience" data-track="click" data-track-action="view subject" data-track-label="link">Neuroscience</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Microscopic features (that is, microstructure) of axons affect neural circuit activity through characteristics such as conduction speed. To what extent axonal microstructure in white matter relates to functional connectivity (synchrony) between brain regions is largely unknown. Using MRI data in 11,354 subjects, we constructed multivariate models that predict functional connectivity of pairs of brain regions from the microstructural signature of white matter pathways that connect them. Microstructure-derived models provided predictions of functional connectivity that explained 3.5% of cross-subject variance on average (ranging from 1–13%, or <i>r</i> = 0.1–0.36) and reached statistical significance in 90% of the brain regions considered. The microstructure–function relationships were associated with genetic variants, co-located with genes <i>DAAM1</i> and <i>LPAR1</i>, that have previously been linked to neural development. Our results demonstrate that variation in white matter microstructure predicts a fraction of functional connectivity across individuals, and that this relationship is underpinned by genetic variability in certain brain areas.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-019-0379-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-019-0379-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-024-46023-2/MediaObjects/41467_2024_46023_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41467-024-46023-2?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41467-024-46023-2">Genetic architecture of the structural connectome
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">04 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Michael Wainberg, Natalie J. Forde, … Shreejoy J. Tripathy</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-021-25184-4/MediaObjects/41467_2021_25184_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41467-021-25184-4?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41467-021-25184-4">Heritability and interindividual variability of regional structure-function coupling
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">12 August 2021</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Zijin Gu, Keith Wakefield Jamison, … Amy Kuceyeski</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41593-023-01376-7/MediaObjects/41593_2023_1376_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41593-023-01376-7?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41593-023-01376-7">Robust estimation of cortical similarity networks from brain MRI
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">17 July 2023</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Isaac Sebenius, Jakob Seidlitz, … Sarah E. Morgan</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711582496,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Communication between brain regions is achieved by axons grouped in white matter pathways. Properties of these structural connections are highly relevant to brain function, often described as functional connectivity. However, it is not simply the presence of a connection, but also the microscopic tissue architecture (that is, microstructure), of white matter that influences brain function. For example, axonal diameter, myelination and length all affect the precise timing of neural signals, which is crucial to synchronizing network dynamics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Buzsáki, G. Rhythms of the Brain (Oxford University Press, 2006)." href="/articles/s41593-019-0379-2#ref-CR1" id="ref-link-section-d121922814e530">1</a></sup>.</p><p>Much of our knowledge about structural connectivity in the brain comes from animals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Kötter, R. &amp; Sommer, F. T. Global relationship between anatomical connectivity and activity propagation in the cerebral cortex. Phil. Trans. R. Soc. Lond. B 355, 127–134 (2000)." href="/articles/s41593-019-0379-2#ref-CR2" id="ref-link-section-d121922814e537">2</a></sup>, human lesions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Rorden, C. &amp; Karnath, H.-O. Using human brain lesions to infer function: a relic from a past era in the fMRI age? Nat. Rev. Neurosci. 5, 813–819 (2004)." href="/articles/s41593-019-0379-2#ref-CR3" id="ref-link-section-d121922814e541">3</a></sup> and postmortem human dissections<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Schmahmann, J. D. &amp; Pandya, D. N. in Fiber Pathways of the Brain 1–37 (Oxford University Press, 2006)." href="/articles/s41593-019-0379-2#ref-CR4" id="ref-link-section-d121922814e545">4</a></sup>. These approaches have relatively high biological specificity and interpretability, but are limited in their ability to characterize inter-individual differences. More recently, diffusion magnetic resonance imaging (dMRI) has emerged as a powerful in vivo tool for studying the brain’s structural connections<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Basser, P. J., Pajevic, S., Pierpaoli, C., Duda, J. &amp; Aldroubi, A. In vivo fiber tractography using DT-MRI data. Magn. Reson. Med. 44, 625–632 (2000)." href="/articles/s41593-019-0379-2#ref-CR5" id="ref-link-section-d121922814e549">5</a></sup>. Although limited in spatial resolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Jbabdi, S. &amp; Johansen-Berg, H. Tractography: where do we go from here? Brain Connect. 1, 169–183 (2011)." href="/articles/s41593-019-0379-2#ref-CR6" id="ref-link-section-d121922814e553">6</a></sup>, dMRI has the unique ability to estimate the trajectories of white matter bundles (that is, tractography) as well as certain microstructural properties of these bundles, through models linking the within-voxel dMRI signal to tissue architecture. An important benefit of dMRI is that it enables us to characterize inter-individual differences, even in large cohorts (for example, UK Biobank<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e558">7</a></sup>). dMRI thus has the potential to relate individual variations in white matter microstructure to differences in brain function, which can also be characterized with MRI.</p><p>Diffusion and functional MRI have been used to investigate structure–function relationships, relating the anatomy of a white matter tract to the functional coupling between the regions it connects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Greicius, M. D., Supekar, K., Menon, V. &amp; Dougherty, R. F. Resting-state functional connectivity reflects structural connectivity in the default mode network. Cereb. Cortex 19, 72–78 (2009)." href="#ref-CR8" id="ref-link-section-d121922814e565">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van den Heuvel, M. P., Mandl, R. C. W., Kahn, R. S. &amp; Hulshoff Pol, H. E. Functionally linked resting-state networks reflect the underlying structural connectivity architecture of the human brain. Hum. Brain Mapp. 30, 3127–3141 (2009)." href="#ref-CR9" id="ref-link-section-d121922814e565_1">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hagmann, P. et al. Mapping the structural core of human cerebral cortex. PLoS Biol. 6, e159 (2008)." href="#ref-CR10" id="ref-link-section-d121922814e565_2">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Koch, M. A., Norris, D. G. &amp; Hund-Georgiadis, M. An investigation of functional and anatomical connectivity using magnetic resonance imaging. Neuroimage 16, 241–250 (2002)." href="/articles/s41593-019-0379-2#ref-CR11" id="ref-link-section-d121922814e568">11</a></sup>. Importantly, these studies relate the macroscopic organization of the network to brain function but did not aim to establish whether the microstructural properties of a white matter tract relate to the functional communication it establishes between brain areas.</p><p>Several studies have demonstrated the potential for dMRI to establish relationships between microstructure and function. For instance, the commonly used metric fractional anisotropy (FA) is a measure of diffusion directionality that is biologically non-specific, being sensitive to many properties including axon density, size and myelination<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Beaulieu, C. The basis of anisotropic water diffusion in the nervous system - a technical review. NMR Biomed. 15, 435–455 (2002)." href="/articles/s41593-019-0379-2#ref-CR12" id="ref-link-section-d121922814e575">12</a></sup>. Mean FA in a given white matter tract has been demonstrated to correlate with strength of functional connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="van den Heuvel, M., Mandl, R., Luigjes, J. &amp; Hulshoff Pol, H. Microstructural organization of the cingulum tract and the level of default mode functional connectivity. J. Neurosci. 28, 10844–10851 (2008)." href="/articles/s41593-019-0379-2#ref-CR13" id="ref-link-section-d121922814e579">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Wahl, M. et al. Human motor corpus callosum: topography, somatotopy, and link between microstructure and function. J. Neurosci. 27, 12132–12138 (2007)." href="/articles/s41593-019-0379-2#ref-CR14" id="ref-link-section-d121922814e582">14</a></sup>. However, these studies focused on the tract connecting a single pair of regions and summarize a tract’s microstructure with a single quantity (for example, FA averaged over the entire tract).</p><p>In this work, we address whether functional connectivity between brain regions is mediated by the microstructure of white matter pathways that connect them. We hypothesize that a data-driven model based on dMRI metrics can predict cross-subject variation in functional connectivity and, more specifically, that this is a general principle that holds across many brain regions and the pathways connecting them. Unlike previous literature, we generated models that capture rich spatial representation of a tract’s microstructural profile (that is, a microstructural signature). In addition to diffusion tensor-based metrics, we incorporated estimates from a more sophisticated biophysical model that aims to provide greater biological specificity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Zhang, H., Schneider, T., Wheeler-Kingshott, C. A. &amp; Alexander, D. C. NODDI: practical in vivo neurite orientation dispersion and density imaging of the human brain. Neuroimage 61, 1000–1016 (2012)." href="/articles/s41593-019-0379-2#ref-CR15" id="ref-link-section-d121922814e590">15</a></sup>. We consider interhemispheric connectivity between pairs of homotopic regions (that is, the homologous region in the two cerebral hemispheres) that are connected by commissural white matter axons that run through the corpus callosum. We built a set of regression models to relate the tract’s microstructural profile to functional connectivity for a large number of paired homotopic regions.</p><p>The models described above linking white matter microstructure to functional connectivity were trained (<i>n</i> = 7,481) and replicated (<i>n</i> = 3,873) on data from the UK Biobank<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e603">7</a></sup>. We show that these models can predict up to 13% of the cross-subject variance in functional connectivity, and demonstrate that the microstructure–function link exists for a large number of brain regions and is highly reproducible. We additionally performed genome-wide association studies (GWASs) to identify single-nucleotide polymorphisms (SNPs) that are significantly associated with functionally relevant microstructure in the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Elliott, L. T. et al. Genome-wide association studies of brain imaging phenotypes in UK Biobank. Nature 562, 210–216 (2018)." href="/articles/s41593-019-0379-2#ref-CR16" id="ref-link-section-d121922814e607">16</a></sup>. The identified SNPs are co-located with genes that have been reported to play an important role in axonal guidance and cortical development.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><p>In our primary analysis, we tested for microstructure–function relationships between homotopic brain regions and the callosal pathways connecting them, using dMRI and resting-state functional MRI (fMRI) data from subjects in the UK Biobank project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e619">7</a></sup>. All subjects were selected based on usable resting-state fMRI and dMRI data, in addition to genetic inclusion criteria (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-019-0379-2#Sec10">Methods</a>). The activity of homotopic region pairs is often synchronized, with high functional connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Stark, D. E. et al. Regional variation in interhemispheric coordination of intrinsic hemodynamic fluctuations. J. Neurosci. 28, 13754–13764 (2008)." href="/articles/s41593-019-0379-2#ref-CR17" id="ref-link-section-d121922814e626">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Shen, K. et al. Stable long-range interhemispheric coordination is supported by direct anatomical projections. Proc. Natl Acad. Sci. USA 112, 6473–6478 (2015)." href="/articles/s41593-019-0379-2#ref-CR18" id="ref-link-section-d121922814e629">18</a></sup>. These pairs are primarily connected through the corpus callosum, the largest commissural pathway in the brain, which is well defined at typical imaging resolutions employed with dMRI.</p><h3 class="c-article__sub-heading" id="Sec3">Functional connectivity</h3><p>We previously conducted a group-average decomposition of resting-state fMRI data using independent component analysis (ICA), which yielded 55 components corresponding to resting-state networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e640">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e643">19</a></sup>. For the work here, more finely grained functional ‘nodes’ were then generated from these components by first splitting each component into its constituent parts for right and left hemispheres, and further splitting if a component still contained non-contiguous brain areas. Homologous regions for the two hemispheres were then identified as nodes with strong similarity, producing 81 homotopic pairs (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig1">1a</a>). Functional connectivity was estimated at the single-subject level by partial correlation of the average blood oxygen level-dependent signal time-series (equivalent to regressing out the time-series from all other regions before calculating pairwise correlations). This gives a connectivity matrix for each subject, which is summarized in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig1">1b</a> as the mean partial correlation across all subjects. Entries in this matrix are ordered first by hemisphere and then by region number, such that interhemispheric connections are given in the upper right and lower left quadrants. Homotopic connections, shown on the diagonals of these quadrants, were found to express on average the strongest connections in the brain, larger than intra-hemispheric or heterotopic interhemispheric connections (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig1">1c</a>), in agreement with previous studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Stark, D. E. et al. Regional variation in interhemispheric coordination of intrinsic hemodynamic fluctuations. J. Neurosci. 28, 13754–13764 (2008)." href="/articles/s41593-019-0379-2#ref-CR17" id="ref-link-section-d121922814e656">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Shen, K. et al. Stable long-range interhemispheric coordination is supported by direct anatomical projections. Proc. Natl Acad. Sci. USA 112, 6473–6478 (2015)." href="/articles/s41593-019-0379-2#ref-CR18" id="ref-link-section-d121922814e659">18</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Definition of homotopic brain regions and dMRI-derived microstructural maps."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Definition of homotopic brain regions and dMRI-derived microstructural maps.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="652"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Functional nodes were defined by applying ICA to the resting-state fMRI data, splitting between the hemispheres and isolating contiguous regions. These were then matched between hemispheres into 81 homotopic pairs, most automatically identified from the same independent component except for 10 manually identified from different components. <b>b</b>, Connectivity between homotopic pairs was estimated by partial correlation of the average time-series of each node, as shown in the connectome (matrix entries are sorted first by hemisphere and then by node number). <b>c</b>, Strength (partial correlation) of different functional connections in the brain, sorted by type. The center line depicts the median correlation coefficient for a specific type of connection; box limits, the 25th and 75th percentiles of the correlation coefficients; the whiskers extend to the most extreme data points excluding outliers (marked with a + symbol). Group-average estimates from <i>n</i> = 11,354 subjects for <i>n</i> = 3,240 connections evaluated. <b>d</b>, Group-averaged microstructure maps derived from the dMRI data. See text for explanation of abbreviations in <b>d</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec4">White matter microstructural signature</h3><p>A range of microstructural features was derived from the dMRI data for the white matter pathway connecting each pair of homotopic gray matter regions. The diffusion tensor model describes the three-dimensional (3D) water displacement profile at each voxel using an ellipsoid<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Basser, P. J., Mattiello, J. &amp; LeBihan, D. Estimation of the effective self-diffusion tensor from the NMR spin echo. J. Magn. Reson. 103, 247–254 (1994)." href="/articles/s41593-019-0379-2#ref-CR20" id="ref-link-section-d121922814e713">20</a></sup>. We extracted estimates of FA, mean diffusivity (MD) and anisotropy mode (MO)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Ennis, D. B. &amp; Kindlmann, G. Orthogonal tensor invariants and the analysis of diffusion tensor magnetic resonance images. Magn. Reson. Med. 55, 136–146 (2006)." href="/articles/s41593-019-0379-2#ref-CR21" id="ref-link-section-d121922814e717">21</a></sup> from this tensor fit. Neurite orientation dispersion and density imaging (NODDI)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Zhang, H., Schneider, T., Wheeler-Kingshott, C. A. &amp; Alexander, D. C. NODDI: practical in vivo neurite orientation dispersion and density imaging of the human brain. Neuroimage 61, 1000–1016 (2012)." href="/articles/s41593-019-0379-2#ref-CR15" id="ref-link-section-d121922814e721">15</a></sup> is a more biologically motivated model that aims to decompose the diffusion signal into an intracellular volume fraction (ICVF) and an isotropic volume fraction (ISOVF), the latter representing interstitial and cerebrospinal fluids. In addition, NODDI estimates an orientation dispersion (OD) index that quantifies the spread of fibers within the intracellular compartment. These dMRI-derived metrics represent an average across thousands of cellular components within each imaging voxel (2 × 2 × 2 mm<sup>3</sup>). Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig1">1d</a> depicts a brain map of each microstructural metric averaged across all subjects. The white matter pathway that connects a given homotopic region pair was identified using probabilistic tractography<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Behrens, T. E. J., Berg, H. J., Jbabdi, S., Rushworth, M. F. S. &amp; Woolrich, M. W. Probabilistic diffusion tractography with multiple fibre orientations: what can we gain? Neuroimage 34, 144–155 (2007)." href="/articles/s41593-019-0379-2#ref-CR22" id="ref-link-section-d121922814e731">22</a></sup> performed on the dMRI data between the regions.</p><h3 class="c-article__sub-heading" id="Sec5">Predicting functional connectivity with microstructure</h3><p>We performed a multiple regression analysis to test whether the microstructural features could predict cross-subject patterns of functional connectivity in the main cohort of 7,481 subjects. For a given homotopic pair of regions, the functional connectivity for all subjects was represented as a vector (<i>N</i><sub>subjects</sub> × 1). To model the spatial patterns of white matter microstructure in a given tract, we begin by constructing a matrix that contains the dMRI-derived metric of interest for every subject (that is, a matrix <i>N</i><sub>subjects</sub> × <i>N</i><sub>voxels</sub>). The included voxels are restricted to the center of the tract of interest using a standard ‘skeletonization’ procedure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Smith, S. M. et al. Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data. Neuroimage 31, 1487–1505 (2006)." href="/articles/s41593-019-0379-2#ref-CR23" id="ref-link-section-d121922814e755">23</a></sup>. Because these microstructure matrices are too large to robustly perform a direct regression (<i>N</i><sub>voxels</sub> = 5,750 ± 4,000), we use principal component analysis (PCA) to reduce the matrix dimensionality. The top 30 principal components (see Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">1</a>) were extracted to serve as a set of regressors, resulting in an <i>N</i><sub>subjects</sub> × 30 regression matrix (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig2">2</a> for an overview). Seven linear models were created for each homotopic pair: one for each of the dMRI-derived metrics (FA, MD, MO, OD, ISOVF, ICVF) and a multimodal approach combining all these microstructural metrics in a single matrix. For the multimodal analysis, the microstructural matrix for each metric was first normalized by its first singular value, and these normalized matrices were concatenated to form a single multimodal matrix (of size <i>N</i><sub>subjects</sub> × 6<i>N</i><sub>voxels</sub>) that was again reduced to include only the top 30 principal components.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Prediction of functional homotopic connectivity from white matter microstructure."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Prediction of functional homotopic connectivity from white matter microstructure.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="545"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p>Between a pair of functionally defined homotopic areas (shown in orange in the brain), probabilistic tractography was performed to delineate the neuronal tract (shown in blue). The white matter skeleton voxels within a tract were stored as rows in a matrix for each subject. To extract the highest cross-subject variance among the tract-based spatial statistics (TBSS) voxels for a given microstructure metric, we performed a dimensionality reduction on this matrix using PCA. The top principal components (<i>n</i> = 30) were fed into a linear regression model as explanatory variables for the functional connectivity between a homotopic pair.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We first tested the hypothesis that dMRI-based microstructure can be used to predict cross-subject variation in functional connectivity consistently across many brain regions. We assessed the statistical significance of each model using permutation testing, performed independently across the homotopic pairs and models, and then corrected for multiple comparisons (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-019-0379-2#Sec10">Methods</a>). The significance (<i>P</i> &lt; 0.05, corrected) is indicated per microstructural metric in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig3">3</a>, and in more detail in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">2</a> as Manhattan plots of corrected <i>P</i> values (family-wise error). The overall regression model was able to predict a statistically significant amount of cross-subject variance in 72–90% of the homotopic brain regions (depending on the dMRI metric). The multimodal microstructure model combining the six dMRI metrics provided a prediction of functional connectivity for the largest number of regions (72, representing 90% of the total brain areas considered). This result is not trivially guaranteed given that this model had the same number of regressors (30) as the other models. These results suggest a general relationship between microstructure and functional connectivity. We can further consider individual regressors (that is, specific principal components). The statistically significant regressors generally correspond to the top principal components (left-most columns in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig3">3</a>). This indicates that the highest cross-subject modes of microstructural variation also explain the most cross-subject variation in functional connectivity. As the regressors reflect the primary modes of variation in the dMRI data but are used to model the fMRI data, this property is not trivially guaranteed. For some regions, no significant associations were found between homotopic functional connectivity and a given microstructure metric. The multimodal microstructure model again resulted in the largest number of significant regressors.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Significant associations between functional connectivity and microstructure of the connecting white matter tract."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Significant associations between functional connectivity and microstructure of the connecting white matter tract.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="863"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>Each row in the matrices represents a homotopic region pair, with each entry a regressor (on the microstructural principal components) of the linear model. Significance of the regressors is color-coded. The graphs depict the number of regions for which a particular rank order principal component yielded a significant regressor (so, for example, in the multimodal models, the first principle component was significant for 28 brain regions). The percentage of homotopic region pairs demonstrating at least one significant regressor is given in the label of each matrix. Statistical significance was determined using permutation testing (two-sided, <i>n</i> = 100,000 permutations, <i>P</i><sub>uncorrected</sub> &lt; 2.9 × 10<sup>−6</sup>, <i>P</i><sub>corrected</sub> &lt; 0.05, corrected for multiple comparisons). See text for definition of abbreviations.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Having established that a microstructure–function link exists in most brain regions, we now consider the apparent strength of this relationship. Effect sizes of the regression models were evaluated in terms of percentage variance explained (equivalent to <i>r</i><sup><i>2</i></sup>) in functional connectivity by the microstructural metrics. The average variance explained across all significant model fits was 3.5% (<i>r</i> = 0.19) for the multimodal model that combines all dMRI microstructure metrics. Substantial variation in variance explained was found across the different brain regions investigated (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig4">4</a>). In the multimodal regression, variance explained was lowest for the middle temporal gyrus (1.1%, <i>r</i> = 0.09) and highest in the posterior cingulate cortex (12.7%, <i>r</i> = 0.36). These effect sizes are mapped back to the 81 homotopic region pairs to visualize how strongly functional connectivity is explained by the underlying microstructure across the cortex (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig5">5a</a>). In addition, <i>z</i>-scores were computed to summarize the overall model fits. The multimodal microstructure regression model yielded on average a higher score than the regressions with any single microstructural metric (<i>z</i> = 12.0, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig4">4</a>), suggesting that the different microstructural metrics explain different variance in functional connectivity. The model incorporating FA shows the highest average <i>z</i>-score of all individual metrics (<i>z</i> = 10.5), although the different metrics perform overall fairly similarly (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig4">4</a>). A list of all brain areas investigated, with their corresponding effect sizes for the multimodal microstructure model, is given in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">1</a>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Percentage variance explained (r2) in the functional connectivity of each homotopic region pair by microstructural metrics derived from the connecting white matter tract in the training cohort (n = 7,481 subjects)."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Percentage variance explained (<i>r</i><sup>2</sup>) in the functional connectivity of each homotopic region pair by microstructural metrics derived from the connecting white matter tract in the training cohort (<i>n</i> = 7,481 subjects).</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="394"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>The box-and-whisker plots at bottom right depict the model performance of each metric in terms of an <i>f</i>-to-<i>z</i> transformed score. The center line depicts the median <i>z</i>-scores across the homotopic region pairs; box limits are the 25th and 75th percentiles of the <i>z</i>-scores; the whiskers extend to the most extreme data points excluding outliers (marked with a + symbol). See text for definition of abbreviations.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Total variance explained by the multimodal regression model in the training and replication cohorts."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Total variance explained by the multimodal regression model in the training and replication cohorts.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="485"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Variance explained mapped onto the brain surface. The maps were smoothed with a 2 mm Gaussian kernel to aid visualization. A similar pattern across the brain was found for the regression models incorporating the individual microstructural metrics. <b>b</b>, Graph showing percentage variance explained for each homotopic region. The model was trained on the main cohort of 7,481 subjects. By applying the regression models trained on the main cohort, we could predict functional connectivity in the replication cohort of 3,873 unseen subjects. The homotopic region numbers on the <i>x</i> axis correspond to the brain areas listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">1</a>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Tensor-based features (FA and MD in particular) have been shown to provide sensitive indicators of changes to tissue microstructure in a broad range of contexts. However, these measures can be influenced by multiple aspects of tissue microstructure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Beaulieu, C. The basis of anisotropic water diffusion in the nervous system - a technical review. NMR Biomed. 15, 435–455 (2002)." href="/articles/s41593-019-0379-2#ref-CR12" id="ref-link-section-d121922814e982">12</a></sup>, making interpretation difficult. We tested whether functional connectivity relates to a microstructure feature with greater biological specificity. We built on our previous work demonstrating quantitative agreement of OD estimates derived from dMRI data and with myelin stains in the same post-mortem human brain tissue<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Mollink, J. et al. Evaluating fibre orientation dispersion in white matter: comparison of diffusion MRI, histology and polarized light imaging. Neuroimage 157, 561–574 (2017)." href="/articles/s41593-019-0379-2#ref-CR24" id="ref-link-section-d121922814e986">24</a></sup>. The callosal OD profile correlated well between the ex vivo imaging data (both MRI and microscopy) and the in vivo dMRI NODDI analyses presented above, with both methods indicating high dispersion on the midline and lower dispersion in the lateral aspects of the callosum (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">3c</a>). Furthermore, OD estimates at the midline of the corpus callosum were able to explain significant variance in interhemispheric functional connectivity (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">3</a>). While the explained variance (0.21% on average) was much less than with the spatially extended microstructure models presented above, the validation against histology demonstrates biological specificity of this particular association.</p><h3 class="c-article__sub-heading" id="Sec6">Model replication</h3><p>We further tested the validity of the above models by applying them to the replication cohort of 3,873 subjects. The data for each replication subject were projected onto the 30 regressors and then multiplied by the regression coefficients estimated from the main cohort to predict that subject’s functional connectivity. That is, the models were applied directly and not retrained on the new subjects. This therefore constitutes a direct prediction of functional connectivity from dMRI data in unseen subjects. As shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig5">5</a>, percentage variance explained was quantitatively very similar among regions (2.5% on average) in the previously unseen subjects, as in the main cohort upon which the model was based.</p><p>Several medial regions demonstrate notably high effect sizes, with the posterior cingulate cortex and the intra-calcarine cortex, in particular, having over 10% variance explained. Regions in the temporal lobe, ventral parts of the frontal lobe and lateral aspect of the occipital lobe demonstrate the lowest variance explained. In addition to the corpus callosum, temporal lobe regions are connected via the anterior commissure. For these regions, we performed additional analyses in which the microstructural signature from the anterior commissure was used to predict functional connectivity (see Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">4</a>). While the anterior commissure microstructure was able to predict functional connectivity, it did not explain the data better than callosal microstructure, nor did a model including both tracts.</p><h3 class="c-article__sub-heading" id="Sec7">Negative control analysis</h3><p>Although the above analyses suggest a general microstructure–function relationship, it is not clear whether these associations are specific to the pathway connecting a given pair of regions, or whether functional connectivity reflects global variance in the microstructural metrics across subjects. A new series of regression analyses were performed similar to those depicted in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig2">2</a>, but instead taking microstructure values from a different ‘wrong’ callosal tract (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig6">6a</a>). From the 81 callosal sub-regions defined above, we selected a subset of 30 distinct tracts with minimal spatial overlap (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">5</a>) for use as control (‘wrong’) tracts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig6">6b</a>). We then assessed whether any of the control tract regressions had similar or better performance compared to the correct tract (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig6">6c</a>). For 70% (60% in the replication cohort) of the homotopic areas, the highest <i>z</i>-score was obtained when the model was performed with the anatomically correct tract; overall, for 81% of brain areas the correct tract ranked among the top three models (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig6">6d</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Negative control analysis."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: Negative control analysis.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="696"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, In the wrong tract approach, general linear model (GLM) analysis was performed with a tract (shown in blue) that does not directly connect between a homotopic pair of interest (shown in orange). <b>b</b>, A total of 30 distinct tracts (12 shown here) were chosen based on minimal spatial overlap between these. <b>c</b>, For each GLM, an <i>f</i>-statistic (across the whole model, with the degrees of freedom for model and error 30 and 7,450, respectively) was calculated and transformed to a <i>z</i>-score to compare the correct tract and 30 wrong tracts. All GLMs in this analysis were derived from the multimodal microstructural information. <b>d</b>, Rows from the matrix in <b>c</b> were sorted in descending order of <i>z</i>-score and labeled according to whether they represented the correct pathway or a different tract. The highest <i>z</i>-scores (left-most column) correspond to the anatomically correct tract in 70% of cases and, overall, the correct tract was in the top three models in 81% of cases.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec8">Genome-wide associations</h3><p>We studied the influence of genetics on the microstructure–function relationships identified above with a series of GWASs. All subjects in this analysis were selected based on recent British ancestry and availability of genotype data that passed the quality control procedures of UK Biobank<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-019-0379-2#ref-CR25" id="ref-link-section-d121922814e1099">25</a></sup>. The target phenotypes used in the GWASs were the cross-subject variation in functional connectivity predicted by the microstructure model (that is, the model fits; see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-019-0379-2#Sec10">Methods</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">6</a>). For each homotopic region pair, the GWAS consisted of a series of univariate correlations of the model fit with 11,734,353 SNPs. These GWASs were fully multiple comparison corrected.</p><p>Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig7">7</a> depicts the association across SNPs for the homotopic pair with the largest variance explained in the multimodal microstructure model (that is, the posterior cingulate cortex). A group of SNPs in chromosome 14 demonstrated a strong association with the microstructure–function phenotype. These SNPs were co-located with the gene Dishevelled Associated Activator of Morphogenesis 1 (<i>DAAM1</i>), while some were also within <i>DAAM1</i>’s promoter region (regulating expression of the gene)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Kent, W. J. et al. The human genome browser at UCSC. Genome Res. 12, 996–1006 (2002)." href="/articles/s41593-019-0379-2#ref-CR26" id="ref-link-section-d121922814e1121">26</a></sup>. The DAAM1 protein plays an important role in the Wnt signaling pathway within the cell, indirectly regulating cell polarity and movement during development. In the central nervous system, this protein has been shown to facilitate the guidance of commissural axons at the embryonic stage in mice and <i>Drosophila</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Matusek, T. et al. Formin proteins of the DAAM subfamily play a role during axon growth. J. Neurosci. 28, 13310–13319 (2008)." href="/articles/s41593-019-0379-2#ref-CR27" id="ref-link-section-d121922814e1127">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Avilés, E. C. &amp; Stoeckli, E. T. Canonical wnt signaling is required for commissural axon guidance. Dev. Neurobiol. 76, 190–208 (2016)." href="/articles/s41593-019-0379-2#ref-CR28" id="ref-link-section-d121922814e1130">28</a></sup>. Expression of the gene Jun n-terminal kinase 1-associated membrane protein (<i>JKAMP</i>) was also regulated by these SNPs, as demonstrated by 3D chromatin interaction data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Won, H. et al. Chromosome conformation elucidates regulatory relationships in developing human brain. Nature 538, 523–527 (2016)." href="/articles/s41593-019-0379-2#ref-CR29" id="ref-link-section-d121922814e1138">29</a></sup> (Virtual 4C<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wang, Y. et al. The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions. Genome Biol. 19, 151 (2018)." href="/articles/s41593-019-0379-2#ref-CR30" id="ref-link-section-d121922814e1142">30</a></sup>). Furthermore, the GWAS revealed many SNPs within the gene lysophosphatidic acid receptor 1 (<i>LPAR</i>1) located in chromosome 9. <i>LPAR1</i> encodes one of the six receptors involved in the lysophosphatidic acid signaling pathway in the cell<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Yung, Y. C., Stoddard, N. C., Mirendil, H. &amp; Chun, J. Lysophosphatidic acid signaling in the nervous system. Neuron 85, 669–682 (2015)." href="/articles/s41593-019-0379-2#ref-CR31" id="ref-link-section-d121922814e1152">31</a></sup>. SNPs co-located with both <i>DAAM1</i> and <i>LPAR1</i> were found for the microstructure–function association of multiple brain areas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig7">7</a> and Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/s41593-019-0379-2#Tab1">1</a>). Detailed Manhattan plots at the location of <i>LPAR1</i> and <i>DAAM1</i> are given in Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">8</a>, respectively. Manhattan plots depicting the GWAS for the microstructure–function model fits of each homotopic region pair in the discovery cohort can be found in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">9</a>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7" data-title="Genome-wide associations with the microstructure–function phenotype (that is, the pattern of functional connectivity that can be predicted from white matter microstructure)."><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 7: Genome-wide associations with the microstructure–function phenotype (that is, the pattern of functional connectivity that can be predicted from white matter microstructure).</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig7_HTML.png?as=webp"><img aria-describedby="Fig7" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_Fig7_HTML.png" alt="figure 7" loading="lazy" width="685" height="642"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p><b>a</b>, The Manhattan plot depicts the associations with each SNP across all chromosomes expressed as the –log<sub>10</sub>(<i>P</i><i>)</i> value. As an example, the genome-wide Manhattan plot is given for the homotopic brain region showing the highest variance explained by the microstructure–function model (<i>n</i> = 7,481 subjects). The strongest association is with an SNP (rs74826997) in chromosome 14 (linear regression, two-sided). <b>b</b>, Single-chromosome Manhattan plots are shown for brain regions that associate with SNPs in either chromosome 9 or 14 that co-located with the genes <i>LPAR1</i> and <i>DAAM1</i>, respectively (linear regression, two-sided). The –log<sub>10</sub>(<i>P</i><i>)</i> values of the SNPs in the discovery GWAS (7,481 subjects) are depicted by blue dots. In an additional cohort of 3,873 subjects, we aimed to replicate the significant hits (black dots in single-chromosome Manhattan plots). The ICA spatial maps of these brain areas are given for each Manhattan plot. The brain area (posterior cingulate cortex, highlighted by max. <i>r</i><sup>2</sup>) corresponds to the genome-wide Manhattan plot in <b>a</b>. A significance threshold is given for a –log<sub>10</sub><i>(</i><i>P</i><i>)</i> value equal to 7.5 corresponding to a <i>P</i> value of ~3 × 10<sup>−8</sup>. The significance threshold for the replication GWAS was determined using Bonferroni correction (<i>P</i> &lt; 1.47 × 10<sup>−4</sup>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-019-0379-2/figures/7" data-track-dest="link:Figure7 Full size image" aria-label="Full size image figure 7" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 Genome-wide associations (linear regression, two-sided) with the microstructure–function phenotype (that is, the pattern of functional connectivity that can be predicted from white matter microstructure).</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/s41593-019-0379-2/tables/1" aria-label="Full size table 1"><span>Full size table</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The GWAS was repeated for subjects in the replication cohort. Rather than using the model prediction approach described above, the multimodal microstructure models were first retrained to better explain functional connectivity with microstructure for these subjects (see Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">10</a> for the effect of retraining). This approach was motivated to make the genetic replication analysis more fully independent of the discovery dataset. Replication GWAS was performed on microstructure–function phenotypes from the homotopic regions showing an association in chromosomes 9 and 14 in the original subjects. Following common practice for replication GWA studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Hibar, D. P. et al. Common genetic variants influence human subcortical brain structures. Nature 520, 224–229 (2015)." href="/articles/s41593-019-0379-2#ref-CR32" id="ref-link-section-d121922814e1795">32</a></sup>, only those SNPs that demonstrated a significant association in the discovery GWAS were tested. For SNPs within the <i>LPAR1</i> gene in chromosome 9, associations with three out of five brain areas were replicated. The SNPs in chromosome 14 corresponding to <i>DAAM1</i> were replicated in two out of three brain areas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig7">7</a>).</p><p>The GWAS results described above used microstructure–function model fits as the target phenotype. These results could simply reflect correlations of these SNPs with both functional connectivity and microstructure. To test for specificity, two additional GWASs were run using the following target phenotypes for each homotopic pair: (1) the functional connectivity that remains unexplained by white matter microstructure (that is, the residuals from each multimodal microstructure–function model) and (2) the first principal component of the multimodal microstructure for the corresponding callosal pathway (that is, the first regressor in the microstructure model). SNPs that were significantly associated with either of these two phenotyes in the discovery and replication cohorts are listed in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">2,</a> <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">3</a>. These GWASs did not find any SNPs co-located with either <i>DAAM1</i> or <i>LPAR1</i> in any homotopic region. This suggests that the relationship to <i>DAAM1</i> and <i>LPAR1</i> is specific to the component of functional connectivity that can be predicted by white matter microstructure. The GWAS associating with the first principal component of multimodal microstructure yielded SNPs within the <i>VCAN</i> gene, which were previously found to associate with ICVF throughout white matter in the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Elliott, L. T. et al. Genome-wide association studies of brain imaging phenotypes in UK Biobank. Nature 562, 210–216 (2018)." href="/articles/s41593-019-0379-2#ref-CR16" id="ref-link-section-d121922814e1833">16</a></sup>.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Discussion</h2><div class="c-article-section__content" id="Sec9-content"><p>Although basic principles relating axonal properties to neural signaling are well established, the degree to which functional connectivity is mediated by microstructural organization at the level of macroscopic tracts is largely unknown. Several studies have related the ‘strength’ and topology of structural connections to functional activity based on fMRI and dMRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Hagmann, P. et al. Mapping the structural core of human cerebral cortex. PLoS Biol. 6, e159 (2008)." href="/articles/s41593-019-0379-2#ref-CR10" id="ref-link-section-d121922814e1846">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Honey, C. J. et al. Predicting human resting-state functional connectivity from structural connectivity. Proc. Natl Acad. Sci. USA 106, 2035–2040 (2009)." href="/articles/s41593-019-0379-2#ref-CR33" id="ref-link-section-d121922814e1849">33</a></sup>, but these studies are uninformative about microstructure. Here we focused on commissural fibers passing through the corpus callosum, a set of connections that can be estimated using MRI both structurally and functionally. Our results are consistent with previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Stark, D. E. et al. Regional variation in interhemispheric coordination of intrinsic hemodynamic fluctuations. J. Neurosci. 28, 13754–13764 (2008)." href="/articles/s41593-019-0379-2#ref-CR17" id="ref-link-section-d121922814e1853">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Shen, K. et al. Stable long-range interhemispheric coordination is supported by direct anatomical projections. Proc. Natl Acad. Sci. USA 112, 6473–6478 (2015)." href="/articles/s41593-019-0379-2#ref-CR18" id="ref-link-section-d121922814e1856">18</a></sup> in that connections between pairs of homotopic areas were shown to be the strongest functional connections in the brain. Furthermore, other studies have demonstrated that severing the corpus callosum reduces or extinguishes interhemispheric functional connectivity, providing evidence that communication between these regions is primarily facilitated by axons running through the callosum<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="O’Reilly, J. X. et al. Causal effect of disconnection lesions on interhemispheric functional connectivity in rhesus monkeys. Proc. Natl Acad. Sci. USA 110, 13982–13987 (2013)." href="/articles/s41593-019-0379-2#ref-CR34" id="ref-link-section-d121922814e1860">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Roland, J. L. et al. On the role of the corpus callosum in interhemispheric functional connectivity in humans. Proc. Natl Acad. Sci. USA 114, 13278–13283 (2017)." href="/articles/s41593-019-0379-2#ref-CR35" id="ref-link-section-d121922814e1863">35</a></sup>.</p><p>In this study, we have demonstrated that white matter microstructure is associated with functional connectivity at the macroscopic level probed by imaging. The majority of brain regions (90%) show statistical evidence for a relationship between white matter microstructure and functional connectivity. Replication in nearly 4,000 subjects demonstrates that the regression models fit in the main cohort have predictive power in unseen subjects.</p><p>On average these models account for 3–4% of the cross-subject variance in a given brain region, with considerable variation across regions—ranging from 1 to 13% of variance explained. It is likely that our results underestimate the true relationships due to methodological limitations. MRI provides indirect estimates of functional connectivity and microstructure. In addition, the model order (linear with 30 regressors), choice of confounds (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">4</a>) and potential for remaining indirect connections (regions not included in the partial correlation) could all lead to unexplained variance. This may be one source of the inter-regional variation, although some true biological variation is also likely. Methodological improvements may well increase the strength of the observed effect sizes. The ability to identify subtle relationships in the order of 1% (<i>r</i> = 0.1) is directly related to our large sample size; indeed, it is common for even smaller effect sizes to be considered valuable in genetic studies providing that replication is demonstrated. The recent advent of population-level imaging requires particular caution in distinguishing between the statistical significance and biological meaningfulness of a given result<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Smith, S. M. &amp; Nichols, T. E. Statistical challenges in “big data” human neuroimaging. Neuron 97, 263–268 (2018)." href="/articles/s41593-019-0379-2#ref-CR36" id="ref-link-section-d121922814e1879">36</a></sup>. Nevertheless, the identification of small effects can be a first step toward aggregate measures with greater explanatory power: for example, polygenic risk factors for disease combining univariate GWAS outcomes with small effect sizes have been enabled by population-level genetics studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Torkamani, A., Wineinger, N. E. &amp; Topol, E. J. The personal and clinical utility of polygenic risk scores. Nat. Rev. Genet. 19, 581–590 (2018)." href="/articles/s41593-019-0379-2#ref-CR37" id="ref-link-section-d121922814e1883">37</a></sup>.</p><p>In both the main and replication cohort, functional connectivity was best explained in regions close to the medial aspect of the brain—for example, the intra-calcarine and posterior cingulate cortices (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig5">5</a>). It should be acknowledged that some of these regions emerged as single contiguous nodes after spatial ICA, where more distal homotopic pairs were separated by other brain structures (see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig1">1</a>). This could reflect fMRI signal blurring between the hemispheres, driving up the apparent functional connectivity. However, it is unclear why this functional connectivity would be better predicted by a completely independent measure of white matter microstructure estimated from diffusion MRI. To investigate this effect further, we conducted a correlation of white matter tract length with variance explained across homotopic region pairs, finding no significant relationship (<i>r</i> = −0.04, <i>P</i> = 0.70).</p><p>For the majority (70%) of the homotopic pairs considered, the strongest model prediction was derived from microstructure in the anatomically correct pathway compared to microstructure obtained from any of the other 30 callosal pathways. This negative control analysis is informative because it establishes that microstructure–function relationships have a degree of regional specificity and do not simply reflect global (brain-wide) inter-individual differences in microstructure and associated function. A similar result has been demonstrated for resting-state functional connectivity between the posterior cingulate and medial frontal cortices, with FA from the correct white matter pathway (cingulum) being more highly correlated than an unrelated tract<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="van den Heuvel, M., Mandl, R., Luigjes, J. &amp; Hulshoff Pol, H. Microstructural organization of the cingulum tract and the level of default mode functional connectivity. J. Neurosci. 28, 10844–10851 (2008)." href="/articles/s41593-019-0379-2#ref-CR13" id="ref-link-section-d121922814e1906">13</a></sup>. Interestingly, for a minority of the brain areas investigated, functional connectivity was better explained by microstructure from a ‘wrong’ white matter tract. Success of control tracts in predicting a given brain region could be driven by either confounded microstructural estimates in the correct tract, partial overlap of tract segmentations or global (brain-wide) variations in microstructure and functional connectivity.</p><p>A general and important confound in our models is ‘partial volume’ (spatial overlap) of tracts. Ideally one would estimate the properties of each axon connecting two brain regions. Instead, dMRI averages within a voxel or region of interest (in our case, a white matter tract). As such, dMRI measurements often mix multiple different white matter bundles: for example, in the centrum semiovale, callosal fibers cross the corticospinal tract and superior longitudinal fasciculus. As a result, our regression models will have included microstructural estimates from other tracts. More advanced modeling to exclude or model these partial volume effects would be valuable to increase specificity without reducing sensitivity.</p><p>Frontotemporal regions were particularly prone to being explained by a control tract. These regions also tend to have lower functional homotopic connectivity, in agreement with previous literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Tobyne, S. M. et al. A surface-based technique for mapping homotopic interhemispheric connectivity:development, characterization, and clinical application. Hum. Brain Mapp. 37, 2849–2868 (2016)." href="/articles/s41593-019-0379-2#ref-CR38" id="ref-link-section-d121922814e1916">38</a></sup>. Such regions may have fractionally less callosal input and be primarily connected to intra-hemispheric brain areas via associations fibers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Schmahmann, J. D. et al. Association fibre pathways of the brain: parallel observations from diffusion spectrum imaging and autoradiography. Brain 130, 630–653 (2007)." href="/articles/s41593-019-0379-2#ref-CR39" id="ref-link-section-d121922814e1920">39</a></sup>. Many temporo-polar regions also have interhemispheric connections via the anterior commissure. We therefore also constructed microstructural models from connections running through the anterior commissure; however, these models did not improve the explained variance in functional connectivity (see Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">4</a>). The effect sizes for models based on the anterior commissure and the corpus callosum varied similarly across brain regions. This may reflect either spatial overlap in the defined tracts close to gray matter, image registration errors or variation in non-relevant variance in functional connectivity across regions (providing a ceiling on the explainable variance). In addition to brain activity, other sources may also contribute to the resting-state signal (see refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Smith, S. M. The future of FMRI connectivity. Neuroimage 62, 1257–1266 (2012)." href="#ref-CR40" id="ref-link-section-d121922814e1927">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cordes, D. et al. Frequencies contributing to functional connectivity in the cerebral cortex in “resting-state” data. Am. J. Neuroradiol. 22, 1326–1333 (2001)." href="#ref-CR41" id="ref-link-section-d121922814e1927_1">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Friston, K. J. Functional and effective connectivity: a review. Brain Connect. 1, 13–36 (2011)." href="/articles/s41593-019-0379-2#ref-CR42" id="ref-link-section-d121922814e1930">42</a></sup> for some excellent reviews).</p><p>Imaging microstructure with dMRI is a rapidly evolving field, including many models only recently developed. The biological interpretation of microstructural metrics is challenging and it is therefore not trivial to decompose the specific contributions of each microstructure parameter in explaining functional connectivity. Each microstructural metric used here explains some unique aspects of the dMRI signal, but they also share some mutual information (for example, both ICVF and OD correlate with FA). The results presented here demonstrate that combining these metrics yields a more comprehensive characterization of the underlying microstructure. In addition, having the rich representation of microstructure along the entire tract better explained functional connectivity than simply using the mean of the tract (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">11</a>).</p><p>To gain further insight into the microscopic tissue features driving the dMRI-derived metrics, evaluation against reference measures such as histology is essential. As such, we demonstrated good correspondence between OD profiles derived from the corpus callosum in ex vivo dMRI and myelin staining<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Mollink, J. et al. Evaluating fibre orientation dispersion in white matter: comparison of diffusion MRI, histology and polarized light imaging. Neuroimage 157, 561–574 (2017)." href="/articles/s41593-019-0379-2#ref-CR24" id="ref-link-section-d121922814e1943">24</a></sup>, providing confidence in the biological meaning of this specific measure. In agreement with histology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Mollink, J. et al. Evaluating fibre orientation dispersion in white matter: comparison of diffusion MRI, histology and polarized light imaging. Neuroimage 157, 561–574 (2017)." href="/articles/s41593-019-0379-2#ref-CR24" id="ref-link-section-d121922814e1947">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Budde, M. D. &amp; Annese, J. Quantification of anisotropy and fiber orientation in human brain histological sections. Front. Integr. Neurosci. 7, 3 (2013)." href="/articles/s41593-019-0379-2#ref-CR43" id="ref-link-section-d121922814e1950">43</a></sup>, the dMRI data used in our study indicate that fibers are more dispersed at the center of the corpus callosum as compared to its lateral aspects. In Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">3</a>, we use this validated measure of fiber dispersion for a simple (single regressor) model to predict functional connectivity. However, this model provided much lower explanatory power (0.21% on average) than the multivariate regression models described above. This is probably because these more comprehensive models capture the spatial richness of microstructure metrics across the white matter tract, demonstrating how pooling of multiple white matter phenotypes can explain more variance in functional connectivity.</p><p>Data richness in the UK Biobank project allowed us to associate genetic variants with the imaging-derived phenotypes in this study. Meta-analyses in the ENIGMA consortium previously revealed genetic variants that were associated with imaging markers such as hippocampal volume<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Hibar, D. P. et al. Novel genetic loci associated with hippocampal volume. Nat. Commun. 8, 13624 (2017)." href="/articles/s41593-019-0379-2#ref-CR44" id="ref-link-section-d121922814e1960">44</a></sup> and other subcortical structures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Hibar, D. P. et al. Common genetic variants influence human subcortical brain structures. Nature 520, 224–229 (2015)." href="/articles/s41593-019-0379-2#ref-CR32" id="ref-link-section-d121922814e1964">32</a></sup> in over 30,000 subjects. ENIGMA pools a vast collection of imaging data from several studies acquired with heterogeneous protocols. It remains to be seen whether the inclusion of a large number of subjects in ENIGMA effectively mitigates this data heterogeneity. In contrast, the UK Biobank project aims to maximize data homogeneity in 100,000 subjects with a common protocol and imaging platform<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e1968">7</a></sup>, which may enable more efficient identification of associations between imaging phenotypes and genetic variants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Elliott, L. T. et al. Genome-wide association studies of brain imaging phenotypes in UK Biobank. Nature 562, 210–216 (2018)." href="/articles/s41593-019-0379-2#ref-CR16" id="ref-link-section-d121922814e1972">16</a></sup>. Combining datasets from the UK Biobank and ENIGMA can be used either to further boost statistical power or can be used separately to replicate discoveries.</p><p>We conducted a GWAS for each homotopic region pair to associate SNPs with the fraction of functional connectivity that was predicted by microstructure. In chromosomes 9 and 14, a group of SNPs was found showing a strong association with the cross-subject pattern of functional connectivity predicted by microstructure for multiple brain areas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-019-0379-2#Fig7">7</a>). Because no SNPs associated with these same genes were found in GWASs relating solely to functional connectivity or microstructure, these associations appear to be unique to the microstructure–function relationship (see Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">2</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">3</a>). For the replication cohort, the SNPs in chromosome 9—co-located with <i>LPAR1—</i>were replicated for three of the five brain areas showing hits in the discovery GWAS. The SNPs in chromosome 14 were replicated in two out of the three brain areas.</p><p>The identified SNPs in chromosomes 9 and 14 are associated with genes that have previously been shown to be important for brain development. The <i>DAAM1</i> gene is expressed in many tissue of the human body and plays an important role in the Wnt signaling pathway<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Habas, R., Kato, Y. &amp; He, X. Wnt/Frizzled activation of Rho regulates vertebrate gastrulation and requires a novel Formin homology protein Daam1. Cell 107, 843–854 (2001)." href="/articles/s41593-019-0379-2#ref-CR45" id="ref-link-section-d121922814e1998">45</a></sup>. In neuronal tissue the DAAM1 protein is primarily found in the shaft of neuronal dendrites<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Salomon, S. N., Haber, M., Murai, K. K. &amp; Dunn, R. J. Localization of the Diaphanous-related formin Daam1 to neuronal dendrites. Neurosci. Lett. 447, 62–67 (2008)." href="/articles/s41593-019-0379-2#ref-CR46" id="ref-link-section-d121922814e2002">46</a></sup>, and in the developing brain it aids axonal guidance in targeting distal brain regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Kida, Y., Shiraishi, T. &amp; Ogura, T. Identification of chick and mouse Daam1 and Daam2 genes and their expression patterns in the central nervous system. Brain Res. Dev. Brain Res. 153, 143–150 (2004)." href="/articles/s41593-019-0379-2#ref-CR47" id="ref-link-section-d121922814e2006">47</a></sup>. Knockout studies in mice and <i>Drosophila</i> have shown deficits in the central nervous system when <i>DAAM1</i> is not expressed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Matusek, T. et al. Formin proteins of the DAAM subfamily play a role during axon growth. J. Neurosci. 28, 13310–13319 (2008)." href="/articles/s41593-019-0379-2#ref-CR27" id="ref-link-section-d121922814e2017">27</a></sup>. In particular, the formation of commissural fibers at an embryonic stage was disturbed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Avilés, E. C. &amp; Stoeckli, E. T. Canonical wnt signaling is required for commissural axon guidance. Dev. Neurobiol. 76, 190–208 (2016)." href="/articles/s41593-019-0379-2#ref-CR28" id="ref-link-section-d121922814e2021">28</a></sup>. Previous work relating cortical thickness to genetic variants also reported SNPs co-located with <i>DAAM1</i> in the cuneus area<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Elliott, L. T. et al. Genome-wide association studies of brain imaging phenotypes in UK Biobank. Nature 562, 210–216 (2018)." href="/articles/s41593-019-0379-2#ref-CR16" id="ref-link-section-d121922814e2028">16</a></sup> (<a href="http://big.stats.ox.ac.uk">http://big.stats.ox.ac.uk</a>). Three-dimensional chromatin data revealed that the SNPs in chromosome 14 also regulate expression of the <i>JKAMP</i> gene<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Won, H. et al. Chromosome conformation elucidates regulatory relationships in developing human brain. Nature 538, 523–527 (2016)." href="/articles/s41593-019-0379-2#ref-CR29" id="ref-link-section-d121922814e2043">29</a></sup>. While diseases associated with <i>JKAMP</i> include medulloblastomas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Behrends, U. et al. Novel tumor antigens identified by autologous antibody screening of childhood medulloblastoma cDNA libraries. Int. J. Cancer 106, 244–251 (2003)." href="/articles/s41593-019-0379-2#ref-CR48" id="ref-link-section-d121922814e2050">48</a></sup>, its exact mechanism in brain development is not well described in the literature. For chromosome 9, several SNPs were located in the <i>LPAR1</i> gene, encoding a receptor involved in the lysophosphatidic acid signaling pathway. These receptors are found on the membranes of most cell types in the central nervous system and have been linked to certain neural processes including, but not limited to, neurogenesis, myelination, microglial activation and astrocyte responses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Yung, Y. C., Stoddard, N. C., Mirendil, H. &amp; Chun, J. Lysophosphatidic acid signaling in the nervous system. Neuron 85, 669–682 (2015)." href="/articles/s41593-019-0379-2#ref-CR31" id="ref-link-section-d121922814e2057">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="González de San Román, E. et al. Anatomical location of LPA1 activation and LPA phospholipid precursors in rodent and human brain. J. Neurochem. 134, 471–485 (2015)." href="/articles/s41593-019-0379-2#ref-CR49" id="ref-link-section-d121922814e2060">49</a></sup>.</p><p>The degree to which functional connectivity between brain regions is mediated by microscopic properties (microstructure) of the white matter pathways is a fundamental question in neuroscience. We demonstrated that a fraction of cross-subject variation in interhemispheric functional connectivity can be predicted from white matter tract microstructure connecting two homotopic regions. Our results suggest that microstructure–function relationships are general (across many brain regions), specific (with the correct tract out-predicting control tracts) and reproducible (as a prediction in a replication cohort). Furthermore, the microstructure–function association was underpinned by genetic variants and, in particular, with SNPs co-located with the genes <i>DAAM1</i> and <i>LPAR1</i>. Attribution of these relationships to specific biological sources, ideally in a causal manner, cannot be achieved with this kind of observational study but would probably require interventional studies in animals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Fields, R. D. A new mechanism of nervous system plasticity: activity-dependent myelination. Nat. Rev. Neurosci. 16, 756–767 (2015)." href="/articles/s41593-019-0379-2#ref-CR50" id="ref-link-section-d121922814e2073">50</a></sup>.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Methods</h2><div class="c-article-section__content" id="Sec10-content"><h3 class="c-article__sub-heading" id="Sec11">Data acquisition and preprocessing</h3><p>We used resting-state functional MRI and diffusion MRI data provided by the UK Biobank project. An extensive overview of the data acquisition protocols and image processing carried out on behalf of UK Biobank can be found elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e2089">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2092">19</a></sup>. Description of post-processing pipelines and acquisition protocols of MRI data in UK Biobank are available at <a href="http://biobank.ctsu.ox.ac.uk/crystal/docs/brain_mri.pdf">http://biobank.ctsu.ox.ac.uk/crystal/docs/brain_mri.pdf</a>. Unless stated otherwise, processing of the MR images was performed using FSL v5.0<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Jenkinson, M., Beckmann, C. F., Behrens, T. E. J., Woolrich, M. W. &amp; Smith, S. M. FSL. Neuroimage 62, 782–790 (2012)." href="/articles/s41593-019-0379-2#ref-CR51" id="ref-link-section-d121922814e2103">51</a></sup>. All imaging data were acquired on a 3 T Siemens Skyra MRI scanner (software platform VD13) using a 32-channel receiving head coil.</p><p>Resting-state fMRI data with 2.4 mm isotropic resolution and whole-brain coverage (field of view, 88 × 88 × 64 matrix) were acquired in a 6 min session (multiband acceleration 8, repetition time (TR) = 0.735 ms, 490 time-points). The functional data were motion corrected<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Jenkinson, M., Bannister, P., Brady, M. &amp; Smith, S. Improved optimization for the robust and accurate linear registration and motion correction of brain images. Neuroimage 17, 825–841 (2002)." href="/articles/s41593-019-0379-2#ref-CR52" id="ref-link-section-d121922814e2110">52</a></sup> and FIX-cleaned<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Salimi-Khorshidi, G. et al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. Neuroimage 90, 449–468 (2014)." href="/articles/s41593-019-0379-2#ref-CR53" id="ref-link-section-d121922814e2114">53</a></sup> to remove physiological noise and image artifacts, before transformation to a 2 mm MNI-template.</p><p>Diffusion MRI data were acquired at 2 mm isotropic resolution achieving whole-brain coverage (field of view, 104 × 104 × 72 matrix) with two diffusion weightings (<i>b</i>-values) (<i>b</i> = 1,000, 2,000 s per mm<sup>2</sup>), with 100 unique gradient directions over the two shells (50 directions per shell). The total acquisition time was 7 min (multiband acceleration 3, echo time (TE)/TR 92/3,600 ms). After eddy current correction of all images<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Andersson, J. L. R. &amp; Sotiropoulos, S. N. An integrated approach to correction for off-resonance effects and subject movement in diffusion MR imaging. Neuroimage 125, 1063–1078 (2016)." href="/articles/s41593-019-0379-2#ref-CR54" id="ref-link-section-d121922814e2129">54</a></sup>, tensor metrics (FA, MD, MO) were calculated from the lower shell (<i>b</i> = 1,000 s per mm<sup>2</sup>) using DTIFIT. Both shells were used to estimate the NODDI model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Zhang, H., Schneider, T., Wheeler-Kingshott, C. A. &amp; Alexander, D. C. NODDI: practical in vivo neurite orientation dispersion and density imaging of the human brain. Neuroimage 61, 1000–1016 (2012)." href="/articles/s41593-019-0379-2#ref-CR15" id="ref-link-section-d121922814e2139">15</a></sup> metrics (ICVF, ISOVF, OD) using the AMICO toolbox<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Daducci, A. et al. Accelerated microstructure imaging via convex pptimization (AMICO) from diffusion MRI data. Neuroimage 105, 32–44 (2015)." href="/articles/s41593-019-0379-2#ref-CR55" id="ref-link-section-d121922814e2143">55</a></sup>.</p><p>While not explicitly used in this study, the UK Biobank imaging protocol includes several structural acquisitions that informed the quality control pipeline and served as registration references for the functional and diffusion data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e2150">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2153">19</a></sup>. T1-weighted structural scans were acquired using a 3D MPRAGE protocol (1.0 × 1.0 × 1.0 mm resolution, matrix 208 × 256 × 256, inversion time (TI)/TR = 880/2,000 ms, in-plane acceleration 2). T2-weighted imaging using fluid-attenuated inversion recovery (FLAIR) contrast provided estimates of white matter hyperintensity (3D SPACE, 1.05 × 1.0 × 1.0 mm resolution, 192 × 256 × 56 matrix, TI/TR = 1,800/5,000 ms, in-plane acceleration 2).</p><h3 class="c-article__sub-heading" id="Sec12">Quality control</h3><p>Quality control was applied at several stages in this study. First, all raw data were subject to a standard preprocessing pipeline<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2165">19</a></sup> that generates several quality control measures. The starting point was the T1-weighted structural scan, which is essential for further processing of the other modalities (for example, the generation of brain masks, tissue segmentations and as a reference for registration). Subjects were excluded if registration to standard space failed, probably due to excessive head motion, atypical structure and/or anatomical abnormalities (for example, large ventricles). The full list of quality control measures derived from the T1-weighted images is given elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2169">19</a></sup>. Based on the T1-weighted anatomical image, 98% of all subjects were deemed suitable for further analysis. Next, the volume of white matter hyperintensities, used as confound variable, derived from the T2-FLAIR images was characterized with BIANCA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Griffanti, L. et al. BIANCA (Brain Intensity AbNormality Classification Algorithm): a new tool for automated segmentation of white matter hyperintensities. Neuroimage 141, 191–205 (2016)." href="/articles/s41593-019-0379-2#ref-CR56" id="ref-link-section-d121922814e2173">56</a></sup>. This feature detects atypical structures and individuals with overt pathology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2177">19</a></sup>. Subjects could additionally be excluded from further analysis on the basis of their dMRI and fMRI data due to bad echoplanar imaging (EPI) distortions, failed registration to T1, extreme bias fields, unusable fieldmaps and/or severe motion artifacts; 87% of the dMRI datasets and 94% of the rfMRI datasets were considered suitable for further analysis based on these quality control measures. Exclusion of the relatively large number of dMRI scans was caused by a change in processing protocol that deemed some early scans unsuitable.</p><p>All subjects selected in this study had both usable dMRI and rfMRI data in addition to suitable genetics data (see section UK Biobank genetics data, below, for more information). This yielded a total of 11,354 subjects: 7,481 in the main cohort and 3,873 in the replication cohort (randomly assigned). Overall, 5,393 females were included, the mean age was 62.8 (s.d. 7.4) years and all subjects had recent British ancestry. No power calculation was needed in advance and we used all available samples. UK Biobank is an observational prospective epidemiological study, and all analyses in our study used all available subjects that fulfilled the criteria described above. Hence there is no equivalent process of randomization that comes into this analysis. For the exact same reason, no blinding step was involved.</p><p>Variations in white matter microstructure and/or functional connectivity may be influenced by certain quality control measures (for example, head motion) in a subtle way that does not require subject exclusion, but which could confound associations. A set of variables of no interest (confounds) are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">4</a> that were used to deconfound the data before modeling (see section Predicting functional connectivity from white matter microstructure, below).</p><h3 class="c-article__sub-heading" id="Sec13">fMRI processing</h3><p>The resting-state fMRI data were fed into an ICA using the MELODIC tool<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Beckmann, C. F. &amp; Smith, S. M. Probabilistic independent component analysis for functional magnetic resonance imaging. IEEE Trans. Med. Imaging 23, 137–152 (2004)." href="/articles/s41593-019-0379-2#ref-CR57" id="ref-link-section-d121922814e2198">57</a></sup> to identify resting-state networks present on average in the whole population. First, data were reduced to 100 dimensions using PCA and then fed into spatial ICA, from which 55 components corresponded to functional regions while the remaining 45 were judged to reflect physiological noise or image artifacts (‘noise’)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 1523–1536 (2016)." href="/articles/s41593-019-0379-2#ref-CR7" id="ref-link-section-d121922814e2202">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2205">19</a></sup>. A functional component was split if it consisted of non-contiguous brain regions, yielding 81 bilateral (homotopic) regions that were further split between the hemispheres to estimate interhemispheric connectivity (see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">1</a>). Average time-series were generated for all ICA components (that is, homotopic areas and noise components) by a spatial regression of the subject’s voxel-wise resting-state fMRI time-series with the ICA spatial maps. Further analyses were performed using the FSLNets toolbox (<a href="https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLNets">https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/FSLNets</a>). The average time-series within a homotopic area was demeaned and ‘cleaned’ by regressing out the time-series from the 45 ‘noise’ component time courses. Functional connectivity was estimated between all pairs of components (2 × 81) by means of partial correlation of the cleaned time-series using Ridge regression with a regularization factor <i>ρ</i> = 1. Partial correlation aims to estimate direct connectivity between two areas by first regressing out all other regions’ time-series before calculating the correlation (that is, established through inversion of the covariance matrix).</p><h3 class="c-article__sub-heading" id="Sec14">dMRI tractography</h3><p>White matter tracts between functional regions were delineated using tractography. Up to three fiber orientations were fitted at each dMRI voxel in a Bayesian approach using bedpostX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Behrens, T. E. J. et al. Characterization and propagation of uncertainty in diffusion-weighted MR imaging. Magn. Reson. Med. 50, 1077–1088 (2003)." href="/articles/s41593-019-0379-2#ref-CR58" id="ref-link-section-d121922814e2230">58</a></sup> modified for multi-shell data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Jbabdi, S., Sotiropoulos, S. N., Savio, A. M., Graña, M. &amp; Behrens, T. E. J. Model-based analysis of multishell diffusion MR data for tractography: how to get over fitting problems. Magn. Reson. Med. 68, 1846–1855 (2012)." href="/articles/s41593-019-0379-2#ref-CR59" id="ref-link-section-d121922814e2234">59</a></sup>. Probabilistic tractography was then performed with the probtrackx2 algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Behrens, T. E. J., Berg, H. J., Jbabdi, S., Rushworth, M. F. S. &amp; Woolrich, M. W. Probabilistic diffusion tractography with multiple fibre orientations: what can we gain? Neuroimage 34, 144–155 (2007)." href="/articles/s41593-019-0379-2#ref-CR22" id="ref-link-section-d121922814e2238">22</a></sup> by generating streamlines from a seed region (5,000 voxel<sup>–1</sup>) in one hemisphere and saving only streamlines that passed through the corpus callosum and terminated in the same region in the contralateral hemisphere. This process was repeated by switching the seed and the target area between hemispheres. The overlap of the identified tracts in this two-step approach was used to generate the mask corresponding to the tract of interest. The tracts were generated for all 81 homotopic pairs (each representing either the seed or the target area) for ten subjects drawn from the UK Biobank dataset. Tracts between a given homotopic pair were then averaged across these subjects and served as a tract mask for all subjects stored in 1 mm MNI-space.</p><h3 class="c-article__sub-heading" id="Sec15">Tract-based spatial statistics</h3><p>Tract-based spatial statistics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Smith, S. M. et al. Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data. Neuroimage 31, 1487–1505 (2006)." href="/articles/s41593-019-0379-2#ref-CR23" id="ref-link-section-d121922814e2253">23</a></sup> was used to align white matter tracts between subjects and extract microstructural information from the tract center (skeleton). The version of TBSS used here employs an optimized nonlinear registration (FNIRT) that avoids the need for the projection step in the original version of TBSS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="de Groot, M. et al. Improving alignment in tract-based spatial statistics: evaluation and optimization of image registration. Neuroimage 76, 400–411 (2013)." href="/articles/s41593-019-0379-2#ref-CR60" id="ref-link-section-d121922814e2257">60</a></sup>. This avoids misalignment problems in which voxels can be projected onto a different tract that is in close proximity, an issue that has been highlighted from the original method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Bach, M. et al. Methodological considerations on tract-based spatial statistics (TBSS). Neuroimage 100, 358–369 (2014)." href="/articles/s41593-019-0379-2#ref-CR61" id="ref-link-section-d121922814e2261">61</a></sup>. The choice of FNIRT-based registration was motivated by its performance compared to other registration algorithms, as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2265">19</a></sup>. We also evaluated the use of DTITK registration, which incorporates the full diffusion tensor to further improve the alignment of dMRI scans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Bach, M. et al. Methodological considerations on tract-based spatial statistics (TBSS). Neuroimage 100, 358–369 (2014)." href="/articles/s41593-019-0379-2#ref-CR61" id="ref-link-section-d121922814e2269">61</a></sup>, finding equivalent performance between the two algorithms (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">12</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">13</a>). The tract reconstructions obtained with probabilistic tracking were used to mask the white matter skeleton voxels for a given homotopic region pair. Microstructural features derived from the diffusion tensor and NODDI fits were extracted from this final tract mask.</p><h3 class="c-article__sub-heading" id="Sec16">Predicting functional connectivity from white matter microstructure</h3><p>We used a multiple linear regression model to predict homotopic functional connectivity from a set of regressors describing the spatial pattern of microstructure along a white matter tract. A rank-based inverse normal transformation was applied to all data to ensure normality. The regression model was constructed separately for each pair of homotopic regions:</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$Y_i = X_i\beta + \varepsilon _i,\;{\mathrm {with} }\;i = 1, \ldots ,n$$</span></div></div><p>Here <i>Y</i><sub><i>i</i></sub> (<i>N</i><sub>subjects</sub> × 1) is a vector that contains the functional connectivity values of all subjects derived from homotopic region <i>i</i> (over <i>n</i> = 81 regions). To build a model using <i>p</i> microstructural regressors, we need to estimate a set of regression coefficients <i>β</i> (<i>p</i> × 1) that describe the relative contribution from the microstructural metrics <i>X</i><sub><i>i</i></sub> (<i>N</i><sub>subjects</sub> × <i>p</i>) along the white matter tract.</p><p>The regressors were derived in two stages. First, the microstructural metrics were extracted from the TBSS voxels (white matter skeleton) corresponding to the tract of interest for every subject, yielding a matrix <i>X</i>°<sub><i>i</i></sub> (<i>N</i><sub>subject</sub><sub><i>s</i></sub> × <i>N</i><sub>voxels</sub>). As the matrix <i>X</i>°<sub><i>i</i></sub> is very large, a direct regression with functional connectivity is ill-conditioned. We therefore performed a dimensionality reduction on <i>X</i><sup>†</sup><sub><i>I</i></sub> to derive a set of regressors reflecting the primary modes of variation of a given microstructural metric across space for the cohort of subjects. The microstructural matrices were first demeaned, and then a singular value decomposition was computed from matrix <i>X</i>°<sub><i>i</i></sub>. The top <i>p</i> components were retained, yielding matrix <i>X</i><sub><i>i</i></sub> (<i>N</i><sub>subjects</sub> × <i>p</i>). In practice, <i>p</i> was set to 30 principal components that approximately corresponded to a transition in the spectrum of singular values in terms of variance explained. This provides a somewhat conservative model order below the point around <i>p</i> = 100, at which variance explained roughly tracked noise singular vectors (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">1</a>) and linear regression is prone to overfitting.</p><p>Matrices <i>X</i><sub><i>i</i></sub> were constructed for each of the microstructure metrics separately, yielding six single-metric linear regression models per homotopic region. In addition, a multimodal regression model was created that combined across all microstructure metrics. For the multimodal regression, all raw microstructure matrices (<i>X</i><sup>†</sup><sub><i>i</i></sub>) were demeaned and normalized through division by their first singular value to ensure a comparable range of values. The six normalized matrices were then concatenated along the voxel dimension (<i>N</i><sub>subjects</sub> × 6<i>N</i><sub>voxels</sub>), and this matrix was reduced to the top 30 principal components as described above.</p><p>We defined a set of 64 confound variables of no interest that might bias the estimated regressors by correlating with the estimated microstructural measures (for example, through artifacts such as partial volume). An overview of all confound variables is given in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">4</a>. The confound variables were regressed out of the functional and microstructural data before fitting the regression models.</p><h3 class="c-article__sub-heading" id="Sec17">Statistical analysis</h3><p>Statistical significance of the regression models was assessed by means of permutation testing, evaluating each regressor using a <i>t</i>-statistic. A null distribution was constructed for each model <i>t</i>-statistic by randomly permuting the functional connectivity values across subjects (100,000 permutations). Because multiple models were evaluated, correction for family-wise error is also essential, where we corrected along three different dimensions of multiple comparison, as follows. First, we tested multiple hypotheses in each model, that is, which of the 30 microstructural principal components explained a significant amount of functional connectivity. Second, the models were applied to each of the 81 homotopic region pairs. Finally, a total of seven models (six individual microstructural models and one multimodal model) were evaluated for each homotopic pair. Following the approach demonstrated in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Winkler, A. M., Ridgway, G. R., Webster, M. A., Smith, S. M. &amp; Nichols, T. E. Permutation inference for the general linear model. Neuroimage 92, 381–397 (2014)." href="/articles/s41593-019-0379-2#ref-CR62" id="ref-link-section-d121922814e2533">62</a></sup>, a maximum <i>t</i>-statistic null distribution across all dimensions (regressor, regions and models) was generated from the permuted <i>t</i>-statistics. From this maximum <i>t</i>-statistics null distribution, a corrected <i>P</i> value was estimated for each of the non-permuted <i>t</i>-statistics. Furthermore, an <i>f</i>-statistic was computed to judge the overall performance of each regression model (degrees of freedom model and error, 30 and 7,450, respectively). The <i>f</i>-statistics were converted to <i>z</i>-scores. Finally, the effect size of the regression models was expressed in terms of percentage variance explained (equivalent to <i>r</i><sup><i>2</i></sup>), describing the strength of the relationship between microstructure and functional connectivity.</p><h3 class="c-article__sub-heading" id="Sec18">Negative control analysis</h3><p>The statistical tests described above determine whether there is a relationship between functional connectivity in a given brain region and the microstructure in the white matter pathway that connects them. However, this does not provide any insight into whether these relationships are specific: for example, microstructure and function could correlate at the whole-brain level. In this case, a regression model could indicate a statistically significant relationship even when using a white matter pathway that does not connect a given homotopic pair. Such a relationship could still be biologically meaningful, but the interpretation would change (for example, demonstrating that individual brains vary globally from hypo- to hyperconnected).</p><p>To test this, a negative control analysis was performed to evaluate the uniqueness of the microstructure–function relationships. From the 81 tracts in our study, a subset of 30 with minimal mutual overlap were selected as canonical control (‘wrong’) tracts. To identify the set of canonical control tracts, the Dice similarity index was computed among all tracts to quantify spatial overlap. Using <i>k</i>-means clustering (<i>k</i> = 3 clusters), a cluster of tracts with the lowest average similarity indices was selected (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM1">5</a>).</p><p>The regression models were then re-evaluated for each homotopic area using the control tracts, rather than microstructure from the anatomically correct tract, for the homotopic pair of interest. If, for a homotopic area, the anatomically correct tract was among the control tracts, an additional control tract was selected. To summarize, the regression models of the homotopic regions were performed once for microstructure from the correct tract and 30 times for the control tracts. Comparison between the correct and control tract analyses was conducted using the <i>f</i>-statistic converted to <i>z</i>-scores.</p><h3 class="c-article__sub-heading" id="Sec19">UK Biobank genetics data</h3><p>The GWASs were performed using the BGENIE software<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-019-0379-2#ref-CR25" id="ref-link-section-d121922814e2606">25</a></sup>. Acquisition and processing steps of the genetics dataset for all subjects in the UK Biobank project can be found in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-019-0379-2#ref-CR25" id="ref-link-section-d121922814e2610">25</a></sup>. For the discovery cohort, we began with the set of 12,623 brain-imaged UK Biobank subjects for whom data were released in July 2017. As in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Elliott, L. T. et al. Genome-wide association studies of brain imaging phenotypes in UK Biobank. Nature 562, 210–216 (2018)." href="/articles/s41593-019-0379-2#ref-CR16" id="ref-link-section-d121922814e2614">16</a></sup>, to avoid confounding effects that might arise from population structure or environmental effects, we selected a subset of 11,354 unrelated subjects with recent British ancestry. Ancestry was determined using sample quality control information provided by UK Biobank<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-019-0379-2#ref-CR25" id="ref-link-section-d121922814e2618">25</a></sup>. We then filtered the genetic data to remove SNPs with minor allele frequency &lt;0.01% or a Hardy–Weinberg equilibrium <i>P</i> value of &lt;10<sup>−7</sup>, yielding a total of 11,734,353 SNPs distributed across the 22 autosomes. Not all of the UK Biobank subjects who underwent brain imaging have usable data with a given MRI modality. All 11,354 unrelated samples were subjects which had usable dMRI and fMRI data according to previous quality control<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. Neuroimage 166, 400–424 (2018)." href="/articles/s41593-019-0379-2#ref-CR19" id="ref-link-section-d121922814e2628">19</a></sup>. Subjects were assigned to the discovery and replication cohorts in a fashion similar to the MRI analyses.</p><h3 class="c-article__sub-heading" id="Sec20">Ex vivo MRI and histology data</h3><p>Microscopy and MRI data from three ex vivo corpus callosum specimens were acquired and processed as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Mollink, J. et al. Evaluating fibre orientation dispersion in white matter: comparison of diffusion MRI, histology and polarized light imaging. Neuroimage 157, 561–574 (2017)." href="/articles/s41593-019-0379-2#ref-CR24" id="ref-link-section-d121922814e2640">24</a></sup>. In brief, formalin-fixed human brain tissue sections were scanned on a preclinical 9.4 T Varian MRI system. Diffusion MRI was performed by spin-echo sequence with TE = 29 ms and TR = 2.4 s. Two shells were acquired (<i>b</i> = 2,500 s per mm<sup>2</sup> and <i>b</i> = 5,000 s per mm<sup>2</sup>), each with 120 gradient directions and 0.4 mm isotropic resolution. Eight images with no diffusion weighting were acquired. A parametric model was fit to the dMRI signals from the <i>b</i> = 5,000 s per mm<sup>2</sup> dataset to obtain orientation dispersion estimates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Sotiropoulos, S. N., Behrens, T. E. J. &amp; Jbabdi, S. Ball and rackets: inferring fiber fanning from diffusion-weighted MRI. Neuroimage 60, 1412–1425 (2012)." href="/articles/s41593-019-0379-2#ref-CR63" id="ref-link-section-d121922814e2660">63</a></sup>.</p><p>Following MR scanning, the specimens were histologically sectioned and immunohistochemically stained for myelin (proteolipid protein). The sections were digitized and we obtained fiber orientation estimates at each pixel using structure tensor analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Budde, M. D. &amp; Frank, J. A. Examining brain microstructure using structure tensor analysis of histological sections. Neuroimage 63, 1–10 (2012)." href="/articles/s41593-019-0379-2#ref-CR64" id="ref-link-section-d121922814e2667">64</a></sup>. From a two-dimensional local neighborhood (0.4 × 0.4 mm) corresponding to the size of an MRI voxel, a fiber orientation distribution was computed from which orientation dispersion was derived. After registration of dMRI and microscopy data to the same image space<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Heinrich, M. P. et al. MIND: modality independent neighbourhood descriptor for multi-modal deformable registration. Med. Image Anal. 16, 1423–1435 (2012)." href="/articles/s41593-019-0379-2#ref-CR65" id="ref-link-section-d121922814e2671">65</a></sup>, dispersion estimates were compared against each other in the corpus callosum.</p><h3 class="c-article__sub-heading" id="Sec21">Ethics and informed consent</h3><p>All participants in the UK Biobank project signed an informed consent, which is controlled by a dedicated Ethics and Guidance Council (<a href="http://www.ukbiobank.ac.uk/ethics">http://www.ukbiobank.ac.uk/ethics</a>). The Ethics and Governance Framework can be found at <a href="http://www.ukbiobank.ac.uk/wp-content/uploads/2011/05/EGF20082.pdf">http://www.ukbiobank.ac.uk/wp-content/uploads/2011/05/EGF20082.pdf</a>. Institutional Review Board approval, also from the North West Multi-center Research Ethics Committee, was obtained for the Ethics and Governance Framework.</p><h3 class="c-article__sub-heading" id="Sec22">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-019-0379-2#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                </div>
            

            <div>
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>All source data (including raw and processed brain imaging data and genetics data) are available from UK Biobank via their standard data access procedure (see <a href="http://www.ukbiobank.ac.uk/register-apply">http://www.ukbiobank.ac.uk/register-apply</a>).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>The image processing pipelines of the MRI data in the UK Biobank project can be found at <a href="http://www.fmrib.ox.ac.uk/ukbiobank">http://www.fmrib.ox.ac.uk/ukbiobank</a>. Custom-written Matlab code including the microstructure–function modeling is freely available at <a href="https://users.fmrib.ox.ac.uk/~jmollink/Biobank/Biobank.html">https://users.fmrib.ox.ac.uk/~jmollink/Biobank/Biobank.html</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Buzsáki, G. <i>Rhythms of the Brain</i> (Oxford University Press, 2006).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Kötter, R. &amp; Sommer, F. T. Global relationship between anatomical connectivity and activity propagation in the cerebral cortex. <i>Phil. Trans. R. Soc. Lond. B</i> <b>355</b>, 127–134 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rstb.2000.0553" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2000.0553" aria-label="Article reference 2" data-doi="10.1098/rstb.2000.0553">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20relationship%20between%20anatomical%20connectivity%20and%20activity%20propagation%20in%20the%20cerebral%20cortex&amp;journal=Phil.%20Trans.%20R.%20Soc.%20Lond.%20B&amp;doi=10.1098%2Frstb.2000.0553&amp;volume=355&amp;pages=127-134&amp;publication_year=2000&amp;author=K%C3%B6tter%2CR&amp;author=Sommer%2CFT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Rorden, C. &amp; Karnath, H.-O. Using human brain lesions to infer function: a relic from a past era in the fMRI age? <i>Nat. Rev. Neurosci.</i> <b>5</b>, 813–819 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn1521" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn1521" aria-label="Article reference 3" data-doi="10.1038/nrn1521">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20human%20brain%20lesions%20to%20infer%20function%3A%20a%20relic%20from%20a%20past%20era%20in%20the%20fMRI%20age%3F&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn1521&amp;volume=5&amp;pages=813-819&amp;publication_year=2004&amp;author=Rorden%2CC&amp;author=Karnath%2CHO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Schmahmann, J. D. &amp; Pandya, D. N. in <i>Fiber Pathways of the Brain</i> 1–37 (Oxford University Press, 2006).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Basser, P. J., Pajevic, S., Pierpaoli, C., Duda, J. &amp; Aldroubi, A. In vivo fiber tractography using DT-MRI data. <i>Magn. Reson. Med.</i> <b>44</b>, 625–632 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/1522-2594(200010)44:4<625::AID-MRM17&gt;3.0.CO;2-O" data-track-action="article reference" href="https://doi.org/10.1002%2F1522-2594%28200010%2944%3A4%3C625%3A%3AAID-MRM17%3E3.0.CO%3B2-O" aria-label="Article reference 5" data-doi="10.1002/1522-2594(200010)44:4<625::AID-MRM17&gt;3.0.CO;2-O">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3cvnvFCitA%3D%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20fiber%20tractography%20using%20DT-MRI%20data&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2F1522-2594%28200010%2944%3A4%3C625%3A%3AAID-MRM17%3E3.0.CO%3B2-O&amp;volume=44&amp;pages=625-632&amp;publication_year=2000&amp;author=Basser%2CPJ&amp;author=Pajevic%2CS&amp;author=Pierpaoli%2CC&amp;author=Duda%2CJ&amp;author=Aldroubi%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Jbabdi, S. &amp; Johansen-Berg, H. Tractography: where do we go from here? <i>Brain Connect.</i> <b>1</b>, 169–183 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1089/brain.2011.0033" data-track-action="article reference" href="https://doi.org/10.1089%2Fbrain.2011.0033" aria-label="Article reference 6" data-doi="10.1089/brain.2011.0033">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Tractography%3A%20where%20do%20we%20go%20from%20here%3F&amp;journal=Brain%20Connect.&amp;doi=10.1089%2Fbrain.2011.0033&amp;volume=1&amp;pages=169-183&amp;publication_year=2011&amp;author=Jbabdi%2CS&amp;author=Johansen-Berg%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. <i>Nat. Neurosci.</i> <b>19</b>, 1523–1536 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4393" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4393" aria-label="Article reference 7" data-doi="10.1038/nn.4393">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFenu7nE" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Multimodal%20population%20brain%20imaging%20in%20the%20UK%20Biobank%20prospective%20epidemiological%20study&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4393&amp;volume=19&amp;pages=1523-1536&amp;publication_year=2016&amp;author=Miller%2CKL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Greicius, M. D., Supekar, K., Menon, V. &amp; Dougherty, R. F. Resting-state functional connectivity reflects structural connectivity in the default mode network. <i>Cereb. Cortex</i> <b>19</b>, 72–78 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhn059" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhn059" aria-label="Article reference 8" data-doi="10.1093/cercor/bhn059">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting-state%20functional%20connectivity%20reflects%20structural%20connectivity%20in%20the%20default%20mode%20network&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhn059&amp;volume=19&amp;pages=72-78&amp;publication_year=2009&amp;author=Greicius%2CMD&amp;author=Supekar%2CK&amp;author=Menon%2CV&amp;author=Dougherty%2CRF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">van den Heuvel, M. P., Mandl, R. C. W., Kahn, R. S. &amp; Hulshoff Pol, H. E. Functionally linked resting-state networks reflect the underlying structural connectivity architecture of the human brain. <i>Hum. Brain Mapp.</i> <b>30</b>, 3127–3141 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/hbm.20737" data-track-action="article reference" href="https://doi.org/10.1002%2Fhbm.20737" aria-label="Article reference 9" data-doi="10.1002/hbm.20737">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Functionally%20linked%20resting-state%20networks%20reflect%20the%20underlying%20structural%20connectivity%20architecture%20of%20the%20human%20brain&amp;journal=Hum.%20Brain%20Mapp.&amp;doi=10.1002%2Fhbm.20737&amp;volume=30&amp;pages=3127-3141&amp;publication_year=2009&amp;author=Heuvel%2CMP&amp;author=Mandl%2CRCW&amp;author=Kahn%2CRS&amp;author=Hulshoff%20Pol%2CHE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Hagmann, P. et al. Mapping the structural core of human cerebral cortex. <i>PLoS Biol.</i> <b>6</b>, e159 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.0060159" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.0060159" aria-label="Article reference 10" data-doi="10.1371/journal.pbio.0060159">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20the%20structural%20core%20of%20human%20cerebral%20cortex&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.0060159&amp;volume=6&amp;publication_year=2008&amp;author=Hagmann%2CP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Koch, M. A., Norris, D. G. &amp; Hund-Georgiadis, M. An investigation of functional and anatomical connectivity using magnetic resonance imaging. <i>Neuroimage</i> <b>16</b>, 241–250 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.2001.1052" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.2001.1052" aria-label="Article reference 11" data-doi="10.1006/nimg.2001.1052">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20investigation%20of%20functional%20and%20anatomical%20connectivity%20using%20magnetic%20resonance%20imaging&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.2001.1052&amp;volume=16&amp;pages=241-250&amp;publication_year=2002&amp;author=Koch%2CMA&amp;author=Norris%2CDG&amp;author=Hund-Georgiadis%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Beaulieu, C. The basis of anisotropic water diffusion in the nervous system - a technical review. <i>NMR Biomed.</i> <b>15</b>, 435–455 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/nbm.782" data-track-action="article reference" href="https://doi.org/10.1002%2Fnbm.782" aria-label="Article reference 12" data-doi="10.1002/nbm.782">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20basis%20of%20anisotropic%20water%20diffusion%20in%20the%20nervous%20system%20-%20a%20technical%20review&amp;journal=NMR%20Biomed.&amp;doi=10.1002%2Fnbm.782&amp;volume=15&amp;pages=435-455&amp;publication_year=2002&amp;author=Beaulieu%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">van den Heuvel, M., Mandl, R., Luigjes, J. &amp; Hulshoff Pol, H. Microstructural organization of the cingulum tract and the level of default mode functional connectivity. <i>J. Neurosci.</i> <b>28</b>, 10844–10851 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2964-08.2008" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2964-08.2008" aria-label="Article reference 13" data-doi="10.1523/JNEUROSCI.2964-08.2008">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Microstructural%20organization%20of%20the%20cingulum%20tract%20and%20the%20level%20of%20default%20mode%20functional%20connectivity&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2964-08.2008&amp;volume=28&amp;pages=10844-10851&amp;publication_year=2008&amp;author=Heuvel%2CM&amp;author=Mandl%2CR&amp;author=Luigjes%2CJ&amp;author=Hulshoff%20Pol%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Wahl, M. et al. Human motor corpus callosum: topography, somatotopy, and link between microstructure and function. <i>J. Neurosci.</i> <b>27</b>, 12132–12138 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2320-07.2007" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2320-07.2007" aria-label="Article reference 14" data-doi="10.1523/JNEUROSCI.2320-07.2007">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtlartLjM" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20motor%20corpus%20callosum%3A%20topography%2C%20somatotopy%2C%20and%20link%20between%20microstructure%20and%20function&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2320-07.2007&amp;volume=27&amp;pages=12132-12138&amp;publication_year=2007&amp;author=Wahl%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Zhang, H., Schneider, T., Wheeler-Kingshott, C. A. &amp; Alexander, D. C. NODDI: practical in vivo neurite orientation dispersion and density imaging of the human brain. <i>Neuroimage</i> <b>61</b>, 1000–1016 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.03.072" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.03.072" aria-label="Article reference 15" data-doi="10.1016/j.neuroimage.2012.03.072">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=NODDI%3A%20practical%20in%20vivo%20neurite%20orientation%20dispersion%20and%20density%20imaging%20of%20the%20human%20brain&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.03.072&amp;volume=61&amp;pages=1000-1016&amp;publication_year=2012&amp;author=Zhang%2CH&amp;author=Schneider%2CT&amp;author=Wheeler-Kingshott%2CCA&amp;author=Alexander%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Elliott, L. T. et al. Genome-wide association studies of brain imaging phenotypes in UK Biobank. <i>Nature</i> <b>562</b>, 210–216 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-018-0571-7" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-018-0571-7" aria-label="Article reference 16" data-doi="10.1038/s41586-018-0571-7">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20studies%20of%20brain%20imaging%20phenotypes%20in%20UK%20Biobank&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0571-7&amp;volume=562&amp;pages=210-216&amp;publication_year=2018&amp;author=Elliott%2CLT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Stark, D. E. et al. Regional variation in interhemispheric coordination of intrinsic hemodynamic fluctuations. <i>J. Neurosci.</i> <b>28</b>, 13754–13764 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.4544-08.2008" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.4544-08.2008" aria-label="Article reference 17" data-doi="10.1523/JNEUROSCI.4544-08.2008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsFCqtrnN" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Regional%20variation%20in%20interhemispheric%20coordination%20of%20intrinsic%20hemodynamic%20fluctuations&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.4544-08.2008&amp;volume=28&amp;pages=13754-13764&amp;publication_year=2008&amp;author=Stark%2CDE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Shen, K. et al. Stable long-range interhemispheric coordination is supported by direct anatomical projections. <i>Proc. Natl Acad. Sci. USA</i> <b>112</b>, 6473–6478 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1503436112" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1503436112" aria-label="Article reference 18" data-doi="10.1073/pnas.1503436112">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXns1Olu70%3D" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Stable%20long-range%20interhemispheric%20coordination%20is%20supported%20by%20direct%20anatomical%20projections&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1503436112&amp;volume=112&amp;pages=6473-6478&amp;publication_year=2015&amp;author=Shen%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. <i>Neuroimage</i> <b>166</b>, 400–424 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.10.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.10.034" aria-label="Article reference 19" data-doi="10.1016/j.neuroimage.2017.10.034">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Image%20processing%20and%20quality%20control%20for%20the%20first%2010%2C000%20brain%20imaging%20datasets%20from%20UK%20Biobank&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2017.10.034&amp;volume=166&amp;pages=400-424&amp;publication_year=2018&amp;author=Alfaro-Almagro%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Basser, P. J., Mattiello, J. &amp; LeBihan, D. Estimation of the effective self-diffusion tensor from the NMR spin echo. <i>J. Magn. Reson.</i> <b>103</b>, 247–254 (1994).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/jmrb.1994.1037" data-track-action="article reference" href="https://doi.org/10.1006%2Fjmrb.1994.1037" aria-label="Article reference 20" data-doi="10.1006/jmrb.1994.1037">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2cXivVyitL0%3D" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimation%20of%20the%20effective%20self-diffusion%20tensor%20from%20the%20NMR%20spin%20echo&amp;journal=J.%20Magn.%20Reson.&amp;doi=10.1006%2Fjmrb.1994.1037&amp;volume=103&amp;pages=247-254&amp;publication_year=1994&amp;author=Basser%2CPJ&amp;author=Mattiello%2CJ&amp;author=LeBihan%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Ennis, D. B. &amp; Kindlmann, G. Orthogonal tensor invariants and the analysis of diffusion tensor magnetic resonance images. <i>Magn. Reson. Med.</i> <b>55</b>, 136–146 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.20741" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.20741" aria-label="Article reference 21" data-doi="10.1002/mrm.20741">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Orthogonal%20tensor%20invariants%20and%20the%20analysis%20of%20diffusion%20tensor%20magnetic%20resonance%20images&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.20741&amp;volume=55&amp;pages=136-146&amp;publication_year=2006&amp;author=Ennis%2CDB&amp;author=Kindlmann%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Behrens, T. E. J., Berg, H. J., Jbabdi, S., Rushworth, M. F. S. &amp; Woolrich, M. W. Probabilistic diffusion tractography with multiple fibre orientations: what can we gain? <i>Neuroimage</i> <b>34</b>, 144–155 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2006.09.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2006.09.018" aria-label="Article reference 22" data-doi="10.1016/j.neuroimage.2006.09.018">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD28nos1WmsQ%3D%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Probabilistic%20diffusion%20tractography%20with%20multiple%20fibre%20orientations%3A%20what%20can%20we%20gain%3F&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2006.09.018&amp;volume=34&amp;pages=144-155&amp;publication_year=2007&amp;author=Behrens%2CTEJ&amp;author=Berg%2CHJ&amp;author=Jbabdi%2CS&amp;author=Rushworth%2CMFS&amp;author=Woolrich%2CMW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Smith, S. M. et al. Tract-based spatial statistics: voxelwise analysis of multi-subject diffusion data. <i>Neuroimage</i> <b>31</b>, 1487–1505 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2006.02.024" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2006.02.024" aria-label="Article reference 23" data-doi="10.1016/j.neuroimage.2006.02.024">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Tract-based%20spatial%20statistics%3A%20voxelwise%20analysis%20of%20multi-subject%20diffusion%20data&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2006.02.024&amp;volume=31&amp;pages=1487-1505&amp;publication_year=2006&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Mollink, J. et al. Evaluating fibre orientation dispersion in white matter: comparison of diffusion MRI, histology and polarized light imaging. <i>Neuroimage</i> <b>157</b>, 561–574 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.06.001" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.06.001" aria-label="Article reference 24" data-doi="10.1016/j.neuroimage.2017.06.001">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluating%20fibre%20orientation%20dispersion%20in%20white%20matter%3A%20comparison%20of%20diffusion%20MRI%2C%20histology%20and%20polarized%20light%20imaging&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2017.06.001&amp;volume=157&amp;pages=561-574&amp;publication_year=2017&amp;author=Mollink%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. <i>Nature</i> <b>562</b>, 203–209 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-018-0579-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-018-0579-z" aria-label="Article reference 25" data-doi="10.1038/s41586-018-0579-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2qsbvI" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0579-z&amp;volume=562&amp;pages=203-209&amp;publication_year=2018&amp;author=Bycroft%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Kent, W. J. et al. The human genome browser at UCSC. <i>Genome Res.</i> <b>12</b>, 996–1006 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1101/gr.229102" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.229102" aria-label="Article reference 26" data-doi="10.1101/gr.229102">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38Xks12hs7s%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20genome%20browser%20at%20UCSC&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.229102&amp;volume=12&amp;pages=996-1006&amp;publication_year=2002&amp;author=Kent%2CWJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Matusek, T. et al. Formin proteins of the DAAM subfamily play a role during axon growth. <i>J. Neurosci.</i> <b>28</b>, 13310–13319 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2727-08.2008" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2727-08.2008" aria-label="Article reference 27" data-doi="10.1523/JNEUROSCI.2727-08.2008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhsV2murzJ" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Formin%20proteins%20of%20the%20DAAM%20subfamily%20play%20a%20role%20during%20axon%20growth&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2727-08.2008&amp;volume=28&amp;pages=13310-13319&amp;publication_year=2008&amp;author=Matusek%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Avilés, E. C. &amp; Stoeckli, E. T. Canonical wnt signaling is required for commissural axon guidance. <i>Dev. Neurobiol.</i> <b>76</b>, 190–208 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/dneu.22307" data-track-action="article reference" href="https://doi.org/10.1002%2Fdneu.22307" aria-label="Article reference 28" data-doi="10.1002/dneu.22307">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Canonical%20wnt%20signaling%20is%20required%20for%20commissural%20axon%20guidance&amp;journal=Dev.%20Neurobiol.&amp;doi=10.1002%2Fdneu.22307&amp;volume=76&amp;pages=190-208&amp;publication_year=2016&amp;author=Avil%C3%A9s%2CEC&amp;author=Stoeckli%2CET">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Won, H. et al. Chromosome conformation elucidates regulatory relationships in developing human brain. <i>Nature</i> <b>538</b>, 523–527 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature19847" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature19847" aria-label="Article reference 29" data-doi="10.1038/nature19847">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromosome%20conformation%20elucidates%20regulatory%20relationships%20in%20developing%20human%20brain&amp;journal=Nature&amp;doi=10.1038%2Fnature19847&amp;volume=538&amp;pages=523-527&amp;publication_year=2016&amp;author=Won%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Wang, Y. et al. The 3D Genome Browser: a web-based browser for visualizing 3D genome organization and long-range chromatin interactions. <i>Genome Biol.</i> <b>19</b>, 151 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/s13059-018-1519-9" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-018-1519-9" aria-label="Article reference 30" data-doi="10.1186/s13059-018-1519-9">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=The%203D%20Genome%20Browser%3A%20a%20web-based%20browser%20for%20visualizing%203D%20genome%20organization%20and%20long-range%20chromatin%20interactions&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-018-1519-9&amp;volume=19&amp;publication_year=2018&amp;author=Wang%2CY">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Yung, Y. C., Stoddard, N. C., Mirendil, H. &amp; Chun, J. Lysophosphatidic acid signaling in the nervous system. <i>Neuron</i> <b>85</b>, 669–682 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2015.01.009" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.01.009" aria-label="Article reference 31" data-doi="10.1016/j.neuron.2015.01.009">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXjt1SitbY%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Lysophosphatidic%20acid%20signaling%20in%20the%20nervous%20system&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2015.01.009&amp;volume=85&amp;pages=669-682&amp;publication_year=2015&amp;author=Yung%2CYC&amp;author=Stoddard%2CNC&amp;author=Mirendil%2CH&amp;author=Chun%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Hibar, D. P. et al. Common genetic variants influence human subcortical brain structures. <i>Nature</i> <b>520</b>, 224–229 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature14101" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature14101" aria-label="Article reference 32" data-doi="10.1038/nature14101">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1Omsbc%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Common%20genetic%20variants%20influence%20human%20subcortical%20brain%20structures&amp;journal=Nature&amp;doi=10.1038%2Fnature14101&amp;volume=520&amp;pages=224-229&amp;publication_year=2015&amp;author=Hibar%2CDP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Honey, C. J. et al. Predicting human resting-state functional connectivity from structural connectivity. <i>Proc. Natl Acad. Sci. USA</i> <b>106</b>, 2035–2040 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0811168106" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0811168106" aria-label="Article reference 33" data-doi="10.1073/pnas.0811168106">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXitVKktbY%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Predicting%20human%20resting-state%20functional%20connectivity%20from%20structural%20connectivity&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0811168106&amp;volume=106&amp;pages=2035-2040&amp;publication_year=2009&amp;author=Honey%2CCJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">O’Reilly, J. X. et al. Causal effect of disconnection lesions on interhemispheric functional connectivity in rhesus monkeys. <i>Proc. Natl Acad. Sci. USA</i> <b>110</b>, 13982–13987 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1305062110" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1305062110" aria-label="Article reference 34" data-doi="10.1073/pnas.1305062110">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Causal%20effect%20of%20disconnection%20lesions%20on%20interhemispheric%20functional%20connectivity%20in%20rhesus%20monkeys&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1305062110&amp;volume=110&amp;pages=13982-13987&amp;publication_year=2013&amp;author=O%E2%80%99Reilly%2CJX">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Roland, J. L. et al. On the role of the corpus callosum in interhemispheric functional connectivity in humans. <i>Proc. Natl Acad. Sci. USA</i> <b>114</b>, 13278–13283 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1707050114" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1707050114" aria-label="Article reference 35" data-doi="10.1073/pnas.1707050114">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVOrtrfK" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20role%20of%20the%20corpus%20callosum%20in%20interhemispheric%20functional%20connectivity%20in%20humans&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1707050114&amp;volume=114&amp;pages=13278-13283&amp;publication_year=2017&amp;author=Roland%2CJL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Smith, S. M. &amp; Nichols, T. E. Statistical challenges in “big data” human neuroimaging. <i>Neuron</i> <b>97</b>, 263–268 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2017.12.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2017.12.018" aria-label="Article reference 36" data-doi="10.1016/j.neuron.2017.12.018">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXis1amtrs%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20challenges%20in%20%E2%80%9Cbig%20data%E2%80%9D%20human%20neuroimaging&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2017.12.018&amp;volume=97&amp;pages=263-268&amp;publication_year=2018&amp;author=Smith%2CSM&amp;author=Nichols%2CTE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Torkamani, A., Wineinger, N. E. &amp; Topol, E. J. The personal and clinical utility of polygenic risk scores. <i>Nat. Rev. Genet.</i> <b>19</b>, 581–590 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41576-018-0018-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41576-018-0018-x" aria-label="Article reference 37" data-doi="10.1038/s41576-018-0018-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVSlu73K" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20personal%20and%20clinical%20utility%20of%20polygenic%20risk%20scores&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fs41576-018-0018-x&amp;volume=19&amp;pages=581-590&amp;publication_year=2018&amp;author=Torkamani%2CA&amp;author=Wineinger%2CNE&amp;author=Topol%2CEJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Tobyne, S. M. et al. A surface-based technique for mapping homotopic interhemispheric connectivity:development, characterization, and clinical application. <i>Hum. Brain Mapp.</i> <b>37</b>, 2849–2868 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/hbm.23214" data-track-action="article reference" href="https://doi.org/10.1002%2Fhbm.23214" aria-label="Article reference 38" data-doi="10.1002/hbm.23214">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20surface-based%20technique%20for%20mapping%20homotopic%20interhemispheric%20connectivity%3Adevelopment%2C%20characterization%2C%20and%20clinical%20application&amp;journal=Hum.%20Brain%20Mapp.&amp;doi=10.1002%2Fhbm.23214&amp;volume=37&amp;pages=2849-2868&amp;publication_year=2016&amp;author=Tobyne%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Schmahmann, J. D. et al. Association fibre pathways of the brain: parallel observations from diffusion spectrum imaging and autoradiography. <i>Brain</i> <b>130</b>, 630–653 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/brain/awl359" data-track-action="article reference" href="https://doi.org/10.1093%2Fbrain%2Fawl359" aria-label="Article reference 39" data-doi="10.1093/brain/awl359">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20fibre%20pathways%20of%20the%20brain%3A%20parallel%20observations%20from%20diffusion%20spectrum%20imaging%20and%20autoradiography&amp;journal=Brain&amp;doi=10.1093%2Fbrain%2Fawl359&amp;volume=130&amp;pages=630-653&amp;publication_year=2007&amp;author=Schmahmann%2CJD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Smith, S. M. The future of FMRI connectivity. <i>Neuroimage</i> <b>62</b>, 1257–1266 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.01.022" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.01.022" aria-label="Article reference 40" data-doi="10.1016/j.neuroimage.2012.01.022">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20future%20of%20FMRI%20connectivity&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.01.022&amp;volume=62&amp;pages=1257-1266&amp;publication_year=2012&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Cordes, D. et al. Frequencies contributing to functional connectivity in the cerebral cortex in “resting-state” data. <i>Am. J. Neuroradiol.</i> <b>22</b>, 1326–1333 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3Mvlt1Sluw%3D%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11498421" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Frequencies%20contributing%20to%20functional%20connectivity%20in%20the%20cerebral%20cortex%20in%20%E2%80%9Cresting-state%E2%80%9D%20data&amp;journal=Am.%20J.%20Neuroradiol.&amp;volume=22&amp;pages=1326-1333&amp;publication_year=2001&amp;author=Cordes%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Friston, K. J. Functional and effective connectivity: a review. <i>Brain Connect.</i> <b>1</b>, 13–36 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1089/brain.2011.0008" data-track-action="article reference" href="https://doi.org/10.1089%2Fbrain.2011.0008" aria-label="Article reference 42" data-doi="10.1089/brain.2011.0008">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20and%20effective%20connectivity%3A%20a%20review&amp;journal=Brain%20Connect.&amp;doi=10.1089%2Fbrain.2011.0008&amp;volume=1&amp;pages=13-36&amp;publication_year=2011&amp;author=Friston%2CKJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Budde, M. D. &amp; Annese, J. Quantification of anisotropy and fiber orientation in human brain histological sections. <i>Front. Integr. Neurosci.</i> <b>7</b>, 3 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fnint.2013.00003" data-track-action="article reference" href="https://doi.org/10.3389%2Ffnint.2013.00003" aria-label="Article reference 43" data-doi="10.3389/fnint.2013.00003">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantification%20of%20anisotropy%20and%20fiber%20orientation%20in%20human%20brain%20histological%20sections&amp;journal=Front.%20Integr.%20Neurosci.&amp;doi=10.3389%2Ffnint.2013.00003&amp;volume=7&amp;publication_year=2013&amp;author=Budde%2CMD&amp;author=Annese%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Hibar, D. P. et al. Novel genetic loci associated with hippocampal volume. <i>Nat. Commun.</i> <b>8</b>, 13624 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms13624" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms13624" aria-label="Article reference 44" data-doi="10.1038/ncomms13624">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlars7o%3D" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Novel%20genetic%20loci%20associated%20with%20hippocampal%20volume&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms13624&amp;volume=8&amp;publication_year=2017&amp;author=Hibar%2CDP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Habas, R., Kato, Y. &amp; He, X. Wnt/Frizzled activation of Rho regulates vertebrate gastrulation and requires a novel Formin homology protein Daam1. <i>Cell</i> <b>107</b>, 843–854 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0092-8674(01)00614-6" data-track-action="article reference" href="https://doi.org/10.1016%2FS0092-8674%2801%2900614-6" aria-label="Article reference 45" data-doi="10.1016/S0092-8674(01)00614-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XjtlKmtQ%3D%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Wnt%2FFrizzled%20activation%20of%20Rho%20regulates%20vertebrate%20gastrulation%20and%20requires%20a%20novel%20Formin%20homology%20protein%20Daam1&amp;journal=Cell&amp;doi=10.1016%2FS0092-8674%2801%2900614-6&amp;volume=107&amp;pages=843-854&amp;publication_year=2001&amp;author=Habas%2CR&amp;author=Kato%2CY&amp;author=He%2CX">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Salomon, S. N., Haber, M., Murai, K. K. &amp; Dunn, R. J. Localization of the Diaphanous-related formin Daam1 to neuronal dendrites. <i>Neurosci. Lett.</i> <b>447</b>, 62–67 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neulet.2008.09.051" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neulet.2008.09.051" aria-label="Article reference 46" data-doi="10.1016/j.neulet.2008.09.051">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXht12ltr7F" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Localization%20of%20the%20Diaphanous-related%20formin%20Daam1%20to%20neuronal%20dendrites&amp;journal=Neurosci.%20Lett.&amp;doi=10.1016%2Fj.neulet.2008.09.051&amp;volume=447&amp;pages=62-67&amp;publication_year=2008&amp;author=Salomon%2CSN&amp;author=Haber%2CM&amp;author=Murai%2CKK&amp;author=Dunn%2CRJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Kida, Y., Shiraishi, T. &amp; Ogura, T. Identification of chick and mouse Daam1 and Daam2 genes and their expression patterns in the central nervous system. <i>Brain Res. Dev. Brain Res.</i> <b>153</b>, 143–150 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.devbrainres.2004.07.014" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.devbrainres.2004.07.014" aria-label="Article reference 47" data-doi="10.1016/j.devbrainres.2004.07.014">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXotFShsbs%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20chick%20and%20mouse%20Daam1%20and%20Daam2%20genes%20and%20their%20expression%20patterns%20in%20the%20central%20nervous%20system&amp;journal=Brain%20Res.%20Dev.%20Brain%20Res.&amp;doi=10.1016%2Fj.devbrainres.2004.07.014&amp;volume=153&amp;pages=143-150&amp;publication_year=2004&amp;author=Kida%2CY&amp;author=Shiraishi%2CT&amp;author=Ogura%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Behrends, U. et al. Novel tumor antigens identified by autologous antibody screening of childhood medulloblastoma cDNA libraries. <i>Int. J. Cancer</i> <b>106</b>, 244–251 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/ijc.11208" data-track-action="article reference" href="https://doi.org/10.1002%2Fijc.11208" aria-label="Article reference 48" data-doi="10.1002/ijc.11208">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3sXlvV2qtrg%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Novel%20tumor%20antigens%20identified%20by%20autologous%20antibody%20screening%20of%20childhood%20medulloblastoma%20cDNA%20libraries&amp;journal=Int.%20J.%20Cancer&amp;doi=10.1002%2Fijc.11208&amp;volume=106&amp;pages=244-251&amp;publication_year=2003&amp;author=Behrends%2CU">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">González de San Román, E. et al. Anatomical location of LPA1 activation and LPA phospholipid precursors in rodent and human brain. <i>J. Neurochem.</i> <b>134</b>, 471–485 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/jnc.13112" data-track-action="article reference" href="https://doi.org/10.1111%2Fjnc.13112" aria-label="Article reference 49" data-doi="10.1111/jnc.13112">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Anatomical%20location%20of%20LPA1%20activation%20and%20LPA%20phospholipid%20precursors%20in%20rodent%20and%20human%20brain&amp;journal=J.%20Neurochem.&amp;doi=10.1111%2Fjnc.13112&amp;volume=134&amp;pages=471-485&amp;publication_year=2015&amp;author=Gonz%C3%A1lez%20de%20San%20Rom%C3%A1n%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Fields, R. D. A new mechanism of nervous system plasticity: activity-dependent myelination. <i>Nat. Rev. Neurosci.</i> <b>16</b>, 756–767 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn4023" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn4023" aria-label="Article reference 50" data-doi="10.1038/nrn4023">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvVOgtb%2FE" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20mechanism%20of%20nervous%20system%20plasticity%3A%20activity-dependent%20myelination&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn4023&amp;volume=16&amp;pages=756-767&amp;publication_year=2015&amp;author=Fields%2CRD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Jenkinson, M., Beckmann, C. F., Behrens, T. E. J., Woolrich, M. W. &amp; Smith, S. M. FSL. <i>Neuroimage</i> <b>62</b>, 782–790 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.09.015" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.09.015" aria-label="Article reference 51" data-doi="10.1016/j.neuroimage.2011.09.015">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=FSL&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2011.09.015&amp;volume=62&amp;pages=782-790&amp;publication_year=2012&amp;author=Jenkinson%2CM&amp;author=Beckmann%2CCF&amp;author=Behrens%2CTEJ&amp;author=Woolrich%2CMW&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Jenkinson, M., Bannister, P., Brady, M. &amp; Smith, S. Improved optimization for the robust and accurate linear registration and motion correction of brain images. <i>Neuroimage</i> <b>17</b>, 825–841 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.2002.1132" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.2002.1132" aria-label="Article reference 52" data-doi="10.1006/nimg.2002.1132">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20optimization%20for%20the%20robust%20and%20accurate%20linear%20registration%20and%20motion%20correction%20of%20brain%20images&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.2002.1132&amp;volume=17&amp;pages=825-841&amp;publication_year=2002&amp;author=Jenkinson%2CM&amp;author=Bannister%2CP&amp;author=Brady%2CM&amp;author=Smith%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Salimi-Khorshidi, G. et al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. <i>Neuroimage</i> <b>90</b>, 449–468 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.11.046" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.11.046" aria-label="Article reference 53" data-doi="10.1016/j.neuroimage.2013.11.046">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Automatic%20denoising%20of%20functional%20MRI%20data%3A%20combining%20independent%20component%20analysis%20and%20hierarchical%20fusion%20of%20classifiers&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.11.046&amp;volume=90&amp;pages=449-468&amp;publication_year=2014&amp;author=Salimi-Khorshidi%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Andersson, J. L. R. &amp; Sotiropoulos, S. N. An integrated approach to correction for off-resonance effects and subject movement in diffusion MR imaging. <i>Neuroimage</i> <b>125</b>, 1063–1078 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.10.019" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.10.019" aria-label="Article reference 54" data-doi="10.1016/j.neuroimage.2015.10.019">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20integrated%20approach%20to%20correction%20for%20off-resonance%20effects%20and%20subject%20movement%20in%20diffusion%20MR%20imaging&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.10.019&amp;volume=125&amp;pages=1063-1078&amp;publication_year=2016&amp;author=Andersson%2CJLR&amp;author=Sotiropoulos%2CSN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Daducci, A. et al. Accelerated microstructure imaging via convex pptimization (AMICO) from diffusion MRI data. <i>Neuroimage</i> <b>105</b>, 32–44 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.10.026" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.10.026" aria-label="Article reference 55" data-doi="10.1016/j.neuroimage.2014.10.026">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerated%20microstructure%20imaging%20via%20convex%20pptimization%20%28AMICO%29%20from%20diffusion%20MRI%20data&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.10.026&amp;volume=105&amp;pages=32-44&amp;publication_year=2015&amp;author=Daducci%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Griffanti, L. et al. BIANCA (Brain Intensity AbNormality Classification Algorithm): a new tool for automated segmentation of white matter hyperintensities. <i>Neuroimage</i> <b>141</b>, 191–205 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2016.07.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2016.07.018" aria-label="Article reference 56" data-doi="10.1016/j.neuroimage.2016.07.018">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=BIANCA%20%28Brain%20Intensity%20AbNormality%20Classification%20Algorithm%29%3A%20a%20new%20tool%20for%20automated%20segmentation%20of%20white%20matter%20hyperintensities&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2016.07.018&amp;volume=141&amp;pages=191-205&amp;publication_year=2016&amp;author=Griffanti%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Beckmann, C. F. &amp; Smith, S. M. Probabilistic independent component analysis for functional magnetic resonance imaging. <i>IEEE Trans. Med. Imaging</i> <b>23</b>, 137–152 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1109/TMI.2003.822821" data-track-action="article reference" href="https://doi.org/10.1109%2FTMI.2003.822821" aria-label="Article reference 57" data-doi="10.1109/TMI.2003.822821">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Probabilistic%20independent%20component%20analysis%20for%20functional%20magnetic%20resonance%20imaging&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;doi=10.1109%2FTMI.2003.822821&amp;volume=23&amp;pages=137-152&amp;publication_year=2004&amp;author=Beckmann%2CCF&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Behrens, T. E. J. et al. Characterization and propagation of uncertainty in diffusion-weighted MR imaging. <i>Magn. Reson. Med.</i> <b>50</b>, 1077–1088 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.10609" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.10609" aria-label="Article reference 58" data-doi="10.1002/mrm.10609">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3srgvFynsw%3D%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20and%20propagation%20of%20uncertainty%20in%20diffusion-weighted%20MR%20imaging&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.10609&amp;volume=50&amp;pages=1077-1088&amp;publication_year=2003&amp;author=Behrens%2CTEJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Jbabdi, S., Sotiropoulos, S. N., Savio, A. M., Graña, M. &amp; Behrens, T. E. J. Model-based analysis of multishell diffusion MR data for tractography: how to get over fitting problems. <i>Magn. Reson. Med.</i> <b>68</b>, 1846–1855 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/mrm.24204" data-track-action="article reference" href="https://doi.org/10.1002%2Fmrm.24204" aria-label="Article reference 59" data-doi="10.1002/mrm.24204">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Model-based%20analysis%20of%20multishell%20diffusion%20MR%20data%20for%20tractography%3A%20how%20to%20get%20over%20fitting%20problems&amp;journal=Magn.%20Reson.%20Med.&amp;doi=10.1002%2Fmrm.24204&amp;volume=68&amp;pages=1846-1855&amp;publication_year=2012&amp;author=Jbabdi%2CS&amp;author=Sotiropoulos%2CSN&amp;author=Savio%2CAM&amp;author=Gra%C3%B1a%2CM&amp;author=Behrens%2CTEJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">de Groot, M. et al. Improving alignment in tract-based spatial statistics: evaluation and optimization of image registration. <i>Neuroimage</i> <b>76</b>, 400–411 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.03.015" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.03.015" aria-label="Article reference 60" data-doi="10.1016/j.neuroimage.2013.03.015">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Improving%20alignment%20in%20tract-based%20spatial%20statistics%3A%20evaluation%20and%20optimization%20of%20image%20registration&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.03.015&amp;volume=76&amp;pages=400-411&amp;publication_year=2013&amp;author=Groot%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Bach, M. et al. Methodological considerations on tract-based spatial statistics (TBSS). <i>Neuroimage</i> <b>100</b>, 358–369 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.06.021" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.06.021" aria-label="Article reference 61" data-doi="10.1016/j.neuroimage.2014.06.021">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Methodological%20considerations%20on%20tract-based%20spatial%20statistics%20%28TBSS%29&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.06.021&amp;volume=100&amp;pages=358-369&amp;publication_year=2014&amp;author=Bach%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Winkler, A. M., Ridgway, G. R., Webster, M. A., Smith, S. M. &amp; Nichols, T. E. Permutation inference for the general linear model. <i>Neuroimage</i> <b>92</b>, 381–397 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.01.060" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.01.060" aria-label="Article reference 62" data-doi="10.1016/j.neuroimage.2014.01.060">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Permutation%20inference%20for%20the%20general%20linear%20model&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.01.060&amp;volume=92&amp;pages=381-397&amp;publication_year=2014&amp;author=Winkler%2CAM&amp;author=Ridgway%2CGR&amp;author=Webster%2CMA&amp;author=Smith%2CSM&amp;author=Nichols%2CTE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Sotiropoulos, S. N., Behrens, T. E. J. &amp; Jbabdi, S. Ball and rackets: inferring fiber fanning from diffusion-weighted MRI. <i>Neuroimage</i> <b>60</b>, 1412–1425 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.01.056" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.01.056" aria-label="Article reference 63" data-doi="10.1016/j.neuroimage.2012.01.056">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Ball%20and%20rackets%3A%20inferring%20fiber%20fanning%20from%20diffusion-weighted%20MRI&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.01.056&amp;volume=60&amp;pages=1412-1425&amp;publication_year=2012&amp;author=Sotiropoulos%2CSN&amp;author=Behrens%2CTEJ&amp;author=Jbabdi%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Budde, M. D. &amp; Frank, J. A. Examining brain microstructure using structure tensor analysis of histological sections. <i>Neuroimage</i> <b>63</b>, 1–10 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.06.042" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.06.042" aria-label="Article reference 64" data-doi="10.1016/j.neuroimage.2012.06.042">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Examining%20brain%20microstructure%20using%20structure%20tensor%20analysis%20of%20histological%20sections&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.06.042&amp;volume=63&amp;pages=1-10&amp;publication_year=2012&amp;author=Budde%2CMD&amp;author=Frank%2CJA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Heinrich, M. P. et al. MIND: modality independent neighbourhood descriptor for multi-modal deformable registration. <i>Med. Image Anal.</i> <b>16</b>, 1423–1435 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.media.2012.05.008" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.media.2012.05.008" aria-label="Article reference 65" data-doi="10.1016/j.media.2012.05.008">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=MIND%3A%20modality%20independent%20neighbourhood%20descriptor%20for%20multi-modal%20deformable%20registration&amp;journal=Med.%20Image%20Anal.&amp;doi=10.1016%2Fj.media.2012.05.008&amp;volume=16&amp;pages=1423-1435&amp;publication_year=2012&amp;author=Heinrich%2CMP">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-019-0379-2?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>All data in this study were obtained from the UK Biobank project (access no. 8107). We are very grateful to all individuals who donated their time to participate in the UK Biobank study. K.L.M., M.K. and J.Mo. are supported by the Wellcome Trust (nos. 091509/Z/10/Z, 202788/Z/16/Z, 098369/Z/12/Z). The authors gratefully acknowledge funding from the Wellcome Trust UK Strategic Award (no. 098369/Z/12/Z). UK Biobank brain imaging and F.A.-A. are funded by the UK Medical Research Council and the Wellcome Trust. J.Ma. acknowledges funding for this work from the European Research Council (grant no. 617306) and the Leverhulme Trust. S.J. is supported by the UK Medical Research Council (no. MR/L009013/1). The Wellcome Centre for Integrative Neuroimaging is supported by core funding from the Wellcome Trust (no. 203139/Z/16/Z). Finally, we would like to thank D. Norris and J. Marques for their valuable input on this work.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Saad Jbabdi and Karla L. Miller.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Wellcome Centre for Integrative Neuroimaging, University of Oxford, Oxford, United Kingdom</p><p class="c-article-author-affiliation__authors-list">Jeroen Mollink, Stephen M. Smith, Michiel Kleinnijenhuis, Fidel Alfaro-Almagro, Saad Jbabdi &amp; Karla L. Miller</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Anatomy, Donders Institute for Brain, Cognition and Behaviour, Radboud University Medical Center, Nijmegen, the Netherlands</p><p class="c-article-author-affiliation__authors-list">Jeroen Mollink, Marlies Hiemstra &amp; Anne-Marie van Cappellen van Walsum</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Department of Statistics and Actuarial Science, Simon Fraser University, Burnaby, British Columbia, Canada</p><p class="c-article-author-affiliation__authors-list">Lloyd T. Elliott</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Department of Statistics, University of Oxford, Oxford, United Kingdom</p><p class="c-article-author-affiliation__authors-list">Lloyd T. Elliott &amp; Jonathan Marchini</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">The Wellcome Centre for Human Genetics, University of Oxford, Oxford, United Kingdom</p><p class="c-article-author-affiliation__authors-list">Jonathan Marchini</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Jeroen-Mollink-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Jeroen Mollink</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jeroen%20Mollink" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jeroen%20Mollink" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jeroen%20Mollink%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stephen_M_-Smith-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Stephen M. Smith</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stephen%20M.%20Smith" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stephen%20M.%20Smith" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stephen%20M.%20Smith%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lloyd_T_-Elliott-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Lloyd T. Elliott</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lloyd%20T.%20Elliott" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lloyd%20T.%20Elliott" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lloyd%20T.%20Elliott%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michiel-Kleinnijenhuis-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Michiel Kleinnijenhuis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michiel%20Kleinnijenhuis" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michiel%20Kleinnijenhuis" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michiel%20Kleinnijenhuis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marlies-Hiemstra-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Marlies Hiemstra</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marlies%20Hiemstra" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marlies%20Hiemstra" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marlies%20Hiemstra%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Fidel-Alfaro_Almagro-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Fidel Alfaro-Almagro</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Fidel%20Alfaro-Almagro" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Fidel%20Alfaro-Almagro" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Fidel%20Alfaro-Almagro%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jonathan-Marchini-Aff4-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Jonathan Marchini</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jonathan%20Marchini" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jonathan%20Marchini" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jonathan%20Marchini%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anne_Marie-van_Cappellen_van_Walsum-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Anne-Marie van Cappellen van Walsum</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anne-Marie%20van%20Cappellen%20van%20Walsum" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anne-Marie%20van%20Cappellen%20van%20Walsum" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anne-Marie%20van%20Cappellen%20van%20Walsum%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Saad-Jbabdi-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Saad Jbabdi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Saad%20Jbabdi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Saad%20Jbabdi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Saad%20Jbabdi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Karla_L_-Miller-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Karla L. Miller</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Karla%20L.%20Miller" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Karla%20L.%20Miller" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Karla%20L.%20Miller%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>J.Mo., S.M.S., S.J. and K.L.M. designed the research. J.Mo. performed the research. K.L.M., F.A.-A. and S.M.S., developed acquisition and processing pipelines for the MRI data. L.T.E. and J.Ma. processed genetics data, provided tools for genome-wide associations analysis and gave feedback on genetics results. J.Mo., S.M.S., M.K., M.H., A.M.C.W., S.J. and K.L.M. analyzed the data and interpreted its outcomes. J.Mo. and K.L.M. wrote the manuscript, which was edited by all authors.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:jeroen.mollink@radboudumc.nl">Jeroen Mollink</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
                <p>J.Ma. is a co-founder and director of GENSCI Ltd. S.S. is a co-founder of SBGneuro.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Journal peer review information</b>: Nature Neuroscience thanks Genevieve Konopka and other anonymous reviewer(s) for their contribution to the peer review of this work.</p><p><b>Publisher’s note:</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec24-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec24">Supplementary information</h2><div class="c-article-section__content" id="Sec24-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary figs. 1–13 and supplementary tables " href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Figs. 1–13 and Supplementary Tables 1–4.</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-019-0379-2/MediaObjects/41593_2019_379_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=The%20spatial%20correspondence%20and%20genetic%20influence%20of%20interhemispheric%20connectivity%20with%20white%20matter%20microstructure&amp;author=Jeroen%20Mollink%20et%20al&amp;contentID=10.1038%2Fs41593-019-0379-2&amp;copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2019-04-15&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-019-0379-2" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-019-0379-2" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Mollink, J., Smith, S.M., Elliott, L.T. <i>et al.</i> The spatial correspondence and genetic influence of interhemispheric connectivity with white matter microstructure.
                    <i>Nat Neurosci</i> <b>22</b>, 809–819 (2019). https://doi.org/10.1038/s41593-019-0379-2</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-019-0379-2?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2018-11-05">05 November 2018</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-03-08">08 March 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-04-15">15 April 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-05">May 2019</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-019-0379-2</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Impact of corpus callosum integrity on functional interhemispheric connectivity and cognition in healthy subjects" href="https://doi.org/10.1007/s11682-023-00814-1">
                                        Impact of corpus callosum integrity on functional interhemispheric connectivity and cognition in healthy subjects
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Michele Porcu</li><li>Luigi Cocco</li><li>Luca Saba</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Brain Imaging and Behavior</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Genomics of perivascular space burden unravels early mechanisms of cerebral small vessel disease" href="https://doi.org/10.1038/s41591-023-02268-w">
                                        Genomics of perivascular space burden unravels early mechanisms of cerebral small vessel disease
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Marie-Gabrielle Duperron</li><li>Maria J. Knol</li><li>Stéphanie Debette</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Medicine</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:The interindividual variability of multimodal brain connectivity maintains spatial heterogeneity and relates to tissue microstructure" href="https://doi.org/10.1038/s42003-022-03974-w">
                                        The interindividual variability of multimodal brain connectivity maintains spatial heterogeneity and relates to tissue microstructure
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Esin Karahan</li><li>Luke Tait</li><li>Jiaxiang Zhang</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2022)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Corticospinal excitability and conductivity are related to the anatomy of the corticospinal tract" href="https://doi.org/10.1007/s00429-021-02410-9">
                                        Corticospinal excitability and conductivity are related to the anatomy of the corticospinal tract
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Sonia Betti</li><li>Marta Fedele</li><li>Sanja Budisavljević</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Brain Structure and Function</i> (2022)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Functional cortical associations and their intraclass correlations and heritability as revealed by the fMRI Human Connectome Project" href="https://doi.org/10.1007/s00221-022-06346-2">
                                        Functional cortical associations and their intraclass correlations and heritability as revealed by the fMRI Human Connectome Project
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Peka Christova</li><li>Jasmine Joseph</li><li>Apostolos P. Georgopoulos</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Experimental Brain Research</i> (2022)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-019-0379-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-019-0379-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-019-0379-2;doi=10.1038/s41593-019-0379-2;techmeta=36,43,45,59;subjmeta=208,378,631;kwrd=Genetics,Neuroscience">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=421152701&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-019-0379-2%26doi%3D10.1038/s41593-019-0379-2%26techmeta%3D36,43,45,59%26subjmeta%3D208,378,631%26kwrd%3DGenetics,Neuroscience">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=421152701&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-019-0379-2%26doi%3D10.1038/s41593-019-0379-2%26techmeta%3D36,43,45,59%26subjmeta%3D208,378,631%26kwrd%3DGenetics,Neuroscience"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-019-0379-2&amp;format=js&amp;last_modified=2019-04-15" async></script>
<img src="/k3x8wuw3/article/s41593-019-0379-2" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>