<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Topographic organization of the human subcortex unveiled with functional connectivity gradients | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"functional-magnetic-resonance-imaging;neural-circuit","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41593-020-00711-6"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Ye Tian","Daniel S. Margulies","Michael Breakspear","Andrew Zalesky"],"publishedAt":1601251200,"publishedAtString":"2020-09-28","title":"Topographic organization of the human subcortex unveiled with functional connectivity gradients","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"23","issue":"11"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Topographic organization of the human subcortex unveiled with functional connectivity gradients","description":"Brain atlases are fundamental to understanding the topographic organization of the human brain, yet many contemporary human atlases cover only the cerebral cortex, leaving the subcortex a terra incognita. We use functional MRI (fMRI) to map the complex topographic organization of the human subcortex, revealing large-scale connectivity gradients and new areal boundaries. We unveil four scales of subcortical organization that recapitulate well-known anatomical nuclei at the coarsest scale and delineate 27 new bilateral regions at the finest. Ultrahigh field strength fMRI corroborates and extends this organizational structure, enabling the delineation of finer subdivisions of the hippocampus and the amygdala, while task-evoked fMRI reveals a subtle subcortical reorganization in response to changing cognitive demands. A new subcortical atlas is delineated, personalized to represent individual differences and used to uncover reproducible brain–behavior relationships. Linking cortical networks to subcortical regions recapitulates a task-positive to task-negative axis. This new atlas enables holistic connectome mapping and characterization of cortico–subcortical connectivity. This work by Tian and colleagues unveils the extraordinarily complex layout of the human subcortex by identifying 27 new functional regions that organize hierarchically across four scales and adapt to changing cognitive demands.","datePublished":"2020-09-28T00:00:00Z","dateModified":"2020-09-28T00:00:00Z","pageStart":"1421","pageEnd":"1432","sameAs":"https://doi.org/10.1038/s41593-020-00711-6","keywords":["Functional magnetic resonance imaging","Neural circuits","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig7_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig8_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"23","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Ye Tian","url":"http://orcid.org/0000-0003-3107-5550","affiliation":[{"name":"The University of Melbourne","address":{"name":"Melbourne Neuropsychiatry Centre, Department of Psychiatry, The University of Melbourne, Parkville, Australia","@type":"PostalAddress"},"@type":"Organization"}],"email":"ye.tian2@unimelb.edu.au","@type":"Person"},{"name":"Daniel S. Margulies","affiliation":[{"name":"Université de Paris","address":{"name":"Centre National de la Recherche Scientifique (CNRS) UMR 8002, Integrative Neuroscience and Cognition Center, Université de Paris, Paris, France","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael Breakspear","url":"http://orcid.org/0000-0003-4943-3969","affiliation":[{"name":"University of Newcastle","address":{"name":"Discipline of Psychiatry, Faculty of Health and Medicine, University of Newcastle, Newcastle, Australia","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Newcastle","address":{"name":"School of Psychology, Faculty of Science, University of Newcastle, Newcastle, Australia","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Andrew Zalesky","url":"http://orcid.org/0000-0003-2298-9908","affiliation":[{"name":"The University of Melbourne","address":{"name":"Melbourne Neuropsychiatry Centre, Department of Psychiatry, The University of Melbourne, Parkville, Australia","@type":"PostalAddress"},"@type":"Organization"},{"name":"The University of Melbourne","address":{"name":"Department of Biomedical Engineering, Melbourne School of Engineering, The University of Melbourne, Parkville, Australia","@type":"PostalAddress"},"@type":"Organization"}],"email":"azalesky@unimelb.edu.au","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-020-00711-6">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="Topographic organization of the human subcortex unveiled with functional connectivity gradients"/>
    <meta name="dc.source" content="Nature Neuroscience 2020 23:11"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2020-09-28"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="Brain atlases are fundamental to understanding the topographic organization of the human brain, yet many contemporary human atlases cover only the cerebral cortex, leaving the subcortex a terra incognita. We use functional MRI (fMRI) to map the complex topographic organization of the human subcortex, revealing large-scale connectivity gradients and new areal boundaries. We unveil four scales of subcortical organization that recapitulate well-known anatomical nuclei at the coarsest scale and delineate 27 new bilateral regions at the finest. Ultrahigh field strength fMRI corroborates and extends this organizational structure, enabling the delineation of finer subdivisions of the hippocampus and the amygdala, while task-evoked fMRI reveals a subtle subcortical reorganization in response to changing cognitive demands. A new subcortical atlas is delineated, personalized to represent individual differences and used to uncover reproducible brain&#8211;behavior relationships. Linking cortical networks to subcortical regions recapitulates a task-positive to task-negative axis. This new atlas enables holistic connectome mapping and characterization of cortico&#8211;subcortical connectivity. This work by Tian and colleagues unveils the extraordinarily complex layout of the human subcortex by identifying 27 new functional regions that organize hierarchically across four scales and adapt to changing cognitive demands."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2020-09-28"/>
    <meta name="prism.volume" content="23"/>
    <meta name="prism.number" content="11"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="1421"/>
    <meta name="prism.endingPage" content="1432"/>
    <meta name="prism.copyright" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-020-00711-6"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-020-00711-6"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-020-00711-6.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-020-00711-6"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="Topographic organization of the human subcortex unveiled with functional connectivity gradients"/>
    <meta name="citation_volume" content="23"/>
    <meta name="citation_issue" content="11"/>
    <meta name="citation_publication_date" content="2020/11"/>
    <meta name="citation_online_date" content="2020/09/28"/>
    <meta name="citation_firstpage" content="1421"/>
    <meta name="citation_lastpage" content="1432"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-020-00711-6"/>
    <meta name="DOI" content="10.1038/s41593-020-00711-6"/>
    <meta name="size" content="354451"/>
    <meta name="citation_doi" content="10.1038/s41593-020-00711-6"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-020-00711-6&amp;api_key="/>
    <meta name="description" content="Brain atlases are fundamental to understanding the topographic organization of the human brain, yet many contemporary human atlases cover only the cerebral cortex, leaving the subcortex a terra incognita. We use functional MRI (fMRI) to map the complex topographic organization of the human subcortex, revealing large-scale connectivity gradients and new areal boundaries. We unveil four scales of subcortical organization that recapitulate well-known anatomical nuclei at the coarsest scale and delineate 27 new bilateral regions at the finest. Ultrahigh field strength fMRI corroborates and extends this organizational structure, enabling the delineation of finer subdivisions of the hippocampus and the amygdala, while task-evoked fMRI reveals a subtle subcortical reorganization in response to changing cognitive demands. A new subcortical atlas is delineated, personalized to represent individual differences and used to uncover reproducible brain&#8211;behavior relationships. Linking cortical networks to subcortical regions recapitulates a task-positive to task-negative axis. This new atlas enables holistic connectome mapping and characterization of cortico&#8211;subcortical connectivity. This work by Tian and colleagues unveils the extraordinarily complex layout of the human subcortex by identifying 27 new functional regions that organize hierarchically across four scales and adapt to changing cognitive demands."/>
    <meta name="dc.creator" content="Tian, Ye"/>
    <meta name="dc.creator" content="Margulies, Daniel S."/>
    <meta name="dc.creator" content="Breakspear, Michael"/>
    <meta name="dc.creator" content="Zalesky, Andrew"/>
    <meta name="dc.subject" content="Functional magnetic resonance imaging"/>
    <meta name="dc.subject" content="Neural circuits"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Imaging-based parcellations of the human brain; citation_author=SB Eickhoff, BTT Yeo, S Genon; citation_volume=19; citation_publication_date=2018; citation_pages=672-686; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A multi-modal parcellation of human cerebral cortex; citation_author=MF Glasser; citation_volume=536; citation_publication_date=2016; citation_pages=171-178; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurophysiol.; citation_title=The organization of the human cerebral cortex estimated by intrinsic functional connectivity; citation_author=BT Yeo; citation_volume=106; citation_publication_date=2011; citation_pages=1125-1165; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Corticostriatal connectivity and its role in disease; citation_author=GM Shepherd; citation_volume=14; citation_publication_date=2013; citation_pages=278-291; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurophysiol.; citation_title=The organization of the human striatum estimated by intrinsic functional connectivity; citation_author=EY Choi, BT Yeo, RL Buckner; citation_volume=108; citation_publication_date=2012; citation_pages=2242-2263; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Mapping the human brain&#8217;s cortical&#8211;subcortical functional network organization; citation_author=JL Ji; citation_volume=185; citation_publication_date=2019; citation_pages=35-57; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Defining functional areas in individual human brains using resting functional connectivity MRI; citation_author=AL Cohen; citation_volume=41; citation_publication_date=2008; citation_pages=45-57; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Generation and evaluation of a cortical area parcellation from resting-state correlations; citation_author=EM Gordon; citation_volume=26; citation_publication_date=2016; citation_pages=288-303; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Local&#8211;global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI; citation_author=A Schaefer; citation_volume=28; citation_publication_date=2018; citation_pages=3095-3114; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=Large-scale gradients in human cortical organization; citation_author=JM Huntenburg, PL Bazin, DS Margulies; citation_volume=22; citation_publication_date=2018; citation_pages=21-31; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Connectopic mapping with resting-state fMRI; citation_author=KV Haak, AF Marquand, CF Beckmann; citation_volume=170; citation_publication_date=2018; citation_pages=83-94; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Situating the default-mode network along a principal gradient of macroscale cortical organization; citation_author=DS Margulies; citation_volume=113; citation_publication_date=2016; citation_pages=12574-12579; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=Functional gradients of the cerebellum; citation_author=X Guell, JD Schmahmann, J Gabrieli, SS Ghosh; citation_volume=7; citation_publication_date=2018; citation_pages=e36652; citation_id=CR13"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Functional boundaries in the human cerebellum revealed by a multi-domain task battery; citation_author=M King, CR Hernandez-Castillo, RA Poldrack, RB Ivry, J Diedrichsen; citation_volume=22; citation_publication_date=2019; citation_pages=1371-1378; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Characterizing the functional connectivity diversity of the insula cortex: subregions, diversity curves and behavior; citation_author=Y Tian, A Zalesky; citation_volume=183; citation_publication_date=2018; citation_pages=716-733; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=The impact of traditional neuroimaging methods on the spatial localization of cortical areas; citation_author=TS Coalson, DC Essen, MF Glasser; citation_volume=115; citation_publication_date=2018; citation_pages=E6356-E6365; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=The human brain is intrinsically organized into dynamic, anticorrelated functional networks; citation_author=MD Fox; citation_volume=102; citation_publication_date=2005; citation_pages=9673-9678; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=The WU&#8211;Minn Human Connectome Project: an overview; citation_author=DC Essen; citation_volume=80; citation_publication_date=2013; citation_pages=62-79; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity; citation_author=ES Finn; citation_volume=18; citation_publication_date=2015; citation_pages=1664; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Core and matrix thalamic sub-populations relate to spatio-temporal cortical connectivity gradients; citation_author=E M&#252;ller; citation_volume=222; citation_publication_date=2020; citation_pages=117224; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=Long-axis specialization of the human hippocampus; citation_author=J Poppenk, HR Evensmoen, M Moscovitch, L Nadel; citation_volume=17; citation_publication_date=2013; citation_pages=230-240; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=Anat. Embryol.; citation_title=Cytoarchitectonic mapping of the human amygdala, hippocampal region and entorhinal cortex: intersubject variability and probability maps; citation_author=K Amunts; citation_volume=210; citation_publication_date=2005; citation_pages=343-352; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Hum. Behav.; citation_title=Functional corticostriatal connection topographies predict goal directed behaviour in humans; citation_author=AF Marquand, KV Haak, CF Beckmann; citation_volume=1; citation_publication_date=2017; citation_pages=0146; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Tradeoffs in pushing the spatial resolution of fMRI for the 7T Human Connectome Project; citation_author=A Vu; citation_volume=154; citation_publication_date=2017; citation_pages=23-32; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Precision functional mapping of individual human brains; citation_author=EM Gordon; citation_volume=95; citation_publication_date=2017; citation_pages=791-807.e7; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion; citation_author=R Kong; citation_volume=29; citation_publication_date=2019; citation_pages=2533-2551; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=There is no single functional atlas even for a single individual: functional parcel definitions change with task; citation_author=M Salehi; citation_volume=208; citation_publication_date=2020; citation_pages=116366; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Network-based statistic: identifying differences in brain networks; citation_author=A Zalesky, A Fornito, ET Bullmore; citation_volume=53; citation_publication_date=2010; citation_pages=1197-1207; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=From reinforcement learning models to psychiatric and neurological disorders; citation_author=TV Maia, MJ Frank; citation_volume=14; citation_publication_date=2011; citation_pages=154-162; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Towards a mechanistic understanding of the human subcortex; citation_author=BU Forstmann, G Hollander, L Maanen, A Alkemade, MC Keuken; citation_volume=18; citation_publication_date=2016; citation_pages=57-65; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Anatomical and microstructural determinants of hippocampal subfield functional connectome embedding; citation_author=R Vos de Wael; citation_volume=115; citation_publication_date=2018; citation_pages=10154-10159; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Perspectives on cognitive neuroscience; citation_author=PS Churchland, TJ Sejnowski; citation_volume=242; citation_publication_date=1988; citation_pages=741-745; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=The Human Brainnetome Atlas: a new brain atlas based on connectional architecture; citation_author=L Fan; citation_volume=26; citation_publication_date=2016; citation_pages=3508-3526; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Sci. Data; citation_title=A high-resolution probabilistic in vivo atlas of human subcortical brain nuclei; citation_author=WM Pauli, AN Nili, JM Tyszka; citation_volume=5; citation_publication_date=2018; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Multimodal parcellations and extensive behavioral profiling tackling the hippocampus gradient; citation_author=A Plachti; citation_volume=29; citation_publication_date=2019; citation_pages=4595-4612; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Probabilistic model-based functional parcellation reveals a robust, fine-grained subdivision of the striatum; citation_author=RJ Janssen, P Jyl&#228;nki, RPC Kessels, MAJ Gerven; citation_volume=119; citation_publication_date=2015; citation_pages=398-405; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging; citation_author=TE Behrens; citation_volume=6; citation_publication_date=2003; citation_pages=750-757; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Neuropsychopharmacology; citation_title=The reward circuit: linking primate anatomy and human imaging; citation_author=SN Haber, B Knutson; citation_volume=35; citation_publication_date=2010; citation_pages=4-26; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=J. Comp. Neurol.; citation_title=Multiarchitectonic and stereotactic atlas of the human thalamus; citation_author=A Morel, M Magnin, D Jeanmonod; citation_volume=387; citation_publication_date=1997; citation_pages=588-630; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=J. Comp. Neurol.; citation_title=Shell and core in monkey and human nucleus accumbens identified with antibodies to calbindin-D28k; citation_author=GE Meredith, A Pattiselanno, HJ Groenewegen, SN Haber; citation_volume=365; citation_publication_date=1996; citation_pages=628-639; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiatry Cogn. Neurosci. Neuroimaging; citation_title=Personalized transcranial magnetic stimulation in psychiatry; citation_author=L Cocchi, A Zalesky; citation_volume=3; citation_publication_date=2018; citation_pages=731-741; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=A motivational and neuropeptidergic hub: anatomical and functional diversity within the nucleus accumbens shell; citation_author=DC Castro, MR Bruchas; citation_volume=102; citation_publication_date=2019; citation_pages=529-552; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Regional specialization within the human striatum for diverse psychological functions; citation_author=WM Pauli, RC O&#8217;Reilly, T Yarkoni, TD Wager; citation_volume=113; citation_publication_date=2016; citation_pages=1907-1912; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Subcellular connectivity underlies pathway-specific signaling in the nucleus accumbens; citation_author=AF MacAskill, JP Little, JM Cassel, AG Carter; citation_volume=15; citation_publication_date=2012; citation_pages=1624-1626; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Convergent inputs from the hippocampus and thalamus to the nucleus accumbens regulate dopamine neuron activity; citation_author=SM Perez, DJ Lodge; citation_volume=38; citation_publication_date=2018; citation_pages=10607-10618; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Effects of nicotine on the nucleus accumbens and similarity to those of addictive drugs; citation_author=FE Pontieri, G Tanda, F Orzi, G Chiara; citation_volume=382; citation_publication_date=1996; citation_pages=255-257; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Psychopharmacol.; citation_title=Abnormal hippocampal neurochemistry in smokers: evidence from proton magnetic resonance spectroscopy at 3T; citation_author=J Gallinat; citation_volume=27; citation_publication_date=2007; citation_pages=80-84; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Res.; citation_title=Fetal and adolescent nicotine administration: effects on CNS serotonergic systems; citation_author=Z Xu, FJ Seidler, SF Ali, W Slikker, TA Slotkin; citation_volume=914; citation_publication_date=2001; citation_pages=166-178; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Schizophrenia Res.; citation_title=A meta-analysis of worldwide studies demonstrates an association between schizophrenia and tobacco smoking behaviors; citation_author=J Leon, FJ Diaz; citation_volume=76; citation_publication_date=2005; citation_pages=135-157; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Nicotine reverses hypofrontality in animal models of addiction and schizophrenia; citation_author=F Koukouli; citation_volume=23; citation_publication_date=2017; citation_pages=347-354; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Permutation inference for the general linear model; citation_author=AM Winkler, GR Ridgway, MA Webster, SM Smith, TE Nichols; citation_volume=92; citation_publication_date=2014; citation_pages=381-397; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Multi-level block permutation; citation_author=AM Winkler, MA Webster, D Vidaurre, TE Nichols, SM Smith; citation_volume=123; citation_publication_date=2015; citation_pages=253-268; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Resting-state fMRI in the Human Connectome Project; citation_author=SM Smith; citation_volume=80; citation_publication_date=2013; citation_pages=144-168; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Function in the human connectome: task-fMRI and individual differences in behavior; citation_author=DM Barch; citation_volume=80; citation_publication_date=2013; citation_pages=169-189; citation_id=CR54"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=The minimal preprocessing pipelines for the Human Connectome Project; citation_author=MF Glasser; citation_volume=80; citation_publication_date=2013; citation_pages=105-124; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers; citation_author=G Salimi-Khorshidi; citation_volume=90; citation_publication_date=2014; citation_pages=449-468; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging; citation_author=L Griffanti; citation_volume=95; citation_publication_date=2014; citation_pages=232-247; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI; citation_author=L Parkes, B Fulcher, M Yucel, A Fornito; citation_volume=171; citation_publication_date=2018; citation_pages=415-436; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion; citation_author=JD Power, KA Barnes, AZ Snyder, BL Schlaggar, SE Petersen; citation_volume=59; citation_publication_date=2012; citation_pages=2142-2154; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=An improved framework for confound regression and filtering for control of motion artifact in the preprocessing of resting-state functional connectivity data; citation_author=TD Satterthwaite; citation_volume=64; citation_publication_date=2013; citation_pages=240-256; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=The influence of head motion on intrinsic functional connectivity MRI; citation_author=KR Dijk, MR Sabuncu, RL Buckner; citation_volume=59; citation_publication_date=2012; citation_pages=431-438; citation_id=CR61"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Using multi-echo simultaneous multi-slice (SMS) EPI to improve functional MRI of the subcortical nuclei of the basal ganglia at ultra-high field (7T); citation_author=AM Puckett; citation_volume=172; citation_publication_date=2018; citation_pages=886-895; citation_id=CR62"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=The Human Connectome Project&#8217;s neuroimaging approach; citation_author=MF Glasser; citation_volume=19; citation_publication_date=2016; citation_pages=1175-1187; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data; citation_author=MF Glasser; citation_volume=181; citation_publication_date=2018; citation_pages=692-717; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Multi-task connectivity reveals flexible hubs for adaptive task control; citation_author=MW Cole; citation_volume=16; citation_publication_date=2013; citation_pages=1348-1355; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=fMRIPrep: a robust preprocessing pipeline for functional MRI; citation_author=O Esteban; citation_volume=16; citation_publication_date=2019; citation_pages=111-116; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=J. Comp. Neurol.; citation_title=A Golgi analysis of the primate globus pallidus. III. Spatial organization of the striato&#8211;pallidal complex; citation_author=G Percheron, J Yelnik, C Francois; citation_volume=227; citation_publication_date=1984; citation_pages=214-227; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Quantitative comparison of 21 protocols for labeling hippocampal subfields and parahippocampal subregions in in vivo MRI: towards a harmonized segmentation protocol; citation_author=PA Yushkevich; citation_volume=111; citation_publication_date=2015; citation_pages=526-541; citation_id=CR68"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Imaging Behav.; citation_title=Human subcortical brain asymmetries in 15,847 people worldwide reveal effects of age and sex; citation_author=T Guadalupe; citation_volume=11; citation_publication_date=2017; citation_pages=1497-1514; citation_id=CR69"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Comparing functional MRI protocols for small, iron-rich basal ganglia nuclei such as the subthalamic nucleus at 7&#8201;T and 3&#8201;T; citation_author=G Hollander, MC Keuken, W Zwaag, BU Forstmann, R Trampel; citation_volume=38; citation_publication_date=2017; citation_pages=3226-3248; citation_id=CR70"/>
    <meta name="citation_reference" content="citation_journal_title=Neural Comput.; citation_title=Laplacian eigenmaps for dimensionality reduction and data representation; citation_author=M Belkin, P Niyogi; citation_volume=15; citation_publication_date=2003; citation_pages=1373-1396; citation_id=CR71"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Graph analysis of the human connectome: promise, progress, and pitfalls; citation_author=A Fornito, A Zalesky, M Breakspear; citation_volume=80; citation_publication_date=2013; citation_pages=426-444; citation_id=CR72"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Probabilistic tractography recovers a rostrocaudal trajectory of connectivity variability in the human insular cortex; citation_author=L Cerliani; citation_volume=33; citation_publication_date=2012; citation_pages=2005-2034; citation_id=CR73"/>
    <meta name="citation_reference" content="citation_journal_title=NMR Biomed.; citation_title=Diffusion MRI fiber tractography of the brain; citation_author=B Jeurissen, M Descoteaux, S Mori, A Leemans; citation_volume=32; citation_publication_date=2019; citation_pages=e3785; citation_id=CR74"/>
    <meta name="citation_reference" content="Alexander, D. C. in Visualization and Processing of Tensor Fields (eds Weickert J. &amp; Hagen, H.) Ch. 5 (Springer, 2006)."/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=A Bayesian approach for stochastic white matter tractography; citation_author=O Friman, G Farneback, CF Westin; citation_volume=25; citation_publication_date=2006; citation_pages=965-978; citation_id=CR76"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Imaging; citation_title=Geometrically constrained two-tensor model for crossing tracts in DWI; citation_author=S Peled, O Friman, F Jolesz, CF Westin; citation_volume=24; citation_publication_date=2006; citation_pages=1263-1270; citation_id=CR77"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Probabilistic diffusion tractography with multiple fibre orientations: what can we gain?; citation_author=TE Behrens, HJ Berg, S Jbabdi, MF Rushworth, MW Woolrich; citation_volume=34; citation_publication_date=2007; citation_pages=144-155; citation_id=CR78"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Int. Soc. Magn. Reson. Med.; citation_title=Diffusion Toolkit: a software package for diffusion imaging data processing and tractography; citation_author=R Wang, T Benner, A Sorensen, V Wedeen; citation_volume=15; citation_publication_date=2007; citation_pages=3720; citation_id=CR79"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=Automatic tractography segmentation using a high-dimensional white matter atlas; citation_author=LJ O&#8217;Donnell, CF Westin; citation_volume=26; citation_publication_date=2007; citation_pages=1562-1575; citation_id=CR80"/>
    <meta name="citation_reference" content="M&#252;ller, M. in Information Retrieval for Music and Motion 69&#8211;84 (Springer, 2007)."/>
    <meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. B; citation_title=Discovering the false discovery rate; citation_author=Y Benjamini; citation_volume=72; citation_publication_date=2010; citation_pages=405-416; citation_id=CR82"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Extracting the multiscale backbone of complex weighted networks; citation_author=M&#193; Serrano, M Bogu&#241;&#225;, A Vespignani; citation_volume=106; citation_publication_date=2009; citation_pages=6483-6488; citation_id=CR83"/>
    <meta name="citation_reference" content="citation_journal_title=Signal Process.; citation_title=Topographic distance and watershed lines; citation_author=F Meyer; citation_volume=38; citation_publication_date=1994; citation_pages=113-125; citation_id=CR84"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data; citation_author=SB Eickhoff; citation_volume=25; citation_publication_date=2005; citation_pages=1325-1335; citation_id=CR85"/>
    <meta name="citation_reference" content="citation_journal_title=N. J. Phys.; citation_title=Detecting the overlapping and hierarchical community structure in complex networks; citation_author=A Lancichinetti, S Fortunato, J Kert&#233;sz; citation_volume=11; citation_publication_date=2009; citation_pages=033015; citation_id=CR86"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Task-induced brain state manipulation improves prediction of individual traits; citation_author=AS Greene, S Gao, D Scheinost, RT Constable; citation_volume=9; citation_publication_date=2018; citation_id=CR87"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Intrinsic and task-evoked network architectures of the human brain; citation_author=MW Cole, DS Bassett, JD Power, TS Braver, SE Petersen; citation_volume=83; citation_publication_date=2014; citation_pages=238-251; citation_id=CR88"/>
    <meta name="citation_reference" content="citation_journal_title=J. Cogn. Neurosci.; citation_title=Cognitive network neuroscience; citation_author=JD Medaglia, ME Lynall, DS Bassett; citation_volume=27; citation_publication_date=2015; citation_pages=1471-1491; citation_id=CR89"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A positive-negative mode of population covariation links brain connectivity, demographics and behavior; citation_author=SM Smith; citation_volume=18; citation_publication_date=2015; citation_pages=1565-1567; citation_id=CR90"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Schizophrenia, neuroimaging and connectomics; citation_author=A Fornito, A Zalesky, C Pantelis, ET Bullmore; citation_volume=62; citation_publication_date=2012; citation_pages=2296-2314; citation_id=CR91"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Linked dimensions of psychopathology and connectivity in functional brain networks; citation_author=CH Xia; citation_volume=9; citation_publication_date=2018; citation_id=CR92"/>
    <meta name="citation_reference" content="citation_journal_title=J. Stat. Mech. Theory Exp.; citation_title=Fast unfolding of communities in large networks; citation_author=VD Blondel, J-L Guillaume, R Lambiotte, E Lefebvre; citation_volume=2008; citation_publication_date=2008; citation_pages=P10008; citation_id=CR93"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. K. Nederlandse Akademie van Wetenschappen. Ser. A; citation_title=Oder tests for two-sample problem and their power; citation_author=B Waerden; citation_volume=55; citation_publication_date=1952; citation_pages=453-458; citation_id=CR94"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Validating the independent components of neuroimaging time series via clustering and visualization; citation_author=J Himberg, A Hyvarinen, F Esposito; citation_volume=22; citation_publication_date=2004; citation_pages=1214-1222; citation_id=CR95"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Neural Netw.; citation_title=Fast and robust fixed-point algorithms for independent component analysis; citation_author=A Hyvarinen; citation_volume=10; citation_publication_date=1999; citation_pages=626-634; citation_id=CR96"/>
    <meta name="citation_reference" content="citation_journal_title=Nav. Res. Logist. Q; citation_title=The Hungarian method for the assignment problem; citation_author=HW Kuhn; citation_volume=2; citation_publication_date=1955; citation_pages=83-97; citation_id=CR97"/>
    <meta name="citation_reference" content="citation_journal_title=J. Bus. Economic Stat.; citation_title=A nonstochastic interpretation of reported significance levels; citation_author=D Freedman, D Lane; citation_volume=1; citation_publication_date=1983; citation_pages=292-298; citation_id=CR98"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A repeated molecular architecture across thalamic pathways; citation_author=JW Phillips; citation_volume=22; citation_publication_date=2019; citation_pages=1925-1935; citation_id=CR99"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=The human thalamus is an integrative hub for functional brain networks; citation_author=K Hwang, MA Bertolero, WB Liu, M D&#8217;Esposito; citation_volume=37; citation_publication_date=2017; citation_pages=5594-5607; citation_id=CR100"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=The low-dimensional neural architecture of cognitive complexity is related to activity in medial thalamic nuclei; citation_author=JM Shine; citation_volume=104; citation_publication_date=2019; citation_pages=849-855.e3; citation_id=CR101"/>
    <meta name="citation_reference" content="citation_journal_title=Prog. Neurobiol.; citation_title=The basal ganglia: focused selection and inhibition of competing motor programs; citation_author=JW Mink; citation_volume=50; citation_publication_date=1996; citation_pages=381-425; citation_id=CR102"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=From circuits to behaviour in the amygdala; citation_author=PH Janak, KM Tye; citation_volume=517; citation_publication_date=2015; citation_pages=284-292; citation_id=CR103"/>
    <meta name="citation_author" content="Tian, Ye"/>
    <meta name="citation_author_institution" content="Melbourne Neuropsychiatry Centre, Department of Psychiatry, The University of Melbourne, Parkville, Australia"/>
    <meta name="citation_author" content="Margulies, Daniel S."/>
    <meta name="citation_author_institution" content="Centre National de la Recherche Scientifique (CNRS) UMR 8002, Integrative Neuroscience and Cognition Center, Universit&#233; de Paris, Paris, France"/>
    <meta name="citation_author" content="Breakspear, Michael"/>
    <meta name="citation_author_institution" content="Discipline of Psychiatry, Faculty of Health and Medicine, University of Newcastle, Newcastle, Australia"/>
    <meta name="citation_author_institution" content="School of Psychology, Faculty of Science, University of Newcastle, Newcastle, Australia"/>
    <meta name="citation_author" content="Zalesky, Andrew"/>
    <meta name="citation_author_institution" content="Melbourne Neuropsychiatry Centre, Department of Psychiatry, The University of Melbourne, Parkville, Australia"/>
    <meta name="citation_author_institution" content="Department of Biomedical Engineering, Melbourne School of Engineering, The University of Melbourne, Parkville, Australia"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="Topographic organization of the human subcortex unveiled with functional connectivity gradients"/>
    <meta name="twitter:description" content="Nature Neuroscience - This work by Tian and colleagues unveils the extraordinarily complex layout of the human subcortex by identifying 27 new functional regions that organize hierarchically across..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-020-00711-6"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="Topographic organization of the human subcortex unveiled with functional connectivity gradients - Nature Neuroscience"/>
    <meta property="og:description" content="This work by Tian and colleagues unveils the extraordinarily complex layout of the human subcortex by identifying 27 new functional regions that organize hierarchically across four scales and adapt to changing cognitive demands."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-020-00711-6;doi=10.1038/s41593-020-00711-6;techmeta=36,57,59;subjmeta=1627,1647,245,378,3920,631;kwrd=Functional+magnetic+resonance+imaging,Neural+circuits">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-1700184643&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-020-00711-6%26doi%3D10.1038/s41593-020-00711-6%26techmeta%3D36,57,59%26subjmeta%3D1627,1647,245,378,3920,631%26kwrd%3DFunctional+magnetic+resonance+imaging,Neural+circuits">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-1700184643&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-020-00711-6%26doi%3D10.1038/s41593-020-00711-6%26techmeta%3D36,57,59%26subjmeta%3D1627,1647,245,378,3920,631%26kwrd%3DFunctional+magnetic+resonance+imaging,Neural+circuits"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-020-00711-6'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        Topographic organization of the human subcortex unveiled with functional connectivity gradients
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00711-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2020-09-28">28 September 2020</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Topographic organization of the human subcortex unveiled with functional connectivity gradients</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ye-Tian-Aff1" data-author-popup="auth-Ye-Tian-Aff1" data-author-search="Tian, Ye" data-corresp-id="c1">Ye Tian<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-3107-5550"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3107-5550</a></span><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Daniel_S_-Margulies-Aff2" data-author-popup="auth-Daniel_S_-Margulies-Aff2" data-author-search="Margulies, Daniel S.">Daniel S. Margulies</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael-Breakspear-Aff3-Aff4" data-author-popup="auth-Michael-Breakspear-Aff3-Aff4" data-author-search="Breakspear, Michael">Michael Breakspear</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-4943-3969"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4943-3969</a></span><sup class="u-js-hide"><a href="#Aff3">3</a>,<a href="#Aff4">4</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 4 authors for this article" title="Show all 4 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Andrew-Zalesky-Aff1-Aff5" data-author-popup="auth-Andrew-Zalesky-Aff1-Aff5" data-author-search="Zalesky, Andrew" data-corresp-id="c2">Andrew Zalesky<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-2298-9908"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2298-9908</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff5">5</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 23</b>, <span class="u-visually-hidden">pages </span>1421–1432 (<span data-test="article-publication-year">2020</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">20k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">199 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">258 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-020-00711-6/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/functional-magnetic-resonance-imaging" data-track="click" data-track-action="view subject" data-track-label="link">Functional magnetic resonance imaging</a></li><li class="c-article-subject-list__subject"><a href="/subjects/neural-circuit" data-track="click" data-track-action="view subject" data-track-label="link">Neural circuits</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Brain atlases are fundamental to understanding the topographic organization of the human brain, yet many contemporary human atlases cover only the cerebral cortex, leaving the subcortex a terra incognita. We use functional MRI (fMRI) to map the complex topographic organization of the human subcortex, revealing large-scale connectivity gradients and new areal boundaries. We unveil four scales of subcortical organization that recapitulate well-known anatomical nuclei at the coarsest scale and delineate 27 new bilateral regions at the finest. Ultrahigh field strength fMRI corroborates and extends this organizational structure, enabling the delineation of finer subdivisions of the hippocampus and the amygdala, while task-evoked fMRI reveals a subtle subcortical reorganization in response to changing cognitive demands. A new subcortical atlas is delineated, personalized to represent individual differences and used to uncover reproducible brain–behavior relationships. Linking cortical networks to subcortical regions recapitulates a task-positive to task-negative axis. This new atlas enables holistic connectome mapping and characterization of cortico–subcortical connectivity.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00711-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00711-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41593-024-01597-4/MediaObjects/41593_2024_1597_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41593-024-01597-4?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41593-024-01597-4">Co-dependent excitatory and inhibitory plasticity accounts for quick, stable and long-lasting memories in biological networks
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">20 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Everton J. Agnes &amp; Tim P. Vogels</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-023-07011-6/MediaObjects/41586_2023_7011_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-023-07011-6?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41586-023-07011-6">Spatial transcriptomics reveal neuron–astrocyte synergy in long-term memory
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">07 February 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Wenfei Sun, Zhihui Liu, … Stephen R. Quake</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-023-06812-z/MediaObjects/41586_2023_6812_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-023-06812-z?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41586-023-06812-z">A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">13 December 2023</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Zizhen Yao, Cindy T. J. van Velthoven, … Hongkui Zeng</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'topic',
                        model: 'visits_v2',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711540482,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>The ubiquity of MRI has enabled high-throughput mapping of standardized brain parcellation atlases that represent consensus cortical boundaries across hundreds of scanned people<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672–686 (2018)." href="/articles/s41593-020-00711-6#ref-CR1" id="ref-link-section-d27595475e512">1</a></sup>. Compared with the cerebral cortex, efforts to parcellate the human subcortex are scarcer. Indeed, many leading brain atlases delineate only cortical regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e516">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Yeo, B. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-020-00711-6#ref-CR3" id="ref-link-section-d27595475e519">3</a></sup>.</p><p>The subcortex is widely implicated in the pathophysiology of many neuropsychiatric disorders, and its role in cognition is unequivocal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Shepherd, G. M. Corticostriatal connectivity and its role in disease. Nat. Rev. Neurosci. 14, 278–291 (2013)." href="/articles/s41593-020-00711-6#ref-CR4" id="ref-link-section-d27595475e526">4</a></sup>. To better elucidate the role of the subcortex in health and disease, further work is needed to advance recent efforts to map the functional architecture of the subcortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Choi, E. Y., Yeo, B. T. &amp; Buckner, R. L. The organization of the human striatum estimated by intrinsic functional connectivity. J. Neurophysiol. 108, 2242–2263 (2012)." href="/articles/s41593-020-00711-6#ref-CR5" id="ref-link-section-d27595475e530">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ji, J. L. et al. Mapping the human brain’s cortical–subcortical functional network organization. NeuroImage 185, 35–57 (2019)." href="/articles/s41593-020-00711-6#ref-CR6" id="ref-link-section-d27595475e533">6</a></sup>. This study aims to unravel the incredibly complex functional organization of the human subcortex by developing and validating a comprehensive subcortical atlas using 3 Tesla (3T) and 7T fMRI.</p><p>Cortical boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Cohen, A. L. et al. Defining functional areas in individual human brains using resting functional connectivity MRI. NeuroImage 41, 45–57 (2008)." href="/articles/s41593-020-00711-6#ref-CR7" id="ref-link-section-d27595475e540">7</a></sup> within a brain atlas are often delineated at locations of abrupt spatial change in cortical microstructure, function and/or connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e544">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e547">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-020-00711-6#ref-CR9" id="ref-link-section-d27595475e550">9</a></sup>. Cortical gradients<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Huntenburg, J. M., Bazin, P. L. &amp; Margulies, D. S. Large-scale gradients in human cortical organization. Trends Cogn. Sci. 22, 21–31 (2018)." href="/articles/s41593-020-00711-6#ref-CR10" id="ref-link-section-d27595475e554">10</a></sup> and the related connectopy concept<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="/articles/s41593-020-00711-6#ref-CR11" id="ref-link-section-d27595475e558">11</a></sup> offer a complementary representation of cortical topography. Rather than delineating discrete spatial boundaries, topographic change is represented continuously along overlapping organizational axes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-020-00711-6#ref-CR12" id="ref-link-section-d27595475e562">12</a></sup>.</p><p>Brain cartography studies tend to espouse one or the other of these two representations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672–686 (2018)." href="/articles/s41593-020-00711-6#ref-CR1" id="ref-link-section-d27595475e569">1</a></sup>. This is exemplified by current contention about whether the functional topography of the cerebellum is best represented by continuous gradients<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Guell, X., Schmahmann, J. D., Gabrieli, J. &amp; Ghosh, S. S. Functional gradients of the cerebellum. eLife 7, e36652 (2018)." href="/articles/s41593-020-00711-6#ref-CR13" id="ref-link-section-d27595475e573">13</a></sup> or discrete boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="King, M., Hernandez-Castillo, C. R., Poldrack, R. A., Ivry, R. B. &amp; Diedrichsen, J. Functional boundaries in the human cerebellum revealed by a multi-domain task battery. Nat. Neurosci. 22, 1371–1378 (2019)." href="/articles/s41593-020-00711-6#ref-CR14" id="ref-link-section-d27595475e577">14</a></sup>. However, the two paradigms are not necessarily contradictory. Spatial variation across expanses of the cerebrum may be evident but too gradual to warrant boundary delineation; thus, a continuum might be deemed to provide the most parsimonious representation. This leads to the following question of model selection: how sharp and conspicuous must a change be to warrant boundary delineation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Tian, Y. &amp; Zalesky, A. Characterizing the functional connectivity diversity of the insula cortex: subregions, diversity curves and behavior. NeuroImage 183, 716–733 (2018)." href="/articles/s41593-020-00711-6#ref-CR15" id="ref-link-section-d27595475e581">15</a></sup>? Here, we map functional connectivity gradients of the human subcortex and delineate subcortical boundaries only when justified by a formal model selection process.</p><p>Subcortical geometry is fundamentally different to the convoluted sheet-like structure of the cerebral cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Coalson, T. S., Van Essen, D. C. &amp; Glasser, M. F. The impact of traditional neuroimaging methods on the spatial localization of cortical areas. Proc. Natl Acad. Sci. USA 115, E6356–E6365 (2018)." href="/articles/s41593-020-00711-6#ref-CR16" id="ref-link-section-d27595475e589">16</a></sup>. New paradigms are therefore required to facilitate subcortical cartography. To this end, we developed gradientography, a fMRI analog of diffusion MRI tractography that enables the quantification of subcortical connectivity gradients and a statistically principled formalism to guide boundary delineation. This allows us to comprehensively parcellate the subcortex on the basis of resting-state functional connectivity profiles and to produce a standardized atlas representing consensus boundaries among more than 1,000 healthy adults. The new atlas comprises four scales that recapitulate well-known anatomical nuclei at the coarsest scale and reveal 27 bilateral regions at the finest. The new atlas is replicated using ultrahigh field strength fMRI (7T) and is personalized to account for individual connectivity differences. An analysis of task-evoked fMRI (tfMRI) reveals a subtle reorganization of subcortical topography in response to changing cognitive demands. We use the new atlas to uncover correlational relationships between subcortical functional connectivity and individual variation in five behavioral domains and to reveal a new task-positive to task-negative organizational axis in cortico–subcortical connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Fox, M. D. et al. The human brain is intrinsically organized into dynamic, anticorrelated functional networks. Proc. Natl Acad. Sci. USA 102, 9673–9678 (2005)." href="/articles/s41593-020-00711-6#ref-CR17" id="ref-link-section-d27595475e593">17</a></sup>. This study provides an important step toward understanding the functional organization of the human subcortex.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><p>We investigated the functional connectivity architecture of the human subcortex using resting-state fMRI (rfMRI) data acquired at 3T (<i>n</i> = 1,080) and 7T (<i>n</i> = 183) and tfMRI data acquired at 3T (<i>n</i> = 725) from healthy adults participating in the Human Connectome Project (HCP)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Van Essen, D. C. et al. The WU–Minn Human Connectome Project: an overview. NeuroImage 80, 62–79 (2013)." href="/articles/s41593-020-00711-6#ref-CR18" id="ref-link-section-d27595475e614">18</a></sup>. Demographics are shown in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a>. Two 3T rfMRI sessions were acquired for each individual, with the first session used as the primary dataset (REST1) and the second used for replication (REST2). Gradients in functional connectivity were mapped using Laplacian eigenmaps and a newly developed procedure called gradientography. Formal model selection was used to determine whether gradient magnitude peaks were sufficiently large to warrant boundary delineation. The competing model posited a gradual spatial change in connectivity without discrete boundaries. This resulted in a new fMRI-based multiscale parcellation atlas of the subcortex, which we used to reveal new insights into subcortical function and architecture.</p><h3 class="c-article__sub-heading" id="Sec3">Connectivity gradients and gradientography</h3><p>Whole-brain functional connectivity fingerprints<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 1664 (2015)." href="/articles/s41593-020-00711-6#ref-CR19" id="ref-link-section-d27595475e628">19</a></sup> were mapped for each subcortical voxel and individual. Spatial gradients characterizing continuous modes of functional connectivity variation across the topography of the subcortex were computed using Laplacian eigenmaps and the connectopy method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="/articles/s41593-020-00711-6#ref-CR11" id="ref-link-section-d27595475e632">11</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1a</a>). The eigenmap, or connectopy, explaining the most variance in functional connectivity was called gradient I (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>), and the next two successive eigenmaps were called gradient II and gradient III, respectively (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a>). Further gradients explained substantially less variance and were therefore not considered (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">2</a>). A new technique called gradientography was developed to visualize the connectivity gradients. Gradientography involves estimation of the principal gradient direction for each subcortical voxel, representing gradient directions and magnitudes with tensors, and then propagation of streamlines through the tensor field using established tools for diffusion MRI tractography. In this way, approximately 15,000 streamlines were initialized and propagated throughout the entire subcortical volume to map the principal organizational axes of the subcortex for gradient I (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>) and for gradients II and III (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Gradientography in the human subcortex."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Gradientography in the human subcortex.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="315"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, A schematic of functional connectivity gradient mapping. The similarity in connectional fingerprints between pairs of subcortical voxels was measured using the <i>η</i><sup>2</sup> coefficient. Pairs of voxels with similar fingerprints connected to similar brain areas. The similarity matrix (<i>S</i>) shown represents a group consensus for 1,080 healthy adults. The similarity matrix was transformed into a sparse graph with adjacency matrix <i>W</i>. The graph nodes correspond to subcortical voxels. Eigenvalues and eigenvectors were computed for the graph Laplacian. The eigenvectors with the second, third and fourth smallest eigenvalue are called gradients I, II and III, respectively. Each gradient provides a mapping of continuous variation in functional connectivity across the topography of the subcortex. <i>N</i>, number of subcortical voxels; <i>M</i>, number of whole-brain gray matter voxels; <i>T</i>, number of time frames. <b>b</b>, A schematic of the gradientography pipeline. Gradients were projected onto subcortical voxels to enable anatomical visualization. The axial slice shown is for gradient I. The Sobel operator was used to estimate the local gradient direction and magnitude for each subcortical voxel. The arrows shown point in the direction of the estimated gradients. Arrow lengths are normalized to unit length. Tensors were fitted to the gradient field. Tensors are colored according to the gradient direction (red, left–right; green, posterior–anterior; blue, superior–inferior). Streamlines were propagated through the tensor field using methods for diffusion MRI tractography. Streamlines are shown from a posterior viewpoint and colored according to the gradient magnitude. The red bands indicate peaks in the gradient magnitude that demarcate hippocampus–thalamus and caudate–thalamus boundaries.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Gradient magnitude peaks indicated the locations of putative functional boundaries between regions. To visualize peak locations, streamlines were colored according to the gradient magnitude. Peak locations are shown as circumscribed red bands (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>) and recapitulated boundaries between well-known anatomical nuclei. Local maxima in the gradient magnitude images showed moderate consistency among the three gradients (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">3</a>); thus, only gradient I was used for boundary delineation. Gradient I streamlines were partitioned into dorsal and ventral groups at the location of the anatomical discontinuity between the caudate tail and the anterior extent of the thalamus. The dorsal group covered the globus pallidus (GP) and the striatum (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2a</a>), whereas the amygdala, hippocampus and thalamus constituted the ventral group (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Model selection: which gradient magnitude peaks are sufficiently large to warrant boundary delineation?"><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Model selection: which gradient magnitude peaks are sufficiently large to warrant boundary delineation?</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="387"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>,<b>b</b>, Streamlines were partitioned into bilaterally symmetric dorsal (<b>a</b>) and ventral (<b>b</b>) groups. The streamlines shown are colored according to the gradient magnitude. Dorsal streamlines traverse the GP, the putamen, the NAc and the caudate. Ventral streamlines traverse the left and right thalamus (THA; where a is anterior and p is posterior), the hippocampus (HIP) and the amygdala (AMY). <b>c</b>,<b>d</b>, Dorsal (<b>c</b>) and ventral (<b>d</b>) diversity curves were mapped to parameterize the gradient magnitude as a function of spatial position along the length of dorsal and ventral streamline trajectories. Black triangles indicate diversity curve peaks (local maxima) demarcating locations of putative functional boundaries. Diversity curves are colored according to eigenmap values for gradient I. Null diversity curves (<i>n</i> = 100) for the null hypothesis of an absence of discrete boundaries are shown in gray, with shading used to show 95% confidence intervals (CI) and solid lines indicating the mean. Peaks in the null diversity curves are due to the effects of geometry. Peaks for which the null hypothesis could be rejected are demarcated with blue asterisks (<i>P</i> &lt; 0.05 one-sided, FDR-corrected for all 12 peaks).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We next sought to quantify the size of each gradient magnitude peak to determine whether gradients were sufficiently abrupt to warrant boundary delineation. To this end, gradient magnitude images were projected onto two-dimensional trajectories called diversity curves<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Tian, Y. &amp; Zalesky, A. Characterizing the functional connectivity diversity of the insula cortex: subregions, diversity curves and behavior. NeuroImage 183, 716–733 (2018)." href="/articles/s41593-020-00711-6#ref-CR15" id="ref-link-section-d27595475e772">15</a></sup>. A separate diversity curve was mapped for the dorsal and ventral streamline group. The dorsal diversity curve comprised three distinct local maxima indicating the locations of putative boundaries between the GP–putamen, the putamen–nucleus accumbens (NAc) and the NAc–caudate (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2c</a>). Nine local maxima were evident in the ventral diversity curve, which suggested boundary locations that distinguished the amygdala, the anterior and posterior hippocampus, the anterior and posterior thalamus, as well as the left and right thalamus (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2d</a>). Locations of gradient magnitude peaks could not be explained by structural MRI signal properties and potential confounds, including the fMRI signal-to-noise ratio (SNR), the fMRI signal magnitude and the T1-weighted and T1/T2-weighted contrast (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">4</a>).</p><h3 class="c-article__sub-heading" id="Sec4">A new subcortical atlas</h3><p>The magnitude of local maxima in the diversity curves varied by a factor of more than ten, which suggests that putative boundaries between some regions were markedly more abrupt than others. How large should a peak be to warrant boundary delineation? Small peaks can emerge due to the convoluted geometry of the subcortex and other confounds (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">5</a>). False-positive boundaries should not be delineated at such peaks. Therefore, for each peak, a formal model selection procedure was used to test the null hypothesis that the peak magnitude could be explained by geometry and/or other confounds. Rejection of the null hypothesis indicated that boundary delineation was warranted. Otherwise, if the peak magnitude was not sufficiently large to enable rejection of the null hypothesis, a continuous representation of spatial variation in connectivity was deemed to be the most parsimonious model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Tian, Y. &amp; Zalesky, A. Characterizing the functional connectivity diversity of the insula cortex: subregions, diversity curves and behavior. NeuroImage 183, 716–733 (2018)." href="/articles/s41593-020-00711-6#ref-CR15" id="ref-link-section-d27595475e796">15</a></sup>. The null hypothesis was tested using a geometry-preserving null model (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00711-6#Sec18">Methods</a>). Diversity curves generated using the null model are colored gray in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2c,d</a>. The actual peaks significantly exceeded all null data (<i>P</i> &lt; 0.05, false discovery rate (FDR)-corrected for all 12 peaks), except for peaks separating the bilateral hippocampus/thalamus and the anterior–posterior hippocampus. However, given that the hippocampus and the thalamus are anatomically and functionally distinct and that the peak separating these two regions was unequivocal, we made an exception and delineated a boundary here, whereas the putative anterior–posterior hippocampus boundary was not delineated, which is consistent with the null data. While peaks in the null diversity curves often corresponded with locations to peaks in the actual diversity curves, including the functional boundaries between the putamen–NAc, the posterior hippocampus–posterior thalamus and the left–right thalamus, distinct peaks were also evident where the effect of geometry is minimal (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">5a</a>), such as the amygadala–anterior hippocampus, the anterior–posterior thalamus and the GP–putamen. This indicates that subcortical geometry can predict and account for the location of some functional boundaries, but geometry per se cannot explain the magnitude of all boundaries, except for the hippocampus–thalamus (see “Further methodological considerations” in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00711-6#Sec18">Methods</a>). Finally, the watershed transform algorithm was used to segment subcortical voxels into contiguous regions such that boundaries demarcated watershed lines.</p><p>This resulted in eight bilateral regions that constituted a relatively coarse atlas, referred to as the scale I atlas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig3">3a</a>). The scale I atlas recapitulated well-known anatomical nuclei, namely, the amygdala, the hippocampus, the thalamus, the GP, the NAc, the caudate and the putamen, as well as an anteroposterior thalamic partition, which corresponded to the spatial distribution of the two subpopulations of thalamic projection cells (matrix and core)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Müller, E. et al. Core and matrix thalamic sub-populations relate to spatio-temporal cortical connectivity gradients. NeuroImage 222, 117224 (2020)." href="/articles/s41593-020-00711-6#ref-CR20" id="ref-link-section-d27595475e822">20</a></sup>. Gradient mapping, boundary delineation and model selection were then iteratively performed separately for each region constituting the scale I atlas. This resulted in a finer atlas, called the scale II atlas, comprising 16 bilateral regions. In turn, the null hypothesis was independently tested for each region in the scale II atlas. Recursively continuing this process until the null hypothesis could not be rejected for any regions yielded four progressively more detailed atlases of the subcortex (scale I: 16 regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig3">3a</a>); scale II: 32 regions; scale III: 50 regions; scale IV: 54 regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig3">3b</a>); Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">6</a>). The new atlas is hierarchically organized, with the entire subcortex residing at the base of a hierarchical tree and the smallest regions constituting the scale IV atlas defining the leaves (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig4">4a,b</a>). Subcortical gradients were also hierarchically organized (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig4">4b,c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">7</a>). A schematic of the recursive procedure is provided in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">8</a>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="A multiscale group-consensus parcellation atlas derived from 3T-rfMRI acquired in 1,080 healthy adults."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: A multiscale group-consensus parcellation atlas derived from 3T-rfMRI acquired in 1,080 healthy adults.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="408"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Scale I is the coarsest atlas and comprises eight bilateral regions that recapitulate anatomical nuclei, including the caudate nucleus (CAU), the NAc, the putamen (PUT), the amygdala, the hippocampus and the thalamus. An anteroposterior division within the thalamus is also present. <b>b</b>, Scale IV is the finest scale of the parcellation hierarchy and comprises 27 bilateral regions. Homologous regions are identically colored. d, dorsal; i, inferior; l, lateral; m, medial; s, superior; v, ventral (detailed nomenclature for all regions is provided in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">2</a>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Organization of the human subcortex."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Organization of the human subcortex.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="512"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>Model selection and null hypothesis testing was used to determine whether gradient magnitude peaks were sufficiently large to warrant boundary delineation. This process was recursively repeated for each new region delineated, unveiling a multiscale parcellation architecture. The null hypothesis of an absence of boundaries could not be rejected for any of the regions constituting scale IV; thus, further (finer) parcellation was not warranted. <b>a</b>, A top-down representation of the parcellation hierarchy. Regions are shown atop a reference structural MRI image. Homologous regions are identically colored. <sup>†</sup>The caudate was removed for visualization purposes; <sup>‡</sup>the putamen and the GP were removed for visualization purposes. <b>b</b>, A parcellation hierarchy visualized in the form of a circular tree. The central node represents the entire subcortex and the other nodes represent distinct regions. Regions are arranged within four concentric circles, where the innermost circle (gray) is the first level (scale I). Nodes are colored according to principal gradient orientations within each region (red, left–right; green, posterior–anterior; blue, superior–inferior). The principal gradient orientation was estimated by averaging the local gradient directions across all voxels within each region. <b>c</b>, Region-averaged gradient directions represented with quiver lines for scales I–IV. The same images without region averaging are shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">7</a>. Detailed nomenclature for all regions is provided in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">2</a>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Parcellation of the amygdala terminated at scale II, and striatal parcellation continued until scale III, while only the thalamus and the hippocampus progressed to scale IV. A nomenclature scheme for all delineated regions was developed (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">2</a>). In particular, regions that could be unequivocally matched to homologous anatomical structures were labeled as such, while the remaining regions were labeled with reference to standard anatomical planes (for example, anterior–posterior, medial–lateral, dorsal–ventral and superior–inferior).</p><p>Scales I and II generally recapitulated anatomical boundaries, whereas some divergence emerged at finer scales between anatomy and the boundaries delineated here based on functional connectivity. The eight thalamic subregions constituting scale IV showed moderate spatial correspondence with histologically delineated thalamic nuclei (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">9a</a>). The hippocampus was segmented along its long axis into anterior (head) and posterior (body and tail) components (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">9b</a>), which is consistent with previous functional-based hippocampal parcellations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Poppenk, J., Evensmoen, H. R., Moscovitch, M. &amp; Nadel, L. Long-axis specialization of the human hippocampus. Trends Cogn. Sci. 17, 230–240 (2013)." href="/articles/s41593-020-00711-6#ref-CR21" id="ref-link-section-d27595475e931">21</a></sup>. Although the head of the hippocampus was further divided into medial and lateral components, the overall long-axis organization diverged from the characteristic mediolateral organization that is histologically found<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Amunts, K. et al. Cytoarchitectonic mapping of the human amygdala, hippocampal region and entorhinal cortex: intersubject variability and probability maps. Anat. Embryol. 210, 343–352 (2005)." href="/articles/s41593-020-00711-6#ref-CR22" id="ref-link-section-d27595475e935">22</a></sup>.</p><p>Connectivity gradients (gradients I–III) for the hippocampus, the striatum and the thalamus were projected onto the cortical surface on the basis of the cortical vertices with which they were most strongly connected<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="/articles/s41593-020-00711-6#ref-CR11" id="ref-link-section-d27595475e943">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. Nat. Hum. Behav. 1, 0146 (2017)." href="/articles/s41593-020-00711-6#ref-CR23" id="ref-link-section-d27595475e946">23</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">10a–c</a>).</p><h3 class="c-article__sub-heading" id="Sec5">Parcellation homogeneity</h3><p>Having delineated a new subcortical atlas, we next sought to investigate the validity and reproducibility of the atlas. A key marker of parcellation validity is within-parcel homogeneity of functional signals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672–686 (2018)." href="/articles/s41593-020-00711-6#ref-CR1" id="ref-link-section-d27595475e961">1</a></sup>. Homogeneity was measured on the basis of within-parcel synchrony of fMRI time series acquired in the second fMRI session (REST2), whereas the first session (REST1) was used for parcellation. Homogeneity was separately computed for each region and then averaged across all regions to yield parcellation homogeneity estimates for scales I–IV (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">11a</a>).</p><p>As expected<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e971">8</a></sup>, homogeneity monotonically increased from scale I to scale IV (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig5">5a</a>). To test whether parcellation homogeneity was significantly greater than expected due to chance, homogeneity was also estimated for ensembles of random subcortical parcellations (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">11b</a>). Random parcellations were generated by delineating boundaries at arbitrary locations, subject to preservation of key geometric properties (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00711-6#Sec18">Methods</a>). Scales II–IV were significantly more homogeneous than expected due to chance (<i>P</i> &lt; 0.01), whereas this was not the case for scale I (<i>P</i> = 0.23; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig5">5a</a>). Scale I regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig3">3a</a>) encompassed multiple functionally distinct substructures, which potentially explains the relatively low within-parcel homogeneity of this scale. Parcellation homogeneity was compared between the new atlas and existing imaging- and histology-based subcortical parcellations (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">3</a>). Striatal, hippocampal and thalamic parcellations were isolated from the new atlas to enable comparisons with existing parcellations of these regions. The new atlas was significantly more homogeneous than existing imaging-based parcellations of the entire subcortex (subcortex-I: <i>P</i> &lt; 0.001; subcortex-II: <i>P</i> &lt; 0.001), the hippocampus (<i>P</i> &lt; 0.001), the thalamus (<i>P</i> &lt; 0.001) or the striatum (<i>P</i> &lt; 0.001), as well as a histological atlas of the hippocampus (Hippocampus-His: <i>P</i> &lt; 0.001; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig5">5b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">12</a>). These results were replicated in an independent dataset comprising ten healthy adults (six males, mean age of 26 ± 2.1 years; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">13</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Parcellation homogeneity."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Parcellation homogeneity.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="497"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Parcellation homogeneity of the new atlas was benchmarked to random parcellations of the subcortex comprising an identical number of regions and a comparable distribution of region sizes. Violin plots show the distribution of parcellation homogeneity measured within ensembles of 100 random parcellations. The <i>P</i> value (one-sided) shown for each scale is the proportion of random parcellations comprising the ensemble that were more homogeneous than the observed parcellation. The horizontal red lines indicate the observed parcellation homogeneity values. The bottom and top edges of the boxes indicate the 25th and 75th percentiles of the distribution, respectively. The central mark indicates the median. The whiskers extend to the most extreme data points that are not considered outliers (1.5-times the interquartile range). The vertical axis is omitted to aid visualization. The null hypothesis of parcellation homogeneity that is no better than chance could be rejected for all scales except scale I. <b>b</b>, Parcellation homogeneity comparisons between the new atlas and existing parcellation atlases of the entire subcortex and specific subcortical nuclei (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">3</a>). Parcellation homogeneity was separately computed for each individual (REST2, <i>n</i> = 1,021) and normalized by the random parcellation homogeneity, yielding an observed-to-null ratio. The box plots show the distribution of this ratio across individuals for the new (turquoise) and existing (violet) parcellation atlases. The bottom and top edges of the boxes indicate the 25th and 75th percentiles of the distribution, respectively. The central mark indicates the median. The whiskers extend to the most extreme data points that are not considered outliers (1.5-times the interquartile range). Homogenous parcellations have a ratio that exceeds one. Asterisks indicate that the observed-to-null ratio is significantly higher for the new atlas relative to the existing atlas used for comparison (<i>P</i> &lt; 0.001, two-sided two-sample <i>t</i>-test). Anatomical visualizations of the new and comparison atlases are shown above and below the box plots, respectively.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>In auxiliary analyses, we tested whether task-evoked activity mapped onto the new atlas and respective atlas boundaries. To this end, the standard deviation in task-evoked activity was computed across voxels constituting each region and then averaged over all regions. This was repeated for scales I–IV. The lower the standard deviation, the more circumscribed task-evoked activity was to particular atlas regions. For all task conditions, we found that standard deviations were significantly lower for the new atlas than comparable random parcellations of the subcortex (<i>P</i> &lt; 0.01; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">14</a>). This cross-modality validation suggests that task-evoked activity is circumscribed to specific subcortical atlas regions more than expected by chance.</p><h3 class="c-article__sub-heading" id="Sec6">Parcellation replication</h3><p>Ultrahigh field strength MRI can alleviate several technical challenges associated with subcortical imaging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Vu, A. et al. Tradeoffs in pushing the spatial resolution of fMRI for the 7T Human Connectome Project. NeuroImage 154, 23–32 (2017)." href="/articles/s41593-020-00711-6#ref-CR24" id="ref-link-section-d27595475e1088">24</a></sup>. Hence, we sought to investigate whether the new atlas could be replicated using 7T fMRI. Qualitatively, gradient magnitude peaks were sharper and more circumscribed at 7T than at 3T (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig6">6</a>). Gradientography, model selection and boundary delineation applied to 7T data yielded a comparable but more detailed atlas of the subcortex that also comprised four scales (scale I: 16 regions; scale II: 34 regions; scale III: 54 regions; scale IV: 62 regions). The number of regions delineated at scale I was identical between 3T and 7T data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig6">6</a>). However, additional boundaries could be identified with 7T data at scales II–IV (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">15</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">16</a>). In particular, the amygdala was partitioned into medial and lateral subregions at 3T, whereas its medial extent was further divided into centromedial and superomedial components at 7T, and the body of the hippocampus was further divided mediolaterally. Based on normalized mutual information (NMI), the spatial correspondence between the 3T and 7T atlases was excellent (NMI = 0.93, 0.88, 0.84 and 0.83 for scale I, scale II, scale III and scale IV, respectively; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">16</a>). Correspondence between 3T and 7T atlases was significantly higher than expected due to chance at all parcellation scales (<i>P</i> &lt; 0.01; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">16</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Replication of 3T parcellation using 7T-rfMRI."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: Replication of 3T parcellation using 7T-rfMRI.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="339"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Group-consensus gradient magnitude image (top row) and scale I atlas (bottom three rows) delineated using 3T-rfMRI (<i>n</i> = 1,080). Bands colored in yellow–orange indicate local maxima in the gradient magnitude image. Homologous regions are identically colored. <b>b</b>, Same as <b>a</b>, but delineated using 7T-rfMRI (<i>n</i> = 183). The eight regions comprising scale I were replicated with 7T imaging. All slice coordinates are indicated in MNI space (mm). A reference structural MRI image is used as the background.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec7">Personalized parcellation</h3><p>Having delineated a group-consensus atlas of the subcortex, we next sought to personalize the new atlas to account for individual variation in connectivity architecture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Gordon, E. M. et al. Precision functional mapping of individual human brains. Neuron 95, 791–807.e7 (2017)." href="/articles/s41593-020-00711-6#ref-CR25" id="ref-link-section-d27595475e1156">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Kong, R. et al. Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion. Cereb. Cortex 29, 2533–2551 (2019)." href="/articles/s41593-020-00711-6#ref-CR26" id="ref-link-section-d27595475e1159">26</a></sup>. Following previous work on cortical atlas individualization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e1163">2</a></sup>, a machine-learning classifier was trained (<i>n</i> = 100, REST2) to recognize putative connectivity fingerprints characteristic of each scale IV region. The trained classifier was then used to delineate personalized boundaries for each region in an independent group (<i>n</i> = 921, REST2). In particular, for each individual, voxels were assigned a classification probability of belonging to each region and a winner-takes-all rule was used to delineate personalized parcellations (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">17</a>).</p><p>Quantification using the Dice coefficient showed that the extent to which the personalized atlases recapitulated the group-consensus atlas is markedly varied among individuals and regions. This was most evident within the striatum (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig7">7a</a>) and the medial subdivisions of the hippocampal head. Group-averaged Dice coefficients (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig7">7b</a>) varied from 0.22 (subdivision of the medial hippocampal head (HIP-head-m2-lh)) to 0.85 (dorsoposterior putamen (PUT-DP-rh)) for a classification probability threshold of 50%. Hotspots in the group-averaged classification probability maps were encircled by the group-consensus boundaries (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig7">7c</a>), which indicates self-consistency in classification performance. Individual variation in Dice coefficients was highly skewed, as evidenced for the dorsoposterior putamen and the hippocampal body (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig7">7d</a>). Specifically, a majority of individuals recapitulated the group-consensus (Dice &gt; 0.8), but 2–7% showed marked divergence (Dice &lt; 0.5). Boundary contraction was the most common source of divergence among individuals and the group-consensus atlas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig7">7c</a>). The same individuals consistently deviated from the group-consensus atlas for all regions, which suggests that this deviation was specific to individuals rather than regions (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">18</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7" data-title="Personalization of the scale IV atlas."><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 7: Personalization of the scale IV atlas.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig7_HTML.png?as=webp"><img aria-describedby="Fig7" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig7_HTML.png" alt="figure 7" loading="lazy" width="685" height="421"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p><b>a</b>, Group-consensus atlas for striatal regions (top row) and personalized atlases for representative individuals with relatively high (middle row, Dice = 0.81) and low (bottom row, Dice = 0.15) similarity with the group-consensus atlas. <b>b</b>, Curves showing group-averaged Dice coefficients as a function of the classification probability threshold (0–100%). A separate curve is shown for each of the 54 regions constituting the scale IV atlas. The classification probability is the probability that a subcortical voxel is classified as belonging to a specific region in a given individual. Parcellations were realized for each classification probability threshold and compared to the group-consensus atlas using the Dice coefficient. The area under the curve was computed for each region, and the legend is shown in descending order from top to bottom and left to right. <b>c</b>, Group-averaged classification probability maps exemplified in seven regions. The voxel color is scaled by the classification probability of the voxel belonging to the given region. The group-consensus boundary is shown in green. <b>d</b>, Box plots showing the distribution of individual variation in Dice coefficients of the dorsoposterior putamen (PUT-DP, left) and the hippocampal body (HIP-body, right). Each data point represents the Dice coefficient of one of the 921 individuals used to evaluate classification performance. The bottom and top edges of the boxes indicate the 25th and 75th percentiles of the distribution, respectively. The central mark indicates the median. The whiskers extend to the most extreme data points that are not considered outliers (1.5-times the interquartile range). Classification probability maps are shown in anatomical space for representative individuals with relatively high (top), moderate (middle) and low (bottom) Dice coefficients. The group-consensus boundary is shown in green. <b>e</b>, Curves showing the regional detection rates as a function of the classification probability threshold. The inset shows anatomical renderings of regional detection rates for a nominal classification probability threshold of 50%. The two hippocampal regions (bilateral HIP-head-m1 and HIP-head-m2) with the lowest regional detection rates are obscured in the slices shown. The dashed red arrow indicates the threshold (50%) used for the images shown in the inset. All slice coordinates are indicated relative to MNI space (mm). A reference structural MRI image is used as the background in anatomical visualizations.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/7" data-track-dest="link:Figure7 Full size image" aria-label="Full size image figure 7" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>It was possible for the classifier to fail to detect particular regions in some individuals. For each region, the proportion of individuals for whom at least one voxel was assigned to that region, called the regional detection rate, was analyzed as a function of the classification probability threshold (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig7">7e</a>). The regional detection rate for a classification probability threshold of 50% was rendered on the group-consensus atlas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig7">7e</a>, inset). The averaged regional detection rate was 97.8%, with detection rates falling below 95% for only two bilateral regions (that is, the HIP-head-m1 and the HIP-head-m2). This suggests that the group-consensus atlas is representative of most individuals, although a minority of individuals may benefit from delineation of individual-specific atlases to account for inter-individual variation in regional boundaries.</p><h3 class="c-article__sub-heading" id="Sec8">Rest compared to task-evoked conditions</h3><p>We have thus far focused on rfMRI data. Using tfMRI, we next investigated the extent to which the functional topography of the subcortex reconfigures in response to changing cognitive demands<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Salehi, M. et al. There is no single functional atlas even for a single individual: functional parcel definitions change with task. NeuroImage 208, 116366 (2020)." href="/articles/s41593-020-00711-6#ref-CR27" id="ref-link-section-d27595475e1249">27</a></sup>. For each of the seven tasks and the two rest sessions, gradient magnitudes (gradient I) were projected onto the previously mapped ventral and dorsal streamlines, which yielded task-evoked diversity curves.</p><p>Qualitatively, diversity curve peaks were consistent among the seven task conditions, but some peak locations and magnitudes differed relative to the rest condition (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19a</a>). The most prominent task–rest variation was bifurcation of the boundary separating the putamen–NAc and the NAc–caudate during the task conditions. Functional separation between the amygdala and the anterior extreme of the hippocampus, the posterior hippocampus and the thalamus, as well as anterior–posterior partition within the thalamus were also weaker during task compared to rest conditions, whereas separation between the anterior and posterior hippocampus was stronger and anteriorly extended (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19a</a>). To investigate this further, the Pearson correlation coefficient was used to quantify the similarity in diversity curves between all pairs of conditions, which yielded correlation matrices (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19b</a>) that were visualized as correlation graphs (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19c</a>). Diversity curves were highly consistent between the two rest sessions (dorsal: <i>r</i> = 0.91, <i>P</i> &lt; 0.0001; ventral: <i>r</i> = 0.98, <i>P</i> &lt; 0.0001), but were distinct from the seven task conditions, thereby resulting in distinct task and rest modules (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19d</a>). Task-evoked reconfiguration of functional organization was not due to differences in the number of fMRI volumes acquired between task and rest conditions (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00711-6#Sec18">Methods</a>).</p><h3 class="c-article__sub-heading" id="Sec9">Links between subcortical topography and cortical networks</h3><p>Next, we investigated functional connectivity between canonical cortical networks and subcortical regions (scale IV). Similarities in patterns of functional connectivity with the subcortex were quantified between pairs of 12 established cortical networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e1295">8</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">20a</a>). Based on the similarity in the subcortical areas to which they most strongly connected, the 12 cortical networks could be parsed into three distinct groups and positioned on a task-positive to task-negative<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Fox, M. D. et al. The human brain is intrinsically organized into dynamic, anticorrelated functional networks. Proc. Natl Acad. Sci. USA 102, 9673–9678 (2005)." href="/articles/s41593-020-00711-6#ref-CR17" id="ref-link-section-d27595475e1302">17</a></sup> organizational axis (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">20b</a>). One extreme of the axis comprised cortical networks that typically showed increased activity during attention-demanding cognitive tasks (that is, task-positive), including the salience, cingulo–parietal, cingulo–opercular, frontoparietal and dorsal attention network. In contrast, the default mode network, which is often characterized by task-related deactivation (that is, task-negative) occupied the other extreme. Between them, sensorimotor-related networks (visual, auditory, dorsal and ventral somatomotor) and ventral attention and retrosplenial–temporal networks formed a third group, in which the latter two also linked to the default mode network. This result suggests that there is a transition from sensorimotor functions toward task-negative functions.</p><h3 class="c-article__sub-heading" id="Sec10">Using the new atlas to relate human behavior to subcortical connectivity</h3><p>Finally, we used the new atlas to unveil new relationships between subcortical functional connectivity and individual variation in human behaviors. Five behavioral dimensions characterizing (1) cognition, (2) illicit substance use, (3) tobacco use, (4) personality and emotional traits and (5) mental health were considered (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig8">8a,b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">21</a>). These dimensions were derived from 109 behavioral items (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">4</a>) using independent component analysis (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00711-6#Sec18">Methods</a>). Using the network-based statistic (NBS)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. T. Network-based statistic: identifying differences in brain networks. NeuroImage 53, 1197–1207 (2010)." href="/articles/s41593-020-00711-6#ref-CR28" id="ref-link-section-d27595475e1329">28</a></sup>, higher tobacco use was significantly associated with lower functional connectivity in the thalamus–caudate, the thalamus–NAc and the hippocampus–NAc circuits (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig8">8c–f</a>). This is consistent with the involvement of the cortical–basal-ganglia–thalamus–cortical circuit and mesolimbic dopamine pathways in reward and goal-directed behaviors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Maia, T. V. &amp; Frank, M. J. From reinforcement learning models to psychiatric and neurological disorders. Nat. Neurosci. 14, 154–162 (2011)." href="/articles/s41593-020-00711-6#ref-CR29" id="ref-link-section-d27595475e1337">29</a></sup> such as smoking (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00711-6#Sec11">Discussion</a>). Although connectivity explained 4% of the individual variation in the behavioral dimension characterizing tobacco use, the relationship was highly consistent across all four parcellation scales (scale I: <i>r</i> = −0.16, <i>P</i> = 4.6×10<sup>−7</sup>; scale II: <i>r</i> = −0.17, <i>P</i> = 7.1×10<sup>−8</sup>; scale III: <i>r</i> = −0.19, <i>P</i> = 3.2×10<sup>−9</sup>; scale IV: <i>r</i> = −0.19, <i>P</i> = 3.4×10<sup>−9</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig8">8e</a>) and reproducible across fMRI sessions (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">22</a>). The remaining four dimensions did not significantly relate to subcortical connectivity after correction for multiple testing.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-8" data-title="Associations between individual variation in behavioral dimensions and subcortical functional connectivity."><figure><figcaption><b id="Fig8" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 8: Associations between individual variation in behavioral dimensions and subcortical functional connectivity.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/8" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig8_HTML.png?as=webp"><img aria-describedby="Fig8" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_Fig8_HTML.png" alt="figure 8" loading="lazy" width="685" height="606"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-8-desc"><p><b>a</b>, Word clouds characterizing five behavioral dimensions derived from 109 behavioral items using ICA. Only the most highly weighted behavioral items are shown. The font size is scaled according to the absolute weight of the item in the ICA de-mixing matrix. The font color indicates weight polarity (red, positive; blue, negative). <b>b</b>, A consensus de-mixing matrix estimated using ICA. Rows correspond to behavioral items. Columns correspond to the five latent dimensions. Cell color is modulated by de-mixing weight. <b>c</b>, Left: group-averaged functional connectivity (FC) matrices for scale I–IV atlases. Rows and columns are reordered to accentuate the modular structure (white blocks). Right: modules are also rendered in anatomical space, where regions constituting the same module are identically colored. Scale I comprises two modules (blue, the hippocampus and the amygdala; pink, the basal ganglia and the thalamus). The latter module was divided into two submodules that separated the basal ganglia (green) and the thalamus (orange) at scales II–IV. <b>d</b>, The subcortical network for which functional connectivity significantly associated with individual variation in the behavioral dimension characterizing tobacco use (<i>n</i> = 958). The linear association was significant for all atlas scales (scales I–IV, <i>P</i> &lt; 0.0001, permutation test, <i>n</i> = 10,000, one-sided, as implemented in NBS). Nodes and edges are colored according to module allegiance. Thicker edges are associated with stronger effects. Each row shows results for a distinct atlas scale. <b>e</b>, Scatter plots and lines of best fit for the linear association between subcortical functional connectivity and individual variation in the dimension characterizing tobacco use. Each data point represents one of the 958 individuals. Lines of best fit are indicated with solid gray lines and dashed gray lines show 95% CI. <i>R</i> values are the Pearson correlation coefficients and <i>P</i> values assess the null hypothesis of zero correlation. Each row shows results for a distinct atlas scale. <b>f</b>, The subcortical network associated with tobacco use shown as a circular network for the scale IV atlas, highlighting the role of thalamus–striatum–hippocampal circuits. Nodes and connections are colored according to module allegiance. The histogram (gray bar) shown for each node indicates the nodal degree.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00711-6/figures/8" data-track-dest="link:Figure8 Full size image" aria-label="Full size image figure 8" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec11-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec11">Discussion</h2><div class="c-article-section__content" id="Sec11-content"><p>The preponderance of the research field toward the cerebral cortex, coupled with technical challenges inherent to subcortical imaging, has led to the subcortex remaining a terra incognita<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Forstmann, B. U., de Hollander, G., van Maanen, L., Alkemade, A. &amp; Keuken, M. C. Towards a mechanistic understanding of the human subcortex. Nat. Rev. Neurosci. 18, 57–65 (2016)." href="/articles/s41593-020-00711-6#ref-CR30" id="ref-link-section-d27595475e1446">30</a></sup>. Through the provision of high-quality, ultrahigh field strength MRI data, collaborative initiatives such as the HCP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Van Essen, D. C. et al. The WU–Minn Human Connectome Project: an overview. NeuroImage 80, 62–79 (2013)." href="/articles/s41593-020-00711-6#ref-CR18" id="ref-link-section-d27595475e1450">18</a></sup> herald new opportunities to overcome some of the challenges associated with reliably imaging this deep and anatomically complex brain structure.</p><p>To address the incomplete representation of subcortical regions in many leading MRI atlases, we used multi-session MRI data of brain function acquired in more than 1,000 healthy adults participating in the HCP to provide fundamental new insight into the organizational architecture and connectivity of the human subcortex. Putative functional boundaries were recursively delineated at locations of sharp spatial change in subcortical-to-whole-brain functional connectivity fingerprints, which led to the semiautomated delineation of a new multiscale parcellation atlas. The new atlas is openly available as a standalone parcellation and has been integrated into established cortex-only<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e1457">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e1460">8</a></sup> and cerebellar<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="King, M., Hernandez-Castillo, C. R., Poldrack, R. A., Ivry, R. B. &amp; Diedrichsen, J. Functional boundaries in the human cerebellum revealed by a multi-domain task battery. Nat. Neurosci. 22, 1371–1378 (2019)." href="/articles/s41593-020-00711-6#ref-CR14" id="ref-link-section-d27595475e1464">14</a></sup> atlases. Crucially, the new subcortical atlas was replicated using ultrahigh field strength MRI, and the delineated regions were functionally homogeneous and identifiable within individual participants, thus enabling the delineation of personalized atlases that accounted for individual variation in connectivity. We used the new atlas to link cortical networks with the subcortex and to uncover a robust and reproducible relationship between individual variation in behavioral dimensions and subcortical functional connectivity.</p><h3 class="c-article__sub-heading" id="Sec12">Reconciling parcellations and gradients</h3><p>Topographic representations of variation in brain structure, function and connectivity usually follows one of two paradigms: (1) a mosaic of discrete parcels separated by hard boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e1475">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Yeo, B. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-020-00711-6#ref-CR3" id="ref-link-section-d27595475e1478">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e1481">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="King, M., Hernandez-Castillo, C. R., Poldrack, R. A., Ivry, R. B. &amp; Diedrichsen, J. Functional boundaries in the human cerebellum revealed by a multi-domain task battery. Nat. Neurosci. 22, 1371–1378 (2019)." href="/articles/s41593-020-00711-6#ref-CR14" id="ref-link-section-d27595475e1484">14</a></sup> and, more recently, (2) gradients of continuous topographic variation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="#ref-CR11" id="ref-link-section-d27595475e1488">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl Acad. Sci. USA 113, 12574–12579 (2016)." href="#ref-CR12" id="ref-link-section-d27595475e1488_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Guell, X., Schmahmann, J. D., Gabrieli, J. &amp; Ghosh, S. S. Functional gradients of the cerebellum. eLife 7, e36652 (2018)." href="/articles/s41593-020-00711-6#ref-CR13" id="ref-link-section-d27595475e1491">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Tian, Y. &amp; Zalesky, A. Characterizing the functional connectivity diversity of the insula cortex: subregions, diversity curves and behavior. NeuroImage 183, 716–733 (2018)." href="/articles/s41593-020-00711-6#ref-CR15" id="ref-link-section-d27595475e1494">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. Nat. Hum. Behav. 1, 0146 (2017)." href="/articles/s41593-020-00711-6#ref-CR23" id="ref-link-section-d27595475e1497">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Vos de Wael, R. et al. Anatomical and microstructural determinants of hippocampal subfield functional connectome embedding. Proc. Natl Acad. Sci. USA 115, 10154–10159 (2018)." href="/articles/s41593-020-00711-6#ref-CR31" id="ref-link-section-d27595475e1500">31</a></sup>. We introduced a principled way forward to reconcile these two seemingly incompatible paradigms that resulted in a hybrid representation that parsimoniously accommodated both discrete boundaries and continuous gradients. Boundaries were only delineated at locations where gradient magnitudes were significantly larger than what would be expected under an appropriate null hypothesis. This was formalized with a model selection process and an accompanying null model to test the null hypothesis of an absence of discrete spatial boundaries at candidate locations. Without model selection, boundaries would be delineated for all gradient magnitude peaks, which could potentially result in spurious boundaries that represent geometric constrictions rather than sharp changes in connectional topography. The bow-tie constriction shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">5b</a> provides an example of spurious boundary delineation, whereby a gradient peak appears at the bow knot, despite uniform spatial variation in connectivity. While the sheet-like geometry of the cerebral cortex is less likely to be affected by this phenomenon, complex subcortical geometry is more vulnerable to false-positive boundaries and over-parcellation.</p><p>Gradientography was developed to enable gradient visualization and accurate localization of candidate boundaries. Gradientography is a fMRI analog of diffusion MRI tractography and can be considered a form of fMRI tractography. Whereas tractography is conventionally used to map white matter fiber trajectories, gradientography, as introduced here, was used to map trajectories of maximum change in connectopic gradients. Future applications of gradientography are not limited to the subcortex, and the method could provide a bridge between cortical gradients and parcellations.</p><h3 class="c-article__sub-heading" id="Sec13">A new multiscale subcortical atlas</h3><p>The new subcortical atlas comprises four scales, labeled scale I to scale IV, each providing a self-contained parcellation atlas. Additional scales were not warranted because the model selection procedure decided against boundary delineation for all regions constituting scale IV. Researchers seeking to use the new atlas should select the atlas scale that provides adequate resolution to address the research question at hand and approximately matches the scale of any cortical atlas that is used in terms of the total number of regions. While the new atlas is unimodal, rfMRI connectivity is argued to be one the most useful modalities in predicting areal boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e1518">2</a></sup>.</p><p>Hierarchy is a hallmark of brain organization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Churchland, P. S. &amp; Sejnowski, T. J. Perspectives on cognitive neuroscience. Science 242, 741–745 (1988)." href="/articles/s41593-020-00711-6#ref-CR32" id="ref-link-section-d27595475e1525">32</a></sup>, and hierarchical organization is not necessarily a fait accompli of the recursive application of the model selection procedure used in this study (see “Further methodological considerations” in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00711-6#Sec18">Methods</a>). Scale I of the atlas recapitulated subcortical regions delineated in existing MRI atlases, such as those available as part of Lead-DBS, Freesurfer and FSL. The position of gradient peaks enabled automated boundary demarcation among the amygdala, the hippocampus, the thalamus, the caudate, the putamen, the NAc and the GP. Discrete partition of the basal ganglia into the latter four regions and separation between the amygdala and the hippocampus were clearly supported by model selection. Contrastingly, the few existing connectivity-based subcortical atlases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ji, J. L. et al. Mapping the human brain’s cortical–subcortical functional network organization. NeuroImage 185, 35–57 (2019)." href="/articles/s41593-020-00711-6#ref-CR6" id="ref-link-section-d27595475e1532">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Fan, L. et al. The Human Brainnetome Atlas: a new brain atlas based on connectional architecture. Cereb. Cortex 26, 3508–3526 (2016)." href="/articles/s41593-020-00711-6#ref-CR33" id="ref-link-section-d27595475e1535">33</a></sup> generally do not preserve established anatomical boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Pauli, W. M., Nili, A. N. &amp; Tyszka, J. M. A high-resolution probabilistic in vivo atlas of human subcortical brain nuclei. Sci. Data 5, 180063 (2018)." href="/articles/s41593-020-00711-6#ref-CR34" id="ref-link-section-d27595475e1539">34</a></sup>, which suggests that there is structure–function divergence or methodological variation. Given that these broad subcortical regions are anatomically distinguishable, rudimentary alignment with functional architecture might be expected<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672–686 (2018)." href="/articles/s41593-020-00711-6#ref-CR1" id="ref-link-section-d27595475e1543">1</a></sup>. Indeed, this assumption motivates the parcellation of regions in isolation, such as the hippocampus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Vos de Wael, R. et al. Anatomical and microstructural determinants of hippocampal subfield functional connectome embedding. Proc. Natl Acad. Sci. USA 115, 10154–10159 (2018)." href="/articles/s41593-020-00711-6#ref-CR31" id="ref-link-section-d27595475e1548">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Plachti, A. et al. Multimodal parcellations and extensive behavioral profiling tackling the hippocampus gradient. Cereb. Cortex 29, 4595–4612 (2019)." href="/articles/s41593-020-00711-6#ref-CR35" id="ref-link-section-d27595475e1551">35</a></sup>, the striatum<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. Nat. Hum. Behav. 1, 0146 (2017)." href="/articles/s41593-020-00711-6#ref-CR23" id="ref-link-section-d27595475e1555">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Janssen, R. J., Jylänki, P., Kessels, R. P. C. &amp; van Gerven, M. A. J. Probabilistic model-based functional parcellation reveals a robust, fine-grained subdivision of the striatum. NeuroImage 119, 398–405 (2015)." href="/articles/s41593-020-00711-6#ref-CR36" id="ref-link-section-d27595475e1558">36</a></sup> and the thalamus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Behrens, T. E. et al. Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging. Nat. Neurosci. 6, 750–757 (2003)." href="/articles/s41593-020-00711-6#ref-CR37" id="ref-link-section-d27595475e1562">37</a></sup>, without first establishing functional boundaries between them. Despite correspondence between the principal connectional topography (gradient I) and anatomy at scale I, fine variation within specific structures was captured by the secondary connectional gradients. As shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">10b</a>, the dominant striatal gradient separates the putamen, the NAc and the caudate, whereas the second and third gradients are organized along a dorsal-to-ventral axis, thus reflecting connectional variation between the dorsal (putamen and caudate) and ventral (NAc) striatum<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Haber, S. N. &amp; Knutson, B. The reward circuit: linking primate anatomy and human imaging. Neuropsychopharmacology 35, 4–26 (2010)." href="/articles/s41593-020-00711-6#ref-CR38" id="ref-link-section-d27595475e1569">38</a></sup>. Similarly, the dominant thalamic gradient follows an anteroposterior trajectory, whereas the second and third gradients capture connectional variation between the mediodorsal and ventrolateral thalamus (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">10c</a>), which may relate to organizational differences between subgroups of thalamic nuclei<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Morel, A., Magnin, M. &amp; Jeanmonod, D. Multiarchitectonic and stereotactic atlas of the human thalamus. J. Comp. Neurol. 387, 588–630 (1997)." href="/articles/s41593-020-00711-6#ref-CR39" id="ref-link-section-d27595475e1577">39</a></sup>.</p><p>Scale II delineates functional divisions within each of the scale I regions. The NAc was partitioned into regions potentially representing its shell and core, and anterior (head) and posterior (body and tail) caudate subdivisions were identified, and the amygdala was divided into lateral and medial subregions. A striking divergence between anatomy and function was evident for the GP. The GP is anatomically divided along a mediolateral axis into the internal and external GP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Pauli, W. M., Nili, A. N. &amp; Tyszka, J. M. A high-resolution probabilistic in vivo atlas of human subcortical brain nuclei. Sci. Data 5, 180063 (2018)." href="/articles/s41593-020-00711-6#ref-CR34" id="ref-link-section-d27595475e1584">34</a></sup>, whereas functional connectivity gradients indicated an anteroposterior partition. Scales III and IV differed only in the parcellation of the thalamus and the hippocampus.</p><h3 class="c-article__sub-heading" id="Sec14">Parcellation validation</h3><p>While histological validation is the gold standard, functional parcellations do not necessarily converge toward anatomical or cytoarchitectural divisions to be deemed useful<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672–686 (2018)." href="/articles/s41593-020-00711-6#ref-CR1" id="ref-link-section-d27595475e1597">1</a></sup>. To establish confidence in the new atlas, we first demonstrated that all atlas scales, except scale I, were significantly more homogenous<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e1601">8</a></sup> than expected due to chance. While scale I regions converged with subcortical anatomy, they encompassed multiple functional systems, thereby rendering scale I too coarse to achieve homogeneity. Homogeneity of functional signals within delineated subregions is a fundamental principle of functional parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672–686 (2018)." href="/articles/s41593-020-00711-6#ref-CR1" id="ref-link-section-d27595475e1605">1</a></sup>. Interestingly, existing structural connectivity and histological parcellations of the thalamus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Behrens, T. E. et al. Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging. Nat. Neurosci. 6, 750–757 (2003)." href="/articles/s41593-020-00711-6#ref-CR37" id="ref-link-section-d27595475e1609">37</a></sup> and the hippocampus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Amunts, K. et al. Cytoarchitectonic mapping of the human amygdala, hippocampal region and entorhinal cortex: intersubject variability and probability maps. Anat. Embryol. 210, 343–352 (2005)." href="/articles/s41593-020-00711-6#ref-CR22" id="ref-link-section-d27595475e1613">22</a></sup>, respectively, were not functionally homogenous (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">12</a>), which indicates that there is a need for modality-specific atlases.</p><p>Second, we reproduced the majority of regions using 7T fMRI. The improved contrast-to-noise ratio and spatial resolution of 7T compared with 3T MRI is particularly important for subcortical imaging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Vu, A. et al. Tradeoffs in pushing the spatial resolution of fMRI for the 7T Human Connectome Project. NeuroImage 154, 23–32 (2017)." href="/articles/s41593-020-00711-6#ref-CR24" id="ref-link-section-d27595475e1624">24</a></sup>. Despite remarkable consistency between the 3T and 7T atlases, 7T fMRI enabled the delineation of finer structures. For example, a bipartite partition of the amygdala (medial and lateral) was delineated at 3T, whereas 7T fMRI aided further division of the medial amygdala into central and superior components, which broadly recapitulated the three cytoarchitectural zones of the amygdala<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Amunts, K. et al. Cytoarchitectonic mapping of the human amygdala, hippocampal region and entorhinal cortex: intersubject variability and probability maps. Anat. Embryol. 210, 343–352 (2005)." href="/articles/s41593-020-00711-6#ref-CR22" id="ref-link-section-d27595475e1628">22</a></sup>. Conversely, the NAc was consistently parcellated into two subdivisions at 3T and 7T, thus possibly representing the immunohistochemically distinct shell and core subdivision that is common to primates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Meredith, G. E., Pattiselanno, A., Groenewegen, H. J. &amp; Haber, S. N. Shell and core in monkey and human nucleus accumbens identified with antibodies to calbindin-D28k. J. Comp. Neurol. 365, 628–639 (1996)." href="/articles/s41593-020-00711-6#ref-CR40" id="ref-link-section-d27595475e1632">40</a></sup>. Therefore, while subcortical cartography is viable with functional connectivity inferred from high-quality 3T fMRI data, the level of detail that can be resolved at 7T is superior for several regions.</p><p>The subcortex atlas was personalized to account for individual connectivity differences. We showed that while the group-consensus atlas was sufficient to represent the majority of individuals, personalized subcortex atlases were warranted for a small proportion of others. Deviations from the group-consensus atlas were specific to particular individuals rather than specific regions (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">18</a>), which suggests that deviations may capture inherent inter-individual differences in functional architecture. Further work is needed to understand whether individual variation in subcortical connectional topography relates to anatomy, task-evoked activation and behavior. Individual-specific atlases of the subcortex can potentially facilitate personalization of targeted therapies in psychiatry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Cocchi, L. &amp; Zalesky, A. Personalized transcranial magnetic stimulation in psychiatry. Biol. Psychiatry Cogn. Neurosci. Neuroimaging 3, 731–741 (2018)." href="/articles/s41593-020-00711-6#ref-CR41" id="ref-link-section-d27595475e1642">41</a></sup> and to improve prediction of behavior<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Kong, R. et al. Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion. Cereb. Cortex 29, 2533–2551 (2019)." href="/articles/s41593-020-00711-6#ref-CR26" id="ref-link-section-d27595475e1646">26</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec15">Dynamic and intrinsic subcortical topography</h3><p>Using tfMRI, we identified subtle reconfigurations in the functional topography of the subcortex in response to changing cognitive demands. While a previous study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Salehi, M. et al. There is no single functional atlas even for a single individual: functional parcel definitions change with task. NeuroImage 208, 116366 (2020)." href="/articles/s41593-020-00711-6#ref-CR27" id="ref-link-section-d27595475e1658">27</a></sup> examined variation in parcel size in response to task stimuli, diversity curves enabled the characterization of changes in functional boundaries. During task conditions, the anterior putamen integrated more extensively with higher-order association networks, whereas the hippocampus segregated along its long axis, yielding a more integrated anterior hippocampal–amygdala system that may have served cognitive–emotional processing, and a more integrated posterior hippocampal–thalamus that potentially benefited learning and spatial memory<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Poppenk, J., Evensmoen, H. R., Moscovitch, M. &amp; Nadel, L. Long-axis specialization of the human hippocampus. Trends Cogn. Sci. 17, 230–240 (2013)." href="/articles/s41593-020-00711-6#ref-CR21" id="ref-link-section-d27595475e1662">21</a></sup>. These task-induced dynamic reconfigurations may reflect a process of neuronal recruitment into specific functional processing systems to cope with cognitive demands. The flexible reconfiguration of neural networks may largely depend on the intrinsic functional organization within local cortical circuits given that a task-positive to task-negative organizational axis was evident in the subcortex (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">20</a>). However, context-dependent recruitment across intrinsic parcels, leading to the adjustment of functional boundaries, may additionally assist adaptive task responses. Further work is needed to understand task-evoked reconfigurations of functional boundaries, not only in the subcortex but also across the brain.</p><h3 class="c-article__sub-heading" id="Sec16">Tobacco and subcortical connectivity</h3><p>We performed a hypothesis-free decomposition of HCP cognitive and behavioral measures into five latent dimensions to facilitate an investigation of relationships between brain connectivity and behavior. Previous HCP investigations typically investigate hundreds of individual measures, which leads to a formidable multiple testing problem, or preselect a limited number of specific behavioral domains or measures. The five latent dimensions identified here condense information from more than 100 individual measures.</p><p>The new atlas unveiled a novel relationship between individual variation in functional connectivity and the behavioral dimension characterizing tobacco use. Higher tobacco use was associated with lower thalamus–caudate, thalamus–NAc and hippocampus–NAc functional connectivity. The shell of the NAc and the adjacent ventroanterior caudate (CAU–VA) were most implicated (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig8">8f</a>). This accords with the primary role of the shell of the NAc and the anterior caudate in reward anticipation, incentive motivation and action<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Castro, D. C. &amp; Bruchas, M. R. A motivational and neuropeptidergic hub: anatomical and functional diversity within the nucleus accumbens shell. Neuron 102, 529–552 (2019)." href="/articles/s41593-020-00711-6#ref-CR42" id="ref-link-section-d27595475e1683">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Pauli, W. M., O’Reilly, R. C., Yarkoni, T. &amp; Wager, T. D. Regional specialization within the human striatum for diverse psychological functions. Proc. Natl Acad. Sci. USA 113, 1907–1912 (2016)." href="/articles/s41593-020-00711-6#ref-CR43" id="ref-link-section-d27595475e1686">43</a></sup>. The NAc integrates glutamatergic (excitatory) inputs from the hippocampus and the thalamus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="MacAskill, A. F., Little, J. P., Cassel, J. M. &amp; Carter, A. G. Subcellular connectivity underlies pathway-specific signaling in the nucleus accumbens. Nat. Neurosci. 15, 1624–1626 (2012)." href="/articles/s41593-020-00711-6#ref-CR44" id="ref-link-section-d27595475e1690">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Perez, S. M. &amp; Lodge, D. J. Convergent inputs from the hippocampus and thalamus to the nucleus accumbens regulate dopamine neuron activity. J. Neurosci. 38, 10607–10618 (2018)." href="/articles/s41593-020-00711-6#ref-CR45" id="ref-link-section-d27595475e1693">45</a></sup>, which enables the regulation of dopamine neuron activity in the ventral tegmental area<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Perez, S. M. &amp; Lodge, D. J. Convergent inputs from the hippocampus and thalamus to the nucleus accumbens regulate dopamine neuron activity. J. Neurosci. 38, 10607–10618 (2018)." href="/articles/s41593-020-00711-6#ref-CR45" id="ref-link-section-d27595475e1697">45</a></sup>. While nicotine, the main neuroactive compound of tobacco, stimulates dopamine release and transmission in the NAc, especially the medial shell<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Pontieri, F. E., Tanda, G., Orzi, F. &amp; Di Chiara, G. Effects of nicotine on the nucleus accumbens and similarity to those of addictive drugs. Nature 382, 255–257 (1996)." href="/articles/s41593-020-00711-6#ref-CR46" id="ref-link-section-d27595475e1701">46</a></sup>, chronic nicotine administration yields neuronal damage in the hippocampus and the striatum<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Gallinat, J. et al. Abnormal hippocampal neurochemistry in smokers: evidence from proton magnetic resonance spectroscopy at 3T. J. Clin. Psychopharmacol. 27, 80–84 (2007)." href="/articles/s41593-020-00711-6#ref-CR47" id="ref-link-section-d27595475e1706">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Xu, Z., Seidler, F. J., Ali, S. F., Slikker, W. Jr. &amp; Slotkin, T. A. Fetal and adolescent nicotine administration: effects on CNS serotonergic systems. Brain Res. 914, 166–178 (2001)." href="/articles/s41593-020-00711-6#ref-CR48" id="ref-link-section-d27595475e1709">48</a></sup>, which potentially manifests in lower functional connectivity in the thalamus–striatum–hippocampus system. Therefore, the putative downregulation of the mesolimbic circuit by nicotine may provide insight into the increased prevalence of tobacco smoking in individuals with schizophrenia<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="de Leon, J. &amp; Diaz, F. J. A meta-analysis of worldwide studies demonstrates an association between schizophrenia and tobacco smoking behaviors. Schizophrenia Res. 76, 135–157 (2005)." href="/articles/s41593-020-00711-6#ref-CR49" id="ref-link-section-d27595475e1713">49</a></sup>. In particular, this downregulation effect of nicotine on the hyperdopaminergic state in the mesolimbic system may correspond with the upregulation effect of nicotine in the prefrontal cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Koukouli, F. et al. Nicotine reverses hypofrontality in animal models of addiction and schizophrenia. Nat. Med. 23, 347–354 (2017)." href="/articles/s41593-020-00711-6#ref-CR50" id="ref-link-section-d27595475e1717">50</a></sup>, which alleviates symptoms in the disorder.</p></div></div></section><section data-title="Conclusions"><div class="c-article-section" id="Sec17-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec17">Conclusions</h2><div class="c-article-section__content" id="Sec17-content"><p>We unveiled the complex topographic organization of the human subcortex and parsed this complexity into four scales constituting a new atlas for the brain’s remaining terra incognita. The new atlas is scalable, personalizable, reproducible at ultrahigh field strengths and openly available. Future work will focus on the addition of structures such as subthalamic regions, the red nucleus and the hypothalamus. The new atlas can be incorporated with existing cortex-only atlases to aid the investigation of cortico–subcortical interactions and the mapping of holistic connectomes using MRI.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec18-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec18">Methods</h2><div class="c-article-section__content" id="Sec18-content"><h3 class="c-article__sub-heading" id="Sec19">Data and preprocessing</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">HCP dataset</h4><p>Human neuroimaging data acquired as part of the HCP S1200 release<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Van Essen, D. C. et al. The WU–Minn Human Connectome Project: an overview. NeuroImage 80, 62–79 (2013)." href="/articles/s41593-020-00711-6#ref-CR18" id="ref-link-section-d27595475e1746">18</a></sup> were analyzed here. Participants were healthy young adults ranging between 22 and 37 years old. Some participants were genetically related. These relationships were controlled in statistical analyses by constraining permutations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Winkler, A. M., Ridgway, G. R., Webster, M. A., Smith, S. M. &amp; Nichols, T. E. Permutation inference for the general linear model. NeuroImage 92, 381–397 (2014)." href="/articles/s41593-020-00711-6#ref-CR51" id="ref-link-section-d27595475e1750">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Winkler, A. M., Webster, M. A., Vidaurre, D., Nichols, T. E. &amp; Smith, S. M. Multi-level block permutation. NeuroImage 123, 253–268 (2015)." href="/articles/s41593-020-00711-6#ref-CR52" id="ref-link-section-d27595475e1753">52</a></sup>. HCP datasets used in this study include three main imaging sessions. (1) Two sessions (REST1 and REST2) of rfMRI) acquired using multiband echo planar imaging (EPI) on a customized Siemens 3T MR scanner (Skyra system), where each session comprised two runs (left-to-right and right-to-left phase encoding) of 14 min and 33 s each (repetition time (TR) = 720 ms, echo time (TE) = 33.1 ms, voxel dimension: 2-mm isotropic). The two runs were temporally concatenated for each session, yielding 29 min and 6 s of data in each session. Concatenation of the two different phase-encoded data ensured that any potential effect of phase encoding on gradient direction was counterbalanced by the opposing phase encoding. (2) tfMRI data acquired using the identical multiband EPI sequence as the rfMRI session; however, the run duration ranged between 2 and 5 min depending on the specific task (7 tasks). (3) rfMRI acquired on a Siemens Magnetom 7T MR scanner using a multiband acquisition (TR = 1,000 ms, TE = 22.2 ms, voxel dimension: 1.6-mm isotropic), where participants completed 4 runs of approximately 16 min each. The first two runs (posterior-to-anterior and anterior-to-posterior phase encoding) were used in this study. We refer to this dataset as 7T-rfMRI. Details pertaining to the acquisition of rfMRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Vu, A. et al. Tradeoffs in pushing the spatial resolution of fMRI for the 7T Human Connectome Project. NeuroImage 154, 23–32 (2017)." href="/articles/s41593-020-00711-6#ref-CR24" id="ref-link-section-d27595475e1757">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Smith, S. M. et al. Resting-state fMRI in the Human Connectome Project. NeuroImage 80, 144–168 (2013)." href="/articles/s41593-020-00711-6#ref-CR53" id="ref-link-section-d27595475e1760">53</a></sup> and tfMRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Barch, D. M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. NeuroImage 80, 169–189 (2013)." href="/articles/s41593-020-00711-6#ref-CR54" id="ref-link-section-d27595475e1764">54</a></sup> are described elsewhere. The seven tfMRI tasks were chosen to tap a broad range of cognitive and affective processes and activated a wide range of neural systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Barch, D. M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. NeuroImage 80, 169–189 (2013)." href="/articles/s41593-020-00711-6#ref-CR54" id="ref-link-section-d27595475e1768">54</a></sup>. Participants that had completed at least two runs in each dataset were included. As a consequence, 33 of the original 1,113 individuals were excluded from the primary dataset (REST1) and 92 individuals were excluded from the replication dataset (REST2). Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a> shows the sample size and basic participant demographics for each dataset.</p><p>The second 3T-rfMRI session (REST2) was used for replication. Minimally preprocessed volumetric rfMRI and tfMRI data were sourced from the online HCP repository. Details of the minimal preprocessing pipeline can be found elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. NeuroImage 80, 105–124 (2013)." href="/articles/s41593-020-00711-6#ref-CR55" id="ref-link-section-d27595475e1779">55</a></sup>. The pipeline includes correction of head motion and spatially structured physiological noise with ICA+FIX<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Salimi-Khorshidi, G. et al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. NeuroImage 90, 449–468 (2014)." href="/articles/s41593-020-00711-6#ref-CR56" id="ref-link-section-d27595475e1783">56</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Griffanti, L. et al. ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging. NeuroImage 95, 232–247 (2014)." href="/articles/s41593-020-00711-6#ref-CR57" id="ref-link-section-d27595475e1786">57</a></sup>. ICA-based denoising methods outperform, or equal, the performance of alternative approaches on most benchmarks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Parkes, L., Fulcher, B., Yucel, M. &amp; Fornito, A. An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI. NeuroImage 171, 415–436 (2018)." href="/articles/s41593-020-00711-6#ref-CR58" id="ref-link-section-d27595475e1790">58</a></sup>. After eliminating spatial artifacts and distortions, the functional images were spatially aligned to the Montreal Neurological Institute (MNI) standard space using the FNIRT nonlinear registration algorithm. MNI standard space herein always refers to the MNI ICBM 152 nonlinear sixth generation stereotaxic registration model. Additional nuisance regression and temporal filtering were not performed as part of the pipeline. However, head motion registration parameters estimated as part of spatial alignment were curated, reduced to a single summary measure known as framewise displacement<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Power, J. D., Barnes, K. A., Snyder, A. Z., Schlaggar, B. L. &amp; Petersen, S. E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. NeuroImage 59, 2142–2154 (2012)." href="/articles/s41593-020-00711-6#ref-CR59" id="ref-link-section-d27595475e1794">59</a></sup> and included as a confound in statistical analyses to control for individual variation in head motion<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Satterthwaite, T. D. et al. An improved framework for confound regression and filtering for control of motion artifact in the preprocessing of resting-state functional connectivity data. NeuroImage 64, 240–256 (2013)." href="/articles/s41593-020-00711-6#ref-CR60" id="ref-link-section-d27595475e1798">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Van Dijk, K. R., Sabuncu, M. R. &amp; Buckner, R. L. The influence of head motion on intrinsic functional connectivity MRI. NeuroImage 59, 431–438 (2012)." href="/articles/s41593-020-00711-6#ref-CR61" id="ref-link-section-d27595475e1801">61</a></sup>.</p><p>The subcortical SNR and sensitivity of the blood-oxygenation-level-dependent (BOLD) signal was lower compared with the cortex because T2* signal decay times are shorter in subcortical regions compared with cortical regions due to iron enrichment of subcortical tissue<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Puckett, A. M. et al. Using multi-echo simultaneous multi-slice (SMS) EPI to improve functional MRI of the subcortical nuclei of the basal ganglia at ultra-high field (7T). NeuroImage 172, 886–895 (2018)." href="/articles/s41593-020-00711-6#ref-CR62" id="ref-link-section-d27595475e1808">62</a></sup>. Therefore, to enhance subcortical SNR and BOLD sensitivity, the minimally preprocessed data were subjected to two additional operations (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">8a</a>). First, spatial smoothing was performed with a Gaussian smoothing kernel of 6-mm full width at half maximum (FWHM) for 3T images and 4-mm FWHM for 7T images. This choice of kernel width was guided by evaluations of gradient magnitude images derived from a range of smoothing kernels (4, 6 and 8 mm) and filtering options (that is, the Wishart filter, see below). Second, to further improve subcortical SNR and to suppress the effects of unstructured and autocorrelated noise that share similar temporal and spatial properties, the fMRI time series were reconstructed from a principal component analysis (PCA), whereby the components were filtered according to an empirically fitted Wishart distribution to dampen the effects of noise<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e1815">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Glasser, M. F. et al. The Human Connectome Project’s neuroimaging approach. Nat. Neurosci. 19, 1175–1187 (2016)." href="/articles/s41593-020-00711-6#ref-CR63" id="ref-link-section-d27595475e1818">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Glasser, M. F. et al. Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data. NeuroImage 181, 692–717 (2018)." href="/articles/s41593-020-00711-6#ref-CR64" id="ref-link-section-d27595475e1821">64</a></sup>. This PCA-based data reconstruction (that is, the Wishart filter) has been previously used in a multimodal cortical parcellation study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e1825">2</a></sup> and comprehensively documented by Glasser and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Glasser, M. F. et al. The Human Connectome Project’s neuroimaging approach. Nat. Neurosci. 19, 1175–1187 (2016)." href="/articles/s41593-020-00711-6#ref-CR63" id="ref-link-section-d27595475e1829">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Glasser, M. F. et al. Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data. NeuroImage 181, 692–717 (2018)." href="/articles/s41593-020-00711-6#ref-CR64" id="ref-link-section-d27595475e1832">64</a></sup>. The Wishart filter was performed over all gray matter voxels for each individual separately in all datasets. In addition, task block regressors were created for each task condition based on the time of onset, the duration and the relative magnitude of each stimulus, and then convolved with hemodynamic response functions (spm_hrf.m function from SPM12, <a href="https://www.fil.ion.ucl.au.uk/spm">https://www.fil.ion.ucl.au.uk/spm</a>). The relevant regressors corresponding to each task were then regressed from the tfMRI data to ensure that functional connectivity was disambiguated from the potential confound of task co-activation effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Cole, M. W. et al. Multi-task connectivity reveals flexible hubs for adaptive task control. Nat. Neurosci. 16, 1348–1355 (2013)." href="/articles/s41593-020-00711-6#ref-CR65" id="ref-link-section-d27595475e1844">65</a></sup>. The residuals remaining from this regression were used for all subsequent analyses.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Independent validation dataset</h4><p>An independent fMRI dataset was acquired in Australia to test the reproducibility of the parcellation homogeneity results. Ten healthy adults were recruited and confirmed not to have any current or history of any neurological or psychiatric disorders at the time of MRI scanning (mean age 26 ± 2.1 years, 6 males). Imaging was performed in all participants using a Siemens Prisma 3T MRI scanner. Structural images of brain anatomy were acquired using an optimized magnetization-prepared rapid acquisition gradient echo (MPRAGE) T1-weighted sequence with 176 sagittal slices of 1-mm thickness, field of view of 240 × 256 mm<sup>2</sup>, flip angle of 9°, TR of 1,900 ms, TE of 2.98 ms and voxel size of 1.0 × 1.0 × 1.0 mm<sup>3</sup>. rfMRI was acquired using a T2*-weighted multiband gradient-echo EPI sequence of approximately 12 min, resulting in 880 volumes (TR = 810 ms, TE = 30 ms, voxel size = 2.0 × 2.0 × 2.0 mm<sup>3</sup>).</p><p>rfMRI data were preprocessed using the fMRIPrep-1.5.9 pipeline<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Esteban, O. et al. fMRIPrep: a robust preprocessing pipeline for functional MRI. Nat. Methods 16, 111–116 (2019)." href="/articles/s41593-020-00711-6#ref-CR66" id="ref-link-section-d27595475e1865">66</a></sup>. In brief, each T1-weighted volume was corrected for intensity non-uniformity and skull-stripped. The extracted brain was spatially normalized to MNI standard space through nonlinear registration using ANTs. Cortical surfaces were reconstructed using FreeSurfer. Functional data were motion-corrected using mcflirt (FSL) followed by co-registration to the corresponding T1-weighted image using boundary-based registration with six degrees of freedom. Motion-correcting transformations, BOLD-to-T1w transformation and T1w-to-MNI warp were concatenated and applied in a single step using antsApplyTransforms (ANTs).</p><p>Confounds including the 24 head-motion parameters (6 basic motion parameters + 6 temporal derivatives + 12 quadratic terms and their 6 temporal derivatives), mean white matter and cerebrospinal fluid signals were computed and regressed from the preprocessed fMRI data for each individual. The residuals of this regression were then subjected to spatial smoothing with a Gaussian-smoothing kernel of 6-mm FWHM and Wishart filtering<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Glasser, M. F. et al. The Human Connectome Project’s neuroimaging approach. Nat. Neurosci. 19, 1175–1187 (2016)." href="/articles/s41593-020-00711-6#ref-CR63" id="ref-link-section-d27595475e1872">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Glasser, M. F. et al. Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data. NeuroImage 181, 692–717 (2018)." href="/articles/s41593-020-00711-6#ref-CR64" id="ref-link-section-d27595475e1875">64</a></sup>.</p><p>For all datasets, statistical methods were not used to predetermine sample sizes. All relevant and appropriate data provisioned by the HCP were included to maximize samples. Our use of HCP datasets and our recruitment of the independent dataset complied with the Australian NHMRC national framework for the conduct of human research. Informed consent was obtained from all the participants. Data collection and analyses were not performed blind to the conditions of the experiments.</p><h3 class="c-article__sub-heading" id="Sec22">Mask delineation</h3><p>A binary subcortex mask comprising the left and right basal ganglia, the thalamus, the hippocampus and the amygdala was delineated using three steps. First, the probabilistic Harvard–Oxford Subcortical Structural Atlas (<a href="https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Atlases">https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/Atlases</a>) was used to delineate a binary mask for each of the following subcortical regions: the thalamus, the caudate, the putamen, the NAc, the GP, the hippocampus and the amygdala. Voxels with a probability of 50% or more of belonging to one of these regions were included in the relevant mask, except for the GP. A higher threshold (60%) was applied to the GP to minimize potential white matter contamination of the fMRI signal due to the numerous myelinated axons of the striato–pallido–nigral bundle abutting the periphery of this region<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Percheron, G., Yelnik, J. &amp; Francois, C. A Golgi analysis of the primate globus pallidus. III. Spatial organization of the striato–pallidal complex. J. Comp. Neurol. 227, 214–227 (1984)." href="/articles/s41593-020-00711-6#ref-CR67" id="ref-link-section-d27595475e1898">67</a></sup>. Second, the masks for each individual subcortical region were merged to yield a binary mask for the entire subcortex. Finally, the subcortex mask was left–right symmetrized. Left–right symmetrization accords with previous subcortical atlas literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Pauli, W. M., Nili, A. N. &amp; Tyszka, J. M. A high-resolution probabilistic in vivo atlas of human subcortical brain nuclei. Sci. Data 5, 180063 (2018)." href="/articles/s41593-020-00711-6#ref-CR34" id="ref-link-section-d27595475e1902">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Yushkevich, P. A. et al. Quantitative comparison of 21 protocols for labeling hippocampal subfields and parahippocampal subregions in in vivo MRI: towards a harmonized segmentation protocol. NeuroImage 111, 526–541 (2015)." href="/articles/s41593-020-00711-6#ref-CR68" id="ref-link-section-d27595475e1905">68</a></sup> and enabled symmetrization of subsequent computations to improve the SNR. Symmetrization is not an unprecedented step given that intra-hemispheric asymmetries in the volume of subcortical nuclei are less evident in healthy young adults compared with cortical asymmetries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Guadalupe, T. et al. Human subcortical brain asymmetries in 15,847 people worldwide reveal effects of age and sex. Brain Imaging Behav. 11, 1497–1514 (2017)." href="/articles/s41593-020-00711-6#ref-CR69" id="ref-link-section-d27595475e1909">69</a></sup>. Mask symmetrization was achieved by reflecting the left hemisphere of the mask around the mid-sagittal plane and computing the intersection between the right hemisphere of the mask and this reflection. Voxels constituting the intersection yielded a left subcortex mask, which was merged with its reflection to yield the final left–right symmetric subcortex mask. The final mask comprised a total of <i>Ν</i> = 7,984 voxels (63.87 cm<sup>3</sup>). The subthalamic nuclei, the substantia nigra and comparably small subcortical nuclei were not included because the spatial resolution of the fMRI data was deemed insufficient to accurately resolve putative substructures within these nuclei (for example, the pars reticulata and compacta). Submillimeter fMRI and multi-echo EPI would benefit the investigation of the functional architecture of these exceedingly small nuclei<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="de Hollander, G., Keuken, M. C., van der Zwaag, W., Forstmann, B. U. &amp; Trampel, R. Comparing functional MRI protocols for small, iron-rich basal ganglia nuclei such as the subthalamic nucleus at 7 T and 3 T. Hum. Brain Mapp. 38, 3226–3248 (2017)." href="/articles/s41593-020-00711-6#ref-CR70" id="ref-link-section-d27595475e1919">70</a></sup>. A binary gray matter mask was delineated using the MNI152 probabilistic gray matter atlas. Voxels belonging to cortical gray matter were included in the gray matter mask as well as any voxels constituting the final subcortex mask. fMRI data were consistently absent for a majority of individuals in a small proportion of voxels constituting the gray matter mask. After eliminating these voxels, the final gray matter mask comprised <i>M</i> = 164,360 voxels (1,314.9 cm<sup>3</sup>), which included the midbrain, the pons and the cerebellum. The subcortex and gray matter masks were delineated in 2-mm isotropic space and were used for the 3T fMRI data. To suit the 7T-rfMRI data, which was in 1.6-mm isotropic space, the masks were resliced to this resolution and median filtered to smooth the mask edges.</p><h3 class="c-article__sub-heading" id="Sec23">Functional connectivity and eigenmaps</h3><p>Whole-brain functional connectivity was mapped for each subcortex voxel. Spatial gradients in the resulting maps were then computed to yield a continuous representation of functional connectivity variation across the subcortex. Connectopic mapping<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="/articles/s41593-020-00711-6#ref-CR11" id="ref-link-section-d27595475e1936">11</a></sup> was used to map spatial gradients for each individual, which involved computing a sequence of eigendecompositions to yield a Laplacian eigenmap<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Belkin, M. &amp; Niyogi, P. Laplacian eigenmaps for dimensionality reduction and data representation. Neural Comput. 15, 1373–1396 (2003)." href="/articles/s41593-020-00711-6#ref-CR71" id="ref-link-section-d27595475e1940">71</a></sup>. Specifically, following the procedure described by Haak and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="/articles/s41593-020-00711-6#ref-CR11" id="ref-link-section-d27595475e1944">11</a></sup>, the temporally concatenated fMRI signals were represented for each individual in a matrix of dimension <i>T</i> × <i>M</i>, where <i>T</i> denotes the number of time frames and <i>M</i> denotes the number of gray matter voxels. PCA was used to reduce the dimensionality of this matrix to <i>T</i> × (<i>T</i> – 1). The fMRI signal at each subcortical voxel was then correlated (Pearson correlation) with each column of the PCA-transformed matrix, resulting in a connectivity matrix of dimension <i>N</i> × (<i>T</i> – 1), where <i>N</i> denotes the number of subcortex voxels. Correlation coefficients were <i>r</i>-to-<i>z</i> transformed using the Fisher transformation. Each row of this matrix provided a connectional fingerprint<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 1664 (2015)." href="/articles/s41593-020-00711-6#ref-CR19" id="ref-link-section-d27595475e1983">19</a></sup> for a particular subcortical voxel in the PCA-transformed space. Similarity in the connectional fingerprints between each pair of subcortical voxels was then quantified with the <i>η</i><sup>2</sup> coefficient, which results in a symmetric matrix of dimension <i>N</i> × <i>N</i> for every individual. The similarity matrix was transformed into a sparse graph using the weighted adjacency matrix, whereby the weights of all connections with a Euclidean distance less than <i>ε</i> were set to zero. The connection density was determined by the smallest value of <i>ε</i> that ensured a connected graph<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="/articles/s41593-020-00711-6#ref-CR11" id="ref-link-section-d27595475e2004">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Fornito, A., Zalesky, A. &amp; Breakspear, M. Graph analysis of the human connectome: promise, progress, and pitfalls. NeuroImage 80, 426–444 (2013)." href="/articles/s41593-020-00711-6#ref-CR72" id="ref-link-section-d27595475e2007">72</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Cerliani, L. et al. Probabilistic tractography recovers a rostrocaudal trajectory of connectivity variability in the human insular cortex. Hum. Brain Mapp. 33, 2005–2034 (2012)." href="/articles/s41593-020-00711-6#ref-CR73" id="ref-link-section-d27595475e2010">73</a></sup>. This threshold varied between individuals, yielding graphs with a connection density of 0.4% on average. The Laplacian matrix, <i>L</i>, was then computed according to <i>L</i> = <i>D</i> – <i>W</i>, where <i>D</i> denotes the diagonal matrix of node strengths and <i>W</i> denotes the sparse adjacency matrix. In particular, <span class="mathjax-tex">\(D_{ij} = \mathop {\sum }\limits_i W_{ij}\)</span> if <i>i</i> = <i>j</i>, otherwise <i>D</i><sub><i>ij</i> </sub>= 0. Finally, the eigenvectors and eigenvalues of the Laplacian matrix were computed. The smallest eigenvalue was necessarily zero, while all other eigenvalues were positive due to the connectedness of the graph. The eigenvector with a zero eigenvalue was a constant and discarded. The eigenvectors with the second, third and fourth smallest eigenvalues were referred to as gradients I, II and III respectively. These steps are shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1a</a>. Each of the three gradients, or eigenmaps, characterized a continuous mode of spatial variation in functional connectivity across the spatial extent of the subcortex. We use the terms eigenmap and gradient interchangeably. Eigenvectors corresponding to the next largest eigenvalues typically explained substantially less variance in the Laplacian matrix than the first four eigenvectors (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">2</a>) and were thus given no further consideration here. The <i>N</i> values defining each gradient were projected onto the three-dimensional (3D) anatomy of the subcortex in MNI standard space for visualization and further analyses. Unless otherwise specified, the same process described above was used to estimate gradients for each individual and for each imaging modality, including 3T-rfMRI, 7T-rfMRI and tfMRI.</p><h3 class="c-article__sub-heading" id="Sec24">Gradientography</h3><p>Tractography is an established 3D modeling technique to visually represent axonal fiber bundles using diffusion MRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Jeurissen, B., Descoteaux, M., Mori, S. &amp; Leemans, A. Diffusion MRI fiber tractography of the brain. NMR Biomed. 32, e3785 (2019)." href="/articles/s41593-020-00711-6#ref-CR74" id="ref-link-section-d27595475e2128">74</a></sup>. We developed an analogous technique for fMRI called gradientography to characterize spatial gradients in functional connectivity. Gradientography was performed at the group level using a group-consensus representation of each eigenmap. In particular, similarity matrices were averaged across all individuals, and eigenmaps were computed for the group-averaged similarity matrix using the same process described above. The resulting group-consensus eigenmaps were projected onto the 3D anatomy of the subcortex in MNI standard space (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>, column I and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a>, column I). Using the Sobel gradient operator, directional gradients were computed for each subcortical voxel constituting the 3D image representing each eigenmap. Each subcortical voxel was therefore endowed with a gradient directed along each of the three canonical axes as well as a gradient magnitude (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>, column II and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a>, column II). We denote the gradient direction and magnitude at a given voxel with <span class="mathjax-tex">\(g = \left[ {g_x,g_y,g_z} \right]^T\)</span> and |<i>g</i>|, respectively. The gradient magnitude indicated how rapidly the functional connectivity of a local area of subcortex changed per unit length. Before application of the Sobel operator, the peripheral boundary of the subcortex in each eigenmap image was dilated by approximately one voxel to diminish spurious gradients from appearing along the direction that is perpendicular to the boundary. Erosion was performed after application of the Sobel operator to reverse this dilation and restore the original volume of the subcortex. The gradient directions and magnitudes were approximately mirror symmetric in the sagittal plane (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">3a</a>). Therefore, to enhance the SNR, the gradient directions were left–right symmetrized (to be consistent with the symmetrization of the subcortex mask). In particular, gradient directions in the left hemisphere were reflected into the right hemisphere around the mid-sagittal plane and averaged with the corresponding directions in the right hemisphere. These averaged gradient directions were then reflected back into the left hemisphere, and gradient magnitudes were recomputed (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">3b</a>). This was performed separately for each of the three eigenmaps, and the difference in the gradient direction before and after symmetrization was quantified (gradient I: <i>θ</i> = 5.42 ± 4.98°; gradient II: <i>θ</i> = 5.04 ± 5.0°; gradient III: <i>θ</i> = 4.95 ± 5.36°).</p><p>We developed the new gradientography procedure to provide a principled framework to visualize and analyze spatial gradients in functional connectivity. First, 3D tensors were fitted to each subcortical voxel using the constrained tensor model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Alexander, D. C. in Visualization and Processing of Tensor Fields (eds Weickert J. &amp; Hagen, H.) Ch. 5 (Springer, 2006)." href="/articles/s41593-020-00711-6#ref-CR75" id="ref-link-section-d27595475e2249">75</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Friman, O., Farneback, G. &amp; Westin, C. F. A Bayesian approach for stochastic white matter tractography. IEEE Trans. Med. Imaging 25, 965–978 (2006)." href="/articles/s41593-020-00711-6#ref-CR76" id="ref-link-section-d27595475e2252">76</a></sup>, which is synonymous with diffusion tensor imaging. The constrained tensor model is given by</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$D = \beta gg^T + \alpha I$$</span></div></div><p>where <i>D</i> denotes a 3 × 3 symmetric tensor, <i>g</i> is a column vector denoting the normalized gradient direction computed with the Sobel operator, <i>I</i> is the identity matrix, and <i>β</i> and <i>α</i> are parameters that quantify the strength of anisotropic and isotropic diffusion, respectively. Here, we set <i>β</i> = <i>Kc</i>|<i>g</i>| and <i>α</i> = <i>K</i>(1 – <i>c</i>), where <i>c</i> = 0.995 is a weighting factor that determines the extent of tensor anisotropy and <i>K</i> is a scale factor that governs the tensor size for visualization purposes. In this way, cigar-shaped tensors indicated rapid change in subcortical functional connectivity along the orientation of the longest axis of the tensor. In contrast, spherical tensors indicated functional connectivity that was constant along all spatial directions. It is important to note that tractography was only dependent on the gradient direction, <i>g</i>. The choice of <i>β</i>, <i>α</i> and <i>c</i> only influenced visualization of the tensor field (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>, column III and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a>, column III). While tensors were separately fitted for each of the three eigenmaps, an alternative methodology would have been to fit a bi-tensor or multi-tensor model to each voxel<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Peled, S., Friman, O., Jolesz, F. &amp; Westin, C. F. Geometrically constrained two-tensor model for crossing tracts in DWI. Magn. Reson. Imaging 24, 1263–1270 (2006)." href="/articles/s41593-020-00711-6#ref-CR77" id="ref-link-section-d27595475e2362">77</a></sup>, enabling representation of multiple eigenmaps within a single tensor field. This is conceptually analogous to modeling crossing fibers in diffusion MRI tractography<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Behrens, T. E., Berg, H. J., Jbabdi, S., Rushworth, M. F. &amp; Woolrich, M. W. Probabilistic diffusion tractography with multiple fibre orientations: what can we gain? NeuroImage 34, 144–155 (2007)." href="/articles/s41593-020-00711-6#ref-CR78" id="ref-link-section-d27595475e2366">78</a></sup>. However, the potential utility of modeling interactions between crossing gradients in this way remains unclear; thus, each eigenmap was independently analyzed. We relegate multidirectional gradientography to future work. Streamline propagation was performed using the interpolated streamline method with a step length of 1 mm, angle threshold of 30° and seeding 20 streamlines from randomly chosen coordinates within each subcortical voxel, yielding a total of approximately 15,000 streamlines. Streamline propagation was terminated when reaching the subcortex mask boundary. Gradientography was performed using Diffusion Toolkit software and visualized with TrackVis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Wang, R., Benner, T., Sorensen, A. &amp; Wedeen, V. Diffusion Toolkit: a software package for diffusion imaging data processing and tractography. Proc. Int. Soc. Magn. Reson. Med. 15, 3720 (2007)." href="/articles/s41593-020-00711-6#ref-CR79" id="ref-link-section-d27595475e2371">79</a></sup>. Streamlines were virtually unchanged when left–right symmetrization of the gradient directions was not performed. All streamlines were permitted to follow curvilinear trajectories of arbitrary complexity. The gradient magnitude image was projected onto the streamlines to parameterize the magnitude of change in functional connectivity and to visualize transitions in the gradient field with respect to the curvature of streamlines. In particular, each spatial coordinate constituting a streamline was assigned a gradient magnitude based on trilinear interpolation of the gradient magnitude image (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>, right most). Gradientography was also performed for gradients II and III using the same procedure (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a>). While streamline trajectories varied among gradients I, II and III, the locations of local maxima in the gradient magnitude images of the three gradients were consistent (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">3</a>). The gradient magnitude images for gradients II and III were spatially correlated (Spearman correlation) to gradient I (gradient I/II: <i>r</i> = 0.44, gradient I/III: <i>r</i> = 0.73); thus, only gradient I was used to delineate functional boundaries.</p><p>We also observed that head motion (framewise displacement (FD)) showed minimal effect on the group-consensus connectivity gradients. Specifically, individuals were ranked according to head motion, and individuals in the top 20% (mean FD: 0.26 ± 0.06 mm) and bottom 20% (mean FD: 0.10 ± 0.01 mm) defined two distinct subgroups. Group-consensus gradient magnitude images were computed for each of two subgroups and compared with the group-consensus gradient magnitude image generated within all individuals. We found that the group-consensus gradient magnitude images were strongly correlated between each pair of the three groups (Spearman correlation, <i>r</i> &gt; 0.9).</p><p>To preclude streamlines from traversing the slender anatomical gap between the tail of the caudate nucleus and the most anterior extent of the thalamus, the subcortex mask was separated at this precise location, giving rise to three spatially contiguous subcortical components: (1) the left basal ganglia, which comprised the striatum and the GP; (2) the right basal ganglia; and (3) the left and right thalamus, the hippocampus and the amygdala. Because these three components were spatially discontiguous, tractography was performed separately for each component. A boundary between the caudate and the thalamus was therefore imposed a priori, consistent with established anatomical knowledge. Due to left–right symmetrization of gradient directions, streamlines for the left and right basal ganglia were mirror symmetric; thus, it was sufficient to analyze only one of the hemispheres of the basal ganglia. We refer to the streamlines traversing the basal ganglia as the dorsal streamlines and the dorsal connectional topography (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2a</a>), while collectively referring to the streamlines traversing the left and right thalamus, the hippocampus and the amygdala as the ventral streamlines and ventral connectional topography (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2b</a>). Permitting streamlines to traverse the caudate–thalamus gap did not alter the location of local maxima in gradient magnitudes, although they were less prominent in this case (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig1">1b</a>).</p><h3 class="c-article__sub-heading" id="Sec25">Diversity curves</h3><p>The dorsal and ventral group of streamlines each comprised hundreds of distinct streamlines that characterized the connectional topographies associated with these regions. Streamlines shorter than 60 mm and 160 mm were discarded from the dorsal and ventral group, respectively. The remaining streamlines within each group traversed trajectories that shared similar profiles; thus, it was feasible to determine a single representative streamline for each group. To determine a representative streamline, the mean closest point distance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="O’Donnell, L. J. &amp; Westin, C. F. Automatic tractography segmentation using a high-dimensional white matter atlas. IEEE Trans. Med. Imaging 26, 1562–1575 (2007)." href="/articles/s41593-020-00711-6#ref-CR80" id="ref-link-section-d27595475e2421">80</a></sup> was computed between all pairs of streamlines constituting a group. In particular, the distance from one streamline to another was determined by first computing the distance between each point on the first streamline to all points on the second streamline. The minimum distance was then determined for each point on the first streamline. The distance from the first to second streamline was defined as the mean of these minimum distances. This was repeated for all pairs of streamline to generate a <i>J</i> × <i>J</i> distance matrix, where <i>J</i> denotes the number of streamlines. The distance matrix was symmetrized in a way that the mean of the two distances between a pair of streamlines was used<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="O’Donnell, L. J. &amp; Westin, C. F. Automatic tractography segmentation using a high-dimensional white matter atlas. IEEE Trans. Med. Imaging 26, 1562–1575 (2007)." href="/articles/s41593-020-00711-6#ref-CR80" id="ref-link-section-d27595475e2434">80</a></sup>. The representative streamline was chosen as the streamline with the shortest distance, on average, to all other streamlines. Having defined a representative streamline for the dorsal and ventral group of streamlines, it was then possible to compute a diversity curve with respect to each of the representative streamlines. This yielded a curvilinear parameterization of the group-consensus eigenmaps and their gradient magnitudes, thereby enabling statistical inference. The diversity curve concept is described in detail elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Tian, Y. &amp; Zalesky, A. Characterizing the functional connectivity diversity of the insula cortex: subregions, diversity curves and behavior. NeuroImage 183, 716–733 (2018)." href="/articles/s41593-020-00711-6#ref-CR15" id="ref-link-section-d27595475e2439">15</a></sup>.</p><p>Diversity curves were mapped in three steps: (1) projection of the gradient magnitude image onto each streamline; (2) spatial alignment of streamlines to the representative streamline using dynamic time warping<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Müller, M. in Information Retrieval for Music and Motion 69–84 (Springer, 2007)." href="/articles/s41593-020-00711-6#ref-CR81" id="ref-link-section-d27595475e2446">81</a></sup>; and (3) averaging across the aligned streamlines to yield a group-representative diversity curve to provide a basis for statistical inference. Specifically, each spatial coordinate constituting a streamline was assigned a gradient magnitude based on trilinear interpolation of the gradient magnitude image. While each streamline comprised the same number of equidistant coordinates, the exact lengths of streamlines varied, resulting in some degree of misalignment between streamlines in terms of coordinate-to-anatomy correspondence. For example, the tenth coordinate on a given streamline was not necessarily closest to the tenth coordinate on a neighboring streamline. Thus, averaging across a set of streamlines could not be performed based on correspondence in the index of coordinates. Dynamic time warping was used to spatially align individual streamlines to the representative streamline and to facilitate averaging of any measure parameterized along the length of each individual streamline.</p><p>Dynamic time warping is typically used to align signals in time, whereas the method was used here to achieve spatial alignment. A warping path was computed for each individual streamline such that the sum of the Euclidean distance was minimal between the warped and representative streamline. This involved either stretching or compressing coordinates along the trajectory of each individual streamline to achieve alignment with the representative streamline. Alignment quality was visually assessed, and streamlines with a warp distance exceeding the averaged warping distance over all streamlines (dorsal: 1.72 mm, ventral: 0.54 mm) were omitted. This yielded a total of 4,693 and 3,862 aligned streamlines for the dorsal and ventral group, respectively. The gradient (eigenmap) and gradient magnitude images projected onto each of these streamlines were then averaged for each point along the streamline trajectory, yielding a single gradient and gradient magnitude diversity curve for the dorsal (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2c</a>) and ventral group (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2d</a>). Gradient magnitude peaks in the diversity curves indicated putative functional boundaries between regions. Consistent with the gradient magnitude image across the entire subcortex, the gradient magnitude images for gradients II and III were also spatially correlated (Spearman correlation) to gradient I for both dorsal (gradient I/II: <i>r</i> = 0.79, gradient I/III: <i>r</i> = 0.79) and ventral (gradient I/II: <i>r</i> = 0.80, gradient I/III: <i>r</i> = 0.54) components. Statistical inference was therefore confined to the dominant gradient (gradient I).</p><p>Focusing on the dominant gradient is not without precedent, and this gradient shows good correspondence with underlying neuroanatomy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. Nat. Hum. Behav. 1, 0146 (2017)." href="/articles/s41593-020-00711-6#ref-CR23" id="ref-link-section-d27595475e2475">23</a></sup>, although non-dominant gradients can reveal complementary connectivity topographies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. NeuroImage 170, 83–94 (2018)." href="/articles/s41593-020-00711-6#ref-CR11" id="ref-link-section-d27595475e2479">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. Nat. Hum. Behav. 1, 0146 (2017)." href="/articles/s41593-020-00711-6#ref-CR23" id="ref-link-section-d27595475e2482">23</a></sup>. Three local maxima were evident in the dorsal streamline group, corresponding to established neuroanatomical boundaries among the GP–putamen, the putamen–NAc and the NAc–caudate (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2c</a>). For the ventral streamline group, nine local maxima were evident in the diversity curve representing the gradient magnitude, which suggests that there are putative functional boundaries among the amygdala–anterior hippocampus, the anterior–posterior hippocampus, the posterior hippocampus–posterior thalamus, the anterior–posterior thalamus and the left–right thalamus (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2d</a>).</p><h3 class="c-article__sub-heading" id="Sec26">Null model</h3><p>Although local maxima in the gradient magnitude images or the diversity curves representing these images indicated putative functional boundaries, these maxima could have also potentially arisen as a matter of random fluctuations reflecting inter-individual variation and finite sample effects or due to the geometry of the subcortex (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">5</a>). Therefore, null data were generated to statistically determine whether local maxima in the gradient images were larger than expected due to these random (sample) effects and/or confounds. The null hypothesis was of a spatially homogeneous gradient magnitude such that the rate at which whole-brain connectivity changed per unit length was constant at all points in the subcortex. Under this null hypothesis, any spatial heterogeneity was assumed to be due to sample effects or geometry. Functional boundaries were deemed to present at local maxima in the empirical gradient magnitude images when this null hypothesis could be locally rejected. Null data were generated by rewiring edges in the graph defined by the sparse adjacency matrix <i>W</i>. As detailed above (see “Functional connectivity and eigenmaps”), <i>W</i> defines a graph in which each node corresponds to a subcortical voxel, and the edge weights quantify similarity in functional connectivity profiles. A rewiring algorithm was developed to randomly reposition each edge, subject to the constraint that edges were more likely to be placed between pairs of voxels in close spatial proximity. This constraint was imposed to preserve the effect of smoothing in the null data. The following methodology was used to determine where edges should be repositioned to respect fMRI smoothing and subcortex geometry.</p><p>First, the fMRI data for each subcortical voxel was replaced with independent and randomly sampled Gaussian data that were matched in length and amplitude distribution. Second, the Gaussian data were smoothed using the same FWHM as the fMRI data. Third, the Pearson correlation coefficient was computed using the smoothed Gaussian data between all pairs of subcortical voxels. Finally, all voxel pairs were ranked from highest to lowest on the basis of the correlation coefficient. At each iteration of the rewiring algorithm, the edge being rewired was repositioned to the highest-ranking pair of voxels that had not yet received a rewired edge. To ensure that the rewired graphs were not fragmented, the first <i>N</i> – 1 edges were repositioned according to a minimum spanning tree (MST), where the MST was computed for a lattice graph in which each node corresponded to a subcortical voxel. Edges in the rewired graphs inherited their edge weights from <i>W</i>. An ensemble of 100 rewired graphs and corresponding sparse adjacency matrices <span class="mathjax-tex">\(\bar W_1,\bar W_2, \ldots ,\bar W_{100}\)</span> were generated by repeatedly initializing this rewiring procedure with randomly sampled Gaussian data. We found that repositioning edges to randomly chosen pairs of spatially neighboring voxels yielded virtually identical null data.</p><p>Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">5a</a> shows the graph corresponding to <i>W</i> and <span class="mathjax-tex">\(\bar W_1\)</span> for an axial slice that exemplifies the functional boundary between the GP and the putamen. Very few edges in the graph for <i>W</i> intersect this boundary, whereas the graph for <span class="mathjax-tex">\(\bar W_1\)</span> comprises numerous rewired edges that interconnect the GP and the putamen. Consistent with anatomy, this suggests that the strong gradient in functional connectivity that yields the boundary between these two regions is most likely not due to chance or the effect of smoothing and/or subcortex geometry. Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">5b</a> shows an example that demonstrates the importance of preserving geometry in the null data. In the bow-tie-shaped region constituting this example, binary edges are randomly positioned between neighboring pixels constituting the bow tie. This positioning of edges is consistent with the null hypothesis of a spatially homogenous gradient magnitude. However, the bow knot is the location of a geometric constriction, and if this geometry is not preserved in the null data, a specious boundary between the left and right sides of the bow is suggested.</p><p>Laplacian eigenmaps and gradient magnitude images were computed for each of <span class="mathjax-tex">\(\bar W_1,\bar W_2, \ldots ,\bar W_{100}\)</span> by following the same procedure described above for <i>W</i> (see “Functional connectivity and eigenmaps”). An ensemble of 100 diversity curves were then computed by projecting each of these gradient magnitude images onto the original streamlines that were computed with respect to <i>W</i>. These diversity curves were consistent with the null hypothesis and we refer to them as null diversity curves (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2c,d</a>).</p><h3 class="c-article__sub-heading" id="Sec27">Boundary delineation</h3><p>Statistical inference was performed to determine whether each local maximum in the gradient magnitude images was sufficiently large to warrant delineation of a discrete functional boundary. If the null hypothesis could not be rejected and thus delineation was not warranted, connectional topography was deemed to be most parsimoniously represented as a continuum. The decision to delineate a boundary or not was informed by statistical testing of the null hypothesis performed on both the gradient magnitude images and the diversity curves onto which these images were projected. All local maxima in the gradient magnitude images were considered as candidate boundaries, but candidate boundaries were delineated only if the null hypothesis was rejected; that is, the gradient magnitude was larger than expected due to chance and the effects of smoothing and subcortex geometry. In particular, a <i>P</i> value for each local maximum on a diversity curve was estimated as the proportion of null diversity curves with local maxima that (1) exceeded or equaled the observed maximum and (2) were located within the same vicinity as the observed maximum. The FDR was controlled at a threshold of 5% across the set of all maxima. The decision to delineate a boundary was supported for maxima with <i>P</i> values that survived FDR correction. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2c,d</a> shows examples of diversity curve maxima for which the null hypothesis was rejected (denoted by blue asterisks).</p><p>The null hypothesis could also be tested with respect to the gradient magnitude images. In this case, the Kolmogorov–Smirnov (KS) test was used to assess the null hypothesis of equality in the distribution across voxels between the observed gradient magnitudes and the null gradient magnitudes. The null hypothesis was rejected if the tail of the distribution of gradient magnitudes was longer in the observed data compared with the null data. The main reasons for reverting to the KS test at finer scales were that (1) generally, only one candidate boundary was evident within each region constituting scales II–IV (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">23</a>) and (2) we found that mapping diversity curves for relatively small regions did not provide substantial additional insight. Diversity curves and gradientography yield most insight when applied across a relatively large expanse that comprises multiple candidate boundaries (that is, scale I). Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">23</a> shows examples of using the KS test to determine whether or not to delineate a candidate boundary in two specific subcortical regions. A strong gradient is evident in the anterior putamen (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">23a</a>, yellow strip), and the KS test indicates that the gradient magnitude distribution across the voxels of this region (green curve) is significantly longer tailed than the null data (gray curves, <i>P</i> &lt; 0.01). Therefore, a boundary was delineated that resulted in the parcellation of the anterior putamen into one dorsal and one ventral component. In contrast, the null hypothesis could not be rejected with the KS test for the NAc-shell (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">23b</a>, <i>P</i> = 0.22); thus, subdivision of this region was not warranted despite the presence of a candidate boundary (yellow strip).</p><p>The final decision to delineate a boundary was not exclusively determined by whether the null hypothesis could be rejected or not. Other criteria that informed this decision were as follows:</p><ol class="u-list-style-none">
                  <li>
                    <span class="u-custom-list-number">1.</span>
                    
                      <p>Size criterion: a candidate boundary was not delineated if at least one of the regions resulting from boundary delineation would have comprised fewer than 100 voxels, irrespective of whether the null hypothesis was rejected.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">2.</span>
                    
                      <p>Prior knowledge: a candidate boundary that corresponded with established subcortex anatomy or cytoarchitecture was delineated, irrespective of whether the null hypothesis was rejected.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">3.</span>
                    
                      <p>Inter-hemispheric homolog: a candidate boundary was not delineated if the null hypothesis could not be rejected for the corresponding contralateral boundary. Edges in the rewired graphs (null data) inherited their edge weights from <i>W</i>, which was not necessarily symmetric. Therefore, the null data were not symmetrized; thus, discrepancies in statistical inference were possible between contralateral boundaries.</p>
                    
                  </li>
                </ol><p>These criteria were only ever used for three boundaries of the more than 50 decisions pertaining to boundary delineation (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">6</a>). In particular, the size criterion prevented further segmentation of a region of the caudate (CAU-DA), the prior knowledge criterion was used to delineate the thalamus–hippocampus boundary at scale I, despite failure to reject the null hypothesis, and the homolog criterion was used for the medial amygdala (mAMY) at scale II. These criteria were not used for any other cases, and all other boundary delineations were supported by the model selection procedure.</p><h3 class="c-article__sub-heading" id="Sec28">Hierarchical parcellation</h3><p>The subcortex was parcellated into distinct functional regions in a hierarchical manner. Each successive scale of the hierarchy involved delineation of additional boundaries that conveyed finer details in connectional topography. Foremost, boundaries were delineated on the basis of maxima in the dorsal and ventral diversity curves for which the null hypothesis was rejected. This resulted in a subcortex parcellation comprising eight bilateral regions that recapitulated well-known subcortical regions and established anatomical boundaries (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig3">3a</a>). These eight bilateral regions defined scale I of the hierarchical parcellation (16 regions in total). Scale I is not intended to provide a novel parcellation, but rather establishes the validity of the model selection procedure and represents a necessary first step to delineate the novel scale II–IV atlases at finer scales. The next scale of the hierarchy (scale II) was delineated by separately computing Laplacian eigenmaps and gradient magnitudes for each of the 16 regions using the same procedure that was initially applied to the entire subcortex. This process was repeated in a recursive manner until the null hypothesis could not be rejected for any candidate boundaries, a condition that was satisfied at scale IV of the hierarchy (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig3">3b</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig4">4</a>). A given region in scale <i>k</i> appeared as either two or more subregions in scale <i>k</i> + 1, if the null hypothesis was rejected, otherwise it appeared unaltered. For each scale, the FDR was controlled at 5% across the family of null hypothesis tests performed for all candidate boundaries. Unlike alternative multiple testing corrections, the FDR is scalable and adaptive<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Benjamini, Y. Discovering the false discovery rate. J. R. Stat. Soc. Ser. B 72, 405–416 (2010)." href="/articles/s41593-020-00711-6#ref-CR82" id="ref-link-section-d27595475e2908">82</a></sup>, meaning that the greater number of candidate boundaries tested at scale IV, compared with scale I, did not make FDR correction at scale IV inherently more stringent. Note that the FDR test was performed independently for each scale. Successive scales of the parcellation hierarchy conveyed finer boundaries and more regions (scale I: 16 regions; scale II: 32 regions; scale III: 50 regions; scale IV: 54 regions). The diameter of all regions constituting scale IV substantially exceeded the FWHM of the smoothing kernel width (minimum: 10.2 mm, maximum: 24.4 mm, median: 17.2 mm). While exceedance of the FWHM is a heuristic criterion, this suggests that the extent of smoothing was sufficiently constrained to enable delineation of the smallest region.</p><p>For scale I, the null hypothesis was tested by performing inference on maxima in the dorsal and ventral diversity curves. In particular, inference was performed to determine whether each maximum in the gradient magnitude was significantly larger than expected due to chance, geometry and smoothing (see “Boundary delineation”). For scale II and beyond, as described above, the KS statistic was used to test the null hypothesis due to challenges in mapping diversity curves for the relatively small regions constituting these scales. Delineation of small regions was also more susceptible to the effects of noise, especially signal contamination from surrounding white matter and cerebrospinal fluid. To reduce these effects, for scale II and beyond, the adjacency matrix (<i>W</i>) used to compute Laplacian eigenmaps was thresholded using the disparity filter<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Serrano, M. Á., Boguñá, M. &amp; Vespignani, A. Extracting the multiscale backbone of complex weighted networks. Proc. Natl Acad. Sci. USA 106, 6483–6488 (2009)." href="/articles/s41593-020-00711-6#ref-CR83" id="ref-link-section-d27595475e2918">83</a></sup>. The significance level for the disparity filter was set to the minimum value required to yield a graph that was not fragmented.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">Watershed algorithm</h4><p>The watershed transform algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Meyer, F. Topographic distance and watershed lines. Signal Process. 38, 113–125 (1994)." href="/articles/s41593-020-00711-6#ref-CR84" id="ref-link-section-d27595475e2929">84</a></sup> was used to cluster voxels and thereby identify spatially contiguous regions that resulted from delineation of a boundary, as applied elsewhere to parcellate the cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e2933">8</a></sup>. The watershed transform was applied to the 3D gradient magnitude images. To initiate the watershed, seed voxels were first chosen such that exactly one seed voxel resided on each side of a boundary. Seed voxels were approximately positioned to reside within a regional minimum of the gradient magnitude image. A regional minimum that was distant from the boundary was preferred, although the parcellation was robust against the precise seed voxel location. Clusters were then iteratively grown for each seed region until locations were reached where voxels could no longer be unequivocally assigned to one of the clusters. Clusters were therefore analogous to water catchment basins (that is, minima in the gradient magnitude images), while boundaries represented watershed ridge lines (that is, maxima in these images). Before applying the watershed transform, the gradient magnitude images were (1) normalized to ensure that all values resided on the unit interval and (2) morphologically reconstructed to ensure that regional minima were only present at the location of the chosen seed voxels. The latter step aimed to enhance the contrast between voxels within the expected cluster and boundary voxels. The output of the watershed transform was a cluster of voxels for each seed voxel. Voxels that resided at boundary locations where not necessarily assigned to any cluster, yielding empty space between parcellated subregions. To fill the empty space, for each unallocated voxel, the shortest path was computed to each seed voxel. Shortest paths were computed with respect to the sparse adjacency matrix (<i>W</i>) following a strength-to-distance remapping of the edge weights. Unallocated voxels were then assigned to subregions with the minimum shortest path distance.</p><h3 class="c-article__sub-heading" id="Sec30">Parcellation validation and replication</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec31">Homogeneity estimation</h4><p>A valid functional parcellation should comprise regions that are internally homogenous with respect to the functional connectivity profile of each constituent voxel. The regional homogeneity of scale I–IV parcellations was compared with the extent of homogeneity achieved with alternative subcortex parcellations as well as the homogeneity that could be expected due to chance.</p><p>The fMRI data for a region was represented as a matrix of dimension <i>N</i> × <i>T</i>, where <i>N</i> and <i>T</i> denote the number of voxels constituting the region and the number of time frames, respectively. Regional homogeneity was defined as the variance explained by the first principal component of this matrix, which was estimated using PCA. Homogeneity therefore quantified the extent of synchrony in fMRI activity across voxels constituting a region. While homogeneity can be computed with respect to functional connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e2968">8</a></sup>, we performed estimation directly on the fMRI time series acquired in the second session (REST2). This ensured that validation was performed using independent data and an independent quantity. Homogeneity was separately computed for each region and each individual. For each individual, homogeneity was then averaged across all regions to yield an overall estimate of homogeneity for the parcellation as a whole, referred to as parcellation homogeneity (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">11a</a>).</p><p>Parcellation homogeneity increased from scale I to scale IV (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig5">5a</a>, red dotted line). This was because small regions were inherently more homogeneous than large regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288–303 (2016)." href="/articles/s41593-020-00711-6#ref-CR8" id="ref-link-section-d27595475e2982">8</a></sup>. To determine whether the observed homogeneity values were larger than expected due to chance, the subcortex was randomly parcellated such that the total number of regions and the distribution of region size were matched between the random and observed parcellations (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">11b</a>). Parcellation homogeneity was then estimated for 100 such random parcellations to yield a null distribution for homogeneity under the null hypothesis of randomly delineated regions. A <i>P</i> value for this null hypothesis was estimated as the proportion of random parcellations with homogeneity exceeding or equaling the observed homogeneity. Parcellations were deemed homogeneous if the null hypothesis was rejected (<i>P</i> &lt; 0.05). Random parcellations were generated by (1) randomly placing seed voxels throughout the subcortex mask in such a way that the Euclidean distance between all pairs of seed voxels was approximately maximized and (2) assigning all non-seed voxels to their closest seed voxel in terms of the Euclidean distance. This assignment process defined a set of randomly delineated regions. The number of seed voxels was set to the number of desired regions. Parcellations were discarded if they contained any regions that were not spatially contiguous or not matched with respect to the distribution of region sizes. Discarded parcellations were replaced until 100 valid random parcellations were generated.</p><p>Parcellation homogeneity was also computed for existing subcortex parcellation atlases (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">3</a>). The scale (I–IV) that comprised the same or fewer parcels than the existing parcellation was chosen for comparison. Choosing a scale with fewer parcels provided an advantage to the existing parcellations. We specifically evaluated the homogeneity of two existing parcellation atlases for the entire subcortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ji, J. L. et al. Mapping the human brain’s cortical–subcortical functional network organization. NeuroImage 185, 35–57 (2019)." href="/articles/s41593-020-00711-6#ref-CR6" id="ref-link-section-d27595475e3001">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Fan, L. et al. The Human Brainnetome Atlas: a new brain atlas based on connectional architecture. Cereb. Cortex 26, 3508–3526 (2016)." href="/articles/s41593-020-00711-6#ref-CR33" id="ref-link-section-d27595475e3004">33</a></sup> as well as parcellation atlases for the hippocampus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Amunts, K. et al. Cytoarchitectonic mapping of the human amygdala, hippocampal region and entorhinal cortex: intersubject variability and probability maps. Anat. Embryol. 210, 343–352 (2005)." href="/articles/s41593-020-00711-6#ref-CR22" id="ref-link-section-d27595475e3008">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Plachti, A. et al. Multimodal parcellations and extensive behavioral profiling tackling the hippocampus gradient. Cereb. Cortex 29, 4595–4612 (2019)." href="/articles/s41593-020-00711-6#ref-CR35" id="ref-link-section-d27595475e3011">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Eickhoff, S. B. et al. A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. NeuroImage 25, 1325–1335 (2005)." href="/articles/s41593-020-00711-6#ref-CR85" id="ref-link-section-d27595475e3014">85</a></sup>, the thalamus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Behrens, T. E. et al. Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging. Nat. Neurosci. 6, 750–757 (2003)." href="/articles/s41593-020-00711-6#ref-CR37" id="ref-link-section-d27595475e3018">37</a></sup> and the striatum<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Janssen, R. J., Jylänki, P., Kessels, R. P. C. &amp; van Gerven, M. A. J. Probabilistic model-based functional parcellation reveals a robust, fine-grained subdivision of the striatum. NeuroImage 119, 398–405 (2015)." href="/articles/s41593-020-00711-6#ref-CR36" id="ref-link-section-d27595475e3022">36</a></sup>. Cerebellar and brainstem regions were omitted from the parcellation delineated by Ji et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ji, J. L. et al. Mapping the human brain’s cortical–subcortical functional network organization. NeuroImage 185, 35–57 (2019)." href="/articles/s41593-020-00711-6#ref-CR6" id="ref-link-section-d27595475e3027">6</a></sup> and Fan et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Fan, L. et al. The Human Brainnetome Atlas: a new brain atlas based on connectional architecture. Cereb. Cortex 26, 3508–3526 (2016)." href="/articles/s41593-020-00711-6#ref-CR33" id="ref-link-section-d27595475e3031">33</a></sup> to ensure homogeneity estimates were specific to the subcortex and therefore comparable to the other parcellations. To facilitate a fair comparison across parcellations containing a disparate number of regions, observed parcellation homogeneity values were normalized with respect to the mean homogeneity across the set of random parcellations. Ratios exceeding one indicated that parcellations were more homogeneous than expected due to chance.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec32">7T replication</h4><p>High-resolution fMRI acquired at ultrahigh magnetic field strength (7T) can enhance functional contrast-to-noise ratios, improve spatial localization and alleviate partial volume effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Vu, A. et al. Tradeoffs in pushing the spatial resolution of fMRI for the 7T Human Connectome Project. NeuroImage 154, 23–32 (2017)." href="/articles/s41593-020-00711-6#ref-CR24" id="ref-link-section-d27595475e3043">24</a></sup>. We sought to test the extent to which parcellations delineated using 3T-rfMRI could be replicated using 7T-rfMRI. To this end, the subcortex was hierarchically parcellated by applying the same procedures described above to the 7T-rfMRI data. This involved mapping Laplacian eigenmaps and gradient magnitudes, null hypothesis testing for each maximum comprising the gradient magnitude images and application of the watershed transform to delineate subregions. This yielded a 7T parcellation atlas of the subcortex that also comprised four scales (scale I: 16 regions; scale II: 34 regions; scale III: 54 regions; scale IV: 62 regions).</p><p>To enable comparisons between 3T and 7T parcellations, the 3T parcellations were up-sampled to the same spatial resolution as 7T (1.6-mm isotropic), and the spatial similarity between each scale of 3T and 7T parcellations was estimated using normalized mutual information (NMI)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Lancichinetti, A., Fortunato, S. &amp; Kertész, J. Detecting the overlapping and hierarchical community structure in complex networks. N. J. Phys. 11, 033015 (2009)." href="/articles/s41593-020-00711-6#ref-CR86" id="ref-link-section-d27595475e3050">86</a></sup>. NMI is an information theoretic measure that quantifies the extent to which knowledge of a parcellation reduces the uncertainty in the location of regions in a second parcellation. NMI ranges between zero and one, with higher NMI values indicating greater spatial correspondence between parcellations.</p><h3 class="c-article__sub-heading" id="Sec33">Personalized parcellation</h3><p>Following Glasser and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e3063">2</a></sup>, a machine-learning approach was used to personalize the group-consensus parcellations and to distinguish individual differences in functional organization, thereby yielding a unique parcellation for each individual. This enabled quantification of the extent of variation among individuals for each region. The subcortex mask remained fixed for all individuals; thus, only variation in functional boundaries within the subcortex was permitted, without variation in the overall subcortical volume. The periphery of each region comprising the scale IV parcellation was first dilated by approximately 4–6 mm to define a zone of uncertainty around each region. The uncertainty zone enabled flexibility in the location of boundaries for individuals who exceeded the group-consensus boundary. The extent of dilation was such that each region in the group-consensus atlas comprised approximately the same number of voxels as its uncertainty zone, meaning that a voxel was equally likely to be randomly classified as belonging to the region compared to its uncertainty zone. Enlarging the uncertainty zone would have permitted individual boundary locations to deviate to a greater extent from the group-consensus boundaries, although major deviations were not expected in young and healthy adults, which is consistent with previous work on the cortex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-020-00711-6#ref-CR2" id="ref-link-section-d27595475e3067">2</a></sup>. A binary support vector machine (SVM) classifier was trained to classify whether a voxel resided in the region or its uncertainty zone. We use <i>M</i> to denote the combined number of voxels constituting the region and its uncertainty zone.</p><p>Combining REST1 and REST2 would potentially yield more accurate estimates of functional connectivity compared with any of the individual sessions. However, given that parcellation was performed using data from the first 3T-rfMRI session (REST1), to assess inter-session reproducibility and to minimize circularity, the SVM was trained and evaluated using data from the second session (REST2). One hundred individuals were randomly selected to train the SVM, and the trained SVM was then applied to delineate a personalized parcellation for the remaining 921 individuals. It is important to remark that each individual contributed <i>M</i> training samples because classification was undertaken on a per-voxel basis. The SVM feature space was defined over the symmetric <i>N</i> × <i>N</i> similarity matrix, whereby each cell in this matrix stored the similarity in functional connectivity fingerprints between a pair of subcortex voxels (see “Functional connectivity and eigenmaps”). The feature vector for a particular voxel was selected as the relevant row in this matrix. The feature space was separately mapped for each individual and required computation of individual similarity matrices. A radial basis function kernel was used, and the feature space was standardized.</p><p>For each of the 921 individuals in the test set, the SVM was used to predict the likelihood (posterior probability) of each voxel belonging to the region under consideration, as opposed to belonging to the uncertainty zone of the region. See Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">17</a> for a schematic of this binary classification problem. The threshold for the posterior probability of a voxel belonging to the region was varied between 0 and 100%, whereby a threshold of 0% led to all voxels, including those in the uncertainty zone, being classified as belonging to the region. Increasing the threshold eventually resulted in loss of spatial contiguity. The extent to which the threshold could be increased while preserving these properties provided insight into parcellation robustness. This process was repeated separately for each region comprising the scale IV parcellation, requiring an SVM to be independently trained for each region. It was possible for voxels to be included in the uncertainty zones of multiple regions. A voxel could therefore be assigned a probability of belonging to each of many regions. To delineate a personalized parcellation for each individual, a winner-takes-all rule was used to break ties, such that voxels were allocated to regions with the highest probability. Crucially, if the highest probability region did not exceed the posterior probability threshold, the voxel remained unassigned. This introduced the possibility of certain regions in the group-consensus parcellation being absent from the personalized parcellation of an individual. The regional detection rate was the proportion of individuals for which a given region was allocated at least one voxel. The detection rate of each region was evaluated as a function of the posterior probability threshold. The group-consensus parcellation was assumed to provide a valid representation of individuals for regions with high detection rates.</p><p>The spatial correspondence between the group-consensus and personalized parcellations was also evaluated with the Dice coefficient. In particular, the Dice coefficient was separately computed for each individual and each region as a function of the posterior probability threshold. If <i>X</i> and <i>Y</i> denote the indices of the voxels constituting the personalized region of an individual and the group-consensus region, respectively, the Dice coefficient was computed as <span class="mathjax-tex">\(2\left| {X \cap Y} \right|/\left( {\left| X \right| + \left| Y \right|} \right)\)</span>. An average Dice coefficient was then computed for each individual by averaging the regional Dice coefficients.</p><h3 class="c-article__sub-heading" id="Sec34">Task-evoked connectional gradients</h3><p>We aimed to investigate whether the connectional topography of the subcortex would exhibit significant variation between task conditions and rest<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Salehi, M. et al. There is no single functional atlas even for a single individual: functional parcel definitions change with task. NeuroImage 208, 116366 (2020)." href="/articles/s41593-020-00711-6#ref-CR27" id="ref-link-section-d27595475e3168">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Greene, A. S., Gao, S., Scheinost, D. &amp; Constable, R. T. Task-induced brain state manipulation improves prediction of individual traits. Nat. Commun. 9, 2807 (2018)." href="#ref-CR87" id="ref-link-section-d27595475e3171">87</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cole, M. W., Bassett, D. S., Power, J. D., Braver, T. S. &amp; Petersen, S. E. Intrinsic and task-evoked network architectures of the human brain. Neuron 83, 238–251 (2014)." href="#ref-CR88" id="ref-link-section-d27595475e3171_1">88</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Medaglia, J. D., Lynall, M. E. &amp; Bassett, D. S. Cognitive network neuroscience. J. Cogn. Neurosci. 27, 1471–1491 (2015)." href="/articles/s41593-020-00711-6#ref-CR89" id="ref-link-section-d27595475e3174">89</a></sup>. The tasks considered were emotion processing, gambling, language processing, motor execution, relational matching, social inference and working memory. Details pertaining to each task condition are described elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Barch, D. M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. NeuroImage 80, 169–189 (2013)." href="/articles/s41593-020-00711-6#ref-CR54" id="ref-link-section-d27595475e3178">54</a></sup>.</p><p>Following the same procedure described above (see “Functional connectivity and eigenmaps”), group-consensus eigenmaps and gradient magnitude images were computed for each of the seven tasks. For each task, the gradient magnitude image for gradient I was projected onto the dorsal and ventral streamlines. This yielded diversity curves for each task (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19a</a>). Tractography of the task-derived gradients was not performed, and streamlines previously determined at rest (see “Diversity curves”) were used as references. To facilitate comparisons of diversity curves between different tasks, only the 725 individuals (360 males) that completed all nine conditions (two rest and seven tasks) were included. The mean diversity curve across the nine conditions was first computed and regressed from the diversity curves for each of the nine conditions. This eliminated the common variance across the conditions. The Pearson correlation coefficient was computed between the resulting residuals to generate a correlation matrix of dimension 9 × 9, whereby each element corresponded to a pair of task conditions. Permutation testing was used to determine a <i>P</i> value for each correlation coefficient. Specifically, for each pair of comparisons, the points on one of the diversity curves were randomly permutated and the correlation was recomputed on the permuted diversity curves. This was repeated for 10,000 permutations, yielding a null distribution for the correlation coefficient. The <i>P</i> value was given by the proportion of permutations with the correlation coefficient that exceeded or equalled the correlation coefficient in the unpermuted data. Bonferroni correction was controlled at 0.05/36 = 0.0014 across the set of all unique pairs of conditions. The pairs of conditions that survived correction are shown in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19b</a> and visualized as a graph to show similarities in task conditions (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19c</a>). Furthermore, a matrix of dimension <i>H</i> × 9 was then constructed, where <i>H</i> denotes the number of points along the diversity curve and each column corresponds to one of the nine conditions. Each cell in the matrix stored the residual value of each diversity curve point. Rows and columns were reordered according to average-linkage clustering based on the Pearson correlation coefficient (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">19d</a>).</p><p>Given that substantially fewer fMRI volumes were acquired during task compared to rest (see “Data and pre-processing”), supplementary analyses were undertaken to test whether task-related changes in connectional gradients could be attributed to the acquisition of fewer volumes. Specifically, the group-consensus eigenmaps and the gradient magnitude images were recomputed using only the first quarter of time frames (<i>T</i> = 278) from each run of the rfMRI data (278 × 2 in total), thereby matching the average number of time frames across the seven tasks. We found that the gradient magnitude images computed using the shortened and full-length rfMRI acquisitions were highly correlated (Spearman correlation: <i>r</i> &gt; 0.98 for both REST1 and REST2). This suggests that differences in the number of volumes between the tfMRI and rfMRI acquisitions do not explain the task-evoked reconfiguration of connectional gradients and diversity curves.</p><h3 class="c-article__sub-heading" id="Sec35">Relating subcortical functional network to behaviors</h3><p>Individual variation in functional connectivity associates with human behaviors that relate to cognition, emotion and psychopathology in both health and mental illness<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 1664 (2015)." href="/articles/s41593-020-00711-6#ref-CR19" id="ref-link-section-d27595475e3227">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Smith, S. M. et al. A positive-negative mode of population covariation links brain connectivity, demographics and behavior. Nat. Neurosci. 18, 1565–1567 (2015)." href="#ref-CR90" id="ref-link-section-d27595475e3230">90</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fornito, A., Zalesky, A., Pantelis, C. &amp; Bullmore, E. T. Schizophrenia, neuroimaging and connectomics. NeuroImage 62, 2296–2314 (2012)." href="#ref-CR91" id="ref-link-section-d27595475e3230_1">91</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Xia, C. H. et al. Linked dimensions of psychopathology and connectivity in functional brain networks. Nat. Commun. 9, 3003 (2018)." href="/articles/s41593-020-00711-6#ref-CR92" id="ref-link-section-d27595475e3233">92</a></sup>. While cortico–cortical connections have been well studied and predict individual variation in human behavior, less is known about the predictive utility of functional connectivity within the subcortex. Our multiscale subcortex parcellation facilitated investigations of associations between intrasubcortical functional connectivity and behavioral phenotypes for each atlas scale.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec36">Functional connectivity and modular structure</h4><p>Subcortical functional connectivity matrices were mapped for each individual and each scale of the parcellation. To this end, representative fMRI time series were determined for each region by averaging across all relevant voxels. Pearson correlation coefficients in regional time series were computed for all pairs of regions and represented in the form of a symmetric functional connectivity matrix. The connectivity matrices were then <i>r</i>-to-<i>z</i> transformed (Fisher transformation). The rows and columns of the connectivity matrices were reordered to accentuate modular structure, whereby the modular structure was determined using the Louvain community detection algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. J. Stat. Mech. Theory Exp. 2008, P10008 (2008)." href="/articles/s41593-020-00711-6#ref-CR93" id="ref-link-section-d27595475e3250">93</a></sup>. Separate connectivity matrices were mapped for the first (REST1) and second (REST2) 3T-rfMRI sessions to provide a discovery and within-sample replication set.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec37">Behavioral phenotypes</h4><p>Behavioral measurements and procedures for the HCP are described in detail elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Barch, D. M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. NeuroImage 80, 169–189 (2013)." href="/articles/s41593-020-00711-6#ref-CR54" id="ref-link-section-d27595475e3262">54</a></sup>. Items that tapped alertness, cognition, emotion, sensory–motor function, personality, psychiatric symptoms, substance use and life function were selected for further analyses, subject to the following conditions:</p><ol class="u-list-style-none">
                    <li>
                      <span class="u-custom-list-number">1.</span>
                      
                        <p>Only total or subtotal scores were selected for psychological assessments.</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">2.</span>
                      
                        <p>Only raw scores were selected for items with both raw and age-adjusted scores.</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">3.</span>
                      
                        <p>Items based on strictly binary responses were excluded.</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">4.</span>
                      
                        <p>Items with missing responses in more than 10% of individuals were excluded.</p>
                      
                    </li>
                  </ol><p>Items relating to task performance in emotional processing, language processing, social recognition, motor execution, working memory, relational matching and reward/decision-making processing were also included. This resulted in a total of 109 behavioral measures for each individual (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">4</a>). Participants with missing responses for any of these 109 items were then excluded, resulting in a final sample comprising 958 individuals (453 males, mean age of 28.7 ± 3.7 years). Two 3T-rfMRI sessions were acquired in all these individuals.</p><p>Independent component analysis (ICA) was used to reduce the set of behavioral measures to a set of latent dimensions with the aim of performing inference on the dimensions rather than the full set of behaviors. Before performing ICA, behavioral items with continuously distributed scores (87 out of 109) were normalized with a rank-based inverse Gaussian transformation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Van der Waerden, B. Oder tests for two-sample problem and their power. Proc. K. Nederlandse Akademie van Wetenschappen. Ser. A 55, 453–458 (1952)." href="/articles/s41593-020-00711-6#ref-CR94" id="ref-link-section-d27595475e3321">94</a></sup>. Age and sex were then regressed from all items and ICA was performed on the residuals that resulted from this regression. Other confounding factors such as blood pressure, hemoglobin, height and weight were not included due to missing responses for several individuals. ICA was performed using the Icasso package<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Himberg, J., Hyvarinen, A. &amp; Esposito, F. Validating the independent components of neuroimaging time series via clustering and visualization. NeuroImage 22, 1214–1222 (2004)." href="/articles/s41593-020-00711-6#ref-CR95" id="ref-link-section-d27595475e3325">95</a></sup> for Matlab, which provides an implementation of the FastICA algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Hyvarinen, A. Fast and robust fixed-point algorithms for independent component analysis. IEEE Trans. Neural Netw. 10, 626–634 (1999)." href="/articles/s41593-020-00711-6#ref-CR96" id="ref-link-section-d27595475e3329">96</a></sup>. Under ICA, the matrix of residuals (behaviors × individuals), denoted with <i>Y</i>, was factorized into the product <i>Y</i> = <i>A</i><sup>−1</sup><i>S</i>, where <i>A</i><sup>−1</sup> denotes the mixing matrix (behaviors × components) and <i>S</i> denotes the estimated independent sources (components × individuals). The inverse of <i>A</i><sup>−1</sup> is denoted <i>A</i> (components × behaviors) and referred to as the de-mixing matrix.</p><p>To estimate the de-mixing matrix, FastICA was performed using default parameter settings, including a cubic nonlinearity and deflationary orthogonalization. Bootstrapping was used to sample individuals with replacement, and ICA was performed independently on each bootstrapped sample using randomly chosen initial conditions. A total of 500 bootstrapped samples were generated. Independent components that represented a consensus across the 500 samples were then derived using a clustering procedure. In particular, agglomerative clustering with average linkage was used to partition the de-mixing matrix into putative clusters. The number of dimensions in the feature space was the number of behaviors, and each cluster comprised groups of components (that is, rows from <i>A</i>). Clearly separated clusters indicated that independent components were consistently and reliably estimated despite randomization of initial conditions and bootstrapping. A cluster quality index was used to quantify cluster separation and provided a metric to select the number of independent components (see below). Cluster separation was quantified by the difference in average intra-cluster similarity and average inter-similarity, denoted by <i>I</i>q. A higher <i>I</i>q suggests compact clusters, whereas a lower <i>I</i>q suggests that clusters are not well separated. These operations were performed internally in the Icasso package, and further details are available in the documentation of the package.</p><p>In addition to these internal procedures, we performed a sampling and matching process to ensure stability and reliability of the estimated independent components. Specifically, the ICA as described above was repeated for ten trials, yielding an ensemble of de-mixing matrices, <i>A</i><sub>1</sub>,<i>A</i><sub>2</sub>,…,<i>A</i><sub>10</sub> and corresponding independent components, <i>S</i><sub>1</sub>,<i>S</i><sub>2</sub>,…,<i>S</i><sub>10</sub>. Variability in <i>A</i><sub><i>u</i></sub> and <i>S</i><sub><i>u</i></sub> across trials was due to randomization of initial conditions and random sampling inherent to bootstrapping. Each row of <i>A</i> corresponded to an independent component, and each column quantified the weighting of a behavioral item in the given component. We sought to evaluate the similarity of the resulting estimates across trials to investigate whether the Icasso procedure yielded reproducible independent components across the ten trials. For each pair of trials <i>u</i> and <i>v</i>, a correlation matrix of dimension <i>I</i> × <i>I</i> was computed, where <i>I</i> denotes the number of independent components. Element (<i>i,j</i>) of this correlation matrix stored the Pearson correlation coefficient between row <i>i</i> of <i>A</i><sub><i>u</i></sub> and row <i>j</i> of <i>A</i><sub><i>v</i></sub>. We denote the correlation matrix between trial <i>u</i> and <i>v</i> with <i>C</i><sub><i>uv</i></sub>, <i>u,v</i> = 1,2…,10. Given that independent components are resolved up to an arbitrary sign flip, the absolute value of all correlation coefficients was considered. The <i>n</i>th component in trial <i>u</i> did not necessarily correspond to the <i>n</i>th component in trial <i>v</i>; thus, a one-to-one matching of components was required for each pair of trials. The Hungarian algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Kuhn, H. W. The Hungarian method for the assignment problem. Nav. Res. Logist. Q 2, 83–97 (1955)." href="/articles/s41593-020-00711-6#ref-CR97" id="ref-link-section-d27595475e3487">97</a></sup> was applied to <i>C</i><sub><i>uv</i></sub> to determine a one-to-one matching that maximized the correlation coefficients between the matched pairs of components. If ICA estimates were reproducible between trials <i>u</i> and <i>v</i>, each column of <i>C</i><sub><i>uv</i></sub> should comprise a correlation coefficient that is substantially larger than all others in the column, indicating an obvious matching. The trial that was best matched to all other trials was chosen as the reference trial, whereby matching quality was quantified as the average correlation coefficient computed over all matched pairs of components in <i>C</i><sub><i>uv</i></sub>.</p><p>Once the reference trial was determined, the components in each of the other trials were reordered to match the reference, as dictated by the optimal one-to-one matching computed using the Hungarian algorithm. The polarity of the component was potentially flipped to ensure consistency with the reference trial. The reordered de-mixing matrices <i>A</i><sub>1</sub>,<i>A</i><sub>2</sub>,…<i>A</i><sub>10</sub> were then averaged across trials to yield a single de-mixing matrix that represented a consensus across the ten trials. The independent components <i>S</i><sub>1</sub>,<i>S</i><sub>2</sub>,…,<i>S</i><sub>10</sub> were reordered and then averaged in the same way.</p><p>Due to the effects of bootstrapping and randomization, the de-mixing matrix was not necessarily perfectly orthogonal, and some components were minimally correlated. The Pearson correlation coefficient was computed between all pairs of components in the de-mixing matrix. The reciprocal of the largest correlation coefficient was used to quantify the extent of interdependence between components such that higher values of the reciprocal were desired. This was performed for each of the reordered de-mixing matrices <i>A</i><sub>1</sub>,<i>A</i><sub>2</sub>,…,<i>A</i><sub>10</sub>, and the average across trials resulted in a consensus index of the independence.</p><p>Three criteria were therefore used to evaluate ICA performance and to assist in determining the best number of components to estimate:</p><ol class="u-list-style-none">
                    <li>
                      <span class="u-custom-list-number">1.</span>
                      
                        <p>Cluster separation: the extent of separation between clusters among independent components found across bootstrapped samples. This was estimated internally as part of the Icasso package and provides an estimate of reliability and stability of independent components.</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">2.</span>
                      
                        <p>Trial matching quality: the quality of the one-to-one matching of independent components across ICA trials. This quantified the reproducibility of independent components and was measured as the average distance between matching independent components of each trial and the reference trial.</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">3.</span>
                      
                        <p>Independence index: the independence of components in the de-mixing matrix. This was quantified as the reciprocal of the largest correlation coefficient between all pairs of components.</p>
                      
                    </li>
                  </ol><p>The ICA procedure described above, including bootstrap resampling and trial matching, was separately repeated for candidate models ranging from 3 to 30 independent components. The optimal model (that is, the number of components) was informed by plotting the above measures of cluster separation, trial matching quality and independence index as a function of the number of components and identifying abrupt changes based on the elbow criterion (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">24</a>). While both cluster separation (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">24a</a>) and trial matching quality (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">24b</a>) showed abrupt reductions when moving from six to seven components, which suggests that six components was optimal, the independence index (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">24c</a>) suggested that five components was most suitable. We could therefore narrow down model selection from 28 candidates to only 2 candidate models: 5 or 6 independent components. To break the tie between these two models, the independent components constituting both models were visualized as word clouds, with the font size of each behavior scaled according to its de-mixing weight. Although, the five-component and six-component models were highly comparable, we selected the five-component model because it was the simpler model and enabled clear labeling of each independent component. The five independent components characterized cognitive performance, illicit substance use, tobacco use, personality and emotional traits, as well as mental health. Hence, the 109 behavioral items initially selected were decomposed into five independent components, which we refer to as behavioral dimensions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig8">8a,b</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">21</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec38">NBS analysis</h4><p>The NBS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. T. Network-based statistic: identifying differences in brain networks. NeuroImage 53, 1197–1207 (2010)." href="/articles/s41593-020-00711-6#ref-CR28" id="ref-link-section-d27595475e3628">28</a></sup> was used to test whether functional connectivity mapped within each parcellation scale (scales I–IV) significantly associated with any of the five behavioral dimensions. The NBS was applied independently to each dimension and parcellation scale. For each pair of regions, a general linear model was formulated in which functional connectivity was the dependent variable and the independent variables included an intercept term, the behavioral dimension (that is, the relevant column from the estimated independent sources) and confounds including head motion parameters summarized by the framewise displacement<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Power, J. D., Barnes, K. A., Snyder, A. Z., Schlaggar, B. L. &amp; Petersen, S. E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. NeuroImage 59, 2142–2154 (2012)." href="/articles/s41593-020-00711-6#ref-CR59" id="ref-link-section-d27595475e3632">59</a></sup> and total subcortical volume estimated by FreeSurfer (<a href="https://surfer.nmr.mgh.harvard.edu/">https://surfer.nmr.mgh.harvard.edu/</a>), as provided by the HCP. Given that age and sex were initially regressed from all behavioral scores before performing ICA, these potential confounds were not included in the statistical model. Each connection was endowed a <i>t</i>-statistic and a corresponding uncorrected <i>P</i> value for the null hypothesis that the association between functional connectivity and the behavioral dimension was not significantly different from zero. The premise of the NBS is to reject the null hypothesis for connected graph components<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. T. Network-based statistic: identifying differences in brain networks. NeuroImage 53, 1197–1207 (2010)." href="/articles/s41593-020-00711-6#ref-CR28" id="ref-link-section-d27595475e3650">28</a></sup> rather than for individual connections, thereby relinquishing some precision in effect localization for a gain in statistical power. To this end, connected components were identified among the set of connections with a <i>t</i>-statistic exceeding 4. This <i>t</i>-statistic threshold was chosen to ensure that the smallest effect size of interest that could be detected was approximately a Cohen’s <i>d</i> of 0.1. The number of suprathreshold connections constituting each component defined the size of the component. Permutation testing was used to generate an empirical null distribution for component size. For each of the 10,000 permutations, an established method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Freedman, D. &amp; Lane, D. A nonstochastic interpretation of reported significance levels. J. Bus. Economic Stat. 1, 292–298 (1983)." href="/articles/s41593-020-00711-6#ref-CR98" id="ref-link-section-d27595475e3663">98</a></sup> was used to refit the general linear model to data in which the correspondence between individuals and behaviors was permuted. Permutation sequences were obtained using multilevel block permutation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Winkler, A. M., Ridgway, G. R., Webster, M. A., Smith, S. M. &amp; Nichols, T. E. Permutation inference for the general linear model. NeuroImage 92, 381–397 (2014)." href="/articles/s41593-020-00711-6#ref-CR51" id="ref-link-section-d27595475e3667">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Winkler, A. M., Webster, M. A., Vidaurre, D., Nichols, T. E. &amp; Smith, S. M. Multi-level block permutation. NeuroImage 123, 253–268 (2015)." href="/articles/s41593-020-00711-6#ref-CR52" id="ref-link-section-d27595475e3670">52</a></sup> to ensure that the data were shuffled in a way that respected known familial relationships among individuals. In particular, each family could only be shuffled as a whole, and family members could only be permuted within their family. Dizygotic twins and non-twin siblings were assumed to share the same kinship and they could therefore be permuted. The size of the largest connected component was stored for each permutation to ensure control of the family-wise error rate across all components identified. The family-wise-error-corrected <i>P</i> value for an observed component was given by the proportion of permutations for which the largest component identified was larger than or equal in size to the observed component. The NBS was repeated independently for each of the five behavioral dimensions and for the two alternative hypotheses of positive and negative correlation. This resulted in a total of ten multiple comparisons. The FDR was controlled at 5% across these multiple comparisons.</p><p>The NBS was performed on functional connectivity mapped using 3T-rfMRI data from the second acquisition (REST2) given that the new atlas was delineated using the first acquisition (REST1). The reproducibility of any significant associations identified with the NBS in REST2 were evaluated using functional connectivity mapped using 3T-rfMRI data from REST1. To enable comparisons, functional connectivity mapped with an alternative subcortex atlas (Subcortex-II)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Fan, L. et al. The Human Brainnetome Atlas: a new brain atlas based on connectional architecture. Cereb. Cortex 26, 3508–3526 (2016)." href="/articles/s41593-020-00711-6#ref-CR33" id="ref-link-section-d27595475e3681">33</a></sup> was also tested for associations with behavioral dimensions following the same procedure described above.</p><h3 class="c-article__sub-heading" id="Sec39">Statistical analysis</h3><p>Null hypothesis tests were used to perform statistical analyses. Statistical significance was determined using permutation tests and resampling. Specifically, for atlas delineation, the null hypothesis was that peaks in the gradient magnitude images were consistent with the geometry of the subcortex and/or fMRI confounds. The alternative hypothesis was that the peaks were larger than attributable to subcortical geometry and confounds. Each rejection of this null hypothesis provided evidence of a discrete boundary in subcortical topography. A dedicated geometry-preserving null model was developed to perform inference on the local maxima of the gradient magnitude images (see “Null model”). This facilitated the generation of an empirical null distribution for peak magnitudes by repeatedly drawing random samples from the null model. The <i>P</i> value for each peak was estimated as the proportion of samples constituting the null distribution with peaks that equalled or exceeded the magnitude of the actual peak (see “Boundary delineation”). The FDR was controlled at 5% across the set of all peaks identified at each scale using the Benjamini–Hochberg procedure. For the behavioral analyses, the null hypothesis was that the five behavioral dimensions did not explain significant individual variation in subcortical functional connectivity. This null hypothesis was tested using linear regression analysis and the NBS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. T. Network-based statistic: identifying differences in brain networks. NeuroImage 53, 1197–1207 (2010)." href="/articles/s41593-020-00711-6#ref-CR28" id="ref-link-section-d27595475e3697">28</a></sup>. All null hypothesis tests were nonparametric and did not mandate strong assumptions about data normality.</p><h3 class="c-article__sub-heading" id="Sec40">Further methodological considerations</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec41">Connectivity gradients and subcortical geometry</h4><p>A clear peak in the diversity curve separates the hippocampus and the thalamus (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2d</a>). However, the transition between the hippocampus and the thalamus is narrow and represents a local constriction in the geometry of the subcortex. Because our resampling preserves the geometry of the empirical data, a relatively abrupt transition is also present between the thalamus and the hippocampus in the null data. In other words, the functional separation between the hippocampus and the thalamus can be largely explained by the intrinsic geometry of the subcortex. This does not imply that the hippocampus and the thalamus are not functionally separated, but simply suggests that the functional separation is consistent with the underlying geometry of the two structures and does not require an additional disruption in the functional connectivity gradients. Conversely, there are occasional peaks in the null diversity curves that are not present in the real data (for example, the broad peak within the caudate shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00711-6#Fig2">2c</a>). This presumably arises when the local functional connectivity gradient is smoother (more homogenous) in the real data across a geometric feature than predicted by the null data.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec42">Multiscale organization and model selection</h4><p>Multiscale (hierarchical) organization is not necessarily a fait accompli of the recursive application of model selection. Indeed, we explicitly tested for the possibility of an absence of multiscale structure for each region comprising scale I, but found that the null hypothesis could be rejected for all regions, which is suggestive of a multiscale organizational structure. If we were unable to reject the null hypothesis, this would have instead suggested an absence of any multiscale structure. Furthermore, the number of scales substantially differed between subcortical nuclei. For example, the thalamus, the hippocampus and the caudate each comprised four levels, which suggests a deeper and more complex multiscale structure relative to the amygdala and the GP, which only comprised two levels. This is consistent with the greater organizational complexity of the thalamic nuclei, from molecular<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Phillips, J. W. et al. A repeated molecular architecture across thalamic pathways. Nat. Neurosci. 22, 1925–1935 (2019)." href="/articles/s41593-020-00711-6#ref-CR99" id="ref-link-section-d27595475e3727">99</a></sup> and cellular<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Morel, A., Magnin, M. &amp; Jeanmonod, D. Multiarchitectonic and stereotactic atlas of the human thalamus. J. Comp. Neurol. 387, 588–630 (1997)." href="/articles/s41593-020-00711-6#ref-CR39" id="ref-link-section-d27595475e3731">39</a></sup> to connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Müller, E. et al. Core and matrix thalamic sub-populations relate to spatio-temporal cortical connectivity gradients. NeuroImage 222, 117224 (2020)." href="/articles/s41593-020-00711-6#ref-CR20" id="ref-link-section-d27595475e3735">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Behrens, T. E. et al. Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging. Nat. Neurosci. 6, 750–757 (2003)." href="/articles/s41593-020-00711-6#ref-CR37" id="ref-link-section-d27595475e3738">37</a></sup> and function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Hwang, K., Bertolero, M. A., Liu, W. B. &amp; D’Esposito, M. The human thalamus is an integrative hub for functional brain networks. J. Neurosci. 37, 5594–5607 (2017)." href="/articles/s41593-020-00711-6#ref-CR100" id="ref-link-section-d27595475e3742">100</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 101" title="Shine, J. M. et al. The low-dimensional neural architecture of cognitive complexity is related to activity in medial thalamic nuclei. Neuron 104, 849–855.e3 (2019)." href="/articles/s41593-020-00711-6#ref-CR101" id="ref-link-section-d27595475e3745">101</a></sup>. In contrast, the GP is primarily involved in the regulation of movement control<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="Mink, J. W. The basal ganglia: focused selection and inhibition of competing motor programs. Prog. Neurobiol. 50, 381–425 (1996)." href="/articles/s41593-020-00711-6#ref-CR102" id="ref-link-section-d27595475e3749">102</a></sup> and the amygdala plays an essential role in emotional processes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Janak, P. H. &amp; Tye, K. M. From circuits to behaviour in the amygdala. Nature 517, 284–292 (2015)." href="/articles/s41593-020-00711-6#ref-CR103" id="ref-link-section-d27595475e3754">103</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec43">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                </div>
            

            <div>
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>Neuroimaging data analyses were undertaken using publicly available human neuroimaging datasets acquired and maintained by the HCP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Van Essen, D. C. et al. The WU–Minn Human Connectome Project: an overview. NeuroImage 80, 62–79 (2013)." href="/articles/s41593-020-00711-6#ref-CR18" id="ref-link-section-d27595475e3849">18</a></sup>. These datasets are available for download to anyone agreeing to the Open Access Data Use Terms (<a href="https://db.humanconnectome.org/">https://db.humanconnectome.org/</a>). Access to family structure data and several behavioral measures requires acceptance of the HCP Restricted Data Use Terms (<a href="https://www.humanconnectome.org/study/hcp-young-adult/document/restricted-data-usage">https://www.humanconnectome.org/study/hcp-young-adult/document/restricted-data-usage</a>). The independent validation dataset used to assess the reproducibility of the parcellation homogeneity results is currently not publicly available. The new atlas is openly available in the form of NIFTI (Neuroimaging Informatics Technology Initiative) and CIFTI (Connectivity Informatics Technology Initiative) files. To facilitate mapping of whole-brain connectomes, we have also integrated the new atlas into several well-known cortex-only parcellation atlases and the combined cortex–subcortex atlases are made openly available. Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00711-6#MOESM1">5</a> provides details about the atlas formats that can be downloaded from the GitHub repository (<a href="https://github.com/yetianmed/subcortex">https://github.com/yetianmed/subcortex</a>).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>The Matlab (R2018b) codes to compute Laplacian eigenmaps, gradient magnitudes, diversity curves and other computational analyses undertaken as part of this study are openly available at <a href="https://github.com/yetianmed/subcortex">https://github.com/yetianmed/subcortex</a>. The following additional software packages used for this study are freely and openly available: Diffusion Toolkit (v.0.6.4.1) and TrackVis (v.0.6.1): <a href="http://trackvis.org/">http://trackvis.org/</a>; NBS (v.1.2): <a href="https://www.nitrc.org/projects/nbs/">https://www.nitrc.org/projects/nbs/</a>; Icasso (v.1.21): <a href="https://research.ics.aalto.fi/ica/icasso/">https://research.ics.aalto.fi/ica/icasso/</a>; NeuroMArVL: <a href="https://immersive.erc.monash.edu/neuromarvl/">https://immersive.erc.monash.edu/neuromarvl/</a>; PALM (v.alpha116): <a href="https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PALM/ExchangeabilityBlocks">https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/PALM/ExchangeabilityBlocks</a>; and fMRIPrep (v.1.5.9): <a href="https://fmriprep.org/en/stable/index.html">https://fmriprep.org/en/stable/index.html</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. <i>Nat. Rev. Neurosci.</i> <b>19</b>, 672–686 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFygurjE" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30305712" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Imaging-based%20parcellations%20of%20the%20human%20brain&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=19&amp;pages=672-686&amp;publication_year=2018&amp;author=Eickhoff%2CSB&amp;author=Yeo%2CBTT&amp;author=Genon%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. <i>Nature</i> <b>536</b>, 171–178 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1GrurnK" aria-label="CAS reference 2">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27437579" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990127" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-modal%20parcellation%20of%20human%20cerebral%20cortex&amp;journal=Nature&amp;volume=536&amp;pages=171-178&amp;publication_year=2016&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Yeo, B. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. <i>J. Neurophysiol.</i> <b>106</b>, 1125–1165 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21653723" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20organization%20of%20the%20human%20cerebral%20cortex%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=J.%20Neurophysiol.&amp;volume=106&amp;pages=1125-1165&amp;publication_year=2011&amp;author=Yeo%2CBT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Shepherd, G. M. Corticostriatal connectivity and its role in disease. <i>Nat. Rev. Neurosci.</i> <b>14</b>, 278–291 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXktFOksrk%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23511908" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4096337" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Corticostriatal%20connectivity%20and%20its%20role%20in%20disease&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=14&amp;pages=278-291&amp;publication_year=2013&amp;author=Shepherd%2CGM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Choi, E. Y., Yeo, B. T. &amp; Buckner, R. L. The organization of the human striatum estimated by intrinsic functional connectivity. <i>J. Neurophysiol.</i> <b>108</b>, 2242–2263 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22832566" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3545026" aria-label="PubMed Central reference 5">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20organization%20of%20the%20human%20striatum%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=J.%20Neurophysiol.&amp;volume=108&amp;pages=2242-2263&amp;publication_year=2012&amp;author=Choi%2CEY&amp;author=Yeo%2CBT&amp;author=Buckner%2CRL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Ji, J. L. et al. Mapping the human brain’s cortical–subcortical functional network organization. <i>NeuroImage</i> <b>185</b>, 35–57 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30291974" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20the%20human%20brain%E2%80%99s%20cortical%E2%80%93subcortical%20functional%20network%20organization&amp;journal=NeuroImage&amp;volume=185&amp;pages=35-57&amp;publication_year=2019&amp;author=Ji%2CJL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Cohen, A. L. et al. Defining functional areas in individual human brains using resting functional connectivity MRI. <i>NeuroImage</i> <b>41</b>, 45–57 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18367410" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Defining%20functional%20areas%20in%20individual%20human%20brains%20using%20resting%20functional%20connectivity%20MRI&amp;journal=NeuroImage&amp;volume=41&amp;pages=45-57&amp;publication_year=2008&amp;author=Cohen%2CAL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. <i>Cereb. Cortex</i> <b>26</b>, 288–303 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25316338" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20and%20evaluation%20of%20a%20cortical%20area%20parcellation%20from%20resting-state%20correlations&amp;journal=Cereb.%20Cortex&amp;volume=26&amp;pages=288-303&amp;publication_year=2016&amp;author=Gordon%2CEM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. <i>Cereb. Cortex</i> <b>28</b>, 3095–3114 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28981612" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Local%E2%80%93global%20parcellation%20of%20the%20human%20cerebral%20cortex%20from%20intrinsic%20functional%20connectivity%20MRI&amp;journal=Cereb.%20Cortex&amp;volume=28&amp;pages=3095-3114&amp;publication_year=2018&amp;author=Schaefer%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Huntenburg, J. M., Bazin, P. L. &amp; Margulies, D. S. Large-scale gradients in human cortical organization. <i>Trends Cogn. Sci.</i> <b>22</b>, 21–31 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29203085" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20gradients%20in%20human%20cortical%20organization&amp;journal=Trends%20Cogn.%20Sci.&amp;volume=22&amp;pages=21-31&amp;publication_year=2018&amp;author=Huntenburg%2CJM&amp;author=Bazin%2CPL&amp;author=Margulies%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Haak, K. V., Marquand, A. F. &amp; Beckmann, C. F. Connectopic mapping with resting-state fMRI. <i>NeuroImage</i> <b>170</b>, 83–94 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28666880" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Connectopic%20mapping%20with%20resting-state%20fMRI&amp;journal=NeuroImage&amp;volume=170&amp;pages=83-94&amp;publication_year=2018&amp;author=Haak%2CKV&amp;author=Marquand%2CAF&amp;author=Beckmann%2CCF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, 12574–12579 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ykurnM" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27791099" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5098630" aria-label="PubMed Central reference 12">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Situating%20the%20default-mode%20network%20along%20a%20principal%20gradient%20of%20macroscale%20cortical%20organization&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=113&amp;pages=12574-12579&amp;publication_year=2016&amp;author=Margulies%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Guell, X., Schmahmann, J. D., Gabrieli, J. &amp; Ghosh, S. S. Functional gradients of the cerebellum. <i>eLife</i> <b>7</b>, e36652 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30106371" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6092123" aria-label="PubMed Central reference 13">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20gradients%20of%20the%20cerebellum&amp;journal=eLife&amp;volume=7&amp;publication_year=2018&amp;author=Guell%2CX&amp;author=Schmahmann%2CJD&amp;author=Gabrieli%2CJ&amp;author=Ghosh%2CSS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">King, M., Hernandez-Castillo, C. R., Poldrack, R. A., Ivry, R. B. &amp; Diedrichsen, J. Functional boundaries in the human cerebellum revealed by a multi-domain task battery. <i>Nat. Neurosci.</i> <b>22</b>, 1371–1378 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtleis7nP" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31285616" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8312478" aria-label="PubMed Central reference 14">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20boundaries%20in%20the%20human%20cerebellum%20revealed%20by%20a%20multi-domain%20task%20battery&amp;journal=Nat.%20Neurosci.&amp;volume=22&amp;pages=1371-1378&amp;publication_year=2019&amp;author=King%2CM&amp;author=Hernandez-Castillo%2CCR&amp;author=Poldrack%2CRA&amp;author=Ivry%2CRB&amp;author=Diedrichsen%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Tian, Y. &amp; Zalesky, A. Characterizing the functional connectivity diversity of the insula cortex: subregions, diversity curves and behavior. <i>NeuroImage</i> <b>183</b>, 716–733 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1GqtrnF" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30172005" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterizing%20the%20functional%20connectivity%20diversity%20of%20the%20insula%20cortex%3A%20subregions%2C%20diversity%20curves%20and%20behavior&amp;journal=NeuroImage&amp;volume=183&amp;pages=716-733&amp;publication_year=2018&amp;author=Tian%2CY&amp;author=Zalesky%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Coalson, T. S., Van Essen, D. C. &amp; Glasser, M. F. The impact of traditional neuroimaging methods on the spatial localization of cortical areas. <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, E6356–E6365 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFKhtrnP" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29925602" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6142239" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20traditional%20neuroimaging%20methods%20on%20the%20spatial%20localization%20of%20cortical%20areas&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=115&amp;pages=E6356-E6365&amp;publication_year=2018&amp;author=Coalson%2CTS&amp;author=Essen%2CDC&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Fox, M. D. et al. The human brain is intrinsically organized into dynamic, anticorrelated functional networks. <i>Proc. Natl Acad. Sci. USA</i> <b>102</b>, 9673–9678 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXmsVaktb8%3D" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15976020" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1157105" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20brain%20is%20intrinsically%20organized%20into%20dynamic%2C%20anticorrelated%20functional%20networks&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=102&amp;pages=9673-9678&amp;publication_year=2005&amp;author=Fox%2CMD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Van Essen, D. C. et al. The WU–Minn Human Connectome Project: an overview. <i>NeuroImage</i> <b>80</b>, 62–79 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23684880" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20WU%E2%80%93Minn%20Human%20Connectome%20Project%3A%20an%20overview&amp;journal=NeuroImage&amp;volume=80&amp;pages=62-79&amp;publication_year=2013&amp;author=Essen%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. <i>Nat. Neurosci.</i> <b>18</b>, 1664 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1ChurfI" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26457551" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5008686" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectome%20fingerprinting%3A%20identifying%20individuals%20using%20patterns%20of%20brain%20connectivity&amp;journal=Nat.%20Neurosci.&amp;volume=18&amp;publication_year=2015&amp;author=Finn%2CES">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Müller, E. et al. Core and matrix thalamic sub-populations relate to spatio-temporal cortical connectivity gradients. <i>NeuroImage</i> <b>222</b>, 117224 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32795658" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Core%20and%20matrix%20thalamic%20sub-populations%20relate%20to%20spatio-temporal%20cortical%20connectivity%20gradients&amp;journal=NeuroImage&amp;volume=222&amp;publication_year=2020&amp;author=M%C3%BCller%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Poppenk, J., Evensmoen, H. R., Moscovitch, M. &amp; Nadel, L. Long-axis specialization of the human hippocampus. <i>Trends Cogn. Sci.</i> <b>17</b>, 230–240 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23597720" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-axis%20specialization%20of%20the%20human%20hippocampus&amp;journal=Trends%20Cogn.%20Sci.&amp;volume=17&amp;pages=230-240&amp;publication_year=2013&amp;author=Poppenk%2CJ&amp;author=Evensmoen%2CHR&amp;author=Moscovitch%2CM&amp;author=Nadel%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Amunts, K. et al. Cytoarchitectonic mapping of the human amygdala, hippocampal region and entorhinal cortex: intersubject variability and probability maps. <i>Anat. Embryol.</i> <b>210</b>, 343–352 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD2MnkvVKqsw%3D%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Cytoarchitectonic%20mapping%20of%20the%20human%20amygdala%2C%20hippocampal%20region%20and%20entorhinal%20cortex%3A%20intersubject%20variability%20and%20probability%20maps&amp;journal=Anat.%20Embryol.&amp;volume=210&amp;pages=343-352&amp;publication_year=2005&amp;author=Amunts%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. <i>Nat. Hum. Behav.</i> <b>1</b>, 0146 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28804783" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5549843" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20corticostriatal%20connection%20topographies%20predict%20goal%20directed%20behaviour%20in%20humans&amp;journal=Nat.%20Hum.%20Behav.&amp;volume=1&amp;publication_year=2017&amp;author=Marquand%2CAF&amp;author=Haak%2CKV&amp;author=Beckmann%2CCF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Vu, A. et al. Tradeoffs in pushing the spatial resolution of fMRI for the 7T Human Connectome Project. <i>NeuroImage</i> <b>154</b>, 23–32 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Tradeoffs%20in%20pushing%20the%20spatial%20resolution%20of%20fMRI%20for%20the%207T%20Human%20Connectome%20Project&amp;journal=NeuroImage&amp;volume=154&amp;pages=23-32&amp;publication_year=2017&amp;author=Vu%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Gordon, E. M. et al. Precision functional mapping of individual human brains. <i>Neuron</i> <b>95</b>, 791–807.e7 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1CmsLjF" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28757305" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5576360" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Precision%20functional%20mapping%20of%20individual%20human%20brains&amp;journal=Neuron&amp;volume=95&amp;pages=791-807.e7&amp;publication_year=2017&amp;author=Gordon%2CEM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Kong, R. et al. Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion. <i>Cereb. Cortex</i> <b>29</b>, 2533–2551 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29878084" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20topography%20of%20individual-specific%20cortical%20networks%20predicts%20human%20cognition%2C%20personality%2C%20and%20emotion&amp;journal=Cereb.%20Cortex&amp;volume=29&amp;pages=2533-2551&amp;publication_year=2019&amp;author=Kong%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Salehi, M. et al. There is no single functional atlas even for a single individual: functional parcel definitions change with task. <i>NeuroImage</i> <b>208</b>, 116366 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740342" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=There%20is%20no%20single%20functional%20atlas%20even%20for%20a%20single%20individual%3A%20functional%20parcel%20definitions%20change%20with%20task&amp;journal=NeuroImage&amp;volume=208&amp;publication_year=2020&amp;author=Salehi%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Zalesky, A., Fornito, A. &amp; Bullmore, E. T. Network-based statistic: identifying differences in brain networks. <i>NeuroImage</i> <b>53</b>, 1197–1207 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20600983" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Network-based%20statistic%3A%20identifying%20differences%20in%20brain%20networks&amp;journal=NeuroImage&amp;volume=53&amp;pages=1197-1207&amp;publication_year=2010&amp;author=Zalesky%2CA&amp;author=Fornito%2CA&amp;author=Bullmore%2CET">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Maia, T. V. &amp; Frank, M. J. From reinforcement learning models to psychiatric and neurological disorders. <i>Nat. Neurosci.</i> <b>14</b>, 154–162 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtFShs70%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21270784" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4408000" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=From%20reinforcement%20learning%20models%20to%20psychiatric%20and%20neurological%20disorders&amp;journal=Nat.%20Neurosci.&amp;volume=14&amp;pages=154-162&amp;publication_year=2011&amp;author=Maia%2CTV&amp;author=Frank%2CMJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Forstmann, B. U., de Hollander, G., van Maanen, L., Alkemade, A. &amp; Keuken, M. C. Towards a mechanistic understanding of the human subcortex. <i>Nat. Rev. Neurosci.</i> <b>18</b>, 57–65 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27974841" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20a%20mechanistic%20understanding%20of%20the%20human%20subcortex&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=18&amp;pages=57-65&amp;publication_year=2016&amp;author=Forstmann%2CBU&amp;author=Hollander%2CG&amp;author=Maanen%2CL&amp;author=Alkemade%2CA&amp;author=Keuken%2CMC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Vos de Wael, R. et al. Anatomical and microstructural determinants of hippocampal subfield functional connectome embedding. <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, 10154–10159 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30249658" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6176604" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Anatomical%20and%20microstructural%20determinants%20of%20hippocampal%20subfield%20functional%20connectome%20embedding&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=115&amp;pages=10154-10159&amp;publication_year=2018&amp;author=Vos%20de%20Wael%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Churchland, P. S. &amp; Sejnowski, T. J. Perspectives on cognitive neuroscience. <i>Science</i> <b>242</b>, 741–745 (1988).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaL1M%2Fktl2ntg%3D%3D" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=3055294" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Perspectives%20on%20cognitive%20neuroscience&amp;journal=Science&amp;volume=242&amp;pages=741-745&amp;publication_year=1988&amp;author=Churchland%2CPS&amp;author=Sejnowski%2CTJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Fan, L. et al. The Human Brainnetome Atlas: a new brain atlas based on connectional architecture. <i>Cereb. Cortex</i> <b>26</b>, 3508–3526 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27230218" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4961028" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Human%20Brainnetome%20Atlas%3A%20a%20new%20brain%20atlas%20based%20on%20connectional%20architecture&amp;journal=Cereb.%20Cortex&amp;volume=26&amp;pages=3508-3526&amp;publication_year=2016&amp;author=Fan%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Pauli, W. M., Nili, A. N. &amp; Tyszka, J. M. A high-resolution probabilistic in vivo atlas of human subcortical brain nuclei. <i>Sci. Data</i> <b>5</b>, 180063 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29664465" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5903366" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20high-resolution%20probabilistic%20in%20vivo%20atlas%20of%20human%20subcortical%20brain%20nuclei&amp;journal=Sci.%20Data&amp;volume=5&amp;publication_year=2018&amp;author=Pauli%2CWM&amp;author=Nili%2CAN&amp;author=Tyszka%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Plachti, A. et al. Multimodal parcellations and extensive behavioral profiling tackling the hippocampus gradient. <i>Cereb. Cortex</i> <b>29</b>, 4595–4612 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30721944" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6917521" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Multimodal%20parcellations%20and%20extensive%20behavioral%20profiling%20tackling%20the%20hippocampus%20gradient&amp;journal=Cereb.%20Cortex&amp;volume=29&amp;pages=4595-4612&amp;publication_year=2019&amp;author=Plachti%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Janssen, R. J., Jylänki, P., Kessels, R. P. C. &amp; van Gerven, M. A. J. Probabilistic model-based functional parcellation reveals a robust, fine-grained subdivision of the striatum. <i>NeuroImage</i> <b>119</b>, 398–405 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC28%2FitlegtQ%3D%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26163800" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Probabilistic%20model-based%20functional%20parcellation%20reveals%20a%20robust%2C%20fine-grained%20subdivision%20of%20the%20striatum&amp;journal=NeuroImage&amp;volume=119&amp;pages=398-405&amp;publication_year=2015&amp;author=Janssen%2CRJ&amp;author=Jyl%C3%A4nki%2CP&amp;author=Kessels%2CRPC&amp;author=Gerven%2CMAJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Behrens, T. E. et al. Non-invasive mapping of connections between human thalamus and cortex using diffusion imaging. <i>Nat. Neurosci.</i> <b>6</b>, 750–757 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3sXkvVKmtLY%3D" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12808459" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Non-invasive%20mapping%20of%20connections%20between%20human%20thalamus%20and%20cortex%20using%20diffusion%20imaging&amp;journal=Nat.%20Neurosci.&amp;volume=6&amp;pages=750-757&amp;publication_year=2003&amp;author=Behrens%2CTE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Haber, S. N. &amp; Knutson, B. The reward circuit: linking primate anatomy and human imaging. <i>Neuropsychopharmacology</i> <b>35</b>, 4–26 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19812543" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20reward%20circuit%3A%20linking%20primate%20anatomy%20and%20human%20imaging&amp;journal=Neuropsychopharmacology&amp;volume=35&amp;pages=4-26&amp;publication_year=2010&amp;author=Haber%2CSN&amp;author=Knutson%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Morel, A., Magnin, M. &amp; Jeanmonod, D. Multiarchitectonic and stereotactic atlas of the human thalamus. <i>J. Comp. Neurol.</i> <b>387</b>, 588–630 (1997).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1c%2FjvVymsA%3D%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9373015" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiarchitectonic%20and%20stereotactic%20atlas%20of%20the%20human%20thalamus&amp;journal=J.%20Comp.%20Neurol.&amp;volume=387&amp;pages=588-630&amp;publication_year=1997&amp;author=Morel%2CA&amp;author=Magnin%2CM&amp;author=Jeanmonod%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Meredith, G. E., Pattiselanno, A., Groenewegen, H. J. &amp; Haber, S. N. Shell and core in monkey and human nucleus accumbens identified with antibodies to calbindin-D28k. <i>J. Comp. Neurol.</i> <b>365</b>, 628–639 (1996).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28XhslyltLc%3D" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8742307" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Shell%20and%20core%20in%20monkey%20and%20human%20nucleus%20accumbens%20identified%20with%20antibodies%20to%20calbindin-D28k&amp;journal=J.%20Comp.%20Neurol.&amp;volume=365&amp;pages=628-639&amp;publication_year=1996&amp;author=Meredith%2CGE&amp;author=Pattiselanno%2CA&amp;author=Groenewegen%2CHJ&amp;author=Haber%2CSN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Cocchi, L. &amp; Zalesky, A. Personalized transcranial magnetic stimulation in psychiatry. <i>Biol. Psychiatry Cogn. Neurosci. Neuroimaging</i> <b>3</b>, 731–741 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29571586" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Personalized%20transcranial%20magnetic%20stimulation%20in%20psychiatry&amp;journal=Biol.%20Psychiatry%20Cogn.%20Neurosci.%20Neuroimaging&amp;volume=3&amp;pages=731-741&amp;publication_year=2018&amp;author=Cocchi%2CL&amp;author=Zalesky%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Castro, D. C. &amp; Bruchas, M. R. A motivational and neuropeptidergic hub: anatomical and functional diversity within the nucleus accumbens shell. <i>Neuron</i> <b>102</b>, 529–552 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXptFCjtrw%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31071288" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6528838" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20motivational%20and%20neuropeptidergic%20hub%3A%20anatomical%20and%20functional%20diversity%20within%20the%20nucleus%20accumbens%20shell&amp;journal=Neuron&amp;volume=102&amp;pages=529-552&amp;publication_year=2019&amp;author=Castro%2CDC&amp;author=Bruchas%2CMR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Pauli, W. M., O’Reilly, R. C., Yarkoni, T. &amp; Wager, T. D. Regional specialization within the human striatum for diverse psychological functions. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, 1907–1912 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhslWgs70%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26831091" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4763751" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Regional%20specialization%20within%20the%20human%20striatum%20for%20diverse%20psychological%20functions&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=113&amp;pages=1907-1912&amp;publication_year=2016&amp;author=Pauli%2CWM&amp;author=O%E2%80%99Reilly%2CRC&amp;author=Yarkoni%2CT&amp;author=Wager%2CTD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">MacAskill, A. F., Little, J. P., Cassel, J. M. &amp; Carter, A. G. Subcellular connectivity underlies pathway-specific signaling in the nucleus accumbens. <i>Nat. Neurosci.</i> <b>15</b>, 1624–1626 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xhs1CktLjP" aria-label="CAS reference 44">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23143514" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3986679" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Subcellular%20connectivity%20underlies%20pathway-specific%20signaling%20in%20the%20nucleus%20accumbens&amp;journal=Nat.%20Neurosci.&amp;volume=15&amp;pages=1624-1626&amp;publication_year=2012&amp;author=MacAskill%2CAF&amp;author=Little%2CJP&amp;author=Cassel%2CJM&amp;author=Carter%2CAG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Perez, S. M. &amp; Lodge, D. J. Convergent inputs from the hippocampus and thalamus to the nucleus accumbens regulate dopamine neuron activity. <i>J. Neurosci.</i> <b>38</b>, 10607–10618 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXktFCqsrc%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30355626" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6290296" aria-label="PubMed Central reference 45">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Convergent%20inputs%20from%20the%20hippocampus%20and%20thalamus%20to%20the%20nucleus%20accumbens%20regulate%20dopamine%20neuron%20activity&amp;journal=J.%20Neurosci.&amp;volume=38&amp;pages=10607-10618&amp;publication_year=2018&amp;author=Perez%2CSM&amp;author=Lodge%2CDJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Pontieri, F. E., Tanda, G., Orzi, F. &amp; Di Chiara, G. Effects of nicotine on the nucleus accumbens and similarity to those of addictive drugs. <i>Nature</i> <b>382</b>, 255–257 (1996).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28XksVWhur8%3D" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8717040" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20nicotine%20on%20the%20nucleus%20accumbens%20and%20similarity%20to%20those%20of%20addictive%20drugs&amp;journal=Nature&amp;volume=382&amp;pages=255-257&amp;publication_year=1996&amp;author=Pontieri%2CFE&amp;author=Tanda%2CG&amp;author=Orzi%2CF&amp;author=Chiara%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Gallinat, J. et al. Abnormal hippocampal neurochemistry in smokers: evidence from proton magnetic resonance spectroscopy at 3T. <i>J. Clin. Psychopharmacol.</i> <b>27</b>, 80–84 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXlt1ykuw%3D%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17224719" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Abnormal%20hippocampal%20neurochemistry%20in%20smokers%3A%20evidence%20from%20proton%20magnetic%20resonance%20spectroscopy%20at%203T&amp;journal=J.%20Clin.%20Psychopharmacol.&amp;volume=27&amp;pages=80-84&amp;publication_year=2007&amp;author=Gallinat%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Xu, Z., Seidler, F. J., Ali, S. F., Slikker, W. Jr. &amp; Slotkin, T. A. Fetal and adolescent nicotine administration: effects on CNS serotonergic systems. <i>Brain Res.</i> <b>914</b>, 166–178 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXntV2jsL0%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11578609" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Fetal%20and%20adolescent%20nicotine%20administration%3A%20effects%20on%20CNS%20serotonergic%20systems&amp;journal=Brain%20Res.&amp;volume=914&amp;pages=166-178&amp;publication_year=2001&amp;author=Xu%2CZ&amp;author=Seidler%2CFJ&amp;author=Ali%2CSF&amp;author=Slikker%2CW&amp;author=Slotkin%2CTA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">de Leon, J. &amp; Diaz, F. J. A meta-analysis of worldwide studies demonstrates an association between schizophrenia and tobacco smoking behaviors. <i>Schizophrenia Res.</i> <b>76</b>, 135–157 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20meta-analysis%20of%20worldwide%20studies%20demonstrates%20an%20association%20between%20schizophrenia%20and%20tobacco%20smoking%20behaviors&amp;journal=Schizophrenia%20Res.&amp;volume=76&amp;pages=135-157&amp;publication_year=2005&amp;author=Leon%2CJ&amp;author=Diaz%2CFJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Koukouli, F. et al. Nicotine reverses hypofrontality in animal models of addiction and schizophrenia. <i>Nat. Med.</i> <b>23</b>, 347–354 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlGqu7Y%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28112735" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5819879" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Nicotine%20reverses%20hypofrontality%20in%20animal%20models%20of%20addiction%20and%20schizophrenia&amp;journal=Nat.%20Med.&amp;volume=23&amp;pages=347-354&amp;publication_year=2017&amp;author=Koukouli%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Winkler, A. M., Ridgway, G. R., Webster, M. A., Smith, S. M. &amp; Nichols, T. E. Permutation inference for the general linear model. <i>NeuroImage</i> <b>92</b>, 381–397 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24530839" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Permutation%20inference%20for%20the%20general%20linear%20model&amp;journal=NeuroImage&amp;volume=92&amp;pages=381-397&amp;publication_year=2014&amp;author=Winkler%2CAM&amp;author=Ridgway%2CGR&amp;author=Webster%2CMA&amp;author=Smith%2CSM&amp;author=Nichols%2CTE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Winkler, A. M., Webster, M. A., Vidaurre, D., Nichols, T. E. &amp; Smith, S. M. Multi-level block permutation. <i>NeuroImage</i> <b>123</b>, 253–268 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26074200" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-level%20block%20permutation&amp;journal=NeuroImage&amp;volume=123&amp;pages=253-268&amp;publication_year=2015&amp;author=Winkler%2CAM&amp;author=Webster%2CMA&amp;author=Vidaurre%2CD&amp;author=Nichols%2CTE&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Smith, S. M. et al. Resting-state fMRI in the Human Connectome Project. <i>NeuroImage</i> <b>80</b>, 144–168 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23702415" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting-state%20fMRI%20in%20the%20Human%20Connectome%20Project&amp;journal=NeuroImage&amp;volume=80&amp;pages=144-168&amp;publication_year=2013&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Barch, D. M. et al. Function in the human connectome: task-fMRI and individual differences in behavior. <i>NeuroImage</i> <b>80</b>, 169–189 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23684877" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Function%20in%20the%20human%20connectome%3A%20task-fMRI%20and%20individual%20differences%20in%20behavior&amp;journal=NeuroImage&amp;volume=80&amp;pages=169-189&amp;publication_year=2013&amp;author=Barch%2CDM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. <i>NeuroImage</i> <b>80</b>, 105–124 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23668970" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20minimal%20preprocessing%20pipelines%20for%20the%20Human%20Connectome%20Project&amp;journal=NeuroImage&amp;volume=80&amp;pages=105-124&amp;publication_year=2013&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Salimi-Khorshidi, G. et al. Automatic denoising of functional MRI data: combining independent component analysis and hierarchical fusion of classifiers. <i>NeuroImage</i> <b>90</b>, 449–468 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24389422" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Automatic%20denoising%20of%20functional%20MRI%20data%3A%20combining%20independent%20component%20analysis%20and%20hierarchical%20fusion%20of%20classifiers&amp;journal=NeuroImage&amp;volume=90&amp;pages=449-468&amp;publication_year=2014&amp;author=Salimi-Khorshidi%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Griffanti, L. et al. ICA-based artefact removal and accelerated fMRI acquisition for improved resting state network imaging. <i>NeuroImage</i> <b>95</b>, 232–247 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24657355" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=ICA-based%20artefact%20removal%20and%20accelerated%20fMRI%20acquisition%20for%20improved%20resting%20state%20network%20imaging&amp;journal=NeuroImage&amp;volume=95&amp;pages=232-247&amp;publication_year=2014&amp;author=Griffanti%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Parkes, L., Fulcher, B., Yucel, M. &amp; Fornito, A. An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI. <i>NeuroImage</i> <b>171</b>, 415–436 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29278773" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20evaluation%20of%20the%20efficacy%2C%20reliability%2C%20and%20sensitivity%20of%20motion%20correction%20strategies%20for%20resting-state%20functional%20MRI&amp;journal=NeuroImage&amp;volume=171&amp;pages=415-436&amp;publication_year=2018&amp;author=Parkes%2CL&amp;author=Fulcher%2CB&amp;author=Yucel%2CM&amp;author=Fornito%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Power, J. D., Barnes, K. A., Snyder, A. Z., Schlaggar, B. L. &amp; Petersen, S. E. Spurious but systematic correlations in functional connectivity MRI networks arise from subject motion. <i>NeuroImage</i> <b>59</b>, 2142–2154 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22019881" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Spurious%20but%20systematic%20correlations%20in%20functional%20connectivity%20MRI%20networks%20arise%20from%20subject%20motion&amp;journal=NeuroImage&amp;volume=59&amp;pages=2142-2154&amp;publication_year=2012&amp;author=Power%2CJD&amp;author=Barnes%2CKA&amp;author=Snyder%2CAZ&amp;author=Schlaggar%2CBL&amp;author=Petersen%2CSE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Satterthwaite, T. D. et al. An improved framework for confound regression and filtering for control of motion artifact in the preprocessing of resting-state functional connectivity data. <i>NeuroImage</i> <b>64</b>, 240–256 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22926292" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20improved%20framework%20for%20confound%20regression%20and%20filtering%20for%20control%20of%20motion%20artifact%20in%20the%20preprocessing%20of%20resting-state%20functional%20connectivity%20data&amp;journal=NeuroImage&amp;volume=64&amp;pages=240-256&amp;publication_year=2013&amp;author=Satterthwaite%2CTD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Van Dijk, K. R., Sabuncu, M. R. &amp; Buckner, R. L. The influence of head motion on intrinsic functional connectivity MRI. <i>NeuroImage</i> <b>59</b>, 431–438 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21810475" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20influence%20of%20head%20motion%20on%20intrinsic%20functional%20connectivity%20MRI&amp;journal=NeuroImage&amp;volume=59&amp;pages=431-438&amp;publication_year=2012&amp;author=Dijk%2CKR&amp;author=Sabuncu%2CMR&amp;author=Buckner%2CRL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Puckett, A. M. et al. Using multi-echo simultaneous multi-slice (SMS) EPI to improve functional MRI of the subcortical nuclei of the basal ganglia at ultra-high field (7T). <i>NeuroImage</i> <b>172</b>, 886–895 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29208571" aria-label="PubMed reference 62">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20multi-echo%20simultaneous%20multi-slice%20%28SMS%29%20EPI%20to%20improve%20functional%20MRI%20of%20the%20subcortical%20nuclei%20of%20the%20basal%20ganglia%20at%20ultra-high%20field%20%287T%29&amp;journal=NeuroImage&amp;volume=172&amp;pages=886-895&amp;publication_year=2018&amp;author=Puckett%2CAM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Glasser, M. F. et al. The Human Connectome Project’s neuroimaging approach. <i>Nat. Neurosci.</i> <b>19</b>, 1175–1187 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27571196" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6172654" aria-label="PubMed Central reference 63">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Human%20Connectome%20Project%E2%80%99s%20neuroimaging%20approach&amp;journal=Nat.%20Neurosci.&amp;volume=19&amp;pages=1175-1187&amp;publication_year=2016&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Glasser, M. F. et al. Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data. <i>NeuroImage</i> <b>181</b>, 692–717 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29753843" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20temporal%20ICA%20to%20selectively%20remove%20global%20noise%20while%20preserving%20global%20signal%20in%20functional%20MRI%20data&amp;journal=NeuroImage&amp;volume=181&amp;pages=692-717&amp;publication_year=2018&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Cole, M. W. et al. Multi-task connectivity reveals flexible hubs for adaptive task control. <i>Nat. Neurosci.</i> <b>16</b>, 1348–1355 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOhs7bJ" aria-label="CAS reference 65">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23892552" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3758404" aria-label="PubMed Central reference 65">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-task%20connectivity%20reveals%20flexible%20hubs%20for%20adaptive%20task%20control&amp;journal=Nat.%20Neurosci.&amp;volume=16&amp;pages=1348-1355&amp;publication_year=2013&amp;author=Cole%2CMW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Esteban, O. et al. fMRIPrep: a robust preprocessing pipeline for functional MRI. <i>Nat. Methods</i> <b>16</b>, 111–116 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhurnN" aria-label="CAS reference 66">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30532080" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=fMRIPrep%3A%20a%20robust%20preprocessing%20pipeline%20for%20functional%20MRI&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=111-116&amp;publication_year=2019&amp;author=Esteban%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Percheron, G., Yelnik, J. &amp; Francois, C. A Golgi analysis of the primate globus pallidus. III. Spatial organization of the striato–pallidal complex. <i>J. Comp. Neurol.</i> <b>227</b>, 214–227 (1984).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaL2c3ptVSqsA%3D%3D" aria-label="CAS reference 67">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6470214" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Golgi%20analysis%20of%20the%20primate%20globus%20pallidus.%20III.%20Spatial%20organization%20of%20the%20striato%E2%80%93pallidal%20complex&amp;journal=J.%20Comp.%20Neurol.&amp;volume=227&amp;pages=214-227&amp;publication_year=1984&amp;author=Percheron%2CG&amp;author=Yelnik%2CJ&amp;author=Francois%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Yushkevich, P. A. et al. Quantitative comparison of 21 protocols for labeling hippocampal subfields and parahippocampal subregions in in vivo MRI: towards a harmonized segmentation protocol. <i>NeuroImage</i> <b>111</b>, 526–541 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25596463" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantitative%20comparison%20of%2021%20protocols%20for%20labeling%20hippocampal%20subfields%20and%20parahippocampal%20subregions%20in%20in%20vivo%20MRI%3A%20towards%20a%20harmonized%20segmentation%20protocol&amp;journal=NeuroImage&amp;volume=111&amp;pages=526-541&amp;publication_year=2015&amp;author=Yushkevich%2CPA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Guadalupe, T. et al. Human subcortical brain asymmetries in 15,847 people worldwide reveal effects of age and sex. <i>Brain Imaging Behav.</i> <b>11</b>, 1497–1514 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27738994" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20subcortical%20brain%20asymmetries%20in%2015%2C847%20people%20worldwide%20reveal%20effects%20of%20age%20and%20sex&amp;journal=Brain%20Imaging%20Behav.&amp;volume=11&amp;pages=1497-1514&amp;publication_year=2017&amp;author=Guadalupe%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">de Hollander, G., Keuken, M. C., van der Zwaag, W., Forstmann, B. U. &amp; Trampel, R. Comparing functional MRI protocols for small, iron-rich basal ganglia nuclei such as the subthalamic nucleus at 7 T and 3 T. <i>Hum. Brain Mapp.</i> <b>38</b>, 3226–3248 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28345164" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6867009" aria-label="PubMed Central reference 70">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparing%20functional%20MRI%20protocols%20for%20small%2C%20iron-rich%20basal%20ganglia%20nuclei%20such%20as%20the%20subthalamic%20nucleus%20at%207%E2%80%89T%20and%203%E2%80%89T&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=38&amp;pages=3226-3248&amp;publication_year=2017&amp;author=Hollander%2CG&amp;author=Keuken%2CMC&amp;author=Zwaag%2CW&amp;author=Forstmann%2CBU&amp;author=Trampel%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Belkin, M. &amp; Niyogi, P. Laplacian eigenmaps for dimensionality reduction and data representation. <i>Neural Comput.</i> <b>15</b>, 1373–1396 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Laplacian%20eigenmaps%20for%20dimensionality%20reduction%20and%20data%20representation&amp;journal=Neural%20Comput.&amp;volume=15&amp;pages=1373-1396&amp;publication_year=2003&amp;author=Belkin%2CM&amp;author=Niyogi%2CP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Fornito, A., Zalesky, A. &amp; Breakspear, M. Graph analysis of the human connectome: promise, progress, and pitfalls. <i>NeuroImage</i> <b>80</b>, 426–444 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23643999" aria-label="PubMed reference 72">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Graph%20analysis%20of%20the%20human%20connectome%3A%20promise%2C%20progress%2C%20and%20pitfalls&amp;journal=NeuroImage&amp;volume=80&amp;pages=426-444&amp;publication_year=2013&amp;author=Fornito%2CA&amp;author=Zalesky%2CA&amp;author=Breakspear%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Cerliani, L. et al. Probabilistic tractography recovers a rostrocaudal trajectory of connectivity variability in the human insular cortex. <i>Hum. Brain Mapp.</i> <b>33</b>, 2005–2034 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21761507" aria-label="PubMed reference 73">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Probabilistic%20tractography%20recovers%20a%20rostrocaudal%20trajectory%20of%20connectivity%20variability%20in%20the%20human%20insular%20cortex&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=33&amp;pages=2005-2034&amp;publication_year=2012&amp;author=Cerliani%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Jeurissen, B., Descoteaux, M., Mori, S. &amp; Leemans, A. Diffusion MRI fiber tractography of the brain. <i>NMR Biomed.</i> <b>32</b>, e3785 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28945294" aria-label="PubMed reference 74">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Diffusion%20MRI%20fiber%20tractography%20of%20the%20brain&amp;journal=NMR%20Biomed.&amp;volume=32&amp;publication_year=2019&amp;author=Jeurissen%2CB&amp;author=Descoteaux%2CM&amp;author=Mori%2CS&amp;author=Leemans%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Alexander, D. C. in <i>Visualization and Processing of Tensor Fields</i> (eds Weickert J. &amp; Hagen, H.) Ch. 5 (Springer, 2006).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Friman, O., Farneback, G. &amp; Westin, C. F. A Bayesian approach for stochastic white matter tractography. <i>IEEE Trans. Med. Imaging</i> <b>25</b>, 965–978 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16894991" aria-label="PubMed reference 76">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Bayesian%20approach%20for%20stochastic%20white%20matter%20tractography&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;volume=25&amp;pages=965-978&amp;publication_year=2006&amp;author=Friman%2CO&amp;author=Farneback%2CG&amp;author=Westin%2CCF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Peled, S., Friman, O., Jolesz, F. &amp; Westin, C. F. Geometrically constrained two-tensor model for crossing tracts in DWI. <i>Magn. Reson. Imaging</i> <b>24</b>, 1263–1270 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17071347" aria-label="PubMed reference 77">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2000805" aria-label="PubMed Central reference 77">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Geometrically%20constrained%20two-tensor%20model%20for%20crossing%20tracts%20in%20DWI&amp;journal=Magn.%20Reson.%20Imaging&amp;volume=24&amp;pages=1263-1270&amp;publication_year=2006&amp;author=Peled%2CS&amp;author=Friman%2CO&amp;author=Jolesz%2CF&amp;author=Westin%2CCF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Behrens, T. E., Berg, H. J., Jbabdi, S., Rushworth, M. F. &amp; Woolrich, M. W. Probabilistic diffusion tractography with multiple fibre orientations: what can we gain? <i>NeuroImage</i> <b>34</b>, 144–155 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD28nos1WmsQ%3D%3D" aria-label="CAS reference 78">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17070705" aria-label="PubMed reference 78">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Probabilistic%20diffusion%20tractography%20with%20multiple%20fibre%20orientations%3A%20what%20can%20we%20gain%3F&amp;journal=NeuroImage&amp;volume=34&amp;pages=144-155&amp;publication_year=2007&amp;author=Behrens%2CTE&amp;author=Berg%2CHJ&amp;author=Jbabdi%2CS&amp;author=Rushworth%2CMF&amp;author=Woolrich%2CMW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Wang, R., Benner, T., Sorensen, A. &amp; Wedeen, V. Diffusion Toolkit: a software package for diffusion imaging data processing and tractography. <i>Proc. Int. Soc. Magn. Reson. Med.</i> <b>15</b>, 3720 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Diffusion%20Toolkit%3A%20a%20software%20package%20for%20diffusion%20imaging%20data%20processing%20and%20tractography&amp;journal=Proc.%20Int.%20Soc.%20Magn.%20Reson.%20Med.&amp;volume=15&amp;publication_year=2007&amp;author=Wang%2CR&amp;author=Benner%2CT&amp;author=Sorensen%2CA&amp;author=Wedeen%2CV">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">O’Donnell, L. J. &amp; Westin, C. F. Automatic tractography segmentation using a high-dimensional white matter atlas. <i>IEEE Trans. Med. Imaging</i> <b>26</b>, 1562–1575 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18041271" aria-label="PubMed reference 80">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Automatic%20tractography%20segmentation%20using%20a%20high-dimensional%20white%20matter%20atlas&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;volume=26&amp;pages=1562-1575&amp;publication_year=2007&amp;author=O%E2%80%99Donnell%2CLJ&amp;author=Westin%2CCF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Müller, M. in <i>Information Retrieval for Music and Motion</i> 69–84 (Springer, 2007).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Benjamini, Y. Discovering the false discovery rate. <i>J. R. Stat. Soc. Ser. B</i> <b>72</b>, 405–416 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Discovering%20the%20false%20discovery%20rate&amp;journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B&amp;volume=72&amp;pages=405-416&amp;publication_year=2010&amp;author=Benjamini%2CY">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Serrano, M. Á., Boguñá, M. &amp; Vespignani, A. Extracting the multiscale backbone of complex weighted networks. <i>Proc. Natl Acad. Sci. USA</i> <b>106</b>, 6483–6488 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXlsFCrs7k%3D" aria-label="CAS reference 83">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19357301" aria-label="PubMed reference 83">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2672499" aria-label="PubMed Central reference 83">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Extracting%20the%20multiscale%20backbone%20of%20complex%20weighted%20networks&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=6483-6488&amp;publication_year=2009&amp;author=Serrano%2CM%C3%81&amp;author=Bogu%C3%B1%C3%A1%2CM&amp;author=Vespignani%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Meyer, F. Topographic distance and watershed lines. <i>Signal Process.</i> <b>38</b>, 113–125 (1994).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Topographic%20distance%20and%20watershed%20lines&amp;journal=Signal%20Process.&amp;volume=38&amp;pages=113-125&amp;publication_year=1994&amp;author=Meyer%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Eickhoff, S. B. et al. A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. <i>NeuroImage</i> <b>25</b>, 1325–1335 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15850749" aria-label="PubMed reference 85">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20SPM%20toolbox%20for%20combining%20probabilistic%20cytoarchitectonic%20maps%20and%20functional%20imaging%20data&amp;journal=NeuroImage&amp;volume=25&amp;pages=1325-1335&amp;publication_year=2005&amp;author=Eickhoff%2CSB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Lancichinetti, A., Fortunato, S. &amp; Kertész, J. Detecting the overlapping and hierarchical community structure in complex networks. <i>N. J. Phys.</i> <b>11</b>, 033015 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=Detecting%20the%20overlapping%20and%20hierarchical%20community%20structure%20in%20complex%20networks&amp;journal=N.%20J.%20Phys.&amp;volume=11&amp;publication_year=2009&amp;author=Lancichinetti%2CA&amp;author=Fortunato%2CS&amp;author=Kert%C3%A9sz%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Greene, A. S., Gao, S., Scheinost, D. &amp; Constable, R. T. Task-induced brain state manipulation improves prediction of individual traits. <i>Nat. Commun.</i> <b>9</b>, 2807 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30022026" aria-label="PubMed reference 87">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6052101" aria-label="PubMed Central reference 87">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=Task-induced%20brain%20state%20manipulation%20improves%20prediction%20of%20individual%20traits&amp;journal=Nat.%20Commun.&amp;volume=9&amp;publication_year=2018&amp;author=Greene%2CAS&amp;author=Gao%2CS&amp;author=Scheinost%2CD&amp;author=Constable%2CRT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Cole, M. W., Bassett, D. S., Power, J. D., Braver, T. S. &amp; Petersen, S. E. Intrinsic and task-evoked network architectures of the human brain. <i>Neuron</i> <b>83</b>, 238–251 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFWgt7bL" aria-label="CAS reference 88">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24991964" aria-label="PubMed reference 88">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4082806" aria-label="PubMed Central reference 88">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=Intrinsic%20and%20task-evoked%20network%20architectures%20of%20the%20human%20brain&amp;journal=Neuron&amp;volume=83&amp;pages=238-251&amp;publication_year=2014&amp;author=Cole%2CMW&amp;author=Bassett%2CDS&amp;author=Power%2CJD&amp;author=Braver%2CTS&amp;author=Petersen%2CSE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Medaglia, J. D., Lynall, M. E. &amp; Bassett, D. S. Cognitive network neuroscience. <i>J. Cogn. Neurosci.</i> <b>27</b>, 1471–1491 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25803596" aria-label="PubMed reference 89">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4854276" aria-label="PubMed Central reference 89">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=Cognitive%20network%20neuroscience&amp;journal=J.%20Cogn.%20Neurosci.&amp;volume=27&amp;pages=1471-1491&amp;publication_year=2015&amp;author=Medaglia%2CJD&amp;author=Lynall%2CME&amp;author=Bassett%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Smith, S. M. et al. A positive-negative mode of population covariation links brain connectivity, demographics and behavior. <i>Nat. Neurosci.</i> <b>18</b>, 1565–1567 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7vN" aria-label="CAS reference 90">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26414616" aria-label="PubMed reference 90">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4625579" aria-label="PubMed Central reference 90">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20positive-negative%20mode%20of%20population%20covariation%20links%20brain%20connectivity%2C%20demographics%20and%20behavior&amp;journal=Nat.%20Neurosci.&amp;volume=18&amp;pages=1565-1567&amp;publication_year=2015&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Fornito, A., Zalesky, A., Pantelis, C. &amp; Bullmore, E. T. Schizophrenia, neuroimaging and connectomics. <i>NeuroImage</i> <b>62</b>, 2296–2314 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22387165" aria-label="PubMed reference 91">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=Schizophrenia%2C%20neuroimaging%20and%20connectomics&amp;journal=NeuroImage&amp;volume=62&amp;pages=2296-2314&amp;publication_year=2012&amp;author=Fornito%2CA&amp;author=Zalesky%2CA&amp;author=Pantelis%2CC&amp;author=Bullmore%2CET">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Xia, C. H. et al. Linked dimensions of psychopathology and connectivity in functional brain networks. <i>Nat. Commun.</i> <b>9</b>, 3003 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30068943" aria-label="PubMed reference 92">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6070480" aria-label="PubMed Central reference 92">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=Linked%20dimensions%20of%20psychopathology%20and%20connectivity%20in%20functional%20brain%20networks&amp;journal=Nat.%20Commun.&amp;volume=9&amp;publication_year=2018&amp;author=Xia%2CCH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. <i>J. Stat. Mech. Theory Exp.</i> <b>2008</b>, P10008 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20unfolding%20of%20communities%20in%20large%20networks&amp;journal=J.%20Stat.%20Mech.%20Theory%20Exp.&amp;volume=2008&amp;publication_year=2008&amp;author=Blondel%2CVD&amp;author=Guillaume%2CJ-L&amp;author=Lambiotte%2CR&amp;author=Lefebvre%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Van der Waerden, B. Oder tests for two-sample problem and their power. <i>Proc. K. Nederlandse Akademie van Wetenschappen. Ser. A</i> <b>55</b>, 453–458 (1952).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=Oder%20tests%20for%20two-sample%20problem%20and%20their%20power&amp;journal=Proc.%20K.%20Nederlandse%20Akademie%20van%20Wetenschappen.%20Ser.%20A&amp;volume=55&amp;pages=453-458&amp;publication_year=1952&amp;author=Waerden%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Himberg, J., Hyvarinen, A. &amp; Esposito, F. Validating the independent components of neuroimaging time series via clustering and visualization. <i>NeuroImage</i> <b>22</b>, 1214–1222 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15219593" aria-label="PubMed reference 95">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Validating%20the%20independent%20components%20of%20neuroimaging%20time%20series%20via%20clustering%20and%20visualization&amp;journal=NeuroImage&amp;volume=22&amp;pages=1214-1222&amp;publication_year=2004&amp;author=Himberg%2CJ&amp;author=Hyvarinen%2CA&amp;author=Esposito%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Hyvarinen, A. Fast and robust fixed-point algorithms for independent component analysis. <i>IEEE Trans. Neural Netw.</i> <b>10</b>, 626–634 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD1c%2FpsFSmtQ%3D%3D" aria-label="CAS reference 96">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18252563" aria-label="PubMed reference 96">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20robust%20fixed-point%20algorithms%20for%20independent%20component%20analysis&amp;journal=IEEE%20Trans.%20Neural%20Netw.&amp;volume=10&amp;pages=626-634&amp;publication_year=1999&amp;author=Hyvarinen%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Kuhn, H. W. The Hungarian method for the assignment problem. <i>Nav. Res. Logist. Q</i> <b>2</b>, 83–97 (1955).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 97" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Hungarian%20method%20for%20the%20assignment%20problem&amp;journal=Nav.%20Res.%20Logist.%20Q&amp;volume=2&amp;pages=83-97&amp;publication_year=1955&amp;author=Kuhn%2CHW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Freedman, D. &amp; Lane, D. A nonstochastic interpretation of reported significance levels. <i>J. Bus. Economic Stat.</i> <b>1</b>, 292–298 (1983).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20nonstochastic%20interpretation%20of%20reported%20significance%20levels&amp;journal=J.%20Bus.%20Economic%20Stat.&amp;volume=1&amp;pages=292-298&amp;publication_year=1983&amp;author=Freedman%2CD&amp;author=Lane%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Phillips, J. W. et al. A repeated molecular architecture across thalamic pathways. <i>Nat. Neurosci.</i> <b>22</b>, 1925–1935 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslyisbzM" aria-label="CAS reference 99">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31527803" aria-label="PubMed reference 99">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6819258" aria-label="PubMed Central reference 99">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 99" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20repeated%20molecular%20architecture%20across%20thalamic%20pathways&amp;journal=Nat.%20Neurosci.&amp;volume=22&amp;pages=1925-1935&amp;publication_year=2019&amp;author=Phillips%2CJW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100."><p class="c-article-references__text" id="ref-CR100">Hwang, K., Bertolero, M. A., Liu, W. B. &amp; D’Esposito, M. The human thalamus is an integrative hub for functional brain networks. <i>J. Neurosci.</i> <b>37</b>, 5594–5607 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFChtrnP" aria-label="CAS reference 100">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28450543" aria-label="PubMed reference 100">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5469300" aria-label="PubMed Central reference 100">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 100" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20thalamus%20is%20an%20integrative%20hub%20for%20functional%20brain%20networks&amp;journal=J.%20Neurosci.&amp;volume=37&amp;pages=5594-5607&amp;publication_year=2017&amp;author=Hwang%2CK&amp;author=Bertolero%2CMA&amp;author=Liu%2CWB&amp;author=D%E2%80%99Esposito%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101."><p class="c-article-references__text" id="ref-CR101">Shine, J. M. et al. The low-dimensional neural architecture of cognitive complexity is related to activity in medial thalamic nuclei. <i>Neuron</i> <b>104</b>, 849–855.e3 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVaksrzF" aria-label="CAS reference 101">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31653463" aria-label="PubMed reference 101">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 101" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20low-dimensional%20neural%20architecture%20of%20cognitive%20complexity%20is%20related%20to%20activity%20in%20medial%20thalamic%20nuclei&amp;journal=Neuron&amp;volume=104&amp;pages=849-855.e3&amp;publication_year=2019&amp;author=Shine%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102."><p class="c-article-references__text" id="ref-CR102">Mink, J. W. The basal ganglia: focused selection and inhibition of competing motor programs. <i>Prog. Neurobiol.</i> <b>50</b>, 381–425 (1996).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK2s7lvVSksA%3D%3D" aria-label="CAS reference 102">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9004351" aria-label="PubMed reference 102">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 102" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20basal%20ganglia%3A%20focused%20selection%20and%20inhibition%20of%20competing%20motor%20programs&amp;journal=Prog.%20Neurobiol.&amp;volume=50&amp;pages=381-425&amp;publication_year=1996&amp;author=Mink%2CJW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103."><p class="c-article-references__text" id="ref-CR103">Janak, P. H. &amp; Tye, K. M. From circuits to behaviour in the amygdala. <i>Nature</i> <b>517</b>, 284–292 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXpvV2lsg%3D%3D" aria-label="CAS reference 103">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25592533" aria-label="PubMed reference 103">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4565157" aria-label="PubMed Central reference 103">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 103" href="http://scholar.google.com/scholar_lookup?&amp;title=From%20circuits%20to%20behaviour%20in%20the%20amygdala&amp;journal=Nature&amp;volume=517&amp;pages=284-292&amp;publication_year=2015&amp;author=Janak%2CPH&amp;author=Tye%2CKM">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-020-00711-6?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank M. Glasser (Washington University) for provision of the Wishart filter code. We thank L. Cocchi (QIMR Berghofer Medical Research Institute) for provision of the additional validation dataset. All other data were provided by the HCP, the WU–Minn Consortium (1U54MH091657; Principal Investigators: D. Van Essen and K. Ugurbil) funded by the 16 National Institutes of Health (NIH) institutes and centers that support the NIH Blueprint for Neuroscience Research, and by the McDonnell Center for Systems Neuroscience at Washington University. Y.T. was supported by a NHMRC Project Grant awarded to A.Z. (APP1142801). M.B. and A.Z. were each supported by research fellowships from the NHMRC (APP1136649 and APP1118153, respectively).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Melbourne Neuropsychiatry Centre, Department of Psychiatry, The University of Melbourne, Parkville, Victoria, Australia</p><p class="c-article-author-affiliation__authors-list">Ye Tian &amp; Andrew Zalesky</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Centre National de la Recherche Scientifique (CNRS) UMR 8002, Integrative Neuroscience and Cognition Center, Université de Paris, Paris, France</p><p class="c-article-author-affiliation__authors-list">Daniel S. Margulies</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Discipline of Psychiatry, Faculty of Health and Medicine, University of Newcastle, Newcastle, New South Wales, Australia</p><p class="c-article-author-affiliation__authors-list">Michael Breakspear</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">School of Psychology, Faculty of Science, University of Newcastle, Newcastle, New South Wales, Australia</p><p class="c-article-author-affiliation__authors-list">Michael Breakspear</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Department of Biomedical Engineering, Melbourne School of Engineering, The University of Melbourne, Parkville, Victoria, Australia</p><p class="c-article-author-affiliation__authors-list">Andrew Zalesky</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Ye-Tian-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Ye Tian</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ye%20Tian" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ye%20Tian" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ye%20Tian%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Daniel_S_-Margulies-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Daniel S. Margulies</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Daniel%20S.%20Margulies" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Daniel%20S.%20Margulies" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Daniel%20S.%20Margulies%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael-Breakspear-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Michael Breakspear</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20Breakspear" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20Breakspear" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20Breakspear%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Andrew-Zalesky-Aff1-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Andrew Zalesky</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Andrew%20Zalesky" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Andrew%20Zalesky" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Andrew%20Zalesky%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>Y.T. and A.Z. conceived the idea and designed the study. Y.T. compiled the data, performed the analyses and prepared the visualizations. Y.T. and A.Z. drafted the manuscript. D.S.M. and M.B. provided critical conceptual input. All authors provided critical feedback and editing of the final manuscript. A.Z. supervised the research.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:ye.tian2@unimelb.edu.au">Ye Tian</a> or <a id="corresp-c2" href="mailto:azalesky@unimelb.edu.au">Andrew Zalesky</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> <i>Nature Neuroscience</i> thanks Janine Bijsterbosch, Rodrigo Braga and Evan Gordon for their contribution to the peer review of this work.</p><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec45-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec45">Supplementary information</h2><div class="c-article-section__content" id="Sec45-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figs. 1–24 and Supplementary Tables 1–5.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-020-00711-6/MediaObjects/41593_2020_711_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Topographic%20organization%20of%20the%20human%20subcortex%20unveiled%20with%20functional%20connectivity%20gradients&amp;author=Ye%20Tian%20et%20al&amp;contentID=10.1038%2Fs41593-020-00711-6&amp;copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2020-09-28&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-020-00711-6" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-020-00711-6" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Tian, Y., Margulies, D.S., Breakspear, M. <i>et al.</i> Topographic organization of the human subcortex unveiled with functional connectivity gradients.
                    <i>Nat Neurosci</i> <b>23</b>, 1421–1432 (2020). https://doi.org/10.1038/s41593-020-00711-6</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-020-00711-6?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-11-29">29 November 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-08-21">21 August 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-09-28">28 September 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-11">November 2020</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-020-00711-6</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Dynamic functional changes upon thalamotomy in essential tremor depend on baseline brain morphometry" href="https://doi.org/10.1038/s41598-024-52410-y">
                                        Dynamic functional changes upon thalamotomy in essential tremor depend on baseline brain morphometry
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Thomas A. W. Bolton</li><li>Dimitri Van De Ville</li><li>Constantin Tuleasca</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Reports</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Nuclei-specific hypothalamus networks predict a dimensional marker of stress in humans" href="https://doi.org/10.1038/s41467-024-46275-y">
                                        Nuclei-specific hypothalamus networks predict a dimensional marker of stress in humans
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Daria E. A. Jensen</li><li>Klaus P. Ebmeier</li><li>Miriam C. Klein-Flügge</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:High-intensity sweet taste as a predictor of subjective alcohol responses to the ascending limb of an intravenous alcohol prime: an fMRI study" href="https://doi.org/10.1038/s41386-023-01684-3">
                                        High-intensity sweet taste as a predictor of subjective alcohol responses to the ascending limb of an intravenous alcohol prime: an fMRI study
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jonathan Alessi</li><li>Mario Dzemidzic</li><li>David A. Kareken</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Neuropsychopharmacology</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Blunted brain responses to neutral faces in healthy first-degree relatives of patients with schizophrenia: an image-based fMRI meta-analysis" href="https://doi.org/10.1038/s41537-024-00452-6">
                                        Blunted brain responses to neutral faces in healthy first-degree relatives of patients with schizophrenia: an image-based fMRI meta-analysis
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Anna M. Fiorito</li><li>Giuseppe Blasi</li><li>Guillaume Sescousse</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Schizophrenia</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Sex-specific resting state brain network dynamics in patients with major depressive disorder" href="https://doi.org/10.1038/s41386-024-01799-1">
                                        Sex-specific resting state brain network dynamics in patients with major depressive disorder
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Daifeng Dong</li><li>Diego A. Pizzagalli</li><li>Emily L. Belleau</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Neuropsychopharmacology</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00711-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00711-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41593-020-00721-4"
                       class="c-card__link u-link-inherit"
                       data-track="click"
                       data-track-action="view article"
                       data-track-category="associated content"
                       
                       data-track-label="news_and_views">Functional boundaries within human subcortex</a>
                </h3>
                
<ul data-test="author-list" class="c-author-list c-author-list--compact">
    <li>Evan M. Gordon</li>
</ul>

                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Neuroscience</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">News &amp; Views</span></span>
        
        
            <time class="c-meta__item" datetime="2020-09-28">28 Sept 2020</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "news_and_views";
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-020-00711-6;doi=10.1038/s41593-020-00711-6;techmeta=36,57,59;subjmeta=1627,1647,245,378,3920,631;kwrd=Functional+magnetic+resonance+imaging,Neural+circuits">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=-877972627&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-020-00711-6%26doi%3D10.1038/s41593-020-00711-6%26techmeta%3D36,57,59%26subjmeta%3D1627,1647,245,378,3920,631%26kwrd%3DFunctional+magnetic+resonance+imaging,Neural+circuits">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=-877972627&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-020-00711-6%26doi%3D10.1038/s41593-020-00711-6%26techmeta%3D36,57,59%26subjmeta%3D1627,1647,245,378,3920,631%26kwrd%3DFunctional+magnetic+resonance+imaging,Neural+circuits"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-020-00711-6&amp;format=js&amp;last_modified=2020-09-28" async></script>
<img src="/c4pwjb2k/article/s41593-020-00711-6" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>