<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"technical report","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"cognitive-neuroscience;network-models","webtrendsContentCategory":null,"webtrendsContentCollection":"Focus on neuroscience methods","webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Technical Report"}},"article":{"doi":"10.1038/s41593-020-00719-y"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Joshua Faskowitz","Farnaz Zamani Esfahlani","Youngheun Jo","Olaf Sporns","Richard F. Betzel"],"publishedAt":1603065600,"publishedAtString":"2020-10-19","title":"Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"23","issue":"12"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":{"id":"hhgegahcbd"}},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture","description":"Network neuroscience has relied on a node-centric network model in which cells, populations and regions are linked to one another via anatomical or functional connections. This model cannot account for interactions of edges with one another. In this study, we developed an edge-centric network model that generates constructs ‘edge time series’ and ‘edge functional connectivity’ (eFC). Using network analysis, we show that, at rest, eFC is consistent across datasets and reproducible within the same individual over multiple scan sessions. We demonstrate that clustering eFC yields communities of edges that naturally divide the brain into overlapping clusters, with regions in sensorimotor and attentional networks exhibiting the greatest levels of overlap. We show that eFC is systematically modulated by variation in sensory input. In future work, the edge-centric approach could be useful for identifying novel biomarkers of disease, characterizing individual variation and mapping the architecture of highly resolved neural circuits. The authors present an edge-centric model of brain connectivity. Edge networks are stable across datasets, and their structure can be modulated by sensory input. When clustered, edge networks yield pervasively overlapping functional modules.","datePublished":"2020-10-19T00:00:00Z","dateModified":"2020-10-19T00:00:00Z","pageStart":"1644","pageEnd":"1654","sameAs":"https://doi.org/10.1038/s41593-020-00719-y","keywords":["Cognitive neuroscience","Network models","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig7_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"23","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Joshua Faskowitz","url":"http://orcid.org/0000-0003-1814-7206","affiliation":[{"name":"Indiana University","address":{"name":"Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Indiana University","address":{"name":"Program in Neuroscience, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Farnaz Zamani Esfahlani","affiliation":[{"name":"Indiana University","address":{"name":"Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Youngheun Jo","affiliation":[{"name":"Indiana University","address":{"name":"Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Olaf Sporns","url":"http://orcid.org/0000-0001-7265-4036","affiliation":[{"name":"Indiana University","address":{"name":"Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Indiana University","address":{"name":"Program in Neuroscience, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Indiana University","address":{"name":"Cognitive Science Program, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Indiana University","address":{"name":"Network Science Institute, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Richard F. Betzel","url":"http://orcid.org/0000-0001-9200-1681","affiliation":[{"name":"Indiana University","address":{"name":"Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Indiana University","address":{"name":"Program in Neuroscience, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Indiana University","address":{"name":"Cognitive Science Program, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Indiana University","address":{"name":"Network Science Institute, Indiana University, Bloomington, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"rbetzel@indiana.edu","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-020-00719-y">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture"/>
    <meta name="dc.source" content="Nature Neuroscience 2020 23:12"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2020-10-19"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="Network neuroscience has relied on a node-centric network model in which cells, populations and regions are linked to one another via anatomical or functional connections. This model cannot account for interactions of edges with one another. In this study, we developed an edge-centric network model that generates constructs &#8216;edge time series&#8217; and &#8216;edge functional connectivity&#8217; (eFC). Using network analysis, we show that, at rest, eFC is consistent across datasets and reproducible within the same individual over multiple scan sessions. We demonstrate that clustering eFC yields communities of edges that naturally divide the brain into overlapping clusters, with regions in sensorimotor and attentional networks exhibiting the greatest levels of overlap. We show that eFC is systematically modulated by variation in sensory input. In future work, the edge-centric approach could be useful for identifying novel biomarkers of disease, characterizing individual variation and mapping the architecture of highly resolved neural circuits. The authors present an edge-centric model of brain connectivity. Edge networks are stable across datasets, and their structure can be modulated by sensory input. When clustered, edge networks yield pervasively overlapping functional modules."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2020-10-19"/>
    <meta name="prism.volume" content="23"/>
    <meta name="prism.number" content="12"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="1644"/>
    <meta name="prism.endingPage" content="1654"/>
    <meta name="prism.copyright" content="2020 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-020-00719-y"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-020-00719-y"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-020-00719-y.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-020-00719-y"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture"/>
    <meta name="citation_volume" content="23"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_publication_date" content="2020/12"/>
    <meta name="citation_online_date" content="2020/10/19"/>
    <meta name="citation_firstpage" content="1644"/>
    <meta name="citation_lastpage" content="1654"/>
    <meta name="citation_article_type" content="Technical Report"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-020-00719-y"/>
    <meta name="DOI" content="10.1038/s41593-020-00719-y"/>
    <meta name="size" content="226915"/>
    <meta name="citation_doi" content="10.1038/s41593-020-00719-y"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-020-00719-y&amp;api_key="/>
    <meta name="description" content="Network neuroscience has relied on a node-centric network model in which cells, populations and regions are linked to one another via anatomical or functional connections. This model cannot account for interactions of edges with one another. In this study, we developed an edge-centric network model that generates constructs &#8216;edge time series&#8217; and &#8216;edge functional connectivity&#8217; (eFC). Using network analysis, we show that, at rest, eFC is consistent across datasets and reproducible within the same individual over multiple scan sessions. We demonstrate that clustering eFC yields communities of edges that naturally divide the brain into overlapping clusters, with regions in sensorimotor and attentional networks exhibiting the greatest levels of overlap. We show that eFC is systematically modulated by variation in sensory input. In future work, the edge-centric approach could be useful for identifying novel biomarkers of disease, characterizing individual variation and mapping the architecture of highly resolved neural circuits. The authors present an edge-centric model of brain connectivity. Edge networks are stable across datasets, and their structure can be modulated by sensory input. When clustered, edge networks yield pervasively overlapping functional modules."/>
    <meta name="dc.creator" content="Faskowitz, Joshua"/>
    <meta name="dc.creator" content="Esfahlani, Farnaz Zamani"/>
    <meta name="dc.creator" content="Jo, Youngheun"/>
    <meta name="dc.creator" content="Sporns, Olaf"/>
    <meta name="dc.creator" content="Betzel, Richard F."/>
    <meta name="dc.subject" content="Cognitive neuroscience"/>
    <meta name="dc.subject" content="Network models"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Network neuroscience; citation_author=DS Bassett, O Sporns; citation_volume=20; citation_publication_date=2017; citation_pages=353-364; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Micro-connectomics: probing the organization of neuronal networks at the cellular scale; citation_author=M Schr&#246;ter, O Paulsen, ET Bullmore; citation_volume=18; citation_publication_date=2017; citation_pages=131-146; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=Uniting functional network topology and oscillations in the fronto-parietal single unit network of behaving primates; citation_author=B Dann, JA Michaels, S Schaffelhofer, H Scherberger; citation_volume=5; citation_publication_date=2016; citation_pages=e15719; citation_id=CR3"/>
    <meta name="citation_reference" content="Park, H.-J. &amp; Friston, K. Structural and functional brain networks: from connections to cognition. Science 342, 1238411 (2013)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroinformatics; citation_title=The small world of the cerebral cortex; citation_author=O Sporns, JD Zwi; citation_volume=2; citation_publication_date=2004; citation_pages=145-162; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Evidence for hubs in human functional brain networks; citation_author=JD Power, BL Schlaggar, CN Lessov-Schlaggar, SE Petersen; citation_volume=79; citation_publication_date=2013; citation_pages=798-813; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=Annu. Rev. Psychol.; citation_title=Modular brain networks; citation_author=O Sporns, RF Betzel; citation_volume=67; citation_publication_date=2016; citation_pages=613-640; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=The economy of brain network organization; citation_author=E Bullmore, O Sporns; citation_volume=13; citation_publication_date=2012; citation_pages=336-349; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Complex brain networks: graph theoretical analysis of structural and functional systems; citation_author=E Bullmore, O Sporns; citation_volume=10; citation_publication_date=2009; citation_pages=186-198; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Complex network measures of brain connectivity: uses and interpretations; citation_author=M Rubinov, O Sporns; citation_volume=52; citation_publication_date=2010; citation_pages=1059-1069; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Link communities reveal multiscale complexity in networks; citation_author=Y-Y Ahn, JP Bagrow, S Lehmann; citation_volume=466; citation_publication_date=2010; citation_pages=761-764; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Line graphs, link partitions, and overlapping communities; citation_author=T Evans, R Lambiotte; citation_volume=80; citation_publication_date=2009; citation_pages=016105; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Topographic organization of the cerebral cortex and brain cartography; citation_author=SB Eickhoff, RT Constable, BT Yeo; citation_volume=170; citation_publication_date=2018; citation_pages=332-347; citation_id=CR13"/>
    <meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. B Biol. Sci.; citation_title=An edge-centric perspective on the human connectome: link communities in the brain; citation_author=MA Reus, VM Saenger, RS Kahn, MP Heuvel; citation_volume=369; citation_publication_date=2014; citation_pages=20130527; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Network modelling methods for FMRI; citation_author=SM Smith; citation_volume=54; citation_publication_date=2011; citation_pages=875-891; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Advancing functional connectivity research from association to causation; citation_author=AT Reid; citation_volume=22; citation_publication_date=2019; citation_pages=1751-1760; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The WU-Minn Human Connectome Project: an overview; citation_author=DC Essen; citation_volume=80; citation_publication_date=2013; citation_pages=62-79; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Precision functional mapping of individual human brains; citation_author=EM Gordon; citation_volume=95; citation_publication_date=2017; citation_pages=791-807; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Gigascience; citation_title=The healthy brain network serial scanning initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions; citation_author=D O&#8217;Connor; citation_volume=6; citation_publication_date=2017; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Functional parcellation using time courses of instantaneous connectivity; citation_author=ES Oort; citation_volume=170; citation_publication_date=2018; citation_pages=31-40; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Space-independent community and hub structure of functional brain networks; citation_author=FZ Esfahlani, MA Bertolero, DS Bassett, RF Betzel; citation_volume=211; citation_publication_date=2020; citation_pages=116612; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Local&#8211;global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI; citation_author=A Schaefer; citation_volume=28; citation_publication_date=2018; citation_pages=3095-3114; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Functional system and areal organization of a highly sampled individual human brain; citation_author=TO Laumann; citation_volume=87; citation_publication_date=2015; citation_pages=657-670; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Sci. Rep.; citation_title=The quest for identifiability in human functional connectomes; citation_author=E Amico, J Go&#241;i; citation_volume=8; citation_publication_date=2018; citation_pages=8254; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Dynamic reconfiguration of human brain networks during learning; citation_author=DS Bassett; citation_volume=108; citation_publication_date=2011; citation_pages=7641-7646; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity; citation_author=ES Finn; citation_volume=18; citation_publication_date=2015; citation_pages=1664-1671; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Describing functional diversity of brain regions and brain networks; citation_author=ML Anderson, J Kinnison, L Pessoa; citation_volume=73; citation_publication_date=2013; citation_pages=50-58; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Life Rev.; citation_title=Understanding brain networks and brain organization; citation_author=L Pessoa; citation_volume=11; citation_publication_date=2014; citation_pages=400-435; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Estimates of segregation and overlap of functional connectivity networks in the human cerebral cortex; citation_author=BT Yeo, FM Krienen, MW Chee, RL Buckner; citation_volume=88; citation_publication_date=2014; citation_pages=212-227; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Spontaneously emerging patterns in human visual cortex reflect responses to naturalistic sensory stimuli; citation_author=M Wilf; citation_volume=27; citation_publication_date=2017; citation_pages=750-763; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Representation of real-world event schemas during narrative perception; citation_author=C Baldassano, U Hasson, KA Norman; citation_volume=38; citation_publication_date=2018; citation_pages=9689-9699; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Communication dynamics in complex brain networks; citation_author=A Avena-Koenigsberger, B Misic, O Sporns; citation_volume=19; citation_publication_date=2018; citation_pages=17; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Estimation of dynamic functional connectivity using multiplication of temporal derivatives; citation_author=JM Shine; citation_volume=122; citation_publication_date=2015; citation_pages=399-407; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Time-varying functional network information extracted from brief instances of spontaneous brain activity; citation_author=X Liu, JH Duyn; citation_volume=110; citation_publication_date=2013; citation_pages=4392-4397; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Finding and evaluating community structure in networks; citation_author=ME Newman, M Girvan; citation_volume=69; citation_publication_date=2004; citation_pages=026113; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Maps of random walks on complex networks reveal community structure; citation_author=M Rosvall, CT Bergstrom; citation_volume=105; citation_publication_date=2008; citation_pages=1118-1123; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=The modular and integrative functional architecture of the human brain; citation_author=MA Bertolero, BT Yeo, M D&#8217;Esposito; citation_volume=112; citation_publication_date=2015; citation_pages=E6798-E6807; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Correspondence of the brain&#8217;s functional architecture during activation and rest; citation_author=SM Smith; citation_volume=106; citation_publication_date=2009; citation_pages=13040-13045; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Task activations produce spurious but systematic inflation of task functional connectivity estimates; citation_author=MW Cole; citation_volume=189; citation_publication_date=2019; citation_pages=1-18; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Situating the default-mode network along a principal gradient of macroscale cortical organization; citation_author=DS Margulies; citation_volume=113; citation_publication_date=2016; citation_pages=12574-12579; citation_id=CR40"/>
    <meta name="citation_reference" content="Esfahlani, F. Z. et al. High-amplitude co-fluctuations in cortical activity drive functional connectivity. Preprint at bioRxiv 
https://doi.org/10.1101/800045

 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Netw. Neurosci.; citation_title=Questions and controversies in the study of time-varying functional connectivity in resting fMRI; citation_author=DJ Lurie; citation_volume=4; citation_publication_date=2020; citation_pages=30-69; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Functional boundaries in the human cerebellum revealed by a multi-domain task battery; citation_author=M King, CR Hernandez-Castillo, RA Poldrack, RB Ivry, J Diedrichsen; citation_volume=22; citation_publication_date=2019; citation_pages=1371-1378; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Machine learning classifiers and fMRI: a tutorial overview; citation_author=F Pereira, T Mitchell, M Botvinick; citation_volume=45; citation_publication_date=2009; citation_pages=S199-S209; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Computational psychiatry as a bridge from neuroscience to clinical applications; citation_author=QJ Huys, TV Maia, MJ Frank; citation_volume=19; citation_publication_date=2016; citation_pages=404; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Annu. Rev. Psychol.; citation_title=Multivariate statistical analyses for neuroimaging data; citation_author=AR McIntosh, B Mi&#353;i&#263;; citation_volume=64; citation_publication_date=2013; citation_pages=499-525; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=On the use of correlation as a measure of network connectivity; citation_author=A Zalesky, A Fornito, E Bullmore; citation_volume=60; citation_publication_date=2012; citation_pages=2096-2106; citation_id=CR47"/>
    <meta name="citation_reference" content="Owen, L. L., Chang, T. H. &amp; Manning, J. R. High-level cognition during story listening is reflected in high-order dynamic correlations in neural activity patterns. Preprint at bioRxiv 
https://doi.org/10.1101/763821

 (2019)."/>
    <meta name="citation_reference" content="citation_journal_title=Netw. Neurosci.; citation_title=The importance of the whole: topological data analysis for the network neuroscientist; citation_author=AE Sizemore, JE Phillips-Cremins, R Ghrist, DS Bassett; citation_volume=3; citation_publication_date=2019; citation_pages=656-673; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=Dynamic network drivers of seizure generation, propagation and termination in human neocortical epilepsy; citation_author=AN Khambhati; citation_volume=11; citation_publication_date=2015; citation_pages=e1004608; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Human cognition involves the dynamic integration of neural activity and neuromodulatory systems; citation_author=JM Shine; citation_volume=22; citation_publication_date=2019; citation_pages=289-296; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=Brain network adaptability across task states; citation_author=EN Davison; citation_volume=11; citation_publication_date=2015; citation_pages=e1004029; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The minimal preprocessing pipelines for the Human Connectome Project; citation_author=MF Glasser; citation_volume=80; citation_publication_date=2013; citation_pages=105-124; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=MSM: a new flexible framework for multimodal surface matching; citation_author=EC Robinson; citation_volume=100; citation_publication_date=2014; citation_pages=414-426; citation_id=CR54"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=fMRIPrep: a robust preprocessing pipeline for functional MRI; citation_author=O Esteban; citation_volume=16; citation_publication_date=2019; citation_pages=111-116; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Neuroinform.; citation_title=Nipype: a flexible, lightweight and extensible neuroimaging data processing framework in Python; citation_author=K Gorgolewski; citation_volume=5; citation_publication_date=2011; citation_pages=13; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Neuroinform.; citation_title=Machine learning for neuroimaging with scikit-learn; citation_author=A Abraham; citation_volume=8; citation_publication_date=2014; citation_pages=14; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=Med. Image Anal.; citation_title=Symmetric diffeomorphic image registration with cross-correlation: evaluating automated labeling of elderly and neurodegenerative brain; citation_author=BB Avants, CL Epstein, M Grossman, JC Gee; citation_volume=12; citation_publication_date=2008; citation_pages=26-41; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=N4ITK: improved N3 bias correction; citation_author=NJ Tustison; citation_volume=29; citation_publication_date=2010; citation_pages=1310-1320; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Cortical surface-based analysis: I. Segmentation and surface reconstruction; citation_author=AM Dale, B Fischl, MI Sereno; citation_volume=9; citation_publication_date=1999; citation_pages=179-194; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=Mindboggling morphometry of human brains; citation_author=A Klein; citation_volume=13; citation_publication_date=2017; citation_pages=e1005350; citation_id=CR61"/>
    <meta name="citation_reference" content="Fonov, V. S., Evans, A. C., McKinstry, R. C., Almli, C. &amp; Collins, D. Unbiased nonlinear average age-appropriate brain templates from birth to adulthood. Neuroimage 47, S102 (2009)."/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=Segmentation of brain MR images through a hidden Markov random field model and the expectation-maximization algorithm; citation_author=Y Zhang, M Brady, S Smith; citation_volume=20; citation_publication_date=2001; citation_pages=45-57; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Improved optimization for the robust and accurate linear registration and motion correction of brain images; citation_author=M Jenkinson, P Bannister, M Brady, S Smith; citation_volume=17; citation_publication_date=2002; citation_pages=825-841; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Neuroinform.; citation_title=Evaluation of field map and nonlinear registration methods for correction of susceptibility artifacts in diffusion MRI; citation_author=S Wang; citation_volume=11; citation_publication_date=2017; citation_pages=17; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Characterization and correction of geometric distortions in 814 diffusion weighted images; citation_author=JM Treiber; citation_volume=11; citation_publication_date=2016; citation_pages=e0152472; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Accurate and robust brain image alignment using boundary-based registration; citation_author=DN Greve, B Fischl; citation_volume=48; citation_publication_date=2009; citation_pages=63-72; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Methods to detect, characterize, and remove motion artifact in resting state fMRI; citation_author=JD Power; citation_volume=84; citation_publication_date=2014; citation_pages=320-341; citation_id=CR68"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=MRIQC: advancing the automatic prediction of image quality in MRI from unseen sites; citation_author=O Esteban; citation_volume=12; citation_publication_date=2017; citation_pages=e0184661; citation_id=CR69"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI; citation_author=L Parkes, B Fulcher, M Y&#252;cel, A Fornito; citation_volume=171; citation_publication_date=2018; citation_pages=415-436; citation_id=CR70"/>
    <meta name="citation_reference" content="Yeo, B. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1126&#8211;1165 (2011)."/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Automatically parcellating the human cerebral cortex; citation_author=B Fischl; citation_volume=14; citation_publication_date=2004; citation_pages=11-22; citation_id=CR72"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Modular preprocessing pipelines can reintroduce artifacts into fMRI data; citation_author=MA Lindquist, S Geuter, TD Wager, BS Caffo; citation_volume=40; citation_publication_date=2019; citation_pages=2358-2376; citation_id=CR73"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An improved framework for confound regression and filtering for control of motion artifact in the preprocessing of resting-state functional connectivity data; citation_author=TD Satterthwaite; citation_volume=64; citation_publication_date=2013; citation_pages=240-256; citation_id=CR74"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Efficient discovery of overlapping communities in massive networks; citation_author=PK Gopalan, DM Blei; citation_volume=110; citation_publication_date=2013; citation_pages=14534-14539; citation_id=CR75"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. IEEE; citation_title=Overlapping communities explain core-periphery organization of networks; citation_author=J Yang, J Leskovec; citation_volume=102; citation_publication_date=2014; citation_pages=1892-1902; citation_id=CR76"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Overlapping community detection using Bayesian non-negative matrix factorization; citation_author=I Psorakis, S Roberts, M Ebden, B Sheldon; citation_volume=83; citation_publication_date=2011; citation_pages=066114; citation_id=CR77"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Overlapping communities reveal rich structure in large-scale brain networks during rest and task conditions; citation_author=M Najafi, BW McMenamin, JZ Simon, L Pessoa; citation_volume=135; citation_publication_date=2016; citation_pages=92-106; citation_id=CR78"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Functional cartography of complex metabolic networks; citation_author=R Guimera, LAN Amaral; citation_volume=433; citation_publication_date=2005; citation_pages=895-900; citation_id=CR79"/>
    <meta name="citation_reference" content="citation_journal_title=Sci. Rep.; citation_title=Versatility of nodal affiliation to communities; citation_author=M Shinn; citation_volume=7; citation_publication_date=2017; citation_pages=1-10; citation_id=CR80"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Multilayer network switching rate predicts brain performance; citation_author=M Pedersen, A Zalesky, A Omidvarnia, GD Jackson; citation_volume=115; citation_publication_date=2018; citation_pages=13376-13381; citation_id=CR81"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Statistical mechanics of community detection; citation_author=J Reichardt, S Bornholdt; citation_volume=74; citation_publication_date=2006; citation_pages=016110; citation_id=CR82"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Narrow scope for resolution-limit-free community detection; citation_author=VA Traag, P Dooren, Y Nesterov; citation_volume=84; citation_publication_date=2011; citation_pages=016114; citation_id=CR83"/>
    <meta name="citation_reference" content="citation_journal_title=Multiscale Model. Simul.; citation_title=Community detection in temporal multilayer networks, with an application to correlation networks; citation_author=M Bazzi; citation_volume=14; citation_publication_date=2016; citation_pages=1-41; citation_id=CR84"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The community structure of functional brain networks exhibits scale-specific patterns of inter-and intra-subject variability; citation_author=RF Betzel; citation_volume=202; citation_publication_date=2019; citation_pages=115990; citation_id=CR85"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Community structure in time-dependent, multiscale, and multiplex networks; citation_author=PJ Mucha, T Richardson, K Macon, MA Porter, J-P Onnela; citation_volume=328; citation_publication_date=2010; citation_pages=876-878; citation_id=CR86"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A positive&#8211;negative mode of population covariation links brain connectivity, demographics and behavior; citation_author=SM Smith; citation_volume=18; citation_publication_date=2015; citation_pages=1565-1567; citation_id=CR87"/>
    <meta name="citation_author" content="Faskowitz, Joshua"/>
    <meta name="citation_author_institution" content="Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA"/>
    <meta name="citation_author_institution" content="Program in Neuroscience, Indiana University, Bloomington, USA"/>
    <meta name="citation_author" content="Esfahlani, Farnaz Zamani"/>
    <meta name="citation_author_institution" content="Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA"/>
    <meta name="citation_author" content="Jo, Youngheun"/>
    <meta name="citation_author_institution" content="Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA"/>
    <meta name="citation_author" content="Sporns, Olaf"/>
    <meta name="citation_author_institution" content="Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA"/>
    <meta name="citation_author_institution" content="Program in Neuroscience, Indiana University, Bloomington, USA"/>
    <meta name="citation_author_institution" content="Cognitive Science Program, Indiana University, Bloomington, USA"/>
    <meta name="citation_author_institution" content="Network Science Institute, Indiana University, Bloomington, USA"/>
    <meta name="citation_author" content="Betzel, Richard F."/>
    <meta name="citation_author_institution" content="Department of Psychological and Brain Sciences, Indiana University, Bloomington, USA"/>
    <meta name="citation_author_institution" content="Program in Neuroscience, Indiana University, Bloomington, USA"/>
    <meta name="citation_author_institution" content="Cognitive Science Program, Indiana University, Bloomington, USA"/>
    <meta name="citation_author_institution" content="Network Science Institute, Indiana University, Bloomington, USA"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture"/>
    <meta name="twitter:description" content="Nature Neuroscience - The authors present an edge-centric model of brain connectivity. Edge networks are stable across datasets, and their structure can be modulated by sensory input. When..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-020-00719-y"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture - Nature Neuroscience"/>
    <meta property="og:description" content="The authors present an edge-centric model of brain connectivity. Edge networks are stable across datasets, and their structure can be modulated by sensory input. When clustered, edge networks yield pervasively overlapping functional modules."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-020-00719-y;doi=10.1038/s41593-020-00719-y;subjmeta=116,1925,2649,378,631;kwrd=Cognitive+neuroscience,Network+models">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=299613425&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-020-00719-y%26doi%3D10.1038/s41593-020-00719-y%26subjmeta%3D116,1925,2649,378,631%26kwrd%3DCognitive+neuroscience,Network+models">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=299613425&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-020-00719-y%26doi%3D10.1038/s41593-020-00719-y%26subjmeta%3D116,1925,2649,378,631%26kwrd%3DCognitive+neuroscience,Network+models"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-020-00719-y'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;technical-report" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:technical reports"><span itemprop="name">technical reports</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00719-y.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Technical Report</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2020-10-19">19 October 2020</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Joshua-Faskowitz-Aff1-Aff2" data-author-popup="auth-Joshua-Faskowitz-Aff1-Aff2" data-author-search="Faskowitz, Joshua">Joshua Faskowitz</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-1814-7206"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1814-7206</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Farnaz_Zamani-Esfahlani-Aff1" data-author-popup="auth-Farnaz_Zamani-Esfahlani-Aff1" data-author-search="Esfahlani, Farnaz Zamani">Farnaz Zamani Esfahlani</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Youngheun-Jo-Aff1" data-author-popup="auth-Youngheun-Jo-Aff1" data-author-search="Jo, Youngheun">Youngheun Jo</a><sup class="u-js-hide"><a href="#Aff1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Olaf-Sporns-Aff1-Aff2-Aff3-Aff4" data-author-popup="auth-Olaf-Sporns-Aff1-Aff2-Aff3-Aff4" data-author-search="Sporns, Olaf">Olaf Sporns</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-7265-4036"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7265-4036</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a>,<a href="#Aff3">3</a>,<a href="#Aff4">4</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 5 authors for this article" title="Show all 5 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Richard_F_-Betzel-Aff1-Aff2-Aff3-Aff4" data-author-popup="auth-Richard_F_-Betzel-Aff1-Aff2-Aff3-Aff4" data-author-search="Betzel, Richard F." data-corresp-id="c1">Richard F. Betzel<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-9200-1681"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9200-1681</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a>,<a href="#Aff3">3</a>,<a href="#Aff4">4</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 23</b>, <span class="u-visually-hidden">pages </span>1644–1654 (<span data-test="article-publication-year">2020</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">15k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">102 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">179 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-020-00719-y/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/cognitive-neuroscience" data-track="click" data-track-action="view subject" data-track-label="link">Cognitive neuroscience</a></li><li class="c-article-subject-list__subject"><a href="/subjects/network-models" data-track="click" data-track-action="view subject" data-track-label="link">Network models</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Network neuroscience has relied on a node-centric network model in which cells, populations and regions are linked to one another via anatomical or functional connections. This model cannot account for interactions of edges with one another. In this study, we developed an edge-centric network model that generates constructs ‘edge time series’ and ‘edge functional connectivity’ (eFC). Using network analysis, we show that, at rest, eFC is consistent across datasets and reproducible within the same individual over multiple scan sessions. We demonstrate that clustering eFC yields communities of edges that naturally divide the brain into overlapping clusters, with regions in sensorimotor and attentional networks exhibiting the greatest levels of overlap. We show that eFC is systematically modulated by variation in sensory input. In future work, the edge-centric approach could be useful for identifying novel biomarkers of disease, characterizing individual variation and mapping the architecture of highly resolved neural circuits.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00719-y.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00719-y.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-024-07109-5/MediaObjects/41586_2024_7109_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-024-07109-5?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41586-024-07109-5">A concerted neuron–astrocyte program declines in ageing and schizophrenia
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">06 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Emi Ling, James Nemesh, … Steven A. McCarroll</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41593-024-01597-4/MediaObjects/41593_2024_1597_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41593-024-01597-4?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41593-024-01597-4">Co-dependent excitatory and inhibitory plasticity accounts for quick, stable and long-lasting memories in biological networks
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">20 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Everton J. Agnes &amp; Tim P. Vogels</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41583-024-00796-z/MediaObjects/41583_2024_796_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41583-024-00796-z?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41583-024-00796-z">Preparatory activity and the expansive null-space
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">05 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Mark M. Churchland &amp; Krishna V. Shenoy</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'topic',
                        model: 'visits_v2',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711556702,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Network science offers a promising framework for representing and modeling neural systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Bassett, D. S. &amp; Sporns, O. Network neuroscience. Nat. Neurosci. 20, 353–364 (2017)." href="/articles/s41593-020-00719-y#ref-CR1" id="ref-link-section-d30515570e426">1</a></sup>. From interconnected cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Schröter, M., Paulsen, O. &amp; Bullmore, E. T. Micro-connectomics: probing the organization of neuronal networks at the cellular scale. Nat. Rev. Neurosci. 18, 131–146 (2017)." href="/articles/s41593-020-00719-y#ref-CR2" id="ref-link-section-d30515570e430">2</a></sup>, to neuronal populations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Dann, B., Michaels, J. A., Schaffelhofer, S. &amp; Scherberger, H. Uniting functional network topology and oscillations in the fronto-parietal single unit network of behaving primates. eLife 5, e15719 (2016)." href="/articles/s41593-020-00719-y#ref-CR3" id="ref-link-section-d30515570e434">3</a></sup>, to large-scale brain areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Park, H.-J. &amp; Friston, K. Structural and functional brain networks: from connections to cognition. Science 342, 1238411 (2013)." href="/articles/s41593-020-00719-y#ref-CR4" id="ref-link-section-d30515570e438">4</a></sup>, network analysis has contributed insights into the topological principles that govern nervous system organization and shape brain function. These include small-world architecture<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Sporns, O. &amp; Zwi, J. D. The small world of the cerebral cortex. Neuroinformatics 2, 145–162 (2004)." href="/articles/s41593-020-00719-y#ref-CR5" id="ref-link-section-d30515570e442">5</a></sup>, the emergence of integrative hubs and rich clubs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Power, J. D., Schlaggar, B. L., Lessov-Schlaggar, C. N. &amp; Petersen, S. E. Evidence for hubs in human functional brain networks. Neuron 79, 798–813 (2013)." href="/articles/s41593-020-00719-y#ref-CR6" id="ref-link-section-d30515570e447">6</a></sup>, modular structure to promote specialized information processing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Sporns, O. &amp; Betzel, R. F. Modular brain networks. Annu. Rev. Psychol. 67, 613–640 (2016)." href="/articles/s41593-020-00719-y#ref-CR7" id="ref-link-section-d30515570e451">7</a></sup> and tradeoffs between topological features and the material and metabolic costs of wiring<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Bullmore, E. &amp; Sporns, O. The economy of brain network organization. Nat. Rev. Neurosci. 13, 336–349 (2012)." href="/articles/s41593-020-00719-y#ref-CR8" id="ref-link-section-d30515570e455">8</a></sup>.</p><p>Central to these and other discoveries in network neuroscience is a simple representation of the brain in which neural elements and their pairwise interactions are treated as the nodes and edges of a network, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Bullmore, E. &amp; Sporns, O. Complex brain networks: graph theoretical analysis of structural and functional systems. Nat. Rev. Neurosci. 10, 186–198 (2009)." href="/articles/s41593-020-00719-y#ref-CR9" id="ref-link-section-d30515570e462">9</a></sup>. This standard model is fundamentally node-centric in that it treats neural elements (nodes) as the irreducible units of brain structure and function. This emphasis on network nodes is further reinforced by the analyses carried out on brain networks, which tend to focus on properties of nodes or groups of nodes—for example, their centralities or community affiliations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: uses and interpretations. Neuroimage 52, 1059–1069 (2010)." href="/articles/s41593-020-00719-y#ref-CR10" id="ref-link-section-d30515570e466">10</a></sup>.</p><p>A limitation of the node-centric approach is that it cannot capture potentially meaningful features or patterns of inter-relationships among edges. In other scientific domains, prioritizing network edges—for example, by modeling and analyzing edge–edge interactions as a graph—has provided important insights into the organization and function of complex systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Ahn, Y.-Y., Bagrow, J. P. &amp; Lehmann, S. Link communities reveal multiscale complexity in networks. Nature 466, 761–764 (2010)." href="/articles/s41593-020-00719-y#ref-CR11" id="ref-link-section-d30515570e473">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Evans, T. &amp; Lambiotte, R. Line graphs, link partitions, and overlapping communities. Phys. Rev. E 80, 016105 (2009)." href="/articles/s41593-020-00719-y#ref-CR12" id="ref-link-section-d30515570e476">12</a></sup>. Nonetheless, network neuroscience has remained largely focused on nodal features and partitions, paralleling a rich history of parceling, mapping and comparing cortical and subcortical gray matter regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Eickhoff, S. B., Constable, R. T. &amp; Yeo, B. T. Topographic organization of the cerebral cortex and brain cartography. Neuroimage 170, 332–347 (2018)." href="/articles/s41593-020-00719-y#ref-CR13" id="ref-link-section-d30515570e480">13</a></sup>. On the other hand, several recent studies have begun modeling brain networks from the perspective of interacting edges, including one foundational study that applied graph theoretic measures to a ‘line graph’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Evans, T. &amp; Lambiotte, R. Line graphs, link partitions, and overlapping communities. Phys. Rev. E 80, 016105 (2009)." href="/articles/s41593-020-00719-y#ref-CR12" id="ref-link-section-d30515570e484">12</a></sup> of interrelated white matter tracts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="de Reus, M. A., Saenger, V. M., Kahn, R. S. &amp; van den Heuvel, M. P. An edge-centric perspective on the human connectome: link communities in the brain. Philos. Trans. R. Soc. B Biol. Sci. 369, 20130527 (2014)." href="/articles/s41593-020-00719-y#ref-CR14" id="ref-link-section-d30515570e488">14</a></sup>. Although highly novel, line graphs were never adopted widely, as their construction requires users to first specify and then apply a sparsity threshold to a connectivity matrix.</p><p>Here we present a novel modeling framework for investigating functional brain network data from an edge-centric perspective. Our approach can be viewed as a temporal ‘unwrapping’ of the Pearson correlation measure—the metric commonly used for estimating the strength of functional connectivity between pairs of brain regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Smith, S. M. et al. Network modelling methods for FMRI. Neuroimage 54, 875–891 (2011)." href="/articles/s41593-020-00719-y#ref-CR15" id="ref-link-section-d30515570e495">15</a></sup>—thereby generating interpretable time series for each edge that express fluctuations in its weight across time. Importantly, edge time series allow the estimation of edge correlation structure, a construct we refer to as eFC. High-amplitude eFC indexes strong similarity in the co-fluctuation of two edges across time, whereas low-amplitude eFC indicates co-fluctuation patterns that are largely independent.</p><p>From a neuroscientific perspective, eFC can be viewed both as an extension of and a complement to traditional node-centric representations of brain networks. In node-centric network models, functional connections represent the temporal correlation of activity recorded from spatially distinct regions and often interpreted as a measure of inter-regional communication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Reid, A. T. et al. Advancing functional connectivity research from association to causation. Nat. Neurosci. 22, 1751–1760 (2019)." href="/articles/s41593-020-00719-y#ref-CR16" id="ref-link-section-d30515570e503">16</a></sup>. That is, strong functional connections are thought to reflect the time-averaged strength of ‘communication’ between brain regions. eFC, on the other hand, tracks how communication patterns evolve over time and ultimately assesses whether similar patterns are occurring in the brain simultaneously (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">1</a>).</p><p>In this study, we demonstrate that eFC is highly replicable given sufficient amounts of data, stable within individuals across multiple scan sessions and consistent across datasets. Next, we apply data-driven clustering algorithms to eFC, which result in partitions of the eFC network into communities of co-fluctuating edges. Each community can be mapped back to individual nodes, yielding overlapping regional community assignments. We find that brain regions associated with sensorimotor and attention networks participate in disproportionately many communities compared to other brain systems, but that, relative to one another, those same regions participate in similar sets of communities. Finally, we compare the organization of eFC at rest and during passive viewing of movies and find that eFC is consistently and reliably modulated by changes in sensory input.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><p>In this section, we analyze eFC estimated using functional MRI (fMRI) data from three independently acquired datasets: 100 unrelated participants from the Human Connectome Project (HCP)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Van Essen, D. C. et al. The WU-Minn Human Connectome Project: an overview. Neuroimage 80, 62–79 (2013)." href="/articles/s41593-020-00719-y#ref-CR17" id="ref-link-section-d30515570e521">17</a></sup>, ten participants scanned ten times as part of the Midnight Scan Club (MSC)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Gordon, E. M. et al. Precision functional mapping of individual human brains. Neuron 95, 791–807 (2017)." href="/articles/s41593-020-00719-y#ref-CR18" id="ref-link-section-d30515570e525">18</a></sup> and ten participants scanned multiple times as part of the Healthy Brain Network (HBN) Serial Scanning Initiative<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="O’Connor, D. et al. The healthy brain network serial scanning initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions. Gigascience 6, giw011 (2017)." href="/articles/s41593-020-00719-y#ref-CR19" id="ref-link-section-d30515570e529">19</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec3">eFC</h3><p>Many studies have investigated networks whose nodes and edges represent brain regions and pairwise functional interactions, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Park, H.-J. &amp; Friston, K. Structural and functional brain networks: from connections to cognition. Science 342, 1238411 (2013)." href="/articles/s41593-020-00719-y#ref-CR4" id="ref-link-section-d30515570e540">4</a></sup>. Here we extend this framework to consider interactions not between pairs of brain regions but between pairs of edges.</p><p>Extant approaches for estimating edge–edge connectivity matrices include construction of line graphs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Evans, T. &amp; Lambiotte, R. Line graphs, link partitions, and overlapping communities. Phys. Rev. E 80, 016105 (2009)." href="/articles/s41593-020-00719-y#ref-CR12" id="ref-link-section-d30515570e547">12</a></sup> or calculating edge overlap indices<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Ahn, Y.-Y., Bagrow, J. P. &amp; Lehmann, S. Link communities reveal multiscale complexity in networks. Nature 466, 761–764 (2010)." href="/articles/s41593-020-00719-y#ref-CR11" id="ref-link-section-d30515570e551">11</a></sup>. Although suitable for sparse networks with positively weighted edges, these approaches are less appropriate for functional neuroimaging data, where networks are typically fully weighted and signed. Here we introduce a method that is well suited for these types of data, operates directly on time series and is closely related to the Pearson correlation coefficient typically used to assess strength of inter-regional functional connections. We refer to the matrices obtained using this procedure as eFC.</p><p>Beginning with regional time series, calculating eFC can be accomplished in three steps, starting by <i>z</i><i>-</i>scoring the time series (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig1">1a,d</a>). Next, for all pairs of brain regions, we calculate the element-wise product of their <i>z</i>-scored time series (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig1">1b,e</a>). This results in a new set of time series, referred to as ‘edge time series’, whose elements represent the instantaneous co-fluctuation magnitude between pairs of brain regions and whose average across time is exactly equal to the Pearson correlation coefficient (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig1">1c</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="van Oort, E. S. et al. Functional parcellation using time courses of instantaneous connectivity. Neuroimage 170, 31–40 (2018)." href="/articles/s41593-020-00719-y#ref-CR20" id="ref-link-section-d30515570e576">20</a></sup>. Co-fluctuation values are positive when activity of two regions deflects in the same direction at precisely the same moment in time, are negative when activity deflects in the opposite direction and are zero when activity is close to baseline. Importantly, the magnitude of these edge time series is not systematically related to in-scanner motion (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">2</a>). The third and final step involves calculating the element-wise product between pairs of edge time series. When repeated over all pairs of edges, the result is an edge-by-edge matrix whose elements are normalized to the interval [−1,1] (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig1">1f</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2a</a>; see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00719-y#Sec13">Methods</a> for additional details on eFC construction).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Derivation of the eFC matrix."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Derivation of the eFC matrix.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="427"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>Each element of the eFC matrix is calculated based on the fMRI BOLD activity time series from four nodes (brain regions). <b>a</b>,<b>d</b>, We show four representative times series from regions <i>i</i>, <i>j</i>, <i>u</i> and <i>v</i>. <b>c</b>, nFC is typically calculated by standardizing (<i>z</i><i>-</i>scoring) each time series, performing an element-wise product (dot product) of time series pairs and calculating the average value of a product time series (actually the sum of each element divided by <i>T</i> − 1, where <i>T</i> is the number of observations). To calculate eFC, we retain the vectors of element-wise products for every pair of regions. <b>b</b>,<b>e</b>, We show product time series for the pairs {<i>i</i>,<i>j</i>} and {<i>u</i>,<i>v</i>}, respectively. The elements of these product time series represent the magnitude of time-resolved co-fluctuation between region pairs (or edges in the nFC graph). We can calculate the magnitude of eFC by performing an element-wise multiplication of the product time series and normalizing the sum by the squared root standard deviations of both time series, ensuring that the magnitude of eFC is bounded to the interval [−1,1]. <b>f</b>, The resulting value is stored in the eFC matrix.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Organization of the eFC matrix."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Organization of the eFC matrix.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="857"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Force-directed layout of the eFC matrix (largest connected component after thresholding away weak connections). Nodes in this graph represent edges in the traditional nFC matrix. Here nodes are colored according to whether the corresponding edge fell within or between cognitive systems. Within-system edges are encircled in black. <b>b</b>, eFC matrix in which rows and columns correspond to pairs of brain regions. <b>c</b>, Two-dimensional histogram of the relationship between eFC and the product of edges’ respective nFC weights. <b>d</b>, Two-dimensional histogram of the relationship between eFC and the surface area of the quadrilateral defined by the four nodes. <b>e</b>, Mean eFC among edges where both edges fall between systems (between; <i>n</i> = 2.3 × 10<sup>8</sup>), where one edge falls within and the other between systems (mixture; <i>n</i> = 1.7 × 10<sup>8</sup>) and where both edges fall within systems (within; <i>n</i> = 8.1 × 10<sup>5</sup>). <b>f</b>, Mean eFC among edges within 16 cognitive systems (<i>n</i> = 6.5 × 10<sup>4</sup>). All results presented in this figure are derived from HCP data. Box plots, shown in green and overlaid on data points in <b>e</b> and <b>f</b>, depict the interquartile range (IQR) and the median value of the distribution. Whiskers extend to the nearest points ±1.5× IQR above and below the 25th and 75th percentiles. Note that, in <b>e</b>, not all points can be displayed owing to the large number of edge–edge connections.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Although eFC is, to our knowledge, a novel construct, we note that the first two steps in calculating eFC are the same as those used to calculate nodal functional connectivity (nFC); the mean value of any co-fluctuation time series is simply the Pearson correlation coefficient. Given that eFC is mathematically related to nFC, we first asked whether it was possible to approximate eFC using estimates of nFC. This is an important question. Whereas the calculation of eFC can be implemented efficiently, performing certain operations on the eFC matrix can prove computationally expensive (it is a fully weighted [<i>M</i> × <i>M</i>] matrix, where <span class="mathjax-tex">\(M = \frac{{N\left( {N - 1} \right)}}{2}\)</span>, and <i>N</i> is the number of nodes; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2b</a>). However, a direct comparison of eFC and nFC is not possible owing to differences in dimensionality. Still, we can approximate eFC using nFC edge weights. Perhaps the simplest approach is to model the edge connection between region pairs {<i>i</i>,<i>j</i>} and {<i>u</i>,<i>v</i>} as a linear combination of the six edges that can be formed between those regions (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00719-y#Sec13">Methods</a>). Although this model performs poorly (correlation of observed and approximated eFC; <i>r</i> = 0.21; <i>n</i> = 197,995,050 edge–edge pairs), we can improve on its performance by including interaction terms based on node connectivity—that is, <i>nFC</i><sub><i>ij</i></sub> × <i>nFC</i><sub><i>uv</i></sub> (<i>r</i> = 0.73; <i>P</i> &lt; 10<sup>−15</sup>; <i>n</i> = 197,995,050 edge–edge pairs; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2c</a>). Collectively, these observations suggest that eFC is not well approximated using linear combinations of nFC, but, with nonlinear transformations and inclusion of interaction terms, nFC can approximate eFC. However, these transformations are unintuitive, and the approximation still fails to fully explain variance in eFC.</p><p>Next, we explored variation of eFC across acquisitions and processing decisions. We found that eFC weights are similar across three independently acquired datasets (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">3</a>) and that the omission of global signal regression from our pre-processing pipeline induced a consistent upward shift of eFC weights, analogous to its effect on nFC (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">4</a>). Additionally, we found that the overall pattern of eFC calculated using edge time series estimated from observed data was uncorrelated with the pattern of eFC calculated using edge time series estimated from phase-randomized surrogate time series (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">5</a>).</p><p>Next, we asked whether eFC exhibits any clear spatial dependence, as nFC is known to decay as a function of Euclidean distance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Esfahlani, F. Z., Bertolero, M. A., Bassett, D. S. &amp; Betzel, R. F. Space-independent community and hub structure of functional brain networks. Neuroimage 211, 116612 (2020)." href="/articles/s41593-020-00719-y#ref-CR21" id="ref-link-section-d30515570e862">21</a></sup>. We assessed the spatial dispersion of eFC with the surface area of the quadrilateral formed by the centroids of the brain region pairs (we explore an alternative edge-level distance metric in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">6</a>). We found evidence of a weak negative relationship between surface area and eFC (<i>r</i> = −0.14, <i>P</i> &lt; 10<sup>−15</sup>; <i>n</i> = 197,995,050 edge–edge pairs; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2d</a>), suggesting that, unlike traditional nFC, whose connection weights are more strongly influenced by spatial relationships of brain areas to one another, eFC is less constrained by the brain’s geometry.</p><p>Finally, we asked whether eFC bears the imprint of nFC communities—brain regions whose activity is highly correlated with members of its own community but weakly correlated or anti-correlated with members of other communities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Sporns, O. &amp; Betzel, R. F. Modular brain networks. Annu. Rev. Psychol. 67, 613–640 (2016)." href="/articles/s41593-020-00719-y#ref-CR7" id="ref-link-section-d30515570e887">7</a></sup>. To address this question, we classified every edge in the nFC network according to whether it fell within or between brain systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-020-00719-y#ref-CR22" id="ref-link-section-d30515570e891">22</a></sup>, resulting in three possible combinations of connections in the eFC graph: eFC could link edges that both fell within a community, edges that both fell between communities or an edge that fell within and an edge that fell between communities. In general, we found that eFC was significantly stronger for within-community edges compared to the other two categories (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2e</a>). Interestingly, we found that eFC could be distinguished further by dividing within-community edges by cognitive system<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-020-00719-y#ref-CR22" id="ref-link-section-d30515570e898">22</a></sup> (one-way analysis of variance (ANOVA); <i>F</i><sub>15,65819 </sub>= 2,667.4, <i>P</i> &lt; 10<sup>−15</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2f</a>).</p><h3 class="c-article__sub-heading" id="Sec4">eFC is stable within individuals</h3><p>In this section, we describe the robustness of eFC to scan duration—that is, how much data are required before eFC stabilizes and whether eFC is consistent across repeated scans of the same individual. To address these questions, we leveraged the within-individual design of the MSC dataset. For each participant, we aggregated their fMRI data across all scan sessions and estimated a single eFC matrix. Then, we sampled smaller amounts of temporally contiguous data, thus approximately preserving the auto-correlation structure, and estimated eFC, which we compared against the aggregated eFC matrix (this procedure was repeated 25 times). Similarly to other studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Laumann, T. O. et al. Functional system and areal organization of a highly sampled individual human brain. Neuron 87, 657–670 (2015)." href="/articles/s41593-020-00719-y#ref-CR23" id="ref-link-section-d30515570e923">23</a></sup>, we found that, with small amounts of data, eFC was highly variable (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig3">3a</a>). However, we observed a monotonic increase in similarity as a function of time, so that, with 30 min of data, the similarity was much greater (<i>r</i> = 0.78; <i>P</i> &lt; 10<sup>−15</sup>; <i>n</i> = 197,995,050 edge–edge pairs). This is of practical significance; like traditional nFC, it implies that eFC estimated using data from short scan sessions might not deliver accurate representations of an individual’s edge network organization. We note that this relationship is strengthened when data are sub-sampled randomly and uniformly (<i>r</i> = 0.90; <i>P</i> &lt; 10<sup>−15</sup>; <i>n</i> = 197,995,050 edge–edge pairs; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">7</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Intra- and inter-participant similarity of eFC across scan sessions."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Intra- and inter-participant similarity of eFC across scan sessions.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="175"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Correlation of session-averaged eFC matrices with eFC estimated using different amounts of data; the mean value is shown as a black line. <b>b</b>, Similarity of eFC within and between participants. Each block corresponds to data from a single participant; participants are also identifiable by the color of the rectangle alongside each block. <b>c</b>, Violin plots of within- and between-participant similarity values from the matrix shown in <b>b</b>; <i>n</i><sub><i>w</i></sub> = 450 and <i>n</i><sub><i>b</i></sub> = 4,500 within- and between-participant comparisons (two-sample <i>t</i>-test; (4,948) = 62.98; <i>P</i> &lt; 10<sup>−15</sup>). Box plots, shown in green and overlaid on data points in <b>c</b>, depict the IQR (box) and the median value of the distribution. Whiskers extend to the nearest points ±1.5× IQR above and below the 25th and 75th percentiles. <b>d</b>, Scan sessions plotted according to coordinates generated by performing a two-dimensional multidimensional scaling (MDS) analysis of the matrix in <b>b</b>. Note that scans from the same participant (shown here with the same color) are located near each other. All panels from this figure were generated using data from the MSC.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Next, we examined the reliability of eFC over multiple scan sessions. That is, if we imaged an individual on separate days, would their eFC on those days be more similar to each other than to that of a different individual? We estimated eFC and calculated the pairwise similarity (Pearson correlation) between all pairs of MSC participants and scans. We found eFC to be highly reliable in the MSC dataset, where the mean within-participant similarity was <i>r</i> = 0.53 ± 0.10 compared to <i>r</i> = 0.30 ± 0.07 between participants (two-sample <i>t</i>-test; (4,948) = 62.98; <i>P</i> &lt; 10<sup>−15</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig3">3b,c</a>). Indeed, we found that, for each eFC matrix, the matrix to which it was most similar belonged to the same participant (100% accuracy). Additionally, eFC exhibited slightly greater differential identifiability compared to nFC (0.224 to 0.210)—calculated as the difference between mean within- and between-participant similarity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Amico, E. &amp; Goñi, J. The quest for identifiability in human functional connectomes. Sci. Rep. 8, 8254 (2018)." href="/articles/s41593-020-00719-y#ref-CR24" id="ref-link-section-d30515570e1040">24</a></sup>. In Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig3">3d</a>, we show the results of applying multi-dimensional scaling to the similarity matrix from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig3">3b</a>. We found similar results in the HBN and HCP datasets (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">8</a>).</p><p>Collectively, these findings suggest that eFC exhibits a high level of participant specificity and captures idiosyncratic features of an individual, provided that eFC was estimated over a sufficiently long time period. This observation serves as an important validation of eFC and suggests that eFC might be useful in future applications as substrate for biomarker generation and ‘fingerprinting’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 1664–1671 (2015)." href="/articles/s41593-020-00719-y#ref-CR26" id="ref-link-section-d30515570e1056">26</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec5">The overlapping community structure of human cerebral cortex</h3><p>Although many studies have investigated the brain’s community structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Sporns, O. &amp; Betzel, R. F. Modular brain networks. Annu. Rev. Psychol. 67, 613–640 (2016)." href="/articles/s41593-020-00719-y#ref-CR7" id="ref-link-section-d30515570e1068">7</a></sup>, most have relied on methodology that forces each brain region into one and only one community. However, partitioning brain regions into non-overlapping communities clashes with evidence suggesting that cognition and behavior require contributions from regions that span multiple node-defined communities and systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Anderson, M. L., Kinnison, J. &amp; Pessoa, L. Describing functional diversity of brain regions and brain networks. Neuroimage 73, 50–58 (2013)." href="/articles/s41593-020-00719-y#ref-CR27" id="ref-link-section-d30515570e1072">27</a></sup>. Accordingly, a definition of communities is needed that more closely matches the brain’s multifunctional nature and the pervasive overlap of its community structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Pessoa, L. Understanding brain networks and brain organization. Phys. Life Rev. 11, 400–435 (2014)." href="/articles/s41593-020-00719-y#ref-CR28" id="ref-link-section-d30515570e1076">28</a></sup>.</p><p>Although deriving overlapping communities of brain regions can be challenging when using nFC, overlap is inherent (indeed, pervasive<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Ahn, Y.-Y., Bagrow, J. P. &amp; Lehmann, S. Link communities reveal multiscale complexity in networks. Nature 466, 761–764 (2010)." href="/articles/s41593-020-00719-y#ref-CR11" id="ref-link-section-d30515570e1083">11</a></sup>) within the eFC construct. Clustering the eFC matrix assigns each edge to a community. Each edge is associated with two brain regions (the nodes it connects). Thus, edge community assignments can be mapped back onto individual brain regions and, because every region is associated with <i>N</i> − 1 edges, allow regions to simultaneously maintain a plurality of community assignments.</p><p>In this section, we cluster eFC matrices to discover overlapping communities in human cerebral cortex. More specifically, we use a modified <i>k</i>-means algorithm to partition the eFC matrix into non-overlapping communities and map the edge assignments back to individual nodes.</p><p>In Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig4">4</a>, we show representative communities obtained with <i>k</i> = 10 (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">9</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">10</a> show examples with different numbers of communities). To demonstrate that the communities capture meaningful variance in our data, we show the edge co-fluctuation time series, the eFC matrix and the community co-assignment matrix reordered according to the derived communities (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig4">4a–c</a>). Here the elements of the co-assignment matrix represent the probability that two edges were assigned to the same community across partitions as we varied the number of communities from <i>k</i> = 2 to <i>k</i> = 20.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Edge communities."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Edge communities.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="537"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>We applied similarity-based clustering to eFC from the HCP dataset. Here we show results with the number of clusters fixed at <i>k</i> = 10. <b>a</b>, Here we reordered edge time series according to the detected community assignments. Horizontal lines divide communities from each other. The colors to the left of the time series plots identify each of the ten communities. <b>b</b>, We also reordered the rows and columns of the eFC matrix to highlight the same ten communities. Note that, on average, within-community eFC is greater than between-community eFC. <b>c</b>, We calculated the probability that pairs of edges (node pairs) were co-assigned to the same community. Here we show the co-assignment matrix with rows and columns reordered according to community assignments. Note that, in general, the range of co-assignment probabilities goes to 1. Here we truncate the color limits to emphasize the ten-community partition (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">9</a> shows the same co-assignment matrix at different values of <i>k</i> and with non-truncated color limits). We present two visualizations of the edge communities projected back to brain regions (nodes). <b>d</b>, We depict overlapping communities in matrix form. Each column in this matrix represents one of ten communities. For each community and for each node, we calculated the proportion of all edges assigned to the community that included that node as one of its endpoints (‘stubs’), indicated by the color and brightness of each cell. Dark colors indicate few edges; bright colors indicate many. <b>e</b>, Topographic distribution of communities. Note that many of the communities resemble known intrinsic connectivity networks. However, because the communities here can overlap, it is possible for nodes associated with a particular intrinsic connectivity network to participate in multiple edge communities.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Although the communities detected here are defined at the level of edges rather than nodes, we can project edge communities back onto brain regions. This was accomplished by extracting the edges associated with each community, determining which nodes were at the endpoints of each edge (the ‘stubs’) and counting the number of times that each node was represented in this stub list. We show these results in matrix form in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig4">4d</a>. In this panel, rows and columns represent nodes ordered according to the canonical system labels described in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-020-00719-y#ref-CR22" id="ref-link-section-d30515570e1173">22</a></sup>.</p><p>The overlapping nature of communities is made clearer in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig4">4e</a>, in which communities are represented topographically. The edges associated with the same visual nodes are all involved in communities 7, 8, 9 and 10 to some extent, thereby linking the visual system to multiple other brain systems. In community 8, for example, edges incident upon nodes in the visual and somatomotor systems are clustered together, whereas, in community 9, edges incident upon visual and control network nodes are assigned to the same community.</p><h3 class="c-article__sub-heading" id="Sec6">Community overlap and functional diversity of cognitive systems</h3><p>In the previous section, we showed that the human cerebral cortex could be partitioned into overlapping communities based on its edge correlation structure. This observation leads to a series of additional questions. For instance, which brain areas participate in many communities? Which participate in few? If we changed the scale of inquiry—the number of detected communities—do the answers to these questions change? Do the answers depend on which dataset we analyze? In this section, we explore these questions in detail.</p><p>One strategy for assessing community overlap is to simply count the number of different communities to which each nodes’ edges are assigned<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Yeo, B. T., Krienen, F. M., Chee, M. W. &amp; Buckner, R. L. Estimates of segregation and overlap of functional connectivity networks in the human cerebral cortex. Neuroimage 88, 212–227 (2014)." href="/articles/s41593-020-00719-y#ref-CR29" id="ref-link-section-d30515570e1195">29</a></sup>. A more nuanced measure that accounts for the distribution of edge community assignments is the normalized entropy, which indexes the uniformity of a distribution. We therefore calculated normalized entropy for every brain region while varying the number of communities from <i>k</i> = 2 to <i>k</i> = 20. In this section, we focus on results with <i>k</i> = 10.</p><p>We found that normalized community entropy was greatest within sensorimotor and attentional systems and lowest within regions traditionally associated with control and default mode networks (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig5">5a–c</a>). Notably, we obtained similar results from the MSC and HBN datasets (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">11</a>), at the level of individual participants (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">12</a>), as we varied the number of clusters (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">13</a>) and when using different parcellation schemes (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">14</a>). These observations seemingly contradict previous reports in which functional overlap was greatest in control networks and lowest in primary sensory systems (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig5">5d</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Power, J. D., Schlaggar, B. L., Lessov-Schlaggar, C. N. &amp; Petersen, S. E. Evidence for hubs in human functional brain networks. Neuron 79, 798–813 (2013)." href="/articles/s41593-020-00719-y#ref-CR6" id="ref-link-section-d30515570e1230">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Yeo, B. T., Krienen, F. M., Chee, M. W. &amp; Buckner, R. L. Estimates of segregation and overlap of functional connectivity networks in the human cerebral cortex. Neuroimage 88, 212–227 (2014)." href="/articles/s41593-020-00719-y#ref-CR29" id="ref-link-section-d30515570e1233">29</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Edge community entropy and overlap."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Edge community entropy and overlap.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="409"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Topographic distribution of normalized entropies. Normalized entropy, in this case, measures the uniformity of a node’s community assignments across all communities and serves as a measure of overlap. In general, higher entropy corresponds to greater levels of overlap. <b>b</b>, Brain systems associated with the highest levels of normalized entropy. These include visual, attentional, somatmotor and temporoparietal systems. <b>c</b>, Entropy values for all brain systems; <i>n</i> = 200 brain regions. Box plots, shown in green and overlaid on data points, depict the IQR (box) and the median value of the distribution. Whiskers extend to the nearest points ±1.5× IQR above and below the 25th and 75th percentiles. <b>d</b>, Here we highlight communities in which somatomotor (red) and visual (blue) systems are represented.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Is it possible to reconcile these seemingly opposed viewpoints? To address this question, we calculated a second measure of functional diversity. Whereas normalized entropy was defined at the level of individual brain regions based on the edge communities in which they participated, this second measure was defined at the level of brain systems as a whole and assessed the average similarity of edge community assignments among the system regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig6">6a,b</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00719-y#Sec13">Methods</a>). Intuitively, functionally diverse systems are comprised of brain regions whose edge community assignments are unique and dissimilar from one another. We found that regions within sensorimotor networks, which exhibited among the highest levels of entropy, exhibited the greatest levels of within-system similarity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig6">6c</a>). In contrast, sub-networks that make up the control network exhibited the lowest levels of within-system similarity, whereas their constituent nodes had among the lowest entropy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig6">6c</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="System-level similarity of edge communities."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: System-level similarity of edge communities.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="217"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Edge communities can be mapped into a [<i>N</i> × <i>N</i>] matrix. The element at row <i>i</i> and column <i>j</i> of the edge community matrix denotes the community label of edge {<i>i</i>,<i>j</i>}. <b>b</b>, We can then calculate the similarity of edge communities involving nodes <i>i</i> and <i>j</i> by comparing the values of columns <i>i</i> and <i>j</i>. This matrix depicts the similarity for all pairs of nodes. <b>c</b>, Within-system similarity values for each of the 16 pre-defined brain systems; <i>n</i> = 1,272 within-system similarity values. Box plots, shown in green and overlaid on data points, depict the IQR (box) and the median value of the distribution. Whiskers extend to the nearest points ±1.5× IQR above and below the 25th and 75th percentiles.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>In the Supplementary Material, we explore the relationship of normalized entropy with more familiar measures of overlap derived from nFC, including participation coefficient, dynamic flexibility and versatility (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">15</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">16</a>). We also compare patterns of normalized entropy derived from eFC community structure with entropy patterns obtained using alternative methods, including line graphs, the affiliation graph model, Bayesian non-negative matrix factorization and mixed-membership stochastic block models (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">17</a>).</p><h3 class="c-article__sub-heading" id="Sec7">eFC is modulated by changes in sensory input</h3><p>In the previous sections, we demonstrated that eFC is a reliable marker of an individual and that by clustering eFC we naturally obtain overlapping communities. We leveraged this final observation to demonstrate that sensorimotor and attentional systems participate in disproportionately more communities than association cortices. Analogous to previous studies documenting the effect of task on nodal FC, we expect that eFC is also modulated by task.</p><p>To address this question, we analyzed fMRI data from the HBN Serial Scanning Initiative recorded during rest and while participants passively viewed the movie ‘Raiders of the Lost Ark’. We estimated group-averaged eFC separately for each of the movie and rest scans.</p><p>In general, we found that eFC during movie watching was highly correlated with eFC estimated during rest (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7a</a>). Across six movie scans, the mean correlation with resting eFC was <i>r</i> = 0.55 ± 0.02 (all <i>P</i> &lt; 10<sup>−15</sup>; <i>n</i> = 197,995,050 edge–edge pairs). When we compared the pairwise similarity of all movie-watching scans with rest, we found that similarity of eFC was greater within a given task than between tasks (<i>P</i> &lt; 10<sup>−4</sup>, uniform and random permutation of movie and rest conditions; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7b</a>). To better understand what was driving this effect, we generated representative matrices for rest and movie conditions and computed the element-wise difference between these matrices. We contrasted these differences with those estimated after randomly permuting scan (condition) labels and found that 8.63% of all edge connections exhibited significant changes between conditions (permutation test; <i>P</i> &lt; 10<sup>−4</sup>; uncorrected). Although eFC differences were widespread, the most pronounced effects were associated with two specific communities (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7c</a>), one of which exhibited a decrease in its within-module eFC, whereas both decreased eFC with respect to each other. These communities consisted of edges associated with somatomotor and visual systems (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7d</a>). To confirm that these system-level effects were statistically significant, we compared the mean within- and between-system eFC differences against a constrained null model in which edges communities were randomly permuted (1,000 repetitions; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">18</a> shows a detailed schematic illustrating this procedure). As expected, the eFC involving systems 5 and 6 was significantly decreased from rest to movie (permutation test; false discovery rate fixed at 5%; <i>P</i><sub><i>adjusted</i></sub> = 3.7 × 10<sup>−5</sup>). Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">19a</a> shows the complete list of condition differences.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7" data-title="Effect of passive movie watching on eFC."><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 7: Effect of passive movie watching on eFC.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig7_HTML.png?as=webp"><img aria-describedby="Fig7" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_Fig7_HTML.png" alt="figure 7" loading="lazy" width="685" height="361"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p><b>a</b>, Two-dimensional histogram of eFC estimated at rest with eFC estimated during movie watching. <b>b</b>, Similarity of whole-brain eFC estimated at rest with movie watching. Note that within-condition similarity is greater for both conditions. <b>c</b>, Community-averaged differences in eFC. Communities 5 and 6 are associated with the largest magnitude differences, on average. Note that these are communities estimated from HBN data and are not identical to those shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig4">4</a>, which were estimated from HCP data. <b>d</b>, Topographic distribution of communities 5 and 6. Note that these communities involve edges associated with visual and somatomotor systems. <b>e</b>, Averaged differences in community overlap (normalized entropy); <i>n</i> = 200 brain regions whose entropy scores were compared across rest and movie-watching conditions (permutation test; mean difference in paired samples; <i>P</i> = 0.019). <b>f</b>, Similarity of whole-brain normalized entropy estimated at rest with movie watching. <b>g</b>, Violin plot showing system-specific differences in normalized entropy. Note that some of the greatest increases in entropy are concentrated with control and default mode networks; <i>n</i> = 200 brain regions. <b>h</b>, Topographic distribution of differences in entropy. Box plots, shown in green and overlaid on data points in <b>e</b> and <b>g</b>, depict the IQR (box) and the median value of the distribution. Whiskers extend to the nearest points ±1.5× IQR above and below the 25th and 75th percentiles.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00719-y/figures/7" data-track-dest="link:Figure7 Full size image" aria-label="Full size image figure 7" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The differences in the connection weights of eFC between movie watching and rest strongly suggested that the locations of high and low cluster overlap might also differ between conditions. To test this, we used the same clustering algorithm described earlier to partition node pairs into non-overlapping clusters and, based on these clusters, calculated each node’s cluster overlap as a normalized entropy. We found that, compared to rest, entropy increased during movie watching (permutation test; mean difference in paired samples; <i>P</i> = 0.019), indicating increased overlap between communities (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7e</a>), and that the brain-wide pattern of entropy also differed (permutation test; <i>P</i> &lt; 10<sup>−4</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7f</a>). We performed analogous tests at the level of individual brain regions and found that 28% of brain regions passed statistical testing (permutation test; false discovery rate fixed at 5%; <i>P</i><sub><i>adjusted</i></sub> = 0.014; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">19b</a>). We further tested whether these differences exhibited system-specific effects by calculating the mean change in entropy for each system and comparing it against mean changes after randomly and uniformly permuting system labels. We found that seven systems exhibited such effects, with increases concentrated within control and salience/ventral attention networks and decreases in dorsal attention temporal-parietal and visual systems (permutation of system labels; false discovery rate fixed at 5%; <i>P</i><sub><i>adjusted</i></sub> = 0.012; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7g,h</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">19c</a>).</p><p>Collectively, these results suggest that, like nFC, eFC is reconfigurable and can be modulated by sensory inputs. The observed changes in eFC, which implicated two clusters associated with both somatomotor and visual systems, are in close agreement with past studies of passive movie watching that documented changes in activity and nFC in similar areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wilf, M. et al. Spontaneously emerging patterns in human visual cortex reflect responses to naturalistic sensory stimuli. Cereb. Cortex 27, 750–763 (2017)." href="/articles/s41593-020-00719-y#ref-CR30" id="ref-link-section-d30515570e1532">30</a></sup>. We also found increased overlap in areas associated with control and default mode networks, which agrees with evidence that activity throughout these areas is sensitive to movie narrative structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Baldassano, C., Hasson, U. &amp; Norman, K. A. Representation of real-world event schemas during narrative perception. J. Neurosci. 38, 9689–9699 (2018)." href="/articles/s41593-020-00719-y#ref-CR31" id="ref-link-section-d30515570e1536">31</a></sup>. An important area of future research involves systematically assessing the effect of different cognitively demanding tasks on eFC.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Discussion</h2><div class="c-article-section__content" id="Sec8-content"><p>Here we presented a network model of human cerebral cortex that focused on edge–edge interactions. The network formed by these interactions—a construct we referred to as eFC—was similar across datasets and more similar within individuals than between them. When clustered, eFC provided a natural estimate of pervasively overlapping community structure. We found that the amount of overlap varied across the cortex but peaked in sensorimotor and attention networks. We found that brain regions associated with sensorimotor and attention networks participated in disproportionately many communities compared to other brain systems, but that, relative to one another, those same regions participated in similar sets of communities. Lastly, we showed that eFC and community overlap varied systematically during passive viewing of movies.</p><h3 class="c-article__sub-heading" id="Sec9">Edge-centric perspective on functional network organization</h3><p>Node-centric representations have dominated the field of network neuroscience and have served as the basis for nearly every discovery within that field<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Bassett, D. S. &amp; Sporns, O. Network neuroscience. Nat. Neurosci. 20, 353–364 (2017)." href="/articles/s41593-020-00719-y#ref-CR1" id="ref-link-section-d30515570e1556">1</a></sup>. The edge-centric representation shifts focus away from dyadic relationships between nodal activations and, instead, onto the interactions between edges (similarity in patterns of co-fluctuation, a potential hallmark of communication). Although related models have been explored in other scientific domains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Ahn, Y.-Y., Bagrow, J. P. &amp; Lehmann, S. Link communities reveal multiscale complexity in networks. Nature 466, 761–764 (2010)." href="/articles/s41593-020-00719-y#ref-CR11" id="ref-link-section-d30515570e1560">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Evans, T. &amp; Lambiotte, R. Line graphs, link partitions, and overlapping communities. Phys. Rev. E 80, 016105 (2009)." href="/articles/s41593-020-00719-y#ref-CR12" id="ref-link-section-d30515570e1563">12</a></sup>, including neuroscience, where they were first used in a study to represent interacting white matter tracts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="de Reus, M. A., Saenger, V. M., Kahn, R. S. &amp; van den Heuvel, M. P. An edge-centric perspective on the human connectome: link communities in the brain. Philos. Trans. R. Soc. B Biol. Sci. 369, 20130527 (2014)." href="/articles/s41593-020-00719-y#ref-CR14" id="ref-link-section-d30515570e1567">14</a></sup>, they require, as input, sparse node–node connectivity matrices and are poorly suited for continuous-valued time series data.</p><p>In this study, we developed a novel edge-centric representation of functional neuroimaging data that operates directly on observed time series. Our method for estimating connection weights between edges can be viewed as a temporal ‘unwrapping’ of the familiar Pearson correlation—the measure frequently used to estimate the magnitude of nFC between pairs of brain regions. Whereas the Pearson correlation coefficient calculates the time-averaged co-fluctuation magnitude for node pairs, we simply omit the averaging step, yielding ‘edge time series’, which represent the co-fluctuation magnitude between two nodes at every instant in time. This simple step enables us to track fluctuations in edge weight across time and, critically, allow for dyadic relationships between edges, creating an edge-centric representation of nervous system architecture (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig1">1</a>). If we interpret edge time series as a temporal unwrapping of nFC, which is thought to reflect the aggregate effect of communication processes between neural elements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Avena-Koenigsberger, A., Misic, B. &amp; Sporns, O. Communication dynamics in complex brain networks. Nat. Rev. Neurosci. 19, 17 (2018)." href="/articles/s41593-020-00719-y#ref-CR32" id="ref-link-section-d30515570e1577">32</a></sup>, then edge times series track, with high temporal resolution, the communication patterns between distributed neural elements.</p><p>We note that our edge-centric approach is conceptually similar to several existing methods. For instance, ‘multiplication of temporal derivatives’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Shine, J. M. et al. Estimation of dynamic functional connectivity using multiplication of temporal derivatives. Neuroimage 122, 399–407 (2015)." href="/articles/s41593-020-00719-y#ref-CR33" id="ref-link-section-d30515570e1584">33</a></sup> calculates the element-wise products using differenced activity time series for all pairs of nodes. These time series are then convolved with a kernel to generate smooth estimates of time-varying nFC. Although similar, our approach relies on untransformed activity to estimate edge time series, thereby preserving the relationship between static nFC and the mean value of each edge time series. Another related method is ‘co-activation patterns’ (CAPs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Liu, X. &amp; Duyn, J. H. Time-varying functional network information extracted from brief instances of spontaneous brain activity. Proc. Natl Acad. Sci. USA 110, 4392–4397 (2013)." href="/articles/s41593-020-00719-y#ref-CR34" id="ref-link-section-d30515570e1588">34</a></sup>, which extracts and clusters voxel- or vertex-level activity during high-activity frames. Because a voxel can be co-active under different contexts, the cluster centroids spatially overlap with one another. Although both CAPs and eFC result in overlapping structures, they operate on distinct substrates, with CAPs focusing on activity and eFC focusing on similarity of co-activity. Although CAPs requires the specification of additional parameters compared to eFC—for example, the threshold for a high-activity frame—CAPs might scale better owing to the focus on activity rather than connectivity.</p><p>Finally, we note that nFC and eFC are both frameworks for investigating pairwise relationships from neural time series. Critically, however, nFC and eFC differ in terms of what elements are being related to one another and how we interpret those relationships. In the case of nFC, correlations refer to similarities in the activity of individual neural elements, often interpreted as two parts of the brain ‘talking’ to one another. In the case of eFC, on the other hand, correlations express similarities in co-fluctuations along edges, which might loosely be interpreted as ‘conversations’ between node pairs (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">1</a>). In other words, nFC focuses on co-activation between nodes whereas eFC focuses on co-fluctuation along edges. In this way, nFC and eFC should be viewed as complementary approaches that can reveal unique organizational features of nervous systems.</p><h3 class="c-article__sub-heading" id="Sec10">Overlapping communities extend understanding of system-level cortical organization</h3><p>Here we demonstrated that clustering eFC using community detection methods naturally leads to communities that overlap when mapped back to the level of brain regions and nodes. Past investigations of cortical organization have focused almost exclusively on non-overlapping communities. The decision to define communities in this way is partially motivated by interpretability but also by limitations of the methods used to detect communities, which assign nodes to one community only<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Newman, M. E. &amp; Girvan, M. Finding and evaluating community structure in networks. Phys. Rev. E 69, 026113 (2004)." href="/articles/s41593-020-00719-y#ref-CR35" id="ref-link-section-d30515570e1606">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Rosvall, M. &amp; Bergstrom, C. T. Maps of random walks on complex networks reveal community structure. Proc. Natl Acad. Sci. USA 105, 1118–1123 (2008)." href="/articles/s41593-020-00719-y#ref-CR36" id="ref-link-section-d30515570e1609">36</a></sup>. This current view of communities has been profoundly successful<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Sporns, O. &amp; Betzel, R. F. Modular brain networks. Annu. Rev. Psychol. 67, 613–640 (2016)." href="/articles/s41593-020-00719-y#ref-CR7" id="ref-link-section-d30515570e1613">7</a></sup>. It provides a low-dimensional description of the brain, it can be used to define node roles and detect hubs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Bertolero, M. A., Yeo, B. T. &amp; D’Esposito, M. The modular and integrative functional architecture of the human brain. Proc. Natl Acad. Sci. USA 112, E6798–E6807 (2015)." href="/articles/s41593-020-00719-y#ref-CR37" id="ref-link-section-d30515570e1617">37</a></sup>, and it can be applied to both anatomical and functional networks with equal success.</p><p>The dominant non-overlapping perspective of communities has strongly influenced how we think about brain function. Because functional communities exhibit reliable correspondence with patterns of task-evoked activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Smith, S. M. et al. Correspondence of the brain’s functional architecture during activation and rest. Proc. Natl Acad. Sci. USA 106, 13040–13045 (2009)." href="/articles/s41593-020-00719-y#ref-CR38" id="ref-link-section-d30515570e1624">38</a></sup>, we have come to associate individual communities with specific cognitive domains. For instance, it is not uncommon to refer to communities as primarily processing visual information, enacting cognitive control or performing attentional functions. This localization of brain function to communities, although likely a reasonable first-order approximation, perpetuates a view of brain function in which brain areas, systems and communities are fundamentally unifunctional. Such a view, however, disagrees with observations that many aspects of cognition and behavior transcend these traditional community labels.</p><p>Another perspective is that overlap arises from time-varying fluctuations in community structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bassett, D. S. et al. Dynamic reconfiguration of human brain networks during learning. Proc. Natl Acad. Sci. USA 108, 7641–7646 (2011)." href="/articles/s41593-020-00719-y#ref-CR25" id="ref-link-section-d30515570e1631">25</a></sup>. That is, at any given instant, communities are non-overlapping but appear ‘fuzzy’ due to nodes changing their community allegiances over time. The approach developed here is closely aligned with the perspective that brain areas and communities are dynamic and exhibit highly degenerate functionality. Other studies have investigated overlapping and dynamic communities by studying overlap in co-activation or through the use of sliding window analysis and multi-layer models to detect flexible regions that change their community assignment over time. Our approach, however, is distinct, emphasizing a state of pervasive overlap<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Ahn, Y.-Y., Bagrow, J. P. &amp; Lehmann, S. Link communities reveal multiscale complexity in networks. Nature 466, 761–764 (2010)." href="/articles/s41593-020-00719-y#ref-CR11" id="ref-link-section-d30515570e1635">11</a></sup> in which nodes belong to several communities instantaneously.</p><h3 class="c-article__sub-heading" id="Sec11">Limitations</h3><p>One of the most important limitations concerns the estimation of edge time series from functional imaging data. To calculate edge time series, we first <i>z</i>-scored regional time series. Here, the <i>z</i>-score is appropriate only if the time series has a temporally invariant mean and s.d. If there is a sustained increase or decrease in activity—for example, the effect of a blocked task—then the <i>z</i>-scoring procedure can result in a biased mean and s.d., resulting in poor estimates of fluctuations in activity. In future work, investigation of task-evoked changes in eFC could be investigated with already common pre-processing steps—for example, constructing task regressors to remove the first-order effect of tasks on activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Cole, M. W. et al. Task activations produce spurious but systematic inflation of task functional connectivity estimates. Neuroimage 189, 1–18 (2019)." href="/articles/s41593-020-00719-y#ref-CR39" id="ref-link-section-d30515570e1656">39</a></sup>.</p><p>Another limitation concerns the scalability of eFC. Calculating eFC given for a brain divided into N parcels results in an eFC matrix of dimensions <span class="mathjax-tex">\(\frac{{N\left( {N - 1} \right)}}{2}\)</span>. This means that an increase in the number of parcels results in a squared increase in the dimensionality of eFC. If the number of parcels is large, this can result in massive, fully weighted matrices that require large amounts of memory to store and manipulate. In the future, however, it might be necessary to explore dimension reduction methods to retain the most relevant sub-graphs for a given task or set of behaviors.</p><h3 class="c-article__sub-heading" id="Sec12">Future directions</h3><p>Although eFC characterizes interactions between edges rather than nodes, it can still be analyzed using the same methods previously applied to nFC. We can use graph theory to detect its hubs and communities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Power, J. D., Schlaggar, B. L., Lessov-Schlaggar, C. N. &amp; Petersen, S. E. Evidence for hubs in human functional brain networks. Neuron 79, 798–813 (2013)." href="/articles/s41593-020-00719-y#ref-CR6" id="ref-link-section-d30515570e1709">6</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">20</a> shows examples), estimate edge gradients<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-020-00719-y#ref-CR40" id="ref-link-section-d30515570e1716">40</a></sup> and compare eFC connection weights across individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 1664–1671 (2015)." href="/articles/s41593-020-00719-y#ref-CR26" id="ref-link-section-d30515570e1720">26</a></sup> and conditions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Cole, M. W. et al. Task activations produce spurious but systematic inflation of task functional connectivity estimates. Neuroimage 189, 1–18 (2019)." href="/articles/s41593-020-00719-y#ref-CR39" id="ref-link-section-d30515570e1724">39</a></sup>. On the other hand, eFC affords many new opportunities, beginning with the edge time series used to estimate eFC. Essentially, edge time series offer a moment-to-moment assessment of how strongly two nodes (brain regions) co-fluctuate with one another, providing an estimate of time-varying nFC without the requirement that we specify a window<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Esfahlani, F. Z. et al. High-amplitude co-fluctuations in cortical activity drive functional connectivity. Preprint at bioRxiv 
https://doi.org/10.1101/800045

 (2020)." href="/articles/s41593-020-00719-y#ref-CR41" id="ref-link-section-d30515570e1729">41</a></sup>. This overcomes one of the main limitations of sliding window estimates of time-varying nFC, namely that the use of a window leads to a ‘blurring’ of events across time<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Lurie, D. J. et al. Questions and controversies in the study of time-varying functional connectivity in resting fMRI. Netw. Neurosci. 4, 30–69 (2020)." href="/articles/s41593-020-00719-y#ref-CR42" id="ref-link-section-d30515570e1733">42</a></sup>. Other directions for future work include developing whole-brain functional atlases with overlapping system labels and applications to specific brain areas and sub-systems for constructing fine-grained overlapping atlases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="King, M., Hernandez-Castillo, C. R., Poldrack, R. A., Ivry, R. B. &amp; Diedrichsen, J. Functional boundaries in the human cerebellum revealed by a multi-domain task battery. Nat. Neurosci. 22, 1371–1378 (2019)." href="/articles/s41593-020-00719-y#ref-CR43" id="ref-link-section-d30515570e1737">43</a></sup>. We note, also, that, because the derivation of eFC is based on Pearson correlations, it would be straightforward to estimate analogs of eFC based on lagged and partial relationships.</p><p>eFC might be useful in applications of machine learning and classification of neuroimaging data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Pereira, F., Mitchell, T. &amp; Botvinick, M. Machine learning classifiers and fMRI: a tutorial overview. Neuroimage 45, S199–S209 (2009)." href="/articles/s41593-020-00719-y#ref-CR44" id="ref-link-section-d30515570e1744">44</a></sup>. The dimensionality of the eFC matrix is much greater than that of a typical nFC matrix. We speculate that some of the added dimensions might be useful for studying brain–behavior relationships—for example, by identifying manifolds along which individuals, clinical cohorts or behaviors naturally separate, enhancing classification accuracy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Huys, Q. J., Maia, T. V. &amp; Frank, M. J. Computational psychiatry as a bridge from neuroscience to clinical applications. Nat. Neurosci. 19, 404 (2016)." href="/articles/s41593-020-00719-y#ref-CR45" id="ref-link-section-d30515570e1748">45</a></sup> (the results of exploratory analyses of brain–behavior relationships based on eFC are shown in Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">23</a>). On the other hand, the increased dimensionality of eFC requires special considerations, as it presents statistical and interpretational challenges. Multivariate methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="McIntosh, A. R. &amp; Mišić, B. Multivariate statistical analyses for neuroimaging data. Annu. Rev. Psychol. 64, 499–525 (2013)." href="/articles/s41593-020-00719-y#ref-CR46" id="ref-link-section-d30515570e1758">46</a></sup>, such as canonical correlation analysis or partial least squares, both of which can help circumvent multiple comparison issues, might prove useful and should be investigated in future brain–behavior analysis involving eFC.</p><p>Additionally, future studies should investigate appropriate null models for eFC. Like nFC, eFC is correlation based, and the weights of edge–edge connections are not independent of one another<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. Neuroimage 60, 2096–2106 (2012)." href="/articles/s41593-020-00719-y#ref-CR47" id="ref-link-section-d30515570e1765">47</a></sup>. This means that rewiring-based null models (which treat connections as independent) are not appropriate. Consideration should be given to other classes of null models, including time-series-based surrogates. Appropriate null models might help clarify brain–behavior relationships in future studies.</p><p>The framework proposed here for investigating interactions between pairs of nodes can be generalized to study mutual interactions between many more nodes by simply calculating the element-wise product of node triplets, quartets and quintets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Owen, L. L., Chang, T. H. &amp; Manning, J. R. High-level cognition during story listening is reflected in high-order dynamic correlations in neural activity patterns. Preprint at bioRxiv 
https://doi.org/10.1101/763821

 (2019)." href="/articles/s41593-020-00719-y#ref-CR48" id="ref-link-section-d30515570e1772">48</a></sup>. This extension is, in some respects, analogous to recent applications of algebraic topology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Sizemore, A. E., Phillips-Cremins, J. E., Ghrist, R. &amp; Bassett, D. S. The importance of the whole: topological data analysis for the network neuroscientist. Netw. Neurosci. 3, 656–673 (2019)." href="/articles/s41593-020-00719-y#ref-CR49" id="ref-link-section-d30515570e1776">49</a></sup>, which can uncover higher-order relationships in a network (through graph simplices). We note, however, that, although generating higher-order time series is straightforward, it is necessarily accompanied by an increase in dimensionality, potentially making the approach computationally intractable for whole-brain networks. On the other hand, higher-order time series (and their corresponding eFC analogs) might be useful for investigating the organization of predefined circuits composed of relatively few brain regions or nuclei.</p><p>Lastly, the edge-centric framework developed here is not limited to fMRI and can be easily extended to different recording modalities, including scalp/intracranial electroencephalography or magnetoencephalography, which makes it possible to track seizure propagation at the level of edges<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Khambhati, A. N. et al. Dynamic network drivers of seizure generation, propagation and termination in human neocortical epilepsy. PLoS Comput. Biol. 11, e1004608 (2015)." href="/articles/s41593-020-00719-y#ref-CR50" id="ref-link-section-d30515570e1784">50</a></sup>. Similarly, the application of this approach to datasets resolving single-neuron activity could uncover important connection-level insights into circuit organization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Dann, B., Michaels, J. A., Schaffelhofer, S. &amp; Scherberger, H. Uniting functional network topology and oscillations in the fronto-parietal single unit network of behaving primates. eLife 5, e15719 (2016)." href="/articles/s41593-020-00719-y#ref-CR3" id="ref-link-section-d30515570e1788">3</a></sup>.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec13-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec13">Methods</h2><div class="c-article-section__content" id="Sec13-content"><p>In this study, we used data from three independently acquired, openly available neuroimaging datasets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Van Essen, D. C. et al. The WU-Minn Human Connectome Project: an overview. Neuroimage 80, 62–79 (2013)." href="#ref-CR17" id="ref-link-section-d30515570e1801">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gordon, E. M. et al. Precision functional mapping of individual human brains. Neuron 95, 791–807 (2017)." href="#ref-CR18" id="ref-link-section-d30515570e1801_1">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="O’Connor, D. et al. The healthy brain network serial scanning initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions. Gigascience 6, giw011 (2017)." href="/articles/s41593-020-00719-y#ref-CR19" id="ref-link-section-d30515570e1804">19</a></sup> and, therefore, did not collect any data for this study. No statistical methods were used to pre-determine sample sizes, but our sample sizes are similar to those reported in previous publications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Power, J. D., Schlaggar, B. L., Lessov-Schlaggar, C. N. &amp; Petersen, S. E. Evidence for hubs in human functional brain networks. Neuron 79, 798–813 (2013)." href="/articles/s41593-020-00719-y#ref-CR6" id="ref-link-section-d30515570e1808">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="de Reus, M. A., Saenger, V. M., Kahn, R. S. &amp; van den Heuvel, M. P. An edge-centric perspective on the human connectome: link communities in the brain. Philos. Trans. R. Soc. B Biol. Sci. 369, 20130527 (2014)." href="/articles/s41593-020-00719-y#ref-CR14" id="ref-link-section-d30515570e1811">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Gordon, E. M. et al. Precision functional mapping of individual human brains. Neuron 95, 791–807 (2017)." href="/articles/s41593-020-00719-y#ref-CR18" id="ref-link-section-d30515570e1814">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 1664–1671 (2015)." href="/articles/s41593-020-00719-y#ref-CR26" id="ref-link-section-d30515570e1817">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Shine, J. M. et al. Human cognition involves the dynamic integration of neural activity and neuromodulatory systems. Nat. Neurosci. 22, 289–296 (2019)." href="/articles/s41593-020-00719-y#ref-CR51" id="ref-link-section-d30515570e1820">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Davison, E. N. et al. Brain network adaptability across task states. PLoS Comput. Biol. 11, e1004029 (2015)." href="/articles/s41593-020-00719-y#ref-CR52" id="ref-link-section-d30515570e1823">52</a></sup>, and represent either all usable data (MSC and HBN) or a subset pre-selected by the study coordinators (HCP). We did not perform any randomization of participants into experimental groups, and we opted to analyze each dataset separately. Appropriate counterbalancing of task conditions was performed by the authors of the original studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Van Essen, D. C. et al. The WU-Minn Human Connectome Project: an overview. Neuroimage 80, 62–79 (2013)." href="#ref-CR17" id="ref-link-section-d30515570e1827">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gordon, E. M. et al. Precision functional mapping of individual human brains. Neuron 95, 791–807 (2017)." href="#ref-CR18" id="ref-link-section-d30515570e1827_1">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="O’Connor, D. et al. The healthy brain network serial scanning initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions. Gigascience 6, giw011 (2017)." href="/articles/s41593-020-00719-y#ref-CR19" id="ref-link-section-d30515570e1830">19</a></sup>. Data analysis was not performed blinded to the conditions of the experiments. Blinding was not relevant because participants were not evaluated based on group membership, and blinding is not applicable to the whole-group analyses reported in this study. All analyses were performed with MATLAB (MathWorks, Inc.) version 2019a. Further study design and statistical details can be found in the Life Sciences Reporting Summary available online.</p><h3 class="c-article__sub-heading" id="Sec14">Datasets</h3><p>The HCP dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Van Essen, D. C. et al. The WU-Minn Human Connectome Project: an overview. Neuroimage 80, 62–79 (2013)." href="/articles/s41593-020-00719-y#ref-CR17" id="ref-link-section-d30515570e1841">17</a></sup> included resting state functional magnetic resonance imaging (rsfMRI) data from 100 unrelated adult participants (54% female; mean age = 29.11 ± 3.67 years; age range, 22–36 years). These participants were selected as they comprised the ‘100 Unrelated Subjects’ released by the HCP. The study was approved by the Washington University Institutional Review Board, and informed consent was obtained from all participants. Participants underwent four 15-min rsfMRI scans over a 2-day period. A full description of the imaging parameters and image pre-processing can be found in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-020-00719-y#ref-CR53" id="ref-link-section-d30515570e1845">53</a></sup>. The rsfMRI data were acquired with a gradient-echo echo-planar imaging (EPI) sequence (run duration = 14:33 min, TR = 720 ms, TE = 33.1 ms, flip angle = 52°, 2-mm isotropic voxel resolution, multi-band factor = 8) with eyes open and instructions to fixate on a cross. Images were collected on a 3T Siemens Connectome Skyra with a 32-channel head coil.</p><p>The MSC dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Gordon, E. M. et al. Precision functional mapping of individual human brains. Neuron 95, 791–807 (2017)." href="/articles/s41593-020-00719-y#ref-CR18" id="ref-link-section-d30515570e1852">18</a></sup> included rsfMRI from 10 adults (50% female; mean age = 29.1 ± 3.3 years; age range, 24–34 years). The study was approved by the Washington University School of Medicine Human Studies Committee and Institutional Review Board, and informed consent was obtained from all participants. Participants underwent 12 scanning sessions on separate days, each session beginning at 24:00. Ten rsfMRI scans per participant were collected with a gradient-echo EPI sequence (run duration = 30 min, TR = 2,200 ms, TE = 27 ms, flip angle = 90°, 4-mm isotropic voxel resolution) with eyes open and with eye tracking recording to monitor for prolonged eye closure (to assess drowsiness). Images were collected on a 3T Siemens Trio.</p><p>The HBN Serial Scanning Initiative dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="O’Connor, D. et al. The healthy brain network serial scanning initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions. Gigascience 6, giw011 (2017)." href="/articles/s41593-020-00719-y#ref-CR19" id="ref-link-section-d30515570e1859">19</a></sup> included rsfMRI and movie-watching functional magnetic resonance imaging (mvfMRI) from 13 adults (54% female; mean age = 30.3 ± 6.4 years; age range, 21–42 years). Three participants of the HBN dataset did not have enough non-outlier functional scans (see quality control criteria below) to be meaningfully analyzed (non-outlier scan percentage = 7%, 0% and 0%) and were excluded entirely from the current study. This rendered the HBN dataset as ten participants (50% female; mean age = 29.8 ± 5.3 years; age range, 23–37 years). The study was approved by the Chesapeake Institutional Review Board, and informed consent was obtained from all participants. Participants underwent 14 scanning sessions over a 1–2-month period, in which 13 rsfMRI runs were acquired per participant. On the eighth session, participants viewed the movie ‘Raiders of the Lost Ark’ (Lucasfilm) in six approximately 20-min scans. The rsfMRI and mvfMRI were acquired with a gradient-echo EPI sequence (run duration rsfMRI = 10:18 min per segment, run duration mvfMRI = 20 min per segment, TR = 1,450 ms, TE = 40 ms, flip angle = 55°, 2.46 × 2.46 × 2.5-mm voxel resolution, multi-band factor = 3) with participants instructed to keep their eyes open and gaze directed toward a cross during the fsMRI scan. Images were collected on a 1.5T Siemens Avanto with a 32-channel head coil. The mvfMRI was divided into six successive scan sessions, which we further truncated by retaining the first 420 samples so that the duration matched that of the HBN rsfMRI, of which we retained the first six for the sake of balance.</p><h3 class="c-article__sub-heading" id="Sec15">Image pre-processing</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec16">HCP functional pre-processing</h4><p>Functional images in the HCP dataset were minimally pre-processed according to the description provided in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-020-00719-y#ref-CR53" id="ref-link-section-d30515570e1875">53</a></sup>. Briefly, these data were corrected for gradient distortion, susceptibility distortion and motion and then aligned to a corresponding T1-weighted (T1w) image with one spline interpolation step. This volume was further corrected for intensity bias and normalized to a mean of 10,000. This volume was then projected to the 32k_fs_LR mesh, excluding outliers, and aligned to a common space using a multi-modal surface registration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Robinson, E. C. et al. MSM: a new flexible framework for multimodal surface matching. Neuroimage 100, 414–426 (2014)." href="/articles/s41593-020-00719-y#ref-CR54" id="ref-link-section-d30515570e1879">54</a></sup>. The resultant <span class="u-small-caps">cifti</span> file for each HCP individual who participated in this study followed the file naming pattern *REST{1,2}_{LR,RL}_Atlas_MSMAll.dtseries.nii.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">MSC and HBN functional pre-processing</h4><p>Functional images in the MSC and HBN datasets were pre-processed using fMRIPrep 1.3.2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Esteban, O. et al. fMRIPrep: a robust preprocessing pipeline for functional MRI. Nat. Methods 16, 111–116 (2019)." href="/articles/s41593-020-00719-y#ref-CR55" id="ref-link-section-d30515570e1894">55</a></sup>), which is based on Nipype 1.1.9 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Gorgolewski, K. et al. Nipype: a flexible, lightweight and extensible neuroimaging data processing framework in Python. Front. Neuroinform. 5, 13 (2011)." href="/articles/s41593-020-00719-y#ref-CR56" id="ref-link-section-d30515570e1898">56</a></sup>). The following description of fMRIPrep’s pre-processing is based on boilerplate distributed with the software covered by a ‘no rights reserved’ (CC0) license. Internal operations of fMRIPrep use Nilearn 0.5.0 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Abraham, A. et al. Machine learning for neuroimaging with scikit-learn. Front. Neuroinform. 8, 14 (2014)." href="/articles/s41593-020-00719-y#ref-CR57" id="ref-link-section-d30515570e1902">57</a></sup>), ANTs 2.2.0, FreeSurfer 6.0.1, FSL 5.0.9 and AFNI v16.2.07. For more details about the pipeline, see the section corresponding to workflows in fMRIPrep’s documentation.</p><p>T1w images were corrected for intensity non-uniformity with N4BiasFieldCorrection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Avants, B. B., Epstein, C. L., Grossman, M. &amp; Gee, J. C. Symmetric diffeomorphic image registration with cross-correlation: evaluating automated labeling of elderly and neurodegenerative brain. Med. Image Anal. 12, 26–41 (2008)." href="/articles/s41593-020-00719-y#ref-CR58" id="ref-link-section-d30515570e1909">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Tustison, N. J. et al. N4ITK: improved N3 bias correction. IEEE Trans. Med. Imaging 29, 1310–1320 (2010)." href="/articles/s41593-020-00719-y#ref-CR59" id="ref-link-section-d30515570e1912">59</a></sup>, distributed with ANTs and used as T1w-reference throughout the workflow. The T1w-reference was then skull-stripped with a Nipype implementation of the antsBrainExtraction.sh workflow, using NKI as the target template. Brain surfaces were reconstructed using recon-all<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Dale, A. M., Fischl, B. &amp; Sereno, M. I. Cortical surface-based analysis: I. Segmentation and surface reconstruction. Neuroimage 9, 179–194 (1999)." href="/articles/s41593-020-00719-y#ref-CR60" id="ref-link-section-d30515570e1916">60</a></sup>, and the brain mask estimated previously was refined with a custom variation of the method to reconcile ANTs-derived and FreeSurfer-derived segmentations of the cortical gray matter using Mindboggle<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Klein, A. et al. Mindboggling morphometry of human brains. PLoS Comput. Biol. 13, e1005350 (2017)." href="/articles/s41593-020-00719-y#ref-CR61" id="ref-link-section-d30515570e1920">61</a></sup>. Spatial normalization to the ICBM 152 Nonlinear Asymmetrical template version 2009c<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Fonov, V. S., Evans, A. C., McKinstry, R. C., Almli, C. &amp; Collins, D. Unbiased nonlinear average age-appropriate brain templates from birth to adulthood. Neuroimage 47, S102 (2009)." href="/articles/s41593-020-00719-y#ref-CR62" id="ref-link-section-d30515570e1924">62</a></sup> was performed through nonlinear registration with antsRegistration, using brain-extracted versions of both T1w volume and template. Brain tissue segmentation of cerebrospinal fluid, white matter and gray matter was performed on the brain-extracted T1w images using FSL’s fast<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Zhang, Y., Brady, M. &amp; Smith, S. Segmentation of brain MR images through a hidden Markov random field model and the expectation-maximization algorithm. IEEE Trans. Med. Imaging 20, 45–57 (2001)." href="/articles/s41593-020-00719-y#ref-CR63" id="ref-link-section-d30515570e1928">63</a></sup>.</p><p>Functional data were slice-time corrected using AFNI’s 3dTshift and motion corrected using FSL’s mcflirt<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Jenkinson, M., Bannister, P., Brady, M. &amp; Smith, S. Improved optimization for the robust and accurate linear registration and motion correction of brain images. Neuroimage 17, 825–841 (2002)." href="/articles/s41593-020-00719-y#ref-CR64" id="ref-link-section-d30515570e1935">64</a></sup>. Fieldmap-less distortion correction was performed by co-registering the functional image to the same-participant T1w image with intensity inverted<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Wang, S. et al. Evaluation of field map and nonlinear registration methods for correction of susceptibility artifacts in diffusion MRI. Front. Neuroinform. 11, 17 (2017)." href="/articles/s41593-020-00719-y#ref-CR65" id="ref-link-section-d30515570e1939">65</a></sup> and constrained with an average fieldmap template<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Treiber, J. M. et al. Characterization and correction of geometric distortions in 814 diffusion weighted images. PLoS ONE 11, e0152472 (2016)." href="/articles/s41593-020-00719-y#ref-CR66" id="ref-link-section-d30515570e1943">66</a></sup>, implemented with antsRegistration. This was followed by co-registration to the corresponding T1w image using boundary-based registration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Greve, D. N. &amp; Fischl, B. Accurate and robust brain image alignment using boundary-based registration. Neuroimage 48, 63–72 (2009)." href="/articles/s41593-020-00719-y#ref-CR67" id="ref-link-section-d30515570e1947">67</a></sup> with 9° of freedom. Motion-correcting transformations, field-distortion-correcting warp, BOLD-to-T1w transformation and T1w-to-template (MNI) warp were concatenated and applied in a single step using antsApplyTransforms using Lanczos interpolation. Several confounding time series were calculated based on this pre-processed BOLD: framewise displacement (FD), DVARS and three region-wise global signals. FD and DVARS are calculated for each functional run, both using their implementations in Nipype<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Power, J. D. et al. Methods to detect, characterize, and remove motion artifact in resting state fMRI. Neuroimage 84, 320–341 (2014)." href="/articles/s41593-020-00719-y#ref-CR68" id="ref-link-section-d30515570e1951">68</a></sup>. The three global signals are extracted within the cerebrospinal fluid, the white matter and the whole-brain masks. The resultant nifti file for each MSC and HBN individual who participated in this study followed the file naming pattern *space-T1w_desc-preproc_bold.nii.gz.</p><h3 class="c-article__sub-heading" id="Sec18">Image quality control</h3><p>All functional images in the HCP and MSC datasets were retained. The quality of functional images in the MSC and HBN datasets were assessed using fMRIPrep’s visual reports and MRIQC 0.15.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Esteban, O. et al. MRIQC: advancing the automatic prediction of image quality in MRI from unseen sites. PLoS ONE 12, e0184661 (2017)." href="/articles/s41593-020-00719-y#ref-CR69" id="ref-link-section-d30515570e1964">69</a></sup>). Data were visually inspected for whole-brain field of view coverage, signal artifacts and proper alignment to the corresponding anatomical image. Functional data were excluded if more than 25% of the frames exceeded 0.2-mm FD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Parkes, L., Fulcher, B., Yücel, M. &amp; Fornito, A. An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI. Neuroimage 171, 415–436 (2018)." href="/articles/s41593-020-00719-y#ref-CR70" id="ref-link-section-d30515570e1968">70</a></sup>. Furthermore, HBN functional data were excluded if marked as an outlier (exceeding 1.5× interquartile range (IQR) in the adverse direction) in more than half of the following image quality metrics (calculated within datasets, across all functional acquisitions): dvars, tsnr, fd mean, aor, aqi, snr and efc. Information about these image quality metrics can be found in MRIQC’s documentation.</p><h3 class="c-article__sub-heading" id="Sec19">Functional and structural networks pre-processing</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">Parcellation preprocessing</h4><p>A functional parcellation designed to optimize both local gradient and global similarity measures of the fMRI signal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-020-00719-y#ref-CR22" id="ref-link-section-d30515570e1985">22</a></sup> (Schaefer200) was used to define 200 areas on the cerebral cortex. These nodes are also mapped to the Yeo canonical functional networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Yeo, B. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1126–1165 (2011)." href="/articles/s41593-020-00719-y#ref-CR71" id="ref-link-section-d30515570e1989">71</a></sup>. For the HCP dataset, the Schaefer200 is openly available in ‘32k fs LR’ space as a <span class="u-small-caps">cifti</span> file. For the MSC and HBN datasets, a Schaefer200 parcellation was obtained for each participant using a Gaussian classifier surface atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Fischl, B. et al. Automatically parcellating the human cerebral cortex. Cereb. Cortex 14, 11–22 (2004)." href="/articles/s41593-020-00719-y#ref-CR72" id="ref-link-section-d30515570e1996">72</a></sup> (trained on 100 unrelated HCP participants) and FreeSurfer’s mris_ca_label function. These tools use the surface registrations computed in the recon-all pipeline to transfer a group average atlas to subject space based on individual surface curvature and sulcal patterns. This method rendered a T1w space volume for each participant. For use with functional data, the parcellation was resampled to 2-mm T1w space. This process could be repeated for other resolutions of the parcellation (that is, Schaefer100).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Functional network pre-processing</h4><p>Each pre-processed BOLD image was linearly de-trended, band-pass filtered (0.008–0.08 Hz)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Parkes, L., Fulcher, B., Yücel, M. &amp; Fornito, A. An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI. Neuroimage 171, 415–436 (2018)." href="/articles/s41593-020-00719-y#ref-CR70" id="ref-link-section-d30515570e2008">70</a></sup>, confound regressed and standardized using Nilearn signal.clean, which removes confounds orthogonally to the temporal filters<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Lindquist, M. A., Geuter, S., Wager, T. D. &amp; Caffo, B. S. Modular preprocessing pipelines can reintroduce artifacts into fMRI data. Hum. Brain Mapp. 40, 2358–2376 (2019)." href="/articles/s41593-020-00719-y#ref-CR73" id="ref-link-section-d30515570e2012">73</a></sup>. The confound regression employed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Satterthwaite, T. D. et al. An improved framework for confound regression and filtering for control of motion artifact in the preprocessing of resting-state functional connectivity data. Neuroimage 64, 240–256 (2013)." href="/articles/s41593-020-00719-y#ref-CR74" id="ref-link-section-d30515570e2016">74</a></sup> included six motion estimates; time series of the mean cerebrospinal fluid, mean white matter and mean global signal; the derivatives of these nine regressors; and the squares of these 18 terms. Furthermore, a spike regressor was added for each fMRI frame exceeding a motion threshold (HCP = 0.25-mm root mean squared displacement; MSC and HBN = 0.5-mm FD). This confound strategy has been shown to be a relatively effective option for reducing motion-related artifacts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Parkes, L., Fulcher, B., Yücel, M. &amp; Fornito, A. An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI. Neuroimage 171, 415–436 (2018)." href="/articles/s41593-020-00719-y#ref-CR70" id="ref-link-section-d30515570e2020">70</a></sup>. After pre-processing and nuisance regression, residual mean BOLD time series at each node were recovered. eFC matrices for each participant were computed and then averaged across participants to obtain a representative eFC matrix for each dataset. This processing was performed for both resting-state and movie-watching data.</p><h3 class="c-article__sub-heading" id="Sec22">Edge graph construction</h3><p>Constructing networks from fMRI data (or any neural time series data) requires estimating the statistical dependency between every pair of time series. The magnitude of that dependency is usually interpreted as a measure of how strongly (or weakly) those voxels or parcels are functionally connected to each other. By far the most common measure of statistic dependence is the Pearson correlation coefficient. Let <i>x</i><sub><i>i</i></sub> = [<i>x</i><sub><i>i</i></sub>(1),...,<i>x</i><sub><i>i</i></sub>(<i>T</i>)] and <i>x</i><sub><i>j</i></sub> = [<i>x</i><sub><i>j</i></sub>(1),...,<i>x</i><sub><i>j</i></sub>(<i>T</i>)] be the time series recorded from voxels or parcels <i>i</i> and <i>j</i>, respectively. We can calculate the correlation of <i>i</i> and <i>j</i> by first <i>z</i> scoring each time series, such that at <span class="mathjax-tex">\({\boldsymbol{z}}_i = \frac{{{\boldsymbol{x}}_i - \mu _i}}{{\sigma _i}}\)</span> where <span class="mathjax-tex">\(\mu _i = \frac{1}{T}\mathop {\sum}\nolimits_t {x_i\left( t \right)}\)</span> and <span class="mathjax-tex">\(\sigma _i = \sqrt {\frac{1}{{T - 1}}\mathop {\sum}\nolimits_t {\left( {x_i\left( t \right) - \mu _i} \right)^2} }\)</span> are the time-averaged mean and s.d. Then, the correlation of <i>i</i> and <i>j</i> can be calculated as <span class="mathjax-tex">\(r_{ij} = \frac{1}{{T - 1}}\mathop {\sum }\limits_t \left[ {z_i\left( t \right) \cdot z_j\left( t \right)} \right]\)</span>. Repeating this procedure for all pairs of parcels results in a node-by-node correlation matrix—that is, an estimate of functional connectivity. If there are <i>N</i> nodes, this matrix has dimensions [<i>N</i> × <i>N</i>].</p><p>To estimate edge-centric networks, we need to modify the above approach in one small but crucial way. Suppose we have two <i>z</i>-scored parcel time series, <i>z</i><sub><i>i</i></sub> and <i>z</i><sub><i>j</i></sub>. To estimate their correlation, we calculate the mean of their element-wise product (not exactly the average, because we divide by <i>T</i> − 1 rather than <i>T</i>). Suppose, instead, that we never calculate the mean and simply stop after calculating the element-wise product. This operation would result in a vector of length <i>T</i> whose elements encode the moment-by-moment co-fluctuations magnitude of parcels <i>i</i> and <i>j</i>. For instance, suppose at time <i>t</i>, parcels <i>i</i> and <i>j</i> simultaneously increased their activity relative to baseline. These increases are encoded in <i>z</i><sub><i>i</i></sub> and <i>z</i><sub><i>j</i></sub> as positive entries in the tth position, so their product is also positive. The same would be true if <i>i</i> and <i>j</i> decreased their activity simultaneously (because the product of negatives is a positive). On the other hand, if <i>i</i> increased while <i>j</i> decreased (or vice versa), this would manifest as a negative entry. Similarly, if either <i>i</i> or <i>j</i> increased or decreased while the activity of the other was close to baseline, the corresponding entry would be close to zero.</p><p>Accordingly, the vector resulting from the element-wise product of <i>z</i><sub><i>i</i></sub> and <i>z</i><sub><i>j</i></sub> can be viewed as encoding the magnitude of moment-to-moment co-fluctuations between <i>i</i> and <i>j</i>. An analogous vector can easily be calculated for every pair of parcels (network nodes), resulting in a set of co-fluctuation (edge) time series. With <i>N</i> parcels, this results in <span class="mathjax-tex">\(\frac{{N\left( {N - 1} \right)}}{2}\)</span> pairs, each of length <i>T</i>. From these time series, we can estimate the statistical dependency for every pair of edges. We refer to this construct as eFC. Let <i>c</i><sub><i>ij</i></sub> = [<i>z</i><sub><i>i</i></sub>(1) <span class="stix">⋅</span> <i>z</i><sub><i>j</i></sub>(1),...,<i>z</i><sub><i>i</i></sub>(<i>T</i>) <span class="stix">⋅</span> <i>z</i><sub><i>j</i></sub>(<i>T</i>)] and <i>c</i><sub><i>uv</i></sub> = [<i>z</i><sub><i>u</i></sub>(1) <span class="stix">⋅</span> <i>z</i><sub><i>v</i></sub>(1),...,<i>z</i><sub><i>u</i></sub>(<i>T</i>) <span class="stix">⋅</span> <i>z</i><sub><i>v</i></sub>(<i>T</i>)] be the time series for edges {<i>i</i>,<i>j</i>} and {<i>u</i>,<i>v</i>}, respectively. Then, we can calculate eFC as</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$eFC_{ij,uv} = \frac{{\mathop {\sum }\nolimits_t c_{ij}\left( t \right) \cdot c_{uv}\left( t \right)}}{{\sqrt {\mathop {\sum }\nolimits_t c_{ij}\left( t \right)^2} \sqrt {\mathop {\sum }\nolimits_t c_{uv}\left( t \right)^2} }}$$</span></div></div><p>Here, the denominator is necessary to bind eFC to the interval [−1,1].</p><h3 class="c-article__sub-heading" id="Sec23">Clustering algorithm</h3><p>In general, eFC matrices are much larger than traditional nFC matrices. Although most clustering algorithms can be applied to hundreds or even thousands of observations, estimating clusters for eFC (which consists of tens of thousands of observations, each paired with at least as many features) presents a computational challenge, especially if the aim is to explore the space of possible partitions. To address this issue and to cluster eFC, we developed a simple two-step clustering procedure that operates on a low-dimensional representation of the eFC matrix.</p><p>First, we performed an eigen decomposition of the eFC matrix, retaining the top 50 eigenvectors. Eigenvector coefficients were rescaled to the interval [−1,1] by dividing each by its largest magnitude element, and then the rescaled coefficients were simply clustered using a standard <i>k</i>-means algorithm with Euclidean distance. We varied the number of communities, <i>k</i>, from <i>k</i> = 2 to <i>k</i> = 20, repeating the clustering algorithm 250 times at each value. We retained, as a representative partition, the one with the greatest overall similarity to all other partitions. We note that the edge time series can be clustered directly and that, in general, the results were highly similar (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">12</a>).</p><p>We note that, in general, other community detection algorithms could be used in place of <i>k</i>-means. Our decision to use this algorithm was practically motivated, as <i>k</i>-means exhibited significantly faster runtimes than other algorithms—for example, modularity maximization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Newman, M. E. &amp; Girvan, M. Finding and evaluating community structure in networks. Phys. Rev. E 69, 026113 (2004)." href="/articles/s41593-020-00719-y#ref-CR35" id="ref-link-section-d30515570e2984">35</a></sup> and Infomap<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Rosvall, M. &amp; Bergstrom, C. T. Maps of random walks on complex networks reveal community structure. Proc. Natl Acad. Sci. USA 105, 1118–1123 (2008)." href="/articles/s41593-020-00719-y#ref-CR36" id="ref-link-section-d30515570e2988">36</a></sup>, which have been used extensively in previous work to derive communities in both functional and structural brain networks.</p><h3 class="c-article__sub-heading" id="Sec24">Community overlap metrics</h3><p>The clustering algorithm partitioned edges into non-overlapping clusters. That is, every edge {<i>i</i>,<i>j</i>}, where <i>i</i>,<i>j</i><span class="stix">∈</span>{1,...,<i>N</i>}, was assigned to one of the <i>k</i> clusters. In this list of edges, each node appeared <i>N</i>−1 times (we excluded self-connections). Region <i>i</i>’s participation in cluster <i>c</i> was calculated as</p><div id="Equb" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$p_{ic} = \frac{1}{{N - 1}}\mathop {\sum }\limits_{j \ne i} \delta \left( {g_{ij},c} \right)$$</span></div></div><p>where <i>g</i><sub><i>ij</i></sub><span class="stix">∈</span>{1,...,<i>k</i>} was the cluster assignment of the edge linking nodes <i>i</i> and <i>j</i>, and <i>δ</i>(<i>x</i>,<i>y</i>) is the Kronecker delta, whose value is 1 if <i>x</i> = <i>y</i> and 0 otherwise. By definition, <span class="mathjax-tex">\(\mathop {\sum}\nolimits_c {p_{ic} = 1}\)</span>, and we can treat the vector <i>p</i><sub><i>i</i></sub> = [<i>p</i><sub><i>i</i>1</sub>,...,<i>p</i><sub><i>ik</i></sub>] as a probability distribution. The entropy of this distribution measures the extent to which region <i>i</i>’s community affiliations are distributed evenly across all communities (high entropy and high overlap) or concentrated within a small number of communities (low entropy and low overlap). We calculate this entropy as:</p><div id="Equc" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$h_i = - \mathop {\sum }\limits_c p_{ic}\log _2p_{ic}$$</span></div></div><p>To normalize this measure and bind it to the interval [0,1], we divided by <span class="mathjax-tex">\(\log _2k\)</span>. We refer to this measure as community entropy and interpret this value as an index of overlap. Intuitively, as the distribution of edge community assignments approaches uniformity, its normalized entropy is close to 1; when edges are assigned to a single community, normalized entropy is closer to 0.</p><h3 class="c-article__sub-heading" id="Sec25">Edge community similarity</h3><p>When we cluster an eFC matrix, we assign each edge to a single community. These edge communities can be rearranged into the upper triangle of an <i>N</i> × <i>N</i> matrix, <i>Χ</i>, whose element <i>x</i><sub><i>ij</i></sub> denotes the edge community assignment of the edge between nodes <i>i</i> and <i>j</i>. The <i>i</i>th column of <i>Χ</i>, which we denote as <i>x</i><sub><i>i</i></sub> = [<i>x</i><sub>1<i>i</i></sub>,...,<i>x</i><sub><i>Ni</i></sub>], encodes the community labels of all edges in which node <i>i</i> participates. Note that we do not consider self-edges, so the element <i>x</i><sub><i>ii</i></sub> is left empty.</p><p>From this matrix, we can compare the edge communities of nodes <i>i</i> and <i>j</i> by calculating the similarity of vectors <i>x</i><sub><i>i</i></sub> and <i>x</i><sub><i>j</i></sub>. Here, we measure that similarity as the fraction of elements in both vectors with the same community label. That is:</p><div id="Equd" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$s_{ij} = \frac{1}{{N - 2}}\mathop {\sum }\limits_{u \ne i,j} \delta \left( {x_{iu},x_{ju}} \right)$$</span></div></div><p>Here, <i>δ</i>(<i>x</i>,<i>y</i>) is the Kronecker delta and takes on a value of 1 when <i>x</i> and <i>y</i> have the same value but is 0 otherwise. Note the normalization of over <i>N</i> − 2 because we ignore the self-connections <i>x</i><sub><i>ii</i></sub> and <i>x</i><sub><i>jj</i></sub>. Repeating this comparison for all pairs of nodes generates the similarity matrix <i>S</i> = {<i>s</i><sub><i>ij</i></sub>}.</p><h3 class="c-article__sub-heading" id="Sec26">Estimating overlapping community structure from nFC</h3><p>In this study, we applied a clustering algorithm to eFC, which generates overlapping nodal communities. In contrast, field standard community detection algorithms like Infomap<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Rosvall, M. &amp; Bergstrom, C. T. Maps of random walks on complex networks reveal community structure. Proc. Natl Acad. Sci. USA 105, 1118–1123 (2008)." href="/articles/s41593-020-00719-y#ref-CR36" id="ref-link-section-d30515570e3593">36</a></sup> and modularity maximization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Newman, M. E. &amp; Girvan, M. Finding and evaluating community structure in networks. Phys. Rev. E 69, 026113 (2004)." href="/articles/s41593-020-00719-y#ref-CR35" id="ref-link-section-d30515570e3597">35</a></sup> partition nFC into non-overlapping communities. However, there are non-standard methods that can be applied directly to nFC that generate overlapping communities. These include, but are not limited to, stochastic variational inference for the mixed-membership stochastic block model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Gopalan, P. K. &amp; Blei, D. M. Efficient discovery of overlapping communities in massive networks. Proc. Natl Acad. Sci. USA 110, 14534–14539 (2013)." href="/articles/s41593-020-00719-y#ref-CR75" id="ref-link-section-d30515570e3601">75</a></sup> (SVINET), the Affiliation Graph Model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Yang, J. &amp; Leskovec, J. Overlapping communities explain core-periphery organization of networks. Proc. IEEE 102, 1892–1902 (2014)." href="/articles/s41593-020-00719-y#ref-CR76" id="ref-link-section-d30515570e3605">76</a></sup> (AGMFIT), Bayesian non-negative matrix factorization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Psorakis, I., Roberts, S., Ebden, M. &amp; Sheldon, B. Overlapping community detection using Bayesian non-negative matrix factorization. Phys. Rev. E 83, 066114 (2011)." href="/articles/s41593-020-00719-y#ref-CR77" id="ref-link-section-d30515570e3609">77</a></sup> (NMF) and thresholded weighted link clustering<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Evans, T. &amp; Lambiotte, R. Line graphs, link partitions, and overlapping communities. Phys. Rev. E 80, 016105 (2009)." href="/articles/s41593-020-00719-y#ref-CR12" id="ref-link-section-d30515570e3614">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="de Reus, M. A., Saenger, V. M., Kahn, R. S. &amp; van den Heuvel, M. P. An edge-centric perspective on the human connectome: link communities in the brain. Philos. Trans. R. Soc. B Biol. Sci. 369, 20130527 (2014)." href="/articles/s41593-020-00719-y#ref-CR14" id="ref-link-section-d30515570e3617">14</a></sup> (ThrLink).</p><p>We applied these methods to group representative nFC data from the HCP dataset (with the number of communities fixed at <i>k</i> = 10) and compared their patterns of overlap with those obtained from clustering eFC. In general, each of these alternative methods require that the input connectivity matrix contain only positively weighted or binary edges, necessitating that it be thresholded. To do this, we computed the maximum spanning tree of the nFC matrix (to ensure that all nodes form a single connected component) and added edges to this backbone to reach a desired network density. We repeated the following comparisons across densities of 10%, 20%, 30% and 40% (a range in which negative edges were not retained). For each method, 250 overlapping community structures were recovered. We describe each method in more detail below and summarize the results in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">17</a>.</p><p>The SVINET method employs a mixed-membership stochastic block model algorithm, which is a generative model of network communities based on grouping nodes with similar connectivity patterns<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Gopalan, P. K. &amp; Blei, D. M. Efficient discovery of overlapping communities in massive networks. Proc. Natl Acad. Sci. USA 110, 14534–14539 (2013)." href="/articles/s41593-020-00719-y#ref-CR75" id="ref-link-section-d30515570e3633">75</a></sup>. This method has been previously used to demonstrate that the areas of the brain that participate in many cognitive functions also participate in proportionally more communities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Najafi, M., McMenamin, B. W., Simon, J. Z. &amp; Pessoa, L. Overlapping communities reveal rich structure in large-scale brain networks during rest and task conditions. Neuroimage 135, 92–106 (2016)." href="/articles/s41593-020-00719-y#ref-CR78" id="ref-link-section-d30515570e3637">78</a></sup>. This method operates on binary connections; thus, edge weights were discarded. Each run was seeded with a random integer and run for 250 iterations with link sampling. Resulting community assignments with at least 5% membership likelihood were recorded as a membership affiliation.</p><p>The AGMFIT method employs a generative model of communities based on a bipartite graph structure, linking nodes to communities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Yang, J. &amp; Leskovec, J. Overlapping communities explain core-periphery organization of networks. Proc. IEEE 102, 1892–1902 (2014)." href="/articles/s41593-020-00719-y#ref-CR76" id="ref-link-section-d30515570e3644">76</a></sup>. The central concept of the AGMFIT algorithm is that communities overlap in a ‘tiled’ manner, meaning that nodes with overlapping community membership are more densely interconnected than non-overlapping nodes. This model of overlapping structure has been shown to accurately capture core-periphery structure in large-scale social networks. This method operates on binary connections; thus, edge weights were discarded. Each run was seeded with a random integer.</p><p>The NMF method employs a probabilistic data reduction model that results in a soft partitioning of the network<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Psorakis, I., Roberts, S., Ebden, M. &amp; Sheldon, B. Overlapping community detection using Bayesian non-negative matrix factorization. Phys. Rev. E 83, 066114 (2011)." href="/articles/s41593-020-00719-y#ref-CR77" id="ref-link-section-d30515570e3652">77</a></sup>. This method has been shown to avoid over-fitting communities in synthetic random graph data where no real communities exist. Edge weights were retained for this method, and diagonal entries of the adjacency matrix were set to the nodal degree (as suggested in the documentation). Each run was randomly initialized. Runs that did not produce the desired number of communities were rejected, and sampling continued until 250 partitions were obtained. Resulting community assignments with at least 5% membership likelihood were recorded as membership affiliation.</p><p>For the ThrLink method, we created a weighted line graph from the thresholded adjacency matrix<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Evans, T. &amp; Lambiotte, R. Line graphs, link partitions, and overlapping communities. Phys. Rev. E 80, 016105 (2009)." href="/articles/s41593-020-00719-y#ref-CR12" id="ref-link-section-d30515570e3659">12</a></sup>. This matrix was clustered using the generalized Louvain algorithm with the resolution parameter γ tuned to produce the desired number of communities. To tune this parameter, a range of values was used to recover communities of varying sizes. The minimum and maximum values producing the desired number of communities were recorded. Uniformly randomly sampled γ values within this range were used to recover communities of the weighted line graph. Runs that did not produce the desired number of communities were rejected, and sampling continued until 250 partitions were obtained. Community memberships of the weighted line graph were projected to the nodes to gather the overlapping structure.</p><p>We compared community entropy against a series of related statistics that can be easily derived from nFC as opposed to eFC. These include static measures of participation coefficient<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Guimera, R. &amp; Amaral, L. A. N. Functional cartography of complex metabolic networks. Nature 433, 895–900 (2005)." href="/articles/s41593-020-00719-y#ref-CR79" id="ref-link-section-d30515570e3666">79</a></sup> and versatility<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Shinn, M. et al. Versatility of nodal affiliation to communities. Sci. Rep. 7, 1–10 (2017)." href="/articles/s41593-020-00719-y#ref-CR80" id="ref-link-section-d30515570e3670">80</a></sup> and the ‘dynamic’ measure of flexibility<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Bassett, D. S. et al. Dynamic reconfiguration of human brain networks during learning. Proc. Natl Acad. Sci. USA 108, 7641–7646 (2011)." href="/articles/s41593-020-00719-y#ref-CR25" id="ref-link-section-d30515570e3674">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Pedersen, M., Zalesky, A., Omidvarnia, A. &amp; Jackson, G. D. Multilayer network switching rate predicts brain performance. Proc. Natl Acad. Sci. USA 115, 13376–13381 (2018)." href="/articles/s41593-020-00719-y#ref-CR81" id="ref-link-section-d30515570e3677">81</a></sup>. We calculated static measures using a group representative nFC matrix that was the average nFC data from all scans and individuals. Flexibility was calculated first at the single-individual level where time series were divided into ten non-overlapping windows containing <i>L</i> = 120 samples each (approximately 86 s) and subsequently averaged across individuals. Details of how each measure was calculated are presented below.</p><p>Participation coefficient measures the uniformity with which a node’s connections are distributed across (non-overlapping) communities. Values closer to 1 indicate that connections are distributed evenly. Participation coefficient is calculated as:</p><div id="Eque" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${pc}_{i}= 1 - {\sum} {\left( {\frac{{k_{is}}}{{k_i}}} \right)^2}$$</span></div></div><p>Here, <i>k</i><sub><i>i</i></sub> is the total strength of node <i>i</i>, and <i>k</i><sub><i>is</i></sub> is the strength of node <i>i</i> to community <i>s</i>. We calculated several variants of participation coefficient in which we varied how communities were defined. First, we treated the system labels from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-020-00719-y#ref-CR22" id="ref-link-section-d30515570e3805">22</a></sup> as a community structure and calculated the participation coefficient with respect to these labels. We also tested a more data-driven procedure in which we used multi-scale modularity maximization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Reichardt, J. &amp; Bornholdt, S. Statistical mechanics of community detection. Phys. Rev. E 74, 016110 (2006)." href="/articles/s41593-020-00719-y#ref-CR82" id="ref-link-section-d30515570e3809">82</a></sup> to detect the communities of the nFC matrix. In doing so, we used a uniform null model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Traag, V. A., Van Dooren, P. &amp; Nesterov, Y. Narrow scope for resolution-limit-free community detection. Phys. Rev. E 84, 016114 (2011)." href="/articles/s41593-020-00719-y#ref-CR83" id="ref-link-section-d30515570e3813">83</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Bazzi, M. et al. Community detection in temporal multilayer networks, with an application to correlation networks. Multiscale Model. Simul. 14, 1–41 (2016)." href="/articles/s41593-020-00719-y#ref-CR84" id="ref-link-section-d30515570e3816">84</a></sup>, which is appropriate for correlation matrices and has been used extensively in the neuroimaging community (see ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Betzel, R. F. et al. The community structure of functional brain networks exhibits scale-specific patterns of inter-and intra-subject variability. Neuroimage 202, 115990 (2019)." href="/articles/s41593-020-00719-y#ref-CR85" id="ref-link-section-d30515570e3820">85</a></sup> as just one example) and systematically varied the resolution parameter γ over the interval [0,0.5] (repeating a Louvain-like algorithm 1,000 times). In all cases, we separately calculated the participation coefficient using for positive and negative connection weights.</p><p>We also used the detected communities to estimate regional versatility<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Shinn, M. et al. Versatility of nodal affiliation to communities. Sci. Rep. 7, 1–10 (2017)." href="/articles/s41593-020-00719-y#ref-CR80" id="ref-link-section-d30515570e3827">80</a></sup>, which measures the variability of a node’s community assignment across repeated runs of a community detection algorithm. We calculated versatility as:</p><div id="Equf" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$v_i = \frac{{\mathop {\sum }\nolimits_j \sin \left( {\pi \cdot p_{ij}} \right)}}{{\mathop {\sum }\nolimits_j p_{ij}}}$$</span></div></div><p>For a given value of γ, <i>p</i><sub><i>ij</i></sub> denotes the fraction of times that nodes <i>i</i> and <i>j</i> were co-assigned to the same community. We calculated versatility with respect to communities detected using the same values of γ.</p><p>Lastly, we calculated network flexibility, which measures how frequently a brain region changes communities across time. We modeled functional connectivity estimated within each non-overlapping window as a layer in a multi-layer network and a used multi-layer modularity maximization algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Mucha, P. J., Richardson, T., Macon, K., Porter, M. A. &amp; Onnela, J.-P. Community structure in time-dependent, multiscale, and multiplex networks. Science 328, 876–878 (2010)." href="/articles/s41593-020-00719-y#ref-CR86" id="ref-link-section-d30515570e3956">86</a></sup> to cluster all layers simultaneously. The result is a node-by-layer matrix of communities whose element <i>g</i><sub><i>i</i>,<i>s</i></sub> indicates the community assignment of node <i>i</i> in layer <i>s</i>. From this matrix, we calculate flexibility as:</p><div id="Equg" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$f_i = 1 - \frac{1}{{T - 1}}\mathop {\sum }\limits_{s = 1}^{T - 1} \delta \left( {g_{i,s},g_{i,s + 1}} \right)$$</span></div></div><p>Here, <i>T</i> = 10 is the number of layers, and <i>δ</i>(<i>g</i><sub><i>i</i>,<i>s</i></sub>, <i>g</i><sub><i>i</i>,<i>s</i>+1</sub>) is the Kronecker delta function and is equal to 1 when <i>g</i><sub><i>i</i>,<i>s</i></sub> = <i>g</i><sub><i>i</i>,<i>s</i>+1</sub> and is 0 otherwise. In essence, flexibility measures the fraction of times that a node’s community assignment changes in successive layers (time points). In addition to the γ resolution parameter, the output of the multi-layer modularity maximization algorithm depends on a second parameter, ω, that controls the consistency of communities across layers. We systematically varied these parameters over the ranges <i>γ</i> = [0,0.25,0.5] and <i>ω</i> = [0.1,0.5,1] and calculated flexibility for all possible {<i>γ</i>,<i>ω</i>} pairs.</p><h3 class="c-article__sub-heading" id="Sec27">Graph theoretic analysis of eFC</h3><p>We applied graph theoretic measures to the eFC matrix to characterize its topological features<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: uses and interpretations. Neuroimage 52, 1059–1069 (2010)." href="/articles/s41593-020-00719-y#ref-CR10" id="ref-link-section-d30515570e4170">10</a></sup>. We focused on local measures that characterize features at the level of a network’s nodes (in the case of eFC, nodes represent pairs of brain regions). To visualize these measures, we reshaped their values into the upper triangle of a region-by-region matrix (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">20</a>). We focused on several different measures:</p><ol class="u-list-style-none">
<li>
<span class="u-custom-list-number">1.</span>

<p>Degree (±) measures separately the total number of positive and negative connections incident upon a given node in the eFC network.</p>

</li>
<li>
<span class="u-custom-list-number">2.</span>

<p>Strength (±) is the weighted analog of degree and measures separately the total weight of positive and negative connections incident upon a given node in the eFC network. Both degree and strength tell us, on average, how strongly or weakly a given node in the eFC network interacts with other nodes in the eFC network.</p>

</li>
<li>
<span class="u-custom-list-number">3.</span>

<p>Participation coefficient (±) measures the extent to which a node’s connections in the eFC network are concentrated within or distributed across edge communities. Values close to 0 mean that a given node in the eFC network interacts primarily with other nodes in its own edge community; values close to 1 mean that a given node in the eFC network interacts uniformly with all edge communities.</p>

</li>
<li>
<span class="u-custom-list-number">4.</span>

<p>Betweenness centrality measures the number of shortest paths between pairs of nodes in the eFC network that pass through a given node. In general, betweenness centrality implies that a particular node in the eFC network might occupy a position of importance in the network.</p>

</li>
<li>
<span class="u-custom-list-number">5.</span>

<p>Clustering coefficient measures the extent to which a node’s neighbors in the eFC network are also connected to one another.</p>

</li>
</ol><h3 class="c-article__sub-heading" id="Sec28">Exploratory analyses of brain–behavior relationships using eFC</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">Correlations of eFC weights with behavior</h4><p>We also used eFC data to explore brain–behavior relationships<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Smith, S. M. et al. A positive–negative mode of population covariation links brain connectivity, demographics and behavior. Nat. Neurosci. 18, 1565–1567 (2015)." href="/articles/s41593-020-00719-y#ref-CR87" id="ref-link-section-d30515570e4246">87</a></sup>. The overall pipeline begins by calculating each participantʼs edge-by-edge eFC matrix (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21a</a>) and representing its upper triangle elements as a vector (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21b</a>). This procedure is repeated for all individuals in the HCP 100 Unrelated Subjects cohort so that the vectorized eFC is stored in a single matrix (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21c</a>). In parallel, we <i>z</i> scored participants’ behavioral data and performed principal components analysis, resulting in a set of scores that characterize orthogonal modes of behavioral variability (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21d</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">1</a>). We computed the correlation of scores with rows from the matrix of vectorized eFC matrices (each row represents the eFC for a particular edge-edge interaction (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21e</a>)). Repeating this procedure for all rows results in a vector of correlation coefficients that can be reshaped to fit into the upper triangle of an edge-by-edge matrix, resulting in a correlation map (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21f</a>). This entire process is repeated separately for principal components 1–10 and for scans REST1 and REST2.</p><p>We compared the correlation maps from REST1 and REST2 and found good correspondence (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21h,i</a>). To better interpret these maps, we adopted a community-level analysis (see Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">22</a> for a short schematic). Briefly, this involves aggregating and averaging correlation coefficients by edge communities (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">22b</a>); comparing the average correlation coefficients against a null distribution obtained using a constrained permutation test (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">22c</a>); and performing statistical evaluation, controlling for false discovery rate at the level of communities (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">22d</a>). Further details of the permutation test can be found in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">18</a>.</p><p>Using this community-level approach, we investigated the relationship between eFC and PC1 in greater detail. We note that PC1 explains approximately 17% variance in behavioral data (almost three times as much as PC2) and defines a task accuracy/reaction time axis of behavior (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21j,k</a>). We include brief descriptions of the other PCs in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">1</a>. We show the correlation map for PC1 with eFC in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21l</a>. To illustrate how the community-level analysis facilitates a clearer interpretation of brain–behavior correlations, consider eFC of edges in communities 7 and 9 (the block highlighted in Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21l</a>). Community 7 links higher-order cognitive areas in the control and default mode networks with visual cortex, forming an ‘executive–visual’ complex, whereas community 9 links control and default mode to the salience/ventral attention network as part of an ‘executive–insular’ complex (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21m,n</a>). Accordingly, the positive correlation eFC between community 7 and 9 with PC1 means that, as the edges within those communities become more synchronized across time (stronger eFC), the value of PC1 increases proportionally (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21o,p</a>).</p><p>In addition to modeling brain–behavior relationships using the original eFC data, we repeated this same analysis with residual eFC after regressing out the effect of nFC. Specifically, we used the procedure described in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2c</a> to generate an approximation of eFC using only nFC data. We then regressed out the approximated eFC from the actual eFC and assessed brain–behavior relationships using the residual values. As with the previous analysis, we found that brain–behavior correlation maps were reproducible across scan sessions (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">23</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec30">Correlations of regional statistics with behavior</h4><p>We also compared eFC and nFC brain–behavior relationships by deriving a series of regional (local) network statistics from each and calculating the correlations of behavioral measures with these statistics (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">24</a>). We note that the measures derived from both nFC and eFC have identical dimensionality, effectively accounting for any differences in the dimensionality of the original nFC and eFC matrices. In general, we found that the correlation patterns estimated using nFC-derived statistics were highly similar to one another, whereas the correlation pattern derived from the eFC statistic was dissimilar (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">24f</a>).</p><p>These findings demonstrate that eFC has the potential to uniquely explain patterns of inter-individual variability not currently explainable by nFC, opening new opportunities for studying individual differences in individuals’ cognitive, developmental and clinical states.</p><h3 class="c-article__sub-heading" id="Sec31">Modeling eFC in terms of nFC</h3><p>eFC and nFC are both derived from the same substrate: regional fMRI BOLD time series. Can the eFC between edges {<i>i</i>,<i>j</i>} and {<i>u</i>,<i>v</i>} be easily modeled in terms of nFC? We tested whether this was the case using linear regression to explain the eFC between pairs of edges {<i>i</i>,<i>j</i>} and {<i>u</i>,<i>v</i>} using information about the pairwise nFC among the same set of nodes: {<i>i</i>,<i>j</i>}, {<i>i</i>,<i>u</i>}, {<i>i</i>,<i>v</i>}, {<i>j</i>,<i>u</i>}, {<i>j</i>,<i>v</i>} and {<i>u</i>,<i>v</i>}. We considered two classes of models. The first modeled eFC in terms of the six nFC weights:</p><div id="Equh" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$eFC_{ij,uv} = \beta _1nFC_{ij} + \beta _2nFC_{iu} + \beta _3nFC_{iv} + \beta _4nFC_{ju} + \beta _5nFC_{jv} + \beta _6nFC_{uv} + \beta _0 + \varepsilon$$</span></div></div><p>The second modeled eFC in terms of nFC interactions:</p><div id="Equi" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$eFC_{ij,uv} = \beta _1\left[ {nFC_{ij} \times nFC_{uv}} \right] + \beta _0 + \varepsilon$$</span></div></div><p>In this model, we systematically varied the interaction term, <i>nFC</i><sub><i>ij</i></sub> × <i>nFC</i><sub><i>uv</i></sub>, so that we tested all possible pairs of edges.</p><p>In general, we found that neither model 1 nor model 2 could fully reproduce eFC. Model 1 performed particularly poorly (<i>r</i> = 0.21). The results of model 2 were more varied. When all nodes were represented in the interaction term—for example, <i>nFC</i><sub><i>ij</i></sub> × <i>nFC</i><sub><i>uv</i></sub>—the model performed well (<i>r</i> = 0.72 ± 0.05), consistent with what we reported in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig2">2c</a>. When any node is repeated—for example, <i>nFC</i><sub><i>ij</i></sub> × <i>nFC</i><sub><i>iv</i></sub>—the model performed poorly (<i>r</i> = 0.06 ± 0.04).</p><p>Collectively, these observations suggest that eFC is not well approximated using linear combinations of nFC, but, with nonlinear transformations and inclusion of interaction terms, nFC can approximate eFC. However, these transformations are unintuitive, and the approximation still fails to fully explain variance in eFC.</p><h3 class="c-article__sub-heading" id="Sec32">Statistics</h3><p>All statistical comparisons made between individuals and groups were implemented using paired sample and two-sample <i>t</i>-tests, ANOVAs or permutation tests. For <i>t</i>-tests and ANOVAs, the data distribution was assumed to be normal, but this was not formally tested. We used several variants of permutation tests: in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7b,e,f</a>, we permuted condition labels (REST and MOVIE); in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00719-y#Fig7">7g</a>, we permuted system labels; in Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">21</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">22</a>, we mapped edge communities into a region-by-region matrix and permuted its rows and columns before extracting the permuted edge community labels (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">18</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM1">24</a>). For permutation tests, observed values were <i>z</i> scored with respect to the mean and s.d. estimated for the null distribution and converted into <i>P</i> values.</p><h3 class="c-article__sub-heading" id="Sec33">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-020-00719-y#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                </div>
            

            <div>
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">

<p>All imaging data come from publicly available, open access repositories. Human Connectome Project data can be accessed at <a href="https://db.humanconnectome.org/app/template/Login.vm">https://db.humanconnectome.org/app/template/Login.vm</a> after signing a data use agreement. Midnight Scan Club data can be accessed via OpenNeuro at <a href="https://openneuro.org/datasets/ds000224/versions/1.0.1">https://openneuro.org/datasets/ds000224/versions/1.0.1</a>. The Healthy Brain Network Serial Scanning Initiative data can be accessed at https://fcon_1000.projects.nitrc.org/indi/hbn_ssi/download.html.</p>
</div></div></section><section data-title="Code availbility"><div class="c-article-section" id="code-availbility-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availbility">Code availbility</h2><div class="c-article-section__content" id="code-availbility-content">

<p>Code to compute eFC and its related derivatives has been made available at <a href="https://github.com/brain-networks">https://github.com/brain-networks</a>.</p>
</div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Bassett, D. S. &amp; Sporns, O. Network neuroscience. <i>Nat. Neurosci.</i> <b>20</b>, 353–364 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVSmsrY%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28230844" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5485642" aria-label="PubMed Central reference 1">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20neuroscience&amp;journal=Nat.%20Neurosci.&amp;volume=20&amp;pages=353-364&amp;publication_year=2017&amp;author=Bassett%2CDS&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Schröter, M., Paulsen, O. &amp; Bullmore, E. T. Micro-connectomics: probing the organization of neuronal networks at the cellular scale. <i>Nat. Rev. Neurosci.</i> <b>18</b>, 131–146 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28148956" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Micro-connectomics%3A%20probing%20the%20organization%20of%20neuronal%20networks%20at%20the%20cellular%20scale&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=18&amp;pages=131-146&amp;publication_year=2017&amp;author=Schr%C3%B6ter%2CM&amp;author=Paulsen%2CO&amp;author=Bullmore%2CET">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Dann, B., Michaels, J. A., Schaffelhofer, S. &amp; Scherberger, H. Uniting functional network topology and oscillations in the fronto-parietal single unit network of behaving primates. <i>eLife</i> <b>5</b>, e15719 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27525488" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5019840" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Uniting%20functional%20network%20topology%20and%20oscillations%20in%20the%20fronto-parietal%20single%20unit%20network%20of%20behaving%20primates&amp;journal=eLife&amp;volume=5&amp;publication_year=2016&amp;author=Dann%2CB&amp;author=Michaels%2CJA&amp;author=Schaffelhofer%2CS&amp;author=Scherberger%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Park, H.-J. &amp; Friston, K. Structural and functional brain networks: from connections to cognition. <i>Science</i> <b>342</b>, 1238411 (2013).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Sporns, O. &amp; Zwi, J. D. The small world of the cerebral cortex. <i>Neuroinformatics</i> <b>2</b>, 145–162 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15319512" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20small%20world%20of%20the%20cerebral%20cortex&amp;journal=Neuroinformatics&amp;volume=2&amp;pages=145-162&amp;publication_year=2004&amp;author=Sporns%2CO&amp;author=Zwi%2CJD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Power, J. D., Schlaggar, B. L., Lessov-Schlaggar, C. N. &amp; Petersen, S. E. Evidence for hubs in human functional brain networks. <i>Neuron</i> <b>79</b>, 798–813 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtlahtrnI" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23972601" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20for%20hubs%20in%20human%20functional%20brain%20networks&amp;journal=Neuron&amp;volume=79&amp;pages=798-813&amp;publication_year=2013&amp;author=Power%2CJD&amp;author=Schlaggar%2CBL&amp;author=Lessov-Schlaggar%2CCN&amp;author=Petersen%2CSE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Sporns, O. &amp; Betzel, R. F. Modular brain networks. <i>Annu. Rev. Psychol.</i> <b>67</b>, 613–640 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26393868" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Modular%20brain%20networks&amp;journal=Annu.%20Rev.%20Psychol.&amp;volume=67&amp;pages=613-640&amp;publication_year=2016&amp;author=Sporns%2CO&amp;author=Betzel%2CRF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Bullmore, E. &amp; Sporns, O. The economy of brain network organization. <i>Nat. Rev. Neurosci.</i> <b>13</b>, 336–349 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XlsFSqt7w%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22498897" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20economy%20of%20brain%20network%20organization&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=13&amp;pages=336-349&amp;publication_year=2012&amp;author=Bullmore%2CE&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Bullmore, E. &amp; Sporns, O. Complex brain networks: graph theoretical analysis of structural and functional systems. <i>Nat. Rev. Neurosci.</i> <b>10</b>, 186–198 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtlygtrg%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19190637" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Complex%20brain%20networks%3A%20graph%20theoretical%20analysis%20of%20structural%20and%20functional%20systems&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=10&amp;pages=186-198&amp;publication_year=2009&amp;author=Bullmore%2CE&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: uses and interpretations. <i>Neuroimage</i> <b>52</b>, 1059–1069 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19819337" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Complex%20network%20measures%20of%20brain%20connectivity%3A%20uses%20and%20interpretations&amp;journal=Neuroimage&amp;volume=52&amp;pages=1059-1069&amp;publication_year=2010&amp;author=Rubinov%2CM&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Ahn, Y.-Y., Bagrow, J. P. &amp; Lehmann, S. Link communities reveal multiscale complexity in networks. <i>Nature</i> <b>466</b>, 761–764 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXns1Gjur0%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20562860" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Link%20communities%20reveal%20multiscale%20complexity%20in%20networks&amp;journal=Nature&amp;volume=466&amp;pages=761-764&amp;publication_year=2010&amp;author=Ahn%2CY-Y&amp;author=Bagrow%2CJP&amp;author=Lehmann%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Evans, T. &amp; Lambiotte, R. Line graphs, link partitions, and overlapping communities. <i>Phys. Rev. E</i> <b>80</b>, 016105 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD1MrjvFaqsg%3D%3D" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Line%20graphs%2C%20link%20partitions%2C%20and%20overlapping%20communities&amp;journal=Phys.%20Rev.%20E&amp;volume=80&amp;publication_year=2009&amp;author=Evans%2CT&amp;author=Lambiotte%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Eickhoff, S. B., Constable, R. T. &amp; Yeo, B. T. Topographic organization of the cerebral cortex and brain cartography. <i>Neuroimage</i> <b>170</b>, 332–347 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28219775" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Topographic%20organization%20of%20the%20cerebral%20cortex%20and%20brain%20cartography&amp;journal=Neuroimage&amp;volume=170&amp;pages=332-347&amp;publication_year=2018&amp;author=Eickhoff%2CSB&amp;author=Constable%2CRT&amp;author=Yeo%2CBT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">de Reus, M. A., Saenger, V. M., Kahn, R. S. &amp; van den Heuvel, M. P. An edge-centric perspective on the human connectome: link communities in the brain. <i>Philos. Trans. R. Soc. B Biol. Sci.</i> <b>369</b>, 20130527 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20edge-centric%20perspective%20on%20the%20human%20connectome%3A%20link%20communities%20in%20the%20brain&amp;journal=Philos.%20Trans.%20R.%20Soc.%20B%20Biol.%20Sci.&amp;volume=369&amp;publication_year=2014&amp;author=Reus%2CMA&amp;author=Saenger%2CVM&amp;author=Kahn%2CRS&amp;author=Heuvel%2CMP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Smith, S. M. et al. Network modelling methods for FMRI. <i>Neuroimage</i> <b>54</b>, 875–891 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20817103" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20modelling%20methods%20for%20FMRI&amp;journal=Neuroimage&amp;volume=54&amp;pages=875-891&amp;publication_year=2011&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Reid, A. T. et al. Advancing functional connectivity research from association to causation. <i>Nat. Neurosci.</i> <b>22</b>, 1751–1760 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFKgsr%2FF" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31611705" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7289187" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Advancing%20functional%20connectivity%20research%20from%20association%20to%20causation&amp;journal=Nat.%20Neurosci.&amp;volume=22&amp;pages=1751-1760&amp;publication_year=2019&amp;author=Reid%2CAT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Van Essen, D. C. et al. The WU-Minn Human Connectome Project: an overview. <i>Neuroimage</i> <b>80</b>, 62–79 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23684880" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3724347" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20WU-Minn%20Human%20Connectome%20Project%3A%20an%20overview&amp;journal=Neuroimage&amp;volume=80&amp;pages=62-79&amp;publication_year=2013&amp;author=Essen%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Gordon, E. M. et al. Precision functional mapping of individual human brains. <i>Neuron</i> <b>95</b>, 791–807 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1CmsLjF" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28757305" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5576360" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Precision%20functional%20mapping%20of%20individual%20human%20brains&amp;journal=Neuron&amp;volume=95&amp;pages=791-807&amp;publication_year=2017&amp;author=Gordon%2CEM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">O’Connor, D. et al. The healthy brain network serial scanning initiative: a resource for evaluating inter-individual differences and their reliabilities across scan conditions and sessions. <i>Gigascience</i> <b>6</b>, giw011 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20healthy%20brain%20network%20serial%20scanning%20initiative%3A%20a%20resource%20for%20evaluating%20inter-individual%20differences%20and%20their%20reliabilities%20across%20scan%20conditions%20and%20sessions&amp;journal=Gigascience&amp;volume=6&amp;publication_year=2017&amp;author=O%E2%80%99Connor%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">van Oort, E. S. et al. Functional parcellation using time courses of instantaneous connectivity. <i>Neuroimage</i> <b>170</b>, 31–40 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28716715" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20parcellation%20using%20time%20courses%20of%20instantaneous%20connectivity&amp;journal=Neuroimage&amp;volume=170&amp;pages=31-40&amp;publication_year=2018&amp;author=Oort%2CES">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Esfahlani, F. Z., Bertolero, M. A., Bassett, D. S. &amp; Betzel, R. F. Space-independent community and hub structure of functional brain networks. <i>Neuroimage</i> <b>211</b>, 116612 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Space-independent%20community%20and%20hub%20structure%20of%20functional%20brain%20networks&amp;journal=Neuroimage&amp;volume=211&amp;publication_year=2020&amp;author=Esfahlani%2CFZ&amp;author=Bertolero%2CMA&amp;author=Bassett%2CDS&amp;author=Betzel%2CRF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. <i>Cereb. Cortex</i> <b>28</b>, 3095–3114 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28981612" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Local%E2%80%93global%20parcellation%20of%20the%20human%20cerebral%20cortex%20from%20intrinsic%20functional%20connectivity%20MRI&amp;journal=Cereb.%20Cortex&amp;volume=28&amp;pages=3095-3114&amp;publication_year=2018&amp;author=Schaefer%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Laumann, T. O. et al. Functional system and areal organization of a highly sampled individual human brain. <i>Neuron</i> <b>87</b>, 657–670 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1ChsLrP" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26212711" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4642864" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20system%20and%20areal%20organization%20of%20a%20highly%20sampled%20individual%20human%20brain&amp;journal=Neuron&amp;volume=87&amp;pages=657-670&amp;publication_year=2015&amp;author=Laumann%2CTO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Amico, E. &amp; Goñi, J. The quest for identifiability in human functional connectomes. <i>Sci. Rep.</i> <b>8</b>, 8254 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29844466" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5973945" aria-label="PubMed Central reference 24">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20quest%20for%20identifiability%20in%20human%20functional%20connectomes&amp;journal=Sci.%20Rep.&amp;volume=8&amp;publication_year=2018&amp;author=Amico%2CE&amp;author=Go%C3%B1i%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Bassett, D. S. et al. Dynamic reconfiguration of human brain networks during learning. <i>Proc. Natl Acad. Sci. USA</i> <b>108</b>, 7641–7646 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXmtV2lsLk%3D" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21502525" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20reconfiguration%20of%20human%20brain%20networks%20during%20learning&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=7641-7646&amp;publication_year=2011&amp;author=Bassett%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. <i>Nat. Neurosci.</i> <b>18</b>, 1664–1671 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1ChurfI" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26457551" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5008686" aria-label="PubMed Central reference 26">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectome%20fingerprinting%3A%20identifying%20individuals%20using%20patterns%20of%20brain%20connectivity&amp;journal=Nat.%20Neurosci.&amp;volume=18&amp;pages=1664-1671&amp;publication_year=2015&amp;author=Finn%2CES">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Anderson, M. L., Kinnison, J. &amp; Pessoa, L. Describing functional diversity of brain regions and brain networks. <i>Neuroimage</i> <b>73</b>, 50–58 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23396162" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3756684" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Describing%20functional%20diversity%20of%20brain%20regions%20and%20brain%20networks&amp;journal=Neuroimage&amp;volume=73&amp;pages=50-58&amp;publication_year=2013&amp;author=Anderson%2CML&amp;author=Kinnison%2CJ&amp;author=Pessoa%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Pessoa, L. Understanding brain networks and brain organization. <i>Phys. Life Rev.</i> <b>11</b>, 400–435 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24819881" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4157099" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20brain%20networks%20and%20brain%20organization&amp;journal=Phys.%20Life%20Rev.&amp;volume=11&amp;pages=400-435&amp;publication_year=2014&amp;author=Pessoa%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Yeo, B. T., Krienen, F. M., Chee, M. W. &amp; Buckner, R. L. Estimates of segregation and overlap of functional connectivity networks in the human cerebral cortex. <i>Neuroimage</i> <b>88</b>, 212–227 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24185018" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimates%20of%20segregation%20and%20overlap%20of%20functional%20connectivity%20networks%20in%20the%20human%20cerebral%20cortex&amp;journal=Neuroimage&amp;volume=88&amp;pages=212-227&amp;publication_year=2014&amp;author=Yeo%2CBT&amp;author=Krienen%2CFM&amp;author=Chee%2CMW&amp;author=Buckner%2CRL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Wilf, M. et al. Spontaneously emerging patterns in human visual cortex reflect responses to naturalistic sensory stimuli. <i>Cereb. Cortex</i> <b>27</b>, 750–763 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26574501" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Spontaneously%20emerging%20patterns%20in%20human%20visual%20cortex%20reflect%20responses%20to%20naturalistic%20sensory%20stimuli&amp;journal=Cereb.%20Cortex&amp;volume=27&amp;pages=750-763&amp;publication_year=2017&amp;author=Wilf%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Baldassano, C., Hasson, U. &amp; Norman, K. A. Representation of real-world event schemas during narrative perception. <i>J. Neurosci.</i> <b>38</b>, 9689–9699 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXjvFylurk%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30249790" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6222059" aria-label="PubMed Central reference 31">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Representation%20of%20real-world%20event%20schemas%20during%20narrative%20perception&amp;journal=J.%20Neurosci.&amp;volume=38&amp;pages=9689-9699&amp;publication_year=2018&amp;author=Baldassano%2CC&amp;author=Hasson%2CU&amp;author=Norman%2CKA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Avena-Koenigsberger, A., Misic, B. &amp; Sporns, O. Communication dynamics in complex brain networks. <i>Nat. Rev. Neurosci.</i> <b>19</b>, 17 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFOht7zL" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Communication%20dynamics%20in%20complex%20brain%20networks&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=19&amp;publication_year=2018&amp;author=Avena-Koenigsberger%2CA&amp;author=Misic%2CB&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Shine, J. M. et al. Estimation of dynamic functional connectivity using multiplication of temporal derivatives. <i>Neuroimage</i> <b>122</b>, 399–407 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26231247" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimation%20of%20dynamic%20functional%20connectivity%20using%20multiplication%20of%20temporal%20derivatives&amp;journal=Neuroimage&amp;volume=122&amp;pages=399-407&amp;publication_year=2015&amp;author=Shine%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Liu, X. &amp; Duyn, J. H. Time-varying functional network information extracted from brief instances of spontaneous brain activity. <i>Proc. Natl Acad. Sci. USA</i> <b>110</b>, 4392–4397 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXmtV2qsro%3D" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23440216" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Time-varying%20functional%20network%20information%20extracted%20from%20brief%20instances%20of%20spontaneous%20brain%20activity&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=4392-4397&amp;publication_year=2013&amp;author=Liu%2CX&amp;author=Duyn%2CJH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Newman, M. E. &amp; Girvan, M. Finding and evaluating community structure in networks. <i>Phys. Rev. E</i> <b>69</b>, 026113 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD2c7gs1Giuw%3D%3D" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Finding%20and%20evaluating%20community%20structure%20in%20networks&amp;journal=Phys.%20Rev.%20E&amp;volume=69&amp;publication_year=2004&amp;author=Newman%2CME&amp;author=Girvan%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Rosvall, M. &amp; Bergstrom, C. T. Maps of random walks on complex networks reveal community structure. <i>Proc. Natl Acad. Sci. USA</i> <b>105</b>, 1118–1123 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhslSru7Y%3D" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18216267" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Maps%20of%20random%20walks%20on%20complex%20networks%20reveal%20community%20structure&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=1118-1123&amp;publication_year=2008&amp;author=Rosvall%2CM&amp;author=Bergstrom%2CCT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Bertolero, M. A., Yeo, B. T. &amp; D’Esposito, M. The modular and integrative functional architecture of the human brain. <i>Proc. Natl Acad. Sci. USA</i> <b>112</b>, E6798–E6807 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFWqsrfI" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26598686" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20modular%20and%20integrative%20functional%20architecture%20of%20the%20human%20brain&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=E6798-E6807&amp;publication_year=2015&amp;author=Bertolero%2CMA&amp;author=Yeo%2CBT&amp;author=D%E2%80%99Esposito%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Smith, S. M. et al. Correspondence of the brain’s functional architecture during activation and rest. <i>Proc. Natl Acad. Sci. USA</i> <b>106</b>, 13040–13045 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVKru7%2FJ" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19620724" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Correspondence%20of%20the%20brain%E2%80%99s%20functional%20architecture%20during%20activation%20and%20rest&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=13040-13045&amp;publication_year=2009&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Cole, M. W. et al. Task activations produce spurious but systematic inflation of task functional connectivity estimates. <i>Neuroimage</i> <b>189</b>, 1–18 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30597260" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Task%20activations%20produce%20spurious%20but%20systematic%20inflation%20of%20task%20functional%20connectivity%20estimates&amp;journal=Neuroimage&amp;volume=189&amp;pages=1-18&amp;publication_year=2019&amp;author=Cole%2CMW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. <i>Proc. Natl Acad. Sci. USA</i> <b>113</b>, 12574–12579 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ykurnM" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27791099" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Situating%20the%20default-mode%20network%20along%20a%20principal%20gradient%20of%20macroscale%20cortical%20organization&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=113&amp;pages=12574-12579&amp;publication_year=2016&amp;author=Margulies%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Esfahlani, F. Z. et al. High-amplitude co-fluctuations in cortical activity drive functional connectivity. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/800045" data-track="click" data-track-action="external reference" data-track-label="10.1101/800045">https://doi.org/10.1101/800045</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Lurie, D. J. et al. Questions and controversies in the study of time-varying functional connectivity in resting fMRI. <i>Netw. Neurosci.</i> <b>4</b>, 30–69 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32043043" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7006871" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Questions%20and%20controversies%20in%20the%20study%20of%20time-varying%20functional%20connectivity%20in%20resting%20fMRI&amp;journal=Netw.%20Neurosci.&amp;volume=4&amp;pages=30-69&amp;publication_year=2020&amp;author=Lurie%2CDJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">King, M., Hernandez-Castillo, C. R., Poldrack, R. A., Ivry, R. B. &amp; Diedrichsen, J. Functional boundaries in the human cerebellum revealed by a multi-domain task battery. <i>Nat. Neurosci.</i> <b>22</b>, 1371–1378 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtleis7nP" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31285616" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20boundaries%20in%20the%20human%20cerebellum%20revealed%20by%20a%20multi-domain%20task%20battery&amp;journal=Nat.%20Neurosci.&amp;volume=22&amp;pages=1371-1378&amp;publication_year=2019&amp;author=King%2CM&amp;author=Hernandez-Castillo%2CCR&amp;author=Poldrack%2CRA&amp;author=Ivry%2CRB&amp;author=Diedrichsen%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Pereira, F., Mitchell, T. &amp; Botvinick, M. Machine learning classifiers and fMRI: a tutorial overview. <i>Neuroimage</i> <b>45</b>, S199–S209 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19070668" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Machine%20learning%20classifiers%20and%20fMRI%3A%20a%20tutorial%20overview&amp;journal=Neuroimage&amp;volume=45&amp;pages=S199-S209&amp;publication_year=2009&amp;author=Pereira%2CF&amp;author=Mitchell%2CT&amp;author=Botvinick%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Huys, Q. J., Maia, T. V. &amp; Frank, M. J. Computational psychiatry as a bridge from neuroscience to clinical applications. <i>Nat. Neurosci.</i> <b>19</b>, 404 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XjtVOjt7Y%3D" aria-label="CAS reference 45">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26906507" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5443409" aria-label="PubMed Central reference 45">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Computational%20psychiatry%20as%20a%20bridge%20from%20neuroscience%20to%20clinical%20applications&amp;journal=Nat.%20Neurosci.&amp;volume=19&amp;publication_year=2016&amp;author=Huys%2CQJ&amp;author=Maia%2CTV&amp;author=Frank%2CMJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">McIntosh, A. R. &amp; Mišić, B. Multivariate statistical analyses for neuroimaging data. <i>Annu. Rev. Psychol.</i> <b>64</b>, 499–525 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22804773" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Multivariate%20statistical%20analyses%20for%20neuroimaging%20data&amp;journal=Annu.%20Rev.%20Psychol.&amp;volume=64&amp;pages=499-525&amp;publication_year=2013&amp;author=McIntosh%2CAR&amp;author=Mi%C5%A1i%C4%87%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. <i>Neuroimage</i> <b>60</b>, 2096–2106 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22343126" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20use%20of%20correlation%20as%20a%20measure%20of%20network%20connectivity&amp;journal=Neuroimage&amp;volume=60&amp;pages=2096-2106&amp;publication_year=2012&amp;author=Zalesky%2CA&amp;author=Fornito%2CA&amp;author=Bullmore%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Owen, L. L., Chang, T. H. &amp; Manning, J. R. High-level cognition during story listening is reflected in high-order dynamic correlations in neural activity patterns. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/763821" data-track="click" data-track-action="external reference" data-track-label="10.1101/763821">https://doi.org/10.1101/763821</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Sizemore, A. E., Phillips-Cremins, J. E., Ghrist, R. &amp; Bassett, D. S. The importance of the whole: topological data analysis for the network neuroscientist. <i>Netw. Neurosci.</i> <b>3</b>, 656–673 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31410372" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6663305" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20importance%20of%20the%20whole%3A%20topological%20data%20analysis%20for%20the%20network%20neuroscientist&amp;journal=Netw.%20Neurosci.&amp;volume=3&amp;pages=656-673&amp;publication_year=2019&amp;author=Sizemore%2CAE&amp;author=Phillips-Cremins%2CJE&amp;author=Ghrist%2CR&amp;author=Bassett%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Khambhati, A. N. et al. Dynamic network drivers of seizure generation, propagation and termination in human neocortical epilepsy. <i>PLoS Comput. Biol.</i> <b>11</b>, e1004608 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26680762" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4682976" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20network%20drivers%20of%20seizure%20generation%2C%20propagation%20and%20termination%20in%20human%20neocortical%20epilepsy&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=11&amp;publication_year=2015&amp;author=Khambhati%2CAN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Shine, J. M. et al. Human cognition involves the dynamic integration of neural activity and neuromodulatory systems. <i>Nat. Neurosci.</i> <b>22</b>, 289–296 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtF2jtbY%3D" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30664771" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20cognition%20involves%20the%20dynamic%20integration%20of%20neural%20activity%20and%20neuromodulatory%20systems&amp;journal=Nat.%20Neurosci.&amp;volume=22&amp;pages=289-296&amp;publication_year=2019&amp;author=Shine%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Davison, E. N. et al. Brain network adaptability across task states. <i>PLoS Comput. Biol.</i> <b>11</b>, e1004029 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25569227" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4287347" aria-label="PubMed Central reference 52">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Brain%20network%20adaptability%20across%20task%20states&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=11&amp;publication_year=2015&amp;author=Davison%2CEN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 105–124 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23668970" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3720813" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20minimal%20preprocessing%20pipelines%20for%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;volume=80&amp;pages=105-124&amp;publication_year=2013&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Robinson, E. C. et al. MSM: a new flexible framework for multimodal surface matching. <i>Neuroimage</i> <b>100</b>, 414–426 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24939340" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4190319" aria-label="PubMed Central reference 54">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=MSM%3A%20a%20new%20flexible%20framework%20for%20multimodal%20surface%20matching&amp;journal=Neuroimage&amp;volume=100&amp;pages=414-426&amp;publication_year=2014&amp;author=Robinson%2CEC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Esteban, O. et al. fMRIPrep: a robust preprocessing pipeline for functional MRI. <i>Nat. Methods</i> <b>16</b>, 111–116 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhurnN" aria-label="CAS reference 55">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30532080" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=fMRIPrep%3A%20a%20robust%20preprocessing%20pipeline%20for%20functional%20MRI&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=111-116&amp;publication_year=2019&amp;author=Esteban%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Gorgolewski, K. et al. Nipype: a flexible, lightweight and extensible neuroimaging data processing framework in Python. <i>Front. Neuroinform.</i> <b>5</b>, 13 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21897815" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3159964" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Nipype%3A%20a%20flexible%2C%20lightweight%20and%20extensible%20neuroimaging%20data%20processing%20framework%20in%20Python&amp;journal=Front.%20Neuroinform.&amp;volume=5&amp;publication_year=2011&amp;author=Gorgolewski%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Abraham, A. et al. Machine learning for neuroimaging with scikit-learn. <i>Front. Neuroinform.</i> <b>8</b>, 14 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24600388" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3930868" aria-label="PubMed Central reference 57">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Machine%20learning%20for%20neuroimaging%20with%20scikit-learn&amp;journal=Front.%20Neuroinform.&amp;volume=8&amp;publication_year=2014&amp;author=Abraham%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Avants, B. B., Epstein, C. L., Grossman, M. &amp; Gee, J. C. Symmetric diffeomorphic image registration with cross-correlation: evaluating automated labeling of elderly and neurodegenerative brain. <i>Med. Image Anal.</i> <b>12</b>, 26–41 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD1c7gtFyntA%3D%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17659998" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Symmetric%20diffeomorphic%20image%20registration%20with%20cross-correlation%3A%20evaluating%20automated%20labeling%20of%20elderly%20and%20neurodegenerative%20brain&amp;journal=Med.%20Image%20Anal.&amp;volume=12&amp;pages=26-41&amp;publication_year=2008&amp;author=Avants%2CBB&amp;author=Epstein%2CCL&amp;author=Grossman%2CM&amp;author=Gee%2CJC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Tustison, N. J. et al. N4ITK: improved N3 bias correction. <i>IEEE Trans. Med. Imaging</i> <b>29</b>, 1310–1320 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20378467" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3071855" aria-label="PubMed Central reference 59">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=N4ITK%3A%20improved%20N3%20bias%20correction&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;volume=29&amp;pages=1310-1320&amp;publication_year=2010&amp;author=Tustison%2CNJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Dale, A. M., Fischl, B. &amp; Sereno, M. I. Cortical surface-based analysis: I. Segmentation and surface reconstruction. <i>Neuroimage</i> <b>9</b>, 179–194 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1M7jt1Gisg%3D%3D" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20surface-based%20analysis%3A%20I.%20Segmentation%20and%20surface%20reconstruction&amp;journal=Neuroimage&amp;volume=9&amp;pages=179-194&amp;publication_year=1999&amp;author=Dale%2CAM&amp;author=Fischl%2CB&amp;author=Sereno%2CMI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Klein, A. et al. Mindboggling morphometry of human brains. <i>PLoS Comput. Biol.</i> <b>13</b>, e1005350 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28231282" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5322885" aria-label="PubMed Central reference 61">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Mindboggling%20morphometry%20of%20human%20brains&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=13&amp;publication_year=2017&amp;author=Klein%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Fonov, V. S., Evans, A. C., McKinstry, R. C., Almli, C. &amp; Collins, D. Unbiased nonlinear average age-appropriate brain templates from birth to adulthood. <i>Neuroimage</i> <b>47</b>, S102 (2009).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Zhang, Y., Brady, M. &amp; Smith, S. Segmentation of brain MR images through a hidden Markov random field model and the expectation-maximization algorithm. <i>IEEE Trans. Med. Imaging</i> <b>20</b>, 45–57 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3MzjvV2luw%3D%3D" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11293691" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Segmentation%20of%20brain%20MR%20images%20through%20a%20hidden%20Markov%20random%20field%20model%20and%20the%20expectation-maximization%20algorithm&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;volume=20&amp;pages=45-57&amp;publication_year=2001&amp;author=Zhang%2CY&amp;author=Brady%2CM&amp;author=Smith%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Jenkinson, M., Bannister, P., Brady, M. &amp; Smith, S. Improved optimization for the robust and accurate linear registration and motion correction of brain images. <i>Neuroimage</i> <b>17</b>, 825–841 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12377157" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20optimization%20for%20the%20robust%20and%20accurate%20linear%20registration%20and%20motion%20correction%20of%20brain%20images&amp;journal=Neuroimage&amp;volume=17&amp;pages=825-841&amp;publication_year=2002&amp;author=Jenkinson%2CM&amp;author=Bannister%2CP&amp;author=Brady%2CM&amp;author=Smith%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Wang, S. et al. Evaluation of field map and nonlinear registration methods for correction of susceptibility artifacts in diffusion MRI. <i>Front. Neuroinform.</i> <b>11</b>, 17 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28270762" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5318394" aria-label="PubMed Central reference 65">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluation%20of%20field%20map%20and%20nonlinear%20registration%20methods%20for%20correction%20of%20susceptibility%20artifacts%20in%20diffusion%20MRI&amp;journal=Front.%20Neuroinform.&amp;volume=11&amp;publication_year=2017&amp;author=Wang%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Treiber, J. M. et al. Characterization and correction of geometric distortions in 814 diffusion weighted images. <i>PLoS ONE</i> <b>11</b>, e0152472 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27027775" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4814112" aria-label="PubMed Central reference 66">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterization%20and%20correction%20of%20geometric%20distortions%20in%20814%20diffusion%20weighted%20images&amp;journal=PLoS%20ONE&amp;volume=11&amp;publication_year=2016&amp;author=Treiber%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Greve, D. N. &amp; Fischl, B. Accurate and robust brain image alignment using boundary-based registration. <i>Neuroimage</i> <b>48</b>, 63–72 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19573611" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2733527" aria-label="PubMed Central reference 67">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20and%20robust%20brain%20image%20alignment%20using%20boundary-based%20registration&amp;journal=Neuroimage&amp;volume=48&amp;pages=63-72&amp;publication_year=2009&amp;author=Greve%2CDN&amp;author=Fischl%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Power, J. D. et al. Methods to detect, characterize, and remove motion artifact in resting state fMRI. <i>Neuroimage</i> <b>84</b>, 320–341 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23994314" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Methods%20to%20detect%2C%20characterize%2C%20and%20remove%20motion%20artifact%20in%20resting%20state%20fMRI&amp;journal=Neuroimage&amp;volume=84&amp;pages=320-341&amp;publication_year=2014&amp;author=Power%2CJD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Esteban, O. et al. MRIQC: advancing the automatic prediction of image quality in MRI from unseen sites. <i>PLoS ONE</i> <b>12</b>, e0184661 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28945803" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5612458" aria-label="PubMed Central reference 69">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=MRIQC%3A%20advancing%20the%20automatic%20prediction%20of%20image%20quality%20in%20MRI%20from%20unseen%20sites&amp;journal=PLoS%20ONE&amp;volume=12&amp;publication_year=2017&amp;author=Esteban%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Parkes, L., Fulcher, B., Yücel, M. &amp; Fornito, A. An evaluation of the efficacy, reliability, and sensitivity of motion correction strategies for resting-state functional MRI. <i>Neuroimage</i> <b>171</b>, 415–436 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29278773" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20evaluation%20of%20the%20efficacy%2C%20reliability%2C%20and%20sensitivity%20of%20motion%20correction%20strategies%20for%20resting-state%20functional%20MRI&amp;journal=Neuroimage&amp;volume=171&amp;pages=415-436&amp;publication_year=2018&amp;author=Parkes%2CL&amp;author=Fulcher%2CB&amp;author=Y%C3%BCcel%2CM&amp;author=Fornito%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Yeo, B. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. <i>J. Neurophysiol.</i> <b>106</b>, 1126–1165 (2011).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Fischl, B. et al. Automatically parcellating the human cerebral cortex. <i>Cereb. Cortex</i> <b>14</b>, 11–22 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14654453" aria-label="PubMed reference 72">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Automatically%20parcellating%20the%20human%20cerebral%20cortex&amp;journal=Cereb.%20Cortex&amp;volume=14&amp;pages=11-22&amp;publication_year=2004&amp;author=Fischl%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Lindquist, M. A., Geuter, S., Wager, T. D. &amp; Caffo, B. S. Modular preprocessing pipelines can reintroduce artifacts into fMRI data. <i>Hum. Brain Mapp.</i> <b>40</b>, 2358–2376 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30666750" aria-label="PubMed reference 73">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6865661" aria-label="PubMed Central reference 73">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Modular%20preprocessing%20pipelines%20can%20reintroduce%20artifacts%20into%20fMRI%20data&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=40&amp;pages=2358-2376&amp;publication_year=2019&amp;author=Lindquist%2CMA&amp;author=Geuter%2CS&amp;author=Wager%2CTD&amp;author=Caffo%2CBS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Satterthwaite, T. D. et al. An improved framework for confound regression and filtering for control of motion artifact in the preprocessing of resting-state functional connectivity data. <i>Neuroimage</i> <b>64</b>, 240–256 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22926292" aria-label="PubMed reference 74">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20improved%20framework%20for%20confound%20regression%20and%20filtering%20for%20control%20of%20motion%20artifact%20in%20the%20preprocessing%20of%20resting-state%20functional%20connectivity%20data&amp;journal=Neuroimage&amp;volume=64&amp;pages=240-256&amp;publication_year=2013&amp;author=Satterthwaite%2CTD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Gopalan, P. K. &amp; Blei, D. M. Efficient discovery of overlapping communities in massive networks. <i>Proc. Natl Acad. Sci. USA</i> <b>110</b>, 14534–14539 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVKks7nM" aria-label="CAS reference 75">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23950224" aria-label="PubMed reference 75">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20discovery%20of%20overlapping%20communities%20in%20massive%20networks&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=14534-14539&amp;publication_year=2013&amp;author=Gopalan%2CPK&amp;author=Blei%2CDM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Yang, J. &amp; Leskovec, J. Overlapping communities explain core-periphery organization of networks. <i>Proc. IEEE</i> <b>102</b>, 1892–1902 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Overlapping%20communities%20explain%20core-periphery%20organization%20of%20networks&amp;journal=Proc.%20IEEE&amp;volume=102&amp;pages=1892-1902&amp;publication_year=2014&amp;author=Yang%2CJ&amp;author=Leskovec%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Psorakis, I., Roberts, S., Ebden, M. &amp; Sheldon, B. Overlapping community detection using Bayesian non-negative matrix factorization. <i>Phys. Rev. E</i> <b>83</b>, 066114 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Overlapping%20community%20detection%20using%20Bayesian%20non-negative%20matrix%20factorization&amp;journal=Phys.%20Rev.%20E&amp;volume=83&amp;publication_year=2011&amp;author=Psorakis%2CI&amp;author=Roberts%2CS&amp;author=Ebden%2CM&amp;author=Sheldon%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Najafi, M., McMenamin, B. W., Simon, J. Z. &amp; Pessoa, L. Overlapping communities reveal rich structure in large-scale brain networks during rest and task conditions. <i>Neuroimage</i> <b>135</b>, 92–106 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27129758" aria-label="PubMed reference 78">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915991" aria-label="PubMed Central reference 78">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Overlapping%20communities%20reveal%20rich%20structure%20in%20large-scale%20brain%20networks%20during%20rest%20and%20task%20conditions&amp;journal=Neuroimage&amp;volume=135&amp;pages=92-106&amp;publication_year=2016&amp;author=Najafi%2CM&amp;author=McMenamin%2CBW&amp;author=Simon%2CJZ&amp;author=Pessoa%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Guimera, R. &amp; Amaral, L. A. N. Functional cartography of complex metabolic networks. <i>Nature</i> <b>433</b>, 895–900 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXhsFOrtb4%3D" aria-label="CAS reference 79">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15729348" aria-label="PubMed reference 79">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2175124" aria-label="PubMed Central reference 79">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20cartography%20of%20complex%20metabolic%20networks&amp;journal=Nature&amp;volume=433&amp;pages=895-900&amp;publication_year=2005&amp;author=Guimera%2CR&amp;author=Amaral%2CLAN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Shinn, M. et al. Versatility of nodal affiliation to communities. <i>Sci. Rep.</i> <b>7</b>, 1–10 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlaqt7bK" aria-label="CAS reference 80">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Versatility%20of%20nodal%20affiliation%20to%20communities&amp;journal=Sci.%20Rep.&amp;volume=7&amp;pages=1-10&amp;publication_year=2017&amp;author=Shinn%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Pedersen, M., Zalesky, A., Omidvarnia, A. &amp; Jackson, G. D. Multilayer network switching rate predicts brain performance. <i>Proc. Natl Acad. Sci. USA</i> <b>115</b>, 13376–13381 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisF2rsrbK" aria-label="CAS reference 81">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30545918" aria-label="PubMed reference 81">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Multilayer%20network%20switching%20rate%20predicts%20brain%20performance&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=115&amp;pages=13376-13381&amp;publication_year=2018&amp;author=Pedersen%2CM&amp;author=Zalesky%2CA&amp;author=Omidvarnia%2CA&amp;author=Jackson%2CGD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Reichardt, J. &amp; Bornholdt, S. Statistical mechanics of community detection. <i>Phys. Rev. E</i> <b>74</b>, 016110 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20mechanics%20of%20community%20detection&amp;journal=Phys.%20Rev.%20E&amp;volume=74&amp;publication_year=2006&amp;author=Reichardt%2CJ&amp;author=Bornholdt%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Traag, V. A., Van Dooren, P. &amp; Nesterov, Y. Narrow scope for resolution-limit-free community detection. <i>Phys. Rev. E</i> <b>84</b>, 016114 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC3MjovVKqtQ%3D%3D" aria-label="CAS reference 83">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Narrow%20scope%20for%20resolution-limit-free%20community%20detection&amp;journal=Phys.%20Rev.%20E&amp;volume=84&amp;publication_year=2011&amp;author=Traag%2CVA&amp;author=Dooren%2CP&amp;author=Nesterov%2CY">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Bazzi, M. et al. Community detection in temporal multilayer networks, with an application to correlation networks. <i>Multiscale Model. Simul.</i> <b>14</b>, 1–41 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Community%20detection%20in%20temporal%20multilayer%20networks%2C%20with%20an%20application%20to%20correlation%20networks&amp;journal=Multiscale%20Model.%20Simul.&amp;volume=14&amp;pages=1-41&amp;publication_year=2016&amp;author=Bazzi%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Betzel, R. F. et al. The community structure of functional brain networks exhibits scale-specific patterns of inter-and intra-subject variability. <i>Neuroimage</i> <b>202</b>, 115990 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31291606" aria-label="PubMed reference 85">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20community%20structure%20of%20functional%20brain%20networks%20exhibits%20scale-specific%20patterns%20of%20inter-and%20intra-subject%20variability&amp;journal=Neuroimage&amp;volume=202&amp;publication_year=2019&amp;author=Betzel%2CRF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Mucha, P. J., Richardson, T., Macon, K., Porter, M. A. &amp; Onnela, J.-P. Community structure in time-dependent, multiscale, and multiplex networks. <i>Science</i> <b>328</b>, 876–878 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXlvVeltr8%3D" aria-label="CAS reference 86">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20466926" aria-label="PubMed reference 86">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=Community%20structure%20in%20time-dependent%2C%20multiscale%2C%20and%20multiplex%20networks&amp;journal=Science&amp;volume=328&amp;pages=876-878&amp;publication_year=2010&amp;author=Mucha%2CPJ&amp;author=Richardson%2CT&amp;author=Macon%2CK&amp;author=Porter%2CMA&amp;author=Onnela%2CJ-P">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Smith, S. M. et al. A positive–negative mode of population covariation links brain connectivity, demographics and behavior. <i>Nat. Neurosci.</i> <b>18</b>, 1565–1567 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7vN" aria-label="CAS reference 87">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26414616" aria-label="PubMed reference 87">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4625579" aria-label="PubMed Central reference 87">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20positive%E2%80%93negative%20mode%20of%20population%20covariation%20links%20brain%20connectivity%2C%20demographics%20and%20behavior&amp;journal=Nat.%20Neurosci.&amp;volume=18&amp;pages=1565-1567&amp;publication_year=2015&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-020-00719-y?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>This research was supported by the Indiana University Office of the Vice President for Research Emerging Area of Research Initiative, Learning: Brains, Machines and Children (F.Z.E. and R.F.B.). This material is based on work supported by the National Science Foundation Graduate Research Fellowship under grant no. 1342962 (J.F.). This research was supported, in part, by the Lilly Endowment, through its support for the Indiana University Pervasive Technology Institute and, in part, by the Indiana METACyt Initiative. The Indiana METACyt Initiative at Indiana University was also supported, in part, by the Lilly Endowment. Data were provided, in part, by the Human Connectome Project, WU-Minn Consortium (principal investigators: D. Van Essen and K. Ugurbil; 1U54MH091657), funded by the 16 National Institues of Health (NIH) institutes and centers that support the NIH Blueprint for Neuroscience Research and by the McDonnell Center for Systems Neuroscience at Washington University. We thank B. Mišić for reading an early version of this manuscript.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Department of Psychological and Brain Sciences, Indiana University, Bloomington, IN, USA</p><p class="c-article-author-affiliation__authors-list">Joshua Faskowitz, Farnaz Zamani Esfahlani, Youngheun Jo, Olaf Sporns &amp; Richard F. Betzel</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Program in Neuroscience, Indiana University, Bloomington, IN, USA</p><p class="c-article-author-affiliation__authors-list">Joshua Faskowitz, Olaf Sporns &amp; Richard F. Betzel</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Cognitive Science Program, Indiana University, Bloomington, IN, USA</p><p class="c-article-author-affiliation__authors-list">Olaf Sporns &amp; Richard F. Betzel</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Network Science Institute, Indiana University, Bloomington, IN, USA</p><p class="c-article-author-affiliation__authors-list">Olaf Sporns &amp; Richard F. Betzel</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Joshua-Faskowitz-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Joshua Faskowitz</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Joshua%20Faskowitz" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joshua%20Faskowitz" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joshua%20Faskowitz%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Farnaz_Zamani-Esfahlani-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Farnaz Zamani Esfahlani</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Farnaz%20Zamani%20Esfahlani" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Farnaz%20Zamani%20Esfahlani" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Farnaz%20Zamani%20Esfahlani%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Youngheun-Jo-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Youngheun Jo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Youngheun%20Jo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Youngheun%20Jo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Youngheun%20Jo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Olaf-Sporns-Aff1-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Olaf Sporns</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Olaf%20Sporns" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Olaf%20Sporns" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Olaf%20Sporns%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Richard_F_-Betzel-Aff1-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Richard F. Betzel</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Richard%20F.%20Betzel" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Richard%20F.%20Betzel" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Richard%20F.%20Betzel%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>J.F. and R.F.B. conceived of the study, processed data, carried out all analyses and wrote the first draft of the manuscript. F.Z.E., Y.J. and O.S. contributed to project direction via discussion. All authors helped revise and write the submitted manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:rbetzel@indiana.edu">Richard F. Betzel</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">

<h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
<p>The authors declare no competing interests.</p>

</div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> <i>Nature Neuroscience</i> thanks Lucina Uddin, Andrew Zalesky, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec35-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec35">Supplementary information</h2><div class="c-article-section__content" id="Sec35-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figs. 1–24 and Supplementary Table 1.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-020-00719-y/MediaObjects/41593_2020_719_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Edge-centric%20functional%20network%20representations%20of%20human%20cerebral%20cortex%20reveal%20overlapping%20system-level%20architecture&amp;author=Joshua%20Faskowitz%20et%20al&amp;contentID=10.1038%2Fs41593-020-00719-y&amp;copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2020-10-19&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-020-00719-y" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-020-00719-y" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Faskowitz, J., Esfahlani, F.Z., Jo, Y. <i>et al.</i> Edge-centric functional network representations of human cerebral cortex reveal overlapping system-level architecture.
                    <i>Nat Neurosci</i> <b>23</b>, 1644–1654 (2020). https://doi.org/10.1038/s41593-020-00719-y</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-020-00719-y?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-09-09">09 September 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-09-03">03 September 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-10-19">19 October 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-12">December 2020</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-020-00719-y</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Modular subgraphs in large-scale connectomes underpin spontaneous co-fluctuation events in mouse and human brains" href="https://doi.org/10.1038/s42003-024-05766-w">
                                        Modular subgraphs in large-scale connectomes underpin spontaneous co-fluctuation events in mouse and human brains
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Elisabeth Ragone</li><li>Jacob Tanner</li><li>Richard Betzel</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Personalized functional network mapping for autism spectrum disorder and attention-deficit/hyperactivity disorder" href="https://doi.org/10.1038/s41398-024-02797-z">
                                        Personalized functional network mapping for autism spectrum disorder and attention-deficit/hyperactivity disorder
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jiang Zhang</li><li>Zhiwei Zhang</li><li>Jiaojian Wang</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Translational Psychiatry</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Multimodal measures of spontaneous brain activity reveal both common and divergent patterns of cortical functional organization" href="https://doi.org/10.1038/s41467-023-44363-z">
                                        Multimodal measures of spontaneous brain activity reveal both common and divergent patterns of cortical functional organization
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Hadi Vafaii</li><li>Francesca Mandino</li><li>Luiz Pessoa</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Silence Practice Modulates the Resting State Functional Connectivity of Language Network with Default Mode and Dorsal Attention Networks in Long-Term Meditators" href="https://doi.org/10.1007/s12671-024-02316-7">
                                        Silence Practice Modulates the Resting State Functional Connectivity of Language Network with Default Mode and Dorsal Attention Networks in Long-Term Meditators
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Vaibhav Tripathi</li><li>Kathryn J. Devaney</li><li>David C. Somers</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Mindfulness</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:The effect of turbulence in brain dynamics information transfer measured with magnetoencephalography" href="https://doi.org/10.1038/s42005-023-01192-2">
                                        The effect of turbulence in brain dynamics information transfer measured with magnetoencephalography
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Gustavo Deco</li><li>Samuel Liebana Garcia</li><li>Morten L. Kringelbach</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Physics</i> (2023)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00719-y.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00719-y.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                        <div class="c-article-associated-content__collection focus u-mb-24">
                            <section>
                                <p class="c-article-associated-content__collection-label u-sans-serif u-text-bold u-mb-8">Focus</p>
                                <h3 class="c-article-associated-content__collection-title u-h3 u-mb-8">
                                    <a href="https://www.nature.com/collections/hhgegahcbd"
                                       class="u-link-inherit"
                                       data-track="click"
                                       data-track-action="view focus"
                                       data-track-category="associated content"
                                       data-track-label="focus"
                                       data-test="collection-link">Focus on neuroscience methods</a>
                                </h3>
                            </section>
                        </div>
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41593-020-00741-0"
                       class="c-card__link u-link-inherit"
                       data-track="click"
                       data-track-action="view article"
                       data-track-category="associated content"
                       
                       data-track-label="news_and_views">An ‘edgy’ new look</a>
                </h3>
                
<ul data-test="author-list" class="c-author-list c-author-list--compact">
    <li>Lucina Q. Uddin</li>
</ul>

                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Neuroscience</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">News &amp; Views</span></span>
        
        
            <time class="c-meta__item" datetime="2020-11-02">02 Nov 2020</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "focus;news_and_views;";
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-020-00719-y;doi=10.1038/s41593-020-00719-y;subjmeta=116,1925,2649,378,631;kwrd=Cognitive+neuroscience,Network+models">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=5746509&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-020-00719-y%26doi%3D10.1038/s41593-020-00719-y%26subjmeta%3D116,1925,2649,378,631%26kwrd%3DCognitive+neuroscience,Network+models">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=5746509&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-020-00719-y%26doi%3D10.1038/s41593-020-00719-y%26subjmeta%3D116,1925,2649,378,631%26kwrd%3DCognitive+neuroscience,Network+models"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-020-00719-y&amp;format=js&amp;last_modified=2020-10-19" async></script>
<img src="/p87ldvqh/article/s41593-020-00719-y" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>