<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Challenges and future directions for representations of functional brain organization | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"review article","legacy":{"webtrendsPrimaryArticleType":"reviews","webtrendsSubjectTerms":"biomarkers;functional-magnetic-resonance-imaging;neuroscience","webtrendsContentCategory":null,"webtrendsContentCollection":"Focus on neuroscience methods","webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Review Article"}},"article":{"doi":"10.1038/s41593-020-00726-z"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Janine Bijsterbosch","Samuel J. Harrison","Saad Jbabdi","Mark Woolrich","Christian Beckmann","Stephen Smith","Eugene P. Duff"],"publishedAt":1603670400,"publishedAtString":"2020-10-26","title":"Challenges and future directions for representations of functional brain organization","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"23","issue":"12"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":{"id":"hhgegahcbd"}},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Challenges and future directions for representations of functional brain organization","description":"A key principle of brain organization is the functional integration of brain regions into interconnected networks. Functional MRI scans acquired at rest offer insights into functional integration via patterns of coherent fluctuations in spontaneous activity, known as functional connectivity. These patterns have been studied intensively and have been linked to cognition and disease. However, the field is fractionated. Diverging analysis approaches have segregated the community into research silos, limiting the replication and clinical translation of findings. A primary source of this fractionation is the diversity of approaches used to reduce complex brain data into a lower-dimensional set of features for analysis and interpretation, which we refer to as brain representations. In this Primer, we provide an overview of different brain representations, lay out the challenges that have led to the fractionation of the field and that continue to form obstacles for convergence, and propose concrete guidelines to unite the field. In this Primer article, Bijsterbosch and colleagues provide an accessible discussion of the challenges faced in analytical representations of functional brain organization and provide clear recommendations to unite a fractionated field.","datePublished":"2020-10-26T00:00:00Z","dateModified":"2020-10-26T00:00:00Z","pageStart":"1484","pageEnd":"1495","sameAs":"https://doi.org/10.1038/s41593-020-00726-z","keywords":["Biomarkers","Functional magnetic resonance imaging","Neuroscience","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig3_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"23","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Janine Bijsterbosch","url":"http://orcid.org/0000-0002-1385-9178","affiliation":[{"name":"Washington University in St Louis","address":{"name":"Mallinckrodt Institute of Radiology, Washington University in St Louis, Saint Louis, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Oxford. John Radcliffe Hospital","address":{"name":"Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"Janine.Bijsterbosch@wustl.edu","@type":"Person"},{"name":"Samuel J. Harrison","url":"http://orcid.org/0000-0002-5886-2389","affiliation":[{"name":"University of Oxford. John Radcliffe Hospital","address":{"name":"Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Zurich & ETH Zurich","address":{"name":"Translational Neuromodeling Unit, University of Zurich & ETH Zurich, Zurich, Switzerland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Saad Jbabdi","affiliation":[{"name":"University of Oxford. John Radcliffe Hospital","address":{"name":"Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Mark Woolrich","affiliation":[{"name":"University of Oxford, Warneford Hospital","address":{"name":"Oxford Centre for Human Brain Activity (OHBA), Wellcome Centre for Integrative Neuroimaging, Department of Psychiatry, University of Oxford, Warneford Hospital, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christian Beckmann","affiliation":[{"name":"Radboud University Medical Centre","address":{"name":"Donders Institute and Department of Cognitive Neurosciences, Radboud University Medical Centre, Nijmegen, The Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stephen Smith","url":"http://orcid.org/0000-0001-8166-069X","affiliation":[{"name":"University of Oxford. John Radcliffe Hospital","address":{"name":"Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Eugene P. Duff","url":"http://orcid.org/0000-0001-8795-5472","affiliation":[{"name":"University of Oxford. John Radcliffe Hospital","address":{"name":"Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Oxford, John Radcliffe Hospital","address":{"name":"Department of Paediatrics, University of Oxford, John Radcliffe Hospital, Oxford, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"eugene.duff@ndcn.ox.ac.uk","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-020-00726-z">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="Challenges and future directions for representations of functional brain organization"/>
    <meta name="dc.source" content="Nature Neuroscience 2020 23:12"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2020-10-26"/>
    <meta name="dc.type" content="ReviewPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2020 Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2020 Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="A key principle of brain organization is the functional integration of brain regions into interconnected networks. Functional MRI scans acquired at rest offer insights into functional integration via patterns of coherent fluctuations in spontaneous activity, known as functional connectivity. These patterns have been studied intensively and have been linked to cognition and disease. However, the field is fractionated. Diverging analysis approaches have segregated the community into research silos, limiting the replication and clinical translation of findings. A primary source of this fractionation is the diversity of approaches used to reduce complex brain data into a lower-dimensional set of features for analysis and interpretation, which we refer to as brain representations. In this Primer, we provide an overview of different brain representations, lay out the challenges that have led to the fractionation of the field and that continue to form obstacles for convergence, and propose concrete guidelines to unite the field. In this Primer article, Bijsterbosch and colleagues provide an accessible discussion of the challenges faced in analytical representations of functional brain organization and provide clear recommendations to unite a fractionated field."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2020-10-26"/>
    <meta name="prism.volume" content="23"/>
    <meta name="prism.number" content="12"/>
    <meta name="prism.section" content="ReviewPaper"/>
    <meta name="prism.startingPage" content="1484"/>
    <meta name="prism.endingPage" content="1495"/>
    <meta name="prism.copyright" content="2020 Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-020-00726-z"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-020-00726-z"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-020-00726-z.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-020-00726-z"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="Challenges and future directions for representations of functional brain organization"/>
    <meta name="citation_volume" content="23"/>
    <meta name="citation_issue" content="12"/>
    <meta name="citation_publication_date" content="2020/12"/>
    <meta name="citation_online_date" content="2020/10/26"/>
    <meta name="citation_firstpage" content="1484"/>
    <meta name="citation_lastpage" content="1495"/>
    <meta name="citation_article_type" content="Review Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-020-00726-z"/>
    <meta name="DOI" content="10.1038/s41593-020-00726-z"/>
    <meta name="size" content="197926"/>
    <meta name="citation_doi" content="10.1038/s41593-020-00726-z"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-020-00726-z&amp;api_key="/>
    <meta name="description" content="A key principle of brain organization is the functional integration of brain regions into interconnected networks. Functional MRI scans acquired at rest offer insights into functional integration via patterns of coherent fluctuations in spontaneous activity, known as functional connectivity. These patterns have been studied intensively and have been linked to cognition and disease. However, the field is fractionated. Diverging analysis approaches have segregated the community into research silos, limiting the replication and clinical translation of findings. A primary source of this fractionation is the diversity of approaches used to reduce complex brain data into a lower-dimensional set of features for analysis and interpretation, which we refer to as brain representations. In this Primer, we provide an overview of different brain representations, lay out the challenges that have led to the fractionation of the field and that continue to form obstacles for convergence, and propose concrete guidelines to unite the field. In this Primer article, Bijsterbosch and colleagues provide an accessible discussion of the challenges faced in analytical representations of functional brain organization and provide clear recommendations to unite a fractionated field."/>
    <meta name="dc.creator" content="Bijsterbosch, Janine"/>
    <meta name="dc.creator" content="Harrison, Samuel J."/>
    <meta name="dc.creator" content="Jbabdi, Saad"/>
    <meta name="dc.creator" content="Woolrich, Mark"/>
    <meta name="dc.creator" content="Beckmann, Christian"/>
    <meta name="dc.creator" content="Smith, Stephen"/>
    <meta name="dc.creator" content="Duff, Eugene P."/>
    <meta name="dc.subject" content="Biomarkers"/>
    <meta name="dc.subject" content="Functional magnetic resonance imaging"/>
    <meta name="dc.subject" content="Neuroscience"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans; citation_author=DC Essen, MF Glasser; citation_volume=99; citation_publication_date=2018; citation_pages=640-663; citation_id=CR1"/>
    <meta name="citation_reference" content="Bijsterbosch, J., Smith, S.M. &amp; Beckmann, C.F. Introduction to Resting State fMRI Functional Connectivity (Oxford Univ. Press, 2017)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Advancing functional connectivity research from association to causation; citation_author=AT Reid; citation_volume=22; citation_publication_date=2019; citation_pages=1751-1760; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Variability in the analysis of a single neuroimaging dataset by many teams; citation_author=R Botvinik-Nezer; citation_volume=582; citation_publication_date=2020; citation_pages=84-88; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Imaging-based parcellations of the human brain; citation_author=SB Eickhoff, BTT Yeo, S Genon; citation_volume=19; citation_publication_date=2018; citation_pages=672-686; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Psychiatry; citation_title=Computational psychiatry: a report from the 2017 NIMH workshop on opportunities and challenges; citation_author=M Ferrante; citation_volume=24; citation_publication_date=2019; citation_pages=479-483; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=JAMA Psychiatry; citation_title=Establishment of best practices for evidence for prediction: a review; citation_author=RA Poldrack, G Huckins, G Varoquaux; citation_volume=77; citation_publication_date=2020; citation_pages=534-540; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci. Methods; citation_title=Detecting cell assemblies in large neuronal populations; citation_author=V Lopes-dos-Santos, S Ribeiro, ABL Tort; citation_volume=220; citation_publication_date=2013; citation_pages=149-166; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Resting-state fMRI in the Human Connectome Project; citation_author=SM Smith; citation_volume=80; citation_publication_date=2013; citation_pages=144-168; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A multi-modal parcellation of human cerebral cortex; citation_author=MF Glasser; citation_volume=536; citation_publication_date=2016; citation_pages=171-178; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Connectivity-based parcellation: critique and implications; citation_author=SB Eickhoff, B Thirion, G Varoquaux, D Bzdok; citation_volume=36; citation_publication_date=2015; citation_pages=4771-4792; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Topogr.; citation_title=Towards a universal taxonomy of macro-scale functional human brain networks; citation_author=LQ Uddin, BTT Yeo, RN Spreng; citation_volume=32; citation_publication_date=2019; citation_pages=926-942; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Topographic organization of the cerebral cortex and brain cartography; citation_author=SB Eickhoff, RT Constable, BTT Yeo; citation_volume=170; citation_publication_date=2018; citation_pages=332-347; citation_id=CR13"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain; citation_author=N Tzourio-Mazoyer; citation_volume=15; citation_publication_date=2002; citation_pages=273-289; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest; citation_author=RS Desikan; citation_volume=31; citation_publication_date=2006; citation_pages=968-980; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data; citation_author=SB Eickhoff; citation_volume=25; citation_publication_date=2005; citation_pages=1325-1335; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Benchmarking functional connectome-based predictive models for resting-state fMRI; citation_author=K Dadi; citation_volume=192; citation_publication_date=2019; citation_pages=115-134; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Local-global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI; citation_author=A Schaefer; citation_volume=28; citation_publication_date=2018; citation_pages=3095-3114; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Groupwise whole-brain parcellation from resting-state fMRI data for network node identification; citation_author=X Shen, F Tokoglu, X Papademetris, RT Constable; citation_volume=82; citation_publication_date=2013; citation_pages=403-415; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=A whole brain fMRI atlas generated via spatially constrained spectral clustering; citation_author=RC Craddock, GA James, PE Holtzheimer, XP Hu, HS Mayberg; citation_volume=33; citation_publication_date=2012; citation_pages=1914-1928; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Generation and evaluation of a cortical area parcellation from resting-state correlations; citation_author=EM Gordon; citation_volume=26; citation_publication_date=2016; citation_pages=288-303; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurophysiol.; citation_title=The organization of the human cerebral cortex estimated by intrinsic functional connectivity; citation_author=BTT Yeo; citation_volume=106; citation_publication_date=2011; citation_pages=1125-1165; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Phil. Trans. R. Soc. Lond. B; citation_title=Investigations into resting-state connectivity using independent component analysis; citation_author=CF Beckmann, M DeLuca, JT Devlin, SM Smith; citation_volume=360; citation_publication_date=2005; citation_pages=1001-1013; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Large-scale probabilistic functional modes from resting state fMRI; citation_author=SJ Harrison; citation_volume=109; citation_publication_date=2015; citation_pages=217-231; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Inf. Process. Med. Imaging; citation_title=Multi-subject dictionary learning to segment an atlas of brain spontaneous activity; citation_author=G Varoquaux, A Gramfort, F Pedregosa, V Michel, B Thirion; citation_volume=22; citation_publication_date=2011; citation_pages=562-573; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Identifying sparse connectivity patterns in the brain using resting-state fMRI; citation_author=H Eavani; citation_volume=105; citation_publication_date=2015; citation_pages=286-299; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Functional and effective connectivity in neuroimaging: a synthesis; citation_author=KJ Friston; citation_volume=2; citation_publication_date=1994; citation_pages=56-78; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=Functional connectomics from resting-state fMRI; citation_author=SM Smith; citation_volume=17; citation_publication_date=2013; citation_pages=666-682; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Multi-scale brain networks; citation_author=RF Betzel, DS Bassett; citation_volume=160; citation_publication_date=2017; citation_pages=73-83; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity; citation_author=ES Finn; citation_volume=18; citation_publication_date=2015; citation_pages=1664-1671; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A positive-negative mode of population covariation links brain connectivity, demographics and behavior; citation_author=SM Smith; citation_volume=18; citation_publication_date=2015; citation_pages=1565-1567; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Neurosci.; citation_title=Using dual regression to investigate network shape and amplitude in functional connectivity analyses; citation_author=LD Nickerson, SM Smith, D &#214;ng&#252;r, CF Beckmann; citation_volume=11; citation_publication_date=2017; citation_pages=115; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Dynamic functional connectivity: promise, issues, and interpretations; citation_author=RM Hutchison; citation_volume=80; citation_publication_date=2013; citation_pages=360-378; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The dynamic functional connectome: state-of-the-art and perspectives; citation_author=MG Preti, TA Bolton, D Ville; citation_volume=160; citation_publication_date=2017; citation_pages=41-54; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Tracking whole-brain connectivity dynamics in the resting state; citation_author=EA Allen; citation_volume=24; citation_publication_date=2014; citation_pages=663-676; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Spontaneous cortical activity transiently organises into frequency specific phase-coupling networks; citation_author=D Vidaurre; citation_volume=9; citation_publication_date=2018; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci. Methods; citation_title=An improved approach to detection of amplitude of low-frequency fluctuation (ALFF) for resting-state fMRI: fractional ALFF; citation_author=Q-H Zou; citation_volume=172; citation_publication_date=2008; citation_pages=137-141; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Multimodal population brain imaging in the UK Biobank prospective epidemiological study; citation_author=KL Miller; citation_volume=19; citation_publication_date=2016; citation_pages=1523-1536; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Investigations into within- and between-subject resting-state amplitude variations; citation_author=J Bijsterbosch; citation_volume=159; citation_publication_date=2017; citation_pages=57-69; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=The relationship between spatial configuration and functional connectivity of brain regions revisited; citation_author=JD Bijsterbosch, CF Beckmann, MW Woolrich, SM Smith, SJ Harrison; citation_volume=8; citation_publication_date=2019; citation_pages=e44890; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion; citation_author=R Kong; citation_volume=29; citation_publication_date=2019; citation_pages=2533-2551; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Disambiguating brain functional connectivity; citation_author=EP Duff, T Makin, M Cottaar, SM Smith, MW Woolrich; citation_volume=173; citation_publication_date=2018; citation_pages=540-550; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=Large-scale gradients in human cortical organization; citation_author=JM Huntenburg, P-L Bazin, DS Margulies; citation_volume=22; citation_publication_date=2018; citation_pages=21-31; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Hum. Behav.; citation_title=Functional corticostriatal connection topographies predict goal directed behaviour in humans; citation_author=AF Marquand, KV Haak, CF Beckmann; citation_volume=1; citation_publication_date=2017; citation_pages=0146; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Situating the default-mode network along a principal gradient of macroscale cortical organization; citation_author=DS Margulies; citation_volume=113; citation_publication_date=2016; citation_pages=12574-12579; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Spatiotemporal dynamics of low frequency BOLD fluctuations in rats and humans; citation_author=W Majeed; citation_volume=54; citation_publication_date=2011; citation_pages=1140-1150; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Building better biomarkers: brain models in translational neuroimaging; citation_author=C-W Woo, LJ Chang, MA Lindquist, TD Wager; citation_volume=20; citation_publication_date=2017; citation_pages=365-377; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=From the connectome to brain function; citation_author=CI Bargmann, E Marder; citation_volume=10; citation_publication_date=2013; citation_pages=483-490; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Neurobiol.; citation_title=The topographic connectome; citation_author=S Jbabdi, SN Sotiropoulos, TE Behrens; citation_volume=23; citation_publication_date=2013; citation_pages=207-215; citation_id=CR49"/>
    <meta name="citation_reference" content="Paquola, C. et al. The cortical wiring scheme of hierarchical information processing. Preprint at bioRxiv 
https://doi.org/10.1101/2020.01.08.899583

 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The community structure of functional brain networks exhibits scale-specific patterns of inter- and intra-subject variability; citation_author=RF Betzel; citation_volume=202; citation_publication_date=2019; citation_pages=115990; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=What we can do and what we cannot do with fMRI; citation_author=NK Logothetis; citation_volume=453; citation_publication_date=2008; citation_pages=869-878; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Multiband multislice GE-EPI at 7 tesla, with 16-fold acceleration using partial parallel imaging with application to high spatial and temporal whole-brain fMRI; citation_author=S Moeller; citation_volume=63; citation_publication_date=2010; citation_pages=1144-1153; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci. Methods; citation_title=Comparing brain graphs in which nodes are regions of interest or independent components: A simulation study; citation_author=Q Yu; citation_volume=291; citation_publication_date=2017; citation_pages=61-68; citation_id=CR54"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Deriving reproducible biomarkers from multi-site resting-state data: An Autism-based example; citation_author=A Abraham; citation_volume=147; citation_publication_date=2017; citation_pages=736-745; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Task-driven ICA feature generation for accurate and interpretable prediction using fMRI; citation_author=EP Duff; citation_volume=60; citation_publication_date=2012; citation_pages=189-203; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Optimising network modelling methods for fMRI; citation_author=U Pervaiz, D Vidaurre, MW Woolrich, SM Smith; citation_volume=211; citation_publication_date=2020; citation_pages=116604; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Network modelling methods for FMRI; citation_author=SM Smith; citation_volume=54; citation_publication_date=2011; citation_pages=875-891; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Capturing inter-subject variability with group independent component analysis of fMRI data: a simulation study; citation_author=EA Allen, EB Erhardt, Y Wei, T Eichele, VD Calhoun; citation_volume=59; citation_publication_date=2012; citation_pages=4141-4159; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=Inter-individual differences in human brain structure and morphology link to variation in demographics and behavior; citation_author=A Llera, T Wolfers, null Mulders, CF Beckmann; citation_volume=8; citation_publication_date=2019; citation_pages=e44443; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Structural variability in the human brain reflects fine-grained functional architecture at the population level; citation_author=S Smith; citation_volume=39; citation_publication_date=2019; citation_pages=6136-6149; citation_id=CR61"/>
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=The relationship between spatial configuration and functional connectivity of brain regions; citation_author=JD Bijsterbosch; citation_volume=7; citation_publication_date=2018; citation_pages=e32992; citation_id=CR62"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Task-free MRI predicts individual differences in brain activity during task performance; citation_author=I Tavor; citation_volume=352; citation_publication_date=2016; citation_pages=216-220; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=The impact of traditional neuroimaging methods on the spatial localization of cortical areas; citation_author=TS Coalson, DC Essen, MF Glasser; citation_volume=115; citation_publication_date=2018; citation_pages=E6356-E6365; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=MSM: a new flexible framework for multimodal surface matching; citation_author=EC Robinson; citation_volume=100; citation_publication_date=2014; citation_pages=414-426; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Parallel interdigitated distributed networks within the individual estimated by intrinsic functional connectivity; citation_author=RM Braga, RL Buckner; citation_volume=95; citation_publication_date=2017; citation_pages=457-471.e5; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Long-term neural and physiological phenotyping of a single human; citation_author=RA Poldrack; citation_volume=6; citation_publication_date=2015; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Trait-like variants in human functional brain networks; citation_author=BA Seitzman; citation_volume=116; citation_publication_date=2019; citation_pages=22851-22861; citation_id=CR68"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Resting state network estimation in individual subjects; citation_author=CD Hacker; citation_volume=82; citation_publication_date=2013; citation_pages=616-633; citation_id=CR69"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Precision functional mapping of individual human brains; citation_author=EM Gordon; citation_volume=95; citation_publication_date=2017; citation_pages=791-807.e7; citation_id=CR70"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Modelling subject variability in the spatial and temporal characteristics of functional modes; citation_author=SJ Harrison; citation_volume=222; citation_publication_date=2020; citation_pages=117226; citation_id=CR71"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Individual differences in functional connectivity during naturalistic viewing conditions; citation_author=T Vanderwal; citation_volume=157; citation_publication_date=2017; citation_pages=521-530; citation_id=CR72"/>
    <meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.; citation_title=A computational model of shared fine-scale structure in the human connectome; citation_author=JS Guntupalli, M Feilong, JV Haxby; citation_volume=14; citation_publication_date=2018; citation_pages=e1006120; citation_id=CR73"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS One; citation_title=Large-scale intrinsic connectivity is consistent across varying task demands; citation_author=P Kieliba, S Madugula, N Filippini, EP Duff, TR Makin; citation_volume=14; citation_publication_date=2019; citation_pages=e0213861; citation_id=CR74"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The individual functional connectome is unique and stable over months to years; citation_author=C Horien, X Shen, D Scheinost, RT Constable; citation_volume=189; citation_publication_date=2019; citation_pages=676-687; citation_id=CR75"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Functional brain networks are dominated by stable group and individual factors, not cognitive or daily variation; citation_author=C Gratton; citation_volume=98; citation_publication_date=2018; citation_pages=439-452.e5; citation_id=CR76"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=There is no single functional atlas even for a single individual: Functional parcel definitions change with task; citation_author=M Salehi; citation_volume=208; citation_publication_date=2020; citation_pages=116366; citation_id=CR77"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Resting-state &#8220;physiological networks&#8221;; citation_author=JE Chen; citation_volume=213; citation_publication_date=2020; citation_pages=116707; citation_id=CR78"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Benchmarking of participant-level confound regression strategies for the control of motion artifact in studies of functional connectivity; citation_author=R Ciric; citation_volume=154; citation_publication_date=2017; citation_pages=174-187; citation_id=CR79"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Towards a consensus regarding global signal regression for resting state functional connectivity MRI; citation_author=K Murphy, MD Fox; citation_volume=154; citation_publication_date=2017; citation_pages=169-173; citation_id=CR80"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=On global fMRI signals and simulations; citation_author=JD Power, TO Laumann, M Plitt, A Martin, SE Petersen; citation_volume=21; citation_publication_date=2017; citation_pages=911-913; citation_id=CR81"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The nuisance of nuisance regression: spectral misspecification in a common approach to resting-state fMRI preprocessing reintroduces noise and obscures functional connectivity; citation_author=MN Hallquist, K Hwang, B Luna; citation_volume=82; citation_publication_date=2013; citation_pages=208-225; citation_id=CR82"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Correspondence of the brain&#8217;s functional architecture during activation and rest; citation_author=SM Smith; citation_volume=106; citation_publication_date=2009; citation_pages=13040-13045; citation_id=CR83"/>
    <meta name="citation_reference" content="citation_journal_title=PLOS Comput. Biol.; citation_title=The dynamic brain: from spiking neurons to neural masses and cortical fields; citation_author=G Deco, VK Jirsa, PA Robinson, M Breakspear, K Friston; citation_volume=4; citation_publication_date=2008; citation_pages=e1000092; citation_id=CR84"/>
    <meta name="citation_reference" content="citation_journal_title=Neural Comput.; citation_title=A framework for simulating and estimating the state and functional topology of complex dynamic geometric networks; citation_author=M Buibas, GA Silva; citation_volume=23; citation_publication_date=2011; citation_pages=183-214; citation_id=CR85"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Neurophysiological investigation of the basis of the fMRI signal; citation_author=NK Logothetis, J Pauls, M Augath, T Trinath, A Oeltermann; citation_volume=412; citation_publication_date=2001; citation_pages=150-157; citation_id=CR86"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Neuronal correlates of spontaneous fluctuations in fMRI signals in monkey visual cortex: Implications for functional connectivity at rest; citation_author=A Shmuel, DA Leopold; citation_volume=29; citation_publication_date=2008; citation_pages=751-761; citation_id=CR87"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Intracranial electrophysiology reveals reproducible intrinsic functional connectivity within human brain networks; citation_author=A Kucyi; citation_volume=38; citation_publication_date=2018; citation_pages=4230-4242; citation_id=CR88"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Magnetoencephalography for brain electrophysiology and imaging; citation_author=S Baillet; citation_volume=20; citation_publication_date=2017; citation_pages=327-339; citation_id=CR89"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Oxygen level and LFP in task-positive and task-negative areas: bridging BOLD fMRI and electrophysiology; citation_author=WJ Bentley, JM Li, AZ Snyder, ME Raichle, LH Snyder; citation_volume=26; citation_publication_date=2016; citation_pages=346-357; citation_id=CR90"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Neural basis of global resting-state fMRI activity; citation_author=ML Sch&#246;lvinck, A Maier, FQ Ye, JH Duyn, DA Leopold; citation_volume=107; citation_publication_date=2010; citation_pages=10238-10243; citation_id=CR91"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Discovering dynamic brain networks from big data in rest and task; citation_author=D Vidaurre; citation_volume=180; citation_publication_date=2018; citation_pages=646-656; citation_id=CR92"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Neurosci.; citation_title=Hierarchical organization and functional streams in the visual cortex; citation_author=DC Essen, JHR Maunsell; citation_volume=6; citation_publication_date=1983; citation_pages=370-375; citation_id=CR93"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Functional connectivity in the motor cortex of resting human brain using echo-planar MRI; citation_author=B Biswal, FZ Yetkin, VM Haughton, JS Hyde; citation_volume=34; citation_publication_date=1995; citation_pages=537-541; citation_id=CR94"/>
    <meta name="citation_reference" content="citation_journal_title=J. Cogn. Neurosci.; citation_title=Cognitive network neuroscience; citation_author=JD Medaglia, M-E Lynall, DS Bassett; citation_volume=27; citation_publication_date=2015; citation_pages=1471-1491; citation_id=CR95"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Dynamic effective connectivity in resting state fMRI; citation_author=H-J Park, KJ Friston, C Pae, B Park, A Razi; citation_volume=180; citation_publication_date=2018; citation_pages=594-608; citation_id=CR96"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=A Bayesian approach to determining connectivity of the human brain; citation_author=RS Patel, FD Bowman, JK Rilling; citation_volume=27; citation_publication_date=2006; citation_pages=267-276; citation_id=CR97"/>
    <meta name="citation_reference" content="citation_journal_title=J. Mach. Learn. Res.; citation_title=Pairwise likelihood ratios for estimation of non-Gaussian structural equation models; citation_author=A Hyv&#228;rinen, SM Smith; citation_volume=14; citation_publication_date=2013; citation_pages=111-152; citation_id=CR98"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Bayesian networks for fMRI: a primer; citation_author=JA Mumford, JD Ramsey; citation_volume=86; citation_publication_date=2014; citation_pages=573-582; citation_id=CR99"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Directed functional connectivity using dynamic graphical models; citation_author=S Schwab; citation_volume=175; citation_publication_date=2018; citation_pages=340-353; citation_id=CR100"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Spatial parcellations, spectral filtering, and connectivity measures in fMRI: Optimizing for discrimination; citation_author=R Sala-Llonch, SM Smith, M Woolrich, EP Duff; citation_volume=40; citation_publication_date=2019; citation_pages=407-419; citation_id=CR101"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=A decade of test-retest reliability of functional connectivity: a systematic review and meta-analysis; citation_author=S Noble, D Scheinost, RT Constable; citation_volume=203; citation_publication_date=2019; citation_pages=116157; citation_id=CR102"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Evaluation of denoising strategies to address motion-correlated artifacts in resting-state functional magnetic resonance imaging data from the Human Connectome Project; citation_author=GC Burgess; citation_volume=6; citation_publication_date=2016; citation_pages=669-680; citation_id=CR103"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The minimal preprocessing pipelines for the Human Connectome Project; citation_author=MF Glasser; citation_volume=80; citation_publication_date=2013; citation_pages=105-124; citation_id=CR104"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=fMRIPrep: a robust preprocessing pipeline for functional MRI; citation_author=O Esteban; citation_volume=16; citation_publication_date=2019; citation_pages=111-116; citation_id=CR105"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Statistical harmonization corrects site effects in functional connectivity measurements from multi-site fMRI data; citation_author=M Yu; citation_volume=39; citation_publication_date=2018; citation_pages=4213-4227; citation_id=CR106"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Sources and implications of whole-brain fMRI signals in humans; citation_author=JD Power, M Plitt, TO Laumann, A Martin; citation_volume=146; citation_publication_date=2017; citation_pages=609-625; citation_id=CR107"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Trouble at rest: how correlation patterns and group differences become distorted after global signal regression; citation_author=ZS Saad; citation_volume=2; citation_publication_date=2012; citation_pages=25-32; citation_id=CR108"/>
    <meta name="citation_reference" content="citation_journal_title=Magn. Reson. Med.; citation_title=Image-based method for retrospective correction of physiological motion effects in fMRI: RETROICOR; citation_author=GH Glover, TQ Li, D Ress; citation_volume=44; citation_publication_date=2000; citation_pages=162-167; citation_id=CR109"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data; citation_author=MF Glasser; citation_volume=181; citation_publication_date=2018; citation_pages=692-717; citation_id=CR110"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage Clin.; citation_title=Evaluating the evidence for biotypes of depression: methodological replication and extension of Drysdale et al. (2017); citation_author=R Dinga; citation_volume=22; citation_publication_date=2019; citation_pages=101796; citation_id=CR111"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Assessing and tuning brain decoders: cross-validation, caveats, and guidelines; citation_author=G Varoquaux; citation_volume=145; citation_publication_date=2017; citation_pages=166-179; citation_id=CR112"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Cluster failure: why fMRI inferences for spatial extent have inflated false-positive rates; citation_author=A Eklund, TE Nichols, H Knutsson; citation_volume=113; citation_publication_date=2016; citation_pages=7900-7905; citation_id=CR113"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Human brain mapping: a systematic comparison of parcellation methods for the human cerebral cortex; citation_author=S Arslan; citation_volume=170; citation_publication_date=2018; citation_pages=5-30; citation_id=CR114"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Scanning the horizon: towards transparent and reproducible neuroimaging research; citation_author=RA Poldrack; citation_volume=18; citation_publication_date=2017; citation_pages=115-126; citation_id=CR115"/>
    <meta name="citation_author" content="Bijsterbosch, Janine"/>
    <meta name="citation_author_institution" content="Mallinckrodt Institute of Radiology, Washington University in St Louis, Saint Louis, USA"/>
    <meta name="citation_author_institution" content="Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK"/>
    <meta name="citation_author" content="Harrison, Samuel J."/>
    <meta name="citation_author_institution" content="Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK"/>
    <meta name="citation_author_institution" content="Translational Neuromodeling Unit, University of Zurich &amp; ETH Zurich, Zurich, Switzerland"/>
    <meta name="citation_author" content="Jbabdi, Saad"/>
    <meta name="citation_author_institution" content="Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK"/>
    <meta name="citation_author" content="Woolrich, Mark"/>
    <meta name="citation_author_institution" content="Oxford Centre for Human Brain Activity (OHBA), Wellcome Centre for Integrative Neuroimaging, Department of Psychiatry, University of Oxford, Warneford Hospital, Oxford, UK"/>
    <meta name="citation_author" content="Beckmann, Christian"/>
    <meta name="citation_author_institution" content="Donders Institute and Department of Cognitive Neurosciences, Radboud University Medical Centre, Nijmegen, The Netherlands"/>
    <meta name="citation_author" content="Smith, Stephen"/>
    <meta name="citation_author_institution" content="Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK"/>
    <meta name="citation_author" content="Duff, Eugene P."/>
    <meta name="citation_author_institution" content="Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK"/>
    <meta name="citation_author_institution" content="Department of Paediatrics, University of Oxford, John Radcliffe Hospital, Oxford, UK"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="Challenges and future directions for representations of functional brain organization"/>
    <meta name="twitter:description" content="Nature Neuroscience - In this Primer article, Bijsterbosch and colleagues provide an accessible discussion of the challenges faced in analytical representations of functional brain organization and..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-020-00726-z"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="Challenges and future directions for representations of functional brain organization - Nature Neuroscience"/>
    <meta property="og:description" content="In this Primer article, Bijsterbosch and colleagues provide an accessible discussion of the challenges faced in analytical representations of functional brain organization and provide clear recommendations to unite a fractionated field."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-020-00726-z;doi=10.1038/s41593-020-00726-z;techmeta=36,59;subjmeta=1627,1647,245,378,53,631,692;kwrd=Biomarkers,Functional+magnetic+resonance+imaging,Neuroscience">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-21099287&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-020-00726-z%26doi%3D10.1038/s41593-020-00726-z%26techmeta%3D36,59%26subjmeta%3D1627,1647,245,378,53,631,692%26kwrd%3DBiomarkers,Functional+magnetic+resonance+imaging,Neuroscience">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=-21099287&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-020-00726-z%26doi%3D10.1038/s41593-020-00726-z%26techmeta%3D36,59%26subjmeta%3D1627,1647,245,378,53,631,692%26kwrd%3DBiomarkers,Functional+magnetic+resonance+imaging,Neuroscience"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-020-00726-z'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;review-article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:review articles"><span itemprop="name">review articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        Challenges and future directions for representations of functional brain organization
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00726-z.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Review Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2020-10-26">26 October 2020</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Challenges and future directions for representations of functional brain organization</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Janine-Bijsterbosch-Aff1-Aff2" data-author-popup="auth-Janine-Bijsterbosch-Aff1-Aff2" data-author-search="Bijsterbosch, Janine" data-corresp-id="c1">Janine Bijsterbosch<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-1385-9178"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1385-9178</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Samuel_J_-Harrison-Aff2-Aff3" data-author-popup="auth-Samuel_J_-Harrison-Aff2-Aff3" data-author-search="Harrison, Samuel J.">Samuel J. Harrison</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-5886-2389"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-5886-2389</a></span><sup class="u-js-hide"><a href="#Aff2">2</a>,<a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Saad-Jbabdi-Aff2" data-author-popup="auth-Saad-Jbabdi-Aff2" data-author-search="Jbabdi, Saad">Saad Jbabdi</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Mark-Woolrich-Aff4" data-author-popup="auth-Mark-Woolrich-Aff4" data-author-search="Woolrich, Mark">Mark Woolrich</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christian-Beckmann-Aff5" data-author-popup="auth-Christian-Beckmann-Aff5" data-author-search="Beckmann, Christian">Christian Beckmann</a><sup class="u-js-hide"><a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stephen-Smith-Aff2" data-author-popup="auth-Stephen-Smith-Aff2" data-author-search="Smith, Stephen">Stephen Smith</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-8166-069X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8166-069X</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 7 authors for this article" title="Show all 7 authors for this article"></li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Eugene_P_-Duff-Aff2-Aff6" data-author-popup="auth-Eugene_P_-Duff-Aff2-Aff6" data-author-search="Duff, Eugene P." data-corresp-id="c2">Eugene P. Duff<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-8795-5472"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8795-5472</a></span><sup class="u-js-hide"><a href="#Aff2">2</a>,<a href="#Aff6">6</a></sup></li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>23</b>,<span class="u-visually-hidden">pages </span>14841495 (<span data-test="article-publication-year">2020</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">11k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">65 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">51 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-020-00726-z/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/biomarkers" data-track="click" data-track-action="view subject" data-track-label="link">Biomarkers</a></li><li class="c-article-subject-list__subject"><a href="/subjects/functional-magnetic-resonance-imaging" data-track="click" data-track-action="view subject" data-track-label="link">Functional magnetic resonance imaging</a></li><li class="c-article-subject-list__subject"><a href="/subjects/neuroscience" data-track="click" data-track-action="view subject" data-track-label="link">Neuroscience</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>A key principle of brain organization is the functional integration of brain regions into interconnected networks. Functional MRI scans acquired at rest offer insights into functional integration via patterns of coherent fluctuations in spontaneous activity, known as functional connectivity. These patterns have been studied intensively and have been linked to cognition and disease. However, the field is fractionated. Diverging analysis approaches have segregated the community into research silos, limiting the replication and clinical translation of findings. A primary source of this fractionation is the diversity of approaches used to reduce complex brain data into a lower-dimensional set of features for analysis and interpretation, which we refer to as brain representations. In this Primer, we provide an overview of different brain representations, lay out the challenges that have led to the fractionation of the field and that continue to form obstacles for convergence, and propose concrete guidelines to unite the field.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00726-z.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00726-z.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-023-05107-3/MediaObjects/42003_2023_5107_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s42003-023-05107-3?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s42003-023-05107-3">Atlasing white matter and grey matter joint contributions to resting-state networks in the human brain
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">14 July 2023</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Victor Nozais, Stephanie J. Forkel,  Marc Joliot</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-019-56834-9/MediaObjects/41598_2019_56834_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41598-019-56834-9?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41598-019-56834-9">Distinct structure-function relationships across cortical regions and connectivity scales in the rat brain
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">09 January 2020</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Milou Straathof, Michel R. T. Sinke,  Rick M. Dijkhuizen</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41582-022-00753-3/MediaObjects/41582_2022_753_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41582-022-00753-3?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41582-022-00753-3">Functional brain networks in the evaluation of patients with neurodegenerative disorders
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">20 December 2022</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Matej Perovnik, Toma Rus,  David Eidelberg</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711582447,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>The fundamental brain organizing principles of segregation into functionally specialized areas and integration into highly connected macroscale brain networks form the basis of neuroscientific research into brain structure and function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Van Essen, D. C. &amp; Glasser, M. F. Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans. Neuron 99, 640663 (2018). This review of brain mapping (i.e., parcellating the brain into units) summarizes extensive research performed in non-human primates and discusses how these results inform cortical brain parcellation efforts in humans using noninvasive imaging techniques including rfMRI. This article provides both a historical and a future perspective on cortical brain parcellation, which is an important aspect of brain representations (related to our description defining the brain units in section Defining a brain unit)." href="/articles/s41593-020-00726-z#ref-CR1" id="ref-link-section-d30294863e541">1</a></sup>. Over recent decades, the ability to noninvasively measure inherent activity fluctuations from the whole brain using resting-state functional MRI (rfMRI) has offered many insights into this macroscale functional organization. In rfMRI, similarities (between different brain regions) in the spontaneous fluctuations of the blood oxygenation level dependent (BOLD) signalstermed functional connectivitycan be used to explore intrinsic functional relationships across brain areas and examine how these vary across healthy and disease states and through the population<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Bijsterbosch, J., Smith, S.M. &amp; Beckmann, C.F. Introduction to Resting State fMRI Functional Connectivity (Oxford Univ. Press, 2017)." href="/articles/s41593-020-00726-z#ref-CR2" id="ref-link-section-d30294863e545">2</a></sup>. However, the field of rfMRI is fractionated, with ongoing debates regarding preprocessing strategies and brain parcellations, as well as extensive divergence of post-processing analysis methods and endpoints<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Reid, A. T. et al. Advancing functional connectivity research from association to causation. Nat. Neurosci. 22, 17511760 (2019)." href="/articles/s41593-020-00726-z#ref-CR3" id="ref-link-section-d30294863e549">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Botvinik-Nezer, R. et al. Variability in the analysis of a single neuroimaging dataset by many teams. Nature 582, 8488 (2020)." href="/articles/s41593-020-00726-z#ref-CR4" id="ref-link-section-d30294863e552">4</a></sup>. These challenges greatly impact the ability of the field to achieve reproducible progress.</p><p>A key source of these issues is the challenge of brain representation. Like many modern scientific fields, rfMRI generates large amounts of data per participant, comprising ongoing activity measurements from tens of thousands of voxels over a period of up to an hour. A key task of analysis is to distill the enormous complexity of the measured brain activity into an accessible and interpretable form. We use the term brain representation here to refer to the combined methodological steps that are taken to derive a lower dimensional set of features from an individuals rfMRI data set for subsequent analysis and interpretation. Brain representations are multifaceted descriptions of the acquired rfMRI data that often encompass both a spatial definition of brain units (i.e., parcellation) and a summary measure that extracts interpretable features at the level of the brain units (for example, pairwise correlation between brain unit time series, discussed further in A primer on brain representations below). As the determinant of the building blocks for further analyses, the choice of how to represent brain data fundamentally underpins descriptions of brain connectivity and organization.</p><p>Representation of the brain is often considered a task of mapping, aiming to delineate boundaries of functionally and neuroanatomically distinct areas of the neural tissue<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Van Essen, D. C. &amp; Glasser, M. F. Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans. Neuron 99, 640663 (2018). This review of brain mapping (i.e., parcellating the brain into units) summarizes extensive research performed in non-human primates and discusses how these results inform cortical brain parcellation efforts in humans using noninvasive imaging techniques including rfMRI. This article provides both a historical and a future perspective on cortical brain parcellation, which is an important aspect of brain representations (related to our description defining the brain units in section Defining a brain unit)." href="/articles/s41593-020-00726-z#ref-CR1" id="ref-link-section-d30294863e562">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672686 (2018)." href="/articles/s41593-020-00726-z#ref-CR5" id="ref-link-section-d30294863e565">5</a></sup>. However, the task of brain representation is broader than mapping, encompassing a more comprehensive range of representational forms and addressing additional aspects of how data are transformed into these representations. The goal of this article is to provide a primer on the representational challenges of rfMRI, with the intention of improving consensus and reproducibility in the field.</p><p>The lack of consensus on brain representations results in part from a lack of knowledge of the true underlying functional organization of the brain. In the absence of this ground-truth, further divergence arises from differences in opinion about what we want to achieve. The organization of the brain is multifaceted, dynamic and hierarchical, leading to many complexities and trade-offs for the question of brain representation. On the one hand, brain representations should strive to accurately reflect neural units to achieve biological interpretability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ferrante, M. et al. Computational psychiatry: a report from the 2017 NIMH workshop on opportunities and challenges. Mol. Psychiatry 24, 479483 (2019)." href="/articles/s41593-020-00726-z#ref-CR6" id="ref-link-section-d30294863e572">6</a></sup>. On the other hand, the accuracy of clinically relevant predictions derived from representations may be prioritized<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Poldrack, R. A., Huckins, G. &amp; Varoquaux, G. Establishment of best practices for evidence for prediction: a review. JAMA Psychiatry 77, 534540 (2020)." href="/articles/s41593-020-00726-z#ref-CR7" id="ref-link-section-d30294863e576">7</a></sup>. A standardized brain representation that is generalizable across groups, ages and clinical populations would provide a common language for the communication of findings, facilitating replication and aggregation of results. However, standardization would come at the cost of optimization for the specific research question, study population, experimental protocol or individual participant. In this Primer, we aim to clarify these issues and provide guidelines for the choice of brain representation.</p><p>Approaches to representing rfMRI data are diverse, and we therefore start with a short primer on a number of distinct families of brain representations. Despite their differences, each family of representations faces similar fundamental challenges, ranging from subtle mathematical biases and confounds to the interpretability of their final outputs. The section Challenges for brain representations surveys and dissects these issues, which may be addressed in a variety of ways, contributing to the dispersion of brain representations. Finally, we put forward a set of explicit suggestions and guidelines to resolve the fractionation of the field and achieve a consensus that it is expected to improve reproducibility and collaboration (Recommendations and future directions for brain representations). As part of our recommendations, we advocate for the use of state-of-the-art preprocessing methods to avoid unnecessary discordance resulting from outdated practices and for improved transparency in reporting to inform and encourage debate about similarities and differences in brain representations. While we focus on rfMRI brain representations, many of the issues that we discuss are shared with other measurement domains such as diffusion MRI and local field potentials.</p></div></div></section><section data-title="A primer on brain representations"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">A primer on brain representations</h2><div class="c-article-section__content" id="Sec2-content"><p>Brain representations can take a variety of approaches to reduce the complexity of measured BOLD data into a set of features for analysis. A majority of brain representations identify (i) a low-dimensional set of brain units (for example, spatial parcels) and (ii) a summary measure that is applied at the level of brain units (for example, pairwise correlation between brain unit time series). Together, these reduce the rfMRI data down to the features used in subsequent statistical or prediction analyses. We use the term brain unit to indicate any spatially defined neural entity that can be treated as a basic functional processing unit. The concept of a brain unit therefore generalizes beyond the rfMRI domain, with strong parallels to for example, Hebbian cell assemblies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Lopes-dos-Santos, V., Ribeiro, S. &amp; Tort, A. B. L. Detecting cell assemblies in large neuronal populations. J. Neurosci. Methods 220, 149166 (2013)." href="/articles/s41593-020-00726-z#ref-CR8" id="ref-link-section-d30294863e592">8</a></sup>. We define the summary measure of a brain representation as a method for computing features, defined in relation to brain units, from the rfMRI data. The computed features vary freely across participants and/or conditions and are used to answer the research question. The summary measures of a brain representation are therefore relatively domain-specific and informed by the measurement type. A smaller number of brain representations do not use brain units and summary measures and instead estimate features that represent complex spatiotemporal patterns of activity (see section Complex spatiotemporal brain representations).</p><h3 class="c-article__sub-heading" id="Sec3">Defining a brain unit</h3><p>As a whole-brain noninvasive imaging modality, the spatial measurement resolution of rfMRI easily reaches 2  2  2 mm in modern scans. This results in about 100,000 voxels within gray matter across the adult human brain. In rfMRI, these voxels (or vertices when representing the cortical gray matter surface as a tessellated mesh<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Smith, S. M. et al. Resting-state fMRI in the Human Connectome Project. Neuroimage 80, 144168 (2013)." href="/articles/s41593-020-00726-z#ref-CR9" id="ref-link-section-d30294863e603">9</a></sup>) are the smallest measurable brain units. However, the voxel or vertex unit, while ideally chosen with reference to brain anatomy such as cortical thickness<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-020-00726-z#ref-CR10" id="ref-link-section-d30294863e607">10</a></sup>, does not represent any particular level of the neuroanatomical hierarchy. It is therefore common to group voxels or vertices together into a smaller set of brain units to achieve a meaningful lower-rank brain representation. The size of this set varies greatly in the literature, from fewer than ten large networks to hundreds of smaller brain units<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. Nat. Rev. Neurosci. 19, 672686 (2018)." href="/articles/s41593-020-00726-z#ref-CR5" id="ref-link-section-d30294863e611">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Eickhoff, S. B., Thirion, B., Varoquaux, G. &amp; Bzdok, D. Connectivity-based parcellation: critique and implications. Hum. Brain Mapp. 36, 47714792 (2015)." href="/articles/s41593-020-00726-z#ref-CR11" id="ref-link-section-d30294863e614">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Uddin, L. Q., Yeo, B. T. T. &amp; Spreng, R. N. Towards a universal taxonomy of macro-scale functional human brain networks. Brain Topogr. 32, 926942 (2019). This article discusses the challenges of scale and dimensionality of brain representations (related to our discussion of dimensionality in the section Heterogeneity and dimensionality of brain units). Focusing on the level of large-scale brain networks, the authors propose a consistent nomenclature for the naming of networks grounded in anatomy to address widespread problems with inconsistent terminology in the literature." href="/articles/s41593-020-00726-z#ref-CR12" id="ref-link-section-d30294863e617">12</a></sup>.</p><p>A brain unit may be spatially contiguous (i.e., adjacent voxels or vertices) or non-contiguous (i.e., made up of several spatially distinct regions spread throughout the brain). Contiguous brain units are consistent with the concept of functionally specialized cortical areas (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig1">1a</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Van Essen, D. C. &amp; Glasser, M. F. Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans. Neuron 99, 640663 (2018). This review of brain mapping (i.e., parcellating the brain into units) summarizes extensive research performed in non-human primates and discusses how these results inform cortical brain parcellation efforts in humans using noninvasive imaging techniques including rfMRI. This article provides both a historical and a future perspective on cortical brain parcellation, which is an important aspect of brain representations (related to our description defining the brain units in section Defining a brain unit)." href="/articles/s41593-020-00726-z#ref-CR1" id="ref-link-section-d30294863e627">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Eickhoff, S. B., Constable, R. T. &amp; Yeo, B. T. T. Topographic organization of the cerebral cortex and brain cartography. Neuroimage 170, 332347 (2018)." href="/articles/s41593-020-00726-z#ref-CR13" id="ref-link-section-d30294863e630">13</a></sup>. Conversely, non-contiguous brain units are able to capture complex network structure within the hierarchically organized and largely hemispherically symmetric brain (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig1">1b</a>; see Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec4">1</a> for a discussion on brain unit nomenclature). In addition, brain units may be binary, such that each voxel or vertex is assigned uniquely and fully to one unit (i.e., a hard parcellation), or weighted, where each voxel or vertex can contribute to multiple units as described by its weights (i.e., a soft parcellation).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Example brain representations."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Example brain representations.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="276"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Example binary parcellations adapted from Glasser<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-020-00726-z#ref-CR10" id="ref-link-section-d30294863e654">10</a></sup> (left; Nature Publishing Group) and Schaefer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Schaefer, A. et al. Local-global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 30953114 (2018)." href="/articles/s41593-020-00726-z#ref-CR18" id="ref-link-section-d30294863e658">18</a></sup> (right; Oxford Univ. Press). <b>b</b>, Example weighted resting-state networks obtained from a 50-dimensional independent component analysis (ICA) decomposition (publicly released Human Connectome Project maps estimated from 1,003 participants).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>A wide variety of approaches can be used to define brain units. Parcellations based on established atlases defined from histology, lesions, gyrification or other features are obvious choices<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tzourio-Mazoyer, N. et al. Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain. Neuroimage 15, 273289 (2002)." href="#ref-CR14" id="ref-link-section-d30294863e676">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. Neuroimage 31, 968980 (2006)." href="#ref-CR15" id="ref-link-section-d30294863e676_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Eickhoff, S. B. et al. A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. Neuroimage 25, 13251335 (2005)." href="/articles/s41593-020-00726-z#ref-CR16" id="ref-link-section-d30294863e679">16</a></sup>. However, these atlases are typically derived from a small number of participants, and the anatomically defined brain unit boundaries do not necessarily match the functional organization. A recent comparison between anatomically and functionally defined brain units in the contexts of rfMRI predictions reported consistent improvements in prediction accuracy when using functionally defined brain units<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Dadi, K. et al. Benchmarking functional connectome-based predictive models for resting-state fMRI. Neuroimage 192, 115134 (2019). This paper is a great example of the benefit of comparing different brain representations to determine their relative value in addressing a specific research question. The results offer clear-cut insights that are important for brain representations. For example, functionally defined brain units clearly outperform anatomically defined brain units, which has informed our guideline #1 in Box 3." href="/articles/s41593-020-00726-z#ref-CR17" id="ref-link-section-d30294863e683">17</a></sup>. A wide variety of methods have been developed that use rfMRI data to generate parcellations. These may provide brain units that are more targeted to the specific resolution and contrast of rfMRI, and can provide greater flexibility in the parameters of the parcellation.</p><p>Clustering and related approaches have produced a number of high-quality rfMRI-derived binary parcellations (that are often also contiguous)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schaefer, A. et al. Local-global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 30953114 (2018)." href="#ref-CR18" id="ref-link-section-d30294863e690">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shen, X., Tokoglu, F., Papademetris, X. &amp; Constable, R. T. Groupwise whole-brain parcellation from resting-state fMRI data for network node identification. Neuroimage 82, 403415 (2013)." href="#ref-CR19" id="ref-link-section-d30294863e690_1">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Craddock, R. C., James, G. A., Holtzheimer, P. E. III, Hu, X. P. &amp; Mayberg, H. S. A whole brain fMRI atlas generated via spatially constrained spectral clustering. Hum. Brain Mapp. 33, 19141928 (2012)." href="#ref-CR20" id="ref-link-section-d30294863e690_2">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. Cereb. Cortex 26, 288303 (2016)." href="#ref-CR21" id="ref-link-section-d30294863e690_3">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 11251165 (2011)." href="/articles/s41593-020-00726-z#ref-CR22" id="ref-link-section-d30294863e693">22</a></sup>. Algorithms that parcellate rfMRI data into weighted units (that are often also non-contiguous) include principal component analysis (PCA), independent component analysis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig1">1b</a>; spatial or temporal), non-negative matrix factorization, probabilistic functional modes and dictionary learning<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Beckmann, C. F., DeLuca, M., Devlin, J. T. &amp; Smith, S. M. Investigations into resting-state connectivity using independent component analysis. Phil. Trans. R. Soc. Lond. B 360, 10011013 (2005)." href="#ref-CR23" id="ref-link-section-d30294863e700">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Harrison, S. J. et al. Large-scale probabilistic functional modes from resting state fMRI. Neuroimage 109, 217231 (2015)." href="#ref-CR24" id="ref-link-section-d30294863e700_1">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Varoquaux, G., Gramfort, A., Pedregosa, F., Michel, V. &amp; Thirion, B. Multi-subject dictionary learning to segment an atlas of brain spontaneous activity. Inf. Process. Med. Imaging 22, 562573 (2011)." href="#ref-CR25" id="ref-link-section-d30294863e700_2">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Eavani, H. et al. Identifying sparse connectivity patterns in the brain using resting-state fMRI. Neuroimage 105, 286299 (2015)." href="/articles/s41593-020-00726-z#ref-CR26" id="ref-link-section-d30294863e703">26</a></sup>. A number of these methods, in particular independent component analysis, have been extensively employed in rfMRI studies.</p><p>Parcellations defined from rfMRI rely on the ability of the spontaneous BOLD fluctuations to resolve brain units, which possibly limits their wider applicability. For example, specific fMRI task activity may reveal further division of brain units not apparent during rest. Multimodal approaches that use combinations of structural, rest and task functional imaging may provide parcellations with improved generalizability across different modalities. However, these multimodal parcellations may generalize less well to new out-of-sample rfMRI data, compared with unimodal (for example, purely rfMRI) parcellations, reflecting the trade-off between optimal modality-specific fit versus accounting for cross-modal disagreement<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-020-00726-z#ref-CR10" id="ref-link-section-d30294863e711">10</a></sup>.</p><div class="c-article-box" data-expandable-box-container="true"><div class="c-article-box__container" data-expandable-box="true" aria-hidden="true" id="box-Sec4"><h3 class="c-article-box__container-title u-h3 js-expandable-title" id="Sec4">Box 1 Brain unit nomenclature</h3><div class="c-article-box__content"><p>In this article we define a brain unit as a basic functional processing entity. Within the rfMRI literature, brain units have been described in a number of different ways. We briefly clarify the semantic distinctions between some of these different conceptualizations of a brain unit:</p><ul class="u-list-style-bullet">
<li>
<p><b>Cortical area:</b> a cortical area benefits from having the clearest definition, being a region of cortex that is distinct from neighboring cortex in terms of its function, cytoarchitecture, connectivity or topographic organization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Van Essen, D. C. &amp; Maunsell, J. H. R. Hierarchical organization and functional streams in the visual cortex. Trends Neurosci. 6, 370375 (1983)." href="/articles/s41593-020-00726-z#ref-CR93" id="ref-link-section-d30294863e731">93</a></sup>.</p>
</li>
<li>
<p><b>Parcellation:</b> the verb parcellate refers to the process of subdividing the brain into a number of brain units. By extension, an individual brain unit may also be referred to as a parcel.</p>
</li>
<li>
<p><b>Region of interest (ROI):</b> in some sub-disciplines of rfMRI, a brain unit may also be described as a region of interest' or ROI. For example, one possible rfMRI brain representation is to define one brain unit (ROI) as the seed and estimate the whole-brain (voxel-wise) correlation map with the seed ROI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Biswal, B., Yetkin, F. Z., Haughton, V. M. &amp; Hyde, J. S. Functional connectivity in the motor cortex of resting human brain using echo-planar MRI. Magn. Reson. Med. 34, 537541 (1995)." href="/articles/s41593-020-00726-z#ref-CR94" id="ref-link-section-d30294863e751">94</a></sup>.</p>
</li>
<li>
<p><b>Edge:</b> one way to study interactions between brain units is to use the mathematical structure of a graph, where each individual brain unit is referred to as a node and pairwise connections between brain units are known as edges<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Smith, S. M. et al. Functional connectomics from resting-state fMRI. Trends Cogn. Sci. 17, 666682 (2013)." href="/articles/s41593-020-00726-z#ref-CR28" id="ref-link-section-d30294863e763">28</a></sup>. In practice for rfMRI, a node often (but not always) describes a contiguous binary brain unit that is part of a high-dimensional (50+) parcellation.</p>
</li>
<li>
<p><b>Network:</b> another common naming that fits with our intuitive understanding for a grouping of items is a network'. In rfMRI, network may refer to the brain as a whole (i.e., in network neuroscience)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Medaglia, J. D., Lynall, M.-E. &amp; Bassett, D. S. Cognitive network neuroscience. J. Cogn. Neurosci. 27, 14711491 (2015)." href="/articles/s41593-020-00726-z#ref-CR95" id="ref-link-section-d30294863e775">95</a></sup>, to a set of multiple brain units that are similar to each other (for example, defined using hierarchical clustering)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Smith, S. M. et al. Functional connectomics from resting-state fMRI. Trends Cogn. Sci. 17, 666682 (2013)." href="/articles/s41593-020-00726-z#ref-CR28" id="ref-link-section-d30294863e779">28</a></sup> or to a single non-contiguous weighted brain unit (popularly known as resting state networks, or RSNs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Smith, S. M. et al. Correspondence of the brains functional architecture during activation and rest. Proc. Natl. Acad. Sci. USA 106, 1304013045 (2009)." href="/articles/s41593-020-00726-z#ref-CR83" id="ref-link-section-d30294863e783">83</a></sup>. Overall, the term network tends to refer to the highest organizational level of the brain into a relatively small number of macroscale functional patterns (&lt;25).</p>
</li>
</ul></div></div></div><h3 class="c-article__sub-heading" id="Sec5">Defining the summary measures</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec6">Functional connectivity summary measures</h4><p>The most common type of information that is of interest in rfMRI is functional connectivity, which is defined as statistical similarities between signals measured from different brain regions and is thought to be indicative of functional integration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Friston, K. J. Functional and effective connectivity in neuroimaging: a synthesis. Hum. Brain Mapp. 2, 5678 (1994)." href="/articles/s41593-020-00726-z#ref-CR27" id="ref-link-section-d30294863e801">27</a></sup>. For example, graph-based connectomics methods capture functional connectivity information by conceptualizing each individual brain unit as a node within a graph (for reviews see refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Smith, S. M. et al. Functional connectomics from resting-state fMRI. Trends Cogn. Sci. 17, 666682 (2013)." href="/articles/s41593-020-00726-z#ref-CR28" id="ref-link-section-d30294863e805">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Betzel, R. F. &amp; Bassett, D. S. Multi-scale brain networks. Neuroimage 160, 7383 (2017)." href="/articles/s41593-020-00726-z#ref-CR29" id="ref-link-section-d30294863e808">29</a></sup>). Pairwise functional connectivity between nodes is commonly estimated as the correlation coefficient between summary time series from different brain units, but a variety of other summary measures can be used (Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec9">2</a>). Connectomics refers to the study of all possible pairwise node-to-node functional connections (edges), which can be summarized in matrix form (network matrix; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig2">2a</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 16641671 (2015)." href="/articles/s41593-020-00726-z#ref-CR30" id="ref-link-section-d30294863e818">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Smith, S. M. et al. A positive-negative mode of population covariation links brain connectivity, demographics and behavior. Nat. Neurosci. 18, 15651567 (2015)." href="/articles/s41593-020-00726-z#ref-CR31" id="ref-link-section-d30294863e821">31</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Different functional-connectivity-based versions of summary measures in different brain representations."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Different functional-connectivity-based versions of summary measures in different brain representations.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="366"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Edges between nodes are shown in graphical form on the left (positive and negative correlations in red and blue, respectively; line thickness indicates correlation strength) and in matrix form on the right. <b>b</b>, Example of different dynamic brain states and their fluctuations over time derived using a hidden Markov model (figure adapted from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Vidaurre, D. et al. Discovering dynamic brain networks from big data in rest and task. Neuroimage 180, 646656 (2018). Pt B." href="/articles/s41593-020-00726-z#ref-CR92" id="ref-link-section-d30294863e843">92</a></sup>, Elsevier). <b>c</b>, Directed edges may have different reciprocal connections strengths, shown as separate arrows in the graphical form, and resulting in a non-symmetrical network matrix. <b>d</b>, Principal global connectivity gradient from primary sensorimotor regions (blue) to multimodal association cortex (red) (figure adapted from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 1257412579 (2016)." href="/articles/s41593-020-00726-z#ref-CR45" id="ref-link-section-d30294863e853">45</a></sup>, US National Academy of Sciences).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Alternatively, in representations that use weighted brain units, functional connectivity information may be captured in the spatial voxel or vertex weights, which can be used as features that vary freely across participants (for example, using dual regression as the summary measure to estimate participant-specific maps<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Nickerson, L. D., Smith, S. M., ngr, D. &amp; Beckmann, C. F. Using dual regression to investigate network shape and amplitude in functional connectivity analyses. Front. Neurosci. 11, 115 (2017)." href="/articles/s41593-020-00726-z#ref-CR32" id="ref-link-section-d30294863e868">32</a></sup>). Depending on the dimensionality and definition of the brain units, functional connectivity can therefore be represented either temporally (i.e., in terms of the temporal correlation between a pair of separate brain units) or spatially (i.e., by combining multiple brain regions into the same non-contiguous brain unit and using the voxel-wise spatial weights to capture the relative strength of connectivity). This ambiguity leads to challenges in the interpretation of brain representations as discussed further in the Representational ambiguity section.</p><p>While the majority of summary measures consider static functional connectivity (i.e., averaged across the full rfMRI scan), a number of dynamic brain representations explicitly aim to capture time-varying functional connectivity in the summary measures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Hutchison, R. M. et al. Dynamic functional connectivity: promise, issues, and interpretations. Neuroimage 80, 360378 (2013)." href="/articles/s41593-020-00726-z#ref-CR33" id="ref-link-section-d30294863e875">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Preti, M. G., Bolton, T. A. &amp; Van De Ville, D. The dynamic functional connectome: state-of-the-art and perspectives. Neuroimage 160, 4154 (2017)." href="/articles/s41593-020-00726-z#ref-CR34" id="ref-link-section-d30294863e878">34</a></sup>. While static functional connectivity estimates are informed by time-varying signal fluctuations, the key difference is that only one estimate of connectivity is obtained across the entire scan, whereas dynamic connectivity methods calculate multiple estimates of connectivity separately for different periods of time over the course of the scan. Many dynamic functional connectivity methods aim to identify distinct dynamic states, for example by performing clustering on sliding window estimates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Allen, E. A. et al. Tracking whole-brain connectivity dynamics in the resting state. Cereb. Cortex 24, 663676 (2014)." href="/articles/s41593-020-00726-z#ref-CR35" id="ref-link-section-d30294863e882">35</a></sup> or using a hidden Markov model (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig2">2b</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Vidaurre, D. et al. Spontaneous cortical activity transiently organises into frequency specific phase-coupling networks. Nat. Commun. 9, 2987 (2018)." href="/articles/s41593-020-00726-z#ref-CR36" id="ref-link-section-d30294863e889">36</a></sup>. These are then used to generate the features for further analysis.</p><p>A further variant of functional connectivity summary measures is to aim for causal inference by estimating the directed connection from one brain unit to another (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig2">2c</a>; also referred to as effective connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Friston, K. J. Functional and effective connectivity in neuroimaging: a synthesis. Hum. Brain Mapp. 2, 5678 (1994)." href="/articles/s41593-020-00726-z#ref-CR27" id="ref-link-section-d30294863e899">27</a></sup>). Indeed, a recent article by Reid and colleagues proposed causal neural interactions as a unifying conceptualization of functional connectivity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Reid, A. T. et al. Advancing functional connectivity research from association to causation. Nat. Neurosci. 22, 17511760 (2019)." href="/articles/s41593-020-00726-z#ref-CR3" id="ref-link-section-d30294863e903">3</a></sup>. While the conceptual considerations and ambitions of this aspirational framework are of great importance, their suggestions may appear somewhat abstract to investigators grappling with their own rfMRI data, in part because some of the key analytic decisions that researchers need to make are not fully addressed.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec7">Univariate (node-based) summary measures</h4><p>While the summary measures of most rfMRI brain representations assess functional connectivity (i.e., integration) in some form, several alternative summary measures describe aspects of the data at each brain unit. Examples include the localized signal amplitude (i.e., BOLD signal strength)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Zou, Q.-H. et al. An improved approach to detection of amplitude of low-frequency fluctuation (ALFF) for resting-state fMRI: fractional ALFF. J. Neurosci. Methods 172, 137141 (2008)." href="#ref-CR37" id="ref-link-section-d30294863e915">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. Nat. Neurosci. 19, 15231536 (2016)." href="#ref-CR38" id="ref-link-section-d30294863e915_1">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bijsterbosch, J. et al. Investigations into within- and between-subject resting-state amplitude variations. Neuroimage 159, 5769 (2017)." href="/articles/s41593-020-00726-z#ref-CR39" id="ref-link-section-d30294863e918">39</a></sup>, the size of brain units or the spatial overlap of weighted brain units<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Bijsterbosch, J. D., Beckmann, C. F., Woolrich, M. W., Smith, S. M. &amp; Harrison, S. J. The relationship between spatial configuration and functional connectivity of brain regions revisited. eLife 8, e44890 (2019)." href="/articles/s41593-020-00726-z#ref-CR40" id="ref-link-section-d30294863e922">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Kong, R. et al. Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion. Cereb. Cortex 29, 25332551 (2019)." href="/articles/s41593-020-00726-z#ref-CR41" id="ref-link-section-d30294863e925">41</a></sup>. Despite their different nature, these univariate measures are often not independent from functional connectivity. For example, changes in signal amplitude may directly influence functional connectivity (see Representational ambiguity below)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Duff, E. P., Makin, T., Cottaar, M., Smith, S. M. &amp; Woolrich, M. W. Disambiguating brain functional connectivity. Neuroimage 173, 540550 (2018)." href="/articles/s41593-020-00726-z#ref-CR42" id="ref-link-section-d30294863e929">42</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec8">Complex spatiotemporal brain representations</h4><p>While all approaches discussed so far start with the definition of a brain unit, some brain representations avoid this step and estimate complex spatiotemporal patterns from the full data. For example, rfMRI data can be represented as one (or more) connectivity gradient(s) that capture variation in functional connectivity along a continuous axis of spatial location<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Huntenburg, J. M., Bazin, P.-L. &amp; Margulies, D. S. Large-scale gradients in human cortical organization. Trends Cogn. Sci. 22, 2131 (2018)." href="/articles/s41593-020-00726-z#ref-CR43" id="ref-link-section-d30294863e941">43</a></sup>. This approach can be used to identify overlapping modes of organization within a predefined brain unit<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. Nat. Hum. Behav. 1, 0146 (2017)." href="/articles/s41593-020-00726-z#ref-CR44" id="ref-link-section-d30294863e945">44</a></sup> or to map the principal global (whole-brain) pattern of cortical organization from primary sensorimotor to multimodal association cortex (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig2">2d</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 1257412579 (2016)." href="/articles/s41593-020-00726-z#ref-CR45" id="ref-link-section-d30294863e952">45</a></sup>. As a further example of complex brain representations, some dynamic methods directly take voxel or vertex rfMRI data as input to estimate a movie-like brain representation of spatially and temporally varying patterns, known as propagating waves<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Majeed, W. et al. Spatiotemporal dynamics of low frequency BOLD fluctuations in rats and humans. Neuroimage 54, 11401150 (2011)." href="/articles/s41593-020-00726-z#ref-CR46" id="ref-link-section-d30294863e956">46</a></sup>.</p><div class="c-article-box" data-expandable-box-container="true"><div class="c-article-box__container" data-expandable-box="true" aria-hidden="true" id="box-Sec9"><h3 class="c-article-box__container-title u-h3 js-expandable-title" id="Sec9">Box 2 Example summary measures</h3><div class="c-article-box__content"><p>A variety of summary measures can be used to calculate features of interest from the preprocessed rfMRI data that has been parcellated into brain units. Some example summary measures that have been used in the literature are summarized here.</p><ul class="u-list-style-bullet">
<li>
<p>Edge-based measures describe functional connectivity between pairs of brain units, which can be combined to form network matrices. Edge-based summary measures include full or partial correlation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Smith, S. M. et al. Functional connectomics from resting-state fMRI. Trends Cogn. Sci. 17, 666682 (2013)." href="/articles/s41593-020-00726-z#ref-CR28" id="ref-link-section-d30294863e976">28</a></sup> and mutual information.</p>
</li>
<li>
<p>Directed (effective) connectivity measures are edge-based measures that aim to estimate the directionality as well as the strength of connectivity between pairs of brain units. Example measures include dynamic causal models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Park, H.-J., Friston, K. J., Pae, C., Park, B. &amp; Razi, A. Dynamic effective connectivity in resting state fMRI. Neuroimage 180, 594608 (2018). Pt B." href="/articles/s41593-020-00726-z#ref-CR96" id="ref-link-section-d30294863e986">96</a></sup>, Patels conditional dependence measures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Patel, R. S., Bowman, F. D. &amp; Rilling, J. K. A Bayesian approach to determining connectivity of the human brain. Hum. Brain Mapp. 27, 267276 (2006)." href="/articles/s41593-020-00726-z#ref-CR97" id="ref-link-section-d30294863e990">97</a></sup>, pairwise likelihood ratios (LiNGAM)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Hyvrinen, A. &amp; Smith, S. M. Pairwise likelihood ratios for estimation of non-Gaussian structural equation models. J. Mach. Learn. Res. 14, 111152 (2013)." href="/articles/s41593-020-00726-z#ref-CR98" id="ref-link-section-d30294863e994">98</a></sup>, Bayes nets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Mumford, J. A. &amp; Ramsey, J. D. Bayesian networks for fMRI: a primer. Neuroimage 86, 573582 (2014)." href="/articles/s41593-020-00726-z#ref-CR99" id="ref-link-section-d30294863e998">99</a></sup> and dynamic graphical models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Schwab, S. et al. Directed functional connectivity using dynamic graphical models. Neuroimage 175, 340353 (2018)." href="/articles/s41593-020-00726-z#ref-CR100" id="ref-link-section-d30294863e1002">100</a></sup>.</p>
</li>
<li>
<p>Dynamic functional connectivity measures reflect variations in functional connectivity over time (typically without attempting to estimate causality). One option for investigating dynamic functional connectivity is to estimate edge-based measures listed above from short, sliding time windows (instead of from the entire scan) and cluster the resulting network matrices into brain states<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Allen, E. A. et al. Tracking whole-brain connectivity dynamics in the resting state. Cereb. Cortex 24, 663676 (2014)." href="/articles/s41593-020-00726-z#ref-CR35" id="ref-link-section-d30294863e1012">35</a></sup>. To overcome challenges with window-based dynamic analyses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Hutchison, R. M. et al. Dynamic functional connectivity: promise, issues, and interpretations. Neuroimage 80, 360378 (2013)." href="/articles/s41593-020-00726-z#ref-CR33" id="ref-link-section-d30294863e1016">33</a></sup>, statistical approaches such as the hidden Markov model offer alternative summary measures for estimating dynamic features<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Vidaurre, D. et al. Discovering dynamic brain networks from big data in rest and task. Neuroimage 180, 646656 (2018). Pt B." href="/articles/s41593-020-00726-z#ref-CR92" id="ref-link-section-d30294863e1020">92</a></sup>.</p>
</li>
<li>
<p>Univariate (node-based) measures, such as the amplitude of the BOLD signal, can be estimated at the brain unit level or at the voxel level, based on the time series standard deviation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bijsterbosch, J. et al. Investigations into within- and between-subject resting-state amplitude variations. Neuroimage 159, 5769 (2017)." href="/articles/s41593-020-00726-z#ref-CR39" id="ref-link-section-d30294863e1030">39</a></sup> or the relative strength of low-frequency power<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Zou, Q.-H. et al. An improved approach to detection of amplitude of low-frequency fluctuation (ALFF) for resting-state fMRI: fractional ALFF. J. Neurosci. Methods 172, 137141 (2008)." href="/articles/s41593-020-00726-z#ref-CR37" id="ref-link-section-d30294863e1034">37</a></sup>.</p>
</li>
<li>
<p>Spatial maps describe the topographical organization of brain units. Inter-individual variability in spatial properties can also be used as features, for example using summary measures that estimate participant-specific weighted resting-state networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Nickerson, L. D., Smith, S. M., ngr, D. &amp; Beckmann, C. F. Using dual regression to investigate network shape and amplitude in functional connectivity analyses. Front. Neurosci. 11, 115 (2017)." href="/articles/s41593-020-00726-z#ref-CR32" id="ref-link-section-d30294863e1044">32</a></sup>, or participant-specific brain unit boundaries in binary parcellations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Hacker, C. D. et al. Resting state network estimation in individual subjects. Neuroimage 82, 616633 (2013)." href="/articles/s41593-020-00726-z#ref-CR69" id="ref-link-section-d30294863e1048">69</a></sup>.</p>
</li>
</ul></div></div></div></div></div></section><section data-title="Challenges for brain representations"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Challenges for brain representations</h2><div class="c-article-section__content" id="Sec10-content"><p>The current divergence of brain representations (i.e., combined descriptions of brain units and summary measures) of rfMRI data is natural and expected as part of an initial exploration phase that has been echoed in other disciplines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Woo, C.-W., Chang, L. J., Lindquist, M. A. &amp; Wager, T. D. Building better biomarkers: brain models in translational neuroimaging. Nat. Neurosci. 20, 365377 (2017)." href="/articles/s41593-020-00726-z#ref-CR47" id="ref-link-section-d30294863e1063">47</a></sup>. However, now that the field is maturing to include biomarker discovery, efforts toward convergence to validated representations are needed to build a cumulative scientific framework.</p><p>Such efforts are complicated by the absence of a gold standard metric for the validation and comparison of brain representations. The underlying neural organization in an individual is not fully accessible by noninvasive technologies (and such detailed circuitry data would provide major representational challenges of its own<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Bargmann, C. I. &amp; Marder, E. From the connectome to brain function. Nat. Methods 10, 483490 (2013)." href="/articles/s41593-020-00726-z#ref-CR48" id="ref-link-section-d30294863e1070">48</a></sup>). Representations derived from rfMRI may therefore correspond to different features of this underlying neural organization. As such, comparisons across different representations necessarily rely on indirect metrics such as behavioral prediction accuracy, genetic heritability, within-parcel homogeneity, variance explained, testretest reliability, comparisons against other data modalities such as structural imaging and histology, and simulations. In this section we lay out a number of challenges for brain representations that come about as a result of this lack of ground truth knowledge, with the goal of promoting awareness of the issues that are in some cases rarely explicitly considered or taught.</p><h3 class="c-article__sub-heading" id="Sec11">Heterogeneity and dimensionality of brain units</h3><p>A common assumption inherent to the majority of brain representations is that an individual brain unit is functionally homogeneous, such that its relevant activity can be accurately reflected in a single summary time series. However, functional heterogeneity within a brain unit can be produced by measurement noise, structured artifacts, between-participant variability and true heterogeneity at the level of neural processing. In addition to heterogeneity, functional multiplicity occurs when the same part of cortex encodes different types of information. Examples of this multiplicity can be seen in visual cortex, which simultaneously encodes a retinotopic map and stimulus orientation, or in parietal cortex, where different somatotopic maps converge and overlap<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Jbabdi, S., Sotiropoulos, S. N. &amp; Behrens, T. E. The topographic connectome. Curr. Opin. Neurobiol. 23, 207215 (2013)." href="/articles/s41593-020-00726-z#ref-CR49" id="ref-link-section-d30294863e1081">49</a></sup>. The inevitable presence of some neuronal functional heterogeneity and/or multiplicity within brain units is typically acknowledged or even assumed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Van Essen, D. C. &amp; Glasser, M. F. Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans. Neuron 99, 640663 (2018). This review of brain mapping (i.e., parcellating the brain into units) summarizes extensive research performed in non-human primates and discusses how these results inform cortical brain parcellation efforts in humans using noninvasive imaging techniques including rfMRI. This article provides both a historical and a future perspective on cortical brain parcellation, which is an important aspect of brain representations (related to our description defining the brain units in section Defining a brain unit)." href="/articles/s41593-020-00726-z#ref-CR1" id="ref-link-section-d30294863e1085">1</a></sup>, yet the implications for brain representations are rarely considered or accounted for.</p><p>One way to potentially reduce problems with functional heterogeneity and multiplicity is to split parcels up into ever-smaller brain units, to achieve a more fine-grained brain representation. However, an excessively fine parcellation, with multiple brain units representing the same functional entities, may lead to complexities with modelling and interpretation. For example, if a functional region is inappropriately divided into multiple brain units, this can lead to incorrect estimates of functional connectivity when using partial correlation and can detrimentally affect causal connectivity models. Determining the optimal number of brain units in a brain representation that balances the trade-offs between homogeneity and model complexity is challenging.</p><p>No consensus has been reached on the question of the optimal dimensionality of brain representations, with recent suggestions varying from six macroscale systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Uddin, L. Q., Yeo, B. T. T. &amp; Spreng, R. N. Towards a universal taxonomy of macro-scale functional human brain networks. Brain Topogr. 32, 926942 (2019). This article discusses the challenges of scale and dimensionality of brain representations (related to our discussion of dimensionality in the section Heterogeneity and dimensionality of brain units). Focusing on the level of large-scale brain networks, the authors propose a consistent nomenclature for the naming of networks grounded in anatomy to address widespread problems with inconsistent terminology in the literature." href="/articles/s41593-020-00726-z#ref-CR12" id="ref-link-section-d30294863e1095">12</a></sup> to several hundred parcels<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Eickhoff, S. B., Constable, R. T. &amp; Yeo, B. T. T. Topographic organization of the cerebral cortex and brain cartography. Neuroimage 170, 332347 (2018)." href="/articles/s41593-020-00726-z#ref-CR13" id="ref-link-section-d30294863e1099">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Dadi, K. et al. Benchmarking functional connectome-based predictive models for resting-state fMRI. Neuroimage 192, 115134 (2019). This paper is a great example of the benefit of comparing different brain representations to determine their relative value in addressing a specific research question. The results offer clear-cut insights that are important for brain representations. For example, functionally defined brain units clearly outperform anatomically defined brain units, which has informed our guideline #1 in Box 3." href="/articles/s41593-020-00726-z#ref-CR17" id="ref-link-section-d30294863e1102">17</a></sup>. This wide range is in part due to the hierarchical organization of the brain, such that it can be meaningfully represented at multiple different levels of granularity, depending on the research question<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Paquola, C. et al. The cortical wiring scheme of hierarchical information processing. Preprint at bioRxiv &#xA;https://doi.org/10.1101/2020.01.08.899583&#xA;&#xA; (2020)." href="/articles/s41593-020-00726-z#ref-CR50" id="ref-link-section-d30294863e1106">50</a></sup>. For example, topological features of functional brain organization can be studied at different dimensionalities<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Betzel, R. F. &amp; Bassett, D. S. Multi-scale brain networks. Neuroimage 160, 7383 (2017)." href="/articles/s41593-020-00726-z#ref-CR29" id="ref-link-section-d30294863e1110">29</a></sup>, and different patterns of within- and between-participant variability may dominate at different scales<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Betzel, R. F. et al. The community structure of functional brain networks exhibits scale-specific patterns of inter- and intra-subject variability. Neuroimage 202, 115990 (2019)." href="/articles/s41593-020-00726-z#ref-CR51" id="ref-link-section-d30294863e1114">51</a></sup>. However, increased dimensionality estimates can also occur due to inadequate handling of between-participant variability, resulting in a misleadingly detailed granularity (see section Dealing with variability). Notably, the effective dimensionality of rfMRI data is subject to biological limits imposed by dependencies of the hemodynamically mediated BOLD signal on the architecture and latency of the brains microvasculature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Logothetis, N. K. What we can do and what we cannot do with fMRI. Nature 453, 869878 (2008)." href="/articles/s41593-020-00726-z#ref-CR52" id="ref-link-section-d30294863e1119">52</a></sup>.</p><p>These challenges of heterogeneity point to a disconnect that exists between the best model of the human brain and the best model of the rfMRI measures that are acquired. There is ample evidence in humans and other species for the presence of functionally specialized neural populations organized into neuroanatomically distinct cortical areas. Following this evidence, a binary parcellation into contiguous brain units may therefore be the best macroscale model of the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Van Essen, D. C. &amp; Glasser, M. F. Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans. Neuron 99, 640663 (2018). This review of brain mapping (i.e., parcellating the brain into units) summarizes extensive research performed in non-human primates and discusses how these results inform cortical brain parcellation efforts in humans using noninvasive imaging techniques including rfMRI. This article provides both a historical and a future perspective on cortical brain parcellation, which is an important aspect of brain representations (related to our description defining the brain units in section Defining a brain unit)." href="/articles/s41593-020-00726-z#ref-CR1" id="ref-link-section-d30294863e1126">1</a></sup>. However, despite rapid advances in recent years with the help of accelerated acquisition methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Moeller, S. et al. Multiband multislice GE-EPI at 7 tesla, with 16-fold acceleration using partial parallel imaging with application to high spatial and temporal whole-brain fMRI. Magn. Reson. Med. 63, 11441153 (2010)." href="/articles/s41593-020-00726-z#ref-CR53" id="ref-link-section-d30294863e1130">53</a></sup>, the spatial and temporal resolution of fMRI are many orders of magnitude removed from the scale of neural populations and action potentials. Similarly, the physiology of the hemodynamic response implies limits on resolution, independent of advances in MR image acquisition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Logothetis, N. K. What we can do and what we cannot do with fMRI. Nature 453, 869878 (2008)." href="/articles/s41593-020-00726-z#ref-CR52" id="ref-link-section-d30294863e1134">52</a></sup>. Therefore, data obtained with rfMRI provide a crude measurement that collapses information both spatially and temporally. As a result, weighted parcellations that allow for overlapping organization and fuzzy boundaries may provide a better model of the data measured from rfMRI. Indeed, previous comparisons have suggested that weighted brain representations may perform better in terms of predicting behavioral traits than binary parcellations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Dadi, K. et al. Benchmarking functional connectome-based predictive models for resting-state fMRI. Neuroimage 192, 115134 (2019). This paper is a great example of the benefit of comparing different brain representations to determine their relative value in addressing a specific research question. The results offer clear-cut insights that are important for brain representations. For example, functionally defined brain units clearly outperform anatomically defined brain units, which has informed our guideline #1 in Box 3." href="/articles/s41593-020-00726-z#ref-CR17" id="ref-link-section-d30294863e1138">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yu, Q. et al. Comparing brain graphs in which nodes are regions of interest or independent components: A simulation study. J. Neurosci. Methods 291, 6168 (2017)." href="#ref-CR54" id="ref-link-section-d30294863e1141">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Abraham, A. et al. Deriving reproducible biomarkers from multi-site resting-state data: An Autism-based example. Neuroimage 147, 736745 (2017)." href="#ref-CR55" id="ref-link-section-d30294863e1141_1">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Duff, E. P. et al. Task-driven ICA feature generation for accurate and interpretable prediction using fMRI. Neuroimage 60, 189203 (2012)." href="#ref-CR56" id="ref-link-section-d30294863e1141_2">56</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Pervaiz, U., Vidaurre, D., Woolrich, M. W. &amp; Smith, S. M. Optimising network modelling methods for fMRI. NeuroImage 211, 116604 (2020)." href="/articles/s41593-020-00726-z#ref-CR57" id="ref-link-section-d30294863e1144">57</a></sup>. However, care is required when interpreting such weighted parcellations. For example, spatial overlap between brain units may be an important summary measure to take into consideration (see section Representational ambiguity)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Bijsterbosch, J. D., Beckmann, C. F., Woolrich, M. W., Smith, S. M. &amp; Harrison, S. J. The relationship between spatial configuration and functional connectivity of brain regions revisited. eLife 8, e44890 (2019)." href="/articles/s41593-020-00726-z#ref-CR40" id="ref-link-section-d30294863e1148">40</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec12">Dealing with variability</h3><p>Brain representations are often defined based on a large number of participants to achieve correspondence across individuals for group comparisons and to overcome the limited signal-to-noise ratio in single-participant data. However, variability across individuals in measured functional brain organization may result from spatial misalignment across individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Smith, S. M. et al. Network modelling methods for FMRI. Neuroimage 54, 875891 (2011)." href="/articles/s41593-020-00726-z#ref-CR58" id="ref-link-section-d30294863e1160">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Allen, E. A., Erhardt, E. B., Wei, Y., Eichele, T. &amp; Calhoun, V. D. Capturing inter-subject variability with group independent component analysis of fMRI data: a simulation study. Neuroimage 59, 41414159 (2012)." href="/articles/s41593-020-00726-z#ref-CR59" id="ref-link-section-d30294863e1163">59</a></sup> or true individual differences in brain structure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Llera, A., Wolfers, T., Mulders, P. &amp; Beckmann, C. F. Inter-individual differences in human brain structure and morphology link to variation in demographics and behavior. eLife 8, e44443 (2019)." href="/articles/s41593-020-00726-z#ref-CR60" id="ref-link-section-d30294863e1167">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Smith, S. et al. Structural variability in the human brain reflects fine-grained functional architecture at the population level. J. Neurosci. 39, 61366149 (2019)." href="/articles/s41593-020-00726-z#ref-CR61" id="ref-link-section-d30294863e1170">61</a></sup> and/or function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Bijsterbosch, J. D. et al. The relationship between spatial configuration and functional connectivity of brain regions. eLife 7, e32992 (2018). This article uncovers an important source of representational ambiguity by showing that between-participant variance in node-to-node correlations is partly driven by variability in spatial organization. Data-driven simulations are used to interrogate interactions between different elements of brain representations, and shared versus unique variance is used to compare different brain representations." href="/articles/s41593-020-00726-z#ref-CR62" id="ref-link-section-d30294863e1174">62</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Tavor, I. et al. Task-free MRI predicts individual differences in brain activity during task performance. Science 352, 216220 (2016)." href="/articles/s41593-020-00726-z#ref-CR63" id="ref-link-section-d30294863e1177">63</a></sup>. Despite advances in surface-based alignment methods designed to address this variability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Coalson, T. S., Van Essen, D. C. &amp; Glasser, M. F. The impact of traditional neuroimaging methods on the spatial localization of cortical areas. Proc. Natl. Acad. Sci. USA 115, E6356E6365 (2018)." href="/articles/s41593-020-00726-z#ref-CR64" id="ref-link-section-d30294863e1181">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Robinson, E. C. et al. MSM: a new flexible framework for multimodal surface matching. Neuroimage 100, 414426 (2014)." href="/articles/s41593-020-00726-z#ref-CR65" id="ref-link-section-d30294863e1184">65</a></sup>, recent studies of extensively scanned individuals point to detailed individualized features of organization that are misrepresented in group-derived brain representations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Braga, R. M. &amp; Buckner, R. L. Parallel interdigitated distributed networks within the individual estimated by intrinsic functional connectivity. Neuron 95, 457471.e5 (2017)." href="#ref-CR66" id="ref-link-section-d30294863e1188">66</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Poldrack, R. A. et al. Long-term neural and physiological phenotyping of a single human. Nat. Commun. 6, 8885 (2015)." href="#ref-CR67" id="ref-link-section-d30294863e1188_1">67</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Seitzman, B. A. et al. Trait-like variants in human functional brain networks. Proc. Natl. Acad. Sci. USA 116, 2285122861 (2019)." href="/articles/s41593-020-00726-z#ref-CR68" id="ref-link-section-d30294863e1191">68</a></sup>.</p><p>A number of recent approaches aim to address these issues of between-participant variability by estimating individualized parcel boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Nickerson, L. D., Smith, S. M., ngr, D. &amp; Beckmann, C. F. Using dual regression to investigate network shape and amplitude in functional connectivity analyses. Front. Neurosci. 11, 115 (2017)." href="/articles/s41593-020-00726-z#ref-CR32" id="ref-link-section-d30294863e1198">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Kong, R. et al. Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion. Cereb. Cortex 29, 25332551 (2019)." href="/articles/s41593-020-00726-z#ref-CR41" id="ref-link-section-d30294863e1201">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Hacker, C. D. et al. Resting state network estimation in individual subjects. Neuroimage 82, 616633 (2013)." href="/articles/s41593-020-00726-z#ref-CR69" id="ref-link-section-d30294863e1204">69</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Gordon, E. M. et al. Precision functional mapping of individual human brains. Neuron 95, 791807.e7 (2017)." href="/articles/s41593-020-00726-z#ref-CR70" id="ref-link-section-d30294863e1207">70</a></sup>, integrating group and participant estimates in the same Bayesian framework<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Harrison, S. J. et al. Large-scale probabilistic functional modes from resting state fMRI. Neuroimage 109, 217231 (2015)." href="/articles/s41593-020-00726-z#ref-CR24" id="ref-link-section-d30294863e1211">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Harrison, S. J. et al. Modelling subject variability in the spatial and temporal characteristics of functional modes. NeuroImage 222, 117226 (2020)." href="/articles/s41593-020-00726-z#ref-CR71" id="ref-link-section-d30294863e1214">71</a></sup>, adopting naturalistic movie-watching paradigms to control variability during data acquisition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Vanderwal, T. et al. Individual differences in functional connectivity during naturalistic viewing conditions. Neuroimage 157, 521530 (2017)." href="/articles/s41593-020-00726-z#ref-CR72" id="ref-link-section-d30294863e1218">72</a></sup> or moving toward connectivity-based hyperalignment across participants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Guntupalli, J. S., Feilong, M. &amp; Haxby, J. V. A computational model of shared fine-scale structure in the human connectome. PLOS Comput. Biol. 14, e1006120 (2018)." href="/articles/s41593-020-00726-z#ref-CR73" id="ref-link-section-d30294863e1222">73</a></sup>. It is yet to be seen how successful these methods will be in generating brain representations that are robust to variability, yet sensitive to valuable inter-individual information.</p><p>In addition to between-participant variance, instability of brain representations within an individual over time (for example, between-session variance or even within-session dynamics within an individual) is a further source of variance for brain representations. While some studies have reported stable trait-like characteristics of brain representations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kieliba, P., Madugula, S., Filippini, N., Duff, E. P. &amp; Makin, T. R. Large-scale intrinsic connectivity is consistent across varying task demands. PLoS One 14, e0213861 (2019)." href="#ref-CR74" id="ref-link-section-d30294863e1229">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Horien, C., Shen, X., Scheinost, D. &amp; Constable, R. T. The individual functional connectome is unique and stable over months to years. Neuroimage 189, 676687 (2019)." href="#ref-CR75" id="ref-link-section-d30294863e1229_1">75</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Gratton, C. et al. Functional brain networks are dominated by stable group and individual factors, not cognitive or daily variation. Neuron 98, 439452.e5 (2018)." href="/articles/s41593-020-00726-z#ref-CR76" id="ref-link-section-d30294863e1232">76</a></sup>, other work has shown state-dependent changes based on task demands<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Salehi, M. et al. There is no single functional atlas even for a single individual: Functional parcel definitions change with task. Neuroimage 208, 116366 (2020)." href="/articles/s41593-020-00726-z#ref-CR77" id="ref-link-section-d30294863e1236">77</a></sup>, as well as fluctuations associated with arousal states<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bijsterbosch, J. et al. Investigations into within- and between-subject resting-state amplitude variations. Neuroimage 159, 5769 (2017)." href="/articles/s41593-020-00726-z#ref-CR39" id="ref-link-section-d30294863e1240">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Poldrack, R. A. et al. Long-term neural and physiological phenotyping of a single human. Nat. Commun. 6, 8885 (2015)." href="/articles/s41593-020-00726-z#ref-CR67" id="ref-link-section-d30294863e1243">67</a></sup> and physiology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Chen, J. E. et al. Resting-state physiological networks. NeuroImage 213, 116707 (2020)." href="/articles/s41593-020-00726-z#ref-CR78" id="ref-link-section-d30294863e1247">78</a></sup>. Within-participant longitudinal changes that occur as a function of development, aging or disease progression have yet to be characterized. Together, these potential sources of within-participant variability point to the importance of disambiguating trait and state effects in brain representations.</p><p>Complex interactions between within-participant variability, between-participant variability and dimensionality form a major challenge for the definition and interpretation of brain representations. Applications of brain representations mostly aim to investigate between-participant effects (for example, patientcontrol comparisons; fingerprinting identification of individuals; prediction or regression with respect to behavior, cognition or diagnosis). Determining which brain representations are most sensitive to between-participant effects is therefore critically important. For example, estimating individualized brain unit boundaries to remove misalignment as a source of between-participant effects will improve interpretability. Furthermore, comparisons to empirically inform the best dimensionality and summary measure for specific between-participant questions are of great importance. A recent example of such a benchmarking study performed by Dadi and colleagues provided specific recommendations, showing that weighted brain units and tangent space embedding of edges outperformed alternative brain representations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Dadi, K. et al. Benchmarking functional connectome-based predictive models for resting-state fMRI. Neuroimage 192, 115134 (2019). This paper is a great example of the benefit of comparing different brain representations to determine their relative value in addressing a specific research question. The results offer clear-cut insights that are important for brain representations. For example, functionally defined brain units clearly outperform anatomically defined brain units, which has informed our guideline #1 in Box 3." href="/articles/s41593-020-00726-z#ref-CR17" id="ref-link-section-d30294863e1254">17</a></sup>. Additional future comparative work will be of great value.</p><p>Non-neural confounds in functional MRI (fMRI) data add yet another source of unwanted variance. Structured artifacts can be caused by participant head motion, cardiac and respiratory cycles, and interactions of these participant factors with the magnetic field (in-)homogeneity, excitation pulses and image readout. Current preprocessing methods for removing non-neural confounds are imperfect<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Ciric, R. et al. Benchmarking of participant-level confound regression strategies for the control of motion artifact in studies of functional connectivity. Neuroimage 154, 174187 (2017)." href="/articles/s41593-020-00726-z#ref-CR79" id="ref-link-section-d30294863e1262">79</a></sup>, and their application to rfMRI data can have unwanted side-effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Murphy, K. &amp; Fox, M. D. Towards a consensus regarding global signal regression for resting state functional connectivity MRI. Neuroimage 154, 169173 (2017)." href="#ref-CR80" id="ref-link-section-d30294863e1266">80</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Power, J. D., Laumann, T. O., Plitt, M., Martin, A. &amp; Petersen, S. E. On global fMRI signals and simulations. Trends Cogn. Sci. 21, 911913 (2017)." href="#ref-CR81" id="ref-link-section-d30294863e1266_1">81</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Hallquist, M. N., Hwang, K. &amp; Luna, B. The nuisance of nuisance regression: spectral misspecification in a common approach to resting-state fMRI preprocessing reintroduces noise and obscures functional connectivity. Neuroimage 82, 208225 (2013)." href="/articles/s41593-020-00726-z#ref-CR82" id="ref-link-section-d30294863e1269">82</a></sup>. The summary measures of rfMRI brain representations are mostly based on similarities in the measured BOLD signal, and therefore require one or more source(s) of stochastic variation (for example, spontaneous neural activity fluctuations). The uncertain nature of these sources increases the impact of confounds for rfMRI (in general more than with task fMRI). The development and comparison of improved data preprocessing strategies is therefore an active area of research and discussion.</p><h3 class="c-article__sub-heading" id="Sec13">Representational ambiguity</h3><p>The importance of comparing and consolidating across available brain representations is underscored by their diverging interpretations. To demonstrate this, we take the example of a group of patients and a group of healthy controls, where the patients have relatively weaker functional connectivity between the frontal and parietal cortex. When these data are summarized using different brain representations, all will likely capture the group differences in some way. However, depending on the methodology, the results will potentially be represented very differently, leading to conflicting interpretations.</p><p>In a brain representation that defines a low-dimensional set of non-contiguous weighted brain units and allows the spatial voxel-wise weights to change across participants as the features (for example, as estimated from dual regression), the weaker frontoparietal functional connectivity in patients will be captured as lower weights in the frontoparietal brain unit map in patients. The interpretation of this finding would be described as a between-group difference in the spatial topography of the frontoparietal network. Alternatively, in a brain representation that defines a high-dimensional set of contiguous binary brain units and investigates between-unit edge connectivity as the summary measures, the above example will be captured as weaker pairwise functional connectivity estimates (i.e., edges) between frontal and parietal brain units in patients. Here the interpretation would be explained as a between-group difference in the amount of integration (or coupling) between frontal and parietal brain regions. Furthermore, in a connectivity gradient brain representation, the frontal and parietal regions may end up further apart in the embedding space in the patients, as a result of their connectivity patterns with the rest of the brain. This could result in a shift of one region relative to the other along the principal gradient, which would be interpreted as a difference in the organizational hierarchy of processing streams for multimodal integration in patients compared with healthy controls. In a final alternative example of a brain representation in which the signal amplitudes of brain units are considered as the features, the above group difference may be observed as a reduction in the signal strength in the frontal and/or parietal brain unit(s) in patients, leading to an interpretation in terms of between-group differences in levels of activation.</p><p>Depending on the research goal of a given study, these representational ambiguities may be more or less critical to the resulting conclusions. More general insights into functional brain organization (such as the similarity between resting state and task organization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Smith, S. M. et al. Correspondence of the brains functional architecture during activation and rest. Proc. Natl. Acad. Sci. USA 106, 1304013045 (2009)." href="/articles/s41593-020-00726-z#ref-CR83" id="ref-link-section-d30294863e1288">83</a></sup>) should be relatively independent of the specifics of the chosen brain representation. Similarly, the choice of brain representation may not be critical if the goal is to achieve accurate clinical or behavioral predictions (for example, the clinical prediction accuracy is potentially similar for all options described in the example above, because each brain representation captures the patientcontrol differences). However, the different brain representations have strongly diverging implications with respect to the hypothesized mechanism of psychopathology, with potentially conflicting treatment suggestions. Therefore, the best brain representation should inform theories on the origin (as opposed to downstream effects) of disease mechanisms and generate testable hypotheses for follow-up research. The absence of such biological interpretability may be acceptable if the goal is to adopt rfMRI purely for clinical or behavioral prediction. Unfortunately, however, ill-founded explanations often end up being given for such predictions.</p><p>The examples and studies discussed in this section illustrate two possible scenarios of representational ambiguity. First, we observe representational contamination, such that apparent changes in one summary measure (for example, connectivity) are driven by underlying changes in a different summary measure (for example, signal amplitude). This type of contamination is problematic because if researchers only test the contaminated summary measure (for example, connectivity), this would lead to an incorrect interpretation. Second, we observe the ill-posed nature of brain representations, because the same underlying effect may be represented either spatially or temporally depending on the dimensionality and characteristics of the brain representations. As a result, there is no single unique solution, and different internally consistent and potentially equally valid interpretations may be invoked to explain the same aspect of the underlying data (for example, change in spatial network shape or between-unit edge connectivity). Importantly, in the absence of ground-truth knowledge it can be difficult to distinguish whether a specific example constitutes an incorrect interpretation or one of several possible valid interpretations.</p><p>Studies reporting the types of representational ambiguity illustrated above have started to emerge. For example, data-driven simulations revealed that a large proportion of between-participant variance in network matrices was driven by inter-individual differences in spatial network topography, shifting the interpretation from coupling to spatial organization (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig3">3a</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Bijsterbosch, J. D. et al. The relationship between spatial configuration and functional connectivity of brain regions. eLife 7, e32992 (2018). This article uncovers an important source of representational ambiguity by showing that between-participant variance in node-to-node correlations is partly driven by variability in spatial organization. Data-driven simulations are used to interrogate interactions between different elements of brain representations, and shared versus unique variance is used to compare different brain representations." href="/articles/s41593-020-00726-z#ref-CR62" id="ref-link-section-d30294863e1302">62</a></sup>. Furthermore, the additive signal change approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Duff, E. P., Makin, T., Cottaar, M., Smith, S. M. &amp; Woolrich, M. W. Disambiguating brain functional connectivity. Neuroimage 173, 540550 (2018)." href="/articles/s41593-020-00726-z#ref-CR42" id="ref-link-section-d30294863e1306">42</a></sup> explicitly links different types of summary measures, by determining whether observed changes in functional connectivity network matrices are downstream effects of changes in signal amplitude (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig3">3b</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bijsterbosch, J. et al. Investigations into within- and between-subject resting-state amplitude variations. Neuroimage 159, 5769 (2017)." href="/articles/s41593-020-00726-z#ref-CR39" id="ref-link-section-d30294863e1313">39</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Toy examples of representational ambiguity."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Toy examples of representational ambiguity.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="276"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Individual differences in the degree of spatial overlap between two separate networks can be the underlying source of apparent changes in functional connectivity. Depending on the type of representation, network overlap may result in apparent connectivity changes through averaging mixed signals for hard parcellations or subtle biases in dual regression (weighted networks)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Bijsterbosch, J. D., Beckmann, C. F., Woolrich, M. W., Smith, S. M. &amp; Harrison, S. J. The relationship between spatial configuration and functional connectivity of brain regions revisited. eLife 8, e44890 (2019)." href="/articles/s41593-020-00726-z#ref-CR40" id="ref-link-section-d30294863e1332">40</a></sup>. <b>b</b>, The amplitude of the measured BOLD signal may increase in one region due to either added noise or added signal, which can result in apparent connectivity changes between this node and one or more other nodes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Duff, E. P., Makin, T., Cottaar, M., Smith, S. M. &amp; Woolrich, M. W. Disambiguating brain functional connectivity. Neuroimage 173, 540550 (2018)." href="/articles/s41593-020-00726-z#ref-CR42" id="ref-link-section-d30294863e1339">42</a></sup>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Recommendations and future directions for brain representations"><div class="c-article-section" id="Sec14-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec14">Recommendations and future directions for brain representations</h2><div class="c-article-section__content" id="Sec14-content"><p>The previous sections have highlighted tensions related to the question of brain representation. While a single gold standard representation is attractive in terms of generalizability and replicability, it may lack the flexibility to appropriately capture and account for variability, and it may curtail complementary insights that could be derived from alternative representations. Instead of championing one specific brain representation, we therefore propose a set of best practice guidelines intended to improve the transparency and robustness of research.</p><p>Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec15">3</a> provides a number of recommendations aimed at applied researchers who wish to adopt existing brain representations in their basic or translational research. The complexity of MRI acquisition and analysis approaches means that outdated research practices can persist, exacerbating the fractionation of the field. Therefore, several items in Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec15">3</a> (for example, #1, #37) are intended to encourage researchers to review their analysis pipelines and identify state-of-the-art approaches. Important building blocks for unifying the field include ensuring that the challenges of brain representation are widely discussed and increasing validation and replication. Therefore, several of the recommendations in Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec15">3</a> (#2, #9, #10) touch on reporting and open science practices (see also the reporting checklist in Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec16">4</a>), which will inform discussion and enable replication.</p><p>In Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec15">3</a> we recommend that authors should justify the choice of brain representation based on the research question and hypotheses (#2) and encourage comparisons across multiple brain representations (#8). To differentiate between brain representations and determine the most appropriate ones to test, there are a number of considerations to take into account.</p><ul class="u-list-style-bullet">
<li>
<p>Brain representations that address some of the challenges laid out in Challenges for brain representations should be preferred over and above representations that do not. For example, cortical brain units obtained from surface-based data with known improvements in between-participant alignment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Coalson, T. S., Van Essen, D. C. &amp; Glasser, M. F. The impact of traditional neuroimaging methods on the spatial localization of cortical areas. Proc. Natl. Acad. Sci. USA 115, E6356E6365 (2018)." href="/articles/s41593-020-00726-z#ref-CR64" id="ref-link-section-d30294863e1387">64</a></sup> are preferable to volumetrically defined cortical brain units. Additionally, brain units that estimate individualized boundaries for each participant (see section Dealing with variability) are preferable over group-averaged brain units.</p>
</li>
<li>
<p>It is important that the scale of the brain units is matched to the hypothesis. For example, if one expects lateralized effects, then an atlas or parcellation that defines left and right regions as separate brain units is appropriate. Similarly, if the hypothesis is specific to the posterior cingulate cortex, then the scale of the brain units should isolate this region instead of incorporating it within the larger default mode network.</p>
</li>
<li>
<p>The population used for the definition of brain units should be matched to the population of interest. The majority of available functional atlases are derived from healthy young adult participants. These brain units may be less appropriate to study developmental, aging or disease cohorts because the brain representation is biased toward a subset of (control) participants. Related to this point, we recommend that existing and new brain representations should explicitly be tested for their generalizability and applicability across different cohorts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig4">4</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Fig. 4"><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-020-00726-z/MediaObjects/41593_2020_726_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="349"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>Proposal for different phases of validation to provide increasing levels of evidence for brain representations.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-020-00726-z/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div>
</li>
<li>
<p>Interactions between the brain units and the summary measure need to be considered. For example, if the intention is to use partial correlation as the summary measure to improve sensitivity to direct (rather than indirect) connections, then it is likely advantageous to combine left and right homologs together into a single brain unit to improve numerical stability, even with explicit regularization of the model fitting.</p>
</li>
</ul><p>In addition to the best practice guidelines for adopting brain representations (Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec15">3</a>), we also propose a series of validation phases that should be applied to existing and future brain representations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig4">4</a>). Following the development of new brain representations, we propose explicit steps to (i) measure between-participant variance and ideally estimate individualized brain units, (ii) explicitly test the generalizability across different disease and lifespan populations and across scanners, (iii) perform systematic and extensive comparisons against multiple existing types of brain representations and, ultimately, (iv) inform the interpretation of a brain representation based on multimodal experiments.</p><p>To aid our ability to synthesize and compare different representations, targeted theoretical and empirical work to establish direct relationships between different brain representations would constitute a major contribution. For example, bottom-up biophysical simulations of neural firing (in which the ground truth is known) may provide a useful test environment to link different representational definitions of brain units and summary measures to each other and to elements of the underlying biophysiology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Smith, S. M. et al. Network modelling methods for FMRI. Neuroimage 54, 875891 (2011)." href="/articles/s41593-020-00726-z#ref-CR58" id="ref-link-section-d30294863e1444">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Deco, G., Jirsa, V. K., Robinson, P. A., Breakspear, M. &amp; Friston, K. The dynamic brain: from spiking neurons to neural masses and cortical fields. PLOS Comput. Biol. 4, e1000092 (2008)." href="/articles/s41593-020-00726-z#ref-CR84" id="ref-link-section-d30294863e1447">84</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Buibas, M. &amp; Silva, G. A. A framework for simulating and estimating the state and functional topology of complex dynamic geometric networks. Neural Comput. 23, 183214 (2011)." href="/articles/s41593-020-00726-z#ref-CR85" id="ref-link-section-d30294863e1450">85</a></sup>. In addition, mathematical efforts to establish known mapping functions between different brain representations, and/or between a common atlas and customized brain units of individuals or sub-groups, would greatly facilitate cross-study generalizability and replicability. These suggestions for future work will be aided by continued efforts to characterize and disentangle different sources of variance (confounds as well as between- and/or within-participant variation).</p><p>Establishing biological interpretability of fMRI-derived brain representations by studying the underlying neural circuitry and cellular mechanisms is critically important for the next generation of rfMRI research (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-020-00726-z#Fig4">4</a>). Specifically, gaining neurophysiological insights into different summary measures (such as between-region connectivity and signal amplitudes) is expected to reduce representational ambiguity and inform appropriate interpretations. Integrating information across different (potentially invasive) measurement modalities is needed to determine how different brain representations derived from fMRI relate to neural firing and cellular processes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Logothetis, N. K., Pauls, J., Augath, M., Trinath, T. &amp; Oeltermann, A. Neurophysiological investigation of the basis of the fMRI signal. Nature 412, 150157 (2001)." href="#ref-CR86" id="ref-link-section-d30294863e1460">86</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shmuel, A. &amp; Leopold, D. A. Neuronal correlates of spontaneous fluctuations in fMRI signals in monkey visual cortex: Implications for functional connectivity at rest. Hum. Brain Mapp. 29, 751761 (2008). This paper used simultaneous rfMRI and intracortical recordings to show that between-region correlations measured from resting state fMRI (i.e., functional connectivity) are linked to synchronization of neuronal signals. This study is one of the few examples that link rfMRI brain representations to the underlying neurophysiology. Similar research efforts are needed to explicitly differentiate between different brain representations, as we have proposed in section Recommendations and future directions for brain representations and Fig. 4." href="#ref-CR87" id="ref-link-section-d30294863e1460_1">87</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Kucyi, A. et al. Intracranial electrophysiology reveals reproducible intrinsic functional connectivity within human brain networks. J. Neurosci. 38, 42304242 (2018)." href="/articles/s41593-020-00726-z#ref-CR88" id="ref-link-section-d30294863e1463">88</a></sup>. Combining fMRI with complementary non-invasive electrophysiology, for example, electro- and/or magnetoencephalography (EEG/MEG), offers the opportunity to noninvasively probe large-scale brain networks with more direct neuronal measures, albeit with coarser spatial detail, but free from vascular confounds and at faster time-scales, thereby aiding in the validation of dynamic representations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Vidaurre, D. et al. Spontaneous cortical activity transiently organises into frequency specific phase-coupling networks. Nat. Commun. 9, 2987 (2018)." href="/articles/s41593-020-00726-z#ref-CR36" id="ref-link-section-d30294863e1467">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Baillet, S. Magnetoencephalography for brain electrophysiology and imaging. Nat. Neurosci. 20, 327339 (2017)." href="/articles/s41593-020-00726-z#ref-CR89" id="ref-link-section-d30294863e1470">89</a></sup>. Additionally, electrode recordings in animals measuring local field potentials, neural spiking and/or oxygen polarography, as well as tracer studies measuring structural connectivity, can offer more direct biological validation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Bentley, W. J., Li, J. M., Snyder, A. Z., Raichle, M. E. &amp; Snyder, L. H. Oxygen level and LFP in task-positive and task-negative areas: bridging BOLD fMRI and electrophysiology. Cereb. Cortex 26, 346357 (2016)." href="/articles/s41593-020-00726-z#ref-CR90" id="ref-link-section-d30294863e1474">90</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Schlvinck, M. L., Maier, A., Ye, F. Q., Duyn, J. H. &amp; Leopold, D. A. Neural basis of global resting-state fMRI activity. Proc. Natl. Acad. Sci. USA 107, 1023810243 (2010)." href="/articles/s41593-020-00726-z#ref-CR91" id="ref-link-section-d30294863e1477">91</a></sup>. Taken together, multimodal research efforts are expected to provide insights into the biological basis of rfMRI-derived brain representations, which will be critical for determining the most appropriate representation and to inform a biological grounding to inferences.</p><div class="c-article-box" data-expandable-box-container="true"><div class="c-article-box__container" data-expandable-box="true" aria-hidden="true" id="box-Sec15"><h3 class="c-article-box__container-title u-h3 js-expandable-title" id="Sec15">Box 3 Best practice guidelines</h3><div class="c-article-box__content"><ol class="u-list-style-none">
<li>
<span class="u-custom-list-number">1.</span>

<p><b>Avoid the use of outdated anatomic definitions of brain units</b></p>
<p>There is robust evidence that parcellations derived purely from gyral and sulcal anatomy, such as the automated anatomical labeling (AAL), HarvardOxford atlas or FreeSurfer folding-based parcellations, are not suitable brain units for use in functional rfMRI brain representations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Dadi, K. et al. Benchmarking functional connectome-based predictive models for resting-state fMRI. Neuroimage 192, 115134 (2019). This paper is a great example of the benefit of comparing different brain representations to determine their relative value in addressing a specific research question. The results offer clear-cut insights that are important for brain representations. For example, functionally defined brain units clearly outperform anatomically defined brain units, which has informed our guideline #1 in Box 3." href="/articles/s41593-020-00726-z#ref-CR17" id="ref-link-section-d30294863e1501">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 101" title="Sala-Llonch, R., Smith, S. M., Woolrich, M. &amp; Duff, E. P. Spatial parcellations, spectral filtering, and connectivity measures in fMRI: Optimizing for discrimination. Hum. Brain Mapp. 40, 407419 (2019)." href="/articles/s41593-020-00726-z#ref-CR101" id="ref-link-section-d30294863e1504">101</a></sup>, particularly when they are based on only small numbers of participants. Instead, definitions of brain units that take into account functional data (potentially alongside other imaging modalities) should be used.</p>

</li>
<li>
<span class="u-custom-list-number">2.</span>

<p><b>Include a justification for the chosen brain representation</b></p>
<p>We recommend that all articles report a justification for the chosen brain representation and a clear description of the associated assumptions in relation to challenges raised in the section Challenges for brain representations (see checklist in Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec16">4</a>). These assumptions should be explicitly acknowledged and considered when the results are discussed (see guideline #10). This level of transparency will help with the interpretation of findings within the broader literature and will inform follow-up work.</p>

</li>
<li>
<span class="u-custom-list-number">3.</span>

<p><b>Adopt established, state-of-the-art data acquisition standards and appropriate confound monitoring</b></p>
<p>Recent findings clearly indicate advantages of eyes-open instructions (ideally with fixation cross), longer data acquisition and/or multiple sessions per participant, faster repetition time (potentially using accelerated imaging methods) and adequate spatial resolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Smith, S. M. et al. Resting-state fMRI in the Human Connectome Project. Neuroimage 80, 144168 (2013)." href="/articles/s41593-020-00726-z#ref-CR9" id="ref-link-section-d30294863e1541">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-020-00726-z#ref-CR10" id="ref-link-section-d30294863e1544">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="Noble, S., Scheinost, D. &amp; Constable, R. T. A decade of test-retest reliability of functional connectivity: a systematic review and meta-analysis. Neuroimage 203, 116157 (2019)." href="/articles/s41593-020-00726-z#ref-CR102" id="ref-link-section-d30294863e1547">102</a></sup>. In addition, concurrent tracking of physiology and arousal states are beneficial<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Chen, J. E. et al. Resting-state physiological networks. NeuroImage 213, 116707 (2020)." href="/articles/s41593-020-00726-z#ref-CR78" id="ref-link-section-d30294863e1551">78</a></sup>. If these recommendations are not possible to implement (for example in legacy data), then be aware of the limitations this imposes on downstream analyses (in terms of, for example, the resolution of brain units it is possible to utilize).</p>

</li>
<li>
<span class="u-custom-list-number">4.</span>

<p><b>Implement optimal data preprocessing, alignment and harmonization methods</b></p>
<p>While consensus on optimal preprocessing pipelines is lacking (see guideline #5), it is clear that nuisance regression of head motion, white matter and cerebrospinal fluid regressors, combined with low-pass filtering, is insufficient for denoising<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Ciric, R. et al. Benchmarking of participant-level confound regression strategies for the control of motion artifact in studies of functional connectivity. Neuroimage 154, 174187 (2017)." href="/articles/s41593-020-00726-z#ref-CR79" id="ref-link-section-d30294863e1570">79</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Burgess, G. C. et al. Evaluation of denoising strategies to address motion-correlated artifacts in resting-state functional magnetic resonance imaging data from the Human Connectome Project. Brain Connect. 6, 669680 (2016)." href="/articles/s41593-020-00726-z#ref-CR103" id="ref-link-section-d30294863e1573">103</a></sup>. Instead, spatial ICA-based cleanup methods should be used where possible (and/or volume censoring where not). In addition, when possible, studies should use multimodal surface-based alignment to reduce between-participant variability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Coalson, T. S., Van Essen, D. C. &amp; Glasser, M. F. The impact of traditional neuroimaging methods on the spatial localization of cortical areas. Proc. Natl. Acad. Sci. USA 115, E6356E6365 (2018)." href="/articles/s41593-020-00726-z#ref-CR64" id="ref-link-section-d30294863e1577">64</a></sup>. For transparency, the use of validated and published methods and pipelines (for example, refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 104" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105124 (2013)." href="/articles/s41593-020-00726-z#ref-CR104" id="ref-link-section-d30294863e1581">104</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 105" title="Esteban, O. et al. fMRIPrep: a robust preprocessing pipeline for functional MRI. Nat. Methods 16, 111116 (2019)." href="/articles/s41593-020-00726-z#ref-CR105" id="ref-link-section-d30294863e1584">105</a></sup>) is preferable over preprocessing scripts developed in-house. Lastly, when collating data from different studies or centers, data harmonization techniques (beyond a simple site regressor) are required to appropriately account for site effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 106" title="Yu, M. et al. Statistical harmonization corrects site effects in functional connectivity measurements from multi-site fMRI data. Hum. Brain Mapp. 39, 42134227 (2018)." href="/articles/s41593-020-00726-z#ref-CR106" id="ref-link-section-d30294863e1588">106</a></sup>.</p>

</li>
<li>
<span class="u-custom-list-number">5.</span>

<p><b>If global signal regression changes your findings, also present results without it</b></p>
<p>Aside from the recommendations in #4, some studies remove (i.e., regress out) any variance that can be explained by the whole-brain (globally averaged) BOLD time series in an attempt to remove spatially diffuse confounds<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 107" title="Power, J. D., Plitt, M., Laumann, T. O. &amp; Martin, A. Sources and implications of whole-brain fMRI signals in humans. Neuroimage 146, 609625 (2017)." href="/articles/s41593-020-00726-z#ref-CR107" id="ref-link-section-d30294863e1607">107</a></sup>. However, this practice (termed global signal regression or GSR) has been shown to affect the definition of brain units by shifting boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-020-00726-z#ref-CR10" id="ref-link-section-d30294863e1611">10</a></sup> and to affect summary measures by shifting the edge distribution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 108" title="Saad, Z. S. et al. Trouble at rest: how correlation patterns and group differences become distorted after global signal regression. Brain Connect. 2, 2532 (2012)." href="/articles/s41593-020-00726-z#ref-CR108" id="ref-link-section-d30294863e1615">108</a></sup>. Therefore, if authors wish to publish findings with the use of GSR, these should be accompanied by comparable results without GSR (potentially placing one in the supplementary files). This will facilitate comparisons across studies and may inform the discussion. The use of alternative approaches that avoid some of these biases, such as physiological models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 109" title="Glover, G. H., Li, T. Q. &amp; Ress, D. Image-based method for retrospective correction of physiological motion effects in fMRI: RETROICOR. Magn. Reson. Med. 44, 162167 (2000)." href="/articles/s41593-020-00726-z#ref-CR109" id="ref-link-section-d30294863e1619">109</a></sup> and/or temporal ICA cleanup<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 110" title="Glasser, M. F. et al. Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data. Neuroimage 181, 692717 (2018)." href="/articles/s41593-020-00726-z#ref-CR110" id="ref-link-section-d30294863e1623">110</a></sup>, should also be considered. It should also be noted that the use of partial correlation instead of full correlation to estimate edges removes the need for GSR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Smith, S. M. et al. Functional connectomics from resting-state fMRI. Trends Cogn. Sci. 17, 666682 (2013)." href="/articles/s41593-020-00726-z#ref-CR28" id="ref-link-section-d30294863e1628">28</a></sup>.</p>

</li>
<li>
<span class="u-custom-list-number">6.</span>

<p><b>Use well-powered samples and out-of-sample validation</b></p>
<p>A wide range of freely shared large-scale data are now available from healthy participants and various clinical cohorts. To overcome challenges with replicability, studies should make extensive use of these resources to achieve robust sample sizes and to validate results obtained from one dataset by including external replications in other datasets. Funding bodies play a role in recognizing such efforts by supporting ongoing analysis of existing data resources.</p>

</li>
<li>
<span class="u-custom-list-number">7.</span>

<p><b>Adopt robust statistical approaches</b></p>
<p>Quantitative statistical tests should be favored over qualitative descriptive results where possible. Many common methods such as clustering and canonical correlation analyses will by definition return results, and appropriate significance testing and stringent cross-validation are essential for robust inference<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 111" title="Dinga, R. et al. Evaluating the evidence for biotypes of depression: methodological replication and extension of Drysdale et al. (2017). Neuroimage Clin. 22, 101796 (2019)." href="/articles/s41593-020-00726-z#ref-CR111" id="ref-link-section-d30294863e1663">111</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 112" title="Varoquaux, G. et al. Assessing and tuning brain decoders: cross-validation, caveats, and guidelines. Neuroimage 145, 166179 (2017). Pt B." href="/articles/s41593-020-00726-z#ref-CR112" id="ref-link-section-d30294863e1666">112</a></sup>. Robust statistical methods should include correction for multiple comparisons with the use of appropriate statistical thresholds<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 113" title="Eklund, A., Nichols, T. E. &amp; Knutsson, H. Cluster failure: why fMRI inferences for spatial extent have inflated false-positive rates. Proc. Natl. Acad. Sci. USA 113, 79007905 (2016)." href="/articles/s41593-020-00726-z#ref-CR113" id="ref-link-section-d30294863e1670">113</a></sup>.</p>

</li>
<li>
<span class="u-custom-list-number">8.</span>

<p><b>Compare different brain representation where possible</b></p>
<p>To aid the generalizability of results, we recommend that studies show how their findings vary as a function of brain representation. While articles are starting to include results across a variety of dimensionalities and/or using a number of different binary parcellations, broader comparisons are rare (although see refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Dadi, K. et al. Benchmarking functional connectome-based predictive models for resting-state fMRI. Neuroimage 192, 115134 (2019). This paper is a great example of the benefit of comparing different brain representations to determine their relative value in addressing a specific research question. The results offer clear-cut insights that are important for brain representations. For example, functionally defined brain units clearly outperform anatomically defined brain units, which has informed our guideline #1 in Box 3." href="/articles/s41593-020-00726-z#ref-CR17" id="ref-link-section-d30294863e1689">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 114" title="Arslan, S. et al. Human brain mapping: a systematic comparison of parcellation methods for the human cerebral cortex. Neuroimage 170, 530 (2018)." href="/articles/s41593-020-00726-z#ref-CR114" id="ref-link-section-d30294863e1692">114</a></sup>). Providing directly comparable findings will be of great value in building an understanding of how different representations of the same data relate to each other. Additionally, comparative testing of different types of summary measures from the same brain units (such as signal amplitudes, network matrices and individualized brain unit shape and/or size) can reduce ambiguity and inform mechanistic interpretability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bijsterbosch, J. et al. Investigations into within- and between-subject resting-state amplitude variations. Neuroimage 159, 5769 (2017)." href="/articles/s41593-020-00726-z#ref-CR39" id="ref-link-section-d30294863e1696">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Duff, E. P., Makin, T., Cottaar, M., Smith, S. M. &amp; Woolrich, M. W. Disambiguating brain functional connectivity. Neuroimage 173, 540550 (2018)." href="/articles/s41593-020-00726-z#ref-CR42" id="ref-link-section-d30294863e1699">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Bijsterbosch, J. D. et al. The relationship between spatial configuration and functional connectivity of brain regions. eLife 7, e32992 (2018). This article uncovers an important source of representational ambiguity by showing that between-participant variance in node-to-node correlations is partly driven by variability in spatial organization. Data-driven simulations are used to interrogate interactions between different elements of brain representations, and shared versus unique variance is used to compare different brain representations." href="/articles/s41593-020-00726-z#ref-CR62" id="ref-link-section-d30294863e1702">62</a></sup>.</p>

</li>
<li>
<span class="u-custom-list-number">9.</span>

<p><b>Publicly share all code, data and results</b></p>
<p>It is essential for transparency and replicability to make research outputs publicly available<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 115" title="Poldrack, R. A. et al. Scanning the horizon: towards transparent and reproducible neuroimaging research. Nat. Rev. Neurosci. 18, 115126 (2017)." href="/articles/s41593-020-00726-z#ref-CR115" id="ref-link-section-d30294863e1721">115</a></sup>. Many excellent platforms are now available to share code (<a href="https://github.com/git">https://github.com/git</a>), data alongside analysis pipelines (<a href="https://openneuro.org/">https://openneuro.org/</a>) and results (<a href="https://balsa.wustl.edu/">https://balsa.wustl.edu/</a>). Importantly, sharing data output facilitates meta-analytical approaches and allows the academic community as a whole to build on the research of others, as required for productive and cumulative scientific progress.</p>

</li>
<li>
<span class="u-custom-list-number">10.</span>

<p><b>Avoid over-interpretation of findings and clearly state caveats</b></p>
<p>When describing findings, we recommend that authors stay close to the empirical results and avoid terminology that is suggestive of biological mechanisms (unless directly warranted by the results). Explicitly discussing potential interpretational caveats (for example, in relation to the brain representation used, representational ambiguity or BOLD limitations) is encouraged (see the checklist in Box <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-020-00726-z#Sec16">4</a>).</p>

</li>
</ol></div></div></div><div class="c-article-box" data-expandable-box-container="true"><div class="c-article-box__container" data-expandable-box="true" aria-hidden="true" id="box-Sec16"><h3 class="c-article-box__container-title u-h3 js-expandable-title" id="Sec16">Box 4 Reporting checklist</h3><div class="c-article-box__content"><ul class="u-list-style-bullet">
<li>
<p>Have the brain units and summary measures been publicly shared?</p>
</li>
<li>
<p>Does the article include a clear motivation for the following elements?</p><ul class="u-list-style-bullet">
<li>
<p>Preprocessing choices</p>
</li>
<li>
<p>Dimensionality</p>
</li>
<li>
<p>Choice of brain representation</p>
</li>
<li>
<p>Assumptions associated with the chosen brain representation</p>
</li>
<li>
<p>Implications of the chosen brain representation for interpretation</p>
</li>
</ul>
</li>
<li>
<p>Is the algorithm to produce the brain representation from (new) data freely available?</p>
</li>
<li>
<p>Have alternate representations been tested?</p><ul class="u-list-style-bullet">
<li>
<p>Does using different brain units give concordant results?</p>
</li>
<li>
<p>Do different types of summary measures give concordant or illuminating results?</p>
</li>
</ul>
</li>
</ul></div></div></div></div></div></section><section data-title="Conclusion"><div class="c-article-section" id="Sec17-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec17">Conclusion</h2><div class="c-article-section__content" id="Sec17-content"><p>The rfMRI field is fractionated by the divergence of brain representations. Despite the importance of the chosen brain representation for study outcomes and interpretation, articles rarely include a clear justification for adopting a specific representation. Instead, it is common for laboratories to subscribe to a specific approach and apply this across all research projects with relatively little consideration for the implicit assumptions of their chosen brain representation. This tendency can produce research silos of segregated reasoning and assumptions that are at odds with the fundamental principles of cumulative science. Advancing the field beyond these research silos and toward successful collaborative brain mapping and interpretable biomarker discovery critically requires a better understanding of the relationships between different representations of the same data. Once we gain a clearer understanding of the relationships between different brain representations and between representations of rfMRI data and the underlying neurophysiology, some of the key concepts, interpretations, definitions and nomenclature in the field may need to be redefined or updated. This will require a commitment and willingness from members in the field to test, challenge and revise our assumptions and core principles. Improving the interpretability of rfMRI brain representations will increase the replicability of results across different research laboratories and improve the real-life clinical impact of rfMRI to inform diagnosis and treatment. The guidelines and suggestions proposed in this article are intended to bring the broader community together in setting new standards for the field.</p></div></div></section>
                </div>
            

            <div>
                <div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Van Essen, D. C. &amp; Glasser, M. F. Parcellating cerebral cortex: how invasive animal studies inform noninvasive mapmaking in humans. <i>Neuron</i> <b>99</b>, 640663 (2018). <b>This review of brain mapping (i.e., parcellating the brain into units) summarizes extensive research performed in non-human primates and discusses how these results inform cortical brain parcellation efforts in humans using noninvasive imaging techniques including rfMRI. This article provides both a historical and a future perspective on cortical brain parcellation, which is an important aspect of brain representations (related to our description defining the brain units in section Defining a brain unit)</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30138588" aria-label="PubMed reference 1">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6149530" aria-label="PubMed Central reference 1">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Parcellating%20cerebral%20cortex%3A%20how%20invasive%20animal%20studies%20inform%20noninvasive%20mapmaking%20in%20humans&amp;journal=Neuron&amp;volume=99&amp;pages=640-663&amp;publication_year=2018&amp;author=Essen%2CDC&amp;author=Glasser%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Bijsterbosch, J., Smith, S.M. &amp; Beckmann, C.F. <i>Introduction to Resting State fMRI Functional Connectivity</i> (Oxford Univ. Press, 2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Reid, A. T. et al. Advancing functional connectivity research from association to causation. <i>Nat. Neurosci.</i> <b>22</b>, 17511760 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFKgsr%2FF" aria-label="CAS reference 3">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31611705" aria-label="PubMed reference 3">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7289187" aria-label="PubMed Central reference 3">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Advancing%20functional%20connectivity%20research%20from%20association%20to%20causation&amp;journal=Nat.%20Neurosci.&amp;volume=22&amp;pages=1751-1760&amp;publication_year=2019&amp;author=Reid%2CAT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Botvinik-Nezer, R. et al. Variability in the analysis of a single neuroimaging dataset by many teams. <i>Nature</i> <b>582</b>, 8488 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXpvVantLg%3D" aria-label="CAS reference 4">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32483374" aria-label="PubMed reference 4">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Variability%20in%20the%20analysis%20of%20a%20single%20neuroimaging%20dataset%20by%20many%20teams&amp;journal=Nature&amp;volume=582&amp;pages=84-88&amp;publication_year=2020&amp;author=Botvinik-Nezer%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Eickhoff, S. B., Yeo, B. T. T. &amp; Genon, S. Imaging-based parcellations of the human brain. <i>Nat. Rev. Neurosci.</i> <b>19</b>, 672686 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFygurjE" aria-label="CAS reference 5">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30305712" aria-label="PubMed reference 5">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Imaging-based%20parcellations%20of%20the%20human%20brain&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=19&amp;pages=672-686&amp;publication_year=2018&amp;author=Eickhoff%2CSB&amp;author=Yeo%2CBTT&amp;author=Genon%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Ferrante, M. et al. Computational psychiatry: a report from the 2017 NIMH workshop on opportunities and challenges. <i>Mol. Psychiatry</i> <b>24</b>, 479483 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29703948" aria-label="PubMed reference 6">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Computational%20psychiatry%3A%20a%20report%20from%20the%202017%20NIMH%20workshop%20on%20opportunities%20and%20challenges&amp;journal=Mol.%20Psychiatry&amp;volume=24&amp;pages=479-483&amp;publication_year=2019&amp;author=Ferrante%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Poldrack, R. A., Huckins, G. &amp; Varoquaux, G. Establishment of best practices for evidence for prediction: a review. <i>JAMA Psychiatry</i> <b>77</b>, 534540 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31774490" aria-label="PubMed reference 7">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7250718" aria-label="PubMed Central reference 7">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Establishment%20of%20best%20practices%20for%20evidence%20for%20prediction%3A%20a%20review&amp;journal=JAMA%20Psychiatry&amp;volume=77&amp;pages=534-540&amp;publication_year=2020&amp;author=Poldrack%2CRA&amp;author=Huckins%2CG&amp;author=Varoquaux%2CG">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Lopes-dos-Santos, V., Ribeiro, S. &amp; Tort, A. B. L. Detecting cell assemblies in large neuronal populations. <i>J. Neurosci. Methods</i> <b>220</b>, 149166 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23639919" aria-label="PubMed reference 8">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Detecting%20cell%20assemblies%20in%20large%20neuronal%20populations&amp;journal=J.%20Neurosci.%20Methods&amp;volume=220&amp;pages=149-166&amp;publication_year=2013&amp;author=Lopes-dos-Santos%2CV&amp;author=Ribeiro%2CS&amp;author=Tort%2CABL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Smith, S. M. et al. Resting-state fMRI in the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 144168 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23702415" aria-label="PubMed reference 9">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3720828" aria-label="PubMed Central reference 9">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting-state%20fMRI%20in%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;volume=80&amp;pages=144-168&amp;publication_year=2013&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. <i>Nature</i> <b>536</b>, 171178 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1GrurnK" aria-label="CAS reference 10">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27437579" aria-label="PubMed reference 10">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990127" aria-label="PubMed Central reference 10">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-modal%20parcellation%20of%20human%20cerebral%20cortex&amp;journal=Nature&amp;volume=536&amp;pages=171-178&amp;publication_year=2016&amp;author=Glasser%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Eickhoff, S. B., Thirion, B., Varoquaux, G. &amp; Bzdok, D. Connectivity-based parcellation: critique and implications. <i>Hum. Brain Mapp.</i> <b>36</b>, 47714792 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26409749" aria-label="PubMed reference 11">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6869530" aria-label="PubMed Central reference 11">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Connectivity-based%20parcellation%3A%20critique%20and%20implications&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=36&amp;pages=4771-4792&amp;publication_year=2015&amp;author=Eickhoff%2CSB&amp;author=Thirion%2CB&amp;author=Varoquaux%2CG&amp;author=Bzdok%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Uddin, L. Q., Yeo, B. T. T. &amp; Spreng, R. N. Towards a universal taxonomy of macro-scale functional human brain networks. <i>Brain Topogr.</i> <b>32</b>, 926942 (2019). <b>This article discusses the challenges of scale and dimensionality of brain representations (related to our discussion of dimensionality in the section Heterogeneity and dimensionality of brain units). Focusing on the level of large-scale brain networks, the authors propose a consistent nomenclature for the naming of networks grounded in anatomy to address widespread problems with inconsistent terminology in the literature</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31707621" aria-label="PubMed reference 12">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7325607" aria-label="PubMed Central reference 12">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20a%20universal%20taxonomy%20of%20macro-scale%20functional%20human%20brain%20networks&amp;journal=Brain%20Topogr.&amp;volume=32&amp;pages=926-942&amp;publication_year=2019&amp;author=Uddin%2CLQ&amp;author=Yeo%2CBTT&amp;author=Spreng%2CRN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Eickhoff, S. B., Constable, R. T. &amp; Yeo, B. T. T. Topographic organization of the cerebral cortex and brain cartography. <i>Neuroimage</i> <b>170</b>, 332347 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28219775" aria-label="PubMed reference 13">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Topographic%20organization%20of%20the%20cerebral%20cortex%20and%20brain%20cartography&amp;journal=Neuroimage&amp;volume=170&amp;pages=332-347&amp;publication_year=2018&amp;author=Eickhoff%2CSB&amp;author=Constable%2CRT&amp;author=Yeo%2CBTT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Tzourio-Mazoyer, N. et al. Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain. <i>Neuroimage</i> <b>15</b>, 273289 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD38%2FltFCntw%3D%3D" aria-label="CAS reference 14">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11771995" aria-label="PubMed reference 14">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20anatomical%20labeling%20of%20activations%20in%20SPM%20using%20a%20macroscopic%20anatomical%20parcellation%20of%20the%20MNI%20MRI%20single-subject%20brain&amp;journal=Neuroimage&amp;volume=15&amp;pages=273-289&amp;publication_year=2002&amp;author=Tzourio-Mazoyer%2CN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. <i>Neuroimage</i> <b>31</b>, 968980 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20automated%20labeling%20system%20for%20subdividing%20the%20human%20cerebral%20cortex%20on%20MRI%20scans%20into%20gyral%20based%20regions%20of%20interest&amp;journal=Neuroimage&amp;volume=31&amp;pages=968-980&amp;publication_year=2006&amp;author=Desikan%2CRS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Eickhoff, S. B. et al. A new SPM toolbox for combining probabilistic cytoarchitectonic maps and functional imaging data. <i>Neuroimage</i> <b>25</b>, 13251335 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15850749" aria-label="PubMed reference 16">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20SPM%20toolbox%20for%20combining%20probabilistic%20cytoarchitectonic%20maps%20and%20functional%20imaging%20data&amp;journal=Neuroimage&amp;volume=25&amp;pages=1325-1335&amp;publication_year=2005&amp;author=Eickhoff%2CSB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Dadi, K. et al. Benchmarking functional connectome-based predictive models for resting-state fMRI. <i>Neuroimage</i> <b>192</b>, 115134 (2019). <b>This paper is a great example of the benefit of comparing different brain representations to determine their relative value in addressing a specific research question. The results offer clear-cut insights that are important for brain representations. For example, functionally defined brain units clearly outperform anatomically defined brain units, which has informed our guideline #1 in Box 3</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30836146" aria-label="PubMed reference 17">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Benchmarking%20functional%20connectome-based%20predictive%20models%20for%20resting-state%20fMRI&amp;journal=Neuroimage&amp;volume=192&amp;pages=115-134&amp;publication_year=2019&amp;author=Dadi%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Schaefer, A. et al. Local-global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. <i>Cereb. Cortex</i> <b>28</b>, 30953114 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28981612" aria-label="PubMed reference 18">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Local-global%20parcellation%20of%20the%20human%20cerebral%20cortex%20from%20intrinsic%20functional%20connectivity%20MRI&amp;journal=Cereb.%20Cortex&amp;volume=28&amp;pages=3095-3114&amp;publication_year=2018&amp;author=Schaefer%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Shen, X., Tokoglu, F., Papademetris, X. &amp; Constable, R. T. Groupwise whole-brain parcellation from resting-state fMRI data for network node identification. <i>Neuroimage</i> <b>82</b>, 403415 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC3sjhtV2gug%3D%3D" aria-label="CAS reference 19">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23747961" aria-label="PubMed reference 19">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Groupwise%20whole-brain%20parcellation%20from%20resting-state%20fMRI%20data%20for%20network%20node%20identification&amp;journal=Neuroimage&amp;volume=82&amp;pages=403-415&amp;publication_year=2013&amp;author=Shen%2CX&amp;author=Tokoglu%2CF&amp;author=Papademetris%2CX&amp;author=Constable%2CRT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Craddock, R. C., James, G. A., Holtzheimer, P. E. III, Hu, X. P. &amp; Mayberg, H. S. A whole brain fMRI atlas generated via spatially constrained spectral clustering. <i>Hum. Brain Mapp.</i> <b>33</b>, 19141928 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21769991" aria-label="PubMed reference 20">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20whole%20brain%20fMRI%20atlas%20generated%20via%20spatially%20constrained%20spectral%20clustering&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=33&amp;pages=1914-1928&amp;publication_year=2012&amp;author=Craddock%2CRC&amp;author=James%2CGA&amp;author=Holtzheimer%2CPE&amp;author=Hu%2CXP&amp;author=Mayberg%2CHS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Gordon, E. M. et al. Generation and evaluation of a cortical area parcellation from resting-state correlations. <i>Cereb. Cortex</i> <b>26</b>, 288303 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25316338" aria-label="PubMed reference 21">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Generation%20and%20evaluation%20of%20a%20cortical%20area%20parcellation%20from%20resting-state%20correlations&amp;journal=Cereb.%20Cortex&amp;volume=26&amp;pages=288-303&amp;publication_year=2016&amp;author=Gordon%2CEM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. <i>J. Neurophysiol.</i> <b>106</b>, 11251165 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21653723" aria-label="PubMed reference 22">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20organization%20of%20the%20human%20cerebral%20cortex%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=J.%20Neurophysiol.&amp;volume=106&amp;pages=1125-1165&amp;publication_year=2011&amp;author=Yeo%2CBTT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Beckmann, C. F., DeLuca, M., Devlin, J. T. &amp; Smith, S. M. Investigations into resting-state connectivity using independent component analysis. <i>Phil. Trans. R. Soc. Lond. B</i> <b>360</b>, 10011013 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Investigations%20into%20resting-state%20connectivity%20using%20independent%20component%20analysis&amp;journal=Phil.%20Trans.%20R.%20Soc.%20Lond.%20B&amp;volume=360&amp;pages=1001-1013&amp;publication_year=2005&amp;author=Beckmann%2CCF&amp;author=DeLuca%2CM&amp;author=Devlin%2CJT&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Harrison, S. J. et al. Large-scale probabilistic functional modes from resting state fMRI. <i>Neuroimage</i> <b>109</b>, 217231 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25598050" aria-label="PubMed reference 24">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4349633" aria-label="PubMed Central reference 24">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20probabilistic%20functional%20modes%20from%20resting%20state%20fMRI&amp;journal=Neuroimage&amp;volume=109&amp;pages=217-231&amp;publication_year=2015&amp;author=Harrison%2CSJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Varoquaux, G., Gramfort, A., Pedregosa, F., Michel, V. &amp; Thirion, B. Multi-subject dictionary learning to segment an atlas of brain spontaneous activity. <i>Inf. Process. Med. Imaging</i> <b>22</b>, 562573 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21761686" aria-label="PubMed reference 25">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-subject%20dictionary%20learning%20to%20segment%20an%20atlas%20of%20brain%20spontaneous%20activity&amp;journal=Inf.%20Process.%20Med.%20Imaging&amp;volume=22&amp;pages=562-573&amp;publication_year=2011&amp;author=Varoquaux%2CG&amp;author=Gramfort%2CA&amp;author=Pedregosa%2CF&amp;author=Michel%2CV&amp;author=Thirion%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Eavani, H. et al. Identifying sparse connectivity patterns in the brain using resting-state fMRI. <i>Neuroimage</i> <b>105</b>, 286299 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25284301" aria-label="PubMed reference 26">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Identifying%20sparse%20connectivity%20patterns%20in%20the%20brain%20using%20resting-state%20fMRI&amp;journal=Neuroimage&amp;volume=105&amp;pages=286-299&amp;publication_year=2015&amp;author=Eavani%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Friston, K. J. Functional and effective connectivity in neuroimaging: a synthesis. <i>Hum. Brain Mapp.</i> <b>2</b>, 5678 (1994).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20and%20effective%20connectivity%20in%20neuroimaging%3A%20a%20synthesis&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=2&amp;pages=56-78&amp;publication_year=1994&amp;author=Friston%2CKJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Smith, S. M. et al. Functional connectomics from resting-state fMRI. <i>Trends Cogn. Sci.</i> <b>17</b>, 666682 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24238796" aria-label="PubMed reference 28">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4004765" aria-label="PubMed Central reference 28">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectomics%20from%20resting-state%20fMRI&amp;journal=Trends%20Cogn.%20Sci.&amp;volume=17&amp;pages=666-682&amp;publication_year=2013&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Betzel, R. F. &amp; Bassett, D. S. Multi-scale brain networks. <i>Neuroimage</i> <b>160</b>, 7383 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27845257" aria-label="PubMed reference 29">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-scale%20brain%20networks&amp;journal=Neuroimage&amp;volume=160&amp;pages=73-83&amp;publication_year=2017&amp;author=Betzel%2CRF&amp;author=Bassett%2CDS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. <i>Nat. Neurosci.</i> <b>18</b>, 16641671 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1ChurfI" aria-label="CAS reference 30">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26457551" aria-label="PubMed reference 30">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5008686" aria-label="PubMed Central reference 30">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectome%20fingerprinting%3A%20identifying%20individuals%20using%20patterns%20of%20brain%20connectivity&amp;journal=Nat.%20Neurosci.&amp;volume=18&amp;pages=1664-1671&amp;publication_year=2015&amp;author=Finn%2CES">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Smith, S. M. et al. A positive-negative mode of population covariation links brain connectivity, demographics and behavior. <i>Nat. Neurosci.</i> <b>18</b>, 15651567 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7vN" aria-label="CAS reference 31">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26414616" aria-label="PubMed reference 31">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4625579" aria-label="PubMed Central reference 31">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20positive-negative%20mode%20of%20population%20covariation%20links%20brain%20connectivity%2C%20demographics%20and%20behavior&amp;journal=Nat.%20Neurosci.&amp;volume=18&amp;pages=1565-1567&amp;publication_year=2015&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Nickerson, L. D., Smith, S. M., ngr, D. &amp; Beckmann, C. F. Using dual regression to investigate network shape and amplitude in functional connectivity analyses. <i>Front. Neurosci.</i> <b>11</b>, 115 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28348512" aria-label="PubMed reference 32">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5346569" aria-label="PubMed Central reference 32">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20dual%20regression%20to%20investigate%20network%20shape%20and%20amplitude%20in%20functional%20connectivity%20analyses&amp;journal=Front.%20Neurosci.&amp;volume=11&amp;publication_year=2017&amp;author=Nickerson%2CLD&amp;author=Smith%2CSM&amp;author=%C3%96ng%C3%BCr%2CD&amp;author=Beckmann%2CCF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Hutchison, R. M. et al. Dynamic functional connectivity: promise, issues, and interpretations. <i>Neuroimage</i> <b>80</b>, 360378 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23707587" aria-label="PubMed reference 33">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20functional%20connectivity%3A%20promise%2C%20issues%2C%20and%20interpretations&amp;journal=Neuroimage&amp;volume=80&amp;pages=360-378&amp;publication_year=2013&amp;author=Hutchison%2CRM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Preti, M. G., Bolton, T. A. &amp; Van De Ville, D. The dynamic functional connectome: state-of-the-art and perspectives. <i>Neuroimage</i> <b>160</b>, 4154 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28034766" aria-label="PubMed reference 34">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20dynamic%20functional%20connectome%3A%20state-of-the-art%20and%20perspectives&amp;journal=Neuroimage&amp;volume=160&amp;pages=41-54&amp;publication_year=2017&amp;author=Preti%2CMG&amp;author=Bolton%2CTA&amp;author=Ville%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Allen, E. A. et al. Tracking whole-brain connectivity dynamics in the resting state. <i>Cereb. Cortex</i> <b>24</b>, 663676 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23146964" aria-label="PubMed reference 35">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Tracking%20whole-brain%20connectivity%20dynamics%20in%20the%20resting%20state&amp;journal=Cereb.%20Cortex&amp;volume=24&amp;pages=663-676&amp;publication_year=2014&amp;author=Allen%2CEA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Vidaurre, D. et al. Spontaneous cortical activity transiently organises into frequency specific phase-coupling networks. <i>Nat. Commun.</i> <b>9</b>, 2987 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30061566" aria-label="PubMed reference 36">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6065434" aria-label="PubMed Central reference 36">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Spontaneous%20cortical%20activity%20transiently%20organises%20into%20frequency%20specific%20phase-coupling%20networks&amp;journal=Nat.%20Commun.&amp;volume=9&amp;publication_year=2018&amp;author=Vidaurre%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Zou, Q.-H. et al. An improved approach to detection of amplitude of low-frequency fluctuation (ALFF) for resting-state fMRI: fractional ALFF. <i>J. Neurosci. Methods</i> <b>172</b>, 137141 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18501969" aria-label="PubMed reference 37">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3902859" aria-label="PubMed Central reference 37">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20improved%20approach%20to%20detection%20of%20amplitude%20of%20low-frequency%20fluctuation%20%28ALFF%29%20for%20resting-state%20fMRI%3A%20fractional%20ALFF&amp;journal=J.%20Neurosci.%20Methods&amp;volume=172&amp;pages=137-141&amp;publication_year=2008&amp;author=Zou%2CQ-H">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Miller, K. L. et al. Multimodal population brain imaging in the UK Biobank prospective epidemiological study. <i>Nat. Neurosci.</i> <b>19</b>, 15231536 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFenu7nE" aria-label="CAS reference 38">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27643430" aria-label="PubMed reference 38">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5086094" aria-label="PubMed Central reference 38">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Multimodal%20population%20brain%20imaging%20in%20the%20UK%20Biobank%20prospective%20epidemiological%20study&amp;journal=Nat.%20Neurosci.&amp;volume=19&amp;pages=1523-1536&amp;publication_year=2016&amp;author=Miller%2CKL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Bijsterbosch, J. et al. Investigations into within- and between-subject resting-state amplitude variations. <i>Neuroimage</i> <b>159</b>, 5769 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28712995" aria-label="PubMed reference 39">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5678294" aria-label="PubMed Central reference 39">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Investigations%20into%20within-%20and%20between-subject%20resting-state%20amplitude%20variations&amp;journal=Neuroimage&amp;volume=159&amp;pages=57-69&amp;publication_year=2017&amp;author=Bijsterbosch%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Bijsterbosch, J. D., Beckmann, C. F., Woolrich, M. W., Smith, S. M. &amp; Harrison, S. J. The relationship between spatial configuration and functional connectivity of brain regions revisited. <i>eLife</i> <b>8</b>, e44890 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1CrsbbN" aria-label="CAS reference 40">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31066676" aria-label="PubMed reference 40">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6541435" aria-label="PubMed Central reference 40">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20relationship%20between%20spatial%20configuration%20and%20functional%20connectivity%20of%20brain%20regions%20revisited&amp;journal=eLife&amp;volume=8&amp;publication_year=2019&amp;author=Bijsterbosch%2CJD&amp;author=Beckmann%2CCF&amp;author=Woolrich%2CMW&amp;author=Smith%2CSM&amp;author=Harrison%2CSJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Kong, R. et al. Spatial topography of individual-specific cortical networks predicts human cognition, personality, and emotion. <i>Cereb. Cortex</i> <b>29</b>, 25332551 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29878084" aria-label="PubMed reference 41">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20topography%20of%20individual-specific%20cortical%20networks%20predicts%20human%20cognition%2C%20personality%2C%20and%20emotion&amp;journal=Cereb.%20Cortex&amp;volume=29&amp;pages=2533-2551&amp;publication_year=2019&amp;author=Kong%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Duff, E. P., Makin, T., Cottaar, M., Smith, S. M. &amp; Woolrich, M. W. Disambiguating brain functional connectivity. <i>Neuroimage</i> <b>173</b>, 540550 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29476911" aria-label="PubMed reference 42">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5929905" aria-label="PubMed Central reference 42">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Disambiguating%20brain%20functional%20connectivity&amp;journal=Neuroimage&amp;volume=173&amp;pages=540-550&amp;publication_year=2018&amp;author=Duff%2CEP&amp;author=Makin%2CT&amp;author=Cottaar%2CM&amp;author=Smith%2CSM&amp;author=Woolrich%2CMW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Huntenburg, J. M., Bazin, P.-L. &amp; Margulies, D. S. Large-scale gradients in human cortical organization. <i>Trends Cogn. Sci.</i> <b>22</b>, 2131 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29203085" aria-label="PubMed reference 43">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20gradients%20in%20human%20cortical%20organization&amp;journal=Trends%20Cogn.%20Sci.&amp;volume=22&amp;pages=21-31&amp;publication_year=2018&amp;author=Huntenburg%2CJM&amp;author=Bazin%2CP-L&amp;author=Margulies%2CDS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Marquand, A. F., Haak, K. V. &amp; Beckmann, C. F. Functional corticostriatal connection topographies predict goal directed behaviour in humans. <i>Nat. Hum. Behav.</i> <b>1</b>, 0146 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28804783" aria-label="PubMed reference 44">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5549843" aria-label="PubMed Central reference 44">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20corticostriatal%20connection%20topographies%20predict%20goal%20directed%20behaviour%20in%20humans&amp;journal=Nat.%20Hum.%20Behav.&amp;volume=1&amp;publication_year=2017&amp;author=Marquand%2CAF&amp;author=Haak%2CKV&amp;author=Beckmann%2CCF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. <i>Proc. Natl. Acad. Sci. USA</i> <b>113</b>, 1257412579 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ykurnM" aria-label="CAS reference 45">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27791099" aria-label="PubMed reference 45">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Situating%20the%20default-mode%20network%20along%20a%20principal%20gradient%20of%20macroscale%20cortical%20organization&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=113&amp;pages=12574-12579&amp;publication_year=2016&amp;author=Margulies%2CDS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Majeed, W. et al. Spatiotemporal dynamics of low frequency BOLD fluctuations in rats and humans. <i>Neuroimage</i> <b>54</b>, 11401150 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20728554" aria-label="PubMed reference 46">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatiotemporal%20dynamics%20of%20low%20frequency%20BOLD%20fluctuations%20in%20rats%20and%20humans&amp;journal=Neuroimage&amp;volume=54&amp;pages=1140-1150&amp;publication_year=2011&amp;author=Majeed%2CW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Woo, C.-W., Chang, L. J., Lindquist, M. A. &amp; Wager, T. D. Building better biomarkers: brain models in translational neuroimaging. <i>Nat. Neurosci.</i> <b>20</b>, 365377 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVSmsr8%3D" aria-label="CAS reference 47">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28230847" aria-label="PubMed reference 47">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5988350" aria-label="PubMed Central reference 47">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Building%20better%20biomarkers%3A%20brain%20models%20in%20translational%20neuroimaging&amp;journal=Nat.%20Neurosci.&amp;volume=20&amp;pages=365-377&amp;publication_year=2017&amp;author=Woo%2CC-W&amp;author=Chang%2CLJ&amp;author=Lindquist%2CMA&amp;author=Wager%2CTD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Bargmann, C. I. &amp; Marder, E. From the connectome to brain function. <i>Nat. Methods</i> <b>10</b>, 483490 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXosVCitrc%3D" aria-label="CAS reference 48">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23866325" aria-label="PubMed reference 48">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=From%20the%20connectome%20to%20brain%20function&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=483-490&amp;publication_year=2013&amp;author=Bargmann%2CCI&amp;author=Marder%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Jbabdi, S., Sotiropoulos, S. N. &amp; Behrens, T. E. The topographic connectome. <i>Curr. Opin. Neurobiol.</i> <b>23</b>, 207215 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXkt1egsg%3D%3D" aria-label="CAS reference 49">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23298689" aria-label="PubMed reference 49">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3622846" aria-label="PubMed Central reference 49">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20topographic%20connectome&amp;journal=Curr.%20Opin.%20Neurobiol.&amp;volume=23&amp;pages=207-215&amp;publication_year=2013&amp;author=Jbabdi%2CS&amp;author=Sotiropoulos%2CSN&amp;author=Behrens%2CTE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Paquola, C. et al. The cortical wiring scheme of hierarchical information processing. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2020.01.08.899583" data-track="click" data-track-action="external reference" data-track-label="10.1101/2020.01.08.899583">https://doi.org/10.1101/2020.01.08.899583</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Betzel, R. F. et al. The community structure of functional brain networks exhibits scale-specific patterns of inter- and intra-subject variability. <i>Neuroimage</i> <b>202</b>, 115990 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31291606" aria-label="PubMed reference 51">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20community%20structure%20of%20functional%20brain%20networks%20exhibits%20scale-specific%20patterns%20of%20inter-%20and%20intra-subject%20variability&amp;journal=Neuroimage&amp;volume=202&amp;publication_year=2019&amp;author=Betzel%2CRF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Logothetis, N. K. What we can do and what we cannot do with fMRI. <i>Nature</i> <b>453</b>, 869878 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXntVCqur0%3D" aria-label="CAS reference 52">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18548064" aria-label="PubMed reference 52">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20we%20can%20do%20and%20what%20we%20cannot%20do%20with%20fMRI&amp;journal=Nature&amp;volume=453&amp;pages=869-878&amp;publication_year=2008&amp;author=Logothetis%2CNK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Moeller, S. et al. Multiband multislice GE-EPI at 7 tesla, with 16-fold acceleration using partial parallel imaging with application to high spatial and temporal whole-brain fMRI. <i>Magn. Reson. Med.</i> <b>63</b>, 11441153 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20432285" aria-label="PubMed reference 53">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2906244" aria-label="PubMed Central reference 53">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiband%20multislice%20GE-EPI%20at%207%20tesla%2C%20with%2016-fold%20acceleration%20using%20partial%20parallel%20imaging%20with%20application%20to%20high%20spatial%20and%20temporal%20whole-brain%20fMRI&amp;journal=Magn.%20Reson.%20Med.&amp;volume=63&amp;pages=1144-1153&amp;publication_year=2010&amp;author=Moeller%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Yu, Q. et al. Comparing brain graphs in which nodes are regions of interest or independent components: A simulation study. <i>J. Neurosci. Methods</i> <b>291</b>, 6168 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28807861" aria-label="PubMed reference 54">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5610951" aria-label="PubMed Central reference 54">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparing%20brain%20graphs%20in%20which%20nodes%20are%20regions%20of%20interest%20or%20independent%20components%3A%20A%20simulation%20study&amp;journal=J.%20Neurosci.%20Methods&amp;volume=291&amp;pages=61-68&amp;publication_year=2017&amp;author=Yu%2CQ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Abraham, A. et al. Deriving reproducible biomarkers from multi-site resting-state data: An Autism-based example. <i>Neuroimage</i> <b>147</b>, 736745 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27865923" aria-label="PubMed reference 55">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Deriving%20reproducible%20biomarkers%20from%20multi-site%20resting-state%20data%3A%20An%20Autism-based%20example&amp;journal=Neuroimage&amp;volume=147&amp;pages=736-745&amp;publication_year=2017&amp;author=Abraham%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Duff, E. P. et al. Task-driven ICA feature generation for accurate and interpretable prediction using fMRI. <i>Neuroimage</i> <b>60</b>, 189203 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22227050" aria-label="PubMed reference 56">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Task-driven%20ICA%20feature%20generation%20for%20accurate%20and%20interpretable%20prediction%20using%20fMRI&amp;journal=Neuroimage&amp;volume=60&amp;pages=189-203&amp;publication_year=2012&amp;author=Duff%2CEP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Pervaiz, U., Vidaurre, D., Woolrich, M. W. &amp; Smith, S. M. Optimising network modelling methods for fMRI. <i>NeuroImage</i> <b>211</b>, 116604 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32062083" aria-label="PubMed reference 57">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7086233" aria-label="PubMed Central reference 57">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Optimising%20network%20modelling%20methods%20for%20fMRI&amp;journal=NeuroImage&amp;volume=211&amp;publication_year=2020&amp;author=Pervaiz%2CU&amp;author=Vidaurre%2CD&amp;author=Woolrich%2CMW&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Smith, S. M. et al. Network modelling methods for FMRI. <i>Neuroimage</i> <b>54</b>, 875891 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20817103" aria-label="PubMed reference 58">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20modelling%20methods%20for%20FMRI&amp;journal=Neuroimage&amp;volume=54&amp;pages=875-891&amp;publication_year=2011&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Allen, E. A., Erhardt, E. B., Wei, Y., Eichele, T. &amp; Calhoun, V. D. Capturing inter-subject variability with group independent component analysis of fMRI data: a simulation study. <i>Neuroimage</i> <b>59</b>, 41414159 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22019879" aria-label="PubMed reference 59">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Capturing%20inter-subject%20variability%20with%20group%20independent%20component%20analysis%20of%20fMRI%20data%3A%20a%20simulation%20study&amp;journal=Neuroimage&amp;volume=59&amp;pages=4141-4159&amp;publication_year=2012&amp;author=Allen%2CEA&amp;author=Erhardt%2CEB&amp;author=Wei%2CY&amp;author=Eichele%2CT&amp;author=Calhoun%2CVD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Llera, A., Wolfers, T., Mulders, P. &amp; Beckmann, C. F. Inter-individual differences in human brain structure and morphology link to variation in demographics and behavior. <i>eLife</i> <b>8</b>, e44443 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFyjt77M" aria-label="CAS reference 60">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31268418" aria-label="PubMed reference 60">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6663467" aria-label="PubMed Central reference 60">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Inter-individual%20differences%20in%20human%20brain%20structure%20and%20morphology%20link%20to%20variation%20in%20demographics%20and%20behavior&amp;journal=eLife&amp;volume=8&amp;publication_year=2019&amp;author=Llera%2CA&amp;author=Wolfers%2CT&amp;author=Mulders%2C&amp;author=Beckmann%2CCF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Smith, S. et al. Structural variability in the human brain reflects fine-grained functional architecture at the population level. <i>J. Neurosci.</i> <b>39</b>, 61366149 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVGns7jN" aria-label="CAS reference 61">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31152123" aria-label="PubMed reference 61">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6668209" aria-label="PubMed Central reference 61">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20variability%20in%20the%20human%20brain%20reflects%20fine-grained%20functional%20architecture%20at%20the%20population%20level&amp;journal=J.%20Neurosci.&amp;volume=39&amp;pages=6136-6149&amp;publication_year=2019&amp;author=Smith%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Bijsterbosch, J. D. et al. The relationship between spatial configuration and functional connectivity of brain regions. <i>eLife</i> <b>7</b>, e32992 (2018). <b>This article uncovers an important source of representational ambiguity by showing that between-participant variance in node-to-node correlations is partly driven by variability in spatial organization. Data-driven simulations are used to interrogate interactions between different elements of brain representations, and shared versus unique variance is used to compare different brain representations</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29451491" aria-label="PubMed reference 62">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5860869" aria-label="PubMed Central reference 62">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20relationship%20between%20spatial%20configuration%20and%20functional%20connectivity%20of%20brain%20regions&amp;journal=eLife&amp;volume=7&amp;publication_year=2018&amp;author=Bijsterbosch%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Tavor, I. et al. Task-free MRI predicts individual differences in brain activity during task performance. <i>Science</i> <b>352</b>, 216220 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XlsFSmtbc%3D" aria-label="CAS reference 63">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27124457" aria-label="PubMed reference 63">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6309730" aria-label="PubMed Central reference 63">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Task-free%20MRI%20predicts%20individual%20differences%20in%20brain%20activity%20during%20task%20performance&amp;journal=Science&amp;volume=352&amp;pages=216-220&amp;publication_year=2016&amp;author=Tavor%2CI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Coalson, T. S., Van Essen, D. C. &amp; Glasser, M. F. The impact of traditional neuroimaging methods on the spatial localization of cortical areas. <i>Proc. Natl. Acad. Sci. USA</i> <b>115</b>, E6356E6365 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFKhtrnP" aria-label="CAS reference 64">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29925602" aria-label="PubMed reference 64">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20traditional%20neuroimaging%20methods%20on%20the%20spatial%20localization%20of%20cortical%20areas&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=115&amp;pages=E6356-E6365&amp;publication_year=2018&amp;author=Coalson%2CTS&amp;author=Essen%2CDC&amp;author=Glasser%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Robinson, E. C. et al. MSM: a new flexible framework for multimodal surface matching. <i>Neuroimage</i> <b>100</b>, 414426 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24939340" aria-label="PubMed reference 65">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4190319" aria-label="PubMed Central reference 65">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=MSM%3A%20a%20new%20flexible%20framework%20for%20multimodal%20surface%20matching&amp;journal=Neuroimage&amp;volume=100&amp;pages=414-426&amp;publication_year=2014&amp;author=Robinson%2CEC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Braga, R. M. &amp; Buckner, R. L. Parallel interdigitated distributed networks within the individual estimated by intrinsic functional connectivity. <i>Neuron</i> <b>95</b>, 457471.e5 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1SmtbzN" aria-label="CAS reference 66">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28728026" aria-label="PubMed reference 66">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5519493" aria-label="PubMed Central reference 66">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Parallel%20interdigitated%20distributed%20networks%20within%20the%20individual%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=Neuron&amp;volume=95&amp;pages=457-471.e5&amp;publication_year=2017&amp;author=Braga%2CRM&amp;author=Buckner%2CRL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Poldrack, R. A. et al. Long-term neural and physiological phenotyping of a single human. <i>Nat. Commun.</i> <b>6</b>, 8885 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFyktrjM" aria-label="CAS reference 67">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26648521" aria-label="PubMed reference 67">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4682164" aria-label="PubMed Central reference 67">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20neural%20and%20physiological%20phenotyping%20of%20a%20single%20human&amp;journal=Nat.%20Commun.&amp;volume=6&amp;publication_year=2015&amp;author=Poldrack%2CRA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Seitzman, B. A. et al. Trait-like variants in human functional brain networks. <i>Proc. Natl. Acad. Sci. USA</i> <b>116</b>, 2285122861 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFWgu7vF" aria-label="CAS reference 68">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31611415" aria-label="PubMed reference 68">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Trait-like%20variants%20in%20human%20functional%20brain%20networks&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=116&amp;pages=22851-22861&amp;publication_year=2019&amp;author=Seitzman%2CBA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Hacker, C. D. et al. Resting state network estimation in individual subjects. <i>Neuroimage</i> <b>82</b>, 616633 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23735260" aria-label="PubMed reference 69">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3909699" aria-label="PubMed Central reference 69">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting%20state%20network%20estimation%20in%20individual%20subjects&amp;journal=Neuroimage&amp;volume=82&amp;pages=616-633&amp;publication_year=2013&amp;author=Hacker%2CCD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Gordon, E. M. et al. Precision functional mapping of individual human brains. <i>Neuron</i> <b>95</b>, 791807.e7 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1CmsLjF" aria-label="CAS reference 70">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28757305" aria-label="PubMed reference 70">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5576360" aria-label="PubMed Central reference 70">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Precision%20functional%20mapping%20of%20individual%20human%20brains&amp;journal=Neuron&amp;volume=95&amp;pages=791-807.e7&amp;publication_year=2017&amp;author=Gordon%2CEM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Harrison, S. J. et al. Modelling subject variability in the spatial and temporal characteristics of functional modes. <i>NeuroImage</i> <b>222</b>, 117226 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32771617" aria-label="PubMed reference 71">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Modelling%20subject%20variability%20in%20the%20spatial%20and%20temporal%20characteristics%20of%20functional%20modes&amp;journal=NeuroImage&amp;volume=222&amp;publication_year=2020&amp;author=Harrison%2CSJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Vanderwal, T. et al. Individual differences in functional connectivity during naturalistic viewing conditions. <i>Neuroimage</i> <b>157</b>, 521530 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28625875" aria-label="PubMed reference 72">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Individual%20differences%20in%20functional%20connectivity%20during%20naturalistic%20viewing%20conditions&amp;journal=Neuroimage&amp;volume=157&amp;pages=521-530&amp;publication_year=2017&amp;author=Vanderwal%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Guntupalli, J. S., Feilong, M. &amp; Haxby, J. V. A computational model of shared fine-scale structure in the human connectome. <i>PLOS Comput. Biol.</i> <b>14</b>, e1006120 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29664910" aria-label="PubMed reference 73">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5922579" aria-label="PubMed Central reference 73">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20computational%20model%20of%20shared%20fine-scale%20structure%20in%20the%20human%20connectome&amp;journal=PLOS%20Comput.%20Biol.&amp;volume=14&amp;publication_year=2018&amp;author=Guntupalli%2CJS&amp;author=Feilong%2CM&amp;author=Haxby%2CJV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Kieliba, P., Madugula, S., Filippini, N., Duff, E. P. &amp; Makin, T. R. Large-scale intrinsic connectivity is consistent across varying task demands. <i>PLoS One</i> <b>14</b>, e0213861 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXps1Smtbo%3D" aria-label="CAS reference 74">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30970031" aria-label="PubMed reference 74">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6457563" aria-label="PubMed Central reference 74">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20intrinsic%20connectivity%20is%20consistent%20across%20varying%20task%20demands&amp;journal=PLoS%20One&amp;volume=14&amp;publication_year=2019&amp;author=Kieliba%2CP&amp;author=Madugula%2CS&amp;author=Filippini%2CN&amp;author=Duff%2CEP&amp;author=Makin%2CTR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Horien, C., Shen, X., Scheinost, D. &amp; Constable, R. T. The individual functional connectome is unique and stable over months to years. <i>Neuroimage</i> <b>189</b>, 676687 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30721751" aria-label="PubMed reference 75">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6422733" aria-label="PubMed Central reference 75">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20individual%20functional%20connectome%20is%20unique%20and%20stable%20over%20months%20to%20years&amp;journal=Neuroimage&amp;volume=189&amp;pages=676-687&amp;publication_year=2019&amp;author=Horien%2CC&amp;author=Shen%2CX&amp;author=Scheinost%2CD&amp;author=Constable%2CRT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Gratton, C. et al. Functional brain networks are dominated by stable group and individual factors, not cognitive or daily variation. <i>Neuron</i> <b>98</b>, 439452.e5 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXnvFers70%3D" aria-label="CAS reference 76">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29673485" aria-label="PubMed reference 76">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5912345" aria-label="PubMed Central reference 76">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20brain%20networks%20are%20dominated%20by%20stable%20group%20and%20individual%20factors%2C%20not%20cognitive%20or%20daily%20variation&amp;journal=Neuron&amp;volume=98&amp;pages=439-452.e5&amp;publication_year=2018&amp;author=Gratton%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Salehi, M. et al. There is no single functional atlas even for a single individual: Functional parcel definitions change with task. <i>Neuroimage</i> <b>208</b>, 116366 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740342" aria-label="PubMed reference 77">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=There%20is%20no%20single%20functional%20atlas%20even%20for%20a%20single%20individual%3A%20Functional%20parcel%20definitions%20change%20with%20task&amp;journal=Neuroimage&amp;volume=208&amp;publication_year=2020&amp;author=Salehi%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Chen, J. E. et al. Resting-state physiological networks. <i>NeuroImage</i> <b>213</b>, 116707 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32145437" aria-label="PubMed reference 78">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting-state%20%E2%80%9Cphysiological%20networks%E2%80%9D&amp;journal=NeuroImage&amp;volume=213&amp;publication_year=2020&amp;author=Chen%2CJE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Ciric, R. et al. Benchmarking of participant-level confound regression strategies for the control of motion artifact in studies of functional connectivity. <i>Neuroimage</i> <b>154</b>, 174187 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28302591" aria-label="PubMed reference 79">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5483393" aria-label="PubMed Central reference 79">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Benchmarking%20of%20participant-level%20confound%20regression%20strategies%20for%20the%20control%20of%20motion%20artifact%20in%20studies%20of%20functional%20connectivity&amp;journal=Neuroimage&amp;volume=154&amp;pages=174-187&amp;publication_year=2017&amp;author=Ciric%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Murphy, K. &amp; Fox, M. D. Towards a consensus regarding global signal regression for resting state functional connectivity MRI. <i>Neuroimage</i> <b>154</b>, 169173 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27888059" aria-label="PubMed reference 80">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5489207" aria-label="PubMed Central reference 80">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20a%20consensus%20regarding%20global%20signal%20regression%20for%20resting%20state%20functional%20connectivity%20MRI&amp;journal=Neuroimage&amp;volume=154&amp;pages=169-173&amp;publication_year=2017&amp;author=Murphy%2CK&amp;author=Fox%2CMD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Power, J. D., Laumann, T. O., Plitt, M., Martin, A. &amp; Petersen, S. E. On global fMRI signals and simulations. <i>Trends Cogn. Sci.</i> <b>21</b>, 911913 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28939332" aria-label="PubMed reference 81">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20global%20fMRI%20signals%20and%20simulations&amp;journal=Trends%20Cogn.%20Sci.&amp;volume=21&amp;pages=911-913&amp;publication_year=2017&amp;author=Power%2CJD&amp;author=Laumann%2CTO&amp;author=Plitt%2CM&amp;author=Martin%2CA&amp;author=Petersen%2CSE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Hallquist, M. N., Hwang, K. &amp; Luna, B. The nuisance of nuisance regression: spectral misspecification in a common approach to resting-state fMRI preprocessing reintroduces noise and obscures functional connectivity. <i>Neuroimage</i> <b>82</b>, 208225 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23747457" aria-label="PubMed reference 82">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20nuisance%20of%20nuisance%20regression%3A%20spectral%20misspecification%20in%20a%20common%20approach%20to%20resting-state%20fMRI%20preprocessing%20reintroduces%20noise%20and%20obscures%20functional%20connectivity&amp;journal=Neuroimage&amp;volume=82&amp;pages=208-225&amp;publication_year=2013&amp;author=Hallquist%2CMN&amp;author=Hwang%2CK&amp;author=Luna%2CB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Smith, S. M. et al. Correspondence of the brains functional architecture during activation and rest. <i>Proc. Natl. Acad. Sci. USA</i> <b>106</b>, 1304013045 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVKru7%2FJ" aria-label="CAS reference 83">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19620724" aria-label="PubMed reference 83">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Correspondence%20of%20the%20brain%E2%80%99s%20functional%20architecture%20during%20activation%20and%20rest&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=13040-13045&amp;publication_year=2009&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Deco, G., Jirsa, V. K., Robinson, P. A., Breakspear, M. &amp; Friston, K. The dynamic brain: from spiking neurons to neural masses and cortical fields. <i>PLOS Comput. Biol.</i> <b>4</b>, e1000092 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18769680" aria-label="PubMed reference 84">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2519166" aria-label="PubMed Central reference 84">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20dynamic%20brain%3A%20from%20spiking%20neurons%20to%20neural%20masses%20and%20cortical%20fields&amp;journal=PLOS%20Comput.%20Biol.&amp;volume=4&amp;publication_year=2008&amp;author=Deco%2CG&amp;author=Jirsa%2CVK&amp;author=Robinson%2CPA&amp;author=Breakspear%2CM&amp;author=Friston%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Buibas, M. &amp; Silva, G. A. A framework for simulating and estimating the state and functional topology of complex dynamic geometric networks. <i>Neural Comput.</i> <b>23</b>, 183214 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20964542" aria-label="PubMed reference 85">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20framework%20for%20simulating%20and%20estimating%20the%20state%20and%20functional%20topology%20of%20complex%20dynamic%20geometric%20networks&amp;journal=Neural%20Comput.&amp;volume=23&amp;pages=183-214&amp;publication_year=2011&amp;author=Buibas%2CM&amp;author=Silva%2CGA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Logothetis, N. K., Pauls, J., Augath, M., Trinath, T. &amp; Oeltermann, A. Neurophysiological investigation of the basis of the fMRI signal. <i>Nature</i> <b>412</b>, 150157 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXlsFWnsLs%3D" aria-label="CAS reference 86">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11449264" aria-label="PubMed reference 86">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=Neurophysiological%20investigation%20of%20the%20basis%20of%20the%20fMRI%20signal&amp;journal=Nature&amp;volume=412&amp;pages=150-157&amp;publication_year=2001&amp;author=Logothetis%2CNK&amp;author=Pauls%2CJ&amp;author=Augath%2CM&amp;author=Trinath%2CT&amp;author=Oeltermann%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Shmuel, A. &amp; Leopold, D. A. Neuronal correlates of spontaneous fluctuations in fMRI signals in monkey visual cortex: Implications for functional connectivity at rest. <i>Hum. Brain Mapp.</i> <b>29</b>, 751761 (2008). <b>This paper used simultaneous rfMRI and intracortical recordings to show that between-region correlations measured from resting state fMRI (i.e., functional connectivity) are linked to synchronization of neuronal signals. This study is one of the few examples that link rfMRI brain representations to the underlying neurophysiology. Similar research efforts are needed to explicitly differentiate between different brain representations, as we have proposed in section Recommendations and future directions for brain representations and Fig. 4</b>.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18465799" aria-label="PubMed reference 87">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6870786" aria-label="PubMed Central reference 87">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=Neuronal%20correlates%20of%20spontaneous%20fluctuations%20in%20fMRI%20signals%20in%20monkey%20visual%20cortex%3A%20Implications%20for%20functional%20connectivity%20at%20rest&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=29&amp;pages=751-761&amp;publication_year=2008&amp;author=Shmuel%2CA&amp;author=Leopold%2CDA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Kucyi, A. et al. Intracranial electrophysiology reveals reproducible intrinsic functional connectivity within human brain networks. <i>J. Neurosci.</i> <b>38</b>, 42304242 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1Gjt73M" aria-label="CAS reference 88">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29626167" aria-label="PubMed reference 88">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5963853" aria-label="PubMed Central reference 88">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=Intracranial%20electrophysiology%20reveals%20reproducible%20intrinsic%20functional%20connectivity%20within%20human%20brain%20networks&amp;journal=J.%20Neurosci.&amp;volume=38&amp;pages=4230-4242&amp;publication_year=2018&amp;author=Kucyi%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Baillet, S. Magnetoencephalography for brain electrophysiology and imaging. <i>Nat. Neurosci.</i> <b>20</b>, 327339 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVSmsrk%3D" aria-label="CAS reference 89">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28230841" aria-label="PubMed reference 89">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=Magnetoencephalography%20for%20brain%20electrophysiology%20and%20imaging&amp;journal=Nat.%20Neurosci.&amp;volume=20&amp;pages=327-339&amp;publication_year=2017&amp;author=Baillet%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Bentley, W. J., Li, J. M., Snyder, A. Z., Raichle, M. E. &amp; Snyder, L. H. Oxygen level and LFP in task-positive and task-negative areas: bridging BOLD fMRI and electrophysiology. <i>Cereb. Cortex</i> <b>26</b>, 346357 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25385710" aria-label="PubMed reference 90">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=Oxygen%20level%20and%20LFP%20in%20task-positive%20and%20task-negative%20areas%3A%20bridging%20BOLD%20fMRI%20and%20electrophysiology&amp;journal=Cereb.%20Cortex&amp;volume=26&amp;pages=346-357&amp;publication_year=2016&amp;author=Bentley%2CWJ&amp;author=Li%2CJM&amp;author=Snyder%2CAZ&amp;author=Raichle%2CME&amp;author=Snyder%2CLH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Schlvinck, M. L., Maier, A., Ye, F. Q., Duyn, J. H. &amp; Leopold, D. A. Neural basis of global resting-state fMRI activity. <i>Proc. Natl. Acad. Sci. USA</i> <b>107</b>, 1023810243 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20439733" aria-label="PubMed reference 91">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=Neural%20basis%20of%20global%20resting-state%20fMRI%20activity&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=10238-10243&amp;publication_year=2010&amp;author=Sch%C3%B6lvinck%2CML&amp;author=Maier%2CA&amp;author=Ye%2CFQ&amp;author=Duyn%2CJH&amp;author=Leopold%2CDA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Vidaurre, D. et al. Discovering dynamic brain networks from big data in rest and task. <i>Neuroimage</i> <b>180</b>, 646656 (2018). Pt B.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28669905" aria-label="PubMed reference 92">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138951" aria-label="PubMed Central reference 92">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=Discovering%20dynamic%20brain%20networks%20from%20big%20data%20in%20rest%20and%20task&amp;journal=Neuroimage&amp;volume=180&amp;pages=646-656&amp;publication_year=2018&amp;author=Vidaurre%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Van Essen, D. C. &amp; Maunsell, J. H. R. Hierarchical organization and functional streams in the visual cortex. <i>Trends Neurosci.</i> <b>6</b>, 370375 (1983).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=Hierarchical%20organization%20and%20functional%20streams%20in%20the%20visual%20cortex&amp;journal=Trends%20Neurosci.&amp;volume=6&amp;pages=370-375&amp;publication_year=1983&amp;author=Essen%2CDC&amp;author=Maunsell%2CJHR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Biswal, B., Yetkin, F. Z., Haughton, V. M. &amp; Hyde, J. S. Functional connectivity in the motor cortex of resting human brain using echo-planar MRI. <i>Magn. Reson. Med.</i> <b>34</b>, 537541 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK287gslKrsg%3D%3D" aria-label="CAS reference 94">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8524021" aria-label="PubMed reference 94">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectivity%20in%20the%20motor%20cortex%20of%20resting%20human%20brain%20using%20echo-planar%20MRI&amp;journal=Magn.%20Reson.%20Med.&amp;volume=34&amp;pages=537-541&amp;publication_year=1995&amp;author=Biswal%2CB&amp;author=Yetkin%2CFZ&amp;author=Haughton%2CVM&amp;author=Hyde%2CJS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Medaglia, J. D., Lynall, M.-E. &amp; Bassett, D. S. Cognitive network neuroscience. <i>J. Cogn. Neurosci.</i> <b>27</b>, 14711491 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25803596" aria-label="PubMed reference 95">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4854276" aria-label="PubMed Central reference 95">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Cognitive%20network%20neuroscience&amp;journal=J.%20Cogn.%20Neurosci.&amp;volume=27&amp;pages=1471-1491&amp;publication_year=2015&amp;author=Medaglia%2CJD&amp;author=Lynall%2CM-E&amp;author=Bassett%2CDS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Park, H.-J., Friston, K. J., Pae, C., Park, B. &amp; Razi, A. Dynamic effective connectivity in resting state fMRI. <i>Neuroimage</i> <b>180</b>, 594608 (2018). Pt B.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29158202" aria-label="PubMed reference 96">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138953" aria-label="PubMed Central reference 96">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20effective%20connectivity%20in%20resting%20state%20fMRI&amp;journal=Neuroimage&amp;volume=180&amp;pages=594-608&amp;publication_year=2018&amp;author=Park%2CH-J&amp;author=Friston%2CKJ&amp;author=Pae%2CC&amp;author=Park%2CB&amp;author=Razi%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Patel, R. S., Bowman, F. D. &amp; Rilling, J. K. A Bayesian approach to determining connectivity of the human brain. <i>Hum. Brain Mapp.</i> <b>27</b>, 267276 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16092131" aria-label="PubMed reference 97">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 97" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20Bayesian%20approach%20to%20determining%20connectivity%20of%20the%20human%20brain&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=27&amp;pages=267-276&amp;publication_year=2006&amp;author=Patel%2CRS&amp;author=Bowman%2CFD&amp;author=Rilling%2CJK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Hyvrinen, A. &amp; Smith, S. M. Pairwise likelihood ratios for estimation of non-Gaussian structural equation models. <i>J. Mach. Learn. Res.</i> <b>14</b>, 111152 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31695580" aria-label="PubMed reference 98">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6834441" aria-label="PubMed Central reference 98">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Pairwise%20likelihood%20ratios%20for%20estimation%20of%20non-Gaussian%20structural%20equation%20models&amp;journal=J.%20Mach.%20Learn.%20Res.&amp;volume=14&amp;pages=111-152&amp;publication_year=2013&amp;author=Hyv%C3%A4rinen%2CA&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Mumford, J. A. &amp; Ramsey, J. D. Bayesian networks for fMRI: a primer. <i>Neuroimage</i> <b>86</b>, 573582 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24140939" aria-label="PubMed reference 99">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 99" href="http://scholar.google.com/scholar_lookup?&amp;title=Bayesian%20networks%20for%20fMRI%3A%20a%20primer&amp;journal=Neuroimage&amp;volume=86&amp;pages=573-582&amp;publication_year=2014&amp;author=Mumford%2CJA&amp;author=Ramsey%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100."><p class="c-article-references__text" id="ref-CR100">Schwab, S. et al. Directed functional connectivity using dynamic graphical models. <i>Neuroimage</i> <b>175</b>, 340353 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29625233" aria-label="PubMed reference 100">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6153304" aria-label="PubMed Central reference 100">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 100" href="http://scholar.google.com/scholar_lookup?&amp;title=Directed%20functional%20connectivity%20using%20dynamic%20graphical%20models&amp;journal=Neuroimage&amp;volume=175&amp;pages=340-353&amp;publication_year=2018&amp;author=Schwab%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101."><p class="c-article-references__text" id="ref-CR101">Sala-Llonch, R., Smith, S. M., Woolrich, M. &amp; Duff, E. P. Spatial parcellations, spectral filtering, and connectivity measures in fMRI: Optimizing for discrimination. <i>Hum. Brain Mapp.</i> <b>40</b>, 407419 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30259597" aria-label="PubMed reference 101">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 101" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20parcellations%2C%20spectral%20filtering%2C%20and%20connectivity%20measures%20in%20fMRI%3A%20Optimizing%20for%20discrimination&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=40&amp;pages=407-419&amp;publication_year=2019&amp;author=Sala-Llonch%2CR&amp;author=Smith%2CSM&amp;author=Woolrich%2CM&amp;author=Duff%2CEP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102."><p class="c-article-references__text" id="ref-CR102">Noble, S., Scheinost, D. &amp; Constable, R. T. A decade of test-retest reliability of functional connectivity: a systematic review and meta-analysis. <i>Neuroimage</i> <b>203</b>, 116157 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31494250" aria-label="PubMed reference 102">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 102" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20decade%20of%20test-retest%20reliability%20of%20functional%20connectivity%3A%20a%20systematic%20review%20and%20meta-analysis&amp;journal=Neuroimage&amp;volume=203&amp;publication_year=2019&amp;author=Noble%2CS&amp;author=Scheinost%2CD&amp;author=Constable%2CRT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103."><p class="c-article-references__text" id="ref-CR103">Burgess, G. C. et al. Evaluation of denoising strategies to address motion-correlated artifacts in resting-state functional magnetic resonance imaging data from the Human Connectome Project. <i>Brain Connect.</i> <b>6</b>, 669680 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27571276" aria-label="PubMed reference 103">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5105353" aria-label="PubMed Central reference 103">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 103" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluation%20of%20denoising%20strategies%20to%20address%20motion-correlated%20artifacts%20in%20resting-state%20functional%20magnetic%20resonance%20imaging%20data%20from%20the%20Human%20Connectome%20Project&amp;journal=Brain%20Connect.&amp;volume=6&amp;pages=669-680&amp;publication_year=2016&amp;author=Burgess%2CGC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="104."><p class="c-article-references__text" id="ref-CR104">Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 105124 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23668970" aria-label="PubMed reference 104">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3720813" aria-label="PubMed Central reference 104">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 104" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20minimal%20preprocessing%20pipelines%20for%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;volume=80&amp;pages=105-124&amp;publication_year=2013&amp;author=Glasser%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="105."><p class="c-article-references__text" id="ref-CR105">Esteban, O. et al. fMRIPrep: a robust preprocessing pipeline for functional MRI. <i>Nat. Methods</i> <b>16</b>, 111116 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhurnN" aria-label="CAS reference 105">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30532080" aria-label="PubMed reference 105">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 105" href="http://scholar.google.com/scholar_lookup?&amp;title=fMRIPrep%3A%20a%20robust%20preprocessing%20pipeline%20for%20functional%20MRI&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=111-116&amp;publication_year=2019&amp;author=Esteban%2CO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="106."><p class="c-article-references__text" id="ref-CR106">Yu, M. et al. Statistical harmonization corrects site effects in functional connectivity measurements from multi-site fMRI data. <i>Hum. Brain Mapp.</i> <b>39</b>, 42134227 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29962049" aria-label="PubMed reference 106">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6179920" aria-label="PubMed Central reference 106">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 106" href="http://scholar.google.com/scholar_lookup?&amp;title=Statistical%20harmonization%20corrects%20site%20effects%20in%20functional%20connectivity%20measurements%20from%20multi-site%20fMRI%20data&amp;journal=Hum.%20Brain%20Mapp.&amp;volume=39&amp;pages=4213-4227&amp;publication_year=2018&amp;author=Yu%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="107."><p class="c-article-references__text" id="ref-CR107">Power, J. D., Plitt, M., Laumann, T. O. &amp; Martin, A. Sources and implications of whole-brain fMRI signals in humans. <i>Neuroimage</i> <b>146</b>, 609625 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27751941" aria-label="PubMed reference 107">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 107" href="http://scholar.google.com/scholar_lookup?&amp;title=Sources%20and%20implications%20of%20whole-brain%20fMRI%20signals%20in%20humans&amp;journal=Neuroimage&amp;volume=146&amp;pages=609-625&amp;publication_year=2017&amp;author=Power%2CJD&amp;author=Plitt%2CM&amp;author=Laumann%2CTO&amp;author=Martin%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="108."><p class="c-article-references__text" id="ref-CR108">Saad, Z. S. et al. Trouble at rest: how correlation patterns and group differences become distorted after global signal regression. <i>Brain Connect.</i> <b>2</b>, 2532 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22432927" aria-label="PubMed reference 108">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3484684" aria-label="PubMed Central reference 108">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 108" href="http://scholar.google.com/scholar_lookup?&amp;title=Trouble%20at%20rest%3A%20how%20correlation%20patterns%20and%20group%20differences%20become%20distorted%20after%20global%20signal%20regression&amp;journal=Brain%20Connect.&amp;volume=2&amp;pages=25-32&amp;publication_year=2012&amp;author=Saad%2CZS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="109."><p class="c-article-references__text" id="ref-CR109">Glover, G. H., Li, T. Q. &amp; Ress, D. Image-based method for retrospective correction of physiological motion effects in fMRI: RETROICOR. <i>Magn. Reson. Med.</i> <b>44</b>, 162167 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3cvntl2itw%3D%3D" aria-label="CAS reference 109">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10893535" aria-label="PubMed reference 109">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 109" href="http://scholar.google.com/scholar_lookup?&amp;title=Image-based%20method%20for%20retrospective%20correction%20of%20physiological%20motion%20effects%20in%20fMRI%3A%20RETROICOR&amp;journal=Magn.%20Reson.%20Med.&amp;volume=44&amp;pages=162-167&amp;publication_year=2000&amp;author=Glover%2CGH&amp;author=Li%2CTQ&amp;author=Ress%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="110."><p class="c-article-references__text" id="ref-CR110">Glasser, M. F. et al. Using temporal ICA to selectively remove global noise while preserving global signal in functional MRI data. <i>Neuroimage</i> <b>181</b>, 692717 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29753843" aria-label="PubMed reference 110">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6237431" aria-label="PubMed Central reference 110">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 110" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20temporal%20ICA%20to%20selectively%20remove%20global%20noise%20while%20preserving%20global%20signal%20in%20functional%20MRI%20data&amp;journal=Neuroimage&amp;volume=181&amp;pages=692-717&amp;publication_year=2018&amp;author=Glasser%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="111."><p class="c-article-references__text" id="ref-CR111">Dinga, R. et al. Evaluating the evidence for biotypes of depression: methodological replication and extension of Drysdale et al. (2017). <i>Neuroimage Clin.</i> <b>22</b>, 101796 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30935858" aria-label="PubMed reference 111">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6543446" aria-label="PubMed Central reference 111">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 111" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluating%20the%20evidence%20for%20biotypes%20of%20depression%3A%20methodological%20replication%20and%20extension%20of%20Drysdale%20et%20al.%20%282017%29&amp;journal=Neuroimage%20Clin.&amp;volume=22&amp;publication_year=2019&amp;author=Dinga%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="112."><p class="c-article-references__text" id="ref-CR112">Varoquaux, G. et al. Assessing and tuning brain decoders: cross-validation, caveats, and guidelines. <i>Neuroimage</i> <b>145</b>, 166179 (2017). Pt B.</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27989847" aria-label="PubMed reference 112">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 112" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessing%20and%20tuning%20brain%20decoders%3A%20cross-validation%2C%20caveats%2C%20and%20guidelines&amp;journal=Neuroimage&amp;volume=145&amp;pages=166-179&amp;publication_year=2017&amp;author=Varoquaux%2CG">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="113."><p class="c-article-references__text" id="ref-CR113">Eklund, A., Nichols, T. E. &amp; Knutsson, H. Cluster failure: why fMRI inferences for spatial extent have inflated false-positive rates. <i>Proc. Natl. Acad. Sci. USA</i> <b>113</b>, 79007905 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFentrvN" aria-label="CAS reference 113">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27357684" aria-label="PubMed reference 113">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 113" href="http://scholar.google.com/scholar_lookup?&amp;title=Cluster%20failure%3A%20why%20fMRI%20inferences%20for%20spatial%20extent%20have%20inflated%20false-positive%20rates&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=113&amp;pages=7900-7905&amp;publication_year=2016&amp;author=Eklund%2CA&amp;author=Nichols%2CTE&amp;author=Knutsson%2CH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="114."><p class="c-article-references__text" id="ref-CR114">Arslan, S. et al. Human brain mapping: a systematic comparison of parcellation methods for the human cerebral cortex. <i>Neuroimage</i> <b>170</b>, 530 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28412442" aria-label="PubMed reference 114">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 114" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20brain%20mapping%3A%20a%20systematic%20comparison%20of%20parcellation%20methods%20for%20the%20human%20cerebral%20cortex&amp;journal=Neuroimage&amp;volume=170&amp;pages=5-30&amp;publication_year=2018&amp;author=Arslan%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="115."><p class="c-article-references__text" id="ref-CR115">Poldrack, R. A. et al. Scanning the horizon: towards transparent and reproducible neuroimaging research. <i>Nat. Rev. Neurosci.</i> <b>18</b>, 115126 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjs1KjsA%3D%3D" aria-label="CAS reference 115">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28053326" aria-label="PubMed reference 115">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6910649" aria-label="PubMed Central reference 115">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 115" href="http://scholar.google.com/scholar_lookup?&amp;title=Scanning%20the%20horizon%3A%20towards%20transparent%20and%20reproducible%20neuroimaging%20research&amp;journal=Nat.%20Rev.%20Neurosci.&amp;volume=18&amp;pages=115-126&amp;publication_year=2017&amp;author=Poldrack%2CRA">
                    Google Scholar</a>
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-020-00726-z?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>E.P.D. was supported by the SSNAP Support for Sick and Newborn Infants and their Parents Medical Research Fund (University of Oxford Excellence Fellowship). S.J.H. was supported by grant #2017-403 of the Strategic Focal Area Personalized Health and Related Technologies (PHRT) of the ETH Domain. S.S. is supported by a Wellcome Trust Strategic Award 098369/Z/12/Z and a Wellcome Trust Collaborative Award 215573/Z/19/Z. M.W. is supported by the NIHR Oxford Health Biomedical Research Centre and by the Wellcome Trust (106183/Z/14/Z and 215573/Z/19/Z). We thank M. Glasser for his helpful comments and discussions in relation to this article.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Mallinckrodt Institute of Radiology, Washington University in St Louis, Saint Louis, MO, USA</p><p class="c-article-author-affiliation__authors-list">Janine Bijsterbosch</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Centre for Functional MRI of the Brain (FMRIB), Wellcome Centre for Integrative Neuroimaging, Nuffield Department of Clinical Neurosciences, University of Oxford. John Radcliffe Hospital, Oxford, UK</p><p class="c-article-author-affiliation__authors-list">Janine Bijsterbosch,Samuel J. Harrison,Saad Jbabdi,Stephen Smith&amp;Eugene P. Duff</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Translational Neuromodeling Unit, University of Zurich &amp; ETH Zurich, Zurich, Switzerland</p><p class="c-article-author-affiliation__authors-list">Samuel J. Harrison</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Oxford Centre for Human Brain Activity (OHBA), Wellcome Centre for Integrative Neuroimaging, Department of Psychiatry, University of Oxford, Warneford Hospital, Oxford, UK</p><p class="c-article-author-affiliation__authors-list">Mark Woolrich</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Donders Institute and Department of Cognitive Neurosciences, Radboud University Medical Centre, Nijmegen, The Netherlands</p><p class="c-article-author-affiliation__authors-list">Christian Beckmann</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Department of Paediatrics, University of Oxford, John Radcliffe Hospital, Oxford, UK</p><p class="c-article-author-affiliation__authors-list">Eugene P. Duff</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Janine-Bijsterbosch-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Janine Bijsterbosch</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Janine%20Bijsterbosch" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Janine%20Bijsterbosch" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Janine%20Bijsterbosch%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Samuel_J_-Harrison-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Samuel J. Harrison</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Samuel%20J.%20Harrison" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Samuel%20J.%20Harrison" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Samuel%20J.%20Harrison%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Saad-Jbabdi-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Saad Jbabdi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Saad%20Jbabdi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Saad%20Jbabdi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Saad%20Jbabdi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Mark-Woolrich-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Mark Woolrich</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Mark%20Woolrich" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mark%20Woolrich" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mark%20Woolrich%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christian-Beckmann-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Christian Beckmann</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Christian%20Beckmann" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christian%20Beckmann" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christian%20Beckmann%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stephen-Smith-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Stephen Smith</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stephen%20Smith" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stephen%20Smith" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stephen%20Smith%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Eugene_P_-Duff-Aff2-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Eugene P. Duff</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Eugene%20P.%20Duff" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Eugene%20P.%20Duff" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Eugene%20P.%20Duff%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>J.B. and E.P.D. conceived of the topic and structure for this article. J.B. wrote the manuscript with input from E.P.D. All authors took part in extensive discussions to refine the arguments presented in this manuscript, and all authors commented on the final manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:Janine.Bijsterbosch@wustl.edu">Janine Bijsterbosch</a> or <a id="corresp-c2" href="mailto:eugene.duff@ndcn.ox.ac.uk">Eugene P. Duff</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">

<h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
<p>The authors declare that they have no conflicts of interest.</p>

</div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> <i>Nature Neuroscience</i> thanks Finnegan Calabro, Thomas Yeo and the other, anonymous, reviewer for their contribution to the peer review of this work.</p><p><b>Publishers note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Challenges%20and%20future%20directions%20for%20representations%20of%20functional%20brain%20organization&amp;author=Janine%20Bijsterbosch%20et%20al&amp;contentID=10.1038%2Fs41593-020-00726-z&amp;copyright=Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2020-10-26&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-020-00726-z" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-020-00726-z" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Bijsterbosch, J., Harrison, S.J., Jbabdi, S. <i>et al.</i> Challenges and future directions for representations of functional brain organization.
                    <i>Nat Neurosci</i> <b>23</b>, 14841495 (2020). https://doi.org/10.1038/s41593-020-00726-z</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-020-00726-z?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-05-04">04 May 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-09-16">16 September 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-10-26">26 October 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-12">December 2020</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-020-00726-z</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Assortative mixing in micro-architecturally annotated brain connectomes" href="https://doi.org/10.1038/s41467-023-38585-4">
                                        Assortative mixing in micro-architecturally annotated brain connectomes
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Vincent Bazinet</li><li>Justine Y. Hansen</li><li>Bratislav Misic</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Biological subtyping of psychiatric syndromes as a pathway for advances in drug discovery and personalized medicine" href="https://doi.org/10.1038/s44220-023-00019-x">
                                        Biological subtyping of psychiatric syndromes as a pathway for advances in drug discovery and personalized medicine
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Wenjing Zhang</li><li>John A. Sweeney</li><li>Su Lui</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Mental Health</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:The abilities of movie-watching functional connectivity in individual identifications and individualized predictions" href="https://doi.org/10.1007/s11682-023-00785-3">
                                        The abilities of movie-watching functional connectivity in individual identifications and individualized predictions
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Yun Guan</li><li>Hao Ma</li><li>Lixia Tian</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Brain Imaging and Behavior</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Meta-analytic evidence that mindfulness training alters resting state default mode network connectivity" href="https://doi.org/10.1038/s41598-022-15195-6">
                                        Meta-analytic evidence that mindfulness training alters resting state default mode network connectivity
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Hadley Rahrig</li><li>David R. Vago</li><li>Kirk Warren Brown</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Reports</i> (2022)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:The normative modeling framework for computational psychiatry" href="https://doi.org/10.1038/s41596-022-00696-5">
                                        The normative modeling framework for computational psychiatry
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Saige Rutherford</li><li>Seyed Mostafa Kia</li><li>Andre F. Marquand</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Protocols</i> (2022)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00726-z.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-020-00726-z.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                        <div class="c-article-associated-content__collection focus u-mb-24">
                            <section>
                                <p class="c-article-associated-content__collection-label u-sans-serif u-text-bold u-mb-8">Focus</p>
                                <h3 class="c-article-associated-content__collection-title u-h3 u-mb-8">
                                    <a href="https://www.nature.com/collections/hhgegahcbd"
                                       class="u-link-inherit"
                                       data-track="click"
                                       data-track-action="view focus"
                                       data-track-category="associated content"
                                       data-track-label="focus"
                                       data-test="collection-link">Focus on neuroscience methods</a>
                                </h3>
                            </section>
                        </div>
                    
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "focus";
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-020-00726-z;doi=10.1038/s41593-020-00726-z;techmeta=36,59;subjmeta=1627,1647,245,378,53,631,692;kwrd=Biomarkers,Functional+magnetic+resonance+imaging,Neuroscience">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=-2007430185&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-020-00726-z%26doi%3D10.1038/s41593-020-00726-z%26techmeta%3D36,59%26subjmeta%3D1627,1647,245,378,53,631,692%26kwrd%3DBiomarkers,Functional+magnetic+resonance+imaging,Neuroscience">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=-2007430185&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-020-00726-z%26doi%3D10.1038/s41593-020-00726-z%26techmeta%3D36,59%26subjmeta%3D1627,1647,245,378,53,631,692%26kwrd%3DBiomarkers,Functional+magnetic+resonance+imaging,Neuroscience"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-transres"
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: TranslationalResearch">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: TranslationalResearch">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing: Translational Research" src="/static/images/logos/nature-briefing-logo-transres-white-1245a3c374.svg" width="213" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Translational Research</em> newsletter  top stories in biotechnology, drug discovery and pharma.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/translational_research" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: TranslationalResearch">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="TransResBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="TransResBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:translational_research" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: TranslationalResearch">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get what matters in translational research, free to your inbox weekly.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="/briefing/translational-research/?brieferEntryPoint=TransResBriefingBanner">Sign up for Nature Briefing: Translational Research
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-020-00726-z&amp;format=js&amp;last_modified=2020-10-26" async></script>
<img src="/v39e5677/article/s41593-020-00726-z" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>