<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>X-chromosome influences on neuroanatomical variation in humans | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"behavioural-genetics;brain;development-of-the-nervous-system;genetic-association-study;genetics-of-the-nervous-system","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41593-021-00890-w"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Travis T. Mallard","Siyuan Liu","Jakob Seidlitz","Zhiwei Ma","Dustin Moraczewski","Adam Thomas","Armin Raznahan"],"publishedAt":1626912000,"publishedAtString":"2021-07-22","title":"X-chromosome influences on neuroanatomical variation in humans","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"24","issue":"9"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"X-chromosome influences on neuroanatomical variation in humans","description":"The X-chromosome has long been hypothesized to have a disproportionate influence on the brain based on its enrichment for genes that are expressed in the brain and associated with intellectual disability. Here, we verify this hypothesis through partitioned heritability analysis of X-chromosome influences (XIs) on human brain anatomy in 32,256 individuals from the UK Biobank. We first establish evidence for dosage compensation in XIs on brain anatomy—reflecting larger XIs in males compared to females, which correlate with regional sex-biases in neuroanatomical variance. XIs are significantly larger than would be predicted from X-chromosome size for the relative surface area of cortical systems supporting attention, decision-making and motor control. Follow-up association analyses implicate X-linked genes with pleiotropic effects on cognition. Our study reveals a privileged role for the X-chromosome in human neurodevelopment and urges greater inclusion of this chromosome in future genome-wide association studies. Mallard et al. study the often overlooked X-chromosome’s influences on the human brain. They find that X-chromosome influences on cortical surface area are sex biased and concentrated in specific cortical systems.","datePublished":"2021-07-22T00:00:00Z","dateModified":"2021-07-22T00:00:00Z","pageStart":"1216","pageEnd":"1224","sameAs":"https://doi.org/10.1038/s41593-021-00890-w","keywords":["Behavioural genetics","Brain","Development of the nervous system","Genetic association study","Genetics of the nervous system","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig4_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"24","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Travis T. Mallard","url":"http://orcid.org/0000-0002-3265-3001","affiliation":[{"name":"University of Texas at Austin","address":{"name":"Department of Psychology, University of Texas at Austin, Austin, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"National Institute of Mental Health Intramural Research Program","address":{"name":"Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Siyuan Liu","url":"http://orcid.org/0000-0003-3661-6248","affiliation":[{"name":"National Institute of Mental Health Intramural Research Program","address":{"name":"Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jakob Seidlitz","url":"http://orcid.org/0000-0002-8164-7476","affiliation":[{"name":"National Institute of Mental Health Intramural Research Program","address":{"name":"Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Zhiwei Ma","url":"http://orcid.org/0000-0003-2928-402X","affiliation":[{"name":"National Institute of Neurological Disorders and Stroke, National Institutes of Health","address":{"name":"Laboratory of Functional and Molecular Imaging, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Dustin Moraczewski","affiliation":[{"name":"National Institute of Mental Health","address":{"name":"Data Science and Sharing Team, National Institute of Mental Health, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Adam Thomas","url":"http://orcid.org/0000-0002-2850-1419","affiliation":[{"name":"National Institute of Mental Health","address":{"name":"Data Science and Sharing Team, National Institute of Mental Health, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Armin Raznahan","url":"http://orcid.org/0000-0002-5622-1190","affiliation":[{"name":"National Institute of Mental Health Intramural Research Program","address":{"name":"Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"raznahana@mail.nih.gov","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-021-00890-w">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="X-chromosome influences on neuroanatomical variation in humans"/>
    <meta name="dc.source" content="Nature Neuroscience 2021 24:9"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2021-07-22"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2021 This is a U.S. government work and not under copyright protection in the U.S.; foreign copyright protection may apply"/>
    <meta name="dc.rights" content="2021 This is a U.S. government work and not under copyright protection in the U.S.; foreign copyright protection may apply"/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="The X-chromosome has long been hypothesized to have a disproportionate influence on the brain based on its enrichment for genes that are expressed in the brain and associated with intellectual disability. Here, we verify this hypothesis through partitioned heritability analysis of X-chromosome influences (XIs) on human brain anatomy in 32,256 individuals from the UK Biobank. We first establish evidence for dosage compensation in XIs on brain anatomy&#8212;reflecting larger XIs in males compared to females, which correlate with regional sex-biases in neuroanatomical variance. XIs are significantly larger than would be predicted from X-chromosome size for the relative surface area of cortical systems supporting attention, decision-making and motor control. Follow-up association analyses implicate X-linked genes with pleiotropic effects on cognition. Our study reveals a privileged role for the X-chromosome in human neurodevelopment and urges greater inclusion of this chromosome in future genome-wide association studies. Mallard et al. study the often overlooked X-chromosome&#8217;s influences on the human brain. They find that X-chromosome influences on cortical surface area are sex biased and concentrated in specific cortical systems."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2021-07-22"/>
    <meta name="prism.volume" content="24"/>
    <meta name="prism.number" content="9"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="1216"/>
    <meta name="prism.endingPage" content="1224"/>
    <meta name="prism.copyright" content="2021 This is a U.S. government work and not under copyright protection in the U.S.; foreign copyright protection may apply"/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-021-00890-w"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-021-00890-w"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-021-00890-w.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-021-00890-w"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="X-chromosome influences on neuroanatomical variation in humans"/>
    <meta name="citation_volume" content="24"/>
    <meta name="citation_issue" content="9"/>
    <meta name="citation_publication_date" content="2021/09"/>
    <meta name="citation_online_date" content="2021/07/22"/>
    <meta name="citation_firstpage" content="1216"/>
    <meta name="citation_lastpage" content="1224"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-021-00890-w"/>
    <meta name="DOI" content="10.1038/s41593-021-00890-w"/>
    <meta name="size" content="326195"/>
    <meta name="citation_doi" content="10.1038/s41593-021-00890-w"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-021-00890-w&amp;api_key="/>
    <meta name="description" content="The X-chromosome has long been hypothesized to have a disproportionate influence on the brain based on its enrichment for genes that are expressed in the brain and associated with intellectual disability. Here, we verify this hypothesis through partitioned heritability analysis of X-chromosome influences (XIs) on human brain anatomy in 32,256 individuals from the UK Biobank. We first establish evidence for dosage compensation in XIs on brain anatomy&#8212;reflecting larger XIs in males compared to females, which correlate with regional sex-biases in neuroanatomical variance. XIs are significantly larger than would be predicted from X-chromosome size for the relative surface area of cortical systems supporting attention, decision-making and motor control. Follow-up association analyses implicate X-linked genes with pleiotropic effects on cognition. Our study reveals a privileged role for the X-chromosome in human neurodevelopment and urges greater inclusion of this chromosome in future genome-wide association studies. Mallard et al. study the often overlooked X-chromosome&#8217;s influences on the human brain. They find that X-chromosome influences on cortical surface area are sex biased and concentrated in specific cortical systems."/>
    <meta name="dc.creator" content="Mallard, Travis T."/>
    <meta name="dc.creator" content="Liu, Siyuan"/>
    <meta name="dc.creator" content="Seidlitz, Jakob"/>
    <meta name="dc.creator" content="Ma, Zhiwei"/>
    <meta name="dc.creator" content="Moraczewski, Dustin"/>
    <meta name="dc.creator" content="Thomas, Adam"/>
    <meta name="dc.creator" content="Raznahan, Armin"/>
    <meta name="dc.subject" content="Behavioural genetics"/>
    <meta name="dc.subject" content="Brain"/>
    <meta name="dc.subject" content="Development of the nervous system"/>
    <meta name="dc.subject" content="Genetic association study"/>
    <meta name="dc.subject" content="Genetics of the nervous system"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=The genetic architecture of the human cerebral cortex; citation_author=KL Grasby; citation_volume=367; citation_publication_date=2020; citation_pages=eaay6690; citation_doi=10.1126/science.aay6690; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Polygenic architecture of human neuroanatomical diversity; citation_author=A Biton; citation_volume=30; citation_publication_date=2020; citation_pages=2307-2320; citation_doi=10.1093/cercor/bhz241; citation_id=CR2"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Genome-wide association analysis of 19,629 individuals identifies variants influencing regional brain volumes and refines their genetic co-architecture with cognitive and mental health traits; citation_author=B Zhao; citation_volume=51; citation_publication_date=2019; citation_pages=1637-1644; citation_doi=10.1038/s41588-019-0516-6; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=An expanded set of genome-wide association studies of brain imaging phenotypes in UK Biobank; citation_author=SM Smith; citation_volume=24; citation_publication_date=2021; citation_pages=737-745; citation_doi=10.1038/s41593-021-00826-4; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=eXclusion: toward integrating the X chromosome in genome-wide association analyses; citation_author=AL Wise, L Gyi, TA Manolio; citation_volume=92; citation_publication_date=2013; citation_pages=643-647; citation_doi=10.1016/j.ajhg.2013.03.017; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Gene action in the X-chromosome of the mouse (Mus musculus L.); citation_author=MF Lyon; citation_volume=190; citation_publication_date=1961; citation_pages=372-373; citation_doi=10.1038/190372a0; citation_id=CR6"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Dosage compensation of the active X chromosome in mammals; citation_author=DK Nguyen, CM Disteche; citation_volume=38; citation_publication_date=2006; citation_pages=47-53; citation_doi=10.1038/ng1705; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Genet.; citation_title=A high density of X-linked genes for general cognitive ability: a run-away process shaping human evolution?; citation_author=U Zechner; citation_volume=17; citation_publication_date=2001; citation_pages=697-701; citation_doi=10.1016/S0168-9525(01)02446-5; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Globally divergent but locally convergent X- and Y-chromosome influences on cortical development; citation_author=A Raznahan; citation_volume=26; citation_publication_date=2016; citation_pages=70-79; citation_doi=10.1093/cercor/bhu174; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Genome partitioning of genetic variation for complex traits using common SNPs; citation_author=J Yang; citation_volume=43; citation_publication_date=2011; citation_pages=519-525; citation_doi=10.1038/ng.823; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=GCTA: a tool for genome-wide complex trait analysis; citation_author=J Yang, SH Lee, ME Goddard, PM Visscher; citation_volume=88; citation_publication_date=2011; citation_pages=76-82; citation_doi=10.1016/j.ajhg.2010.11.011; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain; citation_author=B Fischl; citation_volume=33; citation_publication_date=2002; citation_pages=341-355; citation_doi=10.1016/S0896-6273(02)00569-X; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Automatically parcellating the human cerebral cortex; citation_author=B Fischl; citation_volume=14; citation_publication_date=2004; citation_pages=11-22; citation_doi=10.1093/cercor/bhg087; citation_id=CR13"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A multi-modal parcellation of human cerebral cortex; citation_author=MF Glasser; citation_volume=536; citation_publication_date=2016; citation_pages=171-178; citation_doi=10.1038/nature18933; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Studying neuroanatomy using MRI; citation_author=JP Lerch; citation_volume=20; citation_publication_date=2017; citation_pages=314-326; citation_doi=10.1038/nn.4501; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Genetic correlations and genome-wide associations of cortical structure in general population samples of 22,824 adults; citation_author=E Hofer; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-18367-y; citation_id=CR16"/>
    <meta name="citation_reference" content="Matoba, N., Love, M. I. &amp; Stein, J. L. Evaluating brain structure traits as endophenotypes using polygenicity and discoverability. Hum. Brain Mapp. 
                  https://doi.org/10.1002/hbm.25257
                  
                 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Sex differences in variability of brain structure across the lifespan; citation_author=NJ Forde; citation_volume=30; citation_publication_date=2020; citation_pages=5420-5430; citation_doi=10.1093/cercor/bhaa123; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Sex differences in the adult human brain: evidence from 5216 UK Biobank participants; citation_author=SJ Ritchie; citation_volume=28; citation_publication_date=2018; citation_pages=2959-2975; citation_doi=10.1093/cercor/bhy109; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Partitioning heritability by functional annotation using genome-wide association summary statistics; citation_author=HK Finucane; citation_volume=47; citation_publication_date=2015; citation_pages=1228-1235; citation_doi=10.1038/ng.3404; citation_id=CR20"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Quantifying the polygenic architecture of the human cerebral cortex: extensive genetic overlap between cortical thickness and surface area; citation_author=D Meer; citation_volume=30; citation_publication_date=2020; citation_pages=5597-5603; citation_doi=10.1093/cercor/bhaa146; citation_id=CR21"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Comparison of methods that use whole genome data to estimate the heritability and genetic architecture of complex traits; citation_author=LM Evans; citation_volume=50; citation_publication_date=2018; citation_pages=737-745; citation_doi=10.1038/s41588-018-0108-x; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Genet.; citation_title=Phenome-wide heritability analysis of the UK Biobank; citation_author=T Ge; citation_volume=13; citation_publication_date=2017; citation_pages=e1006711; citation_doi=10.1371/journal.pgen.1006711; citation_id=CR23"/>
    <meta name="citation_reference" content="Kharabian Masouleh, S., Eickhoff, S. B., Hoffstaedter, F., Genon, S. &amp; Alzheimer&#8217;s Disease Neuroimaging Initiative. Empirical examination of the replicability of associations between brain structure and psychological variables. eLife 8, e43464 (2019)."/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Effects of network resolution on topological properties of human neocortex; citation_author=R Romero-Garcia, M Atienza, LH Clemmensen, JL Cantero; citation_volume=59; citation_publication_date=2012; citation_pages=3522-3532; citation_doi=10.1016/j.neuroimage.2011.10.086; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=On testing for spatial correspondence between maps of human brain structure and function; citation_author=AF Alexander-Bloch; citation_volume=178; citation_publication_date=2018; citation_pages=540-551; citation_doi=10.1016/j.neuroimage.2018.05.070; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Adolescent tuning of association cortex in human structural brain networks; citation_author=F V&#225;&#353;a; citation_volume=28; citation_publication_date=2017; citation_pages=281-294; citation_doi=10.1093/cercor/bhx249; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurophysiol.; citation_title=The organization of the human cerebral cortex estimated by intrinsic functional connectivity; citation_author=BTT Yeo; citation_volume=106; citation_publication_date=2011; citation_pages=1125-1165; citation_doi=10.1152/jn.00338.2011; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Large-scale automated synthesis of human functional neuroimaging data; citation_author=T Yarkoni, RA Poldrack, TE Nichols, DC Essen, TD Wager; citation_volume=8; citation_publication_date=2011; citation_pages=665-670; citation_doi=10.1038/nmeth.1635; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=Decoding brain activity using a large-scale probabilistic functional-anatomical atlas of human cognition; citation_author=TN Rubin; citation_volume=13; citation_publication_date=2017; citation_pages=e1005649; citation_doi=10.1371/journal.pcbi.1005649; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=An atlas of the protein-coding genes in the human, pig, and mouse brain; citation_author=E Sj&#246;stedt; citation_volume=367; citation_publication_date=2020; citation_pages=eaay5947; citation_doi=10.1126/science.aay5947; citation_id=CR31"/>
    <meta name="citation_reference" content="Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. Nature 542, 433&#8211;438 (2017)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=The contribution of X-linked coding variation to severe developmental disorders; citation_author=HC Martin; citation_volume=12; citation_publication_date=2021; citation_doi=10.1038/s41467-020-20852-3; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Integrative functional genomic analysis of human brain development and neuropsychiatric risks; citation_author=M Li; citation_volume=362; citation_publication_date=2018; citation_pages=eaat7615; citation_doi=10.1126/science.aat7615; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=Evolution; citation_title=The variability is in the sex chromosomes: brief communication; citation_author=K Reinhold, L Engqvist; citation_volume=67; citation_publication_date=2013; citation_pages=3662-3668; citation_doi=10.1111/evo.12224; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Genet.; citation_title=Solving the missing heritability problem; citation_author=AI Young; citation_volume=15; citation_publication_date=2019; citation_pages=e1008222; citation_doi=10.1371/journal.pgen.1008222; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=A comprehensive quantitative genetic analysis of cerebral surface area in youth; citation_author=JE Schmitt; citation_volume=39; citation_publication_date=2019; citation_pages=3028-3040; citation_doi=10.1523/JNEUROSCI.2248-18.2019; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Development and arealization of the cerebral cortex; citation_author=CR Cadwell, A Bhaduri, MA Mostajo-Radji, MG Keefe, TJ Nowakowski; citation_volume=103; citation_publication_date=2019; citation_pages=980-1004; citation_doi=10.1016/j.neuron.2019.07.009; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Mol. Autism; citation_title=Mutations in RAB39B in individuals with intellectual disability, autism spectrum disorder, and macrocephaly; citation_author=M Woodbury-Smith; citation_volume=8; citation_publication_date=2017; citation_doi=10.1186/s13229-017-0175-3; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Mol. Genet.; citation_title=Neural cell recognition molecule L1: relating biological complexity to human disease mutations; citation_author=S Kenwrick, A Watkins, E Angelis; citation_volume=9; citation_publication_date=2000; citation_pages=879-886; citation_doi=10.1093/hmg/9.6.879; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=Cytogenet. Genome Res.; citation_title=Evolution of the human X&#8212;a smart and sexy chromosome that controls speciation and development; citation_author=JAM Graves, J G&#233;cz, H Hameister; citation_volume=99; citation_publication_date=2002; citation_pages=141-145; citation_doi=10.1159/000071585; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Bioessays; citation_title=How mammalian sex chromosomes acquired their peculiar gene content; citation_author=EJ Vallender, BT Lahn; citation_volume=26; citation_publication_date=2004; citation_pages=159-169; citation_doi=10.1002/bies.10393; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Sex Differ.; citation_title=Derivation of consensus inactivation status for X-linked genes from genome-wide studies; citation_author=BP Balaton; citation_volume=6; citation_publication_date=2015; citation_pages=35; citation_doi=10.1186/s13293-015-0053-7; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Med.; citation_title=UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age; citation_author=C Sudlow; citation_volume=12; citation_publication_date=2015; citation_pages=e1001779; citation_doi=10.1371/journal.pmed.1001779; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank; citation_author=F Alfaro-Almagro; citation_volume=166; citation_publication_date=2018; citation_pages=400-424; citation_doi=10.1016/j.neuroimage.2017.10.034; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=The minimal preprocessing pipelines for the Human Connectome Project; citation_author=MF Glasser; citation_volume=80; citation_publication_date=2013; citation_pages=105-124; citation_doi=10.1016/j.neuroimage.2013.04.127; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=FreeSurfer; citation_author=B Fischl; citation_volume=62; citation_publication_date=2012; citation_pages=774-781; citation_doi=10.1016/j.neuroimage.2012.01.021; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=A hybrid approach to the skull stripping problem in MRI; citation_author=F S&#233;gonne; citation_volume=22; citation_publication_date=2004; citation_pages=1060-1075; citation_doi=10.1016/j.neuroimage.2004.03.032; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Cortical surface-based analysis. I. Segmentation and surface reconstruction; citation_author=AM Dale, B Fischl, MI Sereno; citation_volume=9; citation_publication_date=1999; citation_pages=179-194; citation_doi=10.1006/nimg.1998.0395; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Cortical surface-based analysis. II. Inflation, flattening, and a surface-based coordinate system; citation_author=B Fischl, MI Sereno, AM Dale; citation_volume=9; citation_publication_date=1999; citation_pages=195-207; citation_doi=10.1006/nimg.1998.0396; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest; citation_author=RS Desikan; citation_volume=31; citation_publication_date=2006; citation_pages=968-980; citation_doi=10.1016/j.neuroimage.2006.01.021; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The UK Biobank resource with deep phenotyping and genomic data; citation_author=C Bycroft; citation_volume=562; citation_publication_date=2018; citation_pages=203-209; citation_doi=10.1038/s41586-018-0579-z; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Haplotype estimation for biobank-scale data sets; citation_author=J O&#8217;Connell; citation_volume=48; citation_publication_date=2016; citation_pages=817-820; citation_doi=10.1038/ng.3583; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=Gigascience; citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets; citation_author=CC Chang; citation_volume=4; citation_publication_date=2015; citation_doi=10.1186/s13742-015-0047-8; citation_id=CR54"/>
    <meta name="citation_reference" content="Burnham, K. P. &amp; Anderson, D. R. Model Selection and Multimodel Inference: A Practical Information-Theoretic Approach. (Springer Science &amp; Business Media, 2002)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types; citation_author=HK Finucane; citation_volume=50; citation_publication_date=2018; citation_pages=621-629; citation_doi=10.1038/s41588-018-0081-4; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Improved heritability estimation from genome-wide SNPs; citation_author=D Speed, G Hemani, MR Johnson, DJ Balding; citation_volume=91; citation_publication_date=2012; citation_pages=1011-1021; citation_doi=10.1016/j.ajhg.2012.10.010; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Common SNPs explain a large proportion of the heritability for human height; citation_author=J Yang; citation_volume=42; citation_publication_date=2010; citation_pages=565-569; citation_doi=10.1038/ng.608; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Morphometric similarity networks detect microscale cortical organization and predict inter-individual cognitive variation; citation_author=J Seidlitz; citation_volume=97; citation_publication_date=2018; citation_pages=231-247.e7; citation_doi=10.1016/j.neuron.2017.11.039; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=Discovering relations between mind, brain, and mental disorders using topic mapping; citation_author=RA Poldrack; citation_volume=8; citation_publication_date=2012; citation_pages=e1002707; citation_doi=10.1371/journal.pcbi.1002707; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Protoc.; citation_title=Quality control and conduct of genome-wide association meta-analyses; citation_author=TW Winkler; citation_volume=9; citation_publication_date=2014; citation_pages=1192-1212; citation_doi=10.1038/nprot.2014.071; citation_id=CR61"/>
    <meta name="citation_reference" content="Mallard, T. T. et al. Multivariate GWAS of psychiatric disorders and their cardinal symptoms reveal two dimensions of cross-cutting genetic liabilities. Preprint at bioRxiv 
                  https://doi.org/10.1101/603134
                  
                 (2020)."/>
    <meta name="citation_reference" content="Mallard, T. T. et al. tem-level genome-wide association study of the Alcohol Use Disorders Identification Test in three population-based cohorts. Am. J. Psychiatry 
                  https://doi.org/10.1176/appi.ajp.2020.20091390
                  
                 (2021)."/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=MAGMA: generalized gene-set analysis of GWAS data; citation_author=CA Leeuw, JM Mooij, T Heskes, D Posthuma; citation_volume=11; citation_publication_date=2015; citation_pages=e1004219; citation_doi=10.1371/journal.pcbi.1004219; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Functional mapping and annotation of genetic associations with FUMA; citation_author=K Watanabe, E Taskesen, A Bochoven, D Posthuma; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/s41467-017-01261-5; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=Enrichr: a comprehensive gene set enrichment analysis web server 2016 update; citation_author=MV Kuleshov; citation_volume=44; citation_publication_date=2016; citation_pages=W90-W97; citation_doi=10.1093/nar/gkw377; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=The DisGeNET knowledge platform for disease genomics: 2019 update; citation_author=J Pi&#241;ero; citation_volume=48; citation_publication_date=2020; citation_pages=D845-D855; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=WGCNA: an R package for weighted correlation network analysis; citation_author=P Langfelder, S Horvath; citation_volume=9; citation_publication_date=2008; citation_pages=559; citation_doi=10.1186/1471-2105-9-559; citation_id=CR68"/>
    <meta name="citation_author" content="Mallard, Travis T."/>
    <meta name="citation_author_institution" content="Department of Psychology, University of Texas at Austin, Austin, USA"/>
    <meta name="citation_author_institution" content="Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA"/>
    <meta name="citation_author" content="Liu, Siyuan"/>
    <meta name="citation_author_institution" content="Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA"/>
    <meta name="citation_author" content="Seidlitz, Jakob"/>
    <meta name="citation_author_institution" content="Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA"/>
    <meta name="citation_author" content="Ma, Zhiwei"/>
    <meta name="citation_author_institution" content="Laboratory of Functional and Molecular Imaging, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, USA"/>
    <meta name="citation_author" content="Moraczewski, Dustin"/>
    <meta name="citation_author_institution" content="Data Science and Sharing Team, National Institute of Mental Health, Bethesda, USA"/>
    <meta name="citation_author" content="Thomas, Adam"/>
    <meta name="citation_author_institution" content="Data Science and Sharing Team, National Institute of Mental Health, Bethesda, USA"/>
    <meta name="citation_author" content="Raznahan, Armin"/>
    <meta name="citation_author_institution" content="Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, USA"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="X-chromosome influences on neuroanatomical variation in humans"/>
    <meta name="twitter:description" content="Nature Neuroscience - Mallard et al. study the often overlooked X-chromosome&#8217;s influences on the human brain. They find that X-chromosome influences on cortical surface area are sex biased..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-021-00890-w"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="X-chromosome influences on neuroanatomical variation in humans - Nature Neuroscience"/>
    <meta property="og:description" content="Mallard et al. study the often overlooked X-chromosome&#8217;s influences on the human brain. They find that X-chromosome influences on cortical surface area are sex biased and concentrated in specific cortical systems."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-021-00890-w;doi=10.1038/s41593-021-00890-w;subjmeta=1515,1688,205,208,2571,2583,378,631,64,692,698;kwrd=Behavioural+genetics,Brain,Development+of+the+nervous+system,Genetic+association+study,Genetics+of+the+nervous+system">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=590485874&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-021-00890-w%26doi%3D10.1038/s41593-021-00890-w%26subjmeta%3D1515,1688,205,208,2571,2583,378,631,64,692,698%26kwrd%3DBehavioural+genetics,Brain,Development+of+the+nervous+system,Genetic+association+study,Genetics+of+the+nervous+system">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=590485874&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-021-00890-w%26doi%3D10.1038/s41593-021-00890-w%26subjmeta%3D1515,1688,205,208,2571,2583,378,631,64,692,698%26kwrd%3DBehavioural+genetics,Brain,Development+of+the+nervous+system,Genetic+association+study,Genetics+of+the+nervous+system"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-021-00890-w'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        X-chromosome influences on neuroanatomical variation in humans
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-021-00890-w.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2021-07-22">22 July 2021</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">X-chromosome influences on neuroanatomical variation in humans</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Travis_T_-Mallard-Aff1-Aff2" data-author-popup="auth-Travis_T_-Mallard-Aff1-Aff2" data-author-search="Mallard, Travis T.">Travis T. Mallard</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-3265-3001"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3265-3001</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Siyuan-Liu-Aff2" data-author-popup="auth-Siyuan-Liu-Aff2" data-author-search="Liu, Siyuan">Siyuan Liu</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-3661-6248"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3661-6248</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jakob-Seidlitz-Aff2" data-author-popup="auth-Jakob-Seidlitz-Aff2" data-author-search="Seidlitz, Jakob">Jakob Seidlitz</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-8164-7476"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8164-7476</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Zhiwei-Ma-Aff3" data-author-popup="auth-Zhiwei-Ma-Aff3" data-author-search="Ma, Zhiwei">Zhiwei Ma</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-2928-402X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2928-402X</a></span><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Dustin-Moraczewski-Aff4" data-author-popup="auth-Dustin-Moraczewski-Aff4" data-author-search="Moraczewski, Dustin">Dustin Moraczewski</a><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Adam-Thomas-Aff4" data-author-popup="auth-Adam-Thomas-Aff4" data-author-search="Thomas, Adam">Adam Thomas</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-2850-1419"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2850-1419</a></span><sup class="u-js-hide"><a href="#Aff4">4</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 7 authors for this article" title="Show all 7 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Armin-Raznahan-Aff2" data-author-popup="auth-Armin-Raznahan-Aff2" data-author-search="Raznahan, Armin" data-corresp-id="c1">Armin Raznahan<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-5622-1190"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-5622-1190</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 24</b>, <span class="u-visually-hidden">pages </span>1216–1224 (<span data-test="article-publication-year">2021</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">5049 <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">15 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">64 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-021-00890-w/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/behavioural-genetics" data-track="click" data-track-action="view subject" data-track-label="link">Behavioural genetics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/brain" data-track="click" data-track-action="view subject" data-track-label="link">Brain</a></li><li class="c-article-subject-list__subject"><a href="/subjects/development-of-the-nervous-system" data-track="click" data-track-action="view subject" data-track-label="link">Development of the nervous system</a></li><li class="c-article-subject-list__subject"><a href="/subjects/genetic-association-study" data-track="click" data-track-action="view subject" data-track-label="link">Genetic association study</a></li><li class="c-article-subject-list__subject"><a href="/subjects/genetics-of-the-nervous-system" data-track="click" data-track-action="view subject" data-track-label="link">Genetics of the nervous system</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>The X-chromosome has long been hypothesized to have a disproportionate influence on the brain based on its enrichment for genes that are expressed in the brain and associated with intellectual disability. Here, we verify this hypothesis through partitioned heritability analysis of X-chromosome influences (XIs) on human brain anatomy in 32,256 individuals from the UK Biobank. We first establish evidence for dosage compensation in XIs on brain anatomy—reflecting larger XIs in males compared to females, which correlate with regional sex-biases in neuroanatomical variance. XIs are significantly larger than would be predicted from X-chromosome size for the relative surface area of cortical systems supporting attention, decision-making and motor control. Follow-up association analyses implicate X-linked genes with pleiotropic effects on cognition. Our study reveals a privileged role for the X-chromosome in human neurodevelopment and urges greater inclusion of this chromosome in future genome-wide association studies.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-021-00890-w.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-021-00890-w.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-020-17368-1/MediaObjects/41467_2020_17368_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41467-020-17368-1?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41467-020-17368-1">Understanding the genetic determinants of the brain with MOSTest
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">14 July 2020</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Dennis van der Meer, Oleksandr Frei, … Anders M. Dale</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-020-19378-5/MediaObjects/41467_2020_19378_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41467-020-19378-5?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41467-020-19378-5">Genome-wide meta-analysis of brain volume identifies genomic loci and genes shared with intelligence
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">05 November 2020</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Philip R. Jansen, Mats Nagel, … Danielle Posthuma</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-019-0516-6/MediaObjects/41588_2019_516_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41588-019-0516-6?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41588-019-0516-6">Genome-wide association analysis of 19,629 individuals identifies variants influencing regional brain volumes and refines their genetic co-architecture with cognitive and mental health traits
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">01 November 2019</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Bingxin Zhao, Tianyou Luo, … Hongtu Zhu</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711582424,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Our understanding of genetic contributions to human brain development has dramatically expanded over the past 5 years through access to large study populations that combine both genotypic and neuroimaging data. In particular, pioneering imaging–genetics studies from the ENIGMA consortium and the UK Biobank (UKB) dataset have started to reveal the relationship between neuroanatomical variation in health and genotypic variation tagged by common single nucleotide polymorphisms (SNPs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Grasby, K. L. et al. The genetic architecture of the human cerebral cortex. Science 367, eaay6690 (2020)." href="#ref-CR1" id="ref-link-section-d15478171e502">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Biton, A. et al. Polygenic architecture of human neuroanatomical diversity. Cereb. Cortex 30, 2307–2320 (2020)." href="#ref-CR2" id="ref-link-section-d15478171e502_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Zhao, B. et al. Genome-wide association analysis of 19,629 individuals identifies variants influencing regional brain volumes and refines their genetic co-architecture with cognitive and mental health traits. Nat. Genet. 51, 1637–1644 (2019)." href="/articles/s41593-021-00890-w#ref-CR3" id="ref-link-section-d15478171e505">3</a></sup>. This work has revealed the pervasive and diverse influence of genes on brain morphology, reflected by moderate-to-high heritability estimates and complex genetic architectures across anatomical subdivisions of the human brain. However, to date, most large-scale studies linking ‘genome-wide’ SNP data to neuroimaging phenotypes have excluded the sex chromosomes (with few exceptions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Smith, S. M. et al. An expanded set of genome-wide association studies of brain imaging phenotypes in UK Biobank. Nat. Neurosci. 24, 737–745 (2021)." href="/articles/s41593-021-00890-w#ref-CR4" id="ref-link-section-d15478171e509">4</a></sup>), presumably because they require a unique analytic approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Wise, A. L., Gyi, L. &amp; Manolio, T. A. eXclusion: toward integrating the X chromosome in genome-wide association analyses. Am. J. Hum. Genet. 92, 643–647 (2013)." href="/articles/s41593-021-00890-w#ref-CR5" id="ref-link-section-d15478171e513">5</a></sup>. Notably, analysis of sex chromosomes must grapple with the following facts: (1) X- and Y-chromosome dosage differs between males (XY) and females (XX), and (2) one X-chromosome is randomly silenced in all cells of females through the process of X-inactivation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Lyon, M. F. Gene action in the X-chromosome of the mouse (Mus musculus L.). Nature 190, 372–373 (1961)." href="/articles/s41593-021-00890-w#ref-CR6" id="ref-link-section-d15478171e517">6</a></sup>.</p><p>The current lack of data regarding sex chromosome influences on human brain organization in health is especially problematic when one considers several lines of evidence indicating that the X-chromosome might in fact have a disproportionately strong influence on brain development relative to other chromosomes. Specifically, it is known that X-chromosome genes show enriched expression in brain tissue relative to other chromosomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Nguyen, D. K. &amp; Disteche, C. M. Dosage compensation of the active X chromosome in mammals. Nat. Genet. 38, 47–53 (2006)." href="/articles/s41593-021-00890-w#ref-CR7" id="ref-link-section-d15478171e524">7</a></sup>, and the X-chromosome is enriched for genes that cause intellectual disability when mutated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Zechner, U. et al. A high density of X-linked genes for general cognitive ability: a run-away process shaping human evolution? Trends Genet. 17, 697–701 (2001)." href="/articles/s41593-021-00890-w#ref-CR8" id="ref-link-section-d15478171e528">8</a></sup>. Furthermore, X-chromosome aneuploidy induces robust regional changes in human brain anatomy as measured by structural MRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Raznahan, A. et al. Globally divergent but locally convergent X- and Y-chromosome influences on cortical development. Cereb. Cortex 26, 70–79 (2016)." href="/articles/s41593-021-00890-w#ref-CR9" id="ref-link-section-d15478171e532">9</a></sup>. These observations naturally lead to the as yet untested prediction that common genetic variation on the X-chromosome exerts an ‘oversized’ influence on human brain development as captured by measures of regional brain anatomy from structural MRI. In the present study, we directly test this prediction by leveraging well-substantiated theory from statistical genetics and large-scale data from the UKB (<i>n</i> = 32,256 individuals of European ancestry, 52.97% female, mean age = 64.31 years) to provide the first map of enriched X-chromosome influences (enriched XIs) on normative neuroanatomical variation in humans. To this aim, we performed a series of genome-based restricted maximum likelihood (GREML)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Yang, J. et al. Genome partitioning of genetic variation for complex traits using common SNPs. Nat. Genet. 43, 519–525 (2011)." href="/articles/s41593-021-00890-w#ref-CR10" id="ref-link-section-d15478171e539">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011)." href="/articles/s41593-021-00890-w#ref-CR11" id="ref-link-section-d15478171e542">11</a></sup> analyses to facilitate the following investigations: (1) evaluate evidence for X-chromosome dosage compensation in males and females; (2) partition total SNP-based heritability (<span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span>) across the autosomes (<span class="mathjax-tex">\(h_{\mathrm{a}}^2\)</span>) and the X-chromosome (<span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span>); and (3) test for enriched XIs on both global and regional measures of brain anatomy. We annotate observed XIs using spatial comparison with neuroimaging-derived maps of brain structure and function and with follow-up brain-region-specific X-chromosome-wide association (XWA) analyses.</p><p>All global and regional measures of brain anatomy were estimated using the FreeSurfer pipeline<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Fischl, B. et al. Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. Neuron 33, 341–355 (2002)." href="/articles/s41593-021-00890-w#ref-CR12" id="ref-link-section-d15478171e652">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Fischl, B. et al. Automatically parcellating the human cerebral cortex. Cereb. Cortex 14, 11–22 (2004)." href="/articles/s41593-021-00890-w#ref-CR13" id="ref-link-section-d15478171e655">13</a></sup> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). We examined diverse regional measures, including subcortical volumes and cortical volumes (CVs), as well as regional cortical surface area (SA) and cortical thickness (CT). Cortical phenotypes were estimated for each of the 358 regions of interest (ROIs) as defined by a recent multimodal parcellation from the Human Connectome Project (HCP-parcellation)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e662">14</a></sup>. Separate analysis of SA, CT and CV was motivated by the following considerations: SA and CT together determine CV, but differ greatly in their genetic, developmental and cellular determinants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Lerch, J. P. et al. Studying neuroanatomy using MRI. Nat. Neurosci. 20, 314–326 (2017)." href="/articles/s41593-021-00890-w#ref-CR15" id="ref-link-section-d15478171e666">15</a></sup>; variation in SA is known to explain the bulk of CV variation across individuals and shows higher genetic correlation with CV variation than CT<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Hofer, E. et al. Genetic correlations and genome-wide associations of cortical structure in general population samples of 22,824 adults. Nat. Commun. 11, 4796 (2020)." href="/articles/s41593-021-00890-w#ref-CR16" id="ref-link-section-d15478171e670">16</a></sup>; and SA and CT differ in their genetic architecture such that SA is a less polygenic trait than CT, and is associated with larger absolute variant-specific effect sizes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Matoba, N., Love, M. I. &amp; Stein, J. L. Evaluating brain structure traits as endophenotypes using polygenicity and discoverability. Hum. Brain Mapp. &#xA;                  https://doi.org/10.1002/hbm.25257&#xA;                  &#xA;                 (2020)." href="/articles/s41593-021-00890-w#ref-CR17" id="ref-link-section-d15478171e675">17</a></sup>. Given these considerations, and the observed distribution of XIs in our analyses (see below), we focus our manuscript on findings for regional SA and provide equivalent results for all other anatomical phenotypes in the supplementary materials.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">X-chromosome dosage compensation and sex-biased XIs</h3><p>To appropriately estimate <span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span> while incorporating the X-chromosome, it is necessary to first determine whether <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span> should be modeled assuming full dosage compensation (under which X-linked genetic variance is greater in males than females), no dosage compensation (which would predict the opposite sex-bias in X-linked variance) or equal variance across men and women<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011)." href="/articles/s41593-021-00890-w#ref-CR11" id="ref-link-section-d15478171e759">11</a></sup>. By comparing the fit of these models, we found evidence of full dosage compensation for all global measures of brain anatomy, as well as the vast majority of regional measures (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">1</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>). This finding implies that the X-chromosome tends to explain a greater proportion of neuroanatomical variation in males than females. To provide a complementary test of this inference, we used sex-stratified analyses to separately estimate <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span> in males and females for the 920 cortical and subcortical ROIs for which the full dosage compensation model fit best. Across these phenotypes, the median <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span> in males was 2.37-times larger (bootstrapped 95% confidence interval (CI) = 1.70–3.25) than the median <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span> in females (Wilcoxon <i>P</i> = 1.18 × 10<sup>−23</sup>; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig5">1</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>).</p><p>Although we observed robust statistical support for full dosage compensation for almost all measures of regional brain anatomy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig1">1a</a> (SA), Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig6">2a</a> (CV) and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig7">3a</a> (CT), and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">1</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>), there was substantial variation in the magnitude of this effect across the cortical sheet (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig1">1b</a> (SA), Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig6">2b</a> (CV) and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig7">3b</a> (CT), and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2–</a><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">4</a>). We reasoned that the sex biases in XI arising from full dosage compensation could potentially contribute to known sex biases in regional anatomical variance, which are particularly pronounced for cortical SA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Forde, N. J. et al. Sex differences in variability of brain structure across the lifespan. Cereb. Cortex 30, 5420–5430 (2020)." href="/articles/s41593-021-00890-w#ref-CR18" id="ref-link-section-d15478171e917">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Ritchie, S. J. et al. Sex differences in the adult human brain: evidence from 5216 UK Biobank participants. Cereb. Cortex 28, 2959–2975 (2018)." href="/articles/s41593-021-00890-w#ref-CR19" id="ref-link-section-d15478171e920">19</a></sup>. To test this hypothesis, we computed the male:female ratio of SA variance for each cortical ROI in the UKB sample (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig1">1c</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>) and correlated inter-ROI differences in this metric, with inter-ROI differences in the strength of statistical support for full dosage compensation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig1">1b</a>). We observed a strong positive correlation between these two features of the cortical sheet (<i>r</i> = 0.76; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig1">1d</a>), which is consistent with a model in which regional sex-biases in XIs on SA variance contribute to regional sex-biases in SA variance.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Regional patterning of X-chromosome dosage compensation and links to sex differences in anatomical variance for cortical SA."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Regional patterning of X-chromosome dosage compensation and links to sex differences in anatomical variance for cortical SA.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="680"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Categorical map showing the best-fitting X-chromosome dosage compensation model for each cortical region. NDC, no dosage compensation, EV, equal variance, FDC, full dosage compensation. <b>b</b>, Continuous map showing regional variation in the strength of statistical evidence for X-chromosome dosage compensation on SA, as indexed by ΔAIC<sub>DC</sub> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). ΔAIC<sub>DC</sub> equals zero when the EV model fits best, and departures from zero reflect quantitative evidence for either NDC or FDC. ΔAIC<sub>DC</sub> ≤ −10 is indicative of strong evidence for NDC while ΔAIC<sub>DC</sub> ≥ 10 is indicative of strong evidence for FDC. Any values less than −10 or greater than 10 were set to −10 and 10, respectively, for illustration. <b>c</b>, Continuous map showing regional variation in the magnitude of sex differences in SA phenotypic variance. <b>d</b>, Scatter plot of inter-regional variation in evidence for FDC (ΔAIC<sub>DC</sub>) and the magnitude of sex biases in SA phenotypic variance (male:female variance ratio).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec4">XIs on neuroanatomical variation</h3><p>Having determined the suitable X-chromosome dosage compensation model for each anatomical measure of interest (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">1</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>), we next used GREML<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Yang, J. et al. Genome partitioning of genetic variation for complex traits using common SNPs. Nat. Genet. 43, 519–525 (2011)." href="/articles/s41593-021-00890-w#ref-CR10" id="ref-link-section-d15478171e999">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011)." href="/articles/s41593-021-00890-w#ref-CR11" id="ref-link-section-d15478171e1002">11</a></sup> to estimate <span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span>, which we further partitioned into <span class="mathjax-tex">\(h_{\mathrm{a}}^2\)</span> and <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span>. This partitioning enabled us to test whether observed XIs on neuroanatomical variation were in excess of those predicted by the size of the X-chromosome genomic partition (corresponding to the theoretical null of equal per-SNP heritability). Drawing on existing methods for partitioning heritability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Yang, J. et al. Genome partitioning of genetic variation for complex traits using common SNPs. Nat. Genet. 43, 519–525 (2011)." href="/articles/s41593-021-00890-w#ref-CR10" id="ref-link-section-d15478171e1109">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011)." href="/articles/s41593-021-00890-w#ref-CR11" id="ref-link-section-d15478171e1112">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. Nat. Genet. 47, 1228–1235 (2015)." href="/articles/s41593-021-00890-w#ref-CR20" id="ref-link-section-d15478171e1115">20</a></sup>, we calculated the ratio of the proportion of heritability explained by the X-chromosome to the proportion of genomic base pairs located on the X-chromosome (~5%; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). This provided an intuitive index of the magnitude of XI enrichment (ratios &gt; 1) or depletion (ratios &lt; 1) for each phenotype. Furthermore, to test whether the observed enrichment or depletion of XI was statistically significant, we used a monotonically related enrichment <i>Z</i>-test to obtain a corresponding <i>P</i> value (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>).</p><p>We first considered global measures of brain anatomy, including total brain volume and surface-based estimates of total SA and mean CT. Our estimate of SNP-based <span class="mathjax-tex">\(h_{\mathrm{a}}^2\)</span> for total brain volume was in line with that reported by a prior study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Zhao, B. et al. Genome-wide association analysis of 19,629 individuals identifies variants influencing regional brain volumes and refines their genetic co-architecture with cognitive and mental health traits. Nat. Genet. 51, 1637–1644 (2019)." href="/articles/s41593-021-00890-w#ref-CR3" id="ref-link-section-d15478171e1168">3</a></sup> using GREML in both releases of the UKB (72% versus 71%; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">1</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). We observed a significant depletion of XIs for total brain volume (<i>P</i> = 3 × 10<sup>−4</sup>), with the observed <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span> reaching only 60% of the expected <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span> based on the size of the X-chromosome compartment (2.3% observed versus 3.8% expected; <span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span> = 74%). Our estimates of <span class="mathjax-tex">\(h_{\mathrm{a}}^2\)</span> for total cortical SA and mean CT (77% and 57%, respectively) were substantially larger than those reported by prior studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Grasby, K. L. et al. The genetic architecture of the human cerebral cortex. Science 367, eaay6690 (2020)." href="/articles/s41593-021-00890-w#ref-CR1" id="ref-link-section-d15478171e1320">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="van der Meer, D. et al. Quantifying the polygenic architecture of the human cerebral cortex: extensive genetic overlap between cortical thickness and surface area. Cereb. Cortex 30, 5597–5603 (2020)." href="/articles/s41593-021-00890-w#ref-CR21" id="ref-link-section-d15478171e1323">21</a></sup> using summary statistics from genome-wide association studies (GWAS; excluding the X-chromosome) in a multisample cohort<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Grasby, K. L. et al. The genetic architecture of the human cerebral cortex. Science 367, eaay6690 (2020)." href="/articles/s41593-021-00890-w#ref-CR1" id="ref-link-section-d15478171e1328">1</a></sup> (34% and 26%, respectively) and both releases of the UKB data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="van der Meer, D. et al. Quantifying the polygenic architecture of the human cerebral cortex: extensive genetic overlap between cortical thickness and surface area. Cereb. Cortex 30, 5597–5603 (2020)." href="/articles/s41593-021-00890-w#ref-CR21" id="ref-link-section-d15478171e1332">21</a></sup> (32% and 23%, respectively). Of note, our observation that <span class="mathjax-tex">\(h_{\mathrm{a}}^2\)</span> estimates from GREML are larger than those produced by methods based on GWAS summary statistics is in line with published methodological comparisons of these methods in both simulated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Evans, L. M. et al. Comparison of methods that use whole genome data to estimate the heritability and genetic architecture of complex traits. Nat. Genet. 50, 737–745 (2018)." href="/articles/s41593-021-00890-w#ref-CR22" id="ref-link-section-d15478171e1370">22</a></sup> and real<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Ge, T. et al. Phenome-wide heritability analysis of the UK Biobank. PLoS Genet. 13, e1006711 (2017)." href="/articles/s41593-021-00890-w#ref-CR23" id="ref-link-section-d15478171e1374">23</a></sup> data. We also observed a significant depletion of XIs for total SA and mean CT (<i>P</i> &lt; 9 × 10<sup>−4</sup>, XIs ~50% of expectation from X-chromosome size, <span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span> = 79% and 59%, respectively). The depletion of XIs for measures of global brain anatomy implies that any enriched XIs on human neuroanatomical variation would need to operate at a more local scale involving the relative size of subcomponents of the brain. To test this idea, we conducted GREML analyses of regional anatomical traits after regressing out the appropriate global trait (for example, regional volume controlling for total brain volume, regional CT controlling for overall mean CT and regional SA controlling for total SA).</p><p>Our GREML-based estimates of <span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span> for measures of regional brain anatomy in the full UKB sample (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>) reinforced prior reports<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Grasby, K. L. et al. The genetic architecture of the human cerebral cortex. Science 367, eaay6690 (2020)." href="/articles/s41593-021-00890-w#ref-CR1" id="ref-link-section-d15478171e1464">1</a></sup> of higher heritability for regional SA compared to regional CT (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">3</a>), and substantial inter-regional variation in heritability across the cortical sheet (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig2">2a</a> (SA) and Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig8">4a</a> (CV) and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig9">5a</a> (CT)). Importantly, our estimates of <span class="mathjax-tex">\(h_{\mathrm{a}}^2\)</span> were closely aligned with those reported in a prior study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Zhao, B. et al. Genome-wide association analysis of 19,629 individuals identifies variants influencing regional brain volumes and refines their genetic co-architecture with cognitive and mental health traits. Nat. Genet. 51, 1637–1644 (2019)." href="/articles/s41593-021-00890-w#ref-CR3" id="ref-link-section-d15478171e1519">3</a></sup> that also used a GREML approach to estimate heritability in an earlier release of the UKB imaging data. Specifically, the ranges of <span class="mathjax-tex">\(h_{\mathrm{a}}^2\)</span> for comparable subcortical volumetric phenotypes examined in both studies were very similar, being 31–63% in our study (mean 48%) and 33–63% (mean 54%) in the study by Zhao and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Zhao, B. et al. Genome-wide association analysis of 19,629 individuals identifies variants influencing regional brain volumes and refines their genetic co-architecture with cognitive and mental health traits. Nat. Genet. 51, 1637–1644 (2019)." href="/articles/s41593-021-00890-w#ref-CR3" id="ref-link-section-d15478171e1557">3</a></sup>. We used autosome-only GREML analyses to test and verify the expectation that estimates of total heritability would be consistently greater with inclusion compared to exclusion of the X-chromosome (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>). The mean proportion of <span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span> lost by exclusion of the X-chromosome was greatest for regional measures of SA (mean cross-ROI heritability loss for SA = 7.58% (95% CI = 7.14–8.02%)), lowest for CT (2.83% (95% CI = 2.47–3.19%)) and intermediate for their product, CV (6.59% (95% CI = 6.23–6.94%)). The proportion of SA heritability lost from exclusion of the X-chromosome was remarkably large for several ROIs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a>). For example, although the X-chromosome accounts for ~5% of the genome, the proportion of <span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span> explained by the X-chromosome partition exceeded 20% for relative SA of several cortical regions, including the right superior parietal cortex (areas 7pl and LIPv in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e1639">14</a></sup>), the right premotor cortex (area PEF<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e1643">14</a></sup>), the right entorhinal cortex (area ec<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e1647">14</a></sup>), the left ventral postcentral cortex (area 43<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e1651">14</a></sup>), the left perisylvian cortex (area PSL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e1656">14</a></sup>) and the left supramarginal cortex (area p_fm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e1660">14</a></sup>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Regional patterning of XIs on cortical SA."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Regional patterning of XIs on cortical SA.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="843"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Continuous map of total SNP heritability (<span class="mathjax-tex">\(h_{\mathrm{g}}^2\)</span>) for regional SA. <b>b</b>, Continuous map of X-linked heritability (<span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span>) for regional SA. <b>c</b>, Continuous map of the regional ratio between the observed proportion of total heritability assigned to the X-chromosome and the expected proportion from X-chromosome size (the XI enrichment ratio). Values greater than one reflect enriched XIs and values less than one reflect depleted XIs. <b>d</b>, Categorical map showing regions with significant enrichment of XIs. <b>e</b>, Scatter plot of the mean heritability per chromosome (<span class="mathjax-tex">\(h_{\mathrm{c}}^2\)</span>) plotted against chromosomal length for regional SA phenotypes with significantly enriched XIs (<i>n</i> = 56 regions). The 23 data points correspond to the mean <span class="mathjax-tex">\(h_{\mathrm{c}}^2\)</span> estimate of chromosomes across those regions (with accompanying 95% CIs). The line of best fit is displayed in red (with the accompanying 95% CI presented as a band), while the theoretical null is displayed as a dashed black line.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The magnitude of XIs (that is, <span class="mathjax-tex">\(h_{\mathrm{X}}^2\)</span>) varied substantially across brain regions and phenotypes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig2">2b</a> (SA), Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig8">4b</a> (CV) and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig9">5b</a> (CT), and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>), as did the ratio between observed and expected XIs on human brain anatomy (XI ratio) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig2">2c</a> (SA), Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig8">4c</a> (CV) and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig9">5c</a> (CT), and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>). We did not find evidence for enriched XIs on any regional measure of subcortical volume (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>). In the cortex, observed XI ratios were consistently greater for regional measures of SA and CV than CT (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig10">6a</a>). For example, ~66% of SA ROIs showed an XI ratio greater than 1, whereas only ~18% of ROIs met this criterion for CT (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">3</a>). We also observed a strikingly stronger spatial coupling between regional variation in the XI ratio for SA and CV than for CT and CV (<i>r</i> = 0.72 and <i>r</i> = 0.11, respectively; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig10">6b,c</a>). There was a weak spatial coupling between the XI ratio for SA and CT (<i>r</i> = 0.15; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig10">6d</a>). Taken together, these results indicate that enriched XIs on cortical anatomy are much more widespread for regional SA than regional CT and that XIs on regional CV are more a function of XIs on SA than CT.</p><p>After correcting for multiple comparisons within each phenotype (that is, SA, CT and CV) across all 358 cortical ROIs, we identified 75 regional measures with significant XI enrichment (that is, XI <i>Z</i>-statistic significantly &gt; 0; Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>), 56 (75%) of which were regional measures of SA. In contrast, 93 regional cortical measures showed significant XI depletion (that is, XI <i>Z</i>-statistic significantly &lt; 0; Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">2</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">5</a>), and the majority of these were regional measures of CT (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig11">7</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig12">8</a>). Notably, the medial occipital visual cortices represented a concentrated locus of significant depletion of XIs for both primary dimensions of the cortical sheet (SA and CT) and their product (CV) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig12">8</a>). Thus, we observed robust statistical evidence for enriched XIs on human neuroanatomical variation, and this effect was most apparent for the relative SA in specific subregions of the cortical sheet centered on the bilateral sensorimotor, the temporoparietal junction, the medial parietal and medial temporal cortices, the right inferior frontal and medial prefrontal cortices, and the left dorsolateral prefrontal cortex (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig2">2d,e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig12">8</a>).</p><p>We took four complementary approaches to assess the reproducibility of our findings for XIs on regional SA. First, we compared spatial patterning of the XI <i>Z</i>-statistic for regional SA between the first and second releases of the UKB neuroimaging data and found this to show moderate reproducibility (cross-ROI Pearson’s correlation in XI <i>Z</i>-statistics between the two releases: <i>r</i> = 0.52; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig13">9a</a>). This compares favorably with the spatial reproducibility seen for associations between behavior and regional brain anatomy, for example, see ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Kharabian Masouleh, S., Eickhoff, S. B., Hoffstaedter, F., Genon, S. &amp; Alzheimer’s Disease Neuroimaging Initiative. Empirical examination of the replicability of associations between brain structure and psychological variables. eLife 8, e43464 (2019)." href="/articles/s41593-021-00890-w#ref-CR24" id="ref-link-section-d15478171e1988">24</a></sup>. Second, we quantified cross-ROI correlations in the XI <i>Z</i>-statistic for SA between 100 randomly generated split-halves of the full UKB sample. The mean cross-ROI Pearson’s correlation in XI <i>Z</i>-statistics across the split-half analyses was <i>r</i> = 0.55 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig13">9b</a>). Third, we confirmed that the spatial patterning of observed XI <i>Z</i>-statistics was highly reproducible when GREML analyses were re-run using a more conservative minor allele frequency (MAF) threshold (cross-ROI <i>r</i> = 0.96; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig13">9c</a>). Fourth, to assess parcellation effects, we also computed regional XI for SA using an alternative cortical parcellation scheme that has been extensively used in prior work, and divides the cortex into 308 similarly sized subdivisions of ROIs from the gyral-based Desikan–Killiany parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Romero-Garcia, R., Atienza, M., Clemmensen, L. H. &amp; Cantero, J. L. Effects of network resolution on topological properties of human neocortex. NeuroImage 59, 3522–3532 (2012)." href="/articles/s41593-021-00890-w#ref-CR25" id="ref-link-section-d15478171e2015">25</a></sup> (henceforth, termed 308-parcellation). This alternative parcellation scheme also provides access to a validated ‘spin’-based technique for comparing the spatial similarity between two cortical features<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. NeuroImage 178, 540–551 (2018)." href="/articles/s41593-021-00890-w#ref-CR26" id="ref-link-section-d15478171e2019">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. Cereb. Cortex 28, 281–294 (2017)." href="/articles/s41593-021-00890-w#ref-CR27" id="ref-link-section-d15478171e2022">27</a></sup> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). We projected regional XI <i>Z</i>-statistics for SA from the HCP-parcellation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig2">2c</a>) into the 308-parcellation (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>) and confirmed that the spatial distribution of XIs from this map was highly similar to XI <i>Z</i>-statistics computed within the 308-parcellation itself (cross-ROI <i>r</i> = 0.76, <i>P</i><sub>spin</sub> &lt; 1 × 10<sup>−4</sup>; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig13">9d,e</a>).</p><h3 class="c-article__sub-heading" id="Sec5">Topographic annotation of enriched XIs on cortical SA</h3><p>Taken together, the findings above indicate that SNP-based variation on the X-chromosome exerts a disproportionately large influence on SA variation in a specific subset of cortical regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig2">2c,d</a>). We hypothesized that the spatial patterning of enriched XIs on SA may align with functional topography of the cortical sheet. To test this notion, we harnessed the 308-parcellation-based map of regional XIs on SA (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig13">9d</a>) to access spin-based methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. NeuroImage 178, 540–551 (2018)." href="/articles/s41593-021-00890-w#ref-CR26" id="ref-link-section-d15478171e2069">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. Cereb. Cortex 28, 281–294 (2017)." href="/articles/s41593-021-00890-w#ref-CR27" id="ref-link-section-d15478171e2072">27</a></sup> to directly compare the spatial distribution of XI effects on cortical SA against cortical annotations from independent functional neuroimaging data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-021-00890-w#ref-CR28" id="ref-link-section-d15478171e2076">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. Nat. Methods 8, 665–670 (2011)." href="/articles/s41593-021-00890-w#ref-CR29" id="ref-link-section-d15478171e2079">29</a></sup> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>).</p><p>We first asked whether the spatial distribution of XIs on SA was organized in a non-random manner relative to the topography of canonical resting-state functional connectivity networks in the human brain (as defined by the well-validated Yeo–Krienen-17-parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-021-00890-w#ref-CR28" id="ref-link-section-d15478171e2089">28</a></sup>; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). The assignment of cortical ROIs from the 308-parcellation to network labels from the Yeo–Krienen-17 atlas explained a significantly large proportion of inter-regional variation in XIs for SA (compared to 10,000 spins of these assignments, omnibus <i>F</i><sub>observed</sub> = 5.9, <i>P</i><sub>spin</sub> = 0.01). We further characterized this significant omnibus alignment between regional XI enrichment for SA and functional connectivity networks with post hoc overlap tests for each of the network components contained within the Yeo–Krienen-17-parcellation. The following six network components overlapped regions of significant XI enrichment for SA with an odds ratio &gt; 1: one subcomponent of the motor network; two subcomponents from the frontoparietal network; and individual subcomponents from the ventral attention, dorsal attention and default-mode networks (<i>P</i><sub>spin</sub> &lt; 0.01 for one motor and one ventral attention subnetwork; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig3">3a,b</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">6</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Topographic annotation of XIs on cortical SA."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Topographic annotation of XIs on cortical SA.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="601"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Categorical map of 17 canonical resting-state functional connectivity networks as defined by Yeo, Krienen et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-021-00890-w#ref-CR28" id="ref-link-section-d15478171e2129">28</a></sup>. <b>b</b>, Bar chart illustrating the degree of spatial overlap between regions of significant XI enrichment for SA and each of the 17 Yeo–Krienen networks. <b>c</b>, Bar chart illustrating the degree of spatial overlap between regions of significant XI enrichment for SA and each of the 30 Neurosynth topics. <b>b</b>,<b>c</b>, Odds ratios greater than one indicate greater than chance overlap between regions of significantly enriched XIs on SA and the Yeo–Krienen or Neurosynth feature in question. Asterisks denote <i>P</i><sub>spin</sub> ≤ 0.01 in post hoc tests (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). <i>P</i><sub>spin</sub> = 6.3 × 10<sup>−3</sup> and 8.6 × 10<sup>−3</sup> for MOT-1 and VAN-1 in <b>b</b>; <i>P</i><sub>spin</sub> = 7 × 10<sup>−4</sup>, 1.6 × 10<sup>−3</sup>, &lt;1 × 10<sup>−4</sup>, 4 × 10<sup>−3</sup> and 1 × 10<sup>−2</sup> for somatosensory, motor, response inhibition, cognitive performance and conflict response topics, respectively, in <b>c</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>To annotate the topographical pattern of enriched XIs on cortical anatomy using finer-grained distinctions between different aspects of human brain function, we also compared the map of significantly enriched XIs on SA with 30 different meta-analytic maps of cortical activation from Neurosynth<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. Nat. Methods 8, 665–670 (2011)." href="/articles/s41593-021-00890-w#ref-CR29" id="ref-link-section-d15478171e2194">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Rubin, T. N. et al. Decoding brain activity using a large-scale probabilistic functional-anatomical atlas of human cognition. PLoS Comput. Biol. 13, e1005649 (2017)." href="/articles/s41593-021-00890-w#ref-CR30" id="ref-link-section-d15478171e2197">30</a></sup> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). These 30 maps span the gamut of cognitive constructs (or ‘topics’) examined across &gt;11,000 individual functional neuroimaging studies contained within the Neurosynth database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Rubin, T. N. et al. Decoding brain activity using a large-scale probabilistic functional-anatomical atlas of human cognition. PLoS Comput. Biol. 13, e1005649 (2017)." href="/articles/s41593-021-00890-w#ref-CR30" id="ref-link-section-d15478171e2204">30</a></sup> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">7</a>). We observed a significant omnibus alignment between the spatial patterning of these 30 Neurosynth topic maps and regional XI enrichment for SA (mean <i>F</i><sub>observed</sub> = 9.3, <i>P</i><sub>spin</sub> = 0.02; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). Post hoc analysis identified 16 Neurosynth topic maps that overlapped regions of significant XI enrichment for SA with an odds ratio &gt; 1, the following five of which were significant at <i>P</i><sub>spin</sub> ≤ 0.01: somatosensory, motor, response inhibition, conflict response and cognitive performance (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig3">3c</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM2">7</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>).</p><p>Thus, our comparison with two complementary functional annotations of the cortical sheet suggests that enriched XIs on SA tend to be concentrated in a subset of functional systems within the cerebral cortex, including the following: (1) a somatomotor system encompassing primary sensory, primary and supplemental motor, and insular regions; (2) a response–inhibition system centered on the inferior frontal gyrus; and (3) higher-order attentional and default-mode systems centered on the temporoparietal junction.</p><h3 class="c-article__sub-heading" id="Sec6">Genomic annotation of enriched XIs on SA</h3><p>To identify which specific genes might be relevant to the observed enriched XIs on human brain anatomy, we conducted a series of gene-based XWA analyses in each of the 56 cortical SA phenotypes that exhibited significant XI enrichment (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). Briefly, we observed substantial inflation of the gene-based test statistics for these phenotypes (median gene-based <span class="mathjax-tex">\(\lambda _{{\mathrm{GC}}}\)</span> = 1.36 (IQR = 1.25–1.48)), which is indicative of a robust polygenic signal on the X-chromosome (but could also reflect remaining population stratification despite our control for this in the analysis). Next, to synthesize results across the enrichment map, we calculated the minimum <i>P</i> value for each gene across all particular ROIs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig4">4a</a>, with results of SNP-based association testing provided in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig14">10</a>). This yielded a list of 20 X-chromosome genes that met the threshold of X-chromosome-wide significance for gene-based testing (<i>P</i> = 6.91 × 10<sup>−5</sup>) across the cortex (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/s41593-021-00890-w#Tab1">1</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig4">4b</a>), and one gene (<i>ZIC3</i>) that met the stricter significance threshold correcting for all phenotypes tested (<i>P</i> = 1.23 × 10<sup>−6</sup>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="X-linked genes associated with regional SA phenotypes with enriched XIs."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: X-linked genes associated with regional SA phenotypes with enriched XIs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="688"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, X-chromosome-wide Manhattan plot of the minimum <i>P</i> value for each gene across all tested phenotypes. The <i>x</i> axis refers to the chromosomal position, the <i>y</i> axis refers to the significance of the gene-based association test as a −log<sub>10</sub>(<i>P</i> value), the horizontal dashed line denotes X-chromosome-wide significance (<i>P</i> = 6.91 × 10<sup>−5</sup>), and the horizontal two-dashed line denotes a more conservative significance threshold accounting for all tested phenotypes (<i>P</i> = 1.23 × 10<sup>−6</sup>). To improve readability, the <i>y</i> axis has been exponentiated, giving more space to significant genes and permitting the inclusion of gene labels. Circle size corresponds to the number of tested SNPs in each gene. <b>b</b>, Categorical map of gene discovery, where black refers to regions that were not tested (that is, those with no significant XI enrichment), yellow refers to regions that were tested but did not identify any genes in the gene-based tests and red refers to regions with significant gene associations. <b>c</b>, Bar chart of the top ten enriched disease ontologies based on Enrichr analysis. Fisher’s exact test was used to compute one-sided <i>P</i> values, which are plotted along the <i>x</i> axis on a −log<sub>10</sub> scale. Asterisks and color denote significance after correcting for multiple comparisons. <b>d</b>, Smoothed line plot of gene set expression across developmental time in the PsychENCODE dataset for the 20 identified genes. The <i>x</i> axis refers to the age of the sample in days (on a log<sub>2</sub> scale), the <i>y</i> axis refers to expression, and each data point refers to a brain sample.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-021-00890-w/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 X-chromosome genes associated with regional SA phenotypes</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/s41593-021-00890-w/tables/1" aria-label="Full size table 1"><span>Full size table</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Biological annotation of the 20 X-linked genes implicated by XWA for regional SA revealed significant enrichment of these genes in the following four disease ontologies: X-linked intellectual disability, intellectual disability, hydrocephalus and autism spectrum disorder (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig4">4c</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). Of note, the enrichment for X-linked intellectual disability is likely reflective of the fact that this annotation is defined by X-linkage status, which was shared by all the genes we submitted to overrepresentation analysis against a genome-wide background gene set. However, this caveat does not apply to other significant disease annotation enrichments. We also independently verified that—after restricting the analysis to only those genes that are expressed in the human brain per the Human Protein Atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Sjöstedt, E. et al. An atlas of the protein-coding genes in the human, pig, and mouse brain. Science 367, eaay5947 (2020)." href="/articles/s41593-021-00890-w#ref-CR31" id="ref-link-section-d15478171e3700">31</a></sup>—XWA significant genes were significantly enriched among a manually curated list of genes showing probable or confirmed associations with developmental disabilities by the Deciphering Developmental Disorders (DDD) project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. Nature 542, 433–438 (2017)." href="/articles/s41593-021-00890-w#ref-CR32" id="ref-link-section-d15478171e3704">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Martin, H. C. et al. The contribution of X-linked coding variation to severe developmental disorders. Nat. Commun. 12, 627 (2021)." href="/articles/s41593-021-00890-w#ref-CR33" id="ref-link-section-d15478171e3707">33</a></sup> (hypergeometric <i>P</i> = 3.24 × 10<sup>−3</sup>; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). Taken together, these analyses indicate that relative to all other brain-expressed genes, those X-linked genes highlighted in our gene-based XWA analyses are enriched for those that can cause neurodevelopmental disorders when mutated. We did not find that XWA-implicated genes were significantly enriched for DDD target genes relative to all other brain-expressed X-chromosome genes (hypergeometric <i>P</i> = 0.1). In keeping with these disease enrichments for disorders with early developmental roots, we found that this set of X-linked genes were most highly expressed in cortical tissue during prenatal life in the PsychENCODE dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Li, M. et al. Integrative functional genomic analysis of human brain development and neuropsychiatric risks. Science 362, eaat7615 (2018)." href="/articles/s41593-021-00890-w#ref-CR34" id="ref-link-section-d15478171e3723">34</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig4">4d</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). Thus, both topographic (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig3">3</a>) and genomic (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig4">4</a>) annotations of enriched XIs on human cortical anatomy converge to highlight a concentration of XIs on brain circuits and genes that are important for adaptive behavior.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec7-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec7">Discussion</h2><div class="c-article-section__content" id="Sec7-content"><p>Our study advances the understanding of genetic influences on human neuroanatomical variation in four key directions, with several implications for neuroscientific theory and practice.</p><p>First, we provide novel evidence from quantitative genetics for X-chromosome dosage compensation effects on human brain anatomy. This evidence is detectable for the majority of structural traits examined, which is in line with evidence for X-chromosome inactivation from molecular genetics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Lyon, M. F. Gene action in the X-chromosome of the mouse (Mus musculus L.). Nature 190, 372–373 (1961)." href="/articles/s41593-021-00890-w#ref-CR6" id="ref-link-section-d15478171e3752">6</a></sup>. Thus, common X-linked genetic variation tends to account for a greater proportion of phenotypic variation in brain anatomy among males compared to females. This hitherto unknown sex difference in the genetic architecture of human neuroanatomical variation echoes the male-biased penetrance of rare X-linked variation for intellectual disability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Zechner, U. et al. A high density of X-linked genes for general cognitive ability: a run-away process shaping human evolution? Trends Genet. 17, 697–701 (2001)." href="/articles/s41593-021-00890-w#ref-CR8" id="ref-link-section-d15478171e3756">8</a></sup>. Both of these phenomena could be explained by the monoallelic status of X-linked genes in males and resultant unmasking of recessive allele effects in males<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Zechner, U. et al. A high density of X-linked genes for general cognitive ability: a run-away process shaping human evolution? Trends Genet. 17, 697–701 (2001)." href="/articles/s41593-021-00890-w#ref-CR8" id="ref-link-section-d15478171e3760">8</a></sup> combined with invariance in X-chromosome parent of origin across male tissues versus the potential for mixed contribution of maternally and paternally inherited X-chromosome alleles in females<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Lyon, M. F. Gene action in the X-chromosome of the mouse (Mus musculus L.). Nature 190, 372–373 (1961)." href="/articles/s41593-021-00890-w#ref-CR6" id="ref-link-section-d15478171e3764">6</a></sup>. Such mechanisms have also been proposed to contribute to greater overall phenotypic variance in the heterogametic sex of species with chromosomal sex-determination systems (XY males in eutherian mammals, and ZW females in birds and butterflies)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Reinhold, K. &amp; Engqvist, L. The variability is in the sex chromosomes: brief communication. Evolution 67, 3662–3668 (2013)." href="/articles/s41593-021-00890-w#ref-CR35" id="ref-link-section-d15478171e3768">35</a></sup>. Our study provides new empirical support for this idea by showing that cortical regions with a larger male bias in SA variance tend to show stronger evidence of full X-chromosome dosage compensation. However, it is important to note that the small size of the X-chromosome partition means that even large sex differences in common X-linked genetic variance cannot be the sole source of those sex differences in overall phenotypic variance observed for cortical SA. An important goal for future work would be to develop approaches to estimate Y-chromosome influences on trait variation, and ask how these relate to those from the X-chromosome, especially for traits with sex-biased means or variances.</p><p>Second, by partitioning total heritability into X-linked and autosomal compartments, we directly quantified the information that can be lost by failing to consider the X-chromosome in genomic studies of brain-related phenotypes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Wise, A. L., Gyi, L. &amp; Manolio, T. A. eXclusion: toward integrating the X chromosome in genome-wide association analyses. Am. J. Hum. Genet. 92, 643–647 (2013)." href="/articles/s41593-021-00890-w#ref-CR5" id="ref-link-section-d15478171e3775">5</a></sup>. The observation that this 5% portion of the genome can account for over 20% of SNP-based heritability for some traits is particularly sobering. We note that the true magnitudes of trait heritability lost by excluding the X-chromosome are likely to be larger than the estimates provided by our study design, which excludes rare genetic variation, non-additive genetic effects and pseudoautosomal regions of the X-chromosome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Young, A. I. Solving the missing heritability problem. PLoS Genet. 15, e1008222 (2019)." href="/articles/s41593-021-00890-w#ref-CR36" id="ref-link-section-d15478171e3779">36</a></sup>. Nevertheless, our findings from the GREML analysis provide new quantitative genetic support for prior theories from earlier transcriptomic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Nguyen, D. K. &amp; Disteche, C. M. Dosage compensation of the active X chromosome in mammals. Nat. Genet. 38, 47–53 (2006)." href="/articles/s41593-021-00890-w#ref-CR7" id="ref-link-section-d15478171e3783">7</a></sup> and clinical<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Zechner, U. et al. A high density of X-linked genes for general cognitive ability: a run-away process shaping human evolution? Trends Genet. 17, 697–701 (2001)." href="/articles/s41593-021-00890-w#ref-CR8" id="ref-link-section-d15478171e3787">8</a></sup> data that predicted an enriched influence of the X-chromosome on neurodevelopmental traits.</p><p>Third, we show that enriched XIs on human brain anatomy appear to be focused on particular morphometric properties. Observed XIs on global measures of brain anatomy, such as total brain volume, total SA and overall mean CT, are in fact significantly smaller than would be predicted from the size of the X-chromosome partition, and we failed to detect evidence for enriched XIs for any regional measure of CT (with the sole exception of an isolated region within the left premotor cortex (area 6d)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e3794">14</a></sup>). In contrast, XIs are markedly enriched for relative SA in several regions of the cortical sheet (which translates into enriched XIs on several measures of regional CV). This morphometric selectivity suggests that enriched XIs on the developing brain are likely to be operating on one or more of the biological processes that are capable of shaping local SA independently of global SA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Schmitt, J. E. et al. A comprehensive quantitative genetic analysis of cerebral surface area in youth. J. Neurosci. 39, 3028–3040 (2019)." href="/articles/s41593-021-00890-w#ref-CR37" id="ref-link-section-d15478171e3798">37</a></sup>. Such processes include morphogen gradients, local rates of neuronal progenitor pool division, the degree of tangential radial migration during neuronal population of the cortical plate and the spatial patterning of thalamocortical inputs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Cadwell, C. R., Bhaduri, A., Mostajo-Radji, M. A., Keefe, M. G. &amp; Nowakowski, T. J. Development and arealization of the cerebral cortex. Neuron 103, 980–1004 (2019)." href="/articles/s41593-021-00890-w#ref-CR38" id="ref-link-section-d15478171e3802">38</a></sup>.</p><p>Fourth, to put our findings in a richer biological context, we combined two complementary approaches for annotation of observed enriched XIs on regional cortical SA: (1) comparison of the enriched XI map with independently generated functional annotations of the cortical sheet and (2) XWA analyses that identify specific X-linked genes influencing regional SA. Using recently developed surface-based methods for functional annotation of cortical maps<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. NeuroImage 178, 540–551 (2018)." href="/articles/s41593-021-00890-w#ref-CR26" id="ref-link-section-d15478171e3810">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. Cereb. Cortex 28, 281–294 (2017)." href="/articles/s41593-021-00890-w#ref-CR27" id="ref-link-section-d15478171e3813">27</a></sup>, we showed that enriched XIs on regional cortical SA are overrepresented within cortical systems involved in motor control, decision-making and attention. These annotational results do not directly demonstrate a relevance of XIs on SA for inter-individual variation in brain function but provide an anatomical context for orthogonal evidence from XWA analyses. Specifically, several of the X-linked genes we found to be linked to SA variation are known to cause intellectual disability when mutated in humans. For example, <i>RAB39B</i> encodes a GTPase that is involved in vesicular trafficking, is associated with X-linked intellectual disability, autism spectrum disorder and macrocephaly when mutated, and is thought to be involved in neurogenesis, neuronal fate and regulation of brain development<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Woodbury-Smith, M. et al. Mutations in RAB39B in individuals with intellectual disability, autism spectrum disorder, and macrocephaly. Mol. Autism 8, 59 (2017)." href="/articles/s41593-021-00890-w#ref-CR39" id="ref-link-section-d15478171e3820">39</a></sup>. Another X-linked intellectual disability risk gene identified by our XWA analyses is <i>L1CAM</i>, which encodes an axonal glycoprotein with roles in neuronal migration and differentiation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Kenwrick, S., Watkins, A. &amp; De Angelis, E. Neural cell recognition molecule L1: relating biological complexity to human disease mutations. Hum. Mol. Genet. 9, 879–886 (2000)." href="/articles/s41593-021-00890-w#ref-CR40" id="ref-link-section-d15478171e3827">40</a></sup>. The association of enriched XIs with cortical systems and genes that are important for adaptive behavior is consistent with evolutionary theories regarding enrichment of the X-chromosome for genes influencing traits that are important for reproductive fitness<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Graves, J. A. M., Gécz, J. &amp; Hameister, H. Evolution of the human X—a smart and sexy chromosome that controls speciation and development. Cytogenet. Genome Res. 99, 141–145 (2002)." href="/articles/s41593-021-00890-w#ref-CR41" id="ref-link-section-d15478171e3832">41</a></sup>. Specifically, as the X-chromosome is hemizygous in XY males, it has been theorized that X-linked variation should be subject to strong negative and positive selection for alleles that reduce and enhance reproductive fitness, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Vallender, E. J. &amp; Lahn, B. T. How mammalian sex chromosomes acquired their peculiar gene content. Bioessays 26, 159–169 (2004)." href="/articles/s41593-021-00890-w#ref-CR42" id="ref-link-section-d15478171e3836">42</a></sup>. In this context, it is notable that some of our topographic and genomic annotations linked enriched XIs on SA to higher-order cognitive processes that may be particularly important for reproductive competitiveness in humans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Zechner, U. et al. A high density of X-linked genes for general cognitive ability: a run-away process shaping human evolution? Trends Genet. 17, 697–701 (2001)." href="/articles/s41593-021-00890-w#ref-CR8" id="ref-link-section-d15478171e3840">8</a></sup>.</p><p>Taken together, our findings suggest that the very chromosome that has been conspicuously excluded from large-scale imaging genetic research in fact exerts a conspicuously large influence on several aspects of human brain anatomy. Moreover, we show that this influence is sex biased in its magnitude and involves both brain networks and genes that are known to be relevant for adaptive function in humans.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Methods</h2><div class="c-article-section__content" id="Sec8-content"><h3 class="c-article__sub-heading" id="Sec9">The UKB sample</h3><p>The UKB is a large-scale biomedical study designed to investigate the influence of genetic and environmental variables on human health and well-being<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)." href="/articles/s41593-021-00890-w#ref-CR44" id="ref-link-section-d15478171e3859">44</a></sup>. All participants provided informed consent, were genotyped and completed a myriad of questionnaires related to complex traits. A subset of participants also took part in a MRI protocol, where they attended a scanning appointment at one of three sites. Following exclusions described below, the sample size for the present study was 32,256 (52.97% female, mean age = 64.31 years). Data and materials are available via UKB (<a href="https://www.ukbiobank.ac.uk">https://www.ukbiobank.ac.uk</a>).</p><h3 class="c-article__sub-heading" id="Sec10">Image acquisition and preparation</h3><p>Detailed information regarding the UKB image acquisition process is available from the UKB website (<a href="https://biobank.ctsu.ox.ac.uk/crystal/crystal/docs/brain_mri.pdf">https://biobank.ctsu.ox.ac.uk/crystal/crystal/docs/brain_mri.pdf</a>). Imaging data were downloaded on 23 April 2020. T1-weighted and T2 FLAIR structural scans were acquired on identical Siemens Skyra 3T scanners. For T1-weighted imaging, a 3D MPRAGE sequence was used with 1-mm sagittal slices, 1 ×1-mm in-plane resolution, in-plane acceleration iPAT = 2 and prescan normalization. For T2 FLAIR imaging, a 3D SPACE sequence was used with 1.05-mm sagittal slices, 1 ×1-mm in-plane resolution, in-plane acceleration iPAT = 2, partial Fourier = 7/8, fat saturation, elliptical <i>k</i>-space scanning and prescan normalization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. NeuroImage 166, 400–424 (2018)." href="/articles/s41593-021-00890-w#ref-CR45" id="ref-link-section-d15478171e3888">45</a></sup>.</p><p>We used the PreFreeSurfer and FreeSurfer pipelines from the HCP pipeline (v.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. NeuroImage 80, 105–124 (2013)." href="/articles/s41593-021-00890-w#ref-CR46" id="ref-link-section-d15478171e3895">46</a></sup> (<a href="https://github.com/Washington-University/HCPpipelines">https://github.com/Washington-University/HCPpipelines</a>) for the data pre-processing of the T1-weighted and T2 FLAIR MRI scans. The PreFreeSurfer pipeline corrected gradient nonlinearity distortion and aligned the T1-weighted and T2 FLAIR images. The FreeSurfer pipeline used FreeSurfer (v.5.3.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Fischl, B. FreeSurfer. NeuroImage 62, 774–781 (2012)." href="/articles/s41593-021-00890-w#ref-CR47" id="ref-link-section-d15478171e3906">47</a></sup> software (<a href="https://surfer.nmr.mgh.harvard.edu">https://surfer.nmr.mgh.harvard.edu</a>) for the surface reconstruction from the T1-weighted images. Briefly, the cortical surface for each participant was reconstructed using the following steps: skull stripping<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Ségonne, F. et al. A hybrid approach to the skull stripping problem in MRI. NeuroImage 22, 1060–1075 (2004)." href="/articles/s41593-021-00890-w#ref-CR48" id="ref-link-section-d15478171e3917">48</a></sup>; segmentation of cortical gray and white matter<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. NeuroImage 166, 400–424 (2018)." href="/articles/s41593-021-00890-w#ref-CR45" id="ref-link-section-d15478171e3922">45</a></sup>; separation of the two hemispheres and subcortical structures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Fischl, B. et al. Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. Neuron 33, 341–355 (2002)." href="/articles/s41593-021-00890-w#ref-CR12" id="ref-link-section-d15478171e3926">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Fischl, B. et al. Automatically parcellating the human cerebral cortex. Cereb. Cortex 14, 11–22 (2004)." href="/articles/s41593-021-00890-w#ref-CR13" id="ref-link-section-d15478171e3929">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Dale, A. M., Fischl, B. &amp; Sereno, M. I. Cortical surface-based analysis. I. Segmentation and surface reconstruction. NeuroImage 9, 179–194 (1999)." href="/articles/s41593-021-00890-w#ref-CR49" id="ref-link-section-d15478171e3932">49</a></sup>; and, finally, construction of a smooth representation of the gray–white interface and the pial surface<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Fischl, B., Sereno, M. I. &amp; Dale, A. M. Cortical surface-based analysis. II. Inflation, flattening, and a surface-based coordinate system. NeuroImage 9, 195–207 (1999)." href="/articles/s41593-021-00890-w#ref-CR50" id="ref-link-section-d15478171e3936">50</a></sup>. To examine regional anatomical phenotypes, we used the multimodal HCP-parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-021-00890-w#ref-CR14" id="ref-link-section-d15478171e3940">14</a></sup> as our primary parcellation of interest. An alternative gyral-based (that is, anatomically driven) 308-parcellation was used for sensitivity analysis and to implement spin-based map-alignment tests (see below). To define the 308 anatomical labels within this alternative parcellation, the 68 cortical regions in the Desikan–Killiany atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. NeuroImage 31, 968–980 (2006)." href="/articles/s41593-021-00890-w#ref-CR51" id="ref-link-section-d15478171e3944">51</a></sup> were subparcellated into 308 spatially contiguous regions (152 for the left hemisphere, 156 for the right hemisphere) of approximately equal size (~5 cm<sup>2</sup>) using a backtracking algorithm as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Romero-Garcia, R., Atienza, M., Clemmensen, L. H. &amp; Cantero, J. L. Effects of network resolution on topological properties of human neocortex. NeuroImage 59, 3522–3532 (2012)." href="/articles/s41593-021-00890-w#ref-CR25" id="ref-link-section-d15478171e3951">25</a></sup>. Both the HCP-parcellation and the 308-parcellation were generated once on the surface of the standard anatomical template (fsaverage) and subsequently transformed to the surface of each individual subject. The surface parcellation for each subject was then interpolated and expanded to their respective T1-weighted volume. Regional SA, CT and CV for each parcellation was computed using Freesurfer’s ‘mris_anatomical_stats’ command and subcortical volume for the standard ‘aseg’ subcortical parcellation was calculated using the ‘mri_segstats’ command. Finally, regional measures were adjusted for their corresponding global phenotype (that is, regional CV for total brain volume, regional SA for total SA and regional CT for mean CT) separately within each sex before analysis.</p><h3 class="c-article__sub-heading" id="Sec11">Genotyping, imputation and quality control</h3><p>Genotyping and imputation procedures were centrally performed by the original UKB investigators<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-021-00890-w#ref-CR52" id="ref-link-section-d15478171e3963">52</a></sup>. Genotype data for the autosomes and X-chromosome were phased and imputed using SHAPEIT3<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="O’Connell, J. et al. Haplotype estimation for biobank-scale data sets. Nat. Genet. 48, 817–820 (2016)." href="/articles/s41593-021-00890-w#ref-CR53" id="ref-link-section-d15478171e3967">53</a></sup> and IMPUTE4<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-021-00890-w#ref-CR52" id="ref-link-section-d15478171e3971">52</a></sup>, respectively. For the X-chromosome, the pseudoautosomal regions and non-pseudoautosomal regions were phased and imputed independently, applying the same approach used for the autosomes. As the genotyping, haplotype estimation (phasing) and imputation procedures for all genetic markers have already been extensively described by the original UKB investigators, we refer the reader to their previously published study for further details<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-021-00890-w#ref-CR52" id="ref-link-section-d15478171e3975">52</a></sup>. Information regarding these procedures is also available at <a href="https://www.ukbiobank.ac.uk/scientists-3/genetic-data/">https://www.ukbiobank.ac.uk/scientists-3/genetic-data/</a>.</p><p>In the present study, we followed best practices for estimating heritability and applied additional quality control filters to this aim. First, using the quality control variables provided by the UKB investigators, we removed participants with putative sex chromosome aneuploidy (as the original UKB investigators inferred from the measured intensities of chromosomes X and Y)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41593-021-00890-w#ref-CR52" id="ref-link-section-d15478171e3989">52</a></sup>, excessive heterozygosity (that is, poor quality genotype data) and mismatched self-reported sex and chromosomal sex. Next, we restricted the sample to participants of non-Hispanic European ancestry with processed neuroimaging data (<i>n</i> = 33,870) to minimize the potential influence of population stratification. We then applied standard thresholds to the imputed variants, excluding those with (1) an imputation score of less than 0.30, (2) a MAF of less than 0.0003, (3) a Hardy–Weinberg equilibrium <i>P</i> &lt; 1 × 10<sup>−6</sup> and/or (4) three or more observed alleles (that is, multiallelic sites). We also excluded the pseudoautosomal regions from analysis as their very small size precluded a reliable estimation of partitioned heritability and statistical enrichment (and they cannot be merged with the non-pseudoautosomal region in GCTA). We note that imputation and MAF thresholds were established based on previous research on optimal parameters for estimating heritability from genomic data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Evans, L. M. et al. Comparison of methods that use whole genome data to estimate the heritability and genetic architecture of complex traits. Nat. Genet. 50, 737–745 (2018)." href="/articles/s41593-021-00890-w#ref-CR22" id="ref-link-section-d15478171e4001">22</a></sup>.</p><p>The sample was then pruned for missingness, removing participants missing 5% or more of genotypes; however, no samples met this criterion after applying the variant-level filters described above. In preparation for the heritability analyses, the sample was pruned for relatedness, removing one participant from a related pair if estimated relatedness exceeded 0.05 (note that the pruning algorithm selectively removed individuals to maximize the remaining sample size). This resulted in the removal of 1,606 participants. Finally, we removed eight participants who had values of zero for any of the regional phenotypes. These preprocessing steps yielded a final sample of 32,256 participants with information on 16,873,171 SNPs (16,255,343 autosomal SNPs and 617,828 X-chromosome SNPs) spanning the genome.</p><h3 class="c-article__sub-heading" id="Sec12">Genotype coding</h3><p>The genetic analyses reported in this study were conducted using PLINK (v.2.00)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. Gigascience 4, 7 (2015)." href="/articles/s41593-021-00890-w#ref-CR54" id="ref-link-section-d15478171e4016">54</a></sup> and GCTA (v.192.1beta6)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011)." href="/articles/s41593-021-00890-w#ref-CR11" id="ref-link-section-d15478171e4020">11</a></sup>. During quality control and preprocessing, imputed dosages were converted to genotype hard calls using the default settings in PLINK, where calls with an uncertainty greater than 0.10 were treated as missing. Both PLINK and GCTA handle the X-chromosome in a similar manner. In GCTA, the X-chromosome genetic relationship matrix (GRM) was constructed with females coded as diploid (0, 1 or 2) and males coded as homozygous diploid (0 or 2), as described below. The GRM was then parameterized under three different assumptions related to genetic variance on the X-chromosome (that is, dosage compensation), and the best-fitting model was selected (also described below). In PLINK, X-chromosome association analyses were conducted by analyzing males as homozygous diploid (0 or 2), which is in accordance with the evidence of full dosage compensation for those phenotypes.</p><h3 class="c-article__sub-heading" id="Sec13">GRMs</h3><p>We used GCTA (v.1.92.1beta6) to construct autosomal and X-chromosome GRMs for the sample, which quantified genetic similarity between all individuals in the sample. First, the genetic relationship between individuals <i>j</i> and <i>k</i> was estimated as <span class="mathjax-tex">\(A_{jk} = \frac{1}{N}\mathop {\sum }\limits_{i = {\mathrm{1}}}^N \frac{{\left( {x_{ij} - {\mathrm{2}}p_i} \right)\left( {x_{ik} - {\mathrm{2}}p_i} \right)}}{{2p_i\left( {{\mathrm{1}} - p_i} \right)}}\)</span> for autosomal SNPs, where <i>N</i> refers to the number of SNPs and <i>x</i> and <i>p</i> refer to the number of copies (0, 1 or 2) and the frequency of the reference allele for SNP <i>i</i>. This equation was then modified to estimate the genetic relationship based on the X-chromosome that accounts for the fact that males have only one X-chromosome. As described by Yang and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011)." href="/articles/s41593-021-00890-w#ref-CR11" id="ref-link-section-d15478171e4249">11</a></sup>, the original equation was modified such that genetic relationships were estimated as <span class="mathjax-tex">\(A_{jk}^{\mathrm{F}} = \mathop {\sum }\limits_{i = {\mathrm{1}}}^N \frac{{\left( {x_{ij}^{\mathrm{F}} - {\mathrm{2}}p_i} \right)\left( {x_{ik}^{\mathrm{F}} - {\mathrm{2}}p_i} \right)}}{{2p_i\left( {{\mathrm{1}} - p_i} \right)}}\)</span> for a female–female pair, <span class="mathjax-tex">\(A_{jk}^{\mathrm{M}} = \mathop {\sum }\limits_{i = {\mathrm{1}}}^N \frac{{\left( {x_{ij}^{\mathrm{M}} - p_i} \right)\left( {x_{ik}^{\mathrm{M}} - p_i} \right)}}{{p_i\left( {{\mathrm{1}} - p_i} \right)}}\)</span> for a male–male pair, and <span class="mathjax-tex">\(A_{jk}^{{\mathrm{FM}}} = \mathop {\sum }\limits_{i = {\mathrm{1}}}^N \frac{{\left( {x_{ij}^{\mathrm{F}} - {\mathrm{2}}p_i} \right)\left( {x_{ik}^{\mathrm{M}} - p_i} \right)}}{{\sqrt 2 p_i\left( {{\mathrm{1}} - p_i} \right)}}\)</span> for a female–male pair.</p><h3 class="c-article__sub-heading" id="Sec14">GREML</h3><p>We also used GCTA (v.1.92.1beta6) to estimate and partition total SNP-based heritability for the 1,091 morphological phenotypes via the GREML method. To estimate and partition heritability across the autosomes and the X-chromosome, we began with a single-component GREML analysis in GCTA. This mixed-linear model can be expressed as <b>y</b> = <b>Xβ</b> + <b>g</b><sub><b>a</b></sub> + <b>ε</b> with <span class="mathjax-tex">\(\text{var(}\mathbf{y}\text{) = V = } A_a\sigma _a^2 + I\sigma _\varepsilon ^2\)</span>, where <b>y</b> is a <i>n</i> × 1 vector of standardized phenotypes with sample size <i>n</i>, <b>β</b> is a vector of fixed effects for covariates (here, sex, age, the first 20 principal components of ancestry, genotyping batch and scanning site), <b>g</b><sub><b>a</b></sub> is a <i>n</i> × 1 vector of genetic effects attributable to the autosomes, <b>ε</b> is a vector of residual effects, <i>A</i> is a GRM that quantifies the genetic similarity between individuals in the sample, <i>I</i> is a <i>n</i> × <i>n</i> identity matrix and <span class="mathjax-tex">\(\sigma _a^2\)</span> is the variance explained by all autosomal SNPs. Following this model, the restricted maximum likelihood method was used to estimate <span class="mathjax-tex">\(\sigma _a^2\)</span>. Note that heritability estimates were derived from a model where variance component estimates were constrained to be greater than zero for the sake of interpretation, as negative variance estimates can skew the proportion of variance explained by SNPs in a given partition.</p><p>The basic model was then extended to estimate the variance explained by the X-chromosome (<span class="mathjax-tex">\(\sigma _{\mathrm{X}}^2\)</span>), such that <b>y</b> = <b>Xβ</b> + <b>g</b><sub><b>a</b></sub> + <b>g</b><sub><b>X</b></sub> + <b>ε</b>, where <b>g</b><sub><b>X</b></sub> is a vector of genetic effects attributable to the X-chromosome. To evaluate evidence of dosage compensation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. Am. J. Hum. Genet. 88, 76–82 (2011)." href="/articles/s41593-021-00890-w#ref-CR11" id="ref-link-section-d15478171e5128">11</a></sup>, we used GCTA to fit this mixed-linear model under three different assumptions for the variance of <b>g</b><sub><b>X</b></sub>: (1) no dosage compensation, where variance equals <span class="mathjax-tex">\(A_{\mathrm{X}}^{{\mathrm{ND}}}\sigma _{\mathrm{X}}^2\)</span>; (2) full dosage compensation, where variance equals <span class="mathjax-tex">\(A_{\mathrm{X}}^{{\mathrm{FD}}}\sigma _{\mathrm{X}}^2\)</span>; and (3) equal variance for males and females, where variance equals <span class="mathjax-tex">\(A_{\mathrm{X}}\sigma _{\mathrm{X}}^2\)</span>. Model log likelihood (<i>L</i>) and Akaike information criterion (AIC) were used to identify the model that best fit the data. The latter was calculated as AIC = −2<i>L</i> + 2<i>k</i>, where <i>L</i> is the log likelihood and <i>k</i> is the number of parameters estimated in each model (note that <i>k</i> is invariant across models). Differences in AIC (ΔAIC) were then computed to further quantify the degree of evidence for the optimal model relative to the equal variance model (that is, the theoretical null model), where ΔAIC = ΔAIC<sub>EV</sub> – ΔAIC<sub>min</sub>. As established in information theory, a difference of ten or greater reflects strong evidence for the optimal model minimizing AIC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Burnham, K. P. &amp; Anderson, D. R. Model Selection and Multimodel Inference: A Practical Information-Theoretic Approach. (Springer Science &amp; Business Media, 2002)." href="/articles/s41593-021-00890-w#ref-CR55" id="ref-link-section-d15478171e5327">55</a></sup>. Parameter estimates from the best fitting model were carried forth and used in subsequent analyses.</p><p>For the purposes of plotting and visualization, we also created a dimensional index of the degree of support for the possible dosage compensation models. Specifically, we adjusted the ΔAIC statistic by multiplying it by −1 when the best fitting model was the no dosage compensation model. As such, this modified statistic, which we call ΔAIC<sub>DC</sub>, is equal to zero when the equal variance model fits best. Departures from zero reflect quantitative and dimensional evidence for either no dosage compensation (negative values) or full dosage compensation (positive values). Accordingly, ΔAIC<sub>DC</sub> values ≤ −10 are indicative of strong evidence for no dosage compensation while values ≥ +10 are indicative of strong evidence for full dosage compensation.</p><h3 class="c-article__sub-heading" id="Sec15">Sex-stratified heritability analysis</h3><p>To provide a complementary test of male–female differences in X-linked genetic variance, we used sex-stratified analyses to separately estimate heritability in males and females for the 920 cortical and subcortical ROIs where the full dosage compensation model fit best. In these sex-stratified GREML analyses, we employed the same procedure used to produce our final estimates of total and partitioned heritability, but we performed the analysis in a manner naive to dosage compensation. A nonparametric Wilcoxon rank-sum test was then used to test whether the X-linked heritability estimates in female-only and male-only subsamples were similar distributions with equivalent medians. Finally, we calculated the ratio of the medians of this distribution and used bootstrapping to compute a 95% CI for this ratio.</p><h3 class="c-article__sub-heading" id="Sec16">Sex ratio of cortical SA variance</h3><p>We calculated the phenotypic variance of cortical SA in males and females separately. To parallel the primary analysis, the regional SA phenotypes were adjusted for total SA, the first 20 principal components of ancestry, sex, age, genotyping batch and scanning site. The phenotypic variance ratio was then computed as <span class="mathjax-tex">\(\frac{{\text{V}_{{\mathrm{PM}}}}}{{\text{V}_{{\mathrm{PF}}}}}\)</span>, where V<sub>PM</sub> and V<sub>PF</sub> refer to the phenotypic variance in males and females, respectively.</p><h3 class="c-article__sub-heading" id="Sec17">Enrichment tests</h3><p>Here, we defined enrichment as the proportion of heritability in the partition divided by the proportion of genomic base pairs that reside on the partition, such that the XI ratio for a given phenotype is <span class="mathjax-tex">\(\frac{{(h_{\mathrm{X}}^2/h_{\mathrm{g}}^2)}}{{({\mathrm{length}}_{\mathrm{X}}/{\mathrm{length}}_{\mathrm{g}})}}\)</span>. This definition is analogous to previous definitions of enriched heritability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. Nat. Genet. 47, 1228–1235 (2015)." href="/articles/s41593-021-00890-w#ref-CR20" id="ref-link-section-d15478171e5536">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Finucane, H. K. et al. Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types. Nat. Genet. 50, 621–629 (2018)." href="/articles/s41593-021-00890-w#ref-CR56" id="ref-link-section-d15478171e5539">56</a></sup>, and establishes a partition-specific null that follows the theoretical assumption of equal per-SNP heritability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Speed, D., Hemani, G., Johnson, M. R. &amp; Balding, D. J. Improved heritability estimation from genome-wide SNPs. Am. J. Hum. Genet. 91, 1011–1021 (2012)." href="/articles/s41593-021-00890-w#ref-CR57" id="ref-link-section-d15478171e5543">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Yang, J. et al. Common SNPs explain a large proportion of the heritability for human height. Nat. Genet. 42, 565–569 (2010)." href="/articles/s41593-021-00890-w#ref-CR58" id="ref-link-section-d15478171e5546">58</a></sup>. To determine whether the observed heritability for a given partition was statistically different than what would be expected given its genomic length, we used a one-sample enrichment <i>Z</i>-test that was monotonically related to the enrichment ratio (<i>ρ</i> = 0.95), where <span class="mathjax-tex">\(Z = \frac{{({\mathrm{observed}}\,h_{\mathrm{X}}^2 - {\mathrm{expected}}\,h_{\mathrm{X}}^2)}}{{{\mathrm{standard}}\,{\mathrm{error}}}}\)</span>. This enrichment <i>Z</i>-statistic was then used to obtain a corresponding <i>P</i> value, which was subsequently adjusted for the false discovery rate (FDR). For each chromosome, the proportion of genomic base pairs residing within the partition was calculated using assembly statistics for Genome Reference Consortium Human Build 37 (GRCh37) available at <a href="https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.13/">https://www.ncbi.nlm.nih.gov/assembly/GCF_000001405.13/</a>.</p><h3 class="c-article__sub-heading" id="Sec18">Extended partitioned heritability analyses</h3><p>To verify that the heritability estimate for a given chromosome was largely proportional to its genomic length (as previously reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Biton, A. et al. Polygenic architecture of human neuroanatomical diversity. Cereb. Cortex 30, 2307–2320 (2020)." href="/articles/s41593-021-00890-w#ref-CR2" id="ref-link-section-d15478171e5678">2</a></sup>), we conducted extended GREML analyses that included a genetic variance component for each chromosome. Specifically, for the 56 regional SA phenotypes with significant enrichment of X-chromosome heritability, we extended the final GREML model to include a genetic variance component for each of the 23 chromosomes (22 autosomes and the X-chromosome). This yielded estimates of heritability per chromosome <span class="mathjax-tex">\(h_{\mathrm{c}}^2\)</span> for each region. We then computed the mean and 95% CIs for each partition’s <span class="mathjax-tex">\(h_{\mathrm{c}}^2\)</span> estimate by averaging across regions, and plotted those values as a linear function of chromosome length. The line of best fit for these values showed a strong concordance to the theoretical expectation based on equal per-SNP heritability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Speed, D., Hemani, G., Johnson, M. R. &amp; Balding, D. J. Improved heritability estimation from genome-wide SNPs. Am. J. Hum. Genet. 91, 1011–1021 (2012)." href="/articles/s41593-021-00890-w#ref-CR57" id="ref-link-section-d15478171e5750">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Yang, J. et al. Common SNPs explain a large proportion of the heritability for human height. Nat. Genet. 42, 565–569 (2010)." href="/articles/s41593-021-00890-w#ref-CR58" id="ref-link-section-d15478171e5753">58</a></sup>, and only the X-chromosome deviated substantially from the expected value.</p><h3 class="c-article__sub-heading" id="Sec19">Sensitivity analyses</h3><p>We performed three additional analyses to interrogate the robustness of the final results for the 358 regional SA phenotypes: (1) release-stratified GREML (that is, two subsamples corresponding to the first and second releases of UKB neuroimaging data); (2) repeated GREML in 100 random split-halves of the sample; and (3) GREML with a more conservative MAF threshold (0.001 versus 0.0003). In the release-stratified GREML analyses, we re-estimated model parameters and computed the enrichment <i>Z</i>-statistics in each subsample. We then computed the cross-ROI correlation of those <i>Z</i>-statistics across releases. In the repeated GREML analyses, we re-estimated model parameters and computed the enrichment <i>Z</i>-statistics in 100 random split-halves of the sample, holding the ratio of females and males to be the same in both matching halves. For this analysis, we computed the cross-ROI correlation for each pair of split-halves, which generated a distribution of correlation values that further indexed the reliability of our results. Finally, in the GREML analyses employing a more conservative MAF threshold, we applied the same quality control procedures and conducted the same statistical analysis with one exception: we filtered out SNPs with a MAF less than 0.001 before constructing the GRMs. Here, the cross-ROI correlation of enrichment <i>Z</i>-statistics across MAF thresholds was computed.</p><p>To test whether the cortical parcellation scheme influenced the estimates of regional XIs on SA, we also repeated the final GREML analyses for SA using a 308-region cortical parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Romero-Garcia, R., Atienza, M., Clemmensen, L. H. &amp; Cantero, J. L. Effects of network resolution on topological properties of human neocortex. NeuroImage 59, 3522–3532 (2012)." href="/articles/s41593-021-00890-w#ref-CR25" id="ref-link-section-d15478171e5781">25</a></sup> that has been widely used in prior work. Also, a pre-calculated scheme based on rotations of this 308-region parcellation provides a nonparametric permutation test of the significance of spatial correlations between two regional maps while accounting for the shared spatial autocorrelation and the arbitrary variations in power that would otherwise arise from changes in spatial resolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. NeuroImage 178, 540–551 (2018)." href="/articles/s41593-021-00890-w#ref-CR26" id="ref-link-section-d15478171e5785">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. Cereb. Cortex 28, 281–294 (2017)." href="/articles/s41593-021-00890-w#ref-CR27" id="ref-link-section-d15478171e5788">27</a></sup>. Based on the coordinates of the HCP and 308 parcellations defined in the shared FreeSurfer cortical space, we projected the XI <i>Z</i>-statistics for SA in the HCP-parcellation to the 308-parcellation (by averaging the values in each of the 308 parcels) and calculated the cross-parcel Pearson’s correlation between these XI <i>Z</i>-statistics for SA (that is, projected from the HCP) and those directly calculated in the 308 parcellation. The statistical significance of this observed correlation was determined by the nonparametric spin test described below.</p><h3 class="c-article__sub-heading" id="Sec20">Spin tests</h3><p>To assess the specificity of the correspondence between pairs of cortical maps, we generated 10,000 rotations (that is, spins) of the cortical parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. Cereb. Cortex 28, 281–294 (2017)." href="/articles/s41593-021-00890-w#ref-CR27" id="ref-link-section-d15478171e5806">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Seidlitz, J. et al. Morphometric similarity networks detect microscale cortical organization and predict inter-individual cognitive variation. Neuron 97, 231–247.e7 (2018)." href="/articles/s41593-021-00890-w#ref-CR59" id="ref-link-section-d15478171e5809">59</a></sup>. A measure of observed spatial correspondence between two maps of interest could then be compared against the null distribution of measures generated from using the spun maps. Importantly, the spin approach allows any regional measure to be permuted while controlling for spatial contiguity and hemispheric symmetry. We first obtained the spherical surface coordinates for the centroid of each ROI within the 308 cortical regions on the fsaverage template in FreeSurfer. These were then rotated about the three principal axes at three randomly generated angles. Given the separate left- and right-hemisphere cortical projections, and to preserve symmetry, the same random angles were applied to both hemispheres with the caveat that the sign of the angles was flipped for the rotation around the <i>y</i> and <i>z</i> axes. Following each rotation, the coordinates of the rotated regions were matched to the coordinates of the initial regions using Euclidean distance, proceeding in descending order of average Euclidean distance between pairs of regions on the rotated and unrotated spheres (that is, starting with the rotated region that is furthest away, on average, from the unrotated regions).</p><h3 class="c-article__sub-heading" id="Sec21">Topographic annotation of brain regions with enriched X-linked heritability</h3><p>We examined the spatial coupling between enriched XIs on cortical SA and topography of the Yeo–Krienen cortical parcellation, which classifies 17 functional intrinsic connectivity networks derived from resting-state functional MRI data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-021-00890-w#ref-CR28" id="ref-link-section-d15478171e5827">28</a></sup>. We assigned a label of 1 to 17 from the Yeo–Krienen parcellation to each parcel of the 308-parcellation by identifying the maximum overlap among the 17 networks with that parcel. A primary omnibus test first was used to examine whether the spatial distribution of XIs on SA was related to topography of the Yeo–Krienen-parcellation. To achieve this test, we recorded the observed <i>F</i>-test value of a regression model with XI <i>Z</i>-statistics for regional SA as the dependent variable, and the Yeo–Krienen assignments as the independent variable. We then compared this observed <i>F</i> value against a distribution of 10,000 null <i>F</i> values from repeating the above regression models using spun versions of XI <i>Z</i>-statistics for regional SA (see ‘Spin tests’ above). Given a detection of a significant omnibus association, follow-up post hoc spin tests were used to identify which specific components of the Yeo–Krienen-parcellation showed overlaps with enriched XIs on SA. Specifically, we used Fisher’s exact tests to compute the observed odds ratio for overlap between each of the 17 Yeo–Krienen networks and regions of significant XI enrichment for SA. A nominal <i>P</i> value was derived for each network by comparing its observed odds ratio with 10,000 null odds ratios from spun versions of significant XI enrichment for SA.</p><p>We also compared the spatial pattern of enriched XIs on SA to functional anatomy of the human brain as described by Neurosynth (<a href="https://neurosynth.org">https://neurosynth.org</a>), an online platform for meta-analysis of functional neuroimaging literature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. Nat. Methods 8, 665–670 (2011)." href="/articles/s41593-021-00890-w#ref-CR29" id="ref-link-section-d15478171e5860">29</a></sup>. Previously, a topic modeling technique was applied to the text of over 11,000 studies in the Neurosynth database to define a set of 50 topics capturing conceptually distinct aspects of human cognition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Poldrack, R. A. et al. Discovering relations between mind, brain, and mental disorders using topic mapping. PLoS Comput. Biol. 8, e1002707 (2012)." href="/articles/s41593-021-00890-w#ref-CR60" id="ref-link-section-d15478171e5864">60</a></sup>. Each topic represents a cluster of weighted text terms with shared relevance for a common overarching cognitive construct and is tagged with a specific association test map, in which each voxel’s association statistic (<i>Z</i>-score) represents the likelihood that that voxel is preferentially activated by the topic in question over all other topics. To exclude association test maps that lack meaningful results, we applied an empirical filter to these 50 maps and retained 30 significant maps with at least one cluster of greater than 100 contiguous voxels after correcting for FDR.</p><p>For each of these 30 Neurosynth association test maps, we assigned each parcel of 308 to 1 if there was an overlap between the significant regions in the association test map and that parcel, to 0 if not. Again, a primary omnibus test was used to determine whether the spatial distribution of XIs on SA was non-randomly related to the topography of regional cortical activation for the 30 Neurosynth topics. Specifically, for each Neurosynth topic term, we calculated the <i>F</i>-test value of a regression model with XI <i>Z</i>-statistics for regional SA as the dependent variable and the binary Neurosynth assignments for that term in the 308 parcellation as the independent variable. We then compared the average of these <i>F</i> values across all 30 Neurosynth topics, with the null distribution of this average <i>F</i> value from 10,000 spins of XI <i>Z</i>-statistics for regional SA. Given a detection of a significant omnibus association, follow-up post hoc spin tests were used to identify which specific Neurosynth topic maps showed an overlap with enriched XIs on SA. Specifically, we used Fisher’s exact tests to compute the observed odds ratio for overlap between each Neurosynth topic map and regions of significant XI enrichment for SA. A nominal <i>P</i> value was derived for each Neurosynth topic by comparing its observed odds ratio with 10,000 null odds ratios from spun versions of significant XI enrichment for SA. Given the aim of annotating enriched XIs on SA, the omnibus tests above excluded those regions with evidence of significant depletion of XIs on SA (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-021-00890-w#Fig12">8</a>).</p><h3 class="c-article__sub-heading" id="Sec22">X-chromosome association analyses</h3><p>We used PLINK (v.2.00) to conduct a series of SNP-based XWA tests for SA phenotypes with significantly enriched X-chromosome heritability. As previously noted, females were coded as diploid (0, 1 or 2) and males were analyzed as homozygous diploid (0 or 2) in accordance with the full dosage compensation model. The final analytic sample in the GREML analyses (that is, the unrelated imaging sample) was selected as the analytic sample for association analyses. Hard call genotypes were analyzed by fitting the linear model <i>y</i> = <i>Xβ</i><sub><i>X</i></sub> + <i>Gβ</i><sub><i>G</i></sub> + <i>ε</i> for each SNP, where <i>y</i> is the phenotype, <i>X</i> is the covariate matrix, <i>G</i> is the genotype matrix for the target SNP and <i>ε</i> is the error term. Paralleling the GREML analyses, the first 20 principal components of ancestry, sex, age, genotyping batch and scanning site were included as covariates. We then used an established EasyQC (v.9.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Winkler, T. W. et al. Quality control and conduct of genome-wide association meta-analyses. Nat. Protoc. 9, 1192–1212 (2014)." href="/articles/s41593-021-00890-w#ref-CR61" id="ref-link-section-d15478171e5935">61</a></sup> pipeline to perform additional quality control on the GWAS summary statistics, as described in previous studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Mallard, T. T. et al. Multivariate GWAS of psychiatric disorders and their cardinal symptoms reveal two dimensions of cross-cutting genetic liabilities. Preprint at bioRxiv &#xA;                  https://doi.org/10.1101/603134&#xA;                  &#xA;                 (2020)." href="/articles/s41593-021-00890-w#ref-CR62" id="ref-link-section-d15478171e5939">62</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Mallard, T. T. et al. tem-level genome-wide association study of the Alcohol Use Disorders Identification Test in three population-based cohorts. Am. J. Psychiatry &#xA;                  https://doi.org/10.1176/appi.ajp.2020.20091390&#xA;                  &#xA;                 (2021)." href="/articles/s41593-021-00890-w#ref-CR63" id="ref-link-section-d15478171e5942">63</a></sup>. Here, we filtered out rare and low-frequency SNPs (MAF &lt; 0.005), SNPs that were not imputed well (imputation score &lt; 0.60), SNPs that were unavailable or discordant with the reference panel (a combination of 1000 Genomes Phase 3 v.5 and UK10K data) or SNPs that had implausible values (for example, <i>P</i> &gt; 1). There were 240,400 X-chromosome SNPs suitable for gene-based testing after applying these quality control filters. Finally, after subjecting the XWA summary statistics to quality control, we used MAGMA (v.1.08)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="de Leeuw, C. A., Mooij, J. M., Heskes, T. &amp; Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. PLoS Comput. Biol. 11, e1004219 (2015)." href="/articles/s41593-021-00890-w#ref-CR64" id="ref-link-section-d15478171e5950">64</a></sup> to conduct gene-based association analyses for 724 X-linked genes. Here, we used default MAGMA parameters and followed standard procedures for gene-based association analyses as implemented in FUMA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Watanabe, K., Taskesen, E., van Bochoven, A. &amp; Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. Nat. Commun. 8, 1826 (2017)." href="/articles/s41593-021-00890-w#ref-CR65" id="ref-link-section-d15478171e5954">65</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec23">Biological annotation of associated genes</h3><p>The disease relevance of the genes identified in gene-based XWA analyses was investigated via Enrichr<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. Nucleic Acids Res. 44, W90–W97 (2016)." href="/articles/s41593-021-00890-w#ref-CR66" id="ref-link-section-d15478171e5966">66</a></sup>, which used Fisher’s exact test to evaluate whether the XWA gene set was enriched in gene–disease ontologies indexed by DisGeNET<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Piñero, J. et al. The DisGeNET knowledge platform for disease genomics: 2019 update. Nucleic Acids Res. 48, D845–D855 (2020)." href="/articles/s41593-021-00890-w#ref-CR67" id="ref-link-section-d15478171e5970">67</a></sup>. In Enrichr, the human genome is used as the ‘background’ set and <i>P</i> values were corrected for FDR. To provide an independent and more conservative test of enrichment, we used a hypergeometric test to evaluate whether the XWA gene set was enriched for a curated list of genes implicated in developmental disorders by the DDD project (‘probable’ or ‘confirmed’ developmental disorder genes)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. Nature 542, 433–438 (2017)." href="/articles/s41593-021-00890-w#ref-CR32" id="ref-link-section-d15478171e5977">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Martin, H. C. et al. The contribution of X-linked coding variation to severe developmental disorders. Nat. Commun. 12, 627 (2021)." href="/articles/s41593-021-00890-w#ref-CR33" id="ref-link-section-d15478171e5980">33</a></sup>. Here, we identified brain-expressed genes (per the Human Protein Atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Sjöstedt, E. et al. An atlas of the protein-coding genes in the human, pig, and mouse brain. Science 367, eaay5947 (2020)." href="/articles/s41593-021-00890-w#ref-CR31" id="ref-link-section-d15478171e5984">31</a></sup>; <a href="https://www.proteinatlas.org/humanproteome/brain">https://www.proteinatlas.org/humanproteome/brain</a>) as the background set to conduct a more conservative test of enrichment.</p><p>To further evaluate the developmental relevance of the identified genes, we characterized their expression trajectory in the PsychENCODE dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Li, M. et al. Integrative functional genomic analysis of human brain development and neuropsychiatric risks. Science 362, eaat7615 (2018)." href="/articles/s41593-021-00890-w#ref-CR34" id="ref-link-section-d15478171e5999">34</a></sup>. Specifically, we applied weighted gene correlation network analysis (using the WGCNA (v.1.69) package in R)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Li, M. et al. Integrative functional genomic analysis of human brain development and neuropsychiatric risks. Science 362, eaat7615 (2018)." href="/articles/s41593-021-00890-w#ref-CR34" id="ref-link-section-d15478171e6003">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Langfelder, P. &amp; Horvath, S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics 9, 559 (2008)." href="/articles/s41593-021-00890-w#ref-CR68" id="ref-link-section-d15478171e6006">68</a></sup> to the PsychENCODE data to compute a weighted average expression profile (eigengene), treating the gene set as a module. We then plotted expression as a function of sample age, using a locally estimated scatter plot smoothing line-of-best-fit to characterize the temporal pattern of gene set expression in cortical tissue. Inspection of this line (and the accompanying 95% CI) indicated that these genes were differentially expressed across development.</p><h3 class="c-article__sub-heading" id="Sec24">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-021-00890-w#MOESM1">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                </div>
            

            <div>
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>No data were collected as part of this study. All genetic and phenotypic data (including neuroimaging data) are available from the UKB via their standard data access procedure, as described at <a href="https://www.ukbiobank.ac.uk/register-apply">https://www.ukbiobank.ac.uk/register-apply</a>. The Neurosynth data for topic-based meta-analyses are available at <a href="https://neurosynth.org/analyses/topics/">https://neurosynth.org/analyses/topics/</a>. The PsychENCODE data for developmental expression analyses are available at <a href="http://development.psychencode.org">http://development.psychencode.org</a>. XWA summary statistics are available at <a href="https://osf.io/cqxdj/">https://osf.io/cqxdj/</a>.</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>No custom software was used in this study. All relevant software and code are described in the text and can be found at the URLs or references cited.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Grasby, K. L. et al. The genetic architecture of the human cerebral cortex. <i>Science</i> <b>367</b>, eaay6690 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aay6690" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aay6690" aria-label="Article reference 1" data-doi="10.1126/science.aay6690">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXltl2nurs%3D" aria-label="CAS reference 1">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32193296" aria-label="PubMed reference 1">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7295264" aria-label="PubMed Central reference 1">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genetic%20architecture%20of%20the%20human%20cerebral%20cortex&amp;journal=Science&amp;doi=10.1126%2Fscience.aay6690&amp;volume=367&amp;publication_year=2020&amp;author=Grasby%2CKL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Biton, A. et al. Polygenic architecture of human neuroanatomical diversity. <i>Cereb. Cortex</i> <b>30</b>, 2307–2320 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhz241" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhz241" aria-label="Article reference 2" data-doi="10.1093/cercor/bhz241">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32109272" aria-label="PubMed reference 2">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7175006" aria-label="PubMed Central reference 2">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20architecture%20of%20human%20neuroanatomical%20diversity&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhz241&amp;volume=30&amp;pages=2307-2320&amp;publication_year=2020&amp;author=Biton%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Zhao, B. et al. Genome-wide association analysis of 19,629 individuals identifies variants influencing regional brain volumes and refines their genetic co-architecture with cognitive and mental health traits. <i>Nat. Genet.</i> <b>51</b>, 1637–1644 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41588-019-0516-6" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0516-6" aria-label="Article reference 3" data-doi="10.1038/s41588-019-0516-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFSjsrnN" aria-label="CAS reference 3">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31676860" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858580" aria-label="PubMed Central reference 3">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20analysis%20of%2019%2C629%20individuals%20identifies%20variants%20influencing%20regional%20brain%20volumes%20and%20refines%20their%20genetic%20co-architecture%20with%20cognitive%20and%20mental%20health%20traits&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0516-6&amp;volume=51&amp;pages=1637-1644&amp;publication_year=2019&amp;author=Zhao%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Smith, S. M. et al. An expanded set of genome-wide association studies of brain imaging phenotypes in UK Biobank. <i>Nat. Neurosci.</i> <b>24</b>, 737–745 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41593-021-00826-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41593-021-00826-4" aria-label="Article reference 4" data-doi="10.1038/s41593-021-00826-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXptFWisL4%3D" aria-label="CAS reference 4">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33875891" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7610742" aria-label="PubMed Central reference 4">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20expanded%20set%20of%20genome-wide%20association%20studies%20of%20brain%20imaging%20phenotypes%20in%20UK%20Biobank&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fs41593-021-00826-4&amp;volume=24&amp;pages=737-745&amp;publication_year=2021&amp;author=Smith%2CSM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Wise, A. L., Gyi, L. &amp; Manolio, T. A. eXclusion: toward integrating the X chromosome in genome-wide association analyses. <i>Am. J. Hum. Genet.</i> <b>92</b>, 643–647 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2013.03.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2013.03.017" aria-label="Article reference 5" data-doi="10.1016/j.ajhg.2013.03.017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXmvF2ksro%3D" aria-label="CAS reference 5">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23643377" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3644627" aria-label="PubMed Central reference 5">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=eXclusion%3A%20toward%20integrating%20the%20X%20chromosome%20in%20genome-wide%20association%20analyses&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2013.03.017&amp;volume=92&amp;pages=643-647&amp;publication_year=2013&amp;author=Wise%2CAL&amp;author=Gyi%2CL&amp;author=Manolio%2CTA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Lyon, M. F. Gene action in the X-chromosome of the mouse (<i>Mus musculus</i> L.). <i>Nature</i> <b>190</b>, 372–373 (1961).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/190372a0" data-track-action="article reference" href="https://doi.org/10.1038%2F190372a0" aria-label="Article reference 6" data-doi="10.1038/190372a0">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaF3c%2FosVGksw%3D%3D" aria-label="CAS reference 6">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=13764598" aria-label="PubMed reference 6">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20action%20in%20the%20X-chromosome%20of%20the%20mouse%20%28Mus%20musculus%20L.%29&amp;journal=Nature&amp;doi=10.1038%2F190372a0&amp;volume=190&amp;pages=372-373&amp;publication_year=1961&amp;author=Lyon%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Nguyen, D. K. &amp; Disteche, C. M. Dosage compensation of the active X chromosome in mammals. <i>Nat. Genet.</i> <b>38</b>, 47–53 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng1705" data-track-action="article reference" href="https://doi.org/10.1038%2Fng1705" aria-label="Article reference 7" data-doi="10.1038/ng1705">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtlCmtrzM" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16341221" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Dosage%20compensation%20of%20the%20active%20X%20chromosome%20in%20mammals&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng1705&amp;volume=38&amp;pages=47-53&amp;publication_year=2006&amp;author=Nguyen%2CDK&amp;author=Disteche%2CCM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Zechner, U. et al. A high density of X-linked genes for general cognitive ability: a run-away process shaping human evolution? <i>Trends Genet.</i> <b>17</b>, 697–701 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0168-9525(01)02446-5" data-track-action="article reference" href="https://doi.org/10.1016%2FS0168-9525%2801%2902446-5" aria-label="Article reference 8" data-doi="10.1016/S0168-9525(01)02446-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXosFKmtLc%3D" aria-label="CAS reference 8">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11718922" aria-label="PubMed reference 8">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20high%20density%20of%20X-linked%20genes%20for%20general%20cognitive%20ability%3A%20a%20run-away%20process%20shaping%20human%20evolution%3F&amp;journal=Trends%20Genet.&amp;doi=10.1016%2FS0168-9525%2801%2902446-5&amp;volume=17&amp;pages=697-701&amp;publication_year=2001&amp;author=Zechner%2CU">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Raznahan, A. et al. Globally divergent but locally convergent X- and Y-chromosome influences on cortical development. <i>Cereb. Cortex</i> <b>26</b>, 70–79 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhu174" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhu174" aria-label="Article reference 9" data-doi="10.1093/cercor/bhu174">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25146371" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Globally%20divergent%20but%20locally%20convergent%20X-%20and%20Y-chromosome%20influences%20on%20cortical%20development&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhu174&amp;volume=26&amp;pages=70-79&amp;publication_year=2016&amp;author=Raznahan%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Yang, J. et al. Genome partitioning of genetic variation for complex traits using common SNPs. <i>Nat. Genet.</i> <b>43</b>, 519–525 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.823" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.823" aria-label="Article reference 10" data-doi="10.1038/ng.823">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvVagsLs%3D" aria-label="CAS reference 10">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21552263" aria-label="PubMed reference 10">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4295936" aria-label="PubMed Central reference 10">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome%20partitioning%20of%20genetic%20variation%20for%20complex%20traits%20using%20common%20SNPs&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.823&amp;volume=43&amp;pages=519-525&amp;publication_year=2011&amp;author=Yang%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Yang, J., Lee, S. H., Goddard, M. E. &amp; Visscher, P. M. GCTA: a tool for genome-wide complex trait analysis. <i>Am. J. Hum. Genet.</i> <b>88</b>, 76–82 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2010.11.011" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2010.11.011" aria-label="Article reference 11" data-doi="10.1016/j.ajhg.2010.11.011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXktVejtg%3D%3D" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21167468" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3014363" aria-label="PubMed Central reference 11">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=GCTA%3A%20a%20tool%20for%20genome-wide%20complex%20trait%20analysis&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2010.11.011&amp;volume=88&amp;pages=76-82&amp;publication_year=2011&amp;author=Yang%2CJ&amp;author=Lee%2CSH&amp;author=Goddard%2CME&amp;author=Visscher%2CPM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Fischl, B. et al. Whole brain segmentation: automated labeling of neuroanatomical structures in the human brain. <i>Neuron</i> <b>33</b>, 341–355 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0896-6273(02)00569-X" data-track-action="article reference" href="https://doi.org/10.1016%2FS0896-6273%2802%2900569-X" aria-label="Article reference 12" data-doi="10.1016/S0896-6273(02)00569-X">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XjtlWnsbg%3D" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11832223" aria-label="PubMed reference 12">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole%20brain%20segmentation%3A%20automated%20labeling%20of%20neuroanatomical%20structures%20in%20the%20human%20brain&amp;journal=Neuron&amp;doi=10.1016%2FS0896-6273%2802%2900569-X&amp;volume=33&amp;pages=341-355&amp;publication_year=2002&amp;author=Fischl%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Fischl, B. et al. Automatically parcellating the human cerebral cortex. <i>Cereb. Cortex</i> <b>14</b>, 11–22 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhg087" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhg087" aria-label="Article reference 13" data-doi="10.1093/cercor/bhg087">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14654453" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Automatically%20parcellating%20the%20human%20cerebral%20cortex&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhg087&amp;volume=14&amp;pages=11-22&amp;publication_year=2004&amp;author=Fischl%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. <i>Nature</i> <b>536</b>, 171–178 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature18933" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature18933" aria-label="Article reference 14" data-doi="10.1038/nature18933">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1GrurnK" aria-label="CAS reference 14">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27437579" aria-label="PubMed reference 14">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990127" aria-label="PubMed Central reference 14">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-modal%20parcellation%20of%20human%20cerebral%20cortex&amp;journal=Nature&amp;doi=10.1038%2Fnature18933&amp;volume=536&amp;pages=171-178&amp;publication_year=2016&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Lerch, J. P. et al. Studying neuroanatomy using MRI. <i>Nat. Neurosci.</i> <b>20</b>, 314–326 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4501" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4501" aria-label="Article reference 15" data-doi="10.1038/nn.4501">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVSmsrs%3D" aria-label="CAS reference 15">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28230838" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Studying%20neuroanatomy%20using%20MRI&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4501&amp;volume=20&amp;pages=314-326&amp;publication_year=2017&amp;author=Lerch%2CJP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Hofer, E. et al. Genetic correlations and genome-wide associations of cortical structure in general population samples of 22,824 adults. <i>Nat. Commun.</i> <b>11</b>, 4796 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-020-18367-y" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-18367-y" aria-label="Article reference 16" data-doi="10.1038/s41467-020-18367-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2jsr%2FP" aria-label="CAS reference 16">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32963231" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7508833" aria-label="PubMed Central reference 16">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20correlations%20and%20genome-wide%20associations%20of%20cortical%20structure%20in%20general%20population%20samples%20of%2022%2C824%20adults&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-18367-y&amp;volume=11&amp;publication_year=2020&amp;author=Hofer%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Matoba, N., Love, M. I. &amp; Stein, J. L. Evaluating brain structure traits as endophenotypes using polygenicity and discoverability. <i>Hum. Brain Mapp.</i> <a href="https://doi.org/10.1002/hbm.25257" data-track="click" data-track-action="external reference" data-track-label="10.1002/hbm.25257">https://doi.org/10.1002/hbm.25257</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Forde, N. J. et al. Sex differences in variability of brain structure across the lifespan. <i>Cereb. Cortex</i> <b>30</b>, 5420–5430 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhaa123" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhaa123" aria-label="Article reference 18" data-doi="10.1093/cercor/bhaa123">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32483605" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7566684" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Sex%20differences%20in%20variability%20of%20brain%20structure%20across%20the%20lifespan&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhaa123&amp;volume=30&amp;pages=5420-5430&amp;publication_year=2020&amp;author=Forde%2CNJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Ritchie, S. J. et al. Sex differences in the adult human brain: evidence from 5216 UK Biobank participants. <i>Cereb. Cortex</i> <b>28</b>, 2959–2975 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhy109" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhy109" aria-label="Article reference 19" data-doi="10.1093/cercor/bhy109">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29771288" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6041980" aria-label="PubMed Central reference 19">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Sex%20differences%20in%20the%20adult%20human%20brain%3A%20evidence%20from%205216%20UK%20Biobank%20participants&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhy109&amp;volume=28&amp;pages=2959-2975&amp;publication_year=2018&amp;author=Ritchie%2CSJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Finucane, H. K. et al. Partitioning heritability by functional annotation using genome-wide association summary statistics. <i>Nat. Genet.</i> <b>47</b>, 1228–1235 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.3404" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3404" aria-label="Article reference 20" data-doi="10.1038/ng.3404">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKqu7rF" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26414678" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4626285" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Partitioning%20heritability%20by%20functional%20annotation%20using%20genome-wide%20association%20summary%20statistics&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3404&amp;volume=47&amp;pages=1228-1235&amp;publication_year=2015&amp;author=Finucane%2CHK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">van der Meer, D. et al. Quantifying the polygenic architecture of the human cerebral cortex: extensive genetic overlap between cortical thickness and surface area. <i>Cereb. Cortex</i> <b>30</b>, 5597–5603 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhaa146" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhaa146" aria-label="Article reference 21" data-doi="10.1093/cercor/bhaa146">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32483632" aria-label="PubMed reference 21">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7472200" aria-label="PubMed Central reference 21">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20the%20polygenic%20architecture%20of%20the%20human%20cerebral%20cortex%3A%20extensive%20genetic%20overlap%20between%20cortical%20thickness%20and%20surface%20area&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhaa146&amp;volume=30&amp;pages=5597-5603&amp;publication_year=2020&amp;author=Meer%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Evans, L. M. et al. Comparison of methods that use whole genome data to estimate the heritability and genetic architecture of complex traits. <i>Nat. Genet.</i> <b>50</b>, 737–745 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0108-x" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0108-x" aria-label="Article reference 22" data-doi="10.1038/s41588-018-0108-x">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXosVCksrY%3D" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29700474" aria-label="PubMed reference 22">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5934350" aria-label="PubMed Central reference 22">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparison%20of%20methods%20that%20use%20whole%20genome%20data%20to%20estimate%20the%20heritability%20and%20genetic%20architecture%20of%20complex%20traits&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0108-x&amp;volume=50&amp;pages=737-745&amp;publication_year=2018&amp;author=Evans%2CLM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Ge, T. et al. Phenome-wide heritability analysis of the UK Biobank. <i>PLoS Genet.</i> <b>13</b>, e1006711 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1006711" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1006711" aria-label="Article reference 23" data-doi="10.1371/journal.pgen.1006711">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28388634" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFCgsL%2FK" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5400281" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Phenome-wide%20heritability%20analysis%20of%20the%20UK%20Biobank&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1006711&amp;volume=13&amp;publication_year=2017&amp;author=Ge%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Kharabian Masouleh, S., Eickhoff, S. B., Hoffstaedter, F., Genon, S. &amp; Alzheimer’s Disease Neuroimaging Initiative. Empirical examination of the replicability of associations between brain structure and psychological variables. <i>eLife</i> <b>8</b>, e43464 (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Romero-Garcia, R., Atienza, M., Clemmensen, L. H. &amp; Cantero, J. L. Effects of network resolution on topological properties of human neocortex. <i>NeuroImage</i> <b>59</b>, 3522–3532 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.10.086" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.10.086" aria-label="Article reference 25" data-doi="10.1016/j.neuroimage.2011.10.086">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22094643" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20network%20resolution%20on%20topological%20properties%20of%20human%20neocortex&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2011.10.086&amp;volume=59&amp;pages=3522-3532&amp;publication_year=2012&amp;author=Romero-Garcia%2CR&amp;author=Atienza%2CM&amp;author=Clemmensen%2CLH&amp;author=Cantero%2CJL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. <i>NeuroImage</i> <b>178</b>, 540–551 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2018.05.070" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2018.05.070" aria-label="Article reference 26" data-doi="10.1016/j.neuroimage.2018.05.070">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29860082" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20testing%20for%20spatial%20correspondence%20between%20maps%20of%20human%20brain%20structure%20and%20function&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2018.05.070&amp;volume=178&amp;pages=540-551&amp;publication_year=2018&amp;author=Alexander-Bloch%2CAF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. <i>Cereb. Cortex</i> <b>28</b>, 281–294 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhx249" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhx249" aria-label="Article reference 27" data-doi="10.1093/cercor/bhx249">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5903415" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Adolescent%20tuning%20of%20association%20cortex%20in%20human%20structural%20brain%20networks&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhx249&amp;volume=28&amp;pages=281-294&amp;publication_year=2017&amp;author=V%C3%A1%C5%A1a%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. <i>J. Neurophysiol.</i> <b>106</b>, 1125–1165 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1152/jn.00338.2011" data-track-action="article reference" href="https://doi.org/10.1152%2Fjn.00338.2011" aria-label="Article reference 28" data-doi="10.1152/jn.00338.2011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21653723" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20organization%20of%20the%20human%20cerebral%20cortex%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=J.%20Neurophysiol.&amp;doi=10.1152%2Fjn.00338.2011&amp;volume=106&amp;pages=1125-1165&amp;publication_year=2011&amp;author=Yeo%2CBTT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. <i>Nat. Methods</i> <b>8</b>, 665–670 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmeth.1635" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.1635" aria-label="Article reference 29" data-doi="10.1038/nmeth.1635">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXotV2gs7w%3D" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21706013" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3146590" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20automated%20synthesis%20of%20human%20functional%20neuroimaging%20data&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1635&amp;volume=8&amp;pages=665-670&amp;publication_year=2011&amp;author=Yarkoni%2CT&amp;author=Poldrack%2CRA&amp;author=Nichols%2CTE&amp;author=Essen%2CDC&amp;author=Wager%2CTD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Rubin, T. N. et al. Decoding brain activity using a large-scale probabilistic functional-anatomical atlas of human cognition. <i>PLoS Comput. Biol.</i> <b>13</b>, e1005649 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1005649" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1005649" aria-label="Article reference 30" data-doi="10.1371/journal.pcbi.1005649">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29059185" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlKku77M" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5683652" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Decoding%20brain%20activity%20using%20a%20large-scale%20probabilistic%20functional-anatomical%20atlas%20of%20human%20cognition&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1005649&amp;volume=13&amp;publication_year=2017&amp;author=Rubin%2CTN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Sjöstedt, E. et al. An atlas of the protein-coding genes in the human, pig, and mouse brain. <i>Science</i> <b>367</b>, eaay5947 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aay5947" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aay5947" aria-label="Article reference 31" data-doi="10.1126/science.aay5947">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32139519" aria-label="PubMed reference 31">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlOisbc%3D" aria-label="CAS reference 31">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20atlas%20of%20the%20protein-coding%20genes%20in%20the%20human%2C%20pig%2C%20and%20mouse%20brain&amp;journal=Science&amp;doi=10.1126%2Fscience.aay5947&amp;volume=367&amp;publication_year=2020&amp;author=Sj%C3%B6stedt%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Deciphering Developmental Disorders Study. Prevalence and architecture of de novo mutations in developmental disorders. <i>Nature</i> <b>542</b>, 433–438 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Martin, H. C. et al. The contribution of X-linked coding variation to severe developmental disorders. <i>Nat. Commun.</i> <b>12</b>, 627 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-020-20852-3" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-20852-3" aria-label="Article reference 33" data-doi="10.1038/s41467-020-20852-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXis1Kgtrw%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33504798" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7840967" aria-label="PubMed Central reference 33">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20contribution%20of%20X-linked%20coding%20variation%20to%20severe%20developmental%20disorders&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-20852-3&amp;volume=12&amp;publication_year=2021&amp;author=Martin%2CHC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Li, M. et al. Integrative functional genomic analysis of human brain development and neuropsychiatric risks. <i>Science</i> <b>362</b>, eaat7615 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.aat7615" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aat7615" aria-label="Article reference 34" data-doi="10.1126/science.aat7615">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhsLjE" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30545854" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6413317" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrative%20functional%20genomic%20analysis%20of%20human%20brain%20development%20and%20neuropsychiatric%20risks&amp;journal=Science&amp;doi=10.1126%2Fscience.aat7615&amp;volume=362&amp;publication_year=2018&amp;author=Li%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Reinhold, K. &amp; Engqvist, L. The variability is in the sex chromosomes: brief communication. <i>Evolution</i> <b>67</b>, 3662–3668 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/evo.12224" data-track-action="article reference" href="https://doi.org/10.1111%2Fevo.12224" aria-label="Article reference 35" data-doi="10.1111/evo.12224">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24299417" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20variability%20is%20in%20the%20sex%20chromosomes%3A%20brief%20communication&amp;journal=Evolution&amp;doi=10.1111%2Fevo.12224&amp;volume=67&amp;pages=3662-3668&amp;publication_year=2013&amp;author=Reinhold%2CK&amp;author=Engqvist%2CL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Young, A. I. Solving the missing heritability problem. <i>PLoS Genet.</i> <b>15</b>, e1008222 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1008222" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1008222" aria-label="Article reference 36" data-doi="10.1371/journal.pgen.1008222">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1WmsL7F" aria-label="CAS reference 36">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31233496" aria-label="PubMed reference 36">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611648" aria-label="PubMed Central reference 36">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Solving%20the%20missing%20heritability%20problem&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1008222&amp;volume=15&amp;publication_year=2019&amp;author=Young%2CAI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Schmitt, J. E. et al. A comprehensive quantitative genetic analysis of cerebral surface area in youth. <i>J. Neurosci.</i> <b>39</b>, 3028–3040 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2248-18.2019" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2248-18.2019" aria-label="Article reference 37" data-doi="10.1523/JNEUROSCI.2248-18.2019">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFSjs7vP" aria-label="CAS reference 37">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30833512" aria-label="PubMed reference 37">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6468099" aria-label="PubMed Central reference 37">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20comprehensive%20quantitative%20genetic%20analysis%20of%20cerebral%20surface%20area%20in%20youth&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2248-18.2019&amp;volume=39&amp;pages=3028-3040&amp;publication_year=2019&amp;author=Schmitt%2CJE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Cadwell, C. R., Bhaduri, A., Mostajo-Radji, M. A., Keefe, M. G. &amp; Nowakowski, T. J. Development and arealization of the cerebral cortex. <i>Neuron</i> <b>103</b>, 980–1004 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2019.07.009" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2019.07.009" aria-label="Article reference 38" data-doi="10.1016/j.neuron.2019.07.009">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvVOlsbrM" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31557462" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20and%20arealization%20of%20the%20cerebral%20cortex&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2019.07.009&amp;volume=103&amp;pages=980-1004&amp;publication_year=2019&amp;author=Cadwell%2CCR&amp;author=Bhaduri%2CA&amp;author=Mostajo-Radji%2CMA&amp;author=Keefe%2CMG&amp;author=Nowakowski%2CTJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Woodbury-Smith, M. et al. Mutations in <i>RAB39B</i> in individuals with intellectual disability, autism spectrum disorder, and macrocephaly. <i>Mol. Autism</i> <b>8</b>, 59 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/s13229-017-0175-3" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13229-017-0175-3" aria-label="Article reference 39" data-doi="10.1186/s13229-017-0175-3">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29152164" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXls12is7c%3D" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5679329" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutations%20in%20RAB39B%20in%20individuals%20with%20intellectual%20disability%2C%20autism%20spectrum%20disorder%2C%20and%20macrocephaly&amp;journal=Mol.%20Autism&amp;doi=10.1186%2Fs13229-017-0175-3&amp;volume=8&amp;publication_year=2017&amp;author=Woodbury-Smith%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Kenwrick, S., Watkins, A. &amp; De Angelis, E. Neural cell recognition molecule L1: relating biological complexity to human disease mutations. <i>Hum. Mol. Genet.</i> <b>9</b>, 879–886 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/hmg/9.6.879" data-track-action="article reference" href="https://doi.org/10.1093%2Fhmg%2F9.6.879" aria-label="Article reference 40" data-doi="10.1093/hmg/9.6.879">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3cXivFyksbg%3D" aria-label="CAS reference 40">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10767310" aria-label="PubMed reference 40">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Neural%20cell%20recognition%20molecule%20L1%3A%20relating%20biological%20complexity%20to%20human%20disease%20mutations&amp;journal=Hum.%20Mol.%20Genet.&amp;doi=10.1093%2Fhmg%2F9.6.879&amp;volume=9&amp;pages=879-886&amp;publication_year=2000&amp;author=Kenwrick%2CS&amp;author=Watkins%2CA&amp;author=Angelis%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Graves, J. A. M., Gécz, J. &amp; Hameister, H. Evolution of the human X—a smart and sexy chromosome that controls speciation and development. <i>Cytogenet. Genome Res.</i> <b>99</b>, 141–145 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1159/000071585" data-track-action="article reference" href="https://doi.org/10.1159%2F000071585" aria-label="Article reference 41" data-doi="10.1159/000071585">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3szms1Sksw%3D%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12900556" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%20of%20the%20human%20X%E2%80%94a%20smart%20and%20sexy%20chromosome%20that%20controls%20speciation%20and%20development&amp;journal=Cytogenet.%20Genome%20Res.&amp;doi=10.1159%2F000071585&amp;volume=99&amp;pages=141-145&amp;publication_year=2002&amp;author=Graves%2CJAM&amp;author=G%C3%A9cz%2CJ&amp;author=Hameister%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Vallender, E. J. &amp; Lahn, B. T. How mammalian sex chromosomes acquired their peculiar gene content. <i>Bioessays</i> <b>26</b>, 159–169 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/bies.10393" data-track-action="article reference" href="https://doi.org/10.1002%2Fbies.10393" aria-label="Article reference 42" data-doi="10.1002/bies.10393">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXitVCks7Y%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14745834" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20mammalian%20sex%20chromosomes%20acquired%20their%20peculiar%20gene%20content&amp;journal=Bioessays&amp;doi=10.1002%2Fbies.10393&amp;volume=26&amp;pages=159-169&amp;publication_year=2004&amp;author=Vallender%2CEJ&amp;author=Lahn%2CBT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Balaton, B. P. et al. Derivation of consensus inactivation status for X-linked genes from genome-wide studies. <i>Biol. Sex Differ.</i> <b>6</b>, 35 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/s13293-015-0053-7" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13293-015-0053-7" aria-label="Article reference 43" data-doi="10.1186/s13293-015-0053-7">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26719789" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXls1ejtbc%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4696107" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Derivation%20of%20consensus%20inactivation%20status%20for%20X-linked%20genes%20from%20genome-wide%20studies&amp;journal=Biol.%20Sex%20Differ.&amp;doi=10.1186%2Fs13293-015-0053-7&amp;volume=6&amp;publication_year=2015&amp;author=Balaton%2CBP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. <i>PLoS Med.</i> <b>12</b>, e1001779 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pmed.1001779" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pmed.1001779" aria-label="Article reference 44" data-doi="10.1371/journal.pmed.1001779">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25826379" aria-label="PubMed reference 44">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380465" aria-label="PubMed Central reference 44">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=UK%20Biobank%3A%20an%20open%20access%20resource%20for%20identifying%20the%20causes%20of%20a%20wide%20range%20of%20complex%20diseases%20of%20middle%20and%20old%20age&amp;journal=PLoS%20Med.&amp;doi=10.1371%2Fjournal.pmed.1001779&amp;volume=12&amp;publication_year=2015&amp;author=Sudlow%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Alfaro-Almagro, F. et al. Image processing and quality control for the first 10,000 brain imaging datasets from UK Biobank. <i>NeuroImage</i> <b>166</b>, 400–424 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.10.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.10.034" aria-label="Article reference 45" data-doi="10.1016/j.neuroimage.2017.10.034">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29079522" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Image%20processing%20and%20quality%20control%20for%20the%20first%2010%2C000%20brain%20imaging%20datasets%20from%20UK%20Biobank&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2017.10.034&amp;volume=166&amp;pages=400-424&amp;publication_year=2018&amp;author=Alfaro-Almagro%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. <i>NeuroImage</i> <b>80</b>, 105–124 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.04.127" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.04.127" aria-label="Article reference 46" data-doi="10.1016/j.neuroimage.2013.04.127">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23668970" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20minimal%20preprocessing%20pipelines%20for%20the%20Human%20Connectome%20Project&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2013.04.127&amp;volume=80&amp;pages=105-124&amp;publication_year=2013&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Fischl, B. FreeSurfer. <i>NeuroImage</i> <b>62</b>, 774–781 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.01.021" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.01.021" aria-label="Article reference 47" data-doi="10.1016/j.neuroimage.2012.01.021">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22248573" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=FreeSurfer&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2012.01.021&amp;volume=62&amp;pages=774-781&amp;publication_year=2012&amp;author=Fischl%2CB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Ségonne, F. et al. A hybrid approach to the skull stripping problem in MRI. <i>NeuroImage</i> <b>22</b>, 1060–1075 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2004.03.032" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2004.03.032" aria-label="Article reference 48" data-doi="10.1016/j.neuroimage.2004.03.032">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15219578" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20hybrid%20approach%20to%20the%20skull%20stripping%20problem%20in%20MRI&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2004.03.032&amp;volume=22&amp;pages=1060-1075&amp;publication_year=2004&amp;author=S%C3%A9gonne%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Dale, A. M., Fischl, B. &amp; Sereno, M. I. Cortical surface-based analysis. I. Segmentation and surface reconstruction. <i>NeuroImage</i> <b>9</b>, 179–194 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.1998.0395" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.1998.0395" aria-label="Article reference 49" data-doi="10.1006/nimg.1998.0395">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1M7jt1Gisg%3D%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9931268" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20surface-based%20analysis.%20I.%20Segmentation%20and%20surface%20reconstruction&amp;journal=NeuroImage&amp;doi=10.1006%2Fnimg.1998.0395&amp;volume=9&amp;pages=179-194&amp;publication_year=1999&amp;author=Dale%2CAM&amp;author=Fischl%2CB&amp;author=Sereno%2CMI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Fischl, B., Sereno, M. I. &amp; Dale, A. M. Cortical surface-based analysis. II. Inflation, flattening, and a surface-based coordinate system. <i>NeuroImage</i> <b>9</b>, 195–207 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.1998.0396" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.1998.0396" aria-label="Article reference 50" data-doi="10.1006/nimg.1998.0396">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1M7jt1Gisw%3D%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9931269" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20surface-based%20analysis.%20II.%20Inflation%2C%20flattening%2C%20and%20a%20surface-based%20coordinate%20system&amp;journal=NeuroImage&amp;doi=10.1006%2Fnimg.1998.0396&amp;volume=9&amp;pages=195-207&amp;publication_year=1999&amp;author=Fischl%2CB&amp;author=Sereno%2CMI&amp;author=Dale%2CAM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. <i>NeuroImage</i> <b>31</b>, 968–980 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2006.01.021" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2006.01.021" aria-label="Article reference 51" data-doi="10.1016/j.neuroimage.2006.01.021">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16530430" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20automated%20labeling%20system%20for%20subdividing%20the%20human%20cerebral%20cortex%20on%20MRI%20scans%20into%20gyral%20based%20regions%20of%20interest&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2006.01.021&amp;volume=31&amp;pages=968-980&amp;publication_year=2006&amp;author=Desikan%2CRS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. <i>Nature</i> <b>562</b>, 203–209 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-018-0579-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-018-0579-z" aria-label="Article reference 52" data-doi="10.1038/s41586-018-0579-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2qsbvI" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30305743" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975" aria-label="PubMed Central reference 52">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0579-z&amp;volume=562&amp;pages=203-209&amp;publication_year=2018&amp;author=Bycroft%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">O’Connell, J. et al. Haplotype estimation for biobank-scale data sets. <i>Nat. Genet.</i> <b>48</b>, 817–820 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.3583" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3583" aria-label="Article reference 53" data-doi="10.1038/ng.3583">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27270105" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XpsVCrsro%3D" aria-label="CAS reference 53">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4926957" aria-label="PubMed Central reference 53">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Haplotype%20estimation%20for%20biobank-scale%20data%20sets&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3583&amp;volume=48&amp;pages=817-820&amp;publication_year=2016&amp;author=O%E2%80%99Connell%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. <i>Gigascience</i> <b>4</b>, 7 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/s13742-015-0047-8" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13742-015-0047-8" aria-label="Article reference 54" data-doi="10.1186/s13742-015-0047-8">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25722852" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsFShsLY%3D" aria-label="CAS reference 54">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193" aria-label="PubMed Central reference 54">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&amp;journal=Gigascience&amp;doi=10.1186%2Fs13742-015-0047-8&amp;volume=4&amp;publication_year=2015&amp;author=Chang%2CCC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Burnham, K. P. &amp; Anderson, D. R. <i>Model Selection and Multimodel Inference: A Practical Information-Theoretic Approach</i>. (Springer Science &amp; Business Media, 2002).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Finucane, H. K. et al. Heritability enrichment of specifically expressed genes identifies disease-relevant tissues and cell types. <i>Nat. Genet.</i> <b>50</b>, 621–629 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0081-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0081-4" aria-label="Article reference 56" data-doi="10.1038/s41588-018-0081-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXnsVGrtr4%3D" aria-label="CAS reference 56">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29632380" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5896795" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Heritability%20enrichment%20of%20specifically%20expressed%20genes%20identifies%20disease-relevant%20tissues%20and%20cell%20types&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0081-4&amp;volume=50&amp;pages=621-629&amp;publication_year=2018&amp;author=Finucane%2CHK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Speed, D., Hemani, G., Johnson, M. R. &amp; Balding, D. J. Improved heritability estimation from genome-wide SNPs. <i>Am. J. Hum. Genet.</i> <b>91</b>, 1011–1021 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2012.10.010" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2012.10.010" aria-label="Article reference 57" data-doi="10.1016/j.ajhg.2012.10.010">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVWis7bP" aria-label="CAS reference 57">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23217325" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3516604" aria-label="PubMed Central reference 57">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20heritability%20estimation%20from%20genome-wide%20SNPs&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2012.10.010&amp;volume=91&amp;pages=1011-1021&amp;publication_year=2012&amp;author=Speed%2CD&amp;author=Hemani%2CG&amp;author=Johnson%2CMR&amp;author=Balding%2CDJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Yang, J. et al. Common SNPs explain a large proportion of the heritability for human height. <i>Nat. Genet.</i> <b>42</b>, 565–569 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ng.608" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.608" aria-label="Article reference 58" data-doi="10.1038/ng.608">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXns1GisL8%3D" aria-label="CAS reference 58">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20562875" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3232052" aria-label="PubMed Central reference 58">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Common%20SNPs%20explain%20a%20large%20proportion%20of%20the%20heritability%20for%20human%20height&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.608&amp;volume=42&amp;pages=565-569&amp;publication_year=2010&amp;author=Yang%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Seidlitz, J. et al. Morphometric similarity networks detect microscale cortical organization and predict inter-individual cognitive variation. <i>Neuron</i> <b>97</b>, 231–247.e7 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2017.11.039" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2017.11.039" aria-label="Article reference 59" data-doi="10.1016/j.neuron.2017.11.039">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVehtL7M" aria-label="CAS reference 59">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29276055" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Morphometric%20similarity%20networks%20detect%20microscale%20cortical%20organization%20and%20predict%20inter-individual%20cognitive%20variation&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2017.11.039&amp;volume=97&amp;pages=231-247.e7&amp;publication_year=2018&amp;author=Seidlitz%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Poldrack, R. A. et al. Discovering relations between mind, brain, and mental disorders using topic mapping. <i>PLoS Comput. Biol.</i> <b>8</b>, e1002707 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1002707" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1002707" aria-label="Article reference 60" data-doi="10.1371/journal.pcbi.1002707">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFOrsLjF" aria-label="CAS reference 60">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23071428" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3469446" aria-label="PubMed Central reference 60">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Discovering%20relations%20between%20mind%2C%20brain%2C%20and%20mental%20disorders%20using%20topic%20mapping&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1002707&amp;volume=8&amp;publication_year=2012&amp;author=Poldrack%2CRA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Winkler, T. W. et al. Quality control and conduct of genome-wide association meta-analyses. <i>Nat. Protoc.</i> <b>9</b>, 1192–1212 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nprot.2014.071" data-track-action="article reference" href="https://doi.org/10.1038%2Fnprot.2014.071" aria-label="Article reference 61" data-doi="10.1038/nprot.2014.071">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24762786" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4083217" aria-label="PubMed Central reference 61">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Quality%20control%20and%20conduct%20of%20genome-wide%20association%20meta-analyses&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2014.071&amp;volume=9&amp;pages=1192-1212&amp;publication_year=2014&amp;author=Winkler%2CTW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Mallard, T. T. et al. Multivariate GWAS of psychiatric disorders and their cardinal symptoms reveal two dimensions of cross-cutting genetic liabilities. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/603134" data-track="click" data-track-action="external reference" data-track-label="10.1101/603134">https://doi.org/10.1101/603134</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Mallard, T. T. et al. tem-level genome-wide association study of the Alcohol Use Disorders Identification Test in three population-based cohorts. <i>Am. J. Psychiatry</i> <a href="https://doi.org/10.1176/appi.ajp.2020.20091390" data-track="click" data-track-action="external reference" data-track-label="10.1176/appi.ajp.2020.20091390">https://doi.org/10.1176/appi.ajp.2020.20091390</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">de Leeuw, C. A., Mooij, J. M., Heskes, T. &amp; Posthuma, D. MAGMA: generalized gene-set analysis of GWAS data. <i>PLoS Comput. Biol.</i> <b>11</b>, e1004219 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1004219" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1004219" aria-label="Article reference 64" data-doi="10.1371/journal.pcbi.1004219">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25885710" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XktlOguro%3D" aria-label="CAS reference 64">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4401657" aria-label="PubMed Central reference 64">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=MAGMA%3A%20generalized%20gene-set%20analysis%20of%20GWAS%20data&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1004219&amp;volume=11&amp;publication_year=2015&amp;author=Leeuw%2CCA&amp;author=Mooij%2CJM&amp;author=Heskes%2CT&amp;author=Posthuma%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Watanabe, K., Taskesen, E., van Bochoven, A. &amp; Posthuma, D. Functional mapping and annotation of genetic associations with FUMA. <i>Nat. Commun.</i> <b>8</b>, 1826 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-017-01261-5" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-017-01261-5" aria-label="Article reference 65" data-doi="10.1038/s41467-017-01261-5">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29184056" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2nt7%2FO" aria-label="CAS reference 65">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5705698" aria-label="PubMed Central reference 65">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20mapping%20and%20annotation%20of%20genetic%20associations%20with%20FUMA&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-01261-5&amp;volume=8&amp;publication_year=2017&amp;author=Watanabe%2CK&amp;author=Taskesen%2CE&amp;author=Bochoven%2CA&amp;author=Posthuma%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Kuleshov, M. V. et al. Enrichr: a comprehensive gene set enrichment analysis web server 2016 update. <i>Nucleic Acids Res.</i> <b>44</b>, W90–W97 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/nar/gkw377" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw377" aria-label="Article reference 66" data-doi="10.1093/nar/gkw377">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfF" aria-label="CAS reference 66">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27141961" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987924" aria-label="PubMed Central reference 66">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Enrichr%3A%20a%20comprehensive%20gene%20set%20enrichment%20analysis%20web%20server%202016%20update&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw377&amp;volume=44&amp;pages=W90-W97&amp;publication_year=2016&amp;author=Kuleshov%2CMV">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Piñero, J. et al. The DisGeNET knowledge platform for disease genomics: 2019 update. <i>Nucleic Acids Res.</i> <b>48</b>, D845–D855 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31680165" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20DisGeNET%20knowledge%20platform%20for%20disease%20genomics%3A%202019%20update&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D845-D855&amp;publication_year=2020&amp;author=Pi%C3%B1ero%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Langfelder, P. &amp; Horvath, S. WGCNA: an R package for weighted correlation network analysis. <i>BMC Bioinformatics</i> <b>9</b>, 559 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/1471-2105-9-559" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2105-9-559" aria-label="Article reference 68" data-doi="10.1186/1471-2105-9-559">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19114008" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXht1Khsbw%3D" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2631488" aria-label="PubMed Central reference 68">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=WGCNA%3A%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-9-559&amp;volume=9&amp;publication_year=2008&amp;author=Langfelder%2CP&amp;author=Horvath%2CS">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-021-00890-w?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>This research was supported by the Intramural Research Program of the National Institute of Mental Health (NIH annual report number ZIA MH002949-04), and conducted using the UKB resource under application number 22875. We thank the UKB participants for making this study possible.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Department of Psychology, University of Texas at Austin, Austin, TX, USA</p><p class="c-article-author-affiliation__authors-list">Travis T. Mallard</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Section on Developmental Neurogenomics, Human Genetics Branch, National Institute of Mental Health Intramural Research Program, Bethesda, MD, USA</p><p class="c-article-author-affiliation__authors-list">Travis T. Mallard, Siyuan Liu, Jakob Seidlitz &amp; Armin Raznahan</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Laboratory of Functional and Molecular Imaging, National Institute of Neurological Disorders and Stroke, National Institutes of Health, Bethesda, MD, USA</p><p class="c-article-author-affiliation__authors-list">Zhiwei Ma</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Data Science and Sharing Team, National Institute of Mental Health, Bethesda, MD, USA</p><p class="c-article-author-affiliation__authors-list">Dustin Moraczewski &amp; Adam Thomas</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Travis_T_-Mallard-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Travis T. Mallard</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Travis%20T.%20Mallard" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Travis%20T.%20Mallard" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Travis%20T.%20Mallard%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Siyuan-Liu-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Siyuan Liu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Siyuan%20Liu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Siyuan%20Liu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Siyuan%20Liu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jakob-Seidlitz-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Jakob Seidlitz</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jakob%20Seidlitz" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jakob%20Seidlitz" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jakob%20Seidlitz%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Zhiwei-Ma-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Zhiwei Ma</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Zhiwei%20Ma" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Zhiwei%20Ma" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Zhiwei%20Ma%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Dustin-Moraczewski-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Dustin Moraczewski</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Dustin%20Moraczewski" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Dustin%20Moraczewski" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Dustin%20Moraczewski%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Adam-Thomas-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Adam Thomas</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Adam%20Thomas" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Adam%20Thomas" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Adam%20Thomas%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Armin-Raznahan-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Armin Raznahan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Armin%20Raznahan" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Armin%20Raznahan" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Armin%20Raznahan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>T.T.M. and A.R. conceived and designed the study. A.R. oversaw the study. T.T.M. was the lead analyst responsible for conducting all genetically informative analyses. S.L. conducted the topographic annotation analyses and spin tests. J.S. assisted with the expression analyses in the PsychENCODE dataset. T.T.M. and S.L. prepared the figures and tables. T.T.M. prepared the genetic data, while J.S., Z.M., D.M. and A.T. prepared the neuroimaging data. T.T.M. and A.R. led the writing of the manuscript, and all authors provided valuable feedback and advice during its preparation.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:raznahana@mail.nih.gov">Armin Raznahan</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar3">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> <i>Nature Neuroscience</i> thanks Jason Stein and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec26-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec26">Extended data</h2><div class="c-article-section__content" id="Sec26-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 distributions of x-linked her" href="/articles/s41593-021-00890-w/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig5_ESM.jpg">Extended Data Fig. 1 Distributions of X-linked heritability in males and females for regional measures of brain anatomy.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Raincloud-style plot that considers all regional measures of brain anatomy where the full dosage compensation model was the best fitting model (<i>n</i> = 1,840 sex-stratified heritability estimates). X-linked heritability for males and females was independently estimated within each sex in a model-naïve manner (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). The median X-linked heritability in males is 2.37 times larger (bootstrapped 95% CI = 1.70-3.25) than the median X-linked heritability in females (two-sided Wilcoxon <i>P</i> = 1.18e-23). Summary statistics for males: minimum = 1e-6, Q1 = 3.98e-4, median = 6.34e-3, Q3 = 1.3e-2, maximum = 3.64e-2. Summary statistics for females: minimum = 1e-6, Q1 = 1e-6, median = 2.67e-3, Q3 = 7.48e-3, maximum = 3.34e-2.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 regional patterning of x-chro" href="/articles/s41593-021-00890-w/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig6_ESM.jpg">Extended Data Fig. 2 Regional patterning of X-chromosome dosage compensation for cortical volume (CV).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Categorical map showing the best-fitting X-chromosome dosage compensation model for each cortical region. NDC = no dosage compensation, EV = equal variance, FDC = full dosage compensation. <b>b</b>, Continuous map showing regional variation in the strength of statistical evidence for X-chromosome dosage compensation on CV, as indexed by ΔAIC<sub>DC</sub> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). ΔAIC<sub>DC</sub> equals zero when the EV model fits best, and departures from zero reflect quantitative evidence for either NDC or FDC. Values ≤ -10 are indicative of strong evidence for NDC while values ≥ +10 are indicative of strong evidence for FDC. Any values less than −10 or greater than 10 were set to −10 and 10, respectively, for illustration.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 regional patterning of x-chro" href="/articles/s41593-021-00890-w/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig7_ESM.jpg">Extended Data Fig. 3 Regional patterning of X-chromosome dosage compensation for cortical thickness (CT).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Categorical map showing the best-fitting X-chromosome dosage compensation model for each cortical region. NDC = no dosage compensation, EV = equal variance, FDC = full dosage compensation. <b>b</b>, Continuous map showing regional variation in the strength of statistical evidence for X-chromosome dosage compensation on CT, as indexed by ΔAIC<sub>DC</sub> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>). ΔAIC<sub>DC</sub> equals zero when the EV model fits best, and departures from zero reflect quantitative evidence for either NDC or FDC. Values ≤ -10 are indicative of strong evidence for NDC while values ≥ +10 are indicative of strong evidence for FDC. Any values less than −10 or greater than 10 were set to −10 and 10, respectively, for illustration.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 regional patterning of x-chro" href="/articles/s41593-021-00890-w/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig8_ESM.jpg">Extended Data Fig. 4 Regional patterning of X-chromosome influences (XIs) on cortical volume (CV).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Continuous map of total SNP heritability <span class="mathjax-tex">\((h_{\mathrm{g}}^2)\)</span> for regional CV. <b>b</b>, Continuous map of X-linked heritability <span class="mathjax-tex">\((h_{\mathrm{x}}^2)\)</span> for regional CV. <b>c</b>, Continuous map of regional ratio between the observed proportion of total heritability assigned to the X-chromosome, and the expected proportion from X-chromosome size (XI ratio). Values greater than one reflect enriched XIs and values less than one reflect depleted XIs (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 regional patterning of x-chro" href="/articles/s41593-021-00890-w/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig9_ESM.jpg">Extended Data Fig. 5 Regional patterning of X-chromosome influences (XIs) on cortical thickness (CT).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Continuous map of total SNP heritability <span class="mathjax-tex">\((h_{\mathrm{g}}^2)\)</span> for regional CT. <b>b</b>, Continuous map of X-linked heritability <span class="mathjax-tex">\((h_{\mathrm{x}}^2)\)</span> for regional CT. <b>c</b>, Continuous map of regional ratio between the observed proportion of total heritability assigned to the X-chromosome, and the expected proportion from X-chromosome size (XI ratio). Values greater than one reflect enriched XIs and values less than one reflect depleted XIs (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 comparison of the enrichment " href="/articles/s41593-021-00890-w/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig10_ESM.jpg">Extended Data Fig. 6 Comparison of the enrichment ratio for X-chromosome influences (XIs) on three morphological features of the cortex.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Ranked scatter plot of the regional XI enrichment ratio for cortical surface area, volume, and thickness, where the 358 cortical regions are ranked by their XI enrichment ratio for surface area. <b>b</b>,<b>c</b>,<b>d</b>, Scatter plots of the regional XI enrichment ratio for (<b>b</b>) surface area and volume, (<b>c</b>) thickness and volume, and (<b>d</b>) thickness and surface area.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 comparison of the enrichment " href="/articles/s41593-021-00890-w/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig11_ESM.jpg">Extended Data Fig. 7 Comparison of the enrichment Z statistic for X-chromosome influences (XIs) on three morphological features of the cortex.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Ranked scatter plot of the regional XI enrichment <i>Z</i>-statistic for cortical surface area, volume, and thickness, where the 358 cortical regions are ranked by their XI enrichment <i>Z</i>-statistic for surface area. <b>b</b>,<b>c</b>,<b>d</b>, Scatter plots of the regional XI enrichment <i>Z</i>-statistic for (<b>b</b>) surface area and volume, (<b>c</b>) thickness and volume, and (<b>d</b>) thickness and surface area.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 statistically-significant dep" href="/articles/s41593-021-00890-w/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig12_ESM.jpg">Extended Data Fig. 8 Statistically-significant depletion and enrichment of X-chromosome influences (XIs) on three morphological features of the cortex.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>,<b>b</b>,<b>c</b>, Categorical maps showing the regions of statistically significant depletion or enrichment of XIs (after correction for multiple comparisons) on (<b>a</b>) cortical surface area, (<b>b</b>) cortical volume, and (<b>c</b>) cortical thickness.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 sensitivity analyses for obse" href="/articles/s41593-021-00890-w/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig13_ESM.jpg">Extended Data Fig. 9 Sensitivity analyses for observed X-chromosome influences (XIs) on cortical surface area (SA).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Cross-region of interest (ROI) correlation in the XI <i>Z</i>-statistic for SA between first and second releases of UKB neuroimaging data. <b>b</b>, Distribution of cross-ROI correlations in the XI <i>Z</i>-statistic for SA for 100 split-halves of the UKB dataset. <b>c</b>, Cross-ROI correlation in the XI <i>Z</i>-statistic for SA between UKB analyses differing in minor allele frequency (MAF) threshold. <b>d</b>, Continuous map of XI enrichment ratio for SA as computed in the 308-parcellation. <b>e</b>, Observed cross-ROI correlation (dashed line) in the 308-parcellation between XI <i>Z</i>-statistics computed in the 308-parcellation vs. those projected into the 308-parcellation from values computed in the primary HCP atlas. Density plot shows a null distribution of these correlations from 10,000 rotational permutations ‘spins’ of the 308 map. The observed correlation (two-sided Pearson’s <i>r</i> = .76) is significantly elevated relative to this null distribution (<i>P</i><sub>spin</sub> &lt; 1e-4) (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-021-00890-w#Sec8">Methods</a>).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 x-linked snps associated wit" href="/articles/s41593-021-00890-w/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_Fig14_ESM.jpg">Extended Data Fig. 10 X-linked SNPs associated with regional surface area phenotypes with enriched X-chromosome influences (XIs).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>An X-chromosome-wide Manhattan plot of the minimum <i>P</i>-value for each SNP across all tested phenotypes. The x-axis refers to the chromosomal position, the y-axis refers to the significance of the SNP-based association test as a -log10 <i>P</i>-value, the horizontal dashed line denotes genome-wide significance (<i>P</i> = 5e-8), and the horizontal two-dashed line denotes a more conservative significance threshold accounting for all tested phenotypes (<i>P</i> = 8.93e-10).</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec27-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec27">Supplementary information</h2><div class="c-article-section__content" id="Sec27-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_MOESM1_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary tables" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-021-00890-w/MediaObjects/41593_2021_890_MOESM2_ESM.xlsx" data-supp-info-image="">Supplementary Tables</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Tables 1–7.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=X-chromosome%20influences%20on%20neuroanatomical%20variation%20in%20humans&amp;author=Travis%20T.%20Mallard%20et%20al&amp;contentID=10.1038%2Fs41593-021-00890-w&amp;copyright=This%20is%20a%20U.S.%20government%20work%20and%20not%20under%20copyright%20protection%20in%20the%20U.S.%3B%20foreign%20copyright%20protection%20may%20apply&amp;publication=1097-6256&amp;publicationDate=2021-07-22&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-021-00890-w" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-021-00890-w" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Mallard, T.T., Liu, S., Seidlitz, J. <i>et al.</i> X-chromosome influences on neuroanatomical variation in humans.
                    <i>Nat Neurosci</i> <b>24</b>, 1216–1224 (2021). https://doi.org/10.1038/s41593-021-00890-w</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-021-00890-w?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-06-04">04 June 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-06-14">14 June 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-07-22">22 July 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-09">September 2021</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-021-00890-w</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Females exhibit smaller volumes of brain activation and lower inter-subject variability during motor tasks" href="https://doi.org/10.1038/s41598-023-44871-4">
                                        Females exhibit smaller volumes of brain activation and lower inter-subject variability during motor tasks
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Justin W. Andrushko</li><li>Shie Rinat</li><li>Lara A. Boyd</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Reports</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Sex differences in number of X chromosomes and X-chromosome inactivation in females promote greater variability in hearing among males" href="https://doi.org/10.1186/s13293-022-00457-9">
                                        Sex differences in number of X chromosomes and X-chromosome inactivation in females promote greater variability in hearing among males
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Van Summers</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Biology of Sex Differences</i> (2022)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Shared genetic architecture between schizophrenia and subcortical brain volumes implicates early neurodevelopmental processes and brain development in childhood" href="https://doi.org/10.1038/s41380-022-01751-z">
                                        Shared genetic architecture between schizophrenia and subcortical brain volumes implicates early neurodevelopmental processes and brain development in childhood
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Weiqiu Cheng</li><li>Dennis van der Meer</li><li>Ole A. Andreassen</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Molecular Psychiatry</i> (2022)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:In search of sex-related mediators of affective illness" href="https://doi.org/10.1186/s13293-021-00400-4">
                                        In search of sex-related mediators of affective illness
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Christopher Sikes-Keilp</li><li>David R. Rubinow</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Biology of Sex Differences</i> (2021)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-021-00890-w.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-021-00890-w.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-021-00890-w;doi=10.1038/s41593-021-00890-w;subjmeta=1515,1688,205,208,2571,2583,378,631,64,692,698;kwrd=Behavioural+genetics,Brain,Development+of+the+nervous+system,Genetic+association+study,Genetics+of+the+nervous+system">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=1659775427&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-021-00890-w%26doi%3D10.1038/s41593-021-00890-w%26subjmeta%3D1515,1688,205,208,2571,2583,378,631,64,692,698%26kwrd%3DBehavioural+genetics,Brain,Development+of+the+nervous+system,Genetic+association+study,Genetics+of+the+nervous+system">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=1659775427&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-021-00890-w%26doi%3D10.1038/s41593-021-00890-w%26subjmeta%3D1515,1688,205,208,2571,2583,378,631,64,692,698%26kwrd%3DBehavioural+genetics,Brain,Development+of+the+nervous+system,Genetic+association+study,Genetics+of+the+nervous+system"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-021-00890-w&amp;format=js&amp;last_modified=2021-07-22" async></script>
<img src="/439h0xs6/article/s41593-021-00890-w" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>