<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>A synergistic core for human brain evolution and cognition | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"cognitive-control;dynamical-systems;genetics-of-the-nervous-system;network-models","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41593-022-01070-0"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Andrea I. Luppi","Pedro A. M. Mediano","Fernando E. Rosas","Negin Holland","Tim D. Fryer","John T. O’Brien","James B. Rowe","David K. Menon","Daniel Bor","Emmanuel A. Stamatakis"],"publishedAt":1653523200,"publishedAtString":"2022-05-26","title":"A synergistic core for human brain evolution and cognition","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"25","issue":"6"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"A synergistic core for human brain evolution and cognition","description":"How does the organization of neural information processing enable humans’ sophisticated cognition? Here we decompose functional interactions between brain regions into synergistic and redundant components, revealing their distinct information-processing roles. Combining functional and structural neuroimaging with meta-analytic results, we demonstrate that redundant interactions are predominantly associated with structurally coupled, modular sensorimotor processing. Synergistic interactions instead support integrative processes and complex cognition across higher-order brain networks. The human brain leverages synergistic information to a greater extent than nonhuman primates, with high-synergy association cortices exhibiting the highest degree of evolutionary cortical expansion. Synaptic density mapping from positron emission tomography and convergent molecular and metabolic evidence demonstrate that synergistic interactions are supported by receptor diversity and human-accelerated genes underpinning synaptic function. This information-resolved approach provides analytic tools to disentangle information integration from coupling, enabling richer, more accurate interpretations of functional connectivity, and illuminating how the human neurocognitive architecture navigates the trade-off between robustness and integration. Decomposing neural information into synergistic and redundant components, Luppi et al. show how core brain regions support higher cognition in virtue of their synergy, revealing that human brains leverage synergistic information more than macaques.","datePublished":"2022-05-26T00:00:00Z","dateModified":"2022-05-26T00:00:00Z","pageStart":"771","pageEnd":"782","sameAs":"https://doi.org/10.1038/s41593-022-01070-0","keywords":["Cognitive control","Dynamical systems","Genetics of the nervous system","Network models","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig6_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig7_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"25","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Andrea I. Luppi","url":"http://orcid.org/0000-0002-3461-6431","affiliation":[{"name":"School of Clinical Medicine, University of Cambridge","address":{"name":"Division of Anaesthesia, School of Clinical Medicine, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Leverhulme Centre for the Future of Intelligence, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"The Alan Turing Institute","address":{"name":"The Alan Turing Institute, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"al857@cam.ac.uk","@type":"Person"},{"name":"Pedro A. M. Mediano","url":"http://orcid.org/0000-0003-1789-5894","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Psychology, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Queen Mary University of London","address":{"name":"Department of Psychology, Queen Mary University of London, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Fernando E. Rosas","url":"http://orcid.org/0000-0001-7790-6183","affiliation":[{"name":"Imperial College London","address":{"name":"Center for Psychedelic Research, Department of Brain Science, Imperial College London, London, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Data Science Institute, Imperial College London","address":{"name":"Data Science Institute, Imperial College London, London, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Complexity Science, Imperial College London","address":{"name":"Center for Complexity Science, Imperial College London, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Negin Holland","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tim D. Fryer","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Wolfson Brain Imaging Centre, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"John T. O’Brien","url":"http://orcid.org/0000-0002-0837-5080","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Psychiatry, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Cambridge University Hospitals NHS Foundation Trust","address":{"name":"Cambridge University Hospitals NHS Foundation Trust, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"James B. Rowe","url":"http://orcid.org/0000-0001-7216-8679","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Cambridge University Hospitals NHS Foundation Trust","address":{"name":"Cambridge University Hospitals NHS Foundation Trust, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"MRC Cognition and Brain Sciences Unit, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"David K. Menon","url":"http://orcid.org/0000-0002-3228-9692","affiliation":[{"name":"School of Clinical Medicine, University of Cambridge","address":{"name":"Division of Anaesthesia, School of Clinical Medicine, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Wolfson Brain Imaging Centre, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Daniel Bor","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Psychology, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Queen Mary University of London","address":{"name":"Department of Psychology, Queen Mary University of London, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Emmanuel A. Stamatakis","url":"http://orcid.org/0000-0001-6955-9601","affiliation":[{"name":"School of Clinical Medicine, University of Cambridge","address":{"name":"Division of Anaesthesia, School of Clinical Medicine, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-022-01070-0">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="A synergistic core for human brain evolution and cognition"/>
    <meta name="dc.source" content="Nature Neuroscience 2022 25:6"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2022-05-26"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2022 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2022 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="How does the organization of neural information processing enable humans&#8217; sophisticated cognition? Here we decompose functional interactions between brain regions into synergistic and redundant components, revealing their distinct information-processing roles. Combining functional and structural neuroimaging with meta-analytic results, we demonstrate that redundant interactions are predominantly associated with structurally coupled, modular sensorimotor processing. Synergistic interactions instead support integrative processes and complex cognition across higher-order brain networks. The human brain leverages synergistic information to a greater extent than nonhuman primates, with high-synergy association cortices exhibiting the highest degree of evolutionary cortical expansion. Synaptic density mapping from positron emission tomography and convergent molecular and metabolic evidence demonstrate that synergistic interactions are supported by receptor diversity and human-accelerated genes underpinning synaptic function. This information-resolved approach provides analytic tools to disentangle information integration from coupling, enabling richer, more accurate interpretations of functional connectivity, and illuminating how the human neurocognitive architecture navigates the trade-off between robustness and integration. Decomposing neural information into synergistic and redundant components, Luppi et al. show how core brain regions support higher cognition in virtue of their synergy, revealing that human brains leverage synergistic information more than macaques."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2022-05-26"/>
    <meta name="prism.volume" content="25"/>
    <meta name="prism.number" content="6"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="771"/>
    <meta name="prism.endingPage" content="782"/>
    <meta name="prism.copyright" content="2022 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-022-01070-0"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-022-01070-0"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-022-01070-0.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-022-01070-0"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="A synergistic core for human brain evolution and cognition"/>
    <meta name="citation_volume" content="25"/>
    <meta name="citation_issue" content="6"/>
    <meta name="citation_publication_date" content="2022/06"/>
    <meta name="citation_online_date" content="2022/05/26"/>
    <meta name="citation_firstpage" content="771"/>
    <meta name="citation_lastpage" content="782"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-022-01070-0"/>
    <meta name="DOI" content="10.1038/s41593-022-01070-0"/>
    <meta name="size" content="429949"/>
    <meta name="citation_doi" content="10.1038/s41593-022-01070-0"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-022-01070-0&amp;api_key="/>
    <meta name="description" content="How does the organization of neural information processing enable humans&#8217; sophisticated cognition? Here we decompose functional interactions between brain regions into synergistic and redundant components, revealing their distinct information-processing roles. Combining functional and structural neuroimaging with meta-analytic results, we demonstrate that redundant interactions are predominantly associated with structurally coupled, modular sensorimotor processing. Synergistic interactions instead support integrative processes and complex cognition across higher-order brain networks. The human brain leverages synergistic information to a greater extent than nonhuman primates, with high-synergy association cortices exhibiting the highest degree of evolutionary cortical expansion. Synaptic density mapping from positron emission tomography and convergent molecular and metabolic evidence demonstrate that synergistic interactions are supported by receptor diversity and human-accelerated genes underpinning synaptic function. This information-resolved approach provides analytic tools to disentangle information integration from coupling, enabling richer, more accurate interpretations of functional connectivity, and illuminating how the human neurocognitive architecture navigates the trade-off between robustness and integration. Decomposing neural information into synergistic and redundant components, Luppi et al. show how core brain regions support higher cognition in virtue of their synergy, revealing that human brains leverage synergistic information more than macaques."/>
    <meta name="dc.creator" content="Luppi, Andrea I."/>
    <meta name="dc.creator" content="Mediano, Pedro A. M."/>
    <meta name="dc.creator" content="Rosas, Fernando E."/>
    <meta name="dc.creator" content="Holland, Negin"/>
    <meta name="dc.creator" content="Fryer, Tim D."/>
    <meta name="dc.creator" content="O&#8217;Brien, John T."/>
    <meta name="dc.creator" content="Rowe, James B."/>
    <meta name="dc.creator" content="Menon, David K."/>
    <meta name="dc.creator" content="Bor, Daniel"/>
    <meta name="dc.creator" content="Stamatakis, Emmanuel A."/>
    <meta name="dc.subject" content="Cognitive control"/>
    <meta name="dc.subject" content="Dynamical systems"/>
    <meta name="dc.subject" content="Genetics of the nervous system"/>
    <meta name="dc.subject" content="Network models"/>
    <meta name="citation_reference" content="Marr, D. Vision: A Computational Investigation into the Human Representation and Processing of Visual Information (MIT Press, 2010)."/>
    <meta name="citation_reference" content="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at 
                  http://arxiv.org/abs/1004.2515
                  
                 (2010)."/>
    <meta name="citation_reference" content="citation_journal_title=J. Comput. Neurosci.; citation_title=Synergy, redundancy and multivariate information measures: an experimentalist&#8217;s perspective; citation_author=N Timme, W Alford, B Flecker, JM Beggs; citation_volume=36; citation_publication_date=2014; citation_pages=119-140; citation_doi=10.1007/s10827-013-0458-4; citation_id=CR3"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Cogn.; citation_title=Partial information decomposition as a unified approach to the specification of neural goal functions; citation_author=M Wibral, V Priesemann, JW Kay, JT Lizier, WA Phillips; citation_volume=112; citation_publication_date=2017; citation_pages=25-38; citation_doi=10.1016/j.bandc.2015.09.004; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=Cognition; citation_title=Basic functional trade-offs in cognition: an integrative framework; citation_author=M Giudice, BJ Crespi; citation_volume=179; citation_publication_date=2018; citation_pages=56-70; citation_doi=10.1016/j.cognition.2018.06.008; citation_id=CR5"/>
    <meta name="citation_reference" content="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at 
                  https://arxiv.org/abs/2109.13186
                  
                 (2021)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Extracting information from neuronal populations: information theory and decoding approaches; citation_author=R Quian Quiroga, S Panzeri; citation_volume=10; citation_publication_date=2009; citation_pages=173-185; citation_doi=10.1038/nrn2578; citation_id=CR7"/>
    <meta name="citation_reference" content="citation_journal_title=Front. Genet.; citation_title=Biological robustness: paradigms, mechanisms, systems principles; citation_author=JM Whitacre; citation_volume=3; citation_publication_date=2012; citation_pages=1-15; citation_doi=10.3389/fgene.2012.00067; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=A measure for brain complexity: relating functional segregation and integration in the nervous system; citation_author=G Tononi, O Sporns, GM Edelman; citation_volume=91; citation_publication_date=1994; citation_pages=5033-5037; citation_doi=10.1073/pnas.91.11.5033; citation_id=CR9"/>
    <meta name="citation_reference" content="Schneidman, E., Still, S., Berry, M. J. &amp; Bialek, W. Network information and connected correlations. Phys. Rev. Lett. 91, 23 (2003)."/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Advancing functional connectivity research from association to causation; citation_author=AT Reid; citation_volume=22; citation_publication_date=2019; citation_pages=1751-1760; citation_doi=10.1038/s41593-019-0510-4; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Quantifying high-order interdependencies via multivariate extensions of the mutual information; citation_author=FE Rosas, PAM Mediano, M Gastpar, HJ Jensen; citation_volume=100; citation_publication_date=2019; citation_pages=32305; citation_doi=10.1103/PhysRevE.100.032305; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. Lett.; citation_title=Measuring information transfer; citation_author=T Schreiber; citation_volume=85; citation_publication_date=2000; citation_pages=461-464; citation_doi=10.1103/PhysRevLett.85.461; citation_id=CR13"/>
    <meta name="citation_reference" content="Massey, J. Causality, feedback and directed information. Proc. Int. Symp. Inf. Theory Applic. 27&#8211;30 (1990)."/>
    <meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_title=The restless brain: how intrinsic activity organizes brain function; citation_author=ME Raichle; citation_volume=370; citation_publication_date=2015; citation_pages=20140172; citation_doi=10.1098/rstb.2014.0172; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Life Rev.; citation_title=Is temporo-spatial dynamics the &#8216;common currency&#8217; of brain and mind? In Quest of &#8216;spatiotemporal neuroscience&#8217;; citation_author=G Northoff, S Wainio-Theberge, K Evers; citation_volume=33; citation_publication_date=2020; citation_pages=34-54; citation_doi=10.1016/j.plrev.2019.05.002; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Brain network dynamics are hierarchically organized in time; citation_author=D Vidaurre, SM Smith, MW Woolrich; citation_volume=114; citation_publication_date=2017; citation_pages=12827-12832; citation_doi=10.1073/pnas.1705120114; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=The dynamics of functional brain networks: integrated network states during cognitive task performance; citation_author=JM Shine; citation_volume=92; citation_publication_date=2016; citation_pages=544-554; citation_doi=10.1016/j.neuron.2016.09.018; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Dynamic models of large-scale brain activity; citation_author=M Breakspear; citation_volume=20; citation_publication_date=2017; citation_pages=340-352; citation_doi=10.1038/nn.4497; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Consciousness-specific dynamic interactions of brain integration and functional diversity; citation_author=AI Luppi; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-12658-9; citation_id=CR20"/>
    <meta name="citation_reference" content="Cover, T. M. &amp; Thomas, J. A. Elements of Information Theory. (Wiley-Interscience, 2005). 
                  https://doi.org/10.1002/047174882X
                  
                "/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Exploration of synergistic and redundant information sharing in static and dynamical Gaussian systems; citation_author=AB Barrett; citation_volume=91; citation_publication_date=2015; citation_pages=52802; citation_doi=10.1103/PhysRevE.91.052802; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes; citation_author=SE Morgan; citation_volume=116; citation_publication_date=2019; citation_pages=9604-9609; citation_doi=10.1073/pnas.1820754116; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurophysiol.; citation_title=The organization of the human cerebral cortex estimated by intrinsic functional connectivity; citation_author=BTT Yeo; citation_volume=106; citation_publication_date=2011; citation_pages=1125-1165; citation_doi=10.1152/jn.00338.2011; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Situating the default-mode network along a principal gradient of macroscale cortical organization; citation_author=DS Margulies; citation_volume=113; citation_publication_date=2016; citation_pages=12574-12579; citation_doi=10.1073/pnas.1608282113; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Human cognition involves the dynamic integration of neural activity and neuromodulatory systems; citation_author=JM Shine; citation_volume=22; citation_publication_date=2019; citation_pages=289-296; citation_doi=10.1038/s41593-018-0312-0; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Decoupling of brain function from structure reveals regional behavioral specialization in humans; citation_author=MG Preti, D Ville; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-12765-7; citation_id=CR27"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Large-scale automated synthesis of human functional neuroimaging data; citation_author=T Yarkoni, RA Poldrack, TE Nichols, DC Essen, TD Wager; citation_volume=8; citation_publication_date=2011; citation_pages=665-670; citation_doi=10.1038/nmeth.1635; citation_id=CR28"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Brain networks and cognitive architectures; citation_author=SE Petersen, O Sporns; citation_volume=88; citation_publication_date=2015; citation_pages=207-219; citation_doi=10.1016/j.neuron.2015.09.027; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Network neuroscience; citation_author=DS Bassett, O Sporns; citation_volume=20; citation_publication_date=2017; citation_pages=353-364; citation_doi=10.1038/nn.4502; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=Topographic gradients of intrinsic dynamics across neocortex; citation_author=G Shafiei; citation_volume=9; citation_publication_date=2020; citation_pages=1-24; citation_doi=10.7554/eLife.62116; citation_id=CR31"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Biol.; citation_title=A multi-scale cortical wiring space links cellular architecture and functional dynamics in the human brain; citation_author=C Paquola; citation_volume=18; citation_publication_date=2020; citation_pages=e3000979; citation_doi=10.1371/journal.pbio.3000979; citation_id=CR32"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Biol.; citation_title=Exceptional evolutionary expansion of prefrontal cortex in great apes and humans; citation_author=JB Smaers, A G&#243;mez-Robles, AN Parks, CC Sherwood; citation_volume=27; citation_publication_date=2017; citation_pages=714-720; citation_doi=10.1016/j.cub.2017.01.020; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Genetic mapping and evolutionary analysis of human-expanded cognitive networks; citation_author=Y Wei; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-12764-8; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome; citation_author=KJ Whitaker; citation_volume=113; citation_publication_date=2016; citation_pages=9105-9110; citation_doi=10.1073/pnas.1601745113; citation_id=CR35"/>
    <meta name="citation_reference" content="Finnema, S. J. et al. Imaging synaptic density in the living human brain. Sci. Transl. Med. 8, 348ra96 (2016)."/>
    <meta name="citation_reference" content="Holland, N. et al. Synaptic loss in primary tauopathies revealed by [11C]UCB-J positron emission tomography. Mov. Disord. 
                  https://doi.org/10.1002/mds.28188
                  
                 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Cell Metabolism; citation_title=Aerobic glycolysis in the human brain is associated with development and neotenous gene expression; citation_author=MS Goyal, M Hawrylycz, JA Miller, AZ Snyder, ME Raichle; citation_volume=19; citation_publication_date=2014; citation_pages=49-57; citation_doi=10.1016/j.cmet.2013.11.020; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Regional aerobic glycolysis in the human brain; citation_author=SN Vaishnavi; citation_volume=107; citation_publication_date=2010; citation_pages=17757-17762; citation_doi=10.1073/pnas.1010459107; citation_id=CR39"/>
    <meta name="citation_reference" content="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)."/>
    <meta name="citation_reference" content="citation_journal_title=Front. Neuroanat.; citation_title=Multiple transmitter receptors in regions and layers of the human cerebral cortex; citation_author=K Zilles, N Palomero-Gallagher; citation_volume=11; citation_publication_date=2017; citation_pages=78; citation_doi=10.3389/fnana.2017.00078; citation_id=CR41"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Gene expression-based modeling of human cortical synaptic density; citation_author=MS Goyal, ME Raichle; citation_volume=110; citation_publication_date=2013; citation_pages=6571-6576; citation_doi=10.1073/pnas.1303453110; citation_id=CR42"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Conscious processing and the global neuronal workspace hypothesis; citation_author=GA Mashour, P Roelfsema, JP Changeux, S Dehaene; citation_volume=105; citation_publication_date=2020; citation_pages=776-798; citation_doi=10.1016/j.neuron.2020.01.026; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=J. Phys. A Math. Theor.; citation_title=An operational information decomposition via synergistic disclosure; citation_author=FE Rosas, PAM Mediano, B Rassouli, AB Barrett; citation_volume=53; citation_publication_date=2020; citation_pages=485001; citation_doi=10.1088/1751-8121/abb723; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=The evolution of distributed association networks in the human brain; citation_author=RL Buckner, FM Krienen; citation_volume=17; citation_publication_date=2013; citation_pages=648-665; citation_doi=10.1016/j.tics.2013.09.017; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=The default mode network: where the idiosyncratic self meets the shared social world; citation_author=Y Yeshurun, M Nguyen, U Hasson; citation_volume=22; citation_publication_date=2021; citation_pages=181-192; citation_doi=10.1038/s41583-020-00420-w; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Brain; citation_title=An anatomical study of converging sensory pathways within the cerebral cortex of the monkey; citation_author=EG Jones, TPS Powell; citation_volume=93; citation_publication_date=1970; citation_pages=793-820; citation_doi=10.1093/brain/93.4.793; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=The human brain is intrinsically organized into dynamic, anticorrelated functional networks; citation_author=MD Fox; citation_volume=102; citation_publication_date=2005; citation_pages=9673-9678; citation_doi=10.1073/pnas.0504136102; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=A default mode of brain function; citation_author=ME Raichle; citation_volume=98; citation_publication_date=2001; citation_pages=676-682; citation_doi=10.1073/pnas.98.2.676; citation_id=CR49"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Default mode dynamics for global functional integration; citation_author=D Vatansever, XDK Menon, AE Manktelow, BJ Sahakian, EA Stamatakis; citation_volume=35; citation_publication_date=2015; citation_pages=15254-15262; citation_doi=10.1523/JNEUROSCI.2135-15.2015; citation_id=CR50"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Controllability of structural brain networks; citation_author=S Gu; citation_volume=6; citation_publication_date=2015; citation_doi=10.1038/ncomms9414; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Multi-task connectivity reveals flexible hubs for adaptive task control; citation_author=MW Cole; citation_volume=16; citation_publication_date=2013; citation_pages=1348-1355; citation_doi=10.1038/nn.3470; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=On testing for spatial correspondence between maps of human brain structure and function; citation_author=AF Alexander-Bloch; citation_volume=178; citation_publication_date=2018; citation_pages=540-551; citation_doi=10.1016/j.neuroimage.2018.05.070; citation_id=CR53"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The WU-minn Human Connectome Project: an overview; citation_author=DC Essen; citation_volume=80; citation_publication_date=2013; citation_pages=62-79; citation_doi=10.1016/j.neuroimage.2013.05.041; citation_id=CR54"/>
    <meta name="citation_reference" content="citation_journal_title=Network Neuroscience; citation_title=Combining network topology and information theory to construct representative brain networks; citation_author=AI Luppi, EA Stamatakis; citation_volume=5; citation_publication_date=2021; citation_pages=96-124; citation_doi=10.1162/netn_a_00170; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Network Neuroscience; citation_title=Toward an information theoretical description of communication in brain networks; citation_author=E Amico; citation_volume=5; citation_publication_date=2021; citation_pages=646-665; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The minimal preprocessing pipelines for the Human Connectome Project; citation_author=MF Glasser; citation_volume=80; citation_publication_date=2013; citation_pages=105-124; citation_doi=10.1016/j.neuroimage.2013.04.127; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Connect.; citation_title=Conn: a functional connectivity toolbox for correlated and anticorrelated brain networks; citation_author=S Whitfield-Gabrieli, A Nieto-Castanon; citation_volume=2; citation_publication_date=2012; citation_pages=125-141; citation_doi=10.1089/brain.2012.0073; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=A component based noise correction method (CompCor) for BOLD and perfusion based fMRI; citation_author=Y Behzadi, K Restom, J Liau, TT Liu; citation_volume=37; citation_publication_date=2007; citation_pages=90-101; citation_doi=10.1016/j.neuroimage.2007.04.042; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=Med. Image Anal.; citation_title=A blind deconvolution approach to recover effective connectivity brain networks from resting state fMRI data; citation_author=G-R Wu; citation_volume=17; citation_publication_date=2013; citation_pages=365-374; citation_doi=10.1016/j.media.2013.01.003; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Estimation of fiber orientation and spin density distribution by diffusion deconvolution; citation_author=F-C Yeh, VJ Wedeen, W-YI Tseng; citation_volume=55; citation_publication_date=2011; citation_pages=1054-1062; citation_doi=10.1016/j.neuroimage.2010.11.087; citation_id=CR61"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Deterministic diffusion fiber tracking improved by quantitative anisotropy; citation_author=F-C Yeh, TD Verstynen, Y Wang, JC Fern&#225;ndez-Miranda, W-Y Tseng; citation_volume=8; citation_publication_date=2013; citation_pages=80713; citation_doi=10.1371/journal.pone.0080713; citation_id=CR62"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=An open resource for nonhuman primate imaging; citation_author=AP Michael Milham; citation_volume=100; citation_publication_date=2018; citation_pages=61-74; citation_doi=10.1016/j.neuron.2018.08.039; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Pypreclin: an automatic pipeline for macaque functional MRI preprocessing; citation_author=J Tasserie; citation_volume=207; citation_publication_date=2020; citation_pages=116353; citation_doi=10.1016/j.neuroimage.2019.116353; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage.; citation_title=A population MRI brain template and analysis tools for the macaque; citation_author=J Seidlitz; citation_volume=170; citation_publication_date=2018; citation_pages=121-131; citation_doi=10.1016/j.neuroimage.2017.04.063; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Signature of consciousness in the dynamics of resting-state brain activity; citation_author=P Barttfeld; citation_volume=112; citation_publication_date=2015; citation_pages=887-892; citation_doi=10.1073/pnas.1418031112; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Local&#8211;global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI; citation_author=A Schaefer; citation_volume=28; citation_publication_date=2018; citation_pages=3095-3114; citation_doi=10.1093/cercor/bhx179; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Topographic organization of the human subcortex unveiled with functional connectivity gradients; citation_author=Y Tian, D Margulies, M Breakspear, A Zalesky; citation_volume=23; citation_publication_date=2020; citation_pages=1421-1432; citation_doi=10.1038/s41593-020-00711-6; citation_id=CR68"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest; citation_author=RS Desikan; citation_volume=31; citation_publication_date=2006; citation_pages=968-980; citation_doi=10.1016/j.neuroimage.2006.01.021; citation_id=CR69"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Effects of network resolution on topological properties of human neocortex; citation_author=R Romero-Garcia, M Atienza, LH Clemmensen, JL Cantero; citation_volume=59; citation_publication_date=2012; citation_pages=3522-3532; citation_doi=10.1016/j.neuroimage.2011.10.086; citation_id=CR70"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A multi-modal parcellation of human cerebral cortex; citation_author=MF Glasser; citation_volume=536; citation_publication_date=2016; citation_pages=171-178; citation_doi=10.1038/nature18933; citation_id=CR71"/>
    <meta name="citation_reference" content="citation_journal_title=Philos. Trans. R Soc. B Biol. Sci.; citation_title=Mapping brains without coordinates; citation_author=R K&#246;tter, E Wanke; citation_volume=360; citation_publication_date=2005; citation_pages=751-766; citation_doi=10.1098/rstb.2005.1625; citation_id=CR72"/>
    <meta name="citation_reference" content="Schulz, M. A. et al. Different scaling of linear models and deep learning in UKBiobank brain images versus machine-learning datasets. Nat. Commun. 11, (2020)."/>
    <meta name="citation_reference" content="Nozari, E. et al. Is the brain macroscopically linear? A system identification of resting state dynamics. Preprint at arXiv 
                  https://doi.org/10.48550/arXiv.2012.12351
                  
                 (2020)."/>
    <meta name="citation_reference" content="B&#237;m, J. et al. A nonnegative measure of feature-related information transfer between neural signals. Preprint at bioRxiv, 
                  https://doi.org/10.1101/758128
                  
                 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Entropy; citation_title=Information decomposition of target effects from multi-source interactions: perspectives on previous, current and future work; citation_author=J Lizier, N Bertschinger, J Jost, M Wibral; citation_volume=20; citation_publication_date=2018; citation_pages=307; citation_doi=10.3390/e20040307; citation_id=CR76"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=Practical measures of integrated information for time-series data; citation_author=AB Barrett, AK Seth; citation_volume=7; citation_publication_date=2011; citation_pages=1001052; citation_doi=10.1371/journal.pcbi.1001052; citation_id=CR77"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Visual areas exert feedforward and feedback influences through distinct frequency channels; citation_author=AM Bastos; citation_volume=85; citation_publication_date=2015; citation_pages=390-401; citation_doi=10.1016/j.neuron.2014.12.018; citation_id=CR78"/>
    <meta name="citation_reference" content="Deco, G., Vidaurre, D. &amp; Kringelbach, M. L. Revisiting the global workspace: orchestration of the functional hierarchical organisation of the human brain. Nat. Hum. Behav. 
                  https://doi.org/10.1101/859579
                  
                 (2021)."/>
    <meta name="citation_reference" content="citation_journal_title=Front. Robot. AI; citation_title=JIDT: an information-theoretic toolkit for studying the dynamics of complex systems; citation_author=JT Lizier; citation_volume=1; citation_publication_date=2014; citation_pages=1-37; citation_doi=10.3389/frobt.2014.00011; citation_id=CR80"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Phys.; citation_title=The physics of higher-order interactions in complex systems; citation_author=F Battiston; citation_volume=17; citation_publication_date=2021; citation_pages=1093-1098; citation_doi=10.1038/s41567-021-01371-4; citation_id=CR81"/>
    <meta name="citation_reference" content="Rosas, F. E. et al. Disentangling high-order mechanisms and high-order behaviours in complex systems. Nat. Physics 
                  https://doi.org/10.1038/s41567-022-01548-5
                  
                 (2022)."/>
    <meta name="citation_reference" content="citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci.; citation_title=Gene transcription profiles associated with inter-modular hubs and connection distance in human functional magnetic resonance imaging networks; citation_author=PE V&#233;rtes; citation_volume=371; citation_publication_date=2016; citation_pages=20150362; citation_doi=10.1098/rstb.2015.0362; citation_id=CR83"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Complex network measures of brain connectivity: Uses and interpretations; citation_author=M Rubinov, O Sporns; citation_volume=52; citation_publication_date=2010; citation_pages=1059-1069; citation_doi=10.1016/j.neuroimage.2009.10.003; citation_id=CR84"/>
    <meta name="citation_reference" content="citation_journal_title=NeuroImage; citation_title=Weight-conserving characterization of complex functional brain networks; citation_author=M Rubinov, O Sporns; citation_volume=56; citation_publication_date=2011; citation_pages=2068-2079; citation_doi=10.1016/j.neuroimage.2011.03.069; citation_id=CR85"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The dynamics of human cognition: increasing global integration coupled with decreasing segregation found using iEEG; citation_author=J Cruzat; citation_volume=172; citation_publication_date=2018; citation_pages=492-505; citation_doi=10.1016/j.neuroimage.2018.01.064; citation_id=CR86"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl. Acad. Sci. USA; citation_title=Modularity and community structure in networks; citation_author=MEJ Newman; citation_volume=103; citation_publication_date=2006; citation_pages=8577-8582; citation_doi=10.1073/pnas.0601602103; citation_id=CR87"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=LSD alters dynamic integration and segregation in the human brain; citation_author=AI Luppi; citation_volume=227; citation_publication_date=2021; citation_pages=117653; citation_doi=10.1016/j.neuroimage.2020.117653; citation_id=CR88"/>
    <meta name="citation_reference" content="citation_journal_title=J. R. Soc. Interface; citation_title=Large-scale signatures of unconsciousness are consistent with a departure from critical dynamics; citation_author=E Tagliazucchi; citation_volume=13; citation_publication_date=2016; citation_pages=20151027; citation_doi=10.1098/rsif.2015.1027; citation_id=CR89"/>
    <meta name="citation_reference" content="citation_journal_title=Sci. Data; citation_title=A macaque connectome for large-scale network simulations in TheVirtualBrain; citation_author=K Shen; citation_volume=6; citation_publication_date=2019; citation_doi=10.1038/s41597-019-0129-z; citation_id=CR90"/>
    <meta name="citation_reference" content="Shen, K., Bezgin, G., Everling, S. &amp; McIntosh, A. R. The virtual macaque brain: a macaque connectome for large-scale network simulations in TheVirtualBrain. 
                  https://doi.org/10.5281/zenodo.1471588
                  
                 (2018)."/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci. Methods; citation_title=Mapping the human connectome at multiple scales with diffusion spectrum MRI; citation_author=Letal Cammoun; citation_volume=203; citation_publication_date=2012; citation_pages=386-397; citation_doi=10.1016/j.jneumeth.2011.09.031; citation_id=CR92"/>
    <meta name="citation_reference" content="citation_journal_title=Curr. Biol.; citation_title=Whole-brain multimodal neuroimaging model using serotonin receptor maps explains non-linear functional effects of LSD; citation_author=G Deco; citation_volume=28; citation_publication_date=2018; citation_pages=3065-3074; citation_doi=10.1016/j.cub.2018.07.083; citation_id=CR93"/>
    <meta name="citation_reference" content="citation_journal_title=Sci. Rep.; citation_title=A mechanistic model of the neural entropy increase elicited by psychedelic drugs; citation_author=R Herzog; citation_volume=10; citation_publication_date=2020; citation_doi=10.1038/s41598-020-74060-6; citation_id=CR94"/>
    <meta name="citation_reference" content="citation_journal_title=Commun. Biol.; citation_title=Whole-brain modelling identifies distinct but convergent paths to unconsciousness in anaesthesia and disorders of consciousness; citation_author=AI Luppi; citation_volume=5; citation_publication_date=2022; citation_pages=384; citation_doi=10.1038/s42003-022-03330-y; citation_id=CR95"/>
    <meta name="citation_reference" content="Mediano, P. A. M., Luppi, A. I., Herzog, R. &amp; Rosas, F. E. FastDMF: fast simulator of the dynamic mean-field model of brain dynamics. 
                  https://doi.org/10.5281/zenodo.6373512
                  
                 (2022)."/>
    <meta name="citation_reference" content="citation_journal_title=Sci. Adv.; citation_title=Inversion of a large-scale circuit model reveals a cortical hierarchy in the dynamic resting human brain; citation_author=P Wang; citation_volume=5; citation_publication_date=2019; citation_pages=1-12; citation_id=CR97"/>
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Mutations in human accelerated regions disrupt cognition and social behavior the homozygosity mapping consortium for autism HHS public access; citation_author=RN Doan; citation_volume=167; citation_publication_date=2016; citation_pages=341-354; citation_doi=10.1016/j.cell.2016.08.071; citation_id=CR98"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=An anatomically comprehensive atlas of the adult human brain transcriptome; citation_author=MJ Hawrylycz; citation_volume=489; citation_publication_date=2012; citation_pages=391-399; citation_doi=10.1038/nature11405; citation_id=CR99"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Spatio-temporal transcriptome of the human brain; citation_author=HJ Kang; citation_volume=478; citation_publication_date=2011; citation_pages=483-489; citation_doi=10.1038/nature10523; citation_id=CR100"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Structural covariance networks are coupled to expression of genes enriched in supragranular layers of the human cortex; citation_author=R Romero-Garcia; citation_volume=171; citation_publication_date=2018; citation_pages=256-267; citation_doi=10.1016/j.neuroimage.2017.12.060; citation_id=CR101"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Partial least-squares methods for neuroimaging: a tutorial and review; citation_author=A Krishnan, LJ Williams, AR McIntosh, H Abdi; citation_volume=56; citation_publication_date=2011; citation_pages=455-475; citation_doi=10.1016/j.neuroimage.2010.07.034; citation_id=CR102"/>
    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists; citation_author=E Eden, R Navon, I Steinfeld, D Lipson, Z Yakhini; citation_volume=10; citation_publication_date=2009; citation_pages=48; citation_doi=10.1186/1471-2105-10-48; citation_id=CR103"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Summarizes and visualizes long lists of Gene Ontology terms; citation_author=F Supek, M Bo&#353;njak, S &#711; Kunca, S &#711; Muc; citation_volume=6; citation_publication_date=2011; citation_pages=21800; citation_doi=10.1371/journal.pone.0021800; citation_id=CR104"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Overcoming false-positive gene-category enrichment in the analysis of spatially resolved transcriptomic brain atlas data; citation_author=BD Fulcher, A Arnatkeviciute, A Fornito; citation_volume=12; citation_publication_date=2021; citation_doi=10.1038/s41467-021-22862-1; citation_id=CR105"/>
    <meta name="citation_reference" content="citation_journal_title=Cerebral Cortex; citation_title=Adolescent tuning of association cortex in human structural brain networks; citation_author=F V&#225;&#353;a; citation_volume=28; citation_publication_date=2018; citation_pages=281-294; citation_doi=10.1093/cercor/bhx249; citation_id=CR106"/>
    <meta name="citation_reference" content="citation_journal_title=J. Labelled Comp. Radiopharm.; citation_title=Automated radiosynthesis of [11C]UCB-J for imaging synaptic density by positron emission tomography; citation_author=S Milicevic Sephton; citation_volume=63; citation_publication_date=2020; citation_pages=151-158; citation_doi=10.1002/jlcr.3828; citation_id=CR107"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Differential expression of synaptic vesicle protein 2 (SV2) isoforms; citation_author=SM Bajjalieh, GD Frantz, JM Weimann, SK McConnell, RH Scheller; citation_volume=14; citation_publication_date=1994; citation_pages=5223-5235; citation_doi=10.1523/JNEUROSCI.14-09-05223.1994; citation_id=CR108"/>
    <meta name="citation_reference" content="citation_journal_title=IEEE Trans. Med. Imaging; citation_title=Attenuation correction synthesis for hybrid PET-MR scanners: application to brain studies; citation_author=N Burgos; citation_volume=33; citation_publication_date=2014; citation_pages=2332-2341; citation_doi=10.1109/TMI.2014.2340135; citation_id=CR109"/>
    <meta name="citation_reference" content="Manavaki, R., Hong, Y. &amp; Fryer, T. D. Effect of brain MRI coil attenuation map processing on PET image quantification and uniformity for the GE SIGNA PET/MR. IEEE Nucl. Sci. Symp. Med. Imaging Conf. NSS/MIC (2019)."/>
    <meta name="citation_reference" content="citation_journal_title=J. Cereb. Blood Flow Metab.; citation_title=Noise reduction in the simplified reference tissue model for neuroreceptor functional imaging; citation_author=Y Wu, RE Carson; citation_volume=22; citation_publication_date=2002; citation_pages=1440-1452; citation_doi=10.1097/01.WCB.0000033967.83623.34; citation_id=CR111"/>
    <meta name="citation_reference" content="citation_journal_title=Eur. J. Nucl. Med. Mol. Imaging; citation_title=Quantifying SV2A density and drug occupancy in the human brain using [11C]UCB-J PET imaging and subcortical white matter as reference tissue; citation_author=M Koole; citation_volume=46; citation_publication_date=2019; citation_pages=396-406; citation_doi=10.1007/s00259-018-4119-8; citation_id=CR112"/>
    <meta name="citation_reference" content="citation_journal_title=J Cereb. Blood Flow Metab.; citation_title=Assessment of a white matter reference region for [11C]UCB-J PET quantification; citation_author=S Rossano; citation_volume=40; citation_publication_date=2020; citation_pages=1890-1901; citation_doi=10.1177/0271678X19879230; citation_id=CR113"/>
    <meta name="citation_reference" content="citation_journal_title=J. Royal Stat. Soc. Ser. B; citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; citation_author=Y Benjamini, Y Hochberg; citation_volume=57; citation_publication_date=1995; citation_pages=289-300; citation_id=CR114"/>
    <meta name="citation_reference" content="Borenstein, M., Hedges, L. V., Higgins, J. P. T. &amp; Rothstein, H. R. Introduction to Meta-Analysis. 1&#8211;421 
                  https://doi.org/10.1002/9780470743386
                  
                 (Wiley, 2009)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Comparing spatial null models for brain maps; citation_author=RD Markello, B Misic; citation_volume=236; citation_publication_date=2021; citation_pages=118052; citation_doi=10.1016/j.neuroimage.2021.118052; citation_id=CR116"/>
    <meta name="citation_author" content="Luppi, Andrea I."/>
    <meta name="citation_author_institution" content="Division of Anaesthesia, School of Clinical Medicine, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Leverhulme Centre for the Future of Intelligence, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="The Alan Turing Institute, London, UK"/>
    <meta name="citation_author" content="Mediano, Pedro A. M."/>
    <meta name="citation_author_institution" content="Department of Psychology, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Department of Psychology, Queen Mary University of London, London, UK"/>
    <meta name="citation_author" content="Rosas, Fernando E."/>
    <meta name="citation_author_institution" content="Center for Psychedelic Research, Department of Brain Science, Imperial College London, London, UK"/>
    <meta name="citation_author_institution" content="Data Science Institute, Imperial College London, London, UK"/>
    <meta name="citation_author_institution" content="Center for Complexity Science, Imperial College London, London, UK"/>
    <meta name="citation_author" content="Holland, Negin"/>
    <meta name="citation_author_institution" content="Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Fryer, Tim D."/>
    <meta name="citation_author_institution" content="Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Wolfson Brain Imaging Centre, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="O&#8217;Brien, John T."/>
    <meta name="citation_author_institution" content="Department of Psychiatry, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Cambridge University Hospitals NHS Foundation Trust, Cambridge, UK"/>
    <meta name="citation_author" content="Rowe, James B."/>
    <meta name="citation_author_institution" content="Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Cambridge University Hospitals NHS Foundation Trust, Cambridge, UK"/>
    <meta name="citation_author_institution" content="MRC Cognition and Brain Sciences Unit, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Menon, David K."/>
    <meta name="citation_author_institution" content="Division of Anaesthesia, School of Clinical Medicine, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Wolfson Brain Imaging Centre, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Bor, Daniel"/>
    <meta name="citation_author_institution" content="Department of Psychology, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Department of Psychology, Queen Mary University of London, London, UK"/>
    <meta name="citation_author" content="Stamatakis, Emmanuel A."/>
    <meta name="citation_author_institution" content="Division of Anaesthesia, School of Clinical Medicine, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author_institution" content="Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="A synergistic core for human brain evolution and cognition"/>
    <meta name="twitter:description" content="Nature Neuroscience - Decomposing neural information into synergistic and redundant components, Luppi et al. show how core brain regions support higher cognition in virtue of their synergy,..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-022-01070-0"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="A synergistic core for human brain evolution and cognition - Nature Neuroscience"/>
    <meta property="og:description" content="Decomposing neural information into synergistic and redundant components, Luppi et al. show how core brain regions support higher cognition in virtue of their synergy, revealing that human brains leverage synergistic information more than macaques."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-022-01070-0;doi=10.1038/s41593-022-01070-0;techmeta=36,38,57,59,61,78;subjmeta=116,1925,2150,2393,2583,2649,378,631;kwrd=Cognitive+control,Dynamical+systems,Genetics+of+the+nervous+system,Network+models">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1830994404&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-022-01070-0%26doi%3D10.1038/s41593-022-01070-0%26techmeta%3D36,38,57,59,61,78%26subjmeta%3D116,1925,2150,2393,2583,2649,378,631%26kwrd%3DCognitive+control,Dynamical+systems,Genetics+of+the+nervous+system,Network+models">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1830994404&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-022-01070-0%26doi%3D10.1038/s41593-022-01070-0%26techmeta%3D36,38,57,59,61,78%26subjmeta%3D116,1925,2150,2393,2583,2649,378,631%26kwrd%3DCognitive+control,Dynamical+systems,Genetics+of+the+nervous+system,Network+models"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-022-01070-0'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        A synergistic core for human brain evolution and cognition
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-022-01070-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-05-26">26 May 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">A synergistic core for human brain evolution and cognition</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Andrea_I_-Luppi-Aff1-Aff2-Aff3-Aff4" data-author-popup="auth-Andrea_I_-Luppi-Aff1-Aff2-Aff3-Aff4" data-author-search="Luppi, Andrea I." data-corresp-id="c1">Andrea I. Luppi<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-3461-6431"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3461-6431</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a>,<a href="#Aff3">3</a>,<a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pedro_A__M_-Mediano-Aff5-Aff6" data-author-popup="auth-Pedro_A__M_-Mediano-Aff5-Aff6" data-author-search="Mediano, Pedro A. M.">Pedro A. M. Mediano</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0003-1789-5894"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1789-5894</a></span><sup class="u-js-hide"><a href="#Aff5">5</a>,<a href="#Aff6">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Fernando_E_-Rosas-Aff7-Aff8-Aff9" data-author-popup="auth-Fernando_E_-Rosas-Aff7-Aff8-Aff9" data-author-search="Rosas, Fernando E.">Fernando E. Rosas</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-7790-6183"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7790-6183</a></span><sup class="u-js-hide"><a href="#Aff7">7</a>,<a href="#Aff8">8</a>,<a href="#Aff9">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Negin-Holland-Aff2" data-author-popup="auth-Negin-Holland-Aff2" data-author-search="Holland, Negin">Negin Holland</a><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tim_D_-Fryer-Aff2-Aff10" data-author-popup="auth-Tim_D_-Fryer-Aff2-Aff10" data-author-search="Fryer, Tim D.">Tim D. Fryer</a><sup class="u-js-hide"><a href="#Aff2">2</a>,<a href="#Aff10">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John_T_-O_Brien-Aff11-Aff12" data-author-popup="auth-John_T_-O_Brien-Aff11-Aff12" data-author-search="O’Brien, John T.">John T. O’Brien</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-0837-5080"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0837-5080</a></span><sup class="u-js-hide"><a href="#Aff11">11</a>,<a href="#Aff12">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-James_B_-Rowe-Aff2-Aff12-Aff13" data-author-popup="auth-James_B_-Rowe-Aff2-Aff12-Aff13" data-author-search="Rowe, James B.">James B. Rowe</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-7216-8679"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7216-8679</a></span><sup class="u-js-hide"><a href="#Aff2">2</a>,<a href="#Aff12">12</a>,<a href="#Aff13">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-David_K_-Menon-Aff1-Aff10" data-author-popup="auth-David_K_-Menon-Aff1-Aff10" data-author-search="Menon, David K.">David K. Menon</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0002-3228-9692"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3228-9692</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff10">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Daniel-Bor-Aff5-Aff6" data-author-popup="auth-Daniel-Bor-Aff5-Aff6" data-author-search="Bor, Daniel">Daniel Bor</a><sup class="u-js-hide"><a href="#Aff5">5</a>,<a href="#Aff6">6</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 10 authors for this article" title="Show all 10 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Emmanuel_A_-Stamatakis-Aff1-Aff2" data-author-popup="auth-Emmanuel_A_-Stamatakis-Aff1-Aff2" data-author-search="Stamatakis, Emmanuel A.">Emmanuel A. Stamatakis</a><span class="u-js-hide"> 
            <a class="js-orcid" href="http://orcid.org/0000-0001-6955-9601"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6955-9601</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup> </li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span> 25</b>, <span class="u-visually-hidden">pages </span>771–782 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">12k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">49 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">342 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-022-01070-0/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/cognitive-control" data-track="click" data-track-action="view subject" data-track-label="link">Cognitive control</a></li><li class="c-article-subject-list__subject"><a href="/subjects/dynamical-systems" data-track="click" data-track-action="view subject" data-track-label="link">Dynamical systems</a></li><li class="c-article-subject-list__subject"><a href="/subjects/genetics-of-the-nervous-system" data-track="click" data-track-action="view subject" data-track-label="link">Genetics of the nervous system</a></li><li class="c-article-subject-list__subject"><a href="/subjects/network-models" data-track="click" data-track-action="view subject" data-track-label="link">Network models</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>How does the organization of neural information processing enable humans’ sophisticated cognition? Here we decompose functional interactions between brain regions into synergistic and redundant components, revealing their distinct information-processing roles. Combining functional and structural neuroimaging with meta-analytic results, we demonstrate that redundant interactions are predominantly associated with structurally coupled, modular sensorimotor processing. Synergistic interactions instead support integrative processes and complex cognition across higher-order brain networks. The human brain leverages synergistic information to a greater extent than nonhuman primates, with high-synergy association cortices exhibiting the highest degree of evolutionary cortical expansion. Synaptic density mapping from positron emission tomography and convergent molecular and metabolic evidence demonstrate that synergistic interactions are supported by receptor diversity and human-accelerated genes underpinning synaptic function. This information-resolved approach provides analytic tools to disentangle information integration from coupling, enabling richer, more accurate interpretations of functional connectivity, and illuminating how the human neurocognitive architecture navigates the trade-off between robustness and integration.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-022-01070-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-022-01070-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-024-07178-6/MediaObjects/41586_2024_7178_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-024-07178-6?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41586-024-07178-6">Neural signatures of natural behaviour in socializing macaques
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">13 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Camille Testard, Sébastien Tremblay, … Michael L. Platt</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-024-07109-5/MediaObjects/41586_2024_7109_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41586-024-07109-5?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41586-024-07109-5">A concerted neuron–astrocyte program declines in ageing and schizophrenia
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">06 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Emi Ling, James Nemesh, … Steven A. McCarroll</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41593-024-01597-4/MediaObjects/41593_2024_1597_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41593-024-01597-4?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41593-024-01597-4">Co-dependent excitatory and inhibitory plasticity accounts for quick, stable and long-lasting memories in biological networks
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">20 March 2024</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Everton J. Agnes &amp; Tim P. Vogels</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'topic',
                        model: 'visits_v2',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711582169,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>In theoretical and cognitive neuroscience, considering the human brain as a distributed information-processing system has emerged as a powerful framework to understand the neural basis of cognition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Marr, D. Vision: A Computational Investigation into the Human Representation and Processing of Visual Information (MIT Press, 2010)." href="/articles/s41593-022-01070-0#ref-CR1" id="ref-link-section-d30343262e798">1</a></sup>. However, information is not all the same; rather, several fundamentally distinct kinds of information can be discerned, each providing specific advantages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at &#xA;                  http://arxiv.org/abs/1004.2515&#xA;                  &#xA;                 (2010)." href="#ref-CR2" id="ref-link-section-d30343262e802">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Timme, N., Alford, W., Flecker, B. &amp; Beggs, J. M. Synergy, redundancy and multivariate information measures: an experimentalist’s perspective. J. Comput. Neurosci. 36, 119–140 (2014)." href="#ref-CR3" id="ref-link-section-d30343262e802_1">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Wibral, M., Priesemann, V., Kay, J. W., Lizier, J. T. &amp; Phillips, W. A. Partial information decomposition as a unified approach to the specification of neural goal functions. Brain Cogn. 112, 25–38 (2017)." href="/articles/s41593-022-01070-0#ref-CR4" id="ref-link-section-d30343262e805">4</a></sup>. Therefore, to properly understand any information-processing architecture, including the human brain, it is necessary to provide an account of what kind of information is being processed.</p><p>As an example, consider the two main sources of information about the world in humans: the eyes. Each eye provides some information about the periphery of the visual field, which the other eye cannot see. This is each eye’s ‘unique information’. In contrast, the information that we still have when we close either eye is known as ‘redundant’ (or ‘shared’) information—because it is information that is provided equally by multiple sources (for instance, information about color is largely redundant between the two eyes). Redundancy provides the system with robustness<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Del Giudice, M. &amp; Crespi, B. J. Basic functional trade-offs in cognition: an integrative framework. Cognition 179, 56–70 (2018)." href="/articles/s41593-022-01070-0#ref-CR5" id="ref-link-section-d30343262e812">5</a></sup>: we can still see with one eye closed, because the same information is largely available from the other eye. However, closing one eye also deprives us of stereoscopic information about depth. This information does not come from either eye alone: one needs both eyes working together, to perceive the third dimension through stereopsis. This is known as ‘synergistic’ (or ‘complementary’) information between two sources: the extra advantage obtained from combining and integrating them, reflecting their complementary nature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at &#xA;                  http://arxiv.org/abs/1004.2515&#xA;                  &#xA;                 (2010)." href="/articles/s41593-022-01070-0#ref-CR2" id="ref-link-section-d30343262e816">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e819">6</a></sup>.</p><p>Thus, beyond their own unique information, different sources can also provide redundant information (equally available from either source) or synergistic information (only available by combining both sources). This is a fundamental distinction because a system will not be able to perform cognitively useful computation, just by having several copies of the same information: computation and cognition demand that information should eventually be combined.</p><p>Crucially, synergy and redundancy are fundamental concepts that apply to any specific content that is being encoded, be it about an apple or a predator<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Quian Quiroga, R. &amp; Panzeri, S. Extracting information from neuronal populations: information theory and decoding approaches. Nat. Rev. Neurosci. 10, 173–185 (2009)." href="/articles/s41593-022-01070-0#ref-CR7" id="ref-link-section-d30343262e829">7</a></sup>. They are fundamentally different kinds of information encoding, and every information-processing system, including the human brain, needs to strike a balance between these distinct kinds of information and the specific advantages they provide: robustness and integration, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Timme, N., Alford, W., Flecker, B. &amp; Beggs, J. M. Synergy, redundancy and multivariate information measures: an experimentalist’s perspective. J. Comput. Neurosci. 36, 119–140 (2014)." href="/articles/s41593-022-01070-0#ref-CR3" id="ref-link-section-d30343262e833">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Whitacre, J. M. Biological robustness: paradigms, mechanisms, systems principles. Front. Genet. 3, 1–15 (2012)." href="#ref-CR8" id="ref-link-section-d30343262e836">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tononi, G., Sporns, O. &amp; Edelman, G. M. A measure for brain complexity: relating functional segregation and integration in the nervous system. Proc. Natl. Acad. Sci. USA 91, 5033–5037 (1994)." href="#ref-CR9" id="ref-link-section-d30343262e836_1">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Schneidman, E., Still, S., Berry, M. J. &amp; Bialek, W. Network information and connected correlations. Phys. Rev. Lett. 91, 23 (2003)." href="/articles/s41593-022-01070-0#ref-CR10" id="ref-link-section-d30343262e839">10</a></sup>. Therefore, understanding how the human brain navigates the trade-off between these different kinds of information could provide fundamental insights about its information-processing architecture.</p><p>However, synergistic and redundant information cannot be adequately captured by traditional measures of ‘functional connectivity’ (FC) in the human brain, which simply quantify the similarity between regional activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e847">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Reid, A. T. et al. Advancing functional connectivity research from association to causation. Nat. Neurosci. 22, 1751–1760 (2019)." href="/articles/s41593-022-01070-0#ref-CR11" id="ref-link-section-d30343262e850">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Rosas, F. E., Mediano, P. A. M., Gastpar, M. &amp; Jensen, H. J. Quantifying high-order interdependencies via multivariate extensions of the mutual information. Phys. Rev. E 100, 32305 (2019)." href="/articles/s41593-022-01070-0#ref-CR12" id="ref-link-section-d30343262e853">12</a></sup>, nor by approaches focused on capturing the direction of information transfer from one region to another<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Schreiber, T. Measuring information transfer. Phys. Rev. Lett. 85, 461–464 (2000)." href="/articles/s41593-022-01070-0#ref-CR13" id="ref-link-section-d30343262e857">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Massey, J. Causality, feedback and directed information. Proc. Int. Symp. Inf. Theory Applic. 27–30 (1990)." href="/articles/s41593-022-01070-0#ref-CR14" id="ref-link-section-d30343262e860">14</a></sup>. Hence, it is currently unknown to what extent the human brain differentially relies on synergistic versus redundant information for processing, and to what extent the involvement of these different kinds of information varies across distinct macroscale neural systems and cognitive domains.</p><p>Here, we address these fundamental questions in neuroscience by providing an ‘information-resolved’ framework to decompose the intrinsic information flow within the brain’s BOLD signals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e867">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Tononi, G., Sporns, O. &amp; Edelman, G. M. A measure for brain complexity: relating functional segregation and integration in the nervous system. Proc. Natl. Acad. Sci. USA 91, 5033–5037 (1994)." href="/articles/s41593-022-01070-0#ref-CR9" id="ref-link-section-d30343262e870">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Raichle, M. E. The restless brain: how intrinsic activity organizes brain function. Philos. Trans. R. Soc. Lond. B Biol. Sci. 370, 20140172 (2015)." href="/articles/s41593-022-01070-0#ref-CR15" id="ref-link-section-d30343262e873">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Northoff, G., Wainio-Theberge, S. &amp; Evers, K. Is temporo-spatial dynamics the ‘common currency’ of brain and mind? In Quest of ‘spatiotemporal neuroscience’. Phys. Life Rev. 33, 34–54 (2020)." href="/articles/s41593-022-01070-0#ref-CR16" id="ref-link-section-d30343262e876">16</a></sup> to quantify synergistic and redundant interactions based on the recently developed Integrated Information Decomposition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e880">6</a></sup>. This approach regards the brain as a dynamical system, whose temporal evolution is influenced by the current state of its constituent parts (brain regions) and their interactions. This means that the brain’s current state intrinsically carries information about its own future—information that we can decompose into synergistic and redundant contributions. Specifically, we quantify redundancy as information about the system’s future that is equally available from the current state of each of the parts (brain regions). In contrast, synergy is quantified as the additional information arising from the interactions between regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e884">6</a></sup>. Crucially, this mathematical definition indicates that synergy provides a rigorous quantification of how brain regions mutually influence each other’s activity over time: that is, the integration of information between those regions.</p><p>Therefore, our information-resolved approach to functional interactions in the brain contributes to reducing the gap between a fundamental concept of interest in neuroscience—integration of information—and neuroscientists’ practical ability to quantify it at the macroscale. Although appealing for its conceptual simplicity, traditional FC essentially reflects the similarity between regions’ temporal fluctuations, and it is therefore further removed from the phenomenon of theoretical interest: integration of information. It is helpful to compare our approach to dynamic FC analysis. Dynamic FC goes beyond traditional ‘static’ FC, by demonstrating that static FC in fact comprises distinct time-resolved patterns, with distinct roles for cognition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vidaurre, D., Smith, S. M. &amp; Woolrich, M. W. Brain network dynamics are hierarchically organized in time. Proc. Natl. Acad. Sci. USA 114, 12827–12832 (2017)." href="#ref-CR17" id="ref-link-section-d30343262e891">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shine, J. M. et al. The dynamics of functional brain networks: integrated network states during cognitive task performance. Neuron 92, 544–554 (2016)." href="#ref-CR18" id="ref-link-section-d30343262e891_1">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Breakspear, M. Dynamic models of large-scale brain activity. Nat. Neurosci. 20, 340–352 (2017)." href="#ref-CR19" id="ref-link-section-d30343262e891_2">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Luppi, A. I. et al. Consciousness-specific dynamic interactions of brain integration and functional diversity. Nat. Commun. 10, 4616 (2019)." href="/articles/s41593-022-01070-0#ref-CR20" id="ref-link-section-d30343262e894">20</a></sup>. Analogously, Integrated Information Decomposition provides insights over and above those offered by traditional FC, because it reveals distinct information-resolved connectivity patterns that could not be discerned from traditional FC. More specifically, synergy corresponds to information that only becomes available when the activity of different regions is considered together (reflecting the value of integration), whereas redundancy corresponds to information that is available from multiple brain regions (providing robustness)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e898">6</a></sup>.</p><p>By providing both a richer conceptual framework and widely applicable computational tools to disentangle different kinds of interactions, our information-resolved approach simultaneously highlights and overcomes some of the fundamental limitations of traditional FC analysis. Its richer perspective on neural information dynamics offers the opportunity to investigate the information-processing principles that govern how cognitive functions arise from the brain’s neural architecture. In this work, we investigate how synergistic and redundant information are organized with respect to the human neurocognitive architecture. Specifically, we seek to establish whether the extent of their involvement varies consistently with the functional roles of well-known macroscale neural systems, and whether studying brain function in terms of synergistic and redundant information provides a fertile new perspective complementing our current understanding of the functional organization of the brain. Combining multimodal neuroimaging (functional, structural and diffusion magnetic resonance imaging (MRI) and positron emission tomography (PET)) with cytoarchitectonics, in vitro autoradiography, and genetic evidence, here we reveal the organization and neurobiological underpinnings of synergy and redundancy. Comparing the brains of humans and nonhuman primates, we further demonstrate the involvement of synergy in humans’ sophisticated cognitive abilities and also in the evolution of the human brain.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Information decomposition</h3><p>We developed an information-resolved approach for the analysis of macroscale functional interactions in brain dynamics, to investigate how synergistic and redundant information are organized in the human brain, their functional relevance, and the neurobiology that supports them. Shannon’s mutual information quantifies the interdependence between two random variables <i>X</i> and <i>Y</i>. Put simply, the mutual information <i>I</i>(<i>X</i>;<i>Y</i>) quantifies the information that source variable <i>X</i> provides about target variable <i>Y</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Cover, T. M. &amp; Thomas, J. A. Elements of Information Theory. (Wiley-Interscience, 2005). &#xA;                  https://doi.org/10.1002/047174882X&#xA;                  &#xA;                " href="/articles/s41593-022-01070-0#ref-CR21" id="ref-link-section-d30343262e938">21</a></sup>. Partial information decomposition (PID) extends Shannon’s theory, showing that the information that two source variables <i>X</i> and <i>Y</i> provide about a third target variable <i>Z</i>, <i>I</i>(<i>X</i>,<i>Y</i>; <i>Z</i>), can be decomposed into three conceptually distinct kinds of information: information provided by one source but not the other (unique information), or by both sources separately (redundant information), or jointly by their combination (synergistic information)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at &#xA;                  http://arxiv.org/abs/1004.2515&#xA;                  &#xA;                 (2010)." href="/articles/s41593-022-01070-0#ref-CR2" id="ref-link-section-d30343262e964">2</a></sup>. Integrated information decomposition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e968">6</a></sup> provides a further extension of this principle, but in time. Consider a system <i>X</i> comprised of two parts evolving jointly over time, <span class="mathjax-tex">\(X_t = \left\{ {X_t^1,X_t^2} \right\}\)</span>. In our case, this corresponds to the time series of the BOLD activity of two brain regions. The amount of information flowing from the system’s past to its future is known as the time-delayed mutual information (TDMI)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e1053">6</a></sup> and given by <span class="mathjax-tex">\(I\left( {X_{t - \tau }^1,X_{t - \tau }^2;X_t^1,X_t^2} \right)\)</span>. The fundamental advancement offered by Integrated Information Decomposition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e1170">6</a></sup> is to decompose TDMI into redundant, unique and synergistic information shared with respect to both the past and the present states of both variables (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-022-01070-0#Sec12">Methods</a>). Here, we focus on the temporally persistent redundancy and synergy. As interdependencies between the past and present states of both <i>X</i><sub><i>t</i></sub> and <i>Y</i><sub><i>t</i></sub>, synergy and redundancy capture phenomena that are not assessed by approaches focused on the transfer of information from the past of one to the present of the other; therefore, they are distinct from (and complementary to) alternative information-theoretic measures such as transfer entropy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Schreiber, T. Measuring information transfer. Phys. Rev. Lett. 85, 461–464 (2000)." href="/articles/s41593-022-01070-0#ref-CR13" id="ref-link-section-d30343262e1190">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Massey, J. Causality, feedback and directed information. Proc. Int. Symp. Inf. Theory Applic. 27–30 (1990)." href="/articles/s41593-022-01070-0#ref-CR14" id="ref-link-section-d30343262e1193">14</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec4">Distinct neuroanatomical profiles for synergy and redundancy</h3><p>Here, we applied Integrated Information Decomposition to resting-state functional MRI data from 100 Human Connectome Project (HCP) participants, allowing us to quantify how much information about the brain’s future trajectory is carried redundantly by distinct brain regions, and how much of it is carried by the synergy between regions. Distinguishing synergistic and redundant interactions between the BOLD time series of each pair of 232 cortical and subcortical brain regions, we reveal that synergy and redundancy are distributed following distinct patterns across the human brain (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig1">1a,b</a>). In particular, we show that the organization of redundancy between pairs of brain regions is significantly more similar to traditional FC (Pearson correlation) than synergy: pairs of regions with more correlated time courses are more likely to provide redundant information, and less likely to provide synergistic information (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig1">1c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig8">1</a>). Note that synergy and redundancy do not need to be anticorrelated, at a theoretical level, and models can be constructed where, for example, both increase, or where one changes while the other remains constant<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Barrett, A. B. Exploration of synergistic and redundant information sharing in static and dynamical Gaussian systems. Phys. Rev. E 91, 52802 (2015)." href="/articles/s41593-022-01070-0#ref-CR22" id="ref-link-section-d30343262e1214">22</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Synergistic and redundant networks exhibit distinct anatomical and cognitive profiles."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Synergistic and redundant networks exhibit distinct anatomical and cognitive profiles.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="500"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>,<b>b</b>, Group-average matrices display the redundant (<b>a</b>) and synergistic (<b>b</b>) interactions between each pair of brain regions (note that color bars do not include entries on the diagonal). Brain plots show the cortical distribution of the strongest redundant (blue) and synergistic (red) connections (thresholded to retain the top 5% of connections, for display purposes only). SOM, somatomotor network; VIS, visual network; SAL, salience/ventral attention network; DAN, dorsal attention network; LIM, limbic network; SUB, subcortical network. <b>c</b>, participant-specific Pearson correlation values of synergy and redundancy matrices with the matrix of traditional FC (redundancy: M = 0.62, s.d. = 0.24; synergy: M = −0.29, s.d. = 0.16; <i>t</i>(99) = 24.06, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = 4.39, effect size confidence interval (CI): (3.88, 5.11), from paired-sample non-parametric permutation <i>t</i>-test (two-sided); <i>n</i> = 100 unrelated HCP participants). Note that the relationships of redundancy and synergy with traditional FC are not equal: the absolute value of the correlation was significantly stronger for redundancy than for synergy (redundancy: M = 0.62, s.d. = 0.24; synergy: M = 0.29, s.d. = 0.16; <i>t</i>(99) = 20.35, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = 1.65, effect size CI: (1.36, 1.92), from paired-sample non-parametric permutation <i>t</i>-test (two-sided); <i>n</i> = 100 unrelated HCP participants). In the violin plots, each colored circle represents one participant; white circle, median; blue line, mean; box limits, upper and lower quartiles; whiskers, 1.5× interquartile range; ***<i>P</i> &lt; 0.001. <b>d</b>, Brain surface projections of regional redundancy-to-synergy gradient scores, based on the respective ranks. These ranks exhibited an inverse correlation (Spearman’s ρ = −0.40, <i>P</i> &lt; 0.001; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig8">1</a>). <b>e</b>, NeuroSynth term-based meta-analysis, relating the distribution of redundancy-to-synergy gradient across the brain to a cognitive gradient of cognitive domains, from lower-level sensorimotor processing to higher-level cognitive tasks. These results are robust to the use of different parcellations (cortical only; having lower or higher number of nodes; and obtained from anatomical rather than functional considerations; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig9">2a,b</a>). Likewise, although these results were obtained after deconvolving the hemodynamic response function from the BOLD signals to account for regional variations (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-022-01070-0#Sec12">Methods</a>), analogous results are also obtained when this step is omitted, or if synergy and redundancy are computed from discretized rather than continuous data (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig9">2c,d</a>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We ranked each brain region separately in terms of how synergistic and redundant its interactions with other brain regions are; the difference between these ranks (synergy minus redundancy) determines the relative relevance of a given region for synergistic versus redundant processing, thereby defining a redundancy-to-synergy gradient across brain regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig1">1d</a>). The results of our information-resolved analysis reveal that redundant interactions are especially prominent in the brain’s somatomotor and salience subnetworks, and most visual regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig2">2a</a>), corresponding to primary sensory, primary motor and insular cortices, in terms of Von Economo’s cytoarchitectonic classification<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e1318">23</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig2">2b</a>). In contrast, regions with higher relative importance for synergy predominate in higher-order association cortices, and are affiliated with the default mode network (DMN), frontoparietal executive control network (FPN) and ‘limbic’ (orbitofrontal cortex and temporal poles) subnetwork<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-022-01070-0#ref-CR24" id="ref-link-section-d30343262e1325">24</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig2">2a,b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Distinct cytoarchitectonic and resting-state network profiles for synergy-dominated and redundancy-dominated regions."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Distinct cytoarchitectonic and resting-state network profiles for synergy-dominated and redundancy-dominated regions.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="346"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Regional redundancy-to-synergy gradient values for each canonical resting-state network, based on the definition of Yeo et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-022-01070-0#ref-CR24" id="ref-link-section-d30343262e1347">24</a></sup>. <b>b</b>, Regional redundancy-to-synergy gradient values for each of seven cytoarchitectonic classes (the five canonical classes identified by Von Economo, plus limbic and insular cortices), for 308 cortical regions of interest (ROIs) of equal size (500 mm<sup>2</sup>), obtained from subdivisions of the Desikan–Killiany cortical parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e1356">23</a></sup>. Each colored circle represents one brain region; <i>n</i> = 232 regions divided into eight resting-state networks (<b>a</b>); <i>n</i> = 308 regions divided into seven cytoarchitectonic classes (<b>b</b>). Each violin plot shows the distribution of brain regions assigned to the subnetwork or cytoarchitectonic class indicated on the <i>x</i> axis (each region is assigned exactly once). White circles represent median values, central lines represent the mean values, box limits represent upper and lower quartiles and whiskers represent 1.5× the interquartile range; *<i>P</i> &lt; 0.05, **<i>P</i> &lt; 0.01 and ***<i>P</i> &lt; 0.001, from one-sample non-parametric permutation <i>t</i>-test (two-sided), corrected for multiple comparisons using the false discovery rate (FDR). Full statistical results are shown in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">1</a> (Yeo RSN) and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">2</a> (Von Economo).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The distinct subnetwork affiliations and cytoarchitectonic profiles further suggest that redundant and synergistic interactions may be involved with radically different cognitive domains. To empirically validate this hypothesis, we performed a term-based meta-analysis using NeuroSynth, which is widely used to characterize macroscale brain patterns in terms of cognitive relevance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 12574–12579 (2016)." href="#ref-CR25" id="ref-link-section-d30343262e1406">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shine, J. M. et al. Human cognition involves the dynamic integration of neural activity and neuromodulatory systems. Nat. Neurosci. 22, 289–296 (2019)." href="#ref-CR26" id="ref-link-section-d30343262e1406_1">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Preti, M. G. &amp; van de Ville, D. Decoupling of brain function from structure reveals regional behavioral specialization in humans. Nat. Commun. 10, 4747 (2019)." href="#ref-CR27" id="ref-link-section-d30343262e1406_2">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. Nat. Methods 8, 665–670 (2011)." href="/articles/s41593-022-01070-0#ref-CR28" id="ref-link-section-d30343262e1409">28</a></sup>. NeuroSynth enables automated probabilistic mappings between broad cognitive domains and neural patterns, by synthesizing thousands of published functional MRI (fMRI) studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. Nat. Methods 8, 665–670 (2011)." href="/articles/s41593-022-01070-0#ref-CR28" id="ref-link-section-d30343262e1413">28</a></sup>. We used 24 topic terms used by previous studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-022-01070-0#ref-CR25" id="ref-link-section-d30343262e1417">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Preti, M. G. &amp; van de Ville, D. Decoupling of brain function from structure reveals regional behavioral specialization in humans. Nat. Commun. 10, 4747 (2019)." href="/articles/s41593-022-01070-0#ref-CR27" id="ref-link-section-d30343262e1420">27</a></sup>, ranging from lower sensorimotor functions (for example, eye movement, motion, visual and auditory perception) to higher cognitive functions (for example, attention, working memory, social and numerical cognition). The redundancy-to-synergy gradient identified in terms of regional rank differences was then related to these 24 terms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-022-01070-0#ref-CR25" id="ref-link-section-d30343262e1424">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Preti, M. G. &amp; van de Ville, D. Decoupling of brain function from structure reveals regional behavioral specialization in humans. Nat. Commun. 10, 4747 (2019)." href="/articles/s41593-022-01070-0#ref-CR27" id="ref-link-section-d30343262e1427">27</a></sup>.</p><p>Supporting the inference from neuroanatomy to cognition, our results reveal that the regional gradient from redundancy to synergy corresponds to a gradient from lower sensorimotor functions to higher cognitive ones, requiring the integration of complex information. Specifically, high-redundancy regions loaded strongly onto auditory, visual and multisensory processing and motion. In contrast, high-synergy regions had the strongest loadings onto social cognition, memory and cognitive control (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig1">1e</a>).</p><h3 class="c-article__sub-heading" id="Sec5">Distinct graph-theoretic profiles for synergy and redundancy</h3><p>From a theoretical standpoint, sensorimotor and higher-order cognitive functions impose distinct and opposite demands on cognitive architectures: sensory processing benefits from segregation of the whole network into segregated modules, whereas integration of information demands a highly interconnected network organization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Tononi, G., Sporns, O. &amp; Edelman, G. M. A measure for brain complexity: relating functional segregation and integration in the nervous system. Proc. Natl. Acad. Sci. USA 91, 5033–5037 (1994)." href="/articles/s41593-022-01070-0#ref-CR9" id="ref-link-section-d30343262e1445">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Petersen, S. E. &amp; Sporns, O. Brain networks and cognitive architectures. Neuron 88, 207–219 (2015)." href="/articles/s41593-022-01070-0#ref-CR29" id="ref-link-section-d30343262e1448">29</a></sup></p><p>The set of all synergistic (respectively, redundant) interactions between pairs of brain regions can be viewed as a whole-brain network, whereby each node is a region, and each edge represents the synergistic (respectively, redundant) information between the corresponding two regions. This makes it possible to combine the advantages of our information-resolved analysis with the powerful mathematics of graph theory<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Bassett, D. S. &amp; Sporns, O. Network neuroscience. Nat. Neurosci. 20, 353–364 (2017)." href="/articles/s41593-022-01070-0#ref-CR30" id="ref-link-section-d30343262e1454">30</a></sup> to obtain insights into the network organizations of synergistic and redundant interactions in the human brain. Indeed, combining Integrated Information Decomposition with subsequent graph-theoretical analysis of the resulting synergistic and redundant whole-brain networks, reveals how the human brain resolves the tension between specialized processing and global integration.</p><p>Across individuals, the whole-brain network of synergistic interactions is more highly interconnected and globally efficient than the corresponding whole-brain network defined by the redundancy between brain regions, owing to the high strength of synergistic connections (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig3">3a</a>). In contrast, redundant interactions delineate a network characterized by a highly modular structure, which is virtually absent in the network of synergistic interactions across the brain (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig3">3b</a>). Thus, when viewed through the lens of graph theory, the whole-brain networks of synergistic and redundant interactions exhibit distinct graph-theoretical properties, favoring global and segregated processing, respectively, as demanded by the cognitive functions they support.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Network analysis indicates global and segregated processing for synergy and redundancy, respectively."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Network analysis indicates global and segregated processing for synergy and redundancy, respectively.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="409"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, The whole-brain network organization of synergistic interactions exhibited significantly higher global efficiency than the whole-brain network of redundant interactions (synergy: M = 2.47, s.d. = 0.08; redundancy: M = 0.07, s.d. = 0.02; <i>t</i>(99) = −285.75, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = −43.58, effect size CI: (−50.46, −38.75), from paired-sample non-parametric permutation <i>t</i>-test (two-sided); <i>n</i> = 100 unrelated HCP participants). <b>b</b>, The whole-brain network of redundant interactions exhibited significantly higher segregation (modularity) than the network of synergistic interactions (synergy: M = 0.003, s.d. = 0.001; redundancy: M = 0.29, s.d. = 0.05; <i>t</i>(99) = 52.12, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = 7.28, effect size CI: (6.22, 8.85), from paired-sample non-parametric permutation <i>t</i>-test (two-sided); <i>n</i> = 100 unrelated HCP participants). For the violin plots, each colored circle represents one participant; white circles represent median values, central lines represent the mean values, box limits represent upper and lower quartiles and whiskers represent 1.5× the interquartile range; ***<i>P</i> &lt; 0.001. <b>c</b>, Redundant connections were significantly stronger within functional resting-state subnetworks (RSNs) than between them (within-RSN: M = 0.087, s.d. = 0.078, <i>n</i> = 7,178 within-RSN connections; between-RSN: M = 0.026, s.d. = 0.027, <i>n</i> = 46,414 between-RSN connections; <i>t</i>(53,590) = 127.45, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = 1.61, effect size CI: (1.58, 1.66); from two-sample non-parametric permutation <i>t</i>-test (two-sided)). <b>d</b>, Synergistic connections were significantly stronger between functional resting-state subnetworks than within them (within-RSN: M = 2.41, s.d. = 0.09, <i>n</i> = 7,178 within-RSN connections; between-RSN: M = 2.47, s.d. = 0.06, <i>n</i> = 46,414 between-RSN connections; <i>t</i>(53,590) = −75.31, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = −0.96, effect size CI: (−0.99, −0.92); from two-sample non-parametric permutation <i>t</i>-test (two-sided)). For the box plots, white circles represent the median values; box limits the upper and lower quartiles and whiskers represent 1.5× the interquartile range. <b>e</b>, Matrix of redundancy-to-synergy gradient scores for each connection between brain regions, highlighting regions’ affiliation with the resting-state subnetworks of Yeo et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-022-01070-0#ref-CR24" id="ref-link-section-d30343262e1566">24</a></sup>. Red indicates synergy &gt; redundancy. These results are robust to the use of alternative ways of defining nodes and edges, and alternative network measures (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig10">3</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig12">5</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">4</a>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Complementing this graph-theoretical analysis, we show that redundant interactions tend to be stronger within resting-state subnetworks than between them (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig3">3c</a>), whereas the opposite is true for synergistic interactions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig3">3d</a>), which are stronger across different resting-state subnetworks, especially between DMN/FPN and other subnetworks (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig3">3e</a>). These results indicate that brain regions can rely on redundant information to interact within their own segregated subnetwork, while simultaneously supporting integrated processing across subnetworks through synergistic interactions.</p><h3 class="c-article__sub-heading" id="Sec6">Distinct structural support for synergy and redundancy</h3><p>Because only a subset of brain regions is directly connected by white matter tracts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Petersen, S. E. &amp; Sporns, O. Brain networks and cognitive architectures. Neuron 88, 207–219 (2015)." href="/articles/s41593-022-01070-0#ref-CR29" id="ref-link-section-d30343262e1611">29</a></sup>, we reasoned that the more an organism’s survival depends on interactions between regions A and B, the more we may expect A and B to be directly physically connected, rather than relying on intermediate polysynaptic connections. Thus, direct anatomical connections may be understood as revealing where the brain’s need for robust communication is highest. Consequently, if redundant interdependencies provide robustness to the system (as they correspond to information that is not contingent on any single brain region), they should be co-located with underlying direct structural connections. Our results support this hypothesis: across participants, the number of white matter streamlines (quantified using diffusion-weighted imaging (DWI)) was significantly more correlated with redundant than synergistic interactions between regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig4">4a</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Redundant interactions are supported by anatomical connections; synergistic interactions connect regions with distinct structural wiring profiles."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Redundant interactions are supported by anatomical connections; synergistic interactions connect regions with distinct structural wiring profiles.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="491"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Whole-brain networks of redundant interactions were significantly more correlated (average Spearman’s ρ) with underlying structural connectivity than synergistic interactions (redundancy: M = 0.16, s.d. = 0.028; synergy: M = −0.008, s.d. = 0.014; <i>t</i>(99) = 48.67, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = 7.79, effect size CI: (6.98, 8.92) from paired-sample non-parametric permutation <i>t</i>-test (two-sided); <i>n</i> = 100 unrelated HCP participants). <b>b</b>, Redundant connections were significantly stronger in the presence of an underlying direct structural connection (direct: M = 0.074, s.d. = 0.083, <i>n</i> = 5,276 direct connections; indirect: M = 0.029, s.d. = 0.034, <i>n</i> = 48,548 indirect connections; <i>t</i>(53,822) = 73.38, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = 1.06, effect size CI: (1.02, 1.11)). <b>c</b>, Synergistic connections were significantly stronger between regions that do not share a direct structural connection (direct: M = 2.43, s.d. = 0.10, <i>n</i> = 5,276 direct connections; indirect: M = 2.46, s.d. = 0.18; <i>t</i>(53,822) = −10.23, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = −0.15, effect size CI: (−0.17, −0.12) from two-sided non-parametric permutation <i>t</i>-test). These results ae robust to the use of alternative ways of defining nodes and edges and to a different measure of structure–function coupling (Hamming distance; Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig11">4</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig12">5</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">4</a>). <b>d</b>, Whole-brain networks of synergistic connections were significantly more associated than redundant networks with cortico-cortical wiring distance, a measure of dissimilarity between anatomical wiring that combines diffusion tractography, spatial proximity and microstructural profile similarity (note that this analysis was performed on a different parcellation; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-022-01070-0#Sec12">Methods</a>; redundancy: M = −0.15, s.d. = 0.04; synergy: M = 0.12, s.d. = 0.05; <i>t</i>(99) = −36.82, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = −6.43, effect size CI: (−7.30, −5.82) from paired-sample non-parametric permutation <i>t</i>-test (two-sided); <i>n</i> = 100 unrelated HCP participants). For the violin plots in <b>a</b> and <b>d</b>, each colored circle represents one participant; white circles represent median values, central lines represent the mean values, box limits represent upper and lower quartiles and whiskers represent 1.5× the interquartile range; ***<i>P</i> &lt; 0.001. For the box plots in <b>b</b> and <b>c</b>, white circles represent median values, box limits represent upper and lower quartiles and whiskers represent 1.5× the interquartile range; ***<i>P</i> &lt; 0.001.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Having established that synergy and redundancy differ in their association with the underlying network of anatomical connections, we sought to obtain more fine-grained insights into their respective relationships with structural connectivity. To this end, we compared the values of redundancy between regions that have direct versus indirect anatomical connections between them, as indicated by the presence of a white matter tract between them, based on diffusion tractography. The same was done for synergy. As expected from our previous analysis, we found that redundancy is relatively stronger in the presence of a direct anatomical connection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig4">4b</a>). In contrast, we found that synergy is relatively stronger between regions that do not share a direct physical connection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig4">4c</a>). These results are in line with recent evidence that regions that share a direct anatomical connection tend to be more similar in terms of the profile of dynamics they exhibit<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Shafiei, G. et al. Topographic gradients of intrinsic dynamics across neocortex. eLife 9, 1–24 (2020)." href="/articles/s41593-022-01070-0#ref-CR31" id="ref-link-section-d30343262e1753">31</a></sup>, corroborating our expectation of greater redundancy between them. Conversely, indirect (polysynaptic) connections offer greater opportunity for different information streams to become integrated along the way from region A to region B, and become influenced by diverse modulatory factors, which should correspond to higher opportunity for synergy to occur.</p><p>More broadly, beyond the presence or absence of a direct connection between two regions, we reasoned that there will be greater scope for synergy between the activity of two brain regions if they are exposed to diverse inputs from the rest of the brain. On the other hand, similarity of inputs should favor the presence of redundant information. To test these hypotheses, we capitalized on a recently developed measure of multi-scale wiring distance between pairs of cortical regions, which combines diffusion tractography with geodesic distance (spatial proximity) and microstructural similarity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Paquola, C. et al. A multi-scale cortical wiring space links cellular architecture and functional dynamics in the human brain. PLoS Biol. 18, e3000979 (2020)." href="/articles/s41593-022-01070-0#ref-CR32" id="ref-link-section-d30343262e1760">32</a></sup>. This measure of wiring distance quantifies the difference between regional profiles of structural connectivity, accounting for both long-range white matter pathways and short-range intracortical connections<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Paquola, C. et al. A multi-scale cortical wiring space links cellular architecture and functional dynamics in the human brain. PLoS Biol. 18, e3000979 (2020)." href="/articles/s41593-022-01070-0#ref-CR32" id="ref-link-section-d30343262e1764">32</a></sup>.</p><p>Corroborating our hypotheses, comparing the matrices of synergy and redundancy with cortico-cortical wiring distance showed that greater difference between the structural connectivity profiles of two regions diminished the extent to which they hold the same information, but increased the potential for synergy between them (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig4">4d</a>).</p><p>Overall, redundant interactions demarcate a modular structural–functional backbone in the human brain, ensuring robust sensorimotor input–output channels, whereas synergistic interactions are poised to facilitate high-level cognition through globally efficient connections across different subnetworks, benefiting from diverse patterns of structural connections. Our approach reveals how the brain balances modular and global information processing in the service of different aspects of cognitive function.</p><h3 class="c-article__sub-heading" id="Sec7">Evolutionary potentiation of high-synergy brain regions</h3><p>The association between synergistic information processing and higher cognitive functions, raises the intriguing possibility that the human brain may enable humans’ uniquely sophisticated cognitive capacities specifically in virtue of its highly synergistic nature. We pursued this hypothesis through three convergent approaches.</p><p>First, we show that the human brain is especially successful at leveraging synergistic information, compared with the brains of nonhuman primates. Synergistic interactions account for a higher proportion of total information flow in the human brain than in the macaque (<i>Macaca mulatta</i>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5a</a>), whereas the two species’ brains are equal in terms of proportion of total information accounted for by redundancy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5b</a>). Further corroborating the key role of synergy, we show that the difference between humans and macaques in terms of synergy is significantly greater than the difference in total strength of FC (grand mean of FC matrix), or in the global efficiency or modularity of FC networks (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">6</a>). These results are also robust to the use of alternative analytic approaches (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">7</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">9</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Human brain evolution favored high synergy."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Human brain evolution favored high synergy.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="398"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, The proportion of synergistic interactions across the brain was significantly higher in humans (<i>H. sapiens</i>) than in macaques (<i>M. mulatta</i>; human M = 0.937, s.d. = 0.007; macaque M = 0.901, s.d. = 0.009; <i>t</i>(117) = 19.86, <i>P</i> &lt; 0.001, Hedges’s <i>g</i> = 4.94, effect size CI: (4.12, 6.10), from two-sample non-parametric permutation <i>t</i>-test (two-sided). Human data: <i>n</i> = 100 unrelated HCP participants. Macaque data: <i>n</i> = 19 distinct sessions from ten individual macaques). <b>b</b>, The proportion of redundant interactions across the brain was equivalent in humans and macaques (redundancy: human M = 0.012, s.d. = 0.005; macaque M = 0.011, s.d. = 0.005; <i>t</i>(117) = 0.90, <i>P</i> = 0.372, Hedges’s <i>g</i> = 0.22, effect size CI: (−0.28, 0.66), from two-sample non-parametric permutation <i>t</i>-test (two-sided). Human data: <i>n</i> = 100 unrelated HCP participants. Macaque data: <i>n</i> = 19 distinct sessions from ten individual macaques). White circles represent median values, central lines represent the mean values, box limits represent upper and lower quartiles and whiskers represent 1.5× the interquartile range; ***<i>P</i> &lt; 0.001; NS, not significant <i>P</i> &gt; 0.05. Note that the proportion of synergy can differ even if the proportion of redundancy remains the same; as mentioned in the introduction, synergy and redundancy are not the only constituent elements of total information flow in the system: each source (here, brain region) also brings a unique contribution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at &#xA;                  http://arxiv.org/abs/1004.2515&#xA;                  &#xA;                 (2010)." href="/articles/s41593-022-01070-0#ref-CR2" id="ref-link-section-d30343262e1877">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e1880">6</a></sup>. The results presented here cannot be solely attributed to either the choice of bandpass filter or the difference in TR between datasets (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig13">6</a>). The proportion of synergistic information was significantly higher in humans than macaques even when the comparison was restricted only to those humans whose total FC strength was in the range of the macaque values (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig13">6</a>). <b>c</b>, Surface projection of regional redundancy-to-synergy gradient scores for the macaque brain. Corresponding matrices of synergy and redundancy are shown in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig14">7</a>. <b>d</b>, Significant Spearman correlation (two-sided CI: (0.16, 0.59)) between human regional redundancy-to-synergy gradient scores and regional cortical expansion from chimpanzee (<i>P. troglodytes</i>) to human (<i>n</i> = 57 regions of the left hemisphere of the DK-114 atlas). This result is not due to spatial autocorrelation, as assessed using spin-based permutation testing; <i>P</i><sub>spin</sub> = 0.010. <b>e</b>, Significant Spearman correlation (two-sided CI: (0.25, 0.66)) between human regional redundancy-to-synergy gradient scores and regional expression of brain-related human-accelerated (HAR-Brain) genes (<i>n</i> = 57 regions of the left hemisphere of the DK-114 atlas). This result is not due to spatial autocorrelation, as assessed using spin-based permutation testing; <i>P</i><sub>spin </sub>= 0.002. These results are also robust to the use of unadjusted scores (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig15">8</a>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The patterns of synergy and redundancy in the macaque brain broadly resemble those observed in humans (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig14">7</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">5</a>), demonstrating their evolutionary stability—including the expected high redundancy in sensorimotor regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5c</a>). However, redundancy is more prevalent than synergy in macaque prefrontal cortex, despite the prefrontal cortex being among the most synergy-dominated cortices in humans (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5c</a>). Intriguingly, PFC underwent substantial cortical expansion in the course of human evolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Smaers, J. B., Gómez-Robles, A., Parks, A. N. &amp; Sherwood, C. C. Exceptional evolutionary expansion of prefrontal cortex in great apes and humans. Curr. Biol. 27, 714–720 (2017)." href="/articles/s41593-022-01070-0#ref-CR33" id="ref-link-section-d30343262e1947">33</a></sup>.</p><p>These findings suggest that the high synergy observed in human brains may be specifically related to evolutionary cortical expansion. To explore this hypothesis, we analyzed cortical morphometry data from in vivo structural MRI, comparing humans and one of the closest evolutionary relatives of <i>Homo sapiens</i>: chimpanzees (<i>Pan troglodytes</i>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. Nat. Commun. 10, 4839 (2019)." href="/articles/s41593-022-01070-0#ref-CR34" id="ref-link-section-d30343262e1960">34</a></sup>. Supporting our hypothesis, we identified a significant positive correlation between relative cortical expansion in humans versus chimpanzees, and the gradient of regional prevalence of synergy previously derived from fMRI (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5d</a>). These findings suggest that the additional cortical tissue in humans may be primarily dedicated to synergy, rather than redundancy.</p><p>To provide further support for the evolutionary relevance of synergistic interactions, we capitalized on human adult brain microarray datasets across 57 regions of the left cortical mantle, made available by the Allen Institute for Brain Science (AIBS)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. Nat. Commun. 10, 4839 (2019)." href="/articles/s41593-022-01070-0#ref-CR34" id="ref-link-section-d30343262e1971">34</a></sup>. We demonstrate that regional predominance of synergy correlates with regional expression of genes that are both (1) related to brain development and function, including intelligence and synaptic transmission<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. Nat. Commun. 10, 4839 (2019)." href="/articles/s41593-022-01070-0#ref-CR34" id="ref-link-section-d30343262e1975">34</a></sup> and (2) selectively accelerated in humans versus nonhuman primates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. Nat. Commun. 10, 4839 (2019)." href="/articles/s41593-022-01070-0#ref-CR34" id="ref-link-section-d30343262e1979">34</a></sup> (‘HAR-Brain genes’; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5e</a>). Thus, the more prominent a brain region is in terms of synergy, the more likely it is to express brain genes that are uniquely human.</p><p>Taken together, these findings provide converging evidence that human brain evolution may have resulted in the increased prominence of synergistic interactions, in terms of dedicated genes, (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5e</a>) dedicated cortical real estate (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5d</a>), and the end result: higher prevalence of synergy in human brains than in the brains of nonhuman primates (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig5">5a,b</a>).</p><h3 class="c-article__sub-heading" id="Sec8">Convergent synaptic underpinnings of synergy in the brain</h3><p>These observations raise the question of how such high synergy in the human brain could have been attained. To address this question from a neurobiological perspective, we explored the association between the redundancy-to-synergy gradient and regional expression profiles of 20,674 genes from AIBS microarray data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e2006">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e2009">35</a></sup>. Using partial least-squares (PLS) regression, we show that the first two PLS components explained 30% of the variance in the regional synergy–redundancy values: significantly more than could be expected by chance (permutation test, <i>P</i> = 0.004) even when accounting for spatial autocorrelation (<i>P</i><sub>spin </sub>= 0.027). Both components were significantly enriched for HAR-Brain genes, corroborating the hypothesis-driven results presented above (PLS1: <i>P</i> &lt; 0.001; PLS2: <i>P</i> &lt; 0.001; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig16">9</a>). We next sought to identify the role played by overexpressed genes related to brain synergy, for each PLS component. Analysis of Gene Ontology (GO) terms revealed that the transcriptional signatures of the PLS components were significantly enriched in genes involved in learning/memory (consistently with our NeuroSynth meta-analysis), as well as synapses, synapse components and synaptic transmission (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig6">6a–f</a>; all <i>P</i> &lt; 10<sup>−4</sup> for significant enrichment.).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Synaptic underpinnings of synergy in the human brain."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: Synaptic underpinnings of synergy in the human brain.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="528"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, First principal component of PLS (PLS1) relating the redundancy-to-synergy regional gradient to <i>n</i> = 20,647 genes from the AIBS. <b>b</b>,<b>c</b>, Dimensionality-reduced GO terms pertaining to biological processes (<b>b</b>) or cellular components (<b>c</b>) that are significantly enriched in PLS1, obtained from GOrilla using the ‘<i>P</i>-value threshold 10<sup>−4</sup>’ setting to best approximate FDR correction with α = 0.05 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e2073">35</a></sup>) and summarized with REVIGO (added red ovals highlight psychologically or neurobiologically relevant terms). Semantic space axes indicate the relative distance between terms in multidimensional space, but have no intrinsic meaning. <b>d</b>, Second principal component of PLS (PLS2) relating the redundancy-to-synergy regional gradient to <i>n</i> = 20,647 genes from the AIBS. <b>e</b>,<b>f</b>, Dimensionality-reduced GO terms pertaining to biological processes (<b>e</b>) or cellular components (<b>f</b>) that are significantly enriched in PLS2, obtained from GOrilla using the ‘<i>P</i>-value threshold 10<sup>−4</sup>’ setting to best approximate FDR correction with α = 0.05 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e2101">35</a></sup>) and summarized with REVIGO (added red ovals highlight psychologically or neurobiologically relevant terms). These results are robust to the use of alternative approaches (ridge-regularized PLS regression on the synergy–redundancy cortical pattern; and controlling for spatial autocorrelation using spin-based permutations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. Neuroimage 178, 540–551 (2018)." href="/articles/s41593-022-01070-0#ref-CR53" id="ref-link-section-d30343262e2106">53</a></sup>; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig16">9</a>). The significant enrichment for synaptic transmission and organization was also preserved when using recently developed advanced null models based on random phenotype ensembles (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">10</a>). <b>g</b>, Spearman correlation between regional redundancy-to-synergy gradient scores and the first principal component of in vivo synaptic density from [<sup>11</sup>C]UCB-J PET (Spearman ρ = 0.22, p = 0.077; two-sided CI: −0.019, 0.436; <i>P</i><sub>spin </sub>= 0.053; <i>n</i> = 68 cortical regions). Colored bars correspond to scatterplot axes. <b>h</b>, Spearman correlation between regional redundancy-to-synergy gradient scores and the second (anteroposterior) principal component of in vivo synaptic density from [<sup>11</sup>C]UCB-J PET (Spearman ρ = 0.28, p = 0.018; two-sided CI: 0.044, 0.486; <i>P</i><sub>spin </sub>= 0.059; <i>n</i> = 68 cortical regions). Colored bars correspond to scatterplot axes.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Synapses are the key structures by which neurons exchange information; therefore, they constitute a prime candidate for the neurobiological underpinning of synergistic interactions in the human brain, as suggested by our genetic analysis. To provide a more direct link between synaptic density and regional prevalence of synergy, we used PET to estimate regional synaptic density in vivo, based on the binding potential of the synapse-specific radioligand [<sup>11</sup>C]UCB-J, which has high affinity for the synaptic vesicle glycoprotein 2A, ubiquitously expressed in all synapses throughout the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Finnema, S. J. et al. Imaging synaptic density in the living human brain. Sci. Transl. Med. 8, 348ra96 (2016)." href="/articles/s41593-022-01070-0#ref-CR36" id="ref-link-section-d30343262e2155">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Holland, N. et al. Synaptic loss in primary tauopathies revealed by [11C]UCB-J positron emission tomography. Mov. Disord. &#xA;                  https://doi.org/10.1002/mds.28188&#xA;                  &#xA;                 (2020)." href="/articles/s41593-022-01070-0#ref-CR37" id="ref-link-section-d30343262e2158">37</a></sup>. Therefore, [<sup>11</sup>C]UCB-J binding potential enables in vivo estimation of synaptic density at a regional level<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Finnema, S. J. et al. Imaging synaptic density in the living human brain. Sci. Transl. Med. 8, 348ra96 (2016)." href="/articles/s41593-022-01070-0#ref-CR36" id="ref-link-section-d30343262e2164">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Holland, N. et al. Synaptic loss in primary tauopathies revealed by [11C]UCB-J positron emission tomography. Mov. Disord. &#xA;                  https://doi.org/10.1002/mds.28188&#xA;                  &#xA;                 (2020)." href="/articles/s41593-022-01070-0#ref-CR37" id="ref-link-section-d30343262e2167">37</a></sup>. Across <i>N</i> = 15 healthy volunteers, we obtained regional values of [<sup>11</sup>C]UCB-J PET binding potential<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Holland, N. et al. Synaptic loss in primary tauopathies revealed by [11C]UCB-J positron emission tomography. Mov. Disord. &#xA;                  https://doi.org/10.1002/mds.28188&#xA;                  &#xA;                 (2020)." href="/articles/s41593-022-01070-0#ref-CR37" id="ref-link-section-d30343262e2177">37</a></sup>, which we then decomposed into two principal components reflecting gradients of synaptic density across cortical regions.</p><p>Supporting the notion that regional brain synergy is related to underlying synaptic density, we found that an anteroposterior principal component of regional synaptic density derived from [<sup>11</sup>C]UCB-J PET correlates with the regional gradient from redundancy to synergy, with a clear trend to significance persisting even after accounting for spatial autocorrelation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig6">6d,e</a>). Thus, regional synaptic density predicts regional prevalence of synergy over redundancy across the cortex.</p><h3 class="c-article__sub-heading" id="Sec9">Metabolic and molecular underpinnings of synergy</h3><p>Although genetic and in vivo data indicate the relevance of synapses for brain synergy, synapses themselves are not fixed, but rather they develop over the human lifespan. In both space and time, synaptic growth has well-established metabolic underpinnings, specifically pertaining to aerobic glycolysis (AG). Temporally, meta-analytic results indicate that during human development AG peaks during peak synaptic growth; spatially, the regional distribution of AG in the adult brain coincides with regional expression of genes promoting synaptic growth, suggesting that even during adulthood, AG may support continuous synapse formation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Goyal, M. S., Hawrylycz, M., Miller, J. A., Snyder, A. Z. &amp; Raichle, M. E. Aerobic glycolysis in the human brain is associated with development and neotenous gene expression. Cell Metabolism 19, 49–57 (2014)." href="/articles/s41593-022-01070-0#ref-CR38" id="ref-link-section-d30343262e2197">38</a></sup>.</p><p>In line with this body of literature and our previous findings, we show that a significant correlation exists between the redundancy-to-synergy gradient, and the cortical distribution of glycolytic index (GI), a measure of AG obtained from PET measurements of cerebral metabolic rates for oxygen and glucose<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Vaishnavi, S. N. et al. Regional aerobic glycolysis in the human brain. Proc. Natl. Acad. Sci. USA 107, 17757–17762 (2010)." href="/articles/s41593-022-01070-0#ref-CR39" id="ref-link-section-d30343262e2204">39</a></sup> (Spearman ρ = 0.40, <i>P</i> &lt; 0.001, <i>P</i><sub>spin</sub> = 0.013; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig7">7a</a>). Additionally, the same genes promoting synaptic development that were previously related to regional AG distribution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Goyal, M. S., Hawrylycz, M., Miller, J. A., Snyder, A. Z. &amp; Raichle, M. E. Aerobic glycolysis in the human brain is associated with development and neotenous gene expression. Cell Metabolism 19, 49–57 (2014)." href="/articles/s41593-022-01070-0#ref-CR38" id="ref-link-section-d30343262e2218">38</a></sup> were also significantly enriched in both PLS components of gene expression related to the synergy–redundancy gradient identified here (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig17">10</a>). Therefore, genetic and PET evidence converge to indicate that the same metabolic process supporting synaptic growth may also underpin synergistic interactions in the human brain.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-7" data-title="Metabolic and molecular underpinnings of synergy in the human brain."><figure><figcaption><b id="Fig7" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 7: Metabolic and molecular underpinnings of synergy in the human brain.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/7" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig7_HTML.png?as=webp"><img aria-describedby="Fig7" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig7_HTML.png" alt="figure 7" loading="lazy" width="685" height="283"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-7-desc"><p><b>a</b>, Significant Spearman correlation (two-sided CI: 0.195, 0.572) between regional redundancy-to-synergy gradient scores, and the mean regional estimate of AG based on PET measurements of cerebral metabolic rates for oxygen and glucose (GI); <i>n</i> = 78 cortical regions. <b>b</b>, Significant Spearman correlation (two-sided CI: 0.270, 0.731) between cortical redundancy-to-synergy gradient scores, and the diversity (entropy) of neurotransmitter receptor expression from in vitro quantitative autoradiography across <i>n</i> = 39 regions of human visual, somatosensory, auditory and multimodal association cortices. Colored bars correspond to scatterplot axes.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-022-01070-0/figures/7" data-track-dest="link:Figure7 Full size image" aria-label="Full size image figure 7" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Although a high density of synapses reflects high potential for integrating multiple inputs, the actual interactions between neurons rely on multiple distinct neurotransmitters acting on a variety of different receptors. Importantly, the distribution of neurotransmitter receptors in the human brain is not uniform, but rather it varies both across cortical regions and across different layers (supragranular, granular and infragranular) of the same region<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e2260">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zilles, K. &amp; Palomero-Gallagher, N. Multiple transmitter receptors in regions and layers of the human cerebral cortex. Front. Neuroanat. 11, 78 (2017)." href="/articles/s41593-022-01070-0#ref-CR41" id="ref-link-section-d30343262e2263">41</a></sup>. In particular, cortical regions differ systematically in terms of the diversity of neurotransmitter receptors they exhibit <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e2267">40</a></sup>. This provides an opportunity to identify potential neurochemical underpinnings of synergy in the human brain; we reasoned that diverse receptor expression across receptor types and layers should endow a region with greater flexibility, so that its activity will be able to reflect the integration of distinct inputs and neuromodulatory influences from a variety of regions and systems.</p><p>Supporting the hypothesis of an association between molecular diversity and synergy, we demonstrate that the redundancy-to-synergy gradient corresponds to increasing diversity of receptor densities across supragranular, granular and infragranular layers for 15 different receptor types, obtained from in vitro quantitative receptor autoradiography<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e2274">40</a></sup> (Spearman ρ = 0.54, <i>P</i> &lt; 0.001). This result suggests that a more diverse receptor profile corresponds to a larger range of information that a region can respond to and integrate, as reflected in higher predominance of synergy for that region.</p><p>Overall, hypothesis-driven and data-driven genetic, metabolic and molecular evidence converge to indicate synapses, synaptic formation, synaptic transmission and the diversity of neurotransmitters as key neurobiological underpinnings of synergy in the brain—in line with the notion that synergy quantifies information integration, and its role in supporting higher cognition.</p><h3 class="c-article__sub-heading" id="Sec10">Validation of results against surrogates</h3><p>Finally, we validated our results pertaining to the association between the redundancy-to-synergy gradient and macroscale features of cortical organization by showing that they cannot be recovered if the synergy ranks are obtained from surrogate data. We do this in three different ways: (1) randomizing the matrix of interregional synergistic connections; (2) replacing the true synergy ranks with a random vector with the same (negative) correlation with the redundancy ranks; and (3) computing both synergy and redundancy after phase randomizing each time series (thereby removing interregional correlations from the signal (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-022-01070-0#Sec12">Methods</a>)). The results of these analyses are summarized in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/s41593-022-01070-0#Tab1">1</a>.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 Comparison of macroscale associations for the redundancy-to-synergy gradient, its randomly generated counterparts from surrogates, and traditional FC (node strength)</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/s41593-022-01070-0/tables/1" aria-label="Full size table 1"><span>Full size table</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec11-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec11">Discussion</h2><div class="c-article-section__content" id="Sec11-content"><p>Our multimodal, multispecies investigation of information-resolved brain dynamics illuminates how the human brain has evolved to address the inherent trade-off between robustness and integration. Leveraging Integrated Information Decomposition to decompose the intrinsic dynamics of human BOLD signals, we quantified how much information about the brain’s dynamics is carried redundantly by the present state of distinct brain regions, reflecting their robustness, and how much of it is carried synergistically between regions, reflecting their integration.</p><p>We provided converging evidence that synergistic interactions play a crucial role in the human neurocognitive architecture, and further identified their neurobiological underpinnings across scales by combining genetic, molecular, cytoarchitectonic, metabolic, structural and neuroanatomical evidence. Our findings reveal that basic sensorimotor functions are supported by a modular structure–function backbone of redundant interactions. As the brain’s input–output systems, reliable sensorimotor channels are vital for survival, warranting the additional robustness provided by redundant interactions and supported by direct anatomical connections, as indicated by our structural–functional analysis.</p><p>In contrast, meta-analytic and graph-theoretical results indicate that synergistic interactions form a globally efficient network throughout the brain, with synergistic interactions bridging across different brain modules to support higher cognitive functions. Intriguingly, high-synergy regions also exhibit high rates of AG: in addition to providing a fast and flexible supply of energy, AG is theorized to provide the substrate for biosynthetic processes supporting ongoing synapse formation and turnover<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Raichle, M. E. The restless brain: how intrinsic activity organizes brain function. Philos. Trans. R. Soc. Lond. B Biol. Sci. 370, 20140172 (2015)." href="/articles/s41593-022-01070-0#ref-CR15" id="ref-link-section-d30343262e2742">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Goyal, M. S. &amp; Raichle, M. E. Gene expression-based modeling of human cortical synaptic density. Proc. Natl. Acad. Sci. USA 110, 6571–6576 (2013)." href="/articles/s41593-022-01070-0#ref-CR42" id="ref-link-section-d30343262e2745">42</a></sup>. Indeed, we show that the neuroanatomical organization of high-synergy regions coincides with synapse-rich association cortices. We also show that high-synergy cortical regions exhibit the most diverse profiles of neurotransmitter receptor expression, enabling flexible neuromodulation. Thus, network organization and neurobiology across scales provide converging evidence that synergistic interactions are ideally poised to act as a ‘global workspace’ in the human brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Mashour, G. A., Roelfsema, P., Changeux, J. P. &amp; Dehaene, S. Conscious processing and the global neuronal workspace hypothesis. Neuron 105, 776–798 (2020)." href="/articles/s41593-022-01070-0#ref-CR43" id="ref-link-section-d30343262e2749">43</a></sup> enabling the integration of complementary information from across the brain in the service of higher cognitive functions.</p><p>It is noteworthy that synergy, which quantifies the extra information gained by integrating multiple sources<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e2756">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Rosas, F. E., Mediano, P. A. M., Rassouli, B. &amp; Barrett, A. B. An operational information decomposition via synergistic disclosure. J. Phys. A Math. Theor. 53, 485001 (2020)." href="/articles/s41593-022-01070-0#ref-CR44" id="ref-link-section-d30343262e2759">44</a></sup>, is especially prevalent in regions of the default mode and FPN subnetworks. Functionally, these regions are recruited by complex tasks that rely on multimodal information, decoupled from immediate sensorimotor contingencies;<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-022-01070-0#ref-CR25" id="ref-link-section-d30343262e2763">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Buckner, R. L. &amp; Krienen, F. M. The evolution of distributed association networks in the human brain. Trends Cogn. Sci. 17, 648–665 (2013)." href="/articles/s41593-022-01070-0#ref-CR45" id="ref-link-section-d30343262e2766">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Yeshurun, Y., Nguyen, M. &amp; Hasson, U. The default mode network: where the idiosyncratic self meets the shared social world. Nat. Rev. Neurosci. 22, 181–192 (2021)." href="/articles/s41593-022-01070-0#ref-CR46" id="ref-link-section-d30343262e2769">46</a></sup> anatomically, they receive multimodal inputs from across the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Jones, E. G. &amp; Powell, T. P. S. An anatomical study of converging sensory pathways within the cerebral cortex of the monkey. Brain 93, 793–820 (1970)." href="/articles/s41593-022-01070-0#ref-CR47" id="ref-link-section-d30343262e2773">47</a></sup>. Therefore, it has been conjectured that these networks are devoted to the integration of information<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-022-01070-0#ref-CR25" id="ref-link-section-d30343262e2777">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Jones, E. G. &amp; Powell, T. P. S. An anatomical study of converging sensory pathways within the cerebral cortex of the monkey. Brain 93, 793–820 (1970)." href="/articles/s41593-022-01070-0#ref-CR47" id="ref-link-section-d30343262e2780">47</a></sup>. Our original findings about regional prevalence of synergy in DMN and FPN provide formal information-theoretic evidence to confirm this long-standing hypothesis.</p><p>Taken together, our results delineate a neural architecture whereby direct anatomical connections predominantly contribute to ensuring that the same information is available to regions within the same specialized module—especially in sensory and motor subnetworks. Conversely, indirect polysynaptic connections are better poised to favor the integration of information between different modules, converging to association cortices in default and FPN subnetworks that are associated with anatomical and cognitive integration of information.</p><p>Indeed, high-order association cortices of the DMN and FPN may be able to support human higher cognition precisely thanks to their extensive involvement with synergistic processing: we discovered that synergistic (as opposed to redundant) interactions are specifically enhanced in humans over other primates, with dedicated cortical real estate and dedicated genes, including those promoting synaptic transmission and formation. This process resulted in a neural architecture that is capable of leveraging synergistic information to a greater extent than other primates.</p><p>At the same time, the DMN and FPN exhibit distinct patterns of activity in tasks and at rest<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fox, M. D. et al. The human brain is intrinsically organized into dynamic, anticorrelated functional networks. Proc. Natl. Acad. Sci. USA 102, 9673–9678 (2005)." href="#ref-CR48" id="ref-link-section-d30343262e2794">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Raichle, M. E. et al. A default mode of brain function. Proc. Natl. Acad. Sci. USA 98, 676–682 (2001)." href="#ref-CR49" id="ref-link-section-d30343262e2794_1">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Vatansever, D., Menon, X. D. K., Manktelow, A. E., Sahakian, B. J. &amp; Stamatakis, E. A. Default mode dynamics for global functional integration. J. Neurosci. 35, 15254–15262 (2015)." href="/articles/s41593-022-01070-0#ref-CR50" id="ref-link-section-d30343262e2797">50</a></sup> and play different roles in controlling brain dynamics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Gu, S. et al. Controllability of structural brain networks. Nat. Commun. 6, 8414 (2015)." href="/articles/s41593-022-01070-0#ref-CR51" id="ref-link-section-d30343262e2801">51</a></sup>. Thus, it will be crucial for future work to delineate their respective roles within the brain’s synergistic core; for instance, by examining how the prevalence and regional distributions of synergy and redundancy vary during cognitive tasks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Cole, M. W. et al. Multi-task connectivity reveals flexible hubs for adaptive task control. Nat. Neurosci. 16, 1348–1355 (2013)." href="/articles/s41593-022-01070-0#ref-CR52" id="ref-link-section-d30343262e2805">52</a></sup> or pathological conditions, where our framework for information-resolved analysis could also find fruitful application. Additionally, future work may seek a convergence between the information-resolved and time-resolved approaches to brain function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vidaurre, D., Smith, S. M. &amp; Woolrich, M. W. Brain network dynamics are hierarchically organized in time. Proc. Natl. Acad. Sci. USA 114, 12827–12832 (2017)." href="#ref-CR17" id="ref-link-section-d30343262e2809">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shine, J. M. et al. The dynamics of functional brain networks: integrated network states during cognitive task performance. Neuron 92, 544–554 (2016)." href="#ref-CR18" id="ref-link-section-d30343262e2809_1">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Breakspear, M. Dynamic models of large-scale brain activity. Nat. Neurosci. 20, 340–352 (2017)." href="#ref-CR19" id="ref-link-section-d30343262e2809_2">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Luppi, A. I. et al. Consciousness-specific dynamic interactions of brain integration and functional diversity. Nat. Commun. 10, 4616 (2019)." href="/articles/s41593-022-01070-0#ref-CR20" id="ref-link-section-d30343262e2812">20</a></sup>, as possible complementary strategies to balance the brain’s competing needs for integration and robustness.</p><p>Disentangling synergistic and redundant contributions to the interactions between two regions makes it possible to distinguish cases of ‘integration of information’ (synergy) versus simply ‘having the same information’ (redundancy). The quantification of synergistic information provides a rigorous approach to capture the integration between the activity of different brain regions in terms of their mutual influence, going beyond measures of traditional FC (for example, Pearson correlation) that are unable to account for higher-order statistical phenomena. Specifically, our results suggest that traditional FC is significantly more similar to redundancy than to synergy. These observations suggest that care should be taken when interpreting the results of traditional FC analyses in terms of ‘integration’. Such an interpretation may well be appropriate if what is meant is simply the similarity between regional time courses—which is conceptually close to redundancy. However, if the intention is to quantify information being brought together and combined, then traditional FC may be insufficient, and the more sophisticated machinery of our information-resolved approach would be called for to provide the full picture in terms of synergy. Indeed, recent complementary work has also emphasized that focusing on time-locked fluctuations alone will miss out on important aspects of interactions between regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Shafiei, G. et al. Topographic gradients of intrinsic dynamics across neocortex. eLife 9, 1–24 (2020)." href="/articles/s41593-022-01070-0#ref-CR31" id="ref-link-section-d30343262e2819">31</a></sup>.</p><p>Our information-resolved approach does not just highlight these limitations of traditional FC, however: it also provides the means to overcome them. Thus, we believe that the richer understanding of macroscale interactions provided by integrated information decomposition will help researchers to interpret their FC results more appropriately; generate more specific hypotheses; and choose more appropriate analytic tools, if what they actually wish to quantify is integration rather than simple coupling (similarity of time courses).</p><p>Finally, a key strength of our approach for information-resolved analysis of brain dynamics is its broad range of possible applications: being grounded in information theory, integrated information decomposition can be applied to neural data across scales, from fMRI to neuron cultures. Thus, our framework holds the promise of critical new insights into a wide array of questions across the breadth of neuroscience, from healthy and pathological development to cognition and its disorders. Overall, this work offers the potential to reveal the information-processing principles that govern how mental phenomena emerge from neurobiology.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec12-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec12">Methods</h2><div class="c-article-section__content" id="Sec12-content"><h3 class="c-article__sub-heading" id="Sec13">Human Connectome Project: dataset description</h3><p>The dataset of human MRI data used in this work came from the HCP (<a href="http://www.humanconnectome.org/">http://www.humanconnectome.org/</a>), release Q3. Per HCP protocol, all participants gave written informed consent to the HCP consortium. These data contained fMRI and DWI acquisitions from the widely used 100 unrelated participants (54 females and 46 males; mean age, 29.1 ± 3.7 years) of the HCP 900 data release<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="van Essen, D. C. et al. The WU-minn Human Connectome Project: an overview. Neuroimage 80, 62–79 (2013)." href="/articles/s41593-022-01070-0#ref-CR54" id="ref-link-section-d30343262e2848">54</a></sup>. We did not use statistical methods to predetermine sample size, but this 100-participant dataset has been extensively studied, so that our sample size is similar to that reported in previous publications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Preti, M. G. &amp; van de Ville, D. Decoupling of brain function from structure reveals regional behavioral specialization in humans. Nat. Commun. 10, 4747 (2019)." href="/articles/s41593-022-01070-0#ref-CR27" id="ref-link-section-d30343262e2852">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Luppi, A. I. &amp; Stamatakis, E. A. Combining network topology and information theory to construct representative brain networks. Network Neuroscience 5, 96–124 (2021)." href="/articles/s41593-022-01070-0#ref-CR55" id="ref-link-section-d30343262e2855">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Amico, E. et al. Toward an information theoretical description of communication in brain networks. Network Neuroscience 5, 646–665 (2021)." href="/articles/s41593-022-01070-0#ref-CR56" id="ref-link-section-d30343262e2858">56</a></sup> and no data points were excluded. Data collection and analysis were not performed blind to the conditions of the experiments. All HCP scanning protocols were approved by the local Institutional Review Board at Washington University in St. Louis.</p><h3 class="c-article__sub-heading" id="Sec14">Human Connectome Project: functional data acquisition</h3><p>The following sequences were used: three-dimensional MPRAGE T1-weighted structural MRI with a repetition time (TR) of 2,400 ms, an echo time (TE) of 2.14 ms, an inversion time (TI) of 1,000 ms, a flip angle of 8°, a field of view (FOV) of 224 × 224 and a voxel size of 0.7 mm isotropic. Two sessions of a 15-min resting-state fMRI using gradient-echo echo planar imaging were performed with the following parameters: TR of 720 ms, TE of 33.1 ms, flip angle of 52°, FOV of 208 × 180, voxel size of 2 mm isotropic. Here, we only used functional data from the first scanning session, in the left–right direction. Minimally preprocessed images from the HCP were used for all acquisitions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-022-01070-0#ref-CR57" id="ref-link-section-d30343262e2870">57</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec15">Functional MRI preprocessing and denoising</h3><p>We used the minimally preprocessed fMRI data from the HCP, which includes bias field correction, functional realignment, motion correction and spatial normalization to Montreal Neurological Institute (MNI-152) standard space with 2 mm of isotropic resampling resolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-022-01070-0#ref-CR57" id="ref-link-section-d30343262e2882">57</a></sup>. We also removed the first ten volumes, to allow magnetization to reach steady state. Additional denoising steps were performed using the CONN toolbox (<a href="http://www.nitrc.org/projects/conn">https://www.nitrc.org/projects/conn/</a>), version 17f<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Whitfield-Gabrieli, S. &amp; Nieto-Castanon, A. Conn: a functional connectivity toolbox for correlated and anticorrelated brain networks. Brain Connect. 2, 125–141 (2012)." href="/articles/s41593-022-01070-0#ref-CR58" id="ref-link-section-d30343262e2893">58</a></sup>.To reduce noise due to cardiac and motion artifacts, we applied the anatomical CompCor method of denoising the functional data. The anatomical CompCor method (also implemented within the CONN toolbox) involves regressing the following confounding effects in the functional data: the first five principal components attributable to each individual’s white matter signal, and the first five components attributable to individual cerebrospinal fluid (CSF) signal; six participant-specific realignment parameters (three translations and three rotations) as well as their first-order temporal derivatives<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Behzadi, Y., Restom, K., Liau, J. &amp; Liu, T. T. A component based noise correction method (CompCor) for BOLD and perfusion based fMRI. Neuroimage 37, 90–101 (2007)." href="/articles/s41593-022-01070-0#ref-CR59" id="ref-link-section-d30343262e2897">59</a></sup>. Linear detrending was also applied, and the participant-specific denoised BOLD signal time series were bandpass filtered to eliminate both low-frequency drift effects and high-frequency noise, thus retaining frequencies between 0.008 and 0.09 Hz.</p><h3 class="c-article__sub-heading" id="Sec16">Hemodynamic response function deconvolution</h3><p>Following previous work on the use of information-theoretic measures in the context of functional MRI data, we used state-of-the-art techniques<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Wu, G.-R. et al. A blind deconvolution approach to recover effective connectivity brain networks from resting state fMRI data. Med. Image Anal. 17, 365–374 (2013)." href="/articles/s41593-022-01070-0#ref-CR60" id="ref-link-section-d30343262e2909">60</a></sup> to deconvolve the hemodynamic response function (HRF) from our regional BOLD signal time series. The only exceptions were (1) the computation of traditional FC (Pearson correlation; see below), for which the un-deconvolved BOLD signal time series were instead used, in line with common practice, and (2) the comparison with macaque fMRI, because no HRF deconvolution was performed for macaque data, they were compared with non-deconvolved human data.</p><h3 class="c-article__sub-heading" id="Sec17">HCP: diffusion-weighted data</h3><p>We used DWI data from the 100 unrelated participants of the HCP 900 Subjects data release<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="van Essen, D. C. et al. The WU-minn Human Connectome Project: an overview. Neuroimage 80, 62–79 (2013)." href="/articles/s41593-022-01070-0#ref-CR54" id="ref-link-section-d30343262e2922">54</a></sup>. The diffusion-weighted acquisition protocol is covered in detail elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-022-01070-0#ref-CR57" id="ref-link-section-d30343262e2926">57</a></sup>.</p><p>The diffusion MRI scan was conducted on a Siemens 3T Skyra scanner using a two-dimensional spin-echo single-shot multiband echo planar imaging sequence with a multiband factor of 3 and monopolar gradient pulse. The spatial resolution was 1.25 mm isotropic voxels, TR of 5,500 ms and TE of 89.50 ms. The <i>b</i>-values were 1,000, 2,000 and 3,000 s per mm<sup>2</sup>. The total number of diffusion sampling directions was 90, 90 and 90 for each of the shells in addition to six b0 images. We used the version of the data made available in a format compatible with DSI Studio at <a href="https://pitt.app.box.com/v/HCP1065">https://pitt.app.box.com/v/HCP1065/</a>.</p><h3 class="c-article__sub-heading" id="Sec18">Diffusion-weighted imaging reconstruction and fiber tracking</h3><p>The minimally preprocessed DWI data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. Neuroimage 80, 105–124 (2013)." href="/articles/s41593-022-01070-0#ref-CR57" id="ref-link-section-d30343262e2953">57</a></sup> were corrected for eddy current and susceptibility artifacts. DWI data were then reconstructed using q-space diffeomorphic reconstruction (QSDR), as implemented in DSI Studio (<a href="https://dsi-studio.labsolver.org/">https://dsi-studio.labsolver.org/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Yeh, F. -C., Wedeen, V. J. &amp; Tseng, W. -Y. I. Estimation of fiber orientation and spin density distribution by diffusion deconvolution. Neuroimage 55, 1054–1062 (2011)." href="/articles/s41593-022-01070-0#ref-CR61" id="ref-link-section-d30343262e2964">61</a></sup> following previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Luppi, A. I. &amp; Stamatakis, E. A. Combining network topology and information theory to construct representative brain networks. Network Neuroscience 5, 96–124 (2021)." href="/articles/s41593-022-01070-0#ref-CR55" id="ref-link-section-d30343262e2968">55</a></sup>. QSDR is a model-free method that calculates the orientational distribution of the density of diffusing water in a standard space, to conserve the diffusible spins and preserve the continuity of fiber geometry for fiber tracking. QSDR first reconstructs diffusion-weighted images in native space and computes the quantitative anisotropy (QA) in each voxel. These QA values are used to warp the brain to a template QA volume in MNI space using the statistical parametric mapping (SPM) nonlinear registration algorithm. A diffusion sampling length ratio of 2.5 was used, and the output resolution was 1 mm. A modified FACT algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Yeh, F.-C., Verstynen, T. D., Wang, Y., Fernández-Miranda, J. C. &amp; Tseng, W.-Y. Deterministic diffusion fiber tracking improved by quantitative anisotropy. PLoS ONE 8, 80713 (2013)." href="/articles/s41593-022-01070-0#ref-CR62" id="ref-link-section-d30343262e2972">62</a></sup> was then used to perform deterministic fiber tracking on the reconstructed data, with the same parameters as those used in previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Luppi, A. I. &amp; Stamatakis, E. A. Combining network topology and information theory to construct representative brain networks. Network Neuroscience 5, 96–124 (2021)." href="/articles/s41593-022-01070-0#ref-CR55" id="ref-link-section-d30343262e2977">55</a></sup>: angular cutoff of 55°, step size of 1.0 mm, minimum length of 10 mm, maximum length of 400 mm, spin density function smoothing of 0.0 and a QA threshold determined by DWI signal in the CSF. Each of the streamlines generated was automatically screened for its termination location. A whole-brain white matter mask was created by applying DSI Studio’s default anisotropy threshold (0.6 Otsu’s threshold) to the SDF’s anisotropy values. The mask was used to eliminate streamlines with premature termination in the white matter region. Deterministic fiber tracking was performed until 1,000,000 streamlines were reconstructed for each individual.</p><h3 class="c-article__sub-heading" id="Sec19">Macaque data from PRIME-DE Initiative</h3><p>The nonhuman primate MRI data were made available as part of the PRIMate neuroimaging Data Exchange (PRIME-DE) monkey MRI data sharing initiative, a recently introduced open resource for nonhuman primate imaging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Michael Milham, A. P. et al. An open resource for nonhuman primate imaging. Neuron 100, 61–74 (2018)." href="/articles/s41593-022-01070-0#ref-CR63" id="ref-link-section-d30343262e2989">63</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec20">Macaque dataset description</h3><p>We used fMRI data from rhesus macaques (<i>M. mulatta</i>) scanned at Newcastle University. This samples includes 14 exemplars (12 males, 2 females); age distribution, 3.9–13.14 years; weight distribution, 7.2–18 kg (a full sample description is available at http://fcon_1000.projects.nitrc.org/indi/PRIME/files/newcastle.csv and http://fcon_1000.projects.nitrc.org/indi/PRIME/newcastle.html).</p><p>To ensure comparability with the human data, we excluded a priori any functional MRI data acquired from anesthetized animals. Of the 14 total animals present in the Newcastle sample, 10 had awake resting-state fMRI data available, and of these 10, all except the first animal had data from two scanning sessions available; to maximize our statistical power, these repeated sessions were included in the analysis. In total, data from 19 distinct sessions across 10 individual macaques were included.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Ethics approval</h4><p>All of the animal procedures performed were approved by the UK Home Office and comply with the Animal Scientific Procedures Act (1986) on the care and use of animals in research and with the European Directive on the protection of animals used in research (2010/63/EU). We support the Animal Research Reporting of In Vivo Experiments (ARRIVE) principles on reporting animal research. All persons involved in this project were certified by the Home Office and the work was strictly regulated by the UK Home Office. Local Animal Welfare Review Body (AWERB) approval was obtained. Assessment of and compliance with the principles of the 3Rs were conducted by The National Centre for the 3Rs. Animal in Sciences Committee (UK) approval was obtained as part of the Home Office Project License approval.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">Animal care and housing</h4><p>All animals were housed and cared for in a group-housed colony, and animals performed behavioral training on various tasks for auditory and visual neuroscience. No training took place before MRI scanning.</p><h3 class="c-article__sub-heading" id="Sec23">Macaque MRI acquisition</h3><p>Animals were scanned in a vertical Bruker 4.7T primate dedicated scanner, using single-channel or four- to eight-channel parallel imaging coils. No contrast agent was used. Optimization of the magnetic field before data acquisition was performed using second-order shimming, Bruker and custom scanning sequence optimization. Animals were scanned upright, with MRI compatible head-post or noninvasive head immobilization, and working on tasks or at rest (here, only resting-state scans were included). Eye tracking, video and audio monitoring were used during scanning.</p><p>Resting-state scanning was performed for 21.6 min, with a TR of 2,600 ms, a TE of 17 ms, effective echo spacing of 0.63 ms and a voxel size of 1.22 × 1.22 × 1.24. Columns described the phase-encoding directions. Structural scans comprised a T1 structural, MDEFT sequence with the following parameters: TE of 6 ms, TR of 750 ms, inversion delay of 700 ms; number of slices, 22; in-plane field of view, 12.8 × 9.6 cm<sup>2</sup> on a grid of 256 × 192 voxels; voxel resolution of 0.5 × 0.5 × 2 mm; number of segments, 8.</p><h3 class="c-article__sub-heading" id="Sec24">Macaque functional MRI preprocessing and denoising</h3><p>The macaque MRI data were preprocessed using the recently developed pipeline for nonhuman primate MRI analysis, Pypreclin, which addresses several specificities of monkey research. The pipeline is described in detail in the associated publication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Tasserie, J. et al. Pypreclin: an automatic pipeline for macaque functional MRI preprocessing. Neuroimage 207, 116353 (2020)." href="/articles/s41593-022-01070-0#ref-CR64" id="ref-link-section-d30343262e3044">64</a></sup>. Briefly, it includes the following steps: (1) slice-timing correction; (2) correction for the motion-induced, time-dependent B0 inhomogeneities; (3) reorientation from acquisition position to template, using the recently developed National Institute of Mental Health Macaque Template (NMT)—a high-resolution template of the average macaque brain generated from in vivo MRI of 31 rhesus macaques (<i>M. mulatta</i>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Seidlitz, J. et al. A population MRI brain template and analysis tools for the macaque. Neuroimage. 170, 121–131 (2018)." href="/articles/s41593-022-01070-0#ref-CR65" id="ref-link-section-d30343262e3051">65</a></sup>; (4) realignment to the middle volume using FSL MCFLIRT function; (5) normalization and masking using Joe’s Image Program (JIP-align) routine (<a href="http://www.nmr.mgh.harvard.edu/jbm/jip/">http://www.nmr.mgh.harvard.edu/jbm/jip/</a>; J. Mandeville, Massachusetts General Hospital, Harvard University), which is specifically designed for preclinical studies—the normalization step aligns (affine) and warps (nonlinear alignment using distortion field) the anatomical data into a generic template space; (6) B1 field correction for low-frequency intensity non-uniformities present in the data; and (7) co-registration of functional and anatomical images, using JIP-align to register the mean functional image (moving image) to the anatomical image (fixed image) by applying a rigid transformation. The anatomical brain mask was obtained by warping the template brain mask using the deformation field previously computed during the normalization step. Then, the functional images were aligned with the template space by composing the normalization and co-registration spatial transformations.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec25">Denoising</h4><p>The CompCor denoising method implemented in the CONN toolbox was used to denoise the macaque functional MRI data, to ensure consistency with the human data analysis pipeline. White matter and CSF masks were obtained from the corresponding probabilistic tissue maps of the high-resolution NMT template (eroded by one voxel); their first five principal components were regressed out of the functional data, as well as linear trends and six motion parameters (three translations and three rotations) and their first derivatives.</p><p>Following previous work on macaque functional MRI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Barttfeld, P. et al. Signature of consciousness in the dynamics of resting-state brain activity. Proc. Natl. Acad. Sci. USA 112, 887–892 (2015)." href="/articles/s41593-022-01070-0#ref-CR66" id="ref-link-section-d30343262e3072">66</a></sup>, the fMRI data were bandpass filtered in the range of 0.0025–0.05 Hz. When comparing directly between human and macaque data, results were also robust to the use of the same bandpass filter of 0.008–0.09 Hz used for human data.</p><h3 class="c-article__sub-heading" id="Sec26">Brain parcellations</h3><p>Human brains were parcellated into 232 cortical and subcortical ROIs. The 200 cortical ROIs were obtained from the scale-200 version of the recent local-global functional parcellation of Schaefer et al. (2018)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-022-01070-0#ref-CR67" id="ref-link-section-d30343262e3086">67</a></sup>. Because this parcellation only includes cortical regions, it was augmented with 32 subcortical ROIs from a recent subcortical functional parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Tian, Y., Margulies, D., Breakspear, M. &amp; Zalesky, A. Topographic organization of the human subcortex unveiled with functional connectivity gradients. Nat. Neurosci. 23, 1421–1432 (2020)." href="/articles/s41593-022-01070-0#ref-CR68" id="ref-link-section-d30343262e3090">68</a></sup>. We refer to this 232-ROI parcellation as the augmented ‘Schaefer-232’ parcellation. A recent study indicates that this parcellation results in networks whose topology is more generalizable than networks obtained from the same data but with alternative node definition schemes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Luppi, A. I. &amp; Stamatakis, E. A. Combining network topology and information theory to construct representative brain networks. Network Neuroscience 5, 96–124 (2021)." href="/articles/s41593-022-01070-0#ref-CR55" id="ref-link-section-d30343262e3094">55</a></sup>.</p><p>The data pertaining to regional PET-derived synaptic density, HAR gene expression, human cortical expansion, Von Economo cytoarchitectonics and cortico-cortical wiring distance were each only available according to specific parcellations: the Desikan–Killiany anatomical atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. Neuroimage 31, 968–980 (2006)." href="/articles/s41593-022-01070-0#ref-CR69" id="ref-link-section-d30343262e3101">69</a></sup> with 68 cortical regions (DK-68) for PET data; a 114-ROI subparcellation of the Desikan–Killiany atlas for HAR genes and cortical expansion (DK-114); and a different subparcellation of the Desikan–Killiany atlas with 308 equally sized ROIs of 500 mm<sup>2</sup> each (DK-308)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Romero-Garcia, R., Atienza, M., Clemmensen, L. H. &amp; Cantero, J. L. Effects of network resolution on topological properties of human neocortex. Neuroimage 59, 3522–3532 (2012)." href="/articles/s41593-022-01070-0#ref-CR70" id="ref-link-section-d30343262e3107">70</a></sup>, for the Von Economo cytoarchitectonic classes; and the 360-ROI HCP Multi-Modal Parcellation (HCP-MMP)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-022-01070-0#ref-CR71" id="ref-link-section-d30343262e3111">71</a></sup> for the cortico-cortical wiring distance. Therefore, we show that our results are robust to the use of these alternative parcellations.</p><p>Macaque functional data were parcellated according to the whole-cortex 82-ROI ‘regional mapping’ atlas of Kotter and Wanke<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Kötter, R. &amp; Wanke, E. Mapping brains without coordinates. Philos. Trans. R Soc. B Biol. Sci. 360, 751–766 (2005)." href="/articles/s41593-022-01070-0#ref-CR72" id="ref-link-section-d30343262e3118">72</a></sup> nonlinearly registered to the NMT template used for preprocessing.</p><h3 class="c-article__sub-heading" id="Sec27">BOLD time series extraction</h3><p>The time courses of denoised BOLD signals were averaged between all voxels belonging to a given atlas-derived ROI, using the CONN toolbox. The resulting region-specific time courses of each participant were then extracted for further analysis in MATLAB version 2019a.</p><h3 class="c-article__sub-heading" id="Sec28">Traditional functional connectivity</h3><p>For each pair of brain regions <i>i</i> and <i>j</i>, their traditional functional connectivity, FC<sub><i>ij</i></sub>, was computed as the Pearson correlation between their denoised BOLD signal time series.</p><h3 class="c-article__sub-heading" id="Sec29">Structural connectome construction</h3><p>To construct matrices of structural connectivity, the edge weights <i>a</i><sub><i>ij</i></sub> of the structural connectivity matrix <i>A</i> were defined as the number of streamlines connecting each of the regions in the atlas end-to-end, normalized to lie between zero and one. Note that deterministic tractography produces naturally sparse matrices, so that no thresholding is required.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec30">Partial information decomposition</h4><p>Shannon’s mutual information quantifies the interdependence between two random variables <i>X</i> and <i>Y</i>. It is calculated according to equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-022-01070-0#Equ1">1</a>):</p><div id="Equ1" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$I\left( {X;Y} \right) = H\left( X \right) - H\left( {X|Y} \right) = H\left( X \right) + H\left( Y \right) - H\left( {X,Y} \right),$$</span></div><div class="c-article-equation__number">
                    (1)
                </div></div><p>where H(<i>X</i>) stands for the Shannon entropy of a variable <i>X</i>. Above, the first equality states that the mutual information is equal to the reduction in entropy (that is, uncertainty) about <i>X</i> after <i>Y</i> becomes accessible. Put simply, the mutual information quantifies the information that one variable provides about another<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Cover, T. M. &amp; Thomas, J. A. Elements of Information Theory. (Wiley-Interscience, 2005). &#xA;                  https://doi.org/10.1002/047174882X&#xA;                  &#xA;                " href="/articles/s41593-022-01070-0#ref-CR21" id="ref-link-section-d30343262e3301">21</a></sup>.</p><p>Crucially, Williams and Beer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at &#xA;                  http://arxiv.org/abs/1004.2515&#xA;                  &#xA;                 (2010)." href="/articles/s41593-022-01070-0#ref-CR2" id="ref-link-section-d30343262e3308">2</a></sup> observed that the information that two source variables <i>X</i> and <i>Y</i> give about a third target variable <i>Z</i>, I(<i>X,Y</i>; <i>Z</i>), should be decomposable in terms of different kinds of information: information provided by one source but not the other (unique information), or by both sources separately (redundant information), or jointly by their combination (synergistic information). Following this intuition, they developed the PID<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at &#xA;                  http://arxiv.org/abs/1004.2515&#xA;                  &#xA;                 (2010)." href="/articles/s41593-022-01070-0#ref-CR2" id="ref-link-section-d30343262e3328">2</a></sup> framework, which leads to the fundamental decomposition given by equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-022-01070-0#Equ2">2</a>):</p><div id="Equ2" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$I\left( {X,Y;Z} \right) = \textrm{red}\left( {X,Y;Z} \right) + \textrm{un}\left( {X;Z|Y} \right) + \textrm{un}\left( {Y;Z|X} \right) + \textrm{syn}\left( {X,Y;Z} \right)$$</span></div><div class="c-article-equation__number">
                    (2)
                </div></div><p>‘Un’ corresponds to the unique information one source has but the other doesn’t, ‘red’ is the redundancy between both sources and ‘syn’ is their synergy—information that neither <i>X</i> nor <i>Y</i> alone can provide, but that can be obtained by considering <i>X</i> and <i>Y</i> together.</p><p>The simplest example of a purely synergistic system is one in which <i>X</i> and <i>Y</i> are independent fair coins, and <i>Z</i> is determined by the ‘exclusive OR’ (XOR) function <i>Z</i> = XOR(<i>X,Y</i>): that is, <i>Z</i> = 0 whenever <i>X</i> and <i>Y</i> have the same value, and <i>Z</i> = 1 otherwise. It can be shown that <i>X</i> and <i>Y</i> are both statistically independent of <i>Z</i>, which implies that neither of them provides—by themselves—information about <i>Z</i>. However, <i>X</i> and <i>Y</i> together fully determine <i>Z</i>; hence, the relationship between <i>Z</i> with <i>X</i> and <i>Y</i> is purely synergistic. Although here we based our analysis on the PID framework, it is worth remarking that alternative formulations of measures related to synergy and redundancy have been developed in other contexts (see Timme et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Timme, N., Alford, W., Flecker, B. &amp; Beggs, J. M. Synergy, redundancy and multivariate information measures: an experimentalist’s perspective. J. Comput. Neurosci. 36, 119–140 (2014)." href="/articles/s41593-022-01070-0#ref-CR3" id="ref-link-section-d30343262e3548">3</a></sup> for a discussion).</p><p>While PID provides a formal framework to decompose information, it does not enforce how the corresponding parts ought to be calculated (for example, what counts as ‘the same information’ for the purpose of measuring redundancy). While there is ongoing research on the advantages of different decompositions for discrete data, most decompositions converge into the same simple form for the case of continuous Gaussian variables<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Barrett, A. B. Exploration of synergistic and redundant information sharing in static and dynamical Gaussian systems. Phys. Rev. E 91, 52802 (2015)." href="/articles/s41593-022-01070-0#ref-CR22" id="ref-link-section-d30343262e3556">22</a></sup>. Known as minimum mutual information PID (MMI-PID), this decomposition quantifies redundancy in terms of the MMI of each individual source with the target; synergy, then, becomes identified with the additional information provided by the weaker source once the stronger source is known.</p><p>Because linear Gaussian models are adequate descriptors of functional MRI time series (and more complex, nonlinear models do not seem to offer additional explanatory power<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Schulz, M. A. et al. Different scaling of linear models and deep learning in UKBiobank brain images versus machine-learning datasets. Nat. Commun. 11, (2020)." href="/articles/s41593-022-01070-0#ref-CR73" id="ref-link-section-d30343262e3563">73</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Nozari, E. et al. Is the brain macroscopically linear? A system identification of resting state dynamics. Preprint at arXiv &#xA;                  https://doi.org/10.48550/arXiv.2012.12351&#xA;                  &#xA;                 (2020)." href="/articles/s41593-022-01070-0#ref-CR74" id="ref-link-section-d30343262e3566">74</a></sup>), here we adopted the MMI-PID decomposition, following previous applications of PID to neuroscientific data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Bím, J. et al. A nonnegative measure of feature-related information transfer between neural signals. Preprint at bioRxiv, &#xA;                  https://doi.org/10.1101/758128&#xA;                  &#xA;                 (2020)." href="/articles/s41593-022-01070-0#ref-CR75" id="ref-link-section-d30343262e3570">75</a></sup>. However, we also demonstrate that our results do not crucially rely on the assumption of Gaussianity, by showing that they are robust to the use of discretized data (see below). That being said, information decomposition is an active field of investigation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Lizier, J., Bertschinger, N., Jost, J. &amp; Wibral, M. Information decomposition of target effects from multi-source interactions: perspectives on previous, current and future work. Entropy 20, 307 (2018)." href="/articles/s41593-022-01070-0#ref-CR76" id="ref-link-section-d30343262e3574">76</a></sup> and further advances may shed new light on improved ways to compute and interpret these quantities.</p><h3 class="c-article__sub-heading" id="Sec31">Synergy and redundancy: Integrated Information Decomposition</h3><p>Let us consider now a stochastic process <i>X</i> comprising two random variables evolving jointly over time, <span class="mathjax-tex">\(X_t = \left\{ {X_t^1,X_t^2} \right\}\)</span>. In our case, this corresponds to the time series of the BOLD activity of two brain regions, although in other applications it could be any form of multivariate time series data. One can first consider the amount of information flowing from the system’s past to its future, known as TDMI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Barrett, A. B. &amp; Seth, A. K. Practical measures of integrated information for time-series data. PLoS Comput. Biol. 7, 1001052 (2011)." href="/articles/s41593-022-01070-0#ref-CR77" id="ref-link-section-d30343262e3667">77</a></sup> and given by <span class="mathjax-tex">\(I\left( {X_{t - \tau }^1,X_{t - \tau }^2;X_t^1,X_t^2} \right)\)</span>.</p><p>The fundamental advancement offered by Integrated Information Decomposition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e3787">6</a></sup> is to decompose TDMI into redundant, unique, and synergistic information shared with respect to both past and present state of both variables. In practice, this involves setting up a linear system of 15 equations with 16 unknowns relating the standard (Shannon) mutual information with the redundant, unique and synergistic components of TDMI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e3791">6</a></sup>. The system can be solved by specifying the redundancy between <i>X</i><sub><i>t</i>−τ</sub> and <i>X</i><sub><i>t</i></sub>, which, following the discussion in the previous section, we compute using MMI, given by equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-022-01070-0#Equ3">3</a>):</p><div id="Equ3" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\textrm{red}\left( {X_{t - \tau };X_t} \right) = \textrm{min}_{ij}I\left( {X_{t - \tau }^i;X_t^j} \right)$$</span></div><div class="c-article-equation__number">
                    (3)
                </div></div><p>This allows us to solve the linear system of equations and obtain all components of the Integrated Information Decomposition of <i>X</i>. Of these, we focus on the temporally persistent redundancy and synergy (denoted by <span class="mathjax-tex">\(I_\partial ^{\left\{ 1 \right\}\left\{ 2 \right\} \to \left\{ 1 \right\}\left\{ 2 \right\}}\)</span> and <span class="mathjax-tex">\(I_\partial ^{\left\{ {12} \right\} \to \left\{ {12} \right\}}\)</span> in standard PID notation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at &#xA;                  http://arxiv.org/abs/1004.2515&#xA;                  &#xA;                 (2010)." href="/articles/s41593-022-01070-0#ref-CR2" id="ref-link-section-d30343262e4067">2</a></sup>). It is worth noting that the Integrated Information Decomposition framework scrutinizes the interdependencies between the past and present states of both <i>X</i><sub><i>t</i></sub> and <i>Y</i><sub><i>t</i></sub>, and therefore captures phenomena that are not assessed by approaches focused on the transfer of information from the past of one to the present of the other. Indeed, the metrics of redundancy and synergy studied here reveal a view on brain dynamics that is fundamentally different—and ultimately complementary—to the one provided by directed information-theoretic measures such as directed information and transfer entropy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Schreiber, T. Measuring information transfer. Phys. Rev. Lett. 85, 461–464 (2000)." href="/articles/s41593-022-01070-0#ref-CR13" id="ref-link-section-d30343262e4084">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Massey, J. Causality, feedback and directed information. Proc. Int. Symp. Inf. Theory Applic. 27–30 (1990)." href="/articles/s41593-022-01070-0#ref-CR14" id="ref-link-section-d30343262e4087">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Bastos, A. M. et al. Visual areas exert feedforward and feedback influences through distinct frequency channels. Neuron 85, 390–401 (2015)." href="/articles/s41593-022-01070-0#ref-CR78" id="ref-link-section-d30343262e4090">78</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Deco, G., Vidaurre, D. &amp; Kringelbach, M. L. Revisiting the global workspace: orchestration of the functional hierarchical organisation of the human brain. Nat. Hum. Behav. &#xA;                  https://doi.org/10.1101/859579&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR79" id="ref-link-section-d30343262e4093">79</a></sup>. See the original article for details<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at &#xA;                  https://arxiv.org/abs/2109.13186&#xA;                  &#xA;                 (2021)." href="/articles/s41593-022-01070-0#ref-CR6" id="ref-link-section-d30343262e4097">6</a></sup>.</p><p>For all the analyses, we used the Gaussian solver implemented in the JIDT toolbox<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Lizier, J. T. JIDT: an information-theoretic toolkit for studying the dynamics of complex systems. Front. Robot. AI 1, 1–37 (2014)." href="/articles/s41593-022-01070-0#ref-CR80" id="ref-link-section-d30343262e4104">80</a></sup> to compute all information-theoretic quantities for each pair of brain regions, based on their HRF-deconvolved BOLD signal time series. In addition, we also show that our analyses are not critically dependent on the Gaussianity assumption by computing synergy and redundancy from discrete (binary) time series. We implemented this by estimating the corresponding information-theoretic quantities on mean-binarized signals with the well-known plug-in estimator. Finally, given the recent formulation of Integrated Information Decomposition, it is noteworthy that these metrics and their implementations for both continuous and discrete signals may be refined in future research, including extensions beyond the current pairwise approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Battiston, F. et al. The physics of higher-order interactions in complex systems. Nat. Phys. 17, 1093–1098 (2021)." href="/articles/s41593-022-01070-0#ref-CR81" id="ref-link-section-d30343262e4108">81</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Rosas, F. E. et al. Disentangling high-order mechanisms and high-order behaviours in complex systems. Nat. Physics &#xA;                  https://doi.org/10.1038/s41567-022-01548-5&#xA;                  &#xA;                 (2022)." href="/articles/s41593-022-01070-0#ref-CR82" id="ref-link-section-d30343262e4111">82</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec32">Gradient of redundancy-to-synergy relative importance</h3><p>After building networks of synergistic and redundant interactions between each pair of ROIs, we determined the role of each ROI in terms of its relative engagement in synergistic or redundant interactions. We first calculated the nodal strength of each brain region as the sum of all its connections in the group-averaged matrix. Then, we ranked all 232 regions based on their nodal strength (with higher-strength regions having higher ranks). This procedure was done separately for networks of synergy and redundancy. Subtracting each region’s redundancy rank from its synergy rank yielded a gradient from negative (that is, ranking higher in terms of redundancy than synergy) to positive (that is, having a synergy rank higher than the corresponding redundancy rank); note that the sign is arbitrary.</p><p>It is important to note that the gradient is based on relative—rather than absolute—differences between regional synergy and redundancy. Consequently, a positive rank difference does not necessarily mean that the region’s synergy is greater than its redundancy; rather, it indicates that the balance between its synergy and redundancy relative to the rest of the brain is in favor of synergy—and vice versa for a negative gradient.</p><p>The same procedure was also repeated for network edges (instead of nodes), using their weights to rank them separately in terms of synergy and redundancy and then calculating their difference. This produced a single connectivity matrix where each edge’s weight represents its relative importance, being higher for synergy (positive edges) or redundancy (negative edges).</p><h3 class="c-article__sub-heading" id="Sec33">NeuroSynth term-based meta-analysis</h3><p>The regional redundancy-to-synergy gradient identified in terms of nodal rank differences was related to specific words using NeuroSynth, an online platform for large-scale, automated synthesis of fMRI data (<a href="https://neurosynth.org/">https://neurosynth.org/</a>). For our analyses, we included 24 topic terms used by previous studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-022-01070-0#ref-CR25" id="ref-link-section-d30343262e4145">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Preti, M. G. &amp; van de Ville, D. Decoupling of brain function from structure reveals regional behavioral specialization in humans. Nat. Commun. 10, 4747 (2019)." href="/articles/s41593-022-01070-0#ref-CR27" id="ref-link-section-d30343262e4148">27</a></sup>, ranging from lower sensorimotor functions (such as eye movement, motion and visual and auditory perception) to higher cognitive functions (for example, attention, working memory and social and numerical cognition).</p><p>A meta-analysis analogous to the one implemented by previous studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. Proc. Natl. Acad. Sci. USA 113, 12574–12579 (2016)." href="/articles/s41593-022-01070-0#ref-CR25" id="ref-link-section-d30343262e4155">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Preti, M. G. &amp; van de Ville, D. Decoupling of brain function from structure reveals regional behavioral specialization in humans. Nat. Commun. 10, 4747 (2019)." href="/articles/s41593-022-01070-0#ref-CR27" id="ref-link-section-d30343262e4158">27</a></sup> was conducted to identify topic terms associated with the redundancy-to-synergy gradient. Twenty binary brain masks were obtained by splitting the values of the redundancy-to-synergy gradient into five-percentile increments. These brain masks served as input for the meta-analysis, based on the chosen 24 topic terms. For visualization, terms were ordered according to the weighted mean of the resulting <i>z</i>-statistics. Note that the term ‘visual semantics’ was excluded from visualization, because it failed to reach the significance threshold of <i>z</i> &gt; 3.1, leaving 23 terms (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig1">1</a>). The analyses were carried out using modified code made freely available at <a href="https://www.github.com/gpreti/GSP_StructuralDecouplingIndex">https://www.github.com/gpreti/GSP_StructuralDecouplingIndex</a>.</p><h3 class="c-article__sub-heading" id="Sec34">Von Economo cytoarchitectonic classes</h3><p>Whitaker et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e4186">35</a></sup> assigned each region in the DK-308 cortical parcellation to one of the cytoarchitectural types delineated by von Economo. This atlas subdivided the cortex into five types according to the laminar structure of the cortex: the primary motor cortex/precentral gyrus, two types association cortices, and secondary and primary sensory areas. Two additional subtypes were added<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Vértes, P. E. et al. Gene transcription profiles associated with inter-modular hubs and connection distance in human functional magnetic resonance imaging networks. Philos. Trans. R. Soc. Lond. B Biol. Sci. 371, 20150362 (2016)." href="/articles/s41593-022-01070-0#ref-CR83" id="ref-link-section-d30343262e4190">83</a></sup>: the limbic cortex (including the entorhinal, retrosplenial, presubicular and cingulate cortices, and thus primarily constitutes allocortex) and the insular cortex.</p><p>Subsequently, synergy and redundancy were obtained for the DK-308 parcellation as described above, as well as the redundancy-to-synergy gradient based on rank differences. The regional values of this gradient were then averaged across all ROIs belonging to each of the seven cytoarchitectonic classes. For each cytoarchitectonic class, a positive overall score indicates that cortical regions belonging to that class have overall higher importance for synergy than for redundancy—and vice versa for negative scores.</p><h3 class="c-article__sub-heading" id="Sec35">Canonical resting-state subnetworks</h3><p>We used the canonical subdivision of the brain comprising the seven cortical subnetworks according to Yeo et al. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. J. Neurophysiol. 106, 1125–1165 (2011)." href="/articles/s41593-022-01070-0#ref-CR24" id="ref-link-section-d30343262e4205">24</a></sup>: DMN, somatomotor (SOM), visual (VIS), salience/ventral attention (SAL), dorsal attention (DAN), FPN and limbic (LIM). Schaefer et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. Cereb. Cortex 28, 3095–3114 (2018)." href="/articles/s41593-022-01070-0#ref-CR67" id="ref-link-section-d30343262e4209">67</a></sup> assigned each ROI in their cortical parcellation to one of these canonical subnetworks. The 32 subcortical regions were all assigned to an eighth subcortical subnetwork (SUB).</p><h3 class="c-article__sub-heading" id="Sec36">Quantifying network efficiency and segregation</h3><p>By considering each brain region as a node in a network, it is possible to obtain two distinct whole-brain networks: one whose edges correspond to the synergistic interactions between each pair of brain regions, and one whose edges correspond to the redundant interactions. The whole-brain networks of synergistic and redundant interactions can then be compared in terms of graph-theoretical properties: our analysis focused on properties quantifying the ease of communication through the network (efficiency) and its level of segregation.</p><p>We quantified the ease of communication in the whole-brain networks of synergistic and redundant connections in terms of the network’s ‘global efficiency’, a well-known measure of the ease of parallel information transfer in the network. More precisely, the global efficiency of a network corresponds to the average of the inverse of the shortest path length between each pair of nodes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: Uses and interpretations. NeuroImage 52, 1059–1069 (2010)." href="/articles/s41593-022-01070-0#ref-CR84" id="ref-link-section-d30343262e4224">84</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Rubinov, M. &amp; Sporns, O. Weight-conserving characterization of complex functional brain networks. NeuroImage 56, 2068–2079 (2011)." href="/articles/s41593-022-01070-0#ref-CR85" id="ref-link-section-d30343262e4227">85</a></sup>:</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$Ge = \frac{1}{n}\mathop {\sum }\limits_i^n \frac{{\mathop {\sum }\nolimits_{j \ne i}^n \left( {d_{ij}} \right)^{ - 1}}}{{n - 1}}$$</span></div></div><p>Here, <i>d</i><sub><i>ij</i></sub> indicates the length of the shortest path between nodes <i>i</i> and <i>j</i>, and <i>n</i> is the total number of nodes in the network. Following Cruzat et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Cruzat, J. et al. The dynamics of human cognition: increasing global integration coupled with decreasing segregation found using iEEG. Neuroimage 172, 492–505 (2018)." href="/articles/s41593-022-01070-0#ref-CR86" id="ref-link-section-d30343262e4375">86</a></sup>, segregation of brain networks was quantified by means of network modularity. Put simply, the modularity function quantifies the extent to which a network can be partitioned such that the number of within-group edges is maximized and the density of between-group edges is minimized. We used an implementation of Newman’s spectral modularity algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Newman, M. E. J. Modularity and community structure in networks. Proc. Natl. Acad. Sci. USA 103, 8577–8582 (2006)." href="/articles/s41593-022-01070-0#ref-CR87" id="ref-link-section-d30343262e4380">87</a></sup> available in the Brain Connectivity Toolbox<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: Uses and interpretations. NeuroImage 52, 1059–1069 (2010)." href="/articles/s41593-022-01070-0#ref-CR84" id="ref-link-section-d30343262e4384">84</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Rubinov, M. &amp; Sporns, O. Weight-conserving characterization of complex functional brain networks. NeuroImage 56, 2068–2079 (2011)." href="/articles/s41593-022-01070-0#ref-CR85" id="ref-link-section-d30343262e4387">85</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec37">Structural–functional similarity</h3><p>Matrices of synergy and redundancy were thresholded proportionally using the same network density as the structural connectivity matrix of the same participant. This procedure was selected to ensure that the same number of edges would be present in both matrices, so that the two matrices can be compared. Then, the upper triangular portion of each connectivity matrix (structural and synergy/redundancy) was flattened into a vector, and the Spearman correlation coefficient between these two vectors was computed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Luppi, A. I. et al. LSD alters dynamic integration and segregation in the human brain. Neuroimage 227, 117653 (2021)." href="/articles/s41593-022-01070-0#ref-CR88" id="ref-link-section-d30343262e4399">88</a></sup>. We used this correlation as a measure of similarity between synergy or redundancy and structural connectivity. The same procedure was also adopted to compare synergy and redundancy matrices with the matrices of traditional FC and cortico-cortical wiring distance (described below). As these matrices are dense (all-to-all) matrices, the thresholding step was omitted.</p><h3 class="c-article__sub-heading" id="Sec38">Alternative network measures</h3><p>As an alternative quantification of how well connected a network is, we also used the measure of global integrative capacity developed by Cruzat and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Cruzat, J. et al. The dynamics of human cognition: increasing global integration coupled with decreasing segregation found using iEEG. Neuroimage 172, 492–505 (2018)." href="/articles/s41593-022-01070-0#ref-CR86" id="ref-link-section-d30343262e4411">86</a></sup>. This measure is calculated by taking the un-thresholded connectivity as input, and rescaling it to the range of 0 to 1 by dividing it by its largest element (note that both synergy and redundancy are guaranteed to be nonnegative, and therefore we did not need to take the absolute value). Then, the matrix is progressively thresholded with an increasing threshold ρ in 1% increments. At each threshold value, the size of the largest connected component of the resulting network is evaluated (normalized by the total number of nodes to lie between 0 and 1). The final measure is the integral of the curve of largest connected component sizes over all thresholds:</p><div id="Equb" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$I_{\textrm{GC}} = {\int}_{\rho = 0.01}^{0.99} {\textrm{\it{GC}}\left( {A_\rho } \right)}\textrm{/\it{n}}$$</span></div></div><p>where <i>GC</i>(<i>A</i><sub><i>ρ</i></sub>) indicates the size (number of nodes) of the largest connected component of the network whose adjacency matrix <i>A</i> has been thresholded at threshold ρ, and <i>N</i> is the total number of nodes in the network. This procedure was applied for both redundancy and synergy whole-brain networks of each participant.</p><p>As an alternative method to quantify the similarity of synergistic and redundant connections to the underlying structural connectome, we computed the Hamming distance between the connectivity patterns of each ROI in the functional and anatomical connectivity matrices<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Tagliazucchi, E. et al. Large-scale signatures of unconsciousness are consistent with a departure from critical dynamics. J. R. Soc. Interface 13, 20151027 (2016)." href="/articles/s41593-022-01070-0#ref-CR89" id="ref-link-section-d30343262e4524">89</a></sup>. The Hamming distance between binary vectors <i>a</i> and <i>b</i> is computed as the number of symbol substitutions required to turn one vector into the other (normalized by their length). The matrices were therefore binarized by setting all supra-threshold entries to unity, and all others to zero. In the present application, the Hamming distance measures the proportion of connections that need to be changed before the two connectivity patterns become the same. This analysis was performed for each ROI in our augmented Schaefer-232 atlas; by averaging over all ROI values, we obtained a value of structural–functional connectivity distance. Both correlation and Hamming distance analysis were performed separately for the matrices of synergy and redundancy between each pair of brain regions.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec39">Network null models for graph-theoretical analysis</h4><p>We compared the modularity and global efficiency of the networks of synergistic and redundant interactions against synthetic null models. The considered null models are based on the fact that, within a given individual, both the synergy and the redundancy between regions <i>i</i> and <i>j</i> are upper-bounded by the TDMI between <i>i</i> and <i>j</i> and lower-bounded by 0, and there is no a priori reason dictating which one should be greater. Therefore, we constructed our null models as follows. For each individual, we obtained a random network whereby the weight of the edge between each pair of regions <i>i</i> and <i>j</i> is randomly sampled from the range between 0 and the TDMI between <i>i</i> and <i>j</i>, thereby being in the same theoretical range as both synergy and redundancy. We then computed the global efficiency and modularity of this synthetic network, and repeated the procedure 100 times, comparing the global efficiency and modularity averaged across 100 simulations, with the corresponding measures obtained from the empirical networks of synergy and redundancy, for each individual.</p><h3 class="c-article__sub-heading" id="Sec40">Cortico-cortical wiring distance</h3><p>Paquola and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Paquola, C. et al. A multi-scale cortical wiring space links cellular architecture and functional dynamics in the human brain. PLoS Biol. 18, e3000979 (2020)." href="/articles/s41593-022-01070-0#ref-CR32" id="ref-link-section-d30343262e4576">32</a></sup> developed a measure of cortico-cortical wiring distance. They used diffusion map embedding, a nonlinear dimensionality reduction approach, to combine diffusion tractography with cortico-cortical spatial proximity (geodesic distance) and microstructural profile similarity between regions, to obtain a multi-scale measure of difference between cortico-cortical wiring profiles, termed wiring distance. For our analysis, we used the matrix of wiring distance provided in terms of the multimodal cortical parcellation of Glasser and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171–178 (2016)." href="/articles/s41593-022-01070-0#ref-CR71" id="ref-link-section-d30343262e4580">71</a></sup> with 360 cortical regions.</p><h3 class="c-article__sub-heading" id="Sec41">Structural–functional similarity in macaque brains</h3><p>Individual structural connectomes were not available for the macaques included in this study. Anatomical connectivity for the macaque brain was instead obtained from the fully weighted, whole-cortex macaque connectome recently developed by Shen and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Shen, K. et al. A macaque connectome for large-scale network simulations in TheVirtualBrain. Sci. Data 6, 123 (2019)." href="/articles/s41593-022-01070-0#ref-CR90" id="ref-link-section-d30343262e4592">90</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Shen, K., Bezgin, G., Everling, S. &amp; McIntosh, A. R. The virtual macaque brain: a macaque connectome for large-scale network simulations in TheVirtualBrain. &#xA;                  https://doi.org/10.5281/zenodo.1471588&#xA;                  &#xA;                 (2018)." href="/articles/s41593-022-01070-0#ref-CR91" id="ref-link-section-d30343262e4595">91</a></sup>. This connectome was generated by combining information from two different axonal tract-tracing studies from the CoCoMac database (<a href="http://cocomac.g-node.org/main/index.php?/">http://cocomac.g-node.org/main/index.php?/</a>) with diffusion-based tractography obtained from nine adult macaques (<i>M. mulatta</i> and <i>Macaca fascicularis</i>). The resulting connectome provides a matrix of weighted and directed anatomical connectivity between each of the 82 cortical ROIs of the regional mapping atlas. Because synergy and redundancy are undirected measures of connectivity, the matrix of directed anatomical connections was made undirected by averaging the strength of connections in the two directions.</p><p>As for humans, networks of synergistic and redundant interactions were thresholded proportionally using the same density as the anatomical connectivity matrix, to ensure that the same number of edges would be present in the two matrices being compared. Then, Spearman correlation was used as the measure of structural–functional similarity.</p><h3 class="c-article__sub-heading" id="Sec42">Human–macaque comparison of synergy and redundancy</h3><p>Separately for humans and macaques, each participant’s matrices of synergistic and redundant interactions were each divided by the corresponding participant’s matrix of TDMI. The global mean of each of the resulting matrices therefore represents the proportion of total information across the brain that is provided by synergistic or redundant interactions, respectively. The human data were parcellated for this analysis according to the 83-ROI Lausanne parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Cammoun, Letal Mapping the human connectome at multiple scales with diffusion spectrum MRI. J. Neurosci. Methods 203, 386–397 (2012)." href="/articles/s41593-022-01070-0#ref-CR92" id="ref-link-section-d30343262e4623">92</a></sup>, thereby ensuring a similar number of ROIs in human (83 ROIs) and macaque (82 ROIs) brains.</p><p>Because macaque brains were not deconvolved using the HRF, for this analysis we also used synergy and redundancy obtained from non-deconvolved human fMRI. As shown above, the use of HRF deconvolution had negligible effects on synergy and redundancy calculations, arguably thanks to the high temporal resolution of HCP data. To ensure that the observed differences in the proportion of synergistic information could not be attributed to differences in bandpass filtering, we also repeated this analysis with macaque data filtered in the range of 0.008–0.09 Hz (that is, the same range as the HCP human data). These data were also used to compare the effect size between humans and macaques in terms of total proportion of synergistic information versus total mean FC (overall mean of the FC matrix), or the modularity or global efficiency of FC networks, computed as described above (after removing negative connections by taking the absolute value). The procedure for comparing effect sizes is described in ‘<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-022-01070-0#Sec60">Statistical analysis</a>’.</p><h3 class="c-article__sub-heading" id="Sec43">Validation with dynamic mean field</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec44">Model construction</h4><p>To determine whether the differences in the proportion of synergy between humans and macaques could be driven by different TR measurements (0.72 s for HCP data and 2.6 s for macaque data), we simulated human functional MRI data with the same TR as the macaque data, using a dynamic neuronal mean-field model derived from the collective behavior of empirically validated integrate-and-fire (80% excitatory and 20% inhibitory) neurons<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Deco, G. et al. Whole-brain multimodal neuroimaging model using serotonin receptor maps explains non-linear functional effects of LSD. Curr. Biol. 28, 3065–3074 (2018)." href="/articles/s41593-022-01070-0#ref-CR93" id="ref-link-section-d30343262e4645">93</a></sup>. The model combines macroscale information about neuroanatomy and structural connectivity (from diffusion tensor imaging) with excitatory and inhibitory neuronal populations interconnected by AMPA, NMDA and GABA synapses, providing a neurobiologically plausible account of regional neuronal firing rate. We set all model parameters to be the same as those used by previous publications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Deco, G. et al. Whole-brain multimodal neuroimaging model using serotonin receptor maps explains non-linear functional effects of LSD. Curr. Biol. 28, 3065–3074 (2018)." href="#ref-CR93" id="ref-link-section-d30343262e4649">93</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Herzog, R. et al. A mechanistic model of the neural entropy increase elicited by psychedelic drugs. Sci. Rep. 10, 17725 (2020)." href="#ref-CR94" id="ref-link-section-d30343262e4649_1">94</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Luppi, A. I. et al. Whole-brain modelling identifies distinct but convergent paths to unconsciousness in anaesthesia and disorders of consciousness. Commun. Biol. 5, 384 (2022)." href="/articles/s41593-022-01070-0#ref-CR95" id="ref-link-section-d30343262e4652">95</a></sup>, except for the global coupling parameter <i>G</i>, which we fit (see below). Code for the dynamic mean-field model is freely available at <a href="https://gitlab.com/concog/fastdmf/">https://gitlab.com/concog/fastdmf/</a> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Mediano, P. A. M., Luppi, A. I., Herzog, R. &amp; Rosas, F. E. FastDMF: fast simulator of the dynamic mean-field model of brain dynamics. &#xA;                  https://doi.org/10.5281/zenodo.6373512&#xA;                  &#xA;                 (2022)." href="/articles/s41593-022-01070-0#ref-CR96" id="ref-link-section-d30343262e4666">96</a></sup>).</p><p>The structural connectivity for the model was obtained by following the procedure of Wang et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Wang, P. et al. Inversion of a large-scale circuit model reveals a cortical hierarchy in the dynamic resting human brain. Sci. Adv. 5, 1–12 (2019)." href="/articles/s41593-022-01070-0#ref-CR97" id="ref-link-section-d30343262e4673">97</a></sup>, which derives a consensus structural connectivity matrix <i>A</i> from the individual SC matrices of the 100 HCP participants included in the present study. Hence, for each individual, the structural connectivity was obtained from deterministic tractography (as described above) performed with the Lausanne-83 parcellation. Subsequently, for each pair of regions <i>i</i> and <i>j</i>, if more than half of participants had nonzero connection <i>i</i> and <i>j</i>, <i>A</i><sub><i>ij</i></sub> was set to the average across all participants with nonzero connections between <i>i</i> and <i>j</i>. Otherwise, <i>A</i><sub><i>ij</i></sub> was set to zero.</p><p>A Balloon–Windkessel hemodynamic model was then used to turn simulated regional neuronal activity into simulated regional BOLD signal. The Balloon–Windkessel model considers the BOLD signal as a static nonlinear function of the normalized total deoxyhemoglobin voxel content, normalized venous volume, resting net oxygen extraction fraction by the capillary bed and resting blood volume fraction. The BOLD signal estimation for each brain area is computed from the level of neuronal activity in that particular area. Finally, simulated regional BOLD signal was bandpass filtered in the same range as the empirical data (0.008–0.09 Hz).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec45">Model fitting</h4><p>The model has only one free parameter <i>G</i>, which scales the global coupling strength. To find the value of <i>G</i> that generates the most realistic data, we first generated 100 simulations with a TR of 0.72 s (that is, the same as the empirical HCP data) for each value of <i>G</i> between 0.1 and 2.5, using increments of 0.1. Following previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Deco, G. et al. Whole-brain multimodal neuroimaging model using serotonin receptor maps explains non-linear functional effects of LSD. Curr. Biol. 28, 3065–3074 (2018)." href="#ref-CR93" id="ref-link-section-d30343262e4731">93</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Herzog, R. et al. A mechanistic model of the neural entropy increase elicited by psychedelic drugs. Sci. Rep. 10, 17725 (2020)." href="#ref-CR94" id="ref-link-section-d30343262e4731_1">94</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Luppi, A. I. et al. Whole-brain modelling identifies distinct but convergent paths to unconsciousness in anaesthesia and disorders of consciousness. Commun. Biol. 5, 384 (2022)." href="/articles/s41593-022-01070-0#ref-CR95" id="ref-link-section-d30343262e4734">95</a></sup>, we selected the value of <i>G</i> that minimized the Kolmogorov–Smirnov distance between the empirical and simulated functional connectivity dynamics, which has been shown to provide a better fit than simply using the global FC. The Kolmogorov–Smirnov distance between empirical and simulated functional connectivity dynamics was minimized for a value of <i>G</i> = 1.6 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig13">6</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec46">Simulated human fMRI data</h4><p>To determine whether the observed difference in the proportion of synergy between human and macaque brains could be explained exclusively by the difference in TR, we then generated another set of 100 simulations, using the empirically determined best-fitting <i>G</i> parameter of 1.6, but now with a TR of 2.6 s, that is, the same as the macaque data. The simulated data were filtered between 0.008 and 0.09 Hz, and their mean normalized synergy across all pairs of regions (that is, the proportion of total information accounted for by synergy) was compared with the normalized synergy empirically observed in macaque brains (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-022-01070-0#Fig13">6</a>).</p><h3 class="c-article__sub-heading" id="Sec47">HAR-Brain genes</h3><p>The maps of regional expression of human-accelerated genes for the DK-114 atlas were made available by Wei et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. Nat. Commun. 10, 4839 (2019)." href="/articles/s41593-022-01070-0#ref-CR34" id="ref-link-section-d30343262e4771">34</a></sup>. Briefly, genes located in a total of 2,737 human-accelerated regions (HARs) of the genome were taken as presented by comparative genome analysis representing genomic loci with accelerated divergence in humans<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Doan, R. N. et al. Mutations in human accelerated regions disrupt cognition and social behavior the homozygosity mapping consortium for autism HHS public access. Cell 167, 341–354 (2016)." href="/articles/s41593-022-01070-0#ref-CR98" id="ref-link-section-d30343262e4775">98</a></sup>. Of 2,143 HAR-associated genes identified from this procedure, 1,711 were described in the Allen Human Brain Atlas (AHBA) microarray dataset (<a href="https://human.brain-map.org/">https://human.brain-map.org/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-022-01070-0#ref-CR99" id="ref-link-section-d30343262e4786">99</a></sup> and were used in the analyses by Wei and colleagues, referred to as ‘HAR genes’.</p><p>HAR genes were subsequently subdivided into HAR-Brain and HAR-Nonbrain genes. BRAIN genes were selected as the set of genes commonly expressed in human brain tissue using the Genotype-Tissue Expression (GTEx) database (GTEx Analysis Release V6p; <a href="https://www.gtexportal.org/">https://www.gtexportal.org/</a>), which includes 56,238 gene expression profiles in 53 body sites collected from 7,333 postmortem samples in 449 individuals. From these 56,238 genes, 2,823 genes were identified as BRAIN genes showing significantly higher expressions in brain sites than non-brain sites (one-sided <i>t</i>-test and an FDR-corrected <i>q</i> &lt; 0.05 were used). HAR-Brain genes were identified as the 405 genes that overlapped between the 2,823 BRAIN genes and the 1,711 HAR genes, whereas the remaining HAR genes were labeled as HAR-Nonbrain genes. Finally, the HAR gene expression data were mapped to the 114-region subdivision of the Desikan–Killiany atlas (DK-114). As only two of the six AHBA donors have data for the right hemisphere, Wei et al. only considered HAR gene expression patterns for the left hemisphere.</p><h3 class="c-article__sub-heading" id="Sec48">Cortical expansion</h3><p>The maps of evolutionary cortical expansion were made available by Wei et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. Nat. Commun. 10, 4839 (2019)." href="/articles/s41593-022-01070-0#ref-CR34" id="ref-link-section-d30343262e4814">34</a></sup> who describe in detail how these data were generated. Briefly, Wei and colleagues analyzed in vivo MRI data from 29 adult chimpanzees, as well as 30 adult human participants from the HCP. Pial surface reconstructions of chimpanzee and human T1-weighted MRI scans (processed with FreeSurfer v5.3.0; <a href="https://surfer.nmr.mgh.harvard.edu/">https://surfer.nmr.mgh.harvard.edu/</a>) were used for both vertex-to-vertex mapping across chimpanzee and humans and also for subsequent computation of region-wise expansion for cortical morphometry. A regional-level cortical surface area (Si) was computed by summing up face areas within each cortical region, for all regions of the DK-114 atlas. Normalized cortical area was obtained by dividing the regional area by the area of the whole cortex. Cortical expansion between every pair of chimpanzee and human participants was calculated based on both the raw (‘unadjusted’) and normalized (‘adjusted’) cortical surface area by:</p><div id="Equc" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$E_{i,j} = \frac{{S_{\textrm{human},i}-S_{\textrm{chimp},j}}}{{S_{\textrm{chimp},j}}}$$</span></div></div><p>with <i>E</i><sub><i>i,j</i></sub> denoting the expansion from chimpanzee <i>j</i> to human <i>i</i>. A group-level region-wise cortical expansion map was calculated by taking averages over the 870 chimpanzee-to-human comparisons.</p><h3 class="c-article__sub-heading" id="Sec49">Regional glycolytic index data</h3><p>The maps of average regional GI for 41 regions (left and right) of the Brodmann atlas were made available by Vaishnavi et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Vaishnavi, S. N. et al. Regional aerobic glycolysis in the human brain. Proc. Natl. Acad. Sci. USA 107, 17757–17762 (2010)." href="/articles/s41593-022-01070-0#ref-CR39" id="ref-link-section-d30343262e4958">39</a></sup> based on PET scans of 33 healthy adults (scanned at rest and with eyes closed). We mapped 78 of these values onto an MNI-space parcellation of the Brodmann atlas, which we used to parcellate our fMRI data. The GI is a measure of regional AG, as it quantifies the extent to which empirically measured glucose consumption exceeds or falls short of the glucose consumption that would be predicted based on oxygen metabolism. Specifically, GI is obtained by linear regression of regional cerebral metabolic rate of glucose (measured using [<sup>18</sup>F]fluorodeoxyglucose) on regional cerebral metabolic rate of oxygen, measured by combining three PET scans administering <sup>15</sup>O-labeled water, carbon monoxide and oxygen.</p><h3 class="c-article__sub-heading" id="Sec50">Genes related to aerobic glycolysis</h3><p>Goyal et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Goyal, M. S., Hawrylycz, M., Miller, J. A., Snyder, A. Z. &amp; Raichle, M. E. Aerobic glycolysis in the human brain is associated with development and neotenous gene expression. Cell Metabolism 19, 49–57 (2014)." href="/articles/s41593-022-01070-0#ref-CR38" id="ref-link-section-d30343262e4974">38</a></sup> performed data-driven analysis using AHBA microarray data and an independent transcriptomic database (BrainSpan Study (BSS); <a href="http://brainspan.org/">http://brainspan.org/</a> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Kang, H. J. et al. Spatio-temporal transcriptome of the human brain. Nature 478, 483–489 (2011)." href="/articles/s41593-022-01070-0#ref-CR100" id="ref-link-section-d30343262e4985">100</a></sup>)), to identify genes whose regional distribution was significantly associated with the regional distribution of AG values (GI) previously obtained by Vaishnavi and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Vaishnavi, S. N. et al. Regional aerobic glycolysis in the human brain. Proc. Natl. Acad. Sci. USA 107, 17757–17762 (2010)." href="/articles/s41593-022-01070-0#ref-CR39" id="ref-link-section-d30343262e4989">39</a></sup>. They identified and made available a list of 116 genes that consistently occurred among the top 1,000 of 17,205 genes whose expression is associated (after correction for multiple comparisons) with the regional distribution of GI in the human brain, both in the AHBA dataset and in five matched BSS adult brains.</p><h3 class="c-article__sub-heading" id="Sec51">AIBS data-driven gene expression analysis</h3><p>Regional gene expression levels for 20,647 human genes were obtained from transcriptomic measurements in six postmortem brains from adult human donors with no history of psychiatric or neuropathological disorders (aged 24–57 years), made available by the AIBS (<a href="https://human.brain-map.org/">https://human.brain-map.org/</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. Nature 489, 391–399 (2012)." href="/articles/s41593-022-01070-0#ref-CR99" id="ref-link-section-d30343262e5008">99</a></sup>. We used code made freely available by Morgan et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e5012">23</a></sup> to obtain a 308 × 20,647 regional transcription matrix, matching gene expression data to each cortical region of the DK-308 atlas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e5016">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e5019">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Vértes, P. E. et al. Gene transcription profiles associated with inter-modular hubs and connection distance in human functional magnetic resonance imaging networks. Philos. Trans. R. Soc. Lond. B Biol. Sci. 371, 20150362 (2016)." href="/articles/s41593-022-01070-0#ref-CR83" id="ref-link-section-d30343262e5022">83</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 101" title="Romero-Garcia, R. et al. Structural covariance networks are coupled to expression of genes enriched in supragranular layers of the human cortex. Neuroimage 171, 256–267 (2018)." href="/articles/s41593-022-01070-0#ref-CR101" id="ref-link-section-d30343262e5025">101</a></sup>. Each tissue sample was assigned to a cortical region using the AIBS MRI data for each donor, pooling samples between bilaterally homologous regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e5029">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Vértes, P. E. et al. Gene transcription profiles associated with inter-modular hubs and connection distance in human functional magnetic resonance imaging networks. Philos. Trans. R. Soc. Lond. B Biol. Sci. 371, 20150362 (2016)." href="/articles/s41593-022-01070-0#ref-CR83" id="ref-link-section-d30343262e5032">83</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec52">Partial least squares</h4><p>To explore the association between the redundancy-to-synergy regional gradient and all 20,647 genes measured in the AHBA microarrays, at each of 308 regions, we used PLS as a dimensionality reduction technique<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e5043">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Vértes, P. E. et al. Gene transcription profiles associated with inter-modular hubs and connection distance in human functional magnetic resonance imaging networks. Philos. Trans. R. Soc. Lond. B Biol. Sci. 371, 20150362 (2016)." href="/articles/s41593-022-01070-0#ref-CR83" id="ref-link-section-d30343262e5046">83</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="Krishnan, A., Williams, L. J., McIntosh, A. R. &amp; Abdi, H. Partial least-squares methods for neuroimaging: a tutorial and review. Neuroimage 56, 455–475 (2011)." href="/articles/s41593-022-01070-0#ref-CR102" id="ref-link-section-d30343262e5049">102</a></sup>. PLS finds components from the predictor variables (308 × 20,647 matrix of regional gene expression scores) that have maximum covariance with the response variables (308 × 1 matrix of regional redundancy-to-synergy gradient). The PLS components (that is, linear combinations of the weighted gene expression scores) are ranked by covariance between predictor and response variables, so that the first few PLS components provide a low-dimensional representation of the covariance between the higher dimensional data matrices.</p><p>Non-parametric goodness of fit of low-dimensional PLS components was performed by repeating the analysis 1,000 times after shuffling the regional labels. The error on the PLS weights associated with each gene was tested by resampling with replacement of 308 ROIs (bootstrapping); the ratio of the weight of each gene to its bootstrap standard error was used to compute gene-level <i>z</i>-scores and rank the genes according to their contributions to each PLS component<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e5059">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Vértes, P. E. et al. Gene transcription profiles associated with inter-modular hubs and connection distance in human functional magnetic resonance imaging networks. Philos. Trans. R. Soc. Lond. B Biol. Sci. 371, 20150362 (2016)." href="/articles/s41593-022-01070-0#ref-CR83" id="ref-link-section-d30343262e5062">83</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec53">Gene Ontology and enrichment analysis</h4><p>We used GOrilla for enrichment analysis of the first two PLS components<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e5074">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Eden, E., Navon, R., Steinfeld, I., Lipson, D. &amp; Yakhini, Z. GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists. BMC Bioinformatics 10, 48 (2009)." href="/articles/s41593-022-01070-0#ref-CR103" id="ref-link-section-d30343262e5077">103</a></sup>. GOrilla identifies enriched GO terms in a ranked gene list, leveraging a large online database of gene annotations corresponding to ‘biological processes’ and ‘cellular components’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Eden, E., Navon, R., Steinfeld, I., Lipson, D. &amp; Yakhini, Z. GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists. BMC Bioinformatics 10, 48 (2009)." href="/articles/s41593-022-01070-0#ref-CR103" id="ref-link-section-d30343262e5081">103</a></sup>. We identified GO terms that were overrepresented among the genes with the strongest positive weightings on each PLS component (that is, those most strongly associated with dominance of synergy over redundancy). For our analyses on the online GOrilla platform (<a href="http://cbl-gorilla.cs.technion.ac.il/">http://cbl-gorilla.cs.technion.ac.il/</a>), we unchecked the ‘Run GOrilla in fast mode’ option and used the ‘<i>P</i>-value threshold 10<sup>−4</sup>’ setting to best approximate FDR correction with α = 0.05 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e5098">35</a></sup>).</p><p>We then used the online tool REVIGO (<a href="http://revigo.irb.hr/">http://revigo.irb.hr/</a>) to summarize the list of significant GO terms and visualize the results of whole-genome enrichment analysis. First, REVIGO uses measures of semantic similarity between terms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 104" title="Supek, F., Bošnjak, M., ˇ Kunca, S. &amp; ˇ Muc, S. Summarizes and visualizes long lists of Gene Ontology terms. PLoS ONE 6, 21800 (2011)." href="/articles/s41593-022-01070-0#ref-CR104" id="ref-link-section-d30343262e5112">104</a></sup> to identify representative clusters of genes. Then, REVIGO plots significant GO terms in semantic space, where semantically similar GO terms are represented clustered near one another and labeled in a representative manner.</p><p>We note that this analysis does not take into account within-category gene–gene correlations; therefore, we also used a more sophisticated null model that was recently developed to estimate gene enrichment relative to an ensemble of null phenotypes (rather than an ensemble of random genes)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 105" title="Fulcher, B. D., Arnatkeviciute, A. &amp; Fornito, A. Overcoming false-positive gene-category enrichment in the analysis of spatially resolved transcriptomic brain atlas data. Nat. Commun. 12, 2669 (2021)." href="/articles/s41593-022-01070-0#ref-CR105" id="ref-link-section-d30343262e5119">105</a></sup> using a freely available toolbox: <a href="https://github.com/benfulcher/GeneCategoryEnrichmentAnalysis/wiki/Ensemble-enrichment">https://github.com/benfulcher/GeneCategoryEnrichmentAnalysis/wiki/Ensemble-enrichment/</a>.</p><p>For our hypothesis-driven analysis, testing for enrichment of HAR-Brain genes, we also used non-parametric permutation testing. Specifically, we randomly drew 1,000 samples of the same number of genes and estimated their PLS weighting, and compared the PLS weights of the HAR-Brain genes to this permutation distribution. This provided an estimate of the probability of HAR-Brain gene enrichment of each PLS component under the null hypothesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. Proc. Natl. Acad. Sci. USA 116, 9604–9609 (2019)." href="/articles/s41593-022-01070-0#ref-CR23" id="ref-link-section-d30343262e5133">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. Proc. Natl. Acad. Sci. USA 113, 9105–9110 (2016)." href="/articles/s41593-022-01070-0#ref-CR35" id="ref-link-section-d30343262e5136">35</a></sup>. The same procedure was used to test for significant enrichment of genes related to regional AG.</p><h3 class="c-article__sub-heading" id="Sec54">Robustness of gene enrichment results</h3><p>Our data-driven analysis of regional gene expression also indicated significant enrichment when performed using an alternative approach: ridge-regularized PLS regression implemented in the R package ‘plsgenomics’ (<a href="https://cran.r-project.org/package=plsgenomics">https://CRAN.R-project.org/package=plsgenomics</a>). As this package requires binary target variables, we therefore turned the regional gradient from synergy to redundancy into a binary vector characterizing each region in terms of the predominance of synergy or redundancy (as indicated by the sign of the gradient), regardless of magnitude. Therefore, this analysis seeks to find regularized coefficients to predict synergy-rich versus redundancy-rich binary status of each region, based on regional gene expression. We obtained regression coefficients for each AHBA gene using a ridge (L2) penalty norm determined by tenfold cross-validation. These coefficients were then turned into <i>z</i>-scores, and enrichment for HAR-Brain genes and genes related to AG was computed using permutation testing as described above.</p><p>The second alternative approach was designed to account for the potential confound of spatial autocorrelation. To this end, we obtained a <i>z</i>-score for each AHBA gene, by dividing its empirical Spearman correlation with the redundancy-to-synergy cortical map, with the standard deviation of the distribution of correlations obtained from 5,000 randomly rotated cortical maps having preserved spatial autocorrelation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. Neuroimage 178, 540–551 (2018)." href="/articles/s41593-022-01070-0#ref-CR53" id="ref-link-section-d30343262e5165">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 106" title="Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. Cerebral Cortex 28, 281–294 (2018)." href="/articles/s41593-022-01070-0#ref-CR106" id="ref-link-section-d30343262e5168">106</a></sup>. Enrichment for HAR-Brain genes and genes related to AG was then computed using permutation testing as described above, by comparing the weights of the genes of interest with the weights of 1,000 random selections of the same number of genes.</p><h3 class="c-article__sub-heading" id="Sec55">Synaptic density from positron emission tomography</h3><p>In vivo estimates of regional synaptic density in the human brain were obtained from PET with the radioligand [<sup>11</sup>C]UCB-J ((R)-1-((3-(methyl-<sup>11</sup>C)pyridin-4-yl)methyl)-4- (3,4,5-trifluorophenyl)pyr-rolidin-2-one)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Holland, N. et al. Synaptic loss in primary tauopathies revealed by [11C]UCB-J positron emission tomography. Mov. Disord. &#xA;                  https://doi.org/10.1002/mds.28188&#xA;                  &#xA;                 (2020)." href="/articles/s41593-022-01070-0#ref-CR37" id="ref-link-section-d30343262e5184">37</a></sup>. This ligand quantifies synaptic density<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Finnema, S. J. et al. Imaging synaptic density in the living human brain. Sci. Transl. Med. 8, 348ra96 (2016)." href="/articles/s41593-022-01070-0#ref-CR36" id="ref-link-section-d30343262e5188">36</a></sup> based on its affinity for the presynaptic vesicle glycoprotein 2A<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 107" title="Milicevic Sephton, S. et al. Automated radiosynthesis of [11C]UCB-J for imaging synaptic density by positron emission tomography. J. Labelled Comp. Radiopharm. 63, 151–158 (2020)." href="/articles/s41593-022-01070-0#ref-CR107" id="ref-link-section-d30343262e5192">107</a></sup>, which is ubiquitously expressed in all brain synapses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 108" title="Bajjalieh, S. M., Frantz, G. D., Weimann, J. M., McConnell, S. K. &amp; Scheller, R. H. Differential expression of synaptic vesicle protein 2 (SV2) isoforms. J. Neurosci. 14, 5223–5235 (1994)." href="/articles/s41593-022-01070-0#ref-CR108" id="ref-link-section-d30343262e5197">108</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec56">PET/magnetic resonance imaging protocol</h4><p>The research protocol was approved by an NHS Research Ethics Committee (18/EE/0059) and the Administration of Radioactive Substances Advisory Committee, and all participants provided written informed consent in accordance with the Declaration of Helsinki.</p><p>Participant recruitment and exclusion criteria are described in detail in the original publication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Holland, N. et al. Synaptic loss in primary tauopathies revealed by [11C]UCB-J positron emission tomography. Mov. Disord. &#xA;                  https://doi.org/10.1002/mds.28188&#xA;                  &#xA;                 (2020)." href="/articles/s41593-022-01070-0#ref-CR37" id="ref-link-section-d30343262e5211">37</a></sup>. Briefly, 14 patients with progressive supranuclear palsy (PSP)–Richardson’s syndrome and 15 patients with corticobasal syndrome (CBS) were recruited from a tertiary specialist clinic for PSP/CBS at the Cambridge University Centre for Parkinson-Plus. Additionally, 15 age-, sex- and education-matched healthy volunteers were recruited from the UK National Institute for Health Research (NIHR) Join Dementia Research register between June 2019 and June 2020. Both healthy controls and patient volunteers were prescreened by telephone; participants were excluded based on any of the following: history of cancer within the last 5 years, concurrent use of the medication levetiracetam, any severe physical illness or comorbidity that limited ability to fully participate in the study, and any contraindications to MRI. Participants were also excluded based on history of ischemic or hemorrhagic stroke evident on MRI available from the clinic. Here, we only included data from the healthy volunteers (<i>N</i> = 15, 8 females; aged 68 ± 7 years).</p><p>The radioligand [<sup>11</sup>C]UCB-J was synthesized at the Radiopharmacy Unit, Wolfson Brain Imaging Centre, Cambridge University<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 107" title="Milicevic Sephton, S. et al. Automated radiosynthesis of [11C]UCB-J for imaging synaptic density by positron emission tomography. J. Labelled Comp. Radiopharm. 63, 151–158 (2020)." href="/articles/s41593-022-01070-0#ref-CR107" id="ref-link-section-d30343262e5223">107</a></sup>. All participants underwent simultaneous 3T MRI and [11C]UCB-J PET on a GE SIGNA PET/magnetic resonance scanner (GE Healthcare). Participants were under continuous visual observation from the adjacent control room, and there was an open microphone channel to allow the participant to contact the radiographers. The radiographer was present throughout with medical cover on site in case of need (such a need did not arise).</p><p>Dynamic PET data acquisition was performed for 90 min starting immediately after [<sup>11</sup>C]UCB-J injection (median (range) injected activity of 408 (192–523) MBq, injected UCB-J mass ≤ 10 µg). Attenuation correction included the use of a multi-subject atlas method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 109" title="Burgos, N. et al. Attenuation correction synthesis for hybrid PET-MR scanners: application to brain studies. IEEE Trans. Med. Imaging 33, 2332–2341 (2014)." href="/articles/s41593-022-01070-0#ref-CR109" id="ref-link-section-d30343262e5232">109</a></sup> and improvements to the MRI brain coil component<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 110" title="Manavaki, R., Hong, Y. &amp; Fryer, T. D. Effect of brain MRI coil attenuation map processing on PET image quantification and uniformity for the GE SIGNA PET/MR. IEEE Nucl. Sci. Symp. Med. Imaging Conf. NSS/MIC (2019)." href="/articles/s41593-022-01070-0#ref-CR110" id="ref-link-section-d30343262e5236">110</a></sup>. Each emission image series was aligned using SPM12 (<a href="http://www.fil.ion.ucl.ac.uk/spm/software/spm12/">www.fil.ion.ucl.ac.uk/spm/software/spm12/</a>), then rigidly registered to a T1-weighted MRI acquired during PET data acquisition (TR of 3.6 ms, TE of 9.2 ms, 192 sagittal slices, in-plane resolution of 0.55 × 0.55 mm (subsequently interpolated to 1.0 × 1.0 mm), slice thickness of 1.0 mm). Regional time–activity curves were extracted following the application of geometric transfer matrix partial volume correction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 110" title="Manavaki, R., Hong, Y. &amp; Fryer, T. D. Effect of brain MRI coil attenuation map processing on PET image quantification and uniformity for the GE SIGNA PET/MR. IEEE Nucl. Sci. Symp. Med. Imaging Conf. NSS/MIC (2019)." href="/articles/s41593-022-01070-0#ref-CR110" id="ref-link-section-d30343262e5247">110</a></sup> to each of the dynamic PET images. To quantify presynaptic vesicle glycoprotein 2A density (and therefore synaptic density), regional [<sup>11</sup>C]UCB-J non-displaceable binding potential (BP<sub>ND</sub>) was determined for a 68-ROI subdivision of the Desikan–Killiany cortical atlas (DK-68), using a basis function implementation of the simplified reference tissue model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 111" title="Wu, Y. &amp; Carson, R. E. Noise reduction in the simplified reference tissue model for neuroreceptor functional imaging. J. Cereb. Blood Flow Metab. 22, 1440–1452 (2002)." href="/articles/s41593-022-01070-0#ref-CR111" id="ref-link-section-d30343262e5256">111</a></sup> with the reference tissue defined in the centrum semiovale<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 112" title="Koole, M. et al. Quantifying SV2A density and drug occupancy in the human brain using [11C]UCB-J PET imaging and subcortical white matter as reference tissue. Eur. J. Nucl. Med. Mol. Imaging 46, 396–406 (2019)." href="/articles/s41593-022-01070-0#ref-CR112" id="ref-link-section-d30343262e5260">112</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 113" title="Rossano, S. et al. Assessment of a white matter reference region for [11C]UCB-J PET quantification. J Cereb. Blood Flow Metab. 40, 1890–1901 (2020)." href="/articles/s41593-022-01070-0#ref-CR113" id="ref-link-section-d30343262e5263">113</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec57">Principal components of synaptic density</h4><p>Principal-component analysis was subsequently used to derive the principal components that explained most of the variance in regional [<sup>11</sup>C]UCB-J BP<sub>ND</sub> across volunteers. Components were selected if their associated eigenvalue was greater than unity; two principal components satisfied this criterion, explaining 45% and 16% of the variance.</p><h3 class="c-article__sub-heading" id="Sec58">Molecular diversity from quantitative autoradiography</h3><p>Following the recent work of Goulas and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e5288">40</a></sup>, we analyzed quantitative data about the density of neurotransmitter receptors obtained from in vitro quantitative autoradiography. Data were obtained for 15 different types of receptors: glutamate (AMPA, NMDA, kainate), GABA (GABA<sub>A</sub>, GABA<sub>A</sub>/BZ, GABA<sub>B</sub>), acetylcholine (muscarinic M<sub>1</sub>, M<sub>2</sub>, M<sub>3</sub>, nicotinic a<sub>4</sub>b<sub>2</sub>), noradrenaline (a<sub>1</sub>, a<sub>2</sub>), serotonin (5-HT<sub>1A</sub>, 5-HT<sub>2</sub>), and dopamine (D1). Thus, both excitatory and inhibitory receptors were considered, and both ionotropic and metabotropic receptors.</p><p>The receptor autoradiography data collection and processing are described in detail in the original publication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zilles, K. &amp; Palomero-Gallagher, N. Multiple transmitter receptors in regions and layers of the human cerebral cortex. Front. Neuroanat. 11, 78 (2017)." href="/articles/s41593-022-01070-0#ref-CR41" id="ref-link-section-d30343262e5321">41</a></sup>. Briefly, in vitro autoradiography was performed on four postmortem cerebral hemispheres obtained from three human donors with no known history of neurological or psychiatric diseases (1 female; age 75 ± 3 years) with their previous written consent. All procedures were in accordance with the ethical requirements of the body donor program of the Department of Anatomy, University of Dusseldorf. The causes of death were reported as cardiac arrest, lung edema and myocardial infarction; postmortem delay was 12 ± 5 h (neurotransmitter binding-site densities are stable up to 70–80 h postmortem). For additional details, see ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Zilles, K. &amp; Palomero-Gallagher, N. Multiple transmitter receptors in regions and layers of the human cerebral cortex. Front. Neuroanat. 11, 78 (2017)." href="/articles/s41593-022-01070-0#ref-CR41" id="ref-link-section-d30343262e5325">41</a></sup>. Laminar data were collected for 44 visual, somatosensory, auditory and multimodal association regions of the human cerebral cortex, and summarized into receptor densities for infra-, supra- and granular layers for each region. The same regions were also used to obtain a redundancy-to-synergy gradient (albeit without full cortical coverage) as described above; for this, areas V2v and V2d were combined together, as were areas 37B, 37L and 37M, and areas 10L and 10M; data from both hemispheres were combined to obtain one value per region, and the corresponding receptor density values were averaged separately for each layer. Following previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e5329">40</a></sup> area 24 was not included in the present analysis due to its lack of a granular layer; thus, a total of 39 areas were considered.</p><p>Molecular diversity was then estimated as described by Goulas and colleagues<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e5336">40</a></sup>. Briefly, a vector <b>N</b> was obtained for each area, encoding its normalized receptor profile so that each entry denotes the density of a specific receptor, divided by the maximum density value observed for this receptor across all areas. Diversity of each area was then quantified as the Shannon entropy <i>H</i> of the normalized receptor profile <b>N</b>, computed as:</p><div id="Equd" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$H = - \textrm{sum}\left( {N \times \textrm{log}\left( N \right)} \right)/\textrm{log}\left( M \right).$$</span></div></div><p>Here, <i>M</i> denotes the length of the vector <b>N</b>, that is, 45 elements (15 receptors times three layers), and log is the natural logarithm (so that entropy was normalized to lie between 0 and 1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e5429">40</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec59">Validation with surrogate data methods</h3><p>We generated pseudo-gradients of redundancy-to-synergy patterns in three different ways. The first way was to generate surrogate pseudo-synergy matrices by randomly rewiring the matrix of interregional synergy before obtaining the pseudo-synergy ranks for the gradient, while using the original redundancy ranks, to test whether our results could be obtained if synergy were randomly distributed across regions. The second approach was to obtain a random vector that has the same (negative) correlation with redundancy, and then using it to obtain pseudo-synergy ranks to compute the redundancy-to-synergy gradient (again with the redundancy ranks being the true ones), to test whether the observed anticorrelation would suffice to produce our results. The third approach was to generate phase-randomized surrogate time series for each participant by taking the Fourier transform of each region’s activity, replacing the Fourier phase with a random (antisymmetric) complex vector, and taking the inverse Fourier transform. Then, matrices of redundancy and synergy were computed from these phase-randomized time series for each participant, and used to build a redundancy-to-synergy gradient, to test whether our results could arise from signals with the same autocorrelation, but no correlation with each other.</p><h3 class="c-article__sub-heading" id="Sec60">Statistical analysis</h3><p>One-sample non-parametric <i>t</i>-tests with 10,000 permutations were used to determine whether the synergy–redundancy scores were significantly different from zero for each of the Yeo resting-state subnetworks and for each cytoarchitectonic class of Von Economo; FDR correction for multiple comparisons was adopted according to the Benjamini–Hochberg procedure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 114" title="Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. Royal Stat. Soc. Ser. B 57, 289–300 (1995)." href="/articles/s41593-022-01070-0#ref-CR114" id="ref-link-section-d30343262e5453">114</a></sup>.</p><p>The statistical significance of within-group differences was determined with non-parametric permutation <i>t</i>-tests (paired-sample), with 10,000 permutations. Two-sample non-parametric <i>t</i>-tests (also with 10,000 permutations) were instead used to test the statistical significance of human–macaque comparisons, and of comparisons between different subsets of connections in the brain. The use of non-parametric tests alleviated the need to assume normality of data distributions (which was not formally tested). All tests were two-sided, with an alpha value of 0.05. The effect sizes were estimated using Hedges’s measure of the standardized mean difference, <i>g</i>. To statistically assess the difference between effect sizes obtained from comparing humans and macaques using different measures, we used a <i>z</i>-test as described by Borenstein et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 115" title="Borenstein, M., Hedges, L. V., Higgins, J. P. T. &amp; Rothstein, H. R. Introduction to Meta-Analysis. 1–421 &#xA;                  https://doi.org/10.1002/9780470743386&#xA;                  &#xA;                 (Wiley, 2009)." href="/articles/s41593-022-01070-0#ref-CR115" id="ref-link-section-d30343262e5472">115</a></sup>.</p><p>To ensure robustness to possible outliers, all correlations were quantified using Spearman’s rank-based non-parametric correlation coefficient. To further ensure the robustness of our results to the potential confounding effect of spatial autocorrelation and contralateral symmetry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. Neuroimage 178, 540–551 (2018)." href="/articles/s41593-022-01070-0#ref-CR53" id="ref-link-section-d30343262e5479">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 116" title="Markello, R. D. &amp; Misic, B. Comparing spatial null models for brain maps. Neuroimage 236, 118052 (2021)." href="/articles/s41593-022-01070-0#ref-CR116" id="ref-link-section-d30343262e5482">116</a></sup>, we also estimated <i>P</i> values from a spatial permutation test which generates a null distribution of 10,000 randomly rotated brain maps with preserved spatial covariance (‘spin test’)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. Neuroimage 178, 540–551 (2018)." href="/articles/s41593-022-01070-0#ref-CR53" id="ref-link-section-d30343262e5489">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 106" title="Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. Cerebral Cortex 28, 281–294 (2018)." href="/articles/s41593-022-01070-0#ref-CR106" id="ref-link-section-d30343262e5492">106</a></sup>. This analysis is only applicable for parcellations with full cortical coverage; when data were only available for the left cortical hemisphere, they were mirrored to the corresponding regions of the right hemisphere to perform the spatial rotations, since this test explicitly controls for contralateral symmetry, and then we only considered this hemisphere for computing the empirical and permuted correlations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. Neuroimage 178, 540–551 (2018)." href="/articles/s41593-022-01070-0#ref-CR53" id="ref-link-section-d30343262e5496">53</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec61">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM1">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                </div>
            

            <div>
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>The HCP DWI data in SRC format are available online (<a href="http://brain.labsolver.org/diffusion-mri-data/hcp-dmri-data">http://brain.labsolver.org/diffusion-mri-data/hcp-dmri-data</a>). The HCP fMRI data are available online (<a href="https://www.humanconnectome.org/study/hcp-young-adult/data-releases">https://www.humanconnectome.org/study/hcp-young-adult/data-releases</a>). Macaque MRI data are available from the PRIME-DE through the Neuroimaging Informatics Tools and Resources Clearinghouse (NITRC; http:\\fcon_1000.projects.nitrc.org/indi/indiPRIME.html). The PET data that support the findings of this study are available from J.B.R. (james.rowe@mrc-cbu.cam.ac.uk) upon reasonable request for academic (noncommercial) purposes, subject to restrictions required to preserve participant confidentiality. The macaque connectome is available on Zenodo at <a href="https://doi.org/10.5281/zenodo.1471588">https://doi.org/10.5281/zenodo.1471588</a>. The CoCoMac database on which it is based, is also available online at <a href="http://cocomac.g-node.org/main/index.php?">http://cocomac.g-node.org/main/index.php?</a>. The GTEx database (release V6p) is available at <a href="https://www.gtexportal.org/">https://www.gtexportal.org/</a>. The BSS database is available at <a href="http://brainspan.org">http://brainspan.org/</a>. Cortical gene expression patterns were taken from the transcriptomic data of the AHBA (<a href="http://human.brain-map.org/static/download">http://human.brain-map.org/static/download</a>). Region-wise maps of chimpanzee-to-human cortical expansion and HAR gene expression are available as supplementary materials from Wei et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. Nat. Commun. 10, 4839 (2019)." href="/articles/s41593-022-01070-0#ref-CR34" id="ref-link-section-d30343262e5664">34</a></sup> (<a href="https://doi.org/10.1038/s41467-019-12764-8">https://doi.org/10.1038/s41467-019-12764-8</a>). The NMT anatomical volume and associated probabilistic tissue segmentation maps (GM, WM and CSF) are freely available online: <a href="https://afni.nimh.nih.gov/pub/dist/atlases/macaque/nmt">https://afni.nimh.nih.gov/pub/dist/atlases/macaque/nmt</a> and <a href="http://github.com/jms290/NMT">http://github.com/jms290/NMT</a>. The maps of average regional GI are available as supplementary materials from Vaishnavi et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Vaishnavi, S. N. et al. Regional aerobic glycolysis in the human brain. Proc. Natl. Acad. Sci. USA 107, 17757–17762 (2010)." href="/articles/s41593-022-01070-0#ref-CR39" id="ref-link-section-d30343262e5690">39</a></sup> (<a href="https://doi.org/10.1073/pnas.1010459107">https://doi.org/10.1073/pnas.1010459107</a>). The genes whose expression is associated with the regional distribution of GI in the human brain are available as supplementary materials from Goyal et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Goyal, M. S., Hawrylycz, M., Miller, J. A., Snyder, A. Z. &amp; Raichle, M. E. Aerobic glycolysis in the human brain is associated with development and neotenous gene expression. Cell Metabolism 19, 49–57 (2014)." href="/articles/s41593-022-01070-0#ref-CR38" id="ref-link-section-d30343262e5701">38</a></sup> (<a href="https://doi.org/10.1016/j.cmet.2013.11.020">https://doi.org/10.1016/j.cmet.2013.11.020</a>). Anonymized receptor autoradiography data from Goulas et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. Proc. Natl Acad. Sci. USA 118, e2020574118 (2021)." href="/articles/s41593-022-01070-0#ref-CR40" id="ref-link-section-d30343262e5712">40</a></sup> are available at <a href="https://github.com/AlGoulas/receptor_principles">https://github.com/AlGoulas/receptor_principles</a>. The measure of cortical wiring distance is available as supplementary information from Paquola et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Paquola, C. et al. A multi-scale cortical wiring space links cellular architecture and functional dynamics in the human brain. PLoS Biol. 18, e3000979 (2020)." href="/articles/s41593-022-01070-0#ref-CR32" id="ref-link-section-d30343262e5724">32</a></sup> (<a href="https://doi.org/10.1371/journal.pbio.3000979">https://doi.org/10.1371/journal.pbio.3000979</a>).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>Data analysis was carried out in MATLAB version 2019a. The Java Information Dynamics Toolbox v1.5 is freely available online at <a href="https://github.com/jlizier/jidt">https://github.com/jlizier/jidt</a>. An updated version with MATLAB/Octave code to compute synergy and redundancy from integrated information decomposition of time series with the Gaussian MMI solver is available as <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM3">Supplementary Software</a>. The CONN toolbox version 17f is freely available at <a href="http://www.nitrc.org/projects/conn/">http://www.nitrc.org/projects/conn/</a>. DSI Studio is freely available at <a href="https://dsi-studio.labsolver.org/">https://dsi-studio.labsolver.org/</a>. The Brain Connectivity Toolbox code used for graph-theoretical analyses is freely available at <a href="https://sites.google.com/site/bctnet/">https://sites.google.com/site/bctnet/</a>. The code used for NeuroSynth meta-analysis is freely available at <a href="https://www.github.com/gpreti/GSP_StructuralDecouplingIndex">https://www.github.com/gpreti/GSP_StructuralDecouplingIndex</a>. The HRF deconvolution toolbox v2.2 is freely available at <a href="https://www.nitrc.org/projects/rshrf/">https://www.nitrc.org/projects/rshrf/</a>. The Pypreclin pipeline code v1.0.1 is freely available at <a href="https://github.com/neurospin/pypreclin">https://github.com/neurospin/pypreclin</a>. The code for PLS analysis of gene expression profiles is freely available at <a href="https://github.com/SarahMorgan/Morphometric_Similarity_SZ">https://github.com/SarahMorgan/Morphometric_Similarity_SZ</a>. The R package plsgenomics v1.5 is freely available at <a href="https://CRAN.R-project.org/package=plsgenomics">https://CRAN.R-project.org/package=plsgenomics</a>. The GOrilla platform is available at <a href="http://cbl-gorilla.cs.technion.ac.il">http://cbl-gorilla.cs.technion.ac.il</a>. The REVIGO platform is available at <a href="http://revigo.irb.hr">http://revigo.irb.hr</a>. The code for the dynamic mean-field model is freely available at <a href="http://www.gitlab.com/concog/fastdmf">http://www.gitlab.com/concog/fastdmf</a>. The code for spin-based permutation testing of cortical correlations is freely available at <a href="https://github.com/frantisekvasa/rotate_parcellation">https://github.com/frantisekvasa/rotate_parcellation</a>. The code for gene enrichment relative to an ensemble of null phenotypes is freely available at <a href="https://github.com/benfulcher/GeneCategoryEnrichmentAnalysis/wiki/Ensemble-enrichment">https://github.com/benfulcher/GeneCategoryEnrichmentAnalysis/wiki/Ensemble-enrichment</a>. FreeSurfer v5.3.0 is available at <a href="https://surfer.nmr.mgh.harvard.edu/">https://surfer.nmr.mgh.harvard.edu/</a>. SPM12 is available at <a href="http://www.fil.ion.ucl.ac.uk/spm/software/spm12/">www.fil.ion.ucl.ac.uk/spm/software/spm12/</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Marr, D. <i>Vision: A Computational Investigation into the Human Representation and Processing of Visual Information</i> (MIT Press, 2010).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Williams, P. L. &amp; Beer, R. D. Nonnegative decomposition of multivariate information. Preprint at <a href="http://arxiv.org/abs/1004.2515" data-track="click" data-track-action="external reference" data-track-label="http://arxiv.org/abs/1004.2515">http://arxiv.org/abs/1004.2515</a> (2010).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Timme, N., Alford, W., Flecker, B. &amp; Beggs, J. M. Synergy, redundancy and multivariate information measures: an experimentalist’s perspective. <i>J. Comput. Neurosci.</i> <b>36</b>, 119–140 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1007/s10827-013-0458-4" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s10827-013-0458-4" aria-label="Article reference 3" data-doi="10.1007/s10827-013-0458-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23820856" aria-label="PubMed reference 3">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Synergy%2C%20redundancy%20and%20multivariate%20information%20measures%3A%20an%20experimentalist%E2%80%99s%20perspective&amp;journal=J.%20Comput.%20Neurosci.&amp;doi=10.1007%2Fs10827-013-0458-4&amp;volume=36&amp;pages=119-140&amp;publication_year=2014&amp;author=Timme%2CN&amp;author=Alford%2CW&amp;author=Flecker%2CB&amp;author=Beggs%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Wibral, M., Priesemann, V., Kay, J. W., Lizier, J. T. &amp; Phillips, W. A. Partial information decomposition as a unified approach to the specification of neural goal functions. <i>Brain Cogn.</i> <b>112</b>, 25–38 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.bandc.2015.09.004" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bandc.2015.09.004" aria-label="Article reference 4" data-doi="10.1016/j.bandc.2015.09.004">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26475739" aria-label="PubMed reference 4">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Partial%20information%20decomposition%20as%20a%20unified%20approach%20to%20the%20specification%20of%20neural%20goal%20functions&amp;journal=Brain%20Cogn.&amp;doi=10.1016%2Fj.bandc.2015.09.004&amp;volume=112&amp;pages=25-38&amp;publication_year=2017&amp;author=Wibral%2CM&amp;author=Priesemann%2CV&amp;author=Kay%2CJW&amp;author=Lizier%2CJT&amp;author=Phillips%2CWA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Del Giudice, M. &amp; Crespi, B. J. Basic functional trade-offs in cognition: an integrative framework. <i>Cognition</i> <b>179</b>, 56–70 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cognition.2018.06.008" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cognition.2018.06.008" aria-label="Article reference 5" data-doi="10.1016/j.cognition.2018.06.008">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29909281" aria-label="PubMed reference 5">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Basic%20functional%20trade-offs%20in%20cognition%3A%20an%20integrative%20framework&amp;journal=Cognition&amp;doi=10.1016%2Fj.cognition.2018.06.008&amp;volume=179&amp;pages=56-70&amp;publication_year=2018&amp;author=Giudice%2CM&amp;author=Crespi%2CBJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at <a href="https://arxiv.org/abs/2109.13186" data-track="click" data-track-action="external reference" data-track-label="https://arxiv.org/abs/2109.13186">https://arxiv.org/abs/2109.13186</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Quian Quiroga, R. &amp; Panzeri, S. Extracting information from neuronal populations: information theory and decoding approaches. <i>Nat. Rev. Neurosci.</i> <b>10</b>, 173–185 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn2578" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn2578" aria-label="Article reference 7" data-doi="10.1038/nrn2578">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXitFKmurk%3D" aria-label="CAS reference 7">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19229240" aria-label="PubMed reference 7">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Extracting%20information%20from%20neuronal%20populations%3A%20information%20theory%20and%20decoding%20approaches&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn2578&amp;volume=10&amp;pages=173-185&amp;publication_year=2009&amp;author=Quian%20Quiroga%2CR&amp;author=Panzeri%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Whitacre, J. M. Biological robustness: paradigms, mechanisms, systems principles. <i>Front. Genet.</i> <b>3</b>, 1–15 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fgene.2012.00067" data-track-action="article reference" href="https://doi.org/10.3389%2Ffgene.2012.00067" aria-label="Article reference 8" data-doi="10.3389/fgene.2012.00067">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20robustness%3A%20paradigms%2C%20mechanisms%2C%20systems%20principles&amp;journal=Front.%20Genet.&amp;doi=10.3389%2Ffgene.2012.00067&amp;volume=3&amp;pages=1-15&amp;publication_year=2012&amp;author=Whitacre%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Tononi, G., Sporns, O. &amp; Edelman, G. M. A measure for brain complexity: relating functional segregation and integration in the nervous system. <i>Proc. Natl. Acad. Sci. USA</i> <b>91</b>, 5033–5037 (1994).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.91.11.5033" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.91.11.5033" aria-label="Article reference 9" data-doi="10.1073/pnas.91.11.5033">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK2c3lsFCltg%3D%3D" aria-label="CAS reference 9">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8197179" aria-label="PubMed reference 9">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC43925" aria-label="PubMed Central reference 9">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20measure%20for%20brain%20complexity%3A%20relating%20functional%20segregation%20and%20integration%20in%20the%20nervous%20system&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.91.11.5033&amp;volume=91&amp;pages=5033-5037&amp;publication_year=1994&amp;author=Tononi%2CG&amp;author=Sporns%2CO&amp;author=Edelman%2CGM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Schneidman, E., Still, S., Berry, M. J. &amp; Bialek, W. Network information and connected correlations. <i>Phys. Rev. Lett</i>. <b>91</b>, 23 (2003).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Reid, A. T. et al. Advancing functional connectivity research from association to causation. <i>Nat. Neurosci.</i> <b>22</b>, 1751–1760 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41593-019-0510-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41593-019-0510-4" aria-label="Article reference 11" data-doi="10.1038/s41593-019-0510-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFKgsr%2FF" aria-label="CAS reference 11">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31611705" aria-label="PubMed reference 11">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7289187" aria-label="PubMed Central reference 11">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Advancing%20functional%20connectivity%20research%20from%20association%20to%20causation&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fs41593-019-0510-4&amp;volume=22&amp;pages=1751-1760&amp;publication_year=2019&amp;author=Reid%2CAT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Rosas, F. E., Mediano, P. A. M., Gastpar, M. &amp; Jensen, H. J. Quantifying high-order interdependencies via multivariate extensions of the mutual information. <i>Phys. Rev. E</i> <b>100</b>, 32305 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1103/PhysRevE.100.032305" data-track-action="article reference" href="https://doi.org/10.1103%2FPhysRevE.100.032305" aria-label="Article reference 12" data-doi="10.1103/PhysRevE.100.032305">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVensbbO" aria-label="CAS reference 12">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20high-order%20interdependencies%20via%20multivariate%20extensions%20of%20the%20mutual%20information&amp;journal=Phys.%20Rev.%20E&amp;doi=10.1103%2FPhysRevE.100.032305&amp;volume=100&amp;publication_year=2019&amp;author=Rosas%2CFE&amp;author=Mediano%2CPAM&amp;author=Gastpar%2CM&amp;author=Jensen%2CHJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Schreiber, T. Measuring information transfer. <i>Phys. Rev. Lett.</i> <b>85</b>, 461–464 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1103/PhysRevLett.85.461" data-track-action="article reference" href="https://doi.org/10.1103%2FPhysRevLett.85.461" aria-label="Article reference 13" data-doi="10.1103/PhysRevLett.85.461">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3cXks1ersL4%3D" aria-label="CAS reference 13">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10991308" aria-label="PubMed reference 13">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20information%20transfer&amp;journal=Phys.%20Rev.%20Lett.&amp;doi=10.1103%2FPhysRevLett.85.461&amp;volume=85&amp;pages=461-464&amp;publication_year=2000&amp;author=Schreiber%2CT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Massey, J. Causality, feedback and directed information. <i>Proc. Int. Symp. Inf. Theory Applic.</i> 27–30 (1990).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Raichle, M. E. The restless brain: how intrinsic activity organizes brain function. <i>Philos. Trans. R. Soc. Lond. B Biol. Sci.</i> <b>370</b>, 20140172 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rstb.2014.0172" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2014.0172" aria-label="Article reference 15" data-doi="10.1098/rstb.2014.0172">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25823869" aria-label="PubMed reference 15">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4387513" aria-label="PubMed Central reference 15">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20restless%20brain%3A%20how%20intrinsic%20activity%20organizes%20brain%20function&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;doi=10.1098%2Frstb.2014.0172&amp;volume=370&amp;publication_year=2015&amp;author=Raichle%2CME">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Northoff, G., Wainio-Theberge, S. &amp; Evers, K. Is temporo-spatial dynamics the ‘common currency’ of brain and mind? In Quest of ‘spatiotemporal neuroscience’. <i>Phys. Life Rev.</i> <b>33</b>, 34–54 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.plrev.2019.05.002" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.plrev.2019.05.002" aria-label="Article reference 16" data-doi="10.1016/j.plrev.2019.05.002">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31221604" aria-label="PubMed reference 16">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Is%20temporo-spatial%20dynamics%20the%20%E2%80%98common%20currency%E2%80%99%20of%20brain%20and%20mind%3F%20In%20Quest%20of%20%E2%80%98spatiotemporal%20neuroscience%E2%80%99&amp;journal=Phys.%20Life%20Rev.&amp;doi=10.1016%2Fj.plrev.2019.05.002&amp;volume=33&amp;pages=34-54&amp;publication_year=2020&amp;author=Northoff%2CG&amp;author=Wainio-Theberge%2CS&amp;author=Evers%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Vidaurre, D., Smith, S. M. &amp; Woolrich, M. W. Brain network dynamics are hierarchically organized in time. <i>Proc. Natl. Acad. Sci. USA</i> <b>114</b>, 12827–12832 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1705120114" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1705120114" aria-label="Article reference 17" data-doi="10.1073/pnas.1705120114">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslemsrjM" aria-label="CAS reference 17">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29087305" aria-label="PubMed reference 17">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5715736" aria-label="PubMed Central reference 17">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Brain%20network%20dynamics%20are%20hierarchically%20organized%20in%20time&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1705120114&amp;volume=114&amp;pages=12827-12832&amp;publication_year=2017&amp;author=Vidaurre%2CD&amp;author=Smith%2CSM&amp;author=Woolrich%2CMW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Shine, J. M. et al. The dynamics of functional brain networks: integrated network states during cognitive task performance. <i>Neuron</i> <b>92</b>, 544–554 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2016.09.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2016.09.018" aria-label="Article reference 18" data-doi="10.1016/j.neuron.2016.09.018">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2qtLfE" aria-label="CAS reference 18">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27693256" aria-label="PubMed reference 18">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5073034" aria-label="PubMed Central reference 18">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20dynamics%20of%20functional%20brain%20networks%3A%20integrated%20network%20states%20during%20cognitive%20task%20performance&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2016.09.018&amp;volume=92&amp;pages=544-554&amp;publication_year=2016&amp;author=Shine%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Breakspear, M. Dynamic models of large-scale brain activity. <i>Nat. Neurosci.</i> <b>20</b>, 340–352 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4497" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4497" aria-label="Article reference 19" data-doi="10.1038/nn.4497">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVSms70%3D" aria-label="CAS reference 19">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28230845" aria-label="PubMed reference 19">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Dynamic%20models%20of%20large-scale%20brain%20activity&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4497&amp;volume=20&amp;pages=340-352&amp;publication_year=2017&amp;author=Breakspear%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Luppi, A. I. et al. Consciousness-specific dynamic interactions of brain integration and functional diversity. <i>Nat. Commun.</i> <b>10</b>, 4616 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-019-12658-9" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-12658-9" aria-label="Article reference 20" data-doi="10.1038/s41467-019-12658-9">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFKkur%2FF" aria-label="CAS reference 20">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31601811" aria-label="PubMed reference 20">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6787094" aria-label="PubMed Central reference 20">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Consciousness-specific%20dynamic%20interactions%20of%20brain%20integration%20and%20functional%20diversity&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-12658-9&amp;volume=10&amp;publication_year=2019&amp;author=Luppi%2CAI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Cover, T. M. &amp; Thomas, J. A. <i>Elements of Information Theory</i>. (Wiley-Interscience, 2005). <a href="https://doi.org/10.1002/047174882X" data-track="click" data-track-action="external reference" data-track-label="10.1002/047174882X">https://doi.org/10.1002/047174882X</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Barrett, A. B. Exploration of synergistic and redundant information sharing in static and dynamical Gaussian systems. <i>Phys. Rev. E</i> <b>91</b>, 52802 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1103/PhysRevE.91.052802" data-track-action="article reference" href="https://doi.org/10.1103%2FPhysRevE.91.052802" aria-label="Article reference 22" data-doi="10.1103/PhysRevE.91.052802">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFejs77E" aria-label="CAS reference 22">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Exploration%20of%20synergistic%20and%20redundant%20information%20sharing%20in%20static%20and%20dynamical%20Gaussian%20systems&amp;journal=Phys.%20Rev.%20E&amp;doi=10.1103%2FPhysRevE.91.052802&amp;volume=91&amp;publication_year=2015&amp;author=Barrett%2CAB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Morgan, S. E. et al. Cortical patterning of abnormal morphometric similarity in psychosis is associated with brain expression of schizophrenia-related genes. <i>Proc. Natl. Acad. Sci. USA</i> <b>116</b>, 9604–9609 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1820754116" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1820754116" aria-label="Article reference 23" data-doi="10.1073/pnas.1820754116">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXovFGltb0%3D" aria-label="CAS reference 23">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31004051" aria-label="PubMed reference 23">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6511038" aria-label="PubMed Central reference 23">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Cortical%20patterning%20of%20abnormal%20morphometric%20similarity%20in%20psychosis%20is%20associated%20with%20brain%20expression%20of%20schizophrenia-related%20genes&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1820754116&amp;volume=116&amp;pages=9604-9609&amp;publication_year=2019&amp;author=Morgan%2CSE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Yeo, B. T. T. et al. The organization of the human cerebral cortex estimated by intrinsic functional connectivity. <i>J. Neurophysiol.</i> <b>106</b>, 1125–1165 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1152/jn.00338.2011" data-track-action="article reference" href="https://doi.org/10.1152%2Fjn.00338.2011" aria-label="Article reference 24" data-doi="10.1152/jn.00338.2011">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21653723" aria-label="PubMed reference 24">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20organization%20of%20the%20human%20cerebral%20cortex%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=J.%20Neurophysiol.&amp;doi=10.1152%2Fjn.00338.2011&amp;volume=106&amp;pages=1125-1165&amp;publication_year=2011&amp;author=Yeo%2CBTT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Margulies, D. S. et al. Situating the default-mode network along a principal gradient of macroscale cortical organization. <i>Proc. Natl. Acad. Sci. USA</i> <b>113</b>, 12574–12579 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1608282113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1608282113" aria-label="Article reference 25" data-doi="10.1073/pnas.1608282113">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1ykurnM" aria-label="CAS reference 25">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27791099" aria-label="PubMed reference 25">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5098630" aria-label="PubMed Central reference 25">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Situating%20the%20default-mode%20network%20along%20a%20principal%20gradient%20of%20macroscale%20cortical%20organization&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1608282113&amp;volume=113&amp;pages=12574-12579&amp;publication_year=2016&amp;author=Margulies%2CDS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Shine, J. M. et al. Human cognition involves the dynamic integration of neural activity and neuromodulatory systems. <i>Nat. Neurosci.</i> <b>22</b>, 289–296 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41593-018-0312-0" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41593-018-0312-0" aria-label="Article reference 26" data-doi="10.1038/s41593-018-0312-0">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXmtF2jtbY%3D" aria-label="CAS reference 26">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30664771" aria-label="PubMed reference 26">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20cognition%20involves%20the%20dynamic%20integration%20of%20neural%20activity%20and%20neuromodulatory%20systems&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fs41593-018-0312-0&amp;volume=22&amp;pages=289-296&amp;publication_year=2019&amp;author=Shine%2CJM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Preti, M. G. &amp; van de Ville, D. Decoupling of brain function from structure reveals regional behavioral specialization in humans. <i>Nat. Commun.</i> <b>10</b>, 4747 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-019-12765-7" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-12765-7" aria-label="Article reference 27" data-doi="10.1038/s41467-019-12765-7">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVajtbnN" aria-label="CAS reference 27">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31628329" aria-label="PubMed reference 27">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6800438" aria-label="PubMed Central reference 27">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Decoupling%20of%20brain%20function%20from%20structure%20reveals%20regional%20behavioral%20specialization%20in%20humans&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-12765-7&amp;volume=10&amp;publication_year=2019&amp;author=Preti%2CMG&amp;author=Ville%2CD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Yarkoni, T., Poldrack, R. A., Nichols, T. E., Van Essen, D. C. &amp; Wager, T. D. Large-scale automated synthesis of human functional neuroimaging data. <i>Nat. Methods</i> <b>8</b>, 665–670 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nmeth.1635" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.1635" aria-label="Article reference 28" data-doi="10.1038/nmeth.1635">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXotV2gs7w%3D" aria-label="CAS reference 28">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21706013" aria-label="PubMed reference 28">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3146590" aria-label="PubMed Central reference 28">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20automated%20synthesis%20of%20human%20functional%20neuroimaging%20data&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.1635&amp;volume=8&amp;pages=665-670&amp;publication_year=2011&amp;author=Yarkoni%2CT&amp;author=Poldrack%2CRA&amp;author=Nichols%2CTE&amp;author=Essen%2CDC&amp;author=Wager%2CTD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Petersen, S. E. &amp; Sporns, O. Brain networks and cognitive architectures. <i>Neuron</i> <b>88</b>, 207–219 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2015.09.027" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2015.09.027" aria-label="Article reference 29" data-doi="10.1016/j.neuron.2015.09.027">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1ers7fL" aria-label="CAS reference 29">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26447582" aria-label="PubMed reference 29">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4598639" aria-label="PubMed Central reference 29">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Brain%20networks%20and%20cognitive%20architectures&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2015.09.027&amp;volume=88&amp;pages=207-219&amp;publication_year=2015&amp;author=Petersen%2CSE&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Bassett, D. S. &amp; Sporns, O. Network neuroscience. <i>Nat. Neurosci.</i> <b>20</b>, 353–364 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4502" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4502" aria-label="Article reference 30" data-doi="10.1038/nn.4502">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVSmsrY%3D" aria-label="CAS reference 30">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28230844" aria-label="PubMed reference 30">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5485642" aria-label="PubMed Central reference 30">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20neuroscience&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4502&amp;volume=20&amp;pages=353-364&amp;publication_year=2017&amp;author=Bassett%2CDS&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Shafiei, G. et al. Topographic gradients of intrinsic dynamics across neocortex. <i>eLife</i> <b>9</b>, 1–24 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.7554/eLife.62116" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.62116" aria-label="Article reference 31" data-doi="10.7554/eLife.62116">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Topographic%20gradients%20of%20intrinsic%20dynamics%20across%20neocortex&amp;journal=eLife&amp;doi=10.7554%2FeLife.62116&amp;volume=9&amp;pages=1-24&amp;publication_year=2020&amp;author=Shafiei%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Paquola, C. et al. A multi-scale cortical wiring space links cellular architecture and functional dynamics in the human brain. <i>PLoS Biol.</i> <b>18</b>, e3000979 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.3000979" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.3000979" aria-label="Article reference 32" data-doi="10.1371/journal.pbio.3000979">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1ensrrK" aria-label="CAS reference 32">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33253185" aria-label="PubMed reference 32">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7728398" aria-label="PubMed Central reference 32">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-scale%20cortical%20wiring%20space%20links%20cellular%20architecture%20and%20functional%20dynamics%20in%20the%20human%20brain&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.3000979&amp;volume=18&amp;publication_year=2020&amp;author=Paquola%2CC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Smaers, J. B., Gómez-Robles, A., Parks, A. N. &amp; Sherwood, C. C. Exceptional evolutionary expansion of prefrontal cortex in great apes and humans. <i>Curr. Biol.</i> <b>27</b>, 714–720 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cub.2017.01.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2017.01.020" aria-label="Article reference 33" data-doi="10.1016/j.cub.2017.01.020">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitFehtb4%3D" aria-label="CAS reference 33">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28162899" aria-label="PubMed reference 33">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Exceptional%20evolutionary%20expansion%20of%20prefrontal%20cortex%20in%20great%20apes%20and%20humans&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2017.01.020&amp;volume=27&amp;pages=714-720&amp;publication_year=2017&amp;author=Smaers%2CJB&amp;author=G%C3%B3mez-Robles%2CA&amp;author=Parks%2CAN&amp;author=Sherwood%2CCC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Wei, Y. et al. Genetic mapping and evolutionary analysis of human-expanded cognitive networks. <i>Nat. Commun.</i> <b>10</b>, 4839 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-019-12764-8" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-12764-8" aria-label="Article reference 34" data-doi="10.1038/s41467-019-12764-8">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVGmtLzJ" aria-label="CAS reference 34">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31649260" aria-label="PubMed reference 34">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6813316" aria-label="PubMed Central reference 34">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20mapping%20and%20evolutionary%20analysis%20of%20human-expanded%20cognitive%20networks&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-12764-8&amp;volume=10&amp;publication_year=2019&amp;author=Wei%2CY">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Whitaker, K. J. et al. Adolescence is associated with genomically patterned consolidation of the hubs of the human brain connectome. <i>Proc. Natl. Acad. Sci. USA</i> <b>113</b>, 9105–9110 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1601745113" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1601745113" aria-label="Article reference 35" data-doi="10.1073/pnas.1601745113">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1ehurbM" aria-label="CAS reference 35">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27457931" aria-label="PubMed reference 35">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987797" aria-label="PubMed Central reference 35">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Adolescence%20is%20associated%20with%20genomically%20patterned%20consolidation%20of%20the%20hubs%20of%20the%20human%20brain%20connectome&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1601745113&amp;volume=113&amp;pages=9105-9110&amp;publication_year=2016&amp;author=Whitaker%2CKJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Finnema, S. J. et al. Imaging synaptic density in the living human brain. <i>Sci. Transl. Med</i>. <b>8</b>, 348ra96 (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Holland, N. et al. Synaptic loss in primary tauopathies revealed by [<sup>11</sup>C]UCB-J positron emission tomography. <i>Mov. Disord.</i> <a href="https://doi.org/10.1002/mds.28188" data-track="click" data-track-action="external reference" data-track-label="10.1002/mds.28188">https://doi.org/10.1002/mds.28188</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Goyal, M. S., Hawrylycz, M., Miller, J. A., Snyder, A. Z. &amp; Raichle, M. E. Aerobic glycolysis in the human brain is associated with development and neotenous gene expression. <i>Cell Metabolism</i> <b>19</b>, 49–57 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cmet.2013.11.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cmet.2013.11.020" aria-label="Article reference 38" data-doi="10.1016/j.cmet.2013.11.020">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXksV2mtw%3D%3D" aria-label="CAS reference 38">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24411938" aria-label="PubMed reference 38">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4389678" aria-label="PubMed Central reference 38">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Aerobic%20glycolysis%20in%20the%20human%20brain%20is%20associated%20with%20development%20and%20neotenous%20gene%20expression&amp;journal=Cell%20Metabolism&amp;doi=10.1016%2Fj.cmet.2013.11.020&amp;volume=19&amp;pages=49-57&amp;publication_year=2014&amp;author=Goyal%2CMS&amp;author=Hawrylycz%2CM&amp;author=Miller%2CJA&amp;author=Snyder%2CAZ&amp;author=Raichle%2CME">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Vaishnavi, S. N. et al. Regional aerobic glycolysis in the human brain. <i>Proc. Natl. Acad. Sci. USA</i> <b>107</b>, 17757–17762 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1010459107" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1010459107" aria-label="Article reference 39" data-doi="10.1073/pnas.1010459107">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtlaltr%2FF" aria-label="CAS reference 39">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20837536" aria-label="PubMed reference 39">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2955101" aria-label="PubMed Central reference 39">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Regional%20aerobic%20glycolysis%20in%20the%20human%20brain&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1010459107&amp;volume=107&amp;pages=17757-17762&amp;publication_year=2010&amp;author=Vaishnavi%2CSN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Goulas, A. et al. The natural axis of transmitter receptor distribution in the human cerebral cortex. <i>Proc. Natl Acad. Sci. USA</i> <b>118</b>, e2020574118 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Zilles, K. &amp; Palomero-Gallagher, N. Multiple transmitter receptors in regions and layers of the human cerebral cortex. <i>Front. Neuroanat.</i> <b>11</b>, 78 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/fnana.2017.00078" data-track-action="article reference" href="https://doi.org/10.3389%2Ffnana.2017.00078" aria-label="Article reference 41" data-doi="10.3389/fnana.2017.00078">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXlsFOju70%3D" aria-label="CAS reference 41">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28970785" aria-label="PubMed reference 41">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5609104" aria-label="PubMed Central reference 41">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiple%20transmitter%20receptors%20in%20regions%20and%20layers%20of%20the%20human%20cerebral%20cortex&amp;journal=Front.%20Neuroanat.&amp;doi=10.3389%2Ffnana.2017.00078&amp;volume=11&amp;publication_year=2017&amp;author=Zilles%2CK&amp;author=Palomero-Gallagher%2CN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Goyal, M. S. &amp; Raichle, M. E. Gene expression-based modeling of human cortical synaptic density. <i>Proc. Natl. Acad. Sci. USA</i> <b>110</b>, 6571–6576 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1303453110" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1303453110" aria-label="Article reference 42" data-doi="10.1073/pnas.1303453110">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXnvVOlu7k%3D" aria-label="CAS reference 42">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23576754" aria-label="PubMed reference 42">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3631628" aria-label="PubMed Central reference 42">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20expression-based%20modeling%20of%20human%20cortical%20synaptic%20density&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1303453110&amp;volume=110&amp;pages=6571-6576&amp;publication_year=2013&amp;author=Goyal%2CMS&amp;author=Raichle%2CME">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Mashour, G. A., Roelfsema, P., Changeux, J. P. &amp; Dehaene, S. Conscious processing and the global neuronal workspace hypothesis. <i>Neuron</i> <b>105</b>, 776–798 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2020.01.026" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2020.01.026" aria-label="Article reference 43" data-doi="10.1016/j.neuron.2020.01.026">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXotFeltbg%3D" aria-label="CAS reference 43">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32135090" aria-label="PubMed reference 43">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8770991" aria-label="PubMed Central reference 43">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Conscious%20processing%20and%20the%20global%20neuronal%20workspace%20hypothesis&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2020.01.026&amp;volume=105&amp;pages=776-798&amp;publication_year=2020&amp;author=Mashour%2CGA&amp;author=Roelfsema%2CP&amp;author=Changeux%2CJP&amp;author=Dehaene%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Rosas, F. E., Mediano, P. A. M., Rassouli, B. &amp; Barrett, A. B. An operational information decomposition via synergistic disclosure. <i>J. Phys. A Math. Theor.</i> <b>53</b>, 485001 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1088/1751-8121/abb723" data-track-action="article reference" href="https://doi.org/10.1088%2F1751-8121%2Fabb723" aria-label="Article reference 44" data-doi="10.1088/1751-8121/abb723">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20operational%20information%20decomposition%20via%20synergistic%20disclosure&amp;journal=J.%20Phys.%20A%20Math.%20Theor.&amp;doi=10.1088%2F1751-8121%2Fabb723&amp;volume=53&amp;publication_year=2020&amp;author=Rosas%2CFE&amp;author=Mediano%2CPAM&amp;author=Rassouli%2CB&amp;author=Barrett%2CAB">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Buckner, R. L. &amp; Krienen, F. M. The evolution of distributed association networks in the human brain. <i>Trends Cogn. Sci.</i> <b>17</b>, 648–665 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.tics.2013.09.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.tics.2013.09.017" aria-label="Article reference 45" data-doi="10.1016/j.tics.2013.09.017">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24210963" aria-label="PubMed reference 45">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evolution%20of%20distributed%20association%20networks%20in%20the%20human%20brain&amp;journal=Trends%20Cogn.%20Sci.&amp;doi=10.1016%2Fj.tics.2013.09.017&amp;volume=17&amp;pages=648-665&amp;publication_year=2013&amp;author=Buckner%2CRL&amp;author=Krienen%2CFM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Yeshurun, Y., Nguyen, M. &amp; Hasson, U. The default mode network: where the idiosyncratic self meets the shared social world. <i>Nat. Rev. Neurosci.</i> <b>22</b>, 181–192 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41583-020-00420-w" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41583-020-00420-w" aria-label="Article reference 46" data-doi="10.1038/s41583-020-00420-w">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvF2nt7k%3D" aria-label="CAS reference 46">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33483717" aria-label="PubMed reference 46">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7959111" aria-label="PubMed Central reference 46">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20default%20mode%20network%3A%20where%20the%20idiosyncratic%20self%20meets%20the%20shared%20social%20world&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fs41583-020-00420-w&amp;volume=22&amp;pages=181-192&amp;publication_year=2021&amp;author=Yeshurun%2CY&amp;author=Nguyen%2CM&amp;author=Hasson%2CU">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Jones, E. G. &amp; Powell, T. P. S. An anatomical study of converging sensory pathways within the cerebral cortex of the monkey. <i>Brain</i> <b>93</b>, 793–820 (1970).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/brain/93.4.793" data-track-action="article reference" href="https://doi.org/10.1093%2Fbrain%2F93.4.793" aria-label="Article reference 47" data-doi="10.1093/brain/93.4.793">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaE3M%2FmsFWnug%3D%3D" aria-label="CAS reference 47">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=4992433" aria-label="PubMed reference 47">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20anatomical%20study%20of%20converging%20sensory%20pathways%20within%20the%20cerebral%20cortex%20of%20the%20monkey&amp;journal=Brain&amp;doi=10.1093%2Fbrain%2F93.4.793&amp;volume=93&amp;pages=793-820&amp;publication_year=1970&amp;author=Jones%2CEG&amp;author=Powell%2CTPS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Fox, M. D. et al. The human brain is intrinsically organized into dynamic, anticorrelated functional networks. <i>Proc. Natl. Acad. Sci. USA</i> <b>102</b>, 9673–9678 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0504136102" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0504136102" aria-label="Article reference 48" data-doi="10.1073/pnas.0504136102">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXmsVaktb8%3D" aria-label="CAS reference 48">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15976020" aria-label="PubMed reference 48">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1157105" aria-label="PubMed Central reference 48">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20brain%20is%20intrinsically%20organized%20into%20dynamic%2C%20anticorrelated%20functional%20networks&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0504136102&amp;volume=102&amp;pages=9673-9678&amp;publication_year=2005&amp;author=Fox%2CMD">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Raichle, M. E. et al. A default mode of brain function. <i>Proc. Natl. Acad. Sci. USA</i> <b>98</b>, 676–682 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.98.2.676" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.98.2.676" aria-label="Article reference 49" data-doi="10.1073/pnas.98.2.676">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXnslKmtw%3D%3D" aria-label="CAS reference 49">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11209064" aria-label="PubMed reference 49">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC14647" aria-label="PubMed Central reference 49">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20default%20mode%20of%20brain%20function&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.98.2.676&amp;volume=98&amp;pages=676-682&amp;publication_year=2001&amp;author=Raichle%2CME">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Vatansever, D., Menon, X. D. K., Manktelow, A. E., Sahakian, B. J. &amp; Stamatakis, E. A. Default mode dynamics for global functional integration. <i>J. Neurosci.</i> <b>35</b>, 15254–15262 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2135-15.2015" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2135-15.2015" aria-label="Article reference 50" data-doi="10.1523/JNEUROSCI.2135-15.2015">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XmtlGhtrc%3D" aria-label="CAS reference 50">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26586814" aria-label="PubMed reference 50">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4649001" aria-label="PubMed Central reference 50">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Default%20mode%20dynamics%20for%20global%20functional%20integration&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2135-15.2015&amp;volume=35&amp;pages=15254-15262&amp;publication_year=2015&amp;author=Vatansever%2CD&amp;author=Menon%2CXDK&amp;author=Manktelow%2CAE&amp;author=Sahakian%2CBJ&amp;author=Stamatakis%2CEA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Gu, S. et al. Controllability of structural brain networks. <i>Nat. Commun.</i> <b>6</b>, 8414 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/ncomms9414" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms9414" aria-label="Article reference 51" data-doi="10.1038/ncomms9414">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsF2kt77O" aria-label="CAS reference 51">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26423222" aria-label="PubMed reference 51">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Controllability%20of%20structural%20brain%20networks&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms9414&amp;volume=6&amp;publication_year=2015&amp;author=Gu%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Cole, M. W. et al. Multi-task connectivity reveals flexible hubs for adaptive task control. <i>Nat. Neurosci.</i> <b>16</b>, 1348–1355 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.3470" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.3470" aria-label="Article reference 52" data-doi="10.1038/nn.3470">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOhs7bJ" aria-label="CAS reference 52">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23892552" aria-label="PubMed reference 52">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3758404" aria-label="PubMed Central reference 52">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-task%20connectivity%20reveals%20flexible%20hubs%20for%20adaptive%20task%20control&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.3470&amp;volume=16&amp;pages=1348-1355&amp;publication_year=2013&amp;author=Cole%2CMW">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Alexander-Bloch, A. F. et al. On testing for spatial correspondence between maps of human brain structure and function. <i>Neuroimage</i> <b>178</b>, 540–551 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2018.05.070" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2018.05.070" aria-label="Article reference 53" data-doi="10.1016/j.neuroimage.2018.05.070">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29860082" aria-label="PubMed reference 53">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20testing%20for%20spatial%20correspondence%20between%20maps%20of%20human%20brain%20structure%20and%20function&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2018.05.070&amp;volume=178&amp;pages=540-551&amp;publication_year=2018&amp;author=Alexander-Bloch%2CAF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">van Essen, D. C. et al. The WU-minn Human Connectome Project: an overview. <i>Neuroimage</i> <b>80</b>, 62–79 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.041" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.041" aria-label="Article reference 54" data-doi="10.1016/j.neuroimage.2013.05.041">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23684880" aria-label="PubMed reference 54">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20WU-minn%20Human%20Connectome%20Project%3A%20an%20overview&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.041&amp;volume=80&amp;pages=62-79&amp;publication_year=2013&amp;author=Essen%2CDC">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Luppi, A. I. &amp; Stamatakis, E. A. Combining network topology and information theory to construct representative brain networks. <i>Network Neuroscience</i> <b>5</b>, 96–124 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1162/netn_a_00170" data-track-action="article reference" href="https://doi.org/10.1162%2Fnetn_a_00170" aria-label="Article reference 55" data-doi="10.1162/netn_a_00170">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33688608" aria-label="PubMed reference 55">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7935031" aria-label="PubMed Central reference 55">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Combining%20network%20topology%20and%20information%20theory%20to%20construct%20representative%20brain%20networks&amp;journal=Network%20Neuroscience&amp;doi=10.1162%2Fnetn_a_00170&amp;volume=5&amp;pages=96-124&amp;publication_year=2021&amp;author=Luppi%2CAI&amp;author=Stamatakis%2CEA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Amico, E. et al. Toward an information theoretical description of communication in brain networks. <i>Network Neuroscience</i> <b>5</b>, 646–665 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34746621" aria-label="PubMed reference 56">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8567835" aria-label="PubMed Central reference 56">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20an%20information%20theoretical%20description%20of%20communication%20in%20brain%20networks&amp;journal=Network%20Neuroscience&amp;volume=5&amp;pages=646-665&amp;publication_year=2021&amp;author=Amico%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Glasser, M. F. et al. The minimal preprocessing pipelines for the Human Connectome Project. <i>Neuroimage</i> <b>80</b>, 105–124 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.04.127" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.04.127" aria-label="Article reference 57" data-doi="10.1016/j.neuroimage.2013.04.127">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23668970" aria-label="PubMed reference 57">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20minimal%20preprocessing%20pipelines%20for%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.04.127&amp;volume=80&amp;pages=105-124&amp;publication_year=2013&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Whitfield-Gabrieli, S. &amp; Nieto-Castanon, A. Conn: a functional connectivity toolbox for correlated and anticorrelated brain networks. <i>Brain Connect.</i> <b>2</b>, 125–141 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1089/brain.2012.0073" data-track-action="article reference" href="https://doi.org/10.1089%2Fbrain.2012.0073" aria-label="Article reference 58" data-doi="10.1089/brain.2012.0073">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22642651" aria-label="PubMed reference 58">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Conn%3A%20a%20functional%20connectivity%20toolbox%20for%20correlated%20and%20anticorrelated%20brain%20networks&amp;journal=Brain%20Connect.&amp;doi=10.1089%2Fbrain.2012.0073&amp;volume=2&amp;pages=125-141&amp;publication_year=2012&amp;author=Whitfield-Gabrieli%2CS&amp;author=Nieto-Castanon%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Behzadi, Y., Restom, K., Liau, J. &amp; Liu, T. T. A component based noise correction method (CompCor) for BOLD and perfusion based fMRI. <i>Neuroimage</i> <b>37</b>, 90–101 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2007.04.042" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2007.04.042" aria-label="Article reference 59" data-doi="10.1016/j.neuroimage.2007.04.042">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17560126" aria-label="PubMed reference 59">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20component%20based%20noise%20correction%20method%20%28CompCor%29%20for%20BOLD%20and%20perfusion%20based%20fMRI&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2007.04.042&amp;volume=37&amp;pages=90-101&amp;publication_year=2007&amp;author=Behzadi%2CY&amp;author=Restom%2CK&amp;author=Liau%2CJ&amp;author=Liu%2CTT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Wu, G.-R. et al. A blind deconvolution approach to recover effective connectivity brain networks from resting state fMRI data. <i>Med. Image Anal.</i> <b>17</b>, 365–374 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.media.2013.01.003" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.media.2013.01.003" aria-label="Article reference 60" data-doi="10.1016/j.media.2013.01.003">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23422254" aria-label="PubMed reference 60">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20blind%20deconvolution%20approach%20to%20recover%20effective%20connectivity%20brain%20networks%20from%20resting%20state%20fMRI%20data&amp;journal=Med.%20Image%20Anal.&amp;doi=10.1016%2Fj.media.2013.01.003&amp;volume=17&amp;pages=365-374&amp;publication_year=2013&amp;author=Wu%2CG-R">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Yeh, F. -C., Wedeen, V. J. &amp; Tseng, W. -Y. I. Estimation of fiber orientation and spin density distribution by diffusion deconvolution. <i>Neuroimage</i> <b>55</b>, 1054–1062 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2010.11.087" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2010.11.087" aria-label="Article reference 61" data-doi="10.1016/j.neuroimage.2010.11.087">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21232611" aria-label="PubMed reference 61">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimation%20of%20fiber%20orientation%20and%20spin%20density%20distribution%20by%20diffusion%20deconvolution&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2010.11.087&amp;volume=55&amp;pages=1054-1062&amp;publication_year=2011&amp;author=Yeh%2CF-C&amp;author=Wedeen%2CVJ&amp;author=Tseng%2CW-YI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Yeh, F.-C., Verstynen, T. D., Wang, Y., Fernández-Miranda, J. C. &amp; Tseng, W.-Y. Deterministic diffusion fiber tracking improved by quantitative anisotropy. <i>PLoS ONE</i> <b>8</b>, 80713 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0080713" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0080713" aria-label="Article reference 62" data-doi="10.1371/journal.pone.0080713">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsVCqtbs%3D" aria-label="CAS reference 62">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Deterministic%20diffusion%20fiber%20tracking%20improved%20by%20quantitative%20anisotropy&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0080713&amp;volume=8&amp;publication_year=2013&amp;author=Yeh%2CF-C&amp;author=Verstynen%2CTD&amp;author=Wang%2CY&amp;author=Fern%C3%A1ndez-Miranda%2CJC&amp;author=Tseng%2CW-Y">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Michael Milham, A. P. et al. An open resource for nonhuman primate imaging. <i>Neuron</i> <b>100</b>, 61–74 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2018.08.039" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2018.08.039" aria-label="Article reference 63" data-doi="10.1016/j.neuron.2018.08.039">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVahsb7L" aria-label="CAS reference 63">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30269990" aria-label="PubMed reference 63">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6231397" aria-label="PubMed Central reference 63">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20open%20resource%20for%20nonhuman%20primate%20imaging&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2018.08.039&amp;volume=100&amp;pages=61-74&amp;publication_year=2018&amp;author=Michael%20Milham%2CAP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Tasserie, J. et al. Pypreclin: an automatic pipeline for macaque functional MRI preprocessing. <i>Neuroimage</i> <b>207</b>, 116353 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2019.116353" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2019.116353" aria-label="Article reference 64" data-doi="10.1016/j.neuroimage.2019.116353">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31743789" aria-label="PubMed reference 64">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Pypreclin%3A%20an%20automatic%20pipeline%20for%20macaque%20functional%20MRI%20preprocessing&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2019.116353&amp;volume=207&amp;publication_year=2020&amp;author=Tasserie%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Seidlitz, J. et al. A population MRI brain template and analysis tools for the macaque. <i>Neuroimage.</i> <b>170</b>, 121–131 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.04.063" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.04.063" aria-label="Article reference 65" data-doi="10.1016/j.neuroimage.2017.04.063">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28461058" aria-label="PubMed reference 65">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20population%20MRI%20brain%20template%20and%20analysis%20tools%20for%20the%20macaque&amp;journal=Neuroimage.&amp;doi=10.1016%2Fj.neuroimage.2017.04.063&amp;volume=170&amp;pages=121-131&amp;publication_year=2018&amp;author=Seidlitz%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Barttfeld, P. et al. Signature of consciousness in the dynamics of resting-state brain activity. <i>Proc. Natl. Acad. Sci. USA</i> <b>112</b>, 887–892 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1418031112" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1418031112" aria-label="Article reference 66" data-doi="10.1073/pnas.1418031112">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXjtVWnsA%3D%3D" aria-label="CAS reference 66">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25561541" aria-label="PubMed reference 66">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4311826" aria-label="PubMed Central reference 66">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Signature%20of%20consciousness%20in%20the%20dynamics%20of%20resting-state%20brain%20activity&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1418031112&amp;volume=112&amp;pages=887-892&amp;publication_year=2015&amp;author=Barttfeld%2CP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Schaefer, A. et al. Local–global parcellation of the human cerebral cortex from intrinsic functional connectivity MRI. <i>Cereb. Cortex</i> <b>28</b>, 3095–3114 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhx179" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhx179" aria-label="Article reference 67" data-doi="10.1093/cercor/bhx179">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28981612" aria-label="PubMed reference 67">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Local%E2%80%93global%20parcellation%20of%20the%20human%20cerebral%20cortex%20from%20intrinsic%20functional%20connectivity%20MRI&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhx179&amp;volume=28&amp;pages=3095-3114&amp;publication_year=2018&amp;author=Schaefer%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Tian, Y., Margulies, D., Breakspear, M. &amp; Zalesky, A. Topographic organization of the human subcortex unveiled with functional connectivity gradients. <i>Nat. Neurosci.</i> <b>23</b>, 1421–1432 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41593-020-00711-6" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41593-020-00711-6" aria-label="Article reference 68" data-doi="10.1038/s41593-020-00711-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFegurjP" aria-label="CAS reference 68">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32989295" aria-label="PubMed reference 68">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Topographic%20organization%20of%20the%20human%20subcortex%20unveiled%20with%20functional%20connectivity%20gradients&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fs41593-020-00711-6&amp;volume=23&amp;pages=1421-1432&amp;publication_year=2020&amp;author=Tian%2CY&amp;author=Margulies%2CD&amp;author=Breakspear%2CM&amp;author=Zalesky%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Desikan, R. S. et al. An automated labeling system for subdividing the human cerebral cortex on MRI scans into gyral based regions of interest. <i>Neuroimage</i> <b>31</b>, 968–980 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2006.01.021" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2006.01.021" aria-label="Article reference 69" data-doi="10.1016/j.neuroimage.2006.01.021">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16530430" aria-label="PubMed reference 69">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20automated%20labeling%20system%20for%20subdividing%20the%20human%20cerebral%20cortex%20on%20MRI%20scans%20into%20gyral%20based%20regions%20of%20interest&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2006.01.021&amp;volume=31&amp;pages=968-980&amp;publication_year=2006&amp;author=Desikan%2CRS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Romero-Garcia, R., Atienza, M., Clemmensen, L. H. &amp; Cantero, J. L. Effects of network resolution on topological properties of human neocortex. <i>Neuroimage</i> <b>59</b>, 3522–3532 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.10.086" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.10.086" aria-label="Article reference 70" data-doi="10.1016/j.neuroimage.2011.10.086">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22094643" aria-label="PubMed reference 70">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20network%20resolution%20on%20topological%20properties%20of%20human%20neocortex&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2011.10.086&amp;volume=59&amp;pages=3522-3532&amp;publication_year=2012&amp;author=Romero-Garcia%2CR&amp;author=Atienza%2CM&amp;author=Clemmensen%2CLH&amp;author=Cantero%2CJL">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. <i>Nature</i> <b>536</b>, 171–178 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature18933" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature18933" aria-label="Article reference 71" data-doi="10.1038/nature18933">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1GrurnK" aria-label="CAS reference 71">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27437579" aria-label="PubMed reference 71">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990127" aria-label="PubMed Central reference 71">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-modal%20parcellation%20of%20human%20cerebral%20cortex&amp;journal=Nature&amp;doi=10.1038%2Fnature18933&amp;volume=536&amp;pages=171-178&amp;publication_year=2016&amp;author=Glasser%2CMF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Kötter, R. &amp; Wanke, E. Mapping brains without coordinates. <i>Philos. Trans. R Soc. B Biol. Sci.</i> <b>360</b>, 751–766 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rstb.2005.1625" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2005.1625" aria-label="Article reference 72" data-doi="10.1098/rstb.2005.1625">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20brains%20without%20coordinates&amp;journal=Philos.%20Trans.%20R%20Soc.%20B%20Biol.%20Sci.&amp;doi=10.1098%2Frstb.2005.1625&amp;volume=360&amp;pages=751-766&amp;publication_year=2005&amp;author=K%C3%B6tter%2CR&amp;author=Wanke%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Schulz, M. A. et al. Different scaling of linear models and deep learning in UKBiobank brain images versus machine-learning datasets. <i>Nat. Commun.</i> <b>11</b>, (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Nozari, E. et al. Is the brain macroscopically linear? A system identification of resting state dynamics. Preprint at <i>arXiv</i> <a href="https://doi.org/10.48550/arXiv.2012.12351" data-track="click" data-track-action="external reference" data-track-label="10.48550/arXiv.2012.12351">https://doi.org/10.48550/arXiv.2012.12351</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Bím, J. et al. A nonnegative measure of feature-related information transfer between neural signals. Preprint at <i>bioRxiv</i>, <a href="https://doi.org/10.1101/758128" data-track="click" data-track-action="external reference" data-track-label="10.1101/758128">https://doi.org/10.1101/758128</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Lizier, J., Bertschinger, N., Jost, J. &amp; Wibral, M. Information decomposition of target effects from multi-source interactions: perspectives on previous, current and future work. <i>Entropy</i> <b>20</b>, 307 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3390/e20040307" data-track-action="article reference" href="https://doi.org/10.3390%2Fe20040307" aria-label="Article reference 76" data-doi="10.3390/e20040307">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7512824" aria-label="PubMed Central reference 76">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Information%20decomposition%20of%20target%20effects%20from%20multi-source%20interactions%3A%20perspectives%20on%20previous%2C%20current%20and%20future%20work&amp;journal=Entropy&amp;doi=10.3390%2Fe20040307&amp;volume=20&amp;publication_year=2018&amp;author=Lizier%2CJ&amp;author=Bertschinger%2CN&amp;author=Jost%2CJ&amp;author=Wibral%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Barrett, A. B. &amp; Seth, A. K. Practical measures of integrated information for time-series data. <i>PLoS Comput. Biol.</i> <b>7</b>, 1001052 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1001052" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1001052" aria-label="Article reference 77" data-doi="10.1371/journal.pcbi.1001052">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtlyjs7w%3D" aria-label="CAS reference 77">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Practical%20measures%20of%20integrated%20information%20for%20time-series%20data&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1001052&amp;volume=7&amp;publication_year=2011&amp;author=Barrett%2CAB&amp;author=Seth%2CAK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Bastos, A. M. et al. Visual areas exert feedforward and feedback influences through distinct frequency channels. <i>Neuron</i> <b>85</b>, 390–401 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2014.12.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2014.12.018" aria-label="Article reference 78" data-doi="10.1016/j.neuron.2014.12.018">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXmvFaiuw%3D%3D" aria-label="CAS reference 78">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25556836" aria-label="PubMed reference 78">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Visual%20areas%20exert%20feedforward%20and%20feedback%20influences%20through%20distinct%20frequency%20channels&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2014.12.018&amp;volume=85&amp;pages=390-401&amp;publication_year=2015&amp;author=Bastos%2CAM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Deco, G., Vidaurre, D. &amp; Kringelbach, M. L. Revisiting the global workspace: orchestration of the functional hierarchical organisation of the human brain. <i>Nat. Hum. Behav.</i> <a href="https://doi.org/10.1101/859579" data-track="click" data-track-action="external reference" data-track-label="10.1101/859579">https://doi.org/10.1101/859579</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Lizier, J. T. JIDT: an information-theoretic toolkit for studying the dynamics of complex systems. <i>Front. Robot. AI</i> <b>1</b>, 1–37 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3389/frobt.2014.00011" data-track-action="article reference" href="https://doi.org/10.3389%2Ffrobt.2014.00011" aria-label="Article reference 80" data-doi="10.3389/frobt.2014.00011">Article</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=JIDT%3A%20an%20information-theoretic%20toolkit%20for%20studying%20the%20dynamics%20of%20complex%20systems&amp;journal=Front.%20Robot.%20AI&amp;doi=10.3389%2Ffrobt.2014.00011&amp;volume=1&amp;pages=1-37&amp;publication_year=2014&amp;author=Lizier%2CJT">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Battiston, F. et al. The physics of higher-order interactions in complex systems. <i>Nat. Phys.</i> <b>17</b>, 1093–1098 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41567-021-01371-4" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41567-021-01371-4" aria-label="Article reference 81" data-doi="10.1038/s41567-021-01371-4">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitF2nsL3I" aria-label="CAS reference 81">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20physics%20of%20higher-order%20interactions%20in%20complex%20systems&amp;journal=Nat.%20Phys.&amp;doi=10.1038%2Fs41567-021-01371-4&amp;volume=17&amp;pages=1093-1098&amp;publication_year=2021&amp;author=Battiston%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Rosas, F. E. et al. Disentangling high-order mechanisms and high-order behaviours in complex systems. <i>Nat. Physics</i> <a href="https://doi.org/10.1038/s41567-022-01548-5" data-track="click" data-track-action="external reference" data-track-label="10.1038/s41567-022-01548-5">https://doi.org/10.1038/s41567-022-01548-5</a> (2022).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Vértes, P. E. et al. Gene transcription profiles associated with inter-modular hubs and connection distance in human functional magnetic resonance imaging networks. <i>Philos. Trans. R. Soc. Lond. B Biol. Sci.</i> <b>371</b>, 20150362 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rstb.2015.0362" data-track-action="article reference" href="https://doi.org/10.1098%2Frstb.2015.0362" aria-label="Article reference 83" data-doi="10.1098/rstb.2015.0362">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFCgs7fL" aria-label="CAS reference 83">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27574314" aria-label="PubMed reference 83">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003862" aria-label="PubMed Central reference 83">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20transcription%20profiles%20associated%20with%20inter-modular%20hubs%20and%20connection%20distance%20in%20human%20functional%20magnetic%20resonance%20imaging%20networks&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;doi=10.1098%2Frstb.2015.0362&amp;volume=371&amp;publication_year=2016&amp;author=V%C3%A9rtes%2CPE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: Uses and interpretations. <i>NeuroImage</i> <b>52</b>, 1059–1069 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2009.10.003" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2009.10.003" aria-label="Article reference 84" data-doi="10.1016/j.neuroimage.2009.10.003">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19819337" aria-label="PubMed reference 84">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Complex%20network%20measures%20of%20brain%20connectivity%3A%20Uses%20and%20interpretations&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2009.10.003&amp;volume=52&amp;pages=1059-1069&amp;publication_year=2010&amp;author=Rubinov%2CM&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Rubinov, M. &amp; Sporns, O. Weight-conserving characterization of complex functional brain networks. <i>NeuroImage</i> <b>56</b>, 2068–2079 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2011.03.069" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2011.03.069" aria-label="Article reference 85" data-doi="10.1016/j.neuroimage.2011.03.069">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21459148" aria-label="PubMed reference 85">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Weight-conserving%20characterization%20of%20complex%20functional%20brain%20networks&amp;journal=NeuroImage&amp;doi=10.1016%2Fj.neuroimage.2011.03.069&amp;volume=56&amp;pages=2068-2079&amp;publication_year=2011&amp;author=Rubinov%2CM&amp;author=Sporns%2CO">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Cruzat, J. et al. The dynamics of human cognition: increasing global integration coupled with decreasing segregation found using iEEG. <i>Neuroimage</i> <b>172</b>, 492–505 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2018.01.064" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2018.01.064" aria-label="Article reference 86" data-doi="10.1016/j.neuroimage.2018.01.064">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29425897" aria-label="PubMed reference 86">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20dynamics%20of%20human%20cognition%3A%20increasing%20global%20integration%20coupled%20with%20decreasing%20segregation%20found%20using%20iEEG&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2018.01.064&amp;volume=172&amp;pages=492-505&amp;publication_year=2018&amp;author=Cruzat%2CJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Newman, M. E. J. Modularity and community structure in networks. <i>Proc. Natl. Acad. Sci. USA</i> <b>103</b>, 8577–8582 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0601602103" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0601602103" aria-label="Article reference 87" data-doi="10.1073/pnas.0601602103">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XlvVCitLw%3D" aria-label="CAS reference 87">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16723398" aria-label="PubMed reference 87">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1482622" aria-label="PubMed Central reference 87">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=Modularity%20and%20community%20structure%20in%20networks&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0601602103&amp;volume=103&amp;pages=8577-8582&amp;publication_year=2006&amp;author=Newman%2CMEJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Luppi, A. I. et al. LSD alters dynamic integration and segregation in the human brain. <i>Neuroimage</i> <b>227</b>, 117653 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2020.117653" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2020.117653" aria-label="Article reference 88" data-doi="10.1016/j.neuroimage.2020.117653">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33338615" aria-label="PubMed reference 88">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=LSD%20alters%20dynamic%20integration%20and%20segregation%20in%20the%20human%20brain&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2020.117653&amp;volume=227&amp;publication_year=2021&amp;author=Luppi%2CAI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Tagliazucchi, E. et al. Large-scale signatures of unconsciousness are consistent with a departure from critical dynamics. <i>J. R. Soc. Interface</i> <b>13</b>, 20151027 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1098/rsif.2015.1027" data-track-action="article reference" href="https://doi.org/10.1098%2Frsif.2015.1027" aria-label="Article reference 89" data-doi="10.1098/rsif.2015.1027">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26819336" aria-label="PubMed reference 89">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4759808" aria-label="PubMed Central reference 89">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20signatures%20of%20unconsciousness%20are%20consistent%20with%20a%20departure%20from%20critical%20dynamics&amp;journal=J.%20R.%20Soc.%20Interface&amp;doi=10.1098%2Frsif.2015.1027&amp;volume=13&amp;publication_year=2016&amp;author=Tagliazucchi%2CE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Shen, K. et al. A macaque connectome for large-scale network simulations in TheVirtualBrain. <i>Sci. Data</i> <b>6</b>, 123 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41597-019-0129-z" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41597-019-0129-z" aria-label="Article reference 90" data-doi="10.1038/s41597-019-0129-z">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31316116" aria-label="PubMed reference 90">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6637142" aria-label="PubMed Central reference 90">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20macaque%20connectome%20for%20large-scale%20network%20simulations%20in%20TheVirtualBrain&amp;journal=Sci.%20Data&amp;doi=10.1038%2Fs41597-019-0129-z&amp;volume=6&amp;publication_year=2019&amp;author=Shen%2CK">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Shen, K., Bezgin, G., Everling, S. &amp; McIntosh, A. R. The virtual macaque brain: a macaque connectome for large-scale network simulations in TheVirtualBrain. <a href="https://doi.org/10.5281/zenodo.1471588" data-track="click" data-track-action="external reference" data-track-label="10.5281/zenodo.1471588">https://doi.org/10.5281/zenodo.1471588</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Cammoun, Letal Mapping the human connectome at multiple scales with diffusion spectrum MRI. <i>J. Neurosci. Methods</i> <b>203</b>, 386–397 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.jneumeth.2011.09.031" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jneumeth.2011.09.031" aria-label="Article reference 92" data-doi="10.1016/j.jneumeth.2011.09.031">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22001222" aria-label="PubMed reference 92">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=Mapping%20the%20human%20connectome%20at%20multiple%20scales%20with%20diffusion%20spectrum%20MRI&amp;journal=J.%20Neurosci.%20Methods&amp;doi=10.1016%2Fj.jneumeth.2011.09.031&amp;volume=203&amp;pages=386-397&amp;publication_year=2012&amp;author=Cammoun%2CLetal">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Deco, G. et al. Whole-brain multimodal neuroimaging model using serotonin receptor maps explains non-linear functional effects of LSD. <i>Curr. Biol.</i> <b>28</b>, 3065–3074 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cub.2018.07.083" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2018.07.083" aria-label="Article reference 93" data-doi="10.1016/j.cub.2018.07.083">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVahsb%2FJ" aria-label="CAS reference 93">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30270185" aria-label="PubMed reference 93">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-brain%20multimodal%20neuroimaging%20model%20using%20serotonin%20receptor%20maps%20explains%20non-linear%20functional%20effects%20of%20LSD&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2018.07.083&amp;volume=28&amp;pages=3065-3074&amp;publication_year=2018&amp;author=Deco%2CG">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Herzog, R. et al. A mechanistic model of the neural entropy increase elicited by psychedelic drugs. <i>Sci. Rep.</i> <b>10</b>, 17725 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41598-020-74060-6" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-020-74060-6" aria-label="Article reference 94" data-doi="10.1038/s41598-020-74060-6">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitF2jsLjL" aria-label="CAS reference 94">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33082424" aria-label="PubMed reference 94">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7575594" aria-label="PubMed Central reference 94">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20mechanistic%20model%20of%20the%20neural%20entropy%20increase%20elicited%20by%20psychedelic%20drugs&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-020-74060-6&amp;volume=10&amp;publication_year=2020&amp;author=Herzog%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Luppi, A. I. et al. Whole-brain modelling identifies distinct but convergent paths to unconsciousness in anaesthesia and disorders of consciousness. <i>Commun. Biol.</i> <b>5</b>, 384 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s42003-022-03330-y" data-track-action="article reference" href="https://doi.org/10.1038%2Fs42003-022-03330-y" aria-label="Article reference 95" data-doi="10.1038/s42003-022-03330-y">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35444252" aria-label="PubMed reference 95">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9021270" aria-label="PubMed Central reference 95">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-brain%20modelling%20identifies%20distinct%20but%20convergent%20paths%20to%20unconsciousness%20in%20anaesthesia%20and%20disorders%20of%20consciousness&amp;journal=Commun.%20Biol.&amp;doi=10.1038%2Fs42003-022-03330-y&amp;volume=5&amp;publication_year=2022&amp;author=Luppi%2CAI">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Mediano, P. A. M., Luppi, A. I., Herzog, R. &amp; Rosas, F. E. FastDMF: fast simulator of the dynamic mean-field model of brain dynamics. <a href="https://doi.org/10.5281/zenodo.6373512" data-track="click" data-track-action="external reference" data-track-label="10.5281/zenodo.6373512">https://doi.org/10.5281/zenodo.6373512</a> (2022).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Wang, P. et al. Inversion of a large-scale circuit model reveals a cortical hierarchy in the dynamic resting human brain. <i>Sci. Adv.</i> <b>5</b>, 1–12 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 97" href="http://scholar.google.com/scholar_lookup?&amp;title=Inversion%20of%20a%20large-scale%20circuit%20model%20reveals%20a%20cortical%20hierarchy%20in%20the%20dynamic%20resting%20human%20brain&amp;journal=Sci.%20Adv.&amp;volume=5&amp;pages=1-12&amp;publication_year=2019&amp;author=Wang%2CP">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Doan, R. N. et al. Mutations in human accelerated regions disrupt cognition and social behavior the homozygosity mapping consortium for autism HHS public access. <i>Cell</i> <b>167</b>, 341–354 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.cell.2016.08.071" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2016.08.071" aria-label="Article reference 98" data-doi="10.1016/j.cell.2016.08.071">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGrurbP" aria-label="CAS reference 98">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27667684" aria-label="PubMed reference 98">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5063026" aria-label="PubMed Central reference 98">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutations%20in%20human%20accelerated%20regions%20disrupt%20cognition%20and%20social%20behavior%20the%20homozygosity%20mapping%20consortium%20for%20autism%20HHS%20public%20access&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.08.071&amp;volume=167&amp;pages=341-354&amp;publication_year=2016&amp;author=Doan%2CRN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Hawrylycz, M. J. et al. An anatomically comprehensive atlas of the adult human brain transcriptome. <i>Nature</i> <b>489</b>, 391–399 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature11405" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature11405" aria-label="Article reference 99" data-doi="10.1038/nature11405">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xhtlymu77N" aria-label="CAS reference 99">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22996553" aria-label="PubMed reference 99">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4243026" aria-label="PubMed Central reference 99">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 99" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20anatomically%20comprehensive%20atlas%20of%20the%20adult%20human%20brain%20transcriptome&amp;journal=Nature&amp;doi=10.1038%2Fnature11405&amp;volume=489&amp;pages=391-399&amp;publication_year=2012&amp;author=Hawrylycz%2CMJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100."><p class="c-article-references__text" id="ref-CR100">Kang, H. J. et al. Spatio-temporal transcriptome of the human brain. <i>Nature</i> <b>478</b>, 483–489 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature10523" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature10523" aria-label="Article reference 100" data-doi="10.1038/nature10523">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtlOnur%2FL" aria-label="CAS reference 100">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22031440" aria-label="PubMed reference 100">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3566780" aria-label="PubMed Central reference 100">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 100" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatio-temporal%20transcriptome%20of%20the%20human%20brain&amp;journal=Nature&amp;doi=10.1038%2Fnature10523&amp;volume=478&amp;pages=483-489&amp;publication_year=2011&amp;author=Kang%2CHJ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101."><p class="c-article-references__text" id="ref-CR101">Romero-Garcia, R. et al. Structural covariance networks are coupled to expression of genes enriched in supragranular layers of the human cortex. <i>Neuroimage</i> <b>171</b>, 256–267 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.12.060" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.12.060" aria-label="Article reference 101" data-doi="10.1016/j.neuroimage.2017.12.060">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29274746" aria-label="PubMed reference 101">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 101" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20covariance%20networks%20are%20coupled%20to%20expression%20of%20genes%20enriched%20in%20supragranular%20layers%20of%20the%20human%20cortex&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2017.12.060&amp;volume=171&amp;pages=256-267&amp;publication_year=2018&amp;author=Romero-Garcia%2CR">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102."><p class="c-article-references__text" id="ref-CR102">Krishnan, A., Williams, L. J., McIntosh, A. R. &amp; Abdi, H. Partial least-squares methods for neuroimaging: a tutorial and review. <i>Neuroimage</i> <b>56</b>, 455–475 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2010.07.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2010.07.034" aria-label="Article reference 102" data-doi="10.1016/j.neuroimage.2010.07.034">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20656037" aria-label="PubMed reference 102">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 102" href="http://scholar.google.com/scholar_lookup?&amp;title=Partial%20least-squares%20methods%20for%20neuroimaging%3A%20a%20tutorial%20and%20review&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2010.07.034&amp;volume=56&amp;pages=455-475&amp;publication_year=2011&amp;author=Krishnan%2CA&amp;author=Williams%2CLJ&amp;author=McIntosh%2CAR&amp;author=Abdi%2CH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103."><p class="c-article-references__text" id="ref-CR103">Eden, E., Navon, R., Steinfeld, I., Lipson, D. &amp; Yakhini, Z. GOrilla: a tool for discovery and visualization of enriched GO terms in ranked gene lists. <i>BMC Bioinformatics</i> <b>10</b>, 48 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1186/1471-2105-10-48" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2105-10-48" aria-label="Article reference 103" data-doi="10.1186/1471-2105-10-48">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19192299" aria-label="PubMed reference 103">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2644678" aria-label="PubMed Central reference 103">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 103" href="http://scholar.google.com/scholar_lookup?&amp;title=GOrilla%3A%20a%20tool%20for%20discovery%20and%20visualization%20of%20enriched%20GO%20terms%20in%20ranked%20gene%20lists&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-10-48&amp;volume=10&amp;publication_year=2009&amp;author=Eden%2CE&amp;author=Navon%2CR&amp;author=Steinfeld%2CI&amp;author=Lipson%2CD&amp;author=Yakhini%2CZ">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="104."><p class="c-article-references__text" id="ref-CR104">Supek, F., Bošnjak, M., ˇ Kunca, S. &amp; ˇ Muc, S. Summarizes and visualizes long lists of Gene Ontology terms. <i>PLoS ONE</i> <b>6</b>, 21800 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0021800" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0021800" aria-label="Article reference 104" data-doi="10.1371/journal.pone.0021800">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhtVeht7zI" aria-label="CAS reference 104">CAS</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 104" href="http://scholar.google.com/scholar_lookup?&amp;title=Summarizes%20and%20visualizes%20long%20lists%20of%20Gene%20Ontology%20terms&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0021800&amp;volume=6&amp;publication_year=2011&amp;author=Supek%2CF&amp;author=Bo%C5%A1njak%2CM&amp;author=%CB%87%20Kunca%2CS&amp;author=%CB%87%20Muc%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="105."><p class="c-article-references__text" id="ref-CR105">Fulcher, B. D., Arnatkeviciute, A. &amp; Fornito, A. Overcoming false-positive gene-category enrichment in the analysis of spatially resolved transcriptomic brain atlas data. <i>Nat. Commun.</i> <b>12</b>, 2669 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41467-021-22862-1" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-22862-1" aria-label="Article reference 105" data-doi="10.1038/s41467-021-22862-1">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFWgt7vL" aria-label="CAS reference 105">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33976144" aria-label="PubMed reference 105">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8113439" aria-label="PubMed Central reference 105">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 105" href="http://scholar.google.com/scholar_lookup?&amp;title=Overcoming%20false-positive%20gene-category%20enrichment%20in%20the%20analysis%20of%20spatially%20resolved%20transcriptomic%20brain%20atlas%20data&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-22862-1&amp;volume=12&amp;publication_year=2021&amp;author=Fulcher%2CBD&amp;author=Arnatkeviciute%2CA&amp;author=Fornito%2CA">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="106."><p class="c-article-references__text" id="ref-CR106">Váša, F. et al. Adolescent tuning of association cortex in human structural brain networks. <i>Cerebral Cortex</i> <b>28</b>, 281–294 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhx249" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhx249" aria-label="Article reference 106" data-doi="10.1093/cercor/bhx249">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29088339" aria-label="PubMed reference 106">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 106" href="http://scholar.google.com/scholar_lookup?&amp;title=Adolescent%20tuning%20of%20association%20cortex%20in%20human%20structural%20brain%20networks&amp;journal=Cerebral%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhx249&amp;volume=28&amp;pages=281-294&amp;publication_year=2018&amp;author=V%C3%A1%C5%A1a%2CF">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="107."><p class="c-article-references__text" id="ref-CR107">Milicevic Sephton, S. et al. Automated radiosynthesis of [<sup>11</sup>C]UCB-J for imaging synaptic density by positron emission tomography. <i>J. Labelled Comp. Radiopharm.</i> <b>63</b>, 151–158 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/jlcr.3828" data-track-action="article reference" href="https://doi.org/10.1002%2Fjlcr.3828" aria-label="Article reference 107" data-doi="10.1002/jlcr.3828">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisV2htLY%3D" aria-label="CAS reference 107">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32027052" aria-label="PubMed reference 107">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7155065" aria-label="PubMed Central reference 107">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 107" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20radiosynthesis%20of%20%5B11C%5DUCB-J%20for%20imaging%20synaptic%20density%20by%20positron%20emission%20tomography&amp;journal=J.%20Labelled%20Comp.%20Radiopharm.&amp;doi=10.1002%2Fjlcr.3828&amp;volume=63&amp;pages=151-158&amp;publication_year=2020&amp;author=Milicevic%20Sephton%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="108."><p class="c-article-references__text" id="ref-CR108">Bajjalieh, S. M., Frantz, G. D., Weimann, J. M., McConnell, S. K. &amp; Scheller, R. H. Differential expression of synaptic vesicle protein 2 (SV2) isoforms. <i>J. Neurosci.</i> <b>14</b>, 5223–5235 (1994).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.14-09-05223.1994" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.14-09-05223.1994" aria-label="Article reference 108" data-doi="10.1523/JNEUROSCI.14-09-05223.1994">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2MXnslClsw%3D%3D" aria-label="CAS reference 108">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8083732" aria-label="PubMed reference 108">PubMed</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6577109" aria-label="PubMed Central reference 108">PubMed Central</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 108" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20expression%20of%20synaptic%20vesicle%20protein%202%20%28SV2%29%20isoforms&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.14-09-05223.1994&amp;volume=14&amp;pages=5223-5235&amp;publication_year=1994&amp;author=Bajjalieh%2CSM&amp;author=Frantz%2CGD&amp;author=Weimann%2CJM&amp;author=McConnell%2CSK&amp;author=Scheller%2CRH">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="109."><p class="c-article-references__text" id="ref-CR109">Burgos, N. et al. Attenuation correction synthesis for hybrid PET-MR scanners: application to brain studies. <i>IEEE Trans. Med. Imaging</i> <b>33</b>, 2332–2341 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1109/TMI.2014.2340135" data-track-action="article reference" href="https://doi.org/10.1109%2FTMI.2014.2340135" aria-label="Article reference 109" data-doi="10.1109/TMI.2014.2340135">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25055381" aria-label="PubMed reference 109">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 109" href="http://scholar.google.com/scholar_lookup?&amp;title=Attenuation%20correction%20synthesis%20for%20hybrid%20PET-MR%20scanners%3A%20application%20to%20brain%20studies&amp;journal=IEEE%20Trans.%20Med.%20Imaging&amp;doi=10.1109%2FTMI.2014.2340135&amp;volume=33&amp;pages=2332-2341&amp;publication_year=2014&amp;author=Burgos%2CN">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="110."><p class="c-article-references__text" id="ref-CR110">Manavaki, R., Hong, Y. &amp; Fryer, T. D. Effect of brain MRI coil attenuation map processing on PET image quantification and uniformity for the GE SIGNA PET/MR. <i>IEEE Nucl. Sci. Symp. Med. Imaging Conf. NSS/MIC</i> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="111."><p class="c-article-references__text" id="ref-CR111">Wu, Y. &amp; Carson, R. E. Noise reduction in the simplified reference tissue model for neuroreceptor functional imaging. <i>J. Cereb. Blood Flow Metab.</i> <b>22</b>, 1440–1452 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1097/01.WCB.0000033967.83623.34" data-track-action="article reference" href="https://doi.org/10.1097%2F01.WCB.0000033967.83623.34" aria-label="Article reference 111" data-doi="10.1097/01.WCB.0000033967.83623.34">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12468889" aria-label="PubMed reference 111">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 111" href="http://scholar.google.com/scholar_lookup?&amp;title=Noise%20reduction%20in%20the%20simplified%20reference%20tissue%20model%20for%20neuroreceptor%20functional%20imaging&amp;journal=J.%20Cereb.%20Blood%20Flow%20Metab.&amp;doi=10.1097%2F01.WCB.0000033967.83623.34&amp;volume=22&amp;pages=1440-1452&amp;publication_year=2002&amp;author=Wu%2CY&amp;author=Carson%2CRE">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="112."><p class="c-article-references__text" id="ref-CR112">Koole, M. et al. Quantifying SV2A density and drug occupancy in the human brain using [<sup>11</sup>C]UCB-J PET imaging and subcortical white matter as reference tissue. <i>Eur. J. Nucl. Med. Mol. Imaging</i> <b>46</b>, 396–406 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1007/s00259-018-4119-8" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s00259-018-4119-8" aria-label="Article reference 112" data-doi="10.1007/s00259-018-4119-8">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFCku7bM" aria-label="CAS reference 112">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30121895" aria-label="PubMed reference 112">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 112" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20SV2A%20density%20and%20drug%20occupancy%20in%20the%20human%20brain%20using%20%5B11C%5DUCB-J%20PET%20imaging%20and%20subcortical%20white%20matter%20as%20reference%20tissue&amp;journal=Eur.%20J.%20Nucl.%20Med.%20Mol.%20Imaging&amp;doi=10.1007%2Fs00259-018-4119-8&amp;volume=46&amp;pages=396-406&amp;publication_year=2019&amp;author=Koole%2CM">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="113."><p class="c-article-references__text" id="ref-CR113">Rossano, S. et al. Assessment of a white matter reference region for [<sup>11</sup>C]UCB-J PET quantification. <i>J Cereb. Blood Flow Metab.</i> <b>40</b>, 1890–1901 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1177/0271678X19879230" data-track-action="article reference" href="https://doi.org/10.1177%2F0271678X19879230" aria-label="Article reference 113" data-doi="10.1177/0271678X19879230">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Sjt77P" aria-label="CAS reference 113">CAS</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31570041" aria-label="PubMed reference 113">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 113" href="http://scholar.google.com/scholar_lookup?&amp;title=Assessment%20of%20a%20white%20matter%20reference%20region%20for%20%5B11C%5DUCB-J%20PET%20quantification&amp;journal=J%20Cereb.%20Blood%20Flow%20Metab.&amp;doi=10.1177%2F0271678X19879230&amp;volume=40&amp;pages=1890-1901&amp;publication_year=2020&amp;author=Rossano%2CS">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="114."><p class="c-article-references__text" id="ref-CR114">Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <i>J. Royal Stat. Soc. Ser. B</i> <b>57</b>, 289–300 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 114" href="http://scholar.google.com/scholar_lookup?&amp;title=Controlling%20the%20false%20discovery%20rate%3A%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;journal=J.%20Royal%20Stat.%20Soc.%20Ser.%20B&amp;volume=57&amp;pages=289-300&amp;publication_year=1995&amp;author=Benjamini%2CY&amp;author=Hochberg%2CY">
                    Google Scholar</a> 
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="115."><p class="c-article-references__text" id="ref-CR115">Borenstein, M., Hedges, L. V., Higgins, J. P. T. &amp; Rothstein, H. R. Introduction to Meta-Analysis. 1–421 <a href="https://doi.org/10.1002/9780470743386" data-track="click" data-track-action="external reference" data-track-label="10.1002/9780470743386">https://doi.org/10.1002/9780470743386</a> (Wiley, 2009).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="116."><p class="c-article-references__text" id="ref-CR116">Markello, R. D. &amp; Misic, B. Comparing spatial null models for brain maps. <i>Neuroimage</i> <b>236</b>, 118052 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2021.118052" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2021.118052" aria-label="Article reference 116" data-doi="10.1016/j.neuroimage.2021.118052">Article</a> 
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33857618" aria-label="PubMed reference 116">PubMed</a> 
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 116" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparing%20spatial%20null%20models%20for%20brain%20maps&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2021.118052&amp;volume=236&amp;publication_year=2021&amp;author=Markello%2CRD&amp;author=Misic%2CB">
                    Google Scholar</a> 
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-022-01070-0?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We are grateful to UCB Pharma for providing the precursor for the radioligand used in PET imaging. We also express our gratitude to the PRIME-DE initiative, to the organizers and managers of PRIME-DE and to all the institutions that contributed to the PRIME-DE dataset (http://fcon_1000.projects.nitrc.org/indi/indiPRIME.html), with special thanks to the Newcastle team. We are also grateful to A. Grigis, J. Tasserie and B. Jarraya for their help with the Pypreclin code, and R. Romero-Garcia for generating and sharing the 500-mm<sup>2</sup> subparcellation of the Desikan–Killiany atlas, and the corresponding Von Economo cytoarchitectonics map. We are also grateful to Y. Wei and colleagues for generating and making available the data pertaining to HAR genes and cortical expansion, to N. Vaishnavi and M. Goyal and colleagues for making available data pertaining to regional GI and its associated genes, to C. Paquola and colleagues for making available their data on cortico-cortical wiring distance, and to A. Goulas and colleagues for making available anonymized receptor autoradiography data. We are grateful to S. Morgan, P. Vértes and K. Whitaker for making available their code pertaining to AHBA gene analysis, and to F. Váša for making available the code for spin-based permutation testing. Finally, we thank S. Panzeri for helpful feedback on an earlier draft of our manuscript. This work was supported by grants from the NIHR, Cambridge Biomedical Research Centre and NIHR Senior Investigator Awards (to D.K.M.); the British Oxygen Professorship of the Royal College of Anaesthetists (to D.K.M.); the Canadian Institute for Advanced Research (CIFAR) (RCZB/072 RG93193; to D.K.M. and E.A.S.); the Stephen Erskine Fellowship (Queens’ College, Cambridge; to E.A.S.); and a Gates Scholarship from the Gates Cambridge Trust (OPP 1144 to A.I.L.) and by The Alan Turing Institute under the EPSRC grant EP/N510129/1. P.A.M.M. and D.B. are funded by the Wellcome Trust (grant no. 210920/Z/18/Z). F.R. is funded by the Ad Astra Chandaria foundation. Computing infrastructure at the Wolfson Brain Imaging Centre (WBIC-HPHI) was funded by the MRC research infrastructure award (MR/M009041/1). The PET study was funded by the Cambridge University Centre for Parkinson-Plus; the NIHR Cambridge Biomedical Research Centre (146281); the Wellcome Trust (103838) and the Association of British Neurologists, Patrick Berthoud Charitable Trust (RG99368). Data were provided (in part) by the HCP, WU-Minn Consortium (Principal Investigators: D. Van Essen and K. Ugurbil; 1U54MH091657) funded by the 16 National Institutes of Health (NIH) institutes and centers that support the NIH Blueprint for Neuroscience Research; and by the McDonnell Center for Systems Neuroscience at Washington University. For the macaque data, primary support for the work by Newcastle University was provided by Wellcome Trust (WT091681MA, WT092606AIA), National Centre for 3Rs (project grant NC/K000802/1; pilot grant NC/K000608/1) and Biotechnology and Biological Sciences Research Council (grant number BB/J009849/1).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Division of Anaesthesia, School of Clinical Medicine, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Andrea I. Luppi, David K. Menon &amp; Emmanuel A. Stamatakis</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Clinical Neurosciences, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Andrea I. Luppi, Negin Holland, Tim D. Fryer, James B. Rowe &amp; Emmanuel A. Stamatakis</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Leverhulme Centre for the Future of Intelligence, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Andrea I. Luppi</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">The Alan Turing Institute, London, UK</p><p class="c-article-author-affiliation__authors-list">Andrea I. Luppi</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Department of Psychology, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Pedro A. M. Mediano &amp; Daniel Bor</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Department of Psychology, Queen Mary University of London, London, UK</p><p class="c-article-author-affiliation__authors-list">Pedro A. M. Mediano &amp; Daniel Bor</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Center for Psychedelic Research, Department of Brain Science, Imperial College London, London, UK</p><p class="c-article-author-affiliation__authors-list">Fernando E. Rosas</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Data Science Institute, Imperial College London, London, UK</p><p class="c-article-author-affiliation__authors-list">Fernando E. Rosas</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Center for Complexity Science, Imperial College London, London, UK</p><p class="c-article-author-affiliation__authors-list">Fernando E. Rosas</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Wolfson Brain Imaging Centre, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Tim D. Fryer &amp; David K. Menon</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Department of Psychiatry, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">John T. O’Brien</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Cambridge University Hospitals NHS Foundation Trust, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">John T. O’Brien &amp; James B. Rowe</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">MRC Cognition and Brain Sciences Unit, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">James B. Rowe</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Andrea_I_-Luppi-Aff1-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Andrea I. Luppi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Andrea%20I.%20Luppi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Andrea%20I.%20Luppi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Andrea%20I.%20Luppi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pedro_A__M_-Mediano-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Pedro A. M. Mediano</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pedro%20A.%20M.%20Mediano" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pedro%20A.%20M.%20Mediano" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pedro%20A.%20M.%20Mediano%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Fernando_E_-Rosas-Aff7-Aff8-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Fernando E. Rosas</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Fernando%20E.%20Rosas" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Fernando%20E.%20Rosas" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Fernando%20E.%20Rosas%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Negin-Holland-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Negin Holland</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Negin%20Holland" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Negin%20Holland" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Negin%20Holland%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tim_D_-Fryer-Aff2-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Tim D. Fryer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Tim%20D.%20Fryer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tim%20D.%20Fryer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tim%20D.%20Fryer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-John_T_-O_Brien-Aff11-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">John T. O’Brien</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=John%20T.%20O%E2%80%99Brien" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=John%20T.%20O%E2%80%99Brien" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20T.%20O%E2%80%99Brien%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-James_B_-Rowe-Aff2-Aff12-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">James B. Rowe</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=James%20B.%20Rowe" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=James%20B.%20Rowe" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22James%20B.%20Rowe%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-David_K_-Menon-Aff1-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">David K. Menon</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=David%20K.%20Menon" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=David%20K.%20Menon" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22David%20K.%20Menon%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Daniel-Bor-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Daniel Bor</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Daniel%20Bor" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Daniel%20Bor" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Daniel%20Bor%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Emmanuel_A_-Stamatakis-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Emmanuel A. Stamatakis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Emmanuel%20A.%20Stamatakis" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Emmanuel%20A.%20Stamatakis" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"> </span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Emmanuel%20A.%20Stamatakis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>A.I.L. conceived the study, analyzed data and wrote first draft of the manuscript. P.A.M.M. conceived the study, contributed to data analysis and reviewed and edited the manuscript. F.R. contributed to data analysis, and reviewed and edited the manuscript. N.H. acquired PET data, reviewed PET analysis and reviewed the manuscript. T.D.F. preprocessed PET data and reviewed the manuscript. J.T.O. conceived the PET project, reviewed PET analysis and reviewed the manuscript. J.B.R. conceived the PET project, reviewed PET analysis and reviewed the manuscript. D.K.M. reviewed the manuscript. D.B. conceived the study and reviewed and edited the manuscript. E.A.S. conceived the study and reviewed and edited the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:al857@cam.ac.uk">Andrea I. Luppi</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>J.B.R. is a non-remunerated trustee of the Guarantors of Brain and the PSP Association (United Kingdom). J.B.R. provides consultancy to Asceneuron, Biogen and UCB and has research grants from AstraZeneca/MedImmune, Janssen and Lilly as industry partners in the Dementias Platform UK. All other authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Neuroscience</i> thanks Michael Hawrylycz, Stefano Panzeri, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec63-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec63">Extended data</h2><div class="c-article-section__content" id="Sec63-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 synergistic and redundant int" href="/articles/s41593-022-01070-0/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig8_ESM.jpg">Extended Data Fig. 1 Synergistic and redundant interactions in the brain.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>-<b>c</b>) Group average matrices of pairwise functional interactions between brain regions of the Schaefer-232 atlas, quantified by (a) redundancy; (b) synergy; (c) traditional functional connectivity (Pearson correlation). (<b>d</b>) Mean regional density of redundant interactions, after thresholding the group-average redundancy matrix to retain the 5% strongest edges, for display purposes. (<b>e</b>) Mean regional density of synergistic interactions, after thresholding the group-average synergy matrix to retain the 5% strongest edges, for display purposes. (<b>f</b>) Spearman correlation (two-sided CI: [−0.51, −0.28]) between synergy vs. redundancy ranks across cortical regions.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 synergy-redundancy identifica" href="/articles/s41593-022-01070-0/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig9_ESM.jpg">Extended Data Fig. 2 Synergy-redundancy identification and NeuroSynth meta-analysis are robust to the use of alternative methods.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Left: Group-average matrices of redundant and synergistic interactions; Middle: Redundancy-to-synergy gradient scores (synergy rank minus redundancy rank) displayed on medial and lateral brain surfaces (left hemisphere); Right: Results of the NeuroSynth term-based meta-analysis, relating the distribution of redundancy-to-synergy gradient across the brain to a gradient of cognitive domains, from lower-level sensorimotor processing to higher-level cognitive tasks (note that one term, ‘visual semantics’, was excluded from visualisation because it failed to reach the threshold of Z &gt; 3.1, leaving 23 terms). (<b>a</b>) DK-308 parcellation with equally-sized cortical areas (500 mm<sup>2</sup>), obtained as subdivisions of the Desikan-Killiany cortical atlas. (<b>b</b>) Lausanne-129 parcellation, comprising the DK-114 cortical ROIs, supplemented with 15 subcortical regions. (<b>c</b>) Synergy and redundancy computed without deconvolution of the hemodynamic response function (HRF) from the BOLD signal timeseries. (<b>d</b>) Synergy and redundancy computed from discretised (binary) BOLD signal timeseries.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 robustness of synergy and red" href="/articles/s41593-022-01070-0/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig10_ESM.jpg">Extended Data Fig. 3 Robustness of synergy and redundancy network results (efficiency, modularity, and within- vs between-resting state subnetwork comparison) to alternative node and edge definitions.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>-<b>d</b>) Robustness of network results to the use of the 308-ROI cortical parcellation. (<b>e</b>-<b>h</b>) Robustness of network results to using synergy and redundancy normalised by TDMI. (<b>i-l</b>) Robustness of network results to using synergy and redundancy obtained from discretised signals. For all violin plots: each colored circle represents one subject; white circle: median; central line: mean; box limits, upper and lower quartiles; whiskers, 1.5x inter-quartile range; *** p &lt; 0.001 from paired-sample non-parametric permutation t-test (two-sided); n = 100 unrelated HCP subjects. For all box-plots: white circle, median; box limits, upper and lower quartiles; whiskers, 1.5x inter-quartile range; *** p &lt; 0.001 from two-sample non-parametric permutation t-test (two-sided). For (c-d) and (k-l), Within-RSN n=7178 connections; Between-RSN n=46414 connections. For (g) and (h), Within-RSN n=14784 connections; Between-RSN n=79772 connections.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 robustness of synergy and red" href="/articles/s41593-022-01070-0/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig11_ESM.jpg">Extended Data Fig. 4 Robustness of synergy and redundancy structure-function results to alternative node and edge definitions.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>-<b>c</b>) Robustness of structure-function results to the use of the 308-ROI cortical parcellation. (<b>d</b>-<b>f</b>) Robustness of structure-function results to using synergy and redundancy normalised by TDMI. (<b>g</b>-<b>i</b>) Robustness of structure-function results to using synergy and redundancy obtained from discretised signals. For all violin plots: each colored circle represents one subject; white circle: median; central line: mean; box limits, upper and lower quartiles; whiskers, 1.5x inter-quartile range; *** p &lt; 0.001 from paired-sample non-parametric permutation t-test (two-sided); n = 100 unrelated HCP subjects. For all box-plots: white circle: median; box limits, upper and lower quartiles; whiskers, 1.5x inter-quartile range; *** p &lt; 0.001 from two-sample non-parametric permutation t-test (two-sided). For (b-c), SC+, n=6864 direct connections; SC-, n=88000 connections. For (e-f) and (h-i), SC+, n=5276 direct connections; SC-, n=48548 indirect connections.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 additional validation of syne" href="/articles/s41593-022-01070-0/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig12_ESM.jpg">Extended Data Fig. 5 Additional validation of synergy and redundancy network results.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Alternative measure of global integration (area under the curve of the size of the largest connected component across thresholds). (<b>b</b>) Alternative structural-functional dissimilarity (mean Hamming distance). For both (a) and (b): *** p &lt; 0.001 from paired-sample non-parametric permutation t-test (two-sided), n=100 unrelated HCP subjects. (<b>c</b>) Comparison of global efficiency of synergy and redundancy networks of each subject with the average global efficiency of 100 synthetic null networks with edges randomly drawn from the distribution between 0 and the empirical TDMI. (<b>d</b>) Comparison of modularity of synergy and redundancy networks of each subject with the average modularity of 100 synthetic null networks with edges randomly drawn from the distribution between 0 and the empirical TDMI. For (c) and (d), *** p &lt; 0.001 (FDR-corrected) from two-sample non-parametric permutation t-test (two-sided); n = 100 unrelated HCP subjects and n=100 synthetic null networks. For all violin plots: each colored circle represents one subject; white circle: median; central line: mean; box limits, upper and lower quartiles; whiskers, 1.5x inter-quartile range.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 validation analysis for human" href="/articles/s41593-022-01070-0/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig13_ESM.jpg">Extended Data Fig. 6 Validation analysis for human-macaque comparison of synergy and redundancy.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>-<b>d</b>) Simulation of human fMRI data with same TR as the macaque data shows that human-macaque differences in synergy cannot be attributed solely to TR differences between datasets. (a) The dynamic mean field (DMF) model used to simulate human fMRI data combines macroscale information about neuroanatomy and structural connectivity (from DTI) with excitatory and inhibitory neuronal populations interconnected by AMPA, NMDA and GABA synapses, providing a neurobiologically plausible account of regional neuronal firing rate, which is turned into simulated BOLD signal by means of the Balloon-Windkessel hemodynamic model. (b) Using a TR of 0.72s (the same as the empirical HCP data), the model is fitted to the empirical HCP data by finding the value of the global coupling parameter G that minimises the Kolmogorov-Smirnov distance between the distributions of empirical and simulated functional connectivity dynamics (FCD). The KS distance is minimised for G=1.6, which is the value of G used for subsequent simulations with TR=2.6s (the same TR as the macaque data). (c) The proportion of synergistic information exchange across the brain is significantly higher in simulated human data than in empirical macaque data with the same TR=2.6s (p&lt;0.001). (d) The proportion of redundant information exchange across the brain is also significantly higher in simulated human data than empirical macaque data (p=0.036). Statistical significance assessed with two-sample non-parametric permutation t-test (two-sided); DMF HCP data: n=100 simulations; macaque data: n=19 distinct sessions from 10 individual macaques. (<b>e</b>-<b>f</b>) The human-macaque comparison of synergy and redundancy proportion is robust to bandpass filtering both human and macaque functional MRI data between 0.008−0.09 Hz. The proportion of synergistic information exchange across the brain is significantly higher in humans (p&lt;0.001) (e) whereas the proportion of redundant information exchange across the brain is equivalent in humans and macaques (p=0.943) (f). Statistical significance assessed with two-sample non-parametric permutation t-test (two-sided). Human data: n=100 unrelated HCP subjects. Macaque data: n=19 distinct sessions from 10 individual macaques. (<b>g</b>-<b>h</b>) The proportion of total synergy is significantly higher in humans than macaques (p&lt;0.001) (h), even when only considering humans whose total FC is in the range of values exhibited by macaques (excluding one outlier with extreme value), such that there is no significant difference in total FC between the two groups (p=0.196), shown in (g). Statistical significance assessed with two-sample non-parametric permutation t-test (two-sided). Human data: n=28 unrelated HCP subjects with FC values in the range of the macaque FC values. Macaque data: n=19 distinct sessions from 10 individual macaques (one outlier excluded in (g)). For all violin plots: each colored circle indicates one data-point; white circle: median; central line: mean; box limits, upper and lower quartiles; whiskers, 1.5x inter-quartile range; n.s., p &gt; 0.05; * p &lt; 0.05; *** p &lt; 0.001.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 characterisation of synergist" href="/articles/s41593-022-01070-0/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig14_ESM.jpg">Extended Data Fig. 7 Characterisation of synergistic and redundant network profiles in macaque brains are similar to humans.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Synergistic interactions between regions of the macaque brain. (<b>b</b>) Redundant interactions between regions of the macaque brain. (<b>c</b>) Anatomical connectivity was estimated from axonal tracing and diffusion MRI (Shen <i>et al</i>., 2019), and Spearman correlation coefficient was used to assess the similarity of redundancy and synergy matrices with structural connectivity, after thresholding to ensure equal numbers of connections. (<b>d</b>) The network organisation of synergistic interactions exhibits significantly higher global efficiency than redundant interactions (p &lt; 0.001). (<b>e</b>) The network organisation of redundant interactions exhibits significantly higher segregation (modularity) than synergistic interactions (p &lt; 0.001). (<b>f</b>) Networks of redundant interactions are significantly more correlated with underlying anatomical connectivity than synergistic interactions (p &lt; 0.001). For all tests: *** p &lt; 0.001 from paired-sample non-parametric permutation t-test (two-sided); n=19 distinct sessions from 10 individual macaques (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-022-01070-0#MOESM2">7</a>). For all violin plots: each colored circle indicates one data-point; white circle: median; central line: mean; box limits, upper and lower quartiles; whiskers, 1.5x inter-quartile range.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 synergy-redundancy gradient c" href="/articles/s41593-022-01070-0/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig15_ESM.jpg">Extended Data Fig. 8 Synergy-redundancy gradient correlates with unadjusted cortical expansion and gene expression.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Significant Spearman correlation (two-sided CI: [0.145, 0.476]) between regional redundancy-to-synergy gradient scores and unadjusted regional cortical expansion from chimpanzee (<i>Pan troglodytes</i>) to human (both on DK-114 cortical atlas, both hemispheres; n =114 cortical regions). (<b>b</b>) Significant Spearman correlation (two-sided CI: [0.109, 0.567]) between regional redundancy-to-synergy gradient scores and unadjusted regional expression of brain-related human-accelerated (HAR) genes (both on left hemisphere of DK-114 atlas: n=57 left-hemisphere regions). (<b>c</b>) Significant Spearman correlation (two-sided CI: [0.010, 0.496]) between regional redundancy-to-synergy gradient scores and unadjusted regional expression of non-brain-related human-accelerated (HAR) genes (both on left hemisphere of DK-114 atlas; n=57 left-hemisphere regions). <i>p_spin</i> indicates the p-value estimated from a spatial permutation test comparing the empirical correlation against 10,000 randomly rotated brain maps with preserved spatial covariance.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig16"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 characterisation of pls compo" href="/articles/s41593-022-01070-0/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig16_ESM.jpg">Extended Data Fig. 9 Characterisation of PLS components of 20,647 genes from the Allen Institute for Brain Science, for the 308-ROI subdivision of the Desikan-Killiany cortical parcellation.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Spearman correlation (two-sided CI: [0.334, 0.517]; n=308 regions) between the redundancy-to-synergy regional pattern, and the first principal component of PLS (PLS1). (<b>b</b>) Spearman correlation (two-sided CI: [0.216, 0.417]; n=308 regions) between the redundancy-to-synergy regional pattern, and the second principal component of PLS (PLS2). For both (a) and (b), color-bars correspond to scatter-plot axes. (<b>c</b>) The variance explained by the first 2 PLS components is significantly higher than would be expected based on random patterns with preserved spatial autocorrelation, assessed using spin-based permutations (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-022-01070-0#Sec12">Methods</a>). (<b>d</b>,<b>e</b>) Significant enrichment of HAR-Brain genes in PLS1 and PLS2. (<b>f</b>) Significant HAR-Brain gene enrichment is also observed using an alternative approach: ridge-regularized PLS regression on the binarised cortical pattern of synergy vs redundancy prevalence. (<b>g</b>,<b>h</b>) HAR-Brain gene enrichment in PLS1 and PLS2 is also observed when controlling for spatial autocorrelation using spin-based permutations. (c-h) Statistical significance is assessed via bootstrap resampling of Z-scores; histograms indicate the relative frequency (over 1,000 bootstraps) of the mean Z-score of a random sample of genes of equal size as the HAR-Brain genes. Red vertical line: empirical mean Z-score of HAR-Brain genes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig17"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 enrichment analysis for gene" href="/articles/s41593-022-01070-0/figures/17" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_Fig17_ESM.jpg">Extended Data Fig. 10 Enrichment analysis for genes pertaining to synaptic formation, whose regional distribution corresponds to the distribution of aerobic glycolysis in the human brain, as reported by Goyal et al. (2014) (‘aerobic glycolysis genes’).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>,<b>b</b>) PLS1 and PLS2 are significantly enriched for aerobic glycolysis genes. (<b>c</b>) Enrichment for genes related to aerobic glycolysis is also observed using an alternative approach: ridge-regularised PLS regression on the binarised cortical pattern of synergy vs redundancy prevalence. (<b>d</b>,<b>e</b>) Significant enrichment for genes related to aerobic glycolysis in PLS1 and PLS2 is also observed when controlling for spatial autocorrelation using spin-based permutations. (a-e) Statistical significance is assessed via bootstrap resampling of Z-scores; histograms indicate the relative frequency (over 1,000 bootstraps) of the mean Z-score of a random sample of genes of equal size as the aerobic glycolysis genes. Red vertical line: empirical mean Z-score of aerobic glycolysis genes.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec64-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec64">Supplementary information</h2><div class="c-article-section__content" id="Sec64-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_MOESM1_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_MOESM2_ESM.xlsx" data-supp-info-image="">Supplementary Table</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Table 1 - Prevalence of synergy and redundancy for each canonical resting-state network. Statistical significance assessed via one-sample permutation-based non-parametric <i>t</i>-test (two-sided), FDR-corrected. Supplementary Table 2 - Prevalence of synergy and redundancy for each Von Economo cytoarchitectonic class. Statistical significance assessed via one-sample permutation-based non-parametric <i>t</i>-test (two-sided), FDR-corrected. Supplementary Table 3 - Synergy and redundancy network results for the Schaefer-232 parcellation. Statistical significance assessed via paired-sample permutation-based non-parametric <i>t</i>-test (two-sided). Supplementary Table 4 - Synergy and redundancy network results for alternative definitions of nodes and edges. Statistical significance assessed via paired-sample permutation-based non-parametric <i>t</i>-test (two-sided). Supplementary Table 5 - Synergy and redundancy network results for macaques. Statistical significance assessed via paired-sample permutation-based non-parametric <i>t</i>-test (two-sided). Supplementary Table 6 - Comparison of effect sizes for different measures, for distinguishing the distributions of humans and macaques (macaque data filtered between 0.008 and 0.09 Hz). Statistical significance assessed via <i>z</i>-score test (two-sided). The effect size obtained from each measure is compared against the effect size obtained for synergy, Hedges’ <i>g</i> = 9.98. Positive <i>z</i>-values indicate that synergy has a larger effect size. Supplementary Table 7 - Comparison of effect sizes for different measures, for distinguishing the distributions of humans and macaques (macaque data filtered between 0.0025 and 0.05 Hz). Statistical significance assessed via <i>z</i>-score test (two-sided). The effect size obtained from each measure is compared against the effect size obtained for synergy, Hedges’ <i>g</i> = 4.94. Positive <i>z</i>-values indicate that synergy has a larger effect size. Supplementary Table 8 - Comparison of effect sizes for different measures, for distinguishing the distributions of humans and macaques, with human data trimmed to ensure same number of time points as the macaque data. Statistical significance assessed via <i>z</i>-score test (two-sided). The effect size obtained from each measure is compared against the effect size obtained for synergy, Hedges’ <i>g</i> = 5.90. Positive <i>z</i>-values indicate that synergy has a larger effect size. Supplementary Table 9 - Comparison of effect sizes for different measures, for distinguishing the distributions of humans and macaques, using the discrete estimator for binarized time series. Statistical significance assessed via <i>z</i>-score test (two-sided). The effect size obtained from each measure is compared against the effect size obtained for synergy, Hedges’ <i>g</i> = 14.01. Positive <i>z</i>-values indicate that synergy has a larger effect size. Supplementary Table 10 - Significant enrichment for synaptic transmission and organization is preserved when using advanced null models based on random phenotype ensembles.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary software" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-022-01070-0/MediaObjects/41593_2022_1070_MOESM3_ESM.zip" data-supp-info-image="">Supplementary Software</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>This library contains Octave (v5.0.0) and MATLAB (&gt;R2016a) functions to compute Integrated Information Decomposition (ΦID) as described in Mediano, P. A. M. et al. Towards an extended taxonomy of information dynamics via integrated information decomposition. Preprint at <a href="https://arxiv.org/abs/2109.13186">https://arxiv.org/abs/2109.13186</a> (2021). The code works with continuous and discrete data using Barrett’s MMI redundancy function, as described in Barrett, A. B. Exploration of synergistic and redundant information sharing in static and dynamical Gaussian systems. <i>Phys. Rev.</i> <i>E</i> 91, 52802 (2015). This code implements the measures used for further analysis in this paper.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=A%20synergistic%20core%20for%20human%20brain%20evolution%20and%20cognition&amp;author=Andrea%20I.%20Luppi%20et%20al&amp;contentID=10.1038%2Fs41593-022-01070-0&amp;copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2022-05-26&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-022-01070-0" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-022-01070-0" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Luppi, A.I., Mediano, P.A.M., Rosas, F.E. <i>et al.</i> A synergistic core for human brain evolution and cognition.
                    <i>Nat Neurosci</i> <b>25</b>, 771–782 (2022). https://doi.org/10.1038/s41593-022-01070-0</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-022-01070-0?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-11-20">20 November 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-03-30">30 March 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-05-26">26 May 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-06">June 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-022-01070-0</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Downstream network transformations dissociate neural activity from causal functional contributions" href="https://doi.org/10.1038/s41598-024-52423-7">
                                        Downstream network transformations dissociate neural activity from causal functional contributions
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Kayson Fakhar</li><li>Shrey Dixit</li><li>Claus C. Hilgetag</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Reports</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Behavioural relevance of redundant and synergistic stimulus information between functionally connected neurons in mouse auditory cortex" href="https://doi.org/10.1186/s40708-023-00212-9">
                                        Behavioural relevance of redundant and synergistic stimulus information between functionally connected neurons in mouse auditory cortex
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Loren Koçillari</li><li>Marco Celotto</li><li>Stefano Panzeri</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Brain Informatics</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Evidence for increased parallel information transmission in human brain networks compared to macaques and male mice" href="https://doi.org/10.1038/s41467-023-43971-z">
                                        Evidence for increased parallel information transmission in human brain networks compared to macaques and male mice
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Alessandra Griffa</li><li>Mathieu Mach</li><li>Enrico Amico</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Multivariate information theory uncovers synergistic subsystems of the human cerebral cortex" href="https://doi.org/10.1038/s42003-023-04843-w">
                                        Multivariate information theory uncovers synergistic subsystems of the human cerebral cortex
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Thomas F. Varley</li><li>Maria Pope</li><li>Olaf Sporns</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2023)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Human-specific evolutionary markers linked to foetal neurodevelopment modulate brain surface area in schizophrenia" href="https://doi.org/10.1038/s42003-023-05356-2">
                                        Human-specific evolutionary markers linked to foetal neurodevelopment modulate brain surface area in schizophrenia
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact c-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Maria Guardiola-Ripoll</li><li>Carmen Almodóvar-Payá</li><li>Mar Fatjó-Vilas</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2023)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-022-01070-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-022-01070-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-022-01070-0;doi=10.1038/s41593-022-01070-0;techmeta=36,38,57,59,61,78;subjmeta=116,1925,2150,2393,2583,2649,378,631;kwrd=Cognitive+control,Dynamical+systems,Genetics+of+the+nervous+system,Network+models">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=-1654860064&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-022-01070-0%26doi%3D10.1038/s41593-022-01070-0%26techmeta%3D36,38,57,59,61,78%26subjmeta%3D116,1925,2150,2393,2583,2649,378,631%26kwrd%3DCognitive+control,Dynamical+systems,Genetics+of+the+nervous+system,Network+models">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=-1654860064&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-022-01070-0%26doi%3D10.1038/s41593-022-01070-0%26techmeta%3D36,38,57,59,61,78%26subjmeta%3D116,1925,2150,2393,2583,2649,378,631%26kwrd%3DCognitive+control,Dynamical+systems,Genetics+of+the+nervous+system,Network+models"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-022-01070-0&amp;format=js&amp;last_modified=2022-05-26" async></script>
<img src="/v39e5677/article/s41593-022-01070-0" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>