<!DOCTYPE html>
<html lang="en" class="grade-c">
<head>
    <title>Functional brain networks reflect spatial and temporal autocorrelation | Nature Neuroscience</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/neuro.rss"/>


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin>

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480" />

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"cognitive-neuroscience;computational-neuroscience;functional-magnetic-resonance-imaging","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Neuroscience","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article"}},"article":{"doi":"10.1038/s41593-023-01299-3"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":false,"legacy":{"webtrendsLicenceType":null}}},"contentInfo":{"authors":["Maxwell Shinn","Amber Hu","Laurel Turner","Stephanie Noble","Katrin H. Preller","Jie Lisa Ji","Flora Moujaes","Sophie Achard","Dustin Scheinost","R. Todd Constable","John H. Krystal","Franz X. Vollenweider","Daeyeol Lee","Alan Anticevic","Edward T. Bullmore","John D. Murray"],"publishedAt":1682294400,"publishedAtString":"2023-04-24","title":"Functional brain networks reflect spatial and temporal autocorrelation","legacy":null,"publishedAtTime":null,"documentType":"aplusplus"},"journal":{"pcode":"neuro","title":"nature neuroscience","volume":"26","issue":"5"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false},{"name":"getftr-entitled","active":false},{"name":"paywall_recommendations","active":true}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"US","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-weight:700;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}html{text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;line-height:1.15;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;font-weight:700;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{-webkit-font-smoothing:antialiased;font-size:1.5rem;font-weight:700;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{-webkit-font-smoothing:antialiased;font-weight:700;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{-webkit-font-smoothing:antialiased;font-weight:700;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}button:focus{outline:3px solid #fece3e;will-change:transform}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-article-license__badge,c-card__section{margin-top:8px}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-status-message--success{border-bottom:2px solid #00b8b0;justify-content:center;margin-bottom:16px;padding-bottom:8px}.c-recommendations-list__item .c-card{flex-basis:100%}.c-recommendations-list__item .c-card__image{align-items:baseline;flex:1 1 40%;margin:0 0 0 16px;max-width:150px}.c-recommendations-list__item .c-card__image img{border:1px solid #cedbe0;height:auto;min-height:0;position:static}@media only screen and (max-width:1023px){.c-recommendations-list__item .c-card__image{display:none}}.c-card__layout{display:flex;flex:1 1 auto;justify-content:space-between}.c-card__title-recommendation{-webkit-box-orient:vertical;-webkit-line-clamp:4;display:-webkit-box;font-size:1rem;font-weight:700;line-height:1.4;margin:0 0 8px;max-height:5.6em;overflow:hidden!important;text-overflow:ellipsis}.c-card__title-recommendation .c-card__link{color:inherit}.c-card__title-recommendation .c-card__link:hover{text-decoration:underline}.c-card__title-recommendation .MathJax_Display{display:inline!important}.c-card__link:not(.c-card__link--no-block-link):before{z-index:1}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a,.c-article-recommendations-card__link{color:inherit}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-body .c-article-recommendations-card__authors{display:none;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.5;margin:0 0 8px}@media only screen and (max-width:539px){.c-article-body .c-article-recommendations-card__authors{display:block;margin:0}}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.c-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.c-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .c-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.c-author-list>li:not(:first-child):not(:last-child):before{content:", "}.c-author-list>li:not(:only-child):last-child:before{content:" & "}.c-author-list--compact{font-size:.875rem;line-height:1.4}.c-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .c-author-list__hide{display:none;visibility:hidden}.js .c-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{fill:#888;height:10px;margin:4px 4px 0;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{clip:rect(0,0,0,0);border:0;height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.u-lazy-ad-wrapper{background-color:#ccc;display:none;min-height:137px}@media only screen and (min-width:768px){.u-lazy-ad-wrapper{display:block}}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="print" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-950e2d5825.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-122346e276.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href=/static/images/favicons/nature/apple-touch-icon-f39cb19454.png>
<link rel="icon" type="image/png" sizes="48x48" href=/static/images/favicons/nature/favicon-48x48-b52890008c.png>
<link rel="icon" type="image/png" sizes="32x32" href=/static/images/favicons/nature/favicon-32x32-3fe59ece92.png>
<link rel="icon" type="image/png" sizes="16x16" href=/static/images/favicons/nature/favicon-16x16-951651ab72.png>
<link rel="manifest" href=/static/manifest.json crossorigin="use-credentials">
<link rel="mask-icon" href=/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg color="#000000">
<link rel="shortcut icon" href=/static/images/favicons/nature/favicon.ico>
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content=/static/browserconfig.xml>
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-54.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-26e142e9c6.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-782fd09f66.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d66d49033d.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-aca08c055a.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-4fba787158.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-1fe07484e5.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-8fc1a30809.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-e0c7186f28.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-2399be388c.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Functional brain networks reflect spatial and temporal autocorrelation","description":"High-throughput experimental methods in neuroscience have led to an explosion of techniques for measuring complex interactions and multi-dimensional patterns. However, whether sophisticated measures of emergent phenomena can be traced back to simpler, low-dimensional statistics is largely unknown. To explore this question, we examined resting-state functional magnetic resonance imaging (rs-fMRI) data using complex topology measures from network neuroscience. Here we show that spatial and temporal autocorrelation are reliable statistics that explain numerous measures of network topology. Surrogate time series with subject-matched spatial and temporal autocorrelation capture nearly all reliable individual and regional variation in these topology measures. Network topology changes during aging are driven by spatial autocorrelation, and multiple serotonergic drugs causally induce the same topographic change in temporal autocorrelation. This reductionistic interpretation of widely used complexity measures may help link them to neurobiology. Individual variation in fMRI-derived brain networks is reproduced in a model using only the smoothness (autocorrelation) of the fMRI time series. Smoothness has implication for aging and can be causally manipulated by psychedelic serotonergic drugs.","datePublished":"2023-04-24T00:00:00Z","dateModified":"2023-04-24T00:00:00Z","pageStart":"867","pageEnd":"878","sameAs":"https://doi.org/10.1038/s41593-023-01299-3","keywords":["Cognitive neuroscience","Computational neuroscience","Functional magnetic resonance imaging","Biomedicine","general","Neurosciences","Behavioral Sciences","Biological Techniques","Neurobiology","Animal Genetics and Genomics"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig5_HTML.png"],"isPartOf":{"name":"Nature Neuroscience","issn":["1546-1726","1097-6256"],"volumeNumber":"26","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Maxwell Shinn","url":"http://orcid.org/0000-0002-7424-4230","affiliation":[{"name":"Yale University","address":{"name":"Interdepartmental Neuroscience Program, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yale University","address":{"name":"Department of Psychiatry, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Amber Hu","affiliation":[{"name":"Yale University","address":{"name":"Yale College, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Laurel Turner","affiliation":[{"name":"Yale University","address":{"name":"Yale College, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stephanie Noble","url":"http://orcid.org/0000-0002-4804-5553","affiliation":[{"name":"Yale University","address":{"name":"Department of Radiology and Biomedical Imaging, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Katrin H. Preller","url":"http://orcid.org/0000-0003-0413-7672","affiliation":[{"name":"Yale University","address":{"name":"Department of Psychiatry, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"University Hospital for Psychiatry","address":{"name":"Department of Psychiatry, Psychotherapy and Psychosomatics, University Hospital for Psychiatry, Zurich, Switzerland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jie Lisa Ji","url":"http://orcid.org/0000-0002-6280-9070","affiliation":[{"name":"Yale University","address":{"name":"Department of Psychiatry, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Flora Moujaes","affiliation":[{"name":"University Hospital for Psychiatry","address":{"name":"Department of Psychiatry, Psychotherapy and Psychosomatics, University Hospital for Psychiatry, Zurich, Switzerland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sophie Achard","url":"http://orcid.org/0000-0001-9448-1199","affiliation":[{"name":"LJK","address":{"name":"University of Grenoble Alpes, CNRS, Inria, Grenoble INP, LJK, Grenoble, France","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Dustin Scheinost","url":"http://orcid.org/0000-0002-6301-1167","affiliation":[{"name":"Yale University","address":{"name":"Interdepartmental Neuroscience Program, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yale University","address":{"name":"Department of Radiology and Biomedical Imaging, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"R. Todd Constable","url":"http://orcid.org/0000-0001-5661-9521","affiliation":[{"name":"Yale University","address":{"name":"Interdepartmental Neuroscience Program, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yale University","address":{"name":"Department of Radiology and Biomedical Imaging, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"John H. Krystal","url":"http://orcid.org/0000-0001-6952-1726","affiliation":[{"name":"Yale University","address":{"name":"Department of Psychiatry, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Franz X. Vollenweider","url":"http://orcid.org/0000-0001-9053-6164","affiliation":[{"name":"University Hospital for Psychiatry","address":{"name":"Department of Psychiatry, Psychotherapy and Psychosomatics, University Hospital for Psychiatry, Zurich, Switzerland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Daeyeol Lee","affiliation":[{"name":"Johns Hopkins University","address":{"name":"Zanvyl Krieger Mind/Brain Institute, Johns Hopkins University, Baltimore, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Johns Hopkins University","address":{"name":"Kavli Discovery Neuroscience Institute, Johns Hopkins University, Baltimore, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Johns Hopkins University","address":{"name":"Department of Psychological and Brain Sciences, Johns Hopkins University, Baltimore, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Johns Hopkins University","address":{"name":"Department of Neuroscience, Johns Hopkins University, Baltimore, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alan Anticevic","affiliation":[{"name":"Yale University","address":{"name":"Interdepartmental Neuroscience Program, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yale University","address":{"name":"Department of Psychiatry, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Edward T. Bullmore","affiliation":[{"name":"University of Cambridge","address":{"name":"Department of Psychiatry, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"John D. Murray","url":"http://orcid.org/0000-0003-4115-8181","affiliation":[{"name":"Yale University","address":{"name":"Interdepartmental Neuroscience Program, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yale University","address":{"name":"Department of Psychiatry, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yale University","address":{"name":"Department of Physics, Yale University, New Haven, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"john.murray@yale.edu","@type":"Person"}],"isAccessibleForFree":false,"hasPart":{"isAccessibleForFree":false,"cssSelector":".main-content","@type":"WebPageElement"},"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41593-023-01299-3">
    
    
    <meta name="journal_id" content="41593"/>
    <meta name="dc.title" content="Functional brain networks reflect spatial and temporal autocorrelation"/>
    <meta name="dc.source" content="Nature Neuroscience 2023 26:5"/>
    <meta name="dc.format" content="text/html"/>
    <meta name="dc.publisher" content="Nature Publishing Group"/>
    <meta name="dc.date" content="2023-04-24"/>
    <meta name="dc.type" content="OriginalPaper"/>
    <meta name="dc.language" content="En"/>
    <meta name="dc.copyright" content="2023 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rights" content="2023 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="dc.description" content="High-throughput experimental methods in neuroscience have led to an explosion of techniques for measuring complex interactions and multi-dimensional patterns. However, whether sophisticated measures of emergent phenomena can be traced back to simpler, low-dimensional statistics is largely unknown. To explore this question, we examined resting-state functional magnetic resonance imaging (rs-fMRI) data using complex topology measures from network neuroscience. Here we show that spatial and temporal autocorrelation are reliable statistics that explain numerous measures of network topology. Surrogate time series with subject-matched spatial and temporal autocorrelation capture nearly all reliable individual and regional variation in these topology measures. Network topology changes during aging are driven by spatial autocorrelation, and multiple serotonergic drugs causally induce the same topographic change in temporal autocorrelation. This reductionistic interpretation of widely used complexity measures may help link them to neurobiology. Individual variation in fMRI-derived brain networks is reproduced in a model using only the smoothness (autocorrelation) of the fMRI time series. Smoothness has implication for aging and can be causally manipulated by psychedelic serotonergic drugs."/>
    <meta name="prism.issn" content="1546-1726"/>
    <meta name="prism.publicationName" content="Nature Neuroscience"/>
    <meta name="prism.publicationDate" content="2023-04-24"/>
    <meta name="prism.volume" content="26"/>
    <meta name="prism.number" content="5"/>
    <meta name="prism.section" content="OriginalPaper"/>
    <meta name="prism.startingPage" content="867"/>
    <meta name="prism.endingPage" content="878"/>
    <meta name="prism.copyright" content="2023 The Author(s), under exclusive licence to Springer Nature America, Inc."/>
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com"/>
    <meta name="prism.url" content="https://www.nature.com/articles/s41593-023-01299-3"/>
    <meta name="prism.doi" content="doi:10.1038/s41593-023-01299-3"/>
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41593-023-01299-3.pdf"/>
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41593-023-01299-3"/>
    <meta name="citation_journal_title" content="Nature Neuroscience"/>
    <meta name="citation_journal_abbrev" content="Nat Neurosci"/>
    <meta name="citation_publisher" content="Nature Publishing Group"/>
    <meta name="citation_issn" content="1546-1726"/>
    <meta name="citation_title" content="Functional brain networks reflect spatial and temporal autocorrelation"/>
    <meta name="citation_volume" content="26"/>
    <meta name="citation_issue" content="5"/>
    <meta name="citation_publication_date" content="2023/05"/>
    <meta name="citation_online_date" content="2023/04/24"/>
    <meta name="citation_firstpage" content="867"/>
    <meta name="citation_lastpage" content="878"/>
    <meta name="citation_article_type" content="Article"/>
    <meta name="citation_language" content="en"/>
    <meta name="dc.identifier" content="doi:10.1038/s41593-023-01299-3"/>
    <meta name="DOI" content="10.1038/s41593-023-01299-3"/>
    <meta name="size" content="525723"/>
    <meta name="citation_doi" content="10.1038/s41593-023-01299-3"/>
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41593-023-01299-3&amp;api_key="/>
    <meta name="description" content="High-throughput experimental methods in neuroscience have led to an explosion of techniques for measuring complex interactions and multi-dimensional patterns. However, whether sophisticated measures of emergent phenomena can be traced back to simpler, low-dimensional statistics is largely unknown. To explore this question, we examined resting-state functional magnetic resonance imaging (rs-fMRI) data using complex topology measures from network neuroscience. Here we show that spatial and temporal autocorrelation are reliable statistics that explain numerous measures of network topology. Surrogate time series with subject-matched spatial and temporal autocorrelation capture nearly all reliable individual and regional variation in these topology measures. Network topology changes during aging are driven by spatial autocorrelation, and multiple serotonergic drugs causally induce the same topographic change in temporal autocorrelation. This reductionistic interpretation of widely used complexity measures may help link them to neurobiology. Individual variation in fMRI-derived brain networks is reproduced in a model using only the smoothness (autocorrelation) of the fMRI time series. Smoothness has implication for aging and can be causally manipulated by psychedelic serotonergic drugs."/>
    <meta name="dc.creator" content="Shinn, Maxwell"/>
    <meta name="dc.creator" content="Hu, Amber"/>
    <meta name="dc.creator" content="Turner, Laurel"/>
    <meta name="dc.creator" content="Noble, Stephanie"/>
    <meta name="dc.creator" content="Preller, Katrin H."/>
    <meta name="dc.creator" content="Ji, Jie Lisa"/>
    <meta name="dc.creator" content="Moujaes, Flora"/>
    <meta name="dc.creator" content="Achard, Sophie"/>
    <meta name="dc.creator" content="Scheinost, Dustin"/>
    <meta name="dc.creator" content="Constable, R. Todd"/>
    <meta name="dc.creator" content="Krystal, John H."/>
    <meta name="dc.creator" content="Vollenweider, Franz X."/>
    <meta name="dc.creator" content="Lee, Daeyeol"/>
    <meta name="dc.creator" content="Anticevic, Alan"/>
    <meta name="dc.creator" content="Bullmore, Edward T."/>
    <meta name="dc.creator" content="Murray, John D."/>
    <meta name="dc.subject" content="Cognitive neuroscience"/>
    <meta name="dc.subject" content="Computational neuroscience"/>
    <meta name="dc.subject" content="Functional magnetic resonance imaging"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Opportunities and limitations of intrinsic functional connectivity MRI; citation_author=RL Buckner, FM Krienen, BTT Yeo; citation_volume=16; citation_publication_date=2013; citation_pages=832-837; citation_doi=10.1038/nn.3423; citation_id=CR1"/>
    <meta name="citation_reference" content="citation_journal_title=Trends Cogn. Sci.; citation_title=Functional connectomics from resting-state fMRI; citation_author=SM Smith; citation_volume=17; citation_publication_date=2013; citation_pages=666-682; citation_doi=10.1016/j.tics.2013.09.016; citation_id=CR2"/>
    <meta name="citation_reference" content="Fornito, A., Zalesky, A. &amp; Bullmore, E. Fundamentals of Brain Network Analysis (Academic Press, 2016)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Complex network measures of brain connectivity: uses and interpretations; citation_author=M Rubinov, O Sporns; citation_volume=52; citation_publication_date=2010; citation_pages=1059-1069; citation_doi=10.1016/j.neuroimage.2009.10.003; citation_id=CR4"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Changes in structural and functional connectivity among resting-state networks across the human lifespan; citation_author=RF Betzel; citation_volume=102; citation_publication_date=2014; citation_pages=345-357; citation_doi=10.1016/j.neuroimage.2014.07.067; citation_id=CR5"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Effects of aging on functional and structural brain connectivity; citation_author=JS Damoiseaux; citation_volume=160; citation_publication_date=2017; citation_pages=32-40; citation_doi=10.1016/j.neuroimage.2017.01.077; citation_id=CR6"/>
    <meta name="citation_reference" content="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=LSD alters dynamic integration and segregation in the human brain; citation_author=AI Luppi; citation_volume=227; citation_publication_date=2021; citation_pages=117653; citation_doi=10.1016/j.neuroimage.2020.117653; citation_id=CR8"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Resting-state fMRI confounds and cleanup; citation_author=K Murphy, RM Birn, PA Bandettini; citation_volume=80; citation_publication_date=2013; citation_pages=349-359; citation_doi=10.1016/j.neuroimage.2013.04.001; citation_id=CR9"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Analysis of fMRI time-series revisited; citation_author=KJ Friston; citation_volume=2; citation_publication_date=1995; citation_pages=45-53; citation_doi=10.1006/nimg.1995.1007; citation_id=CR10"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Generative modeling of brain maps with spatial autocorrelation; citation_author=JB Burt, M Helmer, M Shinn, A Anticevic, JD Murray; citation_volume=220; citation_publication_date=2020; citation_pages=117038; citation_doi=10.1016/j.neuroimage.2020.117038; citation_id=CR11"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=Timescales of intrinsic BOLD signal dynamics and functional connectivity in pharmacologic and neuropathologic states of unconsciousness; citation_author=Z Huang, X Liu, GA Mashour, AG Hudetz; citation_volume=38; citation_publication_date=2018; citation_pages=2304-2317; citation_doi=10.1523/JNEUROSCI.2545-17.2018; citation_id=CR12"/>
    <meta name="citation_reference" content="citation_journal_title=Chaos; citation_title=Structural connectome topology relates to regional BOLD signal dynamics in the mouse brain; citation_author=SS Sethi, V Zerbi, N Wenderoth, A Fornito, BD Fulcher; citation_volume=27; citation_publication_date=2017; citation_pages=047405; citation_doi=10.1063/1.4979281; citation_id=CR13"/>
    <meta name="citation_reference" content="citation_journal_title=Netw. Neurosci.; citation_title=Timescales of spontaneous fMRI fluctuations relate to structural connectivity in the brain; citation_author=J Fallon; citation_volume=4; citation_publication_date=2020; citation_pages=788-806; citation_doi=10.1162/netn_a_00151; citation_id=CR14"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Autocorrelation structure in the macaque dorsolateral, but not orbital or polar, prefrontal cortex predicts response-coding strength in a visually cued strategy task; citation_author=V Fascianelli, S Tsujimoto, E Marcos, A Genovesio; citation_volume=29; citation_publication_date=2019; citation_pages=230-241; citation_doi=10.1093/cercor/bhx321; citation_id=CR15"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci. Methods; citation_title=Autoconnectivity: A new perspective on human brain function; citation_author=MR Arbabshirani; citation_volume=323; citation_publication_date=2019; citation_pages=68-76; citation_doi=10.1016/j.jneumeth.2019.03.015; citation_id=CR16"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Slow cortical dynamics and the accumulation of information over long timescales; citation_author=CJ Honey; citation_volume=76; citation_publication_date=2012; citation_pages=423-434; citation_doi=10.1016/j.neuron.2012.08.011; citation_id=CR17"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Spatial embedding of structural similarity in the cerebral cortex; citation_author=HF Song, H Kennedy, X-J Wang; citation_volume=111; citation_publication_date=2014; citation_pages=16580-16585; citation_doi=10.1073/pnas.1414153111; citation_id=CR18"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A multi-modal parcellation of human cerebral cortex; citation_author=MF Glasser; citation_volume=536; citation_publication_date=2016; citation_pages=171-178; citation_doi=10.1038/nature18933; citation_id=CR19"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Influences on the test&#8211;retest reliability of functional connectivity MRI and its relationship with behavioral utility; citation_author=S Noble; citation_volume=27; citation_publication_date=2017; citation_pages=5415-5429; citation_doi=10.1093/cercor/bhx230; citation_id=CR20"/>
    <meta name="citation_reference" content="Shafto, M. A. et al. The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) study protocol: a cross-sectional, lifespan, multidisciplinary examination of healthy cognitive ageing. BMC Neurol. 14, 204 (2014)."/>
    <meta name="citation_reference" content="citation_journal_title=J. Stat. Mech.; citation_title=The correlation of metrics in complex networks with applications in functional brain networks; citation_author=C Li, H Wang, W Haan, CJ Stam, PV Mieghem; citation_volume=2011; citation_publication_date=2011; citation_pages=P11018; citation_doi=10.1088/1742-5468/2011/11/P11018; citation_id=CR22"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Simple models of human brain functional networks; citation_author=PE V&#233;rtes; citation_volume=109; citation_publication_date=2012; citation_pages=5868-5873; citation_doi=10.1073/pnas.1111738109; citation_id=CR23"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The contribution of geometry to the human connectome; citation_author=JA Roberts; citation_volume=124; citation_publication_date=2016; citation_pages=379-393; citation_doi=10.1016/j.neuroimage.2015.09.009; citation_id=CR24"/>
    <meta name="citation_reference" content="citation_journal_title=Am. J. Ment. Defic.; citation_title=Developing criteria for establishing interrater reliability of specific items: applications to assessment of adaptive behavior; citation_author=DV Cicchetti, SA Sparrow; citation_volume=86; citation_publication_date=1981; citation_pages=127-137; citation_id=CR25"/>
    <meta name="citation_reference" content="citation_journal_title=J. Chiropr. Med.; citation_title=A guideline of selecting and reporting intraclass correlation coefficients for reliability research; citation_author=TK Koo, MY Li; citation_volume=15; citation_publication_date=2016; citation_pages=155-163; citation_doi=10.1016/j.jcm.2016.02.012; citation_id=CR26"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=A hierarchy of intrinsic timescales across primate cortex; citation_author=JD Murray; citation_volume=17; citation_publication_date=2014; citation_pages=1661-1663; citation_doi=10.1038/nn.3862; citation_id=CR27"/>
    <meta name="citation_reference" content="Raut, R. V., Snyder, A. Z. &amp; Raichle, M. E. Hierarchical dynamics as a macroscopic organizing principle of the human brain. Proc. Natl Acad. Sci. USA 117, 20890&#8211;20897 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Impact of autocorrelation on functional connectivity; citation_author=MR Arbabshirani; citation_volume=102; citation_publication_date=2014; citation_pages=294-308; citation_doi=10.1016/j.neuroimage.2014.07.045; citation_id=CR29"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=On the use of correlation as a measure of network connectivity; citation_author=A Zalesky, A Fornito, E Bullmore; citation_volume=60; citation_publication_date=2012; citation_pages=2096-2106; citation_doi=10.1016/j.neuroimage.2012.02.001; citation_id=CR30"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Effective degrees of freedom of the Pearson&#8217;s correlation coefficient under autocorrelation; citation_author=S Afyouni, SM Smith, TE Nichols; citation_volume=199; citation_publication_date=2019; citation_pages=609-625; citation_doi=10.1016/j.neuroimage.2019.05.011; citation_id=CR31"/>
    <meta name="citation_reference" content="Shafiei, G. et al. Topographic gradients of intrinsic dynamics across neocortex. eLife 9, e62116 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=PLoS Biol.; citation_title=Individual human brain areas can be identified from their characteristic spectral activation fingerprints; citation_author=A Keitel, J Gross; citation_volume=14; citation_publication_date=2016; citation_pages=e1002498; citation_doi=10.1371/journal.pbio.1002498; citation_id=CR33"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity; citation_author=ES Finn; citation_volume=18; citation_publication_date=2015; citation_pages=1664-1671; citation_doi=10.1038/nn.4135; citation_id=CR34"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Evaluating the replicability, specificity, and generalizability of connectome fingerprints; citation_author=L Waller; citation_volume=158; citation_publication_date=2017; citation_pages=371-377; citation_doi=10.1016/j.neuroimage.2017.07.016; citation_id=CR35"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Linking human brain local activity fluctuations to structural and functional network architectures; citation_author=A Baria; citation_volume=73; citation_publication_date=2013; citation_pages=144-155; citation_doi=10.1016/j.neuroimage.2013.01.072; citation_id=CR36"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=A decade of test&#8211;retest reliability of functional connectivity: a systematic review and meta-analysis; citation_author=S Noble, D Scheinost, RT Constable; citation_volume=203; citation_publication_date=2019; citation_pages=116157; citation_doi=10.1016/j.neuroimage.2019.116157; citation_id=CR37"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci. Methods; citation_title=Functional connectivity change as shared signal dynamics; citation_author=MW Cole, GJ Yang, JD Murray, G Repov&#353;, A Anticevic; citation_volume=259; citation_publication_date=2016; citation_pages=22-39; citation_doi=10.1016/j.jneumeth.2015.11.011; citation_id=CR38"/>
    <meta name="citation_reference" content="citation_journal_title=J. Neurosci.; citation_title=A resilient, low-frequency, small-world human brain functional network with highly connected association cortical hubs; citation_author=S Achard; citation_volume=26; citation_publication_date=2006; citation_pages=63-72; citation_doi=10.1523/JNEUROSCI.3874-05.2006; citation_id=CR39"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Generative models of the human connectome; citation_author=RF Betzel; citation_volume=124; citation_publication_date=2016; citation_pages=1054-1064; citation_doi=10.1016/j.neuroimage.2015.09.041; citation_id=CR40"/>
    <meta name="citation_reference" content="citation_journal_title=Neurosci. Biobehav. Rev.; citation_title=Diffusion tensor imaging and aging; citation_author=EV Sullivan, A Pfefferbaum; citation_volume=30; citation_publication_date=2006; citation_pages=749-61; citation_doi=10.1016/j.neubiorev.2006.06.002; citation_id=CR41"/>
    <meta name="citation_reference" content="Watanabe, T., Rees, G. &amp; Masuda, N. Atypical intrinsic neural timescale in autism. eLife 8, e42256 (2019)."/>
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiatry; citation_title=Psilocybin induces time-dependent changes in global functional connectivity; citation_author=KH Preller; citation_volume=88; citation_publication_date=2020; citation_pages=197-207; citation_doi=10.1016/j.biopsych.2019.12.027; citation_id=CR43"/>
    <meta name="citation_reference" content="citation_journal_title=Clin. Pharmacokinet.; citation_title=Clinical pharmacokinetics of ketanserin; citation_author=B Persson, J Heykants, T Hedner; citation_volume=20; citation_publication_date=1991; citation_pages=263-279; citation_doi=10.2165/00003088-199120040-00002; citation_id=CR44"/>
    <meta name="citation_reference" content="citation_journal_title=Brain Imaging Behav.; citation_title=Understanding variability in the BOLD signal and why it matters for aging; citation_author=CL Grady, DD Garrett; citation_volume=8; citation_publication_date=2013; citation_pages=274-283; citation_doi=10.1007/s11682-013-9253-0; citation_id=CR45"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Rapid reconfiguration of the functional connectome after chemogenetic locus coeruleus activation; citation_author=V Zerbi; citation_volume=103; citation_publication_date=2019; citation_pages=702&#8211;718; citation_doi=10.1016/j.neuron.2019.05.034; citation_id=CR46"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Dopamine signaling modulates the stability and integration of intrinsic brain networks; citation_author=G Shafiei; citation_volume=29; citation_publication_date=2018; citation_pages=397-409; citation_doi=10.1093/cercor/bhy264; citation_id=CR47"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=The basal forebrain regulates global resting-state fMRI fluctuations; citation_author=J Turchi; citation_volume=97; citation_publication_date=2018; citation_pages=940&#8211;952; citation_doi=10.1016/j.neuron.2018.01.032; citation_id=CR48"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Filtering induces correlation in fMRI resting state data; citation_author=CE Davey, DB Grayden, GF Egan, LA Johnston; citation_volume=64; citation_publication_date=2013; citation_pages=728-740; citation_doi=10.1016/j.neuroimage.2012.08.022; citation_id=CR49"/>
    <meta name="citation_reference" content="Cantwell, G. T. et al. Thresholding normally distributed data creates complex networks. Phys. Rev. E 101, 062302 (2020)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=A cortical hierarchy of localized and distributed processes revealed via dissociation of task activations, connectivity changes, and intrinsic timescales; citation_author=T Ito, LJ Hearne, MW Cole; citation_volume=221; citation_publication_date=2020; citation_pages=117141; citation_doi=10.1016/j.neuroimage.2020.117141; citation_id=CR51"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Resting-state &#8216;physiological networks&#8217;; citation_author=JE Chen; citation_volume=213; citation_publication_date=2020; citation_pages=116707; citation_doi=10.1016/j.neuroimage.2020.116707; citation_id=CR52"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Ultra-slow oscillations in fMRI and resting-state connectivity: neuronal and vascular contributions and technical confounds; citation_author=PJ Drew, C Mateo, KL Turner, X Yu, D Kleinfeld; citation_volume=107; citation_publication_date=2020; citation_pages=782-804; citation_doi=10.1016/j.neuron.2020.07.020; citation_id=CR53"/>
    <meta name="citation_reference" content="Baracchini, G. et al. Inter-regional BOLD signal variability is an organizational feature of functional brain networks. Neuroimage 237, 118149 (2021)."/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=How does an fMRI voxel sample the neuronal activity pattern: compact-kernel or complex spatiotemporal filter?; citation_author=N Kriegeskorte, R Cusack, P Bandettini; citation_volume=49; citation_publication_date=2010; citation_pages=1965-1976; citation_doi=10.1016/j.neuroimage.2009.09.059; citation_id=CR55"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=BOLD hemodynamic response function changes significantly with healthy aging; citation_author=KL West; citation_volume=188; citation_publication_date=2019; citation_pages=198-207; citation_doi=10.1016/j.neuroimage.2018.12.012; citation_id=CR56"/>
    <meta name="citation_reference" content="citation_journal_title=Prog. Neurobiol.; citation_title=Serotonin in the regulation of brain microcirculation; citation_author=Z Cohen, G Bonvento, P Lacombe, E Hamel; citation_volume=50; citation_publication_date=1996; citation_pages=335-362; citation_doi=10.1016/S0301-0082(96)00033-0; citation_id=CR57"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Challenges in measuring individual differences in functional connectivity using fMRI: the case of healthy aging; citation_author=L Geerligs, KA Tsvetanov, RN Henson; citation_volume=38; citation_publication_date=2017; citation_pages=4125-4156; citation_doi=10.1002/hbm.23653; citation_id=CR58"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Predicting human resting-state functional connectivity from structural connectivity; citation_author=CJ Honey; citation_volume=106; citation_publication_date=2009; citation_pages=2035-2040; citation_doi=10.1073/pnas.0811168106; citation_id=CR59"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Parallel interdigitated distributed networks within the individual estimated by intrinsic functional connectivity; citation_author=RM Braga, RL Buckner; citation_volume=95; citation_publication_date=2017; citation_pages=457&#8211;471; citation_doi=10.1016/j.neuron.2017.06.038; citation_id=CR60"/>
    <meta name="citation_reference" content="citation_journal_title=Cereb. Cortex; citation_title=Individual-specific areal-level parcellations improve functional connectivity prediction of behavior; citation_author=R Kong; citation_volume=31; citation_publication_date=2021; citation_pages=4477-4500; citation_doi=10.1093/cercor/bhab101; citation_id=CR61"/>
    <meta name="citation_reference" content="citation_journal_title=Psychiatry Res.; citation_title=Whole cortical and default mode network mean functional connectivity as potential biomarkers for mild Alzheimer&#8217;s disease; citation_author=MLF Balthazar, BM Campos, AR Franco, BP Damasceno, F Cendes; citation_volume=221; citation_publication_date=2014; citation_pages=37-42; citation_doi=10.1016/j.pscychresns.2013.10.010; citation_id=CR62"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Spatial network surrogates for disentangling complex system structure from spatial embedding of nodes; citation_author=M Wiedermann, JF Donges, J Kurths, RV Donner; citation_volume=93; citation_publication_date=2016; citation_pages=042308; citation_doi=10.1103/PhysRevE.93.042308; citation_id=CR63"/>
    <meta name="citation_reference" content="citation_journal_title=Neuron; citation_title=Hierarchical heterogeneity across human cortex shapes large-scale neural dynamics; citation_author=M Demirta&#351;; citation_volume=101; citation_publication_date=2019; citation_pages=1181&#8211;1194; citation_doi=10.1016/j.neuron.2019.01.017; citation_id=CR64"/>
    <meta name="citation_reference" content="citation_journal_title=Netw. Neurosci.; citation_title=Low-dimensional morphospace of topological motifs in human fMRI brain networks; citation_author=SE Morgan, S Achard, M Termenon, ET Bullmore, PE V&#233;rtes; citation_volume=2; citation_publication_date=2018; citation_pages=285-302; citation_doi=10.1162/netn_a_00038; citation_id=CR65"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=MSM: a new flexible framework for multimodal surface matching; citation_author=EC Robinson; citation_volume=100; citation_publication_date=2014; citation_pages=414-426; citation_doi=10.1016/j.neuroimage.2014.05.069; citation_id=CR66"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Reducing inter-scanner variability of activation in a multicenter fMRI study: role of smoothness equalization; citation_author=L Friedman, GH Glover, D Krenz, V Magnotta; citation_volume=32; citation_publication_date=2006; citation_pages=1656-1668; citation_doi=10.1016/j.neuroimage.2006.03.062; citation_id=CR67"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The impact of image smoothness on intrinsic functional connectivity and head motion confounds; citation_author=D Scheinost, X Papademetris, RT Constable; citation_volume=95; citation_publication_date=2014; citation_pages=13-21; citation_doi=10.1016/j.neuroimage.2014.03.035; citation_id=CR68"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Groupwise whole-brain parcellation from resting-state fMRI data for network node identification; citation_author=X Shen, F Tokoglu, X Papademetris, R Constable; citation_volume=82; citation_publication_date=2013; citation_pages=403-415; citation_doi=10.1016/j.neuroimage.2013.05.081; citation_id=CR69"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) data repository: structural and functional MRI, MEG, and cognitive data from a cross-sectional adult lifespan sample; citation_author=JR Taylor; citation_volume=144; citation_publication_date=2017; citation_pages=262-269; citation_doi=10.1016/j.neuroimage.2015.09.018; citation_id=CR70"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain; citation_author=N Tzourio-Mazoyer; citation_volume=15; citation_publication_date=2002; citation_pages=273-289; citation_doi=10.1006/nimg.2001.0978; citation_id=CR71"/>
    <meta name="citation_reference" content="citation_journal_title=Hum. Brain Mapp.; citation_title=Colored noise and computational inference in neurophysiological (fMRI) time series analysis: resampling methods in time and wavelet domains; citation_author=E Bullmore; citation_volume=12; citation_publication_date=2001; citation_pages=61-78; citation_doi=10.1002/1097-0193(200102)12:2&lt;61::AID-HBM1004&gt;3.0.CO;2-W; citation_id=CR72"/>
    <meta name="citation_reference" content="citation_journal_title=Psychon. Bull. Rev.; citation_title=Estimation and interpretation of 1/ noise in human cognition; citation_author=E-J Wagenmakers, S Farrell, R Ratcliff; citation_volume=11; citation_publication_date=2004; citation_pages=579-615; citation_doi=10.3758/BF03196615; citation_id=CR73"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Fractional Gaussian noise, functional MRI and Alzheimer&#8217;s disease; citation_author=V Maxim; citation_volume=25; citation_publication_date=2005; citation_pages=141-158; citation_doi=10.1016/j.neuroimage.2004.10.044; citation_id=CR74"/>
    <meta name="citation_reference" content="citation_journal_title=Phys. Rev. E; citation_title=Fractal connectivity of long-memory networks; citation_author=S Achard, DS Bassett, A Meyer-Lindenberg, E Bullmore; citation_volume=77; citation_publication_date=2008; citation_pages=036104; citation_doi=10.1103/PhysRevE.77.036104; citation_id=CR75"/>
    <meta name="citation_reference" content="citation_journal_title=J. Time Ser. Anal.; citation_title=Computationally efficient methods for two multivariate fractionally integrated models; citation_author=RJ Sela, CM Hurvich; citation_volume=30; citation_publication_date=2009; citation_pages=631-651; citation_doi=10.1111/j.1467-9892.2009.00631.x; citation_id=CR76"/>
    <meta name="citation_reference" content="citation_journal_title=J. Time Ser. Anal.; citation_title=Multivariate wavelet whittle estimation in long-range dependence; citation_author=S Achard, I Gannaz; citation_volume=37; citation_publication_date=2015; citation_pages=476-512; citation_doi=10.1111/jtsa.12170; citation_id=CR77"/>
    <meta name="citation_reference" content="Lu, Z. Analysis of stationary and non-stationary long memory processes: estimation, applications and forecast. PhD thesis, &#201;cole normale sup&#233;rieure de Cachan - ENS Cachan (2019); 
                  https://theses.hal.science/tel-00422376/document
                  
                "/>
    <meta name="citation_reference" content="citation_journal_title=Phys. A; citation_title=The sample autocorrelation function and the detection of long-memory processes; citation_author=H Hassani, N Leonenko, K Patterson; citation_volume=391; citation_publication_date=2012; citation_pages=6367-6379; citation_doi=10.1016/j.physa.2012.07.062; citation_id=CR79"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Early-warning signals for critical transitions; citation_author=M Scheffer; citation_volume=461; citation_publication_date=2009; citation_pages=53-59; citation_doi=10.1038/nature08227; citation_id=CR80"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Neurosci.; citation_title=Complex brain networks: graph theoretical analysis of structural and functional systems; citation_author=E Bullmore, O Sporns; citation_volume=10; citation_publication_date=2009; citation_pages=186-198; citation_doi=10.1038/nrn2575; citation_id=CR81"/>
    <meta name="citation_reference" content="citation_journal_title=Eur. Phys. J. B; citation_title=Hierarchical structure in financial markets; citation_author=R Mantegna; citation_volume=11; citation_publication_date=1999; citation_pages=193-197; citation_doi=10.1007/s100510050929; citation_id=CR82"/>
    <meta name="citation_reference" content="citation_journal_title=Proc. Am. Math. Soc.; citation_title=On the shortest spanning subtree of a graph and the traveling salesman problem; citation_author=JB Kruskal; citation_volume=7; citation_publication_date=1956; citation_pages=48-48; citation_doi=10.1090/S0002-9939-1956-0078686-7; citation_id=CR83"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Reliability of graph analysis of resting state fMRI using test-retest dataset from the Human Connectome Project; citation_author=M Termenon, A Jaillard, C Delon-Martin, S Achard; citation_volume=142; citation_publication_date=2016; citation_pages=172-187; citation_doi=10.1016/j.neuroimage.2016.05.062; citation_id=CR84"/>
    <meta name="citation_reference" content="citation_journal_title=Psychol. Methods; citation_title=Forming inferences about some intraclass correlation coefficients; citation_author=KO McGraw, SP Wong; citation_volume=1; citation_publication_date=1996; citation_pages=30-46; citation_doi=10.1037/1082-989X.1.1.30; citation_id=CR85"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Considering factors affecting the connectome-based identification process: comment on Waller et al; citation_author=C Horien; citation_volume=169; citation_publication_date=2018; citation_pages=172-175; citation_doi=10.1016/j.neuroimage.2017.12.045; citation_id=CR86"/>
    <meta name="citation_reference" content="citation_journal_title=Neuroimage; citation_title=Regional homogeneity approach to fMRI data analysis; citation_author=Y Zang, T Jiang, Y Lu, Y He, L Tian; citation_volume=22; citation_publication_date=2004; citation_pages=394-400; citation_doi=10.1016/j.neuroimage.2003.12.030; citation_id=CR87"/>
    <meta name="citation_reference" content="citation_journal_title=Biometrics; citation_title=A concordance correlation coefficient to evaluate reproducibility; citation_author=LI-K Lin; citation_volume=45; citation_publication_date=1989; citation_pages=255; citation_doi=10.2307/2532051; citation_id=CR88"/>
    <meta name="citation_reference" content="Newman, M. E. J. Assortative mixing in networks. Phys. Rev. Lett. 89, 208701 (2002)."/>
    <meta name="citation_reference" content="Latora, V. &amp; Marchiori, M. Efficient behavior of small-world networks. Phys. Rev. Lett. 87, 198701 (2001)."/>
    <meta name="citation_reference" content="Newman, M. E. J. &amp; Girvan, M. Finding and evaluating community structure in networks. Phys. Rev. E Stat. Nonlin. Soft Matter Phys. 9, 026113 (2004)."/>
    <meta name="citation_reference" content="citation_journal_title=J. Math. Sociol.; citation_title=A faster algorithm for betweenness centrality; citation_author=U Brandes; citation_volume=25; citation_publication_date=2001; citation_pages=163-177; citation_doi=10.1080/0022250X.2001.9990249; citation_id=CR92"/>
    <meta name="citation_reference" content="citation_journal_title=Astron. Astrophys.; citation_title=On generating power law noise; citation_author=J Timmer, M K&#246;nig; citation_volume=300; citation_publication_date=1995; citation_pages=707-710; citation_id=CR93"/>
    <meta name="citation_reference" content="citation_journal_title=BIT Numer. Math.; citation_title=Numerically stable generation of correlation matrices and their factors; citation_author=PI Davies, NJ Higham; citation_volume=40; citation_publication_date=2000; citation_pages=640-651; citation_doi=10.1023/A:1022384216930; citation_id=CR94"/>
    <meta name="citation_reference" content="citation_journal_title=Physica D; citation_title=Testing for nonlinearity in time series: the method of surrogate data; citation_author=J Theiler, S Eubank, A Longtin, B Galdrikian, JD Farmer; citation_volume=58; citation_publication_date=1992; citation_pages=77-94; citation_doi=10.1016/0167-2789(92)90102-S; citation_id=CR95"/>
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Specificity and stability in topology of protein networks; citation_author=S Maslov, K Sneppen; citation_volume=296; citation_publication_date=2002; citation_pages=910-913; citation_doi=10.1126/science.1065103; citation_id=CR96"/>
    <meta name="citation_reference" content="citation_journal_title=J. Glob. Optim.; citation_title=Differential evolution&#8212;a simple and efficient heuristic for global optimization over continuous spaces; citation_author=R Storn, K Price; citation_volume=11; citation_publication_date=1997; citation_pages=341-359; citation_doi=10.1023/A:1008202821328; citation_id=CR97"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Neurosci.; citation_title=Network neuroscience; citation_author=DS Bassett, O Sporns; citation_volume=20; citation_publication_date=2017; citation_pages=353-364; citation_doi=10.1038/nn.4502; citation_id=CR98"/>
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Array programming with NumPy; citation_author=CR Harris; citation_volume=585; citation_publication_date=2020; citation_pages=357-362; citation_doi=10.1038/s41586-020-2649-2; citation_id=CR99"/>
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=SciPy 1.0: fundamental algorithms for scientific computing in python; citation_author=P Virtanen; citation_volume=17; citation_publication_date=2020; citation_pages=261-272; citation_doi=10.1038/s41592-019-0686-2; citation_id=CR100"/>
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=The connectome visualization utility: software for visualization of human brain networks; citation_author=RA LaPlante, L Douw, W Tang, SM Stufflebeam; citation_volume=9; citation_publication_date=2014; citation_pages=e113838; citation_doi=10.1371/journal.pone.0113838; citation_id=CR101"/>
    <meta name="citation_reference" content="citation_journal_title=J. Open Source Softw.; citation_title=Pingouin: statistics in Python; citation_author=R Vallat; citation_volume=3; citation_publication_date=2018; citation_pages=1026; citation_doi=10.21105/joss.01026; citation_id=CR102"/>
    <meta name="citation_reference" content="Shinn, M. Refinement type contracts for verification of scientific investigative software. in Verified Software. Theories, Tools, and Experiments (eds Chakraborty, S. &amp; Navas, J. A.) 143&#8211;160 (Springer, 2020)."/>
    <meta name="citation_author" content="Shinn, Maxwell"/>
    <meta name="citation_author_institution" content="Interdepartmental Neuroscience Program, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Hu, Amber"/>
    <meta name="citation_author_institution" content="Yale College, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Turner, Laurel"/>
    <meta name="citation_author_institution" content="Yale College, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Noble, Stephanie"/>
    <meta name="citation_author_institution" content="Department of Radiology and Biomedical Imaging, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Preller, Katrin H."/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Psychotherapy and Psychosomatics, University Hospital for Psychiatry, Zurich, Switzerland"/>
    <meta name="citation_author" content="Ji, Jie Lisa"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Moujaes, Flora"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Psychotherapy and Psychosomatics, University Hospital for Psychiatry, Zurich, Switzerland"/>
    <meta name="citation_author" content="Achard, Sophie"/>
    <meta name="citation_author_institution" content="University of Grenoble Alpes, CNRS, Inria, Grenoble INP, LJK, Grenoble, France"/>
    <meta name="citation_author" content="Scheinost, Dustin"/>
    <meta name="citation_author_institution" content="Interdepartmental Neuroscience Program, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Radiology and Biomedical Imaging, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Constable, R. Todd"/>
    <meta name="citation_author_institution" content="Interdepartmental Neuroscience Program, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Radiology and Biomedical Imaging, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Krystal, John H."/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Vollenweider, Franz X."/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Psychotherapy and Psychosomatics, University Hospital for Psychiatry, Zurich, Switzerland"/>
    <meta name="citation_author" content="Lee, Daeyeol"/>
    <meta name="citation_author_institution" content="Zanvyl Krieger Mind/Brain Institute, Johns Hopkins University, Baltimore, USA"/>
    <meta name="citation_author_institution" content="Kavli Discovery Neuroscience Institute, Johns Hopkins University, Baltimore, USA"/>
    <meta name="citation_author_institution" content="Department of Psychological and Brain Sciences, Johns Hopkins University, Baltimore, USA"/>
    <meta name="citation_author_institution" content="Department of Neuroscience, Johns Hopkins University, Baltimore, USA"/>
    <meta name="citation_author" content="Anticevic, Alan"/>
    <meta name="citation_author_institution" content="Interdepartmental Neuroscience Program, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University, New Haven, USA"/>
    <meta name="citation_author" content="Bullmore, Edward T."/>
    <meta name="citation_author_institution" content="Department of Psychiatry, University of Cambridge, Cambridge, UK"/>
    <meta name="citation_author" content="Murray, John D."/>
    <meta name="citation_author_institution" content="Interdepartmental Neuroscience Program, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Psychiatry, Yale University, New Haven, USA"/>
    <meta name="citation_author_institution" content="Department of Physics, Yale University, New Haven, USA"/>
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access"/>
    <meta name="twitter:site" content="@natureneuro"/>
    <meta name="twitter:card" content="summary_large_image"/>
    <meta name="twitter:image:alt" content="Content cover image"/>
    <meta name="twitter:title" content="Functional brain networks reflect spatial and temporal autocorrelation"/>
    <meta name="twitter:description" content="Nature Neuroscience - Individual variation in fMRI-derived brain networks is reproduced in a model using only the smoothness (autocorrelation) of the fMRI time series. Smoothness has implication..."/>
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig1_HTML.png"/>
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41593-023-01299-3"/>
    <meta property="og:type" content="article"/>
    <meta property="og:site_name" content="Nature"/>
    <meta property="og:title" content="Functional brain networks reflect spatial and temporal autocorrelation - Nature Neuroscience"/>
    <meta property="og:description" content="Individual variation in fMRI-derived brain networks is reproduced in a model using only the smoothness (autocorrelation) of the fMRI time series. Smoothness has implication for aging and can be causally manipulated by psychedelic serotonergic drugs."/>
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig1_HTML.png"/>
    

    <script>
        window.eligibleForRa21 = 'true'; 
    </script>
</head>
<body class="article-page">

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    <div class="u-lazy-ad-wrapper u-mbs-0">
            <div class="deferred-placeholder" data-replace="true"
                 data-placeholder="/placeholder/v1/institutionalBanner?bpids=[bpids] #institutional-banner-container"></div>
            <aside class="c-ad c-ad--728x90">
                <div class="c-ad__inner" data-container-type="banner-advert">
                    <p class="c-ad__label">Advertisement</p>
                    
        
            
    <div id="div-gpt-ad-top-1"
         class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide"
         data-ad-type="top"
         data-test="top-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="728x90"
         data-gpt-targeting="type=article;pos=top;artid=s41593-023-01299-3;doi=10.1038/s41593-023-01299-3;subjmeta=116,1627,1647,245,2649,378,631;kwrd=Cognitive+neuroscience,Computational+neuroscience,Functional+magnetic+resonance+imaging">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1375154417&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-023-01299-3%26doi%3D10.1038/s41593-023-01299-3%26subjmeta%3D116,1627,1647,245,2649,378,631%26kwrd%3DCognitive+neuroscience,Computational+neuroscience,Functional+magnetic+resonance+imaging">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=728x90&amp;c=1375154417&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41593-023-01299-3%26doi%3D10.1038/s41593-023-01299-3%26subjmeta%3D116,1627,1647,245,2649,378,631%26kwrd%3DCognitive+neuroscience,Computational+neuroscience,Functional+magnetic+resonance+imaging"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
                </div>
            </aside>
        </div>
    <header class="c-header" id="header" data-header data-track-component="nature-150-split-header" style="border-color:#00928c">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/neuro"
                           data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-0ccc487532906d646419e51f647ce79a.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/neuro/header-880e5942f43b9213989c58a04ab5c8e6.svg" height="32" alt="Nature Neuroscience">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link"
                               data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search"
                                href="#search-menu"
                                data-header-expander
                                data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"/></svg><span>Search</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" href='https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41593-023-01299-3'><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="#explore"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--explore"
                                       data-track="click" data-track-action="open explore expander" data-track-label="button">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="#about-the-journal"
                                       class="c-header__link"
                                       data-header-expander
                                       data-test="menu-button--about-the-journal"
                                       data-track="click" data-track-action="open about the journal expander" data-track-label="button">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                    </a>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="#publish-with-us"
                                           class="c-header__link c-header__link--dropdown-menu"
                                           data-header-expander
                                           data-test="menu-button--publish"
                                           data-track="click" data-track-action="open publish with us expander" data-track-label="button">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"/></svg>
                                        </a>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item">
                                    <a class="c-header__link"
                                       href="https://idp.nature.com/auth/personal/springernature?redirect_uri&#x3D;https%3A%2F%2Fwww.nature.com%2Fmy-account%2Falerts%2Fsubscribe-journal%3Flist-id%3D6"
                                       rel="nofollow"
                                       data-track="click"
                                       data-track-action="Sign up for alerts"
                                       data-track-label="link (desktop site header)"
                                       data-track-external>
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"/></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link"
                                       href="https://www.nature.com/neuro.rss"
                                       data-track="click"
                                       data-track-action="rss feed"
                                       data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature neuroscience"><span itemprop="name">nature neuroscience</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link"
                               href="/neuro/articles?type&#x3D;article" itemprop="item"
                               data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10"
                                         xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z"
                                              fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container"  data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            <div class="c-context-bar u-hide"
                 data-test="context-bar"
                 data-context-bar
                 aria-hidden="true">
                <div class="c-context-bar__container u-container">
                    <div class="c-context-bar__title">
                        Functional brain networks reflect spatial and temporal autocorrelation
                    </div>
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-023-01299-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

                </div>
            </div>
        
        <article lang="en">
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2023-04-24">24 April 2023</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Functional brain networks reflect spatial and temporal autocorrelation</h1>
                    <ul class="c-article-author-list c-article-author-list--short" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Maxwell-Shinn-Aff1-Aff2" data-author-popup="auth-Maxwell-Shinn-Aff1-Aff2" data-author-search="Shinn, Maxwell">Maxwell Shinn</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-7424-4230"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-7424-4230</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Amber-Hu-Aff3" data-author-popup="auth-Amber-Hu-Aff3" data-author-search="Hu, Amber">Amber Hu</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Laurel-Turner-Aff3" data-author-popup="auth-Laurel-Turner-Aff3" data-author-search="Turner, Laurel">Laurel Turner</a><sup class="u-js-hide"><a href="#Aff3">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stephanie-Noble-Aff4" data-author-popup="auth-Stephanie-Noble-Aff4" data-author-search="Noble, Stephanie">Stephanie Noble</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-4804-5553"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4804-5553</a></span><sup class="u-js-hide"><a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Katrin_H_-Preller-Aff2-Aff5" data-author-popup="auth-Katrin_H_-Preller-Aff2-Aff5" data-author-search="Preller, Katrin H.">Katrin H. Preller</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0003-0413-7672"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0413-7672</a></span><sup class="u-js-hide"><a href="#Aff2">2</a>,<a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jie_Lisa-Ji-Aff2" data-author-popup="auth-Jie_Lisa-Ji-Aff2" data-author-search="Ji, Jie Lisa">Jie Lisa Ji</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-6280-9070"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6280-9070</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Flora-Moujaes-Aff5" data-author-popup="auth-Flora-Moujaes-Aff5" data-author-search="Moujaes, Flora">Flora Moujaes</a><sup class="u-js-hide"><a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sophie-Achard-Aff6" data-author-popup="auth-Sophie-Achard-Aff6" data-author-search="Achard, Sophie">Sophie Achard</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-9448-1199"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9448-1199</a></span><sup class="u-js-hide"><a href="#Aff6">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Dustin-Scheinost-Aff1-Aff4" data-author-popup="auth-Dustin-Scheinost-Aff1-Aff4" data-author-search="Scheinost, Dustin">Dustin Scheinost</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0002-6301-1167"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6301-1167</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-R__Todd-Constable-Aff1-Aff4" data-author-popup="auth-R__Todd-Constable-Aff1-Aff4" data-author-search="Constable, R. Todd">R. Todd Constable</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-5661-9521"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-5661-9521</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff4">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John_H_-Krystal-Aff2" data-author-popup="auth-John_H_-Krystal-Aff2" data-author-search="Krystal, John H.">John H. Krystal</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-6952-1726"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6952-1726</a></span><sup class="u-js-hide"><a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Franz_X_-Vollenweider-Aff5" data-author-popup="auth-Franz_X_-Vollenweider-Aff5" data-author-search="Vollenweider, Franz X.">Franz X. Vollenweider</a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0001-9053-6164"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9053-6164</a></span><sup class="u-js-hide"><a href="#Aff5">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Daeyeol-Lee-Aff7-Aff8-Aff9-Aff10" data-author-popup="auth-Daeyeol-Lee-Aff7-Aff8-Aff9-Aff10" data-author-search="Lee, Daeyeol">Daeyeol Lee</a><sup class="u-js-hide"><a href="#Aff7">7</a>,<a href="#Aff8">8</a>,<a href="#Aff9">9</a>,<a href="#Aff10">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alan-Anticevic-Aff1-Aff2" data-author-popup="auth-Alan-Anticevic-Aff1-Aff2" data-author-search="Anticevic, Alan">Alan Anticevic</a><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Edward_T_-Bullmore-Aff11" data-author-popup="auth-Edward_T_-Bullmore-Aff11" data-author-search="Bullmore, Edward T.">Edward T. Bullmore</a><sup class="u-js-hide"><a href="#Aff11">11</a></sup><sup class="u-js-hide"><a href="#na1">na1</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 16 authors for this article" title="Show all 16 authors for this article"></li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John_D_-Murray-Aff1-Aff2-Aff12" data-author-popup="auth-John_D_-Murray-Aff1-Aff2-Aff12" data-author-search="Murray, John D." data-corresp-id="c1">John D. Murray<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">
            <a class="js-orcid" href="http://orcid.org/0000-0003-4115-8181"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4115-8181</a></span><sup class="u-js-hide"><a href="#Aff1">1</a>,<a href="#Aff2">2</a>,<a href="#Aff12">12</a></sup><sup class="u-js-hide"><a href="#na1">na1</a></sup></li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/neuro" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Neuroscience</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>26</b>,<span class="u-visually-hidden">pages </span>867878 (<span data-test="article-publication-year">2023</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">13k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item">
                        <p class="c-article-metrics-bar__count">9 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item">
                            <p class="c-article-metrics-bar__count">112 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41593-023-01299-3/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    <div class="u-js-hide" data-component="article-subject-links">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/cognitive-neuroscience" data-track="click" data-track-action="view subject" data-track-label="link">Cognitive neuroscience</a></li><li class="c-article-subject-list__subject"><a href="/subjects/computational-neuroscience" data-track="click" data-track-action="view subject" data-track-label="link">Computational neuroscience</a></li><li class="c-article-subject-list__subject"><a href="/subjects/functional-magnetic-resonance-imaging" data-track="click" data-track-action="view subject" data-track-label="link">Functional magnetic resonance imaging</a></li>
        </ul>
    </div>

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>High-throughput experimental methods in neuroscience have led to an explosion of techniques for measuring complex interactions and multi-dimensional patterns. However, whether sophisticated measures of emergent phenomena can be traced back to simpler, low-dimensional statistics is largely unknown. To explore this question, we examined resting-state functional magnetic resonance imaging (rs-fMRI) data using complex topology measures from network neuroscience. Here we show that spatial and temporal autocorrelation are reliable statistics that explain numerous measures of network topology. Surrogate time series with subject-matched spatial and temporal autocorrelation capture nearly all reliable individual and regional variation in these topology measures. Network topology changes during aging are driven by spatial autocorrelation, and multiple serotonergic drugs causally induce the same topographic change in temporal autocorrelation. This reductionistic interpretation of widely used complexity measures may help link them to neurobiology.</p></div></div></section>

            <noscript>
                
                    
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-023-01299-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                
            </noscript>

            
                <div class="js-context-bar-sticky-point-mobile">
                    
                        <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-entitled-mobile
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-023-01299-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

                    
                </div>
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-022-06497-w/MediaObjects/41598_2022_6497_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41598-022-06497-w?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 1"
                                           data-track-label="10.1038/s41598-022-06497-w">Telling functional networks apart using ranked network features stability
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">15 February 2022</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Massimiliano Zanin, Bahar Gntekin,  David Papo</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41593-022-01118-1/MediaObjects/41593_2022_1118_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41593-022-01118-1?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 2"
                                           data-track-label="10.1038/s41593-022-01118-1">A parsimonious description of global functional brain organization in three spatiotemporal patterns
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">28 July 2022</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Taylor Bolt, Jason S. Nomi,  Shella D. Keilholz</p>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-023-33441-3/MediaObjects/41598_2023_33441_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link"
                                           itemprop="url"
                                           href="https://www.nature.com/articles/s41598-023-33441-3?fromPaywallRec=false"
                                           data-track="click"
                                           data-track-action="click recommendations inline - 3"
                                           data-track-label="10.1038/s41598-023-33441-3">Within-subject reproducibility varies in multi-modal, longitudinal brain networks
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">24 April 2023</span>
                                    </div>
                                </div>
                                <p class="c-article-recommendations-card__authors u-sans-serif">Johan Nakuci, Nick Wasylyshyn,  Sarah F. Muldoon</p>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'speedy-BootstrappedUCB',
                        timestamp: 1711573803,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>As neuroscience data become more complex, so do the methods used to analyze them. Are sophisticated methods necessary for sophisticated data? We focus here on network analysis of functional connectivity (FC)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Buckner, R. L., Krienen, F. M. &amp; Yeo, B. T. T. Opportunities and limitations of intrinsic functional connectivity MRI. Nat. Neurosci. 16, 832837 (2013)." href="/articles/s41593-023-01299-3#ref-CR1" id="ref-link-section-d30110227e855">1</a></sup>, the matrix of pairwise correlations across brain regions in resting-state functional magnetic resonance imaging (rs-fMRI). FC is an extremely successful tool for understanding brain function<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Smith, S. M. et al. Functional connectomics from resting-state fMRI. Trends Cogn. Sci. 17, 666682 (2013)." href="/articles/s41593-023-01299-3#ref-CR2" id="ref-link-section-d30110227e859">2</a></sup>. Non-linear methods from network neuroscience build upon the success of FC by unlocking properties observable only at higher levels of abstraction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Fornito, A., Zalesky, A. &amp; Bullmore, E. Fundamentals of Brain Network Analysis (Academic Press, 2016)." href="/articles/s41593-023-01299-3#ref-CR3" id="ref-link-section-d30110227e863">3</a></sup>. In these methods, FC is interpreted as a network using graph theory, where nodes represent brain regions and edges represent the correlation in activity between those regions over time. Network analysis enables the study of brain topology at a higher level of abstraction, using statistics called graph metrics to look at properties such as clustering, modular organization and regional influence of brain-wide activity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: uses and interpretations. Neuroimage 52, 10591069 (2010)." href="/articles/s41593-023-01299-3#ref-CR4" id="ref-link-section-d30110227e867">4</a></sup>.</p><p>Network neuroscience has profoundly influenced how we understand the organization of rs-fMRI activity across a variety of different domains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Fornito, A., Zalesky, A. &amp; Bullmore, E. Fundamentals of Brain Network Analysis (Academic Press, 2016)." href="/articles/s41593-023-01299-3#ref-CR3" id="ref-link-section-d30110227e874">3</a></sup>. One such example is aging, a complex process that impacts many regions of the brain differently. A rich literature shows that network topology reflects numerous changes in functional organization as we age, including changes in local and global connectivity across the brain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Betzel, R. F. et al. Changes in structural and functional connectivity among resting-state networks across the human lifespan. Neuroimage 102, 345357 (2014)." href="/articles/s41593-023-01299-3#ref-CR5" id="ref-link-section-d30110227e878">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Damoiseaux, J. S. Effects of aging on functional and structural brain connectivity. Neuroimage 160, 3240 (2017)." href="/articles/s41593-023-01299-3#ref-CR6" id="ref-link-section-d30110227e881">6</a></sup>. In addition to observational and cross-sectional variation in network topology, causal pharmacological manipulations have also been shown to impact within-subject topology, an essential ingredient for translational applications. For example, the psychedelic serotonergic receptor agonist lysergic acid diethylamide (LSD) causally impacts the modular organization and global integration of brain networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)." href="/articles/s41593-023-01299-3#ref-CR7" id="ref-link-section-d30110227e885">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Luppi, A. I. et al. LSD alters dynamic integration and segregation in the human brain. Neuroimage 227, 117653 (2021)." href="/articles/s41593-023-01299-3#ref-CR8" id="ref-link-section-d30110227e888">8</a></sup>. However, because graph theoretical analysis involves several levels of abstraction, it is unclear if the differences in topology within or across individuals are associated with differences in simpler time series properties. This can make it challenging to draw neurobiological conclusions about the brain using changes in MRI-derived graph metrics. Is it possible that lower-level factors, such as basic statistical properties of rs-fMRI time series, may mediate the changes in network topology?</p><p>The basic statistical properties of rs-fMRI time series vary within and across individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Murphy, K., Birn, R. M. &amp; Bandettini, P. A. Resting-state fMRI confounds and cleanup. Neuroimage 80, 349359 (2013)." href="/articles/s41593-023-01299-3#ref-CR9" id="ref-link-section-d30110227e895">9</a></sup>. Two of these properties, spatial autocorrelation (SA) and temporal autocorrelation (TA), are highly present in neuroimaging data and have striking consequences for the statistical analysis of data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Friston, K. J. et al. Analysis of fMRI time-series revisited. Neuroimage 2, 4553 (1995)." href="/articles/s41593-023-01299-3#ref-CR10" id="ref-link-section-d30110227e899">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Burt, J. B., Helmer, M., Shinn, M., Anticevic, A. &amp; Murray, J. D. Generative modeling of brain maps with spatial autocorrelation. Neuroimage 220, 117038 (2020)." href="/articles/s41593-023-01299-3#ref-CR11" id="ref-link-section-d30110227e902">11</a></sup>. Qualitatively, SA captures the idea that two data points have similar values if they are nearby in space, whereas TA captures the idea that they have similar values if they are close in time. As a result, SA and TA represent statistical dependencies across variables, which often manifest in ways that are difficult to predict. Although many different methods exist to measure SA, we quantify it as the rate at which similarity decays across space, detailed below. Likewise, despite many competing ways to measure TA, we use the correlation coefficient of neighboring time points in a time series, also detailed below. SA and TA are attractive features because they reflect a myriad of physical and biological mechanisms, including molecular<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Burt, J. B., Helmer, M., Shinn, M., Anticevic, A. &amp; Murray, J. D. Generative modeling of brain maps with spatial autocorrelation. Neuroimage 220, 117038 (2020)." href="/articles/s41593-023-01299-3#ref-CR11" id="ref-link-section-d30110227e906">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Huang, Z., Liu, X., Mashour, G. A. &amp; Hudetz, A. G. Timescales of intrinsic BOLD signal dynamics and functional connectivity in pharmacologic and neuropathologic states of unconsciousness. J. Neurosci. 38, 23042317 (2018)." href="/articles/s41593-023-01299-3#ref-CR12" id="ref-link-section-d30110227e909">12</a></sup>, structural<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Sethi, S. S., Zerbi, V., Wenderoth, N., Fornito, A. &amp; Fulcher, B. D. Structural connectome topology relates to regional BOLD signal dynamics in the mouse brain. Chaos 27, 047405 (2017)." href="/articles/s41593-023-01299-3#ref-CR13" id="ref-link-section-d30110227e913">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Fallon, J. et al. Timescales of spontaneous fMRI fluctuations relate to structural connectivity in the brain. Netw. Neurosci. 4, 788806 (2020)." href="/articles/s41593-023-01299-3#ref-CR14" id="ref-link-section-d30110227e916">14</a></sup>, activity state<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Fascianelli, V., Tsujimoto, S., Marcos, E. &amp; Genovesio, A. Autocorrelation structure in the macaque dorsolateral, but not orbital or polar, prefrontal cortex predicts response-coding strength in a visually cued strategy task. Cereb. Cortex 29, 230241 (2019)." href="/articles/s41593-023-01299-3#ref-CR15" id="ref-link-section-d30110227e920">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Arbabshirani, M. R. et al. Autoconnectivity: A new perspective on human brain function. J. Neurosci. Methods 323, 6876 (2019)." href="/articles/s41593-023-01299-3#ref-CR16" id="ref-link-section-d30110227e923">16</a></sup> and organizational<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Honey, C. J. et al. Slow cortical dynamics and the accumulation of information over long timescales. Neuron 76, 423434 (2012)." href="/articles/s41593-023-01299-3#ref-CR17" id="ref-link-section-d30110227e928">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Song, H. F., Kennedy, H. &amp; Wang, X.-J. Spatial embedding of structural similarity in the cerebral cortex. Proc. Natl Acad. Sci. USA 111, 1658016585 (2014)." href="/articles/s41593-023-01299-3#ref-CR18" id="ref-link-section-d30110227e931">18</a></sup> properties of the brain. However, despite the importance of SA and TA on the mechanistic and organizational level, and the known consequences for statistical data analysis, it is largely unknown whether individual variation in SA and TA reliably relates to individual variation in more complex measures, such as those from network neuroscience.</p><p>Here we show that two simple time series statistics, SA and TA, explain a large fraction of individual variation in network topology. We demonstrate that variation in SA and TA is reliable across subjects and brain regions and that individual variation in SA and TA corresponds to individual variation in graph metrics.</p><p>To understand the association among SA, TA and graph metrics, we developed a spatiotemporal surrogate time series model that is parameterized only by SA and TA. After fitting the model to time series from individual subjects, we found a large similarity in graph topology between the subject-derived and model-derived networks, demonstrating the direct influence of SA and TA on graph theoretical analysis. To examine how this impacts data analysis in practice, we measure the changes in network topology with aging. Not only are SA and TA more sensitive to age-related changes than graph metrics, but our model can decompose the age-related changes in graph metrics into changes in SA and TA. They are also sensitive to age-related cognitive decline.</p><p>Finally, we show that TA captures within-subject variation and is causally linked to biological processes beyond a graph theoretical framework. Causal pharmacological manipulations with two different serotonergic receptor agonists show the same topographic patterns of decrease in TA, which graph metrics do not capture. Additionally, a serotonin receptor antagonist shows an increase in TA along the same topographic pattern, demonstrating a common but opposite effect from a common but opposite molecular mechanism of action. Overall, we show that SA and TA are important properties of rs-fMRI with close ties to functional networks, offering a reductionistic perspective on complex brain network topology.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Low dimensionality of graph topology measures</h3><p>We found that many measures of graph topology are correlated with each other across subjects in networks constructed from rs-fMRI. We analyzed multiple neuroimaging datasets using diverse methodologies implemented by different teams, focusing on the Human Connectome Project (HCP) dataset<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-023-01299-3#ref-CR19" id="ref-link-section-d30110227e957">19</a></sup>, with validation in the Yale TestRetest (Yale-TRT)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Noble, S. et al. Influences on the testretest reliability of functional connectivity MRI and its relationship with behavioral utility. Cereb. Cortex 27, 54155429 (2017)." href="/articles/s41593-023-01299-3#ref-CR20" id="ref-link-section-d30110227e961">20</a></sup> and the Cambridge Centre for Ageing and Neuroscience (Cam-CAN)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Shafto, M. A. et al. The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) study protocol: a cross-sectional, lifespan, multidisciplinary examination of healthy cognitive ageing. BMC Neurol. 14, 204 (2014)." href="/articles/s41593-023-01299-3#ref-CR21" id="ref-link-section-d30110227e965">21</a></sup> datasets. In line with previous work, networks were constructed by thresholding the FC matrix to maintain only the 10% strongest connections and ensuring that no nodes are disconnected from the rest of the network. We used several graph metrics to quantify network topology: assortativity, global efficiency, transitivity, modularity, mean clustering coefficient and mean local efficiency. We also considered two nodal graph metrics: degree and centrality (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-023-01299-3#Sec14">Methods</a> for details).</p><p>In all datasets, most graph metrics are highly correlated with most other graph metrics across subjects (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1b</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Li, C., Wang, H., de Haan, W., Stam, C. J. &amp; Mieghem, P. V. The correlation of metrics in complex networks with applications in functional brain networks. J. Stat. Mech. 2011, P11018 (2011)." href="/articles/s41593-023-01299-3#ref-CR22" id="ref-link-section-d30110227e978">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Vrtes, P. E. et al. Simple models of human brain functional networks. Proc. Natl Acad. Sci. USA 109, 58685873 (2012)." href="/articles/s41593-023-01299-3#ref-CR23" id="ref-link-section-d30110227e981">23</a></sup>. However, unweighted graph metrics, derived from binarized networks, are also highly correlated with the mean (mean-FC), variance (var-FC) and kurtosis (kurt-FC) of FC (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1b</a>). This is surprising because the FC binarization procedure destroys all explicit information about mean-FC, var-FC and kurt-FC (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-023-01299-3#Sec14">Methods</a> for proof). These observations were consistent across datasets (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig6">1</a>). Thus, some unobserved underlying factors must influence both the statistical moments of FC and the topology of the unweighted graph. In what follows, we demonstrate that SA and TA are two such factors.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="SA and TA are important features of rs-fMRI time series."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: SA and TA are important features of rs-fMRI time series.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="509"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Diagram describing the connectomic pipeline and graph metrics. <b>b</b>, Correlation across all subjects in the HCP dataset, with Bonferroni FWER-corrected two-sided <i>P</i> values. <b>c</b>, Schematic demonstrating the calculation of SA- and SA-<span class="mathjax-tex">\(\infty\)</span> (top) and global TA-<sub>1</sub> (bottom). To compute SA- and SA-<span class="mathjax-tex">\(\infty\)</span>, FC values are binned across distances <i>D</i>, and the curve <span class="mathjax-tex">\({\mathrm{SA}\hbox{-}\infty} +(1-{\mathrm{SA}\hbox{-}\infty} )\exp (-D/{\mathrm{SA}\hbox{-}\uplambda} )\)</span> is fit to the binned data. To compute global TA-<sub>1</sub>, the Pearson correlation is taken between the time series and the time series shifted by one time point. <b>d</b>, Testretest reliability of graph metrics, quantified by ICC. Error bars indicate 95% CI. After each bar is a string of three characters indicating significance: the first indicates significantly less than SA-, the second SA-<span class="mathjax-tex">\(\infty\)</span> and the third global TA-<sub>1</sub>, where # indicates <i>P</i>&lt;0.01, + indicates <i>P</i>&lt;0.05 and  indicates <i>P</i>&gt;0.5, by a two-sided bootstrap resampling procedure. Inset scatter plots show correlation across subjects for two example sessions. <b>e</b>, Correlation across subjects between graph metrics and SA-, SA-<span class="mathjax-tex">\(\infty\)</span> or global TA-<sub>1</sub>. SA- + SA-<span class="mathjax-tex">\(\infty\)</span> and all indicate a cross-validated linear model with two or three terms, respectively, using <i>n</i>=334 unrelated subjects to avoid relatedness confounds and Bonferroni FWER-corrected two-sided <i>P</i> values. <b>f</b>, The brain map depicting regional TA-<sub>1</sub>, averaged across all subjects. <b>g</b>, Distribution of reliability for each brain region. <b>h</b>, Reliability of regional TA-<sub>1</sub> is plotted across brain regions. <b>i</b>, Mean fraction of subjects correctly identified by a fingerprinting analysis. Points indicate identification performance on each of six possible testretest pairs from the four sessions. <b>j</b>, Correlation across regions of regional TA-<sub>1</sub> with nodal graph metrics for each subject. <b>k</b>, The regional TA-<sub>1</sub> for each region, from <b>f</b>, is plotted against its reliability, from <b>h</b>. For all subpanels, unless otherwise indicated, <i>n</i>=883 subjects, * indicates <i>P</i>&lt;0.05 and ** indicates <i>P</i>&lt;0.01, and box plots indicate the median, first and third quartiles and range, with outliers hidden for visualization. FWER, family-wise error rate.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec4">Autocorrelation in space</h3><p>SA is the ubiquitous but often ill-defined phenomenon in neuroscience that nearby regions are more similar than distant regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Song, H. F., Kennedy, H. &amp; Wang, X.-J. Spatial embedding of structural similarity in the cerebral cortex. Proc. Natl Acad. Sci. USA 111, 1658016585 (2014)." href="/articles/s41593-023-01299-3#ref-CR18" id="ref-link-section-d30110227e1274">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Roberts, J. A. et al. The contribution of geometry to the human connectome. Neuroimage 124, 379393 (2016)." href="/articles/s41593-023-01299-3#ref-CR24" id="ref-link-section-d30110227e1277">24</a></sup>. To examine SAs testretest reliabilitythat is, how well SA is preserved across different rs-fMRI sessions from the same subjectwe developed a method to quantify SA on a single-subject level by decomposing it into two components: the rate at which FC falls off with physical distance (SA-) and the average correlation between two distant brain regions (SA-<span class="mathjax-tex">\(\infty\)</span>). Our method bins FC by distance and finds the best fit SA- and SA-<span class="mathjax-tex">\(\infty\)</span> for each subjects FC versus distance curve (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1c</a>), using Euclidean distance to accommodate both hemispheres and subcortex (Equation <a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-023-01299-3#Equ1">1</a>) (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-023-01299-3#Sec14">Methods</a>). Testretest reliability is quantified with intraclass correlation coefficient (ICC), which compares the variability of multiple observations of the same subject to the variability across all subjects, where 1 is perfect reliability and 0 is chance. Historically, different qualitative assessments have been assigned to ICCs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Cicchetti, D. V. &amp; Sparrow, S. A. Developing criteria for establishing interrater reliability of specific items: applications to assessment of adaptive behavior. Am. J. Ment. Defic. 86, 127137 (1981)." href="/articles/s41593-023-01299-3#ref-CR25" id="ref-link-section-d30110227e1325">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Koo, T. K. &amp; Li, M. Y. A guideline of selecting and reporting intraclass correlation coefficients for reliability research. J. Chiropr. Med. 15, 155163 (2016)." href="/articles/s41593-023-01299-3#ref-CR26" id="ref-link-section-d30110227e1328">26</a></sup>, with values from 0.5 to 0.7 usually being called moderate to good.</p><p>We found that SA is both reliable and correlated with graph metrics across subjects. Both SA- and SA-<span class="mathjax-tex">\(\infty\)</span> have high testretest reliability compared to graph metrics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1d</a>) (SA- greater than all metrics, <i>P</i>&lt;0.01, and SA-<span class="mathjax-tex">\(\infty\)</span> greater than unweighted graph metrics, <i>P</i>&lt;0.05, bootstrap resampling). The effect cannot be explained by head motion within the scanner (accounting for motion, partial correlation&gt;0.65,<i>P</i>&lt;10<sup>10</sup> for both SA- and SA-<span class="mathjax-tex">\(\infty\)</span>) or global signal regression (GSR) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig6">1</a>) and is strongest in the HCP dataset, which had the smallest motion and parcel size confounds (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig7">2</a>). Both SA- and SA-<span class="mathjax-tex">\(\infty\)</span> are highly correlated with weighted and unweighted graph metrics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e</a>). To test whether the combined influence of SA- and SA-<span class="mathjax-tex">\(\infty\)</span> is greater than either one alone, we constructed a linear model including SA-, SA-<span class="mathjax-tex">\(\infty\)</span> and a constant term, training it on 50% of subjects selected randomly to fit each graph metric. This model significantly predicts graph metrics on the held-out data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e</a>). These results hold in all datasets (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig6">1</a>). Therefore, SA is reliable and topologically informative.</p><h3 class="c-article__sub-heading" id="Sec5">Autocorrelation in time</h3><p>TA describes the smoothness, or memory, of the rs-fMRI time series over time and is known to vary heterogeneously across brain regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 16611663 (2014)." href="/articles/s41593-023-01299-3#ref-CR27" id="ref-link-section-d30110227e1476">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Raut, R. V., Snyder, A. Z. &amp; Raichle, M. E. Hierarchical dynamics as a macroscopic organizing principle of the human brain. Proc. Natl Acad. Sci. USA 117, 2089020897 (2020)." href="/articles/s41593-023-01299-3#ref-CR28" id="ref-link-section-d30110227e1479">28</a></sup> and influence both FC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Arbabshirani, M. R. et al. Autoconnectivity: A new perspective on human brain function. J. Neurosci. Methods 323, 6876 (2019)." href="/articles/s41593-023-01299-3#ref-CR16" id="ref-link-section-d30110227e1483">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Arbabshirani, M. R. et al. Impact of autocorrelation on functional connectivity. Neuroimage 102, 294308 (2014)." href="/articles/s41593-023-01299-3#ref-CR29" id="ref-link-section-d30110227e1486">29</a></sup> and graph topology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. Neuroimage 60, 20962106 (2012)." href="/articles/s41593-023-01299-3#ref-CR30" id="ref-link-section-d30110227e1490">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Afyouni, S., Smith, S. M. &amp; Nichols, T. E. Effective degrees of freedom of the Pearsons correlation coefficient under autocorrelation. Neuroimage 199, 609625 (2019)." href="/articles/s41593-023-01299-3#ref-CR31" id="ref-link-section-d30110227e1493">31</a></sup>. We quantify TA for each parcellated region in each subject as the Pearson correlation between adjacent time points of the regions time seriesthat is, the lag-1 temporal autocorrelation (TA-<sub>1</sub>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1c</a>). TA-<sub>1</sub> is a simple non-parametric measure of TA. Although TA-<sub>1</sub>, in theory, measures only correlation across a single time point in the time series, in practice, it was effective in measuring correlations at much longer time scales, including parametric estimates of long-memory dynamics (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig8">3</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Shafiei, G. et al. Topographic gradients of intrinsic dynamics across neocortex. eLife 9, e62116 (2020)." href="/articles/s41593-023-01299-3#ref-CR32" id="ref-link-section-d30110227e1510">32</a></sup>. Because pre-processing methodology and TR both influence TA-<sub>1</sub>, comparisons between TA-<sub>1</sub> must be made within a single dataset. TA-<sub>1</sub> can be measured at either the level of individual regions or as an average of these at the whole-brain level, so we refer to these as regional TA-<sub>1</sub> and global TA-<sub>1</sub>, respectively. We observed highest regional TA-<sub>1</sub> in occipital and parietal regions and lowest in limbic regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1f</a>).</p><p>TA is reliable across subjects, both at the whole-brain level and the regional level. At the whole-brain level, we computed each subjects global TA-<sub>1</sub> by averaging regional TA-<sub>1</sub> across all regions and found that global TA-<sub>1</sub> is highly reliable compared to graph metrics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1d</a>). For each region, we also computed the reliability across subjects and found that median regional reliability was higher for regional TA-<sub>1</sub> than for other nodal graph metrics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1g</a>). The reliability of regional TA-<sub>1</sub> varies heterogeneously across the brain (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1h</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Keitel, A. &amp; Gross, J. Individual human brain areas can be identified from their characteristic spectral activation fingerprints. PLoS Biol. 14, e1002498 (2016)." href="/articles/s41593-023-01299-3#ref-CR33" id="ref-link-section-d30110227e1553">33</a></sup>.</p><p>This reliability and heterogeneity suggests that regional TA-<sub>1</sub> could be used to identify individual subjects across the population. To identify subjects, we used a fingerprinting analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 16641671 (2015)." href="/articles/s41593-023-01299-3#ref-CR34" id="ref-link-section-d30110227e1562">34</a></sup>. For a given measure, such as regional TA-<sub>1</sub>, we matched each session of each subject to the session with the highest Pearson correlation in that measure across regions, selecting among all sessions from all subjects. Then, we counted the number of pairs for which both sessions belonged to the same subject. We found that fingerprinting using regional TA-<sub>1</sub> identified the single matching session from the pool of 1,765 sessions in over 62.4% of subjects, compared to 62.6% with a more traditional fingerprinting analysis using FC (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1i</a>). Although this is lower than reported in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 16641671 (2015)." href="/articles/s41593-023-01299-3#ref-CR34" id="ref-link-section-d30110227e1574">34</a></sup>, the number of possible matches was nearly an order of magnitude larger in our dataset and is consistent with previous findings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Waller, L. et al. Evaluating the replicability, specificity, and generalizability of connectome fingerprints. Neuroimage 158, 371377 (2017)." href="/articles/s41593-023-01299-3#ref-CR35" id="ref-link-section-d30110227e1578">35</a></sup>. We compared this to fingerprinting using several nodal graph metrics, which were unable to match the performance (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1i</a>). Similar results were obtained in other datasets (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig6">1</a>). We also tested whether this performance is due to differences in parcel boundaries or reliable inhomogeneity within the parcel by comparing regional TA-<sub>1</sub> to the parcels regional homogeneity. Regional homogeneity showed a moderate correlation with regional TA-<sub>1</sub>, and its ability to identify individual subjects through fingerprinting varied greatly between datasets but had no ability to identify subjects in the HCP dataset (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig9">4</a>). This indicates that regional TA-<sub>1</sub> is reliable enough to identify an individual subject from a population.</p><p>TA-<sub>1</sub> is highly correlated with graph topology at the individual and regional levels. At the individual level, we found a strong correlation between global TA-<sub>1</sub> and graph metrics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e</a>). To test its influence in conjunction with SA, we developed a linear model incorporating global TA-<sub>1</sub>, SA-, SA-<span class="mathjax-tex">\(\infty\)</span> and a constant, training it on 50% of randomly selected subjects to predict each graph metric. This model significantly predicted all graph metrics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e</a>) and almost all graph metrics in the other datasets (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig6">1</a>). At the regional level, for each subject, we computed the Spearman correlation between regional TA-<sub>1</sub> and various graph metrics. We found that both weighted and unweighted nodal graph metrics were highly correlated with regional TA-<sub>1</sub> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1j</a>). Most notably, a nodes degree, or the total number of connections a node makes to other regions, was predicted by regional TA-<sub>1</sub> with a median correlation of 0.89 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1j</a>), and this relationship was not driven by parcel size (median partial correlation 0.83 between degree and regional TA-<sub>1</sub>, accounting for parcel size). Remarkably, this implies that network hubs can be discovered without examining the topology of network<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Afyouni, S., Smith, S. M. &amp; Nichols, T. E. Effective degrees of freedom of the Pearsons correlation coefficient under autocorrelation. Neuroimage 199, 609625 (2019)." href="/articles/s41593-023-01299-3#ref-CR31" id="ref-link-section-d30110227e1649">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Baria, A. et al. Linking human brain local activity fluctuations to structural and functional network architectures. Neuroimage 73, 144155 (2013)." href="/articles/s41593-023-01299-3#ref-CR36" id="ref-link-section-d30110227e1652">36</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec6">Multiple sources of TA</h3><p>Heterogeneity in TA across brain regions can be shaped by multiple underlying factors. One such factor is the regions intrinsic time scale, which measures the rate at which a signal in a region decays over time. Intrinsic time scales vary from region to region and have been previously described as a central factor of brain dynamics in both fMRI and electrophysiology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 16611663 (2014)." href="/articles/s41593-023-01299-3#ref-CR27" id="ref-link-section-d30110227e1664">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Raut, R. V., Snyder, A. Z. &amp; Raichle, M. E. Hierarchical dynamics as a macroscopic organizing principle of the human brain. Proc. Natl Acad. Sci. USA 117, 2089020897 (2020)." href="/articles/s41593-023-01299-3#ref-CR28" id="ref-link-section-d30110227e1667">28</a></sup>. If intrinsic time scale was the primary driver of TA heterogeneity, regions with longer time scales would lead to higher TA. However, brain maps of intrinsic time scale<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 16611663 (2014)." href="/articles/s41593-023-01299-3#ref-CR27" id="ref-link-section-d30110227e1671">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Raut, R. V., Snyder, A. Z. &amp; Raichle, M. E. Hierarchical dynamics as a macroscopic organizing principle of the human brain. Proc. Natl Acad. Sci. USA 117, 2089020897 (2020)." href="/articles/s41593-023-01299-3#ref-CR28" id="ref-link-section-d30110227e1674">28</a></sup> do not correspond well to our brain map of regional TA-<sub>1</sub> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1f</a>). This suggests that other factors may be important in driving TA.</p><p>In addition to intrinsic time scale, the noise level of a brain region also influences its TA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Noble, S., Scheinost, D. &amp; Constable, R. T. A decade of testretest reliability of functional connectivity: a systematic review and meta-analysis. Neuroimage 203, 116157 (2019)." href="/articles/s41593-023-01299-3#ref-CR37" id="ref-link-section-d30110227e1686">37</a></sup>. Decreasing the signal-to-noise ratio or increasing the noise level will decrease TA. More generally, adding white noise to any time series will cause its TA to approach 0 (see <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">Supplementary Information</a> for proof). Thus, because different regions of the brain are affected by different sources and quantities of noise, these differences may also influence TA. In the extreme case where noise is the only driving factor of TA, all regions would have an identical slow intrinsic time scale, and different amounts of noise in each region would determine the regions TA. Consistent with the hypothesis that noise shapes regional TA, we found that regions with the lowest regional TA-<sub>1</sub> show the lowest reliability in regional TA-<sub>1</sub> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1k</a>). By this logic, TA may influence FC by altering the fraction of shared variance between pairs of regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Cole, M. W., Yang, G. J., Murray, J. D., Repov, G. &amp; Anticevic, A. Functional connectivity change as shared signal dynamics. J. Neurosci. Methods 259, 2239 (2016)." href="/articles/s41593-023-01299-3#ref-CR38" id="ref-link-section-d30110227e1701">38</a></sup>.</p><p>In what follows, we used this principle to build a spatiotemporal model for generating surrogate time series with regionally heterogeneous noise. We compare this with the intrinsic time scale with SA model whereby intrinsic time scale is the primary driver of TA. The main practical difference between these models is that, in the spatiotemporal model, reductions in TA have a corresponding reduction in the local effect of SA (due to increased noise), whereas, in the intrinsic time scale with SA model, SA is independent of TA by construction. We also compare the spatiotemporal model with models that use only SA or only TA, testing the importance of the joint action between SA and TA. Finally, we compare the spatiotemporal model to several popular null models from the graph theory literature.</p><h3 class="c-article__sub-heading" id="Sec7">Spatiotemporal model for surrogate time series</h3><p>We designed a spatiotemporal surrogate time series model that, when fit to individual subjects, uses SA and TA to capture individual variation in network topology. The model operates at the level of the parcellated time series, meaning that comparisons may be drawn at multiple levels of the analysis pipeline (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1a</a>). Our model can be summarized in a small number of steps (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2a</a>). First, we generate long-memory (1/<i>f</i><sup><i></i></sup> spectrum) time series that have uniformly high TA. Concurrently, we spatially embed these time series according to brain geometry, introducing SA by increasing the correlation of nearby regions, while preserving the frequency spectra. This is made possible with our correlated spectral sampling algorithm (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-023-01299-3#Sec14">Methods</a>). Then, we add uncorrelated white noise with a region-specific variance, thereby lowering TA heterogeneously. The resulting time series can then be analyzed the same way as rs-fMRI time series. Thus, in our model, all variation in graph topology must be caused by variation in SA and/or TA.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="A spatiotemporal model captures connectome topology."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: A spatiotemporal model captures connectome topology.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="544"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Schematic of our spatiotemporal modeling framework for surrogate time series with SA and regionally heterogeneous TA. <b>b</b>, loglog distribution of the eigenvalues of the FC matrix for an example subject (black) compared to the spatiotemporal model (red). <b>c</b>, Schematic demonstrating Lins concordance, our model fit statistic, on example data. Values close to 1 indicate both a correlation and a close match in value (gray). Values close to 0 indicate either no correlation (pink) or a large squared error (brown). <b>d</b>, Example FC matrices (top) and graphs (bottom) for the original data (black) and for each model (colors). Graphs are visualized using a force-directed layout, which positions topographically neighboring nodes nearby. <b>e</b>, Lins concordance between model and data for each model. Bars represent the mean across the four scanning sessions, and points indicate the Lins concordance between the model and data for each session. For comparison, black indicates Lins concordance between separate sessions from the same subject, where dots indicate pairs of sessions. <b>f</b>, loglog degree distribution for each model compared to the data (black). <b>g</b>, Distribution of Lins concordance of nodal metrics between model and data for each region. Box plots show median, first and third quartiles and range, with outliers hidden for visualization, <i>n</i>=883.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Our model contains two SA parameters that are fit on the single-subject level, corresponding to noiseless SA- (SA-<sup>gen</sup>) and noiseless SA-<span class="mathjax-tex">\(\infty\)</span> (SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup>). We also use the subjects observed regional TA-<sub>1</sub> to determine the variance of noise to add. We fit SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> through optimization to the distribution of eigenvalues of the FC matrix, a property that is not a graph metric yet is well captured by the model (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2b</a>). This parameterization ensures that all variation in graph topology must be caused by variation in SA and/or TA.</p><h3 class="c-article__sub-heading" id="Sec8">Model performance</h3><p>We evaluated the ability of our model to capture weighted and unweighted graph metrics using surrogate time series. In principle, model fit could be assessed using several different criteria, such as the models ability to capture as much variance as possible (for example, using <i>R</i><sup>2</sup>) or, alternatively, by its ability to capture individual variation (for example, using Pearson correlation) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1</a>). Here, we assess model fit using Lins concordance, a stricter criterion that is 1 only when the model captures both variance and individual variation and 0 or negative when either assessment is poor. Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2c</a> demonstrates a schematic comparison among Lins concordance, correlation coefficient and <i>R</i><sup>2</sup> on artificial data. An example subjects FC matrix and network diagram demonstrate that the spatiotemporal model is the only model we tested that produces the complex, asymmetric patterns observed in the data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2d</a>).</p><p>Our model captures important features of graph topology. The model exhibits a high Lins concordance to the data for both weighted and unweighted graph metrics, indicating that it matches both the individual variability as well as the precise values of the graph metrics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2e</a>). The modeldata match is close to the total reliable variability, which we estimate using the Lins concordance between two independent sessions from the same subject (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2e</a>). The high Lins concordance corresponds to a close fit to the diagonal of graph metrics plotted between the model and the data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig3">3</a>). As expected, we also confirmed the model matches the SA and TA of the original subjects (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig10">5</a>). Furthermore, it reproduces the degree distribution (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2f</a>) and also correlates with several nodal graph metrics on the subject level (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2g</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Correlation of model and data graph metrics for all models."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Correlation of model and data graph metrics for all models.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="760"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>For each model, each subjects empirical graph metrics are plotted against the graph metrics predicted by the spatiotemporal model. These scatter plots depict the relationships summarized by Lins concordance in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2e</a> for the HCP dataset. Spearman correlation (<i>r</i><sub><i>s</i></sub>) and Lins concordance (Lin) are inset. * indicates Spearman correlation two-tailed <i>P</i>&lt;0.05, and ** indicates <i>P</i>&lt;0.01.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>To test alternative ways that graph metrics might be related to time series properties, we fit several additional models, depicted graphically in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig11">6</a>. First, we tested whether both SA and TA are necessary within our model. To do this, we fit the spatiotemporal model with TA set to 0 (SA only model) as well as the spatiotemporal model with an infinitely small SA- and a SA-<span class="mathjax-tex">\(\infty\)</span> of 0 (TA only model) and compared their performance to the performance of our spatiotemporal model. Second, we designed a model to test the hypothesis that TA is determined by the regions estimated intrinsic time scale in conjunction with SA (Intrinsic time scale + SA). This model estimates a pink noise 1/<i>f</i> <sup><i></i></sup> exponent for each region, as well as SA- and SA-<span class="mathjax-tex">\(\infty\)</span> for the full brain, and then generates surrogates using correlated spectral sampling (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41593-023-01299-3#Sec14">Methods</a>). Third, we tested whether a full reconstruction of TA at all scales, via the power spectrum amplitudes with randomized phases (Phase randomization), can match our models performance. Likewise, given our findings related to mean-FC and var-FC, we tested a model that matches these two statistics (Zalesky matching), previously shown to exhibit more brain-like network topology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. Neuroimage 60, 20962106 (2012)." href="/articles/s41593-023-01299-3#ref-CR30" id="ref-link-section-d30110227e1976">30</a></sup>. Additionally, we tested a model that matches the networks degree distribution (Edge reshuffle) due to its overwhelming popularity in the graph theory literature. Finally, to confirm whether our excellent fit was driven by our use of eigenvalues in the loss function, we also created an eigensurrogate method for creating surrogate FC matrices. Eigensurrogates preserve the eigenvalue distribution of the FC matrix but randomize the eigenvectors, thereby perfectly duplicating the linear dimensionality of the FC matrix. This tests the null hypothesis that an effect is due to restrictions in dimensionality.</p><p>None of these alternative models captures network topology (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2e</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig3">3</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig10">5</a>) or the degree distribution (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2f</a>) as well as the spatiotemporal model. These results are not affected by the use of geodesic instead of Euclidean distance (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig12">7</a>). Across all other datasets, only the eigensurrogate model on the Yale-TRT dataset showed more explanatory power than the spatiotemporal model (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig13">8</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1</a>). The ability of our spatiotemporal model to generally fit the Yale-TRT and Cam-CAN datasets better than alternative models underscores its generality.</p><h3 class="c-article__sub-heading" id="Sec9">Linking autocorrelation to graph topology</h3><p>Our results show that SA and TA predict graph topology, but establishing this relationship mathematically is difficult due to the non-linearity of constructing and analyzing a thresholded graph. Instead, we attempted to provide an intuition of why SA and TA may influence topology by considering their impact on individual edges. The impact of SA is relatively straightforwardSA increases the mean correlation between nearby regions and, thus, increases the probability of an edge. However, it is not immediately clear how TA might influence topology.</p><p>A statistical argument explains the high correlation between regional TA-<sub>1</sub> and degree (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1j</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Afyouni, S., Smith, S. M. &amp; Nichols, T. E. Effective degrees of freedom of the Pearsons correlation coefficient under autocorrelation. Neuroimage 199, 609625 (2019)." href="/articles/s41593-023-01299-3#ref-CR31" id="ref-link-section-d30110227e2021">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Baria, A. et al. Linking human brain local activity fluctuations to structural and functional network architectures. Neuroimage 73, 144155 (2013)." href="/articles/s41593-023-01299-3#ref-CR36" id="ref-link-section-d30110227e2024">36</a></sup> and shows why strong TA creates hubs. Degree is determined by the number of correlations the node makes that exceed the binarization threshold (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1a</a>). This means that, even if the expected value of the correlation (nodal mean-FC) is low, there may still be several correlations above the threshold if it has a high variance (nodal var-FC). In other words, a high variance may increase the probability of crossing the threshold more so than a high mean. Thus, any process that increases a nodes variance should also increase the nodes degree. Two temporally autocorrelated time series are expected to have a higher variance in their pairwise correlation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Afyouni, S., Smith, S. M. &amp; Nichols, T. E. Effective degrees of freedom of the Pearsons correlation coefficient under autocorrelation. Neuroimage 199, 609625 (2019)." href="/articles/s41593-023-01299-3#ref-CR31" id="ref-link-section-d30110227e2031">31</a></sup> (Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1.5)</a>, and this relationship is reflected in our data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e,j</a>). Thus, for individual nodes, one way that TA drives a nodes degree is by increasing the var-FC. If region A is highly correlated with regions B and C, it is also more likely that B and C are correlated with each other<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. Neuroimage 60, 20962106 (2012)." href="/articles/s41593-023-01299-3#ref-CR30" id="ref-link-section-d30110227e2042">30</a></sup>. This means that high-TA-<sub>1</sub> nodes in the graph are more likely to share an edge<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Afyouni, S., Smith, S. M. &amp; Nichols, T. E. Effective degrees of freedom of the Pearsons correlation coefficient under autocorrelation. Neuroimage 199, 609625 (2019)." href="/articles/s41593-023-01299-3#ref-CR31" id="ref-link-section-d30110227e2048">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Achard, S. A resilient, low-frequency, small-world human brain functional network with highly connected association cortical hubs. J. Neurosci. 26, 6372 (2006)." href="/articles/s41593-023-01299-3#ref-CR39" id="ref-link-section-d30110227e2051">39</a></sup>, creating a clustered network topology.</p><p>We confirmed this reasoning using a two-parameter economical clustering model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Vrtes, P. E. et al. Simple models of human brain functional networks. Proc. Natl Acad. Sci. USA 109, 58685873 (2012)." href="/articles/s41593-023-01299-3#ref-CR23" id="ref-link-section-d30110227e2058">23</a></sup>, which builds graphs directly by probabilistically connecting nodes based on their distance and clustering topology (Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">2)</a>. This model is known to reproduce several topological features of brain networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Vrtes, P. E. et al. Simple models of human brain functional networks. Proc. Natl Acad. Sci. USA 109, 58685873 (2012)." href="/articles/s41593-023-01299-3#ref-CR23" id="ref-link-section-d30110227e2065">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Betzel, R. F. et al. Generative models of the human connectome. Neuroimage 124, 10541064 (2016)." href="/articles/s41593-023-01299-3#ref-CR40" id="ref-link-section-d30110227e2068">40</a></sup> and is convenient because the procedure for constructing networks is distinct from the rs-fMRI pipeline. We found that changes in SA and TA correspond to an increased propensity for short and clustered edges, respectively, in the economical clustering model (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig14">9</a>). In other words, the parameters of a graph-level generative process known to reproduce topological features of rs-fMRI networks are closely linked to the dimensions of network topology spanned by SA and TA. Thus, SA and TA can be interpreted directly in terms of graph topology.</p><h3 class="c-article__sub-heading" id="Sec10">SA in healthy aging</h3><p>We have shown that our surrogate model can trace differences in graph metrics among subjects down to differences in the SA and TA of the time series used to produce them. Therefore, we asked if the opposite is possibleif graph metrics change across a population, is SA or TA responsible for those changes in graph metrics? Due to the rich history of graph theoretical analysis of aging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Betzel, R. F. et al. Changes in structural and functional connectivity among resting-state networks across the human lifespan. Neuroimage 102, 345357 (2014)." href="/articles/s41593-023-01299-3#ref-CR5" id="ref-link-section-d30110227e2083">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Damoiseaux, J. S. Effects of aging on functional and structural brain connectivity. Neuroimage 160, 3240 (2017)." href="/articles/s41593-023-01299-3#ref-CR6" id="ref-link-section-d30110227e2086">6</a></sup>, we looked for age-related changes in graph metrics, and we show that perturbations in the SA-<span class="mathjax-tex">\(\infty\)</span> parameter, but not the SA- or global TA-<sub>1</sub> parameters, lead to a change in graph metrics that mirrors that of aging.</p><p>We analyzed the Cam-CAN dataset, containing cross-sectional rs-fMRI data from over 800 subjects ranging from age 18years to 90years. The relationships among SA, TA and graph metrics were unchanged despite the large variation in age (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig6">1</a>), and our surrogate model remained effective in this dataset (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig13">8</a>). Because motion was highly correlated with age in this dataset (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig7">2</a>), we performed all analyses using partial correlation controlling for motion. Several weighted and unweighted graph metrics are correlated with age (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4</a>), with global efficiency, var-FC and kurt-FC showing significant correlations. Age was positively correlated with SA-, negatively correlated with SA-<span class="mathjax-tex">\(\infty\)</span> and uncorrelated with global TA-<sub>1</sub> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4a</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="SA links functional connectome topology to neurobiology during aging."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: SA links functional connectome topology to neurobiology during aging.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="911"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Partial correlation of graph metrics with age is shown, controlling for motion. Asterisks indicate two-tailed significance of partial correlation (<i>P</i>&lt;0.05, <i>n</i>=652). Error bars show 95% CI. <b>b</b>, The spatiotemporal model can be perturbed to test whether each SA parameter mediates aging. Top: Subject data predict global efficiency and kurt-FC decrease with age, whereas var-FC increases with age. The SA-<span class="mathjax-tex">\(\infty\)</span> and SA- parameters were separately perturbed in the spatiotemporal model according to how they change with age. The mean predicted change of each metric with age is shown if the age-related changes are due to SA-<span class="mathjax-tex">\(\infty\)</span> alone (middle) or SA- alone (bottom). Error bars show standard error across ten runs of the model. The direction (positive or negative) of the change with age corresponds to the data for SA-<span class="mathjax-tex">\(\infty\)</span> but not for SA-. <b>c</b>, Partial correlation of regional TA-<sub>1</sub> with age is shown for each region, partial on motion. <b>d</b>, Mean partial correlation of regional TA-<sub>1</sub> with age across brain regions, partial on motion, <i>n</i>=646 subjects. Box plots indicate the median, first and third quartiles and outlier-excluded range of partial correlation distribution. ** indicates <i>P</i>&lt;0.01 significant partial correlation, after Bonferroni FWER correction. <b>e</b>, Partial correlation of measures with the ACE-R assessment, partial on motion and age. Top: partial correlation with SA and global TA. SA- and SA-<span class="mathjax-tex">\(\infty\)</span> are significant after Bonferroni FWER correction, and global TA-<sub>1</sub> is not. Bottom: Var-FC was the only weighted or unweighted graph metric that was significant after Bonferroni FWER correction. Error bars show 95% CI. ** indicates <i>P</i>&lt;0.01, and * indicates <i>P</i>&lt;0.05, two-sided Wilcoxon signed-rank test with Bonferroni FWER correction. FWER, family-wise error rate.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Our spatiotemporal model can be used to determine the extent to which SA- and SA-<span class="mathjax-tex">\(\infty\)</span> govern the change in network topology. Because both SA- and SA-<span class="mathjax-tex">\(\infty\)</span>have significant partial correlations with age, we ran the model in reverse to understand which of these two parameters mediated the effects of graph metrics on aging. To do this, we perturbed each of these parameters in the direction predicted by aging. If graph metrics increase or decrease in the same direction as they do across age, this provides evidence that the perturbed parameter mediates the effects of aging. Because SA-<span class="mathjax-tex">\(\infty\)</span> decreases with age, we perturbed the model by decreasing SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> to obtain predictions for SA-<span class="mathjax-tex">\(\infty\)</span>-mediated aging. We found that all of the graph metrics that show significant correlation with age change in response to this perturbation, and the direction of this change matches the direction of the change over age (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4b</a>). We also perturbed the model by increasing SA-<sup>gen</sup> to obtain predictions for SA--mediated aging. However, graph metrics showed only a slight change in response to this perturbation, and the change was in the opposite direction as age (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4b</a>). This suggests that the impact of aging on network structure is mediated by changes in SA-<span class="mathjax-tex">\(\infty\)</span> rather than SA-. In other words, the effect of aging on graph metrics is driven by the baseline level of SA at long distances rather than the rate at which SA decays at short distances.</p><p>Because the global TA did not change with age, we asked whether this was also true on a regional level. We computed the partial correlation of age and regional TA-<sub>1</sub> across the brain (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4c</a>). Regional TA-<sub>1</sub> decreased with age in the frontal subnetwork but increased with age in cerebellar regions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4d</a>), suggesting a difference in how these regions change over age. This difference may reflect age-related structural changes along an anterior-posterior gradient<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Sullivan, E. V. &amp; Pfefferbaum, A. Diffusion tensor imaging and aging. Neurosci. Biobehav. Rev. 30, 74961 (2006)." href="/articles/s41593-023-01299-3#ref-CR41" id="ref-link-section-d30110227e2403">41</a></sup>. Despite small effect size, this demonstrates that age-related changes in SA and TA are consistent with age-related changes in graph metrics.</p><h3 class="c-article__sub-heading" id="Sec11">Subclinical markers of dementia</h3><p>Although SA and TA relate closely to network topology, are they useful in explaining clinical symptoms beyond network topology? We explored the relationship of SA and TA with early symptoms of dementia, asking whether SA or TA predict cognitive decline beyond the effect of healthy aging. To assess cognitive function, we used the Addenbrookes Cognitive Examination Revisited (ACE-R), a battery of cognitive tests for dementia screening in subclinical populations. Because dementia was an exclusion criterion for participation in the Cam-CAN study, we did not expect to find a relationship between dementia markers and SA or TA. We computed the partial correlation across subjects of ACE-R score with SA-, SA-<span class="mathjax-tex">\(\infty\)</span> and global TA-<sub>1</sub>, partial on age and motion. Surprisingly, we found a significant negative partial correlation with SA- (<i>r</i>=0.184, <i>P</i>&lt;10<sup>5</sup>), associating wider SA with reduced cognitive function (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4e</a>). There was also a weaker correlation with SA-<span class="mathjax-tex">\(\infty\)</span> (<i>r</i>=0.118, <i>P</i>=0.002) but no significant relationship with global TA-<sub>1</sub> (<i>r</i>=0.067, <i>P</i>=0.09) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig4">4e</a>). Of all the graph metrics, only the weighted graph metric var-FC (<i>r</i>=0.13, <i>P</i>=0.001) showed a significant relationship (others, <i>P</i>&gt;0.1). This highlights the sensitivity of SA- and SA-<span class="mathjax-tex">\(\infty\)</span> compared to graph metrics. The SA--driven effect also contrasts with that of healthy aging, which was primarily driven by SA-<span class="mathjax-tex">\(\infty\)</span>. These findings hint at a relationship of clinical symptoms with SA and TA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Watanabe, T., Rees, G. &amp; Masuda, N. Atypical intrinsic neural timescale in autism. eLife 8, e42256 (2019)." href="/articles/s41593-023-01299-3#ref-CR42" id="ref-link-section-d30110227e2525">42</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec12">Pharmacological manipulation</h3><p>Lastly, we tested whether SA and TA causally reflect neurobiological processes. In principle, it is possible that TA and SA are driven exclusively by noise, morphology or other reliable artifacts instead of by differences in brain dynamics. A within-subject pharmacological study allows conclusions about acute changes with a causal mechanism. Therefore, in two human pharmacological fMRI experiments, we measured changes in SA and TA caused by manipulation of a neural circuit with the serotonin receptor agonists LSD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)." href="/articles/s41593-023-01299-3#ref-CR7" id="ref-link-section-d30110227e2537">7</a></sup> and psilocybin<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Preller, K. H. et al. Psilocybin induces time-dependent changes in global functional connectivity. Biol. Psychiatry 88, 197207 (2020)." href="/articles/s41593-023-01299-3#ref-CR43" id="ref-link-section-d30110227e2541">43</a></sup>. In both experiments, subjects were administered drug or placebo on separate days in a double-blind methodology, and rs-fMRI was performed at early and late time points after each administration. The same basic relationships among SA, TA and graph metrics were preserved under the drugs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig15">10</a>), despite a small sample size less than 3% that of HCP.</p><p>We found that both LSD and psilocybin caused robust overall reductions in TA across cortex. LSD and psilocybin reduced cortical TA at both early and late time points, in regional-averaged and subject-averaged TA (Wilcoxon rank-sum test <i>P</i>&lt;10<sup>40</sup> for all conditions) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5a</a>). Subject-level drug effects on global TA-<sub>1</sub> were not due to differences in within-scanner motion, and no effect of drugs on SA and graph metrics was detectable for this sample size (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig15">10</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Pharmacological administration of serotonergic drugs causes changes in TA."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Pharmacological administration of serotonergic drugs causes changes in TA.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="1523"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Cortical maps showing the mean difference in regional TA-<sub>1</sub> between drug and placebo at early (left) and late (right) time points. Purple indicates lower regional TA-<sub>1</sub> in the drug than placebo conditions (LSD <i>n</i>=24, psilocybin <i>n</i>=23). <b>b</b>, Change in the regional TA-<sub>1</sub> mean across subjects. Box plots show the median, first and third quartiles and outlier-excluded range of the distribution of regional TA-<sub>1</sub> mean across subjects for different regions. * indicates <i>P</i>&lt;0.05, and ** indicates <i>P</i>&lt;0.01, two-sided Wilcoxon signed-rank test. <b>c</b>, Similarity, measured by Spearman correlation, between drug minus placebo cortical maps for each drug and time point. * indicates <i>P</i>&lt;0.05, and ** indicates <i>P</i>&lt;0.01, two-tailed permutation test correcting for SA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Burt, J. B., Helmer, M., Shinn, M., Anticevic, A. &amp; Murray, J. D. Generative modeling of brain maps with spatial autocorrelation. Neuroimage 220, 117038 (2020)." href="/articles/s41593-023-01299-3#ref-CR11" id="ref-link-section-d30110227e2613">11</a></sup> and BonferroniHolm correction for FWER. <b>d</b>, The first right singular vector (SV) across all drug minus control contrasts for all subjects. Right top: variance explained by the first SV averaged across each drug condition. Right bottom: the SV score averaged across each condition. Box plots show the median, first and third quartiles and outlier-excluded range of the distribution. * indicates <i>P</i>&lt;0.05, and ** indicates <i>P</i>&lt;0.01, two-sided Wilcoxon signed-rank test, <i>n</i>=646 subjects. FWER, family-wise error rate.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41593-023-01299-3/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Psychedelic effects of LSD and psilocybin are predominantly attributed to agonism of serotonin receptors, in particular the 5-HT<sub>2A</sub> receptor<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)." href="/articles/s41593-023-01299-3#ref-CR7" id="ref-link-section-d30110227e2642">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Preller, K. H. et al. Psilocybin induces time-dependent changes in global functional connectivity. Biol. Psychiatry 88, 197207 (2020)." href="/articles/s41593-023-01299-3#ref-CR43" id="ref-link-section-d30110227e2645">43</a></sup>. A common serotonergic mechanism for LSD and psilocybin should produce similar cortex-wide topographies of the change in regional TA-<sub>1</sub>. Indeed, we found significant positive correlation among the cortical topographies for both drugs and both time points (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5c</a>).</p><p>Because both of these 5-HT<sub>2A</sub> agonists produced a specific topographic pattern of reductions in regional TA-<sub>1</sub>, we predicted that a 5-HT<sub>2A</sub> antagonist would produce the same pattern but as an increase in regional TA-<sub>1</sub> instead of a decrease. In the LSD study, on a third visit, subjects were pre-administered ketanserin, a selective 5-HT<sub>2A</sub> antagonist, before receiving LSD (LSD+Ket). We found that, at the late time point, ketanserin strongly attenuated the effect of LSD on regional TA-<sub>1</sub> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5a</a>). At the early time point, there was a cortex-wide increase in regional TA-<sub>1</sub> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5a</a>). Furthermore, regions with the strongest decrease under LSD showed the strongest increase with pre-administration of ketanserin (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5b</a>). These time-dependent changes in TA are consistent with observed pharmacokinetics of ketanserin, which exhibits relatively fast decrease in plasma levels after initial administration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Persson, B., Heykants, J. &amp; Hedner, T. Clinical pharmacokinetics of ketanserin. Clin. Pharmacokinet. 20, 263279 (1991)." href="/articles/s41593-023-01299-3#ref-CR44" id="ref-link-section-d30110227e2681">44</a></sup>.</p><p>We used these experiments to construct a cortical map of overall regional TA-<sub>1</sub> modulation by serotonergic drugs. To incorporate information from all experimental conditions, we used singular value decomposition (SVD) to compute the first singular vector across all drug versus control contrasts from all participants. We found that a single map (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5d</a>, left) explained approximately 50% of the variance of individual subjects for each of the three experiments at both time points (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5d</a>, top right). This map weighted negatively on the LSD and psilocybin conditions and on the late LSD+Ket condition but positively on the early LSD+Ket condition (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5d</a>, bottom right), consistent with their correlational structure (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig5">5c</a>). These findings demonstrate that TA is sensitive to causal pharmacological perturbation by serotonergic drugs, showing that TA reflects meaningful differences in neurobiology.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec13-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec13">Discussion</h2><div class="c-article-section__content" id="Sec13-content"><p>Here we have shown that SA and TAas parameterized by SA-, SA-<span class="mathjax-tex">\(\infty\)</span> and TA-<sub>1</sub>are highly reliable properties of rs-fMRI time series that correlate with, and are predictive of, network topology. When we use a model to generate surrogate time series with subject-matched SA and TA, the time series produce networks that match the subjects graph metrics. The surrogate time series model was also used to track age-related changes in graph metrics, and SA and TA correlate with subclinical symptoms of dementia, even though graph metrics are not sensitive enough to detect this change. Furthermore, causal pharmacological manipulations with multiple serotonergic drugs modulate TA in a distinct and reliable pattern with a large within-subject effect size. We anticipate that comparable relationships with SA and TA will be present in other datasets stemming from a variety of conditions, diseases and pharmacological states. The high reliability, interpretability, effect size, clinical relevance and sensitivity to neurobiology make SA and TA serious candidates for fMRI-based biomarkers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Keitel, A. &amp; Gross, J. Individual human brain areas can be identified from their characteristic spectral activation fingerprints. PLoS Biol. 14, e1002498 (2016)." href="/articles/s41593-023-01299-3#ref-CR33" id="ref-link-section-d30110227e2731">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Watanabe, T., Rees, G. &amp; Masuda, N. Atypical intrinsic neural timescale in autism. eLife 8, e42256 (2019)." href="/articles/s41593-023-01299-3#ref-CR42" id="ref-link-section-d30110227e2734">42</a></sup>.</p><p>What factors drive SA and TA? We showed SA and TA can be influenced by biological factors, such as aging; pharmacological factors, such as serotonergic drugs; and methodological factors, such as motion, parcel size and noise. These findings are consistent with previous work in brain signal variability during aging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Grady, C. L. &amp; Garrett, D. D. Understanding variability in the BOLD signal and why it matters for aging. Brain Imaging Behav. 8, 274283 (2013)." href="/articles/s41593-023-01299-3#ref-CR45" id="ref-link-section-d30110227e2741">45</a></sup> as well as in non-serotonin neuromodulators, such as norepinephrine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Huang, Z., Liu, X., Mashour, G. A. &amp; Hudetz, A. G. Timescales of intrinsic BOLD signal dynamics and functional connectivity in pharmacologic and neuropathologic states of unconsciousness. J. Neurosci. 38, 23042317 (2018)." href="/articles/s41593-023-01299-3#ref-CR12" id="ref-link-section-d30110227e2745">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Zerbi, V. et al. Rapid reconfiguration of the functional connectome after chemogenetic locus coeruleus activation. Neuron 103, 702718 (2019)." href="/articles/s41593-023-01299-3#ref-CR46" id="ref-link-section-d30110227e2748">46</a></sup>, dopamine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Shafiei, G. et al. Dopamine signaling modulates the stability and integration of intrinsic brain networks. Cereb. Cortex 29, 397409 (2018)." href="/articles/s41593-023-01299-3#ref-CR47" id="ref-link-section-d30110227e2752">47</a></sup> and acetylcholine<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Turchi, J. et al. The basal forebrain regulates global resting-state fMRI fluctuations. Neuron 97, 940952 (2018)." href="/articles/s41593-023-01299-3#ref-CR48" id="ref-link-section-d30110227e2756">48</a></sup>. They are also consistent with methodological studies, showing the statistical considerations of SA and TA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Friston, K. J. et al. Analysis of fMRI time-series revisited. Neuroimage 2, 4553 (1995)." href="/articles/s41593-023-01299-3#ref-CR10" id="ref-link-section-d30110227e2760">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Davey, C. E., Grayden, D. B., Egan, G. F. &amp; Johnston, L. A. Filtering induces correlation in fMRI resting state data. Neuroimage 64, 728740 (2013)." href="/articles/s41593-023-01299-3#ref-CR49" id="ref-link-section-d30110227e2763">49</a></sup> and the ability of null models to form complex networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. Neuroimage 60, 20962106 (2012)." href="/articles/s41593-023-01299-3#ref-CR30" id="ref-link-section-d30110227e2768">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Cantwell, G. T. et al. Thresholding normally distributed data creates complex networks. Phys. Rev. E 101, 062302 (2020)." href="/articles/s41593-023-01299-3#ref-CR50" id="ref-link-section-d30110227e2771">50</a></sup>. SA is particularly important for the emerging literature on diverse cortical gradients<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Shafiei, G. et al. Dopamine signaling modulates the stability and integration of intrinsic brain networks. Cereb. Cortex 29, 397409 (2018)." href="/articles/s41593-023-01299-3#ref-CR47" id="ref-link-section-d30110227e2775">47</a></sup>. Furthermore, SA and TA are influenced by local circuit connectivity and properties such as intrinsic time scale<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. Nat. Neurosci. 17, 16611663 (2014)." href="/articles/s41593-023-01299-3#ref-CR27" id="ref-link-section-d30110227e2779">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Ito, T., Hearne, L. J. &amp; Cole, M. W. A cortical hierarchy of localized and distributed processes revealed via dissociation of task activations, connectivity changes, and intrinsic timescales. Neuroimage 221, 117141 (2020)." href="/articles/s41593-023-01299-3#ref-CR51" id="ref-link-section-d30110227e2782">51</a></sup>. The influence of confounding physiological processes on SA and TA must be better understood<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Chen, J. E. et al. Resting-state physiological networks. Neuroimage 213, 116707 (2020)." href="/articles/s41593-023-01299-3#ref-CR52" id="ref-link-section-d30110227e2786">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Drew, P. J., Mateo, C., Turner, K. L., Yu, X. &amp; Kleinfeld, D. Ultra-slow oscillations in fMRI and resting-state connectivity: neuronal and vascular contributions and technical confounds. Neuron 107, 782804 (2020)." href="/articles/s41593-023-01299-3#ref-CR53" id="ref-link-section-d30110227e2789">53</a></sup>, including the link between TA and signal variability<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Baracchini, G. et al. Inter-regional BOLD signal variability is an organizational feature of functional brain networks. Neuroimage 237, 118149 (2021)." href="/articles/s41593-023-01299-3#ref-CR54" id="ref-link-section-d30110227e2793">54</a></sup>, especially related to aging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Grady, C. L. &amp; Garrett, D. D. Understanding variability in the BOLD signal and why it matters for aging. Brain Imaging Behav. 8, 274283 (2013)." href="/articles/s41593-023-01299-3#ref-CR45" id="ref-link-section-d30110227e2797">45</a></sup>. The hemodynamic response function imposes not only temporal but also spatial filtering to the BOLD signal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Kriegeskorte, N., Cusack, R. &amp; Bandettini, P. How does an fMRI voxel sample the neuronal activity pattern: compact-kernel or complex spatiotemporal filter? Neuroimage 49, 19651976 (2010)." href="/articles/s41593-023-01299-3#ref-CR55" id="ref-link-section-d30110227e2802">55</a></sup>, which could contribute to changes in SA and TA due to age<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="West, K. L. et al. BOLD hemodynamic response function changes significantly with healthy aging. Neuroimage 188, 198207 (2019)." href="/articles/s41593-023-01299-3#ref-CR56" id="ref-link-section-d30110227e2806">56</a></sup> or serotonergic drugs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Cohen, Z., Bonvento, G., Lacombe, P. &amp; Hamel, E. Serotonin in the regulation of brain microcirculation. Prog. Neurobiol. 50, 335362 (1996)." href="/articles/s41593-023-01299-3#ref-CR57" id="ref-link-section-d30110227e2810">57</a></sup>. Because anatomical structure serves as a scaffold for SA and TA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Sethi, S. S., Zerbi, V., Wenderoth, N., Fornito, A. &amp; Fulcher, B. D. Structural connectome topology relates to regional BOLD signal dynamics in the mouse brain. Chaos 27, 047405 (2017)." href="/articles/s41593-023-01299-3#ref-CR13" id="ref-link-section-d30110227e2814">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Fallon, J. et al. Timescales of spontaneous fMRI fluctuations relate to structural connectivity in the brain. Netw. Neurosci. 4, 788806 (2020)." href="/articles/s41593-023-01299-3#ref-CR14" id="ref-link-section-d30110227e2817">14</a></sup>, our testretest reliability and fingerprinting performance could reflect individual differences in brain morphology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Geerligs, L., Tsvetanov, K. A. &amp; Henson, R. N. Challenges in measuring individual differences in functional connectivity using fMRI: the case of healthy aging. Hum. Brain Mapp. 38, 41254156 (2017)." href="/articles/s41593-023-01299-3#ref-CR58" id="ref-link-section-d30110227e2821">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Honey, C. J. et al. Predicting human resting-state functional connectivity from structural connectivity. Proc. Natl Acad. Sci. USA 106, 20352040 (2009)." href="/articles/s41593-023-01299-3#ref-CR59" id="ref-link-section-d30110227e2824">59</a></sup>. Similarly, differences in functional network organization across participants could lead to reliable misalignment between parcel boundaries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Braga, R. M. &amp; Buckner, R. L. Parallel interdigitated distributed networks within the individual estimated by intrinsic functional connectivity. Neuron 95, 457471 (2017)." href="/articles/s41593-023-01299-3#ref-CR60" id="ref-link-section-d30110227e2828">60</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Kong, R. et al. Individual-specific areal-level parcellations improve functional connectivity prediction of behavior. Cereb. Cortex 31, 44774500 (2021)." href="/articles/s41593-023-01299-3#ref-CR61" id="ref-link-section-d30110227e2831">61</a></sup>, which would contribute to the reliability that we observed in regional TA-<sub>1</sub>. Our results showed that the regional homogeneity within a parcel, associated with many of these structural and functional differences, was positively correlated with regional TA-<sub>1</sub> but had varied ability to explain fingerprinting performance. In the HCP dataset, it could not explain fingerprinting performance at all, whereas, in the Yale-TRT dataset, it had better fingerprinting performance than regional TA-<sub>1</sub>. This suggests that the strongest underlying influences on TA-<sub>1</sub> vary across datasets, which may explain the difference in model performance between the HCP and Yale-TRT datasets. TA also has a complex relationship with methodological factors, such as repetition time (TR). On the one hand, a shorter TR causes samples to be more closely spaced in time, meaning that, for smooth 1/<i>f</i>-like power spectra, TA will increase. On the other hand, shorter TR may also result in a noisier signal, causing TA to decrease. These differences, as well as the influence of pre-processing strategy, must be studied systematically to compare TA across studies with different TRs.</p><p>In general, SA and TA constrain the dimensionality of the neural signal, and these dimensionality constraints may explain their link to network topology. Previous work has shown that lower-dimensional subspaces capture network topology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Vrtes, P. E. et al. Simple models of human brain functional networks. Proc. Natl Acad. Sci. USA 109, 58685873 (2012)." href="/articles/s41593-023-01299-3#ref-CR23" id="ref-link-section-d30110227e2850">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Betzel, R. F. et al. Generative models of the human connectome. Neuroimage 124, 10541064 (2016)." href="/articles/s41593-023-01299-3#ref-CR40" id="ref-link-section-d30110227e2853">40</a></sup>, and we showed that SA and TA align with these dimensions through the economic clustering model. These dimensions may map onto other aspects of network topologyfor example, a high-degree backbone and a lattice-like background<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Achard, S. A resilient, low-frequency, small-world human brain functional network with highly connected association cortical hubs. J. Neurosci. 26, 6372 (2006)." href="/articles/s41593-023-01299-3#ref-CR39" id="ref-link-section-d30110227e2857">39</a></sup>. Graph metrics provide complex non-linear projections of these dimensions, making it possible to compare topological properties between networks. The eigensurrogate model tested the broader question of whether another FC matrix with equivalent linear dimensionality could reproduce graph metrics. Although it did not perform as well as our time-series-based model on most datasets, it performed surprisingly well and even outperformed our model on the Yale-TRT dataset, emphasizing the role of constrained dimensionality in shaping network topology. Variations on the eigensurrogate method, including constraints on the eigenvectors, could yield insights into individual variation in brain topology.</p><p>Our present study used a parcellated analysis, which limits the precision with which SA can be measured. Our methods for computing both SA and TA scale well for voxel-level or vertex-level analyses, but fitting the spatiotemporal model is computationally intractable for parcellations with many nodes. SA-<span class="mathjax-tex">\(\infty\)</span> is conceptually similar to mean-FC, which has known links to aging<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Geerligs, L., Tsvetanov, K. A. &amp; Henson, R. N. Challenges in measuring individual differences in functional connectivity using fMRI: the case of healthy aging. Hum. Brain Mapp. 38, 41254156 (2017)." href="/articles/s41593-023-01299-3#ref-CR58" id="ref-link-section-d30110227e2881">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Balthazar, M. L. F., de Campos, B. M., Franco, A. R., Damasceno, B. P. &amp; Cendes, F. Whole cortical and default mode network mean functional connectivity as potential biomarkers for mild Alzheimers disease. Psychiatry Res. 221, 3742 (2014)." href="/articles/s41593-023-01299-3#ref-CR62" id="ref-link-section-d30110227e2884">62</a></sup>. It is also similar to global signal, but it remains reliable even after GSR and, thus, may represent spatial inhomogeneities in the global signal. Nevertheless, even this parcellated analysis is sensitive enough to reveal features inaccessible to graph theoretical analysis. For example, in our pharmacological experiments, our sample size of fewer than 25 participants did not show significance in graph-theoretical measures, but changes in TA revealed highly significant differences in functional organization.</p><p>Our work highlights the need to always study complex properties, such as graph metrics, alongside simpler properties, such as SA and TA. In future studies using graph metrics, specific hypotheses must guide the careful interpretation of graph metrics in light of SA and TA. Our work also highlights that graph metrics derived from rs-fMRI networks cannot be directly interpreted as signatures of regional communication or information processing. Our results may extend more generally beyond graph theory to other analyses of FC, including both resting-state networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Shafiei, G. et al. Topographic gradients of intrinsic dynamics across neocortex. eLife 9, e62116 (2020)." href="/articles/s41593-023-01299-3#ref-CR32" id="ref-link-section-d30110227e2892">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Watanabe, T., Rees, G. &amp; Masuda, N. Atypical intrinsic neural timescale in autism. eLife 8, e42256 (2019)." href="/articles/s41593-023-01299-3#ref-CR42" id="ref-link-section-d30110227e2895">42</a></sup> and task-related changes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Fascianelli, V., Tsujimoto, S., Marcos, E. &amp; Genovesio, A. Autocorrelation structure in the macaque dorsolateral, but not orbital or polar, prefrontal cortex predicts response-coding strength in a visually cued strategy task. Cereb. Cortex 29, 230241 (2019)." href="/articles/s41593-023-01299-3#ref-CR15" id="ref-link-section-d30110227e2899">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Arbabshirani, M. R. et al. Autoconnectivity: A new perspective on human brain function. J. Neurosci. Methods 323, 6876 (2019)." href="/articles/s41593-023-01299-3#ref-CR16" id="ref-link-section-d30110227e2902">16</a></sup>. The use of our spatiotemporal surrogate modeland of previously published spatially informed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Roberts, J. A. et al. The contribution of geometry to the human connectome. Neuroimage 124, 379393 (2016)." href="/articles/s41593-023-01299-3#ref-CR24" id="ref-link-section-d30110227e2906">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Wiedermann, M., Donges, J. F., Kurths, J. &amp; Donner, R. V. Spatial network surrogates for disentangling complex system structure from spatial embedding of nodes. Phys. Rev. E 93, 042308 (2016)." href="/articles/s41593-023-01299-3#ref-CR63" id="ref-link-section-d30110227e2909">63</a></sup> and temporally informed<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. Neuroimage 60, 20962106 (2012)." href="/articles/s41593-023-01299-3#ref-CR30" id="ref-link-section-d30110227e2913">30</a></sup> null modelscould be applied more generally to high-dimensional statistics in neuroimaging. Our work also highlights the informativeness of SA and TA about biological processes such as aging and serotonergic tone. Historically, SA and TA have been considered confounds that need to be corrected for. Our results suggest that SA and TA should not be treated as confounds but, rather, as essential informative properties of the connectome.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec14-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec14">Methods</h2><div class="c-article-section__content" id="Sec14-content"><p>Our research complies with all relevant ethical regulations, as approved by the Washington University institutional review board, the Yale University institutional review board, the Cambridgeshire 2 Research Ethics Committee, the Cantonal Ethics Committee of Zurich and the Swiss Federal Office of Public Health.</p><h3 class="c-article__sub-heading" id="Sec15">Datasets</h3><p>We analyzed the following datasets, which comprise a diversity of pre-processing steps and experimental methodologies, including different parcellations, sampling rates, spatial and temporal smoothings, covariate regressions and noise removal strategies. Pre-processing methodology for each dataset was intended to be as faithful as possible to the standard pipeline of the dataset, showing the generality of our conclusions across pre-processing strategies. Because data were not collected specifically for our study, we used the maximum possible sample size for available data. Our team has substantial experience with these datasets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)." href="/articles/s41593-023-01299-3#ref-CR7" id="ref-link-section-d30110227e2932">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Noble, S. et al. Influences on the testretest reliability of functional connectivity MRI and its relationship with behavioral utility. Cereb. Cortex 27, 54155429 (2017)." href="/articles/s41593-023-01299-3#ref-CR20" id="ref-link-section-d30110227e2935">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Preller, K. H. et al. Psilocybin induces time-dependent changes in global functional connectivity. Biol. Psychiatry 88, 197207 (2020)." href="/articles/s41593-023-01299-3#ref-CR43" id="ref-link-section-d30110227e2938">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Demirta, M. et al. Hierarchical heterogeneity across human cortex shapes large-scale neural dynamics. Neuron 101, 11811194 (2019)." href="/articles/s41593-023-01299-3#ref-CR64" id="ref-link-section-d30110227e2941">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Morgan, S. E., Achard, S., Termenon, M., Bullmore, E. T. &amp; Vrtes, P. E. Low-dimensional morphospace of topological motifs in human fMRI brain networks. Netw. Neurosci. 2, 285302 (2018)." href="/articles/s41593-023-01299-3#ref-CR65" id="ref-link-section-d30110227e2944">65</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec16">HCP</h4><p>A total of 883 subjects aged 2237years (459 females) from the HCP 1,200-subject data release underwent four resting-state scanning sessions spread across 2days. Subjects provided informed consent and were compensated for participation. Resting-state scans lasted for 14.4minutes with a TR of 0.72seconds (sampling rate of 1.39Hz). Data were pre-processed with the HCP minimal pre-processing pipeline. This includes distortion correction using the field map, realignment for head motion and registration to T1 images. Data were further denoised using ICA-FIX, a technique based on independent component analysis to remove structured noise. A high-pass filter was applied at 0.01Hz. Data were parcellated into 360 regions (180 per hemisphere)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-023-01299-3#ref-CR19" id="ref-link-section-d30110227e2955">19</a></sup> with multimodal surface matching based on MSMAll (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Robinson, E. C. et al. MSM: a new flexible framework for multimodal surface matching. Neuroimage 100, 414426 (2014)." href="/articles/s41593-023-01299-3#ref-CR66" id="ref-link-section-d30110227e2959">66</a></sup>) and with 2-mm full width at half maximum (FWHM) surface spatial smoothing constrained to the parcel. The first 100 time points were removed to ensure steady state, consistent with prior work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Demirta, M. et al. Hierarchical heterogeneity across human cortex shapes large-scale neural dynamics. Neuron 101, 11811194 (2019)." href="/articles/s41593-023-01299-3#ref-CR64" id="ref-link-section-d30110227e2963">64</a></sup>. Because many HCP subjects are genetically related, we used a subset of 334 unrelated subjects where indicated to avoid this potential confound. When using geodesic distance instead of Euclidean distance, we used only the right hemisphere (180 regions).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">HCP with GSR</h4><p>Subjects and scan parameters are identical to the HCP dataset, but GSR was added as a pre-processing step before parcellation. Additionally, we did not truncate the first 100 TRs, for a total of 1,200 time points. Thirty-three additional subjects were excluded due to non-convergence in the GSR pipeline, for a total of 850 subjects.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec18">Yale-TRT</h4><p>A total of 12 subjects (six females) aged 2756years were scanned on four different days, and six 6-minute sessions were performed each day. Subjects provided informed consent and were compensated for participation. Each subject was scanned on two scanners, 2days on each scanner, on days spaced approximately 1week apart. Thus, half of the scanning sessions for each subject were performed on a different scanner. Scanning sessions lasted 6minutes, each with a TR of 1.0 (sampling rate of 1.0Hz). Motion correction was applied, and images were spatially smoothed to achieve uniform spatial smoothness of a 2.5-mm Gaussian kernel<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Friedman, L., Glover, G. H., Krenz, D. &amp; Magnotta, V. Reducing inter-scanner variability of activation in a multicenter fMRI study: role of smoothness equalization. Neuroimage 32, 16561668 (2006)." href="/articles/s41593-023-01299-3#ref-CR67" id="ref-link-section-d30110227e2983">67</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Scheinost, D., Papademetris, X. &amp; Constable, R. T. The impact of image smoothness on intrinsic functional connectivity and head motion confounds. Neuroimage 95, 1321 (2014)." href="/articles/s41593-023-01299-3#ref-CR68" id="ref-link-section-d30110227e2986">68</a></sup>. Images were co-registered to a common subject-specific space across days, and subsequently into MNI space, and parcellated using the Shen parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Shen, X., Tokoglu, F., Papademetris, X. &amp; Constable, R. Groupwise whole-brain parcellation from resting-state fMRI data for network node identification. Neuroimage 82, 403415 (2013)." href="/articles/s41593-023-01299-3#ref-CR69" id="ref-link-section-d30110227e2990">69</a></sup>.</p><p>High-pass filtering was performed by regressing out linear, quadratic and cubic trends, and low-pass filtering was performed with a Gaussian kernel with cutoff frequency of 0.19Hz. Mean white matter, mean cerebrospinal fluid, mean global signal and a 24-parameter motion model were also regressed out of the data. No subjects, sessions or regions were excluded.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec19">Cam-CAN</h4><p>A total of 652 subjects aged 1888years (326 females) were scanned using a TR of 1.97seconds (sampling rate of 0.508Hz). Subjects provided informed consent and were compensated for participation. We used the standard pre-processing pipeline and parcellation provided by the Cam-CAN project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Shafto, M. A. et al. The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) study protocol: a cross-sectional, lifespan, multidisciplinary examination of healthy cognitive ageing. BMC Neurol. 14, 204 (2014)." href="/articles/s41593-023-01299-3#ref-CR21" id="ref-link-section-d30110227e3006">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Taylor, J. R. et al. The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) data repository: structural and functional MRI, MEG, and cognitive data from a cross-sectional adult lifespan sample. Neuroimage 144, 262269 (2017)." href="/articles/s41593-023-01299-3#ref-CR70" id="ref-link-section-d30110227e3009">70</a></sup>. In summary, scans underwent motion correction and slice time correction before co-registration to T1 images and normalization to MNI space with DARTEL. We used the default AAL parcellation provided by the Cam-CAN project<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Tzourio-Mazoyer, N. et al. Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain. Neuroimage 15, 273289 (2002)." href="/articles/s41593-023-01299-3#ref-CR71" id="ref-link-section-d30110227e3013">71</a></sup>. We also applied a second-order Butterworth low-pass filter at half the Nyquist frequency (0.127Hz) to account for high-frequency motion artifacts. We excluded six subjects and one cerebellar region due to missing data, for a grand total of 646 subjects with 115 regions in the parcellation.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">LSD dataset</h4><p>The study used a double-blind randomized design<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)." href="/articles/s41593-023-01299-3#ref-CR7" id="ref-link-section-d30110227e3025">7</a></sup>. Subjects provided informed consent and were compensated for participation. On each of the 3days, <i>n</i>=24 participants aged 2034years (five females) received one of the following treatments: (1) placebo (179mg of mannitol and 1mg of aerosil, orally) pre-treatment followed by placebo (179mg of mannitol and 1mg of aerosil, orally) treatment; (2) placebo (179mg of mannitol and 1mg of aerosil, orally) pre-treatment followed by LSD (100g, orally); and (3) ketanserin (40mg, orally) pre-treatment followed by LSD (100g, orally) treatment. Pre-treatment was given 60minutes before the treatment. The early resting-state scan occurred 75minutes after treatment and the late scan 300minutes after treatment. Data were acquired with TR=2,500ms (sampling rate of 0.4Hz). Twenty-five subjects were enrolled in the study. One subject was excluded for technical faults in the data, for a total of 24 subjects.</p><p>We used the same pre-processing pipeline described in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)." href="/articles/s41593-023-01299-3#ref-CR7" id="ref-link-section-d30110227e3035">7</a></sup>, which is summarized below. First, the data were subject to the HCP minimal pre-processing pipeline. This involved correction for field inhomogeneities, phase-encoding directions and magnetic susceptibility artifacts as well as motion correction and registration to structure images with non-brain tissue masking. Data were high-pass filtered (&gt;0.008Hz). Several nuisance variables were regressed out: average signal in the ventricles, average signal in deep white matter, motion parameters and the mean time series across gray matter (that is, the global signal) as well as the first derivative of each of these. Data were motion scrubbed, identifying outlier frames as either frames with a summed framewise displacement over 0.5mm or a root mean square (RMS) of differences in intensity of subsequent frames over 1.6 times the median. All outlier frames, the one frame preceding them and two frames following them, were excluded from analysis. Lastly, data for cortex were parcellated into 360 regions according to the Glasser parcellation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. Nature 536, 171178 (2016)." href="/articles/s41593-023-01299-3#ref-CR19" id="ref-link-section-d30110227e3039">19</a></sup>. To ensure that scrubbed frames did not impact TA-<sub>1</sub> calculations, TA-<sub>1</sub> was computed only on consecutive segments involving no dropped frames, such that TA-<sub>1</sub> was a weighted average of the TA-<sub>1</sub> of each consecutive segment, weighted by the degrees of freedom of that segment. The long-distance correlation parameter <i>d</i> was not computed on scrubbed data.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Psilocybin</h4><p>The study used a double-blind randomized design<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Preller, K. H. et al. Psilocybin induces time-dependent changes in global functional connectivity. Biol. Psychiatry 88, 197207 (2020)." href="/articles/s41593-023-01299-3#ref-CR43" id="ref-link-section-d30110227e3063">43</a></sup>, where 23 participants aged 2040years (11 females) were scanned two separate days. Subjects provided informed consent and were compensated for participation. On each day, participants received either placebo (179mg of mannitol and 1mg of aerosil) or psilocybin (0.2mgkg<sup>1</sup>), administered orally. rs-fMRI was performed at three time points after psilocybin administration: immediate, performed 20minutes after administration; early, performed 40minutes after administration; and late, performed 70minutes after administration. Data were acquired with TR=2,430ms. Twenty-four participants were enrolled in the study. One participant was excluded due to a missing scan, for a total of 23 participants. This sample size was determined based on the prior LSD study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. eLife 7, e35082 (2018)." href="/articles/s41593-023-01299-3#ref-CR7" id="ref-link-section-d30110227e3069">7</a></sup>.</p><p>Data pre-processing was identical to that of the LSD dataset described above.</p><h3 class="c-article__sub-heading" id="Sec22">Estimating SA</h3><p>We quantified SA by decomposing it into two components: the rate at which correlations decrease exponentially with distance (SA-) and the spatially invariant level of correlation to which it decays (SA-<span class="mathjax-tex">\(\infty\)</span>). This can be described quantitatively as</p><div id="Equ1" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mathrm{corr}}}\,({x}_{i},{x}_{j})={\mathrm{SA}\hbox{-}\infty} +(1-{\mathrm{SA}\hbox{-}\infty} )\,{e}^{-{D}_{i,j}/{\mathrm{SA}\hbox{-}\uplambda} }$$</span></div><div class="c-article-equation__number">
                    (1)
                </div></div><p>where <i>x</i><sub><i>i</i></sub> is the time series for region <i>i</i>; <i>D</i><sub><i>i</i>,<i>j</i></sub> is the Euclidean distance between regions <i>i</i> and <i>j</i>; and corr is the Pearson correlation. The quantities SA- and SA-<span class="mathjax-tex">\(\infty\)</span> are constants and do not depend on <i>i</i> or <i>j</i>.</p><p>We estimated the parameters SA- and SA-<span class="mathjax-tex">\(\infty\)</span> for each subject as follows. For each pair of brain regions, we computed both their physical Euclidean distance (from the regions centroid) as well as their Pearson correlation coefficient. In this study, we used Euclidean distance rather than geodesic distance as our primary distance measure because Euclidean distance is defined across cortical hemispheres and within subcortex. However, geodesic and other distance measurements can be used instead and produce similar results in practice (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig12">7</a>). We binned each pair of brain regions according to their Euclidean distance, using 1-mm bins for the HCP, HCP-GSR, LSD and psilocybin datasets, and 5-mm bins for the TRT and Cam-CAN datasets due to the fewer number of parcels. We then computed the mean Pearson correlation of pairs in each bin to generate correlation versus distance curve (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1a</a>). We then found the least square fit of Equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-023-01299-3#Equ1">1</a>) to this curve, optimizing with gradient descent, bounding SA- between 0 and 100 and SA-<span class="mathjax-tex">\(\infty\)</span> between 1 and 1. Although it is possible to fit Equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-023-01299-3#Equ1">1</a>) directly to the distance and correlation of each pair without binning, our approach puts more weight on nearby and distant correlations, which are less represented in the data but most critical for determining SA- and SA-<span class="mathjax-tex">\(\infty\)</span>.</p><p>To account for heteroskedasticity in SA- for the Cam-CAN dataset, we analyzed the logarithm of this parameter for this dataset.</p><h3 class="c-article__sub-heading" id="Sec23">Estimating TA</h3><p>Our primary measure of TA, TA-<sub>1</sub>, is a non-parametric measurement computed by taking the Pearson correlation of neighboring time points in the parcellated time seriesthat is, for a time series <i>x</i>[<i>t</i>], we have corr(<i>x</i>[<i>t</i>],<i>x</i>[<i>t</i>+1]). This measure is computationally efficient and can be implemented with only a few lines of code. However, TA-<sub>1</sub> is not comparable across datasets due to differing TR.</p><p>For datasets that included motion scrubbing, we computed TA-<sub>1</sub> only on consecutive regions without scrubbed frames. More specifically, we split each time series into fragments at the location of the scrubbed frames. Then, we computed TA-<sub>1</sub> on each fragment and took the weighted average. More formally, for each time series, fragment <i>x</i><sub><i>i</i></sub> of length <i></i><sub><i>i</i></sub>, TA-<sub>1</sub> was computed as</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\mathrm{TA}\hbox{-}\Delta_1}=\mathop{\sum}\limits_{i}\,{{\mathrm{corr}}}\,({x}_{i}[t],{x}_{i}[t+1])\frac{{\ell }_{i}-1}{{\sum }_{j}({\ell }_{j}-1)}.$$</span></div></div><h3 class="c-article__sub-heading" id="Sec24">Estimating long-memory dynamics</h3><p>fMRI time series are thought to show characteristics of a long-memory process. In other words, time points that are separated by a large lag show a high correlation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bullmore, E. et al. Colored noise and computational inference in neurophysiological (fMRI) time series analysis: resampling methods in time and wavelet domains. Hum. Brain Mapp. 12, 6178 (2001)." href="#ref-CR72" id="ref-link-section-d30110227e3618">72</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wagenmakers, E.-J., Farrell, S. &amp; Ratcliff, R. Estimation and interpretation of 1/f noise in human cognition. Psychon. Bull. Rev. 11, 579615 (2004)." href="#ref-CR73" id="ref-link-section-d30110227e3618_1">73</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Maxim, V. et al. Fractional Gaussian noise, functional MRI and Alzheimers disease. Neuroimage 25, 141158 (2005)." href="/articles/s41593-023-01299-3#ref-CR74" id="ref-link-section-d30110227e3621">74</a></sup>. These processes are often characterized by their autocorrelation function (ACF). For a time series <i>x</i>, the ACF is defined as</p><div id="Equb" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$AC{F}_{x}(k)=\,{{\mbox{corr}}}\,(x[t],x[t+k])$$</span></div></div><p>where corr is the Pearson correlation, and <i>k</i> is the lag at which to evaluate the ACF. Long-memory processes have an ACF that decays slowly across lags, meaning that relatively high correlations exist even at large lags. In this work, we use TA-<sub>1</sub> as our primary measure of TA, which is identical to the lag-1 term of the ACF, <i>A</i><i>C</i><i>F</i><sub><i>x</i></sub>(1).</p><p>To evaluate the ability of the first term of the ACF to predict higher terms of the ACF, we divided the data into ten randomly selected subsets for the HCP and Cam-CAN datasets and 12 subsets for the TRT data. We fit the regression model</p><div id="Equc" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$AC{F}_{x}(k)={\beta }_{0}+{\beta }_{1}{\mathrm{TA}\hbox{-}\Delta_1}$$</span></div></div><p>to one of these subsets. Then, we evaluated the <i>R</i><sup>2</sup> of the model on each of the other subsets. Note that this is a more difficult prediction than traditional <i>k</i>-fold cross validation, as it uses less training data to make the predictions. We evaluated this fit on the remaining subsets.</p><p>As a more direct measure of long-memory dynamics, we analyzed the fractional integration constant <i>d</i> from an ARFIMA model, frequently used to capture long-memory dynamics. One attractive property of <i>d</i> is its close relationship to other measures of long-memory dynamics, such as the Hurst exponent <i>H</i>under specific conditions, there is a one-to-one mapping<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Achard, S., Bassett, D. S., Meyer-Lindenberg, A. &amp; Bullmore, E. Fractal connectivity of long-memory networks. Phys. Rev. E 77, 036104 (2008)." href="/articles/s41593-023-01299-3#ref-CR75" id="ref-link-section-d30110227e3866">75</a></sup> given by <i>H</i>=<i>d</i>+0.5. Following previous work, we estimated <i>d</i> through a univariate wavelet-based Whittle estimator<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Sela, R. J. &amp; Hurvich, C. M. Computationally efficient methods for two multivariate fractionally integrated models. J. Time Ser. Anal. 30, 631651 (2009)." href="/articles/s41593-023-01299-3#ref-CR76" id="ref-link-section-d30110227e3880">76</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Achard, S. &amp; Gannaz, I. Multivariate wavelet whittle estimation in long-range dependence. J. Time Ser. Anal. 37, 476512 (2015)." href="/articles/s41593-023-01299-3#ref-CR77" id="ref-link-section-d30110227e3883">77</a></sup>. We restricted our analyses of <i>d</i> to the HCP, TRT and Cam-CAN datasets, because they did not use motion scrubbing in their pre-processing pipeline. Under specific assumptions, the relationship between <i>d</i> and TA-<sub>1</sub> can be described analytically<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Lu, Z. Analysis of stationary and non-stationary long memory processes: estimation, applications and forecast. PhD thesis, cole normale suprieure de Cachan - ENS Cachan (2019); 
                  https://theses.hal.science/tel-00422376/document
                  
                " href="/articles/s41593-023-01299-3#ref-CR78" id="ref-link-section-d30110227e3896">78</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Hassani, H., Leonenko, N. &amp; Patterson, K. The sample autocorrelation function and the detection of long-memory processes. Phys. A 391, 63676379 (2012)." href="/articles/s41593-023-01299-3#ref-CR79" id="ref-link-section-d30110227e3899">79</a></sup>. Consistent with prior work, TA-<sub>1</sub> is representative of the first and higher lag terms of the ACF<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Maxim, V. et al. Fractional Gaussian noise, functional MRI and Alzheimers disease. Neuroimage 25, 141158 (2005)." href="/articles/s41593-023-01299-3#ref-CR74" id="ref-link-section-d30110227e3905">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, Z. Analysis of stationary and non-stationary long memory processes: estimation, applications and forecast. PhD thesis, cole normale suprieure de Cachan - ENS Cachan (2019); 
                  https://theses.hal.science/tel-00422376/document
                  
                " href="#ref-CR78" id="ref-link-section-d30110227e3908">78</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hassani, H., Leonenko, N. &amp; Patterson, K. The sample autocorrelation function and the detection of long-memory processes. Phys. A 391, 63676379 (2012)." href="#ref-CR79" id="ref-link-section-d30110227e3908_1">79</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Scheffer, M. et al. Early-warning signals for critical transitions. Nature 461, 5359 (2009)." href="/articles/s41593-023-01299-3#ref-CR80" id="ref-link-section-d30110227e3911">80</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec25">FC and graph construction</h3><p>We defined the FC matrix <i></i><sub><i>i</i>,<i>j</i></sub> as the matrix of Pearson correlation coefficients <i></i><sub><i>i</i>,<i>j</i></sub>=corr(<i>x</i><sub><i>i</i></sub>,<i>x</i><sub><i>j</i></sub>) between each pairwise combination of regional time series <i>x</i><sub><i>i</i></sub> and <i>x</i><sub><i>j</i></sub>.</p><p>We constructed unweighted, undirected graphs from model or data FC matrices using standard techniques<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Fornito, A., Zalesky, A. &amp; Bullmore, E. Fundamentals of Brain Network Analysis (Academic Press, 2016)." href="/articles/s41593-023-01299-3#ref-CR3" id="ref-link-section-d30110227e3969">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Bullmore, E. &amp; Sporns, O. Complex brain networks: graph theoretical analysis of structural and functional systems. Nat. Rev. Neurosci. 10, 186198 (2009)." href="/articles/s41593-023-01299-3#ref-CR81" id="ref-link-section-d30110227e3972">81</a></sup>. We first constructed a spanning tree backbone to ensure connectedness by transforming each element of the FC matrix <i></i><sub><i>i</i>,<i>j</i></sub> by <span class="mathjax-tex">\(\sqrt{2(1-{\rho }_{i,\,j})}\)</span>an operation that turns the correlation similarity measure into a distance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Mantegna, R. Hierarchical structure in financial markets. Eur. Phys. J. B 11, 193197 (1999)." href="/articles/s41593-023-01299-3#ref-CR82" id="ref-link-section-d30110227e4041">82</a></sup>and then applying Kruskals algorithm to find the minimum spanning tree<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Kruskal, J. B. On the shortest spanning subtree of a graph and the traveling salesman problem. Proc. Am. Math. Soc. 7, 4848 (1956)." href="/articles/s41593-023-01299-3#ref-CR83" id="ref-link-section-d30110227e4045">83</a></sup>. We iteratively added the strongest edges in the FC matrix to the spanning tree until the graph contained 10% of all possible edges (proportional thresholding). This produced an unweighted, undirected graph with a fixed number of edges.</p><h3 class="c-article__sub-heading" id="Sec26">Quantifying reliability</h3><p>We quantify univariate reliability using the ICC. ICC measures the reliability of a particular scalar measure across subjects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Noble, S., Scheinost, D. &amp; Constable, R. T. A decade of testretest reliability of functional connectivity: a systematic review and meta-analysis. Neuroimage 203, 116157 (2019)." href="/articles/s41593-023-01299-3#ref-CR37" id="ref-link-section-d30110227e4057">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Termenon, M., Jaillard, A., Delon-Martin, C. &amp; Achard, S. Reliability of graph analysis of resting state fMRI using test-retest dataset from the Human Connectome Project. Neuroimage 142, 172187 (2016)." href="/articles/s41593-023-01299-3#ref-CR84" id="ref-link-section-d30110227e4060">84</a></sup>. Let <i>N</i> be the total number of subjects, <i>M</i> be the number of sessions per subject and <span class="mathjax-tex">\({\gamma }_{n}^{m}\)</span> be some scalar measure of interest for session <i>m</i> of subject <i>n</i>. Similarly to an ANOVA, ICC decomposes the variance across subjects into variance from a common source, <span class="mathjax-tex">\({\sigma }_{\gamma }^{2}\)</span>, and noise, <i></i><sub><i></i></sub>, by assuming that <i></i> can be decomposed into a subject-specific term <i></i><sub><i>n</i></sub> and a noise term <i></i><sub><i>n</i>,<i>m</i></sub>that is, <span class="mathjax-tex">\({\gamma }_{n}^{m}={\gamma }_{n}+{\epsilon }_{n,m}\)</span>. Then, the ICC is defined as the fraction of variance explained by the subject-specific termthat is,</p><div id="Equd" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mbox{ICC}}}\,=\frac{{\sigma }_{\gamma }}{{\sigma }_{\gamma }+{\sigma }_{\epsilon }}.$$</span></div></div><p>In theory, there are generalizations of ICC that can accommodate additional sources of variance. Here, we use the simplest form, ICC(1,1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Noble, S., Scheinost, D. &amp; Constable, R. T. A decade of testretest reliability of functional connectivity: a systematic review and meta-analysis. Neuroimage 203, 116157 (2019)." href="/articles/s41593-023-01299-3#ref-CR37" id="ref-link-section-d30110227e4320">37</a></sup>. Significance compared to 0 and confidence intervals (CIs) were computed using the formulas from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="McGraw, K. O. &amp; Wong, S. P. Forming inferences about some intraclass correlation coefficients. Psychol. Methods 1, 3046 (1996)." href="/articles/s41593-023-01299-3#ref-CR85" id="ref-link-section-d30110227e4324">85</a></sup>. Significant differences between ICC values were computed using bootstrapping. Data distribution was assumed to be normal, but this was not formally tested.</p><p>We quantify multivariate reliability using fingerprinting, similar to that performed in refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Noble, S. et al. Influences on the testretest reliability of functional connectivity MRI and its relationship with behavioral utility. Cereb. Cortex 27, 54155429 (2017)." href="/articles/s41593-023-01299-3#ref-CR20" id="ref-link-section-d30110227e4331">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 16641671 (2015)." href="/articles/s41593-023-01299-3#ref-CR34" id="ref-link-section-d30110227e4334">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Horien, C. et al. Considering factors affecting the connectome-based identification process: comment on Waller et al. Neuroimage 169, 172175 (2018)." href="/articles/s41593-023-01299-3#ref-CR86" id="ref-link-section-d30110227e4337">86</a></sup>. Let <span class="mathjax-tex">\({\gamma }_{n}^{m}\)</span> be a vector describing some measure (for example, regional TA-<sub>1</sub>) from a single session <i>m</i><span class="stix"></span>{1,,<i>M</i>} for subject <i>n</i><span class="stix"></span>{1,,<i>N</i>}. For each session <i>m</i> of each subject <i>n</i>, we compute</p><div id="Eque" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\Gamma (n,m)=\arg \mathop{\max }\limits_{{n}^{{\prime} }}\left(\mathop{\max }\limits_{{m}^{{\prime} },(n,m)\ne ({n}^{{\prime} },{m}^{{\prime} })}\,{{\mbox{corr}}}\,\left({\gamma }_{n}^{m},{\gamma }_{{n}^{{\prime} }}^{{m}^{{\prime} }}\right)\right),$$</span></div></div><p>where corr is Pearson correlation. The fingerprinting performance is given by</p><div id="Equf" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{1}{NM}\mathop{\sum}\limits_{m,n}{I}_{\Gamma (n,m) = n},$$</span></div></div><p>where <i>I</i> is the indicator function. Under this measure, chance performance is (<i>M</i>1)/(<i>N</i><i>M</i>1). For regional measures, the length of vector <span class="mathjax-tex">\({\gamma }_{n}^{m}\)</span> was equal to the number of regions in the parcellation <i>R</i>. For FC, the length of <span class="mathjax-tex">\({\gamma }_{n}^{m}\)</span> was <i>R</i>(<i>R</i>1)/2that is, the number of distinct elements in the FC matrix.</p><p>In the HCP and HCP-GSR datasets, subjects were scanned four times over 2days. For computing ICC, we used <i>M</i>=4. For fingerprinting, to obtain error estimates and increase the difficulty of fingerprinting, we performed fingerprinting on each possible pair of sessions 14 (<i>M</i>=2 performed six times instead of <i>M</i>=4 performed once). This means that each session had only one correct match of the 1,765 sessions in the pool, compared to one correct match for 126 in the pool in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. Nat. Neurosci. 18, 16641671 (2015)." href="/articles/s41593-023-01299-3#ref-CR34" id="ref-link-section-d30110227e4812">34</a></sup>. In the TRT dataset, subjects were scanned in six sessions across four different days for a total of 24 sessions per subject. Likewise, we used <i>M</i>=24 to compute ICC. By contrast, we performed fingerprinting for each session independently (that is, <i>M</i>=4 performed six times instead of <i>M</i>=24 performed once), so subjects had three correct matching sessions out of 47 in the pool. Without such increases in difficulty on these datasets, fingerprinting performance was near 100%. We could not compute ICC or perform fingerprinting on the Cam-CAN dataset, because only one rs-fMRI session was acquired per subject.</p><h3 class="c-article__sub-heading" id="Sec27">Regional homogeneity</h3><p>Regional homogeneity estimates how similar a voxels time series is to its neighbors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Zang, Y., Jiang, T., Lu, Y., He, Y. &amp; Tian, L. Regional homogeneity approach to fMRI data analysis. Neuroimage 22, 394400 (2004)." href="/articles/s41593-023-01299-3#ref-CR87" id="ref-link-section-d30110227e4834">87</a></sup>. Here, because we used a parcellated analysis, we tested the homogeneity of the voxels within a parcel. Following standard practice, we defined regional homogeneity as the Kendalls <i>W</i> of time series within a parcel. More concretely,</p><div id="Equg" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mbox{ReHo}}}\,=\frac{12}{{k}^{2}({n}^{3}-n)}\mathop{\sum }\limits_{i=1}^{n}{\left(\mathop{\sum }\limits_{\,j = 1}^{k}{r}_{i,\,j}-k{T}_{n}/n\right)}^{2}$$</span></div></div><p>where <i>r</i><sub><i>i</i>,<i>j</i></sub> is the rank of the <i>i</i>-th time point in the <i>j</i>-th voxel; <i>k</i> is the number of voxels in the parcel; <i>n</i> is the number of time points; and <i>T</i><sub><i>i</i></sub> is the <i>i-</i>th triangular number.</p><h3 class="c-article__sub-heading" id="Sec28">Lins concordance</h3><p>Lins concordance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Lin, L. I.-K. A concordance correlation coefficient to evaluate reproducibility. Biometrics 45, 255 (1989)." href="/articles/s41593-023-01299-3#ref-CR88" id="ref-link-section-d30110227e5065">88</a></sup> measures how well one variable reproduces another variable. In general, there are multiple ways for variables to be related to each other. On the one hand, they can be correlated with each other, meaning that variation in one corresponds to variation in the other. This is generally quantified using the Pearson correlation coefficient. On the other hand, it is possible for two variables to have very different values while still being highly correlated. In this case, there would be a high Pearson correlation coefficient while also having a high mean squared error. Lins concordance addresses this issue by modifying the definition of Pearsons correlation to account for differences in the values of the variables, returning its maximum only if the data have both a high Pearson correlation and a low mean squared error. In other words, it measures the scaled distance of the sample from the unity line in the scatter plot. Lins concordance is defined for two samples <i>x</i><sub>1</sub> and <i>x</i><sub>2</sub> as</p><div id="Equh" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{2\rm{cov}({x}_{1},{x}_{2})}{\mathrm{var}({x}_{1})+\mathrm{var}({x}_{2})+{(\bar{{x}_{1}}-\bar{{x}_{2}})}^{2}}$$</span></div></div><p>where <span class="mathjax-tex">\(\bar{x}\)</span> is the mean of <i>x</i>. Alternatively, it can be formulated as a reweighted version of the Pearson correlation, using the equivalent definition</p><div id="Equi" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mathrm{corr}}}\,({x}_{1},{x}_{2})\frac{2\sqrt{\mathrm{var}({x}_{1})}\sqrt{\mathrm{var}({x}_{2})}}{\mathrm{var}({x}_{1})+\mathrm{var}({x}_{2})+{(\bar{{x}_{1}}-\bar{{x}_{2}})}^{2}}.$$</span></div></div><p>Like Pearson correlation, Lins concordance can take values ranging from 1 to 1. It is 1 if and only if the variables are identical. It is close to 0 if there is no correlation or if the means or variances of the two populations differ. This can be seen in the second definition based on Pearson correlation, where the weight on the Pearson correlation becomes 1 as the means and variances of <i>x</i><sub>1</sub> and <i>x</i><sub>2</sub> approach each other. Likewise, Lins concordance is negative if the means and variances are similar but the Pearson correlation is negative.</p><h3 class="c-article__sub-heading" id="Sec29">Weighted graph metrics</h3><p>We consider three weighted graph metrics: the mean (mean-FC), variance (var-FC) and kurtosis (kurt-FC) of the FC matrix. Each is calculated by finding the corresponding statistic (mean, variance or kurtosis) of the upper triangular portion of the FC matrix, excluding the diagonal. In other words, for the FC matrix <i>M</i>, this is the mean, variance and kurtosis of the set <span class="mathjax-tex">\({{{\mathcal{M}}}}=\{{m}_{i,j}:i &lt; j\}\)</span>.</p><p>Note that the thresholding procedure destroys all explicit information about mean-FC, var-FC and kurt-FC. To understand why, recall that the thresholding procedure keeps a fixed fraction of edges. Thus, if we threshold an FC matrix, the number of edges depends only on the fixed fraction of edges we keep. For an FC matrix of size <i>N</i>, if we keep <i>q</i> fraction of edges, then the moments are identical to those of the binomial distribution: mean-FC of the thresholded matrix is <i>q</i>; the variance is <i>q</i>(1<i>q</i>); and the kurtosis is 1/<i>p</i><i>q</i>6. Therefore, no explicit information remains about mean-FC, var-FC or kurt-FC after thresholding.</p><p>Because TA can be measured for individual nodes, we also consider how it impacts local topology. To do this, we also define the corresponding nodal graph metrics for each that operates on rows of the FC matrix instead of the upper triangle: nodal mean-FC, the mean of the row; nodal var-FC, the variance of the row; and nodal kurt-FC, the kurtosis of the row. We exclude self-connectivity (that is, the diagonal of ones in the FC matrix) from these calculations. Thresholding does not destroy nodal mean-FC, nodal var-FC or nodal kurt-FC.</p><h3 class="c-article__sub-heading" id="Sec30">Unweighted graph metrics</h3><p>We consider six popular graph metrics to quantify the topology of the connectome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: uses and interpretations. Neuroimage 52, 10591069 (2010)." href="/articles/s41593-023-01299-3#ref-CR4" id="ref-link-section-d30110227e5673">4</a></sup>:</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec31">Assortativity</h4><p>Assortativity is a preference of high-degree nodes, or hubs, to connect to each other. Mathematically, this is the Pearson correlation between the degree of nodes connected by edges<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Newman, M. E. J. Assortative mixing in networks. Phys. Rev. Lett. 89, 208701 (2002)." href="/articles/s41593-023-01299-3#ref-CR89" id="ref-link-section-d30110227e5684">89</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec32">Clustering coefficient</h4><p>The clustering coefficient for a single node is the average relative number of triangles around a node. For adjacency matrix <i>A</i>, the nodal clustering coefficient for node <i>i</i> is</p><div id="Equj" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{1}{\left({\sum }_{j}{A}_{i,\,j}\right)\left(\left({\sum }_{j}{A}_{i,\,j}\right)-1\right)}\mathop{\sum}\limits_{j,k,\,j\ne k}{A}_{i,\,j}{A}_{j,k}{A}_{i,k}.$$</span></div></div><p>The clustering coefficient for the network is the average of the nodal clustering coefficients.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec33">Global efficiency</h4><p>The global efficiency is related to the average topological distance between nodes. Mathematically, it is the mean of the inverse of shortest path lengths between each pair of nodesthat is,</p><div id="Equk" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{1}{N(N-1)}\mathop{\sum}\limits_{i,\,j,i\ne j}\frac{1}{{s}_{i,\,j}},$$</span></div></div><p>where <i>s</i><sub><i>i</i>,<i>j</i></sub> indicates the shortest path between nodes <i>i</i> and <i>j</i>, and <i>N</i> is the number of nodes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Latora, V. &amp; Marchiori, M. Efficient behavior of small-world networks. Phys. Rev. Lett. 87, 198701 (2001)." href="/articles/s41593-023-01299-3#ref-CR90" id="ref-link-section-d30110227e6037">90</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec34">Local efficiency</h4><p>Local efficiency, similar to clustering, is the mean global efficiency on the subgraph of each nodes nearest neighbors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Latora, V. &amp; Marchiori, M. Efficient behavior of small-world networks. Phys. Rev. Lett. 87, 198701 (2001)." href="/articles/s41593-023-01299-3#ref-CR90" id="ref-link-section-d30110227e6050">90</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec35">Modularity</h4><p>Modularity quantifies the ability to break a network into communities such that the number of edges within the community is maximized and outside the community is minimized. Let <i>C</i><sub><i>i</i></sub> be some community assignment of node <i>i</i>, and let <i></i> be the Kronecker delta function. We can compute the quality of the community assignment <i>C</i><sub><i>i</i></sub> with the equation</p><div id="Equl" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\mathop{\sum}\limits_{i,\,j,i\ne j}(2{A}_{i,j}-1)(2\delta ({C}_{i},{C}_{j})-1).$$</span></div></div><p>The first term in the sum is 1 if the neurons are connected and 1 if they are not, and the second term is 1 if they are in the same community and 1 if they are not. Thus, this can be maximized if connected nodes fall into the same community and unconnected nodes do not. The modularity is defined as the maximum value of this function across all potential community assignments <span class="mathjax-tex">\({\{{C}_{i}\}}_{i}\)</span>, rescaled to fall between 0.5 and 1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Newman, M. E. J. &amp; Girvan, M. Finding and evaluating community structure in networks. Phys. Rev. E Stat. Nonlin. Soft Matter Phys. 9, 026113 (2004)." href="/articles/s41593-023-01299-3#ref-CR91" id="ref-link-section-d30110227e6267">91</a></sup>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec36">Transitivity</h4><p>The transitivity is the total number of three-way reciprocal connections compared to the total possible number of such connectionsthat is,</p><div id="Equm" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\frac{1}{{\sum }_{i}\left({\sum }_{j}{A}_{i,j}\right)\left(\left({\sum }_{j}{A}_{i,j}\right)-1\right)}\mathop{\sum}\limits_{i,j,k,i\ne j\ne k}{A}_{i,j}{A}_{j,k}{A}_{i,k}$$</span></div></div><p>Note that this is distinct from local efficiency because it considers the network as a whole rather than considering each node <i>i</i> individually and then averaging.</p><p>In addition to considering these graph metrics, we also consider two nodal graph metrics:</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec37">Degree</h4><p>The degree is a nodal metric that measures the total number of edges connected to a nodethat is,</p><div id="Equn" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mbox{degree}}}\,(i)=\mathop{\sum}\limits_{j}{A}_{i,j}$$</span></div></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec38">Centrality</h4><p>Betweenness centrality is a nodal metric that measures the faction of shortest paths between all pairs of nodes in a network that pass through the given node. It does not have a closed-form equation and must be computed using an algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Brandes, U. A faster algorithm for betweenness centrality. J. Math. Sociol. 25, 163177 (2001)." href="/articles/s41593-023-01299-3#ref-CR92" id="ref-link-section-d30110227e6584">92</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec39">Graph metrics linear model</h3><p>To quantify the impact of SA and TA jointly, we used a linear model. We fit the SA- + SA-<span class="mathjax-tex">\(\infty\)</span> model</p><div id="Equo" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mathrm{metric}}}\,={\beta }_{1}{\mathrm{SA}\hbox{-}\uplambda} +{\beta }_{2}{\mathrm{SA}\hbox{-}\infty}$$</span></div></div><p>or the All model</p><div id="Equp" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mathrm{metric}}}\,={\beta }_{1}{\mathrm{SA}\hbox{-}\uplambda} +{\beta }_{2}{\mathrm{SA}\hbox{-}\infty} +{\beta }_{3}{\mathrm{TA}\hbox{-}\Delta_1}$$</span></div></div><p>on 50% of the data, randomly chosen from all sessions. Shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig6">1</a> is the Spearman correlation of the 50% held-out data with the predictions of this linear model. We used Spearman correlation instead of <i>R</i><sup>2</sup> for fair comparison of SA-, SA-<span class="mathjax-tex">\(\infty\)</span> and global TA-<sub>1</sub> individually. Because the fit uses held-out data, the correlation with the linear model does not necessarily need to be higher than the correlation with any of the individual factors in the model. Because many HCP subjects are related, we used only 334 unrelated subjects to avoid the relatedness confound in our analysis, meaning that there were 167 subjects in the training and test sets.</p><h3 class="c-article__sub-heading" id="Sec40">Correlated spectral sampling</h3><p>The spatiotemporal model relies on generating spatially autocorrelated time series, each with a given power spectrum. We introduce the correlated spectral sampling algorithm to produce a set of time series related by a given covariance matrix, where each time series has a given power spectrum.</p><p>Diverse methods of generating time series that exhibit both TA and SA already exist, such as vector autoregressive (VAR) and OrnsteinUhlenbeck processes. These models directly simulate time series in the time domain, introducing TA and SA locally. However, prior research demonstrating power law dynamics in fMRI data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bullmore, E. et al. Colored noise and computational inference in neurophysiological (fMRI) time series analysis: resampling methods in time and wavelet domains. Hum. Brain Mapp. 12, 6178 (2001)." href="#ref-CR72" id="ref-link-section-d30110227e6833">72</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wagenmakers, E.-J., Farrell, S. &amp; Ratcliff, R. Estimation and interpretation of 1/f noise in human cognition. Psychon. Bull. Rev. 11, 579615 (2004)." href="#ref-CR73" id="ref-link-section-d30110227e6833_1">73</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Maxim, V. et al. Fractional Gaussian noise, functional MRI and Alzheimers disease. Neuroimage 25, 141158 (2005)." href="/articles/s41593-023-01299-3#ref-CR74" id="ref-link-section-d30110227e6836">74</a></sup> suggests that VAR and similar models are inadequate for our purposes, as they are unable to model complex power spectral properties. More complex variants of these models, such as VARFI and FIVAR, are able to capture long-memory dynamics, but they are unable to simulate from arbitrary power spectra<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Sela, R. J. &amp; Hurvich, C. M. Computationally efficient methods for two multivariate fractionally integrated models. J. Time Ser. Anal. 30, 631651 (2009)." href="/articles/s41593-023-01299-3#ref-CR76" id="ref-link-section-d30110227e6840">76</a></sup>.</p><p>Instead, we seek to generate power spectra that represent the tradeoff between long-memory dynamicsthat is, filtered 1/<i>f</i><sup><i></i></sup> noise for frequencies above 0.01Hzand optional white noise, with a flat power spectrum. To introduce SA in our method, we spatially embed these time series with a given covariance matrix <i>C</i>. In our case, <i>C</i> is given in Equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-023-01299-3#Equ2">2</a>). The usual method of spatially embedding time series with a given covariance matrix <i>C</i> is to multiply them by the matrix square root of <i>C</i>, equivalent to sampling from a multivariate normal distribution. However, this process involves the linear combination of distinct time series, which changes their power spectra, destroying the desired power spectrum. To avoid this confound, we introduce the correlated spectral sampling algorithm, a generalization of ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Timmer, J. &amp; Knig, M. On generating power law noise. Astron. Astrophys. 300, 707710 (1995)." href="/articles/s41593-023-01299-3#ref-CR93" id="ref-link-section-d30110227e6869">93</a></sup>, which is able to produce correlated time series with arbitrary power spectra. Correlated spectral sampling can alternatively be seen as a generalization of phase randomization to obtain specified correlational structure.</p><p><b>Algorithm: Correlated spectral sampling.</b></p><p>Let <i>N</i> be the number of desired time series, and let <i>T</i> be the even length of the desired time series. Given an <i>N</i><i>N</i> correlation matrix <i>C</i><sub><i>i</i>,<i>j</i></sub> and a set of power spectra <span class="stix"></span><i>X</i><sub><i>i</i></sub>[<i>k</i>]<span class="stix"></span><sup>2</sup>, we seek to generate a set of time series {<i>x</i><sub>1</sub>[<i>t</i>],,<i>x</i><sub><i>N</i></sub>[<i>t</i>]} such that each time series <i>x</i><sub><i>i</i></sub>[<i>t</i>] has a specified estimate of the power spectrum <span class="stix"></span><i>X</i><sub><i>i</i></sub>[<i>k</i>]<span class="stix"></span><sup>2</sup>, and the correlation matrix of the resulting time series has expected value <i>C</i><sub><i>i</i>,<i>j</i></sub>. Correlated spectral sampling operates by generating a set of complex frequency-domain coefficients in the domain [<i>T</i>/2+1,<i>T</i>/2], introducing SA and TA independently, and then performing an inverse discrete Fourier transform to create the desired set of time series.</p><ol class="u-list-style-none">
                  <li>
                    <span class="u-custom-list-number">1.</span>
                    
                      <p>Pick the desired power spectra estimates {<span class="stix"></span><i>X</i><sub>1</sub>[<i>k</i>]<span class="stix"></span><sup>2</sup>,,<span class="stix"></span><i>X</i><sub><i>N</i></sub>[<i>k</i>]<span class="stix"></span><sup>2</sup>}, representing the desired temporal structure, and <i>N</i><i>N</i> correlation matrix <i>C</i>, representing the desired spatial structure. These choices must satisfy the requirement that the matrix</p><div id="Equq" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\Sigma }_{i,j}=\frac{{C}_{i,j}}{\,{{\mbox{sim}}}\,(| {X}_{i}[k]| ,| {X}_{j}[k]| )}$$</span></div></div><p>is positive semi-definite, where sim denotes cosine similaritythat is,</p><div id="Equr" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mbox{sim}}}\,(| {X}_{i}[k]| ,| {X}_{j}[k]| )=\frac{{\sum }_{k}| {X}_{i}[k]| | {X}_{j}[k]| }{\sqrt{{\sum }_{k}| {X}_{i}[k]{| }^{2}}\sqrt{{\sum }_{k}| {X}_{j}[k]{| }^{2}}}.$$</span></div></div>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">2.</span>
                    
                      <p>For each region <i>n</i><span class="stix"></span>{1,,<i>N</i>}, and for each frequency <i>k</i> in the positive sub-Nyquist Fourier domain, 1<i>k</i><i>T</i>/2, sample two vectors from a multivariate normal distribution <span class="mathjax-tex">\({a}_{n}^{R}[k],{a}_{n}^{I}[k] \sim N(0,\Sigma )\)</span>.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">3.</span>
                    
                      <p>Set <span class="mathjax-tex">\({a}_{n}^{I}[T/2]=0\)</span> for all <i>n</i>. Because <i>T</i> is even, by way of symmetry the frequency-domain coefficients of the Nyquist frequency must be real. Set <span class="mathjax-tex">\({a}_{n}^{I}[0]=0\)</span> so that the resulting time series will be real.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">4.</span>
                    
                      <p>Form the complex frequency-domain coefficients by letting <span class="mathjax-tex">\({c}_{n}[k]=| {X}_{i}[k]| ({a}_{n}^{R}[k]+i{a}_{n}^{I}[k])\)</span> for 0<i>k</i><i>T</i>/2, and <span class="mathjax-tex">\({c}_{n}[-k]=\overline{{c}_{n}[k]}\)</span> for 0&lt;<i>k</i>&lt;<i>T</i>/2.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">5.</span>
                    
                      <p>Perform an inverse discrete Fourier transform for each <i>c</i><sub><i>n</i></sub>[<i>k</i>] to obtain the time series <i>x</i><sub><i>n</i></sub>[<i>t</i>].</p>
                    
                  </li>
                </ol><p>Each of the resulting time series <i>x</i><sub><i>i</i></sub>[<i>t</i>] will have a frequency spectrum approximately equal to <span class="mathjax-tex">\(| {X}_{i}[k]({a}_{n}^{R}[k]+i{a}_{n}^{I}[k]){| }^{2}=| {X}_{i}{| }^{2}\)</span>, the desired temporal structure. The proof of this algorithm is given in Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1.1</a>.</p><h3 class="c-article__sub-heading" id="Sec41">Relationship between TA-<sub>1</sub> and power spectra</h3><p>In addition to correlated spectral sampling, defining the spatiotemporal model requires us to determine the precise relationship between TA-<sub>1</sub> and the power spectrum, with a specific focus on the case when the original time series is mixed with white noise.</p><p>Suppose we have a finite time series <i>x</i>[<i>t</i>] with discrete Fourier transform <i>X</i>[<i>k</i>]. Then, its TA-<sub>1</sub> is</p><div id="Equs" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\,{{\mbox{corr}}}\,(x[t],x[t+1])=\frac{1}{\mathop{\sum }\nolimits_{k = 1}^{N-1}| X[k]{| }^{2}}\mathop{\sum }\limits_{k=1}^{N-1}| X[k]{| }^{2}\cos (2\pi k/N).$$</span></div></div><p>A proof is provided in Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1.2</a>. Notably, for a fixed-length time series with a 1/<i>f</i><sup><i></i></sup> power spectrum, this formula implies a bijective relationship between <i></i> and TA-<sub>1</sub> in the domain <i></i><span class="stix"></span>[0,2].</p><p>Additionally, adding white noise will cause a change in the power spectrum and, hence, will change TA-<sub>1</sub>. For a finite time series <i>y</i>[<i>t</i>]=<i>x</i>[<i>t</i>]+<i>w</i>[<i>t</i>] where <i>w</i>[<i>t</i>]~<i>N</i>(0,<i></i>) of length <i>N</i>,</p><div id="Equt" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\mathbb{E}}[\,{{\mbox{corr}}}\,(y[t],y[t+1])]=\frac{1}{N{\sigma }^{2}+\mathop{\sum }\nolimits_{k = 1}^{N-1}| X[k]{| }^{2}}\mathop{\sum }\limits_{k=1}^{N-1}| X[k]{| }^{2}\cos (2\pi k/N).$$</span></div></div><p>A proof is provided in Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1.3</a>. Note that this formula implies that, as the amount (or variance) of white noise is increased, the TA-<sub>1</sub> approaches 0.</p><h3 class="c-article__sub-heading" id="Sec42">Spatiotemporal model</h3><p>The spatiotemporal model generates surrogate time series that can be analyzed like rs-fMRI time series. It takes two parametersthe noiseless SA- (SA-<sup>gen</sup>) and the noiseless SA-<span class="mathjax-tex">\(\infty\)</span> (SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup>)and also uses two pieces of information from the datathe TA-<sub>1</sub> (<i></i><sub><i>i</i></sub>) from each region <i>i</i> and the Euclidean distance <i>D</i><sub><i>i</i>,<i>j</i></sub> between the centroids of each pair of regions <i>i</i> and <i>j</i>.</p><p>The model operates in two basic steps. First, we generate random time series that are both spatially and temporally autocorrelated. For this first step, all time series have uniformly high TA, and SA is determined by the parameters SA- and SA-<span class="mathjax-tex">\(\infty\)</span>. Using correlated spectral sampling, we generate <i>N</i> high-pass-filtered (cutoff frequency of 0.01Hz, 4th order Butterworth filter) Brownian noise time series (frequency spectrum 1/<i>f</i><sup>2</sup>) of length <i>T</i> such that, for time series <i>x</i><sub><i>i</i></sub>[<i>t</i>] and <i>x</i><sub><i>j</i></sub>[<i>t</i>], we have</p><div id="Equ2" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\mathbb{E}}(\,{{\mathrm{corr}}}\,({x}_{i}[t],{x}_{j}[t]))={\mathrm{SA}\hbox{-}\infty}^{{{{\mathrm{gen}}}}}+(1-{\mathrm{SA}\hbox{-}\infty}^{{{{\mathrm{gen}}}}}){e}^{-{D}_{i,\,j}/{\mathrm{SA}\hbox{-}\uplambda}^{{{{\mathrm{gen}}}}}}$$</span></div><div class="c-article-equation__number">
                    (2)
                </div></div><p>where corr is the Pearson correlation, similar to Equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-023-01299-3#Equ1">1</a>). The matrix consisting of all such expected correlations between regions <i>i</i> and <i>j</i> forms the matrix <i>C</i><sub><i>i</i>,<i>j</i></sub> required by correlated spectral sampling, such that</p><div id="Equu" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${C}_{i,\,j}={\mathbb{E}}(\,{{\mbox{corr}}}\,({x}_{i}[t],{x}_{j}[t])).$$</span></div></div><p>For non-negative SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup>, <i>C</i><sub><i>i</i>,<i>j</i></sub> will be positive semi-definite.</p><p>Second, we reduce the TA of each time series to match that of the original data. Brownian motion time series have high TA, and adding white noise to a time series reduces TA, so we add white noise to each time series <i>x</i>[<i>i</i>] until the TA-<sub>1</sub> of the time series is equal to the empirical TA-<sub>1</sub> (<i></i><sub><i>i</i></sub>). Thus, we have to choose a distinct amount of noise to add to each time series such that corr(<i>x</i><sub><i>i</i></sub>[<i>t</i>],<i>x</i><sub><i>i</i></sub>[<i>t</i>+1])=<i></i><sub><i>i</i></sub>. We update <i>x</i><sub><i>i</i></sub>[<i>t</i>]<i>x</i><sub><i>i</i></sub>[<i>t</i>]+<i>N</i>(0,<i></i><sup>2</sup>(<i></i><sub><i>i</i></sub>)) for some function <i></i><sup>2</sup>(<i></i><sub><i>i</i></sub>) given by</p><div id="Equ3" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\sigma }^{2}(\phi )=\frac{1}{{N}^{2}\phi }\mathop{\sum }\limits_{k=1}^{N-1}| X[k]{| }^{2}\left(\cos (2\pi k/N)-\phi \right).$$</span></div><div class="c-article-equation__number">
                    (3)
                </div></div><p>where <i>N</i> is the length of the time series. The derivation of this formula is provided in Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1.3</a>. The function <i></i><sup>2</sup>(<i></i><sub><i>i</i></sub>) is defined for <i></i><sub><i>i</i></sub>&gt;0, and the variance of a random variable must be non-negative, so <i></i><sub><i>i</i></sub> is truncated such that <i></i><sub><i>i</i></sub>0.0001 and <i></i><sup>2</sup>(<i></i><sub><i>i</i></sub>)0. Note that SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> are parameters of the underlying process and differ from the SA- and SA-<span class="mathjax-tex">\(\infty\)</span> of the generated time series due to the addition of noise. The resulting time series exhibit a spatial embedding given by the parcel centroid distances <i>D</i><sub><i>i</i>,<i>j</i></sub> and parameters SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> and have regional TA-<sub>1</sub> equal to the original time series.</p><h3 class="c-article__sub-heading" id="Sec43">Model variants</h3><p>We test several variants of the model to determine the importance of three key components of the model: added uncorrelated noise, TA and SA.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec44">TA only</h4><p>This model modifies the spatiotemporal model to remove the spatial embedding. Specifically, it fixes the parameters SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> to 0, such that corr(<i>x</i><sub><i>i</i></sub>[<i>t</i>],<i>x</i><sub><i>j</i></sub>[<i>t</i>])=<i>I</i><sub><i>i</i>=<i>j</i></sub> where <i>I</i> is the indicator function. It takes no parameters.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec45">SA only</h4><p>We generate random multivariate Gaussian noise with mean zero and covariance matrix</p><div id="Equv" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${C}_{i,j}={\mathrm{SA}\hbox{-}\infty}^{{{{\mathrm{gen}}}}}+(1-{\mathrm{SA}\hbox{-}\infty}^{{{{\mathrm{gen}}}}})\exp (-{D}_{i,j}/{\mathrm{SA}\hbox{-}\uplambda}^{{{{\mathrm{gen}}}}})$$</span></div></div><p>similar to Equation (<a data-track="click" data-track-label="link" data-track-action="equation anchor" href="/articles/s41593-023-01299-3#Equ1">1</a>). It takes two parameters, SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup>. Unlike the spatiotemporal model, <span class="mathjax-tex">\({\mathbb{E}}({\mathrm{SA}\hbox{-}\uplambda} )={\mathrm{SA}\hbox{-}\uplambda}^{{{{\mathrm{gen}}}}}\)</span> and <span class="mathjax-tex">\({\mathbb{E}}({\mathrm{SA}\hbox{-}\infty} )={\mathrm{SA}\hbox{-}\infty}^{{{{\mathrm{gen}}}}}\)</span>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec46">Intrinsic time scale with SA</h4><p>In our spatiotemporal model, we make sure time series have the desired TA-<sub>1</sub> by generating time series with uniformly high TA-<sub>1</sub> and then adding different magnitudes of white noise to each time series to match TA-<sub>1</sub> to the original time series. In this model, we do not add white noise to the time series. To match TA-<sub>1</sub> to the original time series, we generate time series directly with matched TA-<sub>1</sub>. This is possible because correlated spectral sampling does not require the time series power spectra to be identical. Thus, we achieve this diversity in TA-<sub>1</sub> by assuming that each time series has a filtered pink noise (1/<i>f</i><sup><i></i></sup>) temporal dynamics. Then, for each time series <i>i</i>, we find an exponent <i></i><sub><i>i</i></sub> such that the high-pass-filtered <span class="mathjax-tex">\(1/{f}^{{\alpha }_{i}}\)</span> spectrum has expected TA-<sub>1</sub> equal to <i></i><sub><i>i</i></sub>.</p><p>The success of this procedure requires us to choose <i></i><sub><i>i</i></sub> such that the high-pass-filtered <span class="mathjax-tex">\(1/{f}^{{\alpha }_{i}}\)</span> spectrum has TA-<sub>1</sub> equal to <i></i><sub><i>i</i></sub>. The mapping <i></i><sub><i>i</i></sub><i></i><sub><i>i</i></sub> can be determined by numerically inverting the <i></i><sub><i>i</i></sub><i></i><sub><i>i</i></sub> mapping implied by Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1.2</a>. High-pass filtering is performed at the level of the power spectrum by multiplying the square of the amplitude response of the filter by the power spectrum. Additionally, in this model, it is possible to determine the SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> parameters directly from the data, without the need to fit parameters, using correlated spectral sampling. As in the SA only model, <span class="mathjax-tex">\({\mathbb{E}}({\mathrm{SA}\hbox{-}\uplambda} )={\mathrm{SA}\hbox{-}\uplambda}^{\mathrm{gen}}\)</span> and <span class="mathjax-tex">\({\mathbb{E}}({\mathrm{SA\hbox{-}}\infty})={\mathrm{SA\hbox{-}}\infty}^{{{{\mathrm{gen}}}}}\)</span>.</p><p>Thus, in summary, this model makes the following modifications to the spatiotemporal model: (1) rather than simulating random walks (1/<i>f</i><sup><i></i></sup> where <i></i>=2), the spectral exponents <i></i><sub><i>i</i></sub> for each region <i>i</i> are chosen such that the resulting TA-<sub>1</sub> is equal to each regions TA-<sub>1</sub> value; and (2) no noise is added to the power law time series.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec47">Homogeneous TA-<sub>1</sub>
                           </h4><p>To allow SA and TA to be independently manipulated, we developed a homogeneous variant of the spatiotemporal model that treats TA as a parameter. Rather than use TA-<sub>1</sub> values computed from the original time series, this model uses a single fixed value of TA-<sub>1</sub>, TA-<span class="mathjax-tex">\({\Delta }_{1}^{{{{\rm{gen}}}}}\)</span>, for all regions. For simplicity, we fixed SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> to be the mean SA-<span class="mathjax-tex">\(\infty\)</span> across all networks. Thus, the model takes two parameters: TA-<span class="mathjax-tex">\({\Delta }_{1}^{{{{\rm{gen}}}}}\)</span> and SA-<sup>gen</sup>.</p><h3 class="c-article__sub-heading" id="Sec48">Eigensurrogate model</h3><p>Due to our use of eigenvalues for fitting, we developed the eigensurrogate model to test whether eigenvalues alone are capable of reproducing a phenomenon (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig11">6</a>). This tests the null hypothesis that an effect can be explained by its linear dimensionality. Unlike most of the other models that we considered, this produces surrogate FC matrices instead of surrogate time series.</p><p>We first performed an eigendecomposition of the correlation matrix (FC matrix) and then applied the procedure of ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Davies, P. I. &amp; Higham, N. J. Numerically stable generation of correlation matrices and their factors. BIT Numer. Math. 40, 640651 (2000)." href="/articles/s41593-023-01299-3#ref-CR94" id="ref-link-section-d30110227e10466">94</a></sup>. In brief, we sampled a random set of eigenvectors and then applied a series of rotations to set ones on the diagonal. We used the method as implemented by Scipy in the numerical Python stack.</p><p>Because this model creates surrogate FC matrices, we produced time series by sampling the maximum entropy time series that would produce such a correlation matrix. To do this, we numerically computed the matrix square root and multiplied it by an <i>N</i><i>T</i> matrix of standard normal iid random variables. This is equivalent to sampling each time point independently from a multivariate Gaussian distribution. In principle, it is possible to create temporally autocorrelated time series from this model (multiplying by temporally autocorrelated time series instead of iid standard normal random variables), but, because it operates at the level of the FC matrix, all methods of generating time series from the eigensurrogte method will produce an identical FC matrix.</p><h3 class="c-article__sub-heading" id="Sec49">Null models</h3><p>We also test several popular null models.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec50">Phase randomization</h4><p>The power spectrum amplitude of individual time series is preserved, but the phases are randomized (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig11">6</a>). This procedure is described in detail in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Theiler, J., Eubank, S., Longtin, A., Galdrikian, B. &amp; Farmer, J. D. Testing for nonlinearity in time series: the method of surrogate data. Physica D 58, 7794 (1992)." href="/articles/s41593-023-01299-3#ref-CR95" id="ref-link-section-d30110227e10497">95</a></sup>. A Fourier transform was performed on each regions time series. Each element of the complex-valued Fourier transform was randomly rotated on the unit circle, and the inverse transform was performed on the phase-randomized spectra.</p><p>Note that we used distinct random phases for each time series, contrary to many neuroimaging studies that use the same phase for a given frequency across all time series. This is because using the same phase for each time series preserves all cross-correlations between time series. In practice, this means that the surrogate FC matrix is identical to the original FC matrix, and, hence, it produces an identical graph.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec51">Zalesky matching</h4><p>This model matches the mean and the variance of the correlation matrix (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig11">6</a>) and is described in full in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. Neuroimage 60, 20962106 (2012)." href="/articles/s41593-023-01299-3#ref-CR30" id="ref-link-section-d30110227e10515">30</a></sup>. In brief, it matches the first two moments by iteratively computing correlation matrices from time series of different durations with different ground truth correlations. The process continues until the time series duration is found that maximally reproduces the mean and variance of the correlation matrix.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec52">Edge reshuffle</h4><p>We preserve the degree of each node while scrambling the edges (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig11">6</a>). This is accomplished by an iterative algorithm described in detail in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Maslov, S. &amp; Sneppen, K. Specificity and stability in topology of protein networks. Science 296, 910913 (2002)." href="/articles/s41593-023-01299-3#ref-CR96" id="ref-link-section-d30110227e10530">96</a></sup>. Two edges from a graph were selected at random, and the connections were swapped. This swap was iterated <i>k</i> times, where <i>k</i> was chosen here to be five times the number of edges in the network. The result is that each node has the same number of connections, but those connections are randomized.</p><h3 class="c-article__sub-heading" id="Sec53">Models not considered</h3><p>Despite our parameterization based on the TA-<sub>1</sub> we do not report on time series generated using autoregressive (AR) or vector autoregressive (VAR) models. These models have two limitations within this context. First, they do not reproduce the observed long-memory processes observed within rs-fMRI time series. Second, when fitting data using these models, the TA-<sub>1</sub> parameter fits to values very close to 1, resulting in parameter degeneracy.</p><p>Likewise, we did not directly fit the economical clustering model to data. Due to the variability in individual instantiations of this model and the lack of smoothness of the parameter space, this model is incompatible with our numerical fitting algorithm, thus preventing the use of similar methodology to perform the fitting. These issues, combined with the long execution time of the model, made such individual-level fitting infeasible.</p><p>We did not consider models in which only one of SA- and SA-<span class="mathjax-tex">\(\infty\)</span> is fit and the other is fixed. These two parameters were fit by optimizing a function to the subjects measured SA. However, we were unable to get consistent and interpretable fits to this function when only one of these two parameters was fit. In other words, fixing one of these parameters precludes reliable estimation of the other.</p><p>Lastly, we only considered approaches that operate at the level of the parcellated time series. This excluded approaches such as scrambling using a three-dimensional Fourier transform.</p><h3 class="c-article__sub-heading" id="Sec54">Model fitting procedure</h3><p>Due to the fact that uncorrelated random noise was added after the spatial embedding in our model, the SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> parameters in the spatiotemporal model were not identical to the observed SA- and SA-<span class="mathjax-tex">\(\infty\)</span>. We derived a mathematical method for directly matching the spatiotemporal models SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> parameters to the data without the use of fitting, but technical constraints prevented an implementation of the procedure (Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">1.4)</a>. Therefore, we could not directly estimate the model based on observed SA- and SA-<span class="mathjax-tex">\(\infty\)</span>.</p><p>Instead, the spatiotemporal models SA-<sup>gen</sup> and SA-<span class="mathjax-tex">\(\infty\)</span><sup>gen</sup> parameters were fit to the eigenvalue distribution of each individual subjects FC matrix. A predicted FC matrix was generated by the model, and this procedure was iterated until a match was obtained in the eigenvalue distribution. The model was fit using differential evolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Storn, R. &amp; Price, K. Differential evolutiona simple and efficient heuristic for global optimization over continuous spaces. J. Glob. Optim. 11, 341359 (1997)." href="/articles/s41593-023-01299-3#ref-CR97" id="ref-link-section-d30110227e10688">97</a></sup>, a gradient-free global heuristic search method, to the mean squared error between the sorted eigenvalue distributions of the subject and model FC matrices. Eigenvalues were non-negative due to the positive semi-definiteness of the correlation matrix. The model was implemented in such a way that they preserved smoothness with respect to the parameters, meaning that, for a given random seed, small perturbations of the parameters caused only small changes in the time series and, hence, in the structure of the graph. Optimization was performed on the mean objective function from two random seeds. All reported statistics and metrics about the models come from a single instantiation of the model using a different random seed than either seed used during fitting. Parameters for the SA only model were also fit using the same procedure.</p><p>Most alternative models had no parameters (TA only, Phase randomization, Zalesky matching and Edge reshuffle). For the Noiseless model, parameters could be fit directly using correlated spectral sampling. This formalism is not guaranteed to converge for all subjects. When correlated spectral sampling was unable to produce valid time series, we excluded these subjects from the analysis. Results were qualitatively similar when parameters were fit to the eigenvalue distribution as described above that forced parameters into valid regimes.</p><h3 class="c-article__sub-heading" id="Sec55">Economical clustering model</h3><p>As described previously, the spatiotemporal model produces time series for each brain regiongraphs can then be constructed by processing these time series the same way as subjects rs-fMRI time series. Although the spatiotemporal model can be called a generative model by most definitions, within the graph theory literature, the term generative model often refers to models that construct graphs directly through the iterative addition of nodes or edges<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Vrtes, P. E. et al. Simple models of human brain functional networks. Proc. Natl Acad. Sci. USA 109, 58685873 (2012)." href="/articles/s41593-023-01299-3#ref-CR23" id="ref-link-section-d30110227e10703">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Betzel, R. F. et al. Generative models of the human connectome. Neuroimage 124, 10541064 (2016)." href="/articles/s41593-023-01299-3#ref-CR40" id="ref-link-section-d30110227e10706">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Bassett, D. S. &amp; Sporns, O. Network neuroscience. Nat. Neurosci. 20, 353364 (2017)." href="/articles/s41593-023-01299-3#ref-CR98" id="ref-link-section-d30110227e10709">98</a></sup>. The economical clustering (EC) model is one such model that is popular for studying brain networks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Vrtes, P. E. et al. Simple models of human brain functional networks. Proc. Natl Acad. Sci. USA 109, 58685873 (2012)." href="/articles/s41593-023-01299-3#ref-CR23" id="ref-link-section-d30110227e10713">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Betzel, R. F. et al. Generative models of the human connectome. Neuroimage 124, 10541064 (2016)." href="/articles/s41593-023-01299-3#ref-CR40" id="ref-link-section-d30110227e10716">40</a></sup>. In this model, connections between nodes are determined by one parameter governing the impact of distance and one for the impact of clustered topology. The probability of an edge forming between two brain regions is proportional to the product of the Euclidean distance between the regions raised to some power (the distance parameter) and the fraction of shared neighbors between them raised to some power (the clustering parameter). Full model details are provided in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Vrtes, P. E. et al. Simple models of human brain functional networks. Proc. Natl Acad. Sci. USA 109, 58685873 (2012)." href="/articles/s41593-023-01299-3#ref-CR23" id="ref-link-section-d30110227e10720">23</a></sup>.</p><p>To compare the two models, we simulated the EC model across a spectrum of distance and clustering parameters and then fit a spatiotemporal model to the simulated networks. Because our spatiotemporal model takes two SA-related parameters and obtains TA on a regional level directly from the data, we compared the EC model to the homogeneous variant of the spatiotemporal model, which includes one SA parameter and one TA parameter. We used the EC model to simulate ten networks per combination of parameters, fitting the homogeneous spatiotemporal model to each of these ten networks. Because the EC model produced graphs rather than FC matrices, we could not use our previous approach of fitting by eigenvalues, nor could we derive an analytic approach to parameter estimation. Thus, we fit using the objective function from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Betzel, R. F. et al. Generative models of the human connectome. Neuroimage 124, 10541064 (2016)." href="/articles/s41593-023-01299-3#ref-CR40" id="ref-link-section-d30110227e10727">40</a></sup>. Full details are provided in Supplementary Information <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM1">2</a>.</p><h3 class="c-article__sub-heading" id="Sec56">Spatial correction for brain map similarity</h3><p>To assess the similarity between brain maps, the presence of SA can induce a high false-positive rate. To correct for this, we perform a permutation test using an SA-preserving surrogate method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Burt, J. B., Helmer, M., Shinn, M., Anticevic, A. &amp; Murray, J. D. Generative modeling of brain maps with spatial autocorrelation. Neuroimage 220, 117038 (2020)." href="/articles/s41593-023-01299-3#ref-CR11" id="ref-link-section-d30110227e10742">11</a></sup>. To compare a target brain map to a reference map, we generate 10,000 surrogate brain maps that match the SA of the reference map. Then, we compute the Spearman correlation between the target map and the reference map as well as between the target map and the surrogate maps. The <i>P</i> value of the two-tailed test is determined as the fraction of correlations with the surrogate maps that are at least as extreme in absolute value as with the reference map. For cases in which maps cannot be designated as a target or reference map, we perform this procedure twice, once with each map taking the role of the reference map and the other as the target map, and compute the two-tailed <i>P</i> value as the total number of target-to-surrogate Spearman correlations that are at least as extreme in absolute value as the Spearman correlation between the two maps.</p><h3 class="c-article__sub-heading" id="Sec57">SVD</h3><p>We computed a cortical map of serotonergic modulation using SVD, which bears many similarities to principal component analysis (PCA). For a data matrix <i>M</i>, we can rewrite <i>M</i> as</p><div id="Equw" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$M=U\Sigma {V}^{T}$$</span></div></div><p>where  is a diagonal matrix, and the rows of <i>U</i> and <i>V</i> form orthogonal bases. The diagonal elements of  are called singular values; the rows of <i>V</i> are called singular vectors (each element of which is a loading); and the projection of <i>M</i> onto <i>V</i> (or, equivalently, <i>U</i>) are the scores. Note also that the product <i>M</i><sup><i>T</i></sup><i>M</i>=<i>V</i><sup>2</sup><i>V</i><sup><i>T</i></sup>, so <sup>2</sup> and <i>V</i> are the eigenvalues and eigenvectors, respectively, of <i>M</i><sup><i>T</i></sup><i>M</i>. If <i>M</i> is centered, then <i>M</i><sup><i>T</i></sup><i>M</i> is the covariance matrix, and SVD is equivalent to PCA, meaning that <sup>2</sup> gives the variance explained of each component. But, in our case, because <i>M</i> is not centered, variance explained of the first <i>k</i> components is computed as</p><div id="Equx" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\mathrm{var}\left(\mathop{\sum }\limits_{i=1}^{k}{\sigma }_{i}{u}_{i}{v}_{i}^{T}\right).$$</span></div></div><p>To compute the variance explained by experimental condition, we find the variance explained by each experiment from each subject individually and then average across the experimental condition.</p><h3 class="c-article__sub-heading" id="Sec58">Ethics declaration</h3><p>All participants provided written informed consent statements before participation in the study. The HCP data were acquired using protocols approved by the Washington University institutional review board. The Yale-TRT data were collected with approval by the Yale University institutional review board. The Cam-CAN data were collected with approval by the Cambridgeshire 2 Research Ethics Committee. The LSD and psilocybin data were collected with approval by the Cantonal Ethics Committee of Zurich, and the Swiss Federal Office of Public Health authorized the use of LSD and psilocybin in humans.</p><h3 class="c-article__sub-heading" id="Sec59">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41593-023-01299-3#MOESM2">Nature Portfolio Reporting Summary</a> linked to this article.</p></div></div></section>
                </div>
            

            <div>
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>The HCP data are available at <a href="https://www.humanconnectome.org/study/hcp-young-adult">https://www.humanconnectome.org/study/hcp-young-adult</a>. The Yale-TRT data are available at http://fcon_1000.projects.nitrc.org/indi/retro/yale_trt.html. The Cam-CAN data are available at <a href="https://www.cam-can.org/index.php?content=dataset">https://www.cam-can.org/index.php?content=dataset</a>. The LSD data and psilocybin data are available upon reasonable request.</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>We prepared a software package that allows the principal analyses in this paper to be performed quickly and easily. The spatiotemporal Python package, which can be installed through pip or downloaded at <a href="https://github.com/murraylab/spatiotemporal">https://github.com/murraylab/spatiotemporal</a>, offers a more user-friendly way of applying the analyses described here.</p>
              <p>The raw source code used to perform the analyses in this paper can be downloaded at <a href="https://github.com/murraylab/spatial_and_temporal_paper">https://github.com/murraylab/spatial_and_temporal_paper</a>. Code was implemented using the standard Python stack<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Harris, C. R. et al. Array programming with NumPy. Nature 585, 357362 (2020)." href="/articles/s41593-023-01299-3#ref-CR99" id="ref-link-section-d30110227e11132">99</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in python. Nat. Methods 17, 261272 (2020)." href="/articles/s41593-023-01299-3#ref-CR100" id="ref-link-section-d30110227e11135">100</a></sup> and other libraries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 101" title="LaPlante, R. A., Douw, L., Tang, W. &amp; Stufflebeam, S. M. The connectome visualization utility: software for visualization of human brain networks. PLoS ONE 9, e113838 (2014)." href="/articles/s41593-023-01299-3#ref-CR101" id="ref-link-section-d30110227e11139">101</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="Vallat, R. Pingouin: statistics in Python. J. Open Source Softw. 3, 1026 (2018)." href="/articles/s41593-023-01299-3#ref-CR102" id="ref-link-section-d30110227e11142">102</a></sup>. Source code was checked for correctness using software verification techniques<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Shinn, M. Refinement type contracts for verification of scientific investigative software. in Verified Software. Theories, Tools, and Experiments (eds Chakraborty, S. &amp; Navas, J. A.) 143160 (Springer, 2020)." href="/articles/s41593-023-01299-3#ref-CR103" id="ref-link-section-d30110227e11146">103</a></sup>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Buckner, R. L., Krienen, F. M. &amp; Yeo, B. T. T. Opportunities and limitations of intrinsic functional connectivity MRI. <i>Nat. Neurosci.</i> <b>16</b>, 832837 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.3423" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.3423" aria-label="Article reference 1" data-doi="10.1038/nn.3423">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23799476" aria-label="PubMed reference 1">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Opportunities%20and%20limitations%20of%20intrinsic%20functional%20connectivity%20MRI&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.3423&amp;volume=16&amp;pages=832-837&amp;publication_year=2013&amp;author=Buckner%2CRL&amp;author=Krienen%2CFM&amp;author=Yeo%2CBTT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Smith, S. M. et al. Functional connectomics from resting-state fMRI. <i>Trends Cogn. Sci.</i> <b>17</b>, 666682 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.tics.2013.09.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.tics.2013.09.016" aria-label="Article reference 2" data-doi="10.1016/j.tics.2013.09.016">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24238796" aria-label="PubMed reference 2">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4004765" aria-label="PubMed Central reference 2">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectomics%20from%20resting-state%20fMRI&amp;journal=Trends%20Cogn.%20Sci.&amp;doi=10.1016%2Fj.tics.2013.09.016&amp;volume=17&amp;pages=666-682&amp;publication_year=2013&amp;author=Smith%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Fornito, A., Zalesky, A. &amp; Bullmore, E. <i>Fundamentals of Brain Network Analysis</i> (Academic Press, 2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Rubinov, M. &amp; Sporns, O. Complex network measures of brain connectivity: uses and interpretations. <i>Neuroimage</i> <b>52</b>, 10591069 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2009.10.003" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2009.10.003" aria-label="Article reference 4" data-doi="10.1016/j.neuroimage.2009.10.003">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19819337" aria-label="PubMed reference 4">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Complex%20network%20measures%20of%20brain%20connectivity%3A%20uses%20and%20interpretations&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2009.10.003&amp;volume=52&amp;pages=1059-1069&amp;publication_year=2010&amp;author=Rubinov%2CM&amp;author=Sporns%2CO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Betzel, R. F. et al. Changes in structural and functional connectivity among resting-state networks across the human lifespan. <i>Neuroimage</i> <b>102</b>, 345357 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.07.067" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.07.067" aria-label="Article reference 5" data-doi="10.1016/j.neuroimage.2014.07.067">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25109530" aria-label="PubMed reference 5">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Changes%20in%20structural%20and%20functional%20connectivity%20among%20resting-state%20networks%20across%20the%20human%20lifespan&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.07.067&amp;volume=102&amp;pages=345-357&amp;publication_year=2014&amp;author=Betzel%2CRF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Damoiseaux, J. S. Effects of aging on functional and structural brain connectivity. <i>Neuroimage</i> <b>160</b>, 3240 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.01.077" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.01.077" aria-label="Article reference 6" data-doi="10.1016/j.neuroimage.2017.01.077">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28159687" aria-label="PubMed reference 6">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20aging%20on%20functional%20and%20structural%20brain%20connectivity&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2017.01.077&amp;volume=160&amp;pages=32-40&amp;publication_year=2017&amp;author=Damoiseaux%2CJS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Preller, K. H. et al. Changes in global and thalamic brain connectivity in LSD-induced altered states of consciousness are attributable to the 5-HT2A receptor. <i>eLife</i> <b>7</b>, e35082 (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Luppi, A. I. et al. LSD alters dynamic integration and segregation in the human brain. <i>Neuroimage</i> <b>227</b>, 117653 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2020.117653" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2020.117653" aria-label="Article reference 8" data-doi="10.1016/j.neuroimage.2020.117653">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33338615" aria-label="PubMed reference 8">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=LSD%20alters%20dynamic%20integration%20and%20segregation%20in%20the%20human%20brain&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2020.117653&amp;volume=227&amp;publication_year=2021&amp;author=Luppi%2CAI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Murphy, K., Birn, R. M. &amp; Bandettini, P. A. Resting-state fMRI confounds and cleanup. <i>Neuroimage</i> <b>80</b>, 349359 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.04.001" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.04.001" aria-label="Article reference 9" data-doi="10.1016/j.neuroimage.2013.04.001">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23571418" aria-label="PubMed reference 9">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting-state%20fMRI%20confounds%20and%20cleanup&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.04.001&amp;volume=80&amp;pages=349-359&amp;publication_year=2013&amp;author=Murphy%2CK&amp;author=Birn%2CRM&amp;author=Bandettini%2CPA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Friston, K. J. et al. Analysis of fMRI time-series revisited. <i>Neuroimage</i> <b>2</b>, 4553 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.1995.1007" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.1995.1007" aria-label="Article reference 10" data-doi="10.1006/nimg.1995.1007">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK1c%2FgtlCksQ%3D%3D" aria-label="CAS reference 10">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9343589" aria-label="PubMed reference 10">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Analysis%20of%20fMRI%20time-series%20revisited&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.1995.1007&amp;volume=2&amp;pages=45-53&amp;publication_year=1995&amp;author=Friston%2CKJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Burt, J. B., Helmer, M., Shinn, M., Anticevic, A. &amp; Murray, J. D. Generative modeling of brain maps with spatial autocorrelation. <i>Neuroimage</i> <b>220</b>, 117038 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2020.117038" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2020.117038" aria-label="Article reference 11" data-doi="10.1016/j.neuroimage.2020.117038">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32585343" aria-label="PubMed reference 11">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Generative%20modeling%20of%20brain%20maps%20with%20spatial%20autocorrelation&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2020.117038&amp;volume=220&amp;publication_year=2020&amp;author=Burt%2CJB&amp;author=Helmer%2CM&amp;author=Shinn%2CM&amp;author=Anticevic%2CA&amp;author=Murray%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Huang, Z., Liu, X., Mashour, G. A. &amp; Hudetz, A. G. Timescales of intrinsic BOLD signal dynamics and functional connectivity in pharmacologic and neuropathologic states of unconsciousness. <i>J. Neurosci.</i> <b>38</b>, 23042317 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.2545-17.2018" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.2545-17.2018" aria-label="Article reference 12" data-doi="10.1523/JNEUROSCI.2545-17.2018">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1GjtLnM" aria-label="CAS reference 12">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29386261" aria-label="PubMed reference 12">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5830518" aria-label="PubMed Central reference 12">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Timescales%20of%20intrinsic%20BOLD%20signal%20dynamics%20and%20functional%20connectivity%20in%20pharmacologic%20and%20neuropathologic%20states%20of%20unconsciousness&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.2545-17.2018&amp;volume=38&amp;pages=2304-2317&amp;publication_year=2018&amp;author=Huang%2CZ&amp;author=Liu%2CX&amp;author=Mashour%2CGA&amp;author=Hudetz%2CAG">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Sethi, S. S., Zerbi, V., Wenderoth, N., Fornito, A. &amp; Fulcher, B. D. Structural connectome topology relates to regional BOLD signal dynamics in the mouse brain. <i>Chaos</i> <b>27</b>, 047405 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1063/1.4979281" data-track-action="article reference" href="https://doi.org/10.1063%2F1.4979281" aria-label="Article reference 13" data-doi="10.1063/1.4979281">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28456172" aria-label="PubMed reference 13">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20connectome%20topology%20relates%20to%20regional%20BOLD%20signal%20dynamics%20in%20the%20mouse%20brain&amp;journal=Chaos&amp;doi=10.1063%2F1.4979281&amp;volume=27&amp;publication_year=2017&amp;author=Sethi%2CSS&amp;author=Zerbi%2CV&amp;author=Wenderoth%2CN&amp;author=Fornito%2CA&amp;author=Fulcher%2CBD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Fallon, J. et al. Timescales of spontaneous fMRI fluctuations relate to structural connectivity in the brain. <i>Netw. Neurosci.</i> <b>4</b>, 788806 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1162/netn_a_00151" data-track-action="article reference" href="https://doi.org/10.1162%2Fnetn_a_00151" aria-label="Article reference 14" data-doi="10.1162/netn_a_00151">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33615091" aria-label="PubMed reference 14">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7888482" aria-label="PubMed Central reference 14">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Timescales%20of%20spontaneous%20fMRI%20fluctuations%20relate%20to%20structural%20connectivity%20in%20the%20brain&amp;journal=Netw.%20Neurosci.&amp;doi=10.1162%2Fnetn_a_00151&amp;volume=4&amp;pages=788-806&amp;publication_year=2020&amp;author=Fallon%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Fascianelli, V., Tsujimoto, S., Marcos, E. &amp; Genovesio, A. Autocorrelation structure in the macaque dorsolateral, but not orbital or polar, prefrontal cortex predicts response-coding strength in a visually cued strategy task. <i>Cereb. Cortex</i> <b>29</b>, 230241 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhx321" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhx321" aria-label="Article reference 15" data-doi="10.1093/cercor/bhx321">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29228110" aria-label="PubMed reference 15">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Autocorrelation%20structure%20in%20the%20macaque%20dorsolateral%2C%20but%20not%20orbital%20or%20polar%2C%20prefrontal%20cortex%20predicts%20response-coding%20strength%20in%20a%20visually%20cued%20strategy%20task&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhx321&amp;volume=29&amp;pages=230-241&amp;publication_year=2019&amp;author=Fascianelli%2CV&amp;author=Tsujimoto%2CS&amp;author=Marcos%2CE&amp;author=Genovesio%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Arbabshirani, M. R. et al. Autoconnectivity: A new perspective on human brain function. <i>J. Neurosci. Methods</i> <b>323</b>, 6876 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.jneumeth.2019.03.015" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jneumeth.2019.03.015" aria-label="Article reference 16" data-doi="10.1016/j.jneumeth.2019.03.015">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31005575" aria-label="PubMed reference 16">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Autoconnectivity%3A%20A%20new%20perspective%20on%20human%20brain%20function&amp;journal=J.%20Neurosci.%20Methods&amp;doi=10.1016%2Fj.jneumeth.2019.03.015&amp;volume=323&amp;pages=68-76&amp;publication_year=2019&amp;author=Arbabshirani%2CMR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Honey, C. J. et al. Slow cortical dynamics and the accumulation of information over long timescales. <i>Neuron</i> <b>76</b>, 423434 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2012.08.011" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2012.08.011" aria-label="Article reference 17" data-doi="10.1016/j.neuron.2012.08.011">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFClt7jN" aria-label="CAS reference 17">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23083743" aria-label="PubMed reference 17">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3517908" aria-label="PubMed Central reference 17">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Slow%20cortical%20dynamics%20and%20the%20accumulation%20of%20information%20over%20long%20timescales&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2012.08.011&amp;volume=76&amp;pages=423-434&amp;publication_year=2012&amp;author=Honey%2CCJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Song, H. F., Kennedy, H. &amp; Wang, X.-J. Spatial embedding of structural similarity in the cerebral cortex. <i>Proc. Natl Acad. Sci. USA</i> <b>111</b>, 1658016585 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1414153111" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1414153111" aria-label="Article reference 18" data-doi="10.1073/pnas.1414153111">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVGksbrM" aria-label="CAS reference 18">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25368200" aria-label="PubMed reference 18">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4246295" aria-label="PubMed Central reference 18">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20embedding%20of%20structural%20similarity%20in%20the%20cerebral%20cortex&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1414153111&amp;volume=111&amp;pages=16580-16585&amp;publication_year=2014&amp;author=Song%2CHF&amp;author=Kennedy%2CH&amp;author=Wang%2CX-J">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Glasser, M. F. et al. A multi-modal parcellation of human cerebral cortex. <i>Nature</i> <b>536</b>, 171178 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature18933" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature18933" aria-label="Article reference 19" data-doi="10.1038/nature18933">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1GrurnK" aria-label="CAS reference 19">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27437579" aria-label="PubMed reference 19">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4990127" aria-label="PubMed Central reference 19">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multi-modal%20parcellation%20of%20human%20cerebral%20cortex&amp;journal=Nature&amp;doi=10.1038%2Fnature18933&amp;volume=536&amp;pages=171-178&amp;publication_year=2016&amp;author=Glasser%2CMF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Noble, S. et al. Influences on the testretest reliability of functional connectivity MRI and its relationship with behavioral utility. <i>Cereb. Cortex</i> <b>27</b>, 54155429 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhx230" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhx230" aria-label="Article reference 20" data-doi="10.1093/cercor/bhx230">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28968754" aria-label="PubMed reference 20">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6248395" aria-label="PubMed Central reference 20">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Influences%20on%20the%20test%E2%80%93retest%20reliability%20of%20functional%20connectivity%20MRI%20and%20its%20relationship%20with%20behavioral%20utility&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhx230&amp;volume=27&amp;pages=5415-5429&amp;publication_year=2017&amp;author=Noble%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Shafto, M. A. et al. The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) study protocol: a cross-sectional, lifespan, multidisciplinary examination of healthy cognitive ageing. <i>BMC Neurol.</i> <b>14</b>, 204 (2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Li, C., Wang, H., de Haan, W., Stam, C. J. &amp; Mieghem, P. V. The correlation of metrics in complex networks with applications in functional brain networks. <i>J. Stat. Mech.</i> <b>2011</b>, P11018 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1088/1742-5468/2011/11/P11018" data-track-action="article reference" href="https://doi.org/10.1088%2F1742-5468%2F2011%2F11%2FP11018" aria-label="Article reference 22" data-doi="10.1088/1742-5468/2011/11/P11018">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20correlation%20of%20metrics%20in%20complex%20networks%20with%20applications%20in%20functional%20brain%20networks&amp;journal=J.%20Stat.%20Mech.&amp;doi=10.1088%2F1742-5468%2F2011%2F11%2FP11018&amp;volume=2011&amp;publication_year=2011&amp;author=Li%2CC&amp;author=Wang%2CH&amp;author=Haan%2CW&amp;author=Stam%2CCJ&amp;author=Mieghem%2CPV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Vrtes, P. E. et al. Simple models of human brain functional networks. <i>Proc. Natl Acad. Sci. USA</i> <b>109</b>, 58685873 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.1111738109" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1111738109" aria-label="Article reference 23" data-doi="10.1073/pnas.1111738109">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22467830" aria-label="PubMed reference 23">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3326510" aria-label="PubMed Central reference 23">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Simple%20models%20of%20human%20brain%20functional%20networks&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1111738109&amp;volume=109&amp;pages=5868-5873&amp;publication_year=2012&amp;author=V%C3%A9rtes%2CPE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Roberts, J. A. et al. The contribution of geometry to the human connectome. <i>Neuroimage</i> <b>124</b>, 379393 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.09.009" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.09.009" aria-label="Article reference 24" data-doi="10.1016/j.neuroimage.2015.09.009">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26364864" aria-label="PubMed reference 24">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20contribution%20of%20geometry%20to%20the%20human%20connectome&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.09.009&amp;volume=124&amp;pages=379-393&amp;publication_year=2016&amp;author=Roberts%2CJA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Cicchetti, D. V. &amp; Sparrow, S. A. Developing criteria for establishing interrater reliability of specific items: applications to assessment of adaptive behavior. <i>Am. J. Ment. Defic.</i> <b>86</b>, 127137 (1981).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaL38%2FovVKkuw%3D%3D" aria-label="CAS reference 25">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7315877" aria-label="PubMed reference 25">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Developing%20criteria%20for%20establishing%20interrater%20reliability%20of%20specific%20items%3A%20applications%20to%20assessment%20of%20adaptive%20behavior&amp;journal=Am.%20J.%20Ment.%20Defic.&amp;volume=86&amp;pages=127-137&amp;publication_year=1981&amp;author=Cicchetti%2CDV&amp;author=Sparrow%2CSA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Koo, T. K. &amp; Li, M. Y. A guideline of selecting and reporting intraclass correlation coefficients for reliability research. <i>J. Chiropr. Med.</i> <b>15</b>, 155163 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.jcm.2016.02.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jcm.2016.02.012" aria-label="Article reference 26" data-doi="10.1016/j.jcm.2016.02.012">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27330520" aria-label="PubMed reference 26">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4913118" aria-label="PubMed Central reference 26">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20guideline%20of%20selecting%20and%20reporting%20intraclass%20correlation%20coefficients%20for%20reliability%20research&amp;journal=J.%20Chiropr.%20Med.&amp;doi=10.1016%2Fj.jcm.2016.02.012&amp;volume=15&amp;pages=155-163&amp;publication_year=2016&amp;author=Koo%2CTK&amp;author=Li%2CMY">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Murray, J. D. et al. A hierarchy of intrinsic timescales across primate cortex. <i>Nat. Neurosci.</i> <b>17</b>, 16611663 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.3862" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.3862" aria-label="Article reference 27" data-doi="10.1038/nn.3862">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFGltLbN" aria-label="CAS reference 27">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25383900" aria-label="PubMed reference 27">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4241138" aria-label="PubMed Central reference 27">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20hierarchy%20of%20intrinsic%20timescales%20across%20primate%20cortex&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.3862&amp;volume=17&amp;pages=1661-1663&amp;publication_year=2014&amp;author=Murray%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Raut, R. V., Snyder, A. Z. &amp; Raichle, M. E. Hierarchical dynamics as a macroscopic organizing principle of the human brain. <i>Proc. Natl Acad. Sci. USA</i> <b>117</b>, 2089020897 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Arbabshirani, M. R. et al. Impact of autocorrelation on functional connectivity. <i>Neuroimage</i> <b>102</b>, 294308 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.07.045" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.07.045" aria-label="Article reference 29" data-doi="10.1016/j.neuroimage.2014.07.045">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25072392" aria-label="PubMed reference 29">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Impact%20of%20autocorrelation%20on%20functional%20connectivity&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.07.045&amp;volume=102&amp;pages=294-308&amp;publication_year=2014&amp;author=Arbabshirani%2CMR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Zalesky, A., Fornito, A. &amp; Bullmore, E. On the use of correlation as a measure of network connectivity. <i>Neuroimage</i> <b>60</b>, 20962106 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.02.001" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.02.001" aria-label="Article reference 30" data-doi="10.1016/j.neuroimage.2012.02.001">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22343126" aria-label="PubMed reference 30">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20use%20of%20correlation%20as%20a%20measure%20of%20network%20connectivity&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.02.001&amp;volume=60&amp;pages=2096-2106&amp;publication_year=2012&amp;author=Zalesky%2CA&amp;author=Fornito%2CA&amp;author=Bullmore%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Afyouni, S., Smith, S. M. &amp; Nichols, T. E. Effective degrees of freedom of the Pearsons correlation coefficient under autocorrelation. <i>Neuroimage</i> <b>199</b>, 609625 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2019.05.011" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2019.05.011" aria-label="Article reference 31" data-doi="10.1016/j.neuroimage.2019.05.011">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31158478" aria-label="PubMed reference 31">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Effective%20degrees%20of%20freedom%20of%20the%20Pearson%E2%80%99s%20correlation%20coefficient%20under%20autocorrelation&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2019.05.011&amp;volume=199&amp;pages=609-625&amp;publication_year=2019&amp;author=Afyouni%2CS&amp;author=Smith%2CSM&amp;author=Nichols%2CTE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Shafiei, G. et al. Topographic gradients of intrinsic dynamics across neocortex. <i>eLife</i> <b>9</b>, e62116 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Keitel, A. &amp; Gross, J. Individual human brain areas can be identified from their characteristic spectral activation fingerprints. <i>PLoS Biol.</i> <b>14</b>, e1002498 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.1002498" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.1002498" aria-label="Article reference 33" data-doi="10.1371/journal.pbio.1002498">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27355236" aria-label="PubMed reference 33">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4927181" aria-label="PubMed Central reference 33">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Individual%20human%20brain%20areas%20can%20be%20identified%20from%20their%20characteristic%20spectral%20activation%20fingerprints&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.1002498&amp;volume=14&amp;publication_year=2016&amp;author=Keitel%2CA&amp;author=Gross%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Finn, E. S. et al. Functional connectome fingerprinting: identifying individuals using patterns of brain connectivity. <i>Nat. Neurosci.</i> <b>18</b>, 16641671 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4135" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4135" aria-label="Article reference 34" data-doi="10.1038/nn.4135">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhs1ChurfI" aria-label="CAS reference 34">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26457551" aria-label="PubMed reference 34">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5008686" aria-label="PubMed Central reference 34">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectome%20fingerprinting%3A%20identifying%20individuals%20using%20patterns%20of%20brain%20connectivity&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4135&amp;volume=18&amp;pages=1664-1671&amp;publication_year=2015&amp;author=Finn%2CES">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Waller, L. et al. Evaluating the replicability, specificity, and generalizability of connectome fingerprints. <i>Neuroimage</i> <b>158</b>, 371377 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.07.016" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.07.016" aria-label="Article reference 35" data-doi="10.1016/j.neuroimage.2017.07.016">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28710040" aria-label="PubMed reference 35">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluating%20the%20replicability%2C%20specificity%2C%20and%20generalizability%20of%20connectome%20fingerprints&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2017.07.016&amp;volume=158&amp;pages=371-377&amp;publication_year=2017&amp;author=Waller%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Baria, A. et al. Linking human brain local activity fluctuations to structural and functional network architectures. <i>Neuroimage</i> <b>73</b>, 144155 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.01.072" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.01.072" aria-label="Article reference 36" data-doi="10.1016/j.neuroimage.2013.01.072">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC3szmvFyqsA%3D%3D" aria-label="CAS reference 36">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23396160" aria-label="PubMed reference 36">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Linking%20human%20brain%20local%20activity%20fluctuations%20to%20structural%20and%20functional%20network%20architectures&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.01.072&amp;volume=73&amp;pages=144-155&amp;publication_year=2013&amp;author=Baria%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Noble, S., Scheinost, D. &amp; Constable, R. T. A decade of testretest reliability of functional connectivity: a systematic review and meta-analysis. <i>Neuroimage</i> <b>203</b>, 116157 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2019.116157" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2019.116157" aria-label="Article reference 37" data-doi="10.1016/j.neuroimage.2019.116157">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31494250" aria-label="PubMed reference 37">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20decade%20of%20test%E2%80%93retest%20reliability%20of%20functional%20connectivity%3A%20a%20systematic%20review%20and%20meta-analysis&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2019.116157&amp;volume=203&amp;publication_year=2019&amp;author=Noble%2CS&amp;author=Scheinost%2CD&amp;author=Constable%2CRT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Cole, M. W., Yang, G. J., Murray, J. D., Repov, G. &amp; Anticevic, A. Functional connectivity change as shared signal dynamics. <i>J. Neurosci. Methods</i> <b>259</b>, 2239 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.jneumeth.2015.11.011" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jneumeth.2015.11.011" aria-label="Article reference 38" data-doi="10.1016/j.jneumeth.2015.11.011">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26642966" aria-label="PubMed reference 38">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20connectivity%20change%20as%20shared%20signal%20dynamics&amp;journal=J.%20Neurosci.%20Methods&amp;doi=10.1016%2Fj.jneumeth.2015.11.011&amp;volume=259&amp;pages=22-39&amp;publication_year=2016&amp;author=Cole%2CMW&amp;author=Yang%2CGJ&amp;author=Murray%2CJD&amp;author=Repov%C5%A1%2CG&amp;author=Anticevic%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Achard, S. A resilient, low-frequency, small-world human brain functional network with highly connected association cortical hubs. <i>J. Neurosci.</i> <b>26</b>, 6372 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1523/JNEUROSCI.3874-05.2006" data-track-action="article reference" href="https://doi.org/10.1523%2FJNEUROSCI.3874-05.2006" aria-label="Article reference 39" data-doi="10.1523/JNEUROSCI.3874-05.2006">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XmtV2gtQ%3D%3D" aria-label="CAS reference 39">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16399673" aria-label="PubMed reference 39">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6674299" aria-label="PubMed Central reference 39">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20resilient%2C%20low-frequency%2C%20small-world%20human%20brain%20functional%20network%20with%20highly%20connected%20association%20cortical%20hubs&amp;journal=J.%20Neurosci.&amp;doi=10.1523%2FJNEUROSCI.3874-05.2006&amp;volume=26&amp;pages=63-72&amp;publication_year=2006&amp;author=Achard%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Betzel, R. F. et al. Generative models of the human connectome. <i>Neuroimage</i> <b>124</b>, 10541064 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.09.041" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.09.041" aria-label="Article reference 40" data-doi="10.1016/j.neuroimage.2015.09.041">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26427642" aria-label="PubMed reference 40">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Generative%20models%20of%20the%20human%20connectome&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.09.041&amp;volume=124&amp;pages=1054-1064&amp;publication_year=2016&amp;author=Betzel%2CRF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Sullivan, E. V. &amp; Pfefferbaum, A. Diffusion tensor imaging and aging. <i>Neurosci. Biobehav. Rev.</i> <b>30</b>, 74961 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neubiorev.2006.06.002" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neubiorev.2006.06.002" aria-label="Article reference 41" data-doi="10.1016/j.neubiorev.2006.06.002">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16887187" aria-label="PubMed reference 41">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Diffusion%20tensor%20imaging%20and%20aging&amp;journal=Neurosci.%20Biobehav.%20Rev.&amp;doi=10.1016%2Fj.neubiorev.2006.06.002&amp;volume=30&amp;pages=749-61&amp;publication_year=2006&amp;author=Sullivan%2CEV&amp;author=Pfefferbaum%2CA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Watanabe, T., Rees, G. &amp; Masuda, N. Atypical intrinsic neural timescale in autism. <i>eLife</i> <b>8</b>, e42256 (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Preller, K. H. et al. Psilocybin induces time-dependent changes in global functional connectivity. <i>Biol. Psychiatry</i> <b>88</b>, 197207 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2019.12.027" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2019.12.027" aria-label="Article reference 43" data-doi="10.1016/j.biopsych.2019.12.027">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXjvFWhs7c%3D" aria-label="CAS reference 43">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32111343" aria-label="PubMed reference 43">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Psilocybin%20induces%20time-dependent%20changes%20in%20global%20functional%20connectivity&amp;journal=Biol.%20Psychiatry&amp;doi=10.1016%2Fj.biopsych.2019.12.027&amp;volume=88&amp;pages=197-207&amp;publication_year=2020&amp;author=Preller%2CKH">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Persson, B., Heykants, J. &amp; Hedner, T. Clinical pharmacokinetics of ketanserin. <i>Clin. Pharmacokinet.</i> <b>20</b>, 263279 (1991).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.2165/00003088-199120040-00002" data-track-action="article reference" href="https://doi.org/10.2165%2F00003088-199120040-00002" aria-label="Article reference 44" data-doi="10.2165/00003088-199120040-00002">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK3M3ktlemsw%3D%3D" aria-label="CAS reference 44">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2036747" aria-label="PubMed reference 44">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20pharmacokinetics%20of%20ketanserin&amp;journal=Clin.%20Pharmacokinet.&amp;doi=10.2165%2F00003088-199120040-00002&amp;volume=20&amp;pages=263-279&amp;publication_year=1991&amp;author=Persson%2CB&amp;author=Heykants%2CJ&amp;author=Hedner%2CT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Grady, C. L. &amp; Garrett, D. D. Understanding variability in the BOLD signal and why it matters for aging. <i>Brain Imaging Behav.</i> <b>8</b>, 274283 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1007/s11682-013-9253-0" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s11682-013-9253-0" aria-label="Article reference 45" data-doi="10.1007/s11682-013-9253-0">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20variability%20in%20the%20BOLD%20signal%20and%20why%20it%20matters%20for%20aging&amp;journal=Brain%20Imaging%20Behav.&amp;doi=10.1007%2Fs11682-013-9253-0&amp;volume=8&amp;pages=274-283&amp;publication_year=2013&amp;author=Grady%2CCL&amp;author=Garrett%2CDD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Zerbi, V. et al. Rapid reconfiguration of the functional connectome after chemogenetic locus coeruleus activation. <i>Neuron</i> <b>103</b>, 702718 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2019.05.034" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2019.05.034" aria-label="Article reference 46" data-doi="10.1016/j.neuron.2019.05.034">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Rapid%20reconfiguration%20of%20the%20functional%20connectome%20after%20chemogenetic%20locus%20coeruleus%20activation&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2019.05.034&amp;volume=103&amp;publication_year=2019&amp;author=Zerbi%2CV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Shafiei, G. et al. Dopamine signaling modulates the stability and integration of intrinsic brain networks. <i>Cereb. Cortex</i> <b>29</b>, 397409 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhy264" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhy264" aria-label="Article reference 47" data-doi="10.1093/cercor/bhy264">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6294404" aria-label="PubMed Central reference 47">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Dopamine%20signaling%20modulates%20the%20stability%20and%20integration%20of%20intrinsic%20brain%20networks&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhy264&amp;volume=29&amp;pages=397-409&amp;publication_year=2018&amp;author=Shafiei%2CG">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Turchi, J. et al. The basal forebrain regulates global resting-state fMRI fluctuations. <i>Neuron</i> <b>97</b>, 940952 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2018.01.032" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2018.01.032" aria-label="Article reference 48" data-doi="10.1016/j.neuron.2018.01.032">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20basal%20forebrain%20regulates%20global%20resting-state%20fMRI%20fluctuations&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2018.01.032&amp;volume=97&amp;publication_year=2018&amp;author=Turchi%2CJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Davey, C. E., Grayden, D. B., Egan, G. F. &amp; Johnston, L. A. Filtering induces correlation in fMRI resting state data. <i>Neuroimage</i> <b>64</b>, 728740 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2012.08.022" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2012.08.022" aria-label="Article reference 49" data-doi="10.1016/j.neuroimage.2012.08.022">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22939874" aria-label="PubMed reference 49">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Filtering%20induces%20correlation%20in%20fMRI%20resting%20state%20data&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2012.08.022&amp;volume=64&amp;pages=728-740&amp;publication_year=2013&amp;author=Davey%2CCE&amp;author=Grayden%2CDB&amp;author=Egan%2CGF&amp;author=Johnston%2CLA">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Cantwell, G. T. et al. Thresholding normally distributed data creates complex networks. <i>Phys. Rev.</i> <i>E</i> <b>101</b>, 062302 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Ito, T., Hearne, L. J. &amp; Cole, M. W. A cortical hierarchy of localized and distributed processes revealed via dissociation of task activations, connectivity changes, and intrinsic timescales. <i>Neuroimage</i> <b>221</b>, 117141 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2020.117141" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2020.117141" aria-label="Article reference 51" data-doi="10.1016/j.neuroimage.2020.117141">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32663642" aria-label="PubMed reference 51">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20cortical%20hierarchy%20of%20localized%20and%20distributed%20processes%20revealed%20via%20dissociation%20of%20task%20activations%2C%20connectivity%20changes%2C%20and%20intrinsic%20timescales&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2020.117141&amp;volume=221&amp;publication_year=2020&amp;author=Ito%2CT&amp;author=Hearne%2CLJ&amp;author=Cole%2CMW">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Chen, J. E. et al. Resting-state physiological networks. <i>Neuroimage</i> <b>213</b>, 116707 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2020.116707" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2020.116707" aria-label="Article reference 52" data-doi="10.1016/j.neuroimage.2020.116707">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32145437" aria-label="PubMed reference 52">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Resting-state%20%E2%80%98physiological%20networks%E2%80%99&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2020.116707&amp;volume=213&amp;publication_year=2020&amp;author=Chen%2CJE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Drew, P. J., Mateo, C., Turner, K. L., Yu, X. &amp; Kleinfeld, D. Ultra-slow oscillations in fMRI and resting-state connectivity: neuronal and vascular contributions and technical confounds. <i>Neuron</i> <b>107</b>, 782804 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2020.07.020" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2020.07.020" aria-label="Article reference 53" data-doi="10.1016/j.neuron.2020.07.020">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1SrsLvO" aria-label="CAS reference 53">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32791040" aria-label="PubMed reference 53">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7886622" aria-label="PubMed Central reference 53">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Ultra-slow%20oscillations%20in%20fMRI%20and%20resting-state%20connectivity%3A%20neuronal%20and%20vascular%20contributions%20and%20technical%20confounds&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2020.07.020&amp;volume=107&amp;pages=782-804&amp;publication_year=2020&amp;author=Drew%2CPJ&amp;author=Mateo%2CC&amp;author=Turner%2CKL&amp;author=Yu%2CX&amp;author=Kleinfeld%2CD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Baracchini, G. et al. Inter-regional BOLD signal variability is an organizational feature of functional brain networks. <i>Neuroimage</i> <b>237</b>, 118149 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Kriegeskorte, N., Cusack, R. &amp; Bandettini, P. How does an fMRI voxel sample the neuronal activity pattern: compact-kernel or complex spatiotemporal filter? <i>Neuroimage</i> <b>49</b>, 19651976 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2009.09.059" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2009.09.059" aria-label="Article reference 55" data-doi="10.1016/j.neuroimage.2009.09.059">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19800408" aria-label="PubMed reference 55">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20does%20an%20fMRI%20voxel%20sample%20the%20neuronal%20activity%20pattern%3A%20compact-kernel%20or%20complex%20spatiotemporal%20filter%3F&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2009.09.059&amp;volume=49&amp;pages=1965-1976&amp;publication_year=2010&amp;author=Kriegeskorte%2CN&amp;author=Cusack%2CR&amp;author=Bandettini%2CP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">West, K. L. et al. BOLD hemodynamic response function changes significantly with healthy aging. <i>Neuroimage</i> <b>188</b>, 198207 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2018.12.012" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2018.12.012" aria-label="Article reference 56" data-doi="10.1016/j.neuroimage.2018.12.012">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30529628" aria-label="PubMed reference 56">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=BOLD%20hemodynamic%20response%20function%20changes%20significantly%20with%20healthy%20aging&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2018.12.012&amp;volume=188&amp;pages=198-207&amp;publication_year=2019&amp;author=West%2CKL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Cohen, Z., Bonvento, G., Lacombe, P. &amp; Hamel, E. Serotonin in the regulation of brain microcirculation. <i>Prog. Neurobiol.</i> <b>50</b>, 335362 (1996).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/S0301-0082(96)00033-0" data-track-action="article reference" href="https://doi.org/10.1016%2FS0301-0082%2896%2900033-0" aria-label="Article reference 57" data-doi="10.1016/S0301-0082(96)00033-0">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK2sXhtVSqtrk%3D" aria-label="CAS reference 57">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9004349" aria-label="PubMed reference 57">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Serotonin%20in%20the%20regulation%20of%20brain%20microcirculation&amp;journal=Prog.%20Neurobiol.&amp;doi=10.1016%2FS0301-0082%2896%2900033-0&amp;volume=50&amp;pages=335-362&amp;publication_year=1996&amp;author=Cohen%2CZ&amp;author=Bonvento%2CG&amp;author=Lacombe%2CP&amp;author=Hamel%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Geerligs, L., Tsvetanov, K. A. &amp; Henson, R. N. Challenges in measuring individual differences in functional connectivity using fMRI: the case of healthy aging. <i>Hum. Brain Mapp.</i> <b>38</b>, 41254156 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/hbm.23653" data-track-action="article reference" href="https://doi.org/10.1002%2Fhbm.23653" aria-label="Article reference 58" data-doi="10.1002/hbm.23653">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28544076" aria-label="PubMed reference 58">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5518296" aria-label="PubMed Central reference 58">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Challenges%20in%20measuring%20individual%20differences%20in%20functional%20connectivity%20using%20fMRI%3A%20the%20case%20of%20healthy%20aging&amp;journal=Hum.%20Brain%20Mapp.&amp;doi=10.1002%2Fhbm.23653&amp;volume=38&amp;pages=4125-4156&amp;publication_year=2017&amp;author=Geerligs%2CL&amp;author=Tsvetanov%2CKA&amp;author=Henson%2CRN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Honey, C. J. et al. Predicting human resting-state functional connectivity from structural connectivity. <i>Proc. Natl Acad. Sci. USA</i> <b>106</b>, 20352040 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1073/pnas.0811168106" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0811168106" aria-label="Article reference 59" data-doi="10.1073/pnas.0811168106">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXitVKktbY%3D" aria-label="CAS reference 59">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19188601" aria-label="PubMed reference 59">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2634800" aria-label="PubMed Central reference 59">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Predicting%20human%20resting-state%20functional%20connectivity%20from%20structural%20connectivity&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0811168106&amp;volume=106&amp;pages=2035-2040&amp;publication_year=2009&amp;author=Honey%2CCJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Braga, R. M. &amp; Buckner, R. L. Parallel interdigitated distributed networks within the individual estimated by intrinsic functional connectivity. <i>Neuron</i> <b>95</b>, 457471 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2017.06.038" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2017.06.038" aria-label="Article reference 60" data-doi="10.1016/j.neuron.2017.06.038">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Parallel%20interdigitated%20distributed%20networks%20within%20the%20individual%20estimated%20by%20intrinsic%20functional%20connectivity&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2017.06.038&amp;volume=95&amp;publication_year=2017&amp;author=Braga%2CRM&amp;author=Buckner%2CRL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Kong, R. et al. Individual-specific areal-level parcellations improve functional connectivity prediction of behavior. <i>Cereb. Cortex</i> <b>31</b>, 44774500 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1093/cercor/bhab101" data-track-action="article reference" href="https://doi.org/10.1093%2Fcercor%2Fbhab101" aria-label="Article reference 61" data-doi="10.1093/cercor/bhab101">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33942058" aria-label="PubMed reference 61">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8757323" aria-label="PubMed Central reference 61">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Individual-specific%20areal-level%20parcellations%20improve%20functional%20connectivity%20prediction%20of%20behavior&amp;journal=Cereb.%20Cortex&amp;doi=10.1093%2Fcercor%2Fbhab101&amp;volume=31&amp;pages=4477-4500&amp;publication_year=2021&amp;author=Kong%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Balthazar, M. L. F., de Campos, B. M., Franco, A. R., Damasceno, B. P. &amp; Cendes, F. Whole cortical and default mode network mean functional connectivity as potential biomarkers for mild Alzheimers disease. <i>Psychiatry Res.</i> <b>221</b>, 3742 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.pscychresns.2013.10.010" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.pscychresns.2013.10.010" aria-label="Article reference 62" data-doi="10.1016/j.pscychresns.2013.10.010">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24268581" aria-label="PubMed reference 62">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole%20cortical%20and%20default%20mode%20network%20mean%20functional%20connectivity%20as%20potential%20biomarkers%20for%20mild%20Alzheimer%E2%80%99s%20disease&amp;journal=Psychiatry%20Res.&amp;doi=10.1016%2Fj.pscychresns.2013.10.010&amp;volume=221&amp;pages=37-42&amp;publication_year=2014&amp;author=Balthazar%2CMLF&amp;author=Campos%2CBM&amp;author=Franco%2CAR&amp;author=Damasceno%2CBP&amp;author=Cendes%2CF">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Wiedermann, M., Donges, J. F., Kurths, J. &amp; Donner, R. V. Spatial network surrogates for disentangling complex system structure from spatial embedding of nodes. <i>Phys. Rev. E</i> <b>93</b>, 042308 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1103/PhysRevE.93.042308" data-track-action="article reference" href="https://doi.org/10.1103%2FPhysRevE.93.042308" aria-label="Article reference 63" data-doi="10.1103/PhysRevE.93.042308">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27176313" aria-label="PubMed reference 63">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Spatial%20network%20surrogates%20for%20disentangling%20complex%20system%20structure%20from%20spatial%20embedding%20of%20nodes&amp;journal=Phys.%20Rev.%20E&amp;doi=10.1103%2FPhysRevE.93.042308&amp;volume=93&amp;publication_year=2016&amp;author=Wiedermann%2CM&amp;author=Donges%2CJF&amp;author=Kurths%2CJ&amp;author=Donner%2CRV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Demirta, M. et al. Hierarchical heterogeneity across human cortex shapes large-scale neural dynamics. <i>Neuron</i> <b>101</b>, 11811194 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuron.2019.01.017" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuron.2019.01.017" aria-label="Article reference 64" data-doi="10.1016/j.neuron.2019.01.017">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Hierarchical%20heterogeneity%20across%20human%20cortex%20shapes%20large-scale%20neural%20dynamics&amp;journal=Neuron&amp;doi=10.1016%2Fj.neuron.2019.01.017&amp;volume=101&amp;publication_year=2019&amp;author=Demirta%C5%9F%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Morgan, S. E., Achard, S., Termenon, M., Bullmore, E. T. &amp; Vrtes, P. E. Low-dimensional morphospace of topological motifs in human fMRI brain networks. <i>Netw. Neurosci.</i> <b>2</b>, 285302 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1162/netn_a_00038" data-track-action="article reference" href="https://doi.org/10.1162%2Fnetn_a_00038" aria-label="Article reference 65" data-doi="10.1162/netn_a_00038">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30215036" aria-label="PubMed reference 65">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6130546" aria-label="PubMed Central reference 65">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Low-dimensional%20morphospace%20of%20topological%20motifs%20in%20human%20fMRI%20brain%20networks&amp;journal=Netw.%20Neurosci.&amp;doi=10.1162%2Fnetn_a_00038&amp;volume=2&amp;pages=285-302&amp;publication_year=2018&amp;author=Morgan%2CSE&amp;author=Achard%2CS&amp;author=Termenon%2CM&amp;author=Bullmore%2CET&amp;author=V%C3%A9rtes%2CPE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Robinson, E. C. et al. MSM: a new flexible framework for multimodal surface matching. <i>Neuroimage</i> <b>100</b>, 414426 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.05.069" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.05.069" aria-label="Article reference 66" data-doi="10.1016/j.neuroimage.2014.05.069">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24939340" aria-label="PubMed reference 66">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=MSM%3A%20a%20new%20flexible%20framework%20for%20multimodal%20surface%20matching&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.05.069&amp;volume=100&amp;pages=414-426&amp;publication_year=2014&amp;author=Robinson%2CEC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Friedman, L., Glover, G. H., Krenz, D. &amp; Magnotta, V. Reducing inter-scanner variability of activation in a multicenter fMRI study: role of smoothness equalization. <i>Neuroimage</i> <b>32</b>, 16561668 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2006.03.062" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2006.03.062" aria-label="Article reference 67" data-doi="10.1016/j.neuroimage.2006.03.062">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16875843" aria-label="PubMed reference 67">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Reducing%20inter-scanner%20variability%20of%20activation%20in%20a%20multicenter%20fMRI%20study%3A%20role%20of%20smoothness%20equalization&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2006.03.062&amp;volume=32&amp;pages=1656-1668&amp;publication_year=2006&amp;author=Friedman%2CL&amp;author=Glover%2CGH&amp;author=Krenz%2CD&amp;author=Magnotta%2CV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Scheinost, D., Papademetris, X. &amp; Constable, R. T. The impact of image smoothness on intrinsic functional connectivity and head motion confounds. <i>Neuroimage</i> <b>95</b>, 1321 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2014.03.035" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2014.03.035" aria-label="Article reference 68" data-doi="10.1016/j.neuroimage.2014.03.035">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24657356" aria-label="PubMed reference 68">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20image%20smoothness%20on%20intrinsic%20functional%20connectivity%20and%20head%20motion%20confounds&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2014.03.035&amp;volume=95&amp;pages=13-21&amp;publication_year=2014&amp;author=Scheinost%2CD&amp;author=Papademetris%2CX&amp;author=Constable%2CRT">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Shen, X., Tokoglu, F., Papademetris, X. &amp; Constable, R. Groupwise whole-brain parcellation from resting-state fMRI data for network node identification. <i>Neuroimage</i> <b>82</b>, 403415 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2013.05.081" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2013.05.081" aria-label="Article reference 69" data-doi="10.1016/j.neuroimage.2013.05.081">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC3sjhtV2gug%3D%3D" aria-label="CAS reference 69">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23747961" aria-label="PubMed reference 69">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Groupwise%20whole-brain%20parcellation%20from%20resting-state%20fMRI%20data%20for%20network%20node%20identification&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2013.05.081&amp;volume=82&amp;pages=403-415&amp;publication_year=2013&amp;author=Shen%2CX&amp;author=Tokoglu%2CF&amp;author=Papademetris%2CX&amp;author=Constable%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Taylor, J. R. et al. The Cambridge Centre for Ageing and Neuroscience (Cam-CAN) data repository: structural and functional MRI, MEG, and cognitive data from a cross-sectional adult lifespan sample. <i>Neuroimage</i> <b>144</b>, 262269 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2015.09.018" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2015.09.018" aria-label="Article reference 70" data-doi="10.1016/j.neuroimage.2015.09.018">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26375206" aria-label="PubMed reference 70">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Cambridge%20Centre%20for%20Ageing%20and%20Neuroscience%20%28Cam-CAN%29%20data%20repository%3A%20structural%20and%20functional%20MRI%2C%20MEG%2C%20and%20cognitive%20data%20from%20a%20cross-sectional%20adult%20lifespan%20sample&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2015.09.018&amp;volume=144&amp;pages=262-269&amp;publication_year=2017&amp;author=Taylor%2CJR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Tzourio-Mazoyer, N. et al. Automated anatomical labeling of activations in SPM using a macroscopic anatomical parcellation of the MNI MRI single-subject brain. <i>Neuroimage</i> <b>15</b>, 273289 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1006/nimg.2001.0978" data-track-action="article reference" href="https://doi.org/10.1006%2Fnimg.2001.0978" aria-label="Article reference 71" data-doi="10.1006/nimg.2001.0978">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD38%2FltFCntw%3D%3D" aria-label="CAS reference 71">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11771995" aria-label="PubMed reference 71">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20anatomical%20labeling%20of%20activations%20in%20SPM%20using%20a%20macroscopic%20anatomical%20parcellation%20of%20the%20MNI%20MRI%20single-subject%20brain&amp;journal=Neuroimage&amp;doi=10.1006%2Fnimg.2001.0978&amp;volume=15&amp;pages=273-289&amp;publication_year=2002&amp;author=Tzourio-Mazoyer%2CN">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Bullmore, E. et al. Colored noise and computational inference in neurophysiological (fMRI) time series analysis: resampling methods in time and wavelet domains. <i>Hum. Brain Mapp.</i> <b>12</b>, 6178 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1002/1097-0193(200102)12:2<61::AID-HBM1004&gt;3.0.CO;2-W" data-track-action="article reference" href="https://doi.org/10.1002%2F1097-0193%28200102%2912%3A2%3C61%3A%3AAID-HBM1004%3E3.0.CO%3B2-W" aria-label="Article reference 72" data-doi="10.1002/1097-0193(200102)12:2<61::AID-HBM1004&gt;3.0.CO;2-W">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3M3htleltQ%3D%3D" aria-label="CAS reference 72">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11169871" aria-label="PubMed reference 72">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Colored%20noise%20and%20computational%20inference%20in%20neurophysiological%20%28fMRI%29%20time%20series%20analysis%3A%20resampling%20methods%20in%20time%20and%20wavelet%20domains&amp;journal=Hum.%20Brain%20Mapp.&amp;doi=10.1002%2F1097-0193%28200102%2912%3A2%3C61%3A%3AAID-HBM1004%3E3.0.CO%3B2-W&amp;volume=12&amp;pages=61-78&amp;publication_year=2001&amp;author=Bullmore%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Wagenmakers, E.-J., Farrell, S. &amp; Ratcliff, R. Estimation and interpretation of 1/<sup><i>f</i></sup> noise in human cognition. <i>Psychon. Bull. Rev.</i> <b>11</b>, 579615 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.3758/BF03196615" data-track-action="article reference" href="https://doi.org/10.3758%2FBF03196615" aria-label="Article reference 73" data-doi="10.3758/BF03196615">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15581115" aria-label="PubMed reference 73">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1479451" aria-label="PubMed Central reference 73">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Estimation%20and%20interpretation%20of%201%2Ff%CE%B1%20noise%20in%20human%20cognition&amp;journal=Psychon.%20Bull.%20Rev.&amp;doi=10.3758%2FBF03196615&amp;volume=11&amp;pages=579-615&amp;publication_year=2004&amp;author=Wagenmakers%2CE-J&amp;author=Farrell%2CS&amp;author=Ratcliff%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Maxim, V. et al. Fractional Gaussian noise, functional MRI and Alzheimers disease. <i>Neuroimage</i> <b>25</b>, 141158 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2004.10.044" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2004.10.044" aria-label="Article reference 74" data-doi="10.1016/j.neuroimage.2004.10.044">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15734351" aria-label="PubMed reference 74">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=Fractional%20Gaussian%20noise%2C%20functional%20MRI%20and%20Alzheimer%E2%80%99s%20disease&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2004.10.044&amp;volume=25&amp;pages=141-158&amp;publication_year=2005&amp;author=Maxim%2CV">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Achard, S., Bassett, D. S., Meyer-Lindenberg, A. &amp; Bullmore, E. Fractal connectivity of long-memory networks. <i>Phys. Rev. E</i> <b>77</b>, 036104 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1103/PhysRevE.77.036104" data-track-action="article reference" href="https://doi.org/10.1103%2FPhysRevE.77.036104" aria-label="Article reference 75" data-doi="10.1103/PhysRevE.77.036104">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Fractal%20connectivity%20of%20long-memory%20networks&amp;journal=Phys.%20Rev.%20E&amp;doi=10.1103%2FPhysRevE.77.036104&amp;volume=77&amp;publication_year=2008&amp;author=Achard%2CS&amp;author=Bassett%2CDS&amp;author=Meyer-Lindenberg%2CA&amp;author=Bullmore%2CE">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Sela, R. J. &amp; Hurvich, C. M. Computationally efficient methods for two multivariate fractionally integrated models. <i>J. Time Ser. Anal.</i> <b>30</b>, 631651 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/j.1467-9892.2009.00631.x" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1467-9892.2009.00631.x" aria-label="Article reference 76" data-doi="10.1111/j.1467-9892.2009.00631.x">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Computationally%20efficient%20methods%20for%20two%20multivariate%20fractionally%20integrated%20models&amp;journal=J.%20Time%20Ser.%20Anal.&amp;doi=10.1111%2Fj.1467-9892.2009.00631.x&amp;volume=30&amp;pages=631-651&amp;publication_year=2009&amp;author=Sela%2CRJ&amp;author=Hurvich%2CCM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Achard, S. &amp; Gannaz, I. Multivariate wavelet whittle estimation in long-range dependence. <i>J. Time Ser. Anal.</i> <b>37</b>, 476512 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1111/jtsa.12170" data-track-action="article reference" href="https://doi.org/10.1111%2Fjtsa.12170" aria-label="Article reference 77" data-doi="10.1111/jtsa.12170">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Multivariate%20wavelet%20whittle%20estimation%20in%20long-range%20dependence&amp;journal=J.%20Time%20Ser.%20Anal.&amp;doi=10.1111%2Fjtsa.12170&amp;volume=37&amp;pages=476-512&amp;publication_year=2015&amp;author=Achard%2CS&amp;author=Gannaz%2CI">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Lu, Z. Analysis of stationary and non-stationary long memory processes: estimation, applications and forecast. PhD thesis, cole normale suprieure de Cachan - ENS Cachan (2019); <a href="https://theses.hal.science/tel-00422376/document" data-track="click" data-track-action="external reference" data-track-label="https://theses.hal.science/tel-00422376/document">https://theses.hal.science/tel-00422376/document</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Hassani, H., Leonenko, N. &amp; Patterson, K. The sample autocorrelation function and the detection of long-memory processes. <i>Phys. A</i> <b>391</b>, 63676379 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.physa.2012.07.062" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.physa.2012.07.062" aria-label="Article reference 79" data-doi="10.1016/j.physa.2012.07.062">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20sample%20autocorrelation%20function%20and%20the%20detection%20of%20long-memory%20processes&amp;journal=Phys.%20A&amp;doi=10.1016%2Fj.physa.2012.07.062&amp;volume=391&amp;pages=6367-6379&amp;publication_year=2012&amp;author=Hassani%2CH&amp;author=Leonenko%2CN&amp;author=Patterson%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Scheffer, M. et al. Early-warning signals for critical transitions. <i>Nature</i> <b>461</b>, 5359 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nature08227" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature08227" aria-label="Article reference 80" data-doi="10.1038/nature08227">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtVygsbjI" aria-label="CAS reference 80">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19727193" aria-label="PubMed reference 80">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Early-warning%20signals%20for%20critical%20transitions&amp;journal=Nature&amp;doi=10.1038%2Fnature08227&amp;volume=461&amp;pages=53-59&amp;publication_year=2009&amp;author=Scheffer%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Bullmore, E. &amp; Sporns, O. Complex brain networks: graph theoretical analysis of structural and functional systems. <i>Nat. Rev. Neurosci.</i> <b>10</b>, 186198 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nrn2575" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrn2575" aria-label="Article reference 81" data-doi="10.1038/nrn2575">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhtlygtrg%3D" aria-label="CAS reference 81">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19190637" aria-label="PubMed reference 81">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Complex%20brain%20networks%3A%20graph%20theoretical%20analysis%20of%20structural%20and%20functional%20systems&amp;journal=Nat.%20Rev.%20Neurosci.&amp;doi=10.1038%2Fnrn2575&amp;volume=10&amp;pages=186-198&amp;publication_year=2009&amp;author=Bullmore%2CE&amp;author=Sporns%2CO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Mantegna, R. Hierarchical structure in financial markets. <i>Eur. Phys. J. B</i> <b>11</b>, 193197 (1999).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="noopener" data-track-label="10.1007/s100510050929" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/s100510050929" aria-label="Article reference 82" data-doi="10.1007/s100510050929">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK1MXmslCitbw%3D" aria-label="CAS reference 82">CAS</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Hierarchical%20structure%20in%20financial%20markets&amp;journal=Eur.%20Phys.%20J.%20B&amp;doi=10.1007%2Fs100510050929&amp;volume=11&amp;pages=193-197&amp;publication_year=1999&amp;author=Mantegna%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Kruskal, J. B. On the shortest spanning subtree of a graph and the traveling salesman problem. <i>Proc. Am. Math. Soc.</i> <b>7</b>, 4848 (1956).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1090/S0002-9939-1956-0078686-7" data-track-action="article reference" href="https://doi.org/10.1090%2FS0002-9939-1956-0078686-7" aria-label="Article reference 83" data-doi="10.1090/S0002-9939-1956-0078686-7">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20the%20shortest%20spanning%20subtree%20of%20a%20graph%20and%20the%20traveling%20salesman%20problem&amp;journal=Proc.%20Am.%20Math.%20Soc.&amp;doi=10.1090%2FS0002-9939-1956-0078686-7&amp;volume=7&amp;pages=48-48&amp;publication_year=1956&amp;author=Kruskal%2CJB">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Termenon, M., Jaillard, A., Delon-Martin, C. &amp; Achard, S. Reliability of graph analysis of resting state fMRI using test-retest dataset from the Human Connectome Project. <i>Neuroimage</i> <b>142</b>, 172187 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2016.05.062" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2016.05.062" aria-label="Article reference 84" data-doi="10.1016/j.neuroimage.2016.05.062">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BC2s%2FosVKhtw%3D%3D" aria-label="CAS reference 84">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27282475" aria-label="PubMed reference 84">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Reliability%20of%20graph%20analysis%20of%20resting%20state%20fMRI%20using%20test-retest%20dataset%20from%20the%20Human%20Connectome%20Project&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2016.05.062&amp;volume=142&amp;pages=172-187&amp;publication_year=2016&amp;author=Termenon%2CM&amp;author=Jaillard%2CA&amp;author=Delon-Martin%2CC&amp;author=Achard%2CS">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">McGraw, K. O. &amp; Wong, S. P. Forming inferences about some intraclass correlation coefficients. <i>Psychol. Methods</i> <b>1</b>, 3046 (1996).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1037/1082-989X.1.1.30" data-track-action="article reference" href="https://doi.org/10.1037%2F1082-989X.1.1.30" aria-label="Article reference 85" data-doi="10.1037/1082-989X.1.1.30">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Forming%20inferences%20about%20some%20intraclass%20correlation%20coefficients&amp;journal=Psychol.%20Methods&amp;doi=10.1037%2F1082-989X.1.1.30&amp;volume=1&amp;pages=30-46&amp;publication_year=1996&amp;author=McGraw%2CKO&amp;author=Wong%2CSP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Horien, C. et al. Considering factors affecting the connectome-based identification process: comment on Waller et al. <i>Neuroimage</i> <b>169</b>, 172175 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2017.12.045" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2017.12.045" aria-label="Article reference 86" data-doi="10.1016/j.neuroimage.2017.12.045">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29253655" aria-label="PubMed reference 86">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=Considering%20factors%20affecting%20the%20connectome-based%20identification%20process%3A%20comment%20on%20Waller%20et%20al&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2017.12.045&amp;volume=169&amp;pages=172-175&amp;publication_year=2018&amp;author=Horien%2CC">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Zang, Y., Jiang, T., Lu, Y., He, Y. &amp; Tian, L. Regional homogeneity approach to fMRI data analysis. <i>Neuroimage</i> <b>22</b>, 394400 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/j.neuroimage.2003.12.030" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.neuroimage.2003.12.030" aria-label="Article reference 87" data-doi="10.1016/j.neuroimage.2003.12.030">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15110032" aria-label="PubMed reference 87">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=Regional%20homogeneity%20approach%20to%20fMRI%20data%20analysis&amp;journal=Neuroimage&amp;doi=10.1016%2Fj.neuroimage.2003.12.030&amp;volume=22&amp;pages=394-400&amp;publication_year=2004&amp;author=Zang%2CY&amp;author=Jiang%2CT&amp;author=Lu%2CY&amp;author=He%2CY&amp;author=Tian%2CL">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Lin, L. I.-K. A concordance correlation coefficient to evaluate reproducibility. <i>Biometrics</i> <b>45</b>, 255 (1989).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.2307/2532051" data-track-action="article reference" href="https://doi.org/10.2307%2F2532051" aria-label="Article reference 88" data-doi="10.2307/2532051">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaL1M3kslKrtg%3D%3D" aria-label="CAS reference 88">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=2720055" aria-label="PubMed reference 88">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20concordance%20correlation%20coefficient%20to%20evaluate%20reproducibility&amp;journal=Biometrics&amp;doi=10.2307%2F2532051&amp;volume=45&amp;publication_year=1989&amp;author=Lin%2CLI-K">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Newman, M. E. J. Assortative mixing in networks. <i>Phys. Rev. Lett.</i> <b>89</b>, 208701 (2002).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Latora, V. &amp; Marchiori, M. Efficient behavior of small-world networks. <i>Phys. Rev. Lett.</i> <b>87</b>, 198701 (2001).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Newman, M. E. J. &amp; Girvan, M. Finding and evaluating community structure in networks. <i>Phys. Rev. E Stat. Nonlin. Soft Matter Phys.</i> <b>9</b>, 026113 (2004).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Brandes, U. A faster algorithm for betweenness centrality. <i>J. Math. Sociol.</i> <b>25</b>, 163177 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1080/0022250X.2001.9990249" data-track-action="article reference" href="https://doi.org/10.1080%2F0022250X.2001.9990249" aria-label="Article reference 92" data-doi="10.1080/0022250X.2001.9990249">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20faster%20algorithm%20for%20betweenness%20centrality&amp;journal=J.%20Math.%20Sociol.&amp;doi=10.1080%2F0022250X.2001.9990249&amp;volume=25&amp;pages=163-177&amp;publication_year=2001&amp;author=Brandes%2CU">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Timmer, J. &amp; Knig, M. On generating power law noise. <i>Astron. Astrophys.</i> <b>300</b>, 707710 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=On%20generating%20power%20law%20noise&amp;journal=Astron.%20Astrophys.&amp;volume=300&amp;pages=707-710&amp;publication_year=1995&amp;author=Timmer%2CJ&amp;author=K%C3%B6nig%2CM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Davies, P. I. &amp; Higham, N. J. Numerically stable generation of correlation matrices and their factors. <i>BIT Numer. Math.</i> <b>40</b>, 640651 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1023/A:1022384216930" data-track-action="article reference" href="https://doi.org/10.1023%2FA%3A1022384216930" aria-label="Article reference 94" data-doi="10.1023/A:1022384216930">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=Numerically%20stable%20generation%20of%20correlation%20matrices%20and%20their%20factors&amp;journal=BIT%20Numer.%20Math.&amp;doi=10.1023%2FA%3A1022384216930&amp;volume=40&amp;pages=640-651&amp;publication_year=2000&amp;author=Davies%2CPI&amp;author=Higham%2CNJ">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Theiler, J., Eubank, S., Longtin, A., Galdrikian, B. &amp; Farmer, J. D. Testing for nonlinearity in time series: the method of surrogate data. <i>Physica D</i> <b>58</b>, 7794 (1992).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1016/0167-2789(92)90102-S" data-track-action="article reference" href="https://doi.org/10.1016%2F0167-2789%2892%2990102-S" aria-label="Article reference 95" data-doi="10.1016/0167-2789(92)90102-S">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Testing%20for%20nonlinearity%20in%20time%20series%3A%20the%20method%20of%20surrogate%20data&amp;journal=Physica%20D&amp;doi=10.1016%2F0167-2789%2892%2990102-S&amp;volume=58&amp;pages=77-94&amp;publication_year=1992&amp;author=Theiler%2CJ&amp;author=Eubank%2CS&amp;author=Longtin%2CA&amp;author=Galdrikian%2CB&amp;author=Farmer%2CJD">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Maslov, S. &amp; Sneppen, K. Specificity and stability in topology of protein networks. <i>Science</i> <b>296</b>, 910913 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1126/science.1065103" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1065103" aria-label="Article reference 96" data-doi="10.1126/science.1065103">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XjsFymsr8%3D" aria-label="CAS reference 96">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11988575" aria-label="PubMed reference 96">PubMed</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=Specificity%20and%20stability%20in%20topology%20of%20protein%20networks&amp;journal=Science&amp;doi=10.1126%2Fscience.1065103&amp;volume=296&amp;pages=910-913&amp;publication_year=2002&amp;author=Maslov%2CS&amp;author=Sneppen%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Storn, R. &amp; Price, K. Differential evolutiona simple and efficient heuristic for global optimization over continuous spaces. <i>J. Glob. Optim.</i> <b>11</b>, 341359 (1997).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1023/A:1008202821328" data-track-action="article reference" href="https://doi.org/10.1023%2FA%3A1008202821328" aria-label="Article reference 97" data-doi="10.1023/A:1008202821328">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 97" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20evolution%E2%80%94a%20simple%20and%20efficient%20heuristic%20for%20global%20optimization%20over%20continuous%20spaces&amp;journal=J.%20Glob.%20Optim.&amp;doi=10.1023%2FA%3A1008202821328&amp;volume=11&amp;pages=341-359&amp;publication_year=1997&amp;author=Storn%2CR&amp;author=Price%2CK">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Bassett, D. S. &amp; Sporns, O. Network neuroscience. <i>Nat. Neurosci.</i> <b>20</b>, 353364 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/nn.4502" data-track-action="article reference" href="https://doi.org/10.1038%2Fnn.4502" aria-label="Article reference 98" data-doi="10.1038/nn.4502">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXjsVSmsrY%3D" aria-label="CAS reference 98">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28230844" aria-label="PubMed reference 98">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5485642" aria-label="PubMed Central reference 98">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20neuroscience&amp;journal=Nat.%20Neurosci.&amp;doi=10.1038%2Fnn.4502&amp;volume=20&amp;pages=353-364&amp;publication_year=2017&amp;author=Bassett%2CDS&amp;author=Sporns%2CO">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Harris, C. R. et al. Array programming with NumPy. <i>Nature</i> <b>585</b>, 357362 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41586-020-2649-2" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2649-2" aria-label="Article reference 99" data-doi="10.1038/s41586-020-2649-2">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlWmsbbN" aria-label="CAS reference 99">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32939066" aria-label="PubMed reference 99">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7759461" aria-label="PubMed Central reference 99">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 99" href="http://scholar.google.com/scholar_lookup?&amp;title=Array%20programming%20with%20NumPy&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2649-2&amp;volume=585&amp;pages=357-362&amp;publication_year=2020&amp;author=Harris%2CCR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100."><p class="c-article-references__text" id="ref-CR100">Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in python. <i>Nat. Methods</i> <b>17</b>, 261272 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1038/s41592-019-0686-2" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-019-0686-2" aria-label="Article reference 100" data-doi="10.1038/s41592-019-0686-2">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXislCjuro%3D" aria-label="CAS reference 100">CAS</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32015543" aria-label="PubMed reference 100">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7056644" aria-label="PubMed Central reference 100">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 100" href="http://scholar.google.com/scholar_lookup?&amp;title=SciPy%201.0%3A%20fundamental%20algorithms%20for%20scientific%20computing%20in%20python&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0686-2&amp;volume=17&amp;pages=261-272&amp;publication_year=2020&amp;author=Virtanen%2CP">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101."><p class="c-article-references__text" id="ref-CR101">LaPlante, R. A., Douw, L., Tang, W. &amp; Stufflebeam, S. M. The connectome visualization utility: software for visualization of human brain networks. <i>PLoS ONE</i> <b>9</b>, e113838 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0113838" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0113838" aria-label="Article reference 101" data-doi="10.1371/journal.pone.0113838">Article</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25437873" aria-label="PubMed reference 101">PubMed</a>
    <a data-track="click" rel="nofollow noopener" data-track-label="link" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4250035" aria-label="PubMed Central reference 101">PubMed Central</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 101" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20connectome%20visualization%20utility%3A%20software%20for%20visualization%20of%20human%20brain%20networks&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0113838&amp;volume=9&amp;publication_year=2014&amp;author=LaPlante%2CRA&amp;author=Douw%2CL&amp;author=Tang%2CW&amp;author=Stufflebeam%2CSM">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102."><p class="c-article-references__text" id="ref-CR102">Vallat, R. Pingouin: statistics in Python. <i>J. Open Source Softw.</i> <b>3</b>, 1026 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click" rel="nofollow noopener" data-track-label="10.21105/joss.01026" data-track-action="article reference" href="https://doi.org/10.21105%2Fjoss.01026" aria-label="Article reference 102" data-doi="10.21105/joss.01026">Article</a>
    <a data-track="click" data-track-action="google scholar reference" data-track-label="link" rel="nofollow noopener" aria-label="Google Scholar reference 102" href="http://scholar.google.com/scholar_lookup?&amp;title=Pingouin%3A%20statistics%20in%20Python&amp;journal=J.%20Open%20Source%20Softw.&amp;doi=10.21105%2Fjoss.01026&amp;volume=3&amp;publication_year=2018&amp;author=Vallat%2CR">
                    Google Scholar</a>
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103."><p class="c-article-references__text" id="ref-CR103">Shinn, M. Refinement type contracts for verification of scientific investigative software. in <i>Verified Software. Theories, Tools, and Experiments</i> (eds Chakraborty, S. &amp; Navas, J. A.) 143160 (Springer, 2020).</p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-023-01299-3?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank J. Burt for help with plotting; A. Howell for assistance with HCP data; and T. Ito and J. C. C. Vila for helpful discussions. Funding was provided by European Molecular Biology Organization grant ALTF 712-2021, the Winston Churchill Foundation of the United States and the Gruber Foundation (M.S.); National Institute of Mental Health (NIMH) grant K00MH122372 (S.N.); the Swiss National Science Foundation (P2ZHP1\161626) (K.H.P.); Agence Nationale de la Recherche (ANR-20-NEUC-003-01) and MIAI@Grenoble Alpes (ANR-19-P3IA-0003) (S.A.); the Usona Institute (2015-2056), the Heffter Research Institute (1-190420), the Swiss Neuromatrix Foundation (2015-103 and 2016-0111) and the Swiss National Science Foundation under the framework of Neuron Cofund (01EW1908) (F.X.V.); the National Institute for Health and Care Research (NIHR) (Senior Investigator Award) and the NIHR Cambridge Biomedical Research Centre (E.T.B.); SFARI (Pilot Award) (A.A. and J.D.M.); and the NIMH (R01MH112746) (J.D.M.). Data collection and sharing for this project were provided, in part, by the Cambridge Centre for Ageing and Neuroscience (Cam-CAN). Cam-CAN funding was provided by the UK Biotechnology and Biological Sciences Research Council (grant BB/H008217/1), together with support from the UK Medical Research Council and the University of Cambridge.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Edward T. Bullmore, John D. Murray.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Interdepartmental Neuroscience Program, Yale University, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">Maxwell Shinn,Dustin Scheinost,R. Todd Constable,Alan Anticevic&amp;John D. Murray</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Psychiatry, Yale University, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">Maxwell Shinn,Katrin H. Preller,Jie Lisa Ji,John H. Krystal,Alan Anticevic&amp;John D. Murray</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Yale College, Yale University, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">Amber Hu&amp;Laurel Turner</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Department of Radiology and Biomedical Imaging, Yale University, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">Stephanie Noble,Dustin Scheinost&amp;R. Todd Constable</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Department of Psychiatry, Psychotherapy and Psychosomatics, University Hospital for Psychiatry, Zurich, Switzerland</p><p class="c-article-author-affiliation__authors-list">Katrin H. Preller,Flora Moujaes&amp;Franz X. Vollenweider</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">University of Grenoble Alpes, CNRS, Inria, Grenoble INP, LJK, Grenoble, France</p><p class="c-article-author-affiliation__authors-list">Sophie Achard</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Zanvyl Krieger Mind/Brain Institute, Johns Hopkins University, Baltimore, MD, USA</p><p class="c-article-author-affiliation__authors-list">Daeyeol Lee</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Kavli Discovery Neuroscience Institute, Johns Hopkins University, Baltimore, MD, USA</p><p class="c-article-author-affiliation__authors-list">Daeyeol Lee</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Department of Psychological and Brain Sciences, Johns Hopkins University, Baltimore, MD, USA</p><p class="c-article-author-affiliation__authors-list">Daeyeol Lee</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Department of Neuroscience, Johns Hopkins University, Baltimore, MD, USA</p><p class="c-article-author-affiliation__authors-list">Daeyeol Lee</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Department of Psychiatry, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Edward T. Bullmore</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Department of Physics, Yale University, New Haven, CT, USA</p><p class="c-article-author-affiliation__authors-list">John D. Murray</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Maxwell-Shinn-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Maxwell Shinn</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Maxwell%20Shinn" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Maxwell%20Shinn" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Maxwell%20Shinn%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Amber-Hu-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Amber Hu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Amber%20Hu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Amber%20Hu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Amber%20Hu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Laurel-Turner-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Laurel Turner</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Laurel%20Turner" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Laurel%20Turner" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Laurel%20Turner%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stephanie-Noble-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Stephanie Noble</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stephanie%20Noble" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stephanie%20Noble" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stephanie%20Noble%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Katrin_H_-Preller-Aff2-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Katrin H. Preller</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Katrin%20H.%20Preller" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Katrin%20H.%20Preller" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Katrin%20H.%20Preller%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jie_Lisa-Ji-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Jie Lisa Ji</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jie%20Lisa%20Ji" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jie%20Lisa%20Ji" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jie%20Lisa%20Ji%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Flora-Moujaes-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Flora Moujaes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Flora%20Moujaes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Flora%20Moujaes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Flora%20Moujaes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sophie-Achard-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Sophie Achard</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sophie%20Achard" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sophie%20Achard" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sophie%20Achard%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Dustin-Scheinost-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Dustin Scheinost</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Dustin%20Scheinost" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Dustin%20Scheinost" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Dustin%20Scheinost%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-R__Todd-Constable-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">R. Todd Constable</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=R.%20Todd%20Constable" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=R.%20Todd%20Constable" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22R.%20Todd%20Constable%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-John_H_-Krystal-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">John H. Krystal</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=John%20H.%20Krystal" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=John%20H.%20Krystal" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20H.%20Krystal%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Franz_X_-Vollenweider-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Franz X. Vollenweider</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Franz%20X.%20Vollenweider" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Franz%20X.%20Vollenweider" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Franz%20X.%20Vollenweider%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Daeyeol-Lee-Aff7-Aff8-Aff9-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Daeyeol Lee</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Daeyeol%20Lee" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Daeyeol%20Lee" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Daeyeol%20Lee%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Alan-Anticevic-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Alan Anticevic</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Alan%20Anticevic" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alan%20Anticevic" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alan%20Anticevic%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Edward_T_-Bullmore-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Edward T. Bullmore</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Edward%20T.%20Bullmore" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Edward%20T.%20Bullmore" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Edward%20T.%20Bullmore%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-John_D_-Murray-Aff1-Aff2-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">John D. Murray</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=John%20D.%20Murray" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text">You can also search for this author in
                        <span class="c-article-identifiers"><a class="c-article-identifiers__item" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=John%20D.%20Murray" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide"></span><a class="c-article-identifiers__item" href="http://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20D.%20Murray%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>M.S. and E.T.B. conceived the research. M.S. designed the experiments. M.S. and A.H. performed the experiments. M.S. and J.D.M. analyzed and interpreted results. K.H.P., J.L.J., F.M., S.N., D.S., R.T.C., J.H.K., F.X.V. and A.A. contributed data, methodology and resources. M.S., L.T. and S.A. performed the mathematical analysis. D.L., E.T.B. and J.D.M. provided supervision and funding. M.S., A.H. and L.T. wrote the first draft of the manuscript. All authors edited, revised and approved the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:john.murray@yale.edu">John D. Murray</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>K.H.P. is currently an employee of Hoffmann-La Roche. J.H.K. has consulting agreements (less than $5,000 per year) with the following: Aptinyx; Atai Life Sciences; AstraZeneca Pharmaceuticals; Biogen; Biomedisyn Corporation; Bionomics; Boehringer Ingelheim; Cadent Therapeutics; Clexio Bioscience; COMPASS Pathways; Concert Pharmaceuticals; Epiodyne; EpiVario; Greenwich Biosciences; Heptares Therapeutics; Janssen Research &amp; Development; Jazz Pharmaceuticals; Otsuka America Pharmaceutical; Perception Neuroscience Holdings; Spring Care; Sunovion Pharmaceuticals; Takeda Industries; and Taisho Pharmaceutical Company. J.H.K. serves on the scientific advisory boards of Biohaven Pharmaceuticals; BioXcel Therapeutics (Clinical Advisory Board); Cadent Therapeutics (Clinical Advisory Board); Cerevel Therapeutics; EpiVario; Eisai; Jazz Pharmaceuticals; Lohocla Research Corporation; Novartis Pharmaceuticals Corporation; PsychoGenics; Neumora Therapeutics; Tempero Bio; and Terran Biosciences. J.H.K. is on the board of directors of Freedom Biosciences. J.H.K. has stock and/or stock options in Biohaven Pharmaceuticals; Sage Pharmaceuticals; Spring Care; Biohaven Pharmaceuticals Medical Sciences; EpiVario; Neumora Therapeutics; Terran Biosciences; and Tempero Bio. J.H.K. is editor of <i>Biological Psychiatry</i> with income greater than $10,000. D.L. is a co-founder of Neurogazer. A.A. and J.D.M. are co-founders of Manifest Technologies and serve on the technical advisory board of Neumora Therapeutics. E.T.B. serves on the scientific advisory board of Sosei Heptares and as a consultant for GlaxoSmithKline. The remaining authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Neuroscience</i> thanks Janine Bijsterbosch, Javier Gonzalez-Castillo, Lucina Uddin and the other, anonymous, reviewer(s) for their contribution to the peer review of this work.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publishers note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec61-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec61">Extended data</h2><div class="c-article-section__content" id="Sec61-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 sa and ta are important featu" href="/articles/s41593-023-01299-3/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig6_ESM.jpg">Extended Data Fig. 1 SA and TA are important features of rs-fMRI.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Data from the HCP-GSR (a-h, N=850 subjects), Yale-TRT (i-p, N=12 subjects, 4 sessions each), and Cam-CAN (q-t, N=646 subjects) datasets. For all subpanels, unless otherwise indicated, * indicates p&lt;0.05 and ** indicates <i>P</i>&lt;0.01 on a two-sided test, and boxplots indicate the median, first/third quartiles, and range, with outliers hidden for visualization. (a,i,q) Correlation across all subjects in the HCP dataset, with Bonferroni FWER corrected two-sided <i>P</i>-values. (b,i) Test-retest reliability of graph metrics, quantified by intraclass correlation coefficient (ICC). Error bars indicate 95% CI. Following each bar is a string of three characters indicating significance: the first indicates significantly less than SA-, the second SA-<span class="mathjax-tex">\(\infty\)</span>, and the third global TA-<sub>1</sub>, where # indicates <i>P</i>&lt;.01, + indicates <i>P</i>&lt;.05, and  indicates <i>P</i>&gt;.05, by a one-sided bootstrap resampling procedure. Inset scatterplots show correlation across subjects for two example sessions. (c,k,r) Correlation across subjects between graph metrics and SA-, SA-<span class="mathjax-tex">\(\infty\)</span>, or global TA-<sub>1</sub>. SA- + SA-<span class="mathjax-tex">\(\infty\)</span> and all indicate a cross-validated linear model with two or three terms, respectively, using Bonferroni FWER corrected two-sided <i>P</i>-values. (d,l,s) The brain map depicting regional TA-<sub>1</sub>, averaged across all subjects. (e,m) Distribution of reliability for each brain region. (f,n) Mean fraction of subjects correctly identified by a fingerprinting analysis. Points indicate identification performance on each of six possible test-retest pairs from the four sessions. (g,o,t) Correlation across regions of regional TA-<sub>1</sub> with nodal graph metrics for each subject. (k) The regional TA-<sub>1</sub> for each region plotted against its reliability.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 correlation of motion and par" href="/articles/s41593-023-01299-3/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig7_ESM.jpg">Extended Data Fig. 2 Correlation of motion and parcel size with spatial and temporal autocorrelation.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(a-c) For each dataset, the mean framewise displacement (motion) is compared to (a) SA-, (b) SA-<span class="mathjax-tex">\(\infty\)</span>, and (c) TA-<sub>1</sub>, with inset Spearman correlation (r<sub>s</sub>). (d) The average regional TA-<sub>1</sub> across all subjects is plotted against the regions parcel size, with inset Spearman correlation. Parcel size is measured in surface area for HCP and HCP-GSR, and number of voxels for Yale-TRT and Cam-CAN. (e) Subjects mean framewise displacement (motion) is plotted against the subjects age. * indicates Spearman correlation two-sided <i>P</i>&lt;.05, and ** indicates <i>P</i>&lt;.01. HCP: N=883, HCP-GSR: N=850, Yale-TRT: N=12 subjects, 4 sessions each, Cam-CAN: N=646, LSD: N=24, Psilocybin: N=23 subjects.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 ta-1 captures individual var" href="/articles/s41593-023-01299-3/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig8_ESM.jpg">Extended Data Fig. 3 TA-<sub>1</sub> captures individual variation in long memory dynamics.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(a-c,h-j) TA-<sub>1</sub>, the first lag term in the ACF, is more reliable than higher lag terms. (a,h) For each lag <i>k</i>, we computed the reliability of the corresponding term in the ACF, ACF<sub>x</sub>(k), as measured by ICC. The median reliability across brain regions decreased as <i>k</i> increased, and regional TA-<sub>1</sub> (that is, ACF<sub>x</sub>(1)) maximized median reliability. Additionally, <i>d</i> had similar reliability to regional TA-<sub>1</sub>. (b,i) To confirm this occurs for each brain region individually, we found the difference in reliability between the ACF<sub>x</sub>(1) and ACF<sub>x</sub>(k) (ICC), which also decreased across lags. <i>d</i> was similar to regional TA-<sub>1</sub>. (c,j) Global TA-<sub>1</sub> reliability also decreased with increasing lag, and <i>d</i> averaged across regions had similar reliability to global TA-<sub>1</sub>, as measured by ICC. Error bars indicate 95% confidence interval. Thus, the reliability at short compared to long time lags, and the similarity to <i>d</i>, motivate a prioritization for TA-<sub>1</sub> over higher lags. (d,k,o) TA-<sub>1</sub> is predictive of higher terms of the ACF. TA-<sub>1</sub> was used to predict higher ACF terms in a regression model (see Methods). Mean cross-validated R<sup>2</sup> is shown, where error bars (sometimes hidden under the line) indicate maximum and minimum cross-validated R<sup>2</sup>. (e-g,l-n,p-r) Individual variation in <i>d</i>, the long-memory or fractional integration term from an ARFIMA(0,<i>d</i>,0) model, can be captured by individual variation in TA-<sub>1</sub>. (e,l,p) regional TA-<sub>1</sub> averaged across subjects is highly correlated with regional <i>d</i> averaged across subjects. (e: <i>P</i>=0, l: <i>P</i>=10<sup>260</sup>, p: <i>P</i>=.10) (f,m,q) Global TA-<sub>1</sub> is highly correlated with <i>d</i> averaged across regions within a subject. (f: <i>P</i>=0, m: <i>P</i>=10<sup>101</sup> q: 10<sup>61</sup>) (g,n,r) Without averaging, across regions and subjects, d is correlated with TA-<sub>1</sub>. (g: <i>P</i>=0, n: <i>P</i>=0, r: <i>P</i>=0) For all figures, unless otherwise indicated, r<sub>s</sub> indicates Spearman correlation, where * indicates two-sided <i>P</i>&lt;.05, and ** indicates <i>P</i>&lt;.01, N=883 subjects.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 relationship between regional" href="/articles/s41593-023-01299-3/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig9_ESM.jpg">Extended Data Fig. 4 Relationship between regional TA-<sub>1</sub> and regional homogeneity.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Comparisons are shown for the HCP (a-c, N=883) and Yale-TRT (d-f, N=12, 4 sessions each) datasets. (a,d) The global TA-<sub>1</sub> averaged across subjects, compared to the regional homogeneity (ReHo) averaged across subjects. Each point represents a parcel. ** indicates two-sided Spearman correlation <i>P</i>&lt;.01. (a: <i>P</i>=10<sup>18</sup>, d: <i>P</i>=10<sup>39</sup>) (b,e) Distribution of correlations of global TA-<sub>1</sub> and regional homogeneity. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1j</a>. (c,f) Average fingerprinting performance of regional homogeneity, compared to global TA-<sub>1</sub> and chance. Points are overlaid for each pair of datasets compared. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1i</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 correlation of model and data" href="/articles/s41593-023-01299-3/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig10_ESM.jpg">Extended Data Fig. 5 Correlation of model and data graph metrics for all models.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>For each model, each subjects empirical graph metrics are plotted against model graph metrics for metrics from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2e</a>. Spearman correlation (r<sub>s</sub>) and Lins concordance (Lin) are inset. * indicates Spearman correlation two-sided <i>P</i>&lt;.05, and ** indicates <i>P</i>&lt;.01, N=883 subjects. Zalesky matching operates at the level of the FC matrix, and thus, TA-<sub>1</sub> could not be computed. Edge reshuffle operates at the level of the graph, preventing any of these measures from being computed.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 schematics of all models." href="/articles/s41593-023-01299-3/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig11_ESM.jpg">Extended Data Fig. 6 Schematics of all models.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>All models considered are described in their corresponding schematics. The spatiotemporal model is shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2a</a>.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 model fitting under geodesic " href="/articles/s41593-023-01299-3/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig12_ESM.jpg">Extended Data Fig. 7 Model fitting under geodesic distance.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>We analyzed the data and model using geodesic instead of Euclidean distance in the right hemisphere. (a-b) We computed SA- and SA-<span class="mathjax-tex">\(\infty\)</span> under geodesic distance for the right hemisphere. The geodesic and Euclidean distances lead to highly correlated measurements for SA- (a) and SA-<span class="mathjax-tex">\(\infty\)</span> (b). * indicates Spearman correlation two-sided <i>P</i>&lt;.05, and ** indicates <i>P</i>&lt;.01. (a: <i>P</i>=0, b: <i>P</i>=0) (c) Comparison of reliability of graph measures to SA- and SA-<span class="mathjax-tex">\(\infty\)</span> when computed using geodesic distance, as measured using ICC. Following each bar is a string of three characters: the first indicates significantly less than SA-, the second SA-<span class="mathjax-tex">\(\infty\)</span>, and the third global TA-<sub>1</sub>, where # indicates <i>P</i>&lt;.01, + indicates <i>P</i>&lt;.05, and - indicates <i>P</i>&gt;.05 by a bootstrap resampling procedure on a one-sided test. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1d</a>. (d) Correlation of graph metrics with SA-, SA-<span class="mathjax-tex">\(\infty\)</span>, a linear model of both (SA- + SA-<span class="mathjax-tex">\(\infty\)</span>), and a linear model incorporating both of these plus global TA-<sub>1</sub> (all), when computed using geodesic distance instead of Euclidean distance. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e</a>. * indicates Spearman correlation two-sided <i>P</i>&lt;.05, and ** indicates <i>P</i>&lt;.01. (e-g) The spatiotemporal model and all comparison models were fit using geodesic instead of Euclidean distance. (e) The similarity of graph metrics in the model and data, as measured by Lins concordance, under geodesic distance. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2e</a>. (f) The similarity of degree distribution in the model and data under geodesic distance. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2f</a>. (g) The similarity of nodal graph metrics in the model and data under geodesic distance. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2g</a>. For all panels, N=883 subjects.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 comparison of model fits for " href="/articles/s41593-023-01299-3/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig13_ESM.jpg">Extended Data Fig. 8 Comparison of model fits for all datasets.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(a,d,g) Lins concordance between model and data for each model. Bars represent the mean across four (HCP-GSR), six (Yale-TRT), or one (Cam-CAN) scanning sessions, and points indicate the Lins concordance between the model and data for each session. For comparison, black indicates Lins concordance between separate sessions from the same subject in the HCP-GSR and Yale-TRT datasets, where dots indicate pairs of sessions. (b,e,h) Log-log degree distribution for each model compared to the data (black). (c,f,i) Distribution of Lins concordance of nodal metrics between model and data for each region. Boxplots show median, first/third quartiles, and range, with outliers hidden for visualization, N=883 subjects. Compare to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig2">2e-g</a>. Statistics of these fits can be found in Table S1.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 relationship between the spat" href="/articles/s41593-023-01299-3/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig14_ESM.jpg">Extended Data Fig. 9 Relationship between the spatiotemporal model and economical clustering (EC) model.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(a) For three given values of the EC cluster parameter (<i></i>), the distance parameter (<i></i>) was varied and the TA-<span class="mathjax-tex">\({\Delta }_{1}^{{{{\rm{gen}}}}}\)</span> parameter of the best fit spatiotemporal model is shown. (b) For three given values of the EC distance parameter, the EC cluster parameter was varied and the SA-<sup>gen</sup> parameter of the best fit spatiotemporal model is shown. Error bars show standard error across 10 simulations of the EC model. (e-g) The Homogeneous variant of the model was simulated for different values of parameters SA-<sup>gen</sup> and TA-<span class="mathjax-tex">\({\Delta }_{1}^{{{{\rm{gen}}}}}\)</span>. For each timeseries metric (e), weighted graph metric (f), and unweighted graph metric (g), the metric value is plotted as a heatmap. (h) The economical clustering (EC) model was simulated for different values of its distance and cluster parameters. The value of each of the graph metrics is plotted as a heatmap. For all panels, N=883 subjects.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 sa and ta under serotonergic" href="/articles/s41593-023-01299-3/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_Fig15_ESM.jpg">Extended Data Fig. 10 SA and TA under serotonergic modulation.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(a-c,e) Analysis of scans after administration of LSD (top) or psilocybin (bottom). (a) Correlation of graph metrics across all subjects, with Bonferroni FWER corrected two-sided <i>P</i>-values, * indicates <i>P</i>&lt;.05, ** indicates <i>P</i>&lt;.01. Compare to Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1b</a>. (b) Correlation across subjects between graph metrics and SA-, SA-<span class="mathjax-tex">\(\infty\)</span>, or global TA-<sub>1</sub>. SA-+SA-<span class="mathjax-tex">\(\infty\)</span> and all columns indicate a linear model with leave-one-out cross-validation and Bonferroni FWER corrected two-sided <i>P</i>-values, * indicates <i>P</i>&lt;.05, ** indicates <i>P</i>&lt;.01. Compare to Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1e</a>. (c) The brain map depicting regional TA-<sub>1</sub>, averaged across all subjects. (d) Correlation of regional TA-<sub>1</sub> in HCP with those under LSD (top) or psilocybin (bottom). Spearman correlation&gt;.85, two-sided significance <i>P</i>&lt;.0001 of the correlation of HCP and LSD or psilocybin regional TA-<sub>1</sub> determined with SA-preserving scrambles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Burt, J. B., Helmer, M., Shinn, M., Anticevic, A. &amp; Murray, J. D. Generative modeling of brain maps with spatial autocorrelation. Neuroimage 220, 117038 (2020)." href="/articles/s41593-023-01299-3#ref-CR11" id="ref-link-section-d30110227e11998">11</a></sup>. Gray points indicate the placebo condition. (e) Correlation across regions of regional TA-<sub>1</sub> with nodal graph metrics for each subject. Boxplots show the median, first/third quartiles, and outlier-excluded minimum and maximum of the distribution Compare to Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41593-023-01299-3#Fig1">1j</a>. (f-j) Difference between drug and control across subjects for LSD (blue), psilocybin (pink), and LSD with ketanserin (green), for both early and late scans. Metrics plotted are TA as quantified by TA-<sub>1</sub> (f), SA as quantified by SA- and SA-<span class="mathjax-tex">\(\infty\)</span> (g), the residual of a regression model using motion to predict TA-<sub>1</sub>, subject motion as quantified by mean framewise displacement (i), and graph metrics (j). * indicates <i>P</i>&lt;.05, ** indicates <i>P</i>&lt;.01, two-sided Wilcoxon sign-rank test. Boxplots show the median, first/third quartiles, and outlier-excluded minimum and maximum of the distribution. For all panels, LSD: N=24, Psilocybin: N=23 subjects.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec62-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec62">Supplementary information</h2><div class="c-article-section__content" id="Sec62-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Notes 1 and 2 and Supplementary Table 1.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41593-023-01299-3/MediaObjects/41593_2023_1299_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content"><p>Springer Nature or its licensor (e.g. a society or other partner) holds exclusive rights to this article under a publishing agreement with the author(s) or other rightsholder(s); author self-archiving of the accepted manuscript version of this article is solely governed by the terms of such publishing agreement and applicable law.</p><p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Functional%20brain%20networks%20reflect%20spatial%20and%20temporal%20autocorrelation&amp;author=Maxwell%20Shinn%20et%20al&amp;contentID=10.1038%2Fs41593-023-01299-3&amp;copyright=The%20Author%28s%29%2C%20under%20exclusive%20licence%20to%20Springer%20Nature%20America%2C%20Inc.&amp;publication=1097-6256&amp;publicationDate=2023-04-24&amp;publisherName=SpringerNature&amp;orderBeanReset=true">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41593-023-01299-3" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41593-023-01299-3" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Shinn, M., Hu, A., Turner, L. <i>et al.</i> Functional brain networks reflect spatial and temporal autocorrelation.
                    <i>Nat Neurosci</i> <b>26</b>, 867878 (2023). https://doi.org/10.1038/s41593-023-01299-3</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41593-023-01299-3?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-04-27">27 April 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2023-03-14">14 March 2023</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2023-04-24">24 April 2023</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2023-05">May 2023</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41593-023-01299-3</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-none" hidden=""><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list"></div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article"
                                       data-track-label="link:Complex topology meets simple statistics" href="https://doi.org/10.1038/s41593-023-01295-7">
                                        Complex topology meets simple statistics
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="c-author-list c-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Shiyu Wang</li><li>Catie Chang</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Neuroscience</i> (2023)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

            
        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop">
        

        
            <noscript>
                
<div class="c-nature-box c-nature-box--side " data-component="entitlement-box">
    
        
        <p class="c-nature-box__text js-text">You have full access to this article via your institution.</p>
        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-023-01299-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </noscript>
            <div class="c-nature-box__wrapper c-nature-box__wrapper--placeholder">
                <div class="c-nature-box c-nature-box--side u-display-none u-hide-print" aria-hidden="true" data-component="entitlement-box"
    id=entitlement-box-right-column
    
    >

    
        <p class="c-nature-box__text js-text u-display-none" aria-hidden="true"></p>
        
        
            
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41593-023-01299-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="click" data-track-action="download pdf" data-track-label="link" data-track-external download>
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"/></svg>
            </a>
        </div>
    

        
    
</div>

            </div>
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41593-023-01295-7"
                       class="c-card__link u-link-inherit"
                       data-track="click"
                       data-track-action="view article"
                       data-track-category="associated content"
                       
                       data-track-label="news_and_views">Complex topology meets simple statistics</a>
                </h3>
                
<ul data-test="author-list" class="c-author-list c-author-list--compact">
    <li>Shiyu Wang</li><li>Catie Chang</li>
</ul>

                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Neuroscience</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">News &amp; Views</span></span>
        
        
            <time class="c-meta__item" datetime="2023-04-24">24 Apr 2023</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "news_and_views";
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky">
            <div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections">
                <div class="u-lazy-ad-wrapper u-mt-16 u-hide" data-component-mpu>
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2"
         class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide"
         data-ad-type="right"
         data-test="right-ad"
         data-pa11y-ignore
         data-gpt
         data-gpt-unitpath="/285/neurosci.nature.com/article"
         data-gpt-sizes="300x250"
         data-gpt-targeting="type=article;pos=right;artid=s41593-023-01299-3;doi=10.1038/s41593-023-01299-3;subjmeta=116,1627,1647,245,2649,378,631;kwrd=Cognitive+neuroscience,Computational+neuroscience,Functional+magnetic+resonance+imaging">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=136763497&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-023-01299-3%26doi%3D10.1038/s41593-023-01299-3%26subjmeta%3D116,1627,1647,245,2649,378,631%26kwrd%3DCognitive+neuroscience,Computational+neuroscience,Functional+magnetic+resonance+imaging">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/neurosci.nature.com/article&amp;sz=300x250&amp;c=136763497&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41593-023-01299-3%26doi%3D10.1038/s41593-023-01299-3%26subjmeta%3D116,1627,1647,245,2649,378,631%26kwrd%3DCognitive+neuroscience,Computational+neuroscience,Functional+magnetic+resonance+imaging"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures"></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references"></div>
        </div>
    </div>
</aside>
</div>


    
        <nav class="c-header__dropdown" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-articles"
                                   data-track="click"
                                   data-track-action="research articles"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-and-analysis"
                                   data-track="click"
                                   data-track-action="reviews &amp; analysis"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/news-and-comment"
                                   data-track="click"
                                   data-track-action="news &amp; comment"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/video"
                                   data-track="click"
                                   data-track-action="videos"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Videos
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/current-issue"
                                   data-track="click"
                                   data-track-action="current issue"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/collections"
                                   data-track="click"
                                   data-track-action="collections"
                                   data-track-label="link"
                                   data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link"
                               href="https://twitter.com/natureneuro"
                               data-track="click"
                               data-track-action="twitter"
                               data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id&#x3D;6"
                               rel="nofollow"
                               data-track="click"
                               data-track-action="Sign up for alerts"
                               data-track-external
                               data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"/></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link"
                               href="https://www.nature.com/neuro.rss"
                               data-track="click"
                               data-track-action="rss feed"
                               data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
    
    
        
            <nav class="c-header__dropdown" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/aims"
                                   data-track="click"
                                   data-track-action="aims &amp; scope"
                                   data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-information"
                                   data-track="click"
                                   data-track-action="journal information"
                                   data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/journal-impact"
                                   data-track="click"
                                   data-track-action="journal metrics"
                                   data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editors"
                                   data-track="click"
                                   data-track-action="about the editors"
                                   data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/our-publishing-models"
                                   data-track="click"
                                   data-track-action="our publishing models"
                                   data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-values-statement"
                                   data-track="click"
                                   data-track-action="editorial values statement"
                                   data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/editorial-policies"
                                   data-track="click"
                                   data-track-action="editorial policies"
                                   data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/content"
                                   data-track="click"
                                   data-track-action="content types"
                                   data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/web-feeds"
                                   data-track="click"
                                   data-track-action="web feeds"
                                   data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/posters"
                                   data-track="click"
                                   data-track-action="posters"
                                   data-track-label="link">
                                    Posters
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/contact"
                                   data-track="click"
                                   data-track-action="contact"
                                   data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/research-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="research cross-journal editorial team"
                                   data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/reviews-cross-journal-editorial-team"
                                   data-track="click"
                                   data-track-action="reviews cross-journal editorial team"
                                   data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        

        
            <nav class="c-header__dropdown" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/submission-guidelines"
                                   data-track="click"
                                   data-track-action="submission guidelines"
                                   data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link"
                                   href="/neuro/for-reviewers"
                                   data-track="click"
                                   data-track-action="for reviewers"
                                   data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services"
                                   data-track="click"
                                   data-track-action="manuscript author services"
                                   data-track-label="link manuscript author services"
                                   href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&utm_medium=Website_Nature&utm_campaign=Platform+Experimentation+2022&utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link"
                                   href="https://mts-nn.nature.com/cgi-bin/main.plex?form_type&#x3D;home&amp;from_idp&#x3D;1"
                                   data-track="click"
                                   data-track-action="submit manuscript"
                                   data-track-label="link (publish with us dropdown menu)"
                                   data-track-external>Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"/></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
        
    


<div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width" data-track-component="nature-150-split-header">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected>All journals</option>
                                    <option value="neuro">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced"
               data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>

<footer class="composite-layer" itemscope itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <p class="c-meta u-ma-0 u-flex-shrink">
                <span class="c-meta__item">
                    Nature Neuroscience (<i>Nat Neurosci</i>)
                </span>
                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="onlineIssn">1546-1726</span> (online)
    </span>
    


                
    
    <span class="c-meta__item">
        <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="printIssn">1097-6256</span> (print)
    </span>
    

            </p>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/company_info/index.html"
                                                  data-track="click" data-track-action="about us"
                                                  data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/npg_/press_room/press_releases.html"
                                                  data-track="click" data-track-action="press releases"
                                                  data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/"
                                                  data-track="click" data-track-action="press office"
                                                  data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home"
                                                  data-track="click" data-track-action="contact us"
                                                  data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex"
                                                  data-track="click" data-track-action="journals a-z"
                                                  data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects"
                                                  data-track="click" data-track-action="article by subject"
                                                  data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/protocolexchange/"
                                                  data-track="click" data-track-action="protocol exchange"
                                                  data-track-label="link">Protocol Exchange</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/"
                                                  data-track="click" data-track-action="nature index"
                                                  data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/authors/editorial_policies"
                                                  data-track="click" data-track-action="Nature portfolio policies"
                                                  data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/nature-research/open-access"
                                                  data-track="click" data-track-action="open access"
                                                  data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints"
                                                  data-track="click" data-track-action="reprints and permissions"
                                                  data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/authors/research-data"
                                                  data-track="click" data-track-action="data research service"
                                                  data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/language-editing/"
                                                  data-track="click" data-track-action="language editing"
                                                  data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://authorservices.springernature.com/scientific-editing/"
                                                  data-track="click" data-track-action="scientific editing"
                                                  data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/"
                                                  data-track="click" data-track-action="nature masterclasses"
                                                  data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/"
                                                  data-track="click" data-track-action="research solutions"
                                                  data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/tools-services"
                                                  data-track="click" data-track-action="librarian service and tools"
                                                  data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal"
                                                  data-track="click" data-track-action="librarian portal"
                                                  data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.nature.com/openresearch/about-open-access/information-for-institutions"
                                                  data-track="click" data-track-action="open research"
                                                  data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://www.springernature.com/gp/librarians/recommend-to-your-library"
                                                  data-track="click" data-track-action="Recommend to library"
                                                  data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/digital-advertising/"
                                                  data-track="click" data-track-action="advertising"
                                                  data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/"
                                                  data-track="click" data-track-action="partnerships and services"
                                                  data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/media-kits/" data-track="click"
                                                  data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link"
                                                  href="https://partnerships.nature.com/product/branded-content-native-advertising/"
                                                  data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/"
                                                  data-track="click" data-track-action="nature careers"
                                                  data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com"
                                                  data-track="click" data-track-action="nature conferences"
                                                  data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica"
                                                  data-track="click" data-track-action="nature africa"
                                                  data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com"
                                                  data-track="click" data-track-action="nature china"
                                                  data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia"
                                                  data-track="click" data-track-action="nature india"
                                                  data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly"
                                                  data-track="click" data-track-action="nature Italy"
                                                  data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp"
                                                  data-track="click" data-track-action="nature japan"
                                                  data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ko-kr"
                                                  data-track="click" data-track-action="nature korea"
                                                  data-track-label="link">Nature Korea</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast"
                                                  data-track="click" data-track-action="nature middle east"
                                                  data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy"
                                          data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies"
                                          data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;"
                        data-cc-action="preferences" data-track="click" data-track-action="manage cookies"
                        data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice"
                                          data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link"
                                          href="https://www.nature.com/info/accessibility-statement" data-track="click"
                                          data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions"
                                          data-track="click" data-track-action="terms and conditions"
                                          data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa"
                                          data-track="click" data-track-action="california privacy statement"
                                          data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20"/>
    </a>
    <p class="c-footer__legal" data-test="copyright">&copy; 2024 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <?xml version="1.0" encoding="UTF-8"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"/></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"/></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"/></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"/></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"/></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"/></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"/></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"/></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"/></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"/></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"/></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"/></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"/></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"/></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"/></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"/></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"/></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"/></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"/></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"/></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"/></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"/></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"/></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"/></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"/></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"/></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"/></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"/></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"/></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"/></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"/></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"/></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"/></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"/></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"/></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"/></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"/></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"/></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"/></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"/></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"/></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"/><clipPath id="b"><use xlink:href="#a" style="overflow:visible"/></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"/><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"/></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"/></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"/></symbol><symbol id="icon-eds-i-book-series-medium" viewBox="0 0 24 24"><path id="shape" fill-rule="evenodd" clip-rule="evenodd" d="M1 3.78571C1 2.75867 1.85698 2 2.8209 2H6.1791C7.14302 2 8 2.75867 8 3.78571V4H11.1668C11.885 4 12.5585 4.42017 12.8494 5.07033C12.9893 4.98169 13.1425 4.91101 13.3056 4.86206L16.5222 3.89704C17.4454 3.62005 18.4843 4.10046 18.7794 5.08419L22.9256 18.9042C23.2207 19.8878 22.618 20.8608 21.6947 21.1378L18.4781 22.1029C17.5548 22.3799 16.516 21.8993 16.2209 20.9157L13.0001 10.1804V20.2143C13.0001 21.255 12.1231 22 11.1668 22H7.83346C7.54206 22 7.25803 21.9308 7.00392 21.8052C6.75263 21.9305 6.47077 22 6.1791 22H2.8209C1.85693 22 1 21.2412 1 20.2143V3.78571ZM3 4V15H6V4H3ZM3 20V17H6V20H3ZM18.0749 20.1358L17.2129 17.2623L20.0863 16.4002L20.9484 19.2737L18.0749 20.1358ZM19.5116 14.4846L16.6381 15.3466L14.0519 6.72624L16.9254 5.86416L19.5116 14.4846ZM8.00012 20L8.00012 6H11.0001L11.0001 20H8.00012Z"/></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"/></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"/></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"/></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"/></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"/></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"/></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"/></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"/></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"/></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"/></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"/></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"/></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"/></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif "
data-component-id="nature-briefing-banner"
data-component-expirydays="30"
data-component-trigger-scroll-percentage="15"
data-track="in-view"
data-track-action="in-view"
data-track-category="nature briefing"
data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter  what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="submit||nature_briefing_sign_up" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick">
                        <input type="hidden" value="false" name="marketing" id="marketing">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required>
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close"
        data-track="click"
        data-track-category="nature briefing"
        data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif"
                data-track="click"
                data-track-category="nature briefing"
                data-track-label="Small-screen banner CTA to site"
                data-test-element="briefing-banner-link"
                target="_blank"
                rel="noreferrer noopener"
                href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41593-023-01299-3&amp;format=js&amp;last_modified=2023-04-24" async></script>
<img src="/y0a38gii/article/s41593-023-01299-3" width="1" height="1" alt="" class="u-visually-hidden">
</body>
</html>